<pre>Query         10K_MODEL_53 
</pre><pre>Match_columns 116
</pre><pre>No_of_seqs    58 out of 60
</pre><pre>Neff          3.13415
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 20:20:28 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/7795569.hhr -oa3m ../results/7795569.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 1YD6_D UvrC; DNA BINDING PROTE  92.7    0.66 6.1E-06   29.8   4.2   35    1-36     23-66  (99)
</pre><pre>  2 PF01541.26 ; GIY-YIG ; GIY-YIG  92.3    0.37 3.4E-06   28.7   2.5   25    1-25      9-33  (89)
</pre><pre>  3 1YWL_A Hypothetical UPF0213 pr  87.0     1.3 1.2E-05   29.3   2.2   42    7-49     19-69  (96)
</pre><pre>  4 2LWF_A Monothiol glutaredoxin-  86.5     2.1 1.9E-05   31.7   3.1   24    1-25     33-56  (119)
</pre><pre>  5 1YD0_A UvrABC system protein C  85.3     2.9 2.7E-05   26.9   3.0   24    3-27     24-47  (96)
</pre><pre>  6 1MK0_A Intron-associated endon  80.6     5.3 4.9E-05   25.1   2.8   22    1-23     10-31  (97)
</pre><pre>  7 2WSH_D ENDONUCLEASE II; GIY-YI  73.6     8.6   8E-05   28.4   2.7   22    3-25     49-70  (143)
</pre><pre>  8 1ZG2_A Hypothetical UPF0213 pr  69.4      12 0.00012   25.5   2.6   42    7-49     20-68  (107)
</pre><pre>  9 d1mk0a_ d.226.1.1 (A:) Homing   59.4      29 0.00027   23.2   2.8   22    1-23     10-31  (97)
</pre><pre> 10 4XLG_A Structure-specific endo  56.2      22  0.0002   30.0   2.2   42    7-49     25-79  (312)
</pre><pre> 11 7CQ3_A Structure-specific endo  55.8      24 0.00022   29.7   2.4   42    7-49     28-82  (304)
</pre><pre> 12 3OQG_A Hpy188I; ENDONUCLEASE-D  49.6      39 0.00037   27.8   2.7   21    3-23     78-98  (180)
</pre><pre> 13 6SEI_A Structure-specific endo  46.3      40 0.00037   28.9   2.3   42    7-49     26-79  (324)
</pre><pre> 14 PF09517.12 ; RE_Eco29kI ; Eco2  40.6      76  0.0007   25.7   2.9   25    1-25     28-69  (165)
</pre><pre> 15 TIGR01453 grpIintron_endo; gro  39.5      80 0.00075   24.6   2.8   22    1-23      9-30  (214)
</pre><pre> 16 PF13455.8 ; MUG113 ; Meiotical  37.1      66 0.00062   20.0   1.8   39    9-49      3-42  (78)
</pre><pre> 17 PF13066.8 ; DUF3929 ; Protein   37.1      88 0.00082   23.2   2.5   27   90-116    27-53  (64)
</pre><pre> 18 TIGR00194 uvrC; excinuclease A  30.1 1.5E+02  0.0014   28.0   3.5   27    1-28     19-45  (574)
</pre><pre> 19 PF13754.8 ; Big_3_4 ; Domain o  29.5      81 0.00075   24.8   1.5   46   59-105    10-55  (105)
</pre><pre> 20 PF20029.1 ; DUF6436 ; Domain o  27.8      86  0.0008   20.2   1.2   10    1-10     10-19  (59)
</pre><pre> 21 PF19239.1 ; GIY_YIG_domain ; G  26.8 1.6E+02  0.0015   23.3   2.7   30    2-32     61-90  (162)
</pre><pre> 22 PF10544.11 ; T5orf172 ; T5orf1  25.4 1.8E+02  0.0017   18.7   2.3   18    5-23     15-32  (82)
</pre><pre> 23 3MX1_A Eco29kIR; type II restr  23.5 1.6E+02  0.0015   25.4   2.4   21    5-25     94-131 (235)
</pre><pre> 24 d2gw6a1 c.52.2.1 (A:2-123) tRN  23.3 1.5E+02  0.0014   22.1   1.9   13    1-13    110-122 (122)
</pre><pre> 25 2GW6_A tRNA-splicing endonucle  22.1 1.5E+02  0.0014   22.1   1.7   13    1-13    111-123 (123)
</pre><pre> 26 2K3U_A Chemotaxis inhibitory p  21.0 2.9E+02  0.0026   21.9   2.9   44   15-88     10-57  (91)
</pre><pre> 27 PF11434.10 ; CHIPS ; Chemotaxi  21.0 2.9E+02  0.0026   21.9   2.9   44   15-88     10-57  (91)
</pre><pre> 28 d3f0pa1 a.4.5.79 (A:1-80) Alky  21.0 1.9E+02  0.0018   21.9   2.1   28   88-115    48-75  (80)
</pre><pre>
</pre><pre>No 1
</pre><pre>>1YD6_D UvrC; DNA BINDING PROTEIN; HET: SO4; 2.0A {Bacillus caldotenax}
</pre><pre>Probab=92.69  E-value=0.66  Score=29.83  Aligned_cols=35  Identities=26%  Similarity=0.439  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHhccCC---------CccceeEEe
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEHAGDG---------RVFSKVRYV   36 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH~~~~---------k~F~~v~yv   36 (116)
</pre><pre>                      +..++++|.|.|.. +..|+.+|.+..         ..+..+.++
</pre><pre>T Consensus        23 ~~~~~~~YiG~t~~-l~~r~~~h~~~~~~~~~~~~~~~~~~i~~~   66 (99)
</pre><pre>T 1YD6_D           23 DKHGTVIYVGKAKS-LKERVRSYFTGTHDGKTQRLVEEIADFEYI   66 (99)
</pre><pre>T ss_dssp             CTTSCEEEEEEESS-HHHHHHHTTSSCCCHHHHHHHHHCSEEEEE
</pre><pre>T ss_pred             CCCCCEEEEEecCC-HHHHHHHhhccCCCccHHHHHhcceEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF01541.26 ; GIY-YIG ; GIY-YIG catalytic domain
</pre><pre>Probab=92.27  E-value=0.37  Score=28.70  Aligned_cols=25  Identities=44%  Similarity=0.615  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHhcc
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEHAG   25 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH~~   25 (116)
</pre><pre>                      +..+..+|.|.|.+.+..|+.+|..
</pre><pre>T Consensus         9 ~~~~~~~yvG~t~~~l~~r~~~h~~   33 (89)
</pre><pre>T VF019_IIV6/3-9    9 NNLNDEVYIGSTLRTIEERFSEHKA   33 (89)
</pre><pre>T ss_pred             eCCCCcEEEEeeCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>1YWL_A Hypothetical UPF0213 protein EF2693; Alpha and beta, Structural Genomics, Northeast Structural Genomics Consortium (NESG), Protein Structure Initiative (PSI), UNKNOWN FUNCTION; NMR {Enterococcus faecalis}
</pre><pre>Probab=87.04  E-value=1.3  Score=29.32  Aligned_cols=42  Identities=24%  Similarity=0.313  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEeeccchHHHHHHhcc-------C--CCccceeEEecccccceeeeccc
</pre><pre>Q 10K_MODEL_53      7 IYHGITDRTVQERVIEHAG-------D--GRVFSKVRYVDDLGSRVNARNIE   49 (116)
</pre><pre>Q Consensus         7 VY~gITdr~vq~R~~eH~~-------~--~k~F~~v~yvd~l~~R~~aRnlE   49 (116)
</pre><pre>                      +|.|+|. .+..|+.+|..       .  ....-.+.|+....++.+|+..|
</pre><pre>T Consensus        19 ~YiG~t~-~l~~R~~~H~~~~~~~~~~~~~~~~~~lv~~~~~~~~~~A~~~E   69 (96)
</pre><pre>T 1YWL_A           19 FYGGYTT-EPERRLTEHNSGTGAKYTRLAKRRPVIMIHTEKFETRSEATKAE   69 (96)
</pre><pre>T ss_dssp             CEEEEES-CHHHHHHHHHHHHSCSSCSSCCSSCCEEEEEEEESSHHHHHHHH
</pre><pre>T ss_pred             EEEeecC-CHHHHHHHHHCCCCCchhccccCCCEEEEEEEEeCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>2LWF_A Monothiol glutaredoxin-S16, chloroplastic; Nuclease, Glutaredoxin, HYDROLASE; NMR {Arabidopsis thaliana}
</pre><pre>Probab=86.48  E-value=2.1  Score=31.66  Aligned_cols=24  Identities=21%  Similarity=0.423  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHhcc
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEHAG   25 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH~~   25 (116)
</pre><pre>                      |.+|.++|.|.+.. ++.|+.+|.+
</pre><pre>T Consensus        33 d~~~~vlYVG~s~n-l~~Rl~~H~~   56 (119)
</pre><pre>T 2LWF_A           33 DKSDELQFVGISRN-IAASVSAHLK   56 (119)
</pre><pre>T ss_dssp             CTTCCEEEEEEESC-HHHHHHHHHH
</pre><pre>T ss_pred             CCCCCEEEEEEECC-HHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>1YD0_A UvrABC system protein C; DNA BINDING PROTEIN; 1.5A {Thermotoga maritima}
</pre><pre>Probab=85.26  E-value=2.9  Score=26.88  Aligned_cols=24  Identities=21%  Similarity=0.272  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEEeeccchHHHHHHhccCC
</pre><pre>Q 10K_MODEL_53      3 SGDVIYHGITDRTVQERVIEHAGDG   27 (116)
</pre><pre>Q Consensus         3 ~G~vVY~gITdr~vq~R~~eH~~~~   27 (116)
</pre><pre>                      .++++|.|.|+ .+.+|+.+|.+..
</pre><pre>T Consensus        24 ~~~~lYvG~t~-~l~~ri~~~~~~~   47 (96)
</pre><pre>T 1YD0_A           24 KGVPIYIGKAK-RLSNRLRSYLNPQ   47 (96)
</pre><pre>T ss_dssp             TTEEEEEEEES-SHHHHHHGGGSCS
</pre><pre>T ss_pred             CCeEEEEEEcC-CHHHHHHHhcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>1MK0_A Intron-associated endonuclease 1; alpha/beta fold  CATALYTIC DOMAIN  DNA-BINDING SURFACE, HYDROLASE; HET: CIT; 1.6A {Enterobacteria phage T4} SCOP: d.226.1.1
</pre><pre>Probab=80.64  E-value=5.3  Score=25.12  Aligned_cols=22  Identities=23%  Similarity=0.463  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHh
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEH   23 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH   23 (116)
</pre><pre>                      ++....+|.|.| +.++.|+.+|
</pre><pre>T Consensus        10 ~~~~~~~YvG~t-~~~~~r~~~h   31 (97)
</pre><pre>T 1MK0_A           10 NTLNNKVYVGSA-KDFEKRWKRH   31 (97)
</pre><pre>T ss_dssp             ETTTCCEEEEEE-SSHHHHHHHH
</pre><pre>T ss_pred             ECCCCeEEEEEe-CCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>2WSH_D ENDONUCLEASE II; GIY-YIG, NUCLEASE, HYDROLASE; HET: MSE, PEG; 1.9A {ENTEROBACTERIA PHAGE T4}
</pre><pre>Probab=73.60  E-value=8.6  Score=28.38  Aligned_cols=22  Identities=18%  Similarity=0.415  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEEeeccchHHHHHHhcc
</pre><pre>Q 10K_MODEL_53      3 SGDVIYHGITDRTVQERVIEHAG   25 (116)
</pre><pre>Q Consensus         3 ~G~vVY~gITdr~vq~R~~eH~~   25 (116)
</pre><pre>                      +|+++|.|-| ..+..||.+|..
</pre><pre>T Consensus        49 ~~~ilYIG~a-~~L~~Rl~~~~~   70 (143)
</pre><pre>T 2WSH_D           49 NDELVYIGKT-KNLRKRINYYRT   70 (143)
</pre><pre>T ss_dssp             TTEEEEEEEE-EEHHHHHHHHHH
</pre><pre>T ss_pred             CCEEEEEEEc-CCHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>1ZG2_A Hypothetical UPF0213 protein BH0048; BhR2, Autostructure, Northeast Structural Genomics Consortium, PSI, Protein Structure Initiative, NESG, STRUCTURAL GENOMICS, UNKNOWN FUNCTION; NMR {Bacillus halodurans}
</pre><pre>Probab=69.39  E-value=12  Score=25.49  Aligned_cols=42  Identities=24%  Similarity=0.195  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEeeccchHHHHHHhccCCC-------ccceeEEecccccceeeeccc
</pre><pre>Q 10K_MODEL_53      7 IYHGITDRTVQERVIEHAGDGR-------VFSKVRYVDDLGSRVNARNIE   49 (116)
</pre><pre>Q Consensus         7 VY~gITdr~vq~R~~eH~~~~k-------~F~~v~yvd~l~~R~~aRnlE   49 (116)
</pre><pre>                      +|.|+| ..+..|+.+|.....       ..-.+.|++...++.+|+..|
</pre><pre>T Consensus        20 ~YiG~t-~~l~~rl~~h~~~~~~~~~~~~~~~~lv~~~~~~~~~~A~~~E   68 (107)
</pre><pre>T 1ZG2_A           20 WYTGYT-TDVDRRIKKHASGKGAKYTRGRGPFRLVATWAFPSKEEAMRWE   68 (107)
</pre><pre>T ss_dssp             EEEEEE-CCHHHHHHHHHHHTTCCSCCCCSSCEEEEEEEESCHHHHHHHH
</pre><pre>T ss_pred             EEEEEe-CCHHHHHHHHHCCCCcccccCCCCeEEEEEEECCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>d1mk0a_ d.226.1.1 (A:) Homing endonuclease I-TevI {Bacteriophage T4 [TaxId: 10665]}
</pre><pre>Probab=59.40  E-value=29  Score=23.18  Aligned_cols=22  Identities=23%  Similarity=0.430  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHh
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEH   23 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH   23 (116)
</pre><pre>                      +.....+|.|-|. ++..|+.+|
</pre><pre>T Consensus        10 ~~~~~~~YiG~t~-~l~~R~~~h   31 (97)
</pre><pre>T d1mk0a_          10 NTLNNKVYVGSAK-DFEKRWKRH   31 (97)
</pre><pre>T ss_dssp             ETTTCCEEEEEES-SHHHHHHHH
</pre><pre>T ss_pred             ECCCCeEEEEEeC-CHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>4XLG_A Structure-specific endonuclease subunit SLX1; nuclease, DNA repair, GIY-YIG, homogolous recombination, hydrolase; 1.78A {Candida glabrata (strain ATCC 2001 / CBS 138 / JCM 3761 / NBRC 0622 / NRRL Y-65)}
</pre><pre>Probab=56.16  E-value=22  Score=30.03  Aligned_cols=42  Identities=24%  Similarity=0.314  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEeeccchHHHHHHhccC------------CCccceeEEe-cccccceeeeccc
</pre><pre>Q 10K_MODEL_53      7 IYHGITDRTVQERVIEHAGD------------GRVFSKVRYV-DDLGSRVNARNIE   49 (116)
</pre><pre>Q Consensus         7 VY~gITdr~vq~R~~eH~~~------------~k~F~~v~yv-d~l~~R~~aRnlE   49 (116)
</pre><pre>                      +|.|+|. .+..|+.+|...            ...--.+.|+ ....++.+|+..|
</pre><pre>T Consensus        25 ~YiG~T~-~~~~Rl~qH~~g~~~~ga~~T~~~~~~p~~lv~~v~~f~~~~~Al~~E   79 (312)
</pre><pre>T 4XLG_A           25 FYIGSTP-NPVRRLRQHNGSLSRGGAYRTKRDGTRPWEMVAIVYGFPSRIAALQFQ   79 (312)
</pre><pre>T ss_dssp             EEEEEES-CHHHHHHHHHSCSCC-----CCCCSCCSEEEEEEEECCSSHHHHHHHH
</pre><pre>T ss_pred             eEEEEcC-CHHHHHHHHhcCCCCCcchhhcccCCCCcEEEEEEEeCchHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>7CQ3_A Structure-specific endonuclease subunit SLX1; endonuclease complex, Holliday Junction, SLX1-SLX4, HYDROLASE; HET: SO4; 1.449A {Saccharomyces cerevisiae (strain YJM789)}
</pre><pre>Probab=55.80  E-value=24  Score=29.74  Aligned_cols=42  Identities=26%  Similarity=0.381  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEeeccchHHHHHHhccCC-------------CccceeEEecccccceeeeccc
</pre><pre>Q 10K_MODEL_53      7 IYHGITDRTVQERVIEHAGDG-------------RVFSKVRYVDDLGSRVNARNIE   49 (116)
</pre><pre>Q Consensus         7 VY~gITdr~vq~R~~eH~~~~-------------k~F~~v~yvd~l~~R~~aRnlE   49 (116)
</pre><pre>                      +|.|+| .++..|+.+|....             +.+.-+-++....++.+|...|
</pre><pre>T Consensus        28 ~YiG~T-~d~~rRl~qHn~~~~~~ga~~t~~~~~~p~~~v~~~~~f~~~~~Al~~E   82 (304)
</pre><pre>T 7CQ3_A           28 FYVGST-PNPVRRLRQHNGKLAVGGAYRTKRDGSRPWEMIMIVRGFPSKIAALQFE   82 (304)
</pre><pre>T ss_pred             EEEEEc-CCHHHHHHHHHccCCCCCccccccCCCCCcEEEEEEecCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>3OQG_A Hpy188I; ENDONUCLEASE-DNA COMPLEX, RESTRICTION ENZYME, HPY188I, INTERCALATION, GIY-YIG NUCLEASE, CATALYTIC MECHANISM, PSEUDOPALINDROME, HYDROLASE-DNA COMPLEX; HET: MSE; 1.75A {Helicobacter pylori}
</pre><pre>Probab=49.64  E-value=39  Score=27.84  Aligned_cols=21  Identities=19%  Similarity=0.281  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEEeeccchHHHHHHh
</pre><pre>Q 10K_MODEL_53      3 SGDVIYHGITDRTVQERVIEH   23 (116)
</pre><pre>Q Consensus         3 ~G~vVY~gITdr~vq~R~~eH   23 (116)
</pre><pre>                      +|+|+|.|-|.+....||..+
</pre><pre>T Consensus        78 d~~I~YIG~t~~~Lk~R~~~Y   98 (180)
</pre><pre>T 3OQG_A           78 QGKIFKIGHSITPITKRVQSY   98 (180)
</pre><pre>T ss_dssp             TTEEEEEEEESSCHHHHHHHH
</pre><pre>T ss_pred             CCEEEEEEEcCCCHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>6SEI_A Structure-specific endonuclease subunit SLX1; Resolvase, Structure-Selective endonuclease, Endonuclease, DNA BINDING PROTEIN; 2.69A {Thielavia terrestris}
</pre><pre>Probab=46.31  E-value=40  Score=28.90  Aligned_cols=42  Identities=29%  Similarity=0.384  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEeeccchHHHHHHhcc------------CCCccceeEEecccccceeeeccc
</pre><pre>Q 10K_MODEL_53      7 IYHGITDRTVQERVIEHAG------------DGRVFSKVRYVDDLGSRVNARNIE   49 (116)
</pre><pre>Q Consensus         7 VY~gITdr~vq~R~~eH~~------------~~k~F~~v~yvd~l~~R~~aRnlE   49 (116)
</pre><pre>                      .|.|+|. ++..|+.+|-.            ....+.-+-|+....++.+|...|
</pre><pre>T Consensus        26 ~YiG~T~-d~~~Rl~qHn~~~~~ga~~t~~~~~~p~~lv~~~~~f~~~~~Al~fE   79 (324)
</pre><pre>T 6SEI_A           26 LYIGSTP-NPPRRLKQHNGLVPGGAARTSRSSLRPWEMVALVSGFPSMVAALKFQ   79 (324)
</pre><pre>T ss_dssp             EEEEEES-CHHHHHHHHTTSSSCCCSGGGCTTTCSEEEEEEEECCSCHHHHHHHH
</pre><pre>T ss_pred             EEEEEeC-CHHHHHHHhcCCCCCCCCccccccCCCeEEEEEEEcCcHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF09517.12 ; RE_Eco29kI ; Eco29kI restriction endonuclease
</pre><pre>Probab=40.59  E-value=76  Score=25.69  Aligned_cols=25  Identities=24%  Similarity=0.316  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeec-----------------cchHHHHHHhcc
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITD-----------------RTVQERVIEHAG   25 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITd-----------------r~vq~R~~eH~~   25 (116)
</pre><pre>                      +....+||+|=+.                 +.+..||.+|.+
</pre><pre>T Consensus        28 ~~~~~pIYVGKA~~~~~r~g~~~~~~~~~~~~L~~RL~~H~~   69 (165)
</pre><pre>T D4ZZ96_ARTPN/4   28 TAFRAPIYVGKEVPKGWRKAMQSFSVNSQGYELNNRIREHSR   69 (165)
</pre><pre>T ss_pred             cCCcceEEEEEccCCccccccccccccccCCCHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>TIGR01453 grpIintron_endo; group I intron endonuclease. This model represents one subfamily of endonucleases containing the endo/excinuclease amino terminal domain, pfam01541 at its amino end.
</pre><pre>Probab=39.52  E-value=80  Score=24.56  Aligned_cols=22  Identities=32%  Similarity=0.531  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHh
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEH   23 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH   23 (116)
</pre><pre>                      +.+...+|.|.| +.+..|+.+|
</pre><pre>T Consensus         9 ~~~~~~~YvG~t-~~~~~R~~~h   30 (214)
</pre><pre>T TIGR01453         9 NNINGKIYVGSS-VNLEKRLKEH   30 (214)
</pre><pre>T ss_pred             ECCCCeEEEEEe-CCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>PF13455.8 ; MUG113 ; Meiotically up-regulated gene 113
</pre><pre>Probab=37.14  E-value=66  Score=19.98  Aligned_cols=39  Identities=21%  Similarity=0.169  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             EEeec-cchHHHHHHhccCCCccceeEEecccccceeeeccc
</pre><pre>Q 10K_MODEL_53      9 HGITD-RTVQERVIEHAGDGRVFSKVRYVDDLGSRVNARNIE   49 (116)
</pre><pre>Q Consensus         9 ~gITd-r~vq~R~~eH~~~~k~F~~v~yvd~l~~R~~aRnlE   49 (116)
</pre><pre>                      .|+|. +.+..|+.++.... ....+.++...... .++.+|
</pre><pre>T Consensus         3 IG~T~~~~~~~R~~~~~~~~-~~~~~~~~~~~~~~-~~~~~E   42 (78)
</pre><pre>T A5CTV5_CLAM3/4    3 IGMTRRLEPMDRVVELGDAS-VPFRFDVHALFFAD-DAVGVE   42 (78)
</pre><pre>T ss_pred             eeecCCCCHHHHHHHhcCCC-CCCCeEEEEEEEcC-CHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>PF13066.8 ; DUF3929 ; Protein of unknown function (DUF3929)
</pre><pre>Probab=37.08  E-value=88  Score=23.16  Aligned_cols=27  Identities=26%  Similarity=0.533  Sum_probs=0.0  Template_Neff=1.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhhcceeeeecCCceeeecCC
</pre><pre>Q 10K_MODEL_53     90 QEEINKILKNGTVVDVDNKGHFKSGGC  116 (116)
</pre><pre>Q Consensus        90 ~~~id~~~~~at~~~v~~~Gki~~~~~  116 (116)
</pre><pre>                      ..+....|++...+-||++|.|++-+|
</pre><pre>T Consensus        27 ERV~h~Vmdn~~vTAID~~GtiIsIaC   53 (64)
</pre><pre>T Q818M6_BACCR/8   27 ERVANRVMDNREVTAIDKQGTIISIAC   53 (64)
</pre><pre>T ss_pred             HHHHHHHhcCCceeEecCCCcEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>TIGR00194 uvrC; excinuclease ABC, C subunit. This family consists of the DNA repair enzyme UvrC, an ABC excinuclease subunit which interacts with the UvrA/UvrB complex to excise UV-damaged nucleotide segments.
</pre><pre>Probab=30.15  E-value=1.5e+02  Score=27.99  Aligned_cols=27  Identities=22%  Similarity=0.463  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeeccchHHHHHHhccCCC
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITDRTVQERVIEHAGDGR   28 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITdr~vq~R~~eH~~~~k   28 (116)
</pre><pre>                      |.+|+++|.|... .++.|+..|....+
</pre><pre>T Consensus        19 ~~~~~vlYVGka~-nlr~Rv~~~~~~~~   45 (574)
</pre><pre>T TIGR00194        19 DRNGQVLYVGKAK-NLKKRVSSYFRENN   45 (574)
</pre><pre>T ss_pred             CCCCCEEEEEEeC-CHHHHHHHHhccCC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF13754.8 ; Big_3_4 ; Domain of unknown function
</pre><pre>Probab=29.50  E-value=81  Score=24.77  Aligned_cols=46  Identities=11%  Similarity=0.042  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             CcceeeEEEccCCeEEeccccchhcCCeEEeHHHHHHHhhcceeeee
</pre><pre>Q 10K_MODEL_53     59 DPNIENAIRKDGGFYHSYDANNLAEGRKYLSQEEINKILKNGTVVDV  105 (116)
</pre><pre>Q Consensus        59 n~~l~Na~r~d~G~Yhsy~p~~La~Gr~~ls~~~id~~~~~at~~~v  105 (116)
</pre><pre>                      |..-+.-.|.+++.|....|.++ +|.+.+.--..|++++.+-..++
</pre><pre>T Consensus        10 dg~eViF~~~~g~~W~~~VP~~~-dG~YvVEl~AeD~AGN~sy~a~~   55 (105)
</pre><pre>T R5U481_9FIRM/1   10 GSSKIIFDRTDGNIWKVTVPFLE-SGEYIVALYALDDAGNQAYVATI   55 (105)
</pre><pre>T ss_pred             CCEEEEEEeCCCCeEEEEeecCC-CeeEEEEEEEEechhhheeeEeE
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF20029.1 ; DUF6436 ; Domain of unknown function (DUF6436)
</pre><pre>Probab=27.76  E-value=86  Score=20.18  Aligned_cols=10  Identities=20%  Similarity=0.581  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEE
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHG   10 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~g   10 (116)
</pre><pre>                      |.+|+++|+|
</pre><pre>T Consensus        10 d~~~~l~Y~G   19 (59)
</pre><pre>T A0A0T6URZ6_9PS   10 DRNDQLAYLG   19 (59)
</pre><pre>T ss_pred             cCCCeEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>PF19239.1 ; GIY_YIG_domain ; GIY-YIG domain-containing protein
</pre><pre>Probab=26.75  E-value=1.6e+02  Score=23.31  Aligned_cols=30  Identities=23%  Similarity=0.300  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeeccchHHHHHHhccCCCccce
</pre><pre>Q 10K_MODEL_53      2 SSGDVIYHGITDRTVQERVIEHAGDGRVFSK   32 (116)
</pre><pre>Q Consensus         2 ~~G~vVY~gITdr~vq~R~~eH~~~~k~F~~   32 (116)
</pre><pre>                      +.+.+||.|-++. ++.|+.+|.+.+..+..
</pre><pre>T Consensus        61 ~~~~vvYIG~a~~-Lr~Rl~~y~~~~~~l~~   90 (162)
</pre><pre>T A0A0A0LWE7_CUC   61 DWIVVVYLGEADN-VRTRLQHYGRSGSHLGN   90 (162)
</pre><pre>T ss_pred             cceEEEEEEEccc-HHHHHHHhccccccccc
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>PF10544.11 ; T5orf172 ; T5orf172 domain
</pre><pre>Probab=25.39  E-value=1.8e+02  Score=18.74  Aligned_cols=18  Identities=17%  Similarity=0.008  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeeccchHHHHHHh
</pre><pre>Q 10K_MODEL_53      5 DVIYHGITDRTVQERVIEH   23 (116)
</pre><pre>Q Consensus         5 ~vVY~gITdr~vq~R~~eH   23 (116)
</pre><pre>                      .++-.|+|.+ +..|+.++
</pre><pre>T Consensus        15 ~~~KIG~T~~-~~~R~~~~   32 (82)
</pre><pre>T A2FFP4_TRIVA/1   15 KRYKIGRTYN-ITQRYDSI   32 (82)
</pre><pre>T ss_pred             ccEEeEecCC-HHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>3MX1_A Eco29kIR; type II restriction endonuclease, GIY-YIG endonuclease, HYDROLASE; 2.3A {Escherichia coli}
</pre><pre>Probab=23.51  E-value=1.6e+02  Score=25.45  Aligned_cols=21  Identities=29%  Similarity=0.360  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeecc-----------------chHHHHHHhcc
</pre><pre>Q 10K_MODEL_53      5 DVIYHGITDR-----------------TVQERVIEHAG   25 (116)
</pre><pre>Q Consensus         5 ~vVY~gITdr-----------------~vq~R~~eH~~   25 (116)
</pre><pre>                      .+||+|-++.                 .+..||.+|.+
</pre><pre>T Consensus        94 ~pIYVGKA~~~g~r~~~~~~~~~~~~~~Lr~RL~~H~r  131 (235)
</pre><pre>T 3MX1_A           94 LPIYVGKAVPAGWRQSRISDHETRAGSELSNRIREHGR  131 (235)
</pre><pre>T ss_dssp             CCSEEEEECCTTTSCSSCSSCSSCCCSHHHHHHHHHHH
</pre><pre>T ss_pred             CcEEEEeecCcccccccccccccccCChHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>d2gw6a1 c.52.2.1 (A:2-123) tRNA-splicing endonuclease subunit Sen15 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=23.32  E-value=1.5e+02  Score=22.11  Aligned_cols=13  Identities=31%  Similarity=0.634  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeec
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITD   13 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITd   13 (116)
</pre><pre>                      +++|.|||+-|++
</pre><pre>T Consensus       110 d~DstivYY~i~~  122 (122)
</pre><pre>T d2gw6a1         110 ESDSTIVYYKLTD  122 (122)
</pre><pre>T ss_dssp             CTTSCEEEEEEEC
</pre><pre>T ss_pred             cCCCcEEEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>2GW6_A tRNA-splicing endonuclease subunit Sen15; SEN15_HUMAN, tRNA Endonuclease, Structural Genomics, PSI, Protein Structure Initiative, Center for Eukaryotic Structural Genomics, CESG, PROTEIN; NMR {Homo sapiens} SCOP: l.1.1.1, c.52.2.1
</pre><pre>Probab=22.11  E-value=1.5e+02  Score=22.08  Aligned_cols=13  Identities=31%  Similarity=0.634  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEEEeec
</pre><pre>Q 10K_MODEL_53      1 NSSGDVIYHGITD   13 (116)
</pre><pre>Q Consensus         1 ~~~G~vVY~gITd   13 (116)
</pre><pre>                      ++++.|||+-|+|
</pre><pre>T Consensus       111 d~DstivYY~vs~  123 (123)
</pre><pre>T 2GW6_A          111 ESDSTIVYYKLTD  123 (123)
</pre><pre>T ss_dssp             CTTSCEEEEEEEC
</pre><pre>T ss_pred             cCCCceEEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>2K3U_A Chemotaxis inhibitory protein; Chemotaxis Inhibitory Protein (CHIPS), sulfated tyrosine, GPCR membrane protein C5aR, anaphylotoxin C5a, Staphylococcus Aureus, complement cascade; HET: NH2, TYS, ACE; NMR {Staphylococcus aureus subsp. aureus str. Newman}
</pre><pre>Probab=21.00  E-value=2.9e+02  Score=21.88  Aligned_cols=44  Identities=27%  Similarity=0.458  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHhcc----CCCccceeEEecccccceeeeccccccccccCCCcceeeEEEccCCeEEeccccchhcCCeEE
</pre><pre>Q 10K_MODEL_53     15 TVQERVIEHAG----DGRVFSKVRYVDDLGSRVNARNIEGSILDHAKGDPNIENAIRKDGGFYHSYDANNLAEGRKYL   88 (116)
</pre><pre>Q Consensus        15 ~vq~R~~eH~~----~~k~F~~v~yvd~l~~R~~aRnlEGSaL~H~~Gn~~l~Na~r~d~G~Yhsy~p~~La~Gr~~l   88 (116)
</pre><pre>                      .+-+||...+.    ..+.|.+|..+                              -..+|||..|-..+||+.|..+
</pre><pre>T Consensus        10 ~ld~rlr~~lkkgtkn~~~f~kmvi~------------------------------te~~~~ytv~l~~plaedrknv   57 (91)
</pre><pre>T 2K3U_A           10 KLDERLRNYLKKGTKNSAQFEKMVIL------------------------------TENKGYYTVYLNTPLAEDRKNV   57 (91)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHCSCCTTEEEEEEE------------------------------CSTTCEEEEETTSCCCHHHHTC
</pre><pre>T ss_pred             HHHHHHHHHHhhcCcChhhceeEEEE------------------------------ecCCeEEEEEcCCCccchhcch
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>PF11434.10 ; CHIPS ; Chemotaxis-inhibiting protein CHIPS
</pre><pre>Probab=21.00  E-value=2.9e+02  Score=21.88  Aligned_cols=44  Identities=27%  Similarity=0.458  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHhcc----CCCccceeEEecccccceeeeccccccccccCCCcceeeEEEccCCeEEeccccchhcCCeEE
</pre><pre>Q 10K_MODEL_53     15 TVQERVIEHAG----DGRVFSKVRYVDDLGSRVNARNIEGSILDHAKGDPNIENAIRKDGGFYHSYDANNLAEGRKYL   88 (116)
</pre><pre>Q Consensus        15 ~vq~R~~eH~~----~~k~F~~v~yvd~l~~R~~aRnlEGSaL~H~~Gn~~l~Na~r~d~G~Yhsy~p~~La~Gr~~l   88 (116)
</pre><pre>                      .+-+||...+.    ..+.|.+|..+                              -..+|||..|-..+||+.|..+
</pre><pre>T Consensus        10 ~ld~rlr~~lkkgtkn~~~f~kmvi~------------------------------te~~~~ytv~l~~plaedrknv   57 (91)
</pre><pre>T CHIPS_STAA8/59   10 KLDERLRNYLKKGTKNSAQFEKMVIL------------------------------TENKGYYTVYLNTPLAEDRKNV   57 (91)
</pre><pre>T ss_pred             HHHHHHHHHHhhcCcChhhceeEEEE------------------------------ecCCeEEEEEcCCCccchhcch
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>d3f0pa1 a.4.5.79 (A:1-80) Alkylmercury lyase MerB {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=20.98  E-value=1.9e+02  Score=21.85  Aligned_cols=28  Identities=21%  Similarity=0.408  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             EeHHHHHHHhhcceeeeecCCceeeecC
</pre><pre>Q 10K_MODEL_53     88 LSQEEINKILKNGTVVDVDNKGHFKSGG  115 (116)
</pre><pre>Q Consensus        88 ls~~~id~~~~~at~~~v~~~Gki~~~~  115 (116)
</pre><pre>                      .+.+++.+.+..-..++-|++|.|+.-|
</pre><pre>T Consensus        48 ~~~e~V~~~L~~~p~~EyD~~GrIIG~G   75 (80)
</pre><pre>T d3f0pa1          48 WPAERVAAVLEQATSTEYDKDGNIIGYG   75 (80)
</pre><pre>T ss_dssp             CCHHHHHHHHHHCTTCEECTTSCEEESS
</pre><pre>T ss_pred             CCHHHHHHHHHHcCCCEeCCCCCEEEee
</pre><pre>
</pre><pre>
</pre>