<pre>Query         EHC5102581.1/71-131 
</pre><pre>Match_columns 61
</pre><pre>No_of_seqs    12 out of 15
</pre><pre>Neff          2.22663
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 20:28:36 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/3501753.hhr -oa3m ../results/3501753.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF14412.8 ; AHH ; A nuclease f  93.8    0.75   7E-06   29.1   5.5   55    1-60     23-102 (123)
</pre><pre>  2 d1ussa_ a.4.5.13 (A:) Histone   75.3     7.6 7.1E-05   24.1   2.4   21   38-58      1-21  (88)
</pre><pre>  3 PF05778.14 ; Apo-CIII ; Apolip  74.4     3.1 2.9E-05   27.6   0.5   19   11-29      6-24  (69)
</pre><pre>  4 2JQ3_A Apolipoprotein C-III; A  73.4     3.8 3.5E-05   27.8   0.8   19   11-29      7-25  (79)
</pre><pre>  5 PF14023.8 ; DUF4239 ; Protein   65.9     5.4 5.1E-05   27.1   0.3   10   33-42    197-206 (220)
</pre><pre>  6 d1ghca_ a.4.5.13 (A:) Histone   65.4      12 0.00011   22.6   1.6   16   43-58      2-17  (75)
</pre><pre>  7 d1usta_ a.4.5.13 (A:) Histone   63.9      15 0.00014   23.3   2.0   17   42-58      4-20  (92)
</pre><pre>  8 d1hsta_ a.4.5.13 (A:) Histone   63.3      15 0.00014   22.2   1.8   16   43-58      1-16  (74)
</pre><pre>  9 6ZZ6_A Structural maintenance   60.9      50 0.00047   23.9   4.3   56    6-61    262-317 (360)
</pre><pre> 10 d1y71a1 b.34.16.1 (A:4-127) Ki  57.4      29 0.00027   25.4   2.7   23   38-60     76-98  (124)
</pre><pre> 11 6X3M_E Hyaluronoglucosaminidas  57.0      24 0.00022   28.4   2.5   27   34-60    340-366 (379)
</pre><pre> 12 PF12108.10 ; SF3a60_bindingd ;  51.0      45 0.00042   17.2   2.1   15    5-19      7-21  (27)
</pre><pre> 13 PF04691.14 ; ApoC-I ; Apolipop  50.5      26 0.00024   21.5   1.4   24    5-29      4-27  (60)
</pre><pre> 14 PF12129.10 ; Phtf-FEM1B_bdg ;   48.4      23 0.00021   25.6   1.1   33    3-37     17-50  (153)
</pre><pre> 15 d1gkya_ c.37.1.1 (A:) Guanylat  48.2      82 0.00076   20.0   3.3   31   30-60    122-152 (186)
</pre><pre> 16 4FWI_B ABC-type dipeptide/olig  46.3      81 0.00076   22.6   3.4   32   30-61    172-203 (334)
</pre><pre> 17 g1xew.1 c.37.1.12 (X:,Y:) Smc   44.3      65  0.0006   22.4   2.6   26   36-61    253-278 (330)
</pre><pre> 18 2WH7_A HYALURONIDASE-PHAGE ASS  44.2      65  0.0006   25.3   2.9   27   34-60    145-171 (178)
</pre><pre> 19 TIGR00597 rad10; DNA repair pr  42.6      34 0.00031   23.6   1.1   23   36-61     43-65  (112)
</pre><pre> 20 g1f2t.1 c.37.1.12 (A:,B:) Rad5  42.3      86  0.0008   20.5   2.8   26   36-61    232-257 (293)
</pre><pre> 21 6Z4W_A Cell division ATP-bindi  42.2 1.2E+02  0.0011   20.0   3.4   32   30-61    157-188 (230)
</pre><pre> 22 2DT7_A Splicing factor 3A subu  42.2      72 0.00067   17.5   2.2   15    5-19     11-25  (38)
</pre><pre> 23 PF03215.17 ; Rad17 ; Rad17 P-l  41.3 1.5E+02  0.0014   17.9   4.0   51   11-61    104-154 (180)
</pre><pre> 24 3TIF_A Uncharacterized ABC tra  40.7 1.1E+02  0.0011   18.4   3.0   32   30-61    163-194 (235)
</pre><pre> 25 1GHC_A GH1; CHROMOSOMAL PROTEI  39.8      62 0.00057   17.1   1.7   15   44-58      3-17  (75)
</pre><pre> 26 3QF7_B Rad50; ABC-ATPase, ATPa  39.7 1.3E+02  0.0012   19.1   3.2   33   29-61    302-334 (365)
</pre><pre> 27 PF16205.7 ; Ribosomal_S17_N ;   38.5      45 0.00042   20.1   1.1   12   32-43     44-55  (63)
</pre><pre> 28 3HCN_A Ferrochelatase, mitocho  37.7   1E+02 0.00097   22.6   3.0   23   38-61    161-183 (359)
</pre><pre> 29 d1h2vc1 a.118.1.14 (C:29-290)   37.6 2.9E+02  0.0027   20.1   5.8   53    7-61    162-214 (262)
</pre><pre> 30 1JI0_A ABC transporter; ATP bi  37.4 1.3E+02  0.0012   20.1   3.1   26   36-61    163-188 (240)
</pre><pre> 31 7ARL_D Lipoprotein-releasing s  36.7 1.6E+02  0.0015   18.6   3.3   31   30-60    163-193 (241)
</pre><pre> 32 d2foka1 a.4.5.12 (A:5-143) Res  36.1      23 0.00022   26.1  -0.4   40   18-61      5-44  (139)
</pre><pre> 33 TIGR02673 FtsE; cell division   36.0 1.7E+02  0.0016   18.0   3.3   32   30-61    155-186 (214)
</pre><pre> 34 1USS_A HISTONE H1; DNA BINDING  35.8      81 0.00075   17.3   1.7   16   43-58      6-21  (88)
</pre><pre> 35 2PWW_A Uncharacterized protein  34.1 1.4E+02  0.0013   20.7   3.1   22    1-22     96-117 (127)
</pre><pre> 36 4U00_A Amino acid ABC transpor  34.1 1.6E+02  0.0015   20.0   3.2   32   30-61    156-187 (244)
</pre><pre> 37 2WB3_A HYALURONIDASE-PHAGE ASS  34.1      94 0.00087   24.1   2.4   25   36-60    128-152 (157)
</pre><pre> 38 2PCJ_B Lipoprotein-releasing s  33.5 1.7E+02  0.0016   17.4   3.0   31   31-61    159-189 (224)
</pre><pre> 39 6IC4_I ABC transporter ATP-bin  33.4   2E+02  0.0019   17.5   3.2   31   30-60    149-179 (233)
</pre><pre> 40 PF00538.21 ; Linker_histone ;   32.9 1.1E+02 0.00099   15.5   1.8   15   44-58      1-15  (75)
</pre><pre> 41 2LSO_A Histone H1x; Structural  32.7 1.3E+02  0.0012   16.3   2.3   18   41-58      2-19  (83)
</pre><pre> 42 d5x7ka_ c.37.1.0 (A:) automate  32.2 1.6E+02  0.0015   19.9   3.0   26   36-61    171-196 (243)
</pre><pre> 43 1UHM_A Histone H1; winged heli  32.2 1.1E+02 0.00097   16.2   1.8   15   44-58      4-18  (78)
</pre><pre> 44 TIGR00972 3a0107s01c2; phospha  32.1 1.6E+02  0.0015   18.9   2.8   32   30-61    162-193 (247)
</pre><pre> 45 d2pcja_ c.37.1.0 (A:) automate  32.0 1.7E+02  0.0016   19.2   3.0   26   36-61    163-188 (223)
</pre><pre> 46 4ZRX_A F5/8 type C domain prot  31.9 1.3E+02  0.0012   22.9   2.7   27   35-61    149-175 (586)
</pre><pre> 47 3ZGX_B CHROMOSOME PARTITION PR  31.6 2.8E+02  0.0026   18.3   3.8   48   14-61    334-381 (426)
</pre><pre> 48 6XOR_A Protein swallow; coiled  31.4   1E+02 0.00093   21.0   2.0   12   46-57     28-39  (71)
</pre><pre> 49 PF04690.15 ; YABBY ; YABBY pro  31.2 1.1E+02  0.0011   21.9   2.3   16   46-61    122-137 (164)
</pre><pre> 50 PF14883.8 ; GHL13 ; Hypothetic  31.0 1.4E+02  0.0013   19.8   2.5   19   43-61    121-139 (322)
</pre><pre> 51 1JZS_A Isoleucyl-tRNA syntheta  30.8 3.6E+02  0.0034   22.1   5.0   55    7-61    113-168 (821)
</pre><pre> 52 6UK1_A Cystic fibrosis transme  30.6 1.6E+02  0.0015   18.0   2.5   28   33-60    165-192 (229)
</pre><pre> 53 d2r5xa1 d.129.11.0 (A:2-119) a  30.5 1.7E+02  0.0016   20.4   2.9   21    1-21     94-114 (118)
</pre><pre> 54 PF15388.8 ; FAM117 ; Protein F  30.5 1.7E+02  0.0016   23.5   3.4   35   10-44     69-103 (310)
</pre><pre> 55 PF16707.7 ; CagS ; Cag pathoge  30.0      80 0.00074   25.1   1.5   12   46-57    173-184 (196)
</pre><pre> 56 TIGR02769 nickel_nikE; nickel   29.6 1.6E+02  0.0014   19.4   2.5   28   33-60    171-198 (265)
</pre><pre> 57 6QJ0_A Structural maintenance   29.5 1.5E+02  0.0014   21.5   2.7   33   29-61    340-372 (412)
</pre><pre> 58 TIGR03411 urea_trans_UrtD; ure  29.2 2.2E+02  0.0021   18.1   3.1   32   30-61    161-192 (242)
</pre><pre> 59 4P7O_B Poly-beta-1,6-N-acetyl-  29.1 1.6E+02  0.0015   21.0   2.7   19   43-61    133-151 (367)
</pre><pre> 60 2G3V_B CAG pathogenicity islan  29.0      84 0.00078   25.1   1.5   12   46-57    174-185 (208)
</pre><pre> 61 2G3V_D CAG pathogenicity islan  29.0      84 0.00078   25.1   1.5   12   46-57    174-185 (208)
</pre><pre> 62 d1t0fa1 a.4.5.27 (A:169-268) T  28.9      65  0.0006   19.4   0.7   12   34-45     75-86  (100)
</pre><pre> 63 6BZS_A Multidrug resistance-as  28.7      81 0.00075   21.5   1.2   31   30-60    149-179 (238)
</pre><pre> 64 TIGR02770 nickel_nikD; nickel   28.4 2.6E+02  0.0024   17.7   3.2   31   30-60    143-173 (230)
</pre><pre> 65 d3rlfa1 c.37.1.12 (A:2-235) Ma  28.1 2.1E+02  0.0019   19.3   2.9   31   30-60    150-180 (234)
</pre><pre> 66 3V6B_R Vascular endothelial gr  27.6      61 0.00057   21.9   0.5    9   36-44    415-423 (424)
</pre><pre> 67 1G6H_A HIGH-AFFINITY BRANCHED-  27.1 2.7E+02  0.0025   18.8   3.3   32   30-61    171-202 (257)
</pre><pre> 68 6BZR_A Multidrug resistance-as  26.9 2.4E+02  0.0023   17.6   2.9   32   30-61    167-198 (251)
</pre><pre> 69 1B0U_A HISTIDINE PERMEASE; ABC  26.9 2.7E+02  0.0025   19.0   3.3   31   31-61    172-202 (262)
</pre><pre> 70 2GHI_C transport protein; Plas  26.5 2.2E+02   0.002   18.3   2.7   29   33-61    176-204 (260)
</pre><pre> 71 d1h9aa1 c.2.1.3 (A:1-181,A:413  26.2 4.8E+02  0.0045   19.1   5.0   50    1-58    118-167 (196)
</pre><pre> 72 6QPQ_A Structural maintenance   26.1   2E+02  0.0019   19.6   2.6   30   29-58    376-405 (453)
</pre><pre> 73 7C0J_B Histone H5,Double-stran  25.9 1.9E+02  0.0018   15.3   2.1   17   42-58      5-21  (76)
</pre><pre> 74 d2xwsa1 c.92.1.3 (A:1-125) aut  25.8 1.8E+02  0.0017   17.9   2.3   24   34-58    101-124 (125)
</pre><pre> 75 d2c3fa1 b.108.1.5 (A:7-336) Ph  25.8   2E+02  0.0018   24.7   3.1   37   24-60    290-326 (330)
</pre><pre> 76 1H6K_A CBP80; M7G CAP, CAP-BIN  25.7 6.3E+02  0.0058   21.9   5.8   53    7-61    171-223 (757)
</pre><pre> 77 1H6K_B CBP80; M7G CAP, CAP-BIN  25.7 6.3E+02  0.0058   21.9   5.8   53    7-61    171-223 (757)
</pre><pre> 78 2ZU0_D Probable ATP-dependent   25.6 2.8E+02  0.0026   17.8   3.1   32   30-61    182-213 (267)
</pre><pre> 79 1HST_B HISTONE H5; CHROMOSOMAL  25.6 1.8E+02  0.0017   16.2   2.0   17   42-58      5-21  (90)
</pre><pre> 80 6M2A_A Sirohydrochlorin cobalt  25.5 1.5E+02  0.0014   17.5   1.8   16   42-57    110-125 (126)
</pre><pre> 81 PF07212.13 ; Hyaluronidase_1 ;  25.4 2.2E+02   0.002   23.8   3.2   37   24-60    232-268 (273)
</pre><pre> 82 PF13175.8 ; AAA_15 ; AAA ATPas  25.4 1.6E+02  0.0015   21.0   2.2   34   28-61    357-390 (409)
</pre><pre> 83 TIGR00105 L31; ribosomal prote  25.2      50 0.00046   20.3  -0.2    7   42-48     40-46  (68)
</pre><pre> 84 6AU1_B Putative hemin storage   25.1 1.9E+02  0.0018   20.6   2.5   19   43-61    133-151 (357)
</pre><pre> 85 5B57_C Hemin import ATP-bindin  24.9 3.6E+02  0.0033   17.7   3.5   34   27-60    170-203 (273)
</pre><pre> 86 1P4W_A rcsB; RcsB protein, sol  24.9      71 0.00066   17.9   0.4    6   40-45      5-10  (99)
</pre><pre> 87 4ZQA_A Sin3 histone deacetylas  24.9 1.3E+02  0.0012   18.7   1.5   12   42-53      4-15  (86)
</pre><pre> 88 2WXU_A PHOSPHOLIPASE C; CYTOLY  24.6 1.8E+02  0.0017   21.3   2.4   26   33-58    128-162 (370)
</pre><pre> 89 3QKS_A DNA double-strand break  24.6 1.1E+02   0.001   18.8   1.2   39    7-45    164-203 (203)
</pre><pre> 90 TIGR04406 LPS_export_lptB; LPS  24.6 3.2E+02   0.003   17.6   3.2   26   36-61    159-184 (239)
</pre><pre> 91 2C3F_A HYALURONIDASE, PHAGE AS  24.4   2E+02  0.0018   24.9   2.9   27   34-60    327-353 (358)
</pre><pre> 92 d2d62a1 c.37.1.12 (A:1-243) au  24.3 2.2E+02   0.002   19.4   2.6   32   30-61    160-191 (243)
</pre><pre> 93 d3tifa_ c.37.1.12 (A:) automat  24.2 2.7E+02  0.0025   18.4   2.9   25   36-60    169-193 (230)
</pre><pre> 94 d5lj9a1 c.37.1.0 (A:1-223) aut  23.8 3.2E+02   0.003   17.9   3.2   32   30-61    162-193 (223)
</pre><pre> 95 d6amxa1 c.37.1.0 (A:2-235) aut  23.7 2.6E+02  0.0025   19.1   2.9   26   36-61    164-189 (234)
</pre><pre> 96 PF02638.17 ; GHL10 ; Glycosyl   23.6 2.3E+02  0.0022   19.7   2.6   19   43-61    139-157 (322)
</pre><pre> 97 2D2E_A SufC protein; ABC-ATPas  23.6 2.9E+02  0.0027   18.3   3.0   32   30-61    161-192 (250)
</pre><pre> 98 4HUQ_B Energy-coupling factor   23.6 3.1E+02  0.0029   19.2   3.2   32   30-61    158-189 (279)
</pre><pre> 99 6DVU_B Apolipoprotein C-I; lip  23.5 1.1E+02   0.001   20.5   1.1   21    9-29     30-50  (83)
</pre><pre>100 6YAL_k ATP-binding cassette su  23.1 2.5E+02  0.0023   20.7   2.8   32   30-61    476-507 (595)
</pre><pre>101 5HAB_A Ribonuclease J; exoribo  22.8 1.4E+02  0.0013   22.3   1.7   19   39-58    430-448 (470)
</pre><pre>102 PF14257.8 ; DUF4349 ; Domain o  22.6      78 0.00073   21.9   0.3   30    1-31     54-83  (219)
</pre><pre>103 2RQP_A Heterochromatin protein  22.5   2E+02  0.0019   15.6   1.8   15   44-58      9-23  (88)
</pre><pre>104 TIGR03190 benz_CoA_bzdN; benzo  22.5 3.3E+02  0.0031   20.6   3.4   25   36-60    121-145 (377)
</pre><pre>105 PF02981.17 ; FokI_N ; Restrict  22.4      64  0.0006   24.4  -0.1   39   18-60      4-42  (154)
</pre><pre>106 d1lvga_ c.37.1.1 (A:) Guanylat  22.3 3.1E+02  0.0029   17.3   2.8   32   30-61    120-151 (190)
</pre><pre>107 6YIR_A Oligosaccharides import  22.2   3E+02  0.0028   20.4   3.1   32   30-61    152-183 (373)
</pre><pre>108 TIGR01166 cbiO; cobalt transpo  22.2 3.7E+02  0.0034   16.4   3.2   32   30-61    145-176 (190)
</pre><pre>109 5O6C_A E3 ubiquitin-protein li  22.1      88 0.00082   22.6   0.5    7   39-45    238-244 (263)
</pre><pre>110 4V1J_A FIMBRIAL PROTEIN; CELL   22.0 1.2E+02  0.0012   19.3   1.1   10   45-54      1-10  (142)
</pre><pre>111 6HIY_Cm mS37; mitoribosome, tr  22.0 6.2E+02  0.0058   20.9   4.9   42    7-57    134-177 (215)
</pre><pre>112 PF08745.13 ; PIN_5 ; PINc doma  22.0 2.3E+02  0.0022   22.3   2.6   33    1-33     47-87  (217)
</pre><pre>113 6LDK_A Isoleucine--tRNA ligase  22.0 5.7E+02  0.0053   21.4   4.7   55    7-61    120-175 (856)
</pre><pre>114 d1lxna_ d.58.48.1 (A:) Hypothe  21.7 2.4E+02  0.0022   18.2   2.3   22   39-60     10-31  (99)
</pre><pre>115 6ZG3_F Energy-coupling factor   21.6   3E+02  0.0027   18.5   2.7   29   33-61    182-210 (300)
</pre><pre>116 2R5X_B Uncharacterized conserv  21.6 3.3E+02  0.0031   19.0   3.1   22    1-22     97-118 (129)
</pre><pre>117 d1yqha1 d.58.48.1 (A:1-101) Hy  21.5 2.2E+02   0.002   18.6   2.1   21   39-60     13-33  (101)
</pre><pre>118 6WWX_A Hyaluronoglucosaminidas  21.5   2E+02  0.0018   24.4   2.3   27   34-60    254-280 (293)
</pre><pre>119 d1sgwa1 c.37.1.12 (A:2-200) Pu  21.5 3.5E+02  0.0033   17.2   3.0   32   30-61    142-173 (199)
</pre><pre>120 2IT1_B 362aa long hypothetical  21.4 3.1E+02  0.0029   19.3   2.9   31   30-60    151-181 (362)
</pre><pre>121 d2zu0c_ c.37.1.0 (C:) automate  21.1 3.5E+02  0.0032   18.3   3.0   32   30-61    163-194 (247)
</pre><pre>122 6N9L_A UvrABC system protein A  21.1 3.1E+02  0.0028   20.7   2.9   29   33-61    553-581 (640)
</pre><pre>123 TIGR00422 valS; valyl-tRNA syn  21.1 5.8E+02  0.0054   21.7   4.7   55    7-61    107-162 (861)
</pre><pre>124 4GAK_A Acyl-ACP thioesterase;   21.0 1.2E+02  0.0011   20.1   0.8   21   35-55    170-190 (250)
</pre><pre>125 PF14426.8 ; Imm2 ; Immunity pr  20.8 3.3E+02   0.003   16.8   2.6   20   42-61     12-32  (60)
</pre><pre>126 6XGZ_A Organic solvent ABC tra  20.7 3.3E+02   0.003   17.7   2.7   28   33-60    165-192 (269)
</pre><pre>127 6XGZ_G Organic solvent ABC tra  20.7 3.3E+02   0.003   17.7   2.7   28   33-60    165-192 (269)
</pre><pre>128 3HE4_D SYNZIP5; heterodimeric   20.6      86  0.0008   20.0   0.2   24   10-33      2-25  (46)
</pre><pre>129 3HE4_F SYNZIP5; heterodimeric   20.6      86  0.0008   20.0   0.2   24   10-33      2-25  (46)
</pre><pre>130 6XJI_D ABC transporter ATP-bin  20.6 3.1E+02  0.0028   18.5   2.6   28   33-60    140-167 (290)
</pre><pre>131 TIGR01189 ccmA; heme ABC expor  20.5   4E+02  0.0037   16.1   3.0   26   36-61    151-176 (198)
</pre><pre>132 d2a1ia1 c.52.1.20 (A:99-227) D  20.3 1.5E+02  0.0014   21.1   1.2   23   36-61     43-65  (129)
</pre><pre>133 TIGR02982 heterocyst_DevA; ABC  20.2 2.7E+02  0.0025   17.2   2.2   32   30-61    159-190 (220)
</pre><pre>134 1UST_A HISTONE H1; DNA BINDING  20.1 2.6E+02  0.0024   15.6   2.0   17   42-58      5-21  (93)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF14412.8 ; AHH ; A nuclease family of the HNH/ENDO VII superfamily with conserved AHH
</pre><pre>Probab=93.77  E-value=0.75  Score=29.07  Aligned_cols=55  Identities=20%  Similarity=0.359  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhhhHHHHhhhcccccccccccCC----------CCcC-CCC----------hhHHHHH----HHHH
</pre><pre>Q EHC5102581.1/7    1 IPQEIWKEFKTDLKKVTGYVQNVTKKAADMTNLIDLD----------KPFH-GNH----------PAYSQYV----REAI   55 (61)
</pre><pre>Q Consensus         1 IPn~iyk~f~~~~k~i~gYvQn~skka~d~tNLidLd----------kPFH-GNH----------P~Yn~YV----~~~i   55 (61)
</pre><pre>                      ||......-....     +++...-...+..|++-|+          .|-| |.|          +.|.++|    ..++
</pre><pre>T Consensus        23 I~~~~~~~~~~~~-----~~~~~g~~in~~~Ngv~LP~~~~~~~~~~~~~H~g~H~~~~~~~~~~~~Y~~~v~~~~~~~l   97 (123)
</pre><pre>T B7VTP2_VIBA3/9   23 LSKKGAIESGNKS-----ILESYGYDINEIGNLVALPSTFAGACHLEIQLHRGNHISFSNSDDDDELYHDYVQKLMVKTL   97 (123)
</pre><pre>T ss_pred             ccHHHHHhccHHH-----HHHHHCCCCCCcccccccCCChHHHhhcCCCccCCCCccCCCCCCCCchHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHh
</pre><pre>Q EHC5102581.1/7   56 EDLKA   60 (61)
</pre><pre>Q Consensus        56 ~kl~k   60 (61)
</pre><pre>                      ..+.+
</pre><pre>T Consensus        98 ~~i~~  102 (123)
</pre><pre>T B7VTP2_VIBA3/9   98 KKIEK  102 (123)
</pre><pre>T ss_pred             HHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>d1ussa_ a.4.5.13 (A:) Histone H1 homologue Hho1p {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=75.27  E-value=7.6  Score=24.14  Aligned_cols=21  Identities=14%  Similarity=0.278  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CCcCCCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   38 KPFHGNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        38 kPFHGNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ++-.+.||.|.+.|.+.|.+|
</pre><pre>T Consensus         1 k~~~~~~p~~~~mI~eAI~~l   21 (88)
</pre><pre>T d1ussa_           1 KASSPSSLTYKEMILKSMPQL   21 (88)
</pre><pre>T ss_dssp             CTTCCSSCCHHHHHHHHHHHS
</pre><pre>T ss_pred             CCCCCCCCCHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>PF05778.14 ; Apo-CIII ; Apolipoprotein CIII (Apo-CIII)
</pre><pre>Probab=74.41  E-value=3.1  Score=27.58  Aligned_cols=19  Identities=37%  Similarity=0.573  Sum_probs=0.0  Template_Neff=3.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHhhhHHHHhhhccccc
</pre><pre>Q EHC5102581.1/7   11 TDLKKVTGYVQNVTKKAAD   29 (61)
</pre><pre>Q Consensus        11 ~~~k~i~gYvQn~skka~d   29 (61)
</pre><pre>                      ..|.++.||+|..+++|+|
</pre><pre>T Consensus         6 sLl~~mq~Y~qqAtktaqd   24 (69)
</pre><pre>T APOC3_PIG/22-9    6 SLLDKMQDYVKQATRTAQD   24 (69)
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>2JQ3_A Apolipoprotein C-III; ApoCIII, dynamics, apolipoprotein, receptor, LIPID BINDING PROTEIN; NMR {Homo sapiens}
</pre><pre>Probab=73.39  E-value=3.8  Score=27.85  Aligned_cols=19  Identities=37%  Similarity=0.548  Sum_probs=0.0  Template_Neff=3.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHhhhHHHHhhhccccc
</pre><pre>Q EHC5102581.1/7   11 TDLKKVTGYVQNVTKKAAD   29 (61)
</pre><pre>Q Consensus        11 ~~~k~i~gYvQn~skka~d   29 (61)
</pre><pre>                      ..|.++.||+|..+++|+|
</pre><pre>T Consensus         7 sLl~~mQ~Y~qqAtktaqd   25 (79)
</pre><pre>T 2JQ3_A            7 SLLSFMQGYMKHATKTAKD   25 (79)
</pre><pre>T ss_dssp             CTHHHHHHHHHHHGGGHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF14023.8 ; DUF4239 ; Protein of unknown function (DUF4239)
</pre><pre>Probab=65.88  E-value=5.4  Score=27.07  Aligned_cols=10  Identities=50%  Similarity=1.099  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCC
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHG   42 (61)
</pre><pre>Q Consensus        33 LidLdkPFHG   42 (61)
</pre><pre>                      +.|||.||+|
</pre><pre>T Consensus       197 ~~dl~~Pf~G  206 (220)
</pre><pre>T B7FWE4_PHATC/2  197 CYDLNRPFDG  206 (220)
</pre><pre>T ss_pred             HHHccCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>d1ghca_ a.4.5.13 (A:) Histone H1, globular domain {Chicken (Gallus gallus) [TaxId: 9031]}
</pre><pre>Probab=65.42  E-value=12  Score=22.62  Aligned_cols=16  Identities=13%  Similarity=0.345  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      .||.|.+-|.+.|..|
</pre><pre>T Consensus         2 ~hp~~~~mI~~Ai~~l   17 (75)
</pre><pre>T d1ghca_           2 AGPSVTELITKAVSAS   17 (75)
</pre><pre>T ss_dssp             CSSSHCCCCCHHHHHC
</pre><pre>T ss_pred             CCCCHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>d1usta_ a.4.5.13 (A:) Histone H1 homologue Hho1p {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=63.87  E-value=15  Score=23.28  Aligned_cols=17  Identities=18%  Similarity=0.436  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        42 GNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      .+||.|.+-|.+.|..|
</pre><pre>T Consensus         4 ~~~p~~~~mI~~Ai~~l   20 (92)
</pre><pre>T d1usta_           4 ASSKSYRELIIEGLTAL   20 (92)
</pre><pre>T ss_dssp             SCCCCHHHHHHHHHTTT
</pre><pre>T ss_pred             cCCCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>d1hsta_ a.4.5.13 (A:) Histone H5, globular domain {Chicken (Gallus gallus) [TaxId: 9031]}
</pre><pre>Probab=63.25  E-value=15  Score=22.17  Aligned_cols=16  Identities=38%  Similarity=0.683  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      .||.|.+-|.+.|..|
</pre><pre>T Consensus         1 ~~p~~~~mI~~Ai~~l   16 (74)
</pre><pre>T d1hsta_           1 SHPTYSEMIAAAIRAE   16 (74)
</pre><pre>T ss_dssp             CCCCHHHHHHHHHHTC
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>6ZZ6_A Structural maintenance of chromosomes protein 1,Structural maintenance of chromosomes protein 1,Structural maintenance of chromosomes protein 1; SMC, cohesin, DNA, CELL CYCLE; HET: ATP; 3.4A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=60.91  E-value=50  Score=23.88  Aligned_cols=56  Identities=11%  Similarity=-0.009  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    6 WKEFKTDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         6 yk~f~~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +..+..--+..-...............++=||.||+|-++.....+.+.|.++...
</pre><pre>T Consensus       262 ~~~lS~g~~~~~~la~~l~~~~~~~~~~lllDep~~~l~~~~~~~~~~~l~~~~~~  317 (360)
</pre><pre>T 6ZZ6_A          262 MEYLSGGEKTVAALALLFAINSYQPSPFFVLDQVDAALDITNVQRIAAYIRRHRNP  317 (360)
</pre><pre>T ss_dssp             STTSCHHHHHHHHHHHHHHHHHHSCCSEEEESSTTSSCCHHHHHHHHHHHHHHCCS
</pre><pre>T ss_pred             hhhhchHHHHHHHHHHHHHhhcCCCCCEEEEecchhcCCHHHHHHHHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>d1y71a1 b.34.16.1 (A:4-127) Kinase-associated protein B {Bacillus cereus [TaxId: 1396]}
</pre><pre>Probab=57.40  E-value=29  Score=25.42  Aligned_cols=23  Identities=26%  Similarity=0.537  Sum_probs=0.0  Template_Neff=3.700
</pre><pre>
</pre><pre>Q ss_pred             CCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   38 KPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        38 kPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      +||+|--|.|.+=++..+++++.
</pre><pre>T Consensus        76 kpy~g~vPdY~eSL~~Al~~~~~   98 (124)
</pre><pre>T d1y71a1          76 KKYEGEIPDYTESLKLALETQMN   98 (124)
</pre><pre>T ss_dssp             EECCSCCCCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             cccCCCCCCHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>6X3M_E Hyaluronoglucosaminidase; HylP, Lyase, Hyaluronidase, S. pyogenes phage H4489A, Beta-elimination, unsaturated Hyaluronan Octa-saccharides; HET: BDP, NAG, GCD; 3.581A {Streptococcus pyogenes phage H4489A}
</pre><pre>Probab=57.03  E-value=24  Score=28.43  Aligned_cols=27  Identities=37%  Similarity=0.662  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   34 IDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        34 idLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      +.|..|--++|.+-..||.+++.+|++
</pre><pre>T Consensus       340 ~~~~~~~~~~~~~~~~~~~~~~~~~~~  366 (379)
</pre><pre>T 6X3M_E          340 LTVKDPTSGKHAATKDYVDEKIAELKK  366 (379)
</pre><pre>T ss_pred             eEecCCCCCCccccHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>PF12108.10 ; SF3a60_bindingd ; Splicing factor SF3a60 binding domain
</pre><pre>Probab=51.04  E-value=45  Score=17.21  Aligned_cols=15  Identities=27%  Similarity=0.581  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhhHH
</pre><pre>Q EHC5102581.1/7    5 IWKEFKTDLKKVTGY   19 (61)
</pre><pre>Q Consensus         5 iyk~f~~~~k~i~gY   19 (61)
</pre><pre>                      +|.+|-..|++|+.|
</pre><pre>T Consensus         7 ~~~~FY~rlk~Ikd~   21 (27)
</pre><pre>T H6BSN4_EXODN/7    7 PFEEFYKQLDEVKDF   21 (27)
</pre><pre>T ss_pred             hHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>PF04691.14 ; ApoC-I ; Apolipoprotein C-I (ApoC-1)
</pre><pre>Probab=50.49  E-value=26  Score=21.49  Aligned_cols=24  Identities=13%  Similarity=0.134  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhhHHHHhhhccccc
</pre><pre>Q EHC5102581.1/7    5 IWKEFKTDLKKVTGYVQNVTKKAAD   29 (61)
</pre><pre>Q Consensus         5 iyk~f~~~~k~i~gYvQn~skka~d   29 (61)
</pre><pre>                      | ..|.+...++.+|.|+++.||++
</pre><pre>T Consensus         4 i-~~f~~f~~~v~e~~~~laekak~   27 (60)
</pre><pre>T B6S6L6_RABIT/2    4 F-STLELIPDKLKEFGNTLEEKARM   27 (60)
</pre><pre>T ss_pred             h-HHHhhHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF12129.10 ; Phtf-FEM1B_bdg ; Male germ-cell putative homeodomain transcription factor
</pre><pre>Probab=48.40  E-value=23  Score=25.64  Aligned_cols=33  Identities=33%  Similarity=0.561  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHhhhHHHHhhhcc-cccccccccCC
</pre><pre>Q EHC5102581.1/7    3 QEIWKEFKTDLKKVTGYVQNVTKK-AADMTNLIDLD   37 (61)
</pre><pre>Q Consensus         3 n~iyk~f~~~~k~i~gYvQn~skk-a~d~tNLidLd   37 (61)
</pre><pre>                      .+.|+.--+. +.+.| +.++.+| ++-+++|||+|
</pre><pre>T Consensus        17 kq~WE~~veq-~~l~g-~~~~pkk~~~~k~elIDvD   50 (153)
</pre><pre>T E0VE78_PEDHC/3   17 KQQWEKTVEQ-RILHG-LTHVLKKTAKLKTDLIDVD   50 (153)
</pre><pre>T ss_pred             ccHHHHHHHH-HHHcC-CccCcccccccCccceeee
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>d1gkya_ c.37.1.1 (A:) Guanylate kinase {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=48.16  E-value=82  Score=20.00  Aligned_cols=31  Identities=16%  Similarity=0.267  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...+..||.|+.|-.+.-..-+.+.|..+..
</pre><pre>T Consensus       122 ~~~~~~lde~~~~~d~~~~~~~~~~l~~~~~  152 (186)
</pre><pre>T d1gkya_         122 PPSVEDLKKRLEGRGTETEESINKRLSAAQA  152 (186)
</pre><pre>T ss_dssp             CSCHHHHHHHHHHHSCSCHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCceecccccccCCChhhHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>4FWI_B ABC-type dipeptide/oligopeptide/nickel transport system, ATPase component; NBDs, ABC transporter, nucleotide binding, iron-sulfur cluster, TRANSPORT PROTEIN; HET: ATP; 2.892A {Thermoanaerobacter tengcongensis}
</pre><pre>Probab=46.35  E-value=81  Score=22.62  Aligned_cols=32  Identities=16%  Similarity=0.149  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...|+-||.||.|-.+....-+.+.|.++.+.
</pre><pre>T Consensus       172 ~p~llllDep~~~ld~~~~~~l~~~l~~~~~~  203 (334)
</pre><pre>T 4FWI_B          172 DPVVLILDEPTSALDVLTQAHIIQLLKELKKM  203 (334)
</pre><pre>T ss_dssp             CCSEEEESSTTSSSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCcEeeecCcccCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>g1xew.1 c.37.1.12 (X:,Y:) Smc head domain {Pyrococcus furiosus [TaxId: 2261]}
</pre><pre>Probab=44.34  E-value=65  Score=22.36  Aligned_cols=26  Identities=15%  Similarity=0.075  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +|.|+.|-||....-+.+.+.++.++
</pre><pre>T Consensus       253 iDEp~~~Lhp~~~~~l~~~L~~l~~~  278 (330)
</pre><pre>T g1xew.1         253 FDEIDAHLDDANVKRVADLIKESSKE  278 (330)
</pre><pre>T ss_dssp             EESTTTTSCHHHHHHHHHHHHHHTTT
</pre><pre>T ss_pred             EEccccCCCHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>2WH7_A HYALURONIDASE-PHAGE ASSOCIATED; TRIPLE-STRANDED BETA-HELIX, HYALURONAN LYASE, PHAGE TAIL FIBRE, HYDROLASE, GLYCOSIDASE, HYALURONIDASE, SCARLET FEVER; 1.6A {STREPTOCOCCUS PYOGENES}
</pre><pre>Probab=44.17  E-value=65  Score=25.34  Aligned_cols=27  Identities=37%  Similarity=0.646  Sum_probs=0.0  Template_Neff=2.200
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   34 IDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        34 idLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      +.|..|--+-|.+-.+||.++|.+|++
</pre><pre>T Consensus       145 l~~~~p~~~~ha~tk~yvd~~~~~lk~  171 (178)
</pre><pre>T 2WH7_A          145 LKLKDPIANDHAATKAYVDGEVEKLKA  171 (178)
</pre><pre>T ss_dssp             CCCCCCCSTTSCCCHHHHHHHHHHHHH
</pre><pre>T ss_pred             eeecCCCCCCchhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>TIGR00597 rad10; DNA repair protein rad10. All proteins in this family for which functions are known are components in a multiprotein endonuclease complex (usually made up of Rad1 and Rad10 homologs).
</pre><pre>Probab=42.63  E-value=34  Score=23.56  Aligned_cols=23  Identities=30%  Similarity=0.665  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      |.--||--||.|   +.+++++|.++
</pre><pre>T Consensus        43 LSlkyH~l~P~Y---i~~Ri~~l~~~   65 (112)
</pre><pre>T TIGR00597        43 LSLRYHNLHPDY---IHRRLQSLGKN   65 (112)
</pre><pre>T ss_pred             EchhHHhhCHHH---HHHHHHHhccc
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>g1f2t.1 c.37.1.12 (A:,B:) Rad50 {Pyrococcus furiosus [TaxId: 2261]}
</pre><pre>Probab=42.30  E-value=86  Score=20.50  Aligned_cols=26  Identities=15%  Similarity=0.093  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +|.|.-|-||....-+.+.|.++.++
</pre><pre>T Consensus       232 iDEpe~~Lhp~~~~~l~~~l~~~~~~  257 (293)
</pre><pre>T g1f2t.1         232 LDEPTPYLDEERRRKLITIMERYLKK  257 (293)
</pre><pre>T ss_dssp             EESCSCTTCHHHHHHHHHHHHHTGGG
</pre><pre>T ss_pred             EeCCCccCCHHHHHHHHHHHHHHhcc
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>6Z4W_A Cell division ATP-binding protein FtsE; Cell division, divisome, FtsEX, ATP-binding protein, CELL CYCLE; HET: ADP; 1.36A {Streptococcus pneumoniae}
</pre><pre>Probab=42.22  E-value=1.2e+02  Score=19.99  Aligned_cols=32  Identities=16%  Similarity=0.200  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++-||.||.|-++...+.+.+.+.++.++
</pre><pre>T Consensus       157 ~~~~~~ldep~~~ld~~~~~~~~~~l~~~~~~  188 (230)
</pre><pre>T 6Z4W_A          157 NPKVLIADEPTGNLDPDNSWEIMNLLERINLQ  188 (230)
</pre><pre>T ss_dssp             CCSEEEEESTTTTSCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CCcEEEecCCCCCCChHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>2DT7_A Splicing factor 3A subunit 3; structure genomics, SF3a120, SF3a60, SURP domain, Structural Genomics, NPPSFA, National Project on Protein Structural and Functional; NMR {Homo sapiens}
</pre><pre>Probab=42.19  E-value=72  Score=17.47  Aligned_cols=15  Identities=27%  Similarity=0.565  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhhHH
</pre><pre>Q EHC5102581.1/7    5 IWKEFKTDLKKVTGY   19 (61)
</pre><pre>Q Consensus         5 iyk~f~~~~k~i~gY   19 (61)
</pre><pre>                      +|.+|-..|+.|+.|
</pre><pre>T Consensus        11 ~~~~FY~rlk~Ikd~   25 (38)
</pre><pre>T 2DT7_A           11 EFAEFYNRLKQIKEF   25 (38)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>PF03215.17 ; Rad17 ; Rad17 P-loop domain
</pre><pre>Probab=41.34  E-value=1.5e+02  Score=17.93  Aligned_cols=51  Identities=8%  Similarity=0.049  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   11 TDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        11 ~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      .....+............+...++=||.|+.|-.+.....+.+.+..+.+.
</pre><pre>T Consensus       104 ~~~~~~~~~~~~~~~~~~~~~~llllDE~~~~ld~~~~~~~~~~l~~~~~~  154 (180)
</pre><pre>T RAD24_YEAST/58  104 PQMESFSEFLKGARYLVMSNLSLILIEDLPNVFHIDTRRRFQQLILQWLYS  154 (180)
</pre><pre>T ss_pred             hhhhhHHHHHHhHHhHHhcCCcEEEEeCCcccCCHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>3TIF_A Uncharacterized ABC transporter ATP-binding protein MJ0796; Nucleotide-binding domain, ABC transporter ATPase, ATP binding, RNA BINDING PROTEIN; HET: IPA, ADP, PI; 1.7995A {Methanocaldococcus jannaschii DSM 2661} SCOP: c.37.1.12
</pre><pre>Probab=40.70  E-value=1.1e+02  Score=18.37  Aligned_cols=32  Identities=13%  Similarity=0.184  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++-+|.|+.+-||.....+.+.+..+...
</pre><pre>T Consensus       163 ~~~~~iid~~~~~l~~~~~~~~~~~l~~~~~~  194 (235)
</pre><pre>T 3TIF_A          163 NPPIILADQPTWALDSKTGEKIMQLLKKLNEE  194 (235)
</pre><pre>T ss_dssp             CCSEEEEESTTTTSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEeCccccCCHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>1GHC_A GH1; CHROMOSOMAL PROTEIN; NMR {Gallus gallus} SCOP: a.4.5.13
</pre><pre>Probab=39.83  E-value=62  Score=17.14  Aligned_cols=15  Identities=13%  Similarity=0.375  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             ChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   44 HPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        44 HP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ||.|.+.|.+.|..+
</pre><pre>T Consensus         3 ~~~~~~mv~~ai~~~   17 (75)
</pre><pre>T 1GHC_A            3 GPSVTELITKAVSAS   17 (75)
</pre><pre>T ss_dssp             SSSHCCCCCHHHHHC
</pre><pre>T ss_pred             CCCHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>3QF7_B Rad50; ABC-ATPase, ATPase, Mre11, HYDROLASE; HET: ANP; 1.9A {Thermotoga maritima}
</pre><pre>Probab=39.66  E-value=1.3e+02  Score=19.09  Aligned_cols=33  Identities=9%  Similarity=0.162  Sum_probs=0.0  Template_Neff=13.400
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   29 DMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        29 d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +...++=+|.|.-+-||....-+.+.+.++.++
</pre><pre>T Consensus       302 ~~~~illiDEpe~~Lhp~~q~~l~~~l~~~~~~  334 (365)
</pre><pre>T 3QF7_B          302 GRLDAFFIDEGFSSLDTENKEKIASVLKELERL  334 (365)
</pre><pre>T ss_dssp             TTCCEEEEESCCTTSCHHHHHHHHHHHGGGTTS
</pre><pre>T ss_pred             CCCCeEEEecCCcCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>PF16205.7 ; Ribosomal_S17_N ; Ribosomal_S17 N-terminal
</pre><pre>Probab=38.50  E-value=45  Score=20.09  Aligned_cols=12  Identities=50%  Similarity=0.830  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             ccccCCCCcCCC
</pre><pre>Q EHC5102581.1/7   32 NLIDLDKPFHGN   43 (61)
</pre><pre>Q Consensus        32 NLidLdkPFHGN   43 (61)
</pre><pre>                      ..||-.-||||+
</pre><pre>T Consensus        44 ~~iDkkCPf~g~   55 (63)
</pre><pre>T D6WR02_TRICA/4   44 SYIDKKCPFTGN   55 (63)
</pre><pre>T ss_pred             cccCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>3HCN_A Ferrochelatase, mitochondrial; ferrochelatase, metal selectivity, Disease mutation, Heme biosynthesis, Iron, Iron-sulfur, Lyase, Membrane, Metal-binding, Mitochondrion, Mitochondrion inner membrane; HET: HEM, IMD, SO4, GOL, CHD; 1.6A {Homo sapiens} SCOP: c.92.1.1
</pre><pre>Probab=37.73  E-value=1e+02  Score=22.61  Aligned_cols=23  Identities=13%  Similarity=0.079  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   38 KPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        38 kPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      .|| |.||.|.+++.++|.+....
</pre><pre>T Consensus       161 ~~~-~~~~~~~~~~~~~i~~~~~~  183 (359)
</pre><pre>T 3HCN_A          161 DRW-PTHHLLIQCFADHILKELDH  183 (359)
</pre><pre>T ss_dssp             CCC-TTCHHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             CCc-ccCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>d1h2vc1 a.118.1.14 (C:29-290) CBP80, 80KDa nuclear cap-binding protein {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=37.63  E-value=2.9e+02  Score=20.13  Aligned_cols=53  Identities=21%  Similarity=0.328  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ..+...+..+..|++.+........+...=+.|++.  ..+-+.+.++|+.|.++
</pre><pre>T Consensus       162 ~~l~~il~~i~~~~~~r~~~~~~~l~~~~~~~~~~~--~d~l~~l~~~l~~l~~~  214 (262)
</pre><pre>T d1h2vc1         162 AEMDRIFANTESYLKRRQKTHVPMLQVWTADKPHPQ--EEYLDCLWAQIQKLKKD  214 (262)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTCCCTTHHHHCSBCCCSSCCC--CBHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhccCCCcceeeecCCCCCCc--ccHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>1JI0_A ABC transporter; ATP binding protein, structural genomics, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, TRANSPORT; HET: ATP, MSE; 2.0A {Thermotoga maritima} SCOP: c.37.1.12
</pre><pre>Probab=37.40  E-value=1.3e+02  Score=20.06  Aligned_cols=26  Identities=27%  Similarity=0.413  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ||.||+|-.+...+.+.+.+.++.+.
</pre><pre>T Consensus       163 ldep~~~ld~~~~~~~~~~l~~~~~~  188 (240)
</pre><pre>T 1JI0_A          163 MDEPSLGLAPILVSEVFEVIQKINQE  188 (240)
</pre><pre>T ss_dssp             EECTTTTCCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             eeCcCCCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>7ARL_D Lipoprotein-releasing system ATP-binding protein LolD; LolCDE, lipoprotein, lipoprotein transporter, lipoprotein sorting and transport, PROTEIN TRANSPORT, ABC transporter; HET: PLM, ADP, Z41; 3.2A {Escherichia coli K-12}
</pre><pre>Probab=36.73  E-value=1.6e+02  Score=18.61  Aligned_cols=31  Identities=13%  Similarity=0.169  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...++-+|.|+.|-+|....-+.+.+.++.+
</pre><pre>T Consensus       163 ~~~ilild~~~~~l~~~~~~~~~~~l~~~~~  193 (241)
</pre><pre>T 7ARL_D          163 NPRLVLADEPTGNLDARNADSIFQLLGELNR  193 (241)
</pre><pre>T ss_dssp             CCSEEEECCSSSSSCSSSHHHHHHHHTTSTT
</pre><pre>T ss_pred             CCCEEEEcCCCCCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>d2foka1 a.4.5.12 (A:5-143) Restriction endonuclease FokI, N-terminal (recognition) domain {Flavobacterium okeanokoites [TaxId: 244]}
</pre><pre>Probab=36.10  E-value=23  Score=26.10  Aligned_cols=40  Identities=25%  Similarity=0.287  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   18 GYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        18 gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      |+|||.|    |-.+|+..-.=|--|-+-|..-+...|..|+..
</pre><pre>T Consensus         5 GWvQnps----~f~~Lk~Vv~iF~~~S~~~~~L~~~~i~~li~~   44 (139)
</pre><pre>T d2foka1           5 GWVQNPG----KFENLKRVVQVFDRNSKVHNEVKNIKIPTLVKE   44 (139)
</pre><pre>T ss_dssp             SCCCSCC----CHHHHHHHHTTSCTTSHHHHHCCCCCHHHHCCS
</pre><pre>T ss_pred             cccCCcc----cHHHHHHHHhhccCCCHHHHHHHhcchhHHhhh
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>TIGR02673 FtsE; cell division ATP-binding protein FtsE. This model describes FtsE, a member of the ABC transporter ATP-binding protein family.
</pre><pre>Probab=36.01  E-value=1.7e+02  Score=18.00  Aligned_cols=32  Identities=19%  Similarity=0.286  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+.+-||.....+.+.+.++.+.
</pre><pre>T Consensus       155 ~~~iiilDe~~~~l~~~~~~~l~~~l~~~~~~  186 (214)
</pre><pre>T TIGR02673       155 SPPLLLADEPTGNLDPDLSERILDLLKRLNKR  186 (214)
</pre><pre>T ss_pred             CCCEEEecCcccCCCHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>1USS_A HISTONE H1; DNA BINDING PROTEIN, LINKER HISTONE, DNA BINDING DOMAIN; NMR {SACCHAROMYCES CEREVISIAE} SCOP: a.4.5.13
</pre><pre>Probab=35.76  E-value=81  Score=17.26  Aligned_cols=16  Identities=13%  Similarity=0.387  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      .||.|.+.|.+.|..+
</pre><pre>T Consensus         6 ~~~~~~~mi~~ai~~~   21 (88)
</pre><pre>T 1USS_A            6 SSLTYKEMILKSMPQL   21 (88)
</pre><pre>T ss_dssp             SSCCHHHHHHHHHHHS
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>2PWW_A Uncharacterized protein; structural genomics, unknown function, PSI-2, Protein Structure Initiative, New York SGX Research Center for Structural Genomics; HET: MSE; 1.82A {Bacillus clausii} SCOP: l.1.1.1, d.129.11.1
</pre><pre>Probab=34.13  E-value=1.4e+02  Score=20.71  Aligned_cols=22  Identities=14%  Similarity=0.312  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhhhHHHHh
</pre><pre>Q EHC5102581.1/7    1 IPQEIWKEFKTDLKKVTGYVQN   22 (61)
</pre><pre>Q Consensus         1 IPn~iyk~f~~~~k~i~gYvQn   22 (61)
</pre><pre>                      ||..|-+.-...|..|..|++.
</pre><pre>T Consensus        96 iP~~i~~aa~~kL~el~~~L~~  117 (127)
</pre><pre>T 2PWW_A           96 FPQAVIDKCNNKLALLAKTIKA  117 (127)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHHHHHHHHC-
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>4U00_A Amino acid ABC transporter, ATP-binding protein; ABC amino acid transporter, Nucleotide binding domain, ATP binding protein, TRANSPORT PROTEIN; HET: ADP; 2.1A {Thermus thermophilus HB8} SCOP: c.37.1.0
</pre><pre>Probab=34.12  E-value=1.6e+02  Score=19.96  Aligned_cols=32  Identities=19%  Similarity=0.307  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=||.|+.|-.+...+.+.+.|.++.+.
</pre><pre>T Consensus       156 ~~~~lilDEpt~~LD~~~~~~~~~~l~~~~~~  187 (244)
</pre><pre>T 4U00_A          156 EPKIMLFDEPTSALDPEMVGEVLDVMRDLAQG  187 (244)
</pre><pre>T ss_dssp             CCSEEEEECTTTTSCHHHHHHHHHHHHHHHTS
</pre><pre>T ss_pred             CCcEEEEeCCCcCCChHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>2WB3_A HYALURONIDASE-PHAGE ASSOCIATED; LYASE, HYALURONAN LYASE, PHAGE TAIL FIBRE, TRIPLE-STRANDED BETA- HELIX, HYALURONIDASE, SCARLET FEVER; 1.9A {STREPTOCOCCUS PYOGENES}
</pre><pre>Probab=34.10  E-value=94  Score=24.12  Aligned_cols=25  Identities=40%  Similarity=0.678  Sum_probs=0.0  Template_Neff=2.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      +..|--+-|.+-.+||.++|.+|+|
</pre><pre>T Consensus       128 v~dpt~~~ha~tk~yvd~~~~~lk~  152 (157)
</pre><pre>T 2WB3_A          128 VKDPTSGKHAATKDYVDEKIAELKK  152 (157)
</pre><pre>T ss_dssp             CCCCCSTTSCCCHHHHHHHHHHHHH
</pre><pre>T ss_pred             EeCCCCCCccccHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>2PCJ_B Lipoprotein-releasing system ATP-binding protein lolD; ABC TRANSPORTER, ATP-BINDING PROTEIN, Structural Genomics, NPPSFA, National Project on Protein Structural and Functional Analyses, RIKEN; 1.7A {Aquifex aeolicus} SCOP: c.37.1.0
</pre><pre>Probab=33.49  E-value=1.7e+02  Score=17.45  Aligned_cols=31  Identities=16%  Similarity=0.163  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             cccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   31 TNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        31 tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ..++-+|.|+.+-|+.....+.+.+.++.++
</pre><pre>T Consensus       159 ~~~~~id~p~~~l~~~~~~~~~~~l~~~~~~  189 (224)
</pre><pre>T 2PCJ_B          159 PILLFADEPTGNLDSANTKRVMDIFLKINEG  189 (224)
</pre><pre>T ss_dssp             CSEEEEESTTTTSCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CeEEEEeCCCCCCCHHHHHHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>6IC4_I ABC transporter ATP-binding protein; Membrane protein Antibiotic resistance ABC transporter, protein transport; 8.7A {Acinetobacter baumannii}
</pre><pre>Probab=33.37  E-value=2e+02  Score=17.49  Aligned_cols=31  Identities=29%  Similarity=0.515  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...++=+|.|+.+-||....-+.+.+..+..
</pre><pre>T Consensus       149 ~~~~l~ide~~~~l~~~~~~~~~~~l~~~~~  179 (233)
</pre><pre>T 6IC4_I          149 DPDLIMYDEPFAGQDPIVKGVLTRLIRSLRE  179 (233)
</pre><pre>T ss_dssp             CCSEEEEECTTTTSCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEcCCcCCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF00538.21 ; Linker_histone ; linker histone H1 and H5 family
</pre><pre>Probab=32.92  E-value=1.1e+02  Score=15.50  Aligned_cols=15  Identities=53%  Similarity=1.041  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             ChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   44 HPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        44 HP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ||.|.+.|.+.+..+
</pre><pre>T Consensus         1 ~~~~~~~i~~ai~~~   15 (75)
</pre><pre>T B7P0P6_IXOSC/3    1 HPKYTEMVKESIESL   15 (75)
</pre><pre>T ss_pred             CCcHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>2LSO_A Histone H1x; Structural Genomics, NORTHEAST STRUCTURAL GENOMICS CONSORTIUM, NESG, PSI-Biology, Protein Structure Initiative, NUCLEAR PROTEIN; NMR {Homo sapiens}
</pre><pre>Probab=32.73  E-value=1.3e+02  Score=16.30  Aligned_cols=18  Identities=44%  Similarity=0.492  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   41 HGNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        41 HGNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      +-.||.|.+.|.+.+..+
</pre><pre>T Consensus         2 ~~~~~~~~~~i~~ai~~~   19 (83)
</pre><pre>T 2LSO_A            2 HMQPGKYSQLVVETIRRL   19 (83)
</pre><pre>T ss_dssp             CCCSCHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCCCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>d5x7ka_ c.37.1.0 (A:) automated matches {Serratia marcescens [TaxId: 615]}
</pre><pre>Probab=32.23  E-value=1.6e+02  Score=19.87  Aligned_cols=26  Identities=31%  Similarity=0.261  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ||.|++|-++.....+.+.+.++...
</pre><pre>T Consensus       171 ldep~~~l~~~~~~~~~~~l~~~~~~  196 (243)
</pre><pre>T d5x7ka_         171 LDEPNSNLDDAGEKALNQAIMFLKQR  196 (243)
</pre><pre>T ss_dssp             EESTTTTCCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             EeCCCCCCCHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>1UHM_A Histone H1; winged helix-turn-helix, linker histone, S. cerevisiae, RIKEN Structural Genomics/Proteomics Initiative, RSGI, Structural Genomics, STRUCTURAL PROTEIN; NMR {Saccharomyces cerevisiae} SCOP: a.4.5.13
</pre><pre>Probab=32.21  E-value=1.1e+02  Score=16.15  Aligned_cols=15  Identities=20%  Similarity=0.463  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   44 HPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        44 HP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ||.|.+.|.++|..+
</pre><pre>T Consensus         4 ~~~~~~~i~~ai~~~   18 (78)
</pre><pre>T 1UHM_A            4 SKSYRELIIEGLTAL   18 (78)
</pre><pre>T ss_dssp             CCCHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>TIGR00972 3a0107s01c2; phosphate ABC transporter, ATP-binding protein. This model represents the ATP-binding protein of a family of ABC transporters for inorganic phosphate.
</pre><pre>Probab=32.15  E-value=1.6e+02  Score=18.87  Aligned_cols=32  Identities=25%  Similarity=0.392  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+.+-||....-+.+.+..+.++
</pre><pre>T Consensus       162 ~~~iliidep~~~l~~~~~~~l~~~l~~~~~~  193 (247)
</pre><pre>T TIGR00972       162 EPEVLLLDEPTSALDPIATGKIEELIQELKKK  193 (247)
</pre><pre>T ss_pred             CCcEEEEeChhhcCCHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>d2pcja_ c.37.1.0 (A:) automated matches {Aquifex aeolicus [TaxId: 224324]}
</pre><pre>Probab=32.04  E-value=1.7e+02  Score=19.19  Aligned_cols=26  Identities=15%  Similarity=0.199  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ||.||.|-.+.....+.+.+.++.++
</pre><pre>T Consensus       163 ldep~~~ld~~~~~~~~~~l~~~~~~  188 (223)
</pre><pre>T d2pcja_         163 ADEPTGNLDSANTKRVMDIFLKINEG  188 (223)
</pre><pre>T ss_dssp             EESTTTTCCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             EeCCCCCCCHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>4ZRX_A F5/8 type C domain protein; sugar binding protein, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY, HYDROLASE; HET: MSE, EDO; 1.59A {Bacteroides ovatus}
</pre><pre>Probab=31.88  E-value=1.3e+02  Score=22.88  Aligned_cols=27  Identities=22%  Similarity=0.503  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             cCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   35 DLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        35 dLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      |...|..++++.|.+|+.++|++|+.+
</pre><pre>T Consensus       149 ~~~~~~~~~~~~~~~~~~~~~~el~~~  175 (586)
</pre><pre>T 4ZRX_A          149 DRNAECYGDSPRYNDFFIRQLTELLTN  175 (586)
</pre><pre>T ss_dssp             CSSCTTTTSHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             hcCchhhCCCHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>3ZGX_B CHROMOSOME PARTITION PROTEIN SMC; CELL CYCLE; 3.4A {BACILLUS SUBTILIS}
</pre><pre>Probab=31.62  E-value=2.8e+02  Score=18.29  Aligned_cols=48  Identities=8%  Similarity=-0.020  Sum_probs=0.0  Template_Neff=13.300
</pre><pre>
</pre><pre>Q ss_pred             HhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   14 KKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        14 k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +.+-...............++=+|.|+-+-||....-+-+.+.++.++
</pre><pre>T Consensus       334 ~~~~~l~~~l~~~~~~~~~~l~iDepe~~l~~~~~~~~~~~l~~~~~~  381 (426)
</pre><pre>T 3ZGX_B          334 RALTAIALLFSILKVRPVPFCVLDEVEAALDEANVFRFAQYLKKYSSD  381 (426)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHSCCSCEEEESTTTTSCTHHHHHHHHHHHHCSSS
</pre><pre>T ss_pred             HHHHHHHHHHHHhccCCCCEEEEecccccCCHHHHHHHHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>6XOR_A Protein swallow; coiled coil, self-association domain, RNA BINDING PROTEIN; NMR {Drosophila melanogaster}
</pre><pre>Probab=31.44  E-value=1e+02  Score=21.01  Aligned_cols=12  Identities=25%  Similarity=0.808  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   46 AYSQYVREAIED   57 (61)
</pre><pre>Q Consensus        46 ~Yn~YV~~~i~k   57 (61)
</pre><pre>                      +||+|+++++.+
</pre><pre>T Consensus        28 ~yndYLQ~kLde   39 (71)
</pre><pre>T 6XOR_A           28 GFNEYVQERLDR   39 (71)
</pre><pre>T ss_dssp             HHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>PF04690.15 ; YABBY ; YABBY protein
</pre><pre>Probab=31.15  E-value=1.1e+02  Score=21.86  Aligned_cols=16  Identities=31%  Similarity=0.841  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   46 AYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        46 ~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +||.|+++++..+++.
</pre><pre>T Consensus       122 aYn~Fik~~~~rik~~  137 (164)
</pre><pre>T I1J7S5_SOYBN/1  122 AYNRFIKDEIQRIKAG  137 (164)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>PF14883.8 ; GHL13 ; Hypothetical glycosyl hydrolase family 13
</pre><pre>Probab=30.99  E-value=1.4e+02  Score=19.80  Aligned_cols=19  Identities=21%  Similarity=0.146  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +||.+.+|+.+.+.+++++
</pre><pre>T Consensus       121 ~~p~~~~~~~~~~~~~~~~  139 (322)
</pre><pre>T Q3A0T1_PELCD/2  121 FVPEVRALVHGIYGDLARY  139 (322)
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>1JZS_A Isoleucyl-tRNA synthetase; aminoacyl-tRNA synthetase, RIKEN Structural Genomics/Proteomics Initiative, RSGI, Structural Genomics, LIGASE; HET: MRC; 2.5A {Thermus thermophilus} SCOP: b.51.1.1, a.27.1.1, c.26.1.1
</pre><pre>Probab=30.79  E-value=3.6e+02  Score=22.05  Aligned_cols=55  Identities=13%  Similarity=0.266  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhH-HHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTG-YVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~g-YvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      .+|.+..++... |++..........-.+|.+..+.-.+|.|...|...+.+|.++
</pre><pre>T Consensus       113 ~~~~~~~~~~~~~~~~~~~~~~~~lg~~~d~~~~~~t~~~~~~~~~~~~f~~L~~~  168 (821)
</pre><pre>T 1JZS_A          113 ERFNQACRESVFTYEKEWEAFTERIAYWVDLEDAYATLEPTYIESIWWSLKNLFDR  168 (821)
</pre><pre>T ss_dssp             HHHHHHHHHHTTTTHHHHHHHHHHTTCCCCCTTCEETTSHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHhHHHHHHHHHHHhcccCCCCccccCCHHHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>6UK1_A Cystic fibrosis transmembrane conductance regulator; NBD2, CFTR, ABC transport, HYDROLASE; HET: ATP; 2.693A {Homo sapiens}
</pre><pre>Probab=30.62  E-value=1.6e+02  Score=17.96  Aligned_cols=28  Identities=25%  Similarity=0.347  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++=+|.|+-+-||....-+.+.|..+..
</pre><pre>T Consensus       165 lllldep~~~l~~~~~~~~~~~l~~~~~  192 (229)
</pre><pre>T 6UK1_A          165 ILLLDEPSAHLDPVTYQIIRRTLKQAFA  192 (229)
</pre><pre>T ss_dssp             EEEEESHHHHSCHHHHHHHHHHTTTTTT
</pre><pre>T ss_pred             EEEecCcccCCCHHHHHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>d2r5xa1 d.129.11.0 (A:2-119) automated matches {Geobacillus kaustophilus [TaxId: 235909]}
</pre><pre>Probab=30.51  E-value=1.7e+02  Score=20.39  Aligned_cols=21  Identities=14%  Similarity=0.305  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhhhHHHH
</pre><pre>Q EHC5102581.1/7    1 IPQEIWKEFKTDLKKVTGYVQ   21 (61)
</pre><pre>Q Consensus         1 IPn~iyk~f~~~~k~i~gYvQ   21 (61)
</pre><pre>                      ||.++.+.|...|..|..|+.
</pre><pre>T Consensus        94 ~p~~~~~q~~~~l~ev~~~L~  114 (118)
</pre><pre>T d2r5xa1          94 FPSSLVSQCQNVLAQVKSELE  114 (118)
</pre><pre>T ss_dssp             EEHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>PF15388.8 ; FAM117 ; Protein Family FAM117
</pre><pre>Probab=30.49  E-value=1.7e+02  Score=23.52  Aligned_cols=35  Identities=17%  Similarity=0.082  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhhhHHHHhhhcccccccccccCCCCcCCCC
</pre><pre>Q EHC5102581.1/7   10 KTDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNH   44 (61)
</pre><pre>Q Consensus        10 ~~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNH   44 (61)
</pre><pre>                      .+...+|+.-.|.++++......-.+=-.|+||+|
</pre><pre>T Consensus        69 l~ei~klrqqLQ~rs~~~~~~~~~~~r~sP~~g~h  103 (310)
</pre><pre>T M4AV68_XIPMA/5   69 LREIAKLKQQLQQGSKPAASGVHDKDRQCCYPHGS  103 (310)
</pre><pre>T ss_pred             HHHHHHHHHHHHhCCCCCCCCccccccccCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>PF16707.7 ; CagS ; Cag pathogenicity island protein S of Helicobacter pylori
</pre><pre>Probab=30.04  E-value=80  Score=25.07  Aligned_cols=12  Identities=33%  Similarity=0.780  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   46 AYSQYVREAIED   57 (61)
</pre><pre>Q Consensus        46 ~Yn~YV~~~i~k   57 (61)
</pre><pre>                      +||+||+++|++
</pre><pre>T Consensus       173 qyn~~~~~~~~k  184 (196)
</pre><pre>T CAGS_HELPY/1-1  173 QYNDYVRDLLMK  184 (196)
</pre><pre>T ss_pred             HhhHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>TIGR02769 nickel_nikE; nickel import ATP-binding protein NikE. This family represents the NikE subunit of a multisubunit nickel import ABC transporter complex.
</pre><pre>Probab=29.57  E-value=1.6e+02  Score=19.39  Aligned_cols=28  Identities=21%  Similarity=0.262  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++-+|.|+.+.||.-..-+.+.+..+.+
</pre><pre>T Consensus       171 iiildep~~~l~~~~~~~l~~~l~~~~~  198 (265)
</pre><pre>T TIGR02769       171 LIVLDEAVSNLDMVLQAVILELLRKLQQ  198 (265)
</pre><pre>T ss_pred             EEEEechhhCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>6QJ0_A Structural maintenance of chromosomes protein,Structural maintenance of chromosomes protein; condensin SMC complex ATPase chromosome condensation loop extrusion, CELL CYCLE; 2.0A {Chaetomium thermophilum}
</pre><pre>Probab=29.47  E-value=1.5e+02  Score=21.47  Aligned_cols=33  Identities=12%  Similarity=-0.026  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   29 DMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        29 d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ....++-||.||.+-.+.+...+.+.|..+.++
</pre><pre>T Consensus       340 ~~~~~l~lDep~~~ld~~~~~~~~~~l~~~~~~  372 (412)
</pre><pre>T 6QJ0_A          340 KPAPMYILDEVDAALDLSHTQNIGRLIKTRFKG  372 (412)
</pre><pre>T ss_dssp             SCCSEEEEESCCTTCCHHHHHHHHHHHHHHCTT
</pre><pre>T ss_pred             CCCCEEEEEcccccCCHHHHHHHHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>TIGR03411 urea_trans_UrtD; urea ABC transporter, ATP-binding protein UrtD. Members of this protein family are ABC transporter ATP-binding subunits associated with urea transport and metabolism.
</pre><pre>Probab=29.22  E-value=2.2e+02  Score=18.09  Aligned_cols=32  Identities=22%  Similarity=0.319  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++-+|.|+.+-||.-..-+.+.+.++..+
</pre><pre>T Consensus       161 ~~~iiilde~~~~l~~~~~~~l~~~l~~~~~~  192 (242)
</pre><pre>T TIGR03411       161 DPKLLLLDEPVAGMTDEETEKTAELLKSLAGK  192 (242)
</pre><pre>T ss_pred             CCcEEEEcCcccCCCHHHHHHHHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>4P7O_B Poly-beta-1,6-N-acetyl-D-glucosamine N-deacetylase; beta alpha barrel, carbohydrate binding, glycosyl hydrolase fold, HYDROLASE; 1.48A {Escherichia coli}
</pre><pre>Probab=29.14  E-value=1.6e+02  Score=21.01  Aligned_cols=19  Identities=21%  Similarity=0.088  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +||.+.+|+.+.+++|.++
</pre><pre>T Consensus       133 ~~p~~~~~~~~~i~el~~~  151 (367)
</pre><pre>T 4P7O_B          133 FDDRVRAQVGMLYEDLAGH  151 (367)
</pre><pre>T ss_dssp             TCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>2G3V_B CAG pathogenicity island protein 13; Helicobacter pylori, pathogenicity island, Type IV secretion system, UNKNOWN FUNCTION; HET: MSE; 2.3A {Helicobacter pylori}
</pre><pre>Probab=28.95  E-value=84  Score=25.15  Aligned_cols=12  Identities=33%  Similarity=0.780  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   46 AYSQYVREAIED   57 (61)
</pre><pre>Q Consensus        46 ~Yn~YV~~~i~k   57 (61)
</pre><pre>                      +||+||+++|++
</pre><pre>T Consensus       174 qyn~~~~~~~~k  185 (208)
</pre><pre>T 2G3V_B          174 QYNDYVRDLLMK  185 (208)
</pre><pre>T ss_dssp             HHHHHHHTC---
</pre><pre>T ss_pred             HhhHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>2G3V_D CAG pathogenicity island protein 13; Helicobacter pylori, pathogenicity island, Type IV secretion system, UNKNOWN FUNCTION; HET: MSE; 2.3A {Helicobacter pylori}
</pre><pre>Probab=28.95  E-value=84  Score=25.15  Aligned_cols=12  Identities=33%  Similarity=0.780  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   46 AYSQYVREAIED   57 (61)
</pre><pre>Q Consensus        46 ~Yn~YV~~~i~k   57 (61)
</pre><pre>                      +||+||+++|++
</pre><pre>T Consensus       174 qyn~~~~~~~~k  185 (208)
</pre><pre>T 2G3V_D          174 QYNDYVRDLLMK  185 (208)
</pre><pre>T ss_dssp             HHHHHHHTC---
</pre><pre>T ss_pred             HhhHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>d1t0fa1 a.4.5.27 (A:169-268) TnsA endonuclease, C-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=28.94  E-value=65  Score=19.40  Aligned_cols=12  Identities=25%  Similarity=0.451  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCcCCCCh
</pre><pre>Q EHC5102581.1/7   34 IDLDKPFHGNHP   45 (61)
</pre><pre>Q Consensus        34 idLdkPFHGNHP   45 (61)
</pre><pre>                      +||++||+-++|
</pre><pre>T Consensus        75 ~Dl~~~i~~~~~   86 (100)
</pre><pre>T d1t0fa1          75 FNIYKSFRANKC   86 (100)
</pre><pre>T ss_dssp             CCTTSCGGGCBG
</pre><pre>T ss_pred             eeCCCCcccccc
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>6BZS_A Multidrug resistance-associated protein 6; ABCC6, NBD1, Apo state, ATP-Binding Cassette transporter C6, nucleotide binding domain 1, TRANSPORT PROTEIN; 2.29990463032A {Homo sapiens} SCOP: c.37.1.0
</pre><pre>Probab=28.65  E-value=81  Score=21.54  Aligned_cols=31  Identities=16%  Similarity=0.106  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...++=||.||-|-.+....-+.+.|.++..
</pre><pre>T Consensus       149 ~p~llllDEp~~~ld~~~~~~~~~~l~~~~~  179 (238)
</pre><pre>T 6BZS_A          149 KAAVYLLDDPLAALDAHVGQHVFNQVIGPGG  179 (238)
</pre><pre>T ss_dssp             TCSEEEEESTTSSSCHHHHHHHCCCCCSTTS
</pre><pre>T ss_pred             cCCEEEeccHHhcCCHHHHHHHHHHhhccch
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>TIGR02770 nickel_nikD; nickel import ATP-binding protein NikD. This family represents the NikD subunit of a multisubunit nickel import ABC transporter complex.
</pre><pre>Probab=28.41  E-value=2.6e+02  Score=17.67  Aligned_cols=31  Identities=13%  Similarity=0.155  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...++=+|.|+.+-+|.....+.+.+.++.+
</pre><pre>T Consensus       143 ~~~iliiDe~~~~l~~~~~~~l~~~l~~~~~  173 (230)
</pre><pre>T TIGR02770       143 EPPFLIADEPTTDLDVVNQARVLKLLRELRQ  173 (230)
</pre><pre>T ss_pred             CCCEEEEcCcccCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>d3rlfa1 c.37.1.12 (A:2-235) Maltose transport protein MalK, N-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=28.06  E-value=2.1e+02  Score=19.33  Aligned_cols=31  Identities=23%  Similarity=0.280  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...++=||.||.|-.+.-..-+.+.|.++.+
</pre><pre>T Consensus       150 ~p~illlDep~~~ld~~~~~~~~~~L~~~~~  180 (234)
</pre><pre>T d3rlfa1         150 EPSVFLLDEPLSNLDAALRVQMRIEISRLHK  180 (234)
</pre><pre>T ss_dssp             CCSEEEEESTTTTSCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCcEEEEeCCcccCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>3V6B_R Vascular endothelial growth factor receptor 2; Ig-homology domain, VEGFR-2, growth factor receptor, VEGF ligand, ANGIOGENESIS, MEMBRANE, VEGF-E, Orf virus, HORMONE-SIGNALING PROTEIN complex; 3.205A {Orf virus}
</pre><pre>Probab=27.60  E-value=61  Score=21.89  Aligned_cols=9  Identities=22%  Similarity=0.626  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNH   44 (61)
</pre><pre>Q Consensus        36 LdkPFHGNH   44 (61)
</pre><pre>                      |.++|||.|
</pre><pre>T Consensus       415 ~~~~~~~~~  423 (424)
</pre><pre>T 3V6B_R          415 FHRTHHHHH  423 (424)
</pre><pre>T ss_dssp             ---------
</pre><pre>T ss_pred             eeecCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>1G6H_A HIGH-AFFINITY BRANCHED-CHAIN AMINO ACID TRANSPORT ATP-BINDING PROTEIN; beta-core domain, ABC specific-beta-strand domain alpha-helix domain, TRANSPORT PROTEIN; HET: ADP; 1.6A {Methanocaldococcus jannaschii} SCOP: c.37.1.12
</pre><pre>Probab=27.06  E-value=2.7e+02  Score=18.78  Aligned_cols=32  Identities=25%  Similarity=0.456  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++-+|.|+.|-.+....-+.+.+.++...
</pre><pre>T Consensus       171 ~~~~~llDep~~~ld~~~~~~~~~~l~~~~~~  202 (257)
</pre><pre>T 1G6H_A          171 NPKMIVMDEPIAGVAPGLAHDIFNHVLELKAK  202 (257)
</pre><pre>T ss_dssp             CCSEEEEESTTTTCCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CCcEEEeccccCCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>6BZR_A Multidrug resistance-associated protein 6; ABCC6, NBD2, ADP-bound state, ATP-Binding Cassette transporter C6, nucleotide binding domain 2, TRANSPORT PROTEIN; HET: CIT, ADP; 2.79999446301A {Homo sapiens} SCOP: c.37.1.0
</pre><pre>Probab=26.94  E-value=2.4e+02  Score=17.64  Aligned_cols=32  Identities=16%  Similarity=0.175  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+.+-||.....+.+.|..+.+.
</pre><pre>T Consensus       167 ~~~il~idep~~~l~~~~~~~l~~~l~~~~~~  198 (251)
</pre><pre>T 6BZR_A          167 KTQILILDEATAAVDPGTELQMQAMLGSWFAQ  198 (251)
</pre><pre>T ss_dssp             TCSEEEEECCSSCCCHHHHHHHHHHHHHTTTT
</pre><pre>T ss_pred             cCcEEEEecccCCCCHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>1B0U_A HISTIDINE PERMEASE; ABC TRANSPORTER, HISTIDINE PERMEASE, TRANSPORT PROTEIN; HET: ATP; 1.5A {Salmonella typhimurium} SCOP: l.1.1.1, c.37.1.12
</pre><pre>Probab=26.88  E-value=2.7e+02  Score=19.04  Aligned_cols=31  Identities=16%  Similarity=0.294  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             cccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   31 TNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        31 tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ..++=||.|+.|-.+...+.+.+.|.++.+.
</pre><pre>T Consensus       172 ~~~lilDEp~~~Ld~~~~~~~~~~i~~~~~~  202 (262)
</pre><pre>T 1B0U_A          172 PDVLLFDEPTSALDPELVGEVLRIMQQLAEE  202 (262)
</pre><pre>T ss_dssp             CSEEEEESTTTTSCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CCEEEEcCCCCCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>2GHI_C transport protein; Plasmodium yoelii, Multidrug Resistance Protein, MDR, Structural Genomics, Structural Genomics Consortium, SGC, TRANSPORT PROTEIN; HET: SO4; 2.2A {Plasmodium yoelii yoelii str. 17XNL} SCOP: c.37.1.0
</pre><pre>Probab=26.54  E-value=2.2e+02  Score=18.26  Aligned_cols=29  Identities=21%  Similarity=0.335  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ++=+|.|+.+-||....-+.+.+.++..+
</pre><pre>T Consensus       176 ~llidep~~~l~~~~~~~l~~~l~~~~~~  204 (260)
</pre><pre>T 2GHI_C          176 IVIFDEATSSLDSKTEYLFQKAVEDLRKN  204 (260)
</pre><pre>T ss_dssp             EEEEECCCTTCCHHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             EEEeeCcccCCCHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>d1h9aa1 c.2.1.3 (A:1-181,A:413-426) Glucose 6-phosphate dehydrogenase, N-terminal domain {Leuconostoc mesenteroides [TaxId: 1245]}
</pre><pre>Probab=26.17  E-value=4.8e+02  Score=19.14  Aligned_cols=50  Identities=16%  Similarity=0.314  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7    1 IPQEIWKEFKTDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus         1 IPn~iyk~f~~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      +|.++|..       |...+....--.....+=|=++||| |.--+-..=+.+.|.+.
</pre><pre>T Consensus       118 lPP~~f~~-------i~~~L~~~~l~~~~~~~RiviEKPF-G~dl~SA~~Ln~~l~~~  167 (196)
</pre><pre>T d1h9aa1         118 VAPRFFGT-------IAKYLKSEGLLADTGYNRLMIEKPF-GTSYDTAAELQNDLENA  167 (196)
</pre><pre>T ss_dssp             SCGGGHHH-------HHHHHHHTTCSCSSSCEEEEECSCS-CSSHHHHHHHHHHHTTT
</pre><pre>T ss_pred             CCHHHHHH-------HHHHHHHCCCCCCCCCeEEEEeCCC-CCChHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>6QPQ_A Structural maintenance of chromosomes protein,Structural maintenance of chromosomes protein; Cohesin, cell division, genome regulation, sister chromatid cohesion, SMC, kleisin, CELL CYCLE; 2.1A {Chaetomium thermophilum var. thermophilum DSM 1495} SCOP: l.1.1.1, c.37.1.0
</pre><pre>Probab=26.09  E-value=2e+02  Score=19.65  Aligned_cols=30  Identities=13%  Similarity=0.098  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCCcCCCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   29 DMTNLIDLDKPFHGNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        29 d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ....++=+|.|+..-||....-+.+.+.++
</pre><pre>T Consensus       376 ~~~~~l~iDep~~~L~~~~~~~l~~~l~~~  405 (453)
</pre><pre>T 6QPQ_A          376 QPSPFFVLDEVDAALDNANVEKIKKYIREH  405 (453)
</pre><pre>T ss_dssp             SCCSEEEECSGGGGSCHHHHHHHHHHHHTT
</pre><pre>T ss_pred             CCCCEEEEECccccCCHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>7C0J_B Histone H5,Double-stranded RNA-specific adenosine deaminase; GH5, Protein-DNA complex, protein engineering, Z-DNA binding protein, DNA BINDING PROTEIN, DNA BINDING PROTEIN-DNA complex; 2.75A {Gallus gallus}
</pre><pre>Probab=25.93  E-value=1.9e+02  Score=15.34  Aligned_cols=17  Identities=35%  Similarity=0.653  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        42 GNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ..||.|.+.|.+.+..+
</pre><pre>T Consensus         5 ~~~~~~~~mv~~ai~~~   21 (76)
</pre><pre>T 7C0J_B            5 ASHPTYSEMIAAAIRAE   21 (76)
</pre><pre>T ss_dssp             ---CCHHHHHHHHHHHC
</pre><pre>T ss_pred             CCCCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>d2xwsa1 c.92.1.3 (A:1-125) automated matches {Archaeoglobus fulgidus [TaxId: 2234]}
</pre><pre>Probab=25.83  E-value=1.8e+02  Score=17.89  Aligned_cols=24  Identities=8%  Similarity=0.185  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCcCCCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   34 IDLDKPFHGNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        34 idLdkPFHGNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      +.+-.|+ |.||..-+.+.+++.++
</pre><pre>T Consensus       101 v~~~~~l-~~~~~~~~~l~~r~~~~  124 (125)
</pre><pre>T d2xwsa1         101 VVICEPI-GEDYFVTYAILNSVFRI  124 (125)
</pre><pre>T ss_dssp             EEECCCS-TTSHHHHHHHHHHHHCC
</pre><pre>T ss_pred             EEEeCCC-CCCHHHHHHHHHHHhhh
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>d2c3fa1 b.108.1.5 (A:7-336) Phage-associated hyaluronidase, HylP1 {Streptococcus pyogenes [TaxId: 1314]}
</pre><pre>Probab=25.78  E-value=2e+02  Score=24.73  Aligned_cols=37  Identities=30%  Similarity=0.522  Sum_probs=0.0  Template_Neff=2.900
</pre><pre>
</pre><pre>Q ss_pred             hcccccccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   24 TKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        24 skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++-..-......|..|--+-|.+-.+||.++|.+|+|
</pre><pre>T Consensus       290 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  326 (330)
</pre><pre>T d2c3fa1         290 AKETSQIDGNLKLKDPTANDHAATKAYVDKAISELKK  326 (330)
</pre><pre>T ss_dssp             ESSCCCCSSCCCCCCCCSTTSCCCHHHHHHHHHHHHH
</pre><pre>T ss_pred             eeeeeeecCceeecCCCCCcchhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>1H6K_A CBP80; M7G CAP, CAP-BINDING-COMPLEX, RNP DOMAIN, MIF4G DOMAIN, RNA MATURATION, RNA EXPORT, NUCLEAR PROTEIN; 2.0A {HOMO SAPIENS} SCOP: a.118.1.14
</pre><pre>Probab=25.72  E-value=6.3e+02  Score=21.92  Aligned_cols=53  Identities=21%  Similarity=0.342  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ..+...+..|..|++..+......-+...-+.|..  -..|-+++-++|+.|.++
</pre><pre>T Consensus       171 ~~l~~il~~i~~y~~~r~~~~~~ll~~~~~~~~~~--~~~~l~~l~~~i~~l~~~  223 (757)
</pre><pre>T 1H6K_A          171 AEMDRIFANTESYLKRRQKTHVPMLQVWTADKPHP--QEEYLDCLWAQIQKLKKD  223 (757)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCCCTTHHHHCSBCCCSSCC--CEEHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHcCCCCCcceeEeeCCCCCC--chHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>1H6K_B CBP80; M7G CAP, CAP-BINDING-COMPLEX, RNP DOMAIN, MIF4G DOMAIN, RNA MATURATION, RNA EXPORT, NUCLEAR PROTEIN; 2.0A {HOMO SAPIENS} SCOP: a.118.1.14
</pre><pre>Probab=25.72  E-value=6.3e+02  Score=21.92  Aligned_cols=53  Identities=21%  Similarity=0.342  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhHHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTGYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ..+...+..|..|++..+......-+...-+.|..  -..|-+++-++|+.|.++
</pre><pre>T Consensus       171 ~~l~~il~~i~~y~~~r~~~~~~ll~~~~~~~~~~--~~~~l~~l~~~i~~l~~~  223 (757)
</pre><pre>T 1H6K_B          171 AEMDRIFANTESYLKRRQKTHVPMLQVWTADKPHP--QEEYLDCLWAQIQKLKKD  223 (757)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTCCCTTHHHHCSBCCCSSCC--CCBHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHcCCCCCcceeEeeCCCCCC--chHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>2ZU0_D Probable ATP-dependent transporter sufC; Iron-sulfur cluster, ABC-ATPase, ATP-binding, Cytoplasm, Nucleotide-binding, Transport, Biosynthetic Protein-Protein Binding COMPLEX; HET: MES; 2.2A {Escherichia coli}
</pre><pre>Probab=25.65  E-value=2.8e+02  Score=17.81  Aligned_cols=32  Identities=19%  Similarity=0.192  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+.+-||.-..-+.+.+.++.+.
</pre><pre>T Consensus       182 ~~~ililDep~~~l~~~~~~~l~~~l~~~~~~  213 (267)
</pre><pre>T 2ZU0_D          182 EPELCILDESDSGLDIDALKVVADGVNSLRDG  213 (267)
</pre><pre>T ss_dssp             CC------------------CHHHHHHTTCC-
</pre><pre>T ss_pred             CCCEEEEeCCCCCCCHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>1HST_B HISTONE H5; CHROMOSOMAL PROTEIN; 2.6A {Gallus gallus} SCOP: a.4.5.13
</pre><pre>Probab=25.64  E-value=1.8e+02  Score=16.24  Aligned_cols=17  Identities=35%  Similarity=0.653  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        42 GNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ..||.|.+.|.+.|..+
</pre><pre>T Consensus         5 ~~~~~~~~mi~~ai~~~   21 (90)
</pre><pre>T 1HST_B            5 ASHPTYSEMIAAAIRAE   21 (90)
</pre><pre>T ss_dssp             -CCCCHHHHHHHHHHTC
</pre><pre>T ss_pred             CCCCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>6M2A_A Sirohydrochlorin cobaltochelatase,Sirohydrochlorin cobaltochelatase,Sirohydrochlorin cobaltochelatase; Chelatase, BIOSYNTHETIC PROTEIN; HET: SO4; 2.23A {Methanocaldococcus jannaschii (strain ATCC 43067 / DSM 2661 / JAL-1 / JCM 10045 / NBRC 100440)}
</pre><pre>Probab=25.47  E-value=1.5e+02  Score=17.53  Aligned_cols=16  Identities=6%  Similarity=0.046  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CCChhHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYSQYVREAIED   57 (61)
</pre><pre>Q Consensus        42 GNHP~Yn~YV~~~i~k   57 (61)
</pre><pre>                      |.||.+.+.+.++|..
</pre><pre>T Consensus       110 ~~~~~~~~~l~~~i~~  125 (126)
</pre><pre>T 6M2A_A          110 GADDRIVDIIIDRAFG  125 (126)
</pre><pre>T ss_dssp             CSCHHHHHHHHHHHTT
</pre><pre>T ss_pred             CCChHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>PF07212.13 ; Hyaluronidase_1 ; Hyaluronidase protein (HylP)
</pre><pre>Probab=25.44  E-value=2.2e+02  Score=23.78  Aligned_cols=37  Identities=30%  Similarity=0.522  Sum_probs=0.0  Template_Neff=3.500
</pre><pre>
</pre><pre>Q ss_pred             hcccccccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   24 TKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        24 skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++...-....+.|..|--+-|.+-.+||.++|.+|++
</pre><pre>T Consensus       232 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  268 (273)
</pre><pre>T A0A4D6B8Z8_9CA  232 AKETSQIDGNLKLKDPTANDHAATKAYVDKAISELKK  268 (273)
</pre><pre>T ss_pred             EcccceecCceeecCCCCCcchhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>PF13175.8 ; AAA_15 ; AAA ATPase domain
</pre><pre>Probab=25.41  E-value=1.6e+02  Score=21.00  Aligned_cols=34  Identities=21%  Similarity=0.297  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ccccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   28 ADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        28 ~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      .+...++=||.|+-|-||....-+.+.|.++.+.
</pre><pre>T Consensus       357 ~~~~~iliiDEp~~~Ld~~~~~~l~~~L~~~~~~  390 (409)
</pre><pre>T Q5JGZ5_THEKO/1  357 QKLTLLLVIEELELYQHPQRQKHLHKLFKDLAGN  390 (409)
</pre><pre>T ss_pred             cccceEEEEechhHhcCHHHHHHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>TIGR00105 L31; ribosomal protein L31. This family consists exclusively of bacterial (and organellar) 50S ribosomal protein L31.
</pre><pre>Probab=25.16  E-value=50  Score=20.34  Aligned_cols=7  Identities=43%  Similarity=0.842  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             CCChhHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYS   48 (61)
</pre><pre>Q Consensus        42 GNHP~Yn   48 (61)
</pre><pre>                      +|||.|+
</pre><pre>T Consensus        40 ~~HP~yt   46 (68)
</pre><pre>T TIGR00105        40 KCHPFYT   46 (68)
</pre><pre>T ss_pred             CCCcccC
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>6AU1_B Putative hemin storage protein; deacetylase, glycoside hydrolase, HYDROLASE, PNAG, biofilm; HET: SO4, EDO; 1.76A {Bordetella bronchiseptica}
</pre><pre>Probab=25.06  E-value=1.9e+02  Score=20.60  Aligned_cols=19  Identities=32%  Similarity=0.429  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +||.+.+|+.+.+++++++
</pre><pre>T Consensus       133 ~~p~~~~~~~~~~~el~~~  151 (357)
</pre><pre>T 6AU1_B          133 FHPDAHRVISEIYEDLAKA  151 (357)
</pre><pre>T ss_dssp             TSHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>5B57_C Hemin import ATP-binding protein HmuV; metal-binding, membrane protein, METAL TRANSPORT; HET: SO4, DMU; 2.8A {Burkholderia cenocepacia J2315}
</pre><pre>Probab=24.89  E-value=3.6e+02  Score=17.68  Aligned_cols=34  Identities=12%  Similarity=0.124  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             cccccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   27 AADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        27 a~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      +.....++=+|.|+-+-||....-+.+.|.++.+
</pre><pre>T Consensus       170 ~~~~~~llliDep~~~L~~~~~~~l~~~l~~~~~  203 (273)
</pre><pre>T 5B57_C          170 TEPGPRYLLLDEPTAALDLAHQHRLLDTVRAVAR  203 (273)
</pre><pre>T ss_dssp             ----CCEEEEESCSTTSCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCCCeEEEEcCCCCCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>1P4W_A rcsB; RcsB protein, solution structure, DNA binding domain, DNA BINDING PROTEIN; NMR {Erwinia amylovora} SCOP: a.4.6.2
</pre><pre>Probab=24.89  E-value=71  Score=17.88  Aligned_cols=6  Identities=33%  Similarity=0.589  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             cCCCCh
</pre><pre>Q EHC5102581.1/7   40 FHGNHP   45 (61)
</pre><pre>Q Consensus        40 FHGNHP   45 (61)
</pre><pre>                      |||+|+
</pre><pre>T Consensus         5 ~~~~~~   10 (99)
</pre><pre>T 1P4W_A            5 HHHHHH   10 (99)
</pre><pre>T ss_dssp             ------
</pre><pre>T ss_pred             cccccc
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>4ZQA_A Sin3 histone deacetylase corepressor complex component SDS3; transcription repression, histone deacetylase complex, coiled-coil, corepressor complex, TRANSCRIPTION REPRESSOR; 1.65A {Mus musculus}
</pre><pre>Probab=24.86  E-value=1.3e+02  Score=18.68  Aligned_cols=12  Identities=17%  Similarity=0.506  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CCChhHHHHHHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYSQYVRE   53 (61)
</pre><pre>Q Consensus        42 GNHP~Yn~YV~~   53 (61)
</pre><pre>                      |.||.|..-+.+
</pre><pre>T Consensus         4 Gthpey~~~l~~   15 (86)
</pre><pre>T 4ZQA_A            4 GTLQEYQKRMKK   15 (86)
</pre><pre>T ss_dssp             HHHHHHHHHHHH
</pre><pre>T ss_pred             cchHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>2WXU_A PHOSPHOLIPASE C; CYTOLYSIS, HYDROLASE, HEMOLYSIS, MEMBRANE BINDING, VIRULENCE, GANGRENE DETERMINANT, C2 DOMAIN; 1.8A {CLOSTRIDIUM PERFRINGENS} SCOP: a.124.1.1, b.12.1.0
</pre><pre>Probab=24.60  E-value=1.8e+02  Score=21.28  Aligned_cols=26  Identities=27%  Similarity=0.635  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcC---------CCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFH---------GNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        33 LidLdkPFH---------GNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      +-|+--|+|         ..|-.|-.||.+...++
</pre><pre>T Consensus       128 ~~D~~~P~Ht~~~~~~~~~~H~~fE~~v~~~~~~~  162 (370)
</pre><pre>T 2WXU_A          128 FGDIDTPYHPANVTAVDSAGHVKFETFAEERKEQY  162 (370)
</pre><pre>T ss_dssp             HHHHTSSTTTTTCCTTTCTHHHHHHHHHHHHGGGG
</pre><pre>T ss_pred             HHHcCCCCCcccCccCCCCChHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>3QKS_A DNA double-strand break repair rad50 ATPase; RecA-like fold, coiled-coils, ATPase, exonuclease, endonuclease, ATP binding, DNA binding, REPLICATION; 2.1A {Pyrococcus furiosus}
</pre><pre>Probab=24.60  E-value=1.1e+02  Score=18.78  Aligned_cols=39  Identities=18%  Similarity=0.283  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhHHHHhhhcc-cccccccccCCCCcCCCCh
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTGYVQNVTKK-AADMTNLIDLDKPFHGNHP   45 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~gYvQn~skk-a~d~tNLidLdkPFHGNHP   45 (61)
</pre><pre>                      ..+....+.+...+++.... ..-...+-+++.+-|+-||
</pre><pre>T Consensus       164 ~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  203 (203)
</pre><pre>T 3QKS_A          164 DKFETAYKKLSELKKTINNRIKEYRDILARTEGGHHHHHH  203 (203)
</pre><pre>T ss_dssp             CTTHHHHHHHHHHHHHHHHHHHHHHHHHHTTCSSCC----
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhcCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>TIGR04406 LPS_export_lptB; LPS export ABC transporter ATP-binding protein. Members of this fmaily are LptB, the ATP-binding cassette protein of an ABC transporter involved in lipopolysaccharide export.
</pre><pre>Probab=24.57  E-value=3.2e+02  Score=17.59  Aligned_cols=26  Identities=35%  Similarity=0.611  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +|.|+.+-|+....-+.+.+..+.+.
</pre><pre>T Consensus       159 lde~~~~l~~~~~~~~~~~l~~~~~~  184 (239)
</pre><pre>T TIGR04406       159 LDEPFAGVDPIAVGDIKKIIKHLKER  184 (239)
</pre><pre>T ss_pred             EcCcccCCCHHHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>2C3F_A HYALURONIDASE, PHAGE ASSOCIATED; LYASE, HYALURONAN LYASE, PHAGE TAIL FIBRE, TRIPLE-STRANDED BETA- HELIX, HYALURONIDASE, SCARLET FEVER; 1.81A {STREPTOCOCCUS PYOGENES} SCOP: b.108.1.5
</pre><pre>Probab=24.39  E-value=2e+02  Score=24.86  Aligned_cols=27  Identities=37%  Similarity=0.623  Sum_probs=0.0  Template_Neff=3.500
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   34 IDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        34 idLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ..|..|--+-|.+-.+||.++|.+|++
</pre><pre>T Consensus       327 ~~~~~~~~~~~~~~~~~~~~~~~~~~~  353 (358)
</pre><pre>T 2C3F_A          327 LKLKDPTANDHAATKAYVDKAISELKK  353 (358)
</pre><pre>T ss_dssp             CCCCCCCSTTSCCCHHHHHHHHHHHHH
</pre><pre>T ss_pred             eEEeCCCCCcchhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>d2d62a1 c.37.1.12 (A:1-243) automated matches {Pyrococcus horikoshii [TaxId: 53953]}
</pre><pre>Probab=24.30  E-value=2.2e+02  Score=19.41  Aligned_cols=32  Identities=13%  Similarity=0.277  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=||.|+.|-++.-..-+.+.|.++.++
</pre><pre>T Consensus       160 ~~~llllDep~~~ld~~~~~~l~~~l~~~~~~  191 (243)
</pre><pre>T d2d62a1         160 RPKVFLMDEPLSNLDAKLRVKMRAELKKLQRQ  191 (243)
</pre><pre>T ss_dssp             CCSEEEEESTTTTCCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCcEEEEeCCccCCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>d3tifa_ c.37.1.12 (A:) automated matches {Methanocaldococcus jannaschii [TaxId: 243232]}
</pre><pre>Probab=24.23  E-value=2.7e+02  Score=18.44  Aligned_cols=25  Identities=12%  Similarity=0.226  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ||.||.|-.|....-+.+.+.++.+
</pre><pre>T Consensus       169 ldep~~~l~~~~~~~l~~~l~~~~~  193 (230)
</pre><pre>T d3tifa_         169 ADQPTWALDSKTGEKIMQLLKKLNE  193 (230)
</pre><pre>T ss_dssp             EESTTTTSCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             EeCCCCCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>d5lj9a1 c.37.1.0 (A:1-223) automated matches {Escherichia coli [TaxId: 83333]}
</pre><pre>Probab=23.82  E-value=3.2e+02  Score=17.95  Aligned_cols=32  Identities=16%  Similarity=0.141  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.||.|-++....-+.+.+.++.+.
</pre><pre>T Consensus       162 ~p~~lildep~~~l~~~~~~~~~~~l~~~~~~  193 (223)
</pre><pre>T d5lj9a1         162 GGQVILADEPTGALDSHSGEEVMAILHQLRDR  193 (223)
</pre><pre>T ss_dssp             CCSEEEEESTTTTSCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CCcEEEEeCCCCCCCHHHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>d6amxa1 c.37.1.0 (A:2-235) automated matches {Aquifex aeolicus [TaxId: 224324]}
</pre><pre>Probab=23.71  E-value=2.6e+02  Score=19.14  Aligned_cols=26  Identities=12%  Similarity=0.239  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ||.||.|-+|.....+...+.++...
</pre><pre>T Consensus       164 lDep~~~ld~~~~~~~~~~l~~~~~~  189 (234)
</pre><pre>T d6amxa1         164 IDEALAVGDAHFQQKCFRKLKEHKQK  189 (234)
</pre><pre>T ss_dssp             ECGGGGCCCTTHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             EeCCcccCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>PF02638.17 ; GHL10 ; Glycosyl hydrolase-like 10
</pre><pre>Probab=23.65  E-value=2.3e+02  Score=19.72  Aligned_cols=19  Identities=16%  Similarity=0.412  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   43 NHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        43 NHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +||.+.+|+.+.+.+++++
</pre><pre>T Consensus       139 ~~~~~~~~~~~~i~~l~~~  157 (322)
</pre><pre>T YNGK_BACSU/35-  139 GIPEARDFIVKGIEEVVKH  157 (322)
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>2D2E_A SufC protein; ABC-ATPase, Suf protein, 310-helix, RIKEN Structural Genomics/Proteomics Initiative, RSGI, Structural Genomics, Protein Binding; HET: GOL; 1.7A {Thermus thermophilus} SCOP: c.37.1.0
</pre><pre>Probab=23.55  E-value=2.9e+02  Score=18.32  Aligned_cols=32  Identities=13%  Similarity=0.105  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+.|-.|.....+.+.+.++...
</pre><pre>T Consensus       161 ~p~l~llDep~~~ld~~~~~~i~~~l~~~~~~  192 (250)
</pre><pre>T 2D2E_A          161 EPTYAVLDETDSGLDIDALKVVARGVNAMRGP  192 (250)
</pre><pre>T ss_dssp             CCSEEEEECGGGTTCHHHHHHHHHHHHHHCST
</pre><pre>T ss_pred             CCCEEEEeCCCCCCCHHHHHHHHHHHHHhhcC
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>4HUQ_B Energy-coupling factor transporter ATP-binding protein EcfA 2; transporter, HYDROLASE; 2.998A {Lactobacillus brevis}
</pre><pre>Probab=23.55  E-value=3.1e+02  Score=19.22  Aligned_cols=32  Identities=13%  Similarity=0.132  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++-||.|+.|-.|.....+.+.+.++.+.
</pre><pre>T Consensus       158 ~~~~lllDEp~~~ld~~~~~~~~~~l~~~~~~  189 (279)
</pre><pre>T 4HUQ_B          158 APKILILDEATSMLDPQGRIEMLAIVRQLRQQ  189 (279)
</pre><pre>T ss_dssp             CCSEEEEESCGGGSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCeEEEEcCCCcCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>6DVU_B Apolipoprotein C-I; lipoprotein, alpha-helix, lipoprotein particles, lipid metabolism, LIPID TRANSPORT; 1.8A {Homo sapiens}
</pre><pre>Probab=23.46  E-value=1.1e+02  Score=20.47  Aligned_cols=21  Identities=19%  Similarity=0.368  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhhhHHHHhhhccccc
</pre><pre>Q EHC5102581.1/7    9 FKTDLKKVTGYVQNVTKKAAD   29 (61)
</pre><pre>Q Consensus         9 f~~~~k~i~gYvQn~skka~d   29 (61)
</pre><pre>                      +...|.++.+|.|+++.|+++
</pre><pre>T Consensus        30 ~~~~l~~v~e~~~~l~ekak~   50 (83)
</pre><pre>T 6DVU_B           30 VSSALDKLKEFGNTLEDKARE   50 (83)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>6YAL_k ATP-binding cassette sub-family E member 1 (ABCE1); initiation complex, 48S, eIF1A, eIF3, ABCE1, rabbit, TRANSLATION; HET: T6A, C4J, SF4, GNP; 3.0A {Oryctolagus cuniculus}
</pre><pre>Probab=23.06  E-value=2.5e+02  Score=20.74  Aligned_cols=32  Identities=6%  Similarity=0.046  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+++-||....-+.+.+..+.+.
</pre><pre>T Consensus       476 ~~~l~lidep~~~l~~~~~~~~~~~l~~~~~~  507 (595)
</pre><pre>T 6YAL_k          476 PADVYLIDEPSAYLDSEQRLMAARVVKRFILH  507 (595)
</pre><pre>T ss_dssp             SCCEEEEECSSTTCCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCcEEEEeCcccCCCHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>5HAB_A Ribonuclease J; exoribonuclease, beta-CASP, MBL, RNase J, HYDROLASE; HET: SO4; 2.3A {Methanolobus psychrophilus R15}
</pre><pre>Probab=22.82  E-value=1.4e+02  Score=22.35  Aligned_cols=19  Identities=37%  Similarity=0.614  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CcCCCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   39 PFHGNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        39 PFHGNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      |+||.+++...+++ ...++
</pre><pre>T Consensus       430 pvHg~~~~~~~~~~-~~~~~  448 (470)
</pre><pre>T 5HAB_A          430 PAHGTIQMHSEYIQ-MAEDA  448 (470)
</pre><pre>T ss_dssp             EESSCHHHHHHHHH-HHHHT
</pre><pre>T ss_pred             ecCCCHHHHHHHHH-HHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>PF14257.8 ; DUF4349 ; Domain of unknown function (DUF4349)
</pre><pre>Probab=22.60  E-value=78  Score=21.89  Aligned_cols=30  Identities=30%  Similarity=0.510  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhhhHHHHhhhccccccc
</pre><pre>Q EHC5102581.1/7    1 IPQEIWKEFKTDLKKVTGYVQNVTKKAADMT   31 (61)
</pre><pre>Q Consensus         1 IPn~iyk~f~~~~k~i~gYvQn~skka~d~t   31 (61)
</pre><pre>                      ||.+-+..|-..++.+ |.|...+..+.|.|
</pre><pre>T Consensus        54 VP~~~~~~~l~~l~~l-g~v~~~~~~~~Dvt   83 (219)
</pre><pre>T C7P0R0_HALMD/6   54 VPPENFSEARDDLAVL-GHVEDESVSVQDFT   83 (219)
</pre><pre>T ss_pred             ECHHHHHHHHHHHHhc-CcEEEEEeeccccH
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>2RQP_A Heterochromatin protein 1-binding protein 3; heterochromatin protein 1 binding protein, histone H1, heterochromatin protein 1, Alternative splicing, Chromosomal protein, DNA-binding, Nucleus; NMR {Homo sapiens}
</pre><pre>Probab=22.49  E-value=2e+02  Score=15.55  Aligned_cols=15  Identities=27%  Similarity=0.406  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   44 HPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        44 HP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ||.|.+.|.+.|..+
</pre><pre>T Consensus         9 ~~~~~~mi~~ai~~~   23 (88)
</pre><pre>T 2RQP_A            9 RPKMDAILTEAIKAC   23 (88)
</pre><pre>T ss_dssp             CCCHHHHHHHCCCCC
</pre><pre>T ss_pred             CCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>TIGR03190 benz_CoA_bzdN; benzoyl-CoA reductase, bzd-type, N subunit. Members of this family are the N subunit of one of two related types of four-subunit ATP-dependent benzoyl-CoA reductase.
</pre><pre>Probab=22.47  E-value=3.3e+02  Score=20.62  Aligned_cols=25  Identities=8%  Similarity=0.149  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ||-|.....+.--+|++++|++|++
</pre><pre>T Consensus       121 ~~~p~~~~~~~~~~y~~~ql~~li~  145 (377)
</pre><pre>T TIGR03190       121 VPMPNEVQSPHARKAHYAEVQRFRV  145 (377)
</pre><pre>T ss_pred             ccCCCCCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>PF02981.17 ; FokI_N ; Restriction endonuclease FokI, recognition domain
</pre><pre>Probab=22.41  E-value=64  Score=24.36  Aligned_cols=39  Identities=21%  Similarity=0.217  Sum_probs=0.0  Template_Neff=4.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHhhhcccccccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   18 GYVQNVTKKAADMTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        18 gYvQn~skka~d~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      |+|||.|.    -.+|...-.=|--|-+-|...+..+|..|+.
</pre><pre>T Consensus         4 GWvQnps~----f~~Lk~VV~IFd~~S~~~~~L~~~~i~~li~   42 (154)
</pre><pre>T R6AWC6_9BACT/4    4 GWIQNPSS----TDTLRNILGIFVPGSDFHTFMVSERLPLLAS   42 (154)
</pre><pre>T ss_pred             cccCCCcc----hHHHHHHHhhcCCCCHHHHHHHhcchhHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>d1lvga_ c.37.1.1 (A:) Guanylate kinase {Mouse (Mus musculus) [TaxId: 10090]}
</pre><pre>Probab=22.34  E-value=3.1e+02  Score=17.35  Aligned_cols=32  Identities=6%  Similarity=0.103  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=||.||.|-.+.-..=+.+.|.++.+.
</pre><pre>T Consensus       120 ~p~llilDep~~~ld~~~~~~~~~~l~~~~~~  151 (190)
</pre><pre>T d1lvga_         120 PPSLDVLEQRLRLRNTETEESLAKRLAAARTD  151 (190)
</pre><pre>T ss_dssp             CSCHHHHHHHHHHHTCSCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEeecchhCCCHHHHHHHHHHHHHHhcc
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>6YIR_A Oligosaccharides import ATP-binding protein MsmX; ATPase, Polysaccharide import, Multitask, HYDROLASE; HET: PGE; 1.68A {Bacillus subtilis (strain 168)}
</pre><pre>Probab=22.21  E-value=3e+02  Score=20.45  Aligned_cols=32  Identities=16%  Similarity=0.235  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...|+=||.||.|--+....-+.+.|.++.+.
</pre><pre>T Consensus       152 ~~~llllDEP~~~LD~~~~~~i~~~l~~~~~~  183 (373)
</pre><pre>T 6YIR_A          152 DAKVFLMDEPLSNLDAKLRVQMRAEIIKLHQR  183 (373)
</pre><pre>T ss_dssp             TCSEEEEESTTTTSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEcCCCCCCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>TIGR01166 cbiO; cobalt transport protein ATP-binding subunit. This model describes the ATP binding subunit of the multisubunit cobalt transporter in bacteria and its equivalents in archaea.
</pre><pre>Probab=22.16  E-value=3.7e+02  Score=16.40  Aligned_cols=32  Identities=25%  Similarity=0.387  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+-|-||.-..-+.+.+..+.+.
</pre><pre>T Consensus       145 ~~~~~ilde~~~~l~~~~~~~~~~~l~~~~~~  176 (190)
</pre><pre>T TIGR01166       145 RPDVLLLDEPTAGLDPAGREQMLAILRRLRAE  176 (190)
</pre><pre>T ss_pred             CCCEEEEcCcccCCCHHHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>5O6C_A E3 ubiquitin-protein ligase MYCBP2; RING Zinc binding threonine E3 ligase, LIGASE; 1.75A {Homo sapiens}
</pre><pre>Probab=22.08  E-value=88  Score=22.60  Aligned_cols=7  Identities=57%  Similarity=1.127  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CcCCCCh
</pre><pre>Q EHC5102581.1/7   39 PFHGNHP   45 (61)
</pre><pre>Q Consensus        39 PFHGNHP   45 (61)
</pre><pre>                      |||++||
</pre><pre>T Consensus       238 ~l~~~hp  244 (263)
</pre><pre>T 5O6C_A          238 PLHVVHP  244 (263)
</pre><pre>T ss_dssp             TTCSCCC
</pre><pre>T ss_pred             CCCcccC
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>4V1J_A FIMBRIAL PROTEIN; CELL ADHESION, TYPE IV PILLUS; HET: EDO; 1.43A {NEISSERIA MENINGITIDIS MC58}
</pre><pre>Probab=22.05  E-value=1.2e+02  Score=19.27  Aligned_cols=10  Identities=20%  Similarity=0.321  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             hhHHHHHHHH
</pre><pre>Q EHC5102581.1/7   45 PAYSQYVREA   54 (61)
</pre><pre>Q Consensus        45 P~Yn~YV~~~   54 (61)
</pre><pre>                      |+|.+||.+.
</pre><pre>T Consensus         1 PaYq~Yv~ra   10 (142)
</pre><pre>T 4V1J_A            1 GPMQDYTARA   10 (142)
</pre><pre>T ss_dssp             CHHHHHHHHH
</pre><pre>T ss_pred             CchHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>6HIY_Cm mS37; mitoribosome, translation, Trypanosoma, small ribosomal subunit, 9S rRNA, ribosomal protein, mitochondrial initiation factor IF-3, RIBOSOME; HET: MG, SPD, SPM; 3.27A {Trypanosoma brucei brucei}
</pre><pre>Probab=22.04  E-value=6.2e+02  Score=20.86  Aligned_cols=42  Identities=24%  Similarity=0.426  Sum_probs=0.0  Template_Neff=1.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhH-HHHhhhcccccccccccCCCCc-CCCChhHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTG-YVQNVTKKAADMTNLIDLDKPF-HGNHPAYSQYVREAIED   57 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~g-YvQn~skka~d~tNLidLdkPF-HGNHP~Yn~YV~~~i~k   57 (61)
</pre><pre>                      ....+...+..+ +|||||-         .--+|| |-.|..--.|.+|++.+
</pre><pre>T Consensus       134 ~Am~~A~~efnDWFiqNVSD---------NhTkpfqhRPhDWrh~yaQEklv~  177 (215)
</pre><pre>T 6HIY_Cm         134 RAMSEACWEFNDWFIQNVSD---------NHTKPFQHRPHDWRHFYAQEKLVR  177 (215)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHTCC---------CSCSCCCSSGGGGHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHhhhhhhhhccC---------CCCCCccCCCCchHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>PF08745.13 ; PIN_5 ; PINc domain ribonuclease
</pre><pre>Probab=22.00  E-value=2.3e+02  Score=22.30  Aligned_cols=33  Identities=21%  Similarity=0.323  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CC-HHHHHHHHHHH-------HhhhHHHHhhhccccccccc
</pre><pre>Q EHC5102581.1/7    1 IP-QEIWKEFKTDL-------KKVTGYVQNVTKKAADMTNL   33 (61)
</pre><pre>Q Consensus         1 IP-n~iyk~f~~~~-------k~i~gYvQn~skka~d~tNL   33 (61)
</pre><pre>                      +| ..||+|++..+       .-+...--.+.+|++|+.++
</pre><pre>T Consensus        47 mP~Psv~~El~~F~~~~~~~~~~~~~~~~~i~~KsP~~~~i   87 (217)
</pre><pre>T L0KV82_METHD/5   47 VPFPSVYNELQEFAKNNGCETDVIAKIDTWLVKKAPDRYNV   87 (217)
</pre><pre>T ss_pred             ecCHHHHHHHHHHHHhCCCchHHHHHhhhheEecCcccccc
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>6LDK_A Isoleucine--tRNA ligase; tRNA synthetase, TRANSLATION; HET: AMP; 2.9A {Candida albicans SC5314}
</pre><pre>Probab=21.97  E-value=5.7e+02  Score=21.41  Aligned_cols=55  Identities=13%  Similarity=0.236  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhHHHHhhhccccccccc-ccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTGYVQNVTKKAADMTNL-IDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~gYvQn~skka~d~tNL-idLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      .+|....+.+..+..+.-++...+-++ +|-|+.+--..|.|...|...+.+|.++
</pre><pre>T Consensus       120 ~~~~~~~~~~~~~~~~~~~~~l~~lG~~~d~~~~~~t~~~~~~~~v~~~f~~L~~~  175 (856)
</pre><pre>T 6LDK_A          120 DKYNAECRAIVMRYADEWRRTIKRLGRWIDMDNDYKTLYPEFMESVWWAFKELFNK  175 (856)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHTCCCEEEEEEETTSHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCcccCCHHHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>d1lxna_ d.58.48.1 (A:) Hypothetical protein MTH1187 {Methanobacterium thermoautotrophicum [TaxId: 145262]}
</pre><pre>Probab=21.67  E-value=2.4e+02  Score=18.23  Aligned_cols=22  Identities=36%  Similarity=0.529  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   39 PFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        39 PFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      |+...-|.|.+||.+.|+.+.+
</pre><pre>T Consensus        10 Plg~~~~~~~~~V~~~i~~i~~   31 (99)
</pre><pre>T d1lxna_          10 PLGTCSTSLSSYVAAAVEALKK   31 (99)
</pre><pre>T ss_dssp             ESSCSSSCCHHHHHHHHHHHTT
</pre><pre>T ss_pred             EcCCCCCChHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>6ZG3_F Energy-coupling factor transporter ATP-binding protein EcfA1; ECF transporter, vitamin transport, pantothenate, membrane protein, TRANSPORT PROTEIN; HET: LMT, CIT, PEG; 2.8A {Lama glama}
</pre><pre>Probab=21.62  E-value=3e+02  Score=18.48  Aligned_cols=29  Identities=17%  Similarity=0.256  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ++=+|.|+.|-||....-+.+.+..+.+.
</pre><pre>T Consensus       182 lllldep~~~l~~~~~~~l~~~l~~~~~~  210 (300)
</pre><pre>T 6ZG3_F          182 VIILDESTSMLDPEGKEQILDLVRKIKED  210 (300)
</pre><pre>T ss_dssp             EEEEESTTTTSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             EEEEeccccCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>2R5X_B Uncharacterized conserved protein; STRUCTURAL GENOMICS, UNKNOWN FUNCTION, PSI-2, PROTEIN STRUCTURE INITIATIVE, New York SGX Research Center for Structural Genomics; 2.04A {Geobacillus kaustophilus} SCOP: l.1.1.1, d.129.11.0
</pre><pre>Probab=21.59  E-value=3.3e+02  Score=19.01  Aligned_cols=22  Identities=14%  Similarity=0.303  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhhhHHHHh
</pre><pre>Q EHC5102581.1/7    1 IPQEIWKEFKTDLKKVTGYVQN   22 (61)
</pre><pre>Q Consensus         1 IPn~iyk~f~~~~k~i~gYvQn   22 (61)
</pre><pre>                      ||+++...=+..|+.|..|++.
</pre><pre>T Consensus        97 iP~~~~~~a~~~L~ev~~~L~~  118 (129)
</pre><pre>T 2R5X_B           97 FPSSLVSQCQNVLAQVKSELEK  118 (129)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>d1yqha1 d.58.48.1 (A:1-101) Hypothetical protein BC0424 {Bacillus cereus [TaxId: 1396]}
</pre><pre>Probab=21.55  E-value=2.2e+02  Score=18.63  Aligned_cols=21  Identities=24%  Similarity=0.213  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   39 PFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        39 PFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      |+.+. +.|.+||.+.|+.+.+
</pre><pre>T Consensus        13 Pl~~~-~~~~~~I~~~i~~i~~   33 (101)
</pre><pre>T d1yqha1          13 PQAKT-KDVYSVVDKAIEVVQQ   33 (101)
</pre><pre>T ss_dssp             EECSS-SCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             EcCCC-CChHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>6WWX_A Hyaluronoglucosaminidase; HylP, Lyase, Hyaluronidase, S. pyogenes phage H4489A, Beta-elimination; HET: BDP, NAG, GCD; 2.2A {Streptococcus pyogenes phage H4489A}
</pre><pre>Probab=21.52  E-value=2e+02  Score=24.44  Aligned_cols=27  Identities=37%  Similarity=0.662  Sum_probs=0.0  Template_Neff=2.200
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   34 IDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        34 idLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ..|..|--+-|.+--+||.++|.+|++
</pre><pre>T Consensus       254 l~~~dp~~~~haatk~yvd~~~~~lk~  280 (293)
</pre><pre>T 6WWX_A          254 LTVKDPTSGKHAATKDYVDEKIAELKK  280 (293)
</pre><pre>T ss_dssp             EECCCCCSTTBBCBHHHHHHHHHHHHH
</pre><pre>T ss_pred             eEEeCCCCCCccccHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>d1sgwa1 c.37.1.12 (A:2-200) Putative ABC transporter PF0895 {Pyrococcus furiosus [TaxId: 2261]}
</pre><pre>Probab=21.51  E-value=3.5e+02  Score=17.24  Aligned_cols=32  Identities=16%  Similarity=0.175  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=||.|+.|-.+.-..-+.+.|.++.++
</pre><pre>T Consensus       142 ~~~llilDep~~~ld~~~~~~~~~~l~~~~~~  173 (199)
</pre><pre>T d1sgwa1         142 NAEIYVLDDPVVAIDEDSKHKVLKSILEILKE  173 (199)
</pre><pre>T ss_dssp             CCSEEEEESTTTTSCTTTHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEeCCcCCCCHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>2IT1_B 362aa long hypothetical maltose/maltodextrin transport ATP-binding protein; ATP binding protein, Structural Genomics, NPPSFA, National Project on Protein Structural and Functional Analyses, RIKEN Structural; 1.94A {Pyrococcus horikoshii}
</pre><pre>Probab=21.37  E-value=3.1e+02  Score=19.26  Aligned_cols=31  Identities=19%  Similarity=0.305  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ...++=||.||.|-|+....-+.+.+.++.+
</pre><pre>T Consensus       151 ~~~lllldep~~~ld~~~~~~~~~~l~~~~~  181 (362)
</pre><pre>T 2IT1_B          151 EPEVLLLDEPLSNLDALLRLEVRAELKRLQK  181 (362)
</pre><pre>T ss_dssp             CCSEEEEESGGGGSCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCcEEEEeCcccCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>d2zu0c_ c.37.1.0 (C:) automated matches {Escherichia coli K-12 [TaxId: 83333]}
</pre><pre>Probab=21.15  E-value=3.5e+02  Score=18.25  Aligned_cols=32  Identities=19%  Similarity=0.192  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=||.||-|-+|.-...+.+.+..+.+.
</pre><pre>T Consensus       163 ~p~vlilDep~~~Ld~~~~~~~~~~l~~~~~~  194 (247)
</pre><pre>T d2zu0c_         163 EPELCILDESDSGLDIDALKVVADGVNSLRDG  194 (247)
</pre><pre>T ss_dssp             CCSEEEEESTTTTCCHHHHHHHHHHHHTTCCS
</pre><pre>T ss_pred             CCcEEEEeCCCCCCCHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>6N9L_A UvrABC system protein A; DNA Binding protein, UvrA, Nucleotide Excision Repair, NER, ADP, hydrolase, DNA damage repair; HET: ADP; 2.01A {Thermotoga maritima}
</pre><pre>Probab=21.10  E-value=3.1e+02  Score=20.72  Aligned_cols=29  Identities=28%  Similarity=0.265  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ++=+|.|.-|-||....-+-+.|.++...
</pre><pre>T Consensus       553 illIDEpE~~Lhp~~~~~l~~~L~~~~~~  581 (640)
</pre><pre>T 6N9L_A          553 LYILDEPTVGLHFEDVRKLVEVLHRLVDR  581 (640)
</pre><pre>T ss_dssp             EEEEECTTTTCCHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             EEEEecCCCCCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>TIGR00422 valS; valyl-tRNA synthetase. The valyl-tRNA synthetase (ValS) is a class I amino acyl-tRNA ligase and is particularly closely related to the isoleucyl tRNA synthetase.
</pre><pre>Probab=21.09  E-value=5.8e+02  Score=21.71  Aligned_cols=55  Identities=16%  Similarity=0.155  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhhhHHHHhhhccccccccc-ccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7    7 KEFKTDLKKVTGYVQNVTKKAADMTNL-IDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus         7 k~f~~~~k~i~gYvQn~skka~d~tNL-idLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      .+|.+..+++..+-...-+..-.+-++ .|.+.++.-.+|.|.+.|++.+.+|.++
</pre><pre>T Consensus       107 ~~~~~~~~~~~~~~~~~~~~~l~~lgi~~d~~~~~~T~~~~~~~~v~~~f~~L~~~  162 (861)
</pre><pre>T TIGR00422       107 EEFREKIWEWKEESGGTIKNQIKRLGASLDWSRERFTMDEGLSKAVKEAFVRLYEK  162 (861)
</pre><pre>T ss_pred             HHHHHHHHHHHHHhHHHHHHHHHHhCCCCCCCcccccCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>4GAK_A Acyl-ACP thioesterase; MCSG, PSI-BIOLOGY, STRUCTURAL GENOMICS, Midwest Center for Structural Genomics, HYDROLASE; HET: GOL, MSE; 1.9A {Spirosoma linguale}
</pre><pre>Probab=20.99  E-value=1.2e+02  Score=20.08  Aligned_cols=21  Identities=33%  Similarity=0.578  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             cCCCCcCCCChhHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   35 DLDKPFHGNHPAYSQYVREAI   55 (61)
</pre><pre>Q Consensus        35 dLdkPFHGNHP~Yn~YV~~~i   55 (61)
</pre><pre>                      |+|.--|=|+..|-+|+.+.+
</pre><pre>T Consensus       170 d~D~~~Hvn~~~y~~~~~~a~  190 (250)
</pre><pre>T 4GAK_A          170 NIDQNQHVNNVAYVQWLLEGV  190 (250)
</pre><pre>T ss_dssp             GBCTTSSBCHHHHHHHHHHTS
</pre><pre>T ss_pred             hccCCCCcCHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>PF14426.8 ; Imm2 ; Immunity protein Imm2
</pre><pre>Probab=20.78  E-value=3.3e+02  Score=16.77  Aligned_cols=20  Identities=20%  Similarity=0.295  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CCChhHH-HHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   42 GNHPAYS-QYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        42 GNHP~Yn-~YV~~~i~kl~k~   61 (61)
</pre><pre>                      |.||... .|.+++|.+++..
</pre><pre>T Consensus        12 Gr~~~~~~~~~~~~I~~iL~~   32 (60)
</pre><pre>T A0A0F6RSH6_PSE   12 GRSSDKVKRYHLDIIHGLLKE   32 (60)
</pre><pre>T ss_pred             CCChHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>6XGZ_A Organic solvent ABC transporter ATP-binding protein; ABC transporters, MlaF, MlaB, bacterial outer membrane, lipid transport, ATPase, regulation, STAS domain; HET: GOL, PO4; 2.6A {Escherichia coli LAU-EC10}
</pre><pre>Probab=20.69  E-value=3.3e+02  Score=17.71  Aligned_cols=28  Identities=32%  Similarity=0.563  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++=+|.|+-|-||.....+.+.+.++.+
</pre><pre>T Consensus       165 iliidep~~~l~~~~~~~~~~~l~~~~~  192 (269)
</pre><pre>T 6XGZ_A          165 LIMFDEPFVGQDPITMGVLVKLISELNS  192 (269)
</pre><pre>T ss_dssp             EEEEESTTTTCCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             EEEEeCCCCCCChhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>6XGZ_G Organic solvent ABC transporter ATP-binding protein; ABC transporters, MlaF, MlaB, bacterial outer membrane, lipid transport, ATPase, regulation, STAS domain; HET: PO4, GOL; 2.6A {Escherichia coli LAU-EC10}
</pre><pre>Probab=20.69  E-value=3.3e+02  Score=17.71  Aligned_cols=28  Identities=32%  Similarity=0.563  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++=+|.|+-|-||.....+.+.+.++.+
</pre><pre>T Consensus       165 iliidep~~~l~~~~~~~~~~~l~~~~~  192 (269)
</pre><pre>T 6XGZ_G          165 LIMFDEPFVGQDPITMGVLVKLISELNS  192 (269)
</pre><pre>T ss_dssp             EEEEESTTTTCCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             EEEEeCCCCCCChhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>3HE4_D SYNZIP5; heterodimeric coiled-coil, DE NOVO PROTEIN; 2.46A {artificial gene}
</pre><pre>Probab=20.60  E-value=86  Score=19.96  Aligned_cols=24  Identities=25%  Similarity=0.621  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhhhHHHHhhhccccccccc
</pre><pre>Q EHC5102581.1/7   10 KTDLKKVTGYVQNVTKKAADMTNL   33 (61)
</pre><pre>Q Consensus        10 ~~~~k~i~gYvQn~skka~d~tNL   33 (61)
</pre><pre>                      .+..+.++.|||..-.+.....||
</pre><pre>T Consensus         2 sntvkelknyiqeleernaelknl   25 (46)
</pre><pre>T 3HE4_D            2 SNTVKELKNYIQELEERNAELKNL   25 (46)
</pre><pre>T ss_dssp             -CCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             chHHHHHHHHHHHHHHHhHHHHhH
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>3HE4_F SYNZIP5; heterodimeric coiled-coil, DE NOVO PROTEIN; 2.46A {artificial gene}
</pre><pre>Probab=20.60  E-value=86  Score=19.96  Aligned_cols=24  Identities=25%  Similarity=0.621  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhhhHHHHhhhccccccccc
</pre><pre>Q EHC5102581.1/7   10 KTDLKKVTGYVQNVTKKAADMTNL   33 (61)
</pre><pre>Q Consensus        10 ~~~~k~i~gYvQn~skka~d~tNL   33 (61)
</pre><pre>                      .+..+.++.|||..-.+.....||
</pre><pre>T Consensus         2 sntvkelknyiqeleernaelknl   25 (46)
</pre><pre>T 3HE4_F            2 SNTVKELKNYIQELEERNAELKNL   25 (46)
</pre><pre>T ss_dssp             -CCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             chHHHHHHHHHHHHHHHhHHHHhH
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>6XJI_D ABC transporter ATP-binding protein; ABC transporter, ABC exporter, Peptide transort, MEMBRANE PROTEIN; HET: AGS;{Staphylococcus aureus}
</pre><pre>Probab=20.59  E-value=3.1e+02  Score=18.46  Aligned_cols=28  Identities=32%  Similarity=0.388  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCcCCCChhHHHHHHHHHHHHHh
</pre><pre>Q EHC5102581.1/7   33 LIDLDKPFHGNHPAYSQYVREAIEDLKA   60 (61)
</pre><pre>Q Consensus        33 LidLdkPFHGNHP~Yn~YV~~~i~kl~k   60 (61)
</pre><pre>                      ++=+|.|..+-||....-+.+.+.++.+
</pre><pre>T Consensus       140 llliDep~~~L~~~~~~~l~~~l~~l~~  167 (290)
</pre><pre>T 6XJI_D          140 FLILDEPTNGMDPDGSIDVLTTIKSLVN  167 (290)
</pre><pre>T ss_dssp             EEEEESTTTTCCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             EEEEECcccCCChHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>TIGR01189 ccmA; heme ABC exporter, ATP-binding protein CcmA. This model describes the cyt c biogenesis protein encoded by ccmA in bacteria.
</pre><pre>Probab=20.45  E-value=4e+02  Score=16.10  Aligned_cols=26  Identities=19%  Similarity=0.134  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      +|.|+.+-++....-+.+.+.++.+.
</pre><pre>T Consensus       151 iDe~~~~l~~~~~~~~~~~l~~~~~~  176 (198)
</pre><pre>T TIGR01189       151 LDEPTTALDKAGVALLAGLLRAHLAR  176 (198)
</pre><pre>T ss_pred             ecCccccCCHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>d2a1ia1 c.52.1.20 (A:99-227) DNA excision repair protein ERCC-1 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=20.30  E-value=1.5e+02  Score=21.09  Aligned_cols=23  Identities=30%  Similarity=0.665  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   36 LDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        36 LdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      |.-.||--||   +|+.+++++|.++
</pre><pre>T Consensus        43 LSlkyH~l~P---~YI~~Ri~~l~~~   65 (129)
</pre><pre>T d2a1ia1          43 LSLRYHNLHP---DYIHGRLQSLGKN   65 (129)
</pre><pre>T ss_dssp             EEHHHHHHST---THHHHHHHHHTTS
</pre><pre>T ss_pred             eccchHhcCh---HHHHHHHHHccCC
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>TIGR02982 heterocyst_DevA; ABC exporter ATP-binding subunit, DevA family. Members of this protein family are found mostly in the Cyanobacteria, but also in the Planctomycetes.
</pre><pre>Probab=20.24  E-value=2.7e+02  Score=17.21  Aligned_cols=32  Identities=19%  Similarity=0.207  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             ccccccCCCCcCCCChhHHHHHHHHHHHHHhC
</pre><pre>Q EHC5102581.1/7   30 MTNLIDLDKPFHGNHPAYSQYVREAIEDLKAN   61 (61)
</pre><pre>Q Consensus        30 ~tNLidLdkPFHGNHP~Yn~YV~~~i~kl~k~   61 (61)
</pre><pre>                      ...++=+|.|+.+-+|.....+.+.+..+.++
</pre><pre>T Consensus       159 ~~~iiilde~~~~l~~~~~~~~~~~l~~~~~~  190 (220)
</pre><pre>T TIGR02982       159 HPKLVLADEPTAALDSKSGRDVVELMQKLAKE  190 (220)
</pre><pre>T ss_pred             CCcEEEEcCHHhcCChHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>1UST_A HISTONE H1; DNA BINDING PROTEIN, LINKER HISTONE, DNA BINDING DOMAIN, WINGED HELIX FOLD; NMR {SACCHAROMYCES CEREVISIAE} SCOP: a.4.5.13
</pre><pre>Probab=20.06  E-value=2.6e+02  Score=15.58  Aligned_cols=17  Identities=18%  Similarity=0.436  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCChhHHHHHHHHHHHH
</pre><pre>Q EHC5102581.1/7   42 GNHPAYSQYVREAIEDL   58 (61)
</pre><pre>Q Consensus        42 GNHP~Yn~YV~~~i~kl   58 (61)
</pre><pre>                      ..||.|.+.|.+.|..+
</pre><pre>T Consensus         5 ~~~~~~~~mi~~ai~~~   21 (93)
</pre><pre>T 1UST_A            5 ASSKSYRELIIEGLTAL   21 (93)
</pre><pre>T ss_dssp             SCCCCHHHHHHHHHTTT
</pre><pre>T ss_pred             ccCCCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre>