<pre>Query         MBC1768434.1/339-412 
</pre><pre>Match_columns 74
</pre><pre>No_of_seqs    69 out of 74
</pre><pre>Neff          3.81163
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 16:51:48 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/6618720.hhr -oa3m ../results/6618720.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 d1vz0a2 d.268.1.1 (A:23-115) P  98.2 3.4E-05 3.2E-10   46.3   7.5   68    2-71      3-78  (93)
</pre><pre>  2 d1xw3a1 d.268.1.4 (A:32-137) S  98.1 4.4E-05   4E-10   48.3   6.7   68    2-70     12-89  (106)
</pre><pre>  3 TIGR03764 ICE_PFGI_1_parB; int  97.9 0.00023 2.1E-09   51.7   7.7   68    2-70     10-93  (258)
</pre><pre>  4 TIGR00180 parB_part; ParB/RepB  97.8 0.00026 2.4E-09   48.4   6.8   71    1-72      6-86  (187)
</pre><pre>  5 PF02195.20 ; ParBc ; ParB-like  97.8  0.0014 1.3E-08   37.5   8.3   71    1-72      1-79  (91)
</pre><pre>  6 7NFU_B Nucleoid occlusion prot  97.6 0.00093 8.6E-09   47.9   7.6   71    1-72     29-107 (252)
</pre><pre>  7 TIGR03454 partition_RepB; plas  97.6 0.00079 7.3E-09   51.8   7.2   70    2-72     66-144 (325)
</pre><pre>  8 TIGR04285 nucleoid_noc; nucleo  97.5  0.0013 1.2E-08   47.4   7.5   70    1-72      3-80  (255)
</pre><pre>  9 d1vk1a_ d.268.1.2 (A:) Hypothe  97.5 0.00096 8.9E-09   48.0   6.5   73    1-73      9-85  (232)
</pre><pre> 10 6KY4_A Sulfiredoxin, chloropla  97.3  0.0035 3.3E-08   38.7   6.7   70    2-72     21-96  (112)
</pre><pre> 11 6RYK_B ParB-like nuclease doma  97.2  0.0074 6.8E-08   42.8   7.6   71    1-72     35-111 (229)
</pre><pre> 12 5UJE_A SbnI protein; staphylof  97.0  0.0086   8E-08   43.2   7.0   72    1-72     11-85  (242)
</pre><pre> 13 6SDK_B Stage 0 sporulation pro  96.9    0.02 1.8E-07   38.6   7.6   71    1-72      5-82  (199)
</pre><pre> 14 7BM8_A Chromosome-partitioning  96.8   0.017 1.6E-07   41.5   7.1   71    1-72     35-115 (257)
</pre><pre> 15 1XW3_A SULFIREDOXIN; retroredu  96.8    0.03 2.8E-07   33.9   6.9   71    2-72     16-95  (110)
</pre><pre> 16 6NR6_B Bifunctional transcript  96.7   0.019 1.7E-07   41.6   6.5   72    1-72      9-83  (254)
</pre><pre> 17 2HWJ_F Hypothetical protein At  96.7   0.016 1.5E-07   43.0   6.1   68    2-69     11-89  (205)
</pre><pre> 18 4UMK_C PROBABLE CHROMOSOME-PAR  96.6   0.024 2.2E-07   40.2   6.3   71    1-72     34-111 (240)
</pre><pre> 19 1VZ0_C Chromosome-partitioning  96.5   0.038 3.5E-07   38.7   6.9   71    1-72     24-101 (230)
</pre><pre> 20 1VZ0_F Chromosome-partitioning  96.5   0.038 3.5E-07   38.7   6.9   71    1-72     24-101 (230)
</pre><pre> 21 d2hwja1 d.268.1.3 (A:4-204) Hy  96.0   0.045 4.2E-07   40.7   5.3   69    1-69      7-86  (201)
</pre><pre> 22 5K5D_A ParB domain protein nuc  96.0   0.073 6.7E-07   40.5   6.4   68    1-72     12-86  (300)
</pre><pre> 23 5K5D_B ParB domain protein nuc  96.0   0.073 6.7E-07   40.5   6.4   68    1-72     12-86  (300)
</pre><pre> 24 PF08857.13 ; ParBc_2 ; Putativ  96.0   0.073 6.8E-07   37.8   6.1   64    6-69      1-75  (153)
</pre><pre> 25 1VK1_A Conserved hypothetical   95.6    0.11   1E-06   37.3   5.7   73    1-73     19-95  (242)
</pre><pre> 26 TIGR03734 PRTRC_parB; PRTRC sy  94.7    0.34 3.2E-06   40.1   6.8   53   19-72     18-75  (554)
</pre><pre> 27 1R71_C Transcriptional repress  75.2     4.6 4.3E-05   27.4   1.6   18   52-69      1-19  (178)
</pre><pre> 28 d3awub_ d.386.1.1 (B:) Tyrosin  71.1     8.5 7.9E-05   26.6   2.0   20   31-50     38-57  (84)
</pre><pre> 29 PF06236.13 ; MelC1 ; Tyrosinas  63.3      22  0.0002   25.7   2.9   29   31-59     63-97  (109)
</pre><pre> 30 2P12_B Hypothetical protein DU  58.2      41 0.00038   24.9   3.6   27   35-61    107-133 (176)
</pre><pre> 31 5Z0F_B MelC; tyrosinase, catal  56.1      37 0.00034   25.2   3.1   29   31-59     77-111 (134)
</pre><pre> 32 d1s6ma_ d.89.1.5 (A:) automate  54.6      37 0.00034   27.2   3.1   25   35-59    167-191 (293)
</pre><pre> 33 7D8Q_A UPF0374 protein SAB1800  54.1      55 0.00051   23.5   3.6   27   35-61    109-135 (180)
</pre><pre> 34 d2r6sa_ b.82.2.3 (A:) automate  54.1      16 0.00015   27.6   1.1   52    8-59    227-282 (311)
</pre><pre> 35 5Z0E_B MelC; tyrosinase, catal  53.8      40 0.00037   25.0   3.0   29   31-59     77-111 (134)
</pre><pre> 36 d3l57a_ d.89.1.0 (A:) automate  53.7      34 0.00032   26.3   2.7   25   35-59    166-190 (225)
</pre><pre> 37 5ZDM_A FomD; metalloprotein, h  53.1      56 0.00052   24.4   3.7   27   35-61    132-159 (211)
</pre><pre> 38 2R6S_A Gab protein; jelly-roll  52.7      22  0.0002   26.8   1.6   52    8-59    241-296 (325)
</pre><pre> 39 d2p12a1 b.175.1.1 (A:8-172) Hy  45.7      86  0.0008   23.2   3.6   27   35-61     98-124 (165)
</pre><pre> 40 PF18762.3 ; Kinase-PolyVal ; S  45.5      66 0.00062   23.6   3.0   41    8-49     90-153 (160)
</pre><pre> 41 6J2U_A Tyrosinase co-factor pr  41.9      52 0.00049   23.9   2.0   20   31-50     71-90  (118)
</pre><pre> 42 1JR7_A HYPOTHETICAL 37.4 KDA P  41.6      36 0.00033   25.4   1.2   52    8-59    227-282 (311)
</pre><pre> 43 6I3G_A ABC transporter, substr  39.2 2.7E+02  0.0025   21.9   5.5   71    2-72    295-370 (490)
</pre><pre> 44 d2y5ca1 d.15.4.0 (A:4-109) aut  39.1 1.6E+02  0.0014   18.8   3.6   32   33-64      1-37  (106)
</pre><pre> 45 3EXM_A Phosphatase SC4828; nuc  38.7 1.3E+02  0.0012   23.2   3.8   27   35-61    153-179 (237)
</pre><pre> 46 1OMH_A trwC protein; PROTEIN-D  36.9   1E+02 0.00097   24.2   3.1   25   35-59    167-191 (293)
</pre><pre> 47 2JV8_A Uncharacterized protein  36.3      59 0.00055   21.6   1.5   20   31-50     24-44  (73)
</pre><pre> 48 4ZE8_A ABC transporter, substr  36.2 2.4E+02  0.0023   22.2   4.8   52   19-70    323-376 (499)
</pre><pre> 49 4ZE8_D ABC transporter, substr  36.2 2.4E+02  0.0023   22.2   4.8   52   19-70    323-376 (499)
</pre><pre> 50 6CKA_B Paratox; prophage, natu  34.6 1.6E+02  0.0015   20.2   3.2   29   19-48      5-33  (68)
</pre><pre> 51 4C5W_A GAMMA-BUTYROBETAINE DIO  33.9      29 0.00027   26.9  -0.2   22   38-59    330-351 (387)
</pre><pre> 52 PF08751.13 ; TrwC ; TrwC relax  33.4 1.3E+02  0.0012   23.7   3.0   25   35-59    161-185 (283)
</pre><pre> 53 d4cvya_ b.82.2.0 (A:) automate  32.7      38 0.00035   24.9   0.2   44   19-62    212-255 (289)
</pre><pre> 54 3L6T_B Mobilization protein Tr  30.8 1.3E+02  0.0012   23.7   2.8   25   35-59    169-193 (302)
</pre><pre> 55 d1otja_ b.82.2.5 (A:) Taurine/  29.8      53 0.00049   24.0   0.5   41   19-59    218-258 (281)
</pre><pre> 56 PF19129.2 ; DUF5812 ; Family o  29.0      73 0.00068   24.6   1.1   32   35-66    101-140 (141)
</pre><pre> 57 TIGR03054 photo_alph_chp1; put  29.0 1.8E+02  0.0017   21.9   3.1   27   23-49     85-111 (135)
</pre><pre> 58 TIGR02410 carnitine_TMLD; trim  28.9      35 0.00032   26.7  -0.5   23   37-59    314-336 (362)
</pre><pre> 59 PF08943.12 ; CsiD ; CsiD        28.3      83 0.00077   23.0   1.3   52    8-59    221-276 (294)
</pre><pre> 60 PF03477.18 ; ATP-cone ; ATP co  27.3 1.8E+02  0.0016   14.5   2.3   20   37-56      2-21  (91)
</pre><pre> 61 PF18419.3 ; ATP-grasp_6 ; ATP-  26.5 1.4E+02  0.0013   17.3   1.8   13   53-65     11-23  (54)
</pre><pre> 62 7O81_AU EDF1; Frameshift, viru  26.3 1.7E+02  0.0016   20.1   2.4   35   31-65      5-40  (148)
</pre><pre> 63 TIGR03849 arch_ComA; phosphosu  25.7 3.2E+02  0.0029   22.3   4.1   44   19-67     74-120 (237)
</pre><pre> 64 6NPC_A TmpA, 2-trimethylaminoe  25.5      47 0.00044   26.1  -0.3   22   38-59    330-351 (383)
</pre><pre> 65 PF13224.8 ; DUF4032 ; Domain o  25.1 1.8E+02  0.0017   21.3   2.5   22   51-72     22-44  (165)
</pre><pre> 66 4S36_A Phage baseplate protein  24.9 1.6E+02  0.0015   18.3   1.9   15   35-49      2-16  (90)
</pre><pre> 67 TIGR02409 carnitine_bodg; gamm  24.6      37 0.00035   26.4  -1.0   22   38-59    316-337 (366)
</pre><pre> 68 d5wqsa_ c.1.8.1 (A:) beta-Amyl  24.5 1.4E+02  0.0013   26.4   2.2   33   34-67     20-55  (498)
</pre><pre> 69 d3pvja_ b.82.2.5 (A:) automate  24.3 1.8E+02  0.0017   21.1   2.4   35   19-53    215-249 (275)
</pre><pre> 70 PF19064.2 ; DUF5760 ; Family o  24.1 3.4E+02  0.0032   17.9   3.4   23   19-46     10-32  (84)
</pre><pre> 71 d1wdpa_ c.1.8.1 (A:) automated  23.6 1.7E+02  0.0016   25.9   2.5   33   34-67     17-52  (493)
</pre><pre> 72 d1twda_ c.1.30.1 (A:) Copper h  23.6 6.4E+02   0.006   19.7   5.4   56   16-73     36-102 (247)
</pre><pre> 73 1WDP_A Beta-amylase; (beta/alp  23.5 1.6E+02  0.0015   25.6   2.4   33   34-67     19-54  (495)
</pre><pre> 74 d5h57a_ d.15.4.1 (A:) automate  23.5 2.6E+02  0.0024   17.4   2.7   24   41-64     10-37  (97)
</pre><pre> 75 6TFX_B ABC transporter substra  23.1 2.7E+02  0.0025   21.9   3.2   54   19-72    315-369 (500)
</pre><pre> 76 d1qwga_ c.1.27.1 (A:) (2r)-pho  22.4   4E+02  0.0037   21.8   4.1   44   19-67     88-134 (251)
</pre><pre> 77 d2z15a1 d.370.1.1 (A:10-126) T  22.0 3.7E+02  0.0035   18.9   3.5   32   27-62     47-78  (117)
</pre><pre> 78 4KMD_A Sufu; protein peptide c  21.6 3.4E+02  0.0031   23.1   3.7   58    3-62    223-280 (444)
</pre><pre> 79 PF06792.13 ; UPF0261 ; Unchara  21.5      52 0.00048   28.1  -0.8   55    8-63    214-273 (402)
</pre><pre> 80 d1jx4a2 e.8.1.7 (A:1-240) DinB  20.6 3.5E+02  0.0033   20.2   3.3   37   22-62     15-56  (240)
</pre><pre> 81 d1nx4a_ b.82.2.8 (A:) Carbapen  20.3      50 0.00047   23.7  -1.0   22   38-59    233-254 (271)
</pre><pre>
</pre><pre>No 1
</pre><pre>>d1vz0a2 d.268.1.1 (A:23-115) Putative partitioning protein ParB/Spo0J {Thermus thermophilus [TaxId: 274]}
</pre><pre>Probab=98.21  E-value=3.4e-05  Score=46.29  Aligned_cols=68  Identities=21%  Similarity=0.280  Sum_probs=51.6  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHccccccc----CCCc--HHHHHHHHHcCCCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeCcc
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTS----VNGS--DKIIASMKANGWKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRNYD   71 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~s----Vs~~--~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~   71 (74)
</pre><pre>                      +|++++|.+.+..    ++..  ..+.++|++.|. ..|+.|... +|++.-+| ++|+.||+.+|.+ +++.+++.+
</pre><pre>T Consensus         3 ~i~~~~i~~~~~~~r~~~~~~~~~~l~~~i~~~g~-~~pi~v~~~-~~~~~iidG~~R~~a~~~~~~~~i~~~~~~~~   78 (93)
</pre><pre>T d1vz0a2           3 RLPLASIRPNPRQPRKRFAEESLKELADSIREKGL-LQPLLVRPQ-GDGYELVAGERRYRAALMAGLQEVPAVVKDLT   78 (93)
</pre><pre>T ss_dssp             EEEGGGEECCCCCHHHHHHHHHHHHHHHHHHHHCC-SSCEEEEEE-TTEEEEEECHHHHHHHHHHTCSEEEEEECCCC
</pre><pre>T ss_pred             EccHHHcCCCCCCCCccCCHHHHHHHHHHHHHcCC-CcCEEEEEc-CCeEEEEecchHHHHHHHcCCCeecEEEEcCC
</pre><pre>Confidence            4566666665432    2322  789999999885 689999888 78777776 7899999999999 999887654
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>d1xw3a1 d.268.1.4 (A:32-137) Sulfiredoxin {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=98.11  E-value=4.4e-05  Score=48.33  Aligned_cols=68  Identities=16%  Similarity=0.199  Sum_probs=53.4  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHccccccc--CCCc--HHHHHHHHHcCCCCCCceEEECC----CCCEEecc-hhHHHHHHHcCCC-CeeEEeCc
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTS--VNGS--DKIIASMKANGWKGDPIDVVKMP----DGSLTTLD-NTRVAAAREAGID-VQATVRNY   70 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~s--Vs~~--~~~~~~M~~~gW~g~pi~Vv~~~----DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~   70 (74)
</pre><pre>                      .+++++|.. +++  ++..  +++.++|+..|+++.|++|...+    +++|++++ |||+.|++.++.. ++|.+.+.
</pre><pre>T Consensus        12 ~i~~~~i~~-~~~~~~~~~~~~~l~~si~~~G~~~~Pi~v~~~~~~~~~~~~~ii~G~~R~~a~~~l~~~~i~~~v~~~   89 (106)
</pre><pre>T d1xw3a1          12 NVPLSVLIR-PLPSVLDPAKVQSLVDTIREDPDSVPPIDVLWIKGAQGGDYFYSFGGCHRYAAYQQLQRETIPAKLVQS   89 (106)
</pre><pre>T ss_dssp             EEEGGGEEC-CSCCCCCHHHHHHHHHHHHHCGGGSCCEEEEEEECTTSCEEEECCSCHHHHHHHHHTTCSEEEEEEEEE
</pre><pre>T ss_pred             EeeHHHccC-CCCCCCCHHHHHHHHHHHHhCCCCCCCeEEEEEeCCCCCceEEEEecHHHHHHHHHhCCCcCCEEEEeC
</pre><pre>Confidence            356777765 444  3433  89999999998658899998751    45799999 9999999999999 99987654
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>TIGR03764 ICE_PFGI_1_parB; integrating conjugative element, PFGI_1 class, ParB family protein. Members of this protein family carry the ParB-type nuclease domain and are found in integrating conjugative elements (ICE) in the same class as PFGI-1 of Pseudomonas fluorescens Pf-5.
</pre><pre>Probab=97.87  E-value=0.00023  Score=51.72  Aligned_cols=68  Identities=21%  Similarity=0.344  Sum_probs=54.2  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHccccccc----CCCc-HHHHHHHHHcCCCCCCceEEECC-CCCEEecc--hhHHHHHHHc-------CCC-Cee
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTS----VNGS-DKIIASMKANGWKGDPIDVVKMP-DGSLTTLD--NTRVAAAREA-------GID-VQA   65 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~s----Vs~~-~~~~~~M~~~gW~g~pi~Vv~~~-DG~lvS~D--NrRl~AAr~a-------gi~-Vp~   65 (74)
</pre><pre>                      .|++++|.+++.|    .+.. .+|.++|+..|+ .+|++|+... +++|+|++  |||+.|++.+       +.. ++|
</pre><pre>T Consensus        10 ~i~~~~i~~~~~n~R~~~~~~~~~l~~si~~~G~-~~Pi~v~~~~~~~~y~ii~GG~~R~~a~~~l~~~~~~~~~~~i~~   88 (258)
</pre><pre>T TIGR03764        10 VLTLDQLRPYEDNPRTTRNPKYDEIKASIRERGL-DQPPKVTRRPGESDYIIRNGGNTRLQILKELWQETGDERFFRIHC   88 (258)
</pre><pre>T ss_pred             EEcHHHcCCCCCCcCcCCChhHHHHHHHHHHcCC-CCCCEEEeCCCCCcEEEecCChHHHHHHHHHHHhhCCccceeEEE
</pre><pre>Confidence            4778888887666    2334 899999999999 9999999873 37899998  6999999988       666 888
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEeCc
</pre><pre>Q MBC1768434.1/3   66 TVRNY   70 (74)
</pre><pre>Q Consensus        66 ~~~~~   70 (74)
</pre><pre>                      ++.+.
</pre><pre>T Consensus        89 ~v~~~   93 (258)
</pre><pre>T TIGR03764        89 LFKPW   93 (258)
</pre><pre>T ss_pred             EEeCC
</pre><pre>Confidence            87653
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>TIGR00180 parB_part; ParB/RepB/Spo0J family partition protein. This model represents the most well-conserved core of a set of chromosomal and plasmid partition proteins related to ParB, including Spo0J, RepB, and SopB.
</pre><pre>Probab=97.82  E-value=0.00026  Score=48.36  Aligned_cols=71  Identities=30%  Similarity=0.458  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHc---ccccc-cCCCc--HHHHHHHHHcCCCCCCceEEECCC--CCEEecch-hHHHHHHHcCCC-CeeEEeCc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKI---RFSQT-SVNGS--DKIIASMKANGWKGDPIDVVKMPD--GSLTTLDN-TRVAAAREAGID-VQATVRNY   70 (74)
</pre><pre>Q Consensus         1 e~v~P~~I---~FSQ~-sVs~~--~~~~~~M~~~gW~g~pi~Vv~~~D--G~lvS~DN-rRl~AAr~agi~-Vp~~~~~~   70 (74)
</pre><pre>                      .+|+|+.|   .|.++ ..+..  .+++++|+.+|+.. |+.|...++  |++.-+|- +|++||+++|.. |++.+...
</pre><pre>T Consensus         6 ~~v~~~~l~~~~~~~R~~~~~~~i~~l~~si~~~G~~~-Pi~v~~~~~~~~~~~iidG~~R~~a~~~l~~~~i~~~i~~~   84 (187)
</pre><pre>T TIGR00180         6 IEIDIDLLQPNPYQPRKDFSEESLAELIESIKEQGQLQ-PILVRKHPDQPGRYEIIAGERRWRAAKLAGLKTIPAIVREL   84 (187)
</pre><pre>T ss_pred             EEEeHHHcCCCCCCCCCCCCHHHHHHHHHHHHHcCCCC-CeEEEeCCCCCCcEEEEcCccHHHHHHHcCCCcCCEEEECC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cc
</pre><pre>Q MBC1768434.1/3   71 DD   72 (74)
</pre><pre>Q Consensus        71 ~d   72 (74)
</pre><pre>                      ++
</pre><pre>T Consensus        85 ~~   86 (187)
</pre><pre>T TIGR00180        85 DD   86 (187)
</pre><pre>T ss_pred             CH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF02195.20 ; ParBc ; ParB-like nuclease domain
</pre><pre>Probab=97.77  E-value=0.0014  Score=37.45  Aligned_cols=71  Identities=24%  Similarity=0.410  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecch-hHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDN-TRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DN-rRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      .++++..|...+......      .++.++++..+|. .|+.|...++|++..+|- +|+.|+..+|+. +++.+....+
</pre><pre>T Consensus         1 ~~i~~~~i~~~~~~~~~~~~~~~~~~l~~~i~~~~~~-~p~~v~~~~~~~~~i~~G~~r~~a~~~~~~~~i~~~v~~~~~   79 (91)
</pre><pre>T PARB_TREPA/33-    1 HYLDPTLLQANPHQARRTFAQESLEELAASIREHGVI-QPVLAEKNQDGSWVIIAGERRTRAAILAGLNRIPVIVRTCDH   79 (91)
</pre><pre>T ss_pred             CccCHHHcCCCCCCccccCCHHHHHHHHHHHHHcCCc-cCeEEEECCCCCEEEEcCchHHHHHHHhCCCcccEEEEcCCH
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>7NFU_B Nucleoid occlusion protein; CTP switch, chromosome segregation, protein-DNA recognition, peripheral membrane protein, DNA BINDING PROTEIN; HET: SO4; 2.5A {Geobacillus thermoleovorans CCB_US3_UF5}
</pre><pre>Probab=97.63  E-value=0.00093  Score=47.85  Aligned_cols=71  Identities=18%  Similarity=0.311  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|++..|..+....-..      .++.++|+..| ...|+.|..-+||++.-+| ++|+.||+.+|+. +++.+.+.++
</pre><pre>T Consensus        29 ~~v~l~~i~~~~~~~r~~~~~~~~~~L~~si~~~G-~~~Pi~v~~~~~g~y~iidG~~R~~a~~~~g~~~i~~~v~~~~~  107 (252)
</pre><pre>T 7NFU_B           29 RHIPVKSIIPNRFQPRTMFDEEKIDELALTIRTHG-IIQPIVVRECGNGRFEIIAGERRWRAVQKLGWTEIPAIIKNLND  107 (252)
</pre><pre>T ss_dssp             EEEEGGGEECCTTSCCCCCCHHHHHHHHHHHHHHC-CSSCEEEEEETTTEEEEEECHHHHHHHHHTTCSEEEEEECCCCH
</pre><pre>T ss_pred             eEecHHHccCCCCCCCcCCCHHHHHHHHHHHHHhC-CCCCeEEEEcCCCcEEEeecchHHHHHHHhCCCeeCEEEEeCCH
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>TIGR03454 partition_RepB; plasmid partitioning protein RepB. Members of this family are the RepB protein involved in replicon partitioning.
</pre><pre>Probab=97.59  E-value=0.00079  Score=51.78  Aligned_cols=70  Identities=30%  Similarity=0.440  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHccccccc----CCCc--HHHHHHHHHcCCCCCCceEEECC--CCCEEecc-hhHHHHHHHcCCCCeeEEeCccc
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTS----VNGS--DKIIASMKANGWKGDPIDVVKMP--DGSLTTLD-NTRVAAAREAGIDVQATVRNYDD   72 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~s----Vs~~--~~~~~~M~~~gW~g~pi~Vv~~~--DG~lvS~D-NrRl~AAr~agi~Vp~~~~~~~d   72 (74)
</pre><pre>                      .|+|..|..+...    .+..  .+++++++.+|+.- |+.|...+  +|++.-+| +||++|++++|++|++.+.+.+|
</pre><pre>T Consensus        66 ~i~~~~I~~~~~~~R~~~~~~~~~~L~~sI~~~G~~~-PI~Vr~~~~~~g~y~Iv~G~rR~~A~~~lg~~v~a~V~~~~d  144 (325)
</pre><pre>T TIGR03454        66 ELDPALIDPSFVRDRLDSDDEDFADLVESIREHGQQV-PILVRPHPEAEGRYQIAYGHRRLRAARELGRPVKAVVRELSD  144 (325)
</pre><pre>T ss_pred             EeCHHHcCCccCCCCCCCCHHHHHHHHHHHHHcCCcC-CEEEEeCCCCCCceEEEeCccHHHHHHHhCCCeEEEEECCCH
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>TIGR04285 nucleoid_noc; nucleoid occlusion protein. This model describes nucleoid occlusion protein, a close homolog to ParB chromosome partitioning proteins including Spo0J in Bacillus subtilis.
</pre><pre>Probab=97.55  E-value=0.0013  Score=47.44  Aligned_cols=70  Identities=26%  Similarity=0.336  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      .+|+++.|.-.....-..      .+++++|+..|+.. |+.|... +|++.-+| ++|++||+.+|++ |++.+.+.++
</pre><pre>T Consensus         3 ~~i~i~~i~~~~~n~r~~~~~~~i~~L~~si~~~G~~~-Pi~v~~~-~~~y~ii~G~~R~~a~~~lg~~~i~~~v~~~~~   80 (255)
</pre><pre>T TIGR04285         3 QQIPIDKIVPNPYQPRKVFDEESIEELAQSIKEHGLIQ-PIVVRKK-DDKYEIIAGERRFRACKLLGWEEVPAIVREMND   80 (255)
</pre><pre>T ss_pred             EEecHHHcCCCCCCCcccCCHHHHHHHHHHHHHcCCcc-CEEEEec-CCeEEEecChHHHHHHHHhCCCcCCEEEeCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>d1vk1a_ d.268.1.2 (A:) Hypothetical protein PF0380 {Pyrococcus furiosus [TaxId: 2261]}
</pre><pre>Probab=97.52  E-value=0.00096  Score=48.02  Aligned_cols=73  Identities=18%  Similarity=0.252  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc-HHHHHHHHHcCCCCCCceEEEC-CCCCEEecc-hhHHHHHHHcCCC-CeeEEeCcccC
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS-DKIIASMKANGWKGDPIDVVKM-PDGSLTTLD-NTRVAAAREAGID-VQATVRNYDDL   73 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~-~~~~~~M~~~gW~g~pi~Vv~~-~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~d~   73 (74)
</pre><pre>                      ++|++++|.+....-... .++.++|+..|-...|+-|... .+|++.-.| ++|+.||++.|.+ +++.+++.+++
</pre><pre>T Consensus         9 ~~v~~~~i~~~e~~~~~~~~~l~~~i~~~G~~~~Pi~v~~~~~~~~~~i~dGhhR~~a~~~l~~~~i~~~~~~~~~~   85 (232)
</pre><pre>T d1vk1a_           9 VFIELDKMKPHEQLVQRELEDFIESVTGSGIFWKPMLLAKIPGTDEYLIVDGHHRWAGLQKLGAKRAPSVILDYFDE   85 (232)
</pre><pre>T ss_dssp             EEEEGGGEECSBCCCHHHHHHHHHHHHHHCEECSCEEEEECTTSSCEEEEECHHHHHHHHHHTCCEEEEEEECTTST
</pre><pre>T ss_pred             EEEeHHHCCCCcccCHHHHHHHHHHHHhcCCCCCCEEEEECCCCCcEEEeCCHHHHHHHHHhCCCEEcEEEEeCCCh
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>6KY4_A Sulfiredoxin, chloroplastic/mitochondrial; sulfiredoxin, cysteine, sulfinic acid, peroxiredoxin repair protein, arabidopsis thaliana, antioxidant, oxidoreductase; HET: ADP, PO4; 3.2A {Arabidopsis thaliana}
</pre><pre>Probab=97.35  E-value=0.0035  Score=38.75  Aligned_cols=70  Identities=19%  Similarity=0.287  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHcccccccCCCc-----HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTSVNGS-----DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~sVs~~-----~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +|++..|.+++......     .++.++++..| .-.|+.+...+++-.+---++|+.||+.+|++ +++.++...+
</pre><pre>T Consensus        21 ~v~~~~i~~~~~~~r~~~~~~~~~l~~si~~~g-~~~pi~v~~~~~~~~ii~G~~R~~a~~~~g~~~v~~~v~~~~~   96 (112)
</pre><pre>T 6KY4_A           21 ELPLEKIRRPLMRTRSNDQNKVKELMDSIRQIG-LQVPIDVIEVDGTYYGFSGCHRYEAHQKLGLPTIRCKIRKGTK   96 (112)
</pre><pre>T ss_dssp             EEEGGGEECGGGGGCCCCHHHHHHHHHHHHHHC-CCSCEEEEEETTEEECCSCHHHHHHHHHTTCSEEEEEEEECCH
</pre><pre>T ss_pred             EEeHHHccCCccCCCCCCHHHHHHHHHHHHHcC-CCCCEEEEEECCEEEEECChHHHHHHHHhCCCeeeEEEEeCCH
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>6RYK_B ParB-like nuclease domain protein; ParABS, cytoskeleton, bactofilin, CTP, CELL CYCLE; HET: CTP; 1.7A {Myxococcus xanthus DK 1622}
</pre><pre>Probab=97.18  E-value=0.0074  Score=42.81  Aligned_cols=71  Identities=20%  Similarity=0.109  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHccccc-ccCCCc---HHHHHHHHHcCCCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQ-TSVNGS---DKIIASMKANGWKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ-~sVs~~---~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      .+|++..|.+.. ......   .+++++++..| ...|+.|...++|++.-+| ++|++||+.+|.+ |++.++...+
</pre><pre>T Consensus        35 ~~i~i~~i~~~~~~~~~~~~~~~~L~~si~~~G-~~~Pi~v~~~~~~~y~ii~G~~R~~a~~~~g~~~i~~~v~~~~~  111 (229)
</pre><pre>T 6RYK_B           35 VLLPLEKLQDESAFKLRPEGDVSGLATDIARLG-QLFPVDVRPAGEDRYQLVCGFRRVAALRFLKRDAVQARIHLRLS  111 (229)
</pre><pre>T ss_dssp             EEEEGGGBCSCCTTCCSCCCCCHHHHHHHHHHC-CCSCEEEEECSTTCEEEEECHHHHHHHHHTTCSEEEEEEETTCC
</pre><pre>T ss_pred             EEEeHHHcCCCCccCCCCccchHHHHHHHHHhC-CCCCeEEEEcCCCcEEEEeCchHHHHHHHcCCCcceEEEEcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>5UJE_A SbnI protein; staphyloferrin B, heme, regulator, siderophore, GENE REGULATION; HET: GOL, MSE; 2.5A {Staphylococcus aureus}
</pre><pre>Probab=97.04  E-value=0.0086  Score=43.17  Aligned_cols=72  Identities=22%  Similarity=0.251  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc-HHHHHHHHHcCCCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS-DKIIASMKANGWKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~-~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|++.+|......-... .+++++|++.|....|+.|...++|+++-+| .+|+.||+++|.+ +++.+++.++
</pre><pre>T Consensus        11 ~~i~~~~i~~~~~~~~~~~~~l~~~i~~~g~~~~Pi~v~~~~~~~~~i~dGhhR~~a~~~~~~~~i~~~v~~~~~   85 (242)
</pre><pre>T 5UJE_A           11 KLVPVDKIDLHETFEPLRLEKTKSSIEADDFIRHPILVTAMQHGRYMVIDGVHRYTSLKALGCKKVPVQEIHETQ   85 (242)
</pre><pre>T ss_dssp             EEEEGGGEECSBCCCHHHHHHHHHHHHHHTEESSCEEEEECSTTCEEEEECHHHHHHHHHTTCSEEEEEEECTTT
</pre><pre>T ss_pred             EEeeHHHCCCCCCCCHHHHHHHHHHHHhcCCCCCCeEEEECCCCeEEEcCChHHHHHHHHhCCCeeCEEEECCCc
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>6SDK_B Stage 0 sporulation protein J; ParB, Chromosome segregation, Chromosome organization, DNA BINDING PROTEIN; HET: MSE, CDP; 1.81A {Bacillus subtilis (strain 168)} SCOP: d.268.1.0, a.4.14.0
</pre><pre>Probab=96.94  E-value=0.02  Score=38.63  Aligned_cols=71  Identities=25%  Similarity=0.297  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|+.+.|...+...-..      .++.++++..| ...|+.|...++|-.+---++|+.|++.+|+. +++.+...++
</pre><pre>T Consensus         5 ~~i~l~~i~~~~~~~r~~~~~~~~~~l~~si~~~g-~~~pi~v~~~~~~~~ii~G~~R~~a~~~~g~~~i~~~v~~~~~   82 (199)
</pre><pre>T 6SDK_B            5 EEIKIADLRPNPYQPRKHFDDEALAELKESVLQHG-ILQPLIVRKSLKGYDIVAGERRFRAAKLAGLDTVPAIVRELSE   82 (199)
</pre><pre>T ss_dssp             EEEEGGGEECCTTCCCCCCCHHHHHHHHHHHHHHC-CCSCEEEEECSSSEEEEECHHHHHHHHHTTCSEEEEEECCCCH
</pre><pre>T ss_pred             EEccHHHCCCCCCCCCcCCCHHHHHHHHHHHHHcC-CCCCEEEEeCCCCEEEeechHHHHHHHHcCCCeeCEEEecCCH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>7BM8_A Chromosome-partitioning protein ParB; chromosome segregation, CTP, molecular gates, protein-DNA recognition, DNA BINDING PROTEIN; HET: CTP; 2.73A {Caulobacter vibrioides (strain NA1000 / CB15N)}
</pre><pre>Probab=96.84  E-value=0.017  Score=41.53  Aligned_cols=71  Identities=23%  Similarity=0.313  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEEC--CCCCEEecc-hhHHHHHHHcCCC-CeeEEeCc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKM--PDGSLTTLD-NTRVAAAREAGID-VQATVRNY   70 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~--~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~   70 (74)
</pre><pre>                      .+|+...|.+.+...-..      .+++++|+..| ...|+.|...  .+|++.-+| ++|+.||++.|++ +++.+...
</pre><pre>T Consensus        35 ~~I~i~~i~~~~~~~r~~~~~~~~~~L~~sI~~~G-~~~Pi~v~~~~~~~g~y~ildG~~R~~A~~~~~~~~i~~~v~~~  113 (257)
</pre><pre>T 7BM8_A           35 REAPIEILQRNPDQPRRTFREEDLEDLSNSIREKG-VLQPILVRPSPDTAGEYQIVAGERRWRAAQRAGLKTVPIMVREL  113 (257)
</pre><pre>T ss_dssp             EEEEGGGEEECTTCSCCCCCHHHHHHHHHHHHHHC-CCSCEEEEECSSSTTCEEEEECHHHHHHHHHTTCSEEEEEECCC
</pre><pre>T ss_pred             EEEeHHHccCCCCCccccCCHHHHHHHHHHHHHhC-CCCCeEEEeCCCCCCcEEEEcCccHHHHHHHcCCCccCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cc
</pre><pre>Q MBC1768434.1/3   71 DD   72 (74)
</pre><pre>Q Consensus        71 ~d   72 (74)
</pre><pre>                      ++
</pre><pre>T Consensus       114 ~~  115 (257)
</pre><pre>T 7BM8_A          114 DD  115 (257)
</pre><pre>T ss_dssp             CH
</pre><pre>T ss_pred             CH
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>1XW3_A SULFIREDOXIN; retroreduction, sulfinic acid, peroxiredoxin, ATP, OXIDOREDUCTASE; 1.65A {Homo sapiens} SCOP: d.268.1.4, l.1.1.1
</pre><pre>Probab=96.79  E-value=0.03  Score=33.85  Aligned_cols=71  Identities=20%  Similarity=0.123  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHcccccccCCCc---HHHHHHHHHcCCCCCCceEEEC-----CCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTSVNGS---DKIIASMKANGWKGDPIDVVKM-----PDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~sVs~~---~~~~~~M~~~gW~g~pi~Vv~~-----~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      .|+...|...+...-..   ..+.++++..++...|+.|...     .++-...--.+|+.||+.+|++ |++.+...++
</pre><pre>T Consensus        16 ~v~~~~i~~~~~~~~~~~~~~~l~~~i~~~g~~~~p~~v~~~~~~~~~~~~~l~~G~~R~~a~~~~~~~~v~~~v~~~~~   95 (110)
</pre><pre>T 1XW3_A           16 NVPLSVLIRPLPSVLDPAKVQSLVDTIREDPDSVPPIDVLWIKGAQGGDYFYSFGGCHRYAAYQQLQRETIPAKLVQSTL   95 (110)
</pre><pre>T ss_dssp             EEEGGGEECCSCCCCCHHHHHHHHHHHHHCGGGSCCEEEEEEECTTSCEEEECCSCHHHHHHHHHTTCSEEEEEEEEECH
</pre><pre>T ss_pred             EEEHHHccCCCCCCCCHHHHHHHHHHHHhCCCCCCCeEEEEEeCCCCCCeEEEEcChHHHHHHHHcCCCcCCEEEEcCCH
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>6NR6_B Bifunctional transcriptional regulator/O-phospho-L-serine synthase SbnI; free serine kinase, siderophore, ParB/Srx family, staphyloferrin B, BIOSYNTHETIC PROTEIN; HET: ADP; 1.9A {Staphylococcus pseudintermedius}
</pre><pre>Probab=96.71  E-value=0.019  Score=41.56  Aligned_cols=72  Identities=18%  Similarity=0.216  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc-HHHHHHHHHcCCCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS-DKIIASMKANGWKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~-~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.++++.|.....+-... .++.++|+..|....|+-|...++|.+.-+| ..|+.||+.+|.. +++.+++.++
</pre><pre>T Consensus         9 ~~v~~~~i~~~e~~~~~~~~~l~~~i~~~g~~~~Pi~v~~~~~~~~~i~dG~hR~~a~~~~~~~~i~~~v~~~~~   83 (254)
</pre><pre>T 6NR6_B            9 QLIPVDKIDLHEAFEPSRLEKTKESIAKEQHLRHPVLVVKTLFGRYMVIDGVHRFMSLKALGCEVIPVQVIQRTQ   83 (254)
</pre><pre>T ss_dssp             EEEEGGGEECCBCCCHHHHHHHHHHHHHHCEESSCEEEEECGGGCEEEEECHHHHHHHHHTTCSEEEEEECCGGG
</pre><pre>T ss_pred             EEEeHHHCcCCCCCCHHHHHHHHHHHHHhCCCCCCEEEEEcCCCeEEEeCCHHHHHHHHHcCCCEEeEEEEchhh
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>2HWJ_F Hypothetical protein Atu1540; Agrobacterium tumefaciens, Hypothetical protein, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: MSE; 2.61A {Agrobacterium tumefaciens str. C58} SCOP: d.268.1.3
</pre><pre>Probab=96.68  E-value=0.016  Score=43.01  Aligned_cols=68  Identities=18%  Similarity=0.179  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHcccccccCCCc--HHHHHHHHHcC-------CCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeC
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTSVNGS--DKIIASMKANG-------WKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRN   69 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~sVs~~--~~~~~~M~~~g-------W~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~   69 (74)
</pre><pre>                      +|+..+|+=||.+|+..  ....+.++..+       -...|+-||.-++|++.-+| ..|+.||.++|++ |+|.++.
</pre><pre>T Consensus        11 ~v~i~~L~PTQ~~vg~~~V~~~~~~~~~~~~~~~~~~l~~~~iPVVigp~g~~yiiDGHHr~~A~~~~G~~~v~v~v~~   89 (205)
</pre><pre>T 2HWJ_F           11 RIAIDKLRPTQIAVGFREVELKRKEWRETRKKDGDDFLGNHIVPVVAGPKDRAYLIDHHHLVLALSKEGVEHVLTSEVA   89 (205)
</pre><pre>T ss_dssp             EEEGGGEECCBSEECHHHHHHHHHHHHTC---------CCCEEEEEECSTTCEEECSCHHHHHHHHHTTCCEEEEEEEE
</pre><pre>T ss_pred             EEeHHHccccchhhCHHHHHHHHHHHHhhccccchhhhcCCeeeEEECCCCcEEEeCChHHHHHHHHcCCCEEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>4UMK_C PROBABLE CHROMOSOME-PARTITIONING PROTEIN PARB; DNA BINDING PROTEIN-DNA COMPLEX; HET: SO4; 3.096A {HELICOBACTER PYLORI}
</pre><pre>Probab=96.61  E-value=0.024  Score=40.22  Aligned_cols=71  Identities=15%  Similarity=0.181  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|+++.|.+.+...-..      .++.++++..| .-.|+.|...+++-.+---++|+.||+.+|+. |++.+++.++
</pre><pre>T Consensus        34 ~~i~l~~i~~~~~~~r~~~~~~~~~~l~~si~~~G-~~~Pi~v~~~~g~y~iidG~~R~~a~~~~g~~~i~~~v~~~~~  111 (240)
</pre><pre>T 4UMK_C           34 VELGIDEVMPNPYQPRKVFSEDSLEELAQSIKEHG-LLQPVLVVSENGRYHLIAGERRLRASKLAKMPTIKAIVVDIEQ  111 (240)
</pre><pre>T ss_dssp             -CCHHHHSCSCCSSSSCSSSSTTCCCTTTHHHHCS-SCCSCCCSSSCCCSSCCCCCCCTTTTTTCCSSCCCCCCCCHHH
</pre><pre>T ss_pred             EEecHHHcCCCCCCCcccCCHHHHHHHHHHHHHhC-CCCCeEEEecCCEEEEeeCCChHHHHHHcCCCeeeEEEEeCCH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>1VZ0_C Chromosome-partitioning protein Spo0J; NUCLEAR PROTEIN, CHROMOSOME SEGREGATION, DNA-BINDING, HELIX-TURN-HELIX; HET: CO; 2.3A {Thermus thermophilus HB27} SCOP: d.268.1.1, a.4.14.1
</pre><pre>Probab=96.53  E-value=0.038  Score=38.73  Aligned_cols=71  Identities=24%  Similarity=0.274  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|+++.|...+...-..      ..+.++++..| .-.|+.|...++|-.+---++|+.||++.|++ +++.+.+.++
</pre><pre>T Consensus        24 ~~i~i~~i~~~~~~~r~~~~~~~~~~l~~si~~~G-~~~Pi~v~~~~~~~~ii~G~~R~~a~~~l~~~~i~~~v~~~~~  101 (230)
</pre><pre>T 1VZ0_C           24 VRLPLASIRPNPRQPRKRFAEESLKELADSIREKG-LLQPLLVRPQGDGYELVAGERRYRAALMAGLQEVPAVVKDLTD  101 (230)
</pre><pre>T ss_dssp             EEEEGGGEECCCCCHHHHHHHHHHHHHHHHHHHHC-CSSCEEEEEETTEEEEEECHHHHHHHHHTTCSEEEEEECCCCH
</pre><pre>T ss_pred             EEeeHHHcCCCCCCCcccCCHHHHHHHHHHHHHhC-CCCCEEEEECCCcEEEeCCchHHHHHHHcCCCeeCEEEEeCCH
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>1VZ0_F Chromosome-partitioning protein Spo0J; NUCLEAR PROTEIN, CHROMOSOME SEGREGATION, DNA-BINDING, HELIX-TURN-HELIX; HET: CO; 2.3A {Thermus thermophilus HB27} SCOP: a.4.14.1, d.268.1.1
</pre><pre>Probab=96.53  E-value=0.038  Score=38.73  Aligned_cols=71  Identities=24%  Similarity=0.274  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|+++.|...+...-..      ..+.++++..| .-.|+.|...++|-.+---++|+.||++.|++ +++.+.+.++
</pre><pre>T Consensus        24 ~~i~i~~i~~~~~~~r~~~~~~~~~~l~~si~~~G-~~~Pi~v~~~~~~~~ii~G~~R~~a~~~l~~~~i~~~v~~~~~  101 (230)
</pre><pre>T 1VZ0_F           24 VRLPLASIRPNPRQPRKRFAEESLKELADSIREKG-LLQPLLVRPQGDGYELVAGERRYRAALMAGLQEVPAVVKDLTD  101 (230)
</pre><pre>T ss_dssp             EEEETTSEECCCCCHHHHHHHHHHHHHHHHHHHHC-CSSCEEEEEETTEEEEEECHHHHHHHHHHTCSEEEEEECCCCH
</pre><pre>T ss_pred             EEeeHHHcCCCCCCCcccCCHHHHHHHHHHHHHhC-CCCCEEEEECCCcEEEeCCchHHHHHHHcCCCeeCEEEEeCCH
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>d2hwja1 d.268.1.3 (A:4-204) Hypothetical protein Atu1540 {Agrobacterium tumefaciens [TaxId: 358]}
</pre><pre>Probab=96.00  E-value=0.045  Score=40.67  Aligned_cols=69  Identities=17%  Similarity=0.178  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCcHHHHHHHHHcCCC---------CCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeC
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGSDKIIASMKANGWK---------GDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRN   69 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~~~~~~~M~~~gW~---------g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~   69 (74)
</pre><pre>                      ..|+..+|+=||.+|+...-+.+..+=..|.         ..|+-||.-|||++.-+| ..|+.||.++|++ |++.++.
</pre><pre>T Consensus         7 ~~v~i~~L~PTQ~~vg~~~V~~k~~~~~~~~~~~~~~~~~~~~ipvVigp~~~~yl~DGHHr~~A~~~~G~~~V~v~v~~   86 (201)
</pre><pre>T d2hwja1           7 SRIAIDKLRPTQIAVGFREVELKRKEWRETRKKDGDDFLGNHIVPVVAGPKDRAYLIDHHHLVLALSKEGVEHVLTSEVA   86 (201)
</pre><pre>T ss_dssp             EEEEGGGCBCSBSEECHHHHHHHHHHHHTCC-----CCTTCBEEEEEECSTTCEEECSCHHHHHHHHHTTCCEEEEEEEE
</pre><pre>T ss_pred             EEeeHHHcCCCccccCHHHHHHHHHHHHHHhHhcchhhhhCCeEEEEECCCCcEEEECChHHHHHHHHcCCCEEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>5K5D_A ParB domain protein nuclease; ParB-N, pnob8, partition, HYDROLASE; HET: CIT, MSE; 2.45A {Sulfolobus solfataricus}
</pre><pre>Probab=95.99  E-value=0.073  Score=40.47  Aligned_cols=68  Identities=15%  Similarity=0.156  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|+++.|........-.      .+++++|+..| .-.|+.|  . .+-..---.+|++||+++|+. |++.+....+
</pre><pre>T Consensus        12 ~~i~i~~i~~~~~~~~~~~~~~~~~~L~~sI~~~G-~~~Pi~V--~-~~~~lidG~~R~~A~k~lg~~~i~~~v~~~~~   86 (300)
</pre><pre>T 5K5D_A           12 EVVSIEKLKEDDEFKTMVPSNNSREDLEKSLREKS-QIFPLIA--D-RNYVLIDGYTRLDIMKKLGFKEVKILKYDFDS   86 (300)
</pre><pre>T ss_dssp             EEEEGGGCBCCHHHHTTSCCCSCTTHHHHHHHHHC-CSSCEEE--C-TTCBEEECHHHHHHHHHTTCCEEEEEEESSCT
</pre><pre>T ss_pred             EEEeHHHCccChhHhccCCChhhHHHHHHHHHHcC-CCCCEEE--C-CCCEEEccHHHHHHHHHhCCCEeeEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>5K5D_B ParB domain protein nuclease; ParB-N, pnob8, partition, HYDROLASE; HET: MSE, CIT; 2.45A {Sulfolobus solfataricus}
</pre><pre>Probab=95.99  E-value=0.073  Score=40.47  Aligned_cols=68  Identities=15%  Similarity=0.156  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc------HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS------DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~------~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      +.|+++.|........-.      .+++++|+..| .-.|+.|  . .+-..---.+|++||+++|+. |++.+....+
</pre><pre>T Consensus        12 ~~i~i~~i~~~~~~~~~~~~~~~~~~L~~sI~~~G-~~~Pi~V--~-~~~~lidG~~R~~A~k~lg~~~i~~~v~~~~~   86 (300)
</pre><pre>T 5K5D_B           12 EVVSIEKLKEDDEFKTMVPSNNSREDLEKSLREKS-QIFPLIA--D-RNYVLIDGYTRLDIMKKLGFKEVKILKYDFDS   86 (300)
</pre><pre>T ss_dssp             EEEEGGGCEECHHHHHSSCCCTTHHHHHHHHHHHC-SCSCEEE--C-TTCEEEECHHHHHHHHHTTCCEEEEEEESSCT
</pre><pre>T ss_pred             EEEeHHHCccChhHhccCCChhhHHHHHHHHHHcC-CCCCEEE--C-CCCEEEccHHHHHHHHHhCCCEeeEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>PF08857.13 ; ParBc_2 ; Putative ParB-like nuclease
</pre><pre>Probab=95.99  E-value=0.073  Score=37.75  Aligned_cols=64  Identities=16%  Similarity=0.145  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHcccccccCCCc--HHHHHHHHHcC-------CCCCCceEEECCCCCEEecc-hhHHHHHHHcCCC-CeeEEeC
</pre><pre>Q MBC1768434.1/3    6 SKIRFSQTSVNGS--DKIIASMKANG-------WKGDPIDVVKMPDGSLTTLD-NTRVAAAREAGID-VQATVRN   69 (74)
</pre><pre>Q Consensus         6 ~~I~FSQ~sVs~~--~~~~~~M~~~g-------W~g~pi~Vv~~~DG~lvS~D-NrRl~AAr~agi~-Vp~~~~~   69 (74)
</pre><pre>                      .+|.=||-+|+..  ..-.+.++...       -...|+-|+.-+||++.-+| ..|+.||.++|++ |++.+..
</pre><pre>T Consensus         1 ~~L~PTQ~~lg~~~V~~K~~~~~~~~~~~~~~~~~~~~ipvv~~~~~~~yiiDGHHr~~a~~~~g~~~i~v~v~~   75 (153)
</pre><pre>T S5Y864_PARAH/7    1 IALRPTQMTVGMKEVQRKRHDWREDSQSKEVNFLGHHLLPCVLGPKKRYHLIDNHHLALALHLEGVTAVLVNVVA   75 (153)
</pre><pre>T ss_pred             CCCCCCccccCHHHHHHHHHHHHhcchhHHHhhhccCcccEEECCCCcEEEeCChHHHHHHHHcCCcEEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>1VK1_A Conserved hypothetical protein; reductive methylation, dimethyl lysine, Pyrococcus furiosus, conserved hypothetical protein, structural genomics, PSI, Protein Structure Initiative, Southeast; HET: MLY; 1.2A {Pyrococcus furiosus} SCOP: d.268.1.2
</pre><pre>Probab=95.59  E-value=0.11  Score=37.26  Aligned_cols=73  Identities=21%  Similarity=0.278  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHcccccccCCCc-HHHHHHHHHcCCCCCCceEEECC--CCCEEecchhHHHHHHHcCCC-CeeEEeCcccC
</pre><pre>Q MBC1768434.1/3    1 ENVNPSKIRFSQTSVNGS-DKIIASMKANGWKGDPIDVVKMP--DGSLTTLDNTRVAAAREAGID-VQATVRNYDDL   73 (74)
</pre><pre>Q Consensus         1 e~v~P~~I~FSQ~sVs~~-~~~~~~M~~~gW~g~pi~Vv~~~--DG~lvS~DNrRl~AAr~agi~-Vp~~~~~~~d~   73 (74)
</pre><pre>                      ++++.+.|.....+-... .++++++++.|-...|+-|...+  +.-++---+.|+.||+.+|.+ +++.+++.+++
</pre><pre>T Consensus        19 ~~i~~~~i~~he~~~~~~~~~l~~~i~~~g~~~~Pi~v~~~~~~~~~~i~dGhhR~~a~~~~~~~~i~~~v~~~~~~   95 (242)
</pre><pre>T 1VK1_A           19 VFIELDKMKPHEQLVQRELEDFIESVTGSGIFWKPMLLAKIPGTDEYLIVDGHHRWAGLQKLGAKRAPSVILDYFDE   95 (242)
</pre><pre>T ss_dssp             EEEEGGGEECSBCCCHHHHHHHHHHHHHHCEECSCEEEEECTTSSCEEEEECHHHHHHHHHHTCCEEEEEEECTTST
</pre><pre>T ss_pred             EEEeHHHCCCCchhhHHHHHHHHHHHHhcCCccCCEEEEECCCCCeEEEeCCHHHHHHHHHhCCCEEeEEEEcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>TIGR03734 PRTRC_parB; PRTRC system ParB family protein. A novel genetic system characterized by six major proteins, included a ParB homolog and a ThiF homolog, is designated PRTRC, or ParB-Related,ThiF-Related Cassette.
</pre><pre>Probab=94.71  E-value=0.34  Score=40.11  Aligned_cols=53  Identities=25%  Similarity=0.317  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCC--EEecchhHHHHHHHcCCC---CeeEEeCccc
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGS--LTTLDNTRVAAAREAGID---VQATVRNYDD   72 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~--lvS~DNrRl~AAr~agi~---Vp~~~~~~~d   72 (74)
</pre><pre>                      .+++++|+..|.. .||.|...++|.  .+---+||++|++.+|.+   ||+.+...++
</pre><pre>T Consensus        18 ~~La~SI~~~G~l-~pi~V~~~~~~~~y~ii~G~rR~~A~~~l~~~~~~vp~~v~~~~d   75 (554)
</pre><pre>T TIGR03734        18 AELVESIRAKGVL-QPILVRPVPGSDLYEVVAGERRYRAALEVFGEDYDIPALIKVLTD   75 (554)
</pre><pre>T ss_pred             HHHHHHHHHhCCC-CCEEEEeCCCCCcEEEEeCHHHHHHHHHhcCCCCCceEEEecCCH
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>1R71_C Transcriptional repressor protein korB; IncP, plasmid partitioning, protein-DNA complex, heilx-turn-helix motif, transcription factor, ParB homologue, TRANSCRIPTION-DNA COMPLEX; HET: BRU; 2.2A {Escherichia coli} SCOP: a.4.14.1
</pre><pre>Probab=75.22  E-value=4.6  Score=27.40  Aligned_cols=18  Identities=28%  Similarity=0.374  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHcCCC-CeeEEeC
</pre><pre>Q MBC1768434.1/3   52 RVAAAREAGID-VQATVRN   69 (74)
</pre><pre>Q Consensus        52 Rl~AAr~agi~-Vp~~~~~   69 (74)
</pre><pre>                      |++||+.+|++ |++.++.
</pre><pre>T Consensus         1 R~~A~~~~g~~~i~~~V~~   19 (178)
</pre><pre>T 1R71_C            1 RYRGSKWAGKKSIPAFIDN   19 (178)
</pre><pre>T ss_dssp             -------------------
</pre><pre>T ss_pred             ChHHHHHcCCCeeCEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>d3awub_ d.386.1.1 (B:) Tyrosinase cofactor MelC1 {Streptomyces castaneoglobisporus [TaxId: 79261]}
</pre><pre>Probab=71.08  E-value=8.5  Score=26.62  Aligned_cols=20  Identities=25%  Similarity=0.443  Sum_probs=0.0  Template_Neff=4.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECCCCCEEecch
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMPDGSLTTLDN   50 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~DG~lvS~DN   50 (74)
</pre><pre>                      ||.||++++.+||.+.|.-|
</pre><pre>T Consensus        38 DGr~Lhlmr~aDG~ylS~vn   57 (84)
</pre><pre>T d3awub_          38 DGVQLHVMRNADGSWISVVS   57 (84)
</pre><pre>T ss_dssp             TTEEECEEECTTSCEEETTE
</pre><pre>T ss_pred             CCEEceEEECCCCCEEEcee
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF06236.13 ; MelC1 ; Tyrosinase co-factor MelC1
</pre><pre>Probab=63.31  E-value=22  Score=25.67  Aligned_cols=29  Identities=38%  Similarity=0.462  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECCCCCEEecch------hHHHHHHHc
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMPDGSLTTLDN------TRVAAAREA   59 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~DG~lvS~DN------rRl~AAr~a   59 (74)
</pre><pre>                      ||.||+|++..||+++|.=|      .=+.+||.+
</pre><pre>T Consensus        63 DGr~Lhvmr~adG~yiS~v~hY~~f~s~~~lARaa   97 (109)
</pre><pre>T K4RFH5_STRDJ/1   63 DGVELHVMRNADGSWISVVSHYDPVATPRAAARAA   97 (109)
</pre><pre>T ss_pred             CCEEeeEEECCCCCEEeceeCCeeeCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>2P12_B Hypothetical protein DUF402; APC7392, beta-barrel, Rhodococcus sp. RHA1, structural genomics, PSI-2, protein structure initiative, midwest center for structural genomics; HET: MSE; 1.63A {Rhodococcus sp.} SCOP: b.175.1.1, l.1.1.1
</pre><pre>Probab=58.20  E-value=41  Score=24.87  Aligned_cols=27  Identities=22%  Similarity=0.337  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHcCC
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREAGI   61 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~agi   61 (74)
</pre><pre>                      |+|+..+||+..-+|-.-|.+|.++|+
</pre><pre>T Consensus       107 LDvvv~~~g~~~vlD~DEl~~A~~~g~  133 (176)
</pre><pre>T 2P12_B          107 LDLVVRTGRDTELLDVDELMEAHTTGL  133 (176)
</pre><pre>T ss_dssp             EEEEEETTTEEEEECHHHHHHHHHTTS
</pre><pre>T ss_pred             eEEEEeCCCcEEEEcHHHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>5Z0F_B MelC; tyrosinase, catalytic mechanism, OXIDOREDUCTASE-METAL BINDING PROTEIN complex; HET: NO3, DAH; 1.16A {Streptomyces castaneoglobisporus} SCOP: d.386.1.1
</pre><pre>Probab=56.11  E-value=37  Score=25.20  Aligned_cols=29  Identities=38%  Similarity=0.465  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECCCCCEEecch------hHHHHHHHc
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMPDGSLTTLDN------TRVAAAREA   59 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~DG~lvS~DN------rRl~AAr~a   59 (74)
</pre><pre>                      ||.||+|++..||++.|.=|      .=+.+||.|
</pre><pre>T Consensus        77 DGr~Lhvmr~~dG~y~S~v~hY~~f~s~~~~ARaa  111 (134)
</pre><pre>T 5Z0F_B           77 DGVQLHVMRNADGSWISVVSHXDPVPTPRAAARAA  111 (134)
</pre><pre>T ss_dssp             TTEEECEEECTTSCEEETTEEEEEESSHHHHHHHH
</pre><pre>T ss_pred             CCEECeEEECCCCCEEeCCCCCCCCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>d1s6ma_ d.89.1.5 (A:) automated matches {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=54.58  E-value=37  Score=27.20  Aligned_cols=25  Identities=24%  Similarity=0.423  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ..++..+||+..++||++|..++.+
</pre><pre>T Consensus       167 ~N~~~~~dGkw~ald~~~l~~~~~~  191 (293)
</pre><pre>T d1s6ma_         167 LNMTKRSDGQWRALKNDEIVKATRY  191 (293)
</pre><pre>T ss_dssp             ESCEECTTSCEECCBCHHHHTTHHH
</pre><pre>T ss_pred             EcEEECCCCCEEecchHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>7D8Q_A UPF0374 protein SAB1800c; cytosolic protein, nucleitide phosphatase; HET: GZF; 1.5A {Staphylococcus aureus RF122}
</pre><pre>Probab=54.10  E-value=55  Score=23.52  Aligned_cols=27  Identities=19%  Similarity=0.279  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHcCC
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREAGI   61 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~agi   61 (74)
</pre><pre>                      |+|+..|||...-.|-.-|.+|.++|.
</pre><pre>T Consensus       109 LDv~v~~dg~~~v~DedEf~~~~~~g~  135 (180)
</pre><pre>T 7D8Q_A          109 LDIKVYPNGKYHLLDEDEYEQHMNQMN  135 (180)
</pre><pre>T ss_dssp             EEEEECTTSCEEEECHHHHHHHHHHHT
</pre><pre>T ss_pred             EEEEECCCCcEEEcCHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>d2r6sa_ b.82.2.3 (A:) automated matches {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=54.08  E-value=16  Score=27.56  Aligned_cols=52  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCc----HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3    8 IRFSQTSVNGS----DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus         8 I~FSQ~sVs~~----~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      |+|+...+...    .+..+.+.+--.+..-..-+.+.-|.+.-+||+|++-+|.+
</pre><pre>T Consensus       227 ir~~~~~~~~~~~~~~~al~~l~~~~~~~~~~~~~~l~~GDlli~dN~~vlHgR~~  282 (311)
</pre><pre>T d2r6sa_         227 MRYIDQFVQPKDFEEGVWLSELSDAIETSKGILSVPVPVGKFLLINNLFWLHGRDR  282 (311)
</pre><pre>T ss_dssp             ECCCTTTBCCCSHHHHHHHHHHHHHHHTCTTCEEECCCTTCEEEEETTTEEEEECC
</pre><pre>T ss_pred             EEEecccCCCCCHHHHHHHHHHHHHHhcCCCeEEEecCCCCEEEEeCceEeecCcc
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>5Z0E_B MelC; tyrosinase, catalytic mechanism, OXIDOREDUCTASE-METAL BINDING PROTEIN complex; HET: NO3; 1.16A {Streptomyces castaneoglobisporus} SCOP: d.386.1.1
</pre><pre>Probab=53.77  E-value=40  Score=25.00  Aligned_cols=29  Identities=38%  Similarity=0.465  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECCCCCEEecch------hHHHHHHHc
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMPDGSLTTLDN------TRVAAAREA   59 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~DG~lvS~DN------rRl~AAr~a   59 (74)
</pre><pre>                      ||.||+|++..||++.|.=|      .=+.+||.|
</pre><pre>T Consensus        77 DGr~Lhvmr~~dG~y~S~v~hY~~f~s~~~~ARaa  111 (134)
</pre><pre>T 5Z0E_B           77 DGVQLHVMRNADGSWISVVSHFDPVPTPRAAARAA  111 (134)
</pre><pre>T ss_dssp             TTEEECEEECTTSCEEETTEEEEEESSHHHHHHHH
</pre><pre>T ss_pred             CCEECeEEECCCCCEEEceeCCccCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>d3l57a_ d.89.1.0 (A:) automated matches {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=53.72  E-value=34  Score=26.29  Aligned_cols=25  Identities=16%  Similarity=0.343  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ..++..+||+..++||++|...+.+
</pre><pre>T Consensus       166 ~N~~~~~DGkw~ald~~~l~~~~~~  190 (225)
</pre><pre>T d3l57a_         166 MNMTQREDGQWRALKNDELMRNKMH  190 (225)
</pre><pre>T ss_dssp             ESCEECTTSCEECCCTHHHHTTHHH
</pre><pre>T ss_pred             ccceeCCCCCEEecchHHHHhhHHH
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>5ZDM_A FomD; metalloprotein, hydrolase; HET: GOL; 1.38A {Streptomyces fradiae}
</pre><pre>Probab=53.09  E-value=56  Score=24.43  Aligned_cols=27  Identities=22%  Similarity=0.260  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCC-CEEecchhHHHHHHHcCC
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDG-SLTTLDNTRVAAAREAGI   61 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG-~lvS~DNrRl~AAr~agi   61 (74)
</pre><pre>                      |+|+..+|| .+.-.|-.-|.+|...|.
</pre><pre>T Consensus       132 LDvvv~~dg~~~~v~DeDE~~~a~~~g~  159 (211)
</pre><pre>T 5ZDM_A          132 IDLVVPADSRTFRWKDVEKFEERARIGH  159 (211)
</pre><pre>T ss_dssp             EEEEEETTCSCCEEESHHHHHHHHHTTS
</pre><pre>T ss_pred             ceEEEcCCCCcEEECCHHHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>2R6S_A Gab protein; jelly-roll motif, UNKNOWN FUNCTION; HET: SO4, GOL, BCN; 2.1A {Escherichia coli} SCOP: b.82.2.3
</pre><pre>Probab=52.68  E-value=22  Score=26.77  Aligned_cols=52  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCc----HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3    8 IRFSQTSVNGS----DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus         8 I~FSQ~sVs~~----~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ++|++..+.+.    .+..+.+.+--....-..-+.+..|.++-+||+|..=+|.+
</pre><pre>T Consensus       241 ~r~~~~~~~~~~~~~~~al~~l~~~~~~~~~~~~~~l~~GDlli~dN~~~lH~R~~  296 (325)
</pre><pre>T 2R6S_A          241 MRYIDQFVQPKDFEEGVWLSELSDAIETSKGILSVPVPVGKFLLINNLFWLHGRDR  296 (325)
</pre><pre>T ss_dssp             ECCCTTTBCCCSHHHHHHHHHHHHHHHTCTTCEEECCCTTCEEEEETTTEEEEECC
</pre><pre>T ss_pred             EEEecCccCCCCHHHHHHHHHHHHHHHcCCCceeeecCCCCEEEEeCceEeccccc
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>d2p12a1 b.175.1.1 (A:8-172) Hypothetical protein RHA1_ro00977 {Rhodococcus sp. RHA1 [TaxId: 101510]}
</pre><pre>Probab=45.70  E-value=86  Score=23.25  Aligned_cols=27  Identities=22%  Similarity=0.337  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHcCC
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREAGI   61 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~agi   61 (74)
</pre><pre>                      |+|+..+||...-+|--=|.+|.++|+
</pre><pre>T Consensus        98 LDvvv~~dg~~~ilD~DEl~~a~~~g~  124 (165)
</pre><pre>T d2p12a1          98 LDLVVRTGRDTELLDVDELMEAHTTGL  124 (165)
</pre><pre>T ss_dssp             EEEEEETTTEEEEECHHHHHHHHHTTS
</pre><pre>T ss_pred             CEEEEeCCCcEEEeCHHHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF18762.3 ; Kinase-PolyVal ; Serine/Threonine/Tyrosine Kinase found in polyvalent proteins
</pre><pre>Probab=45.46  E-value=66  Score=23.63  Aligned_cols=41  Identities=15%  Similarity=0.203  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCc-----HHHHHHHHHcCCCC------------------CCceEEECCCCCEEecc
</pre><pre>Q MBC1768434.1/3    8 IRFSQTSVNGS-----DKIIASMKANGWKG------------------DPIDVVKMPDGSLTTLD   49 (74)
</pre><pre>Q Consensus         8 I~FSQ~sVs~~-----~~~~~~M~~~gW~g------------------~pi~Vv~~~DG~lvS~D   49 (74)
</pre><pre>                      |-.||..|.+.     .++.+-|...||..                  .|-+|+.. ||.|.-+|
</pre><pre>T Consensus        90 ~V~sQp~i~g~~~~t~~ei~~~m~~~Gf~~~~~~~~~~~~~~~~v~D~hp~Nv~~~-~g~l~pID  153 (160)
</pre><pre>T F9YRM4_CAPCC/4   90 SVVKQEFIQTDEITDTQQVKRFLEYNNFKNVRHNDYISSELGLILEDLHDENVLTK-NNVLYFID  153 (160)
</pre><pre>T ss_pred             EEEEeeeecCCCCCCHHHHHHHHHHCCCeeccccceeeCCCcEEEEeCCcccEEee-CCEEEEcE
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>6J2U_A Tyrosinase co-factor protein; Citrate synthase, Metallosphaera sedula, OXIDOREDUCTASE; 1.3A {Streptomyces avermitilis} SCOP: d.386.1.0
</pre><pre>Probab=41.93  E-value=52  Score=23.94  Aligned_cols=20  Identities=25%  Similarity=0.381  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECCCCCEEecch
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMPDGSLTTLDN   50 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~DG~lvS~DN   50 (74)
</pre><pre>                      ||.||+|+...||++.|.=|
</pre><pre>T Consensus        71 DGr~Lhvmr~~dG~y~S~~~   90 (118)
</pre><pre>T 6J2U_A           71 DGMELHVMQNVDGSWISVVS   90 (118)
</pre><pre>T ss_dssp             TTEEECEEECTTSCEEETTE
</pre><pre>T ss_pred             CCEEceEEECCCCCEEEcee
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>1JR7_A HYPOTHETICAL 37.4 KDA PROTEIN IN ILEY-GABD INTERGENIC REGION; GAMMA AMINO-BUTYRIC ACID METABOLISM, GABA, FERROUS OXYGENASE, 2-OXOGLUTARATE, OXIDOREDUCTASE, ANTIBIOTICS BIOSYNTHESIS, OXYGENASE, B-LACTAM, STRUCTURAL GENOMICS, PSI; 2.0A {Escherichia coli} SCOP: b.82.2.3
</pre><pre>Probab=41.62  E-value=36  Score=25.36  Aligned_cols=52  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCc----HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3    8 IRFSQTSVNGS----DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus         8 I~FSQ~sVs~~----~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      |+|+...+.+.    .+..+.+.+.-.+.....-+.+.-|.+.-+||.|+.=+|.+
</pre><pre>T Consensus       227 ~r~~~~~~~~~~~~~~~al~~l~~~~~~~~~~~~~~l~~Gdlli~dN~~~lHgR~~  282 (311)
</pre><pre>T 1JR7_A          227 MRYIDQFVQPKDFEEGVWLSELSDAIETSKGILSVPVPVGKFLLINNLFWLHGRDR  282 (311)
</pre><pre>T ss_dssp             ECCCTTTEECCSHHHHHHHHHHHHHHHTCTTCEEECCCTTCEEEEETTTEEEEECC
</pre><pre>T ss_pred             EEEecCccCCCCHHHHHHHHHHHHHHHcCCCceeecCCCCCEEEEECCeEeecccc
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>6I3G_A ABC transporter, substrate-binding protein, family 5; ABC Transporter, Solute binding protein, AppA, PEPTIDE BINDING PROTEIN; 2.0A {Clostridioides difficile}
</pre><pre>Probab=39.23  E-value=2.7e+02  Score=21.86  Aligned_cols=71  Identities=13%  Similarity=0.088  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHcccccccCCCc----HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHH-HHHcCCCCeeEEeCccc
</pre><pre>Q MBC1768434.1/3    2 NVNPSKIRFSQTSVNGS----DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAA-AREAGIDVQATVRNYDD   72 (74)
</pre><pre>Q Consensus         2 ~v~P~~I~FSQ~sVs~~----~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~A-Ar~agi~Vp~~~~~~~d   72 (74)
</pre><pre>                      .+.|..-.|........    ....+.|++.||...++.+....+......--..|.. .+++||+|.+...+..+
</pre><pre>T Consensus       295 ~~~~~~~~~~~~~~~~~~~d~~~A~~ll~~ag~~~~~l~l~~~~~~~~~~~~~~~i~~~l~~~Gi~v~~~~~~~~~  370 (490)
</pre><pre>T 6I3G_A          295 ILVPEADFYTEEGVEGYDQDTNKAKDLLDKSGVKIDKLKIGYNTGRFGHKNYALVAQQELKKIGIEAEIVPYESKA  370 (490)
</pre><pre>T ss_dssp             SSCTTSTTCCCTTCCCCCCCHHHHHHHHHHTCCCCSEEEEEEETTSTTHHHHHHHHHHHHHTTTCEEEEEEECHHH
</pre><pre>T ss_pred             CCCCCcccCCCCCCCCCCCCHHHHHHHHHHcCCccCceEEEEeCCCcchhHHHHHHHHHHHHhCCccEEEecChHH
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>d2y5ca1 d.15.4.0 (A:4-109) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=39.08  E-value=1.6e+02  Score=18.76  Aligned_cols=32  Identities=22%  Similarity=0.428  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCceEEECC-CCCEEec----chhHHHHHHHcCCCCe
</pre><pre>Q MBC1768434.1/3   33 DPIDVVKMP-DGSLTTL----DNTRVAAAREAGIDVQ   64 (74)
</pre><pre>Q Consensus        33 ~pi~Vv~~~-DG~lvS~----DNrRl~AAr~agi~Vp   64 (74)
</pre><pre>                      +...|.... ||+...+    +-+-|.+|+++|+.++
</pre><pre>T Consensus         1 ~~~~v~~~~~~g~~~~~~~~~g~~ll~~~~~~g~~i~   37 (106)
</pre><pre>T d2y5ca1           1 DVVNVVFVDRSGQRIPVSGRVGDNVLHLAQRHGVDLE   37 (106)
</pre><pre>T ss_dssp             CEEEEEEECTTSCEEEEEEETTCBHHHHHHHTTCCCC
</pre><pre>T ss_pred             CceEEEEECCCCCEEEEEEeCCCcHHHHHHHCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>3EXM_A Phosphatase SC4828; nucleoside diphosphatase, GDP/UDP'ase, Streptomyces, non-hydrolysable GDP analogue, lipocalcin fold, metalloprotein, HYDROLASE, Structural Genomics, PSI-2, Protein Structure; HET: GP2; 1.65A {Streptomyces coelicolor A3(2)}
</pre><pre>Probab=38.72  E-value=1.3e+02  Score=23.22  Aligned_cols=27  Identities=26%  Similarity=0.380  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHcCC
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREAGI   61 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~agi   61 (74)
</pre><pre>                      |+++..+||.+.-.|-.-|..|...|.
</pre><pre>T Consensus       153 LDlvv~pdg~~~v~DeDEf~~a~~~g~  179 (237)
</pre><pre>T 3EXM_A          153 LDISVHPDRTWHWRDEDEFAQALRDGL  179 (237)
</pre><pre>T ss_dssp             EEEEECTTSCEEEECHHHHHHHHHTTS
</pre><pre>T ss_pred             eEEEECCCCCEEEecHHHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>1OMH_A trwC protein; PROTEIN-DNA COMPLEX, BACTERIAL CONJUGATION, RELAXASE, DNA REPLICATION, TRANSFERASE-DNA COMPLEX; HET: SO4, MSE; 1.95A {Escherichia coli} SCOP: d.89.1.5
</pre><pre>Probab=36.86  E-value=1e+02  Score=24.20  Aligned_cols=25  Identities=24%  Similarity=0.423  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ...+.-+||+..++||++|+..+.+
</pre><pre>T Consensus       167 ~n~~~~~dG~~~ald~~~l~~~~~~  191 (293)
</pre><pre>T 1OMH_A          167 LNMTKRSDGQWRALKNDEIVKATRY  191 (293)
</pre><pre>T ss_dssp             ESCEECTTSCEECCBCHHHHHTHHH
</pre><pre>T ss_pred             eceEECCCCCEEeCchhHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>2JV8_A Uncharacterized protein NE1242; solution structure, NESG, STRUCTURAL GENOMICS, UNKNOWN FUNCTION, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium; NMR {Nitrosomonas europaea ATCC 19718}
</pre><pre>Probab=36.34  E-value=59  Score=21.64  Aligned_cols=20  Identities=15%  Similarity=0.126  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECC-CCCEEecch
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMP-DGSLTTLDN   50 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~-DG~lvS~DN   50 (74)
</pre><pre>                      +|.||+|+..+ ||++.|--|
</pre><pre>T Consensus        24 DG~~l~v~~~~~dG~y~s~~~   44 (73)
</pre><pre>T 2JV8_A           24 NGKEISYVHDAVKNKWSSRYL   44 (73)
</pre><pre>T ss_dssp             TTEECCCCCCSSSCCCCCTTS
</pre><pre>T ss_pred             CCEEeceEEeCCCCeEEeccc
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>4ZE8_A ABC transporter, substrate binding protein (Agrocinopines A and B); PBP FROM C CLUSTER, TRANSPORT PROTEIN; HET: SO4, GOL, EDO; 1.72A {Agrobacterium tumefaciens (strain C58 / ATCC 33970)}
</pre><pre>Probab=36.18  E-value=2.4e+02  Score=22.24  Aligned_cols=52  Identities=15%  Similarity=0.199  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECC--CCCEEecchhHHHHHHHcCCCCeeEEeCc
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMP--DGSLTTLDNTRVAAAREAGIDVQATVRNY   70 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~--DG~lvS~DNrRl~AAr~agi~Vp~~~~~~   70 (74)
</pre><pre>                      ..-.+.+++.||++.++.+....  +.....+-..=-...+++||+|.+...+.
</pre><pre>T Consensus       323 ~~A~~ll~~ag~~g~~l~l~~~~~~~~~~~~~~~~i~~~l~~~Gi~v~~~~~~~  376 (499)
</pre><pre>T 4ZE8_A          323 EEAKRLVKESGYDGTPITYHTMGNYYANAMPALMMMIEMWKQIGVNVVMKTYAP  376 (499)
</pre><pre>T ss_dssp             HHHHHHHHHSSCCSCCEEEEECTTSSTTHHHHHHHHHHHHHTTTCCEEEEECCT
</pre><pre>T ss_pred             HHHHHHHHHhCCCCCCeEEEEecCCCccchHHHHHHHHHHHHcCceEEEEecCc
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>4ZE8_D ABC transporter, substrate binding protein (Agrocinopines A and B); PBP FROM C CLUSTER, TRANSPORT PROTEIN; HET: GOL, SO4, EDO; 1.72A {Agrobacterium tumefaciens (strain C58 / ATCC 33970)}
</pre><pre>Probab=36.18  E-value=2.4e+02  Score=22.24  Aligned_cols=52  Identities=15%  Similarity=0.199  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECC--CCCEEecchhHHHHHHHcCCCCeeEEeCc
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMP--DGSLTTLDNTRVAAAREAGIDVQATVRNY   70 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~--DG~lvS~DNrRl~AAr~agi~Vp~~~~~~   70 (74)
</pre><pre>                      ..-.+.+++.||++.++.+....  +.....+-..=-...+++||+|.+...+.
</pre><pre>T Consensus       323 ~~A~~ll~~ag~~g~~l~l~~~~~~~~~~~~~~~~i~~~l~~~Gi~v~~~~~~~  376 (499)
</pre><pre>T 4ZE8_D          323 EEAKRLVKESGYDGTPITYHTMGNYYANAMPALMMMIEMWKQIGVNVVMKTYAP  376 (499)
</pre><pre>T ss_dssp             HHHHHHHHHSCCCSCCEEEEECTTSSTTHHHHHHHHHHHHHTTTCCEEEEECCT
</pre><pre>T ss_pred             HHHHHHHHHhCCCCCCeEEEEecCCCccchHHHHHHHHHHHHcCceEEEEecCc
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>6CKA_B Paratox; prophage, natural competence, Streptococcus, protein binding, VIRAL PROTEIN; HET: MSE; 1.559A {Streptococcus pyogenes serotype M3}
</pre><pre>Probab=34.57  E-value=1.6e+02  Score=20.25  Aligned_cols=29  Identities=17%  Similarity=0.375  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEEec
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLTTL   48 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~   48 (74)
</pre><pre>                      .++.+.+.+|--.++-+.||+- |||+.-|
</pre><pre>T Consensus         5 ~efkqAId~G~I~~dtV~IVRk-nGkIfDy   33 (68)
</pre><pre>T 6CKA_B            5 DEFKEAIDKGYILGDTVAIVRK-NGKIFDY   33 (68)
</pre><pre>T ss_dssp             HHHHHHHHTTSSCSSEEEEEEE-TTEEEEE
</pre><pre>T ss_pred             HHHHHHHHcCCCCCCeEEEEEE-CCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>4C5W_A GAMMA-BUTYROBETAINE DIOXYGENASE; OXIDOREDUCTASE; HET: NM3, OGA; 1.7A {HOMO SAPIENS} SCOP: b.82.2.5, d.392.1.0
</pre><pre>Probab=33.88  E-value=29  Score=26.88  Aligned_cols=22  Identities=36%  Similarity=0.531  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             EECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   38 VKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        38 v~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      +.+..|.++-+||+|+.-+|.+
</pre><pre>T Consensus       330 ~~l~~Gd~li~DN~r~lHgR~~  351 (387)
</pre><pre>T 4C5W_A          330 FKMNPGDVITFDNWRLLHGRRS  351 (387)
</pre><pre>T ss_dssp             ECCCTTCEEEEETTTEEEEECC
</pre><pre>T ss_pred             ecCCCCCEEEEECCceeecccc
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>PF08751.13 ; TrwC ; TrwC relaxase
</pre><pre>Probab=33.37  E-value=1.3e+02  Score=23.67  Aligned_cols=25  Identities=20%  Similarity=0.433  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ...+..+||++.++|+++|...+.+
</pre><pre>T Consensus       161 ~n~~~~~dG~~~ald~~~l~~~~~~  185 (283)
</pre><pre>T I0ILY3_LEPFC/9  161 MNITKNSDGNYQAMTNDRMMNYQRL  185 (283)
</pre><pre>T ss_pred             eceEECCCCCEEecCcHHHHHhHHH
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>d4cvya_ b.82.2.0 (A:) automated matches {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=32.66  E-value=38  Score=24.94  Aligned_cols=44  Identities=9%  Similarity=0.009  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID   62 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~   62 (74)
</pre><pre>                      ..+.+.+..--.+..-..-+.+..|+++-+||.|+.-+|.+..+
</pre><pre>T Consensus       212 ~~~~~~l~~~~~~~~~~~~~~~~~Gdili~DN~r~lHgR~~~~~  255 (289)
</pre><pre>T d4cvya_         212 RVLFEVLQRRITMPENTIRWNWAPGDVAIWDNRATQHRAIDDYD  255 (289)
</pre><pre>T ss_dssp             HHHHHHHHHHHHCGGGEEEECCCTTCEEEEETTSEEEEECCCST
</pre><pre>T ss_pred             HHHHHHHHHHhcCcccEEEEccCCCCEEEEeCCCEEEEecCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>3L6T_B Mobilization protein TraI; TrwC Superfamily of relaxase enzymes, conjugative relaxase, mob class relaxase, conjugal nickase, hydrolase, histidine triad, HUH+H; HET: PO4, CIT, EDO; 1.93A {Escherichia coli} SCOP: l.1.1.1, d.89.1.0
</pre><pre>Probab=30.80  E-value=1.3e+02  Score=23.74  Aligned_cols=25  Identities=16%  Similarity=0.343  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ...+..+||+..++||++|...+.+
</pre><pre>T Consensus       169 ~n~~~~~dG~~~ald~~~l~~~~~~  193 (302)
</pre><pre>T 3L6T_B          169 MNMTQREDGQWRALKNDELMRNKMH  193 (302)
</pre><pre>T ss_dssp             ESCEECSSSCEECCCCHHHHHTHHH
</pre><pre>T ss_pred             EeeEECCCCCEEecCcHHHHhhHhH
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>d1otja_ b.82.2.5 (A:) Taurine/alpha-ketoglutarate dioxygenase TauD {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=29.76  E-value=53  Score=23.96  Aligned_cols=41  Identities=12%  Similarity=0.019  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      .++.+.+.+.--+......+.+..|.+.-+||.|+.=+|.+
</pre><pre>T Consensus       218 ~~~~~~l~~~~~~~~~~~~~~~~~GDlli~dN~r~lHgR~~  258 (281)
</pre><pre>T d1otja_         218 EALLSFLFAHITKPEFQVRWRWQPNDIAIWDNRVTQHYANA  258 (281)
</pre><pre>T ss_dssp             HHHHHHHHHHTTCGGGEEEECCCTTCEEEEETTSEEEEECC
</pre><pre>T ss_pred             HHHHHHHHHHhhCcceEEEEEeCCCCEEEEeCCCeeEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>PF19129.2 ; DUF5812 ; Family of unknown function (DUF5812)
</pre><pre>Probab=29.00  E-value=73  Score=24.57  Aligned_cols=32  Identities=25%  Similarity=0.325  Sum_probs=0.0  Template_Neff=3.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCC-------EEecchhHHHHHHHcCCC-CeeE
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGS-------LTTLDNTRVAAAREAGID-VQAT   66 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~-------lvS~DNrRl~AAr~agi~-Vp~~   66 (74)
</pre><pre>                      |+|+.+|.+.       +..=+.++-+|||..|+. |+|+
</pre><pre>T Consensus       101 iHVltVp~~~te~Av~dVldDe~t~~RAARl~gv~RVEvR  140 (141)
</pre><pre>T A0A3M9VWG8_9EU  101 IHVITVPAAETSEAVQDVLEDDGTLERAGRMENVVRVEVR  140 (141)
</pre><pre>T ss_pred             eEEEEcChhhHHHHHHHHHccHhHHHHHhhccCeeEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>TIGR03054 photo_alph_chp1; putative photosynthetic complex assembly protein. In twenty or so anoxygenic photosynthetic alpha-Proteobacteria known so far, a gene for a member of this protein family is present and is found in the vicinity of puhA, which encodes a component of the photosynthetic reaction center, and other genes associated with photosynthesis.
</pre><pre>Probab=28.98  E-value=1.8e+02  Score=21.87  Aligned_cols=27  Identities=26%  Similarity=0.338  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCCCceEEECCCCCEEecc
</pre><pre>Q MBC1768434.1/3   23 ASMKANGWKGDPIDVVKMPDGSLTTLD   49 (74)
</pre><pre>Q Consensus        23 ~~M~~~gW~g~pi~Vv~~~DG~lvS~D   49 (74)
</pre><pre>                      +.++.+-=...|.+++.+.||+|+-.|
</pre><pre>T Consensus        85 eR~~~g~~~~~Pf~L~~~~dGrLtL~D  111 (135)
</pre><pre>T TIGR03054        85 ARARAGVAAEPPFRLTRYDNGRLTLTD  111 (135)
</pre><pre>T ss_pred             HHHHcCCCCCCCeEEEEEeCCcEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>TIGR02410 carnitine_TMLD; trimethyllysine dioxygenase. Members of this family with known function act as trimethyllysine dioxygenase, an enzyme in the pathway for carnitine biosynthesis from lysine.
</pre><pre>Probab=28.91  E-value=35  Score=26.72  Aligned_cols=23  Identities=30%  Similarity=0.443  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             EEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   37 VVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        37 Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ...+..|.++-+||+|++-+|.+
</pre><pre>T Consensus       314 ~~~l~~Gd~ii~DN~rvLHgR~~  336 (362)
</pre><pre>T TIGR02410       314 EFKLRPGTVLIFDNWRVLHSRTS  336 (362)
</pre><pre>T ss_pred             EEEcCCCCEEEEeCCcEEEcCCc
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>PF08943.12 ; CsiD ; CsiD
</pre><pre>Probab=28.35  E-value=83  Score=23.00  Aligned_cols=52  Identities=13%  Similarity=0.093  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCc----HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3    8 IRFSQTSVNGS----DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus         8 I~FSQ~sVs~~----~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ++|....+...    .+..+.+.+.-.+..-...+.+..|.+.-+||+|+.-+|.+
</pre><pre>T Consensus       221 ~r~~~~~~~~~~~~~~~al~~l~~~~~~~~~~~~~~l~~Gd~li~dN~~~lH~R~~  276 (294)
</pre><pre>T GLAH_PSEPK/21-  221 MRYIDQFVQPENYEEGIWLNALSESLEGSGKKVSVPVGVGSFLLINNLFWLHGRDR  276 (294)
</pre><pre>T ss_pred             EEEecCcCCCCCHHHHHHHHHHHHHHhcCCCcEEEeeCCCcEEEEECCeEeeccCc
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>PF03477.18 ; ATP-cone ; ATP cone domain
</pre><pre>Probab=27.35  E-value=1.8e+02  Score=14.53  Aligned_cols=20  Identities=20%  Similarity=0.323  Sum_probs=0.0  Template_Neff=14.100
</pre><pre>
</pre><pre>Q ss_pred             EEECCCCCEEecchhHHHHH
</pre><pre>Q MBC1768434.1/3   37 VVKMPDGSLTTLDNTRVAAA   56 (74)
</pre><pre>Q Consensus        37 Vv~~~DG~lvS~DNrRl~AA   56 (74)
</pre><pre>                      .+..+||....+|..++..+
</pre><pre>T Consensus         2 ~v~~~~~~~~~~~~~~~~~~   21 (91)
</pre><pre>T A5N3P6_CLOK5/2    2 HVVKRDGRKVEFNSIKITNA   21 (91)
</pre><pre>T ss_pred             ceecCCCCeecCCHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>PF18419.3 ; ATP-grasp_6 ; ATP-grasp-like domain
</pre><pre>Probab=26.48  E-value=1.4e+02  Score=17.34  Aligned_cols=13  Identities=23%  Similarity=0.414  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCCCee
</pre><pre>Q MBC1768434.1/3   53 VAAAREAGIDVQA   65 (74)
</pre><pre>Q Consensus        53 l~AAr~agi~Vp~   65 (74)
</pre><pre>                      +++|...||+|++
</pre><pre>T Consensus        11 ~~~A~~~Gi~v~~   23 (54)
</pre><pre>T H6LCP6_ACEWD/5   11 IKEALTRGVKVTP   23 (54)
</pre><pre>T ss_pred             HHHHHHcCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>7O81_AU EDF1; Frameshift, virus, pseudoknot, RIBOSOME; HET: G7M, A2M, UNX, HIC, 1MA, AME, PSU, OMC, 4AC, 5MC, OMG, 5MU, B8N, SAC, NMM, GTP, MG, UY1, MA6, 6MZ, M3L, MLZ, V5N, UR3, OMU, SPD; 3.1A {Oryctolagus cuniculus}
</pre><pre>Probab=26.27  E-value=1.7e+02  Score=20.15  Aligned_cols=35  Identities=20%  Similarity=0.292  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCceEEECCCCCEEecch-hHHHHHHHcCCCCee
</pre><pre>Q MBC1768434.1/3   31 KGDPIDVVKMPDGSLTTLDN-TRVAAAREAGIDVQA   65 (74)
</pre><pre>Q Consensus        31 ~g~pi~Vv~~~DG~lvS~DN-rRl~AAr~agi~Vp~   65 (74)
</pre><pre>                      +|+|..|+.-...+.+..=. .-+.+|+.+|+.+..
</pre><pre>T Consensus         5 ~w~~~~~~~~~~~~~~~~~~~~~~~~a~~~G~~~~t   40 (148)
</pre><pre>T 7O81_AU           5 DWDTVTVLRKKGPTAAQAKSKQAILAAQRRGDDVET   40 (148)
</pre><pre>T ss_pred             CCccceecccCCCChhhhcCHHHHHHHHHhCCccch
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>TIGR03849 arch_ComA; phosphosulfolactate synthase. This model finds the ComA (Coenzyme M biosynthesis A) protein, phosphosulfolactate synthase, in methanogenic archaea.
</pre><pre>Probab=25.67  E-value=3.2e+02  Score=22.30  Aligned_cols=44  Identities=25%  Similarity=0.382  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHc---CCCCeeEE
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREA---GIDVQATV   67 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~a---gi~Vp~~~   67 (74)
</pre><pre>                      .+|.+.+++-|.+     +++++||.+.--+..|++.-+.+   |++|..++
</pre><pre>T Consensus        74 ~~yl~~~k~lGf~-----~vEiS~G~i~l~~~~~~~lI~~~~~~g~~v~~Ev  120 (237)
</pre><pre>T TIGR03849        74 DEYLNECDELGFE-----AVEISDGSMEISLEERCNLIERAKDNGFMVLSEV  120 (237)
</pre><pre>T ss_pred             HHHHHHHHHhCCC-----EEEEcCCccCCCHHHHHHHHHHHHHCCCEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>6NPC_A TmpA, 2-trimethylaminoethylphosphonate hydroxylase; organophosphonate, iron oxygenase, HD-domain, gene annotation, enzyme, OXIDOREDUCTASE; HET: KVV, AKG; 1.7A {Leisingera caerulea}
</pre><pre>Probab=25.54  E-value=47  Score=26.08  Aligned_cols=22  Identities=41%  Similarity=0.463  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   38 VKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        38 v~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      ..+..|.++-+||+|+.=+|.+
</pre><pre>T Consensus       330 ~~l~~Gd~li~dN~rvlHgR~~  351 (383)
</pre><pre>T 6NPC_A          330 FKLEPGESFIVDNTRVLHARLG  351 (383)
</pre><pre>T ss_dssp             ECCCTTCEEEEETTTEEEEECC
</pre><pre>T ss_pred             EeCCCCCEEEEeCCceeecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>PF13224.8 ; DUF4032 ; Domain of unknown function (DUF4032)
</pre><pre>Probab=25.12  E-value=1.8e+02  Score=21.34  Aligned_cols=22  Identities=27%  Similarity=0.424  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHcCCC-CeeEEeCccc
</pre><pre>Q MBC1768434.1/3   51 TRVAAAREAGID-VQATVRNYDD   72 (74)
</pre><pre>Q Consensus        51 rRl~AAr~agi~-Vp~~~~~~~d   72 (74)
</pre><pre>                      .|+..|++.|.+ +++.+....+
</pre><pre>T Consensus        22 hRvs~a~~lG~~~iea~v~~~~~   44 (165)
</pre><pre>T C8X9W5_NAKMY/2   22 QRLRKLNELGFDVAQIHINEHDG   44 (165)
</pre><pre>T ss_pred             HHHHHHHHcCCcEEEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>4S36_A Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; 1.46A {Pseudomonas aeruginosa}
</pre><pre>Probab=24.87  E-value=1.6e+02  Score=18.34  Aligned_cols=15  Identities=20%  Similarity=0.381  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEECCCCCEEecc
</pre><pre>Q MBC1768434.1/3   35 IDVVKMPDGSLTTLD   49 (74)
</pre><pre>Q Consensus        35 i~Vv~~~DG~lvS~D   49 (74)
</pre><pre>                      ..++.++||...+||
</pre><pre>T Consensus         2 ~~~~~~~dG~~i~yd   16 (90)
</pre><pre>T 4S36_A            2 SETWRFDDGASLSYD   16 (90)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             ceEEEeCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>TIGR02409 carnitine_bodg; gamma-butyrobetaine hydroxylase. Members of this protein family are gamma-butyrobetaine hydroxylase, both bacterial and eukarytotic.
</pre><pre>Probab=24.60  E-value=37  Score=26.37  Aligned_cols=22  Identities=45%  Similarity=0.664  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             EECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   38 VKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        38 v~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      +.+..|.++-+||+|++=+|.+
</pre><pre>T Consensus       316 ~~l~~Gd~vi~dN~rvlHgR~~  337 (366)
</pre><pre>T TIGR02409       316 FKLEPGDLILFDNTRLLHARDA  337 (366)
</pre><pre>T ss_pred             EecCCCCEEEEeCCcEEEccCc
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>d5wqsa_ c.1.8.1 (A:) beta-Amylase {Sweet potato (Ipomoea batatas) [TaxId: 4120]}
</pre><pre>Probab=24.47  E-value=1.4e+02  Score=26.42  Aligned_cols=33  Identities=24%  Similarity=0.467  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             CceEEECCCCCEEecc--hhHHHHHHHcCCC-CeeEE
</pre><pre>Q MBC1768434.1/3   34 PIDVVKMPDGSLTTLD--NTRVAAAREAGID-VQATV   67 (74)
</pre><pre>Q Consensus        34 pi~Vv~~~DG~lvS~D--NrRl~AAr~agi~-Vp~~~   67 (74)
</pre><pre>                      |++++.. ++.+..-+  -..|++++++|++ |.+-+
</pre><pre>T Consensus        20 pl~~v~~-~~~~~~~~~~~~~L~~lk~aGv~~V~v~V   55 (498)
</pre><pre>T d5wqsa_          20 PLGVVNA-DNVFPDKEKVEDELKQVKAGGCDGVMVDV   55 (498)
</pre><pre>T ss_dssp             CTTSSCT-TSCCCCHHHHHHHHHHHHHTTCCEEEEEE
</pre><pre>T ss_pred             ccceeeC-CCCCCCHHHHHHHHHHHHHCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>d3pvja_ b.82.2.5 (A:) automated matches {Pseudomonas putida [TaxId: 160488]}
</pre><pre>Probab=24.32  E-value=1.8e+02  Score=21.05  Aligned_cols=35  Identities=11%  Similarity=0.079  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEEecchhHH
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRV   53 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl   53 (74)
</pre><pre>                      ..+.+.+.+.-........+.+..|.+.-+||.|+
</pre><pre>T Consensus       215 ~~~~~~l~~~~~~~~~~~~~~~~~GD~li~dN~~~  249 (275)
</pre><pre>T d3pvja_         215 DALLRLLFAHATRPEFSIRWRWQENDVAFWDNRVT  249 (275)
</pre><pre>T ss_dssp             HHHHHHHHHHHTSGGGEEEECCCTTCEEEEETTSE
</pre><pre>T ss_pred             HHHHHHHHHHhhCcceEEEEeeCCCCEEEEECCcE
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>PF19064.2 ; DUF5760 ; Family of unknown function (DUF5760)
</pre><pre>Probab=24.14  E-value=3.4e+02  Score=17.89  Aligned_cols=23  Identities=26%  Similarity=0.462  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEE
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLT   46 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lv   46 (74)
</pre><pre>                      ..+.+-|+.++.+     ++.++||+|.
</pre><pre>T Consensus        10 ~~I~~~M~~n~i~-----~i~i~~gkL~   32 (84)
</pre><pre>T D8LP92_ECTSI/3   10 GRIARHMHENGIP-----ETISPSGTIK   32 (84)
</pre><pre>T ss_pred             HHHHHHHHHcCCC-----eEECCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>d1wdpa_ c.1.8.1 (A:) automated matches {Soybean (Glycine max) [TaxId: 3847]}
</pre><pre>Probab=23.63  E-value=1.7e+02  Score=25.89  Aligned_cols=33  Identities=33%  Similarity=0.575  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CceEEECCCCCEEecc--hhHHHHHHHcCCC-CeeEE
</pre><pre>Q MBC1768434.1/3   34 PIDVVKMPDGSLTTLD--NTRVAAAREAGID-VQATV   67 (74)
</pre><pre>Q Consensus        34 pi~Vv~~~DG~lvS~D--NrRl~AAr~agi~-Vp~~~   67 (74)
</pre><pre>                      |++++.. +|.+..-+  -..|++++++|++ |.+-+
</pre><pre>T Consensus        17 pl~~v~~-~~~~~~~~~~~~~L~~lk~aGv~~V~v~V   52 (493)
</pre><pre>T d1wdpa_          17 PLGVVNV-DNVFEDPDGLKEQLLQLRAAGVDGVMVDV   52 (493)
</pre><pre>T ss_dssp             CTTSBCT-TSCBCCHHHHHHHHHHHHHTTCCEEEEEE
</pre><pre>T ss_pred             CCCeeec-CCCcCCHHHHHHHHHHHHHcCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>d1twda_ c.1.30.1 (A:) Copper homeostasis protein CutC {Shigella flexneri [TaxId: 623]}
</pre><pre>Probab=23.59  E-value=6.4e+02  Score=19.71  Aligned_cols=56  Identities=21%  Similarity=0.310  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CCcHHHHHHHHHcCCCCCCceEEECCCCCEEecch-------hHHHHHHHcCCC-CeeEEeCcc---cC
</pre><pre>Q MBC1768434.1/3   16 NGSDKIIASMKANGWKGDPIDVVKMPDGSLTTLDN-------TRVAAAREAGID-VQATVRNYD---DL   73 (74)
</pre><pre>Q Consensus        16 s~~~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DN-------rRl~AAr~agi~-Vp~~~~~~~---d~   73 (74)
</pre><pre>                      .++.++.+.+++.  -.-|+.|+..|-|.-..+..       ..++.++++|.+ |-+-....+   |+
</pre><pre>T Consensus        36 tp~~~~i~~i~~~--~~ipv~vmirp~ggg~~~~~~e~~~m~~~i~~~~~~Gad~Vv~G~l~~~~~id~  102 (247)
</pre><pre>T d1twda_          36 TPSLGVLKSVRQR--VTIPVHPIIRPRGGDFCYSDGEFAAILEDVRTVRELGFPGLVTGVLDVDGNVDM  102 (247)
</pre><pre>T ss_dssp             CCCHHHHHHHHHH--CCSCEEEBCCSSSSCSCCCHHHHHHHHHHHHHHHHTTCSEEEECCBCTTSSBCH
</pre><pre>T ss_pred             CCCHHHHHHHHHH--CCCCEEEEEcCCCCCccCCHHHHHHHHHHHHHHHHcCCCEEEEecccCCCCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>1WDP_A Beta-amylase; (beta/alpha)8 barrel, Hydrolase; HET: SO4; 1.27A {Glycine max} SCOP: c.1.8.1
</pre><pre>Probab=23.55  E-value=1.6e+02  Score=25.61  Aligned_cols=33  Identities=33%  Similarity=0.571  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             CceEEECCCCCEEecch--hHHHHHHHcCCC-CeeEE
</pre><pre>Q MBC1768434.1/3   34 PIDVVKMPDGSLTTLDN--TRVAAAREAGID-VQATV   67 (74)
</pre><pre>Q Consensus        34 pi~Vv~~~DG~lvS~DN--rRl~AAr~agi~-Vp~~~   67 (74)
</pre><pre>                      |++++.. ++++...+.  .+|+.++++|++ |.+.+
</pre><pre>T Consensus        19 pl~~~~~-~~~~~~~~~~~~~l~~lk~aGv~~V~v~v   54 (495)
</pre><pre>T 1WDP_A           19 PLGVVNV-DNVFEDPDGLKEQLLQLRAAGVDGVMVDV   54 (495)
</pre><pre>T ss_dssp             CTTSBCT-TSCBCCHHHHHHHHHHHHHTTCCEEEEEE
</pre><pre>T ss_pred             ccceecC-CCCCCCHHHHHHHHHHHHHCCCCEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>d5h57a_ d.15.4.1 (A:) automated matches {Maize (Zea mays) [TaxId: 4577]}
</pre><pre>Probab=23.46  E-value=2.6e+02  Score=17.42  Aligned_cols=24  Identities=29%  Similarity=0.542  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEecch----hHHHHHHHcCCCCe
</pre><pre>Q MBC1768434.1/3   41 PDGSLTTLDN----TRVAAAREAGIDVQ   64 (74)
</pre><pre>Q Consensus        41 ~DG~lvS~DN----rRl~AAr~agi~Vp   64 (74)
</pre><pre>                      ++|...+++-    +-|.+|.++|+.++
</pre><pre>T Consensus        10 ~~g~~~~~~~~~g~~ll~~~~~~gi~~~   37 (97)
</pre><pre>T d5h57a_          10 PEGEEHEFDAPDDAYILDAAETAGVELP   37 (97)
</pre><pre>T ss_dssp             TTSCEEEEEEETTSCHHHHHHHTTBCCC
</pre><pre>T ss_pred             CCCCEEEEEcCCCCcHHHHHHHcCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>6TFX_B ABC transporter substrate-binding protein; Solute binding protein, periplasmic binding protein, TRANSPORT PROTEIN; HET: N72, EDO; 1.56A {Rhizobium radiobacter} SCOP: l.1.1.1, c.94.1.0
</pre><pre>Probab=23.11  E-value=2.7e+02  Score=21.91  Aligned_cols=54  Identities=15%  Similarity=0.164  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCCCceEEECCCCCEEecchhHHHHHHHcCCCCeeEEeCccc
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGW-KGDPIDVVKMPDGSLTTLDNTRVAAAREAGIDVQATVRNYDD   72 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW-~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~Vp~~~~~~~d   72 (74)
</pre><pre>                      ....+.|++.|| ++.++.+....+.....+-..=-...+++||+|.+...+..+
</pre><pre>T Consensus       315 ~~A~~ll~~ag~~~g~~l~l~~~~~~~~~~~~~~i~~~l~~~Gi~v~~~~~~~~~  369 (500)
</pre><pre>T 6TFX_B          315 AQSKKLLAEAGFPNGVTVTMVASQLPGLESLAQLIQAQVAEGGFTLNLQPVEHAA  369 (500)
</pre><pre>T ss_dssp             HHHHHHHHHTTCTTCEEEEEEEECSHHHHHHHHHHHHHHHTTTEEEEEEEECHHH
</pre><pre>T ss_pred             HHHHHHHHHcCCCCCceEEEEcCCCccHHHHHHHHHHHHHhCCCEeEEEecChHH
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>d1qwga_ c.1.27.1 (A:) (2r)-phospho-3-sulfolactate synthase ComA {Methanococcus jannaschii [TaxId: 2190]}
</pre><pre>Probab=22.42  E-value=4e+02  Score=21.83  Aligned_cols=44  Identities=25%  Similarity=0.414  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCCCceEEECCCCCEEecchhH---HHHHHHcCCCCeeEE
</pre><pre>Q MBC1768434.1/3   19 DKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTR---VAAAREAGIDVQATV   67 (74)
</pre><pre>Q Consensus        19 ~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrR---l~AAr~agi~Vp~~~   67 (74)
</pre><pre>                      .+|.+.+++-|     ++.|+++||.+.--...|   ++.|+++|++|-.++
</pre><pre>T Consensus        88 ~~yl~~~~~lG-----f~~VEIS~g~i~i~~~~r~~~I~~a~~~g~~V~~Ev  134 (251)
</pre><pre>T d1qwga_          88 DEFLNECEKLG-----FEAVEISDGSSDISLEERNNAIKRAKDNGFMVLTEV  134 (251)
</pre><pre>T ss_dssp             HHHHHHHHHHT-----CCEEEECCSSSCCCHHHHHHHHHHHHHTTCEEEEEE
</pre><pre>T ss_pred             HHHHHHHHHhC-----CCEEEeCCCCCCCCHHHHHHHHHHHHHCCCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>d2z15a1 d.370.1.1 (A:10-126) TOB1, N-terminal domain {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=21.96  E-value=3.7e+02  Score=18.87  Aligned_cols=32  Identities=22%  Similarity=0.464  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             HcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC
</pre><pre>Q MBC1768434.1/3   27 ANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID   62 (74)
</pre><pre>Q Consensus        27 ~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~   62 (74)
</pre><pre>                      +...+|+.-+.+.+ +|   ..|.-=+.||+++||+
</pre><pre>T Consensus        47 ~~P~rGs~~R~i~~-~~---~~Dp~l~~Aa~~~gl~   78 (117)
</pre><pre>T d2z15a1          47 EKPYKGSGFRCIHI-GE---KVDPVIEQASKESGLD   78 (117)
</pre><pre>T ss_dssp             TCTTTTHHHHCEEE-SS---SBCHHHHHHHHTTTCC
</pre><pre>T ss_pred             CCCCCCCCeeeEEc-CC---CCCHHHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>4KMD_A Sufu; protein peptide complex, PROTEIN BINDING-TRANSCRIPTION complex; HET: GOL; 1.7A {Homo sapiens}
</pre><pre>Probab=21.60  E-value=3.4e+02  Score=23.09  Aligned_cols=58  Identities=16%  Similarity=0.223  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CCHHHcccccccCCCcHHHHHHHHHcCCCCCCceEEECCCCCEEecchhHHHHHHHcCCC
</pre><pre>Q MBC1768434.1/3    3 VNPSKIRFSQTSVNGSDKIIASMKANGWKGDPIDVVKMPDGSLTTLDNTRVAAAREAGID   62 (74)
</pre><pre>Q Consensus         3 v~P~~I~FSQ~sVs~~~~~~~~M~~~gW~g~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~   62 (74)
</pre><pre>                      |...+..|.|+.  +...+.+.|++..+.+.|..+..+.-..+...++-+++++-++|++
</pre><pre>T Consensus       223 IT~~El~~~~~~--g~~~ll~~l~~~~~~~~p~~itDl~R~s~i~~~~p~~~~~v~~g~~  280 (444)
</pre><pre>T 4KMD_A          223 VCTEELHSAQQW--NGQGILELLRTVPIAGGPWLITDMRRGETIFEIDPHLQERVDKGIE  280 (444)
</pre><pre>T ss_dssp             ECHHHHHHHHHS--CHHHHHHHHHTSGGGTTTTTCBCTTCCSCHHHHCTHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHh--CHHHHHHHHHhcCccCCCeeeeeCCCcccccccChhHHHHHHhchh
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>PF06792.13 ; UPF0261 ; Uncharacterised protein family (UPF0261)
</pre><pre>Probab=21.47  E-value=52  Score=28.14  Aligned_cols=55  Identities=22%  Similarity=0.339  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             cccccccCCCcHHHHHHHHHcCCCC----CCceEEECCCCCEEecchhHHHHHHHcCCC-C
</pre><pre>Q MBC1768434.1/3    8 IRFSQTSVNGSDKIIASMKANGWKG----DPIDVVKMPDGSLTTLDNTRVAAAREAGID-V   63 (74)
</pre><pre>Q Consensus         8 I~FSQ~sVs~~~~~~~~M~~~gW~g----~pi~Vv~~~DG~lvS~DNrRl~AAr~agi~-V   63 (74)
</pre><pre>                      +-|..+.+++. .+.+..+++-.++    .+-.+...-=|-+.+-...||.||-++||+ |
</pre><pre>T Consensus       214 ~vFHa~G~GG~-amE~li~~G~~~gVlDlt~~E~~~~~~gG~~~agp~Rl~aA~~~GiPqV  273 (402)
</pre><pre>T L0DAE2_SINAD/5  214 IVFHATGVGGQ-AMEGLVRDGQIAGVLDLTTTEIADELVGGVLSAGPDRLEAAGRHGVPQV  273 (402)
</pre><pre>T ss_pred             EEEEccccchH-HHHHHHhcCCCeeeeeccHHHHHHHHhCCcccCCCChHHHHHhcCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>d1jx4a2 e.8.1.7 (A:1-240) DinB homolog (DBH) {Sulfolobus solfataricus, DNA polymerase IV [TaxId: 2287]}
</pre><pre>Probab=20.56  E-value=3.5e+02  Score=20.19  Aligned_cols=37  Identities=24%  Similarity=0.459  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCCCCCceEEEC-----CCCCEEecchhHHHHHHHcCCC
</pre><pre>Q MBC1768434.1/3   22 IASMKANGWKGDPIDVVKM-----PDGSLTTLDNTRVAAAREAGID   62 (74)
</pre><pre>Q Consensus        22 ~~~M~~~gW~g~pi~Vv~~-----~DG~lvS~DNrRl~AAr~agi~   62 (74)
</pre><pre>                      ++..+...+.+.|+-|+..     ..|.+++.+-.    |+++||+
</pre><pre>T Consensus        15 ~~~~~~p~l~~~P~~v~~~~~~~~~~~~v~~~n~~----Ar~~Gi~   56 (240)
</pre><pre>T d1jx4a2          15 VEEVLNPSLKGKPVVVCVFSGRFEDSGAVATANYE----ARKFGVK   56 (240)
</pre><pre>T ss_dssp             HHHHHCGGGTTSCEEEEEECSSSTTCEEEEEECHH----HHTTTCC
</pre><pre>T ss_pred             HHHHHCHHhCCCeEEEEECCCCccCceEEEEcCHH----HHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>d1nx4a_ b.82.2.8 (A:) Carbapenem synthase, CarC {Erwinia carotovora [TaxId: 554]}
</pre><pre>Probab=20.28  E-value=50  Score=23.68  Aligned_cols=22  Identities=41%  Similarity=0.501  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             EECCCCCEEecchhHHHHHHHc
</pre><pre>Q MBC1768434.1/3   38 VKMPDGSLTTLDNTRVAAAREA   59 (74)
</pre><pre>Q Consensus        38 v~~~DG~lvS~DNrRl~AAr~a   59 (74)
</pre><pre>                      +.+..|.+.-+||.|++-+|.+
</pre><pre>T Consensus       233 ~~~~~GD~li~dN~~~lHgR~~  254 (271)
</pre><pre>T d1nx4a_         233 HFWEDGDLLIMDNRRVIHEREE  254 (271)
</pre><pre>T ss_dssp             ECCCTTCEEEEETTTEEEEECC
</pre><pre>T ss_pred             EeCCCCCEEEEeCCCEEecCCC
</pre><pre>
</pre><pre>
</pre>