<pre>Query         EDP9633565.1/40-156 
</pre><pre>Match_columns 117
</pre><pre>No_of_seqs    28 out of 32
</pre><pre>Neff          2.82892
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Sat Aug 14 15:19:59 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/9147778.hhr -oa3m ../results/9147778.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF13600.8 ; DUF4140 ; N-termin  42.2   2E+02  0.0019   20.0   4.8   40   66-105    13-54  (100)
</pre><pre>  2 d1zswa2 d.32.1.10 (A:145-314)   41.6 1.2E+02  0.0011   21.4   3.8   39   41-85     85-123 (170)
</pre><pre>  3 d1lwha2 c.1.8.1 (A:1-391) 4-al  36.8 1.1E+02   0.001   24.9   3.3   35   38-80     65-99  (391)
</pre><pre>  4 d3omsa_ d.32.1.0 (A:) automate  36.3 1.5E+02  0.0014   19.8   3.4   39   41-85     83-122 (129)
</pre><pre>  5 d5brqa1 c.1.8.0 (A:6-478) auto  35.4 1.2E+02  0.0011   25.5   3.4   35   38-80     72-106 (473)
</pre><pre>  6 d5v91a_ d.32.1.0 (A:) automate  34.0 2.7E+02  0.0025   18.7   4.3   38   41-85     73-110 (138)
</pre><pre>  7 d1twua_ d.32.1.8 (A:) Hypothet  29.3 2.9E+02  0.0027   18.0   3.8   38   41-84     85-123 (137)
</pre><pre>  8 PF08496.12 ; Peptidase_S49_N ;  28.8 1.1E+02 0.00099   24.8   2.1   19   87-105   120-139 (146)
</pre><pre>  9 PF14620.8 ; YPEB ; YpeB sporul  28.7 2.8E+02  0.0026   24.2   4.5   55   45-105   287-346 (361)
</pre><pre> 10 d5do8a1 c.1.8.1 (A:3-477) auto  27.7 2.1E+02   0.002   24.1   3.6   32   38-77     73-104 (475)
</pre><pre> 11 d4m8ua1 c.1.8.1 (A:2-482) auto  26.4 1.9E+02  0.0017   24.6   3.1   35   38-80     72-106 (481)
</pre><pre> 12 PF13670.8 ; PepSY_2 ; Peptidas  26.2 3.4E+02  0.0032   17.5   5.2   48   44-102    28-77  (81)
</pre><pre> 13 d2wl9a1 d.32.1.0 (A:1-134) aut  26.1 2.9E+02  0.0027   18.3   3.4   39   41-85     71-114 (134)
</pre><pre> 14 d1klla_ d.32.1.2 (A:) Mitomyci  26.1 3.2E+02   0.003   17.6   3.5   39   41-84     79-118 (128)
</pre><pre> 15 1TWU_A Hypothetical protein yy  25.8 3.3E+02   0.003   17.8   3.5   39   41-85     87-126 (139)
</pre><pre> 16 2RK0_B Glyoxalase/Bleomycin re  25.2 2.8E+02  0.0026   18.1   3.1   38   42-85     83-120 (136)
</pre><pre> 17 d3l20a1 d.32.1.0 (A:4-151) aut  24.6   3E+02  0.0028   19.3   3.4   39   41-85     94-135 (148)
</pre><pre> 18 2CTF_A Vigilin; K homology typ  24.3      78 0.00072   19.6   0.5   19   37-55      1-19  (102)
</pre><pre> 19 4PAV_B Glyoxalase family prote  24.1 3.3E+02  0.0031   18.2   3.4   40   41-85     99-138 (145)
</pre><pre> 20 4PAV_D Glyoxalase family prote  24.1 3.3E+02  0.0031   18.2   3.4   40   41-85     99-138 (145)
</pre><pre> 21 d1tsja_ d.32.1.7 (A:) Hypothet  23.2 3.4E+02  0.0032   18.1   3.3   39   41-85     82-121 (129)
</pre><pre> 22 d2ehza1 d.32.1.0 (A:4-137) aut  23.1   4E+02  0.0037   17.5   3.6   39   41-85     71-114 (134)
</pre><pre> 23 d2ze0a1 c.1.8.0 (A:2-471) auto  22.6 2.6E+02  0.0024   23.6   3.2   29   38-72     73-101 (474)
</pre><pre> 24 3GHJ_A Putative integron gene   22.4 3.7E+02  0.0034   18.1   3.4   38   43-86     96-135 (141)
</pre><pre> 25 d3vgfa2 c.1.8.1 (A:91-490) Gly  22.1 2.8E+02  0.0026   23.2   3.3   29   38-72     73-101 (400)
</pre><pre> 26 d1npba_ d.32.1.2 (A:) Fosfomyc  21.7 3.9E+02  0.0036   18.2   3.4   36   43-85     75-110 (140)
</pre><pre> 27 d1qhoa4 c.1.8.1 (A:1-407) Cycl  21.7 3.6E+02  0.0033   22.9   3.8   32   38-77    103-134 (407)
</pre><pre> 28 d4aiea1 c.1.8.0 (A:2-465) auto  21.7   3E+02  0.0028   23.2   3.4   35   38-80     72-106 (464)
</pre><pre> 29 d1jaea2 c.1.8.1 (A:1-378) Anim  21.6 2.9E+02  0.0027   23.2   3.3   29   38-72     70-98  (378)
</pre><pre> 30 d1lkda1 d.32.1.3 (A:2-132) 2,3  21.4   4E+02  0.0037   17.4   3.3   39   41-85     68-111 (131)
</pre><pre> 31 d2zica1 c.1.8.0 (A:1-463) auto  20.8 2.8E+02  0.0026   23.4   3.1   29   38-72     74-102 (463)
</pre><pre> 32 TIGR03806 chp_HNE_0200; conser  20.7   3E+02  0.0028   24.2   3.4   51   55-107    46-104 (317)
</pre><pre> 33 PF00903.27 ; Glyoxalase ; Glyo  20.6 3.1E+02  0.0029   16.5   2.5   39   41-85     75-116 (120)
</pre><pre> 34 d1aqma2 c.1.8.1 (A:1-354) Bact  20.4 3.1E+02  0.0029   22.8   3.2   29   38-72     60-88  (354)
</pre><pre> 35 d3vm7a1 c.1.8.0 (A:22-400) aut  20.3 3.7E+02  0.0035   22.4   3.6   32   38-77     92-123 (379)
</pre><pre> 36 PF07790.13 ; Pilin_N ; Archaea  20.1 3.3E+02   0.003   18.4   2.8   24   78-101    53-77  (77)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF13600.8 ; DUF4140 ; N-terminal domain of unknown function (DUF4140)
</pre><pre>Probab=42.24  E-value=2e+02  Score=20.00  Aligned_cols=40  Identities=15%  Similarity=0.221  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             eeeEecCCCCCcEEECCCCCCCCCCeEEEEeCCC--CcEEee
</pre><pre>Q EDP9633565.1/4   66 FYDIPINPDNKSIVYHPDGSTSIPDKIRIRNNGN--GTFHGF  105 (117)
</pre><pre>Q Consensus        66 ~~~F~lp~g~~siv~~PdGtt~~at~i~vrnng~--GT~Hgy  105 (117)
</pre><pre>                      ...++|++|...+++..=.....++.|+|+..++  .++++.
</pre><pre>T Consensus        13 ~~~~~l~~G~~~i~i~~Lp~~i~~~Sl~v~~~~~~~~~i~~v   54 (100)
</pre><pre>T A7HCS6_ANADF/3   13 VAKVELPDGPTRVVLEGLPDELDDDSIRVEGKGAAKARVFGV   54 (100)
</pre><pre>T ss_pred             EEEEEcCCCCeEEEEcCCCCccCccceEEEEeccccEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>d1zswa2 d.32.1.10 (A:145-314) Hypothetical protein BC1024 {Bacillus cereus [TaxId: 1396]}
</pre><pre>Probab=41.59  E-value=1.2e+02  Score=21.42  Aligned_cols=39  Identities=18%  Similarity=-0.016  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      +++|++.+.+++...+.      -.......+......+.+|||.
</pre><pre>T Consensus        85 ~~~dv~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~Dp~G~  123 (170)
</pre><pre>T d1zswa2          85 NDAELAYWEEQVKQRGF------HSSGIIDRFYFKSLYFRESNGI  123 (170)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHTTC------CCCCCEECSSEEEEEEECTTCC
</pre><pre>T ss_pred             CHHHHHHHHHHHHHCCC------cccceeecCCeEEEEEEcCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>d1lwha2 c.1.8.1 (A:1-391) 4-alpha-glucanotransferase {Thermotoga maritima [TaxId: 2336]}
</pre><pre>Probab=36.81  E-value=1.1e+02  Score=24.87  Aligned_cols=35  Identities=6%  Similarity=0.055  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEE
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVY   80 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~   80 (117)
</pre><pre>                      .||+.+||+...++|.+++-        ...+++..++.|.-+
</pre><pre>T Consensus        65 ~~Gt~~~~~~Li~~ah~~gi--------~vilD~v~nh~s~~~   99 (391)
</pre><pre>T d1lwha2          65 EYGSEREFKEMIEAFHDSGI--------KVVLDLPIHHTGFLH   99 (391)
</pre><pre>T ss_dssp             GGCCHHHHHHHHHHHHHTTC--------EEEEEECTTBCCTTS
</pre><pre>T ss_pred             hhCCHHHHHHHHHHHHHCCC--------EEEEEeCCccccccC
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>d3omsa_ d.32.1.0 (A:) automated matches {Bacillus cereus [TaxId: 226900]}
</pre><pre>Probab=36.27  E-value=1.5e+02  Score=19.82  Aligned_cols=39  Identities=8%  Similarity=-0.153  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCC-CCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPD-NKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g-~~siv~~PdGt   85 (117)
</pre><pre>                      +.+|++.+.+++..++.      =....-..+-+ ....+.+|||.
</pre><pre>T Consensus        83 ~~~d~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~Dp~G~  122 (129)
</pre><pre>T d3omsa_          83 TEEEIDTVFHKLAQDGA------ILMPLGSYPFSKKFGWLNDKYGV  122 (129)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHTTCE------EEEEEEEETTEEEEEEEECTTSC
</pre><pre>T ss_pred             CHHHHHHHHHHHhcCce------EEeccccccccccEEEEEcCCce
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>d5brqa1 c.1.8.0 (A:6-478) automated matches {Bacillus licheniformis [TaxId: 279010]}
</pre><pre>Probab=35.42  E-value=1.2e+02  Score=25.48  Aligned_cols=35  Identities=14%  Similarity=0.072  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEE
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVY   80 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~   80 (117)
</pre><pre>                      .||+.+||+...+.|..++-        ...+++..++.|..+
</pre><pre>T Consensus        72 ~~Gt~~d~~~li~~~h~~Gi--------~vilD~v~nh~s~~~  106 (473)
</pre><pre>T d5brqa1          72 EYGTMADFEELLEEAHKRGI--------KVIMDLVVNHTSTEH  106 (473)
</pre><pre>T ss_dssp             GGCCHHHHHHHHHHHHHTTC--------EEEEEECCSEEETTS
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC--------EEEEEeecCCCccCC
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>d5v91a_ d.32.1.0 (A:) automated matches {Klebsiella pneumoniae [TaxId: 573]}
</pre><pre>Probab=34.02  E-value=2.7e+02  Score=18.70  Aligned_cols=38  Identities=11%  Similarity=0.224  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      +.+|++.+.+++..++.       .....+.+.+....+.+|||.
</pre><pre>T Consensus        73 ~~~d~~~~~~~l~~~g~-------~~~~~~~~~~~~~~~~dp~G~  110 (138)
</pre><pre>T d5v91a_          73 SEADFASFAARLEAAGV-------AVWKLNRSEGASHYFLDPDGH  110 (138)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHTTC-------CEEECSCSSSSEEEEECTTCC
</pre><pre>T ss_pred             cHHHHHHHHHHHHHcCC-------eEEEecCCCceEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>d1twua_ d.32.1.8 (A:) Hypothetical protein YycE {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=29.28  E-value=2.9e+02  Score=18.00  Aligned_cols=38  Identities=13%  Similarity=0.051  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCC-CCcEEECCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPD-NKSIVYHPDG   84 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g-~~siv~~PdG   84 (117)
</pre><pre>                      +.+|++.+.+++..++.      -......-..+ ....+..|+|
</pre><pre>T Consensus        85 ~~~d~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~dp~G  123 (137)
</pre><pre>T d1twua_          85 NAVELAAITSKLKHMGY------QEVESENPYWSNGGVTIEDPDG  123 (137)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHTTC------CEECCSSHHHHSSEEEEECTTC
</pre><pre>T ss_pred             CHHHHHHHHHHHHHcCC------CccCCCCCCccCccEEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>PF08496.12 ; Peptidase_S49_N ; Peptidase family S49 N-terminal
</pre><pre>Probab=28.85  E-value=1.1e+02  Score=24.76  Aligned_cols=19  Identities=26%  Similarity=0.632  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCeEEEEeC-CCCcEEee
</pre><pre>Q EDP9633565.1/4   87 SIPDKIRIRNN-GNGTFHGF  105 (117)
</pre><pre>Q Consensus        87 ~~at~i~vrnn-g~GT~Hgy  105 (117)
</pre><pre>                      .+.|.+.||-. +.|++|||
</pre><pre>T Consensus       120 ~~~DeVvvrleSpGG~V~~Y  139 (146)
</pre><pre>T Q6ML34_BDEBA/2  120 TPSDEVVVRVESPGGVVHGY  139 (146)
</pre><pre>T ss_pred             CCCCEEEEEEeeCCceecch
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>PF14620.8 ; YPEB ; YpeB sporulation
</pre><pre>Probab=28.70  E-value=2.8e+02  Score=24.25  Aligned_cols=55  Identities=13%  Similarity=0.112  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECC----CCCCCCCCeEEEEeCC-CCcEEee
</pre><pre>Q EDP9633565.1/4   45 LKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHP----DGSTSIPDKIRIRNNG-NGTFHGF  105 (117)
</pre><pre>Q Consensus        45 l~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~P----dGtt~~at~i~vrnng-~GT~Hgy  105 (117)
</pre><pre>                      .+.|.+..+.+++      .++........+....|.=    +|-+..++.|.|+++. +|.+-+|
</pre><pre>T Consensus       287 ~~~A~~fL~~~g~------~~~~~~~~~~~~~~~~~~f~~~~~Gv~~~~d~i~V~V~~~~G~Vv~~  346 (361)
</pre><pre>T D9S297_THEOJ/1  287 VDKARKFLEARGY------ANMEPTGSLREDNSLTVTFAYKQGDVLVYPDFVKVEVALDDGQVVGF  346 (361)
</pre><pre>T ss_pred             HHHHHHHHHHCCC------CCcEEEEEEEECCEEEEEEEEEECCEEEcCCeEEEEEECCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>d5do8a1 c.1.8.1 (A:3-477) automated matches {Listeria monocytogenes [TaxId: 169963]}
</pre><pre>Probab=27.66  E-value=2.1e+02  Score=24.11  Aligned_cols=32  Identities=9%  Similarity=0.047  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCc
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKS   77 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~s   77 (117)
</pre><pre>                      .||+.+||+...+.|.+.+-        ...+++..++.|
</pre><pre>T Consensus        73 ~~Gt~~d~~~lv~~~h~~gi--------~vi~D~v~nh~~  104 (475)
</pre><pre>T d5do8a1          73 EYGDMATFDQLLQGLHDRDM--------KLVMDLVVNHTS  104 (475)
</pre><pre>T ss_dssp             TTCCHHHHHHHHHHHHHTTC--------EEEEEECCSBCC
</pre><pre>T ss_pred             hhCCHHHHHHHHHHHHHCCC--------EEEEEEecccCc
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>d4m8ua1 c.1.8.1 (A:2-482) automated matches {Bacillus subtilis [TaxId: 224308]}
</pre><pre>Probab=26.38  E-value=1.9e+02  Score=24.64  Aligned_cols=35  Identities=14%  Similarity=0.045  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEE
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVY   80 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~   80 (117)
</pre><pre>                      .||+.+||+...+.|.+.+-        ...+++..++.|..+
</pre><pre>T Consensus        72 ~~Gt~~~~~~li~~~h~~gi--------~vilD~v~nh~s~~~  106 (481)
</pre><pre>T d4m8ua1          72 KFGTNEDMFQLIDEVHKRGM--------KIVMDLVVNHTSDEH  106 (481)
</pre><pre>T ss_dssp             TTCCHHHHHHHHHHHHHTTC--------EEEEEECCSBCCTTS
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC--------EEEEEEeccccccCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>PF13670.8 ; PepSY_2 ; Peptidase propeptide and YPEB domain
</pre><pre>Probab=26.21  E-value=3.4e+02  Score=17.47  Aligned_cols=48  Identities=8%  Similarity=-0.035  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhcCCCCCCCCCcceeeEecCCC-CCcEEECCCCCCCCCCeEEEEeCC-CCcE
</pre><pre>Q EDP9633565.1/4   44 DLKYAGQQAAKLPAGPNGTKGSFYDIPINPD-NKSIVYHPDGSTSIPDKIRIRNNG-NGTF  102 (117)
</pre><pre>Q Consensus        44 Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g-~~siv~~PdGtt~~at~i~vrnng-~GT~  102 (117)
</pre><pre>                      +.+.+.+++...++      +....+++..+ .--+.+.++|     ..+.|+.+. +|.+
</pre><pre>T Consensus        28 ~~~~~~~~~~~~~~------~~~~~~~~~~~~~y~v~~~~~g-----~~~~v~id~~tG~i   77 (81)
</pre><pre>T Q1QMT6_NITHX/5   28 PADQVIQKLMEKGY------TDVSKLEADDGRCWEAKAMLNG-----VRQKIYLDPRSGEV   77 (81)
</pre><pre>T ss_pred             CHHHHHHHHHHCCC------cceEEEEeCCCCeEEEEEecCC-----eEEEEEEcCCCCce
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>d2wl9a1 d.32.1.0 (A:1-134) automated matches {Rhodococcus sp. [TaxId: 186196]}
</pre><pre>Probab=26.15  E-value=2.9e+02  Score=18.30  Aligned_cols=39  Identities=10%  Similarity=0.015  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCC-----CcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDN-----KSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~-----~siv~~PdGt   85 (117)
</pre><pre>                      +.+|++.+.+++..++.      ............     ...+.+|+|.
</pre><pre>T Consensus        71 ~~~~~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~~~~~dp~G~  114 (134)
</pre><pre>T d2wl9a1          71 GPVELDELAEQLKNAGI------PFEVASDADAAERRVLGLVKLHDPGGN  114 (134)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHTTC------CCEECCHHHHHHTTEEEEEEEECTTCC
</pre><pre>T ss_pred             CHHHHHHHHHHHHHCCC------CeEeCCccchhhheeEEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>d1klla_ d.32.1.2 (A:) Mitomycin resistance protein D, MRD {Streptomyces lavendulae [TaxId: 1914]}
</pre><pre>Probab=26.14  E-value=3.2e+02  Score=17.57  Aligned_cols=39  Identities=15%  Similarity=0.124  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCC-CCcEEECCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPD-NKSIVYHPDG   84 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g-~~siv~~PdG   84 (117)
</pre><pre>                      +.+|++.+.++++.++.     +-.......+.+ ....+..|||
</pre><pre>T Consensus        79 ~~~d~~~~~~~l~~~g~-----~~~~~~~~~~~~~~~~~~~dp~G  118 (128)
</pre><pre>T d1klla_          79 DTASVDKKYAELVDAGY-----EGHLKPWNAVWGQRYAIVKDPDG  118 (128)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHTTC-----EEEEEEEECTTSEEEEEEECTTS
</pre><pre>T ss_pred             CHHHHHHHHHHHHHCCC-----cccccccccCCceEEEEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>1TWU_A Hypothetical protein yycE; Structural genomics, Protein structure initiative, MCSG, duplication of the alpha-beta sandwichs. Bacillus subtilis, PSI, Midwest Center; 2.0A {Bacillus subtilis} SCOP: d.32.1.8
</pre><pre>Probab=25.84  E-value=3.3e+02  Score=17.83  Aligned_cols=39  Identities=15%  Similarity=0.170  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCccee-eEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFY-DIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~-~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      |.+|++.+.+++..++.      -... ......+....+.+|+|.
</pre><pre>T Consensus        87 ~~~d~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~dp~G~  126 (139)
</pre><pre>T 1TWU_A           87 NAVELAAITSKLKHMGY------QEVESENPYWSNGGVTIEDPDGW  126 (139)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHTTC------CEECCSSHHHHSSEEEEECTTCC
</pre><pre>T ss_pred             ChhhHHHHHHHHHhCCC------eeecCCCCCCcCccEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>2RK0_B Glyoxalase/Bleomycin resistance protein/dioxygenase domain; 11002z, Glyoxylase, Bleomycin resistance protein, dioxygenase domain, PSI-II, NewYork Structural GenomiX research consortium (NYSGRC), Structural Genomics; HET: MSE; 2.04A {Frankia sp.}
</pre><pre>Probab=25.17  E-value=2.8e+02  Score=18.07  Aligned_cols=38  Identities=16%  Similarity=-0.047  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   42 KNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        42 k~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      .+|++.+.+++...+.      -.......+.+....+.+|||.
</pre><pre>T Consensus        83 ~~~~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~dp~G~  120 (136)
</pre><pre>T 2RK0_B           83 MTDLDVLEERLAKAGA------AFTPTQELPFGWILAFRDADNI  120 (136)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTTC------CEEEEEEETTEEEEEEECTTCC
</pre><pre>T ss_pred             hHHHHHHHHHHHHCCC------ccCCcccCCCeEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>d3l20a1 d.32.1.0 (A:4-151) automated matches {Staphylococcus aureus [TaxId: 367830]}
</pre><pre>Probab=24.61  E-value=3e+02  Score=19.33  Aligned_cols=39  Identities=5%  Similarity=-0.137  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCC--CCCCCCCcceeeEecCCC-CCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLP--AGPNGTKGSFYDIPINPD-NKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~--p~~~~~~g~~~~F~lp~g-~~siv~~PdGt   85 (117)
</pre><pre>                      |.+|++.+.+++..++  .      -....-..+.+ ....+.+|||.
</pre><pre>T Consensus        94 ~~~d~~~~~~~l~~~g~~~------~~~~~~~~~~g~~~~~~~Dp~G~  135 (148)
</pre><pre>T d3l20a1          94 DADKVEAFYEQIKDHSSIE------IELPFADQFWGGKMGVFTDKYGV  135 (148)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTCTTCE------EEEEEEECTTSSEEEEEECTTSC
</pre><pre>T ss_pred             CHHHHHHHHHHHHhCCCcE------EEEechhccccceeEEEEcCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>2CTF_A Vigilin; K homology type I domain, RNA-binding, cell sterol metabolism, beta-alpha-alpha-beta-beta-alpha structure, Structural Genomics, NPPSFA, National Project; NMR {Homo sapiens} SCOP: l.1.1.1, d.51.1.1
</pre><pre>Probab=24.30  E-value=78  Score=19.58  Aligned_cols=19  Identities=21%  Similarity=0.373  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHHHHhcC
</pre><pre>Q EDP9633565.1/4   37 GKWGNKNDLKYAGQQAAKL   55 (117)
</pre><pre>Q Consensus        37 GqwGSk~Dl~YA~~~Aa~L   55 (117)
</pre><pre>                      |.+|++++++.+.+....|
</pre><pre>T Consensus         1 ~~~~~~~~~~~~~~~~~~~   19 (102)
</pre><pre>T 2CTF_A            1 GSSGSSGEPEKLGQALTEV   19 (102)
</pre><pre>T ss_dssp             CCSCCCCCCCCCCCSCCCC
</pre><pre>T ss_pred             CCCCCcCcHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>4PAV_B Glyoxalase family protein; hypothetical protein, UNKNOWN FUNCTION; HET: MSE; 2.3A {Staphylococcus aureus}
</pre><pre>Probab=24.12  E-value=3.3e+02  Score=18.21  Aligned_cols=40  Identities=5%  Similarity=-0.082  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      |.+|++.+.+++...+.     .-.......+.+....+..|+|.
</pre><pre>T Consensus        99 ~~~d~~~~~~~l~~~g~-----~~~~~~~~~~~~~~~~~~dp~G~  138 (145)
</pre><pre>T 4PAV_B           99 TNEEVDQLVNKVKEAGG-----AVVQEPTVSQGFYGAMFKDLDGH  138 (145)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHTC-----EEEEEEEEETTEEEEEEECTTCC
</pre><pre>T ss_pred             CHHHHHHHHHHHHHCCC-----EEecCceeeCCeEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>4PAV_D Glyoxalase family protein; hypothetical protein, UNKNOWN FUNCTION; HET: MSE; 2.3A {Staphylococcus aureus}
</pre><pre>Probab=24.12  E-value=3.3e+02  Score=18.21  Aligned_cols=40  Identities=5%  Similarity=-0.082  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      |.+|++.+.+++...+.     .-.......+.+....+..|+|.
</pre><pre>T Consensus        99 ~~~d~~~~~~~l~~~g~-----~~~~~~~~~~~~~~~~~~dp~G~  138 (145)
</pre><pre>T 4PAV_D           99 TNEEVDQLVNKVKEAGG-----AVVQEPTVSQGFYGAMFKDLDGH  138 (145)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHTC-----EEEEEEEEETTEEEEEEECTTCC
</pre><pre>T ss_pred             CHHHHHHHHHHHHHCCC-----EEecCceeeCCeEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>d1tsja_ d.32.1.7 (A:) Hypothetical protein MW1090 {Staphylococcus aureus [TaxId: 1280]}
</pre><pre>Probab=23.25  E-value=3.4e+02  Score=18.14  Aligned_cols=39  Identities=10%  Similarity=-0.071  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCC-CCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPD-NKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g-~~siv~~PdGt   85 (117)
</pre><pre>                      +.++|+.+.++++..+.      -.......+.+ ....+.+|||.
</pre><pre>T Consensus        82 d~~~~~~~~~~l~~~G~------~~~~~~~~~~~~~~~~~~Dp~G~  121 (129)
</pre><pre>T d1tsja_          82 DTIEMERLFNGLKDEGA------ILMPKTNMPPYREFAWVQDKFGV  121 (129)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHTTCE------EEEEEEEETTEEEEEEEECTTSC
</pre><pre>T ss_pred             CHHHHHHHHHHhcccCe------eeccCCcCCCccceEEEEcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>d2ehza1 d.32.1.0 (A:4-137) automated matches {Pseudomonas sp. [TaxId: 69011]}
</pre><pre>Probab=23.07  E-value=4e+02  Score=17.46  Aligned_cols=39  Identities=13%  Similarity=0.116  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCc-----EEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKS-----IVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~s-----iv~~PdGt   85 (117)
</pre><pre>                      +.+|++.+.+++..++.      ..............     .+..|+|.
</pre><pre>T Consensus        71 ~~~d~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~~~~~dp~G~  114 (134)
</pre><pre>T d2ehza1          71 GKPEFEALGQKLIDAGY------KIRICDKVEAQERMVLGLMKTEDPGGN  114 (134)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHTTC------CCEECCHHHHHHHTEEEEEEEECTTSC
</pre><pre>T ss_pred             ChHHHHHHHHHHHHCCC------eEEEcCCcchhhcceeEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>d2ze0a1 c.1.8.0 (A:2-471) automated matches {Geobacillus sp. [TaxId: 258999]}
</pre><pre>Probab=22.61  E-value=2.6e+02  Score=23.63  Aligned_cols=29  Identities=24%  Similarity=0.298  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecC
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPIN   72 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp   72 (117)
</pre><pre>                      .||+.+||+.-.+.|...+-      ....+|.++
</pre><pre>T Consensus        73 ~~Gt~~df~~Li~~ah~~Gi------~vilD~v~n  101 (474)
</pre><pre>T d2ze0a1          73 EFGTMDDFDELLAQAHRRGL------KVILDLVIN  101 (474)
</pre><pre>T ss_dssp             GGCCHHHHHHHHHHHHHTTC------EEEEEEECS
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC------EEEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>3GHJ_A Putative integron gene cassette protein; Integron Cassette Protein, Mobile Metagenome, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: MSE; 1.471A {uncultured bacterium}
</pre><pre>Probab=22.42  E-value=3.7e+02  Score=18.06  Aligned_cols=38  Identities=5%  Similarity=-0.000  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhcCCCCCCCCCcceeeEecCCCC--CcEEECCCCCC
</pre><pre>Q EDP9633565.1/4   43 NDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDN--KSIVYHPDGST   86 (117)
</pre><pre>Q Consensus        43 ~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~--~siv~~PdGtt   86 (117)
</pre><pre>                      +|++.+.+++..++.      -.......+.+.  ...+..|||..
</pre><pre>T Consensus        96 ~d~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~~dp~G~~  135 (141)
</pre><pre>T 3GHJ_A           96 SEIEPLKKALESKGV------SVHGPVNQEWMQAVSLYFADPNGHA  135 (141)
</pre><pre>T ss_dssp             GGHHHHHHHHHHTTC------CCEEEEEEGGGTEEEEEEECTTCCE
</pre><pre>T ss_pred             HHHHHHHHHHHhCCC------ceeccccccccceEEEEEECCCCCE
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>d3vgfa2 c.1.8.1 (A:91-490) Glycosyltrehalose trehalohydrolase, central domain {Sulfolobus solfataricus, km1 [TaxId: 2287]}
</pre><pre>Probab=22.12  E-value=2.8e+02  Score=23.23  Aligned_cols=29  Identities=21%  Similarity=0.322  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecC
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPIN   72 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp   72 (117)
</pre><pre>                      .+|+.+||+...++|...+-      ....++.++
</pre><pre>T Consensus        73 ~~G~~~~l~~li~~~h~~gi------~vi~D~v~n  101 (400)
</pre><pre>T d3vgfa2          73 SYGGPEGFRKLVDEAHKKGL------GVILDVVYN  101 (400)
</pre><pre>T ss_dssp             GGTHHHHHHHHHHHHHHTTC------EEEEEECCS
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC------EEEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>d1npba_ d.32.1.2 (A:) Fosfomycin resistance protein A (FosA) {Serratia marcescens [TaxId: 615]}
</pre><pre>Probab=21.72  E-value=3.9e+02  Score=18.25  Aligned_cols=36  Identities=14%  Similarity=0.232  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   43 NDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        43 ~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      +|++.+.+++..++.       .....+.+.+....+.+|||.
</pre><pre>T Consensus        75 ~~~~~~~~~l~~~g~-------~~~~~~~~~~~~~~~~Dp~G~  110 (140)
</pre><pre>T d1npba_          75 EDFEPLSQRLEQAGV-------TIWKQNKSEGASFYFLDPDGH  110 (140)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTC-------CEEECCCSSSEEEEEECTTCC
</pre><pre>T ss_pred             HHHHHHHHHHHHcCC-------ceeecCCCceEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>d1qhoa4 c.1.8.1 (A:1-407) Cyclodextrin glycosyltransferase {Bacillus stearothermophilus, maltogenic alpha-amylase [TaxId: 1422]}
</pre><pre>Probab=21.69  E-value=3.6e+02  Score=22.86  Aligned_cols=32  Identities=13%  Similarity=0.133  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCc
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKS   77 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~s   77 (117)
</pre><pre>                      .+|+.+||+...+.|.+++-        ...++++.++.+
</pre><pre>T Consensus       103 ~~G~~~~l~~lv~~~h~~Gi--------~vilD~v~nh~~  134 (407)
</pre><pre>T d1qhoa4         103 HFGNWTTFDTLVNDAHQNGI--------KVIVDFVPNHST  134 (407)
</pre><pre>T ss_dssp             TTCCHHHHHHHHHHHHHTTC--------EEEEEECTTEEE
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC--------EEEEEEcCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>d4aiea1 c.1.8.0 (A:2-465) automated matches {Lactobacillus acidophilus [TaxId: 272621]}
</pre><pre>Probab=21.65  E-value=3e+02  Score=23.18  Aligned_cols=35  Identities=11%  Similarity=0.037  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcEEE
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSIVY   80 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~siv~   80 (117)
</pre><pre>                      .+|+.+||+...+.|...+-        ...+++..++.|.-+
</pre><pre>T Consensus        72 ~~Gt~~~~~~li~~~h~~Gi--------kVilD~v~nh~s~~~  106 (464)
</pre><pre>T d4aiea1          72 QYGTMADMDELISKAKEHHI--------KIVMDLVVNHTSDQH  106 (464)
</pre><pre>T ss_dssp             TTCCHHHHHHHHHHHHHTTC--------EEEEEECCSBCCTTS
</pre><pre>T ss_pred             hhCCHHHHHHHHHHHHHCCC--------EEEEEcccccccccC
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>d1jaea2 c.1.8.1 (A:1-378) Animal alpha-amylase {Yellow mealworm (Tenebrio molitor), larva [TaxId: 7067]}
</pre><pre>Probab=21.61  E-value=2.9e+02  Score=23.24  Aligned_cols=29  Identities=14%  Similarity=0.227  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecC
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPIN   72 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp   72 (117)
</pre><pre>                      .||+.+||+...++|.+++-      ....++.++
</pre><pre>T Consensus        70 ~~Gt~~d~k~Lv~~ah~~gi------~vilD~v~n   98 (378)
</pre><pre>T d1jaea2          70 RSGDESAFTDMTRRCNDAGV------RIYVDAVIN   98 (378)
</pre><pre>T ss_dssp             TTEEHHHHHHHHHHHHHTTC------EEEEEECCS
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHHCCC------EEEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>d1lkda1 d.32.1.3 (A:2-132) 2,3-Dihydroxybiphenyl dioxygenase (DHBD, BPHC enzyme) {Burkholderia cepacia, formerly Pseudomonas cepacia [TaxId: 292]}
</pre><pre>Probab=21.36  E-value=4e+02  Score=17.40  Aligned_cols=39  Identities=8%  Similarity=-0.067  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCccee-----eEecCCCCCcEEECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFY-----DIPINPDNKSIVYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~-----~F~lp~g~~siv~~PdGt   85 (117)
</pre><pre>                      +.+||+.+.+++..++.      -...     .-.........+..|||.
</pre><pre>T Consensus        68 ~~~~~~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~~~~~dp~G~  111 (131)
</pre><pre>T d1lkda1          68 DAAGLAQMADKLKQAGI------AVTTGDASLARRRGVTGLITFADPFGL  111 (131)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHTTC------CCEECCHHHHHHHTCSEEEEEECTTSC
</pre><pre>T ss_pred             CHHHHHHHHHHHHHCCC------ceeeCCHHHHhhcCccEEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>d2zica1 c.1.8.0 (A:1-463) automated matches {Streptococcus mutans}
</pre><pre>Probab=20.84  E-value=2.8e+02  Score=23.35  Aligned_cols=29  Identities=24%  Similarity=0.236  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecC
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPIN   72 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp   72 (117)
</pre><pre>                      .||+.+||+..++.|..++-      ....++.++
</pre><pre>T Consensus        74 ~~Gt~~dl~~lv~~~h~~gi------~VilD~v~n  102 (463)
</pre><pre>T d2zica1          74 IFGNMADMDNLLTQAKMRGI------KIIMDLVVN  102 (463)
</pre><pre>T ss_dssp             GGCCHHHHHHHHHHHHTTTC------EEEEEECCS
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC------EEEEEeccc
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>TIGR03806 chp_HNE_0200; conserved hypothetical protein, HNE_0200 family. The model TIGR03805 describes an uncharacterized protein family that contains repeats associated with the formation of a right-handed helical stack of parallel beta strands, homologous to those found in a number of carbohydrate-binding proteins and sugar hydrolases.
</pre><pre>Probab=20.73  E-value=3e+02  Score=24.21  Aligned_cols=51  Identities=22%  Similarity=0.363  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCcceeeEecCCCC---CcEEECCC-----CCCCCCCeEEEEeCCCCcEEeeec
</pre><pre>Q EDP9633565.1/4   55 LPAGPNGTKGSFYDIPINPDN---KSIVYHPD-----GSTSIPDKIRIRNNGNGTFHGFPI  107 (117)
</pre><pre>Q Consensus        55 L~p~~~~~~g~~~~F~lp~g~---~siv~~Pd-----Gtt~~at~i~vrnng~GT~HgyPi  107 (117)
</pre><pre>                      |..|..-+-.....|.+|.|+   +...|--|     +....-|++.||.. +| +++++.
</pre><pre>T Consensus        46 lP~g~~~~~~~~~~~~fP~gtv~~K~f~~~~~~~~~~~~~~iETRll~~~~-~~-w~~~~Y  104 (317)
</pre><pre>T TIGR03806        46 VPAGTSATYNAEGVLDFPVGTVLVKTFALPSDLRGPTNGRLIETRLLIRRA-QG-WAALPY  104 (317)
</pre><pre>T ss_pred             cCCCCccEecCCCCcccCCCCEEEEEEEeecCCCCCCCCcEEEEEEEEEeC-Cc-EEEece
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>PF00903.27 ; Glyoxalase ; Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily
</pre><pre>Probab=20.59  E-value=3.1e+02  Score=16.55  Aligned_cols=39  Identities=21%  Similarity=0.145  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCcE---EECCCCC
</pre><pre>Q EDP9633565.1/4   41 NKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKSI---VYHPDGS   85 (117)
</pre><pre>Q Consensus        41 Sk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~si---v~~PdGt   85 (117)
</pre><pre>                      +.+|++.+.+++...+.      -..............   +..|+|.
</pre><pre>T Consensus        75 ~~~di~~~~~~l~~~g~------~~~~~~~~~~~~~~~~~~~~dp~G~  116 (120)
</pre><pre>T YWBC_BACSU/4-1   75 LTDDIAAEYTKAEKMNA------KFIDEEITTLPNGYRYFYIEGPDGE  116 (120)
</pre><pre>T ss_pred             EecCHHHHHHHHHHcCC------EEeccccccCCCCceEEEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>d1aqma2 c.1.8.1 (A:1-354) Bacterial alpha-amylase {Pseudoalteromonas haloplanktis [TaxId: 228]}
</pre><pre>Probab=20.39  E-value=3.1e+02  Score=22.77  Aligned_cols=29  Identities=17%  Similarity=0.210  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecC
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPIN   72 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp   72 (117)
</pre><pre>                      .||+.+||+...++|...+-      ....++.++
</pre><pre>T Consensus        60 ~~G~~~~l~~lv~~ah~~Gi------~vi~D~v~n   88 (354)
</pre><pre>T d1aqma2          60 RGGNRAQFIDMVNRCSAAGV------DIYVDTLIN   88 (354)
</pre><pre>T ss_dssp             TTBCHHHHHHHHHHHHHHTC------EEEEEEECS
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHHCCC------EEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>d3vm7a1 c.1.8.0 (A:22-400) automated matches {Malbranchea cinnamomea [TaxId: 5041]}
</pre><pre>Probab=20.31  E-value=3.7e+02  Score=22.36  Aligned_cols=32  Identities=13%  Similarity=0.282  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHhcCCCCCCCCCcceeeEecCCCCCc
</pre><pre>Q EDP9633565.1/4   38 KWGNKNDLKYAGQQAAKLPAGPNGTKGSFYDIPINPDNKS   77 (117)
</pre><pre>Q Consensus        38 qwGSk~Dl~YA~~~Aa~L~p~~~~~~g~~~~F~lp~g~~s   77 (117)
</pre><pre>                      ++|+.+||+..++.|...+-        ...+++..++.+
</pre><pre>T Consensus        92 ~~g~~~~l~~~v~~~h~~Gi--------~vi~D~v~~h~~  123 (379)
</pre><pre>T d3vm7a1          92 HFGTEEDLRALADELHARGM--------FLMVDVVINHNG  123 (379)
</pre><pre>T ss_dssp             TTCCHHHHHHHHHHHHHTTC--------EEEEEECCSBCC
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHHCCC--------EEEEEEecccCC
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF07790.13 ; Pilin_N ; Archaeal Type IV pilin, N-terminal
</pre><pre>Probab=20.12  E-value=3.3e+02  Score=18.40  Aligned_cols=24  Identities=17%  Similarity=0.198  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             EEECCCCCCCCC-CeEEEEeCCCCc
</pre><pre>Q EDP9633565.1/4   78 IVYHPDGSTSIP-DKIRIRNNGNGT  101 (117)
</pre><pre>Q Consensus        78 iv~~PdGtt~~a-t~i~vrnng~GT  101 (117)
</pre><pre>                      .+.+-.|.+... +.+.|+.+++|+
</pre><pre>T Consensus        53 ~l~h~gGd~i~~~~~l~v~v~~~g~   77 (77)
</pre><pre>T Q2FP79_METHJ/6   53 VVTYHGGKDHPDVANLSIGFAGASP   77 (77)
</pre><pre>T ss_pred             EEEEeCCCccCCCCcEEEEEccCCC
</pre><pre>
</pre><pre>
</pre>