<pre>Query         WP_134219109.1/46-114 
</pre><pre>Match_columns 69
</pre><pre>No_of_seqs    99 out of 104
</pre><pre>Neff          4.06674
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Sat Aug 14 15:23:54 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/3912182.hhr -oa3m ../results/3912182.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF14076.8 ; DUF4258 ; Domain o  98.1 8.3E-05 7.7E-10   41.3   6.6   59    5-66      1-72  (72)
</pre><pre>  2 PF12106.10 ; Colicin_E5 ; Coli  94.7     0.9 8.4E-06   28.5   7.3   61    5-68      5-73  (82)
</pre><pre>  3 TIGR02530 flg_new; flagellar o  94.0    0.29 2.7E-06   33.4   4.3   63    5-69     17-81  (96)
</pre><pre>  4 d1v74a_ d.243.1.1 (A:) Colicin  93.5     1.5 1.4E-05   29.9   6.9   61    5-65     10-89  (107)
</pre><pre>  5 3U97_A Ribonuclease toxin BrnT  91.5       2 1.9E-05   28.3   5.5   62    5-69     38-109 (114)
</pre><pre>  6 2FHZ_B Colicin-E5; Protein-Pro  91.4     1.9 1.8E-05   28.7   5.3   64    5-68      9-90  (108)
</pre><pre>  7 5D0O_E Outer membrane protein   90.1     3.2   3E-05   25.7   5.2   44   19-62     48-102 (123)
</pre><pre>  8 5EKQ_E Outer membrane protein   89.2     4.1 3.8E-05   24.7   5.1   44   19-62     29-83  (94)
</pre><pre>  9 2YH9_B SMALL PROTEIN A; LIPOPR  89.2     6.6 6.1E-05   24.1   6.1   50   19-68     15-75  (88)
</pre><pre> 10 PF04355.15 ; SmpA_OmlA ; SmpA   89.1     4.9 4.5E-05   22.5   5.2   45   19-63     14-69  (71)
</pre><pre> 11 PF04365.15 ; BrnT_toxin ; Ribo  88.3     6.1 5.6E-05   22.6   6.9   62    5-69      1-77  (79)
</pre><pre> 12 d2fhzb_ d.243.1.2 (B:) Colicin  85.1     6.7 6.2E-05   27.3   4.8   55    5-62      3-73  (93)
</pre><pre> 13 5WAM_B Outer membrane protein   82.0      16 0.00015   23.1   5.4   46   19-65     32-88  (108)
</pre><pre> 14 1V74_A Colicin D; colicin D -   80.2      21  0.0002   22.1   6.1   46   23-68     41-92  (107)
</pre><pre> 15 PF20132.1 ; DUF6522 ; Family o  74.2      33 0.00031   22.8   5.2   44   20-63     23-76  (81)
</pre><pre> 16 4WBJ_B Blr1131 protein; copper  72.9      33  0.0003   20.6   5.2   43   24-66    106-148 (164)
</pre><pre> 17 PF11429.10 ; Colicin_D ; Colic  72.6      32 0.00029   20.2   5.7   46   23-68     27-78  (85)
</pre><pre> 18 2GGT_B SCO1 protein homolog, m  71.7      34 0.00032   21.0   4.6   32   38-69    116-147 (164)
</pre><pre> 19 4WBR_D Blr1131 protein; copper  69.4      39 0.00037   20.1   5.1   43   24-66     99-141 (157)
</pre><pre> 20 d2rlia1 c.47.1.0 (A:100-266) a  68.0      40 0.00037   21.0   4.4   30   39-68    116-145 (167)
</pre><pre> 21 4BPY_A SCO PROTEIN; COPPER HOM  65.9      44 0.00041   20.4   4.2   28   38-65    125-152 (175)
</pre><pre> 22 6N5U_A Protein SCO1 homolog 1,  64.4      53 0.00049   19.7   5.7   46   23-68    106-151 (170)
</pre><pre> 23 6N5U_C Protein SCO1 homolog 1,  64.4      53 0.00049   19.7   5.7   46   23-68    106-151 (170)
</pre><pre> 24 PF12554.10 ; MOZART1 ; Mitotic  64.3      23 0.00021   21.5   2.7   29    3-31     16-44  (47)
</pre><pre> 25 d1in0a1 d.58.49.1 (A:2-89) Hyp  63.2      25 0.00024   24.1   3.0   45   21-68      9-53  (88)
</pre><pre> 26 d4hqza_ c.47.1.0 (A:) automate  57.3      39 0.00036   19.8   2.8   16   51-66    110-125 (141)
</pre><pre> 27 d1i5ga_ c.47.1.10 (A:) Trypare  55.8      24 0.00022   21.6   1.9   16   51-66    111-127 (144)
</pre><pre> 28 d3ixra1 c.47.1.0 (A:1-158) aut  55.6      36 0.00033   20.8   2.6   16   51-66    123-138 (158)
</pre><pre> 29 d4wbja_ c.47.1.0 (A:) automate  54.5      88 0.00082   19.1   5.3   43   23-65     98-140 (157)
</pre><pre> 30 3GMX_A BLP; 2-layer alpha/beta  54.5 1.1E+02   0.001   20.3   5.5   45   19-63     14-63  (154)
</pre><pre> 31 PF02630.16 ; SCO1-SenC ; SCO1/  54.1      63 0.00058   18.9   3.3   39   23-61     96-136 (136)
</pre><pre> 32 d1xzoa1 c.47.1.10 (A:3-174) Th  52.8 1.1E+02   0.001   19.7   4.5   46   20-65    104-150 (172)
</pre><pre> 33 d4je1a1 c.47.1.10 (A:1-167) au  52.3      46 0.00042   20.9   2.7   15   51-65    134-148 (167)
</pre><pre> 34 4DM5_A Osmotically inducible l  52.2 1.2E+02  0.0011   19.9   5.8   46   19-64     26-80  (103)
</pre><pre> 35 4DM5_B Osmotically inducible l  52.2 1.2E+02  0.0011   19.9   5.8   46   19-64     26-80  (103)
</pre><pre> 36 3ME8_A Putative uncharacterize  52.1      63 0.00059   20.3   3.3   21   46-66    124-144 (170)
</pre><pre> 37 d3zl5a_ c.47.1.0 (A:) automate  51.6      59 0.00055   19.9   3.1   19   51-69    123-141 (162)
</pre><pre> 38 d2l5oa1 c.47.1.0 (A:1-142) aut  51.1      57 0.00052   19.2   2.8   16   51-66    106-121 (142)
</pre><pre> 39 d3hdca1 c.47.1.0 (A:22-168) au  50.9      52 0.00048   19.8   2.7   16   51-66    115-130 (147)
</pre><pre> 40 d2ls5a1 c.47.1.0 (A:4-151) aut  50.8      57 0.00053   19.4   2.8   16   51-66    114-129 (148)
</pre><pre> 41 PF07467.13 ; BLIP ; Beta-lacta  50.7 1.2E+02  0.0012   19.7   5.3   50   19-68     10-64  (121)
</pre><pre> 42 d4hdea1 c.47.1.0 (A:29-195) au  50.6      52 0.00048   20.7   2.8   15   51-65    133-147 (167)
</pre><pre> 43 1XZO_A Hypothetical protein yp  49.8      62 0.00057   19.5   2.9   24   42-65    129-152 (174)
</pre><pre> 44 6L7R_B Mozart1; gamma tubulin   49.8      49 0.00046   22.8   2.7   29    3-31     32-60  (86)
</pre><pre> 45 d1mp9a1 d.129.1.1 (A:5-96) TAT  49.6      87 0.00081   20.2   3.7   52   17-68     19-72  (92)
</pre><pre> 46 d1n8ja_ c.47.1.10 (A:) Alkyl h  49.5      53 0.00049   21.3   2.8   16   51-66    120-135 (186)
</pre><pre> 47 d4k9za_ c.47.1.0 (A:) automate  49.3      44 0.00041   19.8   2.2   16   51-66    110-125 (141)
</pre><pre> 48 d3kh7a1 c.47.1.0 (A:26-175) au  48.6      63 0.00059   19.5   2.8   16   51-66    114-129 (150)
</pre><pre> 49 2B7K_A SCO1 protein; Metalloch  48.5 1.2E+02  0.0011   19.0   4.1   27   39-65    135-161 (200)
</pre><pre> 50 3EWL_A uncharacterized conserv  47.9      53 0.00049   19.4   2.4   16   51-66    112-127 (142)
</pre><pre> 51 d3kcma1 c.47.1.0 (A:28-165) au  47.5      66 0.00061   18.9   2.7   15   51-65    106-120 (138)
</pre><pre> 52 d3or5a_ c.47.1.0 (A:) automate  47.5      70 0.00065   19.0   2.9   16   51-66    113-128 (145)
</pre><pre> 53 PF07504.15 ; FTP ; Fungalysin/  47.5      81 0.00075   16.6   3.8   26   42-67     25-50  (51)
</pre><pre> 54 6HN3_A Phospholipid hydroperox  47.1      81 0.00076   19.5   3.2   28   42-69    130-157 (176)
</pre><pre> 55 d3tuea_ c.47.1.10 (A:) automat  46.7      67 0.00062   20.0   2.8   16   51-66    126-141 (166)
</pre><pre> 56 d2r37a1 c.47.1.0 (A:38-223) au  46.3      84 0.00078   21.5   3.4   39   28-66    124-162 (186)
</pre><pre> 57 3KIJ_A Probable glutathione pe  46.2      64  0.0006   20.4   2.7   15   51-65    138-152 (180)
</pre><pre> 58 5C04_A Putative peroxiredoxin   46.2      76 0.00071   18.9   2.9   19   51-69    118-136 (153)
</pre><pre> 59 d2he3a1 c.47.1.0 (A:4-187) aut  46.1      64  0.0006   21.6   2.8   25   42-66    143-167 (184)
</pre><pre> 60 2FY6_A Peptide methionine sulf  45.8      76  0.0007   18.5   2.8   15   51-65    109-123 (143)
</pre><pre> 61 5IMF_A Bacterioferritin comigr  45.8      72 0.00066   19.0   2.7   15   51-65    124-138 (160)
</pre><pre> 62 PF08534.12 ; Redoxin ; Redoxin  45.7      93 0.00086   17.9   3.1   19   51-69    108-126 (135)
</pre><pre> 63 3EYT_B uncharacterized protein  45.5      73 0.00067   19.1   2.7   16   51-66    120-135 (158)
</pre><pre> 64 d2b7ka_ c.47.1.10 (A:) Thiored  45.3 1.3E+02  0.0012   19.4   3.9   27   42-68    123-149 (169)
</pre><pre> 65 6B9X_B Ragulator complex prote  45.1      91 0.00084   19.6   3.2   22   48-69     19-40  (126)
</pre><pre> 66 d2ljaa1 c.47.1.0 (A:4-144) aut  44.9      60 0.00056   19.1   2.3   18   51-68    108-125 (141)
</pre><pre> 67 2H30_A Peptide methionine sulf  44.8      77 0.00071   19.2   2.8   15   51-65    124-138 (164)
</pre><pre> 68 d1q98a_ c.47.1.10 (A:) Thiol p  44.7      73 0.00068   20.1   2.8   15   51-65    131-145 (164)
</pre><pre> 69 d2p5qa_ c.47.1.0 (A:) automate  44.6      73 0.00068   19.4   2.7   15   51-65    127-141 (161)
</pre><pre> 70 2LS5_A Uncharacterized protein  44.4      75  0.0007   19.2   2.7   16   51-66    117-132 (159)
</pre><pre> 71 d4evma_ c.47.1.0 (A:) automate  44.3      61 0.00056   18.8   2.2   15   51-65    107-121 (138)
</pre><pre> 72 d2p31a_ c.47.1.0 (A:) automate  44.2      76  0.0007   19.3   2.7   15   51-65    123-137 (155)
</pre><pre> 73 d3cmia_ c.47.1.0 (A:) automate  44.2      78 0.00072   19.3   2.8   16   51-66    127-142 (161)
</pre><pre> 74 2V1M_A GLUTATHIONE PEROXIDASE;  44.0      79 0.00073   18.8   2.7   16   51-66    135-150 (169)
</pre><pre> 75 d3hcza1 c.47.1.0 (A:330-474) a  43.9      75 0.00069   18.9   2.6   15   51-65    111-125 (145)
</pre><pre> 76 4PQ1_A Putative electron trans  43.9      81 0.00075   18.9   2.8   15   51-65    127-141 (158)
</pre><pre> 77 d5um7a_ c.47.1.0 (A:) automate  43.9      80 0.00075   18.6   2.7   15   51-65    111-125 (140)
</pre><pre> 78 4FO5_A thioredoxin-like protei  43.8      66 0.00062   19.1   2.4   16   51-66    116-131 (143)
</pre><pre> 79 4NMU_C Thiol-disulfide oxidore  43.5      85 0.00079   18.4   2.8   15   51-65    115-129 (147)
</pre><pre> 80 4NMU_D Thiol-disulfide oxidore  43.5      85 0.00079   18.4   2.8   15   51-65    115-129 (147)
</pre><pre> 81 d4g2ea1 c.47.1.0 (A:1-155) aut  43.3      81 0.00075   19.2   2.7   15   51-65    119-133 (155)
</pre><pre> 82 3EUR_A uncharacterized protein  43.1      65 0.00061   19.1   2.3   15   51-65    116-130 (142)
</pre><pre> 83 TIGR02540 gpx7; putative gluta  43.1      92 0.00085   19.0   3.0   18   51-68    122-139 (153)
</pre><pre> 84 4HDE_A SCO1/SenC family lipopr  42.8 1.4E+02  0.0013   17.8   4.9   45   22-66    107-151 (170)
</pre><pre> 85 4GQC_B Thiol peroxidase; CXXXX  42.7      80 0.00074   19.2   2.6   15   51-65    123-137 (164)
</pre><pre> 86 d1st9a_ c.47.1.10 (A:) Thiol-d  42.3      96 0.00089   18.2   2.9   16   51-66    105-120 (137)
</pre><pre> 87 2YZH_D Probable thiol peroxida  42.1 1.1E+02 0.00099   18.9   3.1   19   51-69    135-153 (171)
</pre><pre> 88 d3p7xa1 c.47.1.0 (A:1-164) aut  42.0   1E+02 0.00094   19.2   3.1   19   51-69    129-147 (164)
</pre><pre> 89 1LU4_A SOLUBLE SECRETED ANTIGE  42.0      89 0.00083   18.3   2.7   15   51-65    102-116 (136)
</pre><pre> 90 d4gqca1 c.47.1.10 (A:5-163) Ba  41.9      80 0.00074   19.5   2.6   15   51-65    119-133 (159)
</pre><pre> 91 4EVM_A Thioredoxin family prot  41.8      59 0.00055   18.6   1.9   16   51-66    107-122 (138)
</pre><pre> 92 2B5X_A YkuV protein; Thioredox  41.5      99 0.00092   18.4   2.9   16   51-66    113-128 (148)
</pre><pre> 93 2M72_A Uncharacterized thiored  41.3      90 0.00084   18.9   2.7   16   51-66    114-129 (156)
</pre><pre> 94 d2kuca1 c.47.1.0 (A:1-119) aut  41.3 1.2E+02  0.0011   17.5   3.1   18   51-68     86-103 (119)
</pre><pre> 95 d2bmxa1 c.47.1.10 (A:2-170) Al  41.1      89 0.00083   19.8   2.8   16   51-66    133-148 (169)
</pre><pre> 96 4JE1_A Probable thiol peroxida  41.1      85 0.00079   20.3   2.7   15   51-65    142-156 (175)
</pre><pre> 97 d3kija_ c.47.1.0 (A:) automate  41.1      89 0.00083   19.5   2.7   16   51-66    129-144 (171)
</pre><pre> 98 2F9S_A Thiol-disulfide oxidore  41.1      91 0.00085   19.0   2.7   15   51-65    106-120 (151)
</pre><pre> 99 d3l7ha_ d.110.7.1 (A:) automat  40.9      76  0.0007   19.7   2.4   20   48-67      9-28  (86)
</pre><pre>100 TIGR03673 uL14_arch; 50S ribos  40.9      91 0.00084   22.3   3.0   24   39-62     81-104 (131)
</pre><pre>101 d2yzha1 c.47.1.0 (A:1-168) aut  40.8      90 0.00083   19.6   2.7   15   51-65    133-147 (168)
</pre><pre>102 d2xhfa1 c.47.1.0 (A:37-190) au  40.6      77 0.00071   20.0   2.4   15   51-66    124-138 (154)
</pre><pre>103 2P5Q_A Glutathione peroxidase   40.5      95 0.00088   18.6   2.7   15   51-65    136-150 (170)
</pre><pre>104 2P5Q_C Glutathione peroxidase   40.5      95 0.00088   18.6   2.7   15   51-65    136-150 (170)
</pre><pre>105 d4fh8a1 c.47.1.10 (A:1-170) au  40.5      94 0.00087   19.3   2.8   16   51-66    126-141 (170)
</pre><pre>106 6L81_D Mitotic-spindle organiz  40.3      85 0.00079   21.7   2.7   29    3-31     40-68  (85)
</pre><pre>107 d2pn8a1 c.47.1.0 (A:84-269) au  40.2 1.1E+02   0.001   19.6   3.1   19   51-69    118-136 (186)
</pre><pre>108 d2obia_ c.47.1.0 (A:) automate  40.1 1.4E+02  0.0013   18.5   3.4   24   42-65    125-148 (165)
</pre><pre>109 d6b9xb1 d.110.7.1 (B:1-125) au  40.1 1.1E+02 0.00099   20.0   3.1   21   48-68     18-38  (125)
</pre><pre>110 d1xvwa1 c.47.1.10 (A:1-153) Pu  39.9      96 0.00089   18.9   2.7   16   51-66    117-132 (153)
</pre><pre>111 2JSZ_A Probable thiol peroxida  39.9      85 0.00079   21.1   2.7   15   51-65    130-144 (167)
</pre><pre>112 4K9Z_A Putative thiol-disulfid  39.7      72 0.00067   20.0   2.2   16   51-66    127-142 (168)
</pre><pre>113 d1psqa_ c.47.1.10 (A:) Probabl  39.4      96 0.00089   19.2   2.7   15   51-65    128-142 (163)
</pre><pre>114 3RAZ_A Thioredoxin-related pro  39.3   1E+02 0.00095   18.6   2.7   16   51-66    108-123 (151)
</pre><pre>115 d1zofa1 c.47.1.10 (A:1-170) Th  39.1   1E+02 0.00095   19.3   2.8   16   51-66    125-140 (170)
</pre><pre>116 d3eura1 c.47.1.0 (A:1-139) aut  39.1      76  0.0007   18.9   2.1   15   51-65    113-127 (139)
</pre><pre>117 d1o73a_ c.47.1.10 (A:) Trypare  38.9      56 0.00052   19.8   1.6   16   51-66    111-127 (144)
</pre><pre>118 d2b1ka_ c.47.1.10 (A:) automat  38.9 1.1E+02 0.00098   18.6   2.7   16   51-66    110-125 (149)
</pre><pre>119 3OR5_A Thiol:disulfide interch  38.9 1.1E+02   0.001   18.6   2.8   16   51-66    119-134 (165)
</pre><pre>120 d4nmua1 c.47.1.0 (A:1-144) aut  38.4 1.2E+02  0.0011   18.1   2.8   16   51-66    112-127 (144)
</pre><pre>121 d3dwva_ c.47.1.0 (A:) automate  38.3 1.1E+02   0.001   18.7   2.8   25   42-66    123-147 (163)
</pre><pre>122 d3gl3a1 c.47.1.0 (A:30-168) au  38.3   1E+02 0.00097   18.1   2.6   15   51-65    105-119 (139)
</pre><pre>123 4G2E_A Peroxiredoxin; redox pr  38.3 1.1E+02   0.001   18.4   2.8   16   51-66    121-136 (157)
</pre><pre>124 2LRT_A Uncharacterized protein  38.3      87 0.00081   19.3   2.4   16   51-66    116-131 (152)
</pre><pre>125 d5epfa1 c.47.1.0 (A:9-161) aut  38.2   1E+02 0.00097   18.7   2.7   16   51-66    117-132 (153)
</pre><pre>126 d1jfua_ c.47.1.10 (A:) Membran  38.0 1.1E+02   0.001   19.4   2.8   15   51-65    142-156 (176)
</pre><pre>127 3LEQ_A uncharacterized protein  37.5 1.1E+02   0.001   19.0   2.7   20   48-67     18-37  (126)
</pre><pre>128 4HQZ_A Thioredoxin family prot  37.3   1E+02 0.00095   20.0   2.7   15   51-65    156-170 (187)
</pre><pre>129 2B1L_A Thiol:disulfide interch  37.3 1.5E+02  0.0014   17.2   3.1   19   51-69     89-107 (129)
</pre><pre>130 3IXR_A Bacterioferritin comigr  37.0 1.1E+02   0.001   19.4   2.7   15   51-65    143-157 (179)
</pre><pre>131 3P7X_C Probable thiol peroxida  37.0 1.2E+02  0.0011   18.5   2.8   16   51-66    131-146 (166)
</pre><pre>132 6L80_B Mitotic-spindle organiz  36.9 1.1E+02   0.001   21.9   2.9   29    3-31     57-85  (100)
</pre><pre>133 4FYU_A Thioredoxin; five paral  36.9 1.1E+02 0.00099   17.9   2.5   16   51-66    110-125 (145)
</pre><pre>134 d2m72a1 c.47.1.0 (A:4-148) aut  36.9 1.1E+02  0.0011   18.0   2.6   15   51-65    111-125 (145)
</pre><pre>135 PF20029.1 ; DUF6436 ; Domain o  36.9      83 0.00077   18.3   2.0   12   51-62      5-16  (59)
</pre><pre>136 PF18545.3 ; HalOD1 ; Halobacte  36.9   2E+02  0.0019   18.1   4.4   42   21-62     32-73  (76)
</pre><pre>137 d3ewla1 c.47.1.0 (A:1-135) aut  36.7      96 0.00089   18.4   2.3   15   51-65    109-123 (135)
</pre><pre>138 3ERW_A Sporulation thiol-disul  36.4 1.2E+02  0.0011   17.8   2.7   15   51-65    117-131 (145)
</pre><pre>139 3ERW_B Sporulation thiol-disul  36.4 1.2E+02  0.0011   17.8   2.7   15   51-65    117-131 (145)
</pre><pre>140 3T1S_A Gliding protein MglB; G  36.4 1.4E+02  0.0013   19.0   3.2   21   48-68     25-45  (136)
</pre><pre>141 d2fega1 c.47.1.0 (A:8-175) aut  36.3 1.2E+02  0.0011   18.9   2.8   16   51-66    119-134 (168)
</pre><pre>142 2RLI_A SCO2 protein homolog, m  36.3   2E+02  0.0019   17.9   4.4   29   38-66    119-147 (171)
</pre><pre>143 d1knga_ c.47.1.10 (A:) Thiored  36.2 1.4E+02  0.0013   17.9   3.0   18   51-68    109-126 (144)
</pre><pre>144 3LOR_B Thiol-disulfide isomera  36.1 1.3E+02  0.0012   18.1   2.8   15   51-65    123-137 (160)
</pre><pre>145 2L5O_A Putative thioredoxin; S  36.0 1.2E+02  0.0011   18.2   2.7   15   51-65    109-123 (153)
</pre><pre>146 3FKF_D thiol-disulfide oxidore  36.0      96 0.00089   18.5   2.3   23   44-66    110-132 (148)
</pre><pre>147 3IA1_B Thio-disulfide isomeras  36.0 1.2E+02  0.0011   18.5   2.7   15   51-65    112-126 (154)
</pre><pre>148 3HVV_A Thiol peroxidase; oxido  36.0 1.1E+02  0.0011   18.9   2.6   15   51-65    133-147 (167)
</pre><pre>149 2LJA_A Putative thiol-disulfid  35.8      84 0.00078   19.1   2.0   16   51-66    111-126 (152)
</pre><pre>150 d1we0a1 c.47.1.10 (A:1-166) Al  35.6 1.3E+02  0.0012   18.9   2.8   16   51-66    121-136 (166)
</pre><pre>151 6ZU5_LV0 uL14; Microsporidia,   35.6   1E+02 0.00096   20.3   2.5   34   30-64     82-115 (140)
</pre><pre>152 5UM7_A Thioredoxin signature p  35.5 1.2E+02  0.0011   18.6   2.7   16   51-66    157-172 (187)
</pre><pre>153 6HY0_I Major Outer Capsid Prot  35.4      93 0.00086   23.4   2.5   28    9-36     23-51  (149)
</pre><pre>154 3LWA_A Secreted thiol-disulfid  35.4 1.2E+02  0.0011   19.5   2.8   15   51-65    149-163 (183)
</pre><pre>155 1KNG_A THIOL:DISULFIDE INTERCH  35.4 1.4E+02  0.0013   18.1   2.8   16   51-66    120-135 (156)
</pre><pre>156 PF18832.3 ; LPD18 ; Large poly  35.4      84 0.00078   20.9   2.1   13   51-63     47-59  (86)
</pre><pre>157 1PSQ_A probable thiol peroxida  35.3 1.2E+02  0.0011   18.4   2.7   15   51-65    128-142 (163)
</pre><pre>158 d2f8aa1 c.47.1.10 (A:12-195) G  35.2 1.2E+02  0.0011   20.6   2.9   24   43-66    144-167 (184)
</pre><pre>159 PF09695.12 ; YtfJ_HI0045 ; Bac  35.1 1.2E+02  0.0011   18.8   2.7   15   51-65    126-140 (159)
</pre><pre>160 d3keba_ c.47.1.0 (A:) automate  35.0 1.3E+02  0.0012   20.3   2.9   17   49-65    134-150 (175)
</pre><pre>161 d5enua1 c.47.1.0 (A:1-153) aut  34.9 1.1E+02   0.001   18.7   2.4   15   51-65    120-134 (153)
</pre><pre>162 4YGT_A Uncharacterized protein  34.8 2.9E+02  0.0027   19.4   5.6   47   18-64     40-91  (160)
</pre><pre>163 1VQ8_K 50S ribosomal protein L  34.8 1.1E+02 0.00098   19.6   2.4   24   41-64     84-107 (132)
</pre><pre>164 d4mh2a_ c.47.1.0 (A:) automate  34.6 1.4E+02  0.0013   18.4   2.8   16   51-66    124-139 (165)
</pre><pre>165 d2h30a_ c.47.1.10 (A:) Peptide  34.5 1.4E+02  0.0013   18.0   2.7   15   51-65    111-125 (151)
</pre><pre>166 3GL3_C Putative Thiol:disulfid  34.4 1.3E+02  0.0012   18.3   2.7   15   51-65    108-122 (152)
</pre><pre>167 PF07887.13 ; Calmodulin_bind ;  34.2 2.7E+02  0.0025   22.4   4.9   52    7-62    200-254 (293)
</pre><pre>168 d1aisa2 d.129.1.1 (A:93-181) T  34.0 2.4E+02  0.0022   18.1   4.5   51   18-68     20-71  (89)
</pre><pre>169 d3erwa_ c.47.1.0 (A:) automate  33.9 1.4E+02  0.0013   17.5   2.7   15   51-65    106-120 (134)
</pre><pre>170 2LUS_A Thioredoxion; Cr-TRP16,  33.9 1.1E+02 0.00098   17.9   2.2   18   51-68    111-128 (143)
</pre><pre>171 3E0U_A Glutathione peroxidase;  33.9 1.1E+02   0.001   18.2   2.3   16   51-66    130-145 (166)
</pre><pre>172 d1y4oa1 d.110.7.1 (A:10-104) D  33.9 1.1E+02  0.0011   19.1   2.4   21   48-68     15-35  (95)
</pre><pre>173 d3hvva_ c.47.1.10 (A:) Thiol p  33.7 1.3E+02  0.0012   19.0   2.7   15   51-65    133-147 (167)
</pre><pre>174 4QL7_D Alkylhydroperoxide Redu  33.6 1.4E+02  0.0013   18.1   2.7   15   51-65    121-135 (172)
</pre><pre>175 PF00578.23 ; AhpC-TSA ; AhpC/T  33.4 1.1E+02   0.001   17.1   2.1   12   51-62    109-120 (122)
</pre><pre>176 1XVQ_A thiol peroxidase; thior  33.3 1.4E+02  0.0013   18.8   2.8   15   51-65    131-145 (175)
</pre><pre>177 4IFA_A Extracellular protein c  33.3 4.2E+02  0.0039   20.7   5.9   51   19-69     55-109 (339)
</pre><pre>178 PF13778.8 ; DUF4174 ; Domain o  33.2 1.1E+02 0.00099   19.0   2.2   15   51-65     76-90  (112)
</pre><pre>179 6ERI_AK 50S ribosomal protein   33.2 1.8E+02  0.0016   19.2   3.3   32   30-62     63-94  (121)
</pre><pre>180 1I5G_A TRYPAREDOXIN II; trypar  33.2 1.1E+02 0.00099   18.4   2.1   16   51-66    111-127 (144)
</pre><pre>181 3FW2_C thiol-disulfide oxidore  33.1 1.1E+02   0.001   18.5   2.2   15   51-65    119-133 (150)
</pre><pre>182 3FW2_D thiol-disulfide oxidore  33.1 1.1E+02   0.001   18.5   2.2   15   51-65    119-133 (150)
</pre><pre>183 d2b5xa1 c.47.1.10 (A:1-143) th  33.1 1.6E+02  0.0015   17.7   2.9   16   51-66    113-128 (143)
</pre><pre>184 d2v1ma_ c.47.1.0 (A:) automate  32.8 1.5E+02  0.0014   18.2   2.7   16   51-66    129-144 (163)
</pre><pre>185 d3drna_ c.47.1.0 (A:) automate  32.7 1.6E+02  0.0014   17.9   2.8   18   51-68    112-129 (152)
</pre><pre>186 3KH7_A Thiol:disulfide interch  32.6 1.4E+02  0.0013   19.4   2.7   15   51-65    135-149 (176)
</pre><pre>187 d3fw2a1 c.47.1.0 (A:214-360) a  32.6 1.3E+02  0.0012   18.4   2.4   15   51-65    116-130 (147)
</pre><pre>188 PF17510.4 ; GP44 ; Gene produc  32.5 1.4E+02  0.0013   21.8   2.9   41   27-68     28-70  (108)
</pre><pre>189 d4kcea_ c.47.1.10 (A:) automat  32.4 1.4E+02  0.0013   18.8   2.6   15   51-65    123-137 (172)
</pre><pre>190 1O8X_A TRYPAREDOXIN; TRYPAREDO  32.3 1.2E+02  0.0012   18.1   2.3   16   51-66    111-127 (146)
</pre><pre>191 3HA9_A uncharacterized Thiored  32.1 1.4E+02  0.0013   18.2   2.6   16   51-66    133-148 (165)
</pre><pre>192 5XXB_U Ribosomal protein uL14;  31.9 1.4E+02  0.0013   19.5   2.7   24   41-64     91-114 (139)
</pre><pre>193 4GRF_A Putative thiol-disulfid  31.9 1.3E+02  0.0012   18.3   2.4   15   51-65    112-126 (152)
</pre><pre>194 3CMI_A Peroxiredoxin HYR1; Thi  31.7 1.5E+02  0.0014   18.5   2.7   16   51-66    136-151 (171)
</pre><pre>195 3DWV_A Glutathione peroxidase-  31.7 1.5E+02  0.0014   18.6   2.7   25   42-66    142-166 (187)
</pre><pre>196 d2moba_ d.137.1.1 (A:) Soluble  31.6   3E+02  0.0027   19.2   4.2   32   23-54     14-52  (94)
</pre><pre>197 PF00238.21 ; Ribosomal_L14 ; R  31.6 1.6E+02  0.0015   19.4   2.9   30   33-62     65-94  (122)
</pre><pre>198 d1vq8k1 b.39.1.1 (K:1-132) Rib  31.4 1.4E+02  0.0013   21.4   2.8   22   41-62     84-105 (132)
</pre><pre>199 d3iosa_ c.47.1.10 (A:) Lipopro  31.4 1.7E+02  0.0016   17.2   2.8   16   51-66    104-119 (134)
</pre><pre>200 1JFU_A THIOL:DISULFIDE INTERCH  31.3 1.6E+02  0.0015   18.5   2.8   15   51-65    146-160 (186)
</pre><pre>201 3F9U_A Putative exported cytoc  31.2 1.6E+02  0.0014   18.8   2.7   16   51-66    133-148 (172)
</pre><pre>202 2BMX_B ALKYL HYDROPEROXIDASE C  31.0 1.6E+02  0.0015   18.6   2.7   16   51-66    134-149 (195)
</pre><pre>203 2BMX_C ALKYL HYDROPEROXIDASE C  31.0 1.6E+02  0.0015   18.6   2.7   16   51-66    134-149 (195)
</pre><pre>204 3KCM_E Thioredoxin family prot  30.9 1.7E+02  0.0015   17.8   2.7   15   51-65    109-123 (154)
</pre><pre>205 3KCM_F Thioredoxin family prot  30.9 1.7E+02  0.0015   17.8   2.7   15   51-65    109-123 (154)
</pre><pre>206 d3gmxa_ d.98.1.0 (A:) automate  30.8 3.3E+02  0.0031   18.7   5.6   44   20-63     15-63  (154)
</pre><pre>207 2B1K_A Thiol:disulfide interch  30.7 1.7E+02  0.0015   17.9   2.7   16   51-66    128-143 (168)
</pre><pre>208 PF11248.10 ; DUF3046 ; Protein  30.7 1.4E+02  0.0013   19.4   2.4   17   11-30     37-53  (62)
</pre><pre>209 1Q98_A Thiol Peroxidase; struc  30.6 1.7E+02  0.0016   18.1   2.8   15   51-65    132-146 (165)
</pre><pre>210 d1qmva_ c.47.1.10 (A:) Thiored  30.6 1.5E+02  0.0014   19.7   2.7   16   51-66    127-142 (197)
</pre><pre>211 3HCZ_A Possible thiol-disulfid  30.6 2.2E+02   0.002   16.7   3.2   23   43-65    106-128 (148)
</pre><pre>212 d3ijfx_ c.97.1.0 (X:) automate  30.5 1.5E+02  0.0014   19.4   2.7   15   51-65     28-42  (123)
</pre><pre>213 d1whia_ b.39.1.1 (A:) Ribosoma  30.3 1.6E+02  0.0015   20.8   2.9   23   40-62     72-94  (122)
</pre><pre>214 d4grfa1 c.47.1.0 (A:252-389) a  30.2 2.1E+02   0.002   16.8   3.0   22   44-65    102-123 (138)
</pre><pre>215 PF05988.14 ; DUF899 ; Bacteria  30.1 1.8E+02  0.0016   20.5   3.0   19   51-69    177-195 (233)
</pre><pre>216 3DRN_A Peroxiredoxin, bacterio  30.1 1.6E+02  0.0015   18.4   2.6   15   51-65    113-127 (161)
</pre><pre>217 d1prxa_ c.47.1.10 (A:) 1-Cys p  29.9 1.7E+02  0.0016   20.5   2.9   18   51-68    129-146 (220)
</pre><pre>218 PF03259.19 ; Robl_LC7 ; Roadbl  29.6 1.2E+02  0.0011   17.5   1.9   15   48-62     14-28  (88)
</pre><pre>219 6VPD_B Glutathione peroxidase;  29.5 2.2E+02   0.002   17.9   3.1   28   42-69    147-174 (192)
</pre><pre>220 TIGR00385 dsbE; periplasmic pr  29.5 1.6E+02  0.0015   18.4   2.6   16   51-66    140-155 (173)
</pre><pre>221 6ELW_A Phospholipid hydroperox  29.5 1.7E+02  0.0016   19.4   2.8   16   51-66    161-176 (192)
</pre><pre>222 PF11006.10 ; DUF2845 ; Protein  29.5 2.3E+02  0.0022   16.6   4.8   45   19-64     14-76  (79)
</pre><pre>223 1WHI_A RIBOSOMAL PROTEIN L14;   29.4   2E+02  0.0018   19.2   3.1   28   33-60     65-92  (122)
</pre><pre>224 PF08894.13 ; DUF1838 ; Protein  29.4 1.8E+02  0.0017   21.8   3.2   31   35-65     34-65  (231)
</pre><pre>225 TIGR01067 rplN_bact; ribosomal  29.3 1.6E+02  0.0015   20.8   2.7   22   41-62     73-94  (122)
</pre><pre>226 d2d30a1 c.97.1.1 (A:1-124) mon  29.3 1.6E+02  0.0015   19.2   2.6   15   51-65     27-41  (124)
</pre><pre>227 PF08923.12 ; MAPKK1_Int ; Mito  29.2 2.1E+02   0.002   17.6   3.0   22   48-69     16-37  (120)
</pre><pre>228 3HDC_A Thioredoxin family prot  29.0 1.8E+02  0.0017   18.1   2.7   16   51-66    118-133 (158)
</pre><pre>229 d1lu4a_ c.47.1.10 (A:) Soluble  28.9 1.7E+02  0.0016   17.2   2.5   15   51-65    102-116 (134)
</pre><pre>230 4XRA_A Alkyl hydroperoxide red  28.8 2.2E+02  0.0021   17.9   3.1   19   51-69    120-138 (186)
</pre><pre>231 4XRA_E Alkyl hydroperoxide red  28.8 2.2E+02  0.0021   17.9   3.1   19   51-69    120-138 (186)
</pre><pre>232 2P31_B Glutathione peroxidase   28.7 1.6E+02  0.0015   19.1   2.5   16   51-66    149-164 (181)
</pre><pre>233 2C0D_B THIOREDOXIN PEROXIDASE   28.4 1.7E+02  0.0016   19.7   2.7   15   51-65    148-162 (221)
</pre><pre>234 d2v2ga_ c.47.1.10 (A:) automat  28.4 1.7E+02  0.0016   20.5   2.8   16   51-66    128-143 (220)
</pre><pre>235 PF18810.3 ; PBECR2 ; phage-Bar  28.4 3.1E+02  0.0029   17.7   4.1   46   24-69     50-117 (132)
</pre><pre>236 d5k2ia1 c.47.1.0 (A:1-153) aut  28.3 1.5E+02  0.0014   18.8   2.3   14   51-65    125-138 (153)
</pre><pre>237 5XY3_V 60S ribosomal protein L  28.2 1.8E+02  0.0017   19.0   2.7   24   41-64     92-115 (140)
</pre><pre>238 d4llra_ c.47.1.10 (A:) Trypare  28.1   2E+02  0.0018   19.2   2.9   16   51-66    126-141 (194)
</pre><pre>239 5Y63_E Alkyl hydroperoxide red  28.0 1.9E+02  0.0018   18.1   2.7   15   51-65    132-146 (183)
</pre><pre>240 7KIZ_B Peroxiredoxin-2; hydrog  27.9 1.8E+02  0.0017   18.7   2.7   15   51-65    127-141 (197)
</pre><pre>241 7KIZ_G Peroxiredoxin-2; hydrog  27.9 1.8E+02  0.0017   18.7   2.7   15   51-65    127-141 (197)
</pre><pre>242 5EPF_A Peroxiredoxin; Mycobact  27.7   2E+02  0.0019   18.0   2.8   16   51-66    125-140 (162)
</pre><pre>243 6GXG_B Tryparedoxin; Covalent   27.6 1.5E+02  0.0014   17.7   2.1   19   51-69    114-133 (147)
</pre><pre>244 6GXG_C Tryparedoxin; Covalent   27.6 1.5E+02  0.0014   17.7   2.1   19   51-69    114-133 (147)
</pre><pre>245 d3tb2a_ c.47.1.10 (A:) 1-Cys p  27.4 1.8E+02  0.0017   20.2   2.7   15   51-65    129-143 (219)
</pre><pre>246 6UDG_C Thiol peroxidase; SSGCI  27.3   2E+02  0.0018   18.3   2.7   15   51-65    140-154 (173)
</pre><pre>247 d1o8xa_ c.47.1.10 (A:) Trypare  27.2 1.5E+02  0.0014   18.1   2.1   16   51-66    109-125 (144)
</pre><pre>248 2JU5_A Thioredoxin Disulfide I  27.0 2.5E+02  0.0024   17.4   3.1   19   51-69    119-137 (154)
</pre><pre>249 2R37_B Glutathione peroxidase   26.7 1.7E+02  0.0015   20.3   2.5   16   51-66    161-176 (207)
</pre><pre>250 6MG6_C Carbon-nitrogen hydrola  26.2   2E+02  0.0019   20.8   2.8   20   50-69    106-125 (300)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF14076.8 ; DUF4258 ; Domain of unknown function (DUF4258)
</pre><pre>Probab=98.06  E-value=8.3e-05  Score=41.32  Aligned_cols=59  Identities=19%  Similarity=0.277  Sum_probs=42.3  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHhcCCCCCCHHHHHHHHhCCceeeccc----CCcEEE----EcCCEEEEECCCC-----CEEEEee
</pre><pre>Q WP_134219109.1    5 FSRHAINQAISRDGVGVSSEAILDAVNNPTKVIPQV----GGTIKY----VGERAVVILNEEG-----KVVTTWA   66 (69)
</pre><pre>Q Consensus         5 ~TkHGl~qai~RdG~GVs~~AildAv~nP~~v~~q~----g~~~~y----~Gk~A~VVlN~~G-----kVVTtw~   66 (69)
</pre><pre>                      ||+|..++.-.|   |++.+.|.+++++|..+....    +..+..    .|+.-.||+...+     .|||+|.
</pre><pre>T Consensus         1 ~s~Ha~~~~~~r---~is~~~i~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~i~vv~~~~~~~~~~~viT~~~   72 (72)
</pre><pre>T F4KS02_HALH1/2    1 WRRHVLEKMLER---GISRGEVLQVLLNGEVIQRYDEDKPFPSMLVLGFSMDRPLHVVTSFDENNQVVFVITTYE   72 (72)
</pre><pre>T ss_pred             CchHHHHHHHHc---CCCHHHHHHHHHhCeeeEecCCCCCCCeEEEEEecCCceEEEEEEEeccCCeEEEEEeeC
</pre><pre>Confidence            689999999999   999999999999999654221    133333    3444466666554     7788873
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF12106.10 ; Colicin_E5 ; Colicin E5 ribonuclease domain
</pre><pre>Probab=94.71  E-value=0.9  Score=28.50  Aligned_cols=61  Identities=23%  Similarity=0.321  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHhcCCCCCCHHHHHHHHhCCceeec----ccC-CcEEEEcC-CEEEEECCC-CCEEEEeec-C
</pre><pre>Q WP_134219109.1    5 FSRHAINQAISRDGVGVSSEAILDAVNNPTKVIP----QVG-GTIKYVGE-RAVVILNEE-GKVVTTWAR-N   68 (69)
</pre><pre>Q Consensus         5 ~TkHGl~qai~RdG~GVs~~AildAv~nP~~v~~----q~g-~~~~y~Gk-~A~VVlN~~-GkVVTtw~~-~   68 (69)
</pre><pre>                      +..+..+|...|   |-+.+.|.+++++|.....    ..+ ....|.-+ ...||+|.. |++|-...+ +
</pre><pre>T Consensus         5 ~~~k~~~q~~~r---gwt~~~i~~ti~~~~~~~~~~~~~~~~~at~Y~~~~g~yvv~~d~tg~iiqvs~~~~   73 (82)
</pre><pre>T M1MKY3_9CLOT/9    5 STQKLSNQMTQR---GWTESTVRDIVSKPYTTRASINKATGNSATVYYNKAGGYVIIDDITKSVVQVSDNIN   73 (82)
</pre><pre>T ss_pred             cchHHHHhhhhc---CCCHHHHHHHHhCCcceeeeeeccCCCceeEEEeCCCCEEEEcCCCCcEEEecCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>TIGR02530 flg_new; flagellar operon protein. Members of this family are found in a subset of bacterial flagellar operons, generally between genes designated flgD and flgE, in species as diverse as Bacillus halodurans and various other Firmicutes, Geobacter sulfurreducens, and Bdellovibrio bacteriovorus.
</pre><pre>Probab=94.01  E-value=0.29  Score=33.42  Aligned_cols=63  Identities=14%  Similarity=0.312  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHhcCCCCCCHHHHHHHHhCCceeecccC--CcEEEEcCCEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1    5 FSRHAINQAISRDGVGVSSEAILDAVNNPTKVIPQVG--GTIKYVGERAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus         5 ~TkHGl~qai~RdG~GVs~~AildAv~nP~~v~~q~g--~~~~y~Gk~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ||+|.+++...| |+-.++..+. -+..-.....+.|  .+......-|.||==+++.|||+..+..
</pre><pre>T Consensus        17 ~S~HA~~Rl~eR-~I~l~~~~~~-~l~~av~~A~~KG~keslvl~~~~AfIVsv~n~TVITa~d~~~   81 (96)
</pre><pre>T TIGR02530        17 LSKHALERMRER-NISINPDDWK-KLLEAVEEAESKGVKDSLILMNDAALVVSLKNRTVITAMDKDE   81 (96)
</pre><pre>T ss_pred             ccHHHHHHHHHC-CCCCCHHHHH-HHHHHHHHHHHcCCCcEEEEeCCeEEEEEcCCCeEEEeeeHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>d1v74a_ d.243.1.1 (A:) Colicin D nuclease domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=93.48  E-value=1.5  Score=29.88  Aligned_cols=61  Identities=25%  Similarity=0.514  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHhcC-CCCCCH------------HHHHHHHhCCceee----cccCCcEEEEc--CCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1    5 FSRHAINQAISRD-GVGVSS------------EAILDAVNNPTKVI----PQVGGTIKYVG--ERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus         5 ~TkHGl~qai~Rd-G~GVs~------------~AildAv~nP~~v~----~q~g~~~~y~G--k~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      |+...|+..+... .-|+.+            ++|.+-+++|..+.    ....+...|..  .+-+||++++|..+|.|
</pre><pre>T Consensus        10 ~~~~ql~kkfkHa~DFG~~~~~~n~~~~~~f~~aI~~h~~~~~tv~~GtYr~~~~~~v~fn~~t~~~Vv~~~dG~f~SGw   89 (107)
</pre><pre>T d1v74a_          10 FSRKQLDKKYKHAGDFGISDTKKNRETLTKFRDAIEEHLSDKDTVEKGTYRREKGSKVYFNPNTMNVVIIKSNGEFLSGW   89 (107)
</pre><pre>T ss_dssp             BCHHHHHHHGGGGGGGTCCCCCCSHHHHHHHHHHHHHHHHCTTCEECCBCTTSTTCEEEECTTTCEEEEECTTSCEEEEE
</pre><pre>T ss_pred             cchHHHHHHhccccccccCCCCCCHHHHHHHHHHHHHHhcCccceeeceeecCCCeEEEECCCCCcEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>3U97_A Ribonuclease toxin BrnT; RNAse Sa/RelE small ribonuclease fold, ribonuclease, BrnA, HYDROLASE; HET: MSE; 1.102A {Brucella abortus}
</pre><pre>Probab=91.52  E-value=2  Score=28.31  Aligned_cols=62  Identities=16%  Similarity=0.243  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             ccHHH-HHHHHhcCCCCCCHHHHHHHHhCCceeecccCCcEEEEcCCE-----EEEECCCC----CEEEEeecCC
</pre><pre>Q WP_134219109.1    5 FSRHA-INQAISRDGVGVSSEAILDAVNNPTKVIPQVGGTIKYVGERA-----VVILNEEG----KVVTTWARNR   69 (69)
</pre><pre>Q Consensus         5 ~TkHG-l~qai~RdG~GVs~~AildAv~nP~~v~~q~g~~~~y~Gk~A-----~VVlN~~G----kVVTtw~~~~   69 (69)
</pre><pre>                      ++.+- ......+   |++.+.+.+++.+|..+.....+.+...|...     +|+....+    ++||+|..+.
</pre><pre>T Consensus        38 wd~~~~~~~~~kh---Gis~~ev~~v~~~~~~~~~~~e~R~~~iG~~~~~r~l~Vv~~~~~~~~iriITar~a~~  109 (114)
</pre><pre>T 3U97_A           38 WDEPKRQTNIAKH---GLDFADLHFEFFLSAKVFPTKADRLMAIGEFNGLIIIAVIFKPVGSEALSVISMRSASQ  109 (114)
</pre><pre>T ss_dssp             CCHHHHHHHHHHH---SCCGGGCCHHHHHTSEEEEEETTEEEEEEEETTTEEEEEEEEEETTTEEEEEEEEECCC
</pre><pre>T ss_pred             cCHHHHHHHHHHH---CCCHHHHHHHhcCCceEeeCCCCcEEEEEEECCEEEEEEEEEEcCCCeEEEEECeeCCH
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>2FHZ_B Colicin-E5; Protein-Protein Complex, Inhibition of Ribonuclease, IMMUNE SYSTEM, HYDROLASE; 1.15A {Escherichia coli} SCOP: d.243.1.2
</pre><pre>Probab=91.35  E-value=1.9  Score=28.66  Aligned_cols=64  Identities=19%  Similarity=0.238  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHhcC--CCCCCHHHHHHHHhCCceeec--------------ccC-CcEEEEcCCEEEEECCC-CCEEEEee
</pre><pre>Q WP_134219109.1    5 FSRHAINQAISRD--GVGVSSEAILDAVNNPTKVIP--------------QVG-GTIKYVGERAVVILNEE-GKVVTTWA   66 (69)
</pre><pre>Q Consensus         5 ~TkHGl~qai~Rd--G~GVs~~AildAv~nP~~v~~--------------q~g-~~~~y~Gk~A~VVlN~~-GkVVTtw~   66 (69)
</pre><pre>                      ++...|...|.+.  .+|-+.+.|.+++.+|.....              ..+ ....|.-+...||+|.. |.||-...
</pre><pre>T Consensus         9 ~~~~~~~~Ki~~q~~~rgwt~~~i~~ti~~~~~~~~~~~r~~~~~~~~~~~~~~~atvY~~~g~yvvv~d~tg~ivqvs~   88 (108)
</pre><pre>T 2FHZ_B            9 IPGLKIDQKIRGQMPERGWTEDDIKNTVSNGATGTSFDKRSPKKTPPDYLGRNDPATVYGSPGKYVVVNDRTGEVTQISD   88 (108)
</pre><pre>T ss_dssp             ---CBCCHHHHHHGGGTTCCHHHHHHHHHTCEEEEEEEEECGGGSTTTTCCEEEEEEEEEETTEEEEEETTTCBEEEECC
</pre><pre>T ss_pred             CCCcccCHhHhccCCcCCCCHHHHHHHHhcCCccceecCCCCCCCCCCCCCCCCCceEEccCCceEEEeCCCCcEEEecC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cC
</pre><pre>Q WP_134219109.1   67 RN   68 (69)
</pre><pre>Q Consensus        67 ~~   68 (69)
</pre><pre>                      ++
</pre><pre>T Consensus        89 ~~   90 (108)
</pre><pre>T 2FHZ_B           89 KT   90 (108)
</pre><pre>T ss_dssp             TT
</pre><pre>T ss_pred             CC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>5D0O_E Outer membrane protein assembly factor BamE; E.coli, Bacterial outer membrane beta barrel assembly machinery, outer membrane biogenesis, protein transport., PROTEIN TRANSPORT; 2.9A {Escherichia coli}
</pre><pre>Probab=90.12  E-value=3.2  Score=25.73  Aligned_cols=44  Identities=23%  Similarity=0.320  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEEE-----------cCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKYV-----------GERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y~-----------Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      .|.+.+++.+.+..|..+....+..+.|.           .....|.++++|+|+
</pre><pre>T Consensus        48 ~G~t~~ev~~~lG~P~~~~~~~~~~~~y~~~~~~~~~~~~~~~~~v~f~~~g~v~  102 (123)
</pre><pre>T 5D0O_E           48 VGMTQQQVAYALGTPLMSDPFGTNTWFYVFRQQPGHEGVTQQTLTLTFNSSGVLT  102 (123)
</pre><pre>T ss_dssp             SSCBHHHHHHHHCCCSBCCCSSSCEEEEEEEEECTTTCCEEEEEEEEECTTSBEE
</pre><pre>T ss_pred             CCCCHHHHHHHHCCCceeCCCCCceEEEEEEecCCCCCceeEEEEEEECCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>5EKQ_E Outer membrane protein assembly factor BamE; membrane protein, insertase, beta-barrel, outer membrane protein; 3.392A {Escherichia coli (strain K12)}
</pre><pre>Probab=89.24  E-value=4.1  Score=24.68  Aligned_cols=44  Identities=23%  Similarity=0.279  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEEEcC-----------CEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKYVGE-----------RAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y~Gk-----------~A~VVlN~~GkVV   62 (69)
</pre><pre>                      .|.+.+.+++.+-.|..........+.|...           ...|.+|++|+|+
</pre><pre>T Consensus        29 ~G~t~~eV~~~lG~P~~~~~~~~~~w~y~~~~~~~~~~~~~~~~~v~F~~~g~v~   83 (94)
</pre><pre>T 5EKQ_E           29 VGMTQQQVAYALGTPLMSDPFGTNTWFYVFRQQPGHEGVTQQTLTLTFNSSGVLT   83 (94)
</pre><pre>T ss_dssp             TTCCSTTTTTSSCCCSEECCSSSCEEEEEEEEECTTTCCEEEEEEEEECTTSCEE
</pre><pre>T ss_pred             CCCCHHHHHHHhCCCeeeCCCCCceEEEEEEEecCCCCeEEEEEEEEECCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>2YH9_B SMALL PROTEIN A; LIPOPROTEIN, 3D DOMAIN SWAP, MEMBRANE PROTEIN; HET: MSE; 1.8A {ESCHERICHIA COLI}
</pre><pre>Probab=89.21  E-value=6.6  Score=24.06  Aligned_cols=50  Identities=20%  Similarity=0.238  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEEEcCCE-----------EEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKYVGERA-----------VVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y~Gk~A-----------~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      .|.+.+++++.+-.|..........+.|.....           .|+++.+|+++-.--.+
</pre><pre>T Consensus        15 ~G~T~~eV~~~lG~P~~~~~~~~~~~~Y~~~~~~~~~~~~~~~~~v~Fd~~g~v~~~~~~~   75 (88)
</pre><pre>T 2YH9_B           15 VGMTQQQVAYALGTPLMSDPFGTNTWFYVFRQQPGHEGVTQQTLTLTFNSSGVLTNIDNKP   75 (88)
</pre><pre>T ss_dssp             TTCCHHHHHHHHCSCSCCCCTTCSEEEEECCSCTTCCSSSCCEEEEEECTTSCEEEEEEES
</pre><pre>T ss_pred             CCCCHHHHHHHhCCCceeCCCCCceEEEEEEeecCCCCceEEEEEEEECCCCeEEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>PF04355.15 ; SmpA_OmlA ; SmpA / OmlA family
</pre><pre>Probab=89.14  E-value=4.9  Score=22.47  Aligned_cols=45  Identities=16%  Similarity=0.136  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEEE-----------cCCEEEEECCCCCEEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKYV-----------GERAVVILNEEGKVVT   63 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y~-----------Gk~A~VVlN~~GkVVT   63 (69)
</pre><pre>                      .|.+.++|.+.+..|..+.......+.|.           .....|.++.+|+|..
</pre><pre>T Consensus        14 ~Gmt~~eV~~~lG~P~~~~~~~~~~~~y~~~~~~~~~~~~~~~~~i~f~~~~~v~~   69 (71)
</pre><pre>T BAME_NEIMB/36-   14 PGMTKDQVLLLLGSPILRDAFHTDRWDYTFNTSRNGIIKERSNLTVYFENGVLVRT   69 (71)
</pre><pre>T ss_pred             CCCCHHHHHHHHCCCeEecCCCCCeEEEEEEeecCCeeeeEEEEEEEEcCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>PF04365.15 ; BrnT_toxin ; Ribonuclease toxin, BrnT, of type II toxin-antitoxin system
</pre><pre>Probab=88.33  E-value=6.1  Score=22.56  Aligned_cols=62  Identities=13%  Similarity=0.147  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ccHH-HHHHHHhcCCCCCCHHHHHHHHhCCceeeccc------CCcEEEEc-----CCEEEEECCCCC---EEEEeecCC
</pre><pre>Q WP_134219109.1    5 FSRH-AINQAISRDGVGVSSEAILDAVNNPTKVIPQV------GGTIKYVG-----ERAVVILNEEGK---VVTTWARNR   69 (69)
</pre><pre>Q Consensus         5 ~TkH-Gl~qai~RdG~GVs~~AildAv~nP~~v~~q~------g~~~~y~G-----k~A~VVlN~~Gk---VVTtw~~~~   69 (69)
</pre><pre>                      |..+ -......|   |++.+.+.+++.+|..+....      ...+...|     .-.+||+...+.   +||+|..+.
</pre><pre>T Consensus         1 wd~~k~~~~~~~~---gi~~~ev~~~~~~~~~~~~~~~~~~~~~~r~~~~g~~~~~~~~~vv~~~~~~~~~iita~~a~~   77 (79)
</pre><pre>T B0JP89_MICAN/5    1 WDEEKAKRNLAKH---GVSFDEAKSVFYDPLFLTFADPEHSLGERRFIIMGESAKGRLLVVSYTDRTETTRLISARLATT   77 (79)
</pre><pre>T ss_pred             CCHHHHHHHHHHh---CCCHHHHHHHhcCCCcceecCcccCCCCceEEEEEEccCCeEEEEEEEeCCCeEEEEEeeeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>d2fhzb_ d.243.1.2 (B:) Colicin E5 {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=85.06  E-value=6.7  Score=27.25  Aligned_cols=55  Identities=20%  Similarity=0.251  Sum_probs=0.0  Template_Neff=4.000
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHhcCCCCCCHHHHHHHHhCCceeeccc---------------CCcEEEEcCCEEEEECCC-CCEE
</pre><pre>Q WP_134219109.1    5 FSRHAINQAISRDGVGVSSEAILDAVNNPTKVIPQV---------------GGTIKYVGERAVVILNEE-GKVV   62 (69)
</pre><pre>Q Consensus         5 ~TkHGl~qai~RdG~GVs~~AildAv~nP~~v~~q~---------------g~~~~y~Gk~A~VVlN~~-GkVV   62 (69)
</pre><pre>                      ++++=.+|...|   |=+.+.|.+++.+|..-..-.               .....|.-+..-||+|.. |+||
</pre><pre>T Consensus         3 ~~~Ki~~qm~~R---GWt~~~i~~~i~~~~~~~s~~~~~~~~t~~d~l~~~d~atvY~~~g~YVvvnd~tg~iV   73 (93)
</pre><pre>T d2fhzb_           3 IDQKIRGQMPER---GWTEDDIKNTVSNGATGTSFDKRSPKKTPPDYLGRNDPATVYGSPGKYVVVNDRTGEVT   73 (93)
</pre><pre>T ss_dssp             CCHHHHHHGGGT---TCCHHHHHHHHHTCEEEEEEEEECGGGSTTTTCCEEEEEEEEEETTEEEEEETTTCBEE
</pre><pre>T ss_pred             cchHHHhccccc---CCCHHHHHHHHhcCCCCcccccCCCCCCCccccCCCCCccccccCCcEEEEeCCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>5WAM_B Outer membrane protein assembly factor BamE; beta-barrel assembly machinery, BAM complex, lipoprotein, MEMBRANE PROTEIN; 2.45A {Neisseria gonorrhoeae (strain ATCC 700825 / FA 1090)}
</pre><pre>Probab=82.03  E-value=16  Score=23.13  Aligned_cols=46  Identities=17%  Similarity=0.195  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEE-----------EcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKY-----------VGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y-----------~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .|.+.+++...+-.|..+....+..+.|           ....-.|.++ +|+++..+
</pre><pre>T Consensus        32 ~Gmt~~eV~~~lG~P~~~~~~~~~~~~Y~~~~~~~~~~~~~~~~~v~F~-~g~v~~~~   88 (108)
</pre><pre>T 5WAM_B           32 PGMTKDQVLLLLGSPILRDAFHTDRWDYTFNTSRNGIIKERSNLTVYFE-NGVLVRTE   88 (108)
</pre><pre>T ss_dssp             TTCBHHHHHHHHCCCSCCSSTTSSEEEEEEEEEETTEEEEEEEEEEEEE-TTEEEEEE
</pre><pre>T ss_pred             CCCCHHHHHHHHCCCeeecCCCCCeEEEEEEeecCCcceeEEEEEEEEe-CCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>1V74_A Colicin D; colicin D - ImmD complex, cytotoxicity, transfer RNase, protein-protein inhibition, ANTIBIOTIC-IMMUNE SYSTEM COMPLEX; HET: 1PE; 2.0A {Escherichia coli} SCOP: d.243.1.1
</pre><pre>Probab=80.18  E-value=21  Score=22.13  Aligned_cols=46  Identities=24%  Similarity=0.468  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCceee----cccCCcEEEEcCCE--EEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   23 SEAILDAVNNPTKVI----PQVGGTIKYVGERA--VVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        23 ~~AildAv~nP~~v~----~q~g~~~~y~Gk~A--~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ..+|.+-+.+|..+.    ....+...|..+..  +|+.+.+|+.+|.|.-+
</pre><pre>T Consensus        41 ~~~i~~~~~~~~~~~~g~y~~~~~~~~~~~~~t~~~vv~~~~G~f~sg~~l~   92 (107)
</pre><pre>T 1V74_A           41 RDAIEEHLSDKDTVEKGTYRREKGSKVYFNPNTMNVVIIKSNGEFLSGWKIN   92 (107)
</pre><pre>T ss_dssp             HHHHHHHHHCTTCEECCBCTTSTTCEEEECTTTCEEEEECTTSCEEEEEECC
</pre><pre>T ss_pred             HHHHHHHhhcCCceeeeEecCCCCcEEEeCCCCCeEEEEcCCCcEEeeEecC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>PF20132.1 ; DUF6522 ; Family of unknown function (DUF6522)
</pre><pre>Probab=74.25  E-value=33  Score=22.76  Aligned_cols=44  Identities=11%  Similarity=0.172  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHhCCceee-------cccC---CcEEEEcCCEEEEECCCCCEEE
</pre><pre>Q WP_134219109.1   20 GVSSEAILDAVNNPTKVI-------PQVG---GTIKYVGERAVVILNEEGKVVT   63 (69)
</pre><pre>Q Consensus        20 GVs~~AildAv~nP~~v~-------~q~g---~~~~y~Gk~A~VVlN~~GkVVT   63 (69)
</pre><pre>                      |++++++.+.++.-.-..       +..|   -+|+|-....-++++++|.|+.
</pre><pre>T Consensus        23 gls~~~l~~~mr~G~Its~~E~G~geD~Gr~Rltf~~~~r~~r~ivd~~G~vl~   76 (81)
</pre><pre>T A0A1M7GRH4_9RH   23 DLTPDDVRTRIRNGHITSLCEAGQDEDEGRWRITFYAGARALRLIVDRDGTHLS   76 (81)
</pre><pre>T ss_pred             CCCHHHHHHHHHCCCeEEEEEeEeCCCCCeEEEEEEECCEEEEEEECCCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>4WBJ_B Blr1131 protein; copper, chaperone, cytochrome oxidase, CuA; HET: SIN; 1.3A {Bradyrhizobium diazoefficiens} SCOP: c.47.1.0
</pre><pre>Probab=72.92  E-value=33  Score=20.56  Aligned_cols=43  Identities=12%  Similarity=0.326  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   24 EAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        24 ~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+...+.-+.....+..+.+.+.+.-..+++|++|+++-.+.
</pre><pre>T Consensus       106 ~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i~~~~~  148 (164)
</pre><pre>T 4WBJ_B          106 AKVITSYRVYAKKVPTKDGDYTMDHTALIYLMDRDGRFVSPFN  148 (164)
</pre><pre>T ss_dssp             HHHHHHTTCCEEEEECSTTCEEEEECCCCEEECTTSCEEEECC
</pre><pre>T ss_pred             HHHHHHcCceEeecCCCCCceeeeccCEEEEECCCCcEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>PF11429.10 ; Colicin_D ; Colicin D
</pre><pre>Probab=72.59  E-value=32  Score=20.24  Aligned_cols=46  Identities=24%  Similarity=0.352  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCceee-cc---cCCcEEEEcCCE--EEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   23 SEAILDAVNNPTKVI-PQ---VGGTIKYVGERA--VVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        23 ~~AildAv~nP~~v~-~q---~g~~~~y~Gk~A--~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ..+|.+-+++|..+. ..   ..+...|..+..  .|+.+++|+.+|.|.-+
</pre><pre>T Consensus        27 ~~~i~~~~~~~~~~~~~g~~~~~~~~~~~~~~t~~~v~~~~~G~~~s~~~~~   78 (85)
</pre><pre>T K0ESY2_9NOCA/2   27 KDALQKHIDNPDTKHIQGTYRGDPAILNYNPNTGLVVVQSPTGGFVTGWKVN   78 (85)
</pre><pre>T ss_pred             HHHHHHhhcCCCceEEeeEEcCCccEEEEeCCCCeEEEECCCCCeeeeeecC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>2GGT_B SCO1 protein homolog, mitochondrial; Copper chaperone, Cu-binding protein, mitochondrial assembly factor, redox, Nickel, Disuplhide, Mitochondrion, CHAPERONE; 2.4A {Homo sapiens} SCOP: c.47.1.10
</pre><pre>Probab=71.70  E-value=34  Score=21.02  Aligned_cols=32  Identities=6%  Similarity=0.156  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             cccCCcEEEEcCCEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   38 PQVGGTIKYVGERAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        38 ~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ....+.+.+.+....++++++|+++..|....
</pre><pre>T Consensus       116 ~~~~~~~~~~~~~~~~lid~~g~v~~~~~~~~  147 (164)
</pre><pre>T 2GGT_B          116 KDEDEDYIVDHTIIMYLIGPDGEFLDYFGQNK  147 (164)
</pre><pre>T ss_dssp             CCTTTCCCEEECCCEEEECTTSCEEEEECSSC
</pre><pre>T ss_pred             CCCCCCeEeecCcEEEEECCCCCEEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>4WBR_D Blr1131 protein; copper, chaperone, cytochrome oxidase, CuA; 1.4A {Bradyrhizobium diazoefficiens} SCOP: c.47.1.0
</pre><pre>Probab=69.45  E-value=39  Score=20.06  Aligned_cols=43  Identities=12%  Similarity=0.326  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   24 EAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        24 ~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..................+.+...+--..+++|++|+++..+.
</pre><pre>T Consensus        99 ~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i~~~~~  141 (157)
</pre><pre>T 4WBR_D           99 AKVITSYRVYAKKVPTKDGDYTMDHTALIYLMDRDGRFVSPFN  141 (157)
</pre><pre>T ss_dssp             HHHHHHTTCCEEEEECSTTCEEEEECCCEEEECTTSCEEEECC
</pre><pre>T ss_pred             HHHHHHcCCeEEEccCCCCCeeeeeccEEEEECCCCCEEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>d2rlia1 c.47.1.0 (A:100-266) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=68.02  E-value=40  Score=20.98  Aligned_cols=30  Identities=17%  Similarity=0.200  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             ccCCcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   39 QVGGTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        39 q~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      .....+...+--.++++|++|+++..|...
</pre><pre>T Consensus       116 ~~~~~~~~~~~p~~~lid~~g~i~~~~~~~  145 (167)
</pre><pre>T d2rlia1         116 DEDQDYIVDHSIAIYLLNPDGLFTDYYGRS  145 (167)
</pre><pre>T ss_dssp             CSSCCCCEECCCEEEEECTTSCEEEEEESS
</pre><pre>T ss_pred             CCCCceEEecccEEEEECCCCcEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>4BPY_A SCO PROTEIN; COPPER HOMEOSTASIS, CHAPERONE, HOMEOSTASIS; 1.4A {STREPTOMYCES LIVIDANS}
</pre><pre>Probab=65.87  E-value=44  Score=20.37  Aligned_cols=28  Identities=14%  Similarity=0.096  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             cccCCcEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   38 PQVGGTIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        38 ~q~g~~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .+..+.+.+...-..+|+|++|+++..|
</pre><pre>T Consensus       125 ~~~~~~~~~~~~~~~~lid~~g~i~~~~  152 (175)
</pre><pre>T 4BPY_A          125 KDKDGKIVSTHGTQVVAFSPKSDAGYVL  152 (175)
</pre><pre>T ss_dssp             -----------CCCEEEECTTTCCEEEE
</pre><pre>T ss_pred             CCCCCCEEeecCceEEEECCCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>6N5U_A Protein SCO1 homolog 1, mitochondrial; thioredoxin fold, metal ion, reduced form, metallochaperone, HCC1, METAL BINDING PROTEIN; 2.66A {Arabidopsis thaliana} SCOP: c.47.1.0
</pre><pre>Probab=64.39  E-value=53  Score=19.65  Aligned_cols=46  Identities=7%  Similarity=0.009  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   23 SEAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        23 ~~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ...+.+.+.-+..........+......+.+++|++|+++..|...
</pre><pre>T Consensus       106 ~~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i~~~~~~~  151 (170)
</pre><pre>T 6N5U_A          106 IKSVARSYRVYYMKTEEEDSDYLVDHSIVMYLMSPEMNFVKFYGKN  151 (170)
</pre><pre>T ss_dssp             HHHHHHHTTCCEEEEECSSSSEEEEECCCEEEECTTCCEEEEECTT
</pre><pre>T ss_pred             HHHHHHHcCCceeecCCCCCceEEecceEEEEECCCCCEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>6N5U_C Protein SCO1 homolog 1, mitochondrial; thioredoxin fold, metal ion, reduced form, metallochaperone, HCC1, METAL BINDING PROTEIN; 2.66A {Arabidopsis thaliana} SCOP: c.47.1.0
</pre><pre>Probab=64.39  E-value=53  Score=19.65  Aligned_cols=46  Identities=7%  Similarity=0.009  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   23 SEAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        23 ~~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ...+.+.+.-+..........+......+.+++|++|+++..|...
</pre><pre>T Consensus       106 ~~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i~~~~~~~  151 (170)
</pre><pre>T 6N5U_C          106 IKSVARSYRVYYMKTEEEDSDYLVDHSIVMYLMSPEMNFVKFYGKN  151 (170)
</pre><pre>T ss_dssp             HHHHHHHTTCCEEEEECSSSSEEEEECCCEEEECTTSCEEEEECTT
</pre><pre>T ss_pred             HHHHHHHcCCceeecCCCCCceEEecceEEEEECCCCCEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>PF12554.10 ; MOZART1 ; Mitotic-spindle organizing gamma-tubulin ring associated
</pre><pre>Probab=64.28  E-value=23  Score=21.51  Aligned_cols=29  Identities=31%  Similarity=0.454  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CcccHHHHHHHHhcCCCCCCHHHHHHHHh
</pre><pre>Q WP_134219109.1    3 TGFSRHAINQAISRDGVGVSSEAILDAVN   31 (69)
</pre><pre>Q Consensus         3 tG~TkHGl~qai~RdG~GVs~~AildAv~   31 (69)
</pre><pre>                      ||.+++-|.-.+.-=..||+|+++.+.|+
</pre><pre>T Consensus        16 tgLd~e~L~ici~L~e~GvnPeaLA~vi~   44 (47)
</pre><pre>T A0A0D2X146_CAP   16 TGLDRETLAICVSLCESGVNPEALAAVVK   44 (47)
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHcCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>d1in0a1 d.58.49.1 (A:2-89) Hypothetical protein HI1034 {Haemophilus influenzae [TaxId: 727]}
</pre><pre>Probab=63.19  E-value=25  Score=24.08  Aligned_cols=45  Identities=22%  Similarity=0.406  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   21 VSSEAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        21 Vs~~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      |...++.+||.+-.+-+.   .+|-|.|-++.+.+|+..+.||..+.+
</pre><pre>T Consensus         9 vd~qev~NAv~qa~kEi~---~RyDFKg~~~~ie~~~~~~~I~l~a~~   53 (88)
</pre><pre>T d1in0a1           9 ITLHEVRNAVENANRVLS---TRYDFRGVEAVIELNEKNETIKITTES   53 (88)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHHHHHT---TBGGGTTCCEEEEEETTTTEEEEEESC
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHh---hccccCCCceEEEEcCCCceEEEEECC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>d4hqza_ c.47.1.0 (A:) automated matches {Streptococcus pneumoniae [TaxId: 170187]}
</pre><pre>Probab=57.33  E-value=39  Score=19.76  Aligned_cols=16  Identities=13%  Similarity=0.239  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..+.
</pre><pre>T Consensus       110 ~~~~id~~g~i~~~~~  125 (141)
</pre><pre>T d4hqza_         110 TEYLIDSQGKIGKIQF  125 (141)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>d1i5ga_ c.47.1.10 (A:) Tryparedoxin II {Crithidia fasciculata [TaxId: 5656]}
</pre><pre>Probab=55.84  E-value=24  Score=21.59  Aligned_cols=16  Identities=31%  Similarity=0.522  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++ |+|+..|.
</pre><pre>T Consensus       111 ~~~l~d~~~g~v~~~~~  127 (144)
</pre><pre>T d1i5ga_         111 TLVGVEADSGNIITTQA  127 (144)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCH
</pre><pre>T ss_pred             EEEEEeCCCCcEEehhh
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>d3ixra1 c.47.1.0 (A:1-158) automated matches {Xylella fastidiosa [TaxId: 2371]}
</pre><pre>Probab=55.60  E-value=36  Score=20.80  Aligned_cols=16  Identities=13%  Similarity=0.594  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+|+..+.
</pre><pre>T Consensus       123 ~~~lid~~g~v~~~~~  138 (158)
</pre><pre>T d3ixra1         123 STFLIGPTHRIVEAWR  138 (158)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEC
</pre><pre>T ss_pred             cEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>d4wbja_ c.47.1.0 (A:) automated matches {Bradyrhizobium diazoefficiens [TaxId: 224911]}
</pre><pre>Probab=54.54  E-value=88  Score=19.09  Aligned_cols=43  Identities=12%  Similarity=0.315  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   23 SEAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        23 ~~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ...+.....-+..........+...+--+.+++|++|+++..|
</pre><pre>T Consensus        98 ~~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i~~~~  140 (157)
</pre><pre>T d4wbja_          98 IAKVITSYRVYAKKVPTKDGDYTMDHTALIYLMDRDGRFVSPF  140 (157)
</pre><pre>T ss_dssp             HHHHHHHTTCCEEEEECSTTCEEEEECCCCEEECTTSCEEEEC
</pre><pre>T ss_pred             HHHHHHHcCceeeecCCCCCCeeeecccEEEEECCCCCEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>3GMX_A BLP; 2-layer alpha/beta sandwich, PROTEIN BINDING; HET: ACT; 1.05A {Streptomyces clavuligerus} SCOP: d.98.1.0
</pre><pre>Probab=54.49  E-value=1.1e+02  Score=20.25  Aligned_cols=45  Identities=16%  Similarity=0.237  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEEEc-----CCEEEEECCCCCEEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKYVG-----ERAVVILNEEGKVVT   63 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y~G-----k~A~VVlN~~GkVVT   63 (69)
</pre><pre>                      .|-+.+.+...+-.|..........+.|..     ....|.++++|+++.
</pre><pre>T Consensus        14 ~G~t~~ev~~~~G~p~~~~~~~~~~~~~~~~~~~~~~~~v~f~~~~~v~~   63 (154)
</pre><pre>T 3GMX_A           14 FGMDRTLVWQLAGADQSCSDQVERIICYNNPDHYGPQGHFFFNAADKLIH   63 (154)
</pre><pre>T ss_dssp             TTCBHHHHHHHHTGGGTEEECSSCEEEESSSCTTSSEEEEEECTTSBEEE
</pre><pre>T ss_pred             CCCCHHHHHHHHCCCceecCCCeEEEEEeCCCCCCCeEEEEEcCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>PF02630.16 ; SCO1-SenC ; SCO1/SenC
</pre><pre>Probab=54.11  E-value=63  Score=18.94  Aligned_cols=39  Identities=5%  Similarity=0.012  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCceeecccCC--cEEEEcCCEEEEECCCCCE
</pre><pre>Q WP_134219109.1   23 SEAILDAVNNPTKVIPQVGG--TIKYVGERAVVILNEEGKV   61 (69)
</pre><pre>Q Consensus        23 ~~AildAv~nP~~v~~q~g~--~~~y~Gk~A~VVlN~~GkV   61 (69)
</pre><pre>                      ...+.+++.-+.......+.  .+.+.+....+++|++|+|
</pre><pre>T Consensus        96 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i  136 (136)
</pre><pre>T A0A0M6XUQ2_9RH   96 IARTADGFHIYYEKVEEAAAPGGYTMGHSSQLFLFGPEAGY  136 (136)
</pre><pre>T ss_pred             HHHHHHHcCCeeeecccccCCCCccccccceEEEECCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>d1xzoa1 c.47.1.10 (A:3-174) Thioredoxin-like protein Sco1 (YpmQ), soluble domain {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=52.79  E-value=1.1e+02  Score=19.68  Aligned_cols=46  Identities=20%  Similarity=0.308  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHhCCceeecccC-CcEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   20 GVSSEAILDAVNNPTKVIPQVG-GTIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        20 GVs~~AildAv~nP~~v~~q~g-~~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +-..+.+............+.. ..+...+--..+++|++|+|+-.|
</pre><pre>T Consensus       104 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~v~~~~  150 (172)
</pre><pre>T d1xzoa1         104 GYSQSEIEEFALKSFKAIVKKPEGEDQVIHQSSFYLVGPDGKVLKDY  150 (172)
</pre><pre>T ss_dssp             CSCHHHHHHHHHHHHCCCCCCCSSCCSCCSCCEEEEECTTSEEEEEE
</pre><pre>T ss_pred             CCCHHHHHHHHHHhccceeeCCCCCCCcccccEEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>d4je1a1 c.47.1.10 (A:1-167) automated matches {Burkholderia cenocepacia [TaxId: 216591]}
</pre><pre>Probab=52.32  E-value=46  Score=20.90  Aligned_cols=15  Identities=40%  Similarity=0.579  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+|+-.|
</pre><pre>T Consensus       134 ~~~lid~~g~i~~~~  148 (167)
</pre><pre>T d4je1a1         134 AVVVLDAQDKVIHAE  148 (167)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>4DM5_A Osmotically inducible lipoprotein OsmE; lipoprotein, UNKNOWN FUNCTION; HET: MSE; 1.5A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=52.19  E-value=1.2e+02  Score=19.90  Aligned_cols=46  Identities=22%  Similarity=0.374  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeeccc---CCcEEEEcCCE------EEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQV---GGTIKYVGERA------VVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~---g~~~~y~Gk~A------~VVlN~~GkVVTt   64 (69)
</pre><pre>                      .|.+.+++++.+-.|.....-.   +..+.|.-.+.      .|.+|.+|+|+..
</pre><pre>T Consensus        26 ~Gmt~~eV~~~lG~P~~~~~~~~~~~~~w~Y~~~~~~~~~~~~V~Fd~~G~V~~~   80 (103)
</pre><pre>T 4DM5_A           26 KGMSQQEVLRIGGTPSGTQKRLMKPGSCNSYILNKDGQQQPFYVSFDGSGKVDGS   80 (103)
</pre><pre>T ss_dssp             TTCBHHHHHHHHCSCSEEEECSSSSSEEEEEEEEETTEEEEEEEEECTTSBEEEE
</pre><pre>T ss_pred             CCCCHHHHHHHhCCCceEEeecCCCCeEEEEEECCCCCeeeEEEEECCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>4DM5_B Osmotically inducible lipoprotein OsmE; lipoprotein, UNKNOWN FUNCTION; HET: MSE; 1.5A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=52.19  E-value=1.2e+02  Score=19.90  Aligned_cols=46  Identities=22%  Similarity=0.374  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeeccc---CCcEEEEcCCE------EEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQV---GGTIKYVGERA------VVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~---g~~~~y~Gk~A------~VVlN~~GkVVTt   64 (69)
</pre><pre>                      .|.+.+++++.+-.|.....-.   +..+.|.-.+.      .|.+|.+|+|+..
</pre><pre>T Consensus        26 ~Gmt~~eV~~~lG~P~~~~~~~~~~~~~w~Y~~~~~~~~~~~~V~Fd~~G~V~~~   80 (103)
</pre><pre>T 4DM5_B           26 KGMSQQEVLRIGGTPSGTQKRLMKPGSCNSYILNKDGQQQPFYVSFDGSGKVDGS   80 (103)
</pre><pre>T ss_dssp             TTCBHHHHHHHHCSCSEEEECSSSSSEEEEEEEEETTEEEEEEEEECTTSBEEEE
</pre><pre>T ss_pred             CCCCHHHHHHHhCCCceEEeecCCCCeEEEEEECCCCCeeeEEEEECCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>3ME8_A Putative uncharacterized protein; electron transfer protein, ELECTRON TRANSPORT, Structural Genomics, PSI-2, Protein Structure Initiative, New York SGX Research Center; 1.5A {Aquifex aeolicus}
</pre><pre>Probab=52.14  E-value=63  Score=20.34  Aligned_cols=21  Identities=19%  Similarity=0.275  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   46 YVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        46 y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +....+++++|++|+++..|.
</pre><pre>T Consensus       124 ~~~~p~~~lid~~g~i~~~~~  144 (170)
</pre><pre>T 3ME8_A          124 FIHPNVVVVLSPELQIKDYIY  144 (170)
</pre><pre>T ss_dssp             EECCCEEEEECTTSBEEEEEE
</pre><pre>T ss_pred             eeeceEEEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>d3zl5a_ c.47.1.0 (A:) automated matches {Blood fluke (Schistosoma mansoni) [TaxId: 6183]}
</pre><pre>Probab=51.60  E-value=59  Score=19.94  Aligned_cols=19  Identities=11%  Similarity=0.289  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      .++++|++|+++..+....
</pre><pre>T Consensus       123 ~~~lid~~g~i~~~~~~~~  141 (162)
</pre><pre>T d3zl5a_         123 GLFIIDPNGILRQITINDK  141 (162)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEEEECT
</pre><pre>T ss_pred             EEEEECCCCeEEEEEeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>d2l5oa1 c.47.1.0 (A:1-142) automated matches {Neisseria meningitidis [TaxId: 491]}
</pre><pre>Probab=51.09  E-value=57  Score=19.20  Aligned_cols=16  Identities=19%  Similarity=0.644  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..|.
</pre><pre>T Consensus       106 ~~~~~d~~g~v~~~~~  121 (142)
</pre><pre>T d2l5oa1         106 TSVLIGKKGEILKTYV  121 (142)
</pre><pre>T ss_dssp             EEEEECSSSCCCEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>d3hdca1 c.47.1.0 (A:22-168) automated matches {Geobacter metallireducens [TaxId: 269799]}
</pre><pre>Probab=50.92  E-value=52  Score=19.84  Aligned_cols=16  Identities=13%  Similarity=0.268  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++-.|.
</pre><pre>T Consensus       115 ~~~~id~~g~v~~~~~  130 (147)
</pre><pre>T d3hdca1         115 DTFIVDRKGIIRQRVT  130 (147)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             eEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>d2ls5a1 c.47.1.0 (A:4-151) automated matches {Bacteroides thetaiotaomicron [TaxId: 226186]}
</pre><pre>Probab=50.78  E-value=57  Score=19.44  Aligned_cols=16  Identities=31%  Similarity=0.488  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+|+-.|.
</pre><pre>T Consensus       114 ~~~lid~~g~i~~~~~  129 (148)
</pre><pre>T d2ls5a1         114 RNVLIDREGKIVKLTR  129 (148)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>PF07467.13 ; BLIP ; Beta-lactamase inhibitor (BLIP)
</pre><pre>Probab=50.72  E-value=1.2e+02  Score=19.71  Aligned_cols=50  Identities=16%  Similarity=0.245  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccCCcEEE-----EcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVGGTIKY-----VGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g~~~~y-----~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      .|.+.+.+...+-.|..........+.|     .+....|.++++|+++.-....
</pre><pre>T Consensus        10 ~Gms~~eV~~~lG~P~~~~~~~~~~y~~~~~~~~~~~~~v~f~~~g~v~~~~~~~   64 (121)
</pre><pre>T B5GLC0_STRCL/3   10 FGMDRTLVWQLAGADQSCSDQVERIICYNNPDHYGPQGHFFFNAADKLIHKRQME   64 (121)
</pre><pre>T ss_pred             CCCCHHHHHHHHCCCCccCCCcceEEEEeCCCCCCCeEEEEEEcCCEEEEEEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>d4hdea1 c.47.1.0 (A:29-195) automated matches {Anthrax bacillus (Bacillus anthracis) [TaxId: 1392]}
</pre><pre>Probab=50.59  E-value=52  Score=20.65  Aligned_cols=15  Identities=20%  Similarity=0.658  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++..|
</pre><pre>T Consensus       133 ~~~lid~~G~i~~~~  147 (167)
</pre><pre>T d4hdea1         133 SFYLIDQNGKVMKKY  147 (167)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>1XZO_A Hypothetical protein ypmQ; thioredoxin-like fold, Structural Genomics, Montreal-Kingston Bacterial Structural Genomics Initiative, BSGI, METAL BINDING PROTEIN; HET: CA; 1.702A {Bacillus subtilis} SCOP: c.47.1.10
</pre><pre>Probab=49.80  E-value=62  Score=19.54  Aligned_cols=24  Identities=17%  Similarity=0.457  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+.+.+--+.+++|++|+++..|
</pre><pre>T Consensus       129 ~~~~~~~~p~~~lid~~g~i~~~~  152 (174)
</pre><pre>T 1XZO_A          129 GEDQVIHQSSFYLVGPDGKVLKDY  152 (174)
</pre><pre>T ss_dssp             SCCSCCSCCEEEEECTTSEEEEEE
</pre><pre>T ss_pred             CCccceeceEEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>6L7R_B Mozart1; gamma tubulin complex, microprotien, microtubule, TRANSLATION; 1.84813131157A {Chaetomium thermophilum (strain DSM 1495 / CBS 144.50 / IMI 039719)}
</pre><pre>Probab=49.80  E-value=49  Score=22.81  Aligned_cols=29  Identities=24%  Similarity=0.365  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             CcccHHHHHHHHhcCCCCCCHHHHHHHHh
</pre><pre>Q WP_134219109.1    3 TGFSRHAINQAISRDGVGVSSEAILDAVN   31 (69)
</pre><pre>Q Consensus         3 tG~TkHGl~qai~RdG~GVs~~AildAv~   31 (69)
</pre><pre>                      ||.+++-|.-.+.-=..||+|+++.+.|+
</pre><pre>T Consensus        32 TGLd~etL~icv~L~e~GVnPeaLA~vIk   60 (86)
</pre><pre>T 6L7R_B           32 CHLDRRTLSICISMIENGVNPEALANVIK   60 (86)
</pre><pre>T ss_dssp             CCCCHHHHHHHHHHHHTTCCHHHHHHHHH
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHcCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>d1mp9a1 d.129.1.1 (A:5-96) TATA-box binding protein (TBP), C-terminal domain {Sulfolobus acidocaldarius [TaxId: 2285]}
</pre><pre>Probab=49.60  E-value=87  Score=20.22  Aligned_cols=52  Identities=19%  Similarity=0.145  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             C-CCCCCHHHHHHHHhCCceeecc-cCCcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   17 D-GVGVSSEAILDAVNNPTKVIPQ-VGGTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        17 d-G~GVs~~AildAv~nP~~v~~q-~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      + +..+....+.+.+.+-.--++. .+-.+++..+++++.+-+.||++-+=+++
</pre><pre>T Consensus        19 ~l~~~idL~~la~~~~~~~Y~p~~fp~~~~r~~~~~~~~~if~sGkiiitGa~s   72 (92)
</pre><pre>T d1mp9a1          19 TLDQTLDLYAMERSVPNVEYDPDQFPGLIFRLESPKITSLIFKSGKMVVTGAKS   72 (92)
</pre><pre>T ss_dssp             ECCSCCCHHHHHHHSTTCBCCTTTCSSEEEEETTTTEEEEECTTSEEEEECCSS
</pre><pre>T ss_pred             EcCCcccHHHHHhhCCCcccCcccCceEEEEccCCcEEEEEEcCCceeecCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>d1n8ja_ c.47.1.10 (A:) Alkyl hydroperoxide reductase AhpC {Salmonella typhimurium [TaxId: 90371]}
</pre><pre>Probab=49.51  E-value=53  Score=21.34  Aligned_cols=16  Identities=13%  Similarity=0.266  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++|+|++|+++..|.
</pre><pre>T Consensus       120 ~~~lid~~g~i~~~~~  135 (186)
</pre><pre>T d1n8ja_         120 ATFVVDPQGIIQAIEV  135 (186)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>d4k9za_ c.47.1.0 (A:) automated matches {Bacteroides vulgatus [TaxId: 435590]}
</pre><pre>Probab=49.27  E-value=44  Score=19.79  Aligned_cols=16  Identities=13%  Similarity=0.501  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..|.
</pre><pre>T Consensus       110 ~~~~vd~~g~v~~~~~  125 (141)
</pre><pre>T d4k9za_         110 RFILIDKDFNIVNAYA  125 (141)
</pre><pre>T ss_dssp             EEEEECTTSBEEETTC
</pre><pre>T ss_pred             EEEEECCCCCEEeCcC
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>d3kh7a1 c.47.1.0 (A:26-175) automated matches {Pseudomonas aeruginosa [TaxId: 287]}
</pre><pre>Probab=48.60  E-value=63  Score=19.45  Aligned_cols=16  Identities=6%  Similarity=0.279  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++..|.
</pre><pre>T Consensus       114 ~~~lid~~g~v~~~~~  129 (150)
</pre><pre>T d3kh7a1         114 ETYLIDKQGIIRHKIV  129 (150)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>2B7K_A SCO1 protein; Metallochaperone, Cytochrome c Oxidase, Sco, Sco1, METAL BINDING PROTEIN; 1.8A {Saccharomyces cerevisiae} SCOP: c.47.1.10
</pre><pre>Probab=48.45  E-value=1.2e+02  Score=18.96  Aligned_cols=27  Identities=15%  Similarity=0.175  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             ccCCcEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   39 QVGGTIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        39 q~g~~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .....+.+.+--..+++|++|+++-.|
</pre><pre>T Consensus       135 ~~~~~~~~~~~p~~~lid~~g~v~~~~  161 (200)
</pre><pre>T 2B7K_A          135 KPGQDYLVDHSIFFYLMDPEGQFVDAL  161 (200)
</pre><pre>T ss_dssp             ------CTTTCCCEEEECTTSCEEEEE
</pre><pre>T ss_pred             CCCCceeeecccEEEEECCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>3EWL_A uncharacterized conserved protein BF1870; alpha-beta fold, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, unknown function; HET: MSE; 2.0A {Bacteroides fragilis} SCOP: c.47.1.0, l.1.1.1
</pre><pre>Probab=47.89  E-value=53  Score=19.39  Aligned_cols=16  Identities=13%  Similarity=0.283  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++++|++|+++..+.
</pre><pre>T Consensus       112 ~~~~id~~g~v~~~~~  127 (142)
</pre><pre>T 3EWL_A          112 TIYLLDGRKRVILKDT  127 (142)
</pre><pre>T ss_dssp             EEEEECTTCBEEECSC
</pre><pre>T ss_pred             EEEEECCCCcEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>d3kcma1 c.47.1.0 (A:28-165) automated matches {Geobacter metallireducens [TaxId: 269799]}
</pre><pre>Probab=47.54  E-value=66  Score=18.91  Aligned_cols=15  Identities=7%  Similarity=0.366  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ++++++++|+++..|
</pre><pre>T Consensus       106 ~~~~i~~~g~v~~~~  120 (138)
</pre><pre>T d3kcma1         106 ETFVIDRHGVILKKV  120 (138)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>d3or5a_ c.47.1.0 (A:) automated matches {Chlorobaculum tepidum [TaxId: 1097]}
</pre><pre>Probab=47.54  E-value=70  Score=18.98  Aligned_cols=16  Identities=13%  Similarity=0.314  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..+.
</pre><pre>T Consensus       113 ~~~~id~~g~i~~~~~  128 (145)
</pre><pre>T d3or5a_         113 TSFVIDASGNVSGVIV  128 (145)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEC
</pre><pre>T ss_pred             EEEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>PF07504.15 ; FTP ; Fungalysin/Thermolysin Propeptide Motif
</pre><pre>Probab=47.53  E-value=81  Score=16.60  Aligned_cols=26  Identities=15%  Similarity=0.101  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEeec
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWAR   67 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~~   67 (69)
</pre><pre>                      ..+...|....+.++++|+++...+.
</pre><pre>T Consensus        25 ~gipV~~~~~~v~~~~~g~v~~~~~~   50 (51)
</pre><pre>T A5I1S5_CLOBH/8   25 EDIPVYYGRIVFTTKKDSSLDSINGR   50 (51)
</pre><pre>T ss_pred             CCEEEEcCEEEEEECCCCCEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>6HN3_A Phospholipid hydroperoxide glutathione peroxidase; ANTI-OXIDATVE DEFENSE SYSTEM, OXIDOREDUCTASE; 1.01A {Homo sapiens} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=47.09  E-value=81  Score=19.50  Aligned_cols=28  Identities=11%  Similarity=0.049  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+...+.-+.+++|++|+++..|....
</pre><pre>T Consensus       130 ~~~~~~~~p~~~lid~~g~i~~~~~~~~  157 (176)
</pre><pre>T 6HN3_A          130 LGNAIKWNFTKFLIDKNGCVVKRYGPME  157 (176)
</pre><pre>T ss_dssp             TBSSCCSSCCEEEECTTSCEEEEECTTS
</pre><pre>T ss_pred             ccccccccceEEEECCCCcEEEEeCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>d3tuea_ c.47.1.10 (A:) automated matches {Leishmania major [TaxId: 5664]}
</pre><pre>Probab=46.73  E-value=67  Score=19.99  Aligned_cols=16  Identities=13%  Similarity=0.258  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++++|++|+++..|.
</pre><pre>T Consensus       126 ~~~lid~~g~i~~~~~  141 (166)
</pre><pre>T d3tuea_         126 GLFIIDPHGMLRQITV  141 (166)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             eEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>d2r37a1 c.47.1.0 (A:38-223) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=46.34  E-value=84  Score=21.50  Aligned_cols=39  Identities=5%  Similarity=0.119  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   28 DAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        28 dAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++..|.....+........+.-+++++|++|+++..|.
</pre><pre>T Consensus       124 ~~~~~~~~~~~~~~~~~~i~~~p~~~lid~~g~i~~~~~  162 (186)
</pre><pre>T d2r37a1         124 ELLGTSDRLFWEPMKVHDIRWNFEKFLVGPDGIPIMRWH  162 (186)
</pre><pre>T ss_dssp             SCCCCGGGBCCSSCCTTCCCSTTCEEEECTTSCEEEEEC
</pre><pre>T ss_pred             hhcCCCccccccccCccccccccEEEEECCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>3KIJ_A Probable glutathione peroxidase 8; Human PDI-peroxidase, glutathione peroxidase, Membrane, Oxidoreductase, Peroxidase, Transmembrane; 1.8A {Homo sapiens} SCOP: c.47.1.0
</pre><pre>Probab=46.24  E-value=64  Score=20.36  Aligned_cols=15  Identities=40%  Similarity=0.713  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+++..|
</pre><pre>T Consensus       138 ~~~lid~~g~i~~~~  152 (180)
</pre><pre>T 3KIJ_A          138 WKYLVNPEGQVVKFW  152 (180)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>5C04_A Putative peroxiredoxin MT2298; OXIDOREDUCTASE, 1-cys peroxiredoxin, peroxiredoxin, thioredoxin fold, Mycobacterium tuberculosis, active site mutant; 1.45A {Mycobacterium tuberculosis}
</pre><pre>Probab=46.15  E-value=76  Score=18.94  Aligned_cols=19  Identities=5%  Similarity=0.174  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++|+++-.|....
</pre><pre>T Consensus       118 ~~~lid~~g~v~~~~~~~~  136 (153)
</pre><pre>T 5C04_A          118 GTFVVDRSGIIRFAEMKQP  136 (153)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEEECT
</pre><pre>T ss_pred             cEEEECCCCeEEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>d2he3a1 c.47.1.0 (A:4-187) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=46.08  E-value=64  Score=21.57  Aligned_cols=25  Identities=8%  Similarity=0.117  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ....+.+..+++++|++|+++-.|.
</pre><pre>T Consensus       143 ~~~~~~~~~~~~lid~~g~v~~~~~  167 (184)
</pre><pre>T d2he3a1         143 RRSDVAWNFEKFLIGPEGEPFRRYS  167 (184)
</pre><pre>T ss_dssp             CTTCCCSTTCEEEECTTSCEEEEEC
</pre><pre>T ss_pred             CccccccceeEEEECCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>2FY6_A Peptide methionine sulfoxide reductase msrA/msrB; PILB, Cytochrome maturation protein, Thioredoxin, Methionine sulfoxide reductase, OXIDOREDUCTASE; 1.9A {Neisseria meningitidis} SCOP: c.47.1.10
</pre><pre>Probab=45.85  E-value=76  Score=18.53  Aligned_cols=15  Identities=13%  Similarity=0.315  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       109 ~~~lid~~g~i~~~~  123 (143)
</pre><pre>T 2FY6_A          109 SWALIGKDGDVQRIV  123 (143)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>5IMF_A Bacterioferritin comigratory protein; PrxQ, BCP, Peroxidase, redox, OXIDOREDUCTASE; 1.04A {Xanthomonas campestris pv. campestris (strain ATCC 33913 / DSM 3586 / NCPPB 528 / LMG 568 / P 25)}
</pre><pre>Probab=45.78  E-value=72  Score=18.96  Aligned_cols=15  Identities=40%  Similarity=0.926  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       124 ~~~lid~~g~i~~~~  138 (160)
</pre><pre>T 5IMF_A          124 STFLLSPEGQVVQAW  138 (160)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>PF08534.12 ; Redoxin ; Redoxin
</pre><pre>Probab=45.70  E-value=93  Score=17.87  Aligned_cols=19  Identities=21%  Similarity=0.305  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++|+++..+....
</pre><pre>T Consensus       108 ~~~~id~~g~i~~~~~~~~  126 (135)
</pre><pre>T Q6AIL8_DESPS/2  108 VGFLVNRLGNVVKRYTGYV  126 (135)
</pre><pre>T ss_pred             EEEEECCCCcEEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>3EYT_B uncharacterized protein SPOA0173; Thioredoxin-like superfamily protein SPOA0173, SILICIBACTER POMEROYI DSS, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for; HET: ACY, MSE, EDO; 1.95A {Silicibacter pomeroyi}
</pre><pre>Probab=45.53  E-value=73  Score=19.06  Aligned_cols=16  Identities=6%  Similarity=0.372  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..+.
</pre><pre>T Consensus       120 ~~~~id~~g~i~~~~~  135 (158)
</pre><pre>T 3EYT_B          120 SLLLIDKAGDLRAHHF  135 (158)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEEcCCCcEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>d2b7ka_ c.47.1.10 (A:) Thioredoxin-like protein Sco1 (YpmQ), soluble domain {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=45.29  E-value=1.3e+02  Score=19.39  Aligned_cols=27  Identities=19%  Similarity=0.208  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ..+...+--+.+++|++|+++-.|...
</pre><pre>T Consensus       123 ~~~~~~~~p~~~lid~~g~i~~~~~g~  149 (169)
</pre><pre>T d2b7ka_         123 QDYLVDHSIFFYLMDPEGQFVDALGRN  149 (169)
</pre><pre>T ss_dssp             ---CTTTCCCEEEECTTSCEEEEECTT
</pre><pre>T ss_pred             CceeeeccceEEEECCCCCEEEEeccC
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>6B9X_B Ragulator complex protein LAMTOR2; Ragulator, Lamtor, SIGNALING PROTEIN; 1.42A {Homo sapiens} SCOP: d.110.7.1, l.1.1.1
</pre><pre>Probab=45.14  E-value=91  Score=19.64  Aligned_cols=22  Identities=23%  Similarity=0.436  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      |-+++++++++|.++..++...
</pre><pre>T Consensus        19 gv~~~~i~~~dG~~i~~~~~~~   40 (126)
</pre><pre>T 6B9X_B           19 GVQSTLLLNNEGSLLAYSGYGD   40 (126)
</pre><pre>T ss_dssp             TEEEEEEECTTCCEEEEEECCS
</pre><pre>T ss_pred             CccEEEEEcCCCCEEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>d2ljaa1 c.47.1.0 (A:4-144) automated matches {Bacteroides vulgatus [TaxId: 435590]}
</pre><pre>Probab=44.91  E-value=60  Score=19.06  Aligned_cols=18  Identities=17%  Similarity=0.530  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      +.+++|++|+++..+...
</pre><pre>T Consensus       108 ~~~~id~~g~i~~~~~~~  125 (141)
</pre><pre>T d2ljaa1         108 RFILLDRDGKIISANMTR  125 (141)
</pre><pre>T ss_dssp             CEEEECTTSCEEESSCCC
</pre><pre>T ss_pred             cEEEECCCCCEEeccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>2H30_A Peptide methionine sulfoxide reductase msrA/msrB; reduced, methionine sulfoxide, thiol-disulfide exchange, OXIDOREDUCTASE; HET: MSE; 1.6A {Neisseria gonorrhoeae} SCOP: c.47.1.10
</pre><pre>Probab=44.83  E-value=77  Score=19.18  Aligned_cols=15  Identities=13%  Similarity=0.315  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.+
</pre><pre>T Consensus       124 ~~~lid~~g~i~~~~  138 (164)
</pre><pre>T 2H30_A          124 SWALIGKDGDVQRIV  138 (164)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>d1q98a_ c.47.1.10 (A:) Thiol peroxidase Tpx {Haemophilus influenzae [TaxId: 727]}
</pre><pre>Probab=44.69  E-value=73  Score=20.09  Aligned_cols=15  Identities=33%  Similarity=0.632  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+|+-.|
</pre><pre>T Consensus       131 ~~~lid~~G~i~~~~  145 (164)
</pre><pre>T d1q98a_         131 AVIVLDEQNNVLHSQ  145 (164)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>d2p5qa_ c.47.1.0 (A:) automated matches {Populus trichocarpa [TaxId: 3695]}
</pre><pre>Probab=44.57  E-value=73  Score=19.45  Aligned_cols=15  Identities=33%  Similarity=0.709  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.+
</pre><pre>T Consensus       127 ~~~lid~~g~i~~~~  141 (161)
</pre><pre>T d2p5qa_         127 AKFLVNKDGQVVDRY  141 (161)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>2LS5_A Uncharacterized protein; STRUCTURAL GENOMICS, UNKNOWN FUNCTION, THIOREDOXIN-LIKE, New York Structural Genomics Research Consortium, NYSGRC, PSI-Biology; NMR {Bacteroides thetaiotaomicron} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=44.43  E-value=75  Score=19.19  Aligned_cols=16  Identities=31%  Similarity=0.488  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++-.+.
</pre><pre>T Consensus       117 ~~~lid~~g~i~~~~~  132 (159)
</pre><pre>T 2LS5_A          117 RNVLIDREGKIVKLTR  132 (159)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>d4evma_ c.47.1.0 (A:) automated matches {Pneumococcus (Streptococcus pneumoniae) [TaxId: 1313]}
</pre><pre>Probab=44.27  E-value=61  Score=18.78  Aligned_cols=15  Identities=33%  Similarity=0.596  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++..+
</pre><pre>T Consensus       107 ~~~~~~~~g~~~~~~  121 (138)
</pre><pre>T d4evma_         107 TQAFIDKEGKLVKTH  121 (138)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEE
</pre><pre>T ss_pred             cEEEECCCCcEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>d2p31a_ c.47.1.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=44.21  E-value=76  Score=19.33  Aligned_cols=15  Identities=33%  Similarity=0.676  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ++++++++|+++-.+
</pre><pre>T Consensus       123 ~~~lid~~g~v~~~~  137 (155)
</pre><pre>T d2p31a_         123 WKYLVAPDGKVVGAW  137 (155)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEE
</pre><pre>T ss_pred             eEEEECCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>d3cmia_ c.47.1.0 (A:) automated matches {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=44.18  E-value=78  Score=19.34  Aligned_cols=16  Identities=19%  Similarity=0.513  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++-.|.
</pre><pre>T Consensus       127 ~~~~id~~g~i~~~~~  142 (161)
</pre><pre>T d3cmia_         127 EKFLVDKKGKVYERYS  142 (161)
</pre><pre>T ss_dssp             CEEEECSSSCEEEEEC
</pre><pre>T ss_pred             EEEEECCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>2V1M_A GLUTATHIONE PEROXIDASE; SELENIUM, SELENOCYSTEINE, OXIDOREDUCTASE, LIPID PEROXIDASE, SCHISTOSOMA DETOXIFICATION PATHWAY; HET: SO4; 1.0A {SCHISTOSOMA MANSONI} SCOP: c.47.1.0
</pre><pre>Probab=43.96  E-value=79  Score=18.81  Aligned_cols=16  Identities=13%  Similarity=0.484  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.|.
</pre><pre>T Consensus       135 ~~~lid~~g~v~~~~~  150 (169)
</pre><pre>T 2V1M_A          135 SKFLVDRQGQPVKRYS  150 (169)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             EEEEECCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>d3hcza1 c.47.1.0 (A:330-474) automated matches {Cytophaga hutchinsonii [TaxId: 269798]}
</pre><pre>Probab=43.94  E-value=75  Score=18.86  Aligned_cols=15  Identities=7%  Similarity=0.328  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++..+
</pre><pre>T Consensus       111 ~~~l~d~~g~~~~~~  125 (145)
</pre><pre>T d3hcza1         111 VLYVLDKNKVIIAKR  125 (145)
</pre><pre>T ss_dssp             EEEEECTTCBEEEES
</pre><pre>T ss_pred             EEEEECCCCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>4PQ1_A Putative electron transport related protein; Thioredoxin-like region, reductant, oxidant, Oxidoreductase; 2.097A {Corynebacterium diphtheriae}
</pre><pre>Probab=43.90  E-value=81  Score=18.94  Aligned_cols=15  Identities=7%  Similarity=0.439  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       127 ~~~lid~~g~v~~~~  141 (158)
</pre><pre>T 4PQ1_A          127 TTIVLDKQHRPAAVF  141 (158)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>d5um7a_ c.47.1.0 (A:) automated matches {Streptococcus gordonii [TaxId: 467705]}
</pre><pre>Probab=43.86  E-value=80  Score=18.59  Aligned_cols=15  Identities=13%  Similarity=0.324  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       111 ~~~~i~~~g~v~~~~  125 (140)
</pre><pre>T d5um7a_         111 TIYLVDKNQKVKKVM  125 (140)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>4FO5_A thioredoxin-like protein; AhpC/TSA family protein, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY, OXIDOREDUCTASE; HET: MSE; 2.02A {Parabacteroides distasonis}
</pre><pre>Probab=43.85  E-value=66  Score=19.10  Aligned_cols=16  Identities=19%  Similarity=0.459  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..|.
</pre><pre>T Consensus       116 ~~~~id~~g~i~~~~~  131 (143)
</pre><pre>T 4FO5_A          116 KNFLINDEGVIIAANV  131 (143)
</pre><pre>T ss_dssp             CEEEECTTSBEEEESC
</pre><pre>T ss_pred             eeEEECCCCcEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>4NMU_C Thiol-disulfide oxidoreductase ResA; Structural Genomics, NIAID, National Institute of Allergy and Infectious Diseases, Center for Structural Genomics of Infectious; HET: GOL, EDO, ACY, MSE; 1.35A {Bacillus anthracis} SCOP: c.47.1.0, l.1.1.1
</pre><pre>Probab=43.52  E-value=85  Score=18.43  Aligned_cols=15  Identities=27%  Similarity=0.581  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.|
</pre><pre>T Consensus       115 ~~~l~d~~g~i~~~~  129 (147)
</pre><pre>T 4NMU_C          115 TSFLIDKDGKVVEQI  129 (147)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>4NMU_D Thiol-disulfide oxidoreductase ResA; Structural Genomics, NIAID, National Institute of Allergy and Infectious Diseases, Center for Structural Genomics of Infectious; HET: MSE, ACY, GOL, EDO; 1.35A {Bacillus anthracis} SCOP: c.47.1.0
</pre><pre>Probab=43.52  E-value=85  Score=18.43  Aligned_cols=15  Identities=27%  Similarity=0.581  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.|
</pre><pre>T Consensus       115 ~~~l~d~~g~i~~~~  129 (147)
</pre><pre>T 4NMU_D          115 TSFLIDKDGKVVEQI  129 (147)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>d4g2ea1 c.47.1.0 (A:1-155) automated matches {Sulfolobus tokodaii [TaxId: 273063]}
</pre><pre>Probab=43.26  E-value=81  Score=19.20  Aligned_cols=15  Identities=47%  Similarity=0.942  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++..+
</pre><pre>T Consensus       119 ~~~lid~~g~i~~~~  133 (155)
</pre><pre>T d4g2ea1         119 AVFVIDKEGKVRYKW  133 (155)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>3EUR_A uncharacterized protein; PSI2, MCSG, conserved protein, Structural Genomics, Protein Structure Initiative, Midwest Center for Structural Genomics, unknown function; HET: MSE; 1.3A {Bacteroides fragilis} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=43.13  E-value=65  Score=19.11  Aligned_cols=15  Identities=13%  Similarity=0.229  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++..|
</pre><pre>T Consensus       116 ~~~lid~~g~v~~~~  130 (142)
</pre><pre>T 3EUR_A          116 TLYLLDKNKTVLLKD  130 (142)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>TIGR02540 gpx7; putative glutathione peroxidase Gpx7. This model represents one of several families of known and probable glutathione peroxidases.
</pre><pre>Probab=43.09  E-value=92  Score=19.01  Aligned_cols=18  Identities=33%  Similarity=0.584  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      +++++|++|+++..+...
</pre><pre>T Consensus       122 ~~~lid~~g~i~~~~~g~  139 (153)
</pre><pre>T TIGR02540       122 WKYLVNPEGQVVKFWRPE  139 (153)
</pre><pre>T ss_pred             eEEEECCCCCEEEEeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>4HDE_A SCO1/SenC family lipoprotein; structural genomics, The Center for Structural Genomics of Infectious Diseases, CSGID, NIAID, National Institute of Allergy; HET: MSE; 1.317A {Bacillus anthracis} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=42.76  E-value=1.4e+02  Score=17.85  Aligned_cols=45  Identities=13%  Similarity=0.219  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   22 SSEAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        22 s~~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ....+.................+...+--..+++|++|+++-.|.
</pre><pre>T Consensus       107 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~p~~~lid~~g~i~~~~~  151 (170)
</pre><pre>T 4HDE_A          107 SLEDITKFSKDNFQSLVDKPENGQVIHGTSFYLIDQNGKVMKKYS  151 (170)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHCCCCBCCTTSCCBCCCEEEEECTTSCEEEEEE
</pre><pre>T ss_pred             CHHHHHHHHHhhcccccccCCCCceEeceEEEEECCCCcEEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>4GQC_B Thiol peroxidase; CXXXXC Motif, fully folded, locally unfolded, peroxide, DTT, Structural Genomics, RIKEN, peroxiredoxin, reduces peroxides, disulfide, OXIDOREDUCTASE; HET: SO4, GOL, DTD; 2.0A {Aeropyrum pernix} SCOP: l.1.1.1, c.47.1.10
</pre><pre>Probab=42.69  E-value=80  Score=19.19  Aligned_cols=15  Identities=40%  Similarity=0.829  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+++-.|
</pre><pre>T Consensus       123 ~~~lid~~g~i~~~~  137 (164)
</pre><pre>T 4GQC_B          123 AVFIVKPDGTVAYKW  137 (164)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>d1st9a_ c.47.1.10 (A:) Thiol-disulfide oxidoreductase ResA {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=42.30  E-value=96  Score=18.19  Aligned_cols=16  Identities=38%  Similarity=0.582  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++-.|.
</pre><pre>T Consensus       105 ~~~~i~~~g~i~~~~~  120 (137)
</pre><pre>T d1st9a_         105 TTFLINPEGKVVKVVT  120 (137)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>2YZH_D Probable thiol peroxidase; Redox protein, Antioxidant, Oxidoreductase, Peroxidase, Structural Genomics, NPPSFA, National Project on Protein Structural and Functional Analyses; HET: SO4; 1.85A {Aquifex aeolicus} SCOP: c.47.1.0, l.1.1.1
</pre><pre>Probab=42.07  E-value=1.1e+02  Score=18.89  Aligned_cols=19  Identities=37%  Similarity=0.434  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++|+++..|....
</pre><pre>T Consensus       135 ~~~lid~~g~i~~~~~~~~  153 (171)
</pre><pre>T 2YZH_D          135 AVFIIDKEGKVAYVQLVPE  153 (171)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEECSB
</pre><pre>T ss_pred             EEEEECCCCeEEEEEeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>d3p7xa1 c.47.1.0 (A:1-164) automated matches {Staphylococcus aureus [TaxId: 93061]}
</pre><pre>Probab=42.02  E-value=1e+02  Score=19.24  Aligned_cols=19  Identities=32%  Similarity=0.365  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      .++++|++|+|+-.|....
</pre><pre>T Consensus       129 ~~~lid~~g~i~~~~~~~~  147 (164)
</pre><pre>T d3p7xa1         129 AVFVLDADNKVVYKEIVSE  147 (164)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEEECSB
</pre><pre>T ss_pred             EEEEECCCCeEEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>1LU4_A SOLUBLE SECRETED ANTIGEN MPT53; THIOREDOXIN-LIKE FOLD, Structural Genomics, PSI, Protein Structure Initiative, TB Structural Genomics Consortium, TBSGC, OXIDOREDUCTASE; 1.12A {Mycobacterium tuberculosis} SCOP: c.47.1.10
</pre><pre>Probab=41.97  E-value=89  Score=18.30  Aligned_cols=15  Identities=20%  Similarity=0.220  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       102 ~~~~id~~g~v~~~~  116 (136)
</pre><pre>T 1LU4_A          102 AFVFYRADGTSTFVN  116 (136)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEC
</pre><pre>T ss_pred             eEEEEcCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>d4gqca1 c.47.1.10 (A:5-163) Bacterioferritin comigratory protein {Aeropyrum pernix [TaxId: 56636]}
</pre><pre>Probab=41.93  E-value=80  Score=19.48  Aligned_cols=15  Identities=40%  Similarity=0.829  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.|
</pre><pre>T Consensus       119 ~~~lid~~g~v~~~~  133 (159)
</pre><pre>T d4gqca1         119 AVFIVKPDGTVAYKW  133 (159)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             eEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>4EVM_A Thioredoxin family protein; Structural Genomics, NIAID, National Institute of Allergy and Infectious Diseases, Center for Structural Genomics of Infectious; 1.506A {Streptococcus pneumoniae} SCOP: c.47.1.0
</pre><pre>Probab=41.79  E-value=59  Score=18.57  Aligned_cols=16  Identities=31%  Similarity=0.565  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.+.
</pre><pre>T Consensus       107 ~~~lid~~g~i~~~~~  122 (138)
</pre><pre>T 4EVM_A          107 TQAFIDKEGKLVKTHP  122 (138)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEEE
</pre><pre>T ss_pred             cEEEECCCCcEeecCC
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>2B5X_A YkuV protein; Thioredoxin-like, OXIDOREDUCTASE; NMR {Bacillus subtilis} SCOP: c.47.1.10
</pre><pre>Probab=41.51  E-value=99  Score=18.42  Aligned_cols=16  Identities=19%  Similarity=0.287  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.+.
</pre><pre>T Consensus       113 ~~~~id~~g~i~~~~~  128 (148)
</pre><pre>T 2B5X_A          113 AYYVFDKTGQLRHFQA  128 (148)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>2M72_A Uncharacterized thioredoxin-like protein; STRUCTURAL GENOMICS, UNKNOWN FUNCTION, THIOREDOXIN-LIKE, New York Structural Genomics Research Consortium, NYSGRC, PSI-Biology; NMR {Porphyromonas gingivalis} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=41.30  E-value=90  Score=18.95  Aligned_cols=16  Identities=38%  Similarity=0.599  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.+.
</pre><pre>T Consensus       114 ~~~lid~~g~v~~~~~  129 (156)
</pre><pre>T 2M72_A          114 AVFIVDGNGKIVYNHT  129 (156)
</pre><pre>T ss_dssp             EEEEECTTSSEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEeee
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>d2kuca1 c.47.1.0 (A:1-119) automated matches {Bacteroides thetaiotaomicron [TaxId: 818]}
</pre><pre>Probab=41.26  E-value=1.2e+02  Score=17.50  Aligned_cols=18  Identities=22%  Similarity=0.423  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ..+++|++|+++..+...
</pre><pre>T Consensus        86 ~~~~~~~~g~~~~~~~g~  103 (119)
</pre><pre>T d2kuca1          86 TLLFINSSGEVVYRLVGA  103 (119)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEEESC
</pre><pre>T ss_pred             EEEEECCCCCEEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>d2bmxa1 c.47.1.10 (A:2-170) Alkyl hydroperoxide reductase AhpC {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=41.10  E-value=89  Score=19.82  Aligned_cols=16  Identities=13%  Similarity=0.183  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ++++++++|+++..|.
</pre><pre>T Consensus       133 ~~~lid~~g~v~~~~~  148 (169)
</pre><pre>T d2bmxa1         133 VTFIVDPNNEIQFVSA  148 (169)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>4JE1_A Probable thiol peroxidase; SSGCID, THIOL PEROXIDASE, Structural Genomics, NIAID, National Institute of Allergy and Infectious Diseases, Seattle Structural Genomics; 1.4A {Burkholderia cenocepacia} SCOP: l.1.1.1, c.47.1.10
</pre><pre>Probab=41.07  E-value=85  Score=20.29  Aligned_cols=15  Identities=40%  Similarity=0.579  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+|+..|
</pre><pre>T Consensus       142 ~~~lid~~g~v~~~~  156 (175)
</pre><pre>T 4JE1_A          142 AVVVLDAQDKVIHAE  156 (175)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>d3kija_ c.47.1.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=41.07  E-value=89  Score=19.47  Aligned_cols=16  Identities=38%  Similarity=0.657  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++++|++|+++..|.
</pre><pre>T Consensus       129 ~~~lid~~g~v~~~~~  144 (171)
</pre><pre>T d3kija_         129 WKYLVNPEGQVVKFWR  144 (171)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             eEEEECCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>2F9S_A Thiol-disulfide oxidoreductase resA; Thioredoxin-like protein, Oxidoreductase; HET: MSE; 1.401A {Bacillus subtilis} SCOP: c.47.1.10
</pre><pre>Probab=41.05  E-value=91  Score=18.96  Aligned_cols=15  Identities=40%  Similarity=0.607  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++..|
</pre><pre>T Consensus       106 ~~~lid~~g~v~~~~  120 (151)
</pre><pre>T 2F9S_A          106 TTFLINPEGKVVKVV  120 (151)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>d3l7ha_ d.110.7.1 (A:) automated matches {Fruit fly (Drosophila melanogaster) [TaxId: 7227]}
</pre><pre>Probab=40.92  E-value=76  Score=19.74  Aligned_cols=20  Identities=25%  Similarity=0.371  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeec
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWAR   67 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~   67 (69)
</pre><pre>                      |=.+++|+|++|.++.+...
</pre><pre>T Consensus         9 gV~g~iv~d~~G~~i~s~~~   28 (86)
</pre><pre>T d3l7ha_           9 GVVGTIVVNNEGIPVKSTLD   28 (86)
</pre><pre>T ss_dssp             TEEEEEEEETTSCEEEESSC
</pre><pre>T ss_pred             CEEEEEEEcCCCCeeEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>TIGR03673 uL14_arch; 50S ribosomal protein uL14, archaeal form. Part of the 50S ribosomal subunit. Forms a cluster with proteins L3 and L24e, part of which may contact the 16S rRNA in 2 intersubunit bridges.
</pre><pre>Probab=40.89  E-value=91  Score=22.32  Aligned_cols=24  Identities=25%  Similarity=0.331  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             ccCCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   39 QVGGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        39 q~g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      ...+.+-+-.++|+|+||++|..+
</pre><pre>T Consensus        81 r~dG~~i~F~~Na~VLln~~g~p~  104 (131)
</pre><pre>T TIGR03673        81 RPDGTRVKFEDNAAVIVTPDGEPK  104 (131)
</pre><pre>T ss_pred             CCCCCeEEeCCCEEEEECCCCCcC
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>d2yzha1 c.47.1.0 (A:1-168) automated matches {Aquifex aeolicus [TaxId: 224324]}
</pre><pre>Probab=40.82  E-value=90  Score=19.64  Aligned_cols=15  Identities=47%  Similarity=0.632  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.|
</pre><pre>T Consensus       133 ~~~lid~~g~i~~~~  147 (168)
</pre><pre>T d2yzha1         133 AVFIIDKEGKVAYVQ  147 (168)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>d2xhfa1 c.47.1.0 (A:37-190) automated matches {Alvinella pompejana [TaxId: 6376]}
</pre><pre>Probab=40.58  E-value=77  Score=20.01  Aligned_cols=15  Identities=7%  Similarity=0.301  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++| +|+|+..|.
</pre><pre>T Consensus       124 ~~~lvd-~g~i~~~~~  138 (154)
</pre><pre>T d2xhfa1         124 YAMLID-DNKIRSVST  138 (154)
</pre><pre>T ss_dssp             EEEEEE-TTEEEEEEE
</pre><pre>T ss_pred             EEEEEe-CCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>2P5Q_A Glutathione peroxidase 5; Thioredoxin fold, OXIDOREDUCTASE; HET: ACT, CD; 2.0A {Populus trichocarpa x Populus deltoides} SCOP: c.47.1.0
</pre><pre>Probab=40.49  E-value=95  Score=18.58  Aligned_cols=15  Identities=33%  Similarity=0.709  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++..|
</pre><pre>T Consensus       136 ~~~~id~~g~i~~~~  150 (170)
</pre><pre>T 2P5Q_A          136 AKFLVNKDGQVVDRY  150 (170)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>2P5Q_C Glutathione peroxidase 5; Thioredoxin fold, OXIDOREDUCTASE; HET: CD, ACT; 2.0A {Populus trichocarpa x Populus deltoides} SCOP: c.47.1.0
</pre><pre>Probab=40.49  E-value=95  Score=18.58  Aligned_cols=15  Identities=33%  Similarity=0.709  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++..|
</pre><pre>T Consensus       136 ~~~~id~~g~i~~~~  150 (170)
</pre><pre>T 2P5Q_C          136 AKFLVNKDGQVVDRY  150 (170)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>d4fh8a1 c.47.1.10 (A:1-170) automated matches {Ancylostoma ceylanicum [TaxId: 53326]}
</pre><pre>Probab=40.47  E-value=94  Score=19.35  Aligned_cols=16  Identities=13%  Similarity=0.241  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++-.|.
</pre><pre>T Consensus       126 ~~~lid~~g~i~~~~~  141 (170)
</pre><pre>T d4fh8a1         126 GLFIIDPKGILRQITI  141 (170)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>6L81_D Mitotic-spindle organizing protein 1; gamma tubulin complex, microprotein, microtubule, TRANSLATION; 2.19650999049A {Homo sapiens}
</pre><pre>Probab=40.34  E-value=85  Score=21.73  Aligned_cols=29  Identities=17%  Similarity=0.353  Sum_probs=0.0  Template_Neff=4.100
</pre><pre>
</pre><pre>Q ss_pred             CcccHHHHHHHHhcCCCCCCHHHHHHHHh
</pre><pre>Q WP_134219109.1    3 TGFSRHAINQAISRDGVGVSSEAILDAVN   31 (69)
</pre><pre>Q Consensus         3 tG~TkHGl~qai~RdG~GVs~~AildAv~   31 (69)
</pre><pre>                      ||++++-|.-.|.-=..||+|+++...|+
</pre><pre>T Consensus        40 TGLdretL~ici~L~e~GVnPeALA~vIk   68 (85)
</pre><pre>T 6L81_D           40 TGLDMETLSICVRLCEQGINPEALSSVIK   68 (85)
</pre><pre>T ss_dssp             CCCCHHHHHHHHHHHHTTCCHHHHHHHHH
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>d2pn8a1 c.47.1.0 (A:84-269) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=40.18  E-value=1.1e+02  Score=19.63  Aligned_cols=19  Identities=11%  Similarity=0.258  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      +++++|++|+++..+....
</pre><pre>T Consensus       118 ~~~lid~~g~v~~~~~~~~  136 (186)
</pre><pre>T d2pn8a1         118 GLFIIDDKGILRQITLNDL  136 (186)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEEECT
</pre><pre>T ss_pred             EEEEECCCCeEEEEEeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>d2obia_ c.47.1.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=40.10  E-value=1.4e+02  Score=18.46  Aligned_cols=24  Identities=13%  Similarity=0.087  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+...+--..++++++|+++..|
</pre><pre>T Consensus       125 ~~~~~~~~P~~~lid~~g~i~~~~  148 (165)
</pre><pre>T d2obia_         125 LGNAIKWNFTKFLIDKNGCVVKRY  148 (165)
</pre><pre>T ss_dssp             SSSSCCSTTCEEEECTTSCEEEEE
</pre><pre>T ss_pred             ccceeeeccEEEEECCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>d6b9xb1 d.110.7.1 (B:1-125) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=40.10  E-value=1.1e+02  Score=20.01  Aligned_cols=21  Identities=24%  Similarity=0.461  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      |-.++++++++|.++......
</pre><pre>T Consensus        18 gv~~~~l~~~dG~~i~~~~~~   38 (125)
</pre><pre>T d6b9xb1          18 GVQSTLLLNNEGSLLAYSGYG   38 (125)
</pre><pre>T ss_dssp             TEEEEEEECTTCCEEEEEECC
</pre><pre>T ss_pred             CccEEEEEcCCCCeEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>d1xvwa1 c.47.1.10 (A:1-153) Putative peroxiredoxin Rv2238c/MT2298 {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=39.94  E-value=96  Score=18.85  Aligned_cols=16  Identities=6%  Similarity=0.154  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.+.
</pre><pre>T Consensus       117 ~~~lid~~g~i~~~~~  132 (153)
</pre><pre>T d1xvwa1         117 GTFVVDRSGIIRFAEM  132 (153)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEEcCCCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>2JSZ_A Probable thiol peroxidase; solution structure, Antioxidant, Oxidoreductase, Peroxidase; NMR {Bacillus subtilis} SCOP: c.47.1.0
</pre><pre>Probab=39.91  E-value=85  Score=21.09  Aligned_cols=15  Identities=47%  Similarity=0.654  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ++++||++|+|+-.|
</pre><pre>T Consensus       130 ~~~lid~~G~v~~~~  144 (167)
</pre><pre>T 2JSZ_A          130 SVFVLDENGKVVYAE  144 (167)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>4K9Z_A Putative thiol-disulfide oxidoreductase; PUTATIVE DISULFIDE OXIDOREDUCTASE, NYSGRC, Structural Genomics, New York Structural Genomics Research Consortium, PSI-Biology, OXIDOREDUCTASE; 1.8A {Bacteroides vulgatus} SCOP: c.47.1.0
</pre><pre>Probab=39.71  E-value=72  Score=19.96  Aligned_cols=16  Identities=13%  Similarity=0.501  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..|.
</pre><pre>T Consensus       127 ~~~lid~~g~v~~~~~  142 (168)
</pre><pre>T 4K9Z_A          127 RFILIDKDFNIVNAYA  142 (168)
</pre><pre>T ss_dssp             EEEEECTTSBEEETTC
</pre><pre>T ss_pred             EEEEECCCCcEEeCcC
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>d1psqa_ c.47.1.10 (A:) Probable thiol peroxidase PsaD {Pneumococcus (Streptococcus pneumoniae) [TaxId: 1313]}
</pre><pre>Probab=39.36  E-value=96  Score=19.23  Aligned_cols=15  Identities=20%  Similarity=0.295  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.|
</pre><pre>T Consensus       128 ~~~lid~~g~i~~~~  142 (163)
</pre><pre>T d1psqa_         128 AVFVLDTDNTIRYVE  142 (163)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>3RAZ_A Thioredoxin-related protein; Structural Genomics, PSI-2, Protein Structure Initiative, New York SGX Research Center for Structural Genomics, NYSGXRC, thioredoxin-related; HET: MSE; 2.0A {Neisseria meningitidis serogroup B}
</pre><pre>Probab=39.35  E-value=1e+02  Score=18.59  Aligned_cols=16  Identities=13%  Similarity=-0.062  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++++|++|+++-.|.
</pre><pre>T Consensus       108 ~~~l~d~~g~v~~~~~  123 (151)
</pre><pre>T 3RAZ_A          108 FTVVEAPKCGYRQTIT  123 (151)
</pre><pre>T ss_dssp             EEEEEETTTTEEEECC
</pre><pre>T ss_pred             EEEEECCCceEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>d1zofa1 c.47.1.10 (A:1-170) Thioredoxin reductase TsaA {Helicobacter pylori [TaxId: 210]}
</pre><pre>Probab=39.11  E-value=1e+02  Score=19.35  Aligned_cols=16  Identities=13%  Similarity=0.181  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++-.|.
</pre><pre>T Consensus       125 ~~~lid~~g~i~~~~~  140 (170)
</pre><pre>T d1zofa1         125 GAFLIDKNMKVRHAVI  140 (170)
</pre><pre>T ss_dssp             EEEEEETTTEEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>d3eura1 c.47.1.0 (A:1-139) automated matches {Bacteroides fragilis [TaxId: 272559]}
</pre><pre>Probab=39.08  E-value=76  Score=18.93  Aligned_cols=15  Identities=13%  Similarity=0.229  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++..+
</pre><pre>T Consensus       113 ~~~~id~~g~i~~~~  127 (139)
</pre><pre>T d3eura1         113 TLYLLDKNKTVLLKD  127 (139)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEeecC
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>d1o73a_ c.47.1.10 (A:) Tryparedoxin I {Trypanosoma brucei brucei [TaxId: 5702]}
</pre><pre>Probab=38.92  E-value=56  Score=19.85  Aligned_cols=16  Identities=25%  Similarity=0.578  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++ |+|+-.+.
</pre><pre>T Consensus       111 ~~~lid~~~g~i~~~~~  127 (144)
</pre><pre>T d1o73a_         111 TLITINADTGAIIGTQA  127 (144)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCH
</pre><pre>T ss_pred             EEEEEeCCCCcEechhh
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>d2b1ka_ c.47.1.10 (A:) automated matches {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=38.91  E-value=1.1e+02  Score=18.58  Aligned_cols=16  Identities=13%  Similarity=0.275  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++..|.
</pre><pre>T Consensus       110 ~~~~~d~~g~~~~~~~  125 (149)
</pre><pre>T d2b1ka_         110 ETFLIDGNGIIRYRHA  125 (149)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>3OR5_A Thiol:disulfide interchange protein, thioredoxin family protein; PSI-II, Structural Genomics, Protein Structure Initiative, New York SGX Research Center for Structural Genomics, NYSGXRC, OXIDOREDUCTASE; HET: MSE; 1.66A {Chlorobaculum tepidum} SCOP: c.47.1.0
</pre><pre>Probab=38.86  E-value=1.1e+02  Score=18.61  Aligned_cols=16  Identities=13%  Similarity=0.314  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.|.
</pre><pre>T Consensus       119 ~~~lid~~g~v~~~~~  134 (165)
</pre><pre>T 3OR5_A          119 TSFVIDASGNVSGVIV  134 (165)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEC
</pre><pre>T ss_pred             EEEEEcCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>d4nmua1 c.47.1.0 (A:1-144) automated matches {Anthrax bacillus (Bacillus anthracis) [TaxId: 1392]}
</pre><pre>Probab=38.40  E-value=1.2e+02  Score=18.12  Aligned_cols=16  Identities=25%  Similarity=0.528  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..+.
</pre><pre>T Consensus       112 ~~~~id~~g~i~~~~~  127 (144)
</pre><pre>T d4nmua1         112 TSFLIDKDGKVVEQII  127 (144)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEEE
</pre><pre>T ss_pred             eeEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>d3dwva_ c.47.1.0 (A:) automated matches {Trypanosome (Trypanosoma brucei) [TaxId: 5691]}
</pre><pre>Probab=38.32  E-value=1.1e+02  Score=18.65  Aligned_cols=25  Identities=12%  Similarity=0.179  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+...+--+++++|++|+++..+.
</pre><pre>T Consensus       123 ~~~~~~~~P~~~lid~~g~v~~~~~  147 (163)
</pre><pre>T d3dwva_         123 ATKAIKWNFTSFLIDRDGVPVERFS  147 (163)
</pre><pre>T ss_dssp             SSSSCCSTTCEEEECTTSCEEEEEC
</pre><pre>T ss_pred             cccccccccEEEEECCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>d3gl3a1 c.47.1.0 (A:30-168) automated matches {Chlorobium tepidum [TaxId: 194439]}
</pre><pre>Probab=38.28  E-value=1e+02  Score=18.09  Aligned_cols=15  Identities=20%  Similarity=0.483  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ++++++++|+++-.|
</pre><pre>T Consensus       105 ~~~~id~~g~~~~~~  119 (139)
</pre><pre>T d3gl3a1         105 TSFLIDRNGKVLLQH  119 (139)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>4G2E_A Peroxiredoxin; redox protein, structural genomics, NPPSFA, National Project on Protein Structural and Functional Analyses, RIKEN Structural Genomics/Proteomics; 1.4A {Sulfolobus tokodaii} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=38.26  E-value=1.1e+02  Score=18.41  Aligned_cols=16  Identities=44%  Similarity=0.885  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++++|++|+++-.|.
</pre><pre>T Consensus       121 ~~~lid~~g~v~~~~~  136 (157)
</pre><pre>T 4G2E_A          121 AVFVIDKEGKVRYKWV  136 (157)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>2LRT_A Uncharacterized protein; STRUCTURAL GENOMICS, THIOREDOXIN-LIKE, New York Structural Genomics Research Consortium, NYSGRC, PSI-Biology, OXIDOREDUCTASE; NMR {Bacteroides vulgatus}
</pre><pre>Probab=38.26  E-value=87  Score=19.32  Aligned_cols=16  Identities=13%  Similarity=0.275  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++-.+.
</pre><pre>T Consensus       116 ~~~lid~~g~v~~~~~  131 (152)
</pre><pre>T 2LRT_A          116 SVFLVNRNNELSARGE  131 (152)
</pre><pre>T ss_dssp             EEEEEETTTEEEEETT
</pre><pre>T ss_pred             EEEEECCCCeEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>d5epfa1 c.47.1.0 (A:9-161) automated matches {Mycobacterium tuberculosis [TaxId: 83332]}
</pre><pre>Probab=38.24  E-value=1e+02  Score=18.69  Aligned_cols=16  Identities=13%  Similarity=0.378  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.++++++|+++-.+.
</pre><pre>T Consensus       117 ~~~~i~~~g~i~~~~~  132 (153)
</pre><pre>T d5epfa1         117 TTFVIDTDRKVLDVIS  132 (153)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEC
</pre><pre>T ss_pred             EEEEECCCCeEEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>d1jfua_ c.47.1.10 (A:) Membrane-anchored thioredoxin-like protein TlpA, soluble domain {Bradyrhizobium japonicum [TaxId: 375]}
</pre><pre>Probab=37.95  E-value=1.1e+02  Score=19.43  Aligned_cols=15  Identities=20%  Similarity=0.386  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       142 ~~~lid~~g~v~~~~  156 (176)
</pre><pre>T d1jfua_         142 TSVLVDPQGCEIATI  156 (176)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>3LEQ_A uncharacterized protein cvnB5; PSI, MCSG, Structural Genomics, Midwest Center for Structural Genomics, Protein Structure Initiative, structure genomics, unknown function; HET: MSE; 1.85A {Streptomyces avermitilis}
</pre><pre>Probab=37.50  E-value=1.1e+02  Score=18.98  Aligned_cols=20  Identities=30%  Similarity=0.388  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeec
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWAR   67 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~   67 (69)
</pre><pre>                      |=.++++++.+|.++..++.
</pre><pre>T Consensus        18 gv~~~~l~~~~G~~i~~~~~   37 (126)
</pre><pre>T 3LEQ_A           18 EVDHAVVLSEDGLVVSKSTG   37 (126)
</pre><pre>T ss_dssp             TEEEEEEEETTSCEEEECTT
</pre><pre>T ss_pred             CeeEEEEEcCCCceeEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>4HQZ_A Thioredoxin family protein; lipoprotein, Disulfide Bridge, Thiorredoxin, membrane, OXIDOREDUCTASE; HET: HED; 1.22A {Streptococcus pneumoniae} SCOP: c.47.1.0
</pre><pre>Probab=37.33  E-value=1e+02  Score=20.00  Aligned_cols=15  Identities=13%  Similarity=0.306  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       156 ~~~lid~~G~v~~~~  170 (187)
</pre><pre>T 4HQZ_A          156 TEYLIDSQGKIGKIQ  170 (187)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>2B1L_A Thiol:disulfide interchange protein dsbE; Comparison with the E.coli CcmG, folding topology change, OXIDOREDUCTASE; 1.9A {Escherichia coli} SCOP: c.47.1.10
</pre><pre>Probab=37.32  E-value=1.5e+02  Score=17.20  Aligned_cols=19  Identities=11%  Similarity=0.214  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++|+++..+....
</pre><pre>T Consensus        89 ~~~~~~~~g~~~~~~~g~~  107 (129)
</pre><pre>T 2B1L_A           89 ETFLIDGNGIIRYRHAGDL  107 (129)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEEESCC
</pre><pre>T ss_pred             EEEEEcCCCeEEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>3IXR_A Bacterioferritin comigratory protein; Alpha Beta Protein, OXIDOREDUCTASE; 1.6A {Xylella fastidiosa} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=37.03  E-value=1.1e+02  Score=19.37  Aligned_cols=15  Identities=13%  Similarity=0.647  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.|
</pre><pre>T Consensus       143 ~~~lid~~g~i~~~~  157 (179)
</pre><pre>T 3IXR_A          143 STFLIGPTHRIVEAW  157 (179)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>3P7X_C Probable thiol peroxidase; THIOREDOXIN FOLD, OXIDOREDUCTASE, PEROXIDASE; HET: SO4, PG4; 1.96A {Staphylococcus aureus} SCOP: c.47.1.0
</pre><pre>Probab=36.98  E-value=1.2e+02  Score=18.48  Aligned_cols=16  Identities=38%  Similarity=0.447  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.+.
</pre><pre>T Consensus       131 ~~~lid~~g~v~~~~~  146 (166)
</pre><pre>T 3P7X_C          131 AVFVLDADNKVVYKEI  146 (166)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>6L80_B Mitotic-spindle organizing protein 1; gamma tubulin complex, microprotien, microtubule, TRANSLATION; 2.00049753238A {Schizosaccharomyces pombe (strain 972 / ATCC 24843)}
</pre><pre>Probab=36.94  E-value=1.1e+02  Score=21.86  Aligned_cols=29  Identities=24%  Similarity=0.276  Sum_probs=0.0  Template_Neff=4.000
</pre><pre>
</pre><pre>Q ss_pred             CcccHHHHHHHHhcCCCCCCHHHHHHHHh
</pre><pre>Q WP_134219109.1    3 TGFSRHAINQAISRDGVGVSSEAILDAVN   31 (69)
</pre><pre>Q Consensus         3 tG~TkHGl~qai~RdG~GVs~~AildAv~   31 (69)
</pre><pre>                      ||+++.-|.-.+.-=..||+|+|+.+.++
</pre><pre>T Consensus        57 TGLd~EtL~iCi~LiE~GVnPeALA~vIk   85 (100)
</pre><pre>T 6L80_B           57 TELDKTTLSLCISLCENNVHPEAIAQIIR   85 (100)
</pre><pre>T ss_dssp             CCCCHHHHHHHHHHHHTTCCHHHHHHHHH
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>4FYU_A Thioredoxin; five parallel and anti parallel beta sheet, Oxidoreductase; HET: PEG; 2.0A {Wuchereria bancrofti}
</pre><pre>Probab=36.93  E-value=1.1e+02  Score=17.87  Aligned_cols=16  Identities=19%  Similarity=0.567  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..++++++|+++..+.
</pre><pre>T Consensus       110 ~~~~id~~g~i~~~~~  125 (145)
</pre><pre>T 4FYU_A          110 MLIVIKSDGNVITKNG  125 (145)
</pre><pre>T ss_dssp             EEEEEETTSCEEESCH
</pre><pre>T ss_pred             EEEEECCCCcEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>d2m72a1 c.47.1.0 (A:4-148) automated matches {Porphyromonas gingivalis [TaxId: 242619]}
</pre><pre>Probab=36.93  E-value=1.1e+02  Score=18.00  Aligned_cols=15  Identities=40%  Similarity=0.625  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.|
</pre><pre>T Consensus       111 ~~~lid~~g~i~~~~  125 (145)
</pre><pre>T d2m72a1         111 AVFIVDGNGKIVYNH  125 (145)
</pre><pre>T ss_dssp             EEEEECTTSSEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>PF20029.1 ; DUF6436 ; Domain of unknown function (DUF6436)
</pre><pre>Probab=36.88  E-value=83  Score=18.33  Aligned_cols=12  Identities=25%  Similarity=0.451  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEE
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVV   62 (69)
</pre><pre>                      .++|||++|+++
</pre><pre>T Consensus         5 ~~~vld~~~~l~   16 (59)
</pre><pre>T A0A0T6URZ6_9PS    5 AVAIWDRNDQLA   16 (59)
</pre><pre>T ss_pred             eEEEEcCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>PF18545.3 ; HalOD1 ; Halobacterial output domain 1
</pre><pre>Probab=36.85  E-value=2e+02  Score=18.14  Aligned_cols=42  Identities=17%  Similarity=0.343  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHhCCceeecccCCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   21 VSSEAILDAVNNPTKVIPQVGGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        21 Vs~~AildAv~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      |.+.|+-..+.....-.......+.|.=.+-.|.+..+|.|.
</pre><pre>T Consensus        32 IDpdALd~L~~~~~~~~~~~~~~v~F~~~~~~VtV~~dG~v~   73 (76)
</pre><pre>T L0AHC4_NATGS/3   32 VDPDALDKLFRPRHDGTARLGGHVSFVVADHRVTVYSDGEIL   73 (76)
</pre><pre>T ss_pred             cCHHHHHHHccccCCCCCCCceEEEEEECCEEEEEecCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>d3ewla1 c.47.1.0 (A:1-135) automated matches {Bacteroides fragilis [TaxId: 272559]}
</pre><pre>Probab=36.72  E-value=96  Score=18.39  Aligned_cols=15  Identities=13%  Similarity=0.288  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++..+
</pre><pre>T Consensus       109 ~~~~~~~~g~i~~~~  123 (135)
</pre><pre>T d3ewla1         109 TIYLLDGRKRVILKD  123 (135)
</pre><pre>T ss_dssp             EEEEECTTCBEEECS
</pre><pre>T ss_pred             EEEEECCCCeEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>3ERW_A Sporulation thiol-disulfide oxidoreductase A; Thioredoxin-like fold, ResA-like fold, disulfide, dithiol, stoa, Oxidoreductase, Redox-active center, Sporulation; HET: MSE; 2.5A {Bacillus subtilis} SCOP: c.47.1.0
</pre><pre>Probab=36.40  E-value=1.2e+02  Score=17.77  Aligned_cols=15  Identities=33%  Similarity=0.517  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.+
</pre><pre>T Consensus       117 ~~~~id~~g~i~~~~  131 (145)
</pre><pre>T 3ERW_A          117 TSFLLNEKGEIEKTK  131 (145)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>3ERW_B Sporulation thiol-disulfide oxidoreductase A; Thioredoxin-like fold, ResA-like fold, disulfide, dithiol, stoa, Oxidoreductase, Redox-active center, Sporulation; HET: MSE; 2.5A {Bacillus subtilis} SCOP: c.47.1.0
</pre><pre>Probab=36.40  E-value=1.2e+02  Score=17.77  Aligned_cols=15  Identities=33%  Similarity=0.517  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.+
</pre><pre>T Consensus       117 ~~~~id~~g~i~~~~  131 (145)
</pre><pre>T 3ERW_B          117 TSFLLNEKGEIEKTK  131 (145)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>3T1S_A Gliding protein MglB; GTPase activating protein, bacterial polarity, motility, Pole localisation, alpha/beta protein, catalytic GAP domain, Homodimer, SIGNALING PROTEIN; 1.67A {Thermus thermophilus}
</pre><pre>Probab=36.37  E-value=1.4e+02  Score=18.95  Aligned_cols=21  Identities=19%  Similarity=0.352  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      |-.++++++++|.++..+...
</pre><pre>T Consensus        25 gv~~~~l~~~dG~~i~~~~~~   45 (136)
</pre><pre>T 3T1S_A           25 GARYALLIDRKGFVLAHKEAL   45 (136)
</pre><pre>T ss_dssp             CCSEEEEEETTSCEEEEEECT
</pre><pre>T ss_pred             CCCEEEEEcCCCcEEEEeccc
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>d2fega1 c.47.1.0 (A:8-175) automated matches {Plasmodium yoelii}
</pre><pre>Probab=36.29  E-value=1.2e+02  Score=18.93  Aligned_cols=16  Identities=25%  Similarity=0.443  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++..+.
</pre><pre>T Consensus       119 ~~~lid~~g~i~~~~~  134 (168)
</pre><pre>T d2fega1         119 AFVLIDKQGVVQHLLV  134 (168)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>2RLI_A SCO2 protein homolog, mitochondrial; Copper protein, thioredoxin fold, Metal transport, Structural Genomics, SPINE2-Complexes, Structural Proteomics in Europe, SPINE; HET: CU1; NMR {Homo sapiens} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=36.25  E-value=2e+02  Score=17.93  Aligned_cols=29  Identities=14%  Similarity=0.135  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             cccCCcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   38 PQVGGTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        38 ~q~g~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ......+...+--+.+++|++|+++..|.
</pre><pre>T Consensus       119 ~~~~~~~~~~~~p~~~lid~~g~v~~~~~  147 (171)
</pre><pre>T 2RLI_A          119 KDEDQDYIVDHSIAIYLLNPDGLFTDYYG  147 (171)
</pre><pre>T ss_dssp             CCSSCCCCEECCCEEEEECTTSCEEEEEE
</pre><pre>T ss_pred             CCCCCCEEEEcCceEEEECCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>d1knga_ c.47.1.10 (A:) Thioredoxin-like protein CcmG (CycY, DsbE) {Bradyrhizobium japonicum [TaxId: 375]}
</pre><pre>Probab=36.21  E-value=1.4e+02  Score=17.92  Aligned_cols=18  Identities=17%  Similarity=0.340  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      +++++|++|+++-.+...
</pre><pre>T Consensus       109 ~~~~i~~~g~~~~~~~g~  126 (144)
</pre><pre>T d1knga_         109 ETFVVGREGTIVYKLVGP  126 (144)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEESC
</pre><pre>T ss_pred             EEEEECCCCeEEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>3LOR_B Thiol-disulfide isomerase and thioredoxins; Thiol, isomerase, PSI, MCSG, Structural Genomics, Midwest Center for Structural Genomics, Protein Structure Initiative; HET: MSE; 2.2A {Corynebacterium glutamicum}
</pre><pre>Probab=36.11  E-value=1.3e+02  Score=18.06  Aligned_cols=15  Identities=7%  Similarity=0.454  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++..+
</pre><pre>T Consensus       123 ~~~lid~~g~i~~~~  137 (160)
</pre><pre>T 3LOR_B          123 SIILADRKGRIRQVQ  137 (160)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>2L5O_A Putative thioredoxin; STRUCTURAL GENOMICS, UNKNOWN FUNCTION, PSI-2, Protein Structure Initiative, New York SGX Research Center for Structural Genomics; NMR {Neisseria meningitidis serogroup B} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=36.05  E-value=1.2e+02  Score=18.23  Aligned_cols=15  Identities=20%  Similarity=0.685  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.+
</pre><pre>T Consensus       109 ~~~lid~~g~v~~~~  123 (153)
</pre><pre>T 2L5O_A          109 TSVLIGKKGEILKTY  123 (153)
</pre><pre>T ss_dssp             EEEEECSSSCCCEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 146
</pre><pre>>3FKF_D thiol-disulfide oxidoreductase; thiol disulfide oxidoreductase, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, OXIDOREDUCTASE; HET: ACT, MSE; 2.2A {Bacteroides fragilis}
</pre><pre>Probab=36.04  E-value=96  Score=18.47  Aligned_cols=23  Identities=13%  Similarity=0.189  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   44 IKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        44 ~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +...+--.++++|++|+++-.|.
</pre><pre>T Consensus       110 ~~~~~~P~~~~id~~g~v~~~~~  132 (148)
</pre><pre>T 3FKF_D          110 YAILTLPTNILLSPTGKILARDI  132 (148)
</pre><pre>T ss_dssp             TTCCSSSEEEEECTTSBEEEESC
</pre><pre>T ss_pred             hccccCCeeEEECCCCcEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 147
</pre><pre>>3IA1_B Thio-disulfide isomerase/thioredoxin; OXIDOREDUCTASE, DISULFIDE ISOMERASE, PSI-2, NYSGXRC, STRUCTURAL GENOMICS, PROTEIN STRUCTURE INITIATIVE, NEW YORK SGX RESEARCH CENTER FOR; 1.76A {Thermus thermophilus}
</pre><pre>Probab=36.02  E-value=1.2e+02  Score=18.49  Aligned_cols=15  Identities=33%  Similarity=0.603  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       112 ~~~lid~~g~i~~~~  126 (154)
</pre><pre>T 3IA1_B          112 WTFVVDREGKVVALF  126 (154)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 148
</pre><pre>>3HVV_A Thiol peroxidase; oxidoreductase, Tpx, peroxiredoxin, peroxidase, Antioxidant; 1.75A {Escherichia coli K-12} SCOP: c.47.1.10
</pre><pre>Probab=36.00  E-value=1.1e+02  Score=18.92  Aligned_cols=15  Identities=33%  Similarity=0.574  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+|+-.|
</pre><pre>T Consensus       133 ~~~lid~~g~v~~~~  147 (167)
</pre><pre>T 3HVV_A          133 AVVVIDENDNVIFSQ  147 (167)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             eEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 149
</pre><pre>>2LJA_A Putative thiol-disulfide oxidoreductase; STRUCTURAL GENOMICS, UNKNOWN FUNCTION, THIOREDOXIN-LIKE, PSI-Biology, New York Structural Genomics Research Consortium, NYSGRC; NMR {Bacteroides vulgatus} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=35.78  E-value=84  Score=19.14  Aligned_cols=16  Identities=19%  Similarity=0.594  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..+.
</pre><pre>T Consensus       111 ~~~lid~~g~i~~~~~  126 (152)
</pre><pre>T 2LJA_A          111 RFILLDRDGKIISANM  126 (152)
</pre><pre>T ss_dssp             CEEEECTTSCEEESSC
</pre><pre>T ss_pred             cEEEECCCCCEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 150
</pre><pre>>d1we0a1 c.47.1.10 (A:1-166) Alkyl hydroperoxide reductase AhpC {Amphibacillus xylanus [TaxId: 1449]}
</pre><pre>Probab=35.59  E-value=1.3e+02  Score=18.93  Aligned_cols=16  Identities=13%  Similarity=0.277  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ++++++++|+++-.|.
</pre><pre>T Consensus       121 ~~~lid~~g~i~~~~~  136 (166)
</pre><pre>T d1we0a1         121 GTFIIDPDGVIQAIEI  136 (166)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 151
</pre><pre>>6ZU5_LV0 uL14; Microsporidia, Pathogen, Ribosome, Hibernation, Genome Compaction; HET: AMP, MG; 2.9A {Paranosema locustae}
</pre><pre>Probab=35.57  E-value=1e+02  Score=20.27  Aligned_cols=34  Identities=18%  Similarity=0.290  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HhCCceeecccCCcEEEEcCCEEEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   30 VNNPTKVIPQVGGTIKYVGERAVVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        30 v~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVVTt   64 (69)
</pre><pre>                      ++.+.......|..++|. ++++|++|.+|..+-+
</pre><pre>T Consensus        82 v~~~~~~~r~~g~~~~f~-~n~~vll~~~~~p~gt  115 (140)
</pre><pre>T 6ZU5_LV0         82 IRQKKAWRRRDGSHICFE-DNAAVVITNKGDPKGT  115 (140)
</pre><pre>T ss_dssp             EECSSCEECTTSCEEEES-SCEEEEBCTTSCBSSS
</pre><pre>T ss_pred             ccccceeccCCCCeEeeC-CceEEEECCCCCcccc
</pre><pre>
</pre><pre>
</pre><pre>No 152
</pre><pre>>5UM7_A Thioredoxin signature protein; THIOREDOXIN-LIKE DOMAIN, ALPHA-BETA PROTEIN, OXIDOREDUCTASE, structural genomics, Center for Structural Genomics of Infectious Diseases, CSGID, National; 1.62A {Streptococcus gordonii (strain Challis / ATCC 35105 / BCRC 15272 / CH1 / DL1 / V288)} SCOP: c.47.1.0
</pre><pre>Probab=35.47  E-value=1.2e+02  Score=18.56  Aligned_cols=16  Identities=13%  Similarity=0.316  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..|.
</pre><pre>T Consensus       157 ~~~~~d~~g~i~~~~~  172 (187)
</pre><pre>T 5UM7_A          157 TIYLVDKNQKVKKVMT  172 (187)
</pre><pre>T ss_dssp             EEEEECTTSEEEEEEC
</pre><pre>T ss_pred             EEEEECCCCeEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 153
</pre><pre>>6HY0_I Major Outer Capsid Protein P8; virus, dsRNA, capsid; 3.5A {Pseudomonas phage phi6}
</pre><pre>Probab=35.42  E-value=93  Score=23.37  Aligned_cols=28  Identities=36%  Similarity=0.407  Sum_probs=0.0  Template_Neff=1.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhcC-CCCCCHHHHHHHHhCCcee
</pre><pre>Q WP_134219109.1    9 AINQAISRD-GVGVSSEAILDAVNNPTKV   36 (69)
</pre><pre>Q Consensus         9 Gl~qai~Rd-G~GVs~~AildAv~nP~~v   36 (69)
</pre><pre>                      ||-..|.-. |.-+||+||+.||+|--.+
</pre><pre>T Consensus        23 glisria~~igsk~spsailaavk~~pvv   51 (149)
</pre><pre>T 6HY0_I           23 GLVSRIAAAIGSKVSPSAILAAVKSNPVV   51 (149)
</pre><pre>T ss_dssp             HHHHHHHHHHCCCSSHHHHHHHHHHCHHH
</pre><pre>T ss_pred             cHHHHHHHHhcCCCCHHHHHHHHhcCccc
</pre><pre>
</pre><pre>
</pre><pre>No 154
</pre><pre>>3LWA_A Secreted thiol-disulfide isomerase; Thiol-disulfide, isomerase, thioredoxin, Corynebacterium, glutamicum, PSI, MCSG, Structural Genomics, Midwest Center for Structural Genomics, Protein Structure; HET: MSE; 1.75A {Corynebacterium glutamicum}
</pre><pre>Probab=35.42  E-value=1.2e+02  Score=19.52  Aligned_cols=15  Identities=7%  Similarity=0.439  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++..|
</pre><pre>T Consensus       149 ~~~lid~~g~v~~~~  163 (183)
</pre><pre>T 3LWA_A          149 TTIVLDKQHRPAAVF  163 (183)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEECCCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 155
</pre><pre>>1KNG_A THIOL:DISULFIDE INTERCHANGE PROTEIN CYCY; thioredoxin fold, cytochrome c maturation, atomic resolution, OXIDOREDUCTASE; 1.14A {Bradyrhizobium japonicum} SCOP: c.47.1.10
</pre><pre>Probab=35.42  E-value=1.4e+02  Score=18.12  Aligned_cols=16  Identities=19%  Similarity=0.422  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++..+.
</pre><pre>T Consensus       120 ~~~lid~~g~i~~~~~  135 (156)
</pre><pre>T 1KNG_A          120 ETFVVGREGTIVYKLV  135 (156)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 156
</pre><pre>>PF18832.3 ; LPD18 ; Large polyvalent protein-associated domain 18
</pre><pre>Probab=35.37  E-value=84  Score=20.91  Aligned_cols=13  Identities=46%  Similarity=0.782  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEE
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVT   63 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVT   63 (69)
</pre><pre>                      |.||+|.+|++|.
</pre><pre>T Consensus        47 a~iivd~~G~~v~   59 (86)
</pre><pre>T R6G1R0_9FIRM/1   47 AYVVLSDDGAVVM   59 (86)
</pre><pre>T ss_pred             ceEEECCCCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 157
</pre><pre>>1PSQ_A probable thiol peroxidase; STRUCTURAL GENOMICS, THIOL, PEROXIDASE, NYSGXRC, T817, PSI, Protein Structure Initiative, New York SGX Research Center for; HET: MSE; 2.3A {Streptococcus pneumoniae} SCOP: c.47.1.10
</pre><pre>Probab=35.31  E-value=1.2e+02  Score=18.37  Aligned_cols=15  Identities=20%  Similarity=0.295  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++..|
</pre><pre>T Consensus       128 ~~~lid~~g~i~~~~  142 (163)
</pre><pre>T 1PSQ_A          128 AVFVLDTDNTIRYVE  142 (163)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             EEEEEcCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 158
</pre><pre>>d2f8aa1 c.47.1.10 (A:12-195) Glutathione peroxidase {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=35.22  E-value=1.2e+02  Score=20.63  Aligned_cols=24  Identities=4%  Similarity=0.079  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   43 TIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        43 ~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .+-+.+.-+.+|+|++|+++..|.
</pre><pre>T Consensus       144 ~~~~~~~p~~~lid~~G~v~~~~~  167 (184)
</pre><pre>T d2f8aa1         144 RNDVAWNFEKFLVGPDGVPLRRYS  167 (184)
</pre><pre>T ss_dssp             TTCCCSTTCEEEECTTSCEEEEEC
</pre><pre>T ss_pred             cccceecceEEEECCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 159
</pre><pre>>PF09695.12 ; YtfJ_HI0045 ; Bacterial protein of unknown function (YtfJ_HI0045)
</pre><pre>Probab=35.10  E-value=1.2e+02  Score=18.76  Aligned_cols=15  Identities=20%  Similarity=0.530  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       126 ~~~lid~~g~i~~~~  140 (159)
</pre><pre>T Q87JM8_VIBPA/2  126 AIIVQNKQGEILFVK  140 (159)
</pre><pre>T ss_pred             EEEEEcCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 160
</pre><pre>>d3keba_ c.47.1.0 (A:) automated matches {Chromobacterium violaceum [TaxId: 536]}
</pre><pre>Probab=34.96  E-value=1.3e+02  Score=20.26  Aligned_cols=17  Identities=12%  Similarity=0.141  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   49 ERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        49 k~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|+|++|+|+-.+
</pre><pre>T Consensus       134 ~p~~~lid~~g~i~~~~  150 (175)
</pre><pre>T d3keba_         134 SPAIILADAANVVHYSE  150 (175)
</pre><pre>T ss_dssp             CCEEEEECTTCBEEEEE
</pre><pre>T ss_pred             CCEEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 161
</pre><pre>>d5enua1 c.47.1.0 (A:1-153) automated matches {Burkholderia ambifaria [TaxId: 398577]}
</pre><pre>Probab=34.90  E-value=1.1e+02  Score=18.70  Aligned_cols=15  Identities=13%  Similarity=0.694  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.|
</pre><pre>T Consensus       120 ~~~lid~~g~v~~~~  134 (153)
</pre><pre>T d5enua1         120 STFLIDADGVLRQAW  134 (153)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             ceEEECCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 162
</pre><pre>>4YGT_A Uncharacterized protein YjgB; putative secreted lipoprotein, the first structural representative of PF14172 family, Structural Genomics, Joint Center for Structural; HET: MSE, EDO; 2.13A {Bacillus subtilis}
</pre><pre>Probab=34.84  E-value=2.9e+02  Score=19.35  Aligned_cols=47  Identities=6%  Similarity=0.003  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCC-CCHHHHHHHHhCCceeecccCCcEEEEc----CCEEEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   18 GVG-VSSEAILDAVNNPTKVIPQVGGTIKYVG----ERAVVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        18 G~G-Vs~~AildAv~nP~~v~~q~g~~~~y~G----k~A~VVlN~~GkVVTt   64 (69)
</pre><pre>                      ..| -+.++|.+.+-.|.......++...|..    ....+-.|+++++...
</pre><pre>T Consensus        40 ~iG~tt~~~V~~~~G~P~~~~~~~g~~~~Y~~~~g~~~~~~~~~~~~~v~~i   91 (160)
</pre><pre>T 4YGT_A           40 TINKSTKGDVHAAFGEPERPVGGDNRFDLYHWNMGQPGYGFSYHKDMTISEI   91 (160)
</pre><pre>T ss_dssp             CTTTSBHHHHHHHHCSCSBCSBTTBCSEEECCBTTBCEEEEEECTTSBEEEE
</pre><pre>T ss_pred             CCCCCcHHHHHHHHCCCCceeecCCceEeccccCccCEEEEEECCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 163
</pre><pre>>1VQ8_K 50S ribosomal protein L14P; ribosome 50S, PROTEIN-PROTEIN COMPLEX, RNA-RNA COMPLEX, PROTEIN-RNA COMPLEX, PEPTIDYL TRANSFERASE REACTION, RIBOSOME; HET: CL, NA, OMG, 1MA, MG, SPS, SR, UR3, OMU, PSU; 2.2A {Haloarcula marismortui} SCOP: b.39.1.1
</pre><pre>Probab=34.79  E-value=1.1e+02  Score=19.59  Aligned_cols=24  Identities=29%  Similarity=0.293  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCEEEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   41 GGTIKYVGERAVVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        41 g~~~~y~Gk~A~VVlN~~GkVVTt   64 (69)
</pre><pre>                      .+.+.+..+++.|++|.+|+.+.+
</pre><pre>T Consensus        84 ~g~~~~f~~n~~vl~~~~~~~~g~  107 (132)
</pre><pre>T 1VQ8_K           84 DGTRVKFEDNAAVIVDENEDPRGT  107 (132)
</pre><pre>T ss_dssp             TSCEEEESSCEEEEECTTSCBSSS
</pre><pre>T ss_pred             CCCeeeeCCceEEEeCCCCCcCCc
</pre><pre>
</pre><pre>
</pre><pre>No 164
</pre><pre>>d4mh2a_ c.47.1.0 (A:) automated matches {Cow (Bos taurus) [TaxId: 9913]}
</pre><pre>Probab=34.64  E-value=1.4e+02  Score=18.38  Aligned_cols=16  Identities=13%  Similarity=0.279  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.++++++|+++-.|.
</pre><pre>T Consensus       124 ~~~lid~~g~i~~~~~  139 (165)
</pre><pre>T d4mh2a_         124 GLFIIDPNGVIKHLSV  139 (165)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 165
</pre><pre>>d2h30a_ c.47.1.10 (A:) Peptide methionine sulfoxide reductase MsrA/MsrB, N-terminal domain {Neisseria gonorrhoeae [TaxId: 485]}
</pre><pre>Probab=34.46  E-value=1.4e+02  Score=17.98  Aligned_cols=15  Identities=13%  Similarity=0.315  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       111 ~~~lid~~g~v~~~~  125 (151)
</pre><pre>T d2h30a_         111 SWALIGKDGDVQRIV  125 (151)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 166
</pre><pre>>3GL3_C Putative Thiol:disulfide interchange protein DsbE; Oxidoreductase, PSI-II, 11210h, Thiol:disulfide interchange protein DsbE, putative, Chlorobium tepidum, Structural Genomics, Protein Structure Initiative, New; HET: MSE; 2.09A {Chlorobium tepidum TLS} SCOP: c.47.1.0
</pre><pre>Probab=34.42  E-value=1.3e+02  Score=18.26  Aligned_cols=15  Identities=20%  Similarity=0.483  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       108 ~~~l~d~~g~i~~~~  122 (152)
</pre><pre>T 3GL3_C          108 TSFLIDRNGKVLLQH  122 (152)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 167
</pre><pre>>PF07887.13 ; Calmodulin_bind ; Calmodulin binding protein-like
</pre><pre>Probab=34.22  E-value=2.7e+02  Score=22.45  Aligned_cols=52  Identities=12%  Similarity=0.107  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhcCCCCCCHHHHHHHHhCCceeecccCCcEEE---EcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1    7 RHAINQAISRDGVGVSSEAILDAVNNPTKVIPQVGGTIKY---VGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus         7 kHGl~qai~RdG~GVs~~AildAv~nP~~v~~q~g~~~~y---~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      +..|.++++.   +++.+.-..++++-....... .-+.|   ...+..|++|.-++++
</pre><pre>T Consensus       200 ~~~L~~il~~---~~s~~~W~~~v~HA~~c~~~~-~~y~y~~~~~~~~glvFn~v~~vv  254 (293)
</pre><pre>T D7SQ62_VITVI/8  200 SQKLRSILGS---GMSNKMWEALIEHAKTCTMSG-KFYVYYSDDTRNVGVIFNNIYELS  254 (293)
</pre><pre>T ss_pred             cHHHHHHhCC---CCChhhHHHHHHHhhceeeCC-cEEEEEcCCCCcEEEEeccceeee
</pre><pre>
</pre><pre>
</pre><pre>No 168
</pre><pre>>d1aisa2 d.129.1.1 (A:93-181) TATA-box binding protein (TBP), C-terminal domain {Pyrococcus woesei [TaxId: 2262]}
</pre><pre>Probab=34.02  E-value=2.4e+02  Score=18.06  Aligned_cols=51  Identities=20%  Similarity=0.221  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHhCCceeecc-cCCcEEEEcCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   18 GVGVSSEAILDAVNNPTKVIPQ-VGGTIKYVGERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        18 G~GVs~~AildAv~nP~~v~~q-~g~~~~y~Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      +..+..+.+.+.+.+-.--++. .+-.++...+++++.+-..||++-+=+++
</pre><pre>T Consensus        20 ~~~i~L~~la~~~~~~~Y~p~~~p~l~~~~~~~~~~~~if~tGki~itGa~s   71 (89)
</pre><pre>T d1aisa2          20 GREFNLDVVALTLPNCEYEPEQFPGVIYRVKEPKSVILLFSSGKIVCSGAKS   71 (89)
</pre><pre>T ss_dssp             SSCCCHHHHHHHSTTEECCTTTCSSEEEEETTTTEEEEECTTSEEEEEEESS
</pre><pre>T ss_pred             CCCccHHHHHHhCCCcccCccccCceeEEecCCCeEEEEEcCCcEEecCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 169
</pre><pre>>d3erwa_ c.47.1.0 (A:) automated matches {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=33.94  E-value=1.4e+02  Score=17.48  Aligned_cols=15  Identities=33%  Similarity=0.517  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .+++++++|+++..+
</pre><pre>T Consensus       106 ~~~~id~~g~i~~~~  120 (134)
</pre><pre>T d3erwa_         106 TSFLLNEKGEIEKTK  120 (134)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 170
</pre><pre>>2LUS_A Thioredoxion; Cr-TRP16, OXIDOREDUCTASE; NMR {Carcinoscorpius rotundicauda}
</pre><pre>Probab=33.91  E-value=1.1e+02  Score=17.92  Aligned_cols=18  Identities=28%  Similarity=0.604  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ..+++|++|+++..+...
</pre><pre>T Consensus       111 ~~~~id~~g~i~~~~~~~  128 (143)
</pre><pre>T 2LUS_A          111 ALVIVKKDGTLISMNGRG  128 (143)
</pre><pre>T ss_dssp             EEEEEETTSCEEESSCHH
</pre><pre>T ss_pred             EEEEEcCCCEEEEccCce
</pre><pre>
</pre><pre>
</pre><pre>No 171
</pre><pre>>3E0U_A Glutathione peroxidase; thioredoxin fold, Oxidoreductase, Peroxidase; HET: GOL; 2.3A {Trypanosoma cruzi} SCOP: c.47.1.0
</pre><pre>Probab=33.88  E-value=1.1e+02  Score=18.18  Aligned_cols=16  Identities=13%  Similarity=0.401  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++-.|.
</pre><pre>T Consensus       130 ~~~lid~~g~v~~~~~  145 (166)
</pre><pre>T 3E0U_A          130 TSFLIDRHGVPVERFS  145 (166)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             eeeEecCCCcEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 172
</pre><pre>>d1y4oa1 d.110.7.1 (A:10-104) Dynein light chain 2A, cytoplasmic {Mouse (Mus musculus) [TaxId: 10090]}
</pre><pre>Probab=33.87  E-value=1.1e+02  Score=19.05  Aligned_cols=21  Identities=24%  Similarity=0.506  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      |-.+++|+|++|.++..+...
</pre><pre>T Consensus        15 gV~g~~i~~~dG~~i~s~~~~   35 (95)
</pre><pre>T d1y4oa1          15 GVQGIIVVNTEGIPIKSTMDN   35 (95)
</pre><pre>T ss_dssp             TEEEEEEEETTTEEEEESSCH
</pre><pre>T ss_pred             CeeEEEEEeCCCCeEEEcCCc
</pre><pre>
</pre><pre>
</pre><pre>No 173
</pre><pre>>d3hvva_ c.47.1.10 (A:) Thiol peroxidase Tpx {Escherichia coli K-12 [TaxId: 83333]}
</pre><pre>Probab=33.67  E-value=1.3e+02  Score=18.98  Aligned_cols=15  Identities=33%  Similarity=0.574  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++|+|++|+|+-.+
</pre><pre>T Consensus       133 ~~~lid~~g~v~~~~  147 (167)
</pre><pre>T d3hvva_         133 AVVVIDENDNVIFSQ  147 (167)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEEcCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 174
</pre><pre>>4QL7_D Alkylhydroperoxide Reductase subunit C; Oxidoreductase, Peroxiredoxin, AhpC; 3.75A {Escherichia coli}
</pre><pre>Probab=33.59  E-value=1.4e+02  Score=18.12  Aligned_cols=15  Identities=13%  Similarity=0.282  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.+
</pre><pre>T Consensus       121 ~~~~id~~g~i~~~~  135 (172)
</pre><pre>T 4QL7_D          121 ATFVVDPQGIIQAIE  135 (172)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 175
</pre><pre>>PF00578.23 ; AhpC-TSA ; AhpC/TSA family
</pre><pre>Probab=33.41  E-value=1.1e+02  Score=17.11  Aligned_cols=12  Identities=8%  Similarity=0.376  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEE
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVV   62 (69)
</pre><pre>                      ..+++|++|+++
</pre><pre>T Consensus       109 ~~~~~~~~g~v~  120 (122)
</pre><pre>T RESA_OCEIH/49-  109 TTYFIKPNGEIE  120 (122)
</pre><pre>T ss_pred             EEEEECCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 176
</pre><pre>>1XVQ_A thiol peroxidase; thioredoxin fold, Structural Genomics, PSI, Protein Structure Initiative, TB Structural Genomics Consortium, TBSGC, OXIDOREDUCTASE; 1.75A {Mycobacterium tuberculosis} SCOP: c.47.1.10, l.1.1.1
</pre><pre>Probab=33.31  E-value=1.4e+02  Score=18.84  Aligned_cols=15  Identities=33%  Similarity=0.559  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+++-.+
</pre><pre>T Consensus       131 ~~~lid~~g~i~~~~  145 (175)
</pre><pre>T 1XVQ_A          131 AIVVIGADGNVAYTE  145 (175)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 177
</pre><pre>>4IFA_A Extracellular protein containing a SCP domain; extracellular protein containing a SCP domain, vaccine candidate, virulence, pathogenesis, Center for Structural Genomics of Infectious; HET: MSE, PEG, TRS; 1.5A {Bacillus anthracis}
</pre><pre>Probab=33.29  E-value=4.2e+02  Score=20.73  Aligned_cols=51  Identities=27%  Similarity=0.336  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceee-cccC-CcEEEEcCCE--EEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVI-PQVG-GTIKYVGERA--VVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~-~q~g-~~~~y~Gk~A--~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      +|-+.+.+++.+-.|.++. ...| ..+.|.-..+  ..|-=.+|+|++.|..+.
</pre><pre>T Consensus        55 iG~~~~~~~~~~G~P~r~~~s~yg~~w~vY~~~~~~y~~vgv~d~~V~~iy~~g~  109 (339)
</pre><pre>T 4IFA_A           55 IGGDSENLLAKWGEPSRIEPSAYGYEWWVYNQDLAQYVQFGVAERKVVTAYVAGE  109 (339)
</pre><pre>T ss_dssp             TTSBHHHHHHHHCSCSEEEECTTSCEEEEECSSGGGCEEEEEETTEEEEEEECCT
</pre><pre>T ss_pred             cCCCHHHHHHHHCCCcEEEecCcCceEEEEcCCcccEEEEEEeCCEEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 178
</pre><pre>>PF13778.8 ; DUF4174 ; Domain of unknown function (DUF4174)
</pre><pre>Probab=33.22  E-value=1.1e+02  Score=18.97  Aligned_cols=15  Identities=7%  Similarity=0.251  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++|++++|+++-.|
</pre><pre>T Consensus        76 ~~~vid~dg~i~~~~   90 (112)
</pre><pre>T F4L4R4_HALH1/2   76 SALLIGKDGTEKLRS   90 (112)
</pre><pre>T ss_pred             EEEEECCCCCcceee
</pre><pre>
</pre><pre>
</pre><pre>No 179
</pre><pre>>6ERI_AK 50S ribosomal protein L14, chloroplastic; Chloroplast, translation, ribosome; HET: MG; 3.0A {Spinacia oleracea}
</pre><pre>Probab=33.19  E-value=1.8e+02  Score=19.22  Aligned_cols=32  Identities=28%  Similarity=0.429  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HhCCceeecccCCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   30 VNNPTKVIPQVGGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        30 v~nP~~v~~q~g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      ++........+|..++|. .++.|++|.+|..+
</pre><pre>T Consensus        63 v~~~k~~~r~~g~~i~f~-~n~~vl~~~~~~p~   94 (121)
</pre><pre>T 6ERI_AK          63 VRTCKELKRDNGMIIRYD-DNAAVIIDQEGNPK   94 (121)
</pre><pre>T ss_dssp             EECSSCEECTTSCEEEES-SCEEEEECSSCCCS
</pre><pre>T ss_pred             hhccceeecCCCCeeEec-CceEEEECCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 180
</pre><pre>>1I5G_A TRYPAREDOXIN II; tryparedoxin, electron transport; HET: TS5; 1.4A {Crithidia fasciculata} SCOP: c.47.1.10
</pre><pre>Probab=33.15  E-value=1.1e+02  Score=18.43  Aligned_cols=16  Identities=31%  Similarity=0.522  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++ |+|+..+.
</pre><pre>T Consensus       111 ~~~l~d~~~g~i~~~~~  127 (144)
</pre><pre>T 1I5G_A          111 TLVGVEADSGNIITTQA  127 (144)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCH
</pre><pre>T ss_pred             EEEEEeCCCCcEEEcCc
</pre><pre>
</pre><pre>
</pre><pre>No 181
</pre><pre>>3FW2_C thiol-disulfide oxidoreductase; structural genomics, APC61456.1, thiol-disulfide oxidoreductase, TlpA-like family, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: ACT, MSE; 1.74A {Bacteroides thetaiotaomicron} SCOP: c.47.1.0, l.1.1.1
</pre><pre>Probab=33.15  E-value=1.1e+02  Score=18.45  Aligned_cols=15  Identities=27%  Similarity=0.572  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       119 ~~~lid~~g~v~~~~  133 (150)
</pre><pre>T 3FW2_C          119 ANILLSSDGKILAKN  133 (150)
</pre><pre>T ss_dssp             EEEEECTTCBEEEES
</pre><pre>T ss_pred             ceEEECCCCcEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 182
</pre><pre>>3FW2_D thiol-disulfide oxidoreductase; structural genomics, APC61456.1, thiol-disulfide oxidoreductase, TlpA-like family, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: MSE, ACT; 1.74A {Bacteroides thetaiotaomicron} SCOP: c.47.1.0, l.1.1.1
</pre><pre>Probab=33.15  E-value=1.1e+02  Score=18.45  Aligned_cols=15  Identities=27%  Similarity=0.572  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.+
</pre><pre>T Consensus       119 ~~~lid~~g~v~~~~  133 (150)
</pre><pre>T 3FW2_D          119 ANILLSSDGKILAKN  133 (150)
</pre><pre>T ss_dssp             EEEEECTTCBEEEES
</pre><pre>T ss_pred             ceEEECCCCcEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 183
</pre><pre>>d2b5xa1 c.47.1.10 (A:1-143) thiol:disulfide oxidoreductase YkuV {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=33.13  E-value=1.6e+02  Score=17.66  Aligned_cols=16  Identities=19%  Similarity=0.287  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++-.+.
</pre><pre>T Consensus       113 ~~~~~d~~g~i~~~~~  128 (143)
</pre><pre>T d2b5xa1         113 AYYVFDKTGQLRHFQA  128 (143)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEEE
</pre><pre>T ss_pred             EEEEEcCCCCEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 184
</pre><pre>>d2v1ma_ c.47.1.0 (A:) automated matches {Blood fluke (Schistosoma mansoni) [TaxId: 6183]}
</pre><pre>Probab=32.79  E-value=1.5e+02  Score=18.20  Aligned_cols=16  Identities=13%  Similarity=0.484  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++-.+.
</pre><pre>T Consensus       129 ~~~lid~~g~~~~~~~  144 (163)
</pre><pre>T d2v1ma_         129 SKFLVDRQGQPVKRYS  144 (163)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             EEEEECCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 185
</pre><pre>>d3drna_ c.47.1.0 (A:) automated matches {Sulfolobus solfataricus [TaxId: 2287]}
</pre><pre>Probab=32.70  E-value=1.6e+02  Score=17.92  Aligned_cols=18  Identities=6%  Similarity=0.349  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ++++++++|+++-.|...
</pre><pre>T Consensus       112 ~~~lid~~g~i~~~~~g~  129 (152)
</pre><pre>T d3drna_         112 ITFVIDKKGIIRHIYNSQ  129 (152)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEECS
</pre><pre>T ss_pred             eEEEECCCCEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 186
</pre><pre>>3KH7_A Thiol:disulfide interchange protein dsbE; TRX-like, thiol-disulfide exchange, Cell inner membrane, Cytochrome c-type biogenesis, Disulfide bond, Redox-active center, Transmembrane, oxidoreductase; 1.75A {Pseudomonas aeruginosa} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=32.59  E-value=1.4e+02  Score=19.37  Aligned_cols=15  Identities=7%  Similarity=0.295  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+|+..+
</pre><pre>T Consensus       135 ~~~lid~~g~v~~~~  149 (176)
</pre><pre>T 3KH7_A          135 ETYLIDKQGIIRHKI  149 (176)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 187
</pre><pre>>d3fw2a1 c.47.1.0 (A:214-360) automated matches {Bacteroides thetaiotaomicron [TaxId: 818]}
</pre><pre>Probab=32.57  E-value=1.3e+02  Score=18.42  Aligned_cols=15  Identities=27%  Similarity=0.572  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.+
</pre><pre>T Consensus       116 ~~~~id~~g~v~~~~  130 (147)
</pre><pre>T d3fw2a1         116 ANILLSSDGKILAKN  130 (147)
</pre><pre>T ss_dssp             EEEEECTTSBEEEES
</pre><pre>T ss_pred             eeEEECCCCcEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 188
</pre><pre>>PF17510.4 ; GP44 ; Gene product 44
</pre><pre>Probab=32.52  E-value=1.4e+02  Score=21.85  Aligned_cols=41  Identities=22%  Similarity=0.409  Sum_probs=0.0  Template_Neff=2.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHhCCceeecccCCcEEEEc--CCEEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   27 LDAVNNPTKVIPQVGGTIKYVG--ERAVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        27 ldAv~nP~~v~~q~g~~~~y~G--k~A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ..|+.-=.++.+-...+++|.|  .+|+|-|... ..+|+|-.|
</pre><pre>T Consensus        28 rkAIelM~eia~VDensVRYk~ate~Ati~lg~~-~~~slWQ~N   70 (108)
</pre><pre>T Q5J5Q8_9CAUD/1   28 DRALAEMEKIAEVDKSSLRFSSASETANISLDDD-RTVCTWQMN   70 (108)
</pre><pre>T ss_pred             HHHHHHHHHHheecccceeeecccceeEEEeCCC-CeEEEeech
</pre><pre>
</pre><pre>
</pre><pre>No 189
</pre><pre>>d4kcea_ c.47.1.10 (A:) automated matches {Leishmania braziliensis [TaxId: 5660]}
</pre><pre>Probab=32.39  E-value=1.4e+02  Score=18.77  Aligned_cols=15  Identities=7%  Similarity=0.381  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.|
</pre><pre>T Consensus       123 ~~~lid~~g~i~~~~  137 (172)
</pre><pre>T d4kcea_         123 GLFVIDKKGTLRHST  137 (172)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             eEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 190
</pre><pre>>1O8X_A TRYPAREDOXIN; TRYPAREDOXIN-I, SYNCHROTRON RADIATION, DISULFIDE BONDS TRYPAREDOXIN, CRITHIDIA FASCICULATA, THIOREDOXIN, TRYPANOSOME, ANOMALOUS DISPERSION, OXIDATIVE STRESS, OXIDOREDUCTASE, ELECTRON; 1.3A {CRITHIDIA FASCICULATA} SCOP: c.47.1.10
</pre><pre>Probab=32.30  E-value=1.2e+02  Score=18.12  Aligned_cols=16  Identities=38%  Similarity=0.572  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++ |+|+..+.
</pre><pre>T Consensus       111 ~~~lid~~~g~v~~~~~  127 (146)
</pre><pre>T 1O8X_A          111 TLIGVDADSGDVVTTRA  127 (146)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCH
</pre><pre>T ss_pred             EEEEEeCCCCcEEeecc
</pre><pre>
</pre><pre>
</pre><pre>No 191
</pre><pre>>3HA9_A uncharacterized Thioredoxin-like protein; thioredoxin-like, aeropyrum, pernix, PSI, MCSG, Structural Genomics, Midwest Center for Structural Genomics, Protein Structure Initiative, unknown; 1.7A {Aeropyrum pernix}
</pre><pre>Probab=32.13  E-value=1.4e+02  Score=18.19  Aligned_cols=16  Identities=19%  Similarity=0.343  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..+.
</pre><pre>T Consensus       133 ~~~lid~~g~v~~~~~  148 (165)
</pre><pre>T 3HA9_A          133 YIVIMDKSSNVLYAGT  148 (165)
</pre><pre>T ss_dssp             EEEEEETTCCEEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 192
</pre><pre>>5XXB_U Ribosomal protein uL14; parasitic ribosome, rRNA, rproteins, RIBOSOME; 3.17A {Toxoplasma gondii}
</pre><pre>Probab=31.92  E-value=1.4e+02  Score=19.55  Aligned_cols=24  Identities=25%  Similarity=0.436  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCEEEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   41 GGTIKYVGERAVVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        41 g~~~~y~Gk~A~VVlN~~GkVVTt   64 (69)
</pre><pre>                      .+.+.+..++++|++|.+|+.+.+
</pre><pre>T Consensus        91 ~g~~~~f~~n~~vl~~~~~~p~g~  114 (139)
</pre><pre>T 5XXB_U           91 EGYFIYFEDNAGVVANNKGEMKGS  114 (139)
</pre><pre>T ss_dssp             TSCEEEESSCEEEEECSSSCBSSS
</pre><pre>T ss_pred             CCCeEEeCCCeEEEECCCCCcccc
</pre><pre>
</pre><pre>
</pre><pre>No 193
</pre><pre>>4GRF_A Putative thiol-disulfide oxidoreductase; OXIDOREDUCTASE, THIOREDOXIN DOMAIN, Structural Genomics; HET: GSH; 1.76A {Bacteroides vulgatus} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=31.89  E-value=1.3e+02  Score=18.27  Aligned_cols=15  Identities=27%  Similarity=0.623  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.+
</pre><pre>T Consensus       112 ~~~lid~~g~i~~~~  126 (152)
</pre><pre>T 4GRF_A          112 HIILVDPEGKIVAKE  126 (152)
</pre><pre>T ss_dssp             EEEEECTTSBEEECC
</pre><pre>T ss_pred             EEEEECCCCCEEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 194
</pre><pre>>3CMI_A Peroxiredoxin HYR1; Thioredoxin-like fold, Oxidoreductase, Peroxidase, Redox-active center; 2.02A {Saccharomyces cerevisiae} SCOP: c.47.1.0
</pre><pre>Probab=31.70  E-value=1.5e+02  Score=18.50  Aligned_cols=16  Identities=19%  Similarity=0.513  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..|.
</pre><pre>T Consensus       136 ~~~lid~~g~i~~~~~  151 (171)
</pre><pre>T 3CMI_A          136 EKFLVDKKGKVYERYS  151 (171)
</pre><pre>T ss_dssp             CEEEECSSSCEEEEEC
</pre><pre>T ss_pred             eEEEECCCCCEEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 195
</pre><pre>>3DWV_A Glutathione peroxidase-like protein; alpha beta, 3-Layer(aba) Sandwich, Glutaredoxin fold, Oxidoreductase, Peroxidase; 1.41A {Trypanosoma brucei} SCOP: c.47.1.0
</pre><pre>Probab=31.69  E-value=1.5e+02  Score=18.61  Aligned_cols=25  Identities=12%  Similarity=0.179  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+.+.+--+.++++++|+++..|.
</pre><pre>T Consensus       142 ~~~~~~~~P~~~lid~~g~i~~~~~  166 (187)
</pre><pre>T 3DWV_A          142 ATKAIKWNFTSFLIDRDGVPVERFS  166 (187)
</pre><pre>T ss_dssp             SSSSCCSTTCEEEECTTSCEEEEEC
</pre><pre>T ss_pred             hcccccccceEEEECCCCCEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 196
</pre><pre>>d2moba_ d.137.1.1 (A:) Soluble methane monooxygenase regulatory protein B {Methylosinus trichosporium [TaxId: 426]}
</pre><pre>Probab=31.61  E-value=3e+02  Score=19.24  Aligned_cols=32  Identities=25%  Similarity=0.412  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHH-------hCCceeecccCCcEEEEcCCEEEE
</pre><pre>Q WP_134219109.1   23 SEAILDAV-------NNPTKVIPQVGGTIKYVGERAVVI   54 (69)
</pre><pre>Q Consensus        23 ~~AildAv-------~nP~~v~~q~g~~~~y~Gk~A~VV   54 (69)
</pre><pre>                      .+||.+|+       .||...+...++-++-.+++-.++
</pre><pre>T Consensus        14 ~~aiveai~~~~~~~dNP~v~V~d~~~yvrI~a~~~l~i   52 (94)
</pre><pre>T d2moba_          14 IDAIIEDIVLKGGKAKNPSIVVEDKAGFWWIKADGAIEI   52 (94)
</pre><pre>T ss_dssp             HHHHCCCCCCTHHHHTSSCCEEEECSSCEEEEESSEEEE
</pre><pre>T ss_pred             HHHHHHHHHhccccccCCCcEEEecCcEEEEEeCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 197
</pre><pre>>PF00238.21 ; Ribosomal_L14 ; Ribosomal protein L14p/L23e
</pre><pre>Probab=31.56  E-value=1.6e+02  Score=19.42  Aligned_cols=30  Identities=17%  Similarity=0.211  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CceeecccCCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   33 PTKVIPQVGGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        33 P~~v~~q~g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      +.+......+.+.....+++|++|.+|+.+
</pre><pre>T Consensus        65 ~k~~~~r~~g~~~~f~~n~~vl~~~~~~~~   94 (122)
</pre><pre>T RL14_CHLAA/1-1   65 TAKEYGRPDGSHIRFDDNAAVLIGKENNPR   94 (122)
</pre><pre>T ss_pred             ccceeCCCCCceeeeCCcceEEeCCCCCcC
</pre><pre>
</pre><pre>
</pre><pre>No 198
</pre><pre>>d1vq8k1 b.39.1.1 (K:1-132) Ribosomal protein L14 {Haloarcula marismortui [TaxId: 2238]}
</pre><pre>Probab=31.45  E-value=1.4e+02  Score=21.44  Aligned_cols=22  Identities=27%  Similarity=0.299  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   41 GGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        41 g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      ++.+-+-.++|+|+||.+|..+
</pre><pre>T Consensus        84 ~G~~i~F~~Na~VLln~k~~p~  105 (132)
</pre><pre>T d1vq8k1          84 DGTRVKFEDNAAVIVDENEDPR  105 (132)
</pre><pre>T ss_dssp             TSCEEEESSCEEEEECTTSCBS
</pre><pre>T ss_pred             CCCEEEeCCCeEEEeCCCCCcC
</pre><pre>
</pre><pre>
</pre><pre>No 199
</pre><pre>>d3iosa_ c.47.1.10 (A:) Lipoprotein DsbF {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=31.38  E-value=1.7e+02  Score=17.24  Aligned_cols=16  Identities=19%  Similarity=0.275  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..++
</pre><pre>T Consensus       104 ~~~~~~~~g~i~~~~~  119 (134)
</pre><pre>T d3iosa_         104 AYAFVDPHGNVDVVRG  119 (134)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEES
</pre><pre>T ss_pred             EEEEECCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 200
</pre><pre>>1JFU_A THIOL:DISULFIDE INTERCHANGE PROTEIN TLPA; THIOREDOXIN-LIKE, DOUBLE DISULFIDE BRIDGE, MEMBRANE PROTEIN; 1.6A {Bradyrhizobium japonicum} SCOP: c.47.1.10
</pre><pre>Probab=31.29  E-value=1.6e+02  Score=18.54  Aligned_cols=15  Identities=20%  Similarity=0.386  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..++++++|+++-.|
</pre><pre>T Consensus       146 ~~~lid~~g~v~~~~  160 (186)
</pre><pre>T 1JFU_A          146 TSVLVDPQGCEIATI  160 (186)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 201
</pre><pre>>3F9U_A Putative exported cytochrome C biogenesis-related protein; exported cytochrome C biogenesis-related protein, Bacteroides fragilis, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for; HET: MSE, NO3; 2.2A {Bacteroides fragilis NCTC 9343}
</pre><pre>Probab=31.17  E-value=1.6e+02  Score=18.83  Aligned_cols=16  Identities=19%  Similarity=0.310  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..+.
</pre><pre>T Consensus       133 ~~~lid~~g~i~~~~~  148 (172)
</pre><pre>T 3F9U_A          133 FYVLIDNEGNPLNKSY  148 (172)
</pre><pre>T ss_dssp             EEEEECTTSCBSSCCB
</pre><pre>T ss_pred             EEEEECCCCCeeeeee
</pre><pre>
</pre><pre>
</pre><pre>No 202
</pre><pre>>2BMX_B ALKYL HYDROPEROXIDASE C; PEROXIREDOXIN, ANTIOXIDANT DEFENSE SYSTEM, MYCOBACTERIUM TUBERCULOSIS, OXIDOREDUCTASE, STRUCTURAL PROTEOMICS IN EUROPE, SPINE, STRUCTURAL GENOMICS; HET: MSE; 2.4A {MYCOBACTERIUM TUBERCULOSIS} SCOP: c.47.1.10
</pre><pre>Probab=31.03  E-value=1.6e+02  Score=18.59  Aligned_cols=16  Identities=13%  Similarity=0.183  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.|.
</pre><pre>T Consensus       134 ~~~lid~~g~i~~~~~  149 (195)
</pre><pre>T 2BMX_B          134 VTFIVDPNNEIQFVSA  149 (195)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 203
</pre><pre>>2BMX_C ALKYL HYDROPEROXIDASE C; PEROXIREDOXIN, ANTIOXIDANT DEFENSE SYSTEM, MYCOBACTERIUM TUBERCULOSIS, OXIDOREDUCTASE, STRUCTURAL PROTEOMICS IN EUROPE, SPINE, STRUCTURAL GENOMICS; HET: MSE; 2.4A {MYCOBACTERIUM TUBERCULOSIS} SCOP: c.47.1.10
</pre><pre>Probab=31.03  E-value=1.6e+02  Score=18.59  Aligned_cols=16  Identities=13%  Similarity=0.183  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.|.
</pre><pre>T Consensus       134 ~~~lid~~g~i~~~~~  149 (195)
</pre><pre>T 2BMX_C          134 VTFIVDPNNEIQFVSA  149 (195)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 204
</pre><pre>>3KCM_E Thioredoxin family protein; SGX, thioredoxin protein, PSI, Structural Genomics, Protein Structure Initiative, New York SGX Research Center for Structural; HET: SO4, MSE; 2.45A {Geobacter metallireducens GS-15} SCOP: c.47.1.0
</pre><pre>Probab=30.94  E-value=1.7e+02  Score=17.78  Aligned_cols=15  Identities=7%  Similarity=0.366  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       109 ~~~lid~~g~i~~~~  123 (154)
</pre><pre>T 3KCM_E          109 ETFVIDRHGVILKKV  123 (154)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 205
</pre><pre>>3KCM_F Thioredoxin family protein; SGX, thioredoxin protein, PSI, Structural Genomics, Protein Structure Initiative, New York SGX Research Center for Structural; HET: MSE, SO4; 2.45A {Geobacter metallireducens GS-15} SCOP: c.47.1.0
</pre><pre>Probab=30.94  E-value=1.7e+02  Score=17.78  Aligned_cols=15  Identities=7%  Similarity=0.366  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       109 ~~~lid~~g~i~~~~  123 (154)
</pre><pre>T 3KCM_F          109 ETFVIDRHGVILKKV  123 (154)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             eEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 206
</pre><pre>>d3gmxa_ d.98.1.0 (A:) automated matches {Streptomyces clavuligerus [TaxId: 1901]}
</pre><pre>Probab=30.79  E-value=3.3e+02  Score=18.74  Aligned_cols=44  Identities=18%  Similarity=0.288  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHhCCceeecccCCcEEEE-----cCCEEEEECCCCCEEE
</pre><pre>Q WP_134219109.1   20 GVSSEAILDAVNNPTKVIPQVGGTIKYV-----GERAVVILNEEGKVVT   63 (69)
</pre><pre>Q Consensus        20 GVs~~AildAv~nP~~v~~q~g~~~~y~-----Gk~A~VVlN~~GkVVT   63 (69)
</pre><pre>                      |.+.+.+...+-.|............|.     |....|.+.++|+++.
</pre><pre>T Consensus        15 G~t~~eV~~~lG~p~~~~~~~~~~~~y~W~~~~~~~~~v~~~~~~~v~~   63 (154)
</pre><pre>T d3gmxa_          15 GMDRTLVWQLAGADQSCSDQVERIICYNNPDHYGPQGHFFFNAADKLIH   63 (154)
</pre><pre>T ss_dssp             TCBHHHHHHHHTGGGTEEECSSCEEEESSSCTTSSEEEEEECTTSBEEE
</pre><pre>T ss_pred             CCCHHHHHHHhCCCceeccccceEEEEEcCCCCCCceEEEEccCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 207
</pre><pre>>2B1K_A Thiol:disulfide interchange protein dsbE; C-terminal thioredoxin-like domain, N-terminal beta-sheet, fingerprint rigion, OXIDOREDUCTASE; 1.9A {Escherichia coli} SCOP: c.47.1.10
</pre><pre>Probab=30.75  E-value=1.7e+02  Score=17.92  Aligned_cols=16  Identities=13%  Similarity=0.275  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.+.
</pre><pre>T Consensus       128 ~~~lid~~g~v~~~~~  143 (168)
</pre><pre>T 2B1K_A          128 ETFLIDGNGIIRYRHA  143 (168)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 208
</pre><pre>>PF11248.10 ; DUF3046 ; Protein of unknown function (DUF3046)
</pre><pre>Probab=30.72  E-value=1.4e+02  Score=19.36  Aligned_cols=17  Identities=29%  Similarity=0.472  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHhcCCCCCCHHHHHHHH
</pre><pre>Q WP_134219109.1   11 NQAISRDGVGVSSEAILDAV   30 (69)
</pre><pre>Q Consensus        11 ~qai~RdG~GVs~~AildAv   30 (69)
</pre><pre>                      +|++.+   ||.|+.|-.||
</pre><pre>T Consensus        37 ~~AL~~---G~~pr~VW~Al   53 (62)
</pre><pre>T F4D0F4_PSEUX/1   37 EEAIEA---GIDPKRVWLAV   53 (62)
</pre><pre>T ss_pred             HHHHHC---CCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 209
</pre><pre>>1Q98_A Thiol Peroxidase; structural genomics, tpx, peroxidase, NYSGXRC, T1429, PSI, Protein Structure Initiative, New York SGX Research Center for; 1.9A {Haemophilus influenzae} SCOP: c.47.1.10
</pre><pre>Probab=30.62  E-value=1.7e+02  Score=18.12  Aligned_cols=15  Identities=33%  Similarity=0.632  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .++++|++|+++-.|
</pre><pre>T Consensus       132 ~~~lid~~g~i~~~~  146 (165)
</pre><pre>T 1Q98_A          132 AVIVLDEQNNVLHSQ  146 (165)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 210
</pre><pre>>d1qmva_ c.47.1.10 (A:) Thioredoxin peroxidase 2 (thioredoxin peroxidase B, 2-cys peroxiredoxin) {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=30.60  E-value=1.5e+02  Score=19.72  Aligned_cols=16  Identities=13%  Similarity=0.262  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .+++++++|+++-.+.
</pre><pre>T Consensus       127 ~~~lid~~G~i~~~~~  142 (197)
</pre><pre>T d1qmva_         127 GLFIIDGKGVLRQITV  142 (197)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 211
</pre><pre>>3HCZ_A Possible thiol-disulfide isomerase; APC61559.2, thiol-disulfide isomerase, Cytophaga hutchinsonii ATCC, structural genomics, PSI-2, protein structure initiative, midwest center for structural; HET: SO4; 1.88A {Cytophaga hutchinsonii} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=30.56  E-value=2.2e+02  Score=16.65  Aligned_cols=23  Identities=9%  Similarity=0.215  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   43 TIKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        43 ~~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .+...+--..+++|++|+++-.+
</pre><pre>T Consensus       106 ~~~~~~~P~~~~id~~g~i~~~~  128 (148)
</pre><pre>T 3HCZ_A          106 TYDIYATPVLYVLDKNKVIIAKR  128 (148)
</pre><pre>T ss_dssp             HHCCCSSCEEEEECTTCBEEEES
</pre><pre>T ss_pred             eeeceecCEEEEECCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 212
</pre><pre>>d3ijfx_ c.97.1.0 (X:) automated matches {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=30.45  E-value=1.5e+02  Score=19.43  Aligned_cols=15  Identities=33%  Similarity=0.525  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.||++++|++++.+
</pre><pre>T Consensus        28 Gavi~~~~g~i~~G~   42 (123)
</pre><pre>T d3ijfx_          28 GAAALVDDGRVVTGC   42 (123)
</pre><pre>T ss_dssp             EEEEEETTSCEEEEE
</pre><pre>T ss_pred             EEEEEeCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 213
</pre><pre>>d1whia_ b.39.1.1 (A:) Ribosomal protein L14 {Bacillus stearothermophilus [TaxId: 1422]}
</pre><pre>Probab=30.29  E-value=1.6e+02  Score=20.76  Aligned_cols=23  Identities=22%  Similarity=0.283  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   40 VGGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        40 ~g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      .++.+-.-.++|+|+||.+|..+
</pre><pre>T Consensus        72 ~dG~~i~F~~N~~VLl~~~~~p~   94 (122)
</pre><pre>T d1whia_          72 PDGSYIRFDENACVIIRDDKSPR   94 (122)
</pre><pre>T ss_dssp             TTSCEECCSSCEEEEECTTSCBS
</pre><pre>T ss_pred             CCCCEEEeCCCeEEEEcCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 214
</pre><pre>>d4grfa1 c.47.1.0 (A:252-389) automated matches {Bacteroides vulgatus [TaxId: 435590]}
</pre><pre>Probab=30.18  E-value=2.1e+02  Score=16.80  Aligned_cols=22  Identities=27%  Similarity=0.441  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCCEEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   44 IKYVGERAVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        44 ~~y~Gk~A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +...+--..+++|++|+++-.+
</pre><pre>T Consensus       102 ~~~~~~P~~~~id~~g~i~~~~  123 (138)
</pre><pre>T d4grfa1         102 YCIVGFPHIILVDPEGKIVAKE  123 (138)
</pre><pre>T ss_dssp             TTCCSSCEEEEECTTSBEEECC
</pre><pre>T ss_pred             CEEeccCEEEEECCCCcEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 215
</pre><pre>>PF05988.14 ; DUF899 ; Bacterial protein of unknown function (DUF899)
</pre><pre>Probab=30.08  E-value=1.8e+02  Score=20.53  Aligned_cols=19  Identities=16%  Similarity=0.373  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      +.+|+|++|+|+-.|....
</pre><pre>T Consensus       177 ~~~lid~~G~i~~~~~~~~  195 (233)
</pre><pre>T Q12HE2_POLSJ/1  177 SLFYKDDAGEVFHTYSTYG  195 (233)
</pre><pre>T ss_pred             EEEEECCCCeEEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 216
</pre><pre>>3DRN_A Peroxiredoxin, bacterioferritin comigratory protein homolog; Peroxiredoxin, Bacterioferritin comigratory protein, OXIDOREDUCTASE; HET: CIT, GOL; 2.15A {Sulfolobus solfataricus} SCOP: c.47.1.0
</pre><pre>Probab=30.06  E-value=1.6e+02  Score=18.43  Aligned_cols=15  Identities=7%  Similarity=0.414  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      .+++++++|+++-.|
</pre><pre>T Consensus       113 ~~~lid~~g~v~~~~  127 (161)
</pre><pre>T 3DRN_A          113 ITFVIDKKGIIRHIY  127 (161)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 217
</pre><pre>>d1prxa_ c.47.1.10 (A:) 1-Cys peroxiredoxin {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=29.93  E-value=1.7e+02  Score=20.47  Aligned_cols=18  Identities=11%  Similarity=0.150  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARN   68 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~   68 (69)
</pre><pre>                      ++++++++|+|+..|...
</pre><pre>T Consensus       129 ~~~lid~~g~v~~~~~~~  146 (220)
</pre><pre>T d1prxa_         129 VVFVFGPDKKLKLSILYP  146 (220)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEECC
</pre><pre>T ss_pred             EEEEECCCCeEEEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 218
</pre><pre>>PF03259.19 ; Robl_LC7 ; Roadblock/LC7 domain
</pre><pre>Probab=29.60  E-value=1.2e+02  Score=17.54  Aligned_cols=15  Identities=27%  Similarity=0.508  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      |-.++++++++|.++
</pre><pre>T Consensus        14 gv~~~~l~~~~G~~i   28 (88)
</pre><pre>T LTOR2_CAEEL/5-   14 GVDGSWLFNKEGLLL   28 (88)
</pre><pre>T ss_pred             CeEEEEEEcCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 219
</pre><pre>>6VPD_B Glutathione peroxidase; glutathione peroxidase, OXIDOREDUCTASE; 2.603A {Hypocrea jecorina (strain QM6a)}
</pre><pre>Probab=29.55  E-value=2.2e+02  Score=17.85  Aligned_cols=28  Identities=21%  Similarity=0.423  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   42 GTIKYVGERAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        42 ~~~~y~Gk~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+...+.-+.+++|++|+++..|....
</pre><pre>T Consensus       147 ~~~~~~~~p~~~lid~~g~i~~~~~~~~  174 (192)
</pre><pre>T 6VPD_B          147 GLERVKWNFEKFLIGRDGKVIGRWASTT  174 (192)
</pre><pre>T ss_dssp             SCCSCCSTTCEEEECTTSSEEEEECTTS
</pre><pre>T ss_pred             CcccccccceEEEECCCCcEEEeccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 220
</pre><pre>>TIGR00385 dsbE; periplasmic protein thiol:disulfide oxidoreductases, DsbE subfamily. Involved in the biogenesis of c-type cytochromes as well as in disulfide bond formation in some periplasmic proteins.
</pre><pre>Probab=29.54  E-value=1.6e+02  Score=18.44  Aligned_cols=16  Identities=13%  Similarity=0.262  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++..+.
</pre><pre>T Consensus       140 ~~~lid~~g~~~~~~~  155 (173)
</pre><pre>T TIGR00385       140 ETFLVDGNGVIRYRHA  155 (173)
</pre><pre>T ss_pred             EEEEECCCCcEEEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 221
</pre><pre>>6ELW_A Phospholipid hydroperoxide glutathione peroxidase, mitochondrial; HUMAN GPX4, PEROXIDASE, SELENOPROTEIN, THIOREDOXIN-FOLD, ANTI-OXIDATVE DEFENSE SYSTEM, OXIDOREDUCTASE; 1.3A {Homo sapiens} SCOP: c.47.1.0
</pre><pre>Probab=29.51  E-value=1.7e+02  Score=19.43  Aligned_cols=16  Identities=19%  Similarity=0.551  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +.+++|++|+++..|.
</pre><pre>T Consensus       161 ~~~lvd~~g~v~~~~~  176 (192)
</pre><pre>T 6ELW_A          161 TKFLIDKNGAVVKRYG  176 (192)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             EEEEECCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 222
</pre><pre>>PF11006.10 ; DUF2845 ; Protein of unknown function (DUF2845)
</pre><pre>Probab=29.46  E-value=2.3e+02  Score=16.56  Aligned_cols=45  Identities=11%  Similarity=0.113  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHhCCceeecccC--------------CcEEEE----cCCEEEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   19 VGVSSEAILDAVNNPTKVIPQVG--------------GTIKYV----GERAVVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        19 ~GVs~~AildAv~nP~~v~~q~g--------------~~~~y~----Gk~A~VVlN~~GkVVTt   64 (69)
</pre><pre>                      .|.+.+++++.+-.|..+.....              ..+.|.    .....|.+. +|+|+..
</pre><pre>T Consensus        14 ~Gms~~~V~~~lG~P~~~~~~~~~~~~~~~~~~~~~~~~w~Y~~~~~~~~~~v~f~-~g~v~~i   76 (79)
</pre><pre>T B3E266_GEOLS/2   14 TGDRLSIVAMKCDPPTYKSARTESEAGYKGATILVSVEEWTYNEGPNRLVHILTFR-NGLLDSV   76 (79)
</pre><pre>T ss_pred             CCCcHHHHHHHCCCCceEeeccccccCcccceEEEEEEEEEEeCCCCCceEEEEEE-CCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 223
</pre><pre>>1WHI_A RIBOSOMAL PROTEIN L14; RIBOSOMAL PROTEIN, RRNA-BINDING; 1.5A {Geobacillus stearothermophilus} SCOP: b.39.1.1
</pre><pre>Probab=29.42  E-value=2e+02  Score=19.16  Aligned_cols=28  Identities=18%  Similarity=0.317  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             CceeecccCCcEEEEcCCEEEEECCCCC
</pre><pre>Q WP_134219109.1   33 PTKVIPQVGGTIKYVGERAVVILNEEGK   60 (69)
</pre><pre>Q Consensus        33 P~~v~~q~g~~~~y~Gk~A~VVlN~~Gk   60 (69)
</pre><pre>                      +.+......+.+.+..++|+|+||.+|.
</pre><pre>T Consensus        65 ~kk~~~r~~g~~~~f~~N~~vll~~~~~   92 (122)
</pre><pre>T 1WHI_A           65 TKRGVRRPDGSYIRFDENACVIIRDDKS   92 (122)
</pre><pre>T ss_dssp             CSSCEECTTSCEECCSSCEEEEECTTSC
</pre><pre>T ss_pred             eCCcccCCCCCeEeeCCceEEeeCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 224
</pre><pre>>PF08894.13 ; DUF1838 ; Protein of unknown function (DUF1838)
</pre><pre>Probab=29.39  E-value=1.8e+02  Score=21.76  Aligned_cols=31  Identities=16%  Similarity=0.320  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             eeecccCCcEEEEcCCEEEEECCC-CCEEEEe
</pre><pre>Q WP_134219109.1   35 KVIPQVGGTIKYVGERAVVILNEE-GKVVTTW   65 (69)
</pre><pre>Q Consensus        35 ~v~~q~g~~~~y~Gk~A~VVlN~~-GkVVTtw   65 (69)
</pre><pre>                      +.....++.+.+......+-.|.+ |+++++|
</pre><pre>T Consensus        34 r~~~~~dg~~~~~~re~~~y~D~~Tgeil~~~   65 (231)
</pre><pre>T K9WMM4_9CYAN/2   34 RFIDNQDGSWDLTSRELSYYLDPITDHPLRTW   65 (231)
</pre><pre>T ss_pred             EEEECCCCCEEEEEEEEEEEECCCCCCcccEE
</pre><pre>
</pre><pre>
</pre><pre>No 225
</pre><pre>>TIGR01067 rplN_bact; ribosomal protein L14, bacterial/organelle. This model distinguishes bacterial and most organellar examples of ribosomal protein L14 from all archaeal and eukaryotic forms.
</pre><pre>Probab=29.33  E-value=1.6e+02  Score=20.81  Aligned_cols=22  Identities=18%  Similarity=0.311  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCEEEEECCCCCEE
</pre><pre>Q WP_134219109.1   41 GGTIKYVGERAVVILNEEGKVV   62 (69)
</pre><pre>Q Consensus        41 g~~~~y~Gk~A~VVlN~~GkVV   62 (69)
</pre><pre>                      .+.+-+-.++|+|+||++|..+
</pre><pre>T Consensus        73 ~G~~i~f~~N~~vlln~~~~p~   94 (122)
</pre><pre>T TIGR01067        73 DGSYIRFDDNACVLINKNKEPR   94 (122)
</pre><pre>T ss_pred             CCCeEEeCCCEEEEEcCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 226
</pre><pre>>d2d30a1 c.97.1.1 (A:1-124) mono-domain cytidine deaminase {Anthrax bacillus (Bacillus anthracis) [TaxId: 1392]}
</pre><pre>Probab=29.28  E-value=1.6e+02  Score=19.21  Aligned_cols=15  Identities=27%  Similarity=0.494  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.||+..+|++++.+
</pre><pre>T Consensus        27 gavi~~~~g~i~~g~   41 (124)
</pre><pre>T d2d30a1          27 GAALLTQDGKVYRGC   41 (124)
</pre><pre>T ss_dssp             EEEEEETTCCEEEEE
</pre><pre>T ss_pred             EEEEEeCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 227
</pre><pre>>PF08923.12 ; MAPKK1_Int ; Mitogen-activated protein kinase kinase 1 interacting
</pre><pre>Probab=29.18  E-value=2.1e+02  Score=17.64  Aligned_cols=22  Identities=18%  Similarity=0.321  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             cCCEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   48 GERAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        48 Gk~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      |-.++++++.+|.++..+....
</pre><pre>T Consensus        16 gv~~~~i~~~dG~~i~~~~~~~   37 (120)
</pre><pre>T Q8WQG5_CAEEL/1   16 GVCAIFITDHDGGLILNIGLPS   37 (120)
</pre><pre>T ss_pred             CeeEEEEEcCCCcEEEEEeCcc
</pre><pre>
</pre><pre>
</pre><pre>No 228
</pre><pre>>3HDC_A Thioredoxin family protein; thioredoxin, Geobacter metallireducens, GS-15, ATCC53774, DSM 7210, 11211i, Structural Genomics, PSI-2, Protein Structure Initiative, New York; HET: MSE; 1.771A {Geobacter metallireducens GS-15} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=28.99  E-value=1.8e+02  Score=18.07  Aligned_cols=16  Identities=13%  Similarity=0.268  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ..+++|++|+++-.|.
</pre><pre>T Consensus       118 ~~~l~d~~g~v~~~~~  133 (158)
</pre><pre>T 3HDC_A          118 DTFIVDRKGIIRQRVT  133 (158)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             eEEEECCCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 229
</pre><pre>>d1lu4a_ c.47.1.10 (A:) Soluble secreted antigen MPT53 {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=28.85  E-value=1.7e+02  Score=17.23  Aligned_cols=15  Identities=20%  Similarity=0.220  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..++++++|+++-.+
</pre><pre>T Consensus       102 ~~~~~~~~g~v~~~~  116 (134)
</pre><pre>T d1lu4a_         102 AFVFYRADGTSTFVN  116 (134)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEC
</pre><pre>T ss_pred             EEEEEcCCCeEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 230
</pre><pre>>4XRA_A Alkyl hydroperoxide reductase subunit C; peroxiredoxin, FF, PrxI, conformation, OXIDOREDUCTASE; HET: CL; 1.75A {Salmonella enterica subsp. enterica serovar Typhimurium} SCOP: c.47.1.10
</pre><pre>Probab=28.76  E-value=2.2e+02  Score=17.95  Aligned_cols=19  Identities=11%  Similarity=0.223  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      .+++++++|+++-.+....
</pre><pre>T Consensus       120 ~~~lid~~g~v~~~~~~~~  138 (186)
</pre><pre>T 4XRA_A          120 ATFVVDPQGIIQAIEVTAE  138 (186)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEEECT
</pre><pre>T ss_pred             EEEEECCCCeEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 231
</pre><pre>>4XRA_E Alkyl hydroperoxide reductase subunit C; peroxiredoxin, FF, PrxI, conformation, OXIDOREDUCTASE; HET: CL; 1.75A {Salmonella enterica subsp. enterica serovar Typhimurium} SCOP: c.47.1.10
</pre><pre>Probab=28.76  E-value=2.2e+02  Score=17.95  Aligned_cols=19  Identities=11%  Similarity=0.223  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      .+++++++|+++-.+....
</pre><pre>T Consensus       120 ~~~lid~~g~v~~~~~~~~  138 (186)
</pre><pre>T 4XRA_E          120 ATFVVDPQGIIQAIEVTAE  138 (186)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEEECT
</pre><pre>T ss_pred             EEEEECCCCeEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 232
</pre><pre>>2P31_B Glutathione peroxidase 7; thioredoxin fold, NPGPX, phospholipid hydroperoxidase, Structural Genomics, Structural Genomics Consortium, SGC, OXIDOREDUCTASE; 2.0A {Homo sapiens} SCOP: c.47.1.0
</pre><pre>Probab=28.66  E-value=1.6e+02  Score=19.14  Aligned_cols=16  Identities=31%  Similarity=0.628  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .+++++++|+++-.|.
</pre><pre>T Consensus       149 ~~~lid~~g~i~~~~~  164 (181)
</pre><pre>T 2P31_B          149 WKYLVAPDGKVVGAWD  164 (181)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             eEEEECCCCcEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 233
</pre><pre>>2C0D_B THIOREDOXIN PEROXIDASE 2; PEROXIREDOXIN, 2-CYS, PEROXIDASE, THIOREDOXIN DEPENDANT, MITOCHONDRIAL, ANTIOXIDANT, OXIDOREDUCTASE, REDOX-ACTIVE CENTER; 1.78A {PLASMODIUM FALCIPARUM} SCOP: c.47.1.0
</pre><pre>Probab=28.43  E-value=1.7e+02  Score=19.68  Aligned_cols=15  Identities=20%  Similarity=0.333  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+++-.+
</pre><pre>T Consensus       148 ~~~lid~~G~i~~~~  162 (221)
</pre><pre>T 2C0D_B          148 GLFIIDKNGCVRHQT  162 (221)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 234
</pre><pre>>d2v2ga_ c.47.1.10 (A:) automated matches {Arenicola marina [TaxId: 6344]}
</pre><pre>Probab=28.42  E-value=1.7e+02  Score=20.49  Aligned_cols=16  Identities=25%  Similarity=0.308  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+|+..+.
</pre><pre>T Consensus       128 ~~~lid~~g~i~~~~~  143 (220)
</pre><pre>T d2v2ga_         128 AVFIIGPDKKLKLSIL  143 (220)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 235
</pre><pre>>PF18810.3 ; PBECR2 ; phage-Barnase-EndoU-ColicinE5/D-RelE like nuclease2
</pre><pre>Probab=28.36  E-value=3.1e+02  Score=17.75  Aligned_cols=46  Identities=11%  Similarity=0.146  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCceeecccC-Cc-----------EEEEcCC-------EEEEECCCCC---EEEEeecCC
</pre><pre>Q WP_134219109.1   24 EAILDAVNNPTKVIPQVG-GT-----------IKYVGER-------AVVILNEEGK---VVTTWARNR   69 (69)
</pre><pre>Q Consensus        24 ~AildAv~nP~~v~~q~g-~~-----------~~y~Gk~-------A~VVlN~~Gk---VVTtw~~~~   69 (69)
</pre><pre>                      ..+.+++++|..|..... ..           ++|...-       .+|+++....   ++|+|..+.
</pre><pre>T Consensus        50 ~~l~~~l~~PdeIw~~~~~~~~~~~~~~~~~~~~yik~~~~~~~~~~~vvv~~~~~~~~~~t~~~~~~  117 (132)
</pre><pre>T A0A0A0DCS5_9PR   50 LALAMAILDPDEIWADWQNIAPKGAAPRWRLRRRYIRRFDMDGRKGGIATFGWMDQGWSGATAFPPDT  117 (132)
</pre><pre>T ss_pred             HHHHHHhhCchhhEeehhhcCCCCCCceEEEEEEEEEEEecCCCeEEEEEEEEcCCeEEEEEeeCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 236
</pre><pre>>d5k2ia1 c.47.1.0 (A:1-153) automated matches {Vibrio vulnificus [TaxId: 914127]}
</pre><pre>Probab=28.30  E-value=1.5e+02  Score=18.76  Aligned_cols=14  Identities=29%  Similarity=0.366  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++| +|+|+..|
</pre><pre>T Consensus       125 ~~~lid-~g~i~~~~  138 (153)
</pre><pre>T d5k2ia1         125 YAMIID-NGVVTTLN  138 (153)
</pre><pre>T ss_dssp             EEEEEE-TTEEEEEE
</pre><pre>T ss_pred             EEEEEe-CCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 237
</pre><pre>>5XY3_V 60S ribosomal protein L23, putative; Trichomonas vaginalis ribosome, rRNA, rprotein, RIBOSOME; 3.2A {Trichomonas vaginalis}
</pre><pre>Probab=28.22  E-value=1.8e+02  Score=19.04  Aligned_cols=24  Identities=25%  Similarity=0.430  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCEEEEECCCCCEEEE
</pre><pre>Q WP_134219109.1   41 GGTIKYVGERAVVILNEEGKVVTT   64 (69)
</pre><pre>Q Consensus        41 g~~~~y~Gk~A~VVlN~~GkVVTt   64 (69)
</pre><pre>                      .+.+....++++|++|.+|+.+.+
</pre><pre>T Consensus        92 ~g~~~~f~~n~~vl~~~~~~~~gt  115 (140)
</pre><pre>T 5XY3_V           92 DGVVIGFQDNAGVIINDKGEMKGS  115 (140)
</pre><pre>T ss_dssp             TSCEEEESSCEEEEECSSSCBSSS
</pre><pre>T ss_pred             CCcEEecCCCeEEEeCCCCCcccc
</pre><pre>
</pre><pre>
</pre><pre>No 238
</pre><pre>>d4llra_ c.47.1.10 (A:) Tryparedoxin peroxidase (thioredoxin peroxidase homologue) {Trypanosoma cruzi [TaxId: 5693]}
</pre><pre>Probab=28.10  E-value=2e+02  Score=19.17  Aligned_cols=16  Identities=6%  Similarity=0.163  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      +++++|++|+++..+.
</pre><pre>T Consensus       126 ~~~lid~~g~v~~~~~  141 (194)
</pre><pre>T d4llra_         126 GLFIIDPKQNLRQITV  141 (194)
</pre><pre>T ss_dssp             EEEEECTTCBEEEEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 239
</pre><pre>>5Y63_E Alkyl hydroperoxide reductase, C subunit; Oxidoreductase, 2Cys peroxiredoxins; 2.87A {Enterococcus faecalis (strain ATCC 700802 / V583)} SCOP: c.47.1.0
</pre><pre>Probab=28.00  E-value=1.9e+02  Score=18.10  Aligned_cols=15  Identities=27%  Similarity=0.337  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++..+
</pre><pre>T Consensus       132 ~~~lid~~g~i~~~~  146 (183)
</pre><pre>T 5Y63_E          132 ASFIVSPEGDIKSYE  146 (183)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 240
</pre><pre>>7KIZ_B Peroxiredoxin-2; hydrogen peroxide, OXIDOREDUCTASE; 1.7A {Homo sapiens}
</pre><pre>Probab=27.94  E-value=1.8e+02  Score=18.68  Aligned_cols=15  Identities=13%  Similarity=0.277  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.|
</pre><pre>T Consensus       127 ~~~lid~~g~i~~~~  141 (197)
</pre><pre>T 7KIZ_B          127 GLFIIDGKGVLRQIT  141 (197)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEEcCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 241
</pre><pre>>7KIZ_G Peroxiredoxin-2; hydrogen peroxide, OXIDOREDUCTASE; 1.7A {Homo sapiens}
</pre><pre>Probab=27.94  E-value=1.8e+02  Score=18.68  Aligned_cols=15  Identities=13%  Similarity=0.277  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +.+++|++|+++-.|
</pre><pre>T Consensus       127 ~~~lid~~g~i~~~~  141 (197)
</pre><pre>T 7KIZ_G          127 GLFIIDGKGVLRQIT  141 (197)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEEcCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 242
</pre><pre>>5EPF_A Peroxiredoxin; Mycobacterium tuberculosis, Peroxidoxin, BcpB, Structural Genomics, Seattle Structural Genomics Center for Infectious Disease, SSGCID, OXIDOREDUCTASE; HET: SO4; 1.35A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)} SCOP: l.1.1.1, c.47.1.0
</pre><pre>Probab=27.71  E-value=2e+02  Score=18.03  Aligned_cols=16  Identities=13%  Similarity=0.378  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      .++++|++|+++..+.
</pre><pre>T Consensus       125 ~~~~id~~g~i~~~~~  140 (162)
</pre><pre>T 5EPF_A          125 TTFVIDTDRKVLDVIS  140 (162)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEC
</pre><pre>T ss_pred             EEEEECCCCcEEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 243
</pre><pre>>6GXG_B Tryparedoxin; Covalent Inhibitor, Photoreduction, Inhibitor-Induced Dimerization, Monomer-Dimer Mixture, OXIDOREDUCTASE; HET: GOL, FFN; 1.6A {Trypanosoma brucei brucei} SCOP: l.1.1.1, c.47.1.10
</pre><pre>Probab=27.60  E-value=1.5e+02  Score=17.67  Aligned_cols=19  Identities=32%  Similarity=0.660  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++ |+|+..+....
</pre><pre>T Consensus       114 ~~~lid~~~g~v~~~~~~~~  133 (147)
</pre><pre>T 6GXG_B          114 TLITINADTGAIIGTQARTR  133 (147)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCHHHH
</pre><pre>T ss_pred             EEEEEeCCCCceecccccch
</pre><pre>
</pre><pre>
</pre><pre>No 244
</pre><pre>>6GXG_C Tryparedoxin; Covalent Inhibitor, Photoreduction, Inhibitor-Induced Dimerization, Monomer-Dimer Mixture, OXIDOREDUCTASE; HET: GOL, FFN; 1.6A {Trypanosoma brucei brucei} SCOP: c.47.1.10, l.1.1.1
</pre><pre>Probab=27.60  E-value=1.5e+02  Score=17.67  Aligned_cols=19  Identities=32%  Similarity=0.660  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++ |+|+..+....
</pre><pre>T Consensus       114 ~~~lid~~~g~v~~~~~~~~  133 (147)
</pre><pre>T 6GXG_C          114 TLITINADTGAIIGTQARTR  133 (147)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCHHHH
</pre><pre>T ss_pred             EEEEEeCCCCceecccccch
</pre><pre>
</pre><pre>
</pre><pre>No 245
</pre><pre>>d3tb2a_ c.47.1.10 (A:) 1-Cys peroxiredoxin {Plasmodium yoelii yoelii [TaxId: 73239]}
</pre><pre>Probab=27.41  E-value=1.8e+02  Score=20.20  Aligned_cols=15  Identities=20%  Similarity=0.317  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      +++++|++|+|+..|
</pre><pre>T Consensus       129 ~~~lid~~g~i~~~~  143 (219)
</pre><pre>T d3tb2a_         129 CLFFISPEKKIKATV  143 (219)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 246
</pre><pre>>6UDG_C Thiol peroxidase; SSGCID, Structural Genomics, Elizabethkingia anophelis, tpx, Thiol peroxidase, Seattle Structural Genomics Center for Infectious Disease, OXIDOREDUCTASE; 2.65A {Elizabethkingia anophelis NUHP1} SCOP: c.47.1.0
</pre><pre>Probab=27.29  E-value=2e+02  Score=18.32  Aligned_cols=15  Identities=53%  Similarity=0.630  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEe
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTW   65 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw   65 (69)
</pre><pre>                      ..+++|++|+++-.|
</pre><pre>T Consensus       140 ~~~lid~~g~v~~~~  154 (173)
</pre><pre>T 6UDG_C          140 AVVVADENGNVVYTE  154 (173)
</pre><pre>T ss_dssp             EEEEECSSCBEEEEE
</pre><pre>T ss_pred             EEEEECCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 247
</pre><pre>>d1o8xa_ c.47.1.10 (A:) Tryparedoxin I {Crithidia fasciculata [TaxId: 5656]}
</pre><pre>Probab=27.15  E-value=1.5e+02  Score=18.12  Aligned_cols=16  Identities=38%  Similarity=0.572  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCC-CCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEE-GKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~-GkVVTtw~   66 (69)
</pre><pre>                      ..++++++ |+++..++
</pre><pre>T Consensus       109 ~~~~id~~~g~i~~~~~  125 (144)
</pre><pre>T d1o8xa_         109 TLIGVDADSGDVVTTRA  125 (144)
</pre><pre>T ss_dssp             EEEEEETTTCCEEESCH
</pre><pre>T ss_pred             EEEEEcCCCCcEEecch
</pre><pre>
</pre><pre>
</pre><pre>No 248
</pre><pre>>2JU5_A Thioredoxin Disulfide Isomerase; protein, oxidoreductase; NMR {Chlamydophila pneumoniae}
</pre><pre>Probab=26.97  E-value=2.5e+02  Score=17.43  Aligned_cols=19  Identities=21%  Similarity=0.380  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      ..+++|++|+++..++...
</pre><pre>T Consensus       119 ~~~~~d~~g~~~~~~~~~~  137 (154)
</pre><pre>T 2JU5_A          119 ELVFIDAEGKQLARMGFEP  137 (154)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEECCCT
</pre><pre>T ss_pred             EEEEEcCCCCEEEeeecCC
</pre><pre>
</pre><pre>
</pre><pre>No 249
</pre><pre>>2R37_B Glutathione peroxidase 3; GLUTATHIONE, PEROXIDASE, PLASMA, STRUCTURAL GENOMICS CONSORTIUM, SGC, Oxidoreductase, Secreted, Selenium, Selenocysteine; 1.85A {Homo sapiens} SCOP: c.47.1.0
</pre><pre>Probab=26.67  E-value=1.7e+02  Score=20.29  Aligned_cols=16  Identities=13%  Similarity=0.501  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCCCEEEEee
</pre><pre>Q WP_134219109.1   51 AVVILNEEGKVVTTWA   66 (69)
</pre><pre>Q Consensus        51 A~VVlN~~GkVVTtw~   66 (69)
</pre><pre>                      ++++++++|+++..|.
</pre><pre>T Consensus       161 ~~~lid~~G~i~~~~~  176 (207)
</pre><pre>T 2R37_B          161 EKFLVGPDGIPIMRWH  176 (207)
</pre><pre>T ss_dssp             CEEEECTTSCEEEEEC
</pre><pre>T ss_pred             EEEeeCCCCCEeEecC
</pre><pre>
</pre><pre>
</pre><pre>No 250
</pre><pre>>6MG6_C Carbon-nitrogen hydrolase; SSGCID, Structural Genomics, Helicobacter pylori, CN-hydrolase, Carbon-Nitrogen hydrolase, Seattle Structural Genomics Center for Infectious Disease, HYDROLASE; 2.1A {Helicobacter pylori (strain G27)} SCOP: d.160.1.0, l.1.1.1
</pre><pre>Probab=26.23  E-value=2e+02  Score=20.84  Aligned_cols=20  Identities=10%  Similarity=0.471  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CEEEEECCCCCEEEEeecCC
</pre><pre>Q WP_134219109.1   50 RAVVILNEEGKVVTTWARNR   69 (69)
</pre><pre>Q Consensus        50 ~A~VVlN~~GkVVTtw~~~~   69 (69)
</pre><pre>                      +..++++++|+++..+.|..
</pre><pre>T Consensus       106 n~~~~i~~~G~~i~~~~K~~  125 (300)
</pre><pre>T 6MG6_C          106 NSAVVFEKDGSIAGVYRKMH  125 (300)
</pre><pre>T ss_dssp             EEEEEECTTSCEEEEEECCS
</pre><pre>T ss_pred             EEEEEEcCCCCEEEEEEccc
</pre><pre>
</pre><pre>
</pre>