<pre>Query         10K_MODEL_38 
</pre><pre>Match_columns 96
</pre><pre>No_of_seqs    46 out of 50
</pre><pre>Neff          3.53066
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 19:39:24 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/6465375.hhr -oa3m ../results/6465375.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF15633.8 ; Tox-ART-HYD1 ; HYD  99.2 1.8E-11 1.7E-16   80.3   4.0   83    1-85      2-99  (101)
</pre><pre>  2 PF18096.3 ; Thump_like ; THUMP  75.0      14 0.00013   23.9   3.6   29   27-56     26-54  (71)
</pre><pre>  3 PF19841.1 ; GldN ; Gliding mot  52.1   2E+02  0.0019   23.1   6.7   67    9-81     44-127 (227)
</pre><pre>  4 PF14467.8 ; DUF4426 ; Domain o  49.0      50 0.00046   24.2   2.9   31   25-56      8-38  (122)
</pre><pre>  5 PF15545.8 ; Ntox8 ; Bacterial   44.0     9.7   9E-05   25.5  -1.2   68    3-71      1-68  (70)
</pre><pre>  6 2JT1_A PefI protein; solution   41.1      27 0.00025   22.5   0.5   47   33-79     22-69  (77)
</pre><pre>  7 d1mvfd_ b.129.1.1 (D:) MazE {E  40.8      72 0.00067   17.5   2.1   29   38-69     15-43  (44)
</pre><pre>  8 3O27_B Putative uncharacterize  38.4      95 0.00088   19.0   2.6   37   38-77     30-68  (68)
</pre><pre>  9 d2fy9a1 b.129.1.3 (A:1-54) Put  34.8 1.3E+02  0.0012   17.6   2.7   32   38-72     19-50  (54)
</pre><pre> 10 3UC2_D hypothetical protein wi  34.1      59 0.00054   25.3   1.4   19   38-56     41-59  (139)
</pre><pre> 11 6T7O_A Ribosome hibernation pr  33.9 1.6E+02  0.0015   19.5   3.2   46   26-71      5-50  (67)
</pre><pre> 12 PF05930.14 ; Phage_AlpA ; Prop  33.8      85 0.00079   15.2   1.6   18   33-50      1-18  (52)
</pre><pre> 13 PF12298.10 ; Bot1p ; Eukaryoti  33.5 1.1E+02   0.001   24.7   2.8   23   27-49    112-134 (171)
</pre><pre> 14 PF12324.10 ; HTH_15 ; Helix-tu  32.4      70 0.00065   22.2   1.4   21   31-51     31-51  (74)
</pre><pre> 15 TIGR03523 GldN; gliding motili  29.9 4.3E+02   0.004   23.0   5.7   72    9-81     94-173 (280)
</pre><pre> 16 TIGR01764 excise; DNA binding   29.9 1.1E+02   0.001   15.0   1.6   14   36-49      2-15  (49)
</pre><pre> 17 d1wi9a1 a.4.5.47 (A:105-163) H  29.0      91 0.00085   19.0   1.4   19   32-50     11-29  (59)
</pre><pre> 18 d3ml1a2 b.52.2.0 (A:682-802) a  28.7      93 0.00086   20.9   1.5   14   36-50     41-54  (121)
</pre><pre> 19 d3f0pa1 a.4.5.79 (A:1-80) Alky  28.6      69 0.00064   23.3   0.9   29   31-59     32-66  (80)
</pre><pre> 20 PF14584.8 ; DUF4446 ; Protein   28.0 1.9E+02  0.0018   22.8   3.2   25   18-42    118-146 (152)
</pre><pre> 21 d1b8mb_ g.17.1.3 (B:) Neurotro  27.6 1.2E+02  0.0012   22.9   2.1   38   57-96      2-50  (119)
</pre><pre> 22 PF20063.1 ; DUF6462 ; Family o  26.2 1.2E+02  0.0012   14.6   1.4   18   33-50      1-18  (51)
</pre><pre> 23 3MUD_B DNA repair protein XRCC  25.9   3E+02  0.0028   20.3   3.8   67   17-86     36-117 (175)
</pre><pre> 24 PF13411.8 ; MerR_1 ; MerR HTH   25.5 1.2E+02  0.0011   16.7   1.4   15   36-50      1-15  (69)
</pre><pre> 25 3K2T_A Lmo2511 protein; Lister  25.4 2.7E+02  0.0025   17.4   3.0   46   26-71      3-48  (57)
</pre><pre> 26 7OMV_AAA DarT domain-containin  25.3      95 0.00088   24.1   1.2   22    2-23     14-35  (210)
</pre><pre> 27 d1yfba_ b.129.1.3 (A:) Transcr  25.0 2.5E+02  0.0023   16.2   2.7   32   38-72     20-51  (52)
</pre><pre> 28 d1bl0a1 a.4.1.8 (A:9-62) MarA   24.3 1.8E+02  0.0017   15.6   1.9   17   33-49     17-33  (54)
</pre><pre> 29 PF15689.7 ; NRIP1_repr_3 ; Nuc  24.2      52 0.00048   24.3  -0.3   16   32-47     51-67  (89)
</pre><pre> 30 5MRC_66 mS45; yeast, mitochond  23.4 2.1E+02  0.0019   25.7   3.0   24   26-49    217-240 (319)
</pre><pre> 31 3I71_B Ethanolamine utilizatio  23.1      46 0.00043   22.7  -0.7   28   27-55     11-38  (68)
</pre><pre> 32 7M3P_B Xrcc4-Spc110p(164-207);  23.1 4.1E+02  0.0038   19.8   4.0   66   18-86     41-121 (183)
</pre><pre> 33 5MYJ_A Ribosome hibernation pr  22.6 1.8E+02  0.0016   21.9   2.1   38   34-71    139-176 (185)
</pre><pre> 34 d1ti6a1 b.52.2.2 (A:729-875) T  22.5 1.3E+02  0.0012   21.3   1.4   14   36-50     46-59  (147)
</pre><pre> 35 TIGR02609 doc_partner; putativ  22.3 2.7E+02  0.0025   17.9   2.7   29   38-69     15-43  (74)
</pre><pre> 36 PF16321.7 ; Ribosom_S30AE_C ;   22.3 2.9E+02  0.0027   16.9   2.7   46   26-71      7-52  (57)
</pre><pre> 37 1WFX_A Probable RNA 2'-phospho  22.1 3.3E+02  0.0031   20.8   3.5   53    2-62     96-148 (186)
</pre><pre> 38 PF13670.8 ; PepSY_2 ; Peptidas  21.9   4E+02  0.0037   16.7   4.3   51   31-82     22-73  (81)
</pre><pre> 39 PF03333.15 ; PapB ; Adhesin bi  21.6 1.4E+02  0.0013   21.0   1.4   15   36-50     56-70  (93)
</pre><pre> 40 PF07453.15 ; NUMOD1 ; NUMOD1 d  21.6 1.6E+02  0.0015   15.7   1.3   14   37-50     21-34  (40)
</pre><pre> 41 6G1T_A AM32; Repressor, Protei  21.6 1.2E+02  0.0011   22.4   1.1   15   36-50     64-78  (122)
</pre><pre> 42 TIGR01439 lp_hng_hel_AbrB; loo  21.4 3.2E+02   0.003   15.4   2.9   31   38-71     12-43  (43)
</pre><pre> 43 1YFB_A Transition state regula  21.2 3.7E+02  0.0034   16.0   3.2   44   23-69     12-55  (59)
</pre><pre> 44 1YSF_B Transition state regula  21.2 3.7E+02  0.0034   16.0   3.2   44   23-69     12-55  (59)
</pre><pre> 45 PF04936.14 ; DUF658 ; Protein   21.1 1.5E+02  0.0014   22.7   1.6   25   25-49      4-28  (186)
</pre><pre> 46 3M8J_B FocB protein; All-alpha  21.0 1.4E+02  0.0013   22.1   1.4   15   36-50     61-75  (111)
</pre><pre> 47 PF12728.9 ; HTH_17 ; Helix-tur  20.9 1.9E+02  0.0018   14.5   1.5   14   36-49      2-15  (57)
</pre><pre> 48 PF07180.13 ; CaiF_GrlA ; CaiF/  20.7      86 0.00079   24.4   0.2   25   32-56     35-59  (137)
</pre><pre> 49 PF08845.12 ; SymE_toxin ; Toxi  20.5 2.1E+02  0.0019   16.7   1.8   29   38-69     23-51  (51)
</pre><pre> 50 TIGR00156 TIGR00156; TIGR00156  20.4 1.8E+02  0.0017   21.4   1.8   44   45-92     75-119 (126)
</pre><pre> 51 6ZHX_K Chromodomain-helicase-D  20.3 1.3E+02  0.0012   18.6   0.8    9   32-40     28-36  (37)
</pre><pre> 52 PF06394.15 ; Pepsin-I3 ; Pepsi  20.1 1.9E+02  0.0017   20.0   1.7   21   23-43     26-46  (74)
</pre><pre> 53 6GZZ_W4 Ribosome hibernation p  20.1 3.7E+02  0.0034   16.6   2.9   46   26-71      4-49  (57)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF15633.8 ; Tox-ART-HYD1 ; HYD1 signature containing ADP-ribosyltransferase
</pre><pre>Probab=99.24  E-value=1.8e-11  Score=80.25  Aligned_cols=83  Identities=17%  Similarity=0.169  Sum_probs=62.1  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCccchhhHHHHHhcCceeecc-----------cCeeEEEEcCCC--cCCHHHHHHHhCCCcccccccEEeecCHHHeEE
</pre><pre>Q 10K_MODEL_38      1 RHYTSNQGFAAIKESMKILAGD-----------QNAVFAVRAKGK--PLSMADAADKFKIKQNHARNYIDFDIDTNRVEF   67 (96)
</pre><pre>Q Consensus         1 RHYTn~kg~~gI~e~~~I~A~D-----------qNrVf~e~Ak~K--plSp~dae~kygI~~Grgr~yVEfDv~~~~~e~   67 (96)
</pre><pre>                      +|||+..|+++|+++|.|+++.           ++..|+..+...  +++..+..-..++..++...|||||++.  +++
</pre><pre>T Consensus         2 yHYT~~~g~~~I~~~g~i~~s~~~~~~~~~~~g~g~y~t~~~p~~~~~~~~~~~~~~~~~~~~k~~~yv~~~v~~--l~~   79 (101)
</pre><pre>T G8M247_HUNCD/5    2 YHYTDEVGLNGILNSKKLKPSLKAVNPKDARYGNGQYLSDIIPGTKTPGQLSKQFINNPFQGKRFTYYIEIDVTD--LNI   79 (101)
</pre><pre>T ss_pred             ceecChhhHHHHHcCCCCCccccccCcCccccCCCEEecCCCCCCCCchhHHhhhhcCcccCCceEEEEEEECcC--Cce
</pre><pre>Confidence            5999999999999999999664           554455444444  5555555555678889999999999988  888
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EECCC-CCeEE-EEEcceec
</pre><pre>Q 10K_MODEL_38     68 RKNDL-GVEEY-KIKGDIEL   85 (96)
</pre><pre>Q Consensus        68 ~~Np~-~~~E~-tvkGdv~L   85 (96)
</pre><pre>                      ++++. ...++ ++.+|+.+
</pre><pre>T Consensus        80 ~~~~~~~~~~~~~~~~di~~   99 (101)
</pre><pre>T G8M247_HUNCD/5   80 IKGRDGVYVVLGEEPLDLTN   99 (101)
</pre><pre>T ss_pred             eeCCCceEEecCCCcCCCcc
</pre><pre>Confidence            88877 44444 77777655
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF18096.3 ; Thump_like ; THUMP domain-like
</pre><pre>Probab=75.04  E-value=14  Score=23.85  Aligned_cols=29  Identities=21%  Similarity=0.415  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCCCcCCHHHHHHHhCCCcccccccE
</pre><pre>Q 10K_MODEL_38     27 FAVRAKGKPLSMADAADKFKIKQNHARNYI   56 (96)
</pre><pre>Q Consensus        27 f~e~Ak~KplSp~dae~kygI~~Grgr~yV   56 (96)
</pre><pre>                      ....+++-|++|.+...+|+++.|. ..++
</pre><pre>T Consensus        26 ~~V~~R~~p~~~~~lrk~l~~~~g~-~~~l   54 (71)
</pre><pre>T K2Q3D0_9FLAO/3   26 ANITSRNFPLTVDKLRSQLKIKDGG-SLYL   54 (71)
</pre><pre>T ss_pred             eEEEECCCCCCHHHHHHHhCCCCCC-cEEE
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>PF19841.1 ; GldN ; Gliding motility associated protein GldN
</pre><pre>Probab=52.08  E-value=2e+02  Score=23.07  Aligned_cols=67  Identities=12%  Similarity=0.146  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhcCcee-ecccCeeEEEEcCCCcCCHHHHHHHhCCCccc--------ccccEEeecCHHHeE-------EEECCC
</pre><pre>Q 10K_MODEL_38      9 FAAIKESMKIL-AGDQNAVFAVRAKGKPLSMADAADKFKIKQNH--------ARNYIDFDIDTNRVE-------FRKNDL   72 (96)
</pre><pre>Q Consensus         9 ~~gI~e~~~I~-A~DqNrVf~e~Ak~KplSp~dae~kygI~~Gr--------gr~yVEfDv~~~~~e-------~~~Np~   72 (96)
</pre><pre>                      +++|..+|.|. |.+++. |.     +|||+.++.+.+++..--        +...+.-+++++++.       |..+..
</pre><pre>T Consensus        44 ~~~i~~~g~i~~aY~~d~-f~-----~~ls~~ei~~~l~~~~~~~~~~d~~~~~~~~~~~~~~~~i~~~~ikEdw~fDk~  117 (227)
</pre><pre>T A0A085L0Z8_9FL   44 IDGILKEATIVDVFGDDR-FE-----IPYTSDEVSRYVRTIDTIFNPDDPSKIDLIDTIEIKSKDVIAYYIKSDWYFDKQ  117 (227)
</pre><pre>T ss_pred             HHHHHHhCCceeccCCCC-cC-----ccCCHHHHHHHhcccccccCCCCCccCceeeeeecCHHHEEEEEEEEEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             -CCeEEEEEc
</pre><pre>Q 10K_MODEL_38     73 -GVEEYKIKG   81 (96)
</pre><pre>Q Consensus        73 -~~~E~tvkG   81 (96)
</pre><pre>                       +..+..|.|
</pre><pre>T Consensus       118 ~~~~~~rIig  127 (227)
</pre><pre>T A0A085L0Z8_9FL  118 RGELKNRIIA  127 (227)
</pre><pre>T ss_pred             CCeEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>PF14467.8 ; DUF4426 ; Domain of unknown function (DUF4426)
</pre><pre>Probab=48.97  E-value=50  Score=24.19  Aligned_cols=31  Identities=6%  Similarity=0.312  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             eeEEEEcCCCcCCHHHHHHHhCCCcccccccE
</pre><pre>Q 10K_MODEL_38     25 AVFAVRAKGKPLSMADAADKFKIKQNHARNYI   56 (96)
</pre><pre>Q Consensus        25 rVf~e~Ak~KplSp~dae~kygI~~Grgr~yV   56 (96)
</pre><pre>                      .||.--....-|+| ++|..|||.+++.+..|
</pre><pre>T Consensus         8 ~v~y~~~~s~~L~~-~iA~~~gi~r~~~~~~l   38 (122)
</pre><pre>T Q2S8L8_HAHCH/3    8 EIHYIALPTTILDP-EVATRYELPRSKSTGFV   38 (122)
</pre><pre>T ss_pred             EEEEEEeeHhhcCH-HHHHHcCCCCCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF15545.8 ; Ntox8 ; Bacterial toxin 8
</pre><pre>Probab=43.97  E-value=9.7  Score=25.47  Aligned_cols=68  Identities=7%  Similarity=-0.060  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             ccchhhHHHHHhcCceeecccCeeEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38      3 YTSNQGFAAIKESMKILAGDQNAVFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus         3 YTn~kg~~gI~e~~~I~A~DqNrVf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np   71 (96)
</pre><pre>                      +.+......+.. |.|+...++........-+..+.-+.|-.-|+...+|..|..+|+...++....+.
</pre><pre>T Consensus         1 l~~d~~~~~~~r-gwik~~~~~~~~g~~k~~r~P~g~~lah~rg~ea~kG~~y~~s~lq~~~~h~~qh~   68 (70)
</pre><pre>T A0Q1W6_CLONN/2    1 LVKDLKLSRSLR-GELKRDINLIKKGRRRTIRVPKGYELAHRRGFEARKGYGYAYSDLQVIRNHRIQHR   68 (70)
</pre><pre>T ss_pred             CCCccccchhhc-ceehHHHhhHhcCCCccccCCCccccccccCchhhcCCCccccccchhHHHcCccc
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>2JT1_A PefI protein; solution NMR structure, winged helix-turn-helix, transcription regulatory protein, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural; NMR {Salmonella typhimurium LT2}
</pre><pre>Probab=41.11  E-value=27  Score=22.50  Aligned_cols=47  Identities=19%  Similarity=0.225  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECCC-CCeEEEE
</pre><pre>Q 10K_MODEL_38     33 GKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKNDL-GVEEYKI   79 (96)
</pre><pre>Q Consensus        33 ~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np~-~~~E~tv   79 (96)
</pre><pre>                      ..++|..|.++++|+.+...+.++..=+....|+...+.. ....|+.
</pre><pre>T Consensus        22 ~~~~t~~eIa~~lgis~~tv~~~L~~L~~~g~I~~~~~~~~~~~~~~~   69 (77)
</pre><pre>T 2JT1_A           22 GAPVKTRDIADAAGLSIYQVRLYLEQLHDVGVLEKVNAGKGVPGLWRL   69 (77)
</pre><pre>T ss_dssp             TSCEEHHHHHHHHTCCHHHHHHHHHHHHHTTSEEEESCSSSSCCEEEE
</pre><pre>T ss_pred             CCCCCHHHHHHHHCCCHHHHHHHHHHHHHCCCEEEecCCCCccceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>d1mvfd_ b.129.1.1 (D:) MazE {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=40.81  E-value=72  Score=17.54  Aligned_cols=29  Identities=7%  Similarity=0.197  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEeecCHHHeEEEE
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFDIDTNRVEFRK   69 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfDv~~~~~e~~~   69 (96)
</pre><pre>                      |.++.+.++|+.|   +.+++....+.+....
</pre><pre>T Consensus        15 P~~~~~~~~~~~g---~~v~i~~~~~~i~i~~   43 (44)
</pre><pre>T d1mvfd_          15 PATLMQALNLNID---DEVKIDLVDGKLIIEP   43 (44)
</pre><pre>T ss_dssp             CHHHHHHTTCCTT---CBEEEEEETTEEEEEE
</pre><pre>T ss_pred             CHHHHHHCCCCCC---CeEEEEEECCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>3O27_B Putative uncharacterized protein; swapped-hairpin fold, transcription factor, DNA binding protein; 2.8A {Sulfolobus islandicus}
</pre><pre>Probab=38.41  E-value=95  Score=19.03  Aligned_cols=37  Identities=11%  Similarity=0.184  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEeec--CHHHeEEEECCCCCeEE
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFDI--DTNRVEFRKNDLGVEEY   77 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfDv--~~~~~e~~~Np~~~~E~   77 (96)
</pre><pre>                      |.+..+.+||..|   +.+++.+  ....+....-+....|+
</pre><pre>T Consensus        30 P~~~~~~l~l~~g---~~~~~~~~~~~~~i~i~~~~~~~~~~   68 (68)
</pre><pre>T 3O27_B           30 PKDIAEALDIKPD---DTFILNMEQKDGDIVLSYKRVKELKI   68 (68)
</pre><pre>T ss_dssp             CHHHHHHTTCCTT---CCEEEEEEEETTEEEEEEEECSSCCC
</pre><pre>T ss_pred             eHHHHHHcCCCCC---CEEEEEEEccCCEEEEEEecccccCC
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>d2fy9a1 b.129.1.3 (A:1-54) Putative transition state regulator ABH {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=34.77  E-value=1.3e+02  Score=17.64  Aligned_cols=32  Identities=16%  Similarity=0.295  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEeecCHHHeEEEECCC
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFDIDTNRVEFRKNDL   72 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np~   72 (96)
</pre><pre>                      |.+..+.+++.+|   +.+++-+....+.....+.
</pre><pre>T Consensus        19 P~~~~~~~~l~~g---~~v~i~~~~~~i~i~~~~~   50 (54)
</pre><pre>T d2fy9a1          19 PIELRRALDIAIK---DSIEFFVDGDKIILKKYKP   50 (54)
</pre><pre>T ss_dssp             CHHHHHHTTCSSS---CCEEEEEETTEEEEECSSS
</pre><pre>T ss_pred             CHHHHHHCCCCCC---CEEEEEEeCCEEEEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>3UC2_D hypothetical protein with immunoglobulin-like fold; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY, UNKNOWN FUNCTION; HET: MSE, SO4, GOL; 2.09A {Pseudomonas aeruginosa}
</pre><pre>Probab=34.14  E-value=59  Score=25.30  Aligned_cols=19  Identities=16%  Similarity=0.254  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccE
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYI   56 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yV   56 (96)
</pre><pre>                      +.++|+.|||.+++.+..|
</pre><pre>T Consensus        41 ~p~vA~~ygI~Rs~~~~ll   59 (139)
</pre><pre>T 3UC2_D           41 QPNVASAVGLVRSKAQGVI   59 (139)
</pre><pre>T ss_dssp             ----------CCCTTEEEE
</pre><pre>T ss_pred             CHHHHHHhCCcccCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>6T7O_A Ribosome hibernation promotion factor; HPF, Ribosome, Staphylococcus aureus, Hibernating Promoting Factor, RIBOSOMAL PROTEIN; 1.60003624659A {Staphylococcus aureus (strain NCTC 8325)}
</pre><pre>Probab=33.87  E-value=1.6e+02  Score=19.50  Aligned_cols=46  Identities=15%  Similarity=0.260  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             eEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38     26 VFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus        26 Vf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np   71 (96)
</pre><pre>                      |-...-.-+|||+.||...+......---|...+-..-.+-++...
</pre><pre>T Consensus         5 v~~~~~~~~~ms~~eAi~~le~~~~~f~vF~n~~t~~~~vlyrr~d   50 (67)
</pre><pre>T 6T7O_A            5 IRSKEFSLKPMDSEEAVLQMNLLGHDFFVFTDRETDGTSIVYRRKD   50 (67)
</pre><pre>T ss_dssp             EEECCSCCSCBCHHHHHHHHHHHTCSEEEEEBTTTSSEEEEEECTT
</pre><pre>T ss_pred             EEEeeecCceeCHHHHHHHHHhcCCcEEEEEECCCCcEEEEEEccC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>PF05930.14 ; Phage_AlpA ; Prophage CP4-57 regulatory protein (AlpA)
</pre><pre>Probab=33.78  E-value=85  Score=15.23  Aligned_cols=18  Identities=11%  Similarity=0.093  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             CCcCCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     33 GKPLSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        33 ~KplSp~dae~kygI~~G   50 (96)
</pre><pre>                      ...|++.++++.+++.+.
</pre><pre>T Consensus         1 ~~~~~~~~~~~~~~~~~~   18 (52)
</pre><pre>T Q9KUL8_VIBCH/1    1 MRFLRLKDVMSLTGLGRS   18 (52)
</pre><pre>T ss_pred             CCcccHHHHHHHHCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>PF12298.10 ; Bot1p ; Eukaryotic mitochondrial regulator protein
</pre><pre>Probab=33.54  E-value=1.1e+02  Score=24.70  Aligned_cols=23  Identities=26%  Similarity=0.459  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCCCcCCHHHHHHHhCCCc
</pre><pre>Q 10K_MODEL_38     27 FAVRAKGKPLSMADAADKFKIKQ   49 (96)
</pre><pre>Q Consensus        27 f~e~Ak~KplSp~dae~kygI~~   49 (96)
</pre><pre>                      |..-+-..+++|.|||+-||+.+
</pre><pre>T Consensus       112 F~~i~Es~~Ft~~DAAk~l~~~p  134 (171)
</pre><pre>T C5E0F6_ZYGRC/1  112 FLTIAESEPFGPVDAAKVLELEP  134 (171)
</pre><pre>T ss_pred             EEEeeCCcCCChhhHHHHcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF12324.10 ; HTH_15 ; Helix-turn-helix domain of alkylmercury lyase
</pre><pre>Probab=32.44  E-value=70  Score=22.21  Aligned_cols=21  Identities=19%  Similarity=0.414  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             cCCCcCCHHHHHHHhCCCccc
</pre><pre>Q 10K_MODEL_38     31 AKGKPLSMADAADKFKIKQNH   51 (96)
</pre><pre>Q Consensus        31 Ak~KplSp~dae~kygI~~Gr   51 (96)
</pre><pre>                      |.++|.||.+.+...|+....
</pre><pre>T Consensus        31 a~G~PVs~~~LA~~~g~~~~~   51 (74)
</pre><pre>T F4CKR9_PSEUX/7   31 AEGEPVTLDRLAEAAGTSVDQ   51 (74)
</pre><pre>T ss_pred             HcCCCCCHHHHHHHHCCCHHH
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>TIGR03523 GldN; gliding motility associated protien GldN. Members of this protein family are exclusive to the Bacteroidetes phylum (previously Cytophaga-Flavobacteria-Bacteroides).
</pre><pre>Probab=29.90  E-value=4.3e+02  Score=23.03  Aligned_cols=72  Identities=17%  Similarity=0.198  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhcCceeecccCeeEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeE-------EEECCC-CCeEEEEE
</pre><pre>Q 10K_MODEL_38      9 FAAIKESMKILAGDQNAVFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVE-------FRKNDL-GVEEYKIK   80 (96)
</pre><pre>Q Consensus         9 ~~gI~e~~~I~A~DqNrVf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e-------~~~Np~-~~~E~tvk   80 (96)
</pre><pre>                      +++|++..+|.|.+ +.=|..-....+||+.|..+++++..-........|+++++|.       |....+ +...+.|.
</pre><pre>T Consensus        94 ~~~i~~G~~~~~Y~-~d~fe~f~~~~~lt~~e~~~~~~~~~~~~~~~~~~~i~~~~v~~y~IKE~wyFDk~~S~~~~rI~  172 (280)
</pre><pre>T TIGR03523        94 FKAIVEGSIIEVYA-DDGFEIFTDTEPITLQEFQDRFYLYYEPSDRLDGDSIFSADVYGYYIKEEWYFDKQRSDMKFRII  172 (280)
</pre><pre>T ss_pred             HHHHHcCCCccccc-CCCCccCCCCCccCHHHHHHHhhhccCCCCCCCcccCCHhHeEEEEEEEEEEEeCCCCceEEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             c
</pre><pre>Q 10K_MODEL_38     81 G   81 (96)
</pre><pre>Q Consensus        81 G   81 (96)
</pre><pre>                      |
</pre><pre>T Consensus       173 ~  173 (280)
</pre><pre>T TIGR03523       173 A  173 (280)
</pre><pre>T ss_pred             E
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>TIGR01764 excise; DNA binding domain, excisionase family. An excisionase, or Xis protein, is a small protein that binds and promotes excisive recombination.
</pre><pre>Probab=29.88  E-value=1.1e+02  Score=14.95  Aligned_cols=14  Identities=21%  Similarity=0.496  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQ   49 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~   49 (96)
</pre><pre>                      +|..++++.+++.+
</pre><pre>T Consensus         2 ~~~~~~~~~~~~~~   15 (49)
</pre><pre>T TIGR01764         2 LTVEEAAEYLGVSK   15 (49)
</pre><pre>T ss_pred             CCHHHHHHHHCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>d1wi9a1 a.4.5.47 (A:105-163) Hypothetical protein C20orf116 homolog {Mouse (Mus musculus) [TaxId: 10090]}
</pre><pre>Probab=29.01  E-value=91  Score=18.98  Aligned_cols=19  Identities=21%  Similarity=0.302  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCcCCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     32 KGKPLSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        32 k~KplSp~dae~kygI~~G   50 (96)
</pre><pre>                      ...++|+.|+++.||+.+.
</pre><pre>T Consensus        11 ~~~~~s~~ela~~~~~s~~   29 (59)
</pre><pre>T d1wi9a1          11 KSKVVLLEDLAFQMGLRTQ   29 (59)
</pre><pre>T ss_dssp             HCSEECHHHHHHHHCSCHH
</pre><pre>T ss_pred             hcCCccHHHHHHHhCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>d3ml1a2 b.52.2.0 (A:682-802) automated matches {Ralstonia eutropha [TaxId: 381666]}
</pre><pre>Probab=28.73  E-value=93  Score=20.89  Aligned_cols=14  Identities=14%  Similarity=0.375  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~G   50 (96)
</pre><pre>                      +||.|| +++||+.|
</pre><pre>T Consensus        41 i~p~dA-~~lgl~~g   54 (121)
</pre><pre>T d3ml1a2          41 MHPEDA-KALGLRRG   54 (121)
</pre><pre>T ss_dssp             ECHHHH-HHHTCCTT
</pre><pre>T ss_pred             ECHHHH-HHcCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>d3f0pa1 a.4.5.79 (A:1-80) Alkylmercury lyase MerB {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=28.58  E-value=69  Score=23.25  Aligned_cols=29  Identities=24%  Similarity=0.415  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             cCCCcCCHHHHHHHhCCCccccccc------EEee
</pre><pre>Q 10K_MODEL_38     31 AKGKPLSMADAADKFKIKQNHARNY------IDFD   59 (96)
</pre><pre>Q Consensus        31 Ak~KplSp~dae~kygI~~Grgr~y------VEfD   59 (96)
</pre><pre>                      |+++|.|+.+.|...|....+=+..      +|+|
</pre><pre>T Consensus        32 a~G~PVt~~~LA~a~g~~~e~V~~~L~~~p~~EyD   66 (80)
</pre><pre>T d3f0pa1          32 AKGRPVSRTTLAGILDWPAERVAAVLEQATSTEYD   66 (80)
</pre><pre>T ss_dssp             TTTSCBCHHHHHHHHTCCHHHHHHHHHHCTTCEEC
</pre><pre>T ss_pred             HcCCCcCHHHHHHHhCCCHHHHHHHHHHcCCCEeC
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF14584.8 ; DUF4446 ; Protein of unknown function (DUF4446)
</pre><pre>Probab=27.97  E-value=1.9e+02  Score=22.78  Aligned_cols=25  Identities=32%  Similarity=0.458  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             eeecccCeeEEEEcCCC----cCCHHHHH
</pre><pre>Q 10K_MODEL_38     18 ILAGDQNAVFAVRAKGK----PLSMADAA   42 (96)
</pre><pre>Q Consensus        18 I~A~DqNrVf~e~Ak~K----plSp~dae   42 (96)
</pre><pre>                      |.++++.++|+.+-+.-    +||++|-+
</pre><pre>T Consensus       118 i~~R~~sr~YaK~I~~G~s~~~LS~EE~~  146 (152)
</pre><pre>T K0B417_GOTA9/1  118 IYAKDESNVYAKPIVDGKSTYPLSVEEMQ  146 (152)
</pre><pre>T ss_pred             EEecCCeEEEEEEeeCCCcCCCCCHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>d1b8mb_ g.17.1.3 (B:) Neurotrophin 4, NT4 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=27.63  E-value=1.2e+02  Score=22.93  Aligned_cols=38  Identities=16%  Similarity=0.336  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             EeecCHHHeEEEECCC-----CCeEEEEEcce------ecCCCCCcceecC
</pre><pre>Q 10K_MODEL_38     57 DFDIDTNRVEFRKNDL-----GVEEYKIKGDI------ELDEKTTEFNKRC   96 (96)
</pre><pre>Q Consensus        57 EfDv~~~~~e~~~Np~-----~~~E~tvkGdv------~L~nrnp~f~krc   96 (96)
</pre><pre>                      |++|+++..+|+.||.     ...+++|-.++      .++.  .-|..+|
</pre><pre>T Consensus         2 e~~vC~S~s~Wv~~~~~a~d~~G~~V~Vl~~~~~~~g~~~~Q--~FyEt~C   50 (119)
</pre><pre>T d1b8mb_           2 ELAVCDAVSGWVTDRRTAVDLRGREVEVLGEVPAAGGSPLRQ--YFFETRC   50 (119)
</pre><pre>T ss_dssp             EEESSCEEEEEECCCCEEEBTTSCEEEECSEECCSSSSCEEC--CEEEEEE
</pre><pre>T ss_pred             ccccccceEEEeecCCEEEeCCCCEEEEEeeeeCCCCCceEE--EEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>PF20063.1 ; DUF6462 ; Family of unknown function (DUF6462)
</pre><pre>Probab=26.16  E-value=1.2e+02  Score=14.62  Aligned_cols=18  Identities=11%  Similarity=0.274  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             CCcCCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     33 GKPLSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        33 ~KplSp~dae~kygI~~G   50 (96)
</pre><pre>                      ...+++.++++.+|+...
</pre><pre>T Consensus         1 ~~~~~~~~~~~~~~~~~~   18 (51)
</pre><pre>T A0A1Q9J2M5_9FI    1 KKYVRYPEGAEMYSMSLS   18 (51)
</pre><pre>T ss_pred             CCcccHHHHHHHhCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>3MUD_B DNA repair protein XRCC4,Tropomyosin alpha-1 chain; tropomysoin, overlap complex, coiled-coils, Contractile Protein; 2.2A {Homo sapiens}
</pre><pre>Probab=25.90  E-value=3e+02  Score=20.27  Aligned_cols=67  Identities=12%  Similarity=0.181  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             ceeecccCeeEEEEcCCC---------cCCHHHHHHHhCCCcccccc---cEEeecCHH--HeEEEECCCCCeEEEEE-c
</pre><pre>Q 10K_MODEL_38     17 KILAGDQNAVFAVRAKGK---------PLSMADAADKFKIKQNHARN---YIDFDIDTN--RVEFRKNDLGVEEYKIK-G   81 (96)
</pre><pre>Q Consensus        17 ~I~A~DqNrVf~e~Ak~K---------plSp~dae~kygI~~Grgr~---yVEfDv~~~--~~e~~~Np~~~~E~tvk-G   81 (96)
</pre><pre>                      .|.+.|.+.+|.......         .+|+.+-.+...-.-.....   -+.|++..+  .+.|.+...+   +.++ |
</pre><pre>T Consensus        36 ~l~~tdg~~~w~g~v~~~~i~~~a~~~~~s~~~y~~~~~~aL~~~~~~~~~~~~~l~~~~~~~~wkk~~~~---i~~~lG  112 (175)
</pre><pre>T 3MUD_B           36 VITLTDGHSAWTGTVSESEISQEADDMAMEKGKYVGELRKALLSGAGPADVYTFNFSKESCYFFFEKNLKD---VSFRLG  112 (175)
</pre><pre>T ss_dssp             EEEEECSSCEEEEEEEHHHHHHHHHHHTCCHHHHHHHHHHHHSSCCCSSCEEEEEEETTTTEEEEEEECSS---CEEEEE
</pre><pre>T ss_pred             EEEEEcCCceEEEEecHHHHHHHHHHccCCHHHHHHHHHHHhhcCCCCcceEEEEEcCCeEEEEEEEeccc---ceEEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ceecC
</pre><pre>Q 10K_MODEL_38     82 DIELD   86 (96)
</pre><pre>Q Consensus        82 dv~L~   86 (96)
</pre><pre>                      .+.|.
</pre><pre>T Consensus       113 si~L~  117 (175)
</pre><pre>T 3MUD_B          113 SFNLE  117 (175)
</pre><pre>T ss_dssp             EEECE
</pre><pre>T ss_pred             EEEeE
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>PF13411.8 ; MerR_1 ; MerR HTH family regulatory protein
</pre><pre>Probab=25.51  E-value=1.2e+02  Score=16.65  Aligned_cols=15  Identities=13%  Similarity=0.259  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~G   50 (96)
</pre><pre>                      ||+.+++..+|+.+.
</pre><pre>T Consensus         1 ~~~~~~a~~~~i~~~   15 (69)
</pre><pre>T C6WDH8_ACTMD/4    1 LTVDELAVRVGLPGS   15 (69)
</pre><pre>T ss_pred             CCHHHHHHHHCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>3K2T_A Lmo2511 protein; Listeria monocytogenes, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, unknown function; 2.4A {Listeria monocytogenes}
</pre><pre>Probab=25.39  E-value=2.7e+02  Score=17.37  Aligned_cols=46  Identities=15%  Similarity=0.249  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38     26 VFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus        26 Vf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np   71 (96)
</pre><pre>                      |-.+...-+|||+.||...+......---|...+-..-.+-++...
</pre><pre>T Consensus         3 v~~~~~~~~~ms~~eAv~~l~~~~~~f~~F~~~~t~~~~vlyrr~~   48 (57)
</pre><pre>T 3K2T_A            3 VRTKQFSLKPMDSEEAVLQMNLLGHSFYVYTDAETNGTNIVYSRKD   48 (57)
</pre><pre>T ss_dssp             CCCCC---CCBCHHHHHHHHHHHTCSEEEEEBSSSCCEEEEEECTT
</pre><pre>T ss_pred             eeEEeeeCceECHHHHHHHHHhhCCcEEEEEECCCCcEEEEEEccC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>7OMV_AAA DarT domain-containing protein; TOXIN-ANTITOXIN, DNA ADP-RIBOSYLATION, ADP-ribosyltransferase activity, DNA binding, TOXIN; 1.29A {Thermus sp. 2.9}
</pre><pre>Probab=25.35  E-value=95  Score=24.05  Aligned_cols=22  Identities=32%  Similarity=0.327  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CccchhhHHHHHhcCceeeccc
</pre><pre>Q 10K_MODEL_38      2 HYTSNQGFAAIKESMKILAGDQ   23 (96)
</pre><pre>Q Consensus         2 HYTn~kg~~gI~e~~~I~A~Dq   23 (96)
</pre><pre>                      |+|...-+..|.+.|.|.+++.
</pre><pre>T Consensus        14 HfT~~~NL~sIl~~G~Llsr~~   35 (210)
</pre><pre>T 7OMV_AAA         14 HITHIDNLKGILRMGKLLAHNQ   35 (210)
</pre><pre>T ss_pred             EEEehhcHHHHHHhCCEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>d1yfba_ b.129.1.3 (A:) Transcription-state regulator AbrB, the N-terminal DNA recognition domain {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=24.98  E-value=2.5e+02  Score=16.20  Aligned_cols=32  Identities=9%  Similarity=0.259  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEeecCHHHeEEEECCC
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFDIDTNRVEFRKNDL   72 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np~   72 (96)
</pre><pre>                      |.+..+.++++.|   +.+++.+..+.+....-..
</pre><pre>T Consensus        20 P~~~~~~~~l~~g---~~~~~~~~~~~i~i~~~~~   51 (52)
</pre><pre>T d1yfba_          20 PIELRRTLGIAEK---DALEIYVDDEKIILKKYKP   51 (52)
</pre><pre>T ss_dssp             CHHHHHHTTCCTT---CEEEEEEETTEEEEEECCC
</pre><pre>T ss_pred             cHHHHHHcCCCCC---CEEEEEEECCEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>d1bl0a1 a.4.1.8 (A:9-62) MarA {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=24.35  E-value=1.8e+02  Score=15.58  Aligned_cols=17  Identities=18%  Similarity=0.393  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             CCcCCHHHHHHHhCCCc
</pre><pre>Q 10K_MODEL_38     33 GKPLSMADAADKFKIKQ   49 (96)
</pre><pre>Q Consensus        33 ~KplSp~dae~kygI~~   49 (96)
</pre><pre>                      ..++|..++++.+|+.+
</pre><pre>T Consensus        17 ~~~~~~~~ia~~~~~~~   33 (54)
</pre><pre>T d1bl0a1          17 ESPLSLEKVSERSGYSK   33 (54)
</pre><pre>T ss_dssp             TSCCCCHHHHHHSSSCH
</pre><pre>T ss_pred             CCCCCHHHHHHHHCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF15689.7 ; NRIP1_repr_3 ; Nuclear receptor-interacting protein 1 repression 3
</pre><pre>Probab=24.18  E-value=52  Score=24.26  Aligned_cols=16  Identities=38%  Similarity=0.455  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             CCCcCCHHHHH-HHhCC
</pre><pre>Q 10K_MODEL_38     32 KGKPLSMADAA-DKFKI   47 (96)
</pre><pre>Q Consensus        32 k~KplSp~dae-~kygI   47 (96)
</pre><pre>                      |..|+||.|+. .|+||
</pre><pre>T Consensus        51 KsEP~sp~e~~~sK~GL   67 (89)
</pre><pre>T G3VXQ5_SARHA/7   51 KSEPLSPQDFSFSKNGL   67 (89)
</pre><pre>T ss_pred             ccCCCCCcchhhccccH
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>5MRC_66 mS45; yeast, mitochondrial, ribosome, saccharomyces cerevisiae; HET: MG, GDP; 3.25A {Saccharomyces cerevisiae}
</pre><pre>Probab=23.36  E-value=2.1e+02  Score=25.74  Aligned_cols=24  Identities=25%  Similarity=0.411  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEEcCCCcCCHHHHHHHhCCCc
</pre><pre>Q 10K_MODEL_38     26 VFAVRAKGKPLSMADAADKFKIKQ   49 (96)
</pre><pre>Q Consensus        26 Vf~e~Ak~KplSp~dae~kygI~~   49 (96)
</pre><pre>                      .|..-+-..+.+|.|||+-|+|.+
</pre><pre>T Consensus       217 ~F~~i~ES~~Ftp~DAAkvl~l~P  240 (319)
</pre><pre>T 5MRC_66         217 RFLTIAESEPFGPVDAAHVLELEP  240 (319)
</pre><pre>T ss_dssp             CEEEEESSCCCCHHHHHHHHTSCC
</pre><pre>T ss_pred             cEEEecCCCCCCHHHHHHHcCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>3I71_B Ethanolamine utilization protein eutK; helix-turn-helix, UNKNOWN FUNCTION; HET: FLC; 2.1A {Escherichia coli}
</pre><pre>Probab=23.12  E-value=46  Score=22.75  Aligned_cols=28  Identities=18%  Similarity=0.255  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCCCcCCHHHHHHHhCCCccccccc
</pre><pre>Q 10K_MODEL_38     27 FAVRAKGKPLSMADAADKFKIKQNHARNY   55 (96)
</pre><pre>Q Consensus        27 f~e~Ak~KplSp~dae~kygI~~Grgr~y   55 (96)
</pre><pre>                      |+..+ ..++++.|+|+.++|.....+.|
</pre><pre>T Consensus        11 ~i~~~-~p~~ta~EVAe~l~~~~~~a~~r   38 (68)
</pre><pre>T 3I71_B           11 LLTSV-RQGMTAGEVAAHFGWPLEKARNA   38 (68)
</pre><pre>T ss_dssp             HHHHH-HHCBCHHHHHHHHTCCHHHHHHH
</pre><pre>T ss_pred             HHHhC-CCCccHHHHHHHHCCCHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>7M3P_B Xrcc4-Spc110p(164-207); microtubule nucleation, CELL CYCLE; 2.00001856449A {Homo sapiens}
</pre><pre>Probab=23.06  E-value=4.1e+02  Score=19.78  Aligned_cols=66  Identities=17%  Similarity=0.290  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             eeecccCeeEEEEcCCCcCC-------------HHHHHHHhCCCcccccccEEeecCHH--HeEEEECCCCCeEEEEEcc
</pre><pre>Q 10K_MODEL_38     18 ILAGDQNAVFAVRAKGKPLS-------------MADAADKFKIKQNHARNYIDFDIDTN--RVEFRKNDLGVEEYKIKGD   82 (96)
</pre><pre>Q Consensus        18 I~A~DqNrVf~e~Ak~KplS-------------p~dae~kygI~~Grgr~yVEfDv~~~--~~e~~~Np~~~~E~tvkGd   82 (96)
</pre><pre>                      |.+.|.+.+|.-......++             -.+....+.-..+.+..| .|++..+  .|.|.+.-.+ ..+.+ |.
</pre><pre>T Consensus        41 l~ltdg~~~w~~~v~~~~i~~~a~~~~~s~~ey~~~~~~aL~~~~~~~~~~-~~~~~~~~~~l~wkk~~~~-i~~~l-G~  117 (183)
</pre><pre>T 7M3P_B           41 ITLTDGHSAWTGTVSESEISQEADDMAMEKGKYVGELRKALLSGAGPADVY-TFNFSKESCYFFFEKNLKD-VSFRL-GS  117 (183)
</pre><pre>T ss_dssp             EEEECSSCEEEEEECHHHHHHHHHHTTCCHHHHHHHHHHHHCC------CE-EEEECTTTCEEEEEEEETT-EEEEE-EE
</pre><pre>T ss_pred             EEEEcCCceEEEEeCHHHHHHHHHHccccHHHHHHHHHHHHhcCCCCCcce-EEEEeCCeEEEEEEEeccc-ceEEE-EE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eecC
</pre><pre>Q 10K_MODEL_38     83 IELD   86 (96)
</pre><pre>Q Consensus        83 v~L~   86 (96)
</pre><pre>                      |.|.
</pre><pre>T Consensus       118 i~L~  121 (183)
</pre><pre>T 7M3P_B          118 FNLE  121 (183)
</pre><pre>T ss_dssp             EECE
</pre><pre>T ss_pred             EEEE
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>5MYJ_A Ribosome hibernation promotion factor; ribosome, 70S, lactoccocus lactis, Cryo-EM; 5.6A {Lactococcus lactis subsp. cremoris MG1363}
</pre><pre>Probab=22.55  E-value=1.8e+02  Score=21.90  Aligned_cols=38  Identities=21%  Similarity=0.388  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38     34 KPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus        34 KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np   71 (96)
</pre><pre>                      +|||+.||...+.+....---|+..|-..-.|-++.++
</pre><pre>T Consensus       139 ~pms~~eA~~~m~~~~~~f~~F~~~~t~~~~vlyrr~d  176 (185)
</pre><pre>T 5MYJ_A          139 KPMDAEEAVLQMDMLGHDFYVFTDADSNGTHVVYRRTD  176 (185)
</pre><pre>T ss_dssp             SCCCTTHHHHHHHHHTCSSCCEEETTTTEEECCCBCTT
</pre><pre>T ss_pred             cccCHHHHHHHHHhhCCcEEEEEECCCCcEEEEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>d1ti6a1 b.52.2.2 (A:729-875) Transhydroxylase alpha subunit, AthL {Pelobacter acidigallici [TaxId: 35816]}
</pre><pre>Probab=22.50  E-value=1.3e+02  Score=21.30  Aligned_cols=14  Identities=29%  Similarity=0.332  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~G   50 (96)
</pre><pre>                      |+|.|| +++||+.|
</pre><pre>T Consensus        46 inp~dA-~~~gi~~G   59 (147)
</pre><pre>T d1ti6a1          46 VNSIDA-EARGIKNG   59 (147)
</pre><pre>T ss_dssp             EEHHHH-HHTTCCTT
</pre><pre>T ss_pred             EcHHHH-HHcCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>TIGR02609 doc_partner; putative addiction module antidote. Members of this protein family are putative addiction module antidote proteins that appear recurringly in two-gene operons with members of the Doc (death-on-curing) family TIGR01550.
</pre><pre>Probab=22.32  E-value=2.7e+02  Score=17.93  Aligned_cols=29  Identities=7%  Similarity=0.294  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEeecCHHHeEEEE
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFDIDTNRVEFRK   69 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfDv~~~~~e~~~   69 (96)
</pre><pre>                      |.+..+.++|.+|   +.|++.+....+....
</pre><pre>T Consensus        15 P~~~~~~l~l~~g---d~v~~~~~~~~i~i~~   43 (74)
</pre><pre>T TIGR02609        15 PKEVLESLGLKEG---DTLYVDEEEGGLKLKR   43 (74)
</pre><pre>T ss_pred             cHHHHHHcCCCCC---CEEEEEEcCCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF16321.7 ; Ribosom_S30AE_C ; Sigma 54 modulation/S30EA ribosomal protein C terminus
</pre><pre>Probab=22.29  E-value=2.9e+02  Score=16.93  Aligned_cols=46  Identities=17%  Similarity=0.288  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             eEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38     26 VFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus        26 Vf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np   71 (96)
</pre><pre>                      |-...-.-+|||+.||-..+......---|...+-..-.+-+....
</pre><pre>T Consensus         7 v~~~~~~~~pmt~~eAi~~l~~~~~~f~~F~~~~t~~~~v~yr~~~   52 (57)
</pre><pre>T R7I472_9FIRM/1    7 VRRKTFPVRPMSVEDAAVQMEMLGHSFFAFVNAETEEINVLYQRKD   52 (57)
</pre><pre>T ss_pred             EeeeeecCceEcHHHHHHHHHHcCCcEEEEEeCCCCcEEEEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>1WFX_A Probable RNA 2'-phosphotransferase; RNA 2'-phosphotransferase, tRNA splicing, NAD, Structural Genomics, RIKEN Structural Genomics/Proteomics Initiative, RSGI, transferase; HET: MSE; 2.8A {Aeropyrum pernix} SCOP: d.166.1.5
</pre><pre>Probab=22.06  E-value=3.3e+02  Score=20.80  Aligned_cols=53  Identities=17%  Similarity=0.192  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CccchhhHHHHHhcCceeecccCeeEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCH
</pre><pre>Q 10K_MODEL_38      2 HYTSNQGFAAIKESMKILAGDQNAVFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDT   62 (96)
</pre><pre>Q Consensus         2 HYTn~kg~~gI~e~~~I~A~DqNrVf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~   62 (96)
</pre><pre>                      |-|..+....|.+.| |.....+.|.+....       +.+...|...+..--+|+.|+..
</pre><pre>T Consensus        96 hgt~~~~~~~I~~~G-l~~~~r~~ihl~~~~-------~~a~~~g~r~~~~~v~i~id~~~  148 (186)
</pre><pre>T 1WFX_A           96 HGTTEEALPLIMERG-IMRGRRLKVHLTSSL-------EDAVSTGRRHGNLVAVLLVDVEC  148 (186)
</pre><pre>T ss_dssp             ECCBGGGHHHHHHHC-BCCCSSSSEEEESCH-------HHHHHHHTTSSSSBEEEEEEHHH
</pre><pre>T ss_pred             EEeCHHHHHHHHHhC-CccCCCceeEeeCCh-------hHHHHhcccCCCceEEEEEcHHH
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>PF13670.8 ; PepSY_2 ; Peptidase propeptide and YPEB domain
</pre><pre>Probab=21.94  E-value=4e+02  Score=16.68  Aligned_cols=51  Identities=8%  Similarity=-0.083  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             cCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECCC-CCeEEEEEcc
</pre><pre>Q 10K_MODEL_38     31 AKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKNDL-GVEEYKIKGD   82 (96)
</pre><pre>Q Consensus        31 Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np~-~~~E~tvkGd   82 (96)
</pre><pre>                      .....+|+.++.+...=..+.--..+|+|-. ...+...++. +..|+.|...
</pre><pre>T Consensus        22 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-~~y~v~~~~~g~~~~v~id~~   73 (81)
</pre><pre>T Q1QMT6_NITHX/5   22 PGSDWMPADQVIQKLMEKGYTDVSKLEADDG-RCWEAKAMLNGVRQKIYLDPR   73 (81)
</pre><pre>T ss_pred             CCCCCCCHHHHHHHHHHCCCcceEEEEeCCC-CeEEEEEecCCeEEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>PF03333.15 ; PapB ; Adhesin biosynthesis transcription regulatory protein
</pre><pre>Probab=21.57  E-value=1.4e+02  Score=21.04  Aligned_cols=15  Identities=13%  Similarity=0.220  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~G   50 (96)
</pre><pre>                      +|..+|++++|+.+.
</pre><pre>T Consensus        56 ~s~~e~A~~~gvs~~   70 (93)
</pre><pre>T Q3ZU00_ECOLX/1   56 CSRKFVCEHYNINNG   70 (93)
</pre><pre>T ss_pred             CCHHHHHHHHCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF07453.15 ; NUMOD1 ; NUMOD1 domain
</pre><pre>Probab=21.57  E-value=1.6e+02  Score=15.66  Aligned_cols=14  Identities=29%  Similarity=0.387  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     37 SMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        37 Sp~dae~kygI~~G   50 (96)
</pre><pre>                      |..+|++.+|+.+.
</pre><pre>T Consensus        21 s~~~a~~~~g~~~~   34 (40)
</pre><pre>T Q02714_PODAN/4   21 SKTQASNILGISRN   34 (40)
</pre><pre>T ss_pred             CHHHHHHHHCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>6G1T_A AM32; Repressor, Protein-DNA complex, Type IV secretion system, DNA BINDING PROTEIN; HET: EPE; 1.93A {Enterococcus faecalis}
</pre><pre>Probab=21.56  E-value=1.2e+02  Score=22.38  Aligned_cols=15  Identities=7%  Similarity=0.215  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~G   50 (96)
</pre><pre>                      ||+.|||+.+|+.++
</pre><pre>T Consensus        64 lT~~EAA~~lgvs~~   78 (122)
</pre><pre>T 6G1T_A           64 ITAKEASQLLGKNDE   78 (122)
</pre><pre>T ss_dssp             EEHHHHHHHTTCCTT
</pre><pre>T ss_pred             eeHHHHHHHHCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>TIGR01439 lp_hng_hel_AbrB; looped-hinge helix DNA binding domain, AbrB family. This DNA-binding domain family includes AbrB, a transition state regulator in Bacillus subtilis, whose DNA-binding domain structure in solution was determined by NMR.
</pre><pre>Probab=21.42  E-value=3.2e+02  Score=15.42  Aligned_cols=31  Identities=10%  Similarity=0.315  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEee-cCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFD-IDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfD-v~~~~~e~~~Np   71 (96)
</pre><pre>                      |.+..+++||.+|   ..|+|. .....+....-+
</pre><pre>T Consensus        12 P~~~~~~l~i~~g---~~v~~~~~~~~~i~i~~~~   43 (43)
</pre><pre>T TIGR01439        12 PKEIREKLGLKEG---DRLEVIRVEDGEIILRPKR   43 (43)
</pre><pre>T ss_pred             CHHHHHHhCCCCC---CEEEEEEccCCeEEEEECC
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>1YFB_A Transition state regulatory protein abrB; NMR  HOMODIMER  BIOINFORMATICS  SWAPPED-HAIRPIN BARREL, TRANSCRIPTION; NMR {Bacillus subtilis} SCOP: b.129.1.3
</pre><pre>Probab=21.22  E-value=3.7e+02  Score=16.04  Aligned_cols=44  Identities=14%  Similarity=0.259  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             cCeeEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEE
</pre><pre>Q 10K_MODEL_38     23 QNAVFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRK   69 (96)
</pre><pre>Q Consensus        23 qNrVf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~   69 (96)
</pre><pre>                      ...+.....++.-.=|.+..+.+||..|.   .|++.+....+...+
</pre><pre>T Consensus        12 ~~~~~~~~~~~~i~iP~~~~~~~~l~~g~---~v~~~~~~~~i~i~~   55 (59)
</pre><pre>T 1YFB_A           12 TGIVRKVDELGRVVIPIELRRTLGIAEKD---ALEIYVDDEKIILKK   55 (59)
</pre><pre>T ss_dssp             CCEEEECCTTCEEECCHHHHHHTTCCTTC---EEEEEEETTEEEEEE
</pre><pre>T ss_pred             cceEEEeCCCCeEEccHHHHHHcCCCCCC---EEEEEEECCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>1YSF_B Transition state regulatory protein abrB; NMR  HOMODIMER  BIOINFORMATICS  SWAPPED-HAIRPIN BARREL, TRANSCRIPTION; NMR {Bacillus subtilis} SCOP: b.129.1.3
</pre><pre>Probab=21.22  E-value=3.7e+02  Score=16.04  Aligned_cols=44  Identities=14%  Similarity=0.259  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             cCeeEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEE
</pre><pre>Q 10K_MODEL_38     23 QNAVFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRK   69 (96)
</pre><pre>Q Consensus        23 qNrVf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~   69 (96)
</pre><pre>                      ...+.....++.-.=|.+..+.+||..|.   .|++.+....+...+
</pre><pre>T Consensus        12 ~~~~~~~~~~~~i~iP~~~~~~~~l~~g~---~v~~~~~~~~i~i~~   55 (59)
</pre><pre>T 1YSF_B           12 TGIVRKVDELGRVVIPIELRRTLGIAEKD---ALEIYVDDEKIILKK   55 (59)
</pre><pre>T ss_dssp             CCEEEECCTTCEEECCHHHHHHTTCCTTC---EEEEEEETTEEEEEE
</pre><pre>T ss_pred             cceEEEeCCCCeEEccHHHHHHcCCCCCC---EEEEEEECCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>PF04936.14 ; DUF658 ; Protein of unknown function (DUF658)
</pre><pre>Probab=21.09  E-value=1.5e+02  Score=22.74  Aligned_cols=25  Identities=20%  Similarity=0.299  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             eeEEEEcCCCcCCHHHHHHHhCCCc
</pre><pre>Q 10K_MODEL_38     25 AVFAVRAKGKPLSMADAADKFKIKQ   49 (96)
</pre><pre>Q Consensus        25 rVf~e~Ak~KplSp~dae~kygI~~   49 (96)
</pre><pre>                      .||.......-||+.++++.||+..
</pre><pre>T Consensus         4 ~~~~~~~~~~~lt~~eia~~lgvs~   28 (186)
</pre><pre>T Q9CGR9_LACLA/4    4 DAYIEGEKKATGTIDEIADYFDISR   28 (186)
</pre><pre>T ss_pred             eeeeCCcccceEeHHHHHHHhCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>3M8J_B FocB protein; All-alpha, helix-turn-helix, TRANSCRIPTION; 1.4A {Escherichia coli}
</pre><pre>Probab=21.00  E-value=1.4e+02  Score=22.08  Aligned_cols=15  Identities=20%  Similarity=0.454  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCcc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQN   50 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~G   50 (96)
</pre><pre>                      +|+.||+++||+.++
</pre><pre>T Consensus        61 ~s~~eaa~~~gvs~~   75 (111)
</pre><pre>T 3M8J_B           61 HSRKDVCEKYQMNNG   75 (111)
</pre><pre>T ss_dssp             CCHHHHHHHTTCCHH
</pre><pre>T ss_pred             CCHHHHHHHHCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>PF12728.9 ; HTH_17 ; Helix-turn-helix domain
</pre><pre>Probab=20.88  E-value=1.9e+02  Score=14.52  Aligned_cols=14  Identities=29%  Similarity=0.465  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHhCCCc
</pre><pre>Q 10K_MODEL_38     36 LSMADAADKFKIKQ   49 (96)
</pre><pre>Q Consensus        36 lSp~dae~kygI~~   49 (96)
</pre><pre>                      +++.+++..+++.+
</pre><pre>T Consensus         2 ~~~~~~~~~~~~~~   15 (57)
</pre><pre>T Q24ZC0_DESHY/1    2 VDTSKAAELLKISK   15 (57)
</pre><pre>T ss_pred             CcHHHHHHHhCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>PF07180.13 ; CaiF_GrlA ; CaiF/GrlA transcriptional regulator
</pre><pre>Probab=20.71  E-value=86  Score=24.40  Aligned_cols=25  Identities=32%  Similarity=0.307  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             CCCcCCHHHHHHHhCCCcccccccE
</pre><pre>Q 10K_MODEL_38     32 KGKPLSMADAADKFKIKQNHARNYI   56 (96)
</pre><pre>Q Consensus        32 k~KplSp~dae~kygI~~Grgr~yV   56 (96)
</pre><pre>                      .++++|..|+++.|+|..-++.+.+
</pre><pre>T Consensus        35 q~r~vt~~dIa~aF~Is~RrAs~il   59 (137)
</pre><pre>T Q8VLD0_ECOLX/1   35 QEKWISRNDIAEAFGINLRRASFII   59 (137)
</pre><pre>T ss_pred             cCCcccHHHHHHHHCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>PF08845.12 ; SymE_toxin ; Toxin SymE, type I toxin-antitoxin system
</pre><pre>Probab=20.50  E-value=2.1e+02  Score=16.70  Aligned_cols=29  Identities=7%  Similarity=0.095  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCcccccccEEeecCHHHeEEEE
</pre><pre>Q 10K_MODEL_38     38 MADAADKFKIKQNHARNYIDFDIDTNRVEFRK   69 (96)
</pre><pre>Q Consensus        38 p~dae~kygI~~Grgr~yVEfDv~~~~~e~~~   69 (96)
</pre><pre>                      |.+..+++||..|   +-|++.+..+.+....
</pre><pre>T Consensus        23 p~~~~~~~~~~~g---~~v~v~~~~~~i~i~~   51 (51)
</pre><pre>T Q832D6_ENTFA/1   23 QGKWLEQAGFSVG---DKITVKCQQGQLTITK   51 (51)
</pre><pre>T ss_pred             CHHHHHHcCCCCC---CEEEEEEECCEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>TIGR00156 TIGR00156; TIGR00156 family protein. As of the last revision, this family consists only of two proteins from Escherichia coli and one from the related species Haemophilus influenzae.
</pre><pre>Probab=20.40  E-value=1.8e+02  Score=21.37  Aligned_cols=44  Identities=7%  Similarity=0.219  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             hCCCcccccccEEeecCHHHeEEEECCC-CCeEEEEEcceecCCCCCcc
</pre><pre>Q 10K_MODEL_38     45 FKIKQNHARNYIDFDIDTNRVEFRKNDL-GVEEYKIKGDIELDEKTTEF   92 (96)
</pre><pre>Q Consensus        45 ygI~~Grgr~yVEfDv~~~~~e~~~Np~-~~~E~tvkGdv~L~nrnp~f   92 (96)
</pre><pre>                      |-+..+.|.-.|+++-..    |..+.. ...++.|.|.|+.....+.+
</pre><pre>T Consensus        75 ~~l~D~tG~i~V~~~~~~----~~~~~l~~g~~V~v~G~v~~~~~~~~i  119 (126)
</pre><pre>T TIGR00156        75 YVFRDKSGEINVVIPAAV----WNGREVQPKDMVNISGSLDKKSAPAEV  119 (126)
</pre><pre>T ss_pred             EEEEcCCCeEEEEeCCCc----cCCCCCCCCCEEEEEEEEeCCCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>6ZHX_K Chromodomain-helicase-DNA-binding protein 1-like; ALC1, CHD1L, chromatin remodeler, DNA damage response, nucleosome, NUCLEAR PROTEIN, GENE REGULATION; 2.5A {Xenopus laevis}
</pre><pre>Probab=20.25  E-value=1.3e+02  Score=18.58  Aligned_cols=9  Identities=22%  Similarity=0.335  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             CCCcCCHHH
</pre><pre>Q 10K_MODEL_38     32 KGKPLSMAD   40 (96)
</pre><pre>Q Consensus        32 k~KplSp~d   40 (96)
</pre><pre>                      ++|+|||.+
</pre><pre>T Consensus        28 krk~Lt~eE   36 (37)
</pre><pre>T 6ZHX_K           28 RKRVLSPEE   36 (37)
</pre><pre>T ss_pred             cccCCChhh
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>PF06394.15 ; Pepsin-I3 ; Pepsin inhibitor-3-like repeated domain
</pre><pre>Probab=20.15  E-value=1.9e+02  Score=20.04  Aligned_cols=21  Identities=14%  Similarity=0.349  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             cCeeEEEEcCCCcCCHHHHHH
</pre><pre>Q 10K_MODEL_38     23 QNAVFAVRAKGKPLSMADAAD   43 (96)
</pre><pre>Q Consensus        23 qNrVf~e~Ak~KplSp~dae~   43 (96)
</pre><pre>                      +|++|.-.--.+.|++.|.++
</pre><pre>T Consensus        26 n~kLying~~~R~Lt~~E~~e   46 (74)
</pre><pre>T A0A1I7VL88_LOA   26 NNKIYVGRTHVRDLTPEEVKE   46 (74)
</pre><pre>T ss_pred             CCEEEECCEEcccCCHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>6GZZ_W4 Ribosome hibernation promoting factor; Ribosome, hibernation, 100S, dimer, cryo-EM; 4.13A {Thermus thermophilus (strain HB8 / ATCC 27634 / DSM 579)}
</pre><pre>Probab=20.09  E-value=3.7e+02  Score=16.62  Aligned_cols=46  Identities=22%  Similarity=0.222  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             eEEEEcCCCcCCHHHHHHHhCCCcccccccEEeecCHHHeEEEECC
</pre><pre>Q 10K_MODEL_38     26 VFAVRAKGKPLSMADAADKFKIKQNHARNYIDFDIDTNRVEFRKND   71 (96)
</pre><pre>Q Consensus        26 Vf~e~Ak~KplSp~dae~kygI~~Grgr~yVEfDv~~~~~e~~~Np   71 (96)
</pre><pre>                      |-...-.-+|||..||-..+......---+...+-..-.+-+....
</pre><pre>T Consensus         4 v~~~~~~~~~ms~~eAi~~l~~~~~~f~~F~~~~~~~~~vvyrr~~   49 (57)
</pre><pre>T 6GZZ_W4           4 VRVKRFEMKPMDPEEAAFQMEALGHSFFVFRNAKTDEINVIYRRKD   49 (57)
</pre><pre>T ss_dssp             CCCCSCCCCCCCHHHHHHHHHHHSSSCCEEECTTTCCEEECCCCTT
</pre><pre>T ss_pred             eEEEEEeCcEECHHHHHHHHHHcCCcEEEEEECCCCcEEEEEEccC
</pre><pre>
</pre><pre>
</pre>