<pre>Query         10K_MODEL_32 
</pre><pre>Match_columns 115
</pre><pre>No_of_seqs    43 out of 48
</pre><pre>Neff          3.43619
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 17:16:17 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/7421405.hhr -oa3m ../results/7421405.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 d3bsua1 d.100.1.0 (A:27-73) au  71.7      14 0.00013   21.8   2.9   30   21-50     17-47  (47)
</pre><pre>  2 d1qhka_ d.100.1.2 (A:) N-termi  57.3      33 0.00031   20.6   2.6   29   21-49     18-47  (47)
</pre><pre>  3 PF18330.3 ; Lig_C ; Ligase Pab  53.1      43  0.0004   26.0   3.2   27   23-49     52-82  (121)
</pre><pre>  4 2LVB_A DE NOVO DESIGNED PFK fo  51.3      74 0.00069   25.6   4.2   40   13-52     57-99  (112)
</pre><pre>  5 4RSI_A Structural maintenance   50.7      83 0.00077   24.9   4.4   36   14-49    138-173 (397)
</pre><pre>  6 4U4P_B Structural maintenance   44.7 1.1E+02   0.001   22.5   4.0   33   19-52     65-97  (247)
</pre><pre>  7 PF15540.8 ; Ntox47 ; Bacterial  43.3      56 0.00052   22.5   2.3   41   73-115    60-101 (108)
</pre><pre>  8 4RSI_B Structural maintenance   41.5 1.3E+02  0.0012   23.8   4.2   40   13-54    141-180 (397)
</pre><pre>  9 1QHK_A PROTEIN (RIBONUCLEASE H  38.7 1.1E+02  0.0011   15.8   2.8   29   21-49     18-47  (47)
</pre><pre> 10 5H69_B Chromosome partition pr  37.1 1.6E+02  0.0015   21.9   4.0   36   19-54     74-109 (261)
</pre><pre> 11 1GXL_D CHROMOSOME SEGREGATION   36.5 2.1E+02  0.0019   20.9   4.3   34   19-52     56-89  (213)
</pre><pre> 12 6YWS_H Ribosomal protein L13;   34.5      42 0.00039   27.6   0.7   15   17-31    164-178 (183)
</pre><pre> 13 4LBH_A 5-chloro-2-hydroxyhydro  32.8      95 0.00089   21.0   2.1   15   36-50     60-74  (100)
</pre><pre> 14 PF08946.12 ; Osmo_CC ; Osmosen  31.0      76 0.00071   21.6   1.4   18   39-56      1-18  (46)
</pre><pre> 15 2WD5_A STRUCTURAL MAINTENANCE   30.6 2.6E+02  0.0024   21.3   4.2   34   19-52     68-101 (233)
</pre><pre> 16 7BQS_A Nomur; de novo designed  29.9      94 0.00088   25.0   1.9   38   13-51     34-72  (122)
</pre><pre> 17 5MG8_B Structural maintenance   29.4 2.5E+02  0.0023   22.2   4.1   37   19-55    136-172 (294)
</pre><pre> 18 3L51_A Structural maintenance   29.2 2.7E+02  0.0025   19.7   3.9   37   13-49     24-60  (161)
</pre><pre> 19 4U4P_A Structural maintenance   28.4 3.1E+02  0.0029   20.3   4.2   31   19-49     61-91  (233)
</pre><pre> 20 PF11363.10 ; DUF3164 ; Protein  28.3 1.4E+02  0.0013   24.6   2.7   22   17-38    150-176 (195)
</pre><pre> 21 d4lbha_ d.58.4.0 (A:) automate  27.7 1.3E+02  0.0012   20.2   2.0   15   36-50     60-74  (94)
</pre><pre> 22 PF03795.16 ; YCII ; YCII-relat  26.4 1.4E+02  0.0013   19.3   2.0   15   36-50     56-70  (88)
</pre><pre> 23 PF19026.2 ; HYPK_UBA ; HYPK UB  25.8 1.4E+02  0.0013   15.6   1.6   18   40-57     18-35  (41)
</pre><pre> 24 6WG6_J Structural maintenance   25.7 3.7E+02  0.0035   20.3   4.2   39   13-52     82-120 (256)
</pre><pre> 25 PF06470.15 ; SMC_hinge ; SMC p  25.6 3.9E+02  0.0036   17.9   4.7   41   18-58     15-56  (125)
</pre><pre> 26 1RP3_D anti sigma factor FlgM;  24.5 1.2E+02  0.0011   21.7   1.5   28   25-52     60-88  (88)
</pre><pre> 27 1RP3_F anti sigma factor FlgM;  24.5 1.2E+02  0.0011   21.7   1.5   28   25-52     60-88  (88)
</pre><pre> 28 d1mwqa_ d.58.4.7 (A:) Hypothet  24.0 1.6E+02  0.0015   20.2   2.0   15   36-50     65-79  (100)
</pre><pre> 29 TIGR04266 NDMA_methanol; NDMA-  23.3 4.2E+02  0.0039   23.7   4.7   44    8-51    370-415 (420)
</pre><pre> 30 4RSJ_D Chromosome partition pr  22.8 4.7E+02  0.0044   19.9   4.3   39   14-52     86-124 (276)
</pre><pre> 31 3BSU_A Ribonuclease H1; RNase   22.4 2.8E+02  0.0026   15.0   3.6   32   21-52     20-52  (53)
</pre><pre> 32 3BSU_G Ribonuclease H1; RNase   22.4 2.8E+02  0.0026   15.0   3.6   32   21-52     20-52  (53)
</pre><pre> 33 1MWQ_B HYPOTHETICAL PROTEIN HI  20.9 2.2E+02   0.002   19.3   2.1   15   36-50     66-80  (101)
</pre><pre> 34 5Z4T_A beta-1,4-mannanas; Comp  20.0 5.7E+02  0.0053   19.7   4.3   38   17-54     19-78  (309)
</pre><pre>
</pre><pre>No 1
</pre><pre>>d3bsua1 d.100.1.0 (A:27-73) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=71.68  E-value=14  Score=21.76  Aligned_cols=30  Identities=13%  Similarity=0.230  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCceeeCCHHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     21 WRLAVEAIGSGK-NTNFRVASEKDAIDLISQ   50 (115)
</pre><pre>Q Consensus        21 W~~AVk~ik~~g-~~n~RV~s~~dA~~LL~e   50 (115)
</pre><pre>                      |.++...+..-. ...-...|.+||+++|++
</pre><pre>T Consensus        17 w~~~~~~v~g~~~~~~k~f~s~~eA~~~~~~   47 (47)
</pre><pre>T d3bsua1          17 WNECRAQVDRFPAARFKKFATEDEAWAFVRK   47 (47)
</pre><pre>T ss_dssp             HHHHHHHHTTCTTCEEEEESSHHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHhCCCCCCeEEccCCHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>d1qhka_ d.100.1.2 (A:) N-terminal domain of RNase HI {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=57.26  E-value=33  Score=20.63  Aligned_cols=29  Identities=10%  Similarity=0.165  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCceeeCCHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     21 WRLAVEAIGSGK-NTNFRVASEKDAIDLIS   49 (115)
</pre><pre>Q Consensus        21 W~~AVk~ik~~g-~~n~RV~s~~dA~~LL~   49 (115)
</pre><pre>                      |.+|.+.+..-. ..--...|.+||+++|+
</pre><pre>T Consensus        18 w~~~~~~v~~~~~~~~k~f~t~~eA~~~~~   47 (47)
</pre><pre>T d1qhka_          18 WNECKNQVDGYGGAIYKKFNSYEQAKSFLG   47 (47)
</pre><pre>T ss_dssp             HHHHHHHSSSCSSCCCEEESCHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHhCCCCCCEEEeeCCHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>PF18330.3 ; Lig_C ; Ligase Pab1020 C-terminal region
</pre><pre>Probab=53.08  E-value=43  Score=26.02  Aligned_cols=27  Identities=7%  Similarity=0.000  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHcCC----CCceeeCCHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     23 LAVEAIGSGK----NTNFRVASEKDAIDLIS   49 (115)
</pre><pre>Q Consensus        23 ~AVk~ik~~g----~~n~RV~s~~dA~~LL~   49 (115)
</pre><pre>                      +||+.++.++    .+.+|+.+.+.|++||+
</pre><pre>T Consensus        52 e~I~~v~~g~~v~e~f~~rf~~~~~~e~~l~   82 (121)
</pre><pre>T O29409_ARCFU/2   52 ESIRECKRGEMVCEESELVFESPDVLELFKL   82 (121)
</pre><pre>T ss_pred             HHHHHHHCCCCeeEEEEEEeCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>2LVB_A DE NOVO DESIGNED PFK fold PROTEIN; Structural Genomics, NORTHEAST STRUCTURAL GENOMICS CONSORTIUM (NESG), PSI-Biology, Protein Structure Initiative, DE NOVO PROTEIN; NMR {artificial gene}
</pre><pre>Probab=51.33  E-value=74  Score=25.63  Aligned_cols=40  Identities=0%  Similarity=-0.075  Sum_probs=0.0  Template_Neff=3.500
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHH-HHHHHHHHcCC--CCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     13 VIRPGTKEW-RLAVEAIGSGK--NTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        13 vikpGTk~W-~~AVk~ik~~g--~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      +|+...++| +++++.++..|  =--.+|.|.+||++.|+++.
</pre><pre>T Consensus        57 ~V~~~dke~l~k~ie~akk~g~g~eVr~vTd~dE~kk~v~ef~   99 (112)
</pre><pre>T 2LVB_A           57 FVNGASQNDVNEFQNEAKKEGVSYDVLKSTDPEELTQRVREFL   99 (112)
</pre><pre>T ss_dssp             EEESCCHHHHHHHHHHHHTTTCEEEEEECCCHHHHHHHHHHHH
</pre><pre>T ss_pred             EECCCChHHHHHHHHHHHHcCCCcEEEEcCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>4RSI_A Structural maintenance of chromosomes protein 2; Smc hinge domain with coiled coil, chromosomal condensation, CELL CYCLE; HET: PO4; 2.9A {Saccharomyces cerevisiae}
</pre><pre>Probab=50.68  E-value=83  Score=24.86  Aligned_cols=36  Identities=19%  Similarity=0.291  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             cCCCCHHHHHHHHHHHcCCCCceeeCCHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     14 IRPGTKEWRLAVEAIGSGKNTNFRVASEKDAIDLIS   49 (115)
</pre><pre>Q Consensus        14 ikpGTk~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~   49 (115)
</pre><pre>                      ++++.+.|..||+.+-...-..|-|.+..|+..|+.
</pre><pre>T Consensus       138 v~~~~~~~~~ave~~l~~~l~~~vv~~~~~~~~l~~  173 (397)
</pre><pre>T 4RSI_A          138 IDNDNIRYATALQTCAGGRLFNVVVQDSQTATQLLE  173 (397)
</pre><pre>T ss_dssp             CCTGGGGGHHHHHHHHGGGGGCEEESSHHHHHHHHH
</pre><pre>T ss_pred             ecCCChHHHHHHHHHhcccceeEEECCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>4U4P_B Structural maintenance of chromosomes protein 4; Condensin, SMC, PROTEIN BINDING; 1.89A {Homo sapiens}
</pre><pre>Probab=44.70  E-value=1.1e+02  Score=22.50  Aligned_cols=33  Identities=12%  Similarity=0.076  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     19 KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        19 k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      +.|..||+.+ -..-..|-|.+..+|..++....
</pre><pre>T Consensus        65 ~~~~~ave~~-~~~~~~~vv~~~~~~~~~~~~~~   97 (247)
</pre><pre>T 4U4P_B           65 EKYDVAISSC-CHALDYIVVDSIDIAQECVNFLK   97 (247)
</pre><pre>T ss_dssp             GGGHHHHHHH-CGGGGSEEESCHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHH-chhcCEEEECCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>PF15540.8 ; Ntox47 ; Bacterial toxin 47
</pre><pre>Probab=43.30  E-value=56  Score=22.50  Aligned_cols=41  Identities=29%  Similarity=0.514  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCCcccCCCCcccCCCCCCCCCcceEEecccCcc-cCCCceecC
</pre><pre>Q 10K_MODEL_32     73 KKGYEVHPNESHTQNAPENNIPHVKWKDWESGKS-SGGRGHVFF  115 (115)
</pre><pre>Q Consensus        73 kkGYE~Hp~e~~~~~ap~NdLpHIKWkDw~~gk~-~g~~GHIFy  115 (115)
</pre><pre>                      +.|-+++....|...  +.+-|||.|.--..... ....|||++
</pre><pre>T Consensus        60 ~~G~~~~~d~~~~~~--g~~~~h~~~~~~~~~~~~~~~~g~i~~  101 (108)
</pre><pre>T Q65M69_BACLD/4   60 KNGAEVNIDVGHTKN--GPDVPHTGYQTAGKRGGGGAIRGHILV  101 (108)
</pre><pre>T ss_pred             CCCCEEEEEeccCCC--CCCCCceeeEeCcccCCCCcccceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>4RSI_B Structural maintenance of chromosomes protein 4; Smc hinge domain with coiled coil, chromosomal condensation, CELL CYCLE; HET: PO4; 2.9A {Saccharomyces cerevisiae}
</pre><pre>Probab=41.49  E-value=1.3e+02  Score=23.78  Aligned_cols=40  Identities=13%  Similarity=0.026  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhccc
</pre><pre>Q 10K_MODEL_32     13 VIRPGTKEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGRGR   54 (115)
</pre><pre>Q Consensus        13 vikpGTk~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~eargn   54 (115)
</pre><pre>                      .|+. .+.|..||+.+- ..-..|.|.+..|+..++...+..
</pre><pre>T Consensus       141 ~i~v-~~~~~~ave~~l-~~l~~~vv~~~~~~~~~~~~~~~~  180 (397)
</pre><pre>T 4RSI_B          141 LGVI-DDSFDVAISTAC-PRLDDVVVDTVECAQHCIDYLRKN  180 (397)
</pre><pre>T ss_dssp             SEEE-CGGGHHHHHHHC-GGGGSEEESCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hccC-CHHHHHHHHHHc-hhcCeEEeCCHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>1QHK_A PROTEIN (RIBONUCLEASE HI); RIBONUCLEASE HI N-TERMINAL DOMAIN, HYDROLASE; NMR {Saccharomyces cerevisiae} SCOP: d.100.1.2
</pre><pre>Probab=38.72  E-value=1.1e+02  Score=15.84  Aligned_cols=29  Identities=10%  Similarity=0.165  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCceeeCCHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     21 WRLAVEAIGSGK-NTNFRVASEKDAIDLIS   49 (115)
</pre><pre>Q Consensus        21 W~~AVk~ik~~g-~~n~RV~s~~dA~~LL~   49 (115)
</pre><pre>                      |..+...+..-. ...-+..+.++|.+++.
</pre><pre>T Consensus        18 ~~~~~~~~~~~~~~~~~~~~~~~~a~~~~~   47 (47)
</pre><pre>T 1QHK_A           18 WNECKNQVDGYGGAIYKKFNSYEQAKSFLG   47 (47)
</pre><pre>T ss_dssp             HHHHHHHSSSCSSCCCEEESCHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHcCCCCCeEeeeCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>5H69_B Chromosome partition protein Smc; SMC protein, DNA BINDING PROTEIN, CELL CYCLE; 2.20040022156A {Geobacillus stearothermophilus 10}
</pre><pre>Probab=37.13  E-value=1.6e+02  Score=21.89  Aligned_cols=36  Identities=22%  Similarity=0.211  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhccc
</pre><pre>Q 10K_MODEL_32     19 KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGRGR   54 (115)
</pre><pre>Q Consensus        19 k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~eargn   54 (115)
</pre><pre>                      +.|..||+.+-...-..|-|.+..|+..++...+..
</pre><pre>T Consensus        74 ~~~~~ave~~lg~~l~~~vv~~~~~~~~~~~~~~~~  109 (261)
</pre><pre>T 5H69_B           74 DRYETAIETALGGAMQHIVVDSEQAARQAIHYLKTN  109 (261)
</pre><pre>T ss_dssp             GGGHHHHHHHHGGGGGCEEESCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHhhhhcceEEeCCHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>1GXL_D CHROMOSOME SEGREGATION SMC PROTEIN; CHROMOSOME SEGREGATION, SMC DIMERISATION DOMAIN, ANTI PARALLEL COILED COIL, SMC PROTEINS; 3.0A {THERMOTOGA MARITIMA} SCOP: d.215.1.1
</pre><pre>Probab=36.51  E-value=2.1e+02  Score=20.95  Aligned_cols=34  Identities=21%  Similarity=0.281  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     19 KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        19 k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      +.|..||+.+-...-..+-|.+..+|..+++...
</pre><pre>T Consensus        56 ~~~~~a~e~~lg~~~~~~vv~~~~~~~~~~~~~~   89 (213)
</pre><pre>T 1GXL_D           56 EKYSLAVSVLLGGTAQNIVVRNVDTAKAIVEFLK   89 (213)
</pre><pre>T ss_dssp             HHHHHHHHHHHGGGGGCEEESSHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhccccCeEEeCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>6YWS_H Ribosomal protein L13; Neurospora crassa, translating Mitoribosomes, tRNA, mRNA, mL108, TRANSLATION; HET: SPM, K, NAD, MG;{Neurospora crassa OR74A}
</pre><pre>Probab=34.53  E-value=42  Score=27.59  Aligned_cols=15  Identities=40%  Similarity=0.683  Sum_probs=0.0  Template_Neff=6.800
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHcC
</pre><pre>Q 10K_MODEL_32     17 GTKEWRLAVEAIGSG   31 (115)
</pre><pre>Q Consensus        17 GTk~W~~AVk~ik~~   31 (115)
</pre><pre>                      |.+.|.+||++|+++
</pre><pre>T Consensus       164 ~~~~~~~~~~~~~~~  178 (183)
</pre><pre>T 6YWS_H          164 GAPGWEEAVKAIREA  178 (183)
</pre><pre>T ss_dssp             TSTTHHHHHHHHHHT
</pre><pre>T ss_pred             CCCChHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>4LBH_A 5-chloro-2-hydroxyhydroquinone dehydrochlorinase (TftG); LYASE; 1.75A {Burkholderia cepacia} SCOP: d.58.4.0
</pre><pre>Probab=32.80  E-value=95  Score=21.00  Aligned_cols=15  Identities=7%  Similarity=0.171  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             eeeCCHHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     36 FRVASEKDAIDLISQ   50 (115)
</pre><pre>Q Consensus        36 ~RV~s~~dA~~LL~e   50 (115)
</pre><pre>                      |++.|.++|++||++
</pre><pre>T Consensus        60 v~a~~~~ea~~~~~~   74 (100)
</pre><pre>T 4LBH_A           60 MEAESWDEVHSFVEN   74 (100)
</pre><pre>T ss_dssp             EECSSHHHHHHHHHT
</pre><pre>T ss_pred             EEeCCHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF08946.12 ; Osmo_CC ; Osmosensory transporter coiled coil
</pre><pre>Probab=30.97  E-value=76  Score=21.64  Aligned_cols=18  Identities=6%  Similarity=0.257  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHhccccc
</pre><pre>Q 10K_MODEL_32     39 ASEKDAIDLISQGRGRMQ   56 (115)
</pre><pre>Q Consensus        39 ~s~~dA~~LL~eargnm~   56 (115)
</pre><pre>                      .+.++|+.||.+...++.
</pre><pre>T Consensus         1 S~~~EAkelL~e~~d~IE   18 (46)
</pre><pre>T PROP_ECOLI/454    1 SDIQEAKEILVEHYDNIE   18 (46)
</pre><pre>T ss_pred             CcHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>2WD5_A STRUCTURAL MAINTENANCE OF CHROMOSOMES PROTEIN 1A; DNA DAMAGE, CELL CYCLE, CELL DIVISION; 2.7A {MUS MUSCULUS}
</pre><pre>Probab=30.64  E-value=2.6e+02  Score=21.29  Aligned_cols=34  Identities=26%  Similarity=0.267  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     19 KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        19 k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      +.|..||+.+-...-..+.|.+..+++.++....
</pre><pre>T Consensus        68 ~~~~~ave~~l~~~l~~~vv~~~~~~~~~~~~~~  101 (233)
</pre><pre>T 2WD5_A           68 KKYQIAVTKVLGKNMDAIIVDSEKTGRDCIQYIK  101 (233)
</pre><pre>T ss_dssp             GGGHHHHHHHHGGGGSCEEESCHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhcccccEEEeCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>7BQS_A Nomur; de novo designed proteins, DE NOVO PROTEIN; NMR {synthetic construct}
</pre><pre>Probab=29.93  E-value=94  Score=24.99  Aligned_cols=38  Identities=21%  Similarity=0.372  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             ccCCCC-HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHh
</pre><pre>Q 10K_MODEL_32     13 VIRPGT-KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQG   51 (115)
</pre><pre>Q Consensus        13 vikpGT-k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~ea   51 (115)
</pre><pre>                      |++-.+ .+|.+|.+.|-++ -..---.+-+.|++||.+|
</pre><pre>T Consensus        34 vlktaspeqwrqaaekifea-freasngntekakklleea   72 (122)
</pre><pre>T 7BQS_A           34 VLKTASPEQWRQAAEKIFEA-FREASNGNTEKAKKLLEEA   72 (122)
</pre><pre>T ss_dssp             HHTTSCHHHHHHHHHHHHHH-HHHHHHTCHHHHHHHHHHH
</pre><pre>T ss_pred             HHHhCCHHHHHHHHHHHHHH-HHHHhcCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>5MG8_B Structural maintenance of chromosomes protein 6; SMC, Structural Maintenance of Chromosomes, Hinge Domain, Smc5, Smc6, Smc5/6, recombination; HET: SO4, GOL; 2.75A {Schizosaccharomyces pombe}
</pre><pre>Probab=29.38  E-value=2.5e+02  Score=22.23  Aligned_cols=37  Identities=27%  Similarity=0.326  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhcccc
</pre><pre>Q 10K_MODEL_32     19 KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGRGRM   55 (115)
</pre><pre>Q Consensus        19 k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~eargnm   55 (115)
</pre><pre>                      +.|..||+.+-...-..|-|.+..+|..++...+.+.
</pre><pre>T Consensus       136 ~~~~~ave~~lg~~l~~~vv~~~~~~~~~~~~l~~~~  172 (294)
</pre><pre>T 5MG8_B          136 QKWHLIIERILGNVINGFIVRSHHDQLILKELMRQSN  172 (294)
</pre><pre>T ss_dssp             GGGHHHHHHHSCSGGGSEEESSHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHcccccEEEeCCHHHHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>3L51_A Structural maintenance of chromosomes protein 2; Structural Maintenance of Chromosomes (SMC), hinge domain, Cell cycle, Cell division, Cytoplasm, DNA condensation, Mitosis, Nucleus; HET: GOL, MSE; 1.506A {Mus musculus}
</pre><pre>Probab=29.24  E-value=2.7e+02  Score=19.72  Aligned_cols=37  Identities=22%  Similarity=0.217  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHcCCCCceeeCCHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     13 VIRPGTKEWRLAVEAIGSGKNTNFRVASEKDAIDLIS   49 (115)
</pre><pre>Q Consensus        13 vikpGTk~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~   49 (115)
</pre><pre>                      .+....+.|..||+.+-...-..+.|.+.+++.++++
</pre><pre>T Consensus        24 ~~~~~~~~~~~a~e~~lg~~~~~~i~~~~~~~~~~~~   60 (161)
</pre><pre>T 3L51_A           24 LINVKDNSTATALEVVAGERLYNVVVDTEVTAKKLLE   60 (161)
</pre><pre>T ss_dssp             SCEESCGGGHHHHHHHHGGGGGCEEESCHHHHHHHHH
</pre><pre>T ss_pred             hEEeCChhHHHHHHHHhhhhhccEEECCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>4U4P_A Structural maintenance of chromosomes protein 2; Condensin, SMC, PROTEIN BINDING; 1.89A {Homo sapiens}
</pre><pre>Probab=28.42  E-value=3.1e+02  Score=20.33  Aligned_cols=31  Identities=19%  Similarity=0.186  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCCCceeeCCHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     19 KEWRLAVEAIGSGKNTNFRVASEKDAIDLIS   49 (115)
</pre><pre>Q Consensus        19 k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~   49 (115)
</pre><pre>                      ..|..||+.+-...-..|-|.+..+|..++.
</pre><pre>T Consensus        61 ~~~~~a~e~~lg~~~~~~vv~~~~~~~~~~~   91 (233)
</pre><pre>T 4U4P_A           61 TSATTALELVAGERLYNVVVDTEVTGKKLLE   91 (233)
</pre><pre>T ss_dssp             GGGHHHHHHHHGGGGGCEEESCHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHhccccceEEECCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF11363.10 ; DUF3164 ; Protein of unknown function (DUF3164)
</pre><pre>Probab=28.28  E-value=1.4e+02  Score=24.61  Aligned_cols=22  Identities=32%  Similarity=0.516  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHcC----C-CCceee
</pre><pre>Q 10K_MODEL_32     17 GTKEWRLAVEAIGSG----K-NTNFRV   38 (115)
</pre><pre>Q Consensus        17 GTk~W~~AVk~ik~~----g-~~n~RV   38 (115)
</pre><pre>                      +.+.|.+|++.|+++    + +..||+
</pre><pre>T Consensus       150 ~d~~w~~a~~~i~~s~~~~~sk~Yir~  176 (195)
</pre><pre>T Q6QIE4_BPBMU/1  150 TDPRWLEAMRAIGEALQVIGSKSYVRV  176 (195)
</pre><pre>T ss_pred             CCHHHHHHHHHHHHhceeecCcceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>d4lbha_ d.58.4.0 (A:) automated matches {Burkholderia cepacia [TaxId: 292]}
</pre><pre>Probab=27.72  E-value=1.3e+02  Score=20.23  Aligned_cols=15  Identities=7%  Similarity=0.171  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             eeeCCHHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     36 FRVASEKDAIDLISQ   50 (115)
</pre><pre>Q Consensus        36 ~RV~s~~dA~~LL~e   50 (115)
</pre><pre>                      |+++|.++|++++++
</pre><pre>T Consensus        60 ~~a~s~~ea~~~~~~   74 (94)
</pre><pre>T d4lbha_          60 MEAESWDEVHSFVEN   74 (94)
</pre><pre>T ss_dssp             EECSSHHHHHHHHHT
</pre><pre>T ss_pred             EEeCCHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>PF03795.16 ; YCII ; YCII-related domain
</pre><pre>Probab=26.37  E-value=1.4e+02  Score=19.35  Aligned_cols=15  Identities=7%  Similarity=0.056  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             eeeCCHHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     36 FRVASEKDAIDLISQ   50 (115)
</pre><pre>Q Consensus        36 ~RV~s~~dA~~LL~e   50 (115)
</pre><pre>                      |.+.|.++|+++|+.
</pre><pre>T Consensus        56 ~~a~~~~ea~~~~~~   70 (88)
</pre><pre>T Q9AJZ7_STRCO/1   56 LDLPDPAAARTFAFD   70 (88)
</pre><pre>T ss_pred             EEeCCHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>PF19026.2 ; HYPK_UBA ; HYPK UBA domain
</pre><pre>Probab=25.77  E-value=1.4e+02  Score=15.58  Aligned_cols=18  Identities=17%  Similarity=0.147  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHhcccccc
</pre><pre>Q 10K_MODEL_32     40 SEKDAIDLISQGRGRMQR   57 (115)
</pre><pre>Q Consensus        40 s~~dA~~LL~eargnm~r   57 (115)
</pre><pre>                      +..+|++.|.+..+++..
</pre><pre>T Consensus        18 ~~~~~~~~l~~~~~d~~~   35 (41)
</pre><pre>T A0A226N5M4_CAL   18 SRAAAERSLREHMGNVVE   35 (41)
</pre><pre>T ss_pred             CHHHHHHHHHHcCCCHHH
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>6WG6_J Structural maintenance of chromosomes protein 3; Protein-DNA complex, ATPase, DNA-binding protein, Genome organization, Sister chromatid cohesion, Transcription regulation, CELL CYCLE, CELL CYCLE-DNA; 3.54A {Homo sapiens}
</pre><pre>Probab=25.68  E-value=3.7e+02  Score=20.27  Aligned_cols=39  Identities=10%  Similarity=0.054  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     13 VIRPGTKEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        13 vikpGTk~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      .++. .+.|..||+.+-...-..|-|.+..++..++...+
</pre><pre>T Consensus        82 ~i~v-~~~~~~ave~~l~~~~~~~vv~~~~~~~~~~~~l~  120 (256)
</pre><pre>T 6WG6_J           82 NFEC-EPAFYTCVEVTAGNRLFYHIVDSDEVSTKILMEFN  120 (256)
</pre><pre>T ss_dssp             GCBC-CGGGHHHHHHHHGGGGGCEEESSHHHHHHHHHHHH
</pre><pre>T ss_pred             heec-CHHHHHHHHHHhcccceEEEeCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>PF06470.15 ; SMC_hinge ; SMC proteins Flexible Hinge Domain
</pre><pre>Probab=25.58  E-value=3.9e+02  Score=17.90  Aligned_cols=41  Identities=15%  Similarity=0.238  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             C-HHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhccccccC
</pre><pre>Q 10K_MODEL_32     18 T-KEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGRGRMQRQ   58 (115)
</pre><pre>Q Consensus        18 T-k~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~eargnm~r~   58 (115)
</pre><pre>                      . +.|..|++.+-...-..+-|.+..++.++++.........
</pre><pre>T Consensus        15 ~~~~~~~a~e~~l~~~~~~~vv~~~~~~~~~~~~~~~~~~~~   56 (125)
</pre><pre>T Q8SS38_ENCCU/5   15 TQDRYEIALSVLLGSHDQSVIVDTERTAMSCINFIKEKRLCK   56 (125)
</pre><pre>T ss_pred             ChHHHHHHHHHHhcccccEEEeCCHHHHHHHHHHHHHcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>1RP3_D anti sigma factor FlgM; TRANSCRIPTION, SIGMA FACTOR; 2.3A {Aquifex aeolicus} SCOP: a.137.11.1
</pre><pre>Probab=24.46  E-value=1.2e+02  Score=21.72  Aligned_cols=28  Identities=21%  Similarity=0.239  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcC-CCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     25 VEAIGSG-KNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        25 Vk~ik~~-g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      |+.|+.. ..+.+.|.+.+=|+.||...+
</pre><pre>T Consensus        60 V~~ik~~I~~G~Y~vd~~~iA~~il~~~~   88 (88)
</pre><pre>T 1RP3_D           60 VKELKEKIEKGEYEVSDEKVVKGLIEFFT   88 (88)
</pre><pre>T ss_dssp             HHHHHHHHHTTCCCCCHHHHHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHcCCCCCCHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>1RP3_F anti sigma factor FlgM; TRANSCRIPTION, SIGMA FACTOR; 2.3A {Aquifex aeolicus} SCOP: a.137.11.1
</pre><pre>Probab=24.46  E-value=1.2e+02  Score=21.72  Aligned_cols=28  Identities=21%  Similarity=0.239  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcC-CCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     25 VEAIGSG-KNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        25 Vk~ik~~-g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      |+.|+.. ..+.+.|.+.+=|+.||...+
</pre><pre>T Consensus        60 V~~ik~~I~~G~Y~vd~~~iA~~il~~~~   88 (88)
</pre><pre>T 1RP3_F           60 VKELKEKIEKGEYEVSDEKVVKGLIEFFT   88 (88)
</pre><pre>T ss_dssp             HHHHHHHHHTTCCCCCHHHHHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHcCCCCCCHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>d1mwqa_ d.58.4.7 (A:) Hypothetical protein HI0828 {Haemophilus influenzae [TaxId: 727]}
</pre><pre>Probab=24.00  E-value=1.6e+02  Score=20.23  Aligned_cols=15  Identities=20%  Similarity=0.133  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             eeeCCHHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     36 FRVASEKDAIDLISQ   50 (115)
</pre><pre>Q Consensus        36 ~RV~s~~dA~~LL~e   50 (115)
</pre><pre>                      |+++|.++|++++++
</pre><pre>T Consensus        65 ~~a~s~eea~~i~~~   79 (100)
</pre><pre>T d1mwqa_          65 AQFENLQAAKDWAAQ   79 (100)
</pre><pre>T ss_dssp             EECSSHHHHHHHHHT
</pre><pre>T ss_pred             EEcCCHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>TIGR04266 NDMA_methanol; NDMA-dependent methanol dehydrogenase. Members of this family belong to the iron-dependent alcohol dehydrogenase family (see pfam00465).
</pre><pre>Probab=23.32  E-value=4.2e+02  Score=23.73  Aligned_cols=44  Identities=25%  Similarity=0.246  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             hhhCCccCCCCHHHHHHHHHHHcCC--CCceeeCCHHHHHHHHHHh
</pre><pre>Q 10K_MODEL_32      8 ASNLPVIRPGTKEWRLAVEAIGSGK--NTNFRVASEKDAIDLISQG   51 (115)
</pre><pre>Q Consensus         8 As~lpvikpGTk~W~~AVk~ik~~g--~~n~RV~s~~dA~~LL~ea   51 (115)
</pre><pre>                      ...+|.+.--.+.....++......  ..|-|..+.+|.+++|+.|
</pre><pre>T Consensus       370 ~~~~~~~~~~~~~~~~la~~a~~~~~~~~np~~~~~~~~~~i~~~~  415 (420)
</pre><pre>T TIGR04266       370 SGRGPVIKGDDKTVRAISEHIQGDWCTPGNPREVTVESMIPVVSHA  415 (420)
</pre><pre>T ss_pred             CCCCCccCCChHHHHHHHHHHhcCCCCCCCCcCCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>4RSJ_D Chromosome partition protein Smc; Smc hinge domain with coiled coils, CELL CYCLE; 3.5A {Pyrococcus furiosus}
</pre><pre>Probab=22.76  E-value=4.7e+02  Score=19.95  Aligned_cols=39  Identities=23%  Similarity=0.205  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             cCCCCHHHHHHHHHHHcCCCCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     14 IRPGTKEWRLAVEAIGSGKNTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        14 ikpGTk~W~~AVk~ik~~g~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      |+.-.+.|..||+.+-...-..|-|.+..|+..++...+
</pre><pre>T Consensus        86 i~v~~~~~~~a~e~~l~~~~~~~vv~~~~d~~~~~~~~~  124 (276)
</pre><pre>T 4RSJ_D           86 IKVKDEAYALAIEVALGNRADNVVVEDELVAEKAIKYLK  124 (276)
</pre><pre>T ss_dssp             CEESSGGGHHHHHHHHGGGGSCEEESCHHHHHHHHHHHH
</pre><pre>T ss_pred             eecChHHHHHHHHHHhccccCEEEECCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>3BSU_A Ribonuclease H1; RNase H, RNA/DNA hybrid, dsRNA, HYDROLASE-RNA-DNA COMPLEX; HET: 5IU; 2.1A {Homo sapiens} SCOP: l.1.1.1, d.100.1.0
</pre><pre>Probab=22.42  E-value=2.8e+02  Score=15.05  Aligned_cols=32  Identities=13%  Similarity=0.213  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     21 WRLAVEAIGSGK-NTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        21 W~~AVk~ik~~g-~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      |.++...+..-. ..--+..+.++|.+++....
</pre><pre>T Consensus        20 ~~~~~~~~~~~~~~~~~~~~~~~~a~~~~~~~~   52 (53)
</pre><pre>T 3BSU_A           20 WNECRAQVDRFPAARFKKFATEDEAWAFVRKSA   52 (53)
</pre><pre>T ss_dssp             HHHHHHHHTTCTTCEEEEESSHHHHHHHHHC--
</pre><pre>T ss_pred             HHHHHHHhcCCCCCEEEeeCCHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>3BSU_G Ribonuclease H1; RNase H, RNA/DNA hybrid, dsRNA, HYDROLASE-RNA-DNA COMPLEX; HET: 5IU; 2.1A {Homo sapiens} SCOP: d.100.1.0
</pre><pre>Probab=22.42  E-value=2.8e+02  Score=15.05  Aligned_cols=32  Identities=13%  Similarity=0.213  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCceeeCCHHHHHHHHHHhc
</pre><pre>Q 10K_MODEL_32     21 WRLAVEAIGSGK-NTNFRVASEKDAIDLISQGR   52 (115)
</pre><pre>Q Consensus        21 W~~AVk~ik~~g-~~n~RV~s~~dA~~LL~ear   52 (115)
</pre><pre>                      |.++...+..-. ..--+..+.++|.+++....
</pre><pre>T Consensus        20 ~~~~~~~~~~~~~~~~~~~~~~~~a~~~~~~~~   52 (53)
</pre><pre>T 3BSU_G           20 WNECRAQVDRFPAARFKKFATEDEAWAFVRKSA   52 (53)
</pre><pre>T ss_dssp             HHHHHHHHTTCTTCEEEEESSHHHHHHHHTC--
</pre><pre>T ss_pred             HHHHHHHhcCCCCCEEEeeCCHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>1MWQ_B HYPOTHETICAL PROTEIN HI0828; HI0828, YCII_HAEIN, hypothetical protein, structural genomics, Structure 2 Function Project, S2F, UNKNOWN FUNCTION; HET: MSE, 1PE; 0.99A {Haemophilus influenzae} SCOP: d.58.4.7
</pre><pre>Probab=20.87  E-value=2.2e+02  Score=19.29  Aligned_cols=15  Identities=20%  Similarity=0.133  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             eeeCCHHHHHHHHHH
</pre><pre>Q 10K_MODEL_32     36 FRVASEKDAIDLISQ   50 (115)
</pre><pre>Q Consensus        36 ~RV~s~~dA~~LL~e   50 (115)
</pre><pre>                      |.+.|.++|++++++
</pre><pre>T Consensus        66 ~~a~~~~~a~~~~~~   80 (101)
</pre><pre>T 1MWQ_B           66 AQFENLQAAKDWAAQ   80 (101)
</pre><pre>T ss_dssp             EECSSHHHHHHHHHT
</pre><pre>T ss_pred             EEcCCHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>5Z4T_A beta-1,4-mannanas; Complex, HYDROLASE; HET: BMA; 1.68A {Amphibacillus xylanus (strain ATCC 51415 / DSM 6626 / JCM 7361 / LMG 17667 / NBRC 15112 / Ep01)}
</pre><pre>Probab=20.02  E-value=5.7e+02  Score=19.67  Aligned_cols=38  Identities=16%  Similarity=0.212  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHcCCC-CceeeC---------------------CHHHHHHHHHHhccc
</pre><pre>Q 10K_MODEL_32     17 GTKEWRLAVEAIGSGKN-TNFRVA---------------------SEKDAIDLISQGRGR   54 (115)
</pre><pre>Q Consensus        17 GTk~W~~AVk~ik~~g~-~n~RV~---------------------s~~dA~~LL~eargn   54 (115)
</pre><pre>                      ....++..++.|++.|- .-||+.                     +.+....+|..|+..
</pre><pre>T Consensus        19 ~~~~~~~~~~~l~~~G~~n~vR~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~i~~a~~~   78 (309)
</pre><pre>T 5Z4T_A           19 KTERAKESLRLMQERTASEYVIVALAALQDTAHSTEVDFQGSHMVDDDELIELIDYAKSL   78 (309)
</pre><pre>T ss_dssp             GSHHHHHHHHHHHHHHCCSEEEEEEEEEESSTTCCCCBCSSTTSCCHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CCHHHHHHHHHHHHCCCCCeEEEEeEEeecCCCCCCceecCCCCCCHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre>