<pre>Query         XP_021345007.1/4-96 
</pre><pre>Match_columns 93
</pre><pre>No_of_seqs    49 out of 54
</pre><pre>Neff          4.5525
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 19:42:38 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/6412114.hhr -oa3m ../results/6412114.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 d1vz0a2 d.268.1.1 (A:23-115) P  97.1  0.0014 1.3E-08   41.4   3.4   77    1-91      3-80  (93)
</pre><pre>  2 TIGR00180 parB_part; ParB/RepB  94.1    0.12 1.1E-06   37.4   2.8   55   29-91     29-87  (187)
</pre><pre>  3 6SDK_B Stage 0 sporulation pro  93.9    0.14 1.3E-06   36.4   2.8   77    1-91      6-83  (199)
</pre><pre>  4 d1xw3a1 d.268.1.4 (A:32-137) S  93.0    0.24 2.2E-06   33.1   2.7   62   23-91     25-92  (106)
</pre><pre>  5 4UMK_C PROBABLE CHROMOSOME-PAR  92.5    0.12 1.1E-06   38.6   0.7   76    2-91     36-112 (240)
</pre><pre>  6 6RYK_B ParB-like nuclease doma  92.1    0.42 3.9E-06   35.8   3.2   74    2-91     37-112 (229)
</pre><pre>  7 6KY4_A Sulfiredoxin, chloropla  91.6    0.74 6.9E-06   30.0   3.6   61   23-91     36-97  (112)
</pre><pre>  8 1VZ0_C Chromosome-partitioning  91.1    0.12 1.1E-06   38.1  -0.4   76    2-91     26-102 (230)
</pre><pre>  9 1VZ0_F Chromosome-partitioning  91.1    0.12 1.1E-06   38.1  -0.4   76    2-91     26-102 (230)
</pre><pre> 10 TIGR04285 nucleoid_noc; nucleo  89.1    0.98 9.1E-06   34.7   2.9   55   29-91     26-81  (255)
</pre><pre> 11 7NFU_B Nucleoid occlusion prot  87.4    0.36 3.3E-06   36.7  -0.3   77    1-91     30-108 (252)
</pre><pre> 12 7BM8_A Chromosome-partitioning  85.7    0.39 3.6E-06   36.5  -0.9   72    2-87     37-112 (257)
</pre><pre> 13 TIGR03454 partition_RepB; plas  85.4     1.7 1.6E-05   35.7   2.5   76    1-91     66-145 (325)
</pre><pre> 14 TIGR03764 ICE_PFGI_1_parB; int  85.1     3.5 3.3E-05   32.1   4.0   72    1-87     10-92  (258)
</pre><pre> 15 PF08857.13 ; ParBc_2 ; Putativ  84.4     1.1 9.8E-06   33.4   0.9   72    1-88      2-76  (153)
</pre><pre> 16 PF02195.20 ; ParBc ; ParB-like  83.8     3.3 3.1E-05   24.8   2.7   51   29-87     24-76  (91)
</pre><pre> 17 1XW3_A SULFIREDOXIN; retroredu  81.5     5.5 5.1E-05   25.4   3.2   56   29-91     35-96  (110)
</pre><pre> 18 2HWJ_F Hypothetical protein At  81.2     1.3 1.2E-05   34.7   0.3   76    2-92     17-94  (205)
</pre><pre> 19 d1vk1a_ d.268.1.2 (A:) Hypothe  76.4     6.3 5.9E-05   30.3   2.7   61   24-91     22-85  (232)
</pre><pre> 20 5K5D_A ParB domain protein nuc  72.3     7.7 7.1E-05   31.3   2.4   72    2-90     14-86  (300)
</pre><pre> 21 5K5D_B ParB domain protein nuc  72.3     7.7 7.1E-05   31.3   2.4   72    2-90     14-86  (300)
</pre><pre> 22 d2hwja1 d.268.1.3 (A:4-204) Hy  67.8      15 0.00014   28.9   3.1   76    2-92     14-91  (201)
</pre><pre> 23 6CKA_B Paratox; prophage, natu  59.8      56 0.00052   22.8   4.2   31   26-59      1-31  (68)
</pre><pre> 24 TIGR03734 PRTRC_parB; PRTRC sy  58.0      17 0.00016   32.1   2.0   71   13-91      1-76  (554)
</pre><pre> 25 d3amra_ b.68.3.1 (A:) automate  49.0      37 0.00034   28.5   2.5   21   49-69    184-205 (352)
</pre><pre> 26 d1zyba1 a.4.5.4 (A:148-220) Pr  48.2      95 0.00088   18.5   3.6   50    4-66     23-72  (73)
</pre><pre> 27 6YBU_E Bacterial cellulose sec  46.1 1.6E+02  0.0015   23.3   5.3   39   31-69     30-86  (179)
</pre><pre> 28 5UJE_A SbnI protein; staphylof  44.3      67 0.00062   24.9   3.1   60   24-90     24-85  (242)
</pre><pre> 29 1VK1_A Conserved hypothetical   39.3 1.1E+02  0.0011   23.4   3.6   56   29-91     37-95  (242)
</pre><pre> 30 PF16288.7 ; DUF4934 ; Domain o  38.4 1.4E+02  0.0013   18.1   3.3   22   48-69     26-47  (102)
</pre><pre> 31 PF11061.10 ; DUF2862 ; Protein  33.0      58 0.00053   21.6   1.1   10   62-71     48-57  (60)
</pre><pre> 32 d2gaua1 a.4.5.4 (A:152-232) Tr  30.8 1.8E+02  0.0017   17.4   2.9   49    6-67     27-75  (81)
</pre><pre> 33 2N5U_A Tsr0524 protein; PetP,   29.2      69 0.00064   22.5   1.0   10   62-71     64-73  (79)
</pre><pre> 34 d3l57a_ d.89.1.0 (A:) automate  26.1      44 0.00041   26.8  -0.4   22   51-72    169-190 (225)
</pre><pre> 35 PF03519.16 ; Invas_SpaK ; Inva  25.5 1.6E+02  0.0015   20.1   2.2   19   41-59     18-36  (78)
</pre><pre> 36 5MPO_B Molybdopterin synthase   24.5   4E+02  0.0037   17.7   4.6   45   25-69     48-100 (106)
</pre><pre> 37 PF18531.3 ; Polo_box_2 ; Polo   23.8 1.7E+02  0.0016   22.4   2.3   31   50-81     14-44  (112)
</pre><pre> 38 1A1X_A HMTCP-1; MTCP-1, ONCOGE  23.4 2.3E+02  0.0022   21.0   2.8   23   47-69     10-32  (108)
</pre><pre> 39 PF09871.11 ; DUF2098 ; Unchara  22.0 2.5E+02  0.0023   20.0   2.7   19   49-67     20-39  (90)
</pre><pre> 40 6NR6_B Bifunctional transcript  21.5 2.8E+02  0.0026   21.7   3.1   56   29-90     27-83  (254)
</pre><pre> 41 1JSG_A ONCOGENE PRODUCT P14TCL  21.4 2.5E+02  0.0023   21.1   2.7   23   47-69     14-36  (114)
</pre><pre> 42 3B02_A Transcriptional regulat  21.0 3.4E+02  0.0032   18.7   3.1   47    7-66    138-184 (195)
</pre><pre> 43 PF02333.17 ; Phytase ; Phytase  21.0 2.1E+02   0.002   23.9   2.5   21   49-69    208-229 (375)
</pre><pre> 44 4NZL_B Uncharacterized protein  20.7   2E+02  0.0019   21.6   2.1   37   49-86     77-114 (114)
</pre><pre> 45 6VTM_E MAP domain-containing p  20.3 2.2E+02   0.002   20.8   2.1   37   49-86     62-99  (99)
</pre><pre> 46 6VTM_F MAP domain-containing p  20.3 2.2E+02   0.002   20.8   2.1   37   49-86     62-99  (99)
</pre><pre>
</pre><pre>No 1
</pre><pre>>d1vz0a2 d.268.1.1 (A:23-115) Putative partitioning protein ParB/Spo0J {Thermus thermophilus [TaxId: 274]}
</pre><pre>Probab=97.14  E-value=0.0014  Score=41.39  Aligned_cols=77  Identities=13%  Similarity=0.099  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCce
</pre><pre>Q XP_021345007.1    1 YLKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCS   79 (93)
</pre><pre>Q Consensus         1 ~l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~   79 (93)
</pre><pre>                      .+++++|...+      +.....+....+..+...+..+.    ..+||.|...+|+++..| ++|+++++++.    ..
</pre><pre>T Consensus         3 ~i~~~~i~~~~------~~~r~~~~~~~~~~l~~~i~~~g----~~~pi~v~~~~~~~~iidG~~R~~a~~~~~----~~   68 (93)
</pre><pre>T d1vz0a2           3 RLPLASIRPNP------RQPRKRFAEESLKELADSIREKG----LLQPLLVRPQGDGYELVAGERRYRAALMAG----LQ   68 (93)
</pre><pre>T ss_dssp             EEEGGGEECCC------CCHHHHHHHHHHHHHHHHHHHHC----CSSCEEEEEETTEEEEEECHHHHHHHHHHT----CS
</pre><pre>T ss_pred             EccHHHcCCCC------CCCCccCCHHHHHHHHHHHHHcC----CCcCEEEEEcCCeEEEEecchHHHHHHHcC----CC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEecchhh
</pre><pre>Q XP_021345007.1   80 KILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        80 ~I~V~~~~~~~~   91 (93)
</pre><pre>                      .|++.+....+.
</pre><pre>T Consensus        69 ~i~~~~~~~~~~   80 (93)
</pre><pre>T d1vz0a2          69 EVPAVVKDLTDR   80 (93)
</pre><pre>T ss_dssp             EEEEEECCCCHH
</pre><pre>T ss_pred             eecEEEEcCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>TIGR00180 parB_part; ParB/RepB/Spo0J family partition protein. This model represents the most well-conserved core of a set of chromosomal and plasmid partition proteins related to ParB, including Spo0J, RepB, and SopB.
</pre><pre>Probab=94.13  E-value=0.12  Score=37.42  Aligned_cols=55  Identities=16%  Similarity=0.177  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHhCCCCcccCCCeEEEEEC---CEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   29 IGKTLDDILVASSSIDQIPKITVCKKN---GQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        29 I~~~l~~i~~G~~~v~diP~I~V~~~~---G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      +.++.+.+..+..    .+||.|...+   |+|+..| ++|+++++.+.    ...|+|.+....+.
</pre><pre>T Consensus        29 i~~l~~si~~~G~----~~Pi~v~~~~~~~~~~~iidG~~R~~a~~~l~----~~~i~~~i~~~~~~   87 (187)
</pre><pre>T TIGR00180        29 LAELIESIKEQGQ----LQPILVRKHPDQPGRYEIIAGERRWRAAKLAG----LKTIPAIVRELDDE   87 (187)
</pre><pre>T ss_pred             HHHHHHHHHHcCC----CCCeEEEeCCCCCCcEEEEcCccHHHHHHHcC----CCcCCEEEECCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>6SDK_B Stage 0 sporulation protein J; ParB, Chromosome segregation, Chromosome organization, DNA BINDING PROTEIN; HET: MSE, CDP; 1.81A {Bacillus subtilis (strain 168)} SCOP: d.268.1.0, a.4.14.0
</pre><pre>Probab=93.87  E-value=0.14  Score=36.37  Aligned_cols=77  Identities=12%  Similarity=0.158  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCce
</pre><pre>Q XP_021345007.1    1 YLKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCS   79 (93)
</pre><pre>Q Consensus         1 ~l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~   79 (93)
</pre><pre>                      .++.++|....      |..........+.++.++|.....    ++||.|...+|.|+..| ++|+++++.+.    ..
</pre><pre>T Consensus         6 ~i~l~~i~~~~------~~~r~~~~~~~~~~l~~si~~~g~----~~pi~v~~~~~~~~ii~G~~R~~a~~~~g----~~   71 (199)
</pre><pre>T 6SDK_B            6 EIKIADLRPNP------YQPRKHFDDEALAELKESVLQHGI----LQPLIVRKSLKGYDIVAGERRFRAAKLAG----LD   71 (199)
</pre><pre>T ss_dssp             EEEGGGEECCT------TCCCCCCCHHHHHHHHHHHHHHCC----CSCEEEEECSSSEEEEECHHHHHHHHHTT----CS
</pre><pre>T ss_pred             EccHHHCCCCC------CCCCcCCCHHHHHHHHHHHHHcCC----CCCEEEEeCCCCEEEeechHHHHHHHHcC----CC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEecchhh
</pre><pre>Q XP_021345007.1   80 KILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        80 ~I~V~~~~~~~~   91 (93)
</pre><pre>                      .|++.+....+.
</pre><pre>T Consensus        72 ~i~~~v~~~~~~   83 (199)
</pre><pre>T 6SDK_B           72 TVPAIVRELSEA   83 (199)
</pre><pre>T ss_dssp             EEEEEECCCCHH
</pre><pre>T ss_pred             eeCEEEecCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>d1xw3a1 d.268.1.4 (A:32-137) Sulfiredoxin {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=93.04  E-value=0.24  Score=33.07  Aligned_cols=62  Identities=21%  Similarity=0.291  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCcchHHHHHHHHhCCCCcccCCCeEEEEE-----CCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   23 RHKNTQIGKTLDDILVASSSIDQIPKITVCKK-----NGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        23 ~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~-----~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      .+....+.++.+.|......   ++||.|...     +|.|+..| ++|+.+++.+.    ...|++.+....++
</pre><pre>T Consensus        25 ~~~~~~~~~l~~si~~~G~~---~~Pi~v~~~~~~~~~~~~~ii~G~~R~~a~~~l~----~~~i~~~v~~~~~~   92 (106)
</pre><pre>T d1xw3a1          25 VLDPAKVQSLVDTIREDPDS---VPPIDVLWIKGAQGGDYFYSFGGCHRYAAYQQLQ----RETIPAKLVQSTLS   92 (106)
</pre><pre>T ss_dssp             CCCHHHHHHHHHHHHHCGGG---SCCEEEEEEECTTSCEEEECCSCHHHHHHHHHTT----CSEEEEEEEEECHH
</pre><pre>T ss_pred             CCCHHHHHHHHHHHHhCCCC---CCCeEEEEEeCCCCCceEEEEecHHHHHHHHHhC----CCcCCEEEEeCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>4UMK_C PROBABLE CHROMOSOME-PARTITIONING PROTEIN PARB; DNA BINDING PROTEIN-DNA COMPLEX; HET: SO4; 3.096A {HELICOBACTER PYLORI}
</pre><pre>Probab=92.47  E-value=0.12  Score=38.64  Aligned_cols=76  Identities=17%  Similarity=0.211  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceE
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSK   80 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~   80 (93)
</pre><pre>                      +++++|..........|....          +..|..--....-++||.|...+|+|+..| ++|+.+++.+.    ...
</pre><pre>T Consensus        36 i~l~~i~~~~~~~r~~~~~~~----------~~~l~~si~~~G~~~Pi~v~~~~g~y~iidG~~R~~a~~~~g----~~~  101 (240)
</pre><pre>T 4UMK_C           36 LGIDEVMPNPYQPRKVFSEDS----------LEELAQSIKEHGLLQPVLVVSENGRYHLIAGERRLRASKLAK----MPT  101 (240)
</pre><pre>T ss_dssp             CHHHHSCSCCSSSSCSSSSTT----------CCCTTTHHHHCSSCCSCCCSSSCCCSSCCCCCCCTTTTTTCC----SSC
</pre><pre>T ss_pred             ecHHHcCCCCCCCcccCCHHH----------HHHHHHHHHHhCCCCCeEEEecCCEEEEeeCCChHHHHHHcC----CCe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEecchhh
</pre><pre>Q XP_021345007.1   81 ILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        81 I~V~~~~~~~~   91 (93)
</pre><pre>                      |++.+....+.
</pre><pre>T Consensus       102 i~~~v~~~~~~  112 (240)
</pre><pre>T 4UMK_C          102 IKAIVVDIEQE  112 (240)
</pre><pre>T ss_dssp             CCCCCCCHHHH
</pre><pre>T ss_pred             eeEEEEeCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>6RYK_B ParB-like nuclease domain protein; ParABS, cytoskeleton, bactofilin, CTP, CELL CYCLE; HET: CTP; 1.7A {Myxococcus xanthus DK 1622}
</pre><pre>Probab=92.10  E-value=0.42  Score=35.83  Aligned_cols=74  Identities=15%  Similarity=0.197  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEE-CCEEEEcc-cceeeeeehhHhhCCce
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKK-NGQWYSFD-NRRLWIFKKLEKYGKCS   79 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~-~G~~~s~D-NRRLwvfk~a~~~~~~~   79 (93)
</pre><pre>                      +++++|....     .|...+   ...+.++.+.|....    -..||.|... +|+|+..| ++|+++.+.+.    ..
</pre><pre>T Consensus        37 i~i~~i~~~~-----~~~~~~---~~~~~~L~~si~~~G----~~~Pi~v~~~~~~~y~ii~G~~R~~a~~~~g----~~  100 (229)
</pre><pre>T 6RYK_B           37 LPLEKLQDES-----AFKLRP---EGDVSGLATDIARLG----QLFPVDVRPAGEDRYQLVCGFRRVAALRFLK----RD  100 (229)
</pre><pre>T ss_dssp             EEGGGBCSCC-----TTCCSC---CCCCHHHHHHHHHHC----CCSCEEEEECSTTCEEEEECHHHHHHHHHTT----CS
</pre><pre>T ss_pred             EeHHHcCCCC-----ccCCCC---ccchHHHHHHHHHhC----CCCCeEEEEcCCCcEEEEeCchHHHHHHHcC----CC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEecchhh
</pre><pre>Q XP_021345007.1   80 KILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        80 ~I~V~~~~~~~~   91 (93)
</pre><pre>                      +|++.+..+...
</pre><pre>T Consensus       101 ~i~~~v~~~~~~  112 (229)
</pre><pre>T 6RYK_B          101 AVQARIHLRLSD  112 (229)
</pre><pre>T ss_dssp             EEEEEEETTCCH
</pre><pre>T ss_pred             cceEEEEcCCCH
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>6KY4_A Sulfiredoxin, chloroplastic/mitochondrial; sulfiredoxin, cysteine, sulfinic acid, peroxiredoxin repair protein, arabidopsis thaliana, antioxidant, oxidoreductase; HET: ADP, PO4; 3.2A {Arabidopsis thaliana}
</pre><pre>Probab=91.62  E-value=0.74  Score=29.99  Aligned_cols=61  Identities=25%  Similarity=0.339  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   23 RHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        23 ~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      ......+.++.+.++...    ..++|.|...+|.++..| ++|+.+++.+.    ...|++.+....+.
</pre><pre>T Consensus        36 ~~~~~~~~~l~~si~~~g----~~~pi~v~~~~~~~~ii~G~~R~~a~~~~g----~~~v~~~v~~~~~~   97 (112)
</pre><pre>T 6KY4_A           36 SNDQNKVKELMDSIRQIG----LQVPIDVIEVDGTYYGFSGCHRYEAHQKLG----LPTIRCKIRKGTKE   97 (112)
</pre><pre>T ss_dssp             CCCHHHHHHHHHHHHHHC----CCSCEEEEEETTEEECCSCHHHHHHHHHTT----CSEEEEEEEECCHH
</pre><pre>T ss_pred             CCCHHHHHHHHHHHHHcC----CCCCEEEEEECCEEEEECChHHHHHHHHhC----CCeeeEEEEeCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>1VZ0_C Chromosome-partitioning protein Spo0J; NUCLEAR PROTEIN, CHROMOSOME SEGREGATION, DNA-BINDING, HELIX-TURN-HELIX; HET: CO; 2.3A {Thermus thermophilus HB27} SCOP: d.268.1.1, a.4.14.1
</pre><pre>Probab=91.13  E-value=0.12  Score=38.11  Aligned_cols=76  Identities=12%  Similarity=0.104  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceE
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSK   80 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~   80 (93)
</pre><pre>                      +++++|...+......+....          ++.|..--....-.+||.|...+|+|+..| ++|+++++.+.    ...
</pre><pre>T Consensus        26 i~i~~i~~~~~~~r~~~~~~~----------~~~l~~si~~~G~~~Pi~v~~~~~~~~ii~G~~R~~a~~~l~----~~~   91 (230)
</pre><pre>T 1VZ0_C           26 LPLASIRPNPRQPRKRFAEES----------LKELADSIREKGLLQPLLVRPQGDGYELVAGERRYRAALMAG----LQE   91 (230)
</pre><pre>T ss_dssp             EEGGGEECCCCCHHHHHHHHH----------HHHHHHHHHHHCCSSCEEEEEETTEEEEEECHHHHHHHHHTT----CSE
</pre><pre>T ss_pred             eeHHHcCCCCCCCcccCCHHH----------HHHHHHHHHHhCCCCCEEEEECCCcEEEeCCchHHHHHHHcC----CCe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEecchhh
</pre><pre>Q XP_021345007.1   81 ILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        81 I~V~~~~~~~~   91 (93)
</pre><pre>                      |++.+....+.
</pre><pre>T Consensus        92 i~~~v~~~~~~  102 (230)
</pre><pre>T 1VZ0_C           92 VPAVVKDLTDR  102 (230)
</pre><pre>T ss_dssp             EEEEECCCCHH
</pre><pre>T ss_pred             eCEEEEeCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>1VZ0_F Chromosome-partitioning protein Spo0J; NUCLEAR PROTEIN, CHROMOSOME SEGREGATION, DNA-BINDING, HELIX-TURN-HELIX; HET: CO; 2.3A {Thermus thermophilus HB27} SCOP: a.4.14.1, d.268.1.1
</pre><pre>Probab=91.13  E-value=0.12  Score=38.11  Aligned_cols=76  Identities=12%  Similarity=0.104  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceE
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSK   80 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~   80 (93)
</pre><pre>                      +++++|...+......+....          ++.|..--....-.+||.|...+|+|+..| ++|+++++.+.    ...
</pre><pre>T Consensus        26 i~i~~i~~~~~~~r~~~~~~~----------~~~l~~si~~~G~~~Pi~v~~~~~~~~ii~G~~R~~a~~~l~----~~~   91 (230)
</pre><pre>T 1VZ0_F           26 LPLASIRPNPRQPRKRFAEES----------LKELADSIREKGLLQPLLVRPQGDGYELVAGERRYRAALMAG----LQE   91 (230)
</pre><pre>T ss_dssp             EETTSEECCCCCHHHHHHHHH----------HHHHHHHHHHHCCSSCEEEEEETTEEEEEECHHHHHHHHHHT----CSE
</pre><pre>T ss_pred             eeHHHcCCCCCCCcccCCHHH----------HHHHHHHHHHhCCCCCEEEEECCCcEEEeCCchHHHHHHHcC----CCe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEecchhh
</pre><pre>Q XP_021345007.1   81 ILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        81 I~V~~~~~~~~   91 (93)
</pre><pre>                      |++.+....+.
</pre><pre>T Consensus        92 i~~~v~~~~~~  102 (230)
</pre><pre>T 1VZ0_F           92 VPAVVKDLTDR  102 (230)
</pre><pre>T ss_dssp             EEEEECCCCHH
</pre><pre>T ss_pred             eCEEEEeCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>TIGR04285 nucleoid_noc; nucleoid occlusion protein. This model describes nucleoid occlusion protein, a close homolog to ParB chromosome partitioning proteins including Spo0J in Bacillus subtilis.
</pre><pre>Probab=89.07  E-value=0.98  Score=34.67  Aligned_cols=55  Identities=20%  Similarity=0.206  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   29 IGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        29 I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      +.++.+.|.....    +.||.|...+|.|+..| ++|+.+++.+.    ...|++.+....+.
</pre><pre>T Consensus        26 i~~L~~si~~~G~----~~Pi~v~~~~~~y~ii~G~~R~~a~~~lg----~~~i~~~v~~~~~~   81 (255)
</pre><pre>T TIGR04285        26 IEELAQSIKEHGL----IQPIVVRKKDDKYEIIAGERRFRACKLLG----WEEVPAIVREMNDE   81 (255)
</pre><pre>T ss_pred             HHHHHHHHHHcCC----ccCEEEEecCCeEEEecChHHHHHHHHhC----CCcCCEEEeCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>7NFU_B Nucleoid occlusion protein; CTP switch, chromosome segregation, protein-DNA recognition, peripheral membrane protein, DNA BINDING PROTEIN; HET: SO4; 2.5A {Geobacillus thermoleovorans CCB_US3_UF5}
</pre><pre>Probab=87.36  E-value=0.36  Score=36.72  Aligned_cols=77  Identities=19%  Similarity=0.218  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEE-CCEEEEcc-cceeeeeehhHhhCCc
</pre><pre>Q XP_021345007.1    1 YLKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKK-NGQWYSFD-NRRLWIFKKLEKYGKC   78 (93)
</pre><pre>Q Consensus         1 ~l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~-~G~~~s~D-NRRLwvfk~a~~~~~~   78 (93)
</pre><pre>                      .+++++|..........+....          +..|..--....-.+||.|... +|+|+..| ++|+.+.+.+.    .
</pre><pre>T Consensus        30 ~v~l~~i~~~~~~~r~~~~~~~----------~~~L~~si~~~G~~~Pi~v~~~~~g~y~iidG~~R~~a~~~~g----~   95 (252)
</pre><pre>T 7NFU_B           30 HIPVKSIIPNRFQPRTMFDEEK----------IDELALTIRTHGIIQPIVVRECGNGRFEIIAGERRWRAVQKLG----W   95 (252)
</pre><pre>T ss_dssp             EEEGGGEECCTTSCCCCCCHHH----------HHHHHHHHHHHCCSSCEEEEEETTTEEEEEECHHHHHHHHHTT----C
</pre><pre>T ss_pred             EecHHHccCCCCCCCcCCCHHH----------HHHHHHHHHHhCCCCCeEEEEcCCCcEEEeecchHHHHHHHhC----C
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eEEEEEEecchhh
</pre><pre>Q XP_021345007.1   79 SKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        79 ~~I~V~~~~~~~~   91 (93)
</pre><pre>                      ..|+|.+....+.
</pre><pre>T Consensus        96 ~~i~~~v~~~~~~  108 (252)
</pre><pre>T 7NFU_B           96 TEIPAIIKNLNDK  108 (252)
</pre><pre>T ss_dssp             SEEEEEECCCCHH
</pre><pre>T ss_pred             CeeCEEEEeCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>7BM8_A Chromosome-partitioning protein ParB; chromosome segregation, CTP, molecular gates, protein-DNA recognition, DNA BINDING PROTEIN; HET: CTP; 2.73A {Caulobacter vibrioides (strain NA1000 / CB15N)}
</pre><pre>Probab=85.67  E-value=0.39  Score=36.52  Aligned_cols=72  Identities=13%  Similarity=0.191  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEE---ECCEEEEcc-cceeeeeehhHhhCC
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCK---KNGQWYSFD-NRRLWIFKKLEKYGK   77 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~---~~G~~~s~D-NRRLwvfk~a~~~~~   77 (93)
</pre><pre>                      ++..+|.+.+......+....          +..|.+--....-+.||.|+.   .+|+|+..| ++|+.+++.+.    
</pre><pre>T Consensus        37 I~i~~i~~~~~~~r~~~~~~~----------~~~L~~sI~~~G~~~Pi~v~~~~~~~g~y~ildG~~R~~A~~~~~----  102 (257)
</pre><pre>T 7BM8_A           37 APIEILQRNPDQPRRTFREED----------LEDLSNSIREKGVLQPILVRPSPDTAGEYQIVAGERRWRAAQRAG----  102 (257)
</pre><pre>T ss_dssp             EEGGGEEECTTCSCCCCCHHH----------HHHHHHHHHHHCCCSCEEEEECSSSTTCEEEEECHHHHHHHHHTT----
</pre><pre>T ss_pred             EeHHHccCCCCCccccCCHHH----------HHHHHHHHHHhCCCCCeEEEeCCCCCCcEEEEcCccHHHHHHHcC----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEEEec
</pre><pre>Q XP_021345007.1   78 CSKILVKISK   87 (93)
</pre><pre>Q Consensus        78 ~~~I~V~~~~   87 (93)
</pre><pre>                      ...|++.+..
</pre><pre>T Consensus       103 ~~~i~~~v~~  112 (257)
</pre><pre>T 7BM8_A          103 LKTVPIMVRE  112 (257)
</pre><pre>T ss_dssp             CSEEEEEECC
</pre><pre>T ss_pred             CCccCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>TIGR03454 partition_RepB; plasmid partitioning protein RepB. Members of this family are the RepB protein involved in replicon partitioning.
</pre><pre>Probab=85.38  E-value=1.7  Score=35.68  Aligned_cols=76  Identities=17%  Similarity=0.177  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEEC---CEEEEcc-cceeeeeehhHhhC
</pre><pre>Q XP_021345007.1    1 YLKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKN---GQWYSFD-NRRLWIFKKLEKYG   76 (93)
</pre><pre>Q Consensus         1 ~l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~---G~~~s~D-NRRLwvfk~a~~~~   76 (93)
</pre><pre>                      .++|++|.-+.  .+.++..    ....+.++.+.|...    ....||.|....   |+|...| +||+.+.+.+.   
</pre><pre>T Consensus        66 ~i~~~~I~~~~--~~~R~~~----~~~~~~~L~~sI~~~----G~~~PI~Vr~~~~~~g~y~Iv~G~rR~~A~~~lg---  132 (325)
</pre><pre>T TIGR03454        66 ELDPALIDPSF--VRDRLDS----DDEDFADLVESIREH----GQQVPILVRPHPEAEGRYQIAYGHRRLRAARELG---  132 (325)
</pre><pre>T ss_pred             EeCHHHcCCcc--CCCCCCC----CHHHHHHHHHHHHHc----CCcCCEEEEeCCCCCCceEEEeCccHHHHHHHhC---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   77 KCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        77 ~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                       . .|++.+..-.+.
</pre><pre>T Consensus       133 -~-~v~a~V~~~~d~  145 (325)
</pre><pre>T TIGR03454       133 -R-PVKAVVRELSDE  145 (325)
</pre><pre>T ss_pred             -C-CeEEEEECCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>TIGR03764 ICE_PFGI_1_parB; integrating conjugative element, PFGI_1 class, ParB family protein. Members of this protein family carry the ParB-type nuclease domain and are found in integrating conjugative elements (ICE) in the same class as PFGI-1 of Pseudomonas fluorescens Pf-5.
</pre><pre>Probab=85.13  E-value=3.5  Score=32.06  Aligned_cols=72  Identities=17%  Similarity=0.184  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEEC--CEEEEcc--cceeeeeehh----
</pre><pre>Q XP_021345007.1    1 YLKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKN--GQWYSFD--NRRLWIFKKL----   72 (93)
</pre><pre>Q Consensus         1 ~l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~--G~~~s~D--NRRLwvfk~a----   72 (93)
</pre><pre>                      .+++++|....       .++..+....+.++.+.|......    .||.|....  |+|+..|  |||+-+++.+    
</pre><pre>T Consensus        10 ~i~~~~i~~~~-------~n~R~~~~~~~~~l~~si~~~G~~----~Pi~v~~~~~~~~y~ii~GG~~R~~a~~~l~~~~   78 (258)
</pre><pre>T TIGR03764        10 VLTLDQLRPYE-------DNPRTTRNPKYDEIKASIRERGLD----QPPKVTRRPGESDYIIRNGGNTRLQILKELWQET   78 (258)
</pre><pre>T ss_pred             EEcHHHcCCCC-------CCcCcCCChhHHHHHHHHHHcCCC----CCCEEEeCCCCCcEEEecCChHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ---HhhCCceEEEEEEec
</pre><pre>Q XP_021345007.1   73 ---EKYGKCSKILVKISK   87 (93)
</pre><pre>Q Consensus        73 ---~~~~~~~~I~V~~~~   87 (93)
</pre><pre>                         .    ...|++.+..
</pre><pre>T Consensus        79 ~~~~----~~~i~~~v~~   92 (258)
</pre><pre>T TIGR03764        79 GDER----FFRIHCLFKP   92 (258)
</pre><pre>T ss_pred             CCcc----ceeEEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>PF08857.13 ; ParBc_2 ; Putative ParB-like nuclease
</pre><pre>Probab=84.40  E-value=1.1  Score=33.37  Aligned_cols=72  Identities=7%  Similarity=0.071  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCC-CCcccCCCeEEEE-ECCEEEEcc-cceeeeeehhHhhCC
</pre><pre>Q XP_021345007.1    1 YLKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVAS-SSIDQIPKITVCK-KNGQWYSFD-NRRLWIFKKLEKYGK   77 (93)
</pre><pre>Q Consensus         1 ~l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~-~~v~diP~I~V~~-~~G~~~s~D-NRRLwvfk~a~~~~~   77 (93)
</pre><pre>                      .|.|+..+.+++.+..+...            +.....+. .......+|.|+. .+|++|..| .-|+-++..+.    
</pre><pre>T Consensus         2 ~L~PTQ~~lg~~~V~~K~~~------------~~~~~~~~~~~~~~~~~ipvv~~~~~~~yiiDGHHr~~a~~~~g----   65 (153)
</pre><pre>T S5Y864_PARAH/7    2 ALRPTQMTVGMKEVQRKRHD------------WREDSQSKEVNFLGHHLLPCVLGPKKRYHLIDNHHLALALHLEG----   65 (153)
</pre><pre>T ss_pred             CCCCCccccCHHHHHHHHHH------------HHhcchhHHHhhhccCcccEEECCCCcEEEeCChHHHHHHHHcC----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEEEecc
</pre><pre>Q XP_021345007.1   78 CSKILVKISKK   88 (93)
</pre><pre>Q Consensus        78 ~~~I~V~~~~~   88 (93)
</pre><pre>                      ...|+|.+..+
</pre><pre>T Consensus        66 ~~~i~v~v~~d   76 (153)
</pre><pre>T S5Y864_PARAH/7   66 VTAVLVNVVAD   76 (153)
</pre><pre>T ss_pred             CcEEEEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>PF02195.20 ; ParBc ; ParB-like nuclease domain
</pre><pre>Probab=83.76  E-value=3.3  Score=24.85  Aligned_cols=51  Identities=16%  Similarity=0.168  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHhCCCCcccCCCeEEEEEC-CEEEEcc-cceeeeeehhHhhCCceEEEEEEec
</pre><pre>Q XP_021345007.1   29 IGKTLDDILVASSSIDQIPKITVCKKN-GQWYSFD-NRRLWIFKKLEKYGKCSKILVKISK   87 (93)
</pre><pre>Q Consensus        29 I~~~l~~i~~G~~~v~diP~I~V~~~~-G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~   87 (93)
</pre><pre>                      +..+.+.+......    .++.|.... |.++..| .+|+.+.+.+.    ..+|++.+..
</pre><pre>T Consensus        24 ~~~l~~~i~~~~~~----~p~~v~~~~~~~~~i~~G~~r~~a~~~~~----~~~i~~~v~~   76 (91)
</pre><pre>T PARB_TREPA/33-   24 LEELAASIREHGVI----QPVLAEKNQDGSWVIIAGERRTRAAILAG----LNRIPVIVRT   76 (91)
</pre><pre>T ss_pred             HHHHHHHHHHcCCc----cCeEEEECCCCCEEEEcCchHHHHHHHhC----CCcccEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>1XW3_A SULFIREDOXIN; retroreduction, sulfinic acid, peroxiredoxin, ATP, OXIDOREDUCTASE; 1.65A {Homo sapiens} SCOP: d.268.1.4, l.1.1.1
</pre><pre>Probab=81.53  E-value=5.5  Score=25.37  Aligned_cols=56  Identities=21%  Similarity=0.323  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHhCCCCcccCCCeEEEEE-----CCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   29 IGKTLDDILVASSSIDQIPKITVCKK-----NGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        29 I~~~l~~i~~G~~~v~diP~I~V~~~-----~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      +..+.+.+.....   .++++.|...     +|.|+..| .+|+-+.+.+.    ...|++.+....+.
</pre><pre>T Consensus        35 ~~~l~~~i~~~g~---~~~p~~v~~~~~~~~~~~~~l~~G~~R~~a~~~~~----~~~v~~~v~~~~~~   96 (110)
</pre><pre>T 1XW3_A           35 VQSLVDTIREDPD---SVPPIDVLWIKGAQGGDYFYSFGGCHRYAAYQQLQ----RETIPAKLVQSTLS   96 (110)
</pre><pre>T ss_dssp             HHHHHHHHHHCGG---GSCCEEEEEEECTTSCEEEECCSCHHHHHHHHHTT----CSEEEEEEEEECHH
</pre><pre>T ss_pred             HHHHHHHHHhCCC---CCCCeEEEEEeCCCCCCeEEEEcChHHHHHHHHcC----CCcCCEEEEcCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>2HWJ_F Hypothetical protein Atu1540; Agrobacterium tumefaciens, Hypothetical protein, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: MSE; 2.61A {Agrobacterium tumefaciens str. C58} SCOP: d.268.1.3
</pre><pre>Probab=81.24  E-value=1.3  Score=34.67  Aligned_cols=76  Identities=13%  Similarity=0.185  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEE-ECCEEEEcc-cceeeeeehhHhhCCce
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCK-KNGQWYSFD-NRRLWIFKKLEKYGKCS   79 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~-~~G~~~s~D-NRRLwvfk~a~~~~~~~   79 (93)
</pre><pre>                      |.|.....+.+.|...-..-...+.+.+.+.+..           .+|.|+. .+|++|..| .-|+-++..+.    ..
</pre><pre>T Consensus        17 L~PTQ~~vg~~~V~~~~~~~~~~~~~~~~~~l~~-----------~~iPVVigp~g~~yiiDGHHr~~A~~~~G----~~   81 (205)
</pre><pre>T 2HWJ_F           17 LRPTQIAVGFREVELKRKEWRETRKKDGDDFLGN-----------HIVPVVAGPKDRAYLIDHHHLVLALSKEG----VE   81 (205)
</pre><pre>T ss_dssp             EECCBSEECHHHHHHHHHHHHTC---------CC-----------CEEEEEECSTTCEEECSCHHHHHHHHHTT----CC
</pre><pre>T ss_pred             ccccchhhCHHHHHHHHHHHHhhccccchhhhcC-----------CeeeEEECCCCcEEEeCChHHHHHHHHcC----CC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEecchhhc
</pre><pre>Q XP_021345007.1   80 KILVKISKKLKKI   92 (93)
</pre><pre>Q Consensus        80 ~I~V~~~~~~~~~   92 (93)
</pre><pre>                      .|+|.+..+...+
</pre><pre>T Consensus        82 ~v~v~v~~d~s~l   94 (205)
</pre><pre>T 2HWJ_F           82 HVLTSEVAKFSHL   94 (205)
</pre><pre>T ss_dssp             EEEEEEEEECTTS
</pre><pre>T ss_pred             EEEEEEecCcccC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>d1vk1a_ d.268.1.2 (A:) Hypothetical protein PF0380 {Pyrococcus furiosus [TaxId: 2261]}
</pre><pre>Probab=76.38  E-value=6.3  Score=30.30  Aligned_cols=61  Identities=10%  Similarity=0.119  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCcchHHHHHHHHhCCCCcccCCCeEEEE--ECCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   24 HKNTQIGKTLDDILVASSSIDQIPKITVCK--KNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        24 h~~~~I~~~l~~i~~G~~~v~diP~I~V~~--~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      +....+.++.+.|.....-   ..||-|..  .+|+++.+| ++|+.+++.+.    ...|.|.+.+..+.
</pre><pre>T Consensus        22 ~~~~~~~~l~~~i~~~G~~---~~Pi~v~~~~~~~~~~i~dGhhR~~a~~~l~----~~~i~~~~~~~~~~   85 (232)
</pre><pre>T d1vk1a_          22 LVQRELEDFIESVTGSGIF---WKPMLLAKIPGTDEYLIVDGHHRWAGLQKLG----AKRAPSVILDYFDE   85 (232)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHHHHCEE---CSCEEEEECTTSSCEEEEECHHHHHHHHHHT----CCEEEEEEECTTST
</pre><pre>T ss_pred             cCHHHHHHHHHHHHhcCCC---CCCEEEEECCCCCcEEEeCCHHHHHHHHHhC----CCEEcEEEEeCCCh
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>5K5D_A ParB domain protein nuclease; ParB-N, pnob8, partition, HYDROLASE; HET: CIT, MSE; 2.45A {Sulfolobus solfataricus}
</pre><pre>Probab=72.26  E-value=7.7  Score=31.32  Aligned_cols=72  Identities=17%  Similarity=0.093  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceE
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSK   80 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~   80 (93)
</pre><pre>                      +++++|......-...+.      ...+.++.+.|....+    +.||.|   .|.|.-.| ++|+.+++.+.    ...
</pre><pre>T Consensus        14 i~i~~i~~~~~~~~~~~~------~~~~~~L~~sI~~~G~----~~Pi~V---~~~~~lidG~~R~~A~k~lg----~~~   76 (300)
</pre><pre>T 5K5D_A           14 VSIEKLKEDDEFKTMVPS------NNSREDLEKSLREKSQ----IFPLIA---DRNYVLIDGYTRLDIMKKLG----FKE   76 (300)
</pre><pre>T ss_dssp             EEGGGCBCCHHHHTTSCC------CSCTTHHHHHHHHHCC----SSCEEE---CTTCBEEECHHHHHHHHHTT----CCE
</pre><pre>T ss_pred             EeHHHCccChhHhccCCC------hhhHHHHHHHHHHcCC----CCCEEE---CCCCEEEccHHHHHHHHHhC----CCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEecchh
</pre><pre>Q XP_021345007.1   81 ILVKISKKLK   90 (93)
</pre><pre>Q Consensus        81 I~V~~~~~~~   90 (93)
</pre><pre>                      |+|.+....+
</pre><pre>T Consensus        77 i~~~v~~~~~   86 (300)
</pre><pre>T 5K5D_A           77 VKILKYDFDS   86 (300)
</pre><pre>T ss_dssp             EEEEEESSCT
</pre><pre>T ss_pred             eeEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>5K5D_B ParB domain protein nuclease; ParB-N, pnob8, partition, HYDROLASE; HET: MSE, CIT; 2.45A {Sulfolobus solfataricus}
</pre><pre>Probab=72.26  E-value=7.7  Score=31.32  Aligned_cols=72  Identities=17%  Similarity=0.093  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceE
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSK   80 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~   80 (93)
</pre><pre>                      +++++|......-...+.      ...+.++.+.|....+    +.||.|   .|.|.-.| ++|+.+++.+.    ...
</pre><pre>T Consensus        14 i~i~~i~~~~~~~~~~~~------~~~~~~L~~sI~~~G~----~~Pi~V---~~~~~lidG~~R~~A~k~lg----~~~   76 (300)
</pre><pre>T 5K5D_B           14 VSIEKLKEDDEFKTMVPS------NNSREDLEKSLREKSQ----IFPLIA---DRNYVLIDGYTRLDIMKKLG----FKE   76 (300)
</pre><pre>T ss_dssp             EEGGGCEECHHHHHSSCC------CTTHHHHHHHHHHHCS----CSCEEE---CTTCEEEECHHHHHHHHHTT----CCE
</pre><pre>T ss_pred             EeHHHCccChhHhccCCC------hhhHHHHHHHHHHcCC----CCCEEE---CCCCEEEccHHHHHHHHHhC----CCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEecchh
</pre><pre>Q XP_021345007.1   81 ILVKISKKLK   90 (93)
</pre><pre>Q Consensus        81 I~V~~~~~~~   90 (93)
</pre><pre>                      |+|.+....+
</pre><pre>T Consensus        77 i~~~v~~~~~   86 (300)
</pre><pre>T 5K5D_B           77 VKILKYDFDS   86 (300)
</pre><pre>T ss_dssp             EEEEEESSCT
</pre><pre>T ss_pred             eeEEEecCCC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>d2hwja1 d.268.1.3 (A:4-204) Hypothetical protein Atu1540 {Agrobacterium tumefaciens [TaxId: 358]}
</pre><pre>Probab=67.83  E-value=15  Score=28.90  Aligned_cols=76  Identities=12%  Similarity=0.097  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CCHHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEE-EECCEEEEcc-cceeeeeehhHhhCCce
</pre><pre>Q XP_021345007.1    2 LKPAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVC-KKNGQWYSFD-NRRLWIFKKLEKYGKCS   79 (93)
</pre><pre>Q Consensus         2 l~P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~-~~~G~~~s~D-NRRLwvfk~a~~~~~~~   79 (93)
</pre><pre>                      |.|.+.....+.+..+           +.+.......-.-......+|.|+ .-+|++|..| .-|+-++..+.    ..
</pre><pre>T Consensus        14 L~PTQ~~vg~~~V~~k-----------~~~~~~~~~~~~~~~~~~~~ipvVigp~~~~yl~DGHHr~~A~~~~G----~~   78 (201)
</pre><pre>T d2hwja1          14 LRPTQIAVGFREVELK-----------RKEWRETRKKDGDDFLGNHIVPVVAGPKDRAYLIDHHHLVLALSKEG----VE   78 (201)
</pre><pre>T ss_dssp             CBCSBSEECHHHHHHH-----------HHHHHTCC-----CCTTCBEEEEEECSTTCEEECSCHHHHHHHHHTT----CC
</pre><pre>T ss_pred             cCCCccccCHHHHHHH-----------HHHHHHHhHhcchhhhhCCeEEEEECCCCcEEEECChHHHHHHHHcC----CC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEecchhhc
</pre><pre>Q XP_021345007.1   80 KILVKISKKLKKI   92 (93)
</pre><pre>Q Consensus        80 ~I~V~~~~~~~~~   92 (93)
</pre><pre>                      .|+|.+..+...+
</pre><pre>T Consensus        79 ~V~v~v~~d~s~l   91 (201)
</pre><pre>T d2hwja1          79 HVLTSEVAKFSHL   91 (201)
</pre><pre>T ss_dssp             EEEEEEEEECTTS
</pre><pre>T ss_pred             EEEEEEecccccC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>6CKA_B Paratox; prophage, natural competence, Streptococcus, protein binding, VIRAL PROTEIN; HET: MSE; 1.559A {Streptococcus pyogenes serotype M3}
</pre><pre>Probab=59.79  E-value=56  Score=22.82  Aligned_cols=31  Identities=19%  Similarity=0.210  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             CcchHHHHHHHHhCCCCcccCCCeEEEEECCEEE
</pre><pre>Q XP_021345007.1   26 NTQIGKTLDDILVASSSIDQIPKITVCKKNGQWY   59 (93)
</pre><pre>Q Consensus        26 ~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~   59 (93)
</pre><pre>                      +-...++.+++-+|.++-+   +|.|++++|+.+
</pre><pre>T Consensus         1 Ml~~~efkqAId~G~I~~d---tV~IVRknGkIf   31 (68)
</pre><pre>T 6CKA_B            1 MLYIDEFKEAIDKGYILGD---TVAIVRKNGKIF   31 (68)
</pre><pre>T ss_dssp             CCCHHHHHHHHHTTSSCSS---EEEEEEETTEEE
</pre><pre>T ss_pred             CccHHHHHHHHHcCCCCCC---eEEEEEECCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>TIGR03734 PRTRC_parB; PRTRC system ParB family protein. A novel genetic system characterized by six major proteins, included a ParB homolog and a ThiF homolog, is designated PRTRC, or ParB-Related,ThiF-Related Cassette.
</pre><pre>Probab=57.99  E-value=17  Score=32.11  Aligned_cols=71  Identities=11%  Similarity=0.027  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEEC--CEEEEcc-cceeeeeehhHhhCCceE--EEEEEec
</pre><pre>Q XP_021345007.1   13 TISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKN--GQWYSFD-NRRLWIFKKLEKYGKCSK--ILVKISK   87 (93)
</pre><pre>Q Consensus        13 sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~--G~~~s~D-NRRLwvfk~a~~~~~~~~--I~V~~~~   87 (93)
</pre><pre>                      .|.+-+..........|.++.+.|....    -+.||.|....  ++|...| +||+...+.+.    ...  |+|.+..
</pre><pre>T Consensus         1 ~~~~p~n~R~~~~~~~i~~La~SI~~~G----~l~pi~V~~~~~~~~y~ii~G~rR~~A~~~l~----~~~~~vp~~v~~   72 (554)
</pre><pre>T TIGR03734         1 LIVPGNNPRRYFDPAEMAELVESIRAKG----VLQPILVRPVPGSDLYEVVAGERRYRAALEVF----GEDYDIPALIKV   72 (554)
</pre><pre>T ss_pred             CCCCCCCcccCCCHHHHHHHHHHHHHhC----CCCCEEEEeCCCCCcEEEEeCHHHHHHHHHhc----CCCCCceEEEec
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhh
</pre><pre>Q XP_021345007.1   88 KLKK   91 (93)
</pre><pre>Q Consensus        88 ~~~~   91 (93)
</pre><pre>                      ..+.
</pre><pre>T Consensus        73 ~~d~   76 (554)
</pre><pre>T TIGR03734        73 LTDE   76 (554)
</pre><pre>T ss_pred             CCHH
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>d3amra_ b.68.3.1 (A:) automated matches {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=49.02  E-value=37  Score=28.54  Aligned_cols=21  Identities=19%  Similarity=0.604  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCEEEEcc-cceeeee
</pre><pre>Q XP_021345007.1   49 ITVCKKNGQWYSFD-NRRLWIF   69 (93)
</pre><pre>Q Consensus        49 I~V~~~~G~~~s~D-NRRLwvf   69 (93)
</pre><pre>                      |-|-...|.+|.+| |.|+|+|
</pre><pre>T Consensus       184 iavD~~~g~LyVaD~~~rV~~f  205 (352)
</pre><pre>T d3amra_         184 MAADDEYGRLYIAEEDEAIWKF  205 (352)
</pre><pre>T ss_dssp             EEEETTTTEEEEEETTTEEEEE
</pre><pre>T ss_pred             EEEECCCCEEEEEeCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>d1zyba1 a.4.5.4 (A:148-220) Probable transcription regulator BT4300, C-terminal domain {Bacteroides thetaiotaomicron [TaxId: 818]}
</pre><pre>Probab=48.20  E-value=95  Score=18.52  Aligned_cols=50  Identities=12%  Similarity=0.193  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             HHHeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEccccee
</pre><pre>Q XP_021345007.1    4 PAEIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFDNRRL   66 (93)
</pre><pre>Q Consensus         4 P~dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~DNRRL   66 (93)
</pre><pre>                      +..+.+++..|...|.-    +..++...++.|..-.+         |....|.++..+-..|
</pre><pre>T Consensus        23 ~~~~~~~~~eia~~~~~----s~~~~~~~l~~L~~~g~---------i~~~~~~~~~~~~~~~   72 (73)
</pre><pre>T d1zyba1          23 EKTFKVKMDDLARCLDD----TRLNISKTLNELQDNGL---------IELHRKEILIPDAQKL   72 (73)
</pre><pre>T ss_dssp             CEEEECCHHHHHHHHTS----CHHHHHHHHHHHHHTTS---------CEEETTEEEESCGGGG
</pre><pre>T ss_pred             CceeeeCHHHHHHHHCC----CHHHHHHHHHHHHHCCC---------EEEeCCeEEECCHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>6YBU_E Bacterial cellulose secretion regulator BcsE, residues 349-523; Bacterial biofilms, Bacterial cellulose, Bacterial secretion system, ATP binding protein, c-di-GMP binding protein, SIGNALING PROTEIN; HET: C2E, ATP; 2.49A {Escherichia coli}
</pre><pre>Probab=46.14  E-value=1.6e+02  Score=23.33  Aligned_cols=39  Identities=13%  Similarity=0.260  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhCCCCcccCCCeEEEEE------------------CCEEEEcccceeeee
</pre><pre>Q XP_021345007.1   31 KTLDDILVASSSIDQIPKITVCKK------------------NGQWYSFDNRRLWIF   69 (93)
</pre><pre>Q Consensus        31 ~~l~~i~~G~~~v~diP~I~V~~~------------------~G~~~s~DNRRLwvf   69 (93)
</pre><pre>                      +.++.+.+......+++..-|...                  +|-+.|.++.+||+|
</pre><pre>T Consensus        30 ~~V~~~l~~~~~~~~~~~~Lv~L~p~~gl~~~~~L~~c~~rR~GDl~T~~~~~lyLF   86 (179)
</pre><pre>T 6YBU_E           30 NAVNNMMNNPLLPAHGKGVLVALRPVPGIRVEQALTLCRPNRTGDIMTIGGNRLVLF   86 (179)
</pre><pre>T ss_dssp             HHHHHHHTCTTSCSSCSEEEEEEEECTTSCHHHHHTTCCCCSTTCEEEEBSSCEEEE
</pre><pre>T ss_pred             HHHHHHHhCCCCCccCCcEEEEeEECCCCCHHHHHHHcCCCCCCeEEEEeCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>5UJE_A SbnI protein; staphyloferrin B, heme, regulator, siderophore, GENE REGULATION; HET: GOL, MSE; 2.5A {Staphylococcus aureus}
</pre><pre>Probab=44.28  E-value=67  Score=24.91  Aligned_cols=60  Identities=22%  Similarity=0.253  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCcchHHHHHHHHhCCCCcccCCCeEEEEE-CCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchh
</pre><pre>Q XP_021345007.1   24 HKNTQIGKTLDDILVASSSIDQIPKITVCKK-NGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLK   90 (93)
</pre><pre>Q Consensus        24 h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~-~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~   90 (93)
</pre><pre>                      +....+.++.+.|.....-.   .||.|... ++.++.+| .+|+.+++.+.    ...|.+.+....+
</pre><pre>T Consensus        24 ~~~~~~~~l~~~i~~~g~~~---~Pi~v~~~~~~~~~i~dGhhR~~a~~~~~----~~~i~~~v~~~~~   85 (242)
</pre><pre>T 5UJE_A           24 FEPLRLEKTKSSIEADDFIR---HPILVTAMQHGRYMVIDGVHRYTSLKALG----CKKVPVQEIHETQ   85 (242)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHHHHTEES---SCEEEEECSTTCEEEEECHHHHHHHHHTT----CSEEEEEEECTTT
</pre><pre>T ss_pred             CCHHHHHHHHHHHHhcCCCC---CCeEEEECCCCeEEEcCChHHHHHHHHhC----CCeeCEEEECCCc
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>1VK1_A Conserved hypothetical protein; reductive methylation, dimethyl lysine, Pyrococcus furiosus, conserved hypothetical protein, structural genomics, PSI, Protein Structure Initiative, Southeast; HET: MLY; 1.2A {Pyrococcus furiosus} SCOP: d.268.1.2
</pre><pre>Probab=39.31  E-value=1.1e+02  Score=23.42  Aligned_cols=56  Identities=11%  Similarity=0.124  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHhCCCCcccCCCeEEEEEC--CEEEEcc-cceeeeeehhHhhCCceEEEEEEecchhh
</pre><pre>Q XP_021345007.1   29 IGKTLDDILVASSSIDQIPKITVCKKN--GQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLKK   91 (93)
</pre><pre>Q Consensus        29 I~~~l~~i~~G~~~v~diP~I~V~~~~--G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~~   91 (93)
</pre><pre>                      +.++.+.+......   ..||-|...+  ++++.+| ++|+.+++.+.    ...|.+.+....+.
</pre><pre>T Consensus        37 ~~~l~~~i~~~g~~---~~Pi~v~~~~~~~~~~i~dGhhR~~a~~~~~----~~~i~~~v~~~~~~   95 (242)
</pre><pre>T 1VK1_A           37 LEDFIESVTGSGIF---WKPMLLAKIPGTDEYLIVDGHHRWAGLQKLG----AKRAPSVILDYFDE   95 (242)
</pre><pre>T ss_dssp             HHHHHHHHHHHCEE---CSCEEEEECTTSSCEEEEECHHHHHHHHHHT----CCEEEEEEECTTST
</pre><pre>T ss_pred             HHHHHHHHHhcCCc---cCCEEEEECCCCCeEEEeCCHHHHHHHHHhC----CCEEeEEEEcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>PF16288.7 ; DUF4934 ; Domain of unknown function (DUF4934)
</pre><pre>Probab=38.36  E-value=1.4e+02  Score=18.06  Aligned_cols=22  Identities=14%  Similarity=0.173  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             CeEEEEECCEEEEcccceeeee
</pre><pre>Q XP_021345007.1   48 KITVCKKNGQWYSFDNRRLWIF   69 (93)
</pre><pre>Q Consensus        48 ~I~V~~~~G~~~s~DNRRLwvf   69 (93)
</pre><pre>                      +..+...+|..|..||.++++|
</pre><pre>T Consensus        26 ~~~i~~~~~~i~v~d~~~i~~~   47 (102)
</pre><pre>T A6L0Z3_BACV8/4   26 SYSIKLSKDHIFVSTNGRCLSF   47 (102)
</pre><pre>T ss_pred             cceEEEeCCEEEEEeCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>PF11061.10 ; DUF2862 ; Protein of unknown function (DUF2862)
</pre><pre>Probab=32.96  E-value=58  Score=21.58  Aligned_cols=10  Identities=30%  Similarity=0.910  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             ccceeeeeeh
</pre><pre>Q XP_021345007.1   62 DNRRLWIFKK   71 (93)
</pre><pre>Q Consensus        62 DNRRLwvfk~   71 (93)
</pre><pre>                      |+.|+|+|..
</pre><pre>T Consensus        48 d~~riW~f~~   57 (60)
</pre><pre>T K9XUE3_STAC7/3   48 DKSATWFFED   57 (60)
</pre><pre>T ss_pred             CCCeEEeehh
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>d2gaua1 a.4.5.4 (A:152-232) Transcriptional regulator PG0396, C-terminal domain {Porphyromonas gingivalis [TaxId: 837]}
</pre><pre>Probab=30.81  E-value=1.8e+02  Score=17.42  Aligned_cols=49  Identities=14%  Similarity=0.139  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             HeeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcccceee
</pre><pre>Q XP_021345007.1    6 EIRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFDNRRLW   67 (93)
</pre><pre>Q Consensus         6 dI~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~DNRRLw   67 (93)
</pre><pre>                      .+.+++..|...++.    +..++..+++.|.+-.+         |....+.++..|-.+|+
</pre><pre>T Consensus        27 ~~~~t~~~la~~~g~----~~~~v~~~l~~L~~~g~---------i~~~~~~~~i~~~~~l~   75 (81)
</pre><pre>T d2gaua1          27 SIYLSREELATLSNM----TVSNAIRTLSTFVSERM---------LALDGKRIKIIDCDRLQ   75 (81)
</pre><pre>T ss_dssp             SCCCCHHHHHHHTTS----CHHHHHHHHHHHHHTTS---------EEEETTEEEESCHHHHH
</pre><pre>T ss_pred             EeeccHHHHHHHcCC----CHHHHHHHHHHHHHCCC---------EEEeCCEEEEeCHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>2N5U_A Tsr0524 protein; PetP, cytochrome b6f complex, SH3 domain, cyanobacteria, Thermosynechococcus elongatus, photosynthesis; NMR {Thermosynechococcus elongatus BP-1}
</pre><pre>Probab=29.20  E-value=69  Score=22.46  Aligned_cols=10  Identities=30%  Similarity=0.906  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             ccceeeeeeh
</pre><pre>Q XP_021345007.1   62 DNRRLWIFKK   71 (93)
</pre><pre>Q Consensus        62 DNRRLwvfk~   71 (93)
</pre><pre>                      |+.|+|+|+.
</pre><pre>T Consensus        64 d~tRiWff~~   73 (79)
</pre><pre>T 2N5U_A           64 DRSSTWFFED   73 (79)
</pre><pre>T ss_dssp             TSCEEEEETT
</pre><pre>T ss_pred             CCCeEEeehh
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>d3l57a_ d.89.1.0 (A:) automated matches {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=26.14  E-value=44  Score=26.77  Aligned_cols=22  Identities=27%  Similarity=0.460  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             EEEECCEEEEcccceeeeeehh
</pre><pre>Q XP_021345007.1   51 VCKKNGQWYSFDNRRLWIFKKL   72 (93)
</pre><pre>Q Consensus        51 V~~~~G~~~s~DNRRLwvfk~a   72 (93)
</pre><pre>                      +...+|+|.++||++|+-.+++
</pre><pre>T Consensus       169 ~~~~DGkw~ald~~~l~~~~~~  190 (225)
</pre><pre>T d3l57a_         169 TQREDGQWRALKNDELMRNKMH  190 (225)
</pre><pre>T ss_dssp             EECTTSCEECCCTHHHHTTHHH
</pre><pre>T ss_pred             eeCCCCCEEecchHHHHhhHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>PF03519.16 ; Invas_SpaK ; Invasion protein B family
</pre><pre>Probab=25.50  E-value=1.6e+02  Score=20.09  Aligned_cols=19  Identities=32%  Similarity=0.448  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             CCcccCCCeEEEEECCEEE
</pre><pre>Q XP_021345007.1   41 SSIDQIPKITVCKKNGQWY   59 (93)
</pre><pre>Q Consensus        41 ~~v~diP~I~V~~~~G~~~   59 (93)
</pre><pre>                      +.+.+.|.|.+...++..|
</pre><pre>T Consensus        18 l~f~d~p~I~Is~~dd~vw   36 (78)
</pre><pre>T Q326Q9_SHIDS/1   18 ISLDNMPAINIALVNEQVM   36 (78)
</pre><pre>T ss_pred             EEecCCCCEEEEEECCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>5MPO_B Molybdopterin synthase sulfur carrier subunit; MOCS2A, MOCS2B, transferase; 2.43A {Homo sapiens}
</pre><pre>Probab=24.50  E-value=4e+02  Score=17.67  Aligned_cols=45  Identities=2%  Similarity=0.122  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc--------cceeeee
</pre><pre>Q XP_021345007.1   25 KNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD--------NRRLWIF   69 (93)
</pre><pre>Q Consensus        25 ~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D--------NRRLwvf   69 (93)
</pre><pre>                      .+.++.+++++|............--....+|+.+..|        +.++.++
</pre><pre>T Consensus        48 ~~~t~~~ll~~l~~~~p~~~~~~~~~~v~vng~~v~~~~~~~~l~~gd~v~i~  100 (106)
</pre><pre>T 5MPO_B           48 QEIKALQLWKEIETRHPGLADVRNQIIFAVRQEYVELGDQLLVLQPGDEIAVI  100 (106)
</pre><pre>T ss_dssp             SEEEHHHHHHHHHHHSGGGGGGGGGCEEEETTEEECSSSCEEECCTTCEEEEE
</pre><pre>T ss_pred             CCCcHHHHHHHHHHHCchhHhhhccEEEEECcceeecCCcceeCCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>PF18531.3 ; Polo_box_2 ; Polo box domain
</pre><pre>Probab=23.83  E-value=1.7e+02  Score=22.43  Aligned_cols=31  Identities=23%  Similarity=0.211  Sum_probs=0.0  Template_Neff=3.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCEEEEcccceeeeeehhHhhCCceEE
</pre><pre>Q XP_021345007.1   50 TVCKKNGQWYSFDNRRLWIFKKLEKYGKCSKI   81 (93)
</pre><pre>Q Consensus        50 ~V~~~~G~~~s~DNRRLwvfk~a~~~~~~~~I   81 (93)
</pre><pre>                      ++....|+|+..||.|+ +|.-|-+.|...+|
</pre><pre>T Consensus        14 r~~ta~gRYii~~d~r~-~FEVA~k~g~itkI   44 (112)
</pre><pre>T ZYG1_CAEBR/350   14 RVRTPGGRYIVEMNTRC-RFEVVSKGNIVLRI   44 (112)
</pre><pre>T ss_pred             EEECCCccEEEEeCCEE-EEEEeccCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>1A1X_A HMTCP-1; MTCP-1, ONCOGENE INVOLVED IN T CELL MALIGNANCIES, PROTO-ONCOGENE; 2.0A {Homo sapiens} SCOP: b.63.1.1
</pre><pre>Probab=23.43  E-value=2.3e+02  Score=20.96  Aligned_cols=23  Identities=17%  Similarity=0.349  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CCeEEEEECCEEEEcccceeeee
</pre><pre>Q XP_021345007.1   47 PKITVCKKNGQWYSFDNRRLWIF   69 (93)
</pre><pre>Q Consensus        47 P~I~V~~~~G~~~s~DNRRLwvf   69 (93)
</pre><pre>                      ||+......=..|.-.|+|+|+.
</pre><pre>T Consensus        10 ~P~~Lw~~~pgvY~DE~~RtWl~   32 (108)
</pre><pre>T 1A1X_A           10 PPDHLWVHQEGIYRDEYQRTWVA   32 (108)
</pre><pre>T ss_dssp             CCSEEEEEETTEEEETTSCEEEE
</pre><pre>T ss_pred             CCCeEEEEecCEEEcCCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>PF09871.11 ; DUF2098 ; Uncharacterized protein conserved in archaea (DUF2098)
</pre><pre>Probab=21.98  E-value=2.5e+02  Score=20.05  Aligned_cols=19  Identities=21%  Similarity=0.613  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCE-EEEcccceee
</pre><pre>Q XP_021345007.1   49 ITVCKKNGQ-WYSFDNRRLW   67 (93)
</pre><pre>Q Consensus        49 I~V~~~~G~-~~s~DNRRLw   67 (93)
</pre><pre>                      +.+...+|+ |+..|+.-||
</pre><pre>T Consensus        20 ~~i~~~~g~~w~~ld~t~l~   39 (90)
</pre><pre>T A0B846_METTP/2   20 KLIKEEDGETWALLDSTDLY   39 (90)
</pre><pre>T ss_pred             EEEEEeCCEEEEEEeCCCcE
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>6NR6_B Bifunctional transcriptional regulator/O-phospho-L-serine synthase SbnI; free serine kinase, siderophore, ParB/Srx family, staphyloferrin B, BIOSYNTHETIC PROTEIN; HET: ADP; 1.9A {Staphylococcus pseudintermedius}
</pre><pre>Probab=21.45  E-value=2.8e+02  Score=21.70  Aligned_cols=56  Identities=23%  Similarity=0.339  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHhCCCCcccCCCeEEEEECCEEEEcc-cceeeeeehhHhhCCceEEEEEEecchh
</pre><pre>Q XP_021345007.1   29 IGKTLDDILVASSSIDQIPKITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKISKKLK   90 (93)
</pre><pre>Q Consensus        29 I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~~~~~   90 (93)
</pre><pre>                      +.++.+.+....  +-..|.+-....+|.++.+| ++|+.+++.+.    ...|.+.+....+
</pre><pre>T Consensus        27 ~~~l~~~i~~~g--~~~~Pi~v~~~~~~~~~i~dG~hR~~a~~~~~----~~~i~~~v~~~~~   83 (254)
</pre><pre>T 6NR6_B           27 LEKTKESIAKEQ--HLRHPVLVVKTLFGRYMVIDGVHRFMSLKALG----CEVIPVQVIQRTQ   83 (254)
</pre><pre>T ss_dssp             HHHHHHHHHHHC--EESSCEEEEECGGGCEEEEECHHHHHHHHHTT----CSEEEEEECCGGG
</pre><pre>T ss_pred             HHHHHHHHHHhC--CCCCCEEEEEcCCCeEEEeCCHHHHHHHHHcC----CCEEeEEEEchhh
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>1JSG_A ONCOGENE PRODUCT P14TCL1; PROTO-ONCOGENE, CL1 GENE, T CELL LEUKEMIA, ONCOPROTEIN, MICROSOME; 2.5A {Homo sapiens} SCOP: b.63.1.1
</pre><pre>Probab=21.39  E-value=2.5e+02  Score=21.10  Aligned_cols=23  Identities=9%  Similarity=0.069  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCeEEEEECCEEEEcccceeeee
</pre><pre>Q XP_021345007.1   47 PKITVCKKNGQWYSFDNRRLWIF   69 (93)
</pre><pre>Q Consensus        47 P~I~V~~~~G~~~s~DNRRLwvf   69 (93)
</pre><pre>                      ||++.+...--.|.-.|+|.|+.
</pre><pre>T Consensus        14 ~P~~Lwi~~~gvY~DE~~RtWl~   36 (114)
</pre><pre>T 1JSG_A           14 HPDRLWAWEKFVYLDEKQHAWLP   36 (114)
</pre><pre>T ss_dssp             CCSEEEEEETTEEEETTCCEEEE
</pre><pre>T ss_pred             CCcceEEeeccEEeCCCCceeEE
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>3B02_A Transcriptional regulator, Crp family; Structural Genomics, RIKEN Structural Genomics/Proteomics Initiative, RSGI, cyclic nucleotide monophosphate-binding domain, Helix-turn-helix DNA-binding motif, Bacterial regulatory; 1.92A {Thermus thermophilus}
</pre><pre>Probab=21.03  E-value=3.4e+02  Score=18.75  Aligned_cols=47  Identities=17%  Similarity=0.236  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             eeEehhHhHHhccCCCCCCCcchHHHHHHHHhCCCCcccCCCeEEEEECCEEEEccccee
</pre><pre>Q XP_021345007.1    7 IRYTQETISGVFRQNTRHKNTQIGKTLDDILVASSSIDQIPKITVCKKNGQWYSFDNRRL   66 (93)
</pre><pre>Q Consensus         7 I~FsQ~sI~~~F~~~~~h~~~~I~~~l~~i~~G~~~v~diP~I~V~~~~G~~~s~DNRRL   66 (93)
</pre><pre>                      +.++|..|+...+-    +..++..++..|..+.+         +....|+++..|-.+|
</pre><pre>T Consensus       138 ~~~~~~~ia~~lg~----sr~~~~r~l~~l~~~g~---------i~~~~~~i~i~~~~~l  184 (195)
</pre><pre>T 3B02_A          138 VTVSHEEIADATAS----IRESVSKVLADLRREGL---------IATAYRRVYLLDLAAL  184 (195)
</pre><pre>T ss_dssp             EECCHHHHHHTTTS----CHHHHHHHHHHHHHHTS---------EEEETTEEEECCHHHH
</pre><pre>T ss_pred             eecCHHHHHHHHcc----CHHHHHHHHHHHHHCCC---------EEecCCeEEEeCHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF02333.17 ; Phytase ; Phytase
</pre><pre>Probab=20.98  E-value=2.1e+02  Score=23.94  Aligned_cols=21  Identities=19%  Similarity=0.593  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCEEEEcc-cceeeee
</pre><pre>Q XP_021345007.1   49 ITVCKKNGQWYSFD-NRRLWIF   69 (93)
</pre><pre>Q Consensus        49 I~V~~~~G~~~s~D-NRRLwvf   69 (93)
</pre><pre>                      +.|-...|.+|..| +.|+|+|
</pre><pre>T Consensus       208 ~avd~~~g~lyV~d~~~~I~~~  229 (375)
</pre><pre>T Q65NG0_BACLD/5  208 LAADDEYGKMYIAEEDAAIWSF  229 (375)
</pre><pre>T ss_pred             EEEECCCCeEEEEeCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>4NZL_B Uncharacterized protein; Primarily beta, Serine protease, protease inhibitor, innate immunity, Azurophilic granules, HYDROLASE-HYDROLASE INHIBITOR complex; HET: BMA, NAG, FUL; 1.85A {Homo sapiens}
</pre><pre>Probab=20.74  E-value=2e+02  Score=21.64  Aligned_cols=37  Identities=11%  Similarity=0.038  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCEEEEcc-cceeeeeehhHhhCCceEEEEEEe
</pre><pre>Q XP_021345007.1   49 ITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKIS   86 (93)
</pre><pre>Q Consensus        49 I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~   86 (93)
</pre><pre>                      |+|...+|..++.| +.+|- .-..-..-.+..|.|.+.
</pre><pre>T Consensus        77 i~V~~kng~k~tidL~~kl~-m~~vId~~~Ik~IdV~lk  114 (114)
</pre><pre>T 4NZL_B           77 YTLTLNDGNKKVVNLKKNDD-AKNSIDPSTIKQIQIVVK  114 (114)
</pre><pre>T ss_dssp             EEEEETTSCEEEEESSCCTT-TTCEECGGGEEEEEEEEC
</pre><pre>T ss_pred             EEEEeCCCCEEEEECCCCCc-cCCeeCHHHcceEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>6VTM_E MAP domain-containing protein; Inhibitor, Protease, Complex, Neutrophil, HYDROLASE-INHIBITOR complex; HET: SO4; 1.6A {Homo sapiens}
</pre><pre>Probab=20.28  E-value=2.2e+02  Score=20.84  Aligned_cols=37  Identities=11%  Similarity=0.050  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCEEEEcc-cceeeeeehhHhhCCceEEEEEEe
</pre><pre>Q XP_021345007.1   49 ITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKIS   86 (93)
</pre><pre>Q Consensus        49 I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~   86 (93)
</pre><pre>                      |+|...+|..++-| +..| -.-..-....+..|.|.+.
</pre><pre>T Consensus        62 i~v~~k~g~k~tidL~~kl-~~~~vid~~~Ik~IdV~vk   99 (99)
</pre><pre>T 6VTM_E           62 YTLTLNDGNKKVVNLKKND-DAKNSIDPSTIKQIQIVVK   99 (99)
</pre><pre>T ss_dssp             EEEEETTSCEEEEESSCCT-TTTSEECGGGEEEEEEEEC
</pre><pre>T ss_pred             EEEEeCCCCEEEEECCCCC-ccCCeeCHHHcceEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>6VTM_F MAP domain-containing protein; Inhibitor, Protease, Complex, Neutrophil, HYDROLASE-INHIBITOR complex; HET: SO4; 1.6A {Homo sapiens}
</pre><pre>Probab=20.28  E-value=2.2e+02  Score=20.84  Aligned_cols=37  Identities=11%  Similarity=0.050  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCEEEEcc-cceeeeeehhHhhCCceEEEEEEe
</pre><pre>Q XP_021345007.1   49 ITVCKKNGQWYSFD-NRRLWIFKKLEKYGKCSKILVKIS   86 (93)
</pre><pre>Q Consensus        49 I~V~~~~G~~~s~D-NRRLwvfk~a~~~~~~~~I~V~~~   86 (93)
</pre><pre>                      |+|...+|..++-| +..| -.-..-....+..|.|.+.
</pre><pre>T Consensus        62 i~v~~k~g~k~tidL~~kl-~~~~vid~~~Ik~IdV~vk   99 (99)
</pre><pre>T 6VTM_F           62 YTLTLNDGNKKVVNLKKND-DAKNSIDPSTIKQIQIVVK   99 (99)
</pre><pre>T ss_dssp             EEEEETTSCEEEEESSCCT-TTTSEECGGGEEEEEEEEC
</pre><pre>T ss_pred             EEEEeCCCCEEEEECCCCC-ccCCeeCHHHcceEEEEeC
</pre><pre>
</pre><pre>
</pre>