<pre>Query         MPZ78938.1/145-235 
</pre><pre>Match_columns 91
</pre><pre>No_of_seqs    45 out of 49
</pre><pre>Neff          3.91333
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 19:45:08 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/5314097.hhr -oa3m ../results/5314097.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF15540.8 ; Ntox47 ; Bacterial  93.7     1.2 1.1E-05   29.2   6.7   63    1-64     21-89  (108)
</pre><pre>  2 PF14090.8 ; HTH_39 ; Helix-tur  69.3      24 0.00023   21.6   3.6   35    3-37     31-67  (69)
</pre><pre>  3 PF15657.8 ; Tox-HNH-EHHH ; HNH  65.1      44 0.00041   20.9   4.2   35   29-64     12-55  (76)
</pre><pre>  4 PF09413.12 ; DUF2007 ; Putativ  33.7      64  0.0006   19.0   1.3   11    6-16     17-27  (69)
</pre><pre>  5 PF12558.10 ; DUF3744 ; ATP-bin  33.0      72 0.00067   21.1   1.5   12    7-18     16-27  (68)
</pre><pre>  6 PF13986.8 ; DUF4224 ; Domain o  32.9   1E+02 0.00094   18.8   2.0   15    1-15     15-29  (45)
</pre><pre>  7 TIGR03936 sam_1_link_chp; radi  31.3      60 0.00055   25.4   1.1   10    8-17     27-36  (208)
</pre><pre>  8 PF09601.12 ; DUF2459 ; Protein  30.2      75  0.0007   24.5   1.4   11    7-17    157-167 (173)
</pre><pre>  9 PF14026.8 ; DUF4242 ; Protein   28.0 1.4E+02  0.0013   19.3   2.2   17    1-17     56-72  (77)
</pre><pre> 10 3BO6_A Hydrophilic protein, vi  27.9 1.6E+02  0.0015   24.4   2.9   28    7-34    164-191 (220)
</pre><pre> 11 PF15385.8 ; SARG ; Specificall  27.4   1E+02 0.00094   28.9   1.9   14    2-15    291-304 (548)
</pre><pre> 12 3I0U_A Phosphothreonine lyase   27.1 1.6E+02  0.0015   24.4   2.8   28    7-34    162-189 (218)
</pre><pre> 13 5VID_F Bot.0671.2; mini-protei  26.4 1.2E+02  0.0011   21.1   1.7   12    2-13     39-50  (66)
</pre><pre> 14 5VID_I Bot.0671.2; mini-protei  26.4 1.2E+02  0.0011   21.1   1.7   12    2-13     39-50  (66)
</pre><pre> 15 d3o69a_ d.113.1.1 (A:) ADP-rib  25.7 1.1E+02   0.001   22.4   1.5   15    3-17     95-109 (190)
</pre><pre> 16 2Z8N_B 27.5 kDa virulence prot  25.3 1.9E+02  0.0018   24.4   2.9   28    7-34    185-212 (241)
</pre><pre> 17 4HAH_B 27.5 kDa virulence prot  25.3 1.9E+02  0.0018   24.4   2.9   28    7-34    185-212 (241)
</pre><pre> 18 TIGR02150 IPP_isom_1; isopente  24.2 1.5E+02  0.0014   20.5   1.9   15    3-17     72-86  (158)
</pre><pre> 19 d2azwa1 d.113.1.1 (A:2-143) Hy  24.1 1.3E+02  0.0012   20.4   1.5   12    3-14     58-69  (142)
</pre><pre> 20 TIGR02117 chp_urease_rgn; cons  23.9 1.1E+02   0.001   24.5   1.4   11    7-17    185-195 (208)
</pre><pre> 21 d4p0ea_ c.51.4.0 (A:) automate  22.6 1.3E+02  0.0012   23.5   1.4   22    1-22      8-29  (189)
</pre><pre> 22 d2feka_ c.44.1.0 (A:) automate  22.4 1.9E+02  0.0017   20.4   2.1   18    6-23     50-67  (147)
</pre><pre> 23 d4mpoa_ d.113.1.0 (A:) automat  22.0 1.6E+02  0.0015   20.6   1.8   13    2-14     51-63  (145)
</pre><pre> 24 d2r7la2 d.142.1.9 (A:124-361)   21.8 1.8E+02  0.0017   22.7   2.1   18    2-19      2-19  (238)
</pre><pre> 25 2KPQ_A Uncharacterized protein  21.4 2.6E+02  0.0024   19.5   2.6   28    2-29     72-99  (100)
</pre><pre> 26 d2p5xa_ c.51.4.0 (A:) automate  21.2 1.4E+02  0.0013   23.5   1.4   22    1-22     11-32  (204)
</pre><pre> 27 4OI3_B Nickel responsive prote  21.0 2.1E+02   0.002   19.7   2.1   18    1-18     64-81  (89)
</pre><pre> 28 4S2X_A RNA pyrophosphohydrolas  20.9 2.1E+02  0.0019   18.6   2.0   16    2-17     49-64  (161)
</pre><pre> 29 PF15255.8 ; CAP-ZIP_m ; WASH c  20.9 1.4E+02  0.0013   23.1   1.4   14    1-14    100-113 (138)
</pre><pre> 30 d5deqa1 d.113.1.6 (A:1-149) Hy  20.7   2E+02  0.0018   20.1   2.0   15    2-16     58-72  (149)
</pre><pre> 31 d1zrja1 a.140.2.1 (A:1-37) Het  20.3   2E+02  0.0019   15.4   1.6   12    6-17     12-23  (37)
</pre><pre> 32 d5c7qa_ d.113.1.0 (A:) automat  20.3 1.6E+02  0.0014   22.3   1.5   15    2-16     87-101 (182)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF15540.8 ; Ntox47 ; Bacterial toxin 47
</pre><pre>Probab=93.73  E-value=1.2  Score=29.23  Aligned_cols=63  Identities=14%  Similarity=0.240  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCCCCcCCCeeee---eCCCCeEEEEEcC-CceEEEEEecCCCCCCCCCc--ccccCCC
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGIPVDAEPVDII---RNASGVQYVYDMD-GRNMLVTENTMDRSHPGQPH--WEAGEMK   64 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGIPTSqqP~sQ~---~~~~grqY~Ye~~-~~~~~V~~~~~D~~H~~~pH--WeAG~~K   64 (91)
</pre><pre>                      +..+..++++++|||.++..+.+.   .....-...|..+ +......-.-.+. .+..||  |.....+
</pre><pre>T Consensus        21 ~~~~~~~~~~~~g~~~~~~~~~~~~~~~~g~~~~~~~~~~~G~~~~~d~~~~~~-g~~~~h~~~~~~~~~   89 (108)
</pre><pre>T Q65M69_BACLD/4   21 YKDALDEAFKRTGVPKKEFKVTKWGKDENGKSFPVEWRSKNGAEVNIDVGHTKN-GPDVPHTGYQTAGKR   89 (108)
</pre><pre>T ss_pred             cHHHHHHHHHHhCCCccccceeecccCCCCCcccEEEEcCCCCEEEEEeccCCC-CCCCCceeeEeCccc
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF14090.8 ; HTH_39 ; Helix-turn-helix domain
</pre><pre>Probab=69.26  E-value=24  Score=21.62  Aligned_cols=35  Identities=14%  Similarity=0.304  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCcCCCeeeeeCCCC--eEEEEEcC
</pre><pre>Q MPZ78938.1/145    3 TVRLLAMRMAGIPVDAEPVDIIRNASG--VQYVYDMD   37 (91)
</pre><pre>Q Consensus         3 aARReAMR~aGIPTSqqP~sQ~~~~~g--rqY~Ye~~   37 (91)
</pre><pre>                      ++|--.||+.|+|+...+.......+.  +.-.|.+.
</pre><pre>T Consensus        31 ~~~I~~Lr~~G~~I~~~~~~~~~~~G~~~~~~~Y~l~   67 (69)
</pre><pre>T Q13T68_PARXL/2   31 RARIVELRKEGYRINTSYVHQLSECGRPHAVGLYTLI   67 (69)
</pre><pre>T ss_pred             HHHHHHHHHCCCceEeEEEEEECCCCCeEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>PF15657.8 ; Tox-HNH-EHHH ; HNH/Endo VII superfamily nuclease toxins
</pre><pre>Probab=65.09  E-value=44  Score=20.89  Aligned_cols=35  Identities=17%  Similarity=0.449  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CeEEEEEcC-CceEEEEEecCCCCC--------CCCCcccccCCC
</pre><pre>Q MPZ78938.1/145   29 GVQYVYDMD-GRNMLVTENTMDRSH--------PGQPHWEAGEMK   64 (91)
</pre><pre>Q Consensus        29 grqY~Ye~~-~~~~~V~~~~~D~~H--------~~~pHWeAG~~K   64 (91)
</pre><pre>                      +++|.|..+ +..++++.|..+...        . .||+.+...+
</pre><pre>T Consensus        12 ~r~y~~~~~~g~~~~i~~h~~gh~~~~~~~~~~~-~~H~n~~~~~   55 (76)
</pre><pre>T C9PMI9_9PAST/1   12 TREYHYTTPSGEKVVFQEHSLGHVYGPKGTLGNQ-GPHFNPRQYD   55 (76)
</pre><pre>T ss_pred             eeEEEEeCCCCCeEEEEEecCCccccCCCCCCCC-CCCcccccCC
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>PF09413.12 ; DUF2007 ; Putative prokaryotic signal transducing protein
</pre><pre>Probab=33.74  E-value=64  Score=19.03  Aligned_cols=11  Identities=18%  Similarity=0.214  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCCC
</pre><pre>Q MPZ78938.1/145    6 LLAMRMAGIPV   16 (91)
</pre><pre>Q Consensus         6 ReAMR~aGIPT   16 (91)
</pre><pre>                      .++|+++|||.
</pre><pre>T Consensus        17 ~~~L~~~gI~~   27 (69)
</pre><pre>T A8MJ48_ALKOO/1   17 KAILDENNIPY   27 (69)
</pre><pre>T ss_pred             HHHHHHCCCCE
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF12558.10 ; DUF3744 ; ATP-binding cassette cobalt transporter
</pre><pre>Probab=32.97  E-value=72  Score=21.13  Aligned_cols=12  Identities=17%  Similarity=0.337  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCcC
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVDA   18 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTSq   18 (91)
</pre><pre>                      +|||.|||+++.
</pre><pre>T Consensus        16 ~ALk~ag~~l~~   27 (68)
</pre><pre>T Y430_STRR6/237   16 TTLRQLGVDLVK   27 (68)
</pre><pre>T ss_pred             HHHHHCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>PF13986.8 ; DUF4224 ; Domain of unknown function (DUF4224)
</pre><pre>Probab=32.87  E-value=1e+02  Score=18.77  Aligned_cols=15  Identities=13%  Similarity=0.330  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCCC
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGIP   15 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGIP   15 (91)
</pre><pre>                      .+++....|++.|||
</pre><pre>T Consensus        15 ~~~~Q~~wL~~~gI~   29 (45)
</pre><pre>T D5BYE7_NITHN/2   15 KPSCMIRQLRSYGLR   29 (45)
</pre><pre>T ss_pred             CHHHHHHHHHHcCCc
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>TIGR03936 sam_1_link_chp; radical SAM-linked protein. This model describes an uncharacterized protein encoded adjacent to, or as a fusion protein with, an uncharacterized radical SAM protein.
</pre><pre>Probab=31.28  E-value=60  Score=25.38  Aligned_cols=10  Identities=60%  Similarity=1.022  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    8 AMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         8 AMR~aGIPTS   17 (91)
</pre><pre>                      |||+||||+.
</pre><pre>T Consensus        27 alrRa~lp~~   36 (208)
</pre><pre>T TIGR03936        27 ALRRAGLPVA   36 (208)
</pre><pre>T ss_pred             HHHHcCCCce
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>PF09601.12 ; DUF2459 ; Protein of unknown function (DUF2459)
</pre><pre>Probab=30.15  E-value=75  Score=24.54  Aligned_cols=11  Identities=18%  Similarity=0.262  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTS   17 (91)
</pre><pre>                      ++||+||+|++
</pre><pre>T Consensus       157 ~~L~~aG~~~~  167 (173)
</pre><pre>T K8GKD6_9CYAN/4  157 NGLRAANVNTP  167 (173)
</pre><pre>T ss_pred             HHHHHCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>PF14026.8 ; DUF4242 ; Protein of unknown function (DUF4242)
</pre><pre>Probab=28.05  E-value=1.4e+02  Score=19.30  Aligned_cols=17  Identities=18%  Similarity=0.225  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGIPTS   17 (91)
</pre><pre>                      |.++=+++++++|+|..
</pre><pre>T Consensus        56 s~e~v~~~~~~~g~p~d   72 (77)
</pre><pre>T H8GQA4_METAL/4   56 DEASVREHAEKGGFPAN   72 (77)
</pre><pre>T ss_pred             CHHHHHHHHHHhCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>3BO6_A Hydrophilic protein, virA protein; alpha/beta fold of phosphothreonine lyase, LYASE; HET: SO4; 1.4A {Chromobacterium violaceum}
</pre><pre>Probab=27.93  E-value=1.6e+02  Score=24.43  Aligned_cols=28  Identities=11%  Similarity=0.071  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCcCCCeeeeeCCCCeEEEE
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVDAEPVDIIRNASGVQYVY   34 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTSqqP~sQ~~~~~grqY~Y   34 (91)
</pre><pre>                      ++|+++||..+.-|.|-....+|.--.|
</pre><pre>T Consensus       164 ~~L~~agI~pG~~p~SD~~~~~~~y~SY  191 (220)
</pre><pre>T 3BO6_A          164 SRLSESGLMPGQYPESDVHPENWKYVSY  191 (220)
</pre><pre>T ss_dssp             HHHHHTTCCBCCCCTTCBCCTTCSSEEE
</pre><pre>T ss_pred             HHHHHCCCCCCCCCCCCCCCCCceeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>PF15385.8 ; SARG ; Specifically androgen-regulated gene protein
</pre><pre>Probab=27.41  E-value=1e+02  Score=28.86  Aligned_cols=14  Identities=29%  Similarity=0.484  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCCC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGIP   15 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGIP   15 (91)
</pre><pre>                      ..||+||||+.|.+
</pre><pre>T Consensus       291 qKvR~EAL~KLGLL  304 (548)
</pre><pre>T L9JBL1_TUPCH/2  291 RKARREALEKLGLP  304 (548)
</pre><pre>T ss_pred             HHHHHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>3I0U_A Phosphothreonine lyase ospF; Shigella flexneri, apo-structure, type III effector, phosphothreonine lyase, Lyase, Secreted, Virulence, Structural Genomics, Center for Structural; 2.7A {Shigella flexneri}
</pre><pre>Probab=27.05  E-value=1.6e+02  Score=24.40  Aligned_cols=28  Identities=14%  Similarity=0.080  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCcCCCeeeeeCCCCeEEEE
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVDAEPVDIIRNASGVQYVY   34 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTSqqP~sQ~~~~~grqY~Y   34 (91)
</pre><pre>                      ++|+++||..+.-|.|-....++.--.|
</pre><pre>T Consensus       162 ~~L~~agI~pG~~p~SD~~~~~~~y~SY  189 (218)
</pre><pre>T 3I0U_A          162 SNLLRSKIAPGEYPASDVRPEDWKYVSY  189 (218)
</pre><pre>T ss_dssp             HHHHHTTCCBCCCCTTCBCCTTCSSEEE
</pre><pre>T ss_pred             HHHHhCCCCCCCCCCCCCCCCCcceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>5VID_F Bot.0671.2; mini-protein binder, inhibitor, TOXIN; 2.75A {Clostridium botulinum}
</pre><pre>Probab=26.42  E-value=1.2e+02  Score=21.14  Aligned_cols=12  Identities=33%  Similarity=0.340  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAG   13 (91)
</pre><pre>Q Consensus         2 raARReAMR~aG   13 (91)
</pre><pre>                      |.|-|.|||+||
</pre><pre>T Consensus        39 rdaarnalrkag   50 (66)
</pre><pre>T 5VID_F           39 RDAARNALRKAG   50 (66)
</pre><pre>T ss_dssp             HHHHHHHHHTTT
</pre><pre>T ss_pred             HHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>5VID_I Bot.0671.2; mini-protein binder, inhibitor, TOXIN; 2.75A {Clostridium botulinum}
</pre><pre>Probab=26.42  E-value=1.2e+02  Score=21.14  Aligned_cols=12  Identities=33%  Similarity=0.340  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAG   13 (91)
</pre><pre>Q Consensus         2 raARReAMR~aG   13 (91)
</pre><pre>                      |.|-|.|||+||
</pre><pre>T Consensus        39 rdaarnalrkag   50 (66)
</pre><pre>T 5VID_I           39 RDAARNALRKAG   50 (66)
</pre><pre>T ss_dssp             HHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>d3o69a_ d.113.1.1 (A:) ADP-ribose pyrophosphatase homologue YffH {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=25.65  E-value=1.1e+02  Score=22.44  Aligned_cols=15  Identities=27%  Similarity=0.375  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    3 TVRLLAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         3 aARReAMR~aGIPTS   17 (91)
</pre><pre>                      +|+||++.+.||...
</pre><pre>T Consensus        95 aa~REl~EE~Gi~~~  109 (190)
</pre><pre>T d3o69a_          95 CIRKAAIEETGYEVG  109 (190)
</pre><pre>T ss_dssp             HHHHHHHHHTSCCCS
</pre><pre>T ss_pred             HHHHHHHHHhCceee
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>2Z8N_B 27.5 kDa virulence protein; short three-helix bundle, distorted beta-strand sheet, LYASE; HET: SO4; 1.8A {Salmonella typhimurium}
</pre><pre>Probab=25.25  E-value=1.9e+02  Score=24.42  Aligned_cols=28  Identities=11%  Similarity=0.075  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCcCCCeeeeeCCCCeEEEE
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVDAEPVDIIRNASGVQYVY   34 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTSqqP~sQ~~~~~grqY~Y   34 (91)
</pre><pre>                      ++|+++||..+.-|.|-....+|.--.|
</pre><pre>T Consensus       185 ~~L~~agI~pG~~p~SD~~~~~~~Y~SY  212 (241)
</pre><pre>T 2Z8N_B          185 SRLSENGVISGQCPESDVHPENWKYLSY  212 (241)
</pre><pre>T ss_dssp             HHHHHTTCCBCCCCTTCBCCTTCSSEEE
</pre><pre>T ss_pred             HHHHHCCCCCCCCCCCCCCCCCceeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>4HAH_B 27.5 kDa virulence protein; Phosphothreonine Lyase, Lyase; 1.8A {Salmonella enterica subsp. enterica serovar Typhimurium}
</pre><pre>Probab=25.25  E-value=1.9e+02  Score=24.42  Aligned_cols=28  Identities=11%  Similarity=0.075  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCcCCCeeeeeCCCCeEEEE
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVDAEPVDIIRNASGVQYVY   34 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTSqqP~sQ~~~~~grqY~Y   34 (91)
</pre><pre>                      ++|+++||..+.-|.|-....+|.--.|
</pre><pre>T Consensus       185 ~~L~~agI~pG~~p~SD~~~~~~~Y~SY  212 (241)
</pre><pre>T 4HAH_B          185 SRLSENGVISGQCPESDVHPENWKYLSY  212 (241)
</pre><pre>T ss_dssp             HHHHHTTCCBCCCCTTCBCCTTCSSEEE
</pre><pre>T ss_pred             HHHHHCCCCCCCCCCCCCCCCCceeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>TIGR02150 IPP_isom_1; isopentenyl-diphosphate delta-isomerase, type 1. This model represents type 1 of two non-homologous families of the enzyme isopentenyl-diphosphate delta-isomerase (IPP isomerase).
</pre><pre>Probab=24.19  E-value=1.5e+02  Score=20.48  Aligned_cols=15  Identities=27%  Similarity=0.255  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    3 TVRLLAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         3 aARReAMR~aGIPTS   17 (91)
</pre><pre>                      +|.||.+.++||+.+
</pre><pre>T Consensus        72 ~a~Rel~ee~gi~~~   86 (158)
</pre><pre>T TIGR02150        72 AAIRRLRHELGIPAD   86 (158)
</pre><pre>T ss_pred             HHHHHHHHHhCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>d2azwa1 d.113.1.1 (A:2-143) Hypothetical protein EF1141 {Enterococcus faecalis [TaxId: 1351]}
</pre><pre>Probab=24.07  E-value=1.3e+02  Score=20.39  Aligned_cols=12  Identities=17%  Similarity=0.279  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC
</pre><pre>Q MPZ78938.1/145    3 TVRLLAMRMAGI   14 (91)
</pre><pre>Q Consensus         3 aARReAMR~aGI   14 (91)
</pre><pre>                      +|+||++.+.||
</pre><pre>T Consensus        58 aa~Rel~EE~g~   69 (142)
</pre><pre>T d2azwa1          58 AIHREVLEELGI   69 (142)
</pre><pre>T ss_dssp             HHHHHHHHHHSE
</pre><pre>T ss_pred             HHHHHHHHHhCe
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>TIGR02117 chp_urease_rgn; conserved hypothetical protein. This conserved hypothetical protein of unknown function is found in several Proteobacteria.
</pre><pre>Probab=23.85  E-value=1.1e+02  Score=24.52  Aligned_cols=11  Identities=36%  Similarity=0.555  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    7 LAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         7 eAMR~aGIPTS   17 (91)
</pre><pre>                      ++||+||+|++
</pre><pre>T Consensus       185 ~~Lr~AG~~~~  195 (208)
</pre><pre>T TIGR02117       185 AALRSAGLRTG  195 (208)
</pre><pre>T ss_pred             HHHHHCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>d4p0ea_ c.51.4.0 (A:) automated matches {Escherichia coli [TaxId: 83333]}
</pre><pre>Probab=22.59  E-value=1.3e+02  Score=23.49  Aligned_cols=22  Identities=9%  Similarity=0.063  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCCCCcCCCee
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGIPVDAEPVD   22 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGIPTSqqP~s   22 (91)
</pre><pre>                      +...|++.|+++|||....|..
</pre><pre>T Consensus         8 ~S~~R~~lL~~~gi~f~v~~~~   29 (189)
</pre><pre>T d4p0ea_           8 GSPRRQELLAQLGVTFERIVTG   29 (189)
</pre><pre>T ss_dssp             CCHHHHHHHHHTTCCEEECCCC
</pre><pre>T ss_pred             CCHHHHHHHHHcCCCeEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>d2feka_ c.44.1.0 (A:) automated matches {Escherichia coli K-12 [TaxId: 83333]}
</pre><pre>Probab=22.44  E-value=1.9e+02  Score=20.39  Aligned_cols=18  Identities=0%  Similarity=-0.014  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCCCcCCCeee
</pre><pre>Q MPZ78938.1/145    6 LLAMRMAGIPVDAEPVDI   23 (91)
</pre><pre>Q Consensus         6 ReAMR~aGIPTSqqP~sQ   23 (91)
</pre><pre>                      .++|++.||+++.....+
</pre><pre>T Consensus        50 ~~~l~~~gi~~~~~~~~~   67 (147)
</pre><pre>T d2feka_          50 ISVAAEHQLSLEGHCARQ   67 (147)
</pre><pre>T ss_dssp             HHHHHHTTCCCTTCCCCB
</pre><pre>T ss_pred             HHHHHHCCCCCCCcccee
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>d4mpoa_ d.113.1.0 (A:) automated matches {Chlamydia trachomatis [TaxId: 471472]}
</pre><pre>Probab=22.05  E-value=1.6e+02  Score=20.55  Aligned_cols=13  Identities=15%  Similarity=0.220  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGI   14 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGI   14 (91)
</pre><pre>                      .+|+||++.+.||
</pre><pre>T Consensus        51 ~aa~REl~EE~G~   63 (145)
</pre><pre>T d4mpoa_          51 EAAERELVEETGL   63 (145)
</pre><pre>T ss_dssp             HHHHHHHHHHHCC
</pre><pre>T ss_pred             HHHHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>d2r7la2 d.142.1.9 (A:124-361) 5-formaminoimidazole-4-carboxamide ribonucleotide synthetase PurP {Methanocaldococcus jannaschii [TaxId: 2190]}
</pre><pre>Probab=21.80  E-value=1.8e+02  Score=22.67  Aligned_cols=18  Identities=22%  Similarity=0.303  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCCCCcCC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGIPVDAE   19 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGIPTSqq   19 (91)
</pre><pre>                      |..-+++|+++|||+..+
</pre><pre>T Consensus         2 K~~~~~~l~~~Gi~~p~~   19 (238)
</pre><pre>T d2r7la2           2 RSLEGKLLREAGLRVPKK   19 (238)
</pre><pre>T ss_dssp             HHHHHHHHHHHTCCCCCE
</pre><pre>T ss_pred             hHHHHHHHHHcCCCCCCE
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>2KPQ_A Uncharacterized protein; Protein, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, UNKNOWN FUNCTION; NMR {Agrobacterium tumefaciens str. C58}
</pre><pre>Probab=21.42  E-value=2.6e+02  Score=19.46  Aligned_cols=28  Identities=25%  Similarity=0.107  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCCCCcCCCeeeeeCCCC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGIPVDAEPVDIIRNASG   29 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGIPTSqqP~sQ~~~~~g   29 (91)
</pre><pre>                      |.|=..|.+++||.+...+.+-....+|
</pre><pre>T Consensus        72 R~AF~~Aa~eagi~v~~~~~~~~~~~~~   99 (100)
</pre><pre>T 2KPQ_A           72 RTAFVAASRQAHCLMEDKAEAPNTIASG   99 (100)
</pre><pre>T ss_dssp             HHHHHHHHHHHTCCCCCCCSSCCCCSCC
</pre><pre>T ss_pred             HHHHHHHHHHCCCccCCccCCCCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>d2p5xa_ c.51.4.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=21.23  E-value=1.4e+02  Score=23.45  Aligned_cols=22  Identities=18%  Similarity=0.253  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCCCCcCCCee
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGIPVDAEPVD   22 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGIPTSqqP~s   22 (91)
</pre><pre>                      +...|++.|+++|||....|..
</pre><pre>T Consensus        11 ~S~~R~~lL~~~g~~f~v~~~~   32 (204)
</pre><pre>T d2p5xa_          11 ASPRRQEILSNAGLRFEVVPSK   32 (204)
</pre><pre>T ss_dssp             CCHHHHHHHHHTTCCCEECCCC
</pre><pre>T ss_pred             CCHHHHHHHHHCCCCeEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>4OI3_B Nickel responsive protein; nickel responsive protein, Structural Genomics, ferredoxin-like fold, a nickel responsive protein, nickel binding, METAL BINDING PROTEIN; HET: MSE; 1.3A {Streptomyces coelicolor}
</pre><pre>Probab=21.01  E-value=2.1e+02  Score=19.71  Aligned_cols=18  Identities=17%  Similarity=0.049  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCCCCcC
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGIPVDA   18 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGIPTSq   18 (91)
</pre><pre>                      |..+=|++.+++|+|...
</pre><pre>T Consensus        64 s~eaV~~~~~~aglp~d~   81 (89)
</pre><pre>T 4OI3_B           64 SAEAVQRVHERAGHKADE   81 (89)
</pre><pre>T ss_dssp             CHHHHHHHHHHHTCCCSE
</pre><pre>T ss_pred             CHHHHHHHHHHhCCCCCE
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>4S2X_A RNA pyrophosphohydrolase; Nudix hydrolase, RNA pyrophosphohydrolase, hydrolase-RNA complex; HET: APC, SO4; 1.5A {Escherichia coli}
</pre><pre>Probab=20.95  E-value=2.1e+02  Score=18.56  Aligned_cols=16  Identities=6%  Similarity=0.092  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGIPTS   17 (91)
</pre><pre>                      .+|+|+++.+.||...
</pre><pre>T Consensus        49 ~~~~re~~ee~gl~~~   64 (161)
</pre><pre>T 4S2X_A           49 QAMYRELFEEVGLSRK   64 (161)
</pre><pre>T ss_dssp             HHHHHHCCCCCCCCGG
</pre><pre>T ss_pred             HHHHHHHHHHhCCCce
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF15255.8 ; CAP-ZIP_m ; WASH complex subunit CAP-Z interacting, central region
</pre><pre>Probab=20.91  E-value=1.4e+02  Score=23.08  Aligned_cols=14  Identities=21%  Similarity=0.178  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             ChHHHHHHHHHcCC
</pre><pre>Q MPZ78938.1/145    1 RETVRLLAMRMAGI   14 (91)
</pre><pre>Q Consensus         1 sraARReAMR~aGI   14 (91)
</pre><pre>                      +|.||+++++..++
</pre><pre>T Consensus       100 TR~aR~~a~~~S~~  113 (138)
</pre><pre>T WASC2_RAT/931-  100 TRAARRLAAQESSE  113 (138)
</pre><pre>T ss_pred             cHHHHHHHhhhccc
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>d5deqa1 d.113.1.6 (A:1-149) Hypothetical protein BT0354, N-terminal domain {Bacteroides thetaiotaomicron [TaxId: 818]}
</pre><pre>Probab=20.73  E-value=2e+02  Score=20.09  Aligned_cols=15  Identities=7%  Similarity=0.319  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCCCC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGIPV   16 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGIPT   16 (91)
</pre><pre>                      .+|+||+..+.||..
</pre><pre>T Consensus        58 ~~a~Re~~EE~Gl~~   72 (149)
</pre><pre>T d5deqa1          58 DAAKRVLAELTGLEN   72 (149)
</pre><pre>T ss_dssp             HHHHHHHHHHHCCCS
</pre><pre>T ss_pred             HHHHHHHHHHHCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>d1zrja1 a.140.2.1 (A:1-37) Heterogeneous nuclear ribonucleoprotein U-like protein 1 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=20.34  E-value=2e+02  Score=15.40  Aligned_cols=12  Identities=8%  Similarity=0.152  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCCCc
</pre><pre>Q MPZ78938.1/145    6 LLAMRMAGIPVD   17 (91)
</pre><pre>Q Consensus         6 ReAMR~aGIPTS   17 (91)
</pre><pre>                      ++.++..|||++
</pre><pre>T Consensus        12 k~~l~~~gl~~~   23 (37)
</pre><pre>T d1zrja1          12 REELQRRGLDTR   23 (37)
</pre><pre>T ss_dssp             HHHHHHTTCCCC
</pre><pre>T ss_pred             HHHHHHCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>d5c7qa_ d.113.1.0 (A:) automated matches {Bdellovibrio bacteriovorus [TaxId: 264462]}
</pre><pre>Probab=20.33  E-value=1.6e+02  Score=22.31  Aligned_cols=15  Identities=13%  Similarity=0.082  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHcCCCC
</pre><pre>Q MPZ78938.1/145    2 ETVRLLAMRMAGIPV   16 (91)
</pre><pre>Q Consensus         2 raARReAMR~aGIPT   16 (91)
</pre><pre>                      .+|+||++.++|+.+
</pre><pre>T Consensus        87 ~aa~REl~EEtG~~~  101 (182)
</pre><pre>T d5c7qa_          87 LTAKRELLEETGYEA  101 (182)
</pre><pre>T ss_dssp             HHHHHHHHHHHCEEE
</pre><pre>T ss_pred             HHHHHHHHHHhCeee
</pre><pre>
</pre><pre>
</pre>