<pre>Query         WP_048874398.1/67-328 
</pre><pre>Match_columns 262
</pre><pre>No_of_seqs    139 out of 601
</pre><pre>Neff          6.02165
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Mon Aug 16 17:06:15 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/7262799.hhr -oa3m ../results/7262799.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 d1cola_ f.1.1.1 (A:) Colicin A  96.0    0.83 7.7E-06   42.7  14.3  186   43-260    10-197 (197)
</pre><pre>  2 PF15967.7 ; Nucleoporin_FG2 ;   95.6    0.42 3.9E-06   51.4  12.6  156    1-157   253-410 (596)
</pre><pre>  3 d1a87a_ f.1.1.1 (A:) Colicin N  94.0       2 1.9E-05   42.7  11.4  182   53-258   100-292 (297)
</pre><pre>  4 1A87_A COLICIN N; BACTERIOCIN,  93.2     3.4 3.1E-05   41.7  11.8  186   53-262   124-320 (321)
</pre><pre>  5 d1rh1a2 f.1.1.1 (A:313-511) Co  92.5     9.5 8.9E-05   36.1  12.9  184   50-258     8-197 (199)
</pre><pre>  6 1COL_B COLICIN A; ANTIBACTERIA  91.3      24 0.00022   33.6  14.9  185   43-258    14-199 (204)
</pre><pre>  7 PF11981.10 ; DUF3482 ; Domain   90.4      20 0.00019   35.3  13.4  159   57-256    26-198 (288)
</pre><pre>  8 PF01024.21 ; Colicin ; Colicin  88.3      39 0.00036   31.8  13.5  155   57-246     9-174 (185)
</pre><pre>  9 3FEW_X Colicin S4; Colicin S4,  88.0      74 0.00069   34.7  18.9  217    1-258   276-494 (505)
</pre><pre> 10 2I88_A Colicin-E1; protein-mem  86.9      48 0.00045   31.4  18.0  169   54-259     9-190 (191)
</pre><pre> 11 4UY3_A SEPTATION RING FORMATIO  85.2      46 0.00042   30.6  11.6  101    2-113    92-198 (198)
</pre><pre> 12 PF15967.7 ; Nucleoporin_FG2 ;   84.2      20 0.00018   39.5  10.2   76    1-91    352-434 (596)
</pre><pre> 13 6THK_A Pyocin S5; Bacteriocin,  84.0 1.2E+02  0.0011   33.3  19.3  207    3-258   283-495 (506)
</pre><pre> 14 PF04048.16 ; Sec8_exocyst ; Se  81.6      42 0.00039   29.7   9.6  130   12-152     2-135 (142)
</pre><pre> 15 5IJN_N Nuclear pore glycoprote  79.8 1.6E+02  0.0015   31.9  15.2  140    3-162   342-488 (522)
</pre><pre> 16 d1ciia1 f.1.1.1 (A:451-624) Co  77.0 1.2E+02  0.0011   28.8  15.0  162   43-246     1-162 (174)
</pre><pre> 17 d2csba1 a.60.2.4 (A:351-409) T  76.5      13 0.00012   27.5   4.1   33   19-72     12-44  (59)
</pre><pre> 18 1CII_A COLICIN IA; COLICIN, BA  75.6 2.4E+02  0.0023   31.8  19.4  203    2-246   383-590 (602)
</pre><pre> 19 3ZQE_A PROTEIN PRGI, CELL INVA  75.3      38 0.00036   34.0   8.4   96    3-106    21-139 (305)
</pre><pre> 20 TIGR03001 Sig-70_gmx1; RNA pol  74.1 1.2E+02  0.0011   27.7  12.6   92    3-104   146-243 (244)
</pre><pre> 21 PF04124.14 ; Dor1 ; Dor1-like   73.4 1.6E+02  0.0015   28.9  14.7  149    2-157     9-168 (317)
</pre><pre> 22 2D4Y_A Flagellar hook-associat  71.9 1.8E+02  0.0017   30.6  12.4  100    1-100    20-149 (463)
</pre><pre> 23 2PFT_A Exocytosis Protein; hel  71.6      58 0.00054   34.6   9.1  111   26-148   412-538 (571)
</pre><pre> 24 5IJN_G NUCLEAR PORE COMPLEX PR  71.1   3E+02  0.0028   30.9  14.3  153    1-157   257-414 (599)
</pre><pre> 25 5IJO_M Nucleoporin p58/p45; Nu  71.1   3E+02  0.0028   30.9  14.3  153    1-157   257-414 (599)
</pre><pre> 26 2B1E_A Exocyst complex compone  69.5      80 0.00074   33.7   9.6  118   26-158   416-550 (564)
</pre><pre> 27 6YS8_C GldL; membrane protein   68.6   1E+02 0.00093   29.8   9.1   92    1-98     99-199 (215)
</pre><pre> 28 6YS8_G GldL; membrane protein   68.6   1E+02 0.00093   29.8   9.1   92    1-98     99-199 (215)
</pre><pre> 29 TIGR03513 GldL_gliding; glidin  66.7 1.4E+02  0.0013   28.8   9.5   82   17-98     98-190 (202)
</pre><pre> 30 PF10475.11 ; Vps54_N ; Vacuola  65.5      33 0.00031   32.2   5.4  131   26-157     8-140 (299)
</pre><pre> 31 5J5L_A Uncharacterized protein  65.1 1.6E+02  0.0015   25.6  10.3  100    2-102    45-146 (160)
</pre><pre> 32 d2b1ea_ a.118.17.2 (A:) automa  64.9 2.4E+02  0.0022   30.5  11.9  142    2-158   369-543 (557)
</pre><pre> 33 4YJW_A DUF3829 family protein;  64.2 1.4E+02  0.0013   27.7   8.8   67   19-97     77-149 (169)
</pre><pre> 34 PF06511.13 ; T3SS_TC ; Type II  63.1 1.1E+02   0.001   31.7   8.7  110    3-120    77-194 (337)
</pre><pre> 35 d3favb_ a.25.3.1 (B:) ESAT-6,   62.6 1.1E+02   0.001   22.6   8.1   69    4-72      1-69  (72)
</pre><pre> 36 PF19658.1 ; DUF6161 ; Family o  61.4 2.7E+02  0.0025   26.8  10.5   91    2-95     10-100 (223)
</pre><pre> 37 PF05120.14 ; GvpG ; Gas vesicl  61.1      65 0.00061   25.4   5.4   38   57-99     31-68  (72)
</pre><pre> 38 3NZZ_A Cell invasion protein s  60.5      65  0.0006   32.8   6.6  107    3-117    46-153 (308)
</pre><pre> 39 PF13949.8 ; ALIX_LYPXL_bnd ; A  60.1 2.8E+02  0.0026   26.6  10.5  118    1-120   143-267 (284)
</pre><pre> 40 6DLC_A Designed protein DHD1:2  59.8      61 0.00056   27.2   5.2   45   83-127    11-55  (115)
</pre><pre> 41 6J6F_A Envelope glycoprotein;   59.8   3E+02  0.0028   26.9  15.9  164    2-178    44-210 (313)
</pre><pre> 42 4YUU_q1 Extrinsic protein in p  58.8 3.2E+02  0.0029   26.8  12.7  103    1-103   105-216 (218)
</pre><pre> 43 TIGR03042 PS_II_psbQ_bact; pho  58.1 2.4E+02  0.0022   25.1   9.9  103    1-103    34-141 (142)
</pre><pre> 44 2YM9_D CELL INVASION PROTEIN S  58.0      69 0.00064   33.2   6.4  107    3-117    84-191 (346)
</pre><pre> 45 4JIO_B BRO1; Membrane Traffick  57.9 3.6E+02  0.0034   27.2  12.4  132    1-157   198-340 (344)
</pre><pre> 46 4LWS_A Uncharacterized protein  57.1 2.1E+02  0.0019   24.1  11.8   95    1-100    39-133 (135)
</pre><pre> 47 1WDF_A E2 glycoprotein; MHV, c  53.0      87  0.0008   26.7   5.1   66    2-72     27-92  (95)
</pre><pre> 48 TIGR02492 flgK_ends; flagellar  52.6 4.5E+02  0.0041   26.6  12.5   97    2-100   109-210 (323)
</pre><pre> 49 4UT1_A FLAGELLAR HOOK-ASSOCIAT  51.8 5.7E+02  0.0052   29.0  12.3  100    1-100    91-218 (673)
</pre><pre> 50 PF03081.17 ; Exo70 ; Exo70 exo  51.7 4.4E+02  0.0041   26.3  15.1  138    2-154   204-351 (373)
</pre><pre> 51 3L1N_A Cell wall antigen; heli  51.6 3.4E+02  0.0031   25.0  10.8  102    1-102    86-189 (194)
</pre><pre> 52 5IJN_G NUCLEAR PORE COMPLEX PR  51.6 2.5E+02  0.0023   31.5   9.3   98    1-115   356-460 (599)
</pre><pre> 53 5IJO_M Nucleoporin p58/p45; Nu  51.6 2.5E+02  0.0023   31.5   9.3   98    1-115   356-460 (599)
</pre><pre> 54 7D6D_A Sorting nexin-1; Coat c  50.5 6.1E+02  0.0056   27.6  12.9  149    1-154   315-482 (529)
</pre><pre> 55 7D6D_I Sorting nexin-1; Coat c  50.5 6.1E+02  0.0056   27.6  12.9  149    1-154   315-482 (529)
</pre><pre> 56 5E7X_A Cell wall antigen; palm  50.0 2.9E+02  0.0027   23.6  10.4  100    2-102    47-148 (155)
</pre><pre> 57 TIGR02131 phaP_Bmeg; polyhydro  49.9 3.9E+02  0.0036   25.2  10.2  100   41-150     6-105 (165)
</pre><pre> 58 d1yaro_ a.24.8.1 (O:) automate  49.9 2.8E+02  0.0026   27.4   8.6   91    1-96     32-156 (228)
</pre><pre> 59 PF19180.2 ; DUF5862 ; Family o  49.6 1.7E+02  0.0016   23.3   6.0   51  200-259     1-52  (68)
</pre><pre> 60 PF05524.15 ; PEP-utilisers_N ;  49.6 1.6E+02  0.0014   24.3   6.2   72    1-73     35-118 (119)
</pre><pre> 61 6S7O_E Dolichyl-diphosphooligo  49.3   6E+02  0.0055   28.3  11.9   91    3-103   510-600 (607)
</pre><pre> 62 PF02252.20 ; PA28_beta ; Prote  48.8   2E+02  0.0019   25.2   7.1   81   10-96      6-86  (145)
</pre><pre> 63 d1zyma1 a.60.10.1 (A:22-144) E  48.7   3E+02  0.0028   23.5   8.3   89    1-96     18-120 (123)
</pre><pre> 64 PF08855.12 ; DUF1825 ; Domain   48.1 1.9E+02  0.0018   24.7   6.5   63   57-120     7-71  (103)
</pre><pre> 65 1AVO_D 11S REGULATOR; PROTEASO  47.6 2.1E+02  0.0019   25.0   6.9   81   10-96      6-86  (140)
</pre><pre> 66 1AVO_J 11S REGULATOR; PROTEASO  47.6 2.1E+02  0.0019   25.0   6.9   81   10-96      6-86  (140)
</pre><pre> 67 1WDG_B E2 glycoprotein; MHV, c  46.6 1.7E+02  0.0016   25.0   5.8   67    2-72     13-91  (94)
</pre><pre> 68 PF15469.8 ; Sec5 ; Exocyst com  46.5 8.1E+02  0.0075   27.9  15.0  130    2-157    49-183 (819)
</pre><pre> 69 PF10157.11 ; BORCS6 ; BLOC-1-r  46.4 4.1E+02  0.0038   24.4   9.6   90    2-97     63-152 (157)
</pre><pre> 70 5CSD_C Envelope glycoprotein;   46.0 3.4E+02  0.0032   23.4  11.3  100    2-102    47-148 (159)
</pre><pre> 71 6W08_A Motility Associated Kil  44.8 5.6E+02  0.0052   25.6  10.8  108    1-112   240-349 (359)
</pre><pre> 72 3R9V_B Invasin ipaD; type III   44.4   3E+02  0.0028   28.1   8.2   84    7-106    54-141 (286)
</pre><pre> 73 PF01601.18 ; CoV_S2 ; Coronavi  44.0 8.9E+02  0.0082   27.6  12.1   89    1-98    211-299 (563)
</pre><pre> 74 3H6P_B ESAT-6 LIKE PROTEIN ESX  43.4 3.4E+02  0.0032   22.7   9.0   71    3-73     23-93  (111)
</pre><pre> 75 PF10303.11 ; DUF2408 ; Protein  43.3 3.1E+02  0.0029   24.6   7.3  115   16-132     1-138 (141)
</pre><pre> 76 d2np3a1 a.4.1.9 (A:35-99) Puta  42.8   1E+02 0.00097   22.2   3.8   38   96-133     2-39  (65)
</pre><pre> 77 PF11101.10 ; DUF2884 ; Protein  42.6 5.5E+02  0.0051   24.8  13.0  125   18-151    42-180 (227)
</pre><pre> 78 d1sgma1 a.4.1.9 (A:5-77) Putat  42.3 1.2E+02  0.0011   23.1   4.3   44   87-133     2-45  (73)
</pre><pre> 79 PF10439.11 ; Bacteriocin_IIc ;  42.3      51 0.00047   26.6   2.3   62  184-245    15-85  (86)
</pre><pre> 80 6GSM_p Eukaryotic translation   42.2 1.7E+02  0.0016   31.7   6.8   54   53-106   594-647 (661)
</pre><pre> 81 3FX7_A Putative uncharacterize  42.2   3E+02  0.0028   21.6  10.1   80    2-98      8-87  (94)
</pre><pre> 82 PF07964.13 ; Red1 ; Rec10 / Re  42.1 3.4E+02  0.0032   31.2   9.0   70   47-133   623-696 (699)
</pre><pre> 83 d2pv4a1 a.286.1.1 (A:1-144) Un  42.0   5E+02  0.0046   24.1   9.8   94    4-120    44-138 (144)
</pre><pre> 84 5XLN_B Threonine--tRNA ligase,  41.6      64  0.0006   22.9   2.5   17   83-99     22-38  (45)
</pre><pre> 85 6J19_B ESAT-6-like protein Esx  41.3 2.6E+02  0.0024   20.7  11.1   90    1-98     11-100 (104)
</pre><pre> 86 5YFP_D Exocyst complex compone  41.3 5.4E+02   0.005   30.8  10.7  144    3-157    26-173 (1065)
</pre><pre> 87 PF11887.10 ; Mce4_CUP1 ; Chole  40.8 6.5E+02   0.006   25.1  11.7   94    2-105    89-182 (268)
</pre><pre> 88 5YFP_E Exocyst complex compone  40.5   1E+03  0.0096   27.4  12.6  139    2-157    81-232 (871)
</pre><pre> 89 6W08_A Motility Associated Kil  40.5 6.6E+02  0.0061   25.1  16.6  204    1-244    19-229 (359)
</pre><pre> 90 TIGR02950 SigM_subfam; RNA pol  39.8 2.4E+02  0.0022   22.9   5.8   61    4-66     91-152 (154)
</pre><pre> 91 TIGR03879 near_KaiC_dom; proba  39.7 2.4E+02  0.0022   19.8   5.8   61   87-148    10-71  (73)
</pre><pre> 92 3S84_A Apolipoprotein A-IV; fo  39.6 5.5E+02  0.0051   23.9  11.4   97    3-106   172-268 (273)
</pre><pre> 93 PF10168.11 ; Nup88 ; Nuclear p  39.4 9.9E+02  0.0092   26.8  16.0  128    2-157   529-657 (706)
</pre><pre> 94 5JG7_A Fur regulated Salmonell  39.4 1.5E+02  0.0014   25.9   4.8   35    2-36    107-141 (160)
</pre><pre> 95 PF11172.10 ; DUF2959 ; Protein  39.3 5.9E+02  0.0054   24.2   9.7  139    2-148    27-167 (190)
</pre><pre> 96 5XSJ_L Signal transduction his  39.1 4.5E+02  0.0042   22.8   8.3   96   56-157    22-122 (148)
</pre><pre> 97 2DII_A TFIIH basal transcripti  38.6 1.5E+02  0.0014   23.2   4.1   41   54-95     17-57  (61)
</pre><pre> 98 TIGR01636 phage_rinA; phage tr  38.3 1.3E+02  0.0012   24.5   4.2   31    1-31     65-95  (134)
</pre><pre> 99 d1c02a_ a.24.10.2 (A:) Phospho  38.1 5.5E+02  0.0051   23.9   8.3   98   30-129     9-112 (166)
</pre><pre>100 d3fava_ a.25.3.1 (A:) automate  37.9 2.9E+02  0.0027   20.3   7.0   74    5-78      1-74  (74)
</pre><pre>101 d1pq4a_ c.92.2.2 (A:) Periplas  37.7 2.9E+02  0.0027   27.1   6.9   69    3-96    139-207 (289)
</pre><pre>102 2P7N_A Pathogenicity island 1   37.7   4E+02  0.0037   28.7   8.2  106    3-117   118-235 (407)
</pre><pre>103 4RL5_B Exocyst complex compone  37.3 9.4E+02  0.0088   26.0  14.0  145    2-158   374-542 (565)
</pre><pre>104 6FTJ_1 Dolichyl-diphosphooligo  37.2 2.5E+02  0.0023   26.2   6.0   74   83-157    46-122 (162)
</pre><pre>105 6JLU_Q Extrinsic protein in ph  36.7 7.1E+02  0.0066   24.4  10.7  100    1-101   110-209 (211)
</pre><pre>106 d4j42a_ a.25.3.0 (A:) automate  36.6   3E+02  0.0028   20.0   8.7   72    1-72      3-74  (80)
</pre><pre>107 TIGR01417 PTS_I_fam; phosphoen  36.4 3.4E+02  0.0031   29.7   7.8   89    1-96     38-140 (565)
</pre><pre>108 7C53_B Spike protein S2',pan-C  36.4 1.1E+02   0.001   26.6   3.5   67    2-72     36-103 (107)
</pre><pre>109 7C53_C Spike protein S2',pan-C  36.4 1.1E+02   0.001   26.6   3.5   67    2-72     36-103 (107)
</pre><pre>110 4I0X_E ESAT-6-like protein MAB  36.0 3.3E+02   0.003   20.3  10.6   90    1-98      1-90  (94)
</pre><pre>111 5YFP_B Exocyst complex compone  35.7 1.4E+03   0.013   27.4  16.4  130    2-157   126-267 (971)
</pre><pre>112 5MX5_G Proteasome activator co  35.1 4.9E+02  0.0046   25.5   7.9   81   10-96    108-188 (249)
</pre><pre>113 PF05277.14 ; DUF726 ; Protein   34.4   2E+02  0.0018   29.3   5.4   61  199-259    11-72  (340)
</pre><pre>114 4XCP_A Nematode fatty acid ret  34.0   6E+02  0.0056   23.1   7.9   95    3-97     76-170 (170)
</pre><pre>115 4AKV_A SORTING NEXIN-33; TRANS  33.7 8.2E+02  0.0076   24.2  11.2  151    2-152   190-364 (386)
</pre><pre>116 d2o1ea_ c.92.2.0 (A:) automate  33.7   3E+02  0.0028   26.6   6.3   70    3-97    119-188 (282)
</pre><pre>117 TIGR03164 UHCUDC; OHCU decarbo  33.6 3.2E+02   0.003   24.9   6.1   65   10-74     38-107 (157)
</pre><pre>118 3CX3_A Lipoprotein; zinc-bindi  33.3 3.7E+02  0.0035   26.0   6.8   69    3-96    126-194 (284)
</pre><pre>119 PF12920.9 ; TcdA_TcdB_pore ; T  33.3 3.5E+02  0.0033   30.6   7.5  111  126-238   154-267 (660)
</pre><pre>120 7BAS_C CC-Type2-(TgLaId)4-W19B  33.2 2.1E+02   0.002   19.1   3.5   21   52-72      4-24  (32)
</pre><pre>121 2CCY_B CYTOCHROME C; ELECTRON   32.9 5.3E+02  0.0049   21.8   9.1  100   81-181     4-120 (128)
</pre><pre>122 5XBJ_A Flagellar hook-associat  32.9 1.1E+03    0.01   25.6  12.6   93    2-96     44-142 (511)
</pre><pre>123 d3gwkc1 a.25.3.0 (C:4-97) auto  32.7 3.7E+02  0.0034   19.9  10.1   90    1-98      5-94  (94)
</pre><pre>124 PF05823.14 ; Gp-FAR-1 ; Nemato  32.7 5.9E+02  0.0054   22.2   7.5   75    3-77     55-129 (149)
</pre><pre>125 TIGR01834 PHA_synth_III_E; pol  32.6 9.7E+02   0.009   24.7  13.5  131   10-158   175-310 (320)
</pre><pre>126 4J2C_C Syntaxin-6; Tethering c  32.5 3.8E+02  0.0035   22.0   5.9  101   46-148     1-104 (110)
</pre><pre>127 5IJN_N Nuclear pore glycoprote  32.5 5.7E+02  0.0053   28.1   8.2  110    1-110   392-519 (522)
</pre><pre>128 d1t33a1 a.4.1.9 (A:1-88) Putat  32.4   2E+02  0.0019   22.7   4.2   43   87-133    12-54  (88)
</pre><pre>129 1ZVA_A E2 glycoprotein; SARS c  32.3 2.4E+02  0.0022   23.3   4.5   71    2-72      2-75  (77)
</pre><pre>130 1LS4_A Apolipophorin-III; Heli  32.0 7.7E+02  0.0072   23.4  11.9  101    5-105    26-136 (180)
</pre><pre>131 5W56_A Periplasmic solute bind  31.9   4E+02  0.0037   25.9   6.8   50    2-76    120-169 (286)
</pre><pre>132 d2xpwa1 a.4.1.9 (A:3-67) Tetra  31.6   2E+02  0.0019   20.5   3.8   38   96-133     8-45  (65)
</pre><pre>133 d1ui5a1 a.4.1.9 (A:5-75) A-fac  31.6 3.6E+02  0.0033   19.5   5.1   69   88-156     2-71  (71)
</pre><pre>134 TIGR03017 EpsF; chain length d  31.5 9.8E+02  0.0091   24.4  12.1   84   20-103   142-227 (444)
</pre><pre>135 3HYI_A Protein DUF199/WhiA; LA  31.5 8.9E+02  0.0083   24.1   9.1  107    9-133   175-283 (295)
</pre><pre>136 PF07200.15 ; Mod_r ; Modifier   31.4   6E+02  0.0056   22.0  10.3  103   17-134     1-103 (146)
</pre><pre>137 4X6G_A OxyR; OxyR, peroxide, T  31.4      61 0.00057   29.6   1.4   76   26-103     4-88  (316)
</pre><pre>138 4X6G_B OxyR; OxyR, peroxide, T  31.4      61 0.00057   29.6   1.4   76   26-103     4-88  (316)
</pre><pre>139 6BFU_A Spike protein; coronavi  31.3 9.5E+02  0.0088   29.4  10.4   80    6-98    779-858 (1024)
</pre><pre>140 PF05461.13 ; ApoL ; Apolipopro  31.3   3E+02  0.0027   27.4   6.0   57  196-258    88-144 (298)
</pre><pre>141 PF18249.3 ; Ca_bind_SSO6904 ;   31.2 2.6E+02  0.0024   23.8   4.6   51   56-106    21-76  (90)
</pre><pre>142 PF10819.10 ; DUF2564 ; Protein  31.2 5.5E+02  0.0051   21.4   9.6   64    5-101    12-75  (78)
</pre><pre>143 6V8I_BF Tape Measure Protein,   31.1 1.6E+03   0.015   26.7  19.6  251    1-255    13-316 (1154)
</pre><pre>144 5MX5_M Proteasome activator co  31.1 6.3E+02  0.0059   24.5   7.9   81   10-96     98-178 (239)
</pre><pre>145 4LWS_B Uncharacterized protein  31.1 3.9E+02  0.0036   19.7  10.5   89    1-97     14-102 (103)
</pre><pre>146 TIGR02984 Sig-70_plancto1; RNA  31.0 4.5E+02  0.0042   22.1   6.3   63    2-66    124-187 (189)
</pre><pre>147 4H0F_B Laminin-binding surface  31.0 4.5E+02  0.0042   25.1   6.9   69    3-96    106-174 (264)
</pre><pre>148 2O1E_B YcdH; alpha-beta protei  30.9 3.6E+02  0.0034   26.8   6.5   70    3-97    139-208 (312)
</pre><pre>149 PF06730.13 ; FAM92 ; FAM92 pro  30.8 7.3E+02  0.0068   22.7  10.3   97    1-102    19-115 (219)
</pre><pre>150 3A6M_A Protein grpE; GRPE, COI  30.8 6.8E+02  0.0063   22.4  12.8  102    1-103    13-114 (177)
</pre><pre>151 6VME_V Vacuolar protein sortin  30.7 6.7E+02  0.0062   22.2   8.2   76   58-141    44-120 (122)
</pre><pre>152 6NZK_C Spike surface glycoprot  30.7 1.1E+03    0.01   29.6  10.9   89    2-98   1025-1125(1322)
</pre><pre>153 6ZT1_E CC-Type2-(LaIdGe)4; coi  30.5 2.5E+02  0.0024   18.8   3.5   21   52-72      4-24  (32)
</pre><pre>154 6ZT1_I CC-Type2-(LaIdGe)4; coi  30.5 2.5E+02  0.0024   18.8   3.5   21   52-72      4-24  (32)
</pre><pre>155 2HWG_B Phosphoenolpyruvate-pro  30.5 7.3E+02  0.0068   27.1   9.0   89    1-96     39-141 (575)
</pre><pre>156 d2o3fa1 a.4.1.20 (A:1-83) Puta  30.5 3.8E+02  0.0036   20.0   5.3   75   83-160     3-83  (83)
</pre><pre>157 1PQ4_B periplasmic binding pro  30.3 3.7E+02  0.0035   26.4   6.4   69    3-96    139-207 (291)
</pre><pre>158 7JJ8_A Zinc-binding lipoprotei  30.2 6.4E+02   0.006   24.7   7.9   69    3-96    120-188 (299)
</pre><pre>159 5ZVK_A Human Coronavirus MERS   30.1 5.9E+02  0.0054   21.4   8.4   73   25-98      6-78  (80)
</pre><pre>160 5L1X_G hMPV F2 subunit; class   30.1   1E+02 0.00092   25.8   2.1   42   50-91     47-88  (89)
</pre><pre>161 7KIP_B Spike glycoprotein; NL6  30.0 9.8E+02  0.0091   30.3  10.5   80    6-98    973-1052(1356)
</pre><pre>162 3LK4_K F-actin-capping protein  30.0 2.5E+02  0.0023   27.8   5.2   56   53-108   207-266 (277)
</pre><pre>163 2BEZ_C Spike glycoprotein; COI  30.0 5.8E+02  0.0054   21.3   7.6   73   25-98      1-73  (77)
</pre><pre>164 5AMO_A NOELIN; SIGNALING PROTE  29.9 1.3E+03   0.012   25.4  13.2  116    1-120    77-195 (471)
</pre><pre>165 PF00440.25 ; TetR_N ; Bacteria  29.9 1.9E+02  0.0018   17.3   3.1   38   96-133     2-39  (47)
</pre><pre>166 7NRC_A GCN1; Ribosome, Disome,  29.8 2.9E+02  0.0027   31.8   5.9  157   12-177   275-433 (1209)
</pre><pre>167 TIGR02959 SigZ; RNA polymerase  29.8 4.5E+02  0.0041   22.3   6.1   64    1-66     83-147 (170)
</pre><pre>168 PF01442.20 ; Apolipoprotein ;   29.8 7.2E+02  0.0067   22.3  10.6  100    3-106    84-186 (192)
</pre><pre>169 3DYT_A Sorting nexin-9; 3-heli  29.7 3.5E+02  0.0033   25.9   6.0  142    5-151   202-343 (366)
</pre><pre>170 6LOD_D Uncharacterized protein  29.7   5E+02  0.0046   23.4   6.7   56  200-255    73-135 (192)
</pre><pre>171 PF15449.8 ; Retinal ; Retinal   29.5   1E+03  0.0097   29.8  10.5   85   10-99    227-331 (1264)
</pre><pre>172 6PZ8_A S protein; MERS-CoV, co  29.5 7.5E+02   0.007   27.4   8.6   73   25-98    237-309 (472)
</pre><pre>173 3RD3_A Probable transcriptiona  29.2 3.5E+02  0.0033   22.3   5.3   53   78-133     1-53  (197)
</pre><pre>174 7M0I_A Fusion glycoprotein F2;  29.1   1E+02 0.00093   25.8   2.0   42   50-91     47-88  (89)
</pre><pre>175 PF14637.8 ; FNIP_M ; Folliculi  29.0 3.6E+02  0.0033   26.3   5.9   88   21-108    23-127 (224)
</pre><pre>176 2RT6_A Primosomal replication   28.7 5.9E+02  0.0055   21.0   7.8   76    1-105    13-88  (98)
</pre><pre>177 6IN7_B RNA polymerase sigma-H   28.7   6E+02  0.0056   21.1   6.8   68    4-73    124-192 (193)
</pre><pre>178 PF10824.10 ; T7SS_ESX_EspC ; E  28.7 4.2E+02  0.0039   19.3   8.0   87    3-97     10-98  (99)
</pre><pre>179 PF04353.15 ; Rsd_AlgQ ; Regula  28.6 7.3E+02  0.0067   22.0   9.0   91   12-102    51-142 (151)
</pre><pre>180 2H7O_A Protein kinase ypkA; Yp  28.5 1.2E+03   0.011   24.4   9.8   96    1-96    178-289 (303)
</pre><pre>181 6J6F_A Envelope glycoprotein;   28.1   1E+03  0.0093   23.5  11.5  100    2-102   204-305 (313)
</pre><pre>182 d4akrb_ e.43.1.2 (B:) automate  28.1 2.5E+02  0.0024   27.3   4.8   40   53-92    208-248 (251)
</pre><pre>183 5ZUV_A Spike glycoprotein,Spik  28.0 6.7E+02  0.0062   23.0   6.8   67    2-72     47-128 (133)
</pre><pre>184 5ZUV_C Spike glycoprotein,Spik  28.0 6.7E+02  0.0062   23.0   6.8   67    2-72     47-128 (133)
</pre><pre>185 6VSJ_A Spike glycoprotein; MHV  27.9 1.3E+03   0.012   29.1  10.8   89    2-98    988-1088(1275)
</pre><pre>186 2KJG_A Archaeal protein SSO690  27.7 2.5E+02  0.0023   24.2   4.0   51   56-106    21-76  (99)
</pre><pre>187 PF05818.14 ; TraT ; Enterobact  27.7 1.5E+02  0.0014   28.3   3.3   41  198-248    87-127 (215)
</pre><pre>188 PF06148.13 ; COG2 ; COG (conse  27.7 6.4E+02  0.0059   21.1  11.0  100    5-104    28-133 (133)
</pre><pre>189 6NSI_A Manganese-binding prote  27.7 6.4E+02   0.006   24.5   7.4   69    3-96    108-178 (287)
</pre><pre>190 6U7K_B Spike glycoprotein; PED  27.5 1.1E+03   0.011   29.9  10.5   80    6-98    994-1073(1399)
</pre><pre>191 PF09632.12 ; Rac1 ; Rac1-bindi  27.2 1.2E+03   0.011   24.2   9.8   96    1-96    168-279 (293)
</pre><pre>192 PF04100.14 ; Vps53_N ; Vps53-l  27.1 1.2E+03   0.011   24.0  13.4  142    2-157    26-173 (413)
</pre><pre>193 5IJN_G NUCLEAR PORE COMPLEX PR  27.0 9.4E+02  0.0087   27.5   8.9  101    1-103   304-406 (599)
</pre><pre>194 5IJO_M Nucleoporin p58/p45; Nu  27.0 9.4E+02  0.0087   27.5   8.9  101    1-103   304-406 (599)
</pre><pre>195 PF11855.10 ; DUF3375 ; Protein  27.0 6.6E+02  0.0061   27.2   7.9  148    1-154   137-304 (468)
</pre><pre>196 6ZZ5_BBB SmhB; Pore forming to  26.9 8.6E+02   0.008   24.5   8.3   68    1-68    232-301 (366)
</pre><pre>197 7JMN_O MED15; Transcription, A  26.9 9.9E+02  0.0092   30.8  10.0  104    2-157  1026-1140(1625)
</pre><pre>198 TIGR02952 Sig70_famx2; RNA pol  26.8 3.6E+02  0.0033   22.2   4.9   59    6-66    110-169 (170)
</pre><pre>199 1EQ1_A APOLIPOPHORIN-III; Five  26.7   9E+02  0.0084   22.5  10.3  101    4-105     8-117 (166)
</pre><pre>200 PF05508.13 ; Ran-binding ; Ran  26.6 1.4E+03   0.013   24.5  10.7   90    3-100    45-145 (347)
</pre><pre>201 PF05263.13 ; DUF722 ; Protein   26.5 2.5E+02  0.0023   23.9   4.0   31    1-31     64-94  (134)
</pre><pre>202 7CYD_B Spike glycoprotein; Alp  26.5 1.2E+03   0.011   29.0  10.2   80    6-98    792-871 (1116)
</pre><pre>203 6U42_5S FAP106; motile cilia,   26.5   9E+02  0.0084   23.5   8.1   65    3-69    145-225 (240)
</pre><pre>204 6U42_5V FAP106; motile cilia,   26.5   9E+02  0.0084   23.5   8.1   65    3-69    145-225 (240)
</pre><pre>205 6PMI_F RNA polymerase sigma fa  26.5 7.9E+02  0.0073   22.0   7.3   69    6-82    176-244 (247)
</pre><pre>206 3S84_A Apolipoprotein A-IV; fo  26.5 9.2E+02  0.0085   22.5  11.0   98    3-103    84-181 (273)
</pre><pre>207 4I0X_J ESAT-6-like protein MAB  26.4   5E+02  0.0047   19.5  10.1   89    1-97     12-102 (103)
</pre><pre>208 6JX7_B Feline Infectious Perit  26.4 1.1E+03    0.01   30.2  10.1   80    6-98   1065-1144(1468)
</pre><pre>209 1TOA_B PROTEIN (PERIPLASMIC BI  26.4 4.7E+02  0.0044   25.9   6.4   69    3-96    144-214 (313)
</pre><pre>210 d3aa0a_ e.43.1.1 (A:) Capz alp  26.4 2.2E+02  0.0021   28.1   4.2   40   53-92    216-256 (270)
</pre><pre>211 TIGR02895 spore_sigI; RNA poly  26.3 6.1E+02  0.0056   22.3   6.5   68    6-81    148-215 (218)
</pre><pre>212 7BUL_A General transcription f  26.3 8.9E+02  0.0083   22.3   7.7   67   28-95     89-155 (161)
</pre><pre>213 3H6P_C ESAT-6-like protein esx  26.3   5E+02  0.0046   19.3  11.1   88    1-96      7-94  (96)
</pre><pre>214 6GY7_A XaxB; bacterial toxin,   26.2 1.3E+03   0.012   24.0   9.6  145    1-157   189-336 (367)
</pre><pre>215 3PLT_C Sphingolipid long chain  26.2   1E+03  0.0096   23.0  11.2  148    2-156    39-190 (234)
</pre><pre>216 2OV3_A Periplasmic binding pro  25.9 4.4E+02   0.004   25.1   5.9   70    3-97    107-176 (260)
</pre><pre>217 PF09577.12 ; Spore_YpjB ; Spor  25.9 9.4E+02  0.0087   22.4  11.7  113    2-120    47-173 (226)
</pre><pre>218 TIGR02989 Sig-70_gvs1; RNA pol  25.8 4.8E+02  0.0045   21.2   5.5   59    6-66     99-158 (159)
</pre><pre>219 TIGR02947 SigH_actino; RNA pol  25.4 5.1E+02  0.0047   21.9   5.7   62   10-73    123-185 (193)
</pre><pre>220 PF05859.14 ; Mis12 ; Mis12 pro  25.3   8E+02  0.0074   21.4   9.2  110   47-157     9-140 (141)
</pre><pre>221 PF15294.8 ; Leu_zip ; Leucine   25.3 1.2E+03   0.012   23.6  12.2  120    1-120   134-259 (275)
</pre><pre>222 d2q9qc_ a.278.1.1 (C:) automat  25.2 3.3E+02  0.0031   23.9   4.7   40    1-40     83-127 (145)
</pre><pre>223 5COF_A Uncharacterized protein  25.1 6.7E+02  0.0062   20.5   6.8  144    2-157    12-169 (173)
</pre><pre>224 6GRK_I AhlB; Tripartite pore-f  25.1 9.7E+02   0.009   24.2   8.3   68    1-68    233-302 (367)
</pre><pre>225 d2hyja1 a.4.1.9 (A:8-82) Putat  25.0   5E+02  0.0046   19.3   5.0   46   88-133     2-48  (75)
</pre><pre>226 2E9X_E DNA replication complex  24.9 3.6E+02  0.0033   23.3   4.8   40    1-40     83-127 (149)
</pre><pre>227 6QAJ_B Endolysin,Transcription  24.8 1.3E+03   0.012   23.7  12.8  101    2-102   405-509 (544)
</pre><pre>228 d2hkua1 a.4.1.9 (A:18-87) Puta  24.8   3E+02  0.0028   20.6   3.8   37   96-133     7-43  (70)
</pre><pre>229 TIGR02954 Sig70_famx3; RNA pol  24.7 4.3E+02   0.004   21.8   5.1   58    7-66    108-166 (169)
</pre><pre>230 TIGR03772 anch_rpt_subst; anch  24.7 5.6E+02  0.0052   27.7   7.0   49    2-75    317-365 (479)
</pre><pre>231 TIGR02937 sigma70-ECF; RNA pol  24.7 4.9E+02  0.0046   21.0   5.3   60    6-66     98-157 (158)
</pre><pre>232 3OGI_C Putative ESAT-6-like pr  24.6 6.7E+02  0.0063   20.3   9.0   72    1-72      6-77  (101)
</pre><pre>233 3ZBH_B ESXA; UNKNOWN FUNCTION,  24.4   5E+02  0.0046   18.7  10.8   89    1-97     10-98  (99)
</pre><pre>234 3ZBH_G ESXA; UNKNOWN FUNCTION,  24.4   5E+02  0.0046   18.7  10.8   89    1-97     10-98  (99)
</pre><pre>235 1WYY_A E2 Glycoprotein; membra  24.3   1E+03  0.0096   22.4   9.2   75    2-84     31-105 (149)
</pre><pre>236 d5x7va_ d.305.1.0 (A:) automat  24.3 2.1E+02   0.002   26.4   3.5   43   54-96      3-48  (189)
</pre><pre>237 6M39_A Spike glycoprotein; SAD  24.3 1.2E+03   0.011   28.4   9.8   69   25-98    751-823 (981)
</pre><pre>238 TIGR02985 Sig70_bacteroi1; RNA  24.3 5.7E+02  0.0053   20.6   5.6   59    6-66    101-160 (161)
</pre><pre>239 7M5F_A CdiI; toxin, antitoxin,  24.2 5.5E+02  0.0051   20.6   5.4   44   57-100    35-86  (98)
</pre><pre>240 4NNO_A Lipoprotein; Manganese   24.1 8.1E+02  0.0075   24.0   7.4   69    3-96    128-198 (291)
</pre><pre>241 5T12_A Phosphoenolpyruvate--pr  24.0 1.1E+03    0.01   22.6   8.2   88    1-96     39-140 (255)
</pre><pre>242 1U5E_A Src-associated adaptor   24.0 5.7E+02  0.0053   22.0   5.8   45   57-103     4-48  (211)
</pre><pre>243 2OTX_A Src kinase-associated p  24.0 5.7E+02  0.0053   22.0   5.8   45   57-103     4-48  (211)
</pre><pre>244 TIGR04449 halocin_C8_dom; halo  23.8 2.7E+02  0.0025   22.2   3.4   30  213-242    17-46  (69)
</pre><pre>245 PF10023.11 ; Aminopep ; Putati  23.8 1.4E+03   0.013   23.6   9.2  102   53-163   208-309 (320)
</pre><pre>246 d2yvea1 a.4.1.9 (A:1-74) Trans  23.8 4.6E+02  0.0043   19.2   4.6   61   96-156    10-70  (74)
</pre><pre>247 d3cx3a_ c.92.2.0 (A:) automate  23.7 3.4E+02  0.0031   26.3   4.8   34    3-36    120-153 (277)
</pre><pre>248 3S84_A Apolipoprotein A-IV; fo  23.7   1E+03  0.0096   22.1  13.8  146    3-154    18-164 (273)
</pre><pre>249 TIGR02835 spore_sigmaE; RNA po  23.6 9.6E+02  0.0089   21.7   7.7   59    6-67    166-234 (234)
</pre><pre>250 3GHG_D Fibrinogen alpha chain;  23.4 1.9E+03   0.018   25.3  10.4  111    1-120    65-185 (562)
</pre><pre>
</pre><pre>No 1
</pre><pre>>d1cola_ f.1.1.1 (A:) Colicin A {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=95.97  E-value=0.83  Score=42.70  Aligned_cols=186  Identities=15%  Similarity=0.216  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             hhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhc
</pre><pre>Q WP_048874398.1   43 KSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALG  121 (262)
</pre><pre>Q Consensus        43 ~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LG  121 (262)
</pre><pre>                      ..+..++++.++.-+...+-.+++.+-..+..+. +|+..+.  +.+=-+.|.|..+.+    .+.++-.|.+.|.++| 
</pre><pre>T Consensus        10 ~a~~~iad~~~~is~~~G~k~~~~A~elA~~~knfkGKkIRn--~~eAl~afeK~k~n~----n~K~n~~Dr~AI~~Al-   82 (197)
</pre><pre>T d1cola_          10 KTSELIAGMGDKIGEHLGDKYKAIAKDIADNIKNFQGKTIRS--FDDAMASLNKITANP----AMKINKADRDALVNAW-   82 (197)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHHTCTTCCCCC--HHHHHHHHHHHHTSG----GGCCCHHHHHHHHHHH-
</pre><pre>T ss_pred             HHHHHHHHhHHHHHHHHHHHHHHHHHHHHHHHHhcCCCCCCC--HHHHHHHHHHHhcCc----CcCCCHHHHHHHHHHH-
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCC-hHHccceeeeecccchh
</pre><pre>Q WP_048874398.1  122 LSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGR-EQSCSKVAVKKYSQFGA  200 (262)
</pre><pre>Q Consensus       122 lStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~-~~~C~k~~~~E~g~~~G  200 (262)
</pre><pre>                                      +--.+.+--+++.+.+|.+   |+++..+++..-..++.++..+|+ ..-..++-..=.|+.++
</pre><pre>T Consensus        83 ----------------~s~d~~~~A~nL~~~sK~f---g~~~~~i~~~~v~~~~~~g~~TgnW~Pl~levEs~~~g~~A~  143 (197)
</pre><pre>T d1cola_          83 ----------------KHVDAQDMANKLGNLSKAF---KVADVVMKVEKVREKSIEGYETGNWGPLMLEVESWVLSGIAS  143 (197)
</pre><pre>T ss_dssp             ----------------HTCCHHHHHHHHHHHCGGG---CCTTHHHHHHHHHHHHHHHHHHCCCHHHHHHHHHHHHTTCCH
</pre><pre>T ss_pred             ----------------HcCCHHHHHHHHHHHHHHh---hHHHHhhcHHHHHHHHHHHHhhCCChhhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>Q WP_048874398.1  201 SLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYVMD  260 (262)
</pre><pre>Q Consensus       201 ~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~Ge  260 (262)
</pre><pre>                      +++.++...     ....+.+.+|.|.+.+|++.. ++.+.+++|.-...-+..-+.+..
</pre><pre>T Consensus       144 ~~a~~~~~~-----~~~~~~~~~g~p~~~~gi~g~-ii~a~~~a~Idd~~~ekiN~~ii~  197 (197)
</pre><pre>T d1cola_         144 SVALGIFSA-----TLGAYALSLGVPAIAVGIAGI-LLAAVVGALIDDKFADALNNEIIR  197 (197)
</pre><pre>T ss_dssp             HHHHHHHHH-----HHHHHHHHCCSCHHHHHHHHH-HHHHHHHHHHCTHHHHHHCCCCCC
</pre><pre>T ss_pred             HHHHHHHHH-----HHHHHHHHcchhHHHHHHHHH-HHHHHHHHhcCHHHHHHHHHhhcC
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF15967.7 ; Nucleoporin_FG2 ; Nucleoporin FG repeated region
</pre><pre>Probab=95.63  E-value=0.42  Score=51.43  Aligned_cols=156  Identities=10%  Similarity=0.139  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      +|+...++-+++++.+..++.++..-+.+..+.|+..+...+..+--....+++..+++.+.|++.|.+.+.+-...+-.
</pre><pre>T Consensus       253 ~~~k~IKeQk~iseeIsr~S~k~i~kV~edi~~L~q~L~~v~s~Lqr~~~ai~kLK~ev~q~l~naEiA~Rtq~~p~~lq  332 (596)
</pre><pre>T W5M8W5_LEPOC/2  253 NFQKFVKEQKQVQEEISRMSSKAMLKVEEDIKSLKQLLSVCASGLQRNSLSIDKLKIETAQELKNADIALRTQKTPPGLQ  332 (596)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhcHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHccCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHH-HHHHHHHHHHHHHHHhhcCCCCh-HHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQ-KLFSELDVLLNKLSRLTLKLKPY-EEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~-~Lf~~Ld~~L~~~~~~~~~l~~~-~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                      +...+..++- +|.+++|.+|..+-..--.|..| ..+-+.-.++..-|...-++.- ..+-.++.+++.+-.-++.+|
</pre><pre>T Consensus       333 ~entap~eYF~~Lv~~fE~rL~~YrqqIeELE~hL~s~~~~~~~TpqdL~~~L~~~~-etF~aLAaqLa~lHe~V~~lK  410 (596)
</pre><pre>T W5M8W5_LEPOC/2  333 HENTAPADYFRGLVEQFEVQLQQYRQQIEELENHLTTQASSSHITPQDLSLAMQKLY-HTFVAVAAQLQSVHENVKILK  410 (596)
</pre><pre>T ss_pred             CCCCccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHcccCCCCCHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>d1a87a_ f.1.1.1 (A:) Colicin N {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=93.98  E-value=2  Score=42.66  Aligned_cols=182  Identities=10%  Similarity=0.136  Sum_probs=0.0  Template_Neff=4.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHH-----HHHHhhcCCCChHHHHHHhcC-----
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLN-----KLSRLTLKLKPYEEIKHALGL-----  122 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~-----~~~~~~~~l~~~~~lK~~LGl-----  122 (262)
</pre><pre>                      ....+.+.++.+=+.+-|++-        ++.|=.+...+-++|-..+.     ++-...--+..+++.|+.++.     
</pre><pre>T Consensus       100 ~~e~e~l~~A~~~~adf~k~i--------~ek~G~k~~~lA~elA~~~KnfqGKkIRn~~eAlkafeK~k~npn~K~n~~  171 (297)
</pre><pre>T d1a87a_         100 KNEKEALLKASELVSGMGDKL--------GEYLGVKYKNVAKEVANDIKNFHGRNIRSYNEAMASLNKVLANPKMKVNKS  171 (297)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH--------HHHHCHHHHHHHHHHHHHHHTCCGGGCCCHHHHHHHHHHHHTCTTCCCCHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHHhhcCCCCCCCHHHHHHHHHHHhcCcCCCCCHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCC-hHHccceeeeecccchhc
</pre><pre>Q WP_048874398.1  123 SSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGR-EQSCSKVAVKKYSQFGAS  201 (262)
</pre><pre>Q Consensus       123 StkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~-~~~C~k~~~~E~g~~~G~  201 (262)
</pre><pre>                      ....|+..|+..-   ...+++++.+++      |..|+++..+++..-..++.++..+|+ ..-.-++-..=.|..+++
</pre><pre>T Consensus       172 Dr~AIanAL~sld---~~~mAknL~~~s------K~Fg~~~~~i~~~~v~~k~~kg~eTGnWkPl~levEsia~g~~As~  242 (297)
</pre><pre>T d1a87a_         172 DKDAIVNAWKQVN---AKDMANKIGNLG------KAFKVADLAIKVEKIREKSIEGYNTGNWGPLLLEVESWIIGGVVAG  242 (297)
</pre><pre>T ss_dssp             HHHHHHHHHHTCC---HHHHHHHHHTTC------GGGCCTTHHHHHHHHHHHHHHHHTTCCCHHHHHHHHHHHHTTCCHH
</pre><pre>T ss_pred             HHHHHHHHHHhCC---HHHHHHHHHHHH------HHhHHHHHHhhHHHHHHHHHHHhhcCCchhhHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  202 LAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYV  258 (262)
</pre><pre>Q Consensus       202 ~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~  258 (262)
</pre><pre>                      ++.++++..      .+..+..++++..+|++.++++ +.++++.-...-+..-+.+
</pre><pre>T Consensus       243 ~a~~~f~~~------ls~~~~~~l~~t~lgI~gi~~i-a~v~s~IDd~~vekiN~~i  292 (297)
</pre><pre>T d1a87a_         243 VAISLFGAV------LSFLPISGLAVTALGVIGIMTI-SYLSSFIDANRVSNINNII  292 (297)
</pre><pre>T ss_dssp             HHHHHHHHH------HHTCCCTTCCHHHHHHHHHHHH-HHHHHHHCHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHH-HHHHHhcCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>1A87_A COLICIN N; BACTERIOCIN, TOXIN, PORE-FORMING ACTIVITY; 3.1A {Escherichia coli K12} SCOP: f.1.1.1
</pre><pre>Probab=93.24  E-value=3.4  Score=41.67  Aligned_cols=186  Identities=10%  Similarity=0.144  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHH-----HHHHhhcCCCChHHHHHHhcC-----
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLN-----KLSRLTLKLKPYEEIKHALGL-----  122 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~-----~~~~~~~~l~~~~~lK~~LGl-----  122 (262)
</pre><pre>                      ....+.+.++.+-+-+.|++.        ++.|=.+...+-++|-....     ++-...--+...++.|..++.     
</pre><pre>T Consensus       124 ~~e~~~l~~a~~~~~d~~~ki--------s~~~G~k~~~lAkeLA~~~KNfqGKkIRn~~dA~aafeK~~~n~~~K~s~a  195 (321)
</pre><pre>T 1A87_A          124 KNEKEALLKASELVSGMGDKL--------GEYLGVKYKNVAKEVANDIKNFHGRNIRSYNEAMASLNKVLANPKMKVNKS  195 (321)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH--------HHHHCHHHHHHHHHHHHHHHTCCGGGCCCHHHHHHHHHHHHTCTTCCCCHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHHhccCCCcCCCHHHHHHHHHHHhhCccccCCHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCC-hHHccceeeeecccchhc
</pre><pre>Q WP_048874398.1  123 SSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGR-EQSCSKVAVKKYSQFGAS  201 (262)
</pre><pre>Q Consensus       123 StkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~-~~~C~k~~~~E~g~~~G~  201 (262)
</pre><pre>                      ....|...|+..-   -..+++++.+++|+.      |+++..++...-..++.+++.+|+ ..--.++...=.++.+++
</pre><pre>T Consensus       196 Dr~AI~nAl~sld---~~~~A~nl~~~sKaF------g~~~~~i~~~~l~~~~~~g~~TGnW~Pl~levEsi~lg~~Aa~  266 (321)
</pre><pre>T 1A87_A          196 DKDAIVNAWKQVN---AKDMANKIGNLGKAF------KVADLAIKVEKIREKSIEGYNTGNWGPLLLEVESWIIGGVVAG  266 (321)
</pre><pre>T ss_dssp             HHHHHHHHHHTCC---HHHHHHHHHTTCGGG------CCTTHHHHHHHHHHHHHHHHTTCCCHHHHHHHHHHHHTTCCHH
</pre><pre>T ss_pred             HHHHHHHHHHhcc---HHHHHHHHHHHHHHH------HHHHHHhhHHHHHHHHhhhccCCCcHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHHhhcC
</pre><pre>Q WP_048874398.1  202 LAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYVMDKL  262 (262)
</pre><pre>Q Consensus       202 ~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~Ge~i  262 (262)
</pre><pre>                      +..++++..      .+.+.+.+++++.++++..+++ +.++++.=...-+..-..++..+
</pre><pre>T Consensus       267 ia~a~~~a~------~~~~~~~~l~~t~~gi~gi~~i-a~~sa~IDd~~vekiNn~i~~~~  320 (321)
</pre><pre>T 1A87_A          267 VAISLFGAV------LSFLPISGLAVTALGVIGIMTI-SYLSSFIDANRVSNINNIISSVI  320 (321)
</pre><pre>T ss_dssp             HHHHHHHHH------HHTCCCTTCCHHHHHHHHHHHH-HHHHHHHCHHHHHHHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHH-HHHHHHhCHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>d1rh1a2 f.1.1.1 (A:313-511) Colicin B C-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=92.51  E-value=9.5  Score=36.13  Aligned_cols=184  Identities=12%  Similarity=0.115  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHH---HHHHHHHHHHHHHHHHHHHHH---hhcCCCChHHHHHHhcCc
</pre><pre>Q WP_048874398.1   50 DAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQ---FFVERQKLFSELDVLLNKLSR---LTLKLKPYEEIKHALGLS  123 (262)
</pre><pre>Q Consensus        50 ~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~---F~~~R~~Lf~~Ld~~L~~~~~---~~~~l~~~~~lK~~LGlS  123 (262)
</pre><pre>                      ......-+-|.+.-+++-+-|-+.|.+..+---++   |.-++-+=+++=-..++++..   +.++-.|...+.++|   
</pre><pre>T Consensus         8 ~~l~~as~~i~~~~~~i~~~~G~ky~~~A~elA~~ikn~kGKkIRn~~dAl~afeK~k~n~n~K~n~~Dr~AI~~Al---   84 (199)
</pre><pre>T d1rh1a2           8 TVLTKTSEVIISVGDKVGEYLGDKYKALSREIAENINNFQGKTIRSYDDAMSSINKLMANPSLKINATDKEAIVNAW---   84 (199)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHHHCTTCCCCCHHHHHHHHHHHTTCTTCCCCHHHHHHHHHHH---
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHhhcCCCcCCCHHHHHHHHHHHhhCCCCCCCHHHHHHHHHHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchh
</pre><pre>Q WP_048874398.1  124 SRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLA  203 (262)
</pre><pre>Q Consensus       124 tkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~  203 (262)
</pre><pre>                                    +--.+.+--+++.+.+|.+   |+++..+++..-..++.++..+|+    =+--+.|.-.++-+-+
</pre><pre>T Consensus        85 --------------~s~d~~~~A~nL~~~sK~f---~~~~~~i~~~~v~~~~~~g~~TGn----W~Pl~levEs~a~g~~  143 (199)
</pre><pre>T d1rh1a2          85 --------------KAFNAEDMGNKFAALGKTF---KAADYAIKANNIREKSIEGYQTGN----WGPLMLEVESWVISGM  143 (199)
</pre><pre>T ss_dssp             --------------HHCCHHHHHHTTSSSSSCS---SSTTHHHHHHHHHHHHHHHHSSSC----CHHHHHHHHHHHHTTC
</pre><pre>T ss_pred             --------------HhCCHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHHhcCC----chhhhHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  204 GGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYV  258 (262)
</pre><pre>Q Consensus       204 Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~  258 (262)
</pre><pre>                      .+.+....-+...+.....+|++...+|++.. ++.+.++++.-...-+..-..+
</pre><pre>T Consensus       144 As~~a~~~fs~~l~~~~~~~~~~~t~~gi~g~-ii~a~v~alIDd~~vekiN~~l  197 (199)
</pre><pre>T d1rh1a2         144 ASAVALSLFSLTLGSALIAFGLSATVVGFVGV-VIAGAIGAFIDDKFVDELNHKI  197 (199)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHTTCCCCHHHHHHHHH-HHHHHHHHTTCSHHHHHCCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhhHHHHHHHHH-HHHHHHHHhcCHHHHHHHHhhh
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>1COL_B COLICIN A; ANTIBACTERIAL PROTEIN; 2.4A {Escherichia coli} SCOP: f.1.1.1
</pre><pre>Probab=91.26  E-value=24  Score=33.62  Aligned_cols=185  Identities=12%  Similarity=0.137  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             hhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhc
</pre><pre>Q WP_048874398.1   43 KSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALG  121 (262)
</pre><pre>Q Consensus        43 ~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LG  121 (262)
</pre><pre>                      ..+..+++..++..++..+-+.++.+-..+..+. +|+..+.  +.+--+-|.|..+.++.    .++-+|...|.++| 
</pre><pre>T Consensus        14 ~a~~~~ad~~~~i~~k~G~k~~~~A~elA~~~kn~qGKkIRn--~~dAl~afeK~~~n~~~----K~s~~Dr~Ai~~Al-   86 (204)
</pre><pre>T 1COL_B           14 KTSELIAGMGDKIGEHLGDKYKAIAKDIADNIKNFQGKTIRS--FDDAMASLNKITANPAM----KINKADRDALVNAW-   86 (204)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHHTCTTCCCCC--HHHHHHHHHHHHTCGGG----CCCHHHHHHHHHHH-
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccCC--HHHHHHHHHHHhcCccc----cCCHHHHHHHHHHH-
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhc
</pre><pre>Q WP_048874398.1  122 LSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGAS  201 (262)
</pre><pre>Q Consensus       122 lStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~  201 (262)
</pre><pre>                                      +--++.+--+++.+.+|.+   |+++..++...=..++.+++.+|+    -+--+.|.-.++-+
</pre><pre>T Consensus        87 ----------------~s~d~~~~a~nl~~~sK~f---g~~~~~~~~~~l~~~~~~~~~Tgn----W~pl~lk~Esi~~~  143 (204)
</pre><pre>T 1COL_B           87 ----------------KHVDAQDMANKLGNLSKAF---KVADVVMKVEKVREKSIEGYETGN----WGPLMLEVESWVLS  143 (204)
</pre><pre>T ss_dssp             ----------------HTCCHHHHHHHHHHHCGGG---CCTTHHHHHHHHHHHHHHHHHHSC----CHHHHHHHHHHHHT
</pre><pre>T ss_pred             ----------------HhcCHHHHHHHHHHHHHHH---HHHHHHhhHHHHHHHHhhhccCCC----cHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  202 LAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYV  258 (262)
</pre><pre>Q Consensus       202 ~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~  258 (262)
</pre><pre>                      -..+.+.....+.....+.+.+.+|+..+|++.. ++.+.++++.=...-+..-..+
</pre><pre>T Consensus       144 ~~Aa~~a~~~~a~~~~~~~~~~~~~~t~~gI~~~-ilmA~~~a~id~~~ve~iN~~l  199 (204)
</pre><pre>T 1COL_B          144 GIASSVALGIFSATLGAYALSLGVPAIAVGIAGI-LLAAVVGALIDDKFADALNNEI  199 (204)
</pre><pre>T ss_dssp             TCCHHHHHHHHHHHHHHHHHTCCCCHHHHHHHHH-HHHHHHHHTTCTHHHHHHCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhhhHHHHHHHHHH-HHHHHHHHHhCHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>PF11981.10 ; DUF3482 ; Domain of unknown function (DUF3482)
</pre><pre>Probab=90.43  E-value=20  Score=35.29  Aligned_cols=159  Identities=14%  Similarity=0.007  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccc
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGV  136 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~  136 (262)
</pre><pre>                      .++-+.|..|...++..+.+--.....+...++++-..-|-..|...+......++.++.++.+                
</pre><pre>T Consensus        26 ~~Ll~~l~~l~~~~~~~l~~~i~~~~~~~~~r~~~a~~~iA~~L~~~~~~~~~~~~~~~~~~~~----------------   89 (288)
</pre><pre>T S6BA96_PSERE/1   26 QRLYDSLALLLERARPQLQRLVDDHQAQRQLRLEAGSRLIAELLLDVAACRRSVPAQEAALQAA----------------   89 (288)
</pre><pre>T ss_pred             HHHHHHHHHHhHHhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCCchHHHHHHH----------------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceee---------eecccchh--chhHH
</pre><pre>Q WP_048874398.1  137 ASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAV---------KKYSQFGA--SLAGG  205 (262)
</pre><pre>Q Consensus       137 g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~---------~E~g~~~G--~~~Gg  205 (262)
</pre><pre>                        ...|.+++.+.-+               .......++|.--....+.+......         .|.-+..|  ...|+
</pre><pre>T Consensus        90 --~~~l~~~l~~~e~---------------~~~~~l~~ly~~~~~~~~~~~~~~~~~~~~~Dlf~~~~~~~~Gl~~~~ga  152 (288)
</pre><pre>T S6BA96_PSERE/1   90 --TETLRQQVRQREQ---------------RCVEALLRLYAFGQEDAATSDLPLLGGRWGDDLFNPETLKQLGVRLGGGM  152 (288)
</pre><pre>T ss_pred             --HHHHHHHHHHHHH---------------HHHHHHHHHhCCChHhhcccCCCCcCCCcccCCCCHHHHHHhhHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHH---HHHHHH
</pre><pre>Q WP_048874398.1  206 IYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEV---AGGATE  256 (262)
</pre><pre>Q Consensus       206 ~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~---G~~~G~  256 (262)
</pre><pre>                      ..|+.+|        +.+-+.|+|.++.+-.++++.+|+.++...   +....+
</pre><pre>T Consensus       153 a~GAa~G--------a~iDl~tGGlSlG~ga~iGa~~Gg~~~~~~~~~~~~~~~  198 (288)
</pre><pre>T S6BA96_PSERE/1  153 AAGAAAG--------AGVDLLVGGLTLGAAAALGAIAGGAWQTFGHYGERLLGK  198 (288)
</pre><pre>T ss_pred             HHHHHHH--------HHhhhhccCccHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>PF01024.21 ; Colicin ; Colicin pore forming domain
</pre><pre>Probab=88.25  E-value=39  Score=31.76  Aligned_cols=155  Identities=17%  Similarity=0.142  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHH-----------hcCcch
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHA-----------LGLSSR  125 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~-----------LGlStk  125 (262)
</pre><pre>                      +++++.++..-+.|++.        +..|=.+...+-++|-+..     ++-.++.+++--..           +....+
</pre><pre>T Consensus         9 ~~~~~a~~~~ad~~~~i--------~~k~G~~~~~~A~ela~~~-----kGKkiRs~~dAl~~f~K~~~n~~~K~s~~Dr   75 (185)
</pre><pre>T Q9I4Y4_PSEAE/3    9 QALQDAIKFTADFYKEV--------TEKFGARTSEMARQLAEGA-----RGKNIRSSAEAIKSFEKHKDALNKKLSLKDR   75 (185)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHc-----CCCCCCCHHHHHHHHHHHHHhhhhcCCHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchhHH
</pre><pre>Q WP_048874398.1  126 SIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLAGG  205 (262)
</pre><pre>Q Consensus       126 S~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~Gg  205 (262)
</pre><pre>                      ..+-+.=+.-  +-..+++++.+++      |..|+++..++...-..++..+..+|+    -+--+.|.-.+.-|.+.+
</pre><pre>T Consensus        76 ~Ai~~Al~s~--d~~~~A~nl~~~s------K~fg~~~~~~~~~~l~~~~~~g~~Tgn----W~pl~l~~Esi~~G~~A~  143 (185)
</pre><pre>T Q9I4Y4_PSEAE/3   76 QAIAKAFDSL--DKQMMAKSLEKFS------KGFGVVGKAIDAASLYQEFKISTETGD----WKPFFVKIETLAAGAAAS  143 (185)
</pre><pre>T ss_pred             HHHHHHHHHc--cHHHHHHHHHHHH------HHHHHHHHHhCHHHHHHHHHHHhccCC----cHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  206 IYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYI  246 (262)
</pre><pre>Q Consensus       206 ~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~  246 (262)
</pre><pre>                      .+-+.       ...+.++.|.   |++..+++.+.++++.
</pre><pre>T Consensus       144 ~~~a~-------~fs~~~~~p~---gii~~a~~~a~~~a~i  174 (185)
</pre><pre>T Q9I4Y4_PSEAE/3  144 WLVGI-------AFATATATPI---GILGFALVMAVTGAMI  174 (185)
</pre><pre>T ss_pred             HHHHH-------HHHHHhHHHH---HHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>3FEW_X Colicin S4; Colicin S4, cell lysis, Y-ray, cell killing, IMMUNE SYSTEM; 2.45A {Escherichia coli}
</pre><pre>Probab=87.99  E-value=74  Score=34.67  Aligned_cols=217  Identities=9%  Similarity=0.123  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHH-HHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQK-VNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QG   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~-v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g   78 (262)
</pre><pre>                      +..+-++++++ ..+.+.+..+++...-.+-...-..+          ++++.++.-+...+.++++.+-..+..+. +|
</pre><pre>T Consensus       276 ~eaqRq~~Akkaa~d~~ks~k~e~~~~~~~~l~~aa~~----------~~d~~~~v~~~~G~k~~~~A~elA~~~kn~qG  345 (505)
</pre><pre>T 3FEW_X          276 EEPQRQNQAKKAAYDVFYSFSMNRDRIQSDVLNKAAEV----------ISDIGNKVGDYLGDAYKSLAREIADDVKNFQG  345 (505)
</pre><pre>T ss_dssp             THHHHHHHHHHHHHHHHHTCHHHHHHHCCCCCHHHHHH----------HHHHHHHHHHHHCHHHHHHHHHHHHHHHTCTT
</pre><pre>T ss_pred             cchHHHHHHHHHHHHHHHhhhcchHHHHHHHHHHHHHH----------HHHHHHHHHHHHHHHHHHHHHHHHHHHhhcCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKM  158 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~  158 (262)
</pre><pre>                      +..+.  +.+-.+-|.|+.+-+    ...++-.|...|.++|                 .--...+.-+++.+.+|.+  
</pre><pre>T Consensus       346 KkIRs--~~dA~~af~K~~~n~----~~K~s~~Dr~AI~~Al-----------------~s~d~~~~A~nl~~~sKaf--  400 (505)
</pre><pre>T 3FEW_X          346 KTIRS--YDDAMASLNKVLSNP----GFKFNRADSDALANVW-----------------RSIDAQDMANKLGNISKAF--  400 (505)
</pre><pre>T ss_dssp             CCCCC--HHHHHHHHHHHHTCT----TCCCCHHHHHHHHHHH-----------------HHCCHHHHHHHHTTTCGGG--
</pre><pre>T ss_pred             CccCC--HHHHHHHHHHHhcCC----CccCCHHHHHHHHHHH-----------------HhCcHHHHHHHHHHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHH
</pre><pre>Q WP_048874398.1  159 GGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVV  238 (262)
</pre><pre>Q Consensus       159 gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv  238 (262)
</pre><pre>                       |+++..++...-..++.+.+.+|+    -+--+.|.-.++-+-+.+.+.+.+-+.....+.+.+++|...++++.+ ++
</pre><pre>T Consensus       401 -g~~~~~~~~~~v~~~~~~g~~Tgn----W~pl~le~Es~~~~~~A~~~~~~~~s~~~~~~~~~~~~p~taigi~g~-~i  474 (505)
</pre><pre>T 3FEW_X          401 -KFADVVMKVEKVREKSIEGYETGN----WGPLMLEVESWVLSGIASAVALGVFSATLGAYALSLGAPAIAVGIVGI-LL  474 (505)
</pre><pre>T ss_dssp             -CSSCCHHHHHHHHHHHHHHHHHSC----CHHHHHHHHHHHHTTCCHHHHHHHHHHHHHHHHHHCCSCSSHHHHHHH-HH
</pre><pre>T ss_pred             -HHHHHhhcHHHHHHHHhhhccCCC----chhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhchhHHHHHHHHH-HH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  239 GAAAGGYIGGEVAGGATEYV  258 (262)
</pre><pre>Q Consensus       239 ~~g~g~~~gg~~G~~~G~~~  258 (262)
</pre><pre>                      .+.++++.=...-...-..+
</pre><pre>T Consensus       475 ~a~~~a~idd~~v~~iN~~i  494 (505)
</pre><pre>T 3FEW_X          475 AAVVGALLDDKFADALNKEI  494 (505)
</pre><pre>T ss_dssp             HHHHHHHHCTHHHHHCCCCC
</pre><pre>T ss_pred             HHHHHHHhCHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>2I88_A Colicin-E1; protein-membrane interactions, toxin-membrane interactions, toxin structure, voltage-gated channel, colicin, MEMBRANE PROTEIN; 2.5A {Escherichia coli}
</pre><pre>Probab=86.91  E-value=48  Score=31.37  Aligned_cols=169  Identities=14%  Similarity=0.063  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHH-----------hcC
</pre><pre>Q WP_048874398.1   54 RYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHA-----------LGL  122 (262)
</pre><pre>Q Consensus        54 ~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~-----------LGl  122 (262)
</pre><pre>                      ...+++++.++-.-+-|++.        ...|=.+...+-++|-...     ++-.++.+++--+.           +..
</pre><pre>T Consensus         9 ~~~~~~k~av~~~~df~~~i--------~~k~G~~~~~~A~eLA~~~-----kGKkIR~~~dAl~afeK~~~n~~~K~s~   75 (191)
</pre><pre>T 2I88_A            9 LLNSQIKDAVDATVSFYQTL--------TEKYGEKYSKMAQELADKS-----KGKKIGNVNEALAAFEKYKDVLNKKFSK   75 (191)
</pre><pre>T ss_dssp             -----CHHHHHHHHHHHHHH--------HHHSCHHHHHHHHHHHHHH-----TTCCCCCHHHHHHHHHHHHHHHHHSSCH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHh-----CCCCCCCHHHHHHHHHHHHhhhcccCCH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhch
</pre><pre>Q WP_048874398.1  123 SSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASL  202 (262)
</pre><pre>Q Consensus       123 StkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~  202 (262)
</pre><pre>                      .-+.-+.++=+.-  +-..+++++.+++|.      .|+++..++...-..++..++.+|+                +..
</pre><pre>T Consensus        76 ~Dr~AI~~Al~s~--d~~~~A~nl~~~sK~------fg~~g~~~~~~d~~~~~~k~~~Tgn----------------W~p  131 (191)
</pre><pre>T 2I88_A           76 ADRDAIFNALASV--KYDDWAKHLDQFAKY------LKITGHVSFGYDVVSDILKIKDTGD----------------WKP  131 (191)
</pre><pre>T ss_dssp             HHHHHHHHHHTTC--CTHHHHTTHHHHHHH------TTCCSCCCTTCCHHHHHHHHHHHCC----------------CHH
</pre><pre>T ss_pred             HHHHHHHHHHHcC--CHHHHHHHHHHHHHH------hHHHhhHhhHHHHHHHHHHHhcCCC----------------cHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHH-HHHhc-ccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1  203 AGGIYGGKYGAMAGAGVC-VALGV-ATGGVGLLACSVVGAAAGGYIGGEVAGGATEYVM  259 (262)
</pre><pre>Q Consensus       203 ~Gg~~gg~~g~~~a~~vc-~~~G~-~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~G  259 (262)
</pre><pre>                      +.=-+=....+.++..+. +++++ .+..+|++..+++.+.++++.-...-+..-..+|
</pre><pre>T Consensus       132 l~le~Es~~~g~~a~~~~a~~fs~~~~t~~gi~~~a~~~a~~~a~id~~~~~~iN~~l~  190 (191)
</pre><pre>T 2I88_A          132 LFLTLEKKAADAGVSYVVALLFSLLAGTTLGIWGIAIVTGILCSYIDKNKLNTINEVLG  190 (191)
</pre><pre>T ss_dssp             HHHHHHHHHGGGTHHHHHHHHHHHHHTCCCCHHHHHHHHHHHHHHSSCCCHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>4UY3_A SEPTATION RING FORMATION REGULATOR EZRA; STRUCTURAL PROTEIN, DIVISOME, FTSZ, CELL WALL, TUBULIN, SPECTRIN, Z-RING; 2.6A {STAPHYLOCOCCUS AUREUS}
</pre><pre>Probab=85.17  E-value=46  Score=30.59  Aligned_cols=101  Identities=11%  Similarity=0.088  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHH------HHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTD------EANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYR   75 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~------ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~   75 (262)
</pre><pre>                      |..+....+.+...|..|...      +..-+.+.|.-+..-+..-+...|.+...++..+++|++-+.+.+.+..+.  
</pre><pre>T Consensus        92 l~~~e~~~~~l~~~l~~l~~~e~~~r~~~~~lk~~yr~lkk~l~~~~~~~g~~~~~le~~l~~ie~~f~~f~~l~~~g--  169 (198)
</pre><pre>T 4UY3_A           92 MDSYEQSYQQQLEDVNEIIALYKDNDELYDKCKVDYREMKRDVLANRHQFGEAASLLETEIEKFEPRLEQYEVLKADG--  169 (198)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCSGGGGGGHHHHHHHHHTHHHHHHHHHHHHHTT--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHhhhhHHHHHHHHHHHHHHHHHHHHHHHHcC--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCCh
</pre><pre>Q WP_048874398.1   76 TQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPY  113 (262)
</pre><pre>Q Consensus        76 ~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~  113 (262)
</pre><pre>                               -|.+=+.++.+|+..+..|-..--.||.+
</pre><pre>T Consensus       170 ---------dy~~A~e~l~~i~~~i~~l~~~l~~iP~~  198 (198)
</pre><pre>T 4UY3_A          170 ---------NYVQAHNHIAALNEQMKQLRSYMHGSSGN  198 (198)
</pre><pre>T ss_dssp             ---------CHHHHHHHHHHHHHHHHHHHHHTCC----
</pre><pre>T ss_pred             ---------CHHHHHHHHHHHHHHHHHHHHHHhhcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>PF15967.7 ; Nucleoporin_FG2 ; Nucleoporin FG repeated region
</pre><pre>Probab=84.21  E-value=20  Score=39.46  Aligned_cols=76  Identities=16%  Similarity=0.193  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCC-------HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELN-------TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~-------~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      ||.+++++.+++++.|..+.       +|-.+.|.+.|+.+-.+               ..++.+|.+.++++.+.|.+.
</pre><pre>T Consensus       352 rL~~YrqqIeELE~hL~s~~~~~~~TpqdL~~~L~~~~etF~aL---------------AaqLa~lHe~V~~lKeqYL~~  416 (596)
</pre><pre>T W5M8W5_LEPOC/2  352 QLQQYRQQIEELENHLTTQASSSHITPQDLSLAMQKLYHTFVAV---------------AAQLQSVHENVKILKHQYLGY  416 (596)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHcccCCCCCHHHHHHHHHHHHHHHHHH---------------HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhcCCCccHHHHHHHHH
</pre><pre>Q WP_048874398.1   74 YRTQGTLIGQQFFVERQK   91 (262)
</pre><pre>Q Consensus        74 ~~~~g~l~~~~F~~~R~~   91 (262)
</pre><pre>                      ++..-.-..+-|..+|+.
</pre><pre>T Consensus       417 rR~~~~d~~dpFe~~r~~  434 (596)
</pre><pre>T W5M8W5_LEPOC/2  417 RKAFLEDSTDVFESKRAA  434 (596)
</pre><pre>T ss_pred             HHHHhccccCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>6THK_A Pyocin S5; Bacteriocin, Pyocin, Antibiotic, Ionophore, ANTIMICROBIAL PROTEIN; 2.2A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=83.98  E-value=1.2e+02  Score=33.28  Aligned_cols=207  Identities=20%  Similarity=0.177  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCH------HHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNT------DEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~------~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      ..|-.+.+.++..|..|.+      +|+..-..-.+.+.......+....-++.-++..++.+.+-|.+=-         
</pre><pre>T Consensus       283 evL~sEI~sLq~rL~~L~A~~~rr~~eA~~~~aE~~~lkdAv~~~adf~keIsek~G~k~~~iAkELA~~i---------  353 (506)
</pre><pre>T 6THK_A          283 ELLVDEIASLQARLVKLNAETTRRRTEAERKAAEEQALQDAIKFTADFYKEVTEKFGARTSEMARQLAEGA---------  353 (506)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCHHHHHHHHHHHHHH---------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc---------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   77 QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        77 ~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                      +|+..+.  +.+=-+-|.|..+-++.    .++-.|...|.++|                 .--.+.+--+++.+.+|.+
</pre><pre>T Consensus       354 kGKKIRS--~~dAlaafeK~ksnpn~----Kls~aDr~AIanAL-----------------ksld~~~mA~nL~~fsKaF  410 (506)
</pre><pre>T 6THK_A          354 RGKNIRS--SAEAIKSFEKHKDALNK----KLSLKDRQAIAKAF-----------------DSLDKQMMAKSLEKFSKGF  410 (506)
</pre><pre>T ss_dssp             TTCBCCC--HHHHHHHHHHHHHHHHH----HSCHHHHHHHHHHH-----------------HTCCHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             CCCcCCC--HHHHHHHHHHHHhhccc----cCCHHHHHHHHHHH-----------------HHccHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHH
</pre><pre>Q WP_048874398.1  157 KMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACS  236 (262)
</pre><pre>Q Consensus       157 k~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~  236 (262)
</pre><pre>                      +   +++..+++..-..++.+++.+|+    -+--+.|.-.++-|.+.+.+-+.       +..++.+.|.   |++..+
</pre><pre>T Consensus       411 G---vagk~i~~~dv~~k~i~g~eTGN----WkPl~LelEsiaaG~aAs~l~a~-------~fs~~~~Tpl---GIig~a  473 (506)
</pre><pre>T 6THK_A          411 G---VVGKAIDAASLYQEFKISTETGD----WKPFFVKIETLAAGAAASWLVGI-------AFATATATPI---GILGFA  473 (506)
</pre><pre>T ss_dssp             T---CCCCCCCHHHHHHHHHHHHHSSC----CHHHHHHHHHHHTTCBHHHHHHH-------HHHHHHHSCC---HHHHHH
</pre><pre>T ss_pred             H---HHHHHHcHHHHHHHHHHHhccCC----cHHHHHHHHHHHHHHHHHHHHHH-------HHHHHHHHHH---HHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  237 VVGAAAGGYIGGEVAGGATEYV  258 (262)
</pre><pre>Q Consensus       237 vv~~g~g~~~gg~~G~~~G~~~  258 (262)
</pre><pre>                      ++.+.++++.=...-...-..+
</pre><pre>T Consensus       474 llmAivsAlIDd~lvekiNn~i  495 (506)
</pre><pre>T 6THK_A          474 LVMAVTGAMIDEDLLEKANNLV  495 (506)
</pre><pre>T ss_dssp             HHHHHHHHTTCHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHhhCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF04048.16 ; Sec8_exocyst ; Sec8 exocyst complex component specific domain
</pre><pre>Probab=81.61  E-value=42  Score=29.69  Aligned_cols=130  Identities=10%  Similarity=-0.017  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCHHHHHHHHHhHHHHHHH---hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHH
</pre><pre>Q WP_048874398.1   12 VNGALAELNTDEANFFNKHFSTIAAL---TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVE   88 (262)
</pre><pre>Q Consensus        12 v~~al~~L~~~ea~fl~~n~~~l~~~---l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~   88 (262)
</pre><pre>                      ++..|..+..+-...+.++|..+...   +..+|.+.+---..+.+...++.+.|+++-..|.+.        +..-..-
</pre><pre>T Consensus         2 l~~vl~~i~~~W~~l~~~dfnpl~~aL~lld~ss~g~~~~~~~f~~~~~~~d~~L~~iV~~h~~~--------Fn~si~~   73 (142)
</pre><pre>T A5E514_LODEL/2    2 LKMIHNEIKYDWPQILQSNANPIEVAVAFLDDTSVGLAHRLKEFSGFKRRTEGVLRHVVDEHYEL--------FSNSIGS   73 (142)
</pre><pre>T ss_pred             HHHHHHHHHHhHHHHccCCCChHHHHHHhccCCCCchHHHHHHHHHHHHHHHHHHHHHHHHHHHH--------HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc-ccccchHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   89 RQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG-VASIRGYSTYVDNASKA  152 (262)
</pre><pre>Q Consensus        89 R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g-~g~Ip~y~~~i~~~ak~  152 (262)
</pre><pre>                      ...+...+...=.++..-.-+|   ..-|..|+.....+..-|...- ...+=.+-+.|+.+-++
</pre><pre>T Consensus        74 y~~i~~~i~~sq~~v~~ik~~L---~~~k~~L~~~~~~L~~Lw~~s~~~~~~l~iL~~Ie~lk~v  135 (142)
</pre><pre>T A5E514_LODEL/2   74 YHMLLSSLNQSKQDSKEIKEFL---ELSKKEVSDRSSLLAELAQSSARYEEMIEILDAIKLLNNV  135 (142)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHH---HHHHHHHhCChHHHHHHHHHHHHHHHHHHHHHHHHHHhcc
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>5IJN_N Nuclear pore glycoprotein p62; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=79.80  E-value=1.6e+02  Score=31.88  Aligned_cols=140  Identities=16%  Similarity=0.158  Sum_probs=0.0  Template_Neff=2.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL------TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      .+|.++...-...-.++++ -.+.|++|.|-|-.+      ....-.-+...-.+++++=+++++.|+.+|+--++.+..
</pre><pre>T Consensus       342 ~QLeesek~F~nQA~kINe-WDRvLVqNGEKISqLysDV~kAEQTQ~KIDQ~LdYIErQQdELEslLd~YEkQAE~LLsd  420 (522)
</pre><pre>T 5IJN_N          342 LELEDQERHFLQQATQVNA-WDRTLIENGEKITSLHREVEKVKLDQKRLDQELDFILSQQKELEDLLSPLEELVKEQSGT  420 (522)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTSSTHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHhHHHHHHHHHHHHH-HHHHHHHccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHH-HH
</pre><pre>Q WP_048874398.1   77 QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAA-RF  155 (262)
</pre><pre>Q Consensus        77 ~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~-k~  155 (262)
</pre><pre>                      -.....++=+++--++=..||.+|++|..                 .-++++-+-++..  .-..-.+-+.+|.|+. .+
</pre><pre>T Consensus       421 i~lq~nDqeREkAY~LAE~LDeQLdsMSe-----------------NLsSLIEEINeVS--Ks~d~dDPL~QIVKILNsH  481 (522)
</pre><pre>T 5IJN_N          421 IYLQHADEEREKTYKLAENIDAQLKRMAQ-----------------DLKDIIEHLNTSG--APADTSDPLQQICKILNAH  481 (522)
</pre><pre>T ss_dssp             HHSSSCCCCSCHHHHHHHHHHHHHHHHHH-----------------HHHHHHHHHHHTS--CCSCTTSSHHHHHHHHHHH
</pre><pre>T ss_pred             hhcchhhHHHHHHHHHHHHHHHHHHHHHH-----------------HHHHHHHHHHhcC--CCCCCCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHhhhH
</pre><pre>Q WP_048874398.1  156 LKMGGWL  162 (262)
</pre><pre>Q Consensus       156 lk~gGyv  162 (262)
</pre><pre>                      |...-||
</pre><pre>T Consensus       482 LsSLqWI  488 (522)
</pre><pre>T 5IJN_N          482 MDSLQWI  488 (522)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>d1ciia1 f.1.1.1 (A:451-624) Colicin Ia {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=77.01  E-value=1.2e+02  Score=28.77  Aligned_cols=162  Identities=15%  Similarity=0.097  Sum_probs=0.0  Template_Neff=4.500
</pre><pre>
</pre><pre>Q ss_pred             hhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcC
</pre><pre>Q WP_048874398.1   43 KSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGL  122 (262)
</pre><pre>Q Consensus        43 ~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGl  122 (262)
</pre><pre>                      ..+.-++++.....+...+..+++.+-..+..+-+.-.+.++-.+--.+..+.++..++.-=|        +.|-++|  
</pre><pre>T Consensus         1 dav~ftadf~k~i~ek~G~k~~~~A~elA~~~kGKkIRn~~dAl~afeK~k~n~~~K~n~~Dr--------~AI~nAl--   70 (174)
</pre><pre>T d1ciia1           1 DAINFTTEFLKSVSEKYGAKAEQLAREMAGQAKGKKIRNVEEALKTYEKYRADINKKINAKDR--------AAIAAAL--   70 (174)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCTHHHHHHHHHHHTTTTCBCCCHHHHHHHHHHTHHHHTTSSCSHHH--------HHHHHHH--
</pre><pre>T ss_pred             ChHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCCCcCCCHHHHHHHHHHHHhcccccCCHHHH--------HHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhch
</pre><pre>Q WP_048874398.1  123 SSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASL  202 (262)
</pre><pre>Q Consensus       123 StkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~  202 (262)
</pre><pre>                                     .--.+.+--+++.+.+|.   .||++..++...-..++.++..+|+    =+--+.|.-.++   
</pre><pre>T Consensus        71 ---------------~s~d~~~~A~nL~~~sK~---fg~~~k~~~~~~~~~~~~kg~eTgn----W~Pl~levEs~~---  125 (174)
</pre><pre>T d1ciia1          71 ---------------ESVKLSDISSNLNRFSRG---LGYAGKFTSLADWITEFGKAVRTEN----WRPLFVKTETII---  125 (174)
</pre><pre>T ss_dssp             ---------------HTCCHHHHHHHHHHHHHH---HTCCSCBCCHHHHHHHHHHHHHSTT----CHHHHHHHTTTB---
</pre><pre>T ss_pred             ---------------HhCCHHHHHHHHHHHHHH---hhhHHHHccHHHHHHHHHHHHhcCC----cHhhHHHHHHHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  203 AGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYI  246 (262)
</pre><pre>Q Consensus       203 ~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~  246 (262)
</pre><pre>                          +|...+..++....+.+|.|.   |++..+++.+.++++.
</pre><pre>T Consensus       126 ----~g~~A~~l~a~~fs~~~~t~~---giig~~li~a~~~s~I  162 (174)
</pre><pre>T d1ciia1         126 ----AGNAATALVALVFSILTGSAL---GIIGYGLLMAVTGALI  162 (174)
</pre><pre>T ss_dssp             ----TTCBHHHHHHHHHHHHTTCCC---HHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ----HHHHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>d2csba1 a.60.2.4 (A:351-409) Topoisomerase V {Methanopyrus kandleri [TaxId: 2320]}
</pre><pre>Probab=76.52  E-value=13  Score=27.52  Aligned_cols=33  Identities=33%  Similarity=0.595  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   19 LNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus        19 L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      |+|+.++-+.+||+-+..+                     +...|.+|++.|.+
</pre><pre>T Consensus        12 lspdaadeliehfesiagi---------------------latdleeiermyee   44 (59)
</pre><pre>T d2csba1          12 LSPDAADELIEHFESIAGI---------------------LATDLEEIERMYEE   44 (59)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHSSHHHH---------------------HTSCHHHHHHHHHH
</pre><pre>T ss_pred             CChhHHHHHHHHHHHHHHH---------------------HhcCHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>1CII_A COLICIN IA; COLICIN, BACTERIOCIN, ION CHANNEL FORMATION, TRANSMEMBRANE PROTEIN; 3.0A {Escherichia coli} SCOP: h.4.3.1, f.1.1.1
</pre><pre>Probab=75.61  E-value=2.4e+02  Score=31.81  Aligned_cols=203  Identities=13%  Similarity=0.072  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-----hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-----TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-----l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      .++-+.+-+++++.......-+.++-..+..+-+..     .+.--..++.++.+....-+...+.+..+.+--.+..+.
</pre><pre>T Consensus       383 eqkkK~AedKl~kekke~~~~~~~~~~~~~~~~ee~~~~~e~~~~kdAvk~tadf~kevteK~G~k~s~iAkeLA~~~KG  462 (602)
</pre><pre>T 1CII_A          383 TNEQKHANDALNALLKEKENIRNQLSGINQKIAEEKRKQDELKATKDAINFTTEFLKSVSEKYGAKAEQLAREMAGQAKG  462 (602)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCTHHHHHHHHHHHTTTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   77 QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        77 ~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                      +.-.+..++...-.+....++        +.++-.+...|.++|                 .--.+.+--+++.+.+|.+
</pre><pre>T Consensus       463 KKIRn~dDAlkafeK~k~np~--------kK~s~aDR~AI~nAL-----------------~sld~~~~A~nL~~fsKaF  517 (602)
</pre><pre>T 1CII_A          463 KKIRNVEEALKTYEKYRADIN--------KKINAKDRAAIAAAL-----------------ESVKLSDISSNLNRFSRGL  517 (602)
</pre><pre>T ss_dssp             CBCCCHHHHHHHHHHTHHHHT--------TSSCSHHHHHHHHHH-----------------HTCCHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CcCCCHHHHHHHHHHHHhcHh--------hhcCHHHHHHHHHHH-----------------HHCcHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHH
</pre><pre>Q WP_048874398.1  157 KMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACS  236 (262)
</pre><pre>Q Consensus       157 k~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~  236 (262)
</pre><pre>                      ..   ++..+++..-..++.++..+|+    -+--+.|.-.++       +|..++..++..+.+++|.|.   |++..+
</pre><pre>T Consensus       518 G~---~gk~~~~~dl~~~~~kg~eTGn----WrPl~levEsia-------ag~~A~~lvA~~fs~~~~tpl---GIig~a  580 (602)
</pre><pre>T 1CII_A          518 GY---AGKFTSLADWITEFGKAVRTEN----WRPLFVKTETII-------AGNAATALVALVFSILTGSAL---GIIGYG  580 (602)
</pre><pre>T ss_dssp             TC---CSCBCCHHHHHHHHHHHHHSTT----CHHHHHHHTTTB-------TTCBHHHHHHHHHHHHTTCCC---HHHHHH
</pre><pre>T ss_pred             hh---HHHHhCHHHHHHHHHHHhccCC----cHHHHHHHHHHH-------HHHHHHHHHHHHHHHHHHHHH---HHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHH
</pre><pre>Q WP_048874398.1  237 VVGAAAGGYI  246 (262)
</pre><pre>Q Consensus       237 vv~~g~g~~~  246 (262)
</pre><pre>                      ++.+.++++.
</pre><pre>T Consensus       581 iimA~vsa~I  590 (602)
</pre><pre>T 1CII_A          581 LLMAVTGALI  590 (602)
</pre><pre>T ss_dssp             HHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>3ZQE_A PROTEIN PRGI, CELL INVASION PROTEIN SIPD; CELL INVASION, CELL INVASION COMPLEX, TYPE III SECRETION, T3SS, TIP COMPLEX, HOST PATHOGEN INTERACTION, BACTERIAL PATHOGENESIS; HET: GOL, DXC; 2.19A {SALMONELLA ENTERICA SUBSP. ENTERICA SEROVAR TYPHIMURIUM}
</pre><pre>Probab=75.29  E-value=38  Score=33.97  Aligned_cols=96  Identities=11%  Similarity=0.155  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHH--------HHHHHHHhHHHHHHH---------------hhhchhhhHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTD--------EANFFNKHFSTIAAL---------------TNFMDKSLGVVADAGERYFSEI   59 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~--------ea~fl~~n~~~l~~~---------------l~~~s~~~G~~s~~~~~~~~~i   59 (262)
</pre><pre>                      .......+++.+++..|..+        +.+....+|.++..+               +.....+.+........++++|
</pre><pre>T Consensus        21 ~~~~~~~~~l~~a~~~l~~~p~np~~La~~q~~~~~y~~~r~~qS~~~Kslkd~~~~iiqn~~~~~~~~~~S~~~~~~~i  100 (305)
</pre><pre>T 3ZQE_A           21 TGVDNLQTQVTEALDKLAAKPSDPALLAAYQSKLSEYNLYRNAQSNTVKVFKDIDAAIIQNFRGGSGGTTISDAEIWDMV  100 (305)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHSTTCHHHHHHHHHHHHHHHHHHHHHHTHHHHHHHHHHHHCC----------CCHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhcCCCCHHHHHHHHHHHHHHHHHHHHhHHHHHHHHHHHHHHHHhhcCCCCCCCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1   60 EKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRL  106 (262)
</pre><pre>Q Consensus        60 ~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~  106 (262)
</pre><pre>                      .+.+..|..-|...        ..+.+.+...+|+.+...+.++..+
</pre><pre>T Consensus       101 ~~~I~~i~~~yl~~--------y~~i~~~y~~fy~~~s~i~s~l~~~  139 (305)
</pre><pre>T 3ZQE_A          101 SQNISAIGDSYLGV--------YENVVAVYTDFYQAFSDILSKMGGW  139 (305)
</pre><pre>T ss_dssp             HTSCHHHHHTTHHH--------HHHHHHHHHHHHHHHHHHHTTSGGG
</pre><pre>T ss_pred             HHHHHHhchhhHHH--------HHHHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>TIGR03001 Sig-70_gmx1; RNA polymerase sigma-70 factor, Myxococcales family 1. This group of sigma factors are members of the sigma-70 family (TIGR02937) and are found in multiple copies in the order Myxococcales.
</pre><pre>Probab=74.13  E-value=1.2e+02  Score=27.69  Aligned_cols=92  Identities=20%  Similarity=0.212  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhH------HHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHF------STIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~------~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      ....+....+...+..|++.+...+.-+|      .-|...++.+...+-          +.+.+++++|.+.+.+....
</pre><pre>T Consensus       146 ~~~~~~~~~i~~~l~~L~~~~~~il~l~~~~g~s~~eIA~~lgis~~~V~----------~~~~ra~~kLr~~l~~~~~~  215 (244)
</pre><pre>T TIGR03001       146 RYRQDFRQALREALAALSERERHLLRLHFVDGLSMDRIGAMYQVHRSTVS----------RWVAQARERLLERTRRRLAE  215 (244)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHHHhcCCCHHHHHHHhCCCHHHHH----------HHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCccHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   77 QGTLIGQQFFVERQKLFSELDVLLNKLS  104 (262)
</pre><pre>Q Consensus        77 ~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~  104 (262)
</pre><pre>                      ..+...++.-.-.+.+-..||-.|++++
</pre><pre>T Consensus       216 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~  243 (244)
</pre><pre>T TIGR03001       216 RLKLSSPELESLLGLVRSRLDVSLSRLL  243 (244)
</pre><pre>T ss_pred             hhCCCcHHHHHHHHHHHHHHhhhHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>PF04124.14 ; Dor1 ; Dor1-like family
</pre><pre>Probab=73.44  E-value=1.6e+02  Score=28.85  Aligned_cols=149  Identities=11%  Similarity=0.052  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHH-HHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    2 LKQLQ-NAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         2 l~~L~-~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      +.+|+ ..-.+..+....++.+-...+++||+.+-..... ...+-..-...+..++.+.+.+.+|.....+........
</pre><pre>T Consensus         9 l~~L~~~~~~~L~~~~~~l~~~l~~Lv~~Ny~~FI~a~~~-i~~i~~~~~~l~~~~~~l~~~l~~l~~~~~~~~~~~~~~   87 (317)
</pre><pre>T Q9P3N9_NEUCS/3    9 LTALETTEPQALAQTSHGLLLSIQALSKKSHKQVIESATH-HSTLRTSLPALAASTADLRHSIPKLDHEAVRFSTNYSKS   87 (317)
</pre><pre>T ss_pred             HHHHcccchHHHHHHHHHHHHHHHHHHhhcHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHhHHHHHHHHHHHHHHhHhh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcch--------hhhhhhhhcc--ccccchHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSR--------SIVHEWSTAG--VASIRGYSTYVDNAS  150 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStk--------S~vh~w~~~g--~g~Ip~y~~~i~~~a  150 (262)
</pre><pre>                      ....=..+|+.....|+..  .-.++-+.+|  ..|+..+  ..+        .+..+|+...  ..++|.+..-..++.
</pre><pre>T Consensus        88 ~~~~~l~~r~~~~~~l~~~--~~l~~llelP--~~l~~~i--~~~~~~y~eAv~l~~~~~~l~~~~~~~~~~~~i~~~~~  161 (317)
</pre><pre>T Q9P3N9_NEUCS/3   88 SESDVLLRRKKALLLSRNV--ERLVDVLELP--NLLSTAI--STAPVNYSSALDLNGHIRRLHSLYPDSPLVTSIAAQAD  161 (317)
</pre><pre>T ss_pred             ccChHHHHHHHHHHHHhhH--HHHHHHHhHH--HHHHHHH--HcCCCCHHHHHHHHHHHHHHHHHCCCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1  151 KAARFLK  157 (262)
</pre><pre>Q Consensus       151 k~~k~lk  157 (262)
</pre><pre>                      +....++
</pre><pre>T Consensus       162 ~~~~~l~  168 (317)
</pre><pre>T Q9P3N9_NEUCS/3  162 EAMRTMA  168 (317)
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>2D4Y_A Flagellar hook-associated protein 1; Multi-domain protein, alpha-helical bundle, complex all-beta folds, STRUCTURAL PROTEIN; 2.1A {Salmonella typhimurium}
</pre><pre>Probab=71.85  E-value=1.8e+02  Score=30.60  Aligned_cols=100  Identities=17%  Similarity=0.207  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH---------------------------hhhchhhhHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL---------------------------TNFMDKSLGVVADAGE   53 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~---------------------------l~~~s~~~G~~s~~~~   53 (262)
</pre><pre>                      ..+...+...++...+.+.+..-...|.+.|.-++.+                           ...-...-..+...+.
</pre><pre>T Consensus        20 ~~~~~~~~l~~le~~~~~~~~~l~~~l~~ff~s~~~La~~P~~~~~R~~vv~~a~~l~~~~~~~~~~L~~~~~~~~~~i~   99 (463)
</pre><pre>T 2D4Y_A           20 GLTTRYEQMSKIDNLLADKSSSLSGSLQSFFTSLQTLVSNAEDPAARQALIGKAEGLVNQFKTTDQYLRDQDKQVNIAIG   99 (463)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCTTTCHHHHHHHHHHHHHHHHHTTTCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHhcCCCCChHHHHHHHHHHHHHHHhCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHh-c--CCCccHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   54 RYFSEIEKKLKAIELAYQNQYRT-Q--GTLIGQQFFVERQKLFSELDVLL  100 (262)
</pre><pre>Q Consensus        54 ~~~~~i~~~L~~ie~ly~~~~~~-~--g~l~~~~F~~~R~~Lf~~Ld~~L  100 (262)
</pre><pre>                      ..+++|+..|++|.++=++-... .  ......+...||.+|+.+|-..+
</pre><pre>T Consensus       100 ~~V~~iN~ll~~Ia~LN~~I~~~~~~~~g~~~ndL~DqRD~ll~eLS~~i  149 (463)
</pre><pre>T 2D4Y_A          100 SSVAQINNYAKQIANLNDQISRMTGVGAGASPNDLLDQRDQLVSELNKIV  149 (463)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHC--------CHHHHHHHHHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhHhccCCCCCcccHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>2PFT_A Exocytosis Protein; helix-turn-helix, ENDOCYTOSIS-EXOCYTOSIS COMPLEX; 2.25A {Mus musculus}
</pre><pre>Probab=71.64  E-value=58  Score=34.58  Aligned_cols=111  Identities=10%  Similarity=0.163  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHH-------------HHHHHH
</pre><pre>Q WP_048874398.1   26 FFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQF-------------FVERQK   91 (262)
</pre><pre>Q Consensus        26 fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F-------------~~~R~~   91 (262)
</pre><pre>                      ||++|+..+..-+. .+.-.......+....++....+++--+.|++.|.+ -..+.....             ....+.
</pre><pre>T Consensus       412 FllNN~~yi~~~~~-~~~l~~~l~~~~~~~~~~~~~~~~~~~~~Y~~sW~~~~~~L~~~~~~~~~~~~~l~~~~r~~~K~  490 (571)
</pre><pre>T 2PFT_A          412 FLHNNYNYILKSLE-KSELIQLVAVTQKTAERSYREHIEQQIQTYQRSWLKVTDYIAEKNLPVFQPGVKLRDKERQMIKE  490 (571)
</pre><pre>T ss_dssp             HHHHHHHHHHTTTT-TSTHHHHHHTTCTTHHHHHHHHHHHHHHHHHGGGHHHHTTTSGGGSCSSCCCSCCCHHHHHHHHH
</pre><pre>T ss_pred             HHHhhHHHHHHhcc-cccHHHHHhcccHhHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCCCCCCCCCCcCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHH--hhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHH
</pre><pre>Q WP_048874398.1   92 LFSELDVLLNKLSR--LTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDN  148 (262)
</pre><pre>Q Consensus        92 Lf~~Ld~~L~~~~~--~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~  148 (262)
</pre><pre>                      -|++.+..+..+.+  +...+|+ +.||+.|   .+.+...       -+|.|...+++
</pre><pre>T Consensus       491 ~f~~Fn~~fee~~~~q~~~~v~D-~~LR~~L---r~~i~~~-------v~p~Y~~f~~~  538 (571)
</pre><pre>T 2PFT_A          491 RFKGFNDGLEELCKIQKVWAIPD-TEQRDKI---RQAQKDI-------VKETYGAFLHR  538 (571)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHTTCCCCC-HHHHHHH---HHHHHHH-------HHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhcceecCC-HHHHHHH---HHHHHHH-------HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>5IJN_G NUCLEAR PORE COMPLEX PROTEIN NUP58; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=71.06  E-value=3e+02  Score=30.90  Aligned_cols=153  Identities=10%  Similarity=0.142  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      +|++-..+-++..+.|.+.+.+.-.-+.++.+.|...+..-+..+-.-...++.+++.|++.|+..|.+-+...   .+.
</pre><pre>T Consensus       257 ELEErIrEyRDeSqrLDQ~DSvSL~~fEeDAe~LvQEL~gLstsLERQkesLqELmekVKqmLRNTE~AVRsye---~Pp  333 (599)
</pre><pre>T 5IJN_G          257 NLQKFVKEQKQVQEEISRMSSKAMLKVQEDIKALKQLLSLAANGIQRNTLNIDKLKIETAQELKNAEIALRTQK---TPP  333 (599)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHCCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTTTGGGSS---HHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhcCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc---CCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHH-HHHhhcCCCChHHHHHHhc----CcchhhhhhhhhccccccchHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNK-LSRLTLKLKPYEEIKHALG----LSSRSIVHEWSTAGVASIRGYSTYVDNASKAARF  155 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~-~~~~~~~l~~~~~lK~~LG----lStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~  155 (262)
</pre><pre>                      ....|+..-...|.++...+++ +-...-.|.+-+++-+...    -+..+|...-+..- .-+-.++.+++++-.-++.
</pre><pre>T Consensus       334 GvQ~fYSaPspFFqqTVarFEkrLeEYrQ~IEEIEQlLnS~~nss~qSPQsLpKVMSNQH-D~FVhVAAKVEsLHQ~VEs  412 (599)
</pre><pre>T 5IJN_G          334 GLQHEYAAPADYFRILVQQFEVQLQQYRQQIEELENHLATQANNSHITPQDLSMAMQKIY-QTFVALAAQLQSIHENVKV  412 (599)
</pre><pre>T ss_dssp             HSCCCCSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCCCCSHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ccccccccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCCCHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HH
</pre><pre>Q WP_048874398.1  156 LK  157 (262)
</pre><pre>Q Consensus       156 lk  157 (262)
</pre><pre>                      ||
</pre><pre>T Consensus       413 MK  414 (599)
</pre><pre>T 5IJN_G          413 LK  414 (599)
</pre><pre>T ss_dssp             HH
</pre><pre>T ss_pred             HH
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>5IJO_M Nucleoporin p58/p45; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=71.06  E-value=3e+02  Score=30.90  Aligned_cols=153  Identities=10%  Similarity=0.142  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      +|++-..+-++..+.|.+.+.+.-.-+.++.+.|...+..-+..+-.-...++.+++.|++.|+..|.+-+...   .+.
</pre><pre>T Consensus       257 ELEErIrEyRDeSqrLDQ~DSvSL~~fEeDAe~LvQEL~gLstsLERQkesLqELmekVKqmLRNTE~AVRsye---~Pp  333 (599)
</pre><pre>T 5IJO_M          257 NLQKFVKEQKQVQEEISRMSSKAMLKVQEDIKALKQLLSLAANGIQRNTLNIDKLKIETAQELKNAEIALRTQK---TPP  333 (599)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHCCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTTTGGGSS---HHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhcCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc---CCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHH-HHHhhcCCCChHHHHHHhc----CcchhhhhhhhhccccccchHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNK-LSRLTLKLKPYEEIKHALG----LSSRSIVHEWSTAGVASIRGYSTYVDNASKAARF  155 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~-~~~~~~~l~~~~~lK~~LG----lStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~  155 (262)
</pre><pre>                      ....|+..-...|.++...+++ +-...-.|.+-+++-+...    -+..+|...-+..- .-+-.++.+++++-.-++.
</pre><pre>T Consensus       334 GvQ~fYSaPspFFqqTVarFEkrLeEYrQ~IEEIEQlLnS~~nss~qSPQsLpKVMSNQH-D~FVhVAAKVEsLHQ~VEs  412 (599)
</pre><pre>T 5IJO_M          334 GLQHEYAAPADYFRILVQQFEVQLQQYRQQIEELENHLATQANNSHITPQDLSMAMQKIY-QTFVALAAQLQSIHENVKV  412 (599)
</pre><pre>T ss_dssp             HSCCCCSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCCCCSHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ccccccccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCCCHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HH
</pre><pre>Q WP_048874398.1  156 LK  157 (262)
</pre><pre>Q Consensus       156 lk  157 (262)
</pre><pre>                      ||
</pre><pre>T Consensus       413 MK  414 (599)
</pre><pre>T 5IJO_M          413 LK  414 (599)
</pre><pre>T ss_dssp             HH
</pre><pre>T ss_pred             HH
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>2B1E_A Exocyst complex component EXO70; tethering complex, exocyst, ENDOCYTOSIS-EXOCYTOSIS COMPLEX; 2.0A {Saccharomyces cerevisiae} SCOP: a.118.17.2
</pre><pre>Probab=69.52  E-value=80  Score=33.68  Aligned_cols=118  Identities=14%  Similarity=0.182  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHH----------HHHHHHHHH
</pre><pre>Q WP_048874398.1   26 FFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQF----------FVERQKLFS   94 (262)
</pre><pre>Q Consensus        26 fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F----------~~~R~~Lf~   94 (262)
</pre><pre>                      ||.+|+..|...+..+    .......+...+++.+..+.....|...|.. -..+....+          ...-+..|+
</pre><pre>T Consensus       416 FllNN~~yI~~~~~~s----~l~~~lg~~~~~~~~~~~~~~~~y~~~~W~~~~~~L~~~~~~~~~~~~~~~r~~iK~~f~  491 (564)
</pre><pre>T 2B1E_A          416 FILMNLTLVEQIVEKS----ELNLMLAGEGHSRLERLKKRYISYMVSDWRDLTANLMDSVFIDSSGKKSKDKEQIKEKFR  491 (564)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTS----THHHHTTTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTCSSSCCCCHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHhcc----hHHHHhccchHHHHHHHHHHHHHHHHHhhHHHHHHHhhccccCCCCCChhcHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHH--hhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHH----HHHHHH
</pre><pre>Q WP_048874398.1   95 ELDVLLNKLSR--LTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKA----ARFLKM  158 (262)
</pre><pre>Q Consensus        95 ~Ld~~L~~~~~--~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~----~k~lk~  158 (262)
</pre><pre>                      +.+..+..+.+  +...+|+ +.||+.|   .+.+...       -+|.|..-+++-...    .|++|+
</pre><pre>T Consensus       492 ~Fn~~f~e~~~~~~~~~v~D-~~Lr~~l---~~~i~~~-------v~p~Y~~f~~~~~~~~k~~~kyiky  550 (564)
</pre><pre>T 2B1E_A          492 KFNEGFEDLVSKTKQYKLSD-PSLKVTL---KSEIISL-------VMPMYERFYSRYKDSFKNPRKHIKY  550 (564)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTTCCCCC-HHHHHHH---HHHHHHH-------HHHHHHHHHHHHGGGSSSGGGTCSC
</pre><pre>T ss_pred             HHHHHHHHHHHHHcccCCCC-HHHHHHH---HHHHHHH-------HHHHHHHHHHHhhhcccCccccccc
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>6YS8_C GldL; membrane protein complex ion-driven motor, MOTOR PROTEIN; 3.9A {Flavobacterium johnsoniae}
</pre><pre>Probab=68.61  E-value=1e+02  Score=29.83  Aligned_cols=92  Identities=9%  Similarity=0.108  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGT   79 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~   79 (262)
</pre><pre>                      ++..+..+|+.-.+.++.| .+..+-+.++-.-+..+.+.+.     .+......++.+++.|..|+..|..+++. +..
</pre><pre>T Consensus        99 ~l~~~~~~a~i~~~~~~~l-~~g~~~l~~~~~~l~~~s~~~~-----~~~~y~~q~~~~~~nl~~lN~iYe~ql~~~~~~  172 (215)
</pre><pre>T 6YS8_C           99 KLDAMLKEAKVDGELMASL-GNSIKNFEGAAKAISPTVDSIA-----GQKKYAEEMSMAAAQMESLNSLYKVQLESASRN  172 (215)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHcccCHHHHHHH-HHHHHHHHHHHHhhHHHHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             Ccc--------HHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIG--------QQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        80 l~~--------~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      +..        +.|.+|=++|=++|..
</pre><pre>T Consensus       173 ~~~~~~~~~~~~~~~~e~~~L~~nl~~  199 (215)
</pre><pre>T 6YS8_C          173 AQANSEIAENAAKLKEQMASMTANIAS  199 (215)
</pre><pre>T ss_dssp             ---------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>6YS8_G GldL; membrane protein complex ion-driven motor, MOTOR PROTEIN; 3.9A {Flavobacterium johnsoniae}
</pre><pre>Probab=68.61  E-value=1e+02  Score=29.83  Aligned_cols=92  Identities=9%  Similarity=0.108  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGT   79 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~   79 (262)
</pre><pre>                      ++..+..+|+.-.+.++.| .+..+-+.++-.-+..+.+.+.     .+......++.+++.|..|+..|..+++. +..
</pre><pre>T Consensus        99 ~l~~~~~~a~i~~~~~~~l-~~g~~~l~~~~~~l~~~s~~~~-----~~~~y~~q~~~~~~nl~~lN~iYe~ql~~~~~~  172 (215)
</pre><pre>T 6YS8_G           99 KLDAMLKEAKVDGELMASL-GNSIKNFEGAAKAISPTVDSIA-----GQKKYAEEMSMAAAQMESLNSLYKVQLESASRN  172 (215)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHcccCHHHHHHH-HHHHHHHHHHHHhhHHHHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             Ccc--------HHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIG--------QQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        80 l~~--------~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      +..        +.|.+|=++|=++|..
</pre><pre>T Consensus       173 ~~~~~~~~~~~~~~~~e~~~L~~nl~~  199 (215)
</pre><pre>T 6YS8_G          173 AQANSEIAENAAKLKEQMASMTANIAS  199 (215)
</pre><pre>T ss_dssp             ---------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>TIGR03513 GldL_gliding; gliding motility-associated protein GldL. This protein family, GldL, is named for the member from Flavobacterium johnsoniae, which is required for a type of rapid gliding motility found in certain members of the Bacteriodetes.
</pre><pre>Probab=66.65  E-value=1.4e+02  Score=28.77  Aligned_cols=82  Identities=9%  Similarity=0.101  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             HcCCHHHHHHHHHhHHHHHHHhhhchhhhH--HHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCcc--------HHH
</pre><pre>Q WP_048874398.1   17 AELNTDEANFFNKHFSTIAALTNFMDKSLG--VVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIG--------QQF   85 (262)
</pre><pre>Q Consensus        17 ~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G--~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~--------~~F   85 (262)
</pre><pre>                      .+++++..+-|-+...-+......-+....  ..+......++.+++.|..|+..|+.+++. +.+++.        +.|
</pre><pre>T Consensus        98 a~i~~~~~~~L~~gi~~l~~a~~~l~~~~~sy~~t~~Y~~ql~~l~knL~~LN~iYe~QLk~~~~q~~~~~~~~~ds~~f  177 (202)
</pre><pre>T TIGR03513        98 AKKDATLMQSLGNGINPFEGAAKTLAPMTDSYAQQQKYIEQMSSLAANMEGLNTIYEAQLKGASSHADANNEIAINSSSL  177 (202)
</pre><pre>T ss_pred             cCCCHHHHHHHHhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   86 FVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        86 ~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      .+|-.+|=++|..
</pre><pre>T Consensus       178 kee~~kLa~nL~~  190 (202)
</pre><pre>T TIGR03513       178 KEEMEKMAANLTS  190 (202)
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>PF10475.11 ; Vps54_N ; Vacuolar-sorting protein 54, of GARP complex
</pre><pre>Probab=65.49  E-value=33  Score=32.22  Aligned_cols=131  Identities=11%  Similarity=0.136  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   26 FFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSR  105 (262)
</pre><pre>Q Consensus        26 fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~  105 (262)
</pre><pre>                      |+..+||..++.+..-....+...+.++.++....+.|..-.+.....+...-.-++..|+.. -.-+.+|+..+.....
</pre><pre>T Consensus         8 f~~~~fd~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~L~~~l~~v~~~L~~~v~~~~~~f~~~-~~~~~~l~~~l~~~~~   86 (299)
</pre><pre>T Q54MU4_DICDI/7    8 FFDPKFDTTQFILDLLPPNVNNSSSEFSIFLDSNVKQFTRAMDYVNSKLYNRVRENYTEFVQG-MSQIHEIGVELQRSTV   86 (299)
</pre><pre>T ss_pred             ccCCCCChHHHHHhhCCCCCCCCcchhHHHHHHHHHHHHHHHHHHHHHHHHHHHHcHHHHHHH-HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhcCCCCh-HHHHHHhcCcchhhhhhhhhcc-ccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  106 LTLKLKPY-EEIKHALGLSSRSIVHEWSTAG-VASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus       106 ~~~~l~~~-~~lK~~LGlStkS~vh~w~~~g-~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                      .-.++... ..+|+.+.-+...+++.|.+.- ...+...-+.+..+......++
</pre><pre>T Consensus        87 ~~~~lr~~l~~~~~~l~~~~~~i~~~~~~~~~l~~~~~~L~~i~~~~~~~~~l~  140 (299)
</pre><pre>T Q54MU4_DICDI/7   87 MCSNGRRTLSQTKKNLTQTAFIIMSKYTKRNLLTKMYMDLSQIREIVHLERLLK  140 (299)
</pre><pre>T ss_pred             HHHHHHHHHHHHHhhhcHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>5J5L_A Uncharacterized protein; VIRULENCE FACTOR, ARACHIDONIC ACID, LIPID BINDING PROTEIN; HET: ACD; 1.7A {Neosartorya fumigata (strain ATCC MYA-4609 / Af293 / CBS 101355 / FGSC A1100)}
</pre><pre>Probab=65.11  E-value=1.6e+02  Score=25.56  Aligned_cols=100  Identities=6%  Similarity=-0.045  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERY--FSEIEKKLKAIELAYQNQYRTQGT   79 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~--~~~i~~~L~~ie~ly~~~~~~~g~   79 (262)
</pre><pre>                      .+.+++....++. ..+|+++|...+...+.-+.......-..+-.--..+.+.  -..|...|..+..........-..
</pre><pre>T Consensus        45 ~~~~~~~~~~~~~-~~~ls~~ds~~i~~~~~~l~~~~~~~l~~l~~kk~~f~~~~~~~~v~~~L~~l~~~~~~l~~~l~~  123 (160)
</pre><pre>T 5J5L_A           45 ESDIDQATADAKA-TSALSAADSTSVTNALLGLKPDIVTSLDAIVAKKPQVDSAGVGSLVLSDLNALQSKTDALSGALQD  123 (160)
</pre><pre>T ss_dssp             HHHHHHHHHHHHT-CCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHTHHHHHHHTCHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHh-CCCCCHHHHHHHHHHHHccHHHHHHHHHHHHHcCHHHHhcCcHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        80 l~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      .-+++....=+.+..++|..+.+
</pre><pre>T Consensus       124 k~p~~~~~~~~~~~~~i~~af~~  146 (160)
</pre><pre>T 5J5L_A          124 IATATDKDTIASGTQDIDAAFSS  146 (160)
</pre><pre>T ss_dssp             HSCHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HCCHhhHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>d2b1ea_ a.118.17.2 (A:) automated matches {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=64.91  E-value=2.4e+02  Score=30.54  Aligned_cols=142  Identities=12%  Similarity=0.148  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHH------cCC----------HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALA------ELN----------TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKA   65 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~------~L~----------~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~   65 (262)
</pre><pre>                      |..|...-+.-++...      +-+          +-..=||.+|+..|...+...    ....-..+...+.+++..++
</pre><pre>T Consensus       369 i~~L~~~Le~ks~~y~~~~~~~d~~~~~~~~~~~~~l~~iFllNN~~yI~~~l~~~----~l~~~l~~~~~~~~~~l~~~  444 (557)
</pre><pre>T d2b1ea_         369 IDTLAVNLERKAQIALMPNQEPDVANPNSSKNKHKQRIGFFILMNLTLVEQIVEKS----ELNLMLAGEGHSRLERLKKR  444 (557)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCTTSCCCCSCTTCSSCTTHHHHHHHHHHHHHHHHHHHHTS----THHHHTTTHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhhCCCCCCCCCCCCCcccccHHhHHHHHHHHHHHHHHHHhcC----hHHHHhcCcchHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHh-cCCCccHHH----------HHHHHHHHHHHHHHHHHHHH--hhcCCCChHHHHHHhcCcchhhhhhhh
</pre><pre>Q WP_048874398.1   66 IELAYQNQYRT-QGTLIGQQF----------FVERQKLFSELDVLLNKLSR--LTLKLKPYEEIKHALGLSSRSIVHEWS  132 (262)
</pre><pre>Q Consensus        66 ie~ly~~~~~~-~g~l~~~~F----------~~~R~~Lf~~Ld~~L~~~~~--~~~~l~~~~~lK~~LGlStkS~vh~w~  132 (262)
</pre><pre>                      ....|+..|.+ -..+.....          ...-+..|++.+..+..+.+  +...+|+ +.||+.|   .+.+...  
</pre><pre>T Consensus       445 ~~~y~~~~W~~~~~~L~~~~~~~~~~~~~~~r~~iK~~f~~Fn~~fee~~~~q~~~~v~D-~~LR~~l---~~~i~~~--  518 (557)
</pre><pre>T d2b1ea_         445 YISYMVSDWRDLTANLMDSVFIDSSGKKSKDKEQIKEKFRKFNEGFEDLVSKTKQYKLSD-PSLKVTL---KSEIISL--  518 (557)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHTCSSSCCCCHHHHHHHHHHHHHHHHHHHHHHTTCCCCC-HHHHHHH---HHHHHHH--
</pre><pre>T ss_pred             HHHHHHHhHHHHHHHHhccccccCCCCCcccHHHHHHHHHHHHHHHHHHHHHHccccCCC-HHHHHHH---HHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hccccccchHHHHHHHHHH----HHHHHHH
</pre><pre>Q WP_048874398.1  133 TAGVASIRGYSTYVDNASK----AARFLKM  158 (262)
</pre><pre>Q Consensus       133 ~~g~g~Ip~y~~~i~~~ak----~~k~lk~  158 (262)
</pre><pre>                           -+|.|..-+++...    ..|++|+
</pre><pre>T Consensus       519 -----v~p~Y~~f~~k~~~~~k~~~Kyiky  543 (557)
</pre><pre>T d2b1ea_         519 -----VMPMYERFYSRYKDSFKNPRKHIKY  543 (557)
</pre><pre>T ss_dssp             -----HHHHHHHHHHHHGGGSSSGGGTCSC
</pre><pre>T ss_pred             -----HHHHHHHHHHHhhcccCCccccccC
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>4YJW_A DUF3829 family protein; All alpha protein, PF12889 family, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative; HET: MSE; 1.8A {Bacteroides vulgatus (strain ATCC 8482 / DSM 1447 / NCTC 11154)}
</pre><pre>Probab=64.23  E-value=1.4e+02  Score=27.70  Aligned_cols=67  Identities=9%  Similarity=0.026  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHH-----H-HHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHH
</pre><pre>Q WP_048874398.1   19 LNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGE-----R-YFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKL   92 (262)
</pre><pre>Q Consensus        19 L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~-----~-~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~L   92 (262)
</pre><pre>                      |+++++++|.++++.+...+..--...-...++..     . --......+++|+.+++.            |+..|..|
</pre><pre>T Consensus        77 f~~e~~~~~k~~v~~~~~s~~~~~~~y~~L~~Y~kaEDyKDDkgaKg~~L~~~i~~~~~~------------~~~~~~~i  144 (169)
</pre><pre>T 4YJW_A           77 FSENDRQNLKENYGRLFRSISAFYENYKTYRLYMQDQSYKKDNNALADKIRKEELLLSIA------------LSEYKQVI  144 (169)
</pre><pre>T ss_dssp             SCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTHHHHTTTHHHHHHHHHHHHHHHH------------HHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhhhhcCccHHHHHHHHHHHHHHHH------------HHHhHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHH
</pre><pre>Q WP_048874398.1   93 FSELD   97 (262)
</pre><pre>Q Consensus        93 f~~Ld   97 (262)
</pre><pre>                      |.+|+
</pre><pre>T Consensus       145 ~~~l~  149 (169)
</pre><pre>T 4YJW_A          145 FDILT  149 (169)
</pre><pre>T ss_dssp             HHHHH
</pre><pre>T ss_pred             HHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF06511.13 ; T3SS_TC ; Type III secretion systems tip complex components
</pre><pre>Probab=63.12  E-value=1.1e+02  Score=31.71  Aligned_cols=110  Identities=13%  Similarity=0.152  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      .+|...-+........|++++.+.+.......... ..............-...+++|.+.+..|..-|...        
</pre><pre>T Consensus        77 ~~~~~~l~~l~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~S~~e~~~~i~~~I~~ik~~YL~v--------  148 (337)
</pre><pre>T SIPD_SALTY/5-3   77 QQLTSSLNALAKSGVSLSAEQNENLRSAFSAPTSALFSASPMAQPRTTISDAEIWDMVSQNISAIGDSYLGV--------  148 (337)
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCCCCHHHHHHHHHHhhcChhHhcCCCCCCCCCCCCCHHHHHHHHHHHHHHHhhhhHHH--------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHhhcCCCCh-------HHHHHHh
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELDVLLNKLSRLTLKLKPY-------EEIKHAL  120 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~-------~~lK~~L  120 (262)
</pre><pre>                      ..+...+....|+.+...+.+|..+--.-.+-       ..|++.|
</pre><pre>T Consensus       149 Ye~iv~~Ytdfy~~fs~i~skl~~~I~~~~DGn~V~~~~~~l~~aL  194 (337)
</pre><pre>T SIPD_SALTY/5-3  149 YENVVAVYTDFYQAFSDILSKMGGWLLPGKDGNTVKLDVTSLKNDL  194 (337)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHhhhcccCCCCCeEEEcHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>d3favb_ a.25.3.1 (B:) ESAT-6, EsxA {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=62.62  E-value=1.1e+02  Score=22.65  Aligned_cols=69  Identities=10%  Similarity=0.062  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    4 QLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         4 ~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      +|...+..++.....+...-.+.-.+--.+....-+.++...-..-.-|.+.++++.+.|.++.+.-.+
</pre><pre>T Consensus         1 ~L~~~a~~~~~~~~~i~~~~~~l~~~~~~l~~~W~G~a~~af~~~~~~~~~~~~~~~~~L~~~~~~l~~   69 (72)
</pre><pre>T d3favb_           1 GIEAAASAIQGNVTSIHSLLDEGKQSLTKLAAAWGGSGSEAYQGVQQKWDATATELNNALQNLARTISE   69 (72)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHTGGGGTCTTCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF19658.1 ; DUF6161 ; Family of unknown function (DUF6161)
</pre><pre>Probab=61.39  E-value=2.7e+02  Score=26.77  Aligned_cols=91  Identities=13%  Similarity=0.211  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      ...|.+.-+++.+.+.++..+..++..+--++.+..-..-+........-++...++.++.+++++++|.+.+.-..   
</pre><pre>T Consensus        10 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~l~~l~~~y~e~l~L~a---   86 (223)
</pre><pre>T A0A3A9UHJ4_9FL   10 QKQIVKSENELIEHLTNANKEYENYAGKIDELKTNKETAFDEWYSKIQNKFSTFDSDSKKKILNLENTYTEKLKLEK---   86 (223)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhcchHHHHHHHHHHHHHHHHhccHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSE   95 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~   95 (262)
</pre><pre>                      +..++..|+.-.++
</pre><pre>T Consensus        87 p~~yW~~ra~~h~~  100 (223)
</pre><pre>T A0A3A9UHJ4_9FL   87 PARYWSKKSSKYYK  100 (223)
</pre><pre>T ss_pred             HHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>PF05120.14 ; GvpG ; Gas vesicle protein G
</pre><pre>Probab=61.07  E-value=65  Score=25.37  Aligned_cols=38  Identities=16%  Similarity=0.404  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVL   99 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~   99 (262)
</pre><pre>                      ..+.+.|.+++..|..     |.+..++|.++++.|+.+|+..
</pre><pre>T Consensus        31 ~~i~~~L~~L~~~ye~-----GeIseeEy~~~e~~Ll~rl~~~   68 (72)
</pre><pre>T Q2JJS5_SYNJB/2   31 ENLQKELTALQIQLDL-----GEIDEETYARREEEILLALEAL   68 (72)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHc-----CCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>3NZZ_A Cell invasion protein sipD; needle tip protein, PrgI, SipB, SipC, CELL INVASION; 1.65A {Salmonella enterica}
</pre><pre>Probab=60.52  E-value=65  Score=32.79  Aligned_cols=107  Identities=12%  Similarity=0.137  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHH-HhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFN-KHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~-~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      .+|....+........|++++.+.+. +.-+.+.................-...+++|.+.+..|..-|..-        
</pre><pre>T Consensus        46 ~~~~~~l~~l~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~S~~e~~~~i~~~I~~i~~~YL~~--------  117 (308)
</pre><pre>T 3NZZ_A           46 QQLTSSLNALAKSGVSLSAEQNENLRSAFSAPTSALFSASPMAQPRTTISDAEIWDMVSQNISAIGDSYLGV--------  117 (308)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTCCCCHHHHHHHHHHTCC---------------------HHHHHHHHHHHHHCCCCCHH--------
</pre><pre>T ss_pred             HHHHHHHHHHHhcCCCCCHHHHHHHHHHhhhhhhhHhccCcCCCCCCCCCHHHHHHHHHHHHHHHchhhHHH--------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIK  117 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK  117 (262)
</pre><pre>                      ..+...+....|+.+...+.++..+-..-.+-+++|
</pre><pre>T Consensus       118 Ye~iv~~Yt~fy~~fs~ilskm~~~I~~~~DG~~V~  153 (308)
</pre><pre>T 3NZZ_A          118 YENVVAVYTDFYQAFSDILSKMGGWLLPGKDGNTVK  153 (308)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHTTGGGGEEECSSTTEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhcccCCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>PF13949.8 ; ALIX_LYPXL_bnd ; ALIX V-shaped domain binding to HIV
</pre><pre>Probab=60.06  E-value=2.8e+02  Score=26.64  Aligned_cols=118  Identities=13%  Similarity=0.093  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHH--cCCHHHHHHHHHhHHH-HHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALA--ELNTDEANFFNKHFST-IAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~--~L~~~ea~fl~~n~~~-l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      ++..|+.+-+..-+.|.  -.+++-...+..++.- ++.++..--.-.......+...++.-.+.+.+|...+.+.....
</pre><pre>T Consensus       143 ~l~~l~~~R~~~l~~l~~~~~~ddI~~~L~~~~~~~~~~~f~~eL~k~~~~~~~i~~~~~~Q~~ll~~i~~~~~~f~~~~  222 (284)
</pre><pre>T T1L3V4_TETUR/4  143 KVQEMKNQRASLEKQLREEVQSDDVLKHVIAHSKDELETIFEKELAKYDKTVSLLEQNLAAQENILNALTKASASYGKSR  222 (284)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhcchHHHHHHcCHHHHHHHHHHHHHhcHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCccHHHHHHHHHHHHHHHHHHHHHHH----hhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   78 GTLIGQQFFVERQKLFSELDVLLNKLSR----LTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        78 g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~----~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                      ......  ..+|...+++|+.....|..    -.-++.-|.+|...+
</pre><pre>T Consensus       223 ~~~~~~--~~~r~~~~~~l~~a~~~~~~l~~~l~eG~~FY~~L~~~l  267 (284)
</pre><pre>T T1L3V4_TETUR/4  223 MALNEV--NRLRVNRIQTLLTSFDGFMKVKDNIEKGLEFYKEFLSIT  267 (284)
</pre><pre>T ss_pred             HhHHHH--HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>6DLC_A Designed protein DHD1:234_A; Computational Design, Heterodimer, Coiled-coil, DE NOVO PROTEIN; 3.261A {synthetic construct}
</pre><pre>Probab=59.84  E-value=61  Score=27.20  Aligned_cols=45  Identities=24%  Similarity=0.389  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhh
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSI  127 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~  127 (262)
</pre><pre>                      ..|...-++-+.+||..+.++-.--..-|+....|+++.||.+|+
</pre><pre>T Consensus        11 ktflenlrrhldrldkhikqlrdilsenpedervkdaidlsersv   55 (115)
</pre><pre>T 6DLC_A           11 KTFLENLRRHLDRLDKHIKQLRDILSENPEDERVKDAIDLSERSV   55 (115)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHCTTCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhhCCCchHHHHHhhhhHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>6J6F_A Envelope glycoprotein; T.marenffei Mp1p, LIPID BINDING PROTEIN; 4.2A {Talaromyces marneffei PM1}
</pre><pre>Probab=59.75  E-value=3e+02  Score=26.87  Aligned_cols=164  Identities=13%  Similarity=0.054  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGER--YFSEIEKKLKAIELAYQNQYRTQGT   79 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~--~~~~i~~~L~~ie~ly~~~~~~~g~   79 (262)
</pre><pre>                      +..+++....++. ..+|+++|+..+...+.-|.......-..+-.--..+.+  .-..|...|..|..........--.
</pre><pre>T Consensus        44 ~~~i~~~~~~~~~-~~~ls~~ea~~l~~~~~~l~~~~~~~l~~l~~Kk~~~~~~~~~~~v~~~L~~l~~~~~~l~~al~~  122 (313)
</pre><pre>T 6J6F_A           44 IKATENGLQELGP-QPPLSLTEALALVGPVQGVNKLIMKTVDHLIEKKGPLVGGGYGPQVKASLQKQAHAAVTLSELVSS  122 (313)
</pre><pre>T ss_dssp             HHHHHHHHHHHTT-CCCCCHHHHHHTHHHHHHHHHHHHHHHHHHHHTHHHHHHHTCHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhcC-CCCCCHHHHHHhHHHHHHHHHHHHHHHHHHHHhHHHHHhCCchHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CccHHHHHHHHHHHHHHHHHHHH-HHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIGQQFFVERQKLFSELDVLLNK-LSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKM  158 (262)
</pre><pre>Q Consensus        80 l~~~~F~~~R~~Lf~~Ld~~L~~-~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~  158 (262)
</pre><pre>                      .-++++...-+.+++++|..|.+ +..+...-......++.    ..++++.        |..+...+..+....+....
</pre><pre>T Consensus       123 k~p~~~~~~a~~~~~~i~~~~~~~i~~~~~~~~~~~~~~~~----~~~v~~~--------i~~i~~~~~~l~~~v~~~~g  190 (313)
</pre><pre>T 6J6F_A          123 KVPSPLAPISKQLSDQVAQALQKGIEAFSISARQATKVKRD----ISAFQKV--------IQDISLAVNKFNVDIERYVG  190 (313)
</pre><pre>T ss_dssp             HSCTTSHHHHHHHHHHHHHHHHHHHHHTTTCSSCCCCCCCC----HHHHHHH--------HHHHHHHHHHHHHHHHHCCS
</pre><pre>T ss_pred             cCCcchHHHHHHHHHHHHHHHHHHHHHhhhchhhhhhhhcc----hHHHHHH--------HHHHHHHHHHHHHHHHhhcC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhHHHHhHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  159 GGWLAIGFAGANTTNEVYDA  178 (262)
</pre><pre>Q Consensus       159 gGyvgi~l~~~~s~~~I~eA  178 (262)
</pre><pre>                      +-.-.|.-....-...|.++
</pre><pre>T Consensus       191 ~~~~~i~~~~~~l~~~i~~~  210 (313)
</pre><pre>T 6J6F_A          191 GDASHLLADGNVLIKATLDG  210 (313)
</pre><pre>T ss_dssp             SCCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>4YUU_q1 Extrinsic protein in photosystem II; TRANSMEMBRANE ALPHA-HELIX, ELECTRON TRANSPORT, PHOTOSYNTHESIS, PHOTOSYSTEM II, WATER-SPLITTING, OXYGEN-EVOLVING, IRON BINDING, CALCIUM BINDING, MANGANESE BINDING, CHLORIDE; HET: CLA, LMG, PHO, PL9, LHG, LMT, OEX, HEM, BCT, GOL, SQD, DGD, UNL, BCR; 2.77000368116A {Cyanidium caldarium}
</pre><pre>Probab=58.78  E-value=3.2e+02  Score=26.77  Aligned_cols=103  Identities=11%  Similarity=0.095  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHH---HH--HHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGER---YF--SEIEKKLKAIELAYQNQYR   75 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~---~~--~~i~~~L~~ie~ly~~~~~   75 (262)
</pre><pre>                      +++++++.+..+.+.-+.|.+-+.-.-.++|+.+..++....+.+...-..+.+   .+  .+-++.+.++.+-..+.+.
</pre><pre>T Consensus       105 ~~~~i~~y~~rI~~~~~RL~el~~lI~~k~W~~V~~~Lrgplg~LR~dm~~l~~~~~~l~~p~~kk~a~~lak~lf~~Le  184 (218)
</pre><pre>T 4YUU_q1         105 TPELLNEYRASINGAKKRIEDAGEAIAKKSWEDVRSALRLAVGTLRSVCSKVNSYAMAQATKQEKQNIEKAYREFLKRIE  184 (218)
</pre><pre>T ss_dssp             ----CHHHHHHHHHHHHHHHHHHHHHHTTCHHHHHHHHHHHHHHHHHHHHHHHHHC----CHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHhccHHHHHHHHHhhHHHHHHHHHHHHHHHHHcCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcC----CCccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   76 TQG----TLIGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        76 ~~g----~l~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      .-.    ..+.+.....+......||.-|..+
</pre><pre>T Consensus       185 ~LD~AAr~kd~~~A~~~y~~a~~~ld~fl~~i  216 (218)
</pre><pre>T 4YUU_q1         185 DMDFAARMKDQDRAERLRTASISALDNWMSVV  216 (218)
</pre><pre>T ss_dssp             HHHHHHHTTCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HhHHHHHcCCHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>TIGR03042 PS_II_psbQ_bact; photosystem II protein PsbQ. This protein through the member sll1638 from Synechocystis sp.
</pre><pre>Probab=58.11  E-value=2.4e+02  Score=25.15  Aligned_cols=103  Identities=15%  Similarity=0.025  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHH-----HHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFS-----EIEKKLKAIELAYQNQYR   75 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~-----~i~~~L~~ie~ly~~~~~   75 (262)
</pre><pre>                      ++++++.....+.+.=..|.+=+...=.+.|.-+...+......+.-.-+.+-..+.     +..+.+++|.+.-.+.-.
</pre><pre>T Consensus        34 ~~~~l~~~~~~i~~iq~~Le~i~~~l~~k~W~~v~~~ir~~~~~L~~~~~~Il~~ip~~~k~~a~~l~~~l~~~L~~Ld~  113 (142)
</pre><pre>T TIGR03042        34 QLAQIQRQAEGIEAAKDRLPELASLVAKGDWVFTRNLIHGPMGEVRREMTYLNQSLLPKDQKEALALAKELKDDLEKLDE  113 (142)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHcCChHHHHHHhHhhHHHHHHHHHHHHHcCCHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcCCCccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   76 TQGTLIGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        76 ~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      .-...+.+.|...|+..++.|+.-+.-+
</pre><pre>T Consensus       114 aa~~kd~~~~~~~~~~al~~l~~le~ll  141 (142)
</pre><pre>T TIGR03042       114 AARLQDGPQAQKAYQEAAADFDAYLDLL  141 (142)
</pre><pre>T ss_pred             HHHcCCHHHHHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>2YM9_D CELL INVASION PROTEIN SIPD; CELL INVASION, TYPE 3 SECRETION, T3SS, NEEDLE TIP PROTEIN, HOST PATHOGEN INTERACTION, BACTERIAL PATHOGENESIS, TIP COMPLEX; HET: SO4, GOL; 3.0A {SALMONELLA ENTERICA SUBSP. ENTERICA SEROVAR TYPHIMURIUM}
</pre><pre>Probab=57.99  E-value=69  Score=33.20  Aligned_cols=107  Identities=12%  Similarity=0.137  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHH-HhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFN-KHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~-~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      .+|....+........|++++.+.+. +.-+.+...........+.....-...+++|.+.+..|..-|...        
</pre><pre>T Consensus        84 ~~~~~~l~~l~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~S~~e~~~~I~~~I~~ik~~YL~v--------  155 (346)
</pre><pre>T 2YM9_D           84 QQLTSSLNALAKSGVSLSAEQNENLRSAFSAPTSALFSASPMAQPRTTISDAEIWDMVSQNISAIGDSYLGV--------  155 (346)
</pre><pre>T ss_dssp             ----------------------------------------------CCCCHHHHHHTTCCC-----CTHHHH--------
</pre><pre>T ss_pred             HHHHHHHHHHHhcCCCCCHHHHHHHHHHhhhchhhHhccCCCCCCCCCCCHHHHHHHHHHHHHHHchhhHHH--------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIK  117 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK  117 (262)
</pre><pre>                      ..+...+....|+.+...+.++..+--.-.+-+++|
</pre><pre>T Consensus       156 Ye~iv~~Yt~Fy~~fs~ilskm~~~I~~~~DG~~V~  191 (346)
</pre><pre>T 2YM9_D          156 YENVVAVYTDFYQAFSDILSKMGGWLLPGKDGNTVK  191 (346)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTCC--------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhcccCCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>4JIO_B BRO1; Membrane Trafficking, ESCRT-I, ESCRT-III, Ubiquitin, Endosome, PROTEIN BINDING; HET: MSE; 3.6A {Naumovozyma castellii}
</pre><pre>Probab=57.90  E-value=3.6e+02  Score=27.19  Aligned_cols=132  Identities=13%  Similarity=0.217  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcC--C-HHHHHHHHHhH----HHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAEL--N-TDEANFFNKHF----STIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L--~-~~ea~fl~~n~----~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      ++..|+.+-+.+-+.|...  + ++-...++.++    ..++.++..-=.-.......+...++.-.+.|.+|..++.+.
</pre><pre>T Consensus       198 ~l~~l~~~R~~~l~~lk~~~~~~ddI~~~Ll~~~~~~~~~~~~if~~eL~k~~~~~~~l~~~~~~q~~ll~~l~~~~~~~  277 (344)
</pre><pre>T 4JIO_B          198 DLRLLKEERARTMGELKDYVNNNDDITSLLLANKDKSDAELQSLFESELAKFEPLTTRIEATIFKQGSTINGVKIELDNI  277 (344)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHSCC--------CCCCCCSCSTTHHHHHHHTTSHHHHHHHHHHHHHTTHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhcCCChHHHHHHccCCCHHHHHHHHHHHHHccHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhcCCC----ccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHH
</pre><pre>Q WP_048874398.1   74 YRTQGTL----IGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNA  149 (262)
</pre><pre>Q Consensus        74 ~~~~g~l----~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~  149 (262)
</pre><pre>                      ....+..    ....-..+|+..+++|+.....|..-..++.+                         ++.=|.+-...+
</pre><pre>T Consensus       278 ~~~~~~~~~~~~~~~~~~~r~~~~~~l~~a~~~~~~i~~~l~e-------------------------G~~FY~~L~~~l  332 (344)
</pre><pre>T 4JIO_B          278 FGLTGIQDKTPEEQKAELERKLFFDKLEEATTKFTIFSSDIQK-------------------------GLNFYNSLLKMS  332 (344)
</pre><pre>T ss_dssp             HHHSSSCSCCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHH
</pre><pre>T ss_pred             hcCcccccCCchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHH
</pre><pre>Q WP_048874398.1  150 SKAARFLK  157 (262)
</pre><pre>Q Consensus       150 ak~~k~lk  157 (262)
</pre><pre>                      .+..+..+
</pre><pre>T Consensus       333 ~~l~~~v~  340 (344)
</pre><pre>T 4JIO_B          333 KDLQNSVE  340 (344)
</pre><pre>T ss_dssp             HHHHHTTC
</pre><pre>T ss_pred             HHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>4LWS_A Uncharacterized protein; ESX Secretion, Signal Sequence, Type VII Secretion, UNKNOWN FUNCTION; HET: ACT, GOL, MSE; 2.0A {Thermomonospora curvata}
</pre><pre>Probab=57.08  E-value=2.1e+02  Score=24.10  Aligned_cols=95  Identities=12%  Similarity=0.051  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|++.++.+.+....+...-..+...--.+-..+...+..........|...++++.+.|.++.....+.-..    
</pre><pre>T Consensus        39 d~~~L~~~a~~l~~~~~~l~~~~~~l~~~~~~~~~~w~g~aa~af~~~~~~~~~~~~~l~~~l~~l~~~l~~~a~~----  114 (135)
</pre><pre>T 4LWS_A           39 DRAAMAQAAQDIEQSANAIRGMQNQLASAKDQLRSHWEGDASMAFEAVFNRFNEDFSRVLKALDGMHESLVQTRIT----  114 (135)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTCCSGGGGTTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH----
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccCcHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLL  100 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L  100 (262)
</pre><pre>                       ..+--.+-...|+++...+
</pre><pre>T Consensus       115 -y~~~D~~~a~~~~~~~~~l  133 (135)
</pre><pre>T 4LWS_A          115 -YEAREEAAQQSVNRVQALL  133 (135)
</pre><pre>T ss_dssp             -HHHHHHHHHTTHHHHHHHH
</pre><pre>T ss_pred             -HHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>1WDF_A E2 glycoprotein; MHV, coronavirus, heptad repeat, fusion core, viral entry, Viral protein; 2.5A {Murine hepatitis virus} SCOP: h.3.3.1
</pre><pre>Probab=52.98  E-value=87  Score=26.70  Aligned_cols=66  Identities=20%  Similarity=0.241  Sum_probs=0.0  Template_Neff=3.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      |...+....+-.++|..|+++    +.+||..+.+ -.........+---+.+.++.+.+..++|+.-|.+
</pre><pre>T Consensus        27 L~kiQ~vVN~q~~aL~~l~~q----L~~nF~aiSS-~~~~~~~~N~T~ldl~~Em~~LqevIk~LN~S~Id   92 (95)
</pre><pre>T 1WDF_A           27 LGKIQSVVNANAEALNNLLNQ----LSNRFGAISD-LSLDFEKLNVTLLDLTYEMNRIQDAIKKLNESYIN   92 (95)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH----HHHHTTCC-C-CCCCGGGCCCCCCCCHHHHHHHHHHHHHHTTCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH----HHHhhccccc-CCCCHHHhchhhhHHHHHHHHHHHHHHHHHHhhcc
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>TIGR02492 flgK_ends; flagellar hook-associated protein FlgK. The flagellar hook-associated protein FlgK of bacterial flagella has conserved N- and C-terminal domains.
</pre><pre>Probab=52.55  E-value=4.5e+02  Score=26.61  Aligned_cols=97  Identities=16%  Similarity=0.101  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhch----hhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMD----KSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-   76 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s----~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-   76 (262)
</pre><pre>                      |..+..+-+.+.....  ++..+..+.+.-+.|..-+...+    ..-..+...+....++|+..|++|.++=++-... 
</pre><pre>T Consensus       109 l~~f~~al~~la~~P~--~~~~r~~vi~~a~~l~~~~~~~~~~l~~~~~~~~~~i~~~v~~iN~ll~~Ia~lN~~I~~~~  186 (323)
</pre><pre>T TIGR02492       109 LNNFFNALQELAKNPD--SEALRQAVLESAQALANSFNQTSNELQDLRKGINAEIKSAVTEINSLLKQIASLNKEIKQVE  186 (323)
</pre><pre>T ss_pred             HHHHHHHHHHHHhCCC--CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCccHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   77 QGTLIGQQFFVERQKLFSELDVLL  100 (262)
</pre><pre>Q Consensus        77 ~g~l~~~~F~~~R~~Lf~~Ld~~L  100 (262)
</pre><pre>                      .......+...||.+|+.+|-..+
</pre><pre>T Consensus       187 ~~g~~~~~L~DqRD~ll~eLS~~~  210 (323)
</pre><pre>T TIGR02492       187 ASGQDANDLLDQRDLLLKELSQLI  210 (323)
</pre><pre>T ss_pred             hcCCChhhHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>4UT1_A FLAGELLAR HOOK-ASSOCIATED PROTEIN; MOTOR PROTEIN, ANTIGEN, EPITOPE DISCOVERY; HET: EDO; 1.8A {BURKHOLDERIA PSEUDOMALLEI K96243}
</pre><pre>Probab=51.76  E-value=5.7e+02  Score=29.03  Aligned_cols=100  Identities=12%  Similarity=0.068  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH---------------------------hhhchhhhHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL---------------------------TNFMDKSLGVVADAGE   53 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~---------------------------l~~~s~~~G~~s~~~~   53 (262)
</pre><pre>                      ..+...+..++++..+.+-+..-...|.+.|.-++.+                           -..=...-..+..-+.
</pre><pre>T Consensus        91 ~~~~~~~~l~~le~~~~~~~~gl~~~l~~ff~s~~~la~~P~~~~~R~~vi~~a~~la~~~~~~~~~l~~~~~~~~~~i~  170 (673)
</pre><pre>T 4UT1_A           91 SLSTYYTLVAQLNNYVGSPTAGIATAITNYFTGLQTVANNAADPSARQTAMSNAQTLASQLVAAGQQYSQLRQSVNSQLT  170 (673)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCCTTSTHHHHHHHHHHHHHHHHHSTTCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHCCCCCCHHHHHHHHHHHHHHHHHCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   54 RYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKLFSELDVLL  100 (262)
</pre><pre>Q Consensus        54 ~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~Lf~~Ld~~L  100 (262)
</pre><pre>                      ..+++|++.|++|.++=++-... ...-...+...||..|+++|-..+
</pre><pre>T Consensus       171 ~~V~~iN~ll~~Ia~lN~~I~~~~~~g~~~n~L~DqRd~ll~eLS~~v  218 (673)
</pre><pre>T 4UT1_A          171 DTVTQINSYTSQIAQLNEQIASASSQGQPPNQLLDQRDLAVSKLSQLA  218 (673)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHTC----CHHHHHHHHHHHHHHHTTS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhcCCCcchHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>PF03081.17 ; Exo70 ; Exo70 exocyst complex subunit
</pre><pre>Probab=51.74  E-value=4.4e+02  Score=26.29  Aligned_cols=138  Identities=10%  Similarity=0.085  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHHh-cCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAI-ELAYQNQYRT-QGT   79 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~i-e~ly~~~~~~-~g~   79 (262)
</pre><pre>                      |..|...-+.-.... ..+....-||.+|+..+...+.......-.......++-+.+.+..+.+ ...+..-+.. ...
</pre><pre>T Consensus       204 i~~L~~~l~~ks~~~-~~~~~~~iFllNN~~~I~~~l~~~~l~~~l~~~~~~~~~~~~~~~~~~y~~~~w~~~~~~l~~~  282 (373)
</pre><pre>T D8SPE6_SELML/2  204 LEALMRHTDENARLY-KDTVLSTIFLMNNTHYIVQKAKHAGIQFVIGDSWLRRHSSLVRYHAMNYQRVAWGKIFSYLRDE  282 (373)
</pre><pre>T ss_pred             HHHHHHHHHHHHHhc-CCHHHHHHHHHHHHHHHHHHHHhcCHHHHhCHHHHHHHHHHHHHHHHHHHHHhcHHHHHhHhhc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             Cc------cHHHHHHHHHHHHHHHHHHHHHHH--hhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LI------GQQFFVERQKLFSELDVLLNKLSR--LTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASK  151 (262)
</pre><pre>Q Consensus        80 l~------~~~F~~~R~~Lf~~Ld~~L~~~~~--~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak  151 (262)
</pre><pre>                      ..      ..+-.  .+..|...+..|+.+.+  +...+|+ + ||+.|   .+.+.+.       -+|.|.+-+++...
</pre><pre>T Consensus       283 ~~~~~l~~~~~~~--~k~~f~~Fn~~f~~~~~~~~~~~v~d-~-LR~~l---~~~i~~~-------v~~~Y~~f~~~~~~  348 (373)
</pre><pre>T D8SPE6_SELML/2  283 GIRGPGYNISKEI--LKERFKNFNAAFEEIHRTQAGWVVSD-G-LRDEL---RVLISDK-------LIPAYRSFLGRYRV  348 (373)
</pre><pre>T ss_pred             CCCCCCccCCHHH--HHHHHHHHHHHHHHHHHHhCCCccch-H-HHHHH---HHHHHHh-------HHHHHHHHHHHhHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHH
</pre><pre>Q WP_048874398.1  152 AAR  154 (262)
</pre><pre>Q Consensus       152 ~~k  154 (262)
</pre><pre>                      ..|
</pre><pre>T Consensus       349 ~~~  351 (373)
</pre><pre>T D8SPE6_SELML/2  349 HLE  351 (373)
</pre><pre>T ss_pred             Hhc
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>3L1N_A Cell wall antigen; helix-turn-helix, protein-ligand complex, LIPID BINDING PROTEIN; HET: MSE, PLM; 1.3A {Penicillium marneffei}
</pre><pre>Probab=51.64  E-value=3.4e+02  Score=24.96  Aligned_cols=102  Identities=13%  Similarity=-0.012  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHH--HHHHHHHHHHHHHHHHHHHhcC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYF--SEIEKKLKAIELAYQNQYRTQG   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~--~~i~~~L~~ie~ly~~~~~~~g   78 (262)
</pre><pre>                      ++....+++...-+...+++++|...+.+.++-+.......-..+-.--..+.+.-  ..|...|+.+..........-.
</pre><pre>T Consensus        86 ~l~~~l~~a~~~~~~s~~ls~~ds~~i~~~~~~l~~~~~~~l~~l~aKk~~f~~~g~~~~v~~~L~~l~~~~~~l~~al~  165 (194)
</pre><pre>T 3L1N_A           86 VLIKATLDGVQSLQNEPPLSSMEALALVGPVQDLSNQIMLAIQNLIDKKEPLVQAGFGGKVENNLRQQEEAAQKLSELVS  165 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTTCCCCCHHHHHHHHCHHHHHHHHHHHHHHHHHHTHHHHHHTTCHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHhcCCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHchHHHHHCCChHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      +.-++++...-+.+.+++|..+.+
</pre><pre>T Consensus       166 ~k~p~~~~~~a~~~~~~i~~~f~~  189 (194)
</pre><pre>T 3L1N_A          166 TKVPHELADISRQLSDGIAAGIKK  189 (194)
</pre><pre>T ss_dssp             HHSCGGGHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hhCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>5IJN_G NUCLEAR PORE COMPLEX PROTEIN NUP58; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=51.61  E-value=2.5e+02  Score=31.46  Aligned_cols=98  Identities=15%  Similarity=0.114  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCC-------HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELN-------TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~-------~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      +|+++.+..+++.+.|....       ++-.+.|.+.+|.|-++               ....++|.+-++++..+|.+.
</pre><pre>T Consensus       356 rLeEYrQ~IEEIEQlLnS~~nss~qSPQsLpKVMSNQHD~FVhV---------------AAKVEsLHQ~VEsMKtAYLA~  420 (599)
</pre><pre>T 5IJN_G          356 QLQQYRQQIEELENHLATQANNSHITPQDLSMAMQKIYQTFVAL---------------AAQLQSIHENVKVLKEQYLGY  420 (599)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHCCCCCSHHHHHHHHHHHHHHHHH---------------HHHHHHHHHHHHHHHHHHC--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhcccCCCCHHHHHHHHHHHHHHHHHH---------------HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHH
</pre><pre>Q WP_048874398.1   74 YRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEE  115 (262)
</pre><pre>Q Consensus        74 ~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~  115 (262)
</pre><pre>                      .++...--.+-|.+.++  ..+....-.+.-+-...+.+.++
</pre><pre>T Consensus       421 dqR~rGD~~DPFleAdR--~Akqeaaa~rvh~~~~~~~~~~~  460 (599)
</pre><pre>T 5IJN_G          421 RKMFLGDAVDVFETRRA--EAKKWQNTPRVTTGPTPFSTMPN  460 (599)
</pre><pre>T ss_dssp             ------------------------------------------
</pre><pre>T ss_pred             HHHHhcCCCCHHHHHHH--HHHHHHcCCccCCCCCCCCCChh
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>5IJO_M Nucleoporin p58/p45; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=51.61  E-value=2.5e+02  Score=31.46  Aligned_cols=98  Identities=15%  Similarity=0.114  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCC-------HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELN-------TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~-------~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      +|+++.+..+++.+.|....       ++-.+.|.+.+|.|-++               ....++|.+-++++..+|.+.
</pre><pre>T Consensus       356 rLeEYrQ~IEEIEQlLnS~~nss~qSPQsLpKVMSNQHD~FVhV---------------AAKVEsLHQ~VEsMKtAYLA~  420 (599)
</pre><pre>T 5IJO_M          356 QLQQYRQQIEELENHLATQANNSHITPQDLSMAMQKIYQTFVAL---------------AAQLQSIHENVKVLKEQYLGY  420 (599)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHCCCCCSHHHHHHHHHHHHHHHHH---------------HHHHHHHHHHHHHHHHHHC--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhcccCCCCHHHHHHHHHHHHHHHHHH---------------HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHH
</pre><pre>Q WP_048874398.1   74 YRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEE  115 (262)
</pre><pre>Q Consensus        74 ~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~  115 (262)
</pre><pre>                      .++...--.+-|.+.++  ..+....-.+.-+-...+.+.++
</pre><pre>T Consensus       421 dqR~rGD~~DPFleAdR--~Akqeaaa~rvh~~~~~~~~~~~  460 (599)
</pre><pre>T 5IJO_M          421 RKMFLGDAVDVFETRRA--EAKKWQNTPRVTTGPTPFSTMPN  460 (599)
</pre><pre>T ss_dssp             ------------------------------------------
</pre><pre>T ss_pred             HHHHhcCCCCHHHHHHH--HHHHHHcCCccCCCCCCCCCChh
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>7D6D_A Sorting nexin-1; Coat complex, Membrane deformation, LIPID BINDING PROTEIN, helical assembly, PROTEIN TRANSPORT; 9.0A {Mus musculus}
</pre><pre>Probab=50.51  E-value=6.1e+02  Score=27.56  Aligned_cols=149  Identities=14%  Similarity=0.056  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH--hhhchhhhHHHH--HHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL--TNFMDKSLGVVA--DAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~--l~~~s~~~G~~s--~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      ...+.+...+.+...+..|-..-........++-..+  ++..-...+..-  ..+.+.+.++.++++.++.+|+.+...
</pre><pre>T Consensus       315 ~fe~~~~~v~~l~~~l~~L~~~~~~l~~~~~~ls~~~~~~a~sl~~l~~~e~~~~ls~~l~~la~~~~~~~~l~~~qa~~  394 (529)
</pre><pre>T 7D6D_A          315 WFEEKLQEVECEEQRLRKLHAVVETLVNHRKELALNTALFAKSLAMLGSSEDNTALSRALSQLAEVEEKIEQLHQEQANN  394 (529)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTSCCCHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCHHHHHHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cC-------------CCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc--ccccch
</pre><pre>Q WP_048874398.1   77 QG-------------TLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG--VASIRG  141 (262)
</pre><pre>Q Consensus        77 ~g-------------~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g--~g~Ip~  141 (262)
</pre><pre>                      .-             --.-.+.+.+|.+++++++..-..+.++.-.+   .+|+..-  +...+-.......  ...+..
</pre><pre>T Consensus       395 ~~~~l~e~L~~y~~~~~avk~~~~~R~~~~~~~~~a~~~L~kkre~~---~kl~~~~--k~~k~~qa~~ei~e~e~~~~~  469 (529)
</pre><pre>T 7D6D_A          395 DFFLLAELLSDYIRLLAIVRAAFDQRMKTWQRWQDAQATLQKKRESE---ARLLWAN--KPDKLQQAKDEITEWESRVTQ  469 (529)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHTTTSC--CSHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHHHHcc--CHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  142 YSTYVDNASKAAR  154 (262)
</pre><pre>Q Consensus       142 y~~~i~~~ak~~k  154 (262)
</pre><pre>                      +.+.++.+.+..+
</pre><pre>T Consensus       470 ~~~~~e~is~~l~  482 (529)
</pre><pre>T 7D6D_A          470 YERDFERISTVVR  482 (529)
</pre><pre>T ss_dssp             HHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>7D6D_I Sorting nexin-1; Coat complex, Membrane deformation, LIPID BINDING PROTEIN, helical assembly, PROTEIN TRANSPORT; 9.0A {Mus musculus}
</pre><pre>Probab=50.51  E-value=6.1e+02  Score=27.56  Aligned_cols=149  Identities=14%  Similarity=0.056  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH--hhhchhhhHHHH--HHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL--TNFMDKSLGVVA--DAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~--l~~~s~~~G~~s--~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      ...+.+...+.+...+..|-..-........++-..+  ++..-...+..-  ..+.+.+.++.++++.++.+|+.+...
</pre><pre>T Consensus       315 ~fe~~~~~v~~l~~~l~~L~~~~~~l~~~~~~ls~~~~~~a~sl~~l~~~e~~~~ls~~l~~la~~~~~~~~l~~~qa~~  394 (529)
</pre><pre>T 7D6D_I          315 WFEEKLQEVECEEQRLRKLHAVVETLVNHRKELALNTALFAKSLAMLGSSEDNTALSRALSQLAEVEEKIEQLHQEQANN  394 (529)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTSCCCHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCHHHHHHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cC-------------CCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc--ccccch
</pre><pre>Q WP_048874398.1   77 QG-------------TLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG--VASIRG  141 (262)
</pre><pre>Q Consensus        77 ~g-------------~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g--~g~Ip~  141 (262)
</pre><pre>                      .-             --.-.+.+.+|.+++++++..-..+.++.-.+   .+|+..-  +...+-.......  ...+..
</pre><pre>T Consensus       395 ~~~~l~e~L~~y~~~~~avk~~~~~R~~~~~~~~~a~~~L~kkre~~---~kl~~~~--k~~k~~qa~~ei~e~e~~~~~  469 (529)
</pre><pre>T 7D6D_I          395 DFFLLAELLSDYIRLLAIVRAAFDQRMKTWQRWQDAQATLQKKRESE---ARLLWAN--KPDKLQQAKDEITEWESRVTQ  469 (529)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHTTTSC--CSHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHHHHcc--CHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  142 YSTYVDNASKAAR  154 (262)
</pre><pre>Q Consensus       142 y~~~i~~~ak~~k  154 (262)
</pre><pre>                      +.+.++.+.+..+
</pre><pre>T Consensus       470 ~~~~~e~is~~l~  482 (529)
</pre><pre>T 7D6D_I          470 YERDFERISTVVR  482 (529)
</pre><pre>T ss_dssp             HHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>5E7X_A Cell wall antigen; palmitic acid, MP1, LIPID BINDING PROTEIN; HET: PLM; 1.8A {Talaromyces marneffei PM1}
</pre><pre>Probab=50.00  E-value=2.9e+02  Score=23.64  Aligned_cols=100  Identities=14%  Similarity=0.038  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGE--RYFSEIEKKLKAIELAYQNQYRTQGT   79 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~--~~~~~i~~~L~~ie~ly~~~~~~~g~   79 (262)
</pre><pre>                      ...++.....++. ...|+++|..-+...+.-+.......-..+-.--..+.  ..-..|...|..+..........-..
</pre><pre>T Consensus        47 ~~~l~~~~~~~~~-~~~ls~~d~~~i~~~~~~l~~~~~~~l~~l~~kk~~~~~~~~~~~v~~~L~~l~~~~~~l~~al~~  125 (155)
</pre><pre>T 5E7X_A           47 IKATENGLQELGP-QPPLSLTEALALVGPVQGVNKLIMKTVDHLIEKKGPLVGGGYGPQVKASLQKQAHAAVTLSELVSS  125 (155)
</pre><pre>T ss_dssp             HHHHHHHHHHHTT-CCCCCHHHHHTTHHHHHHHHHHHHHHHHHHHTTHHHHHHTTCHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhcC-CCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHchHHHHhCCchHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        80 l~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      .-++++...=+.+..+++..+.+
</pre><pre>T Consensus       126 k~p~~~~~~~~~~~~~i~~~~~~  148 (155)
</pre><pre>T 5E7X_A          126 KVPSPLAPISKQLSDQVAQALQK  148 (155)
</pre><pre>T ss_dssp             HSCGGGHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hCChhhHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>TIGR02131 phaP_Bmeg; polyhydroxyalkanoic acid inclusion protein PhaP. This model describes a protein found in polyhydroxyalkanoic acid (PHA) gene regions and incorporated into PHA inclusions in Bacillus cereus and Bacillus megaterium.
</pre><pre>Probab=49.87  E-value=3.9e+02  Score=25.18  Aligned_cols=100  Identities=12%  Similarity=0.049  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             chhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   41 MDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        41 ~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                      ....+.+.=..|.+-++.+.+.-+++|.+|-+.+..     -.+|...-+.-+.+++....++..--..- .-+.+|+.=
</pre><pre>T Consensus         6 ~~~~vD~~Wd~W~nsl~~~~~~qe~~E~~tLqal~~-----Qke~~~~~~~~l~~~eee~Kk~~~e~~~~-~~~~~~~~~   79 (165)
</pre><pre>T TIGR02131         6 YDTLIDAFWENWSKGLQLFADAGKQIEQLTLEALEQ-----QQDALHKATEGLDALDKELKAELADFNNK-TTDNLKKLA   79 (165)
</pre><pre>T ss_pred             hhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCcchhhhhhhhhccccccchHHHHHHHHH
</pre><pre>Q WP_048874398.1  121 GLSSRSIVHEWSTAGVASIRGYSTYVDNAS  150 (262)
</pre><pre>Q Consensus       121 GlStkS~vh~w~~~g~g~Ip~y~~~i~~~a  150 (262)
</pre><pre>                      |-...-.+-.|.    ..+.++..+++.++
</pre><pre>T Consensus        80 ~e~~~~~~~ew~----~ql~Ei~~~~q~Ls  105 (165)
</pre><pre>T TIGR02131        80 GNALADQIEEWQ----DKTHEALAHLHELF  105 (165)
</pre><pre>T ss_pred             ccHHHHHHHHHH----HHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>d1yaro_ a.24.8.1 (O:) automated matches {Trypanosome (Trypanosoma brucei) [TaxId: 5691]}
</pre><pre>Probab=49.86  E-value=2.8e+02  Score=27.42  Aligned_cols=91  Identities=11%  Similarity=0.132  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCC--------------------HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELN--------------------TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIE   60 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~--------------------~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~   60 (262)
</pre><pre>                      +|+++-+++.++...+...+                    .++...+.+..+.+...+...-.-+--..++.-..-+++-
</pre><pre>T Consensus        32 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~N~~i~~ll~~vk~ei~~l~e~~~~Ik~WI~L~IPkieDGnNFGV~VQe~VL  111 (228)
</pre><pre>T d1yaro_          32 EIEGIAKAAAEAHGVIRNSTYGRAQAEKSPEQLLGVLQRYQDLCHNVYCQAETIRTVIAIRIPEHKEEDNLGVAVQHAVL  111 (228)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHTCCCCTHHHHSCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTCCCCCSSSCHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHcCCCCCcccCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhCCCCCCCCccchHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH--------------HHhcCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   61 KKLKAIELAYQNQ--------------YRTQGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        61 ~~L~~ie~ly~~~--------------~~~~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      +.|++++......              +..     -.+||..|..+..|+
</pre><pre>T Consensus       112 ~~L~~~e~~~~~~~~~~~~~~~~~~~~~~~-----i~~Y~~~RA~l~~Kl  156 (228)
</pre><pre>T d1yaro_         112 KIIDELEIKTLGSGEKSGSGGAPTPIGMYA-----LREYLSARSTVEDKL  156 (228)
</pre><pre>T ss_dssp             HHHHCCCCCCSCCSSBCCTTCBCSSSCTTH-----HHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhcCCccCCCCCCCCchhHHH-----HHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>PF19180.2 ; DUF5862 ; Family of unknown function (DUF5862)
</pre><pre>Probab=49.65  E-value=1.7e+02  Score=23.30  Aligned_cols=51  Identities=27%  Similarity=0.335  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             hchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHH-h
</pre><pre>Q WP_048874398.1  200 ASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYV-M  259 (262)
</pre><pre>Q Consensus       200 G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~-G  259 (262)
</pre><pre>                      |++.|+..++..+        .++|-..+|-+.+..+++.-++|.+.|. .|...+-.. |
</pre><pre>T Consensus         1 gai~G~~~g~~~G--------~~iGg~~gGgG~lg~G~I~q~VG~i~G~-~G~i~G~v~~G   52 (68)
</pre><pre>T A0A095T1P9_9GA    1 GSILGAIDGFGTG--------AAVGGKYGGAGGWIVGGIAQLVGLTASA-VGAGIGGYLYG   52 (68)
</pre><pre>T ss_pred             ChhHHHHHHHHHH--------HHHhhhhccccchhHhHHHHHHHHHHHH-HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>PF05524.15 ; PEP-utilisers_N ; PEP-utilising enzyme, N-terminal
</pre><pre>Probab=49.56  E-value=1.6e+02  Score=24.31  Aligned_cols=72  Identities=17%  Similarity=0.253  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcC-----CHHHHHHHHHhHHHHHH--HhhhchhhhH----HHHHHHHHHHHHHHHHHHHH-HH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAEL-----NTDEANFFNKHFSTIAA--LTNFMDKSLG----VVADAGERYFSEIEKKLKAI-EL   68 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L-----~~~ea~fl~~n~~~l~~--~l~~~s~~~G----~~s~~~~~~~~~i~~~L~~i-e~   68 (262)
</pre><pre>                      +++.|.++..++.+.|..+     ++++++.|.-|..++++  |......-+.    .+..++.+.++++.+.|.++ ++
</pre><pre>T Consensus        35 e~~~l~~a~~~~~~~l~~~~~~~~~~~~~~I~~a~~~il~D~~l~~~i~~~i~~~~~~a~~Av~~~~~~~~~~~~~~~~d  114 (119)
</pre><pre>T E3PU46_ACESD/3   35 ESQRVTLAFHKAKLEIEEFISTLTNEEEKNIINTQLMMLEDPELLKEIDNYIKDNSVNAESAIDKILDAYIAMFEQMPDE  114 (119)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhCCCHHHHHHHHHHHHhhhCHHHHHHHHHHHHhCCCCHHHHHHHHHHHHHHHHhcCCcc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHH
</pre><pre>Q WP_048874398.1   69 AYQNQ   73 (262)
</pre><pre>Q Consensus        69 ly~~~   73 (262)
</pre><pre>                       |-++
</pre><pre>T Consensus       115 -yl~e  118 (119)
</pre><pre>T E3PU46_ACESD/3  115 -YMSQ  118 (119)
</pre><pre>T ss_pred             -hhhc
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>6S7O_E Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1; N-glycosylation, Oligosaccharyltransferase, OSTA, TRANSFERASE; HET: MAN, KZB, EGY, NAG, BMA, KZE; 3.5A {Homo sapiens}
</pre><pre>Probab=49.26  E-value=6e+02  Score=28.28  Aligned_cols=91  Identities=9%  Similarity=0.143  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      .+++...+.++.-++.++.+        -.-+..-+...+....-....++..-.++.+...+|+++..+ |+ .|++..
</pre><pre>T Consensus       510 ~~y~~~~~~l~~~l~~l~~~--------l~~l~~~l~~~~~~~~~~i~~le~~e~e~~~~~~~l~~l~~r-~r-~g~isr  579 (607)
</pre><pre>T 6S7O_E          510 STLNSGKKSLETEHKALTSE--------IALLQSRLKTEGSDLCDRVSEMQKLDAQVKELVLKSAVEAER-LV-AGKLKK  579 (607)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHH--------HHHHTSSSCCSHHHHHHHHHHHHHHHHHHHHHHHHTTSSSCC-SH-HHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHH--------HHHHHHHHHhcCccHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HH-cCCCcH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      .+|...++.+.++++..-+++
</pre><pre>T Consensus       580 ~~Y~~l~~~~~~ri~k~~~~I  600 (607)
</pre><pre>T 6S7O_E          580 DTYIENEKLISGKRQELVTKI  600 (607)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHCC------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>PF02252.20 ; PA28_beta ; Proteasome activator pa28 beta subunit
</pre><pre>Probab=48.78  E-value=2e+02  Score=25.18  Aligned_cols=81  Identities=10%  Similarity=0.077  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVER   89 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R   89 (262)
</pre><pre>                      +.|.+.+..+.+ |...+.+..+.+...+...-.-+....++.-...+++-+.+.++++.-...+..     -..|+.+|
</pre><pre>T Consensus         6 ~~v~~l~~~vk~-e~~~l~~~~~~i~~Wi~l~iPkiedGnnfgv~vq~~vl~~l~~~~~~~~~~~~~-----i~~Y~~~R   79 (145)
</pre><pre>T T1JYH5_TETUR/1    6 NKIKAMSNLVKP-KMKEFIEDLNLIKLWIQLLIPKIEDGNNFGVEIQEEILAEVRFALSDGFTYYSF-----MIKYHNTR   79 (145)
</pre><pre>T ss_pred             HHHHHHHHHHHH-HHHHHHHHHhHHHHHHHhcccccCCCCCccHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   90 QKLFSEL   96 (262)
</pre><pre>Q Consensus        90 ~~Lf~~L   96 (262)
</pre><pre>                      .++..++
</pre><pre>T Consensus        80 a~l~~k~   86 (145)
</pre><pre>T T1JYH5_TETUR/1   80 AGIVKKI   86 (145)
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>d1zyma1 a.60.10.1 (A:22-144) Enzyme I of the PEP:sugar phosphotransferase system HPr-binding (sub)domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=48.66  E-value=3e+02  Score=23.49  Aligned_cols=89  Identities=9%  Similarity=0.231  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcC--------CHHHHHHHHHhHHHHH------HHhhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAEL--------NTDEANFFNKHFSTIA------ALTNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L--------~~~ea~fl~~n~~~l~------~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      ++..|..+...+.+.|..+        +++++..+.-|-.+|+      .+...-....-.+..++.+.++++.+.|.++
</pre><pre>T Consensus        18 E~~rl~~Ai~~~~~~l~~~~~~~~~~~~~~~~~I~~~~~~~L~D~~l~~~v~~~I~~~~~~a~~Av~~~~~~~~~~f~~~   97 (123)
</pre><pre>T d1zyma1          18 EVERFLSGRAKASAQLETIKTKAGETFGEEKEAIFEGHIMLLEDEELEQEIIALIKDKHMTADAAAHEVIEGQASALEEL   97 (123)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHTCHHHHHHHHHHHHHHCCCHHHHHHHHHHHHHTTTSSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhhhCHHHHHHHHHHHHHHcCHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhcCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   67 ELAYQNQYRTQGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        67 e~ly~~~~~~~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ++-|-++       +..++..-+++|+..|
</pre><pre>T Consensus        98 ~d~ylre-------R~~Di~dv~~rll~~L  120 (123)
</pre><pre>T d1zyma1          98 DDEYLKE-------RAADVRDIGKRLLRNI  120 (123)
</pre><pre>T ss_dssp             CCHHHHH-------HHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCHHHHH-------HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>PF08855.12 ; DUF1825 ; Domain of unknown function (DUF1825)
</pre><pre>Probab=48.15  E-value=1.9e+02  Score=24.73  Aligned_cols=63  Identities=6%  Similarity=0.086  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHH--HHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNK--LSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~--~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                      +.|.+.+.+|.++|++-+....+...-. ...|..-+.++...+.+  ++-..+.|.+.+.-|..+
</pre><pre>T Consensus         7 eiVq~El~~i~~~~~~l~~~~~~~~~fd-~e~K~~~Id~~~~LieKqkify~Rl~LsdD~~A~~~~   71 (103)
</pre><pre>T K9TWP3_CHRTP/1    7 DIVQQEAKQLFEDYQALIKLGNSYGKFD-REGKKLYIDQMEAMMERYRIFMKRFELSEDFMAQMTV   71 (103)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHhhccCC-hHHHHHHHHHHHHHHHHHHHHHHHHHcCCChHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>1AVO_D 11S REGULATOR; PROTEASOME ACTIVATOR, CELL ADHESION, INTERFERON INDUCTION; 2.8A {Homo sapiens}
</pre><pre>Probab=47.60  E-value=2.1e+02  Score=24.95  Aligned_cols=81  Identities=10%  Similarity=0.104  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVER   89 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R   89 (262)
</pre><pre>                      +.|.+.+..+.+ +...+.+..+.+...+..--.-+....++.....+++.+.+.++++.-...+..     ...|+..|
</pre><pre>T Consensus         6 ~~i~~l~~~lk~-~~~~l~~~~~~l~~WI~l~iPkiedgnnfgv~vq~~vl~~l~~~~~~~~~~~~~-----~~~Y~~~R   79 (140)
</pre><pre>T 1AVO_D            6 EKIVVLLQRLKP-EIKDVIEQLNLVTTWLQLQIPRIEDGNNFGVAVQEKVFELMTSLHTKLEGFHTQ-----ISKYFSER   79 (140)
</pre><pre>T ss_dssp             HHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHTCCCCCSSCCHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHH-HHHHHHHHHhHHhHHHHhcCccccCCCCcchHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   90 QKLFSEL   96 (262)
</pre><pre>Q Consensus        90 ~~Lf~~L   96 (262)
</pre><pre>                      ..+..++
</pre><pre>T Consensus        80 a~lv~k~   86 (140)
</pre><pre>T 1AVO_D           80 GDAVTKA   86 (140)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>1AVO_J 11S REGULATOR; PROTEASOME ACTIVATOR, CELL ADHESION, INTERFERON INDUCTION; 2.8A {Homo sapiens}
</pre><pre>Probab=47.60  E-value=2.1e+02  Score=24.95  Aligned_cols=81  Identities=10%  Similarity=0.104  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVER   89 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R   89 (262)
</pre><pre>                      +.|.+.+..+.+ +...+.+..+.+...+..--.-+....++.....+++.+.+.++++.-...+..     ...|+..|
</pre><pre>T Consensus         6 ~~i~~l~~~lk~-~~~~l~~~~~~l~~WI~l~iPkiedgnnfgv~vq~~vl~~l~~~~~~~~~~~~~-----~~~Y~~~R   79 (140)
</pre><pre>T 1AVO_J            6 EKIVVLLQRLKP-EIKDVIEQLNLVTTWLQLQIPRIEDGNNFGVAVQEKVFELMTSLHTKLEGFHTQ-----ISKYFSER   79 (140)
</pre><pre>T ss_dssp             HHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHTCCCCCSSCCHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHH-HHHHHHHHHhHHhHHHHhcCccccCCCCcchHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   90 QKLFSEL   96 (262)
</pre><pre>Q Consensus        90 ~~Lf~~L   96 (262)
</pre><pre>                      ..+..++
</pre><pre>T Consensus        80 a~lv~k~   86 (140)
</pre><pre>T 1AVO_J           80 GDAVTKA   86 (140)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>1WDG_B E2 glycoprotein; MHV, coronavirus, heptad repeat, fusion core, viral entry, Viral protein; 2.06A {Murine hepatitis virus} SCOP: h.3.3.1
</pre><pre>Probab=46.64  E-value=1.7e+02  Score=25.02  Aligned_cols=67  Identities=19%  Similarity=0.251  Sum_probs=0.0  Template_Neff=3.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHH------------HHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAG------------ERYFSEIEKKLKAIELA   69 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~------------~~~~~~i~~~L~~ie~l   69 (262)
</pre><pre>                      +....+....++.+|.+    --+.+.+|-+.|..++..=+...|+.++.+            ...++++.+..++|++.
</pre><pre>T Consensus        13 ~~~i~~~f~t~~~AL~k----iQ~vVN~~~~aL~~l~~qL~~nF~AIs~sf~~~~fN~T~LnL~~EmdrLqevik~LN~S   88 (94)
</pre><pre>T 1WDG_B           13 LGAIQDGFDATNSALGK----IQSVVNANAEALNNLLNQLSLVPRGSGGSGGSGGLNVTLLDLTYEMNRIQDAIKKLNES   88 (94)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHTTCCC-------------------CCHHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHhCcccccCCCCCCcccHhHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHH
</pre><pre>Q WP_048874398.1   70 YQN   72 (262)
</pre><pre>Q Consensus        70 y~~   72 (262)
</pre><pre>                      |.+
</pre><pre>T Consensus        89 ~ID   91 (94)
</pre><pre>T 1WDG_B           89 YIN   91 (94)
</pre><pre>T ss_dssp             CCC
</pre><pre>T ss_pred             hcc
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>PF15469.8 ; Sec5 ; Exocyst complex component Sec5
</pre><pre>Probab=46.53  E-value=8.1e+02  Score=27.87  Aligned_cols=130  Identities=11%  Similarity=0.134  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVAD-----AGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~-----~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      +..|+.......+.+..++++-..++.+||+-+-.....-...-.....     ......+.+.+.+.++.....+.+..
</pre><pre>T Consensus        49 ~~~L~~~~~~L~~~~~~~~~~l~~lV~~N~~~Fi~~~~~i~~l~~~~~~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~  128 (819)
</pre><pre>T C5DMT2_LACTC/7   49 FKDLSLSLDHLDQSLQAQSEDLKQLVQRNFVKYVRSKNNLDRIYEQFNKFSLGESRDFGTDDLGETVDESIREITIKVKP  128 (819)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHcHHHHHHHHHHhHHHHHHHHHHHHHHHHHHHhcccCCccccchHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   77 QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        77 ~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                               .-.|+.-..++.+.++-+-+..+=+.-...|+..+                 .=..|.+-++...++-..+
</pre><pre>T Consensus       129 ---------~l~~~~~~~~l~~~l~~l~~~~~lf~LP~~L~~~i-----------------~~~~y~~av~~y~~~k~l~  182 (819)
</pre><pre>T C5DMT2_LACTC/7  129 ---------ILDISTKRRNAQTTIAFLQDHKQFFDAPKKLKHCL-----------------IEKDFANLVVEYNNAHSTF  182 (819)
</pre><pre>T ss_pred             ---------HhhhHHHHHHHHHHHHHHHHcHHHhchhHHHHHHH-----------------HcCCHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             H
</pre><pre>Q WP_048874398.1  157 K  157 (262)
</pre><pre>Q Consensus       157 k  157 (262)
</pre><pre>                      +
</pre><pre>T Consensus       183 ~  183 (819)
</pre><pre>T C5DMT2_LACTC/7  183 K  183 (819)
</pre><pre>T ss_pred             H
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>PF10157.11 ; BORCS6 ; BLOC-1-related complex sub-unit 6
</pre><pre>Probab=46.38  E-value=4.1e+02  Score=24.37  Aligned_cols=90  Identities=17%  Similarity=0.114  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      |..|..+|+++...+..|...-...|.+--++-..-+..-...+..+.+.+...++.....+.+.|.+.+..-.      
</pre><pre>T Consensus        63 L~~LE~~A~~ia~~ld~ll~~L~~~L~~~S~lT~~~~~~y~~av~~~cd~vD~sIk~my~liakceeL~~~m~~------  136 (157)
</pre><pre>T A0A212EXV4_DAN   63 LNDLEYEAQKIAQSVDNLIENLSSILHSNSSLTAENMEVYRDAVGKTCDAMDNNIKSMYTILAKGEEVSQAMVP------  136 (157)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHccccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHhccc------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                      -..+-++.+.+-+.||
</pre><pre>T Consensus       137 v~~La~qIk~ik~~ld  152 (157)
</pre><pre>T A0A212EXV4_DAN  137 VQAQAARIAEIKRLLH  152 (157)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>5CSD_C Envelope glycoprotein; arachidonic acid, Ligand binding domain, LIPID BINDING PROTEIN; HET: ACD, GOL; 1.45A {Talaromyces marneffei PM1}
</pre><pre>Probab=45.95  E-value=3.4e+02  Score=23.37  Aligned_cols=100  Identities=14%  Similarity=0.071  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGER--YFSEIEKKLKAIELAYQNQYRTQGT   79 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~--~~~~i~~~L~~ie~ly~~~~~~~g~   79 (262)
</pre><pre>                      .+.+++....++. ..+|+++|...+...+.-|.......-..+-.--..+..  .-..|...|..+...-......-..
</pre><pre>T Consensus        47 ~~~l~~a~~~~~~-~~~ls~~d~~~l~~~~~~l~~~~~~~l~~l~~kk~~~~~~~~~~~v~~~l~~l~~~~~~l~~~l~~  125 (159)
</pre><pre>T 5CSD_C           47 IKATLDGVQSLQN-EPPLSSMEALALVGPVQDLSNQILLAIQNLIDKKEPLVQAGFGGKVENNLRQQEEAAQKLSELVST  125 (159)
</pre><pre>T ss_dssp             HHHHHHHHHHHTT-CCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHTHHHHHHTTCHHHHHHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHhhc-CCCCCHHHHHHhHHHHHHHHHHHHHHHHHHHHcHHHHHHCCChHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        80 l~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      .-++++...=+.+..+++..|..
</pre><pre>T Consensus       126 k~p~~~~~~~~~~~~~i~~~f~~  148 (159)
</pre><pre>T 5CSD_C          126 KVPHELADISRQLSDGIAAGIKK  148 (159)
</pre><pre>T ss_dssp             TSCHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>6W08_A Motility Associated Killing Factor E; Tripartite pore forming toxin, Cytotoxin, Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID, TOXIN; HET: EDO; 1.75A {Vibrio cholerae O1 biovar El Tor str. N16961}
</pre><pre>Probab=44.83  E-value=5.6e+02  Score=25.56  Aligned_cols=108  Identities=9%  Similarity=0.140  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHH--HHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTD--EANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~--ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g   78 (262)
</pre><pre>                      ++++++.+-+...+.|.++..+  ....+..+.+-+...+......+....+.|.....++.+.+++|+..-..    ..
</pre><pre>T Consensus       240 ~i~~~~~~i~~~~~~l~~~~~~~~~l~~~~~~l~~~~~~i~~a~~~l~~l~~~W~~l~~~l~~l~~~l~~~~~~----~~  315 (359)
</pre><pre>T 6W08_A          240 DIQSSLNTINHDQQTLSQDQQQIASLNALLLSVDQVNNDCAAISRSLDTLQTTVLSLYNETNNVVSNLTKAQDS----QA  315 (359)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTTSCHHHHHHHTGGGGGHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCSH----HH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHcccc----hh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCC
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKP  112 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~  112 (262)
</pre><pre>                      .......+..=..-.++|......|.......+.
</pre><pre>T Consensus       316 ~~~~~~~l~~a~~~W~~l~~~a~~~~~~~~~~~~  349 (359)
</pre><pre>T 6W08_A          316 VILEQVWYQSAYNEWQDILEVASTLNNAQPQITK  349 (359)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHTCCCEEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhhCCCCcce
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>3R9V_B Invasin ipaD; type III secretion system, tip protein, deoxycholate, CELL INVASION; HET: GOL, DXC; 1.9A {Shigella flexneri} SCOP: a.250.1.1
</pre><pre>Probab=44.37  E-value=3e+02  Score=28.09  Aligned_cols=84  Identities=11%  Similarity=0.177  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCC----HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    7 NAKQKVNGALAELN----TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         7 ~~A~~v~~al~~L~----~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      .+++++.++|...+    ++.++++......-+        .-+.....-..++++|.+.+..|..-|...        .
</pre><pre>T Consensus        54 ~~~~qll~~L~~s~~~~~~~~r~~l~~~~~~~~--------~~~~~~~S~~ei~~~I~~~I~~i~~~YLdv--------Y  117 (286)
</pre><pre>T 3R9V_B           54 NKSAQLLDILSRNEYPINKDARELLHSAPKEAE--------LDGDQMISHRELWAKIANSINDINEQYLKV--------Y  117 (286)
</pre><pre>T ss_dssp             -------------------------------------------CCCEEEHHHHHHHHHHHHHHHCCCCCHH--------H
</pre><pre>T ss_pred             HHHHHHHHHHHhCCCCCCHHHHHHHhcCCchhh--------cCCCCCCCHHHHHHHHHHHHHHHhHHHHHH--------H
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKLSRL  106 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~~~~  106 (262)
</pre><pre>                      .+.+++...+|+.+...+.++..+
</pre><pre>T Consensus       118 ~~iv~kYt~fy~d~s~~ls~ls~~  141 (286)
</pre><pre>T 3R9V_B          118 EHAVSSYTQMYQDFSAVLSSLAGW  141 (286)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHCC---
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>PF01601.18 ; CoV_S2 ; Coronavirus spike glycoprotein S2
</pre><pre>Probab=44.03  E-value=8.9e+02  Score=27.60  Aligned_cols=89  Identities=10%  Similarity=0.127  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      |..-|++..+.+.++..+    -...+..-|+-+.+.+..+|.+..+++.++.+--+.|+.--+.|..++++.-..    
</pre><pre>T Consensus       211 q~~vL~~Nqk~LA~sFN~----ai~~I~~a~~~v~~~~~~~~~gf~tva~AL~kiQ~vVN~n~~aL~~L~~qL~~n----  282 (563)
</pre><pre>T A0A125R5A5_9AL  211 QTDVLQENQKILANAFNN----AIGNITLALGKVSNAITTISDGFNTMSSALTKIQSVVNQQGEALSQLTSQLQKN----  282 (563)
</pre><pre>T ss_pred             chhHHHHHHHHHHHHHHH----HHHHHHHHhhhhhhhhHHhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                       +...-.-=+.++.|||.
</pre><pre>T Consensus       283 -F~AISssi~dI~~rLd~  299 (563)
</pre><pre>T A0A125R5A5_9AL  283 -FQAISSSIAEIYNRLEK  299 (563)
</pre><pre>T ss_pred             -HHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>3H6P_B ESAT-6 LIKE PROTEIN ESXS; Four-helix bundle, Structural Genomics, PSI-2, Protein Structure Initiative, TB Structural Genomics Consortium, TBSGC, Integrated Center for; 1.91A {Mycobacterium tuberculosis}
</pre><pre>Probab=43.37  E-value=3.4e+02  Score=22.68  Aligned_cols=71  Identities=7%  Similarity=-0.066  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      ..|...+..+......+...-.+.-.+--.+....-+.++......-.-|.+...++.+.|.+|.+.-...
</pre><pre>T Consensus        23 ~~l~~~a~~i~~~~~~i~~~l~~l~~~v~~l~~~W~G~Aa~Af~~~~~~W~~~~~~l~~~L~~i~~~l~~a   93 (111)
</pre><pre>T 3H6P_B           23 PQLIASHTAFAAKAGLMRHTIGQAEQQAMSAQAFHQGESAAAFQGAHARFVAAAAKVNTLLDIAQANLGEA   93 (111)
</pre><pre>T ss_dssp             -----CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHGGG
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>PF10303.11 ; DUF2408 ; Protein of unknown function (DUF2408)
</pre><pre>Probab=43.30  E-value=3.1e+02  Score=24.57  Aligned_cols=115  Identities=15%  Similarity=0.254  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             HHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHH---------HHHHHHHHHHHHHHHHHHHH--HHHh-cCCCccH
</pre><pre>Q WP_048874398.1   16 LAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAG---------ERYFSEIEKKLKAIELAYQN--QYRT-QGTLIGQ   83 (262)
</pre><pre>Q Consensus        16 l~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~---------~~~~~~i~~~L~~ie~ly~~--~~~~-~g~l~~~   83 (262)
</pre><pre>                      ++.++|+-...+.+=..+-..+.+.....--...+..         ..-+..+++.|.+|+....+  .|.. .++...+
</pre><pre>T Consensus         1 l~~i~~~L~pi~~kLv~Irr~L~~l~~~~~~~~~~~~~~~~~~~~~~~dl~~lq~~L~~Id~~r~d~GkF~~~~g~~~~~   80 (141)
</pre><pre>T G8ZZV2_TORDC/2    1 VEEIDPHLTTIMEKLFEIRRGLLSLAASSKRSNPSLEKSPLSQEHVHEQVEKLESQLTELESHRDENGKFKSLETDKVED   80 (141)
</pre><pre>T ss_pred             CccCCHHHHHHHHHHHHHHHHHHHHHhcCCCCCcccCCCCCCccccHHHHHHHHHHHHHHHHcccccCCccCCCCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhh---cCCCCh--------HHHHHHhcCcchhhhhhhh
</pre><pre>Q WP_048874398.1   84 QFFVERQKLFSELDVLLNKLSRLT---LKLKPY--------EEIKHALGLSSRSIVHEWS  132 (262)
</pre><pre>Q Consensus        84 ~F~~~R~~Lf~~Ld~~L~~~~~~~---~~l~~~--------~~lK~~LGlStkS~vh~w~  132 (262)
</pre><pre>                      .-.+.=..|+.+--..+..+....   -.+++.        ..||+.|  ..-.+.|.|+
</pre><pre>T Consensus        81 ~GQa~l~~LL~~C~~ll~~l~~~~~~~~~v~~~L~piy~~L~~ik~~L--e~L~~t~rWt  138 (141)
</pre><pre>T G8ZZV2_TORDC/2   81 KGQNVLNGLLDDCHDLISDLSHQKNGGIELDAKLQPIYEQLIDIKTSL--ENLLVTRRWT  138 (141)
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHhccCCCCCCCHHhHHHHHHHHHHHHHH--HHHhcccccc
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>d2np3a1 a.4.1.9 (A:35-99) Putative transcriptional regulator SCO0857 {Streptomyces coelicolor [TaxId: 1902]}
</pre><pre>Probab=42.82  E-value=1e+02  Score=22.20  Aligned_cols=38  Identities=13%  Similarity=0.168  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   96 LDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        96 Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      ++..+.-+...++.-.....+-+.+|+|..++-|+|..
</pre><pre>T Consensus         2 ~~a~~~l~~~~g~~~~t~~~ia~~~gvs~~~~~~~f~~   39 (65)
</pre><pre>T d2np3a1           2 LTAARVCFAERGFDATSLRRIAETAGVDQSLVHHFYGT   39 (65)
</pre><pre>T ss_dssp             HHHHHHHC---------------------------CCC
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHHCCCHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>PF11101.10 ; DUF2884 ; Protein of unknown function (DUF2884)
</pre><pre>Probab=42.60  E-value=5.5e+02  Score=24.77  Aligned_cols=125  Identities=13%  Similarity=0.196  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             cCCHHHHHHHHHhHHHHHHH----hhhchhhhHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHHhcC--CCccH
</pre><pre>Q WP_048874398.1   18 ELNTDEANFFNKHFSTIAAL----TNFMDKSLGVVADAGERYF--------SEIEKKLKAIELAYQNQYRTQG--TLIGQ   83 (262)
</pre><pre>Q Consensus        18 ~L~~~ea~fl~~n~~~l~~~----l~~~s~~~G~~s~~~~~~~--------~~i~~~L~~ie~ly~~~~~~~g--~l~~~   83 (262)
</pre><pre>                      .|++++.+.+.+++.-+..+    ...+-.++..+..++...+        .++++.+.++..--.+.+...+  .+.+.
</pre><pre>T Consensus        42 ~L~~~Qq~~l~~Y~~~ir~~vP~~~~ia~e~~~la~~a~~~v~~~l~~~~~~~l~~~~~~l~~~~~~~~~~~~~~~~~~~  121 (227)
</pre><pre>T Q3IDE0_PSET1/3   42 ALNHSQQKALTQYSDSLRTQLPEVANIALEGVKIAGVALQEVANAFNIDGLDNISSLMDDIKLEVQNTFYQQDTFVMDQQ  121 (227)
</pre><pre>T ss_pred             cCCHHHHHHHHHHHHHHHhhhHHHHHHHHHHHHHHHHHHHHHHHhcChhHHHHHHHHHHHHHHHHHHhheeCCeEEEChH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHH
</pre><pre>Q WP_048874398.1   84 QFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASK  151 (262)
</pre><pre>Q Consensus        84 ~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak  151 (262)
</pre><pre>                      .|-.-...+=..++..+..+..        ..+++.+|-=-.++....-..+ ++.+.+.++|+++.+
</pre><pre>T Consensus       122 ~~~~~~~~~~~e~e~~ie~~v~--------~~v~~~~g~i~~~lg~~~~~g~-~~~~~~~~rme~~~~  180 (227)
</pre><pre>T Q3IDE0_PSET1/3  122 SFNQFGENFEQQFEQQIETAVE--------SAVMQSMGSILVALGSELLGSG-GDMQAFEQRMENMGT  180 (227)
</pre><pre>T ss_pred             HhhhhhhhhHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHhcCC-CCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>d1sgma1 a.4.1.9 (A:5-77) Putative transcriptional regulator YxaF {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=42.32  E-value=1.2e+02  Score=23.09  Aligned_cols=44  Identities=14%  Similarity=0.130  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   87 VERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        87 ~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      ..|++++   +....-+...+..-.....|-+.+|+|..++-|++..
</pre><pre>T Consensus         2 ~~~~~i~---~~a~~l~~~~g~~~~t~~~ia~~~gvs~~t~y~~f~~   45 (73)
</pre><pre>T d1sgma1           2 DSREKIL---HTASRLSQLQGYHATGLNQIVKESGAPKGSLYHFFPN   45 (73)
</pre><pre>T ss_dssp             CHHHHHH---HHHHHHHHHHCTTTCCHHHHHHHHCCCSCHHHHSTTT
</pre><pre>T ss_pred             cHHHHHH---HHHHHHHHHHChhcCCHHHHHHHHCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>PF10439.11 ; Bacteriocin_IIc ; Bacteriocin class II with double-glycine leader peptide
</pre><pre>Probab=42.27  E-value=51  Score=26.64  Aligned_cols=62  Identities=24%  Similarity=0.366  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             hHHccceeeeecccchhc--------hhHHHHHHHHHHHHHHHHH-HHHhcccCcchHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  184 EQSCSKVAVKKYSQFGAS--------LAGGIYGGKYGAMAGAGVC-VALGVATGGVGLLACSVVGAAAGGY  245 (262)
</pre><pre>Q Consensus       184 ~~~C~k~~~~E~g~~~G~--------~~Gg~~gg~~g~~~a~~vc-~~~G~~t~G~g~l~c~vv~~g~g~~  245 (262)
</pre><pre>                      .+..+.....|-..+.|+        ...+..++..+...+.+.. ..++.+.+.++.++|+++++..|++
</pre><pre>T Consensus        15 ~~~f~~L~~~eL~~I~GG~~~~~~~~~~~~~~~~~~~ga~~Ga~~G~~~~~~~g~~G~~~Ga~~Ga~~G~~   85 (86)
</pre><pre>T B7GL22_ANOFW/1   15 NGSISFLSEEELKEIDGGRGSWTNAVIGAGTLSPIVASAVRGAQQGVRFGRLGGPWGVVAGAVVGAVVGGY   85 (86)
</pre><pre>T ss_pred             hhccccCCHHHHhhCcCccCccccccccccccchHHHHHHHHHHHHHhhccccchHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>6GSM_p Eukaryotic translation initiation factor 3 subunit B; ribosome, translation, initiation factors, 40S, eIF1A, eIF3, eIF2, tRNAi, 48S PIC, small ribosome subunit; HET: T6A, RIA, 2MG, 1MG, GCP, 5MC, M2G, 7MG, 1MA, H2U, MG; 5.15A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=42.20  E-value=1.7e+02  Score=31.73  Aligned_cols=54  Identities=9%  Similarity=0.172  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRL  106 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~  106 (262)
</pre><pre>                      +...++|.+.|+++.+-|.++-.........++.++|++++++...-..+....
</pre><pre>T Consensus       594 ~~~~~~i~~~~~~~~~~f~~~d~~~~~~~~~~~~~~r~~~~~~~~~~~~~~~~~  647 (661)
</pre><pre>T 6GSM_p          594 NAERKKVRKNLAEWSAQFEEQDAMEADTAMRDLILHQRELLKQWTEYREKIGQE  647 (661)
</pre><pre>T ss_dssp             TTCSSSSSSSTTTCC---CCSCSSSSSTTCC----CHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>3FX7_A Putative uncharacterized protein; double helix, UNKNOWN FUNCTION; 1.65A {Helicobacter pylori} SCOP: l.1.1.1, a.25.5.1
</pre><pre>Probab=42.15  E-value=3e+02  Score=21.62  Aligned_cols=80  Identities=9%  Similarity=0.114  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      ..+|+..+.++...-.++.+.        ..-|...+..-+...|.++..+.+.++++.+.++++.....+         
</pre><pre>T Consensus         8 ~~~l~~~a~~l~~~~~~i~~~--------~~~l~~~~~~l~~W~G~a~~~f~~~~~~~~~~~~~~~~~l~~---------   70 (94)
</pre><pre>T 3FX7_A            8 TEEVREFVGHLERFKELLREE--------VNSLSNHFHNLESWRDARRDKFSEVLDNLKSTFNEFDEAAQE---------   70 (94)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH--------HHHHHHHHHHCCSCCSHHHHHHHHHHHHHHHHHHHHHHHHHH---------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH--------HHHHHHHHHcccccccHHHHHHHHHHHHHHHHHHHHHHHHHH---------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      ...+..+...-|.+.|.
</pre><pre>T Consensus        71 ~~~~L~~~a~~~~~~~~   87 (94)
</pre><pre>T 3FX7_A           71 QIAWLKERIRVLEEDYL   87 (94)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>PF07964.13 ; Red1 ; Rec10 / Red1
</pre><pre>Probab=42.13  E-value=3.4e+02  Score=31.21  Aligned_cols=70  Identities=23%  Similarity=0.306  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHH----HHHHHhhcCCCChHHHHHHhcC
</pre><pre>Q WP_048874398.1   47 VVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLL----NKLSRLTLKLKPYEEIKHALGL  122 (262)
</pre><pre>Q Consensus        47 ~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L----~~~~~~~~~l~~~~~lK~~LGl  122 (262)
</pre><pre>                      ..+.+-......|.-+-+||+.--.+++.           ++++.||++|+..+    +.|..+      +..+|++|.|
</pre><pre>T Consensus       623 SI~~FSnel~rKi~IIN~ElNnKI~~ELS-----------eKYq~Lf~eLQ~sFqnDt~~M~~F------vgeIk~mL~L  685 (699)
</pre><pre>T RED1_YEAST/124  623 SINHFSNELVRKISIINQELNKKILKELS-----------EKYQKLFSELQDNFQNDTNEMLKF------MGEIKDMMNL  685 (699)
</pre><pre>T ss_pred             chhhhcHHHHHHHHHHHHHHHHHHHHHHH-----------HHHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhhhhhh
</pre><pre>Q WP_048874398.1  123 SSRSIVHEWST  133 (262)
</pre><pre>Q Consensus       123 StkS~vh~w~~  133 (262)
</pre><pre>                      ....|||.-+.
</pre><pre>T Consensus       686 pEdeLv~~IR~  696 (699)
</pre><pre>T RED1_YEAST/124  686 PEDQLVHAIRT  696 (699)
</pre><pre>T ss_pred             CHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>d2pv4a1 a.286.1.1 (A:1-144) Uncharacterized protein Sama2622 {Shewanella amazonensis [TaxId: 60478]}
</pre><pre>Probab=41.95  E-value=5e+02  Score=24.10  Aligned_cols=94  Identities=9%  Similarity=0.060  Sum_probs=0.0  Template_Neff=2.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccH
</pre><pre>Q WP_048874398.1    4 QLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQ   83 (262)
</pre><pre>Q Consensus         4 ~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~   83 (262)
</pre><pre>                      .|.+.-..+.++-..|+. .|+-+.+|| -++.|                    -|.++|-++-+.| +.+..-.....+
</pre><pre>T Consensus        44 ll~dse~~~r~aWd~LPa-SAqnvld~f-qFHgf--------------------yianA~v~lSq~~-~~i~E~~~~d~~  100 (144)
</pre><pre>T d2pv4a1          44 LLADRDGRFTRAWDQLPD-SASSLFERC-VFHGF--------------------YLANAWIQLSIVA-RDISELQDTDEA  100 (144)
</pre><pre>T ss_dssp             HHHTTTCHHHHHHHHSCH-HHHTSSCHH-HHHHH--------------------HHHHHHHHHHHHC-CCHHHHHHTTTT
</pre><pre>T ss_pred             HHhccchHHHHHHHhCCH-HHHHHHhhh-hhchH--------------------HHHHHHHHHHHHH-HHHHHHhcCcHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHH-HHHHHHHHHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   84 QFFVERQKLFSE-LDVLLNKLSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        84 ~F~~~R~~Lf~~-Ld~~L~~~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                      ..-.+|..+|.| |++.+.-+.|+--+-+..+.||+.+
</pre><pre>T Consensus       101 ~~e~~~a~il~rvl~~~lKd~vKqLKKAR~d~~lkn~f  138 (144)
</pre><pre>T d2pv4a1         101 IAEQEYSGLYVRVAEAALKESVKKLKKARTDRSMYNSM  138 (144)
</pre><pre>T ss_dssp             SCHHHHHHHHHHHHHTHHHHHHHHHHHHTTCHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHccCHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>5XLN_B Threonine--tRNA ligase, cytoplasmic; TRS, 4EHP, Complex, RNA BINDING PROTEIN-LIGASE complex; 1.9A {Homo sapiens}
</pre><pre>Probab=41.61  E-value=64  Score=22.89  Aligned_cols=17  Identities=12%  Similarity=0.366  Sum_probs=0.0  Template_Neff=6.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVL   99 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~   99 (262)
</pre><pre>                      |+|..+|-.||.+|...
</pre><pre>T Consensus        22 P~fi~~Rl~lfe~lka~   38 (45)
</pre><pre>T 5XLN_B           22 PEYIYTRLEMYNILKAE   38 (45)
</pre><pre>T ss_dssp             CSCCCCHHHHHHHHHHH
</pre><pre>T ss_pred             chHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>6J19_B ESAT-6-like protein EsxB; ATPase, MOTOR PROTEIN; HET: ATP; 1.978A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)}
</pre><pre>Probab=41.30  E-value=2.6e+02  Score=20.68  Aligned_cols=90  Identities=10%  Similarity=0.030  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|...++.+.+....+...-.++-...-.+...+-.............|...++.+...|.++.......       
</pre><pre>T Consensus        11 ~~~~l~~~a~~~~~~~~~~~~~~~~~~~~~~~~~~~w~g~~~~a~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   83 (104)
</pre><pre>T 6J19_B           11 DAATLAQEAGNFERISGDLKTQIDQVESTAGSLQGQWRGAAGTAAQAAVVRFQEAANKQKQELDEISTNIRQA-------   83 (104)
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhhccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                       ...|...-+..-..+..
</pre><pre>T Consensus        84 -~~~~~~~d~~~~~~~~~  100 (104)
</pre><pre>T 6J19_B           84 -GVQYSRADEEQQQALSS  100 (104)
</pre><pre>T ss_pred             -HHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>5YFP_D Exocyst complex component SEC8; exocyst, coiled-coil, EXOCYTOSIS; 4.4A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=41.26  E-value=5.4e+02  Score=30.81  Aligned_cols=144  Identities=9%  Similarity=0.034  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH---hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL---TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGT   79 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~---l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~   79 (262)
</pre><pre>                      ..+...-+++++.|.....+-...+.+.|.-+...   +..+|.+.+---.-+.+..+++++.|+++-.-|.+.      
</pre><pre>T Consensus        26 ~~~~~~~~~l~~~l~~I~~~W~~~~~~dfnPl~~AL~lLD~sS~G~~~r~~ef~~l~~~l~~~L~~iV~eh~~~------   99 (1065)
</pre><pre>T 5YFP_D           26 SAMNSSLDHLQNDLNRINLQWNRILSDNTNPLELALAFLDDTSVGLGHRYEEFNQLKSQIGSHLQDVVNEHSQV------   99 (1065)
</pre><pre>T ss_dssp             CCCSCSSSHHHHHHTGGGGGTTHHHHCCCSCSCSTTTTSCSSSSCCSSHHHHHHCCCCCTTSTTTTTTTTSTTT------
</pre><pre>T ss_pred             HHhhHHHHHHHHHHHHHHHHHHHHhCCCCccHHHHHHhcCCCCCCccccHHHHHHHHHHHHHHHHHHHHHHHHH------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc-ccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG-VASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus        80 l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g-~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                        +..=......+...+...=+++-.-.-.|   .+-|+.|.-....+..-|.+.- ..++=.+-+.|+.+-++-..+.
</pre><pre>T Consensus       100 --Fn~sI~sf~~i~~~i~~Sq~~v~~lk~~L---~~~k~~L~~~r~~L~~Lw~~s~~~k~mi~iLd~Ie~lk~vP~~ie  173 (1065)
</pre><pre>T 5YFP_D          100 --FNTNVASYGKAVSSIMQAQEQTLNLKNCL---KEANEKITTDKGSLQELNDNNLKYTKMIDVLVNIEELLQIPEKIE  173 (1065)
</pre><pre>T ss_dssp             --TTTTTHHHHHHHHHHHHCCCCCCCCHHHH---HHHHSSSCSSCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             --HHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHHHHHhcCChHHHHHHHHHHHHHHHHHHHHHHHHHHHhcHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>PF11887.10 ; Mce4_CUP1 ; Cholesterol uptake porter CUP1 of Mce4, putative
</pre><pre>Probab=40.85  E-value=6.5e+02  Score=25.12  Aligned_cols=94  Identities=9%  Similarity=-0.113  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      ...+.+.+..+.+.+..+ ..-...+.++.+-++.++.............+..+-+.+...+.+++..-+.         
</pre><pre>T Consensus        89 ~~~l~~~~~dL~~~l~nl-~~t~~tl~~~~~~L~~ll~~~~~~~~~l~~ll~~~~~~L~~~l~~l~~~~~~---------  158 (268)
</pre><pre>T Q79FZ9_MYCTU/1   89 GDVYADAAPDLFDFLDSS-VTTARTINAQQAELDSALLAAAGFGNTTADVFDRGGPYLQRGVADLVPTATL---------  158 (268)
</pre><pre>T ss_pred             HHHHHHHhHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhchhhHHHHHHHHHHHHHH---------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELDVLLNKLSR  105 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld~~L~~~~~  105 (262)
</pre><pre>                      -.++..+-..+|+.++..+..+.+
</pre><pre>T Consensus       159 L~~~s~~l~~~l~~l~~~~~~~~~  182 (268)
</pre><pre>T Q79FZ9_MYCTU/1  159 LDTYSPELFCTIRNFYDADPLAKA  182 (268)
</pre><pre>T ss_pred             HHHhCHHHHHHHHhHHhhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>5YFP_E Exocyst complex component SEC10; exocyst, coiled-coil, EXOCYTOSIS; 4.4A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=40.52  E-value=1e+03  Score=27.35  Aligned_cols=139  Identities=11%  Similarity=-0.021  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH---hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL---TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG   78 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~---l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g   78 (262)
</pre><pre>                      +.+|+...+++++.+..|..+-.+...+|..-+..+   +......+...-..+.+.-..+..+-++|+++....-+-..
</pre><pre>T Consensus        81 l~~L~~l~~~v~~~~~~l~~~~~~~~~~~~~~l~~l~~~~~~~~~~~~~L~~~i~~v~~~~~~i~~~L~~ld~~k~r~~~  160 (871)
</pre><pre>T 5YFP_E           81 LKELKNLNEETLNKRQYFSEQVATQEVIHSENVIKLSKDLHTTLLTFDKLDDRLTNVTQVVSPLGDKLETAIKKKQNYIQ  160 (871)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCCCCHHHCCCCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTHHHHHHHTTCCTTHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHhhhhHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHH--HHHHHHHHH-----HHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccc---cchHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQK--LFSELDVLL-----NKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVAS---IRGYSTYVDN  148 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~--Lf~~Ld~~L-----~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~---Ip~y~~~i~~  148 (262)
</pre><pre>                      ...--..+..-..  .+.+|+..+     ..+..-..-+..-..|.+.+                 .   +|++.+..+.
</pre><pre>T Consensus       161 a~~ll~~~~~~~~~g~~~~L~~l~~~~~~~~~~~aA~~l~~L~~l~~~l-----------------~~~~~~~~~~~~~~  223 (871)
</pre><pre>T 5YFP_E          161 SVELIRRYNDFYSMGKSDIVEQLRLSKNWKLNLKSVKLMKNLLILSSKL-----------------ETSSIPKTINTKLV  223 (871)
</pre><pre>T ss_dssp             HHHHHHHHHCCCCCCCCCCSTTTTTCSSSSCCTTTTTTHHHHTTTSCCS-----------------SSSSCTHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHhcCCchhHHHHhhcCChhhHHHHHHHHHHHHHHhhhc-----------------CcCCCCchHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHH
</pre><pre>Q WP_048874398.1  149 ASKAARFLK  157 (262)
</pre><pre>Q Consensus       149 ~ak~~k~lk  157 (262)
</pre><pre>                      +.+..+.|.
</pre><pre>T Consensus       224 i~~~~~~le  232 (871)
</pre><pre>T 5YFP_E          224 IEKYSEMME  232 (871)
</pre><pre>T ss_dssp             HHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>6W08_A Motility Associated Killing Factor E; Tripartite pore forming toxin, Cytotoxin, Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID, TOXIN; HET: EDO; 1.75A {Vibrio cholerae O1 biovar El Tor str. N16961}
</pre><pre>Probab=40.49  E-value=6.6e+02  Score=25.11  Aligned_cols=204  Identities=12%  Similarity=0.047  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHH-HHcCC-HHHHHHHHHhHHHHHHH----hhhchhhhH-HHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGA-LAELN-TDEANFFNKHFSTIAAL----TNFMDKSLG-VVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~a-l~~L~-~~ea~fl~~n~~~l~~~----l~~~s~~~G-~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      .+..++..++.+... +.+++ |+....|.++|..+..-    .......+- .....+..+-+.+......|....++.
</pre><pre>T Consensus        19 ~~~~~~~ya~~i~~~~~~~~~~~~~~~~l~~~~~~~~~~a~~W~~~~~p~~~~~l~~~i~~~a~~~~~~~~~l~~~~~~~   98 (359)
</pre><pre>T 6W08_A           19 AMVLLNSACQGVVETYIDQGNAEHWYAQVEQNLNAVQKLVRQWRLSGNLYFSNDIMDSVLSIANTFKDSNVQILTLFKAL   98 (359)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTCEECCTTCCTHHHHHHHHHHHHHHHHHHHHHTTCSSSCHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHhHHhhccCCCCCCchHHHHHHHHHHHHHHHHHHHHHcCcccchHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   74 YRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAA  153 (262)
</pre><pre>Q Consensus        74 ~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~  153 (262)
</pre><pre>                      -....+...+.|...-..|-++++....+.-.-...|   .+.++.+.=....+-..-+... ..+.+..+.|+.+.+-.
</pre><pre>T Consensus        99 ~~~~d~~~~~~~~~~L~~L~~~~~~~~~~~~~~~~~l---~~f~~~l~~~~~~l~~~~~~~~-~~~~~~~~~i~~L~~~i  174 (359)
</pre><pre>T 6W08_A           99 ETRFDTAQLQQLTSLILTLQNPIQSLTSNIKRYDEGL---NAWARQVEDAHNTLQQTIAQIQ-QEEVSIQAEIIATNAQI  174 (359)
</pre><pre>T ss_dssp             HHSCCHHHHHHHHHHHHTTHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHHHH-HHCGGGHHHHHHHHHHH
</pre><pre>T ss_pred             HhCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHH
</pre><pre>Q WP_048874398.1  154 RFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLL  233 (262)
</pre><pre>Q Consensus       154 k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l  233 (262)
</pre><pre>                      +.|+               .+|.+.             ..+..........|.++..++        +++...|+|++.+
</pre><pre>T Consensus       175 ~~L~---------------~~i~~~-------------~~~i~~~~~~~~~g~~~~~~g--------~~~~p~t~g~~~i  218 (359)
</pre><pre>T 6W08_A          175 DLMK---------------QQIAAF-------------KTAIANAQSQRKKGIFETIFG--------VVLAPFTLGGSLI  218 (359)
</pre><pre>T ss_dssp             HHHH---------------HHHHHH-------------HHHHHHCC--CSEEEEEEEEE--------EEEEECTTSCEEE
</pre><pre>T ss_pred             HHHH---------------HHHHHH-------------HHHHHHHHhhhHHHHHHHHHH--------HHHHhhccchHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHH
</pre><pre>Q WP_048874398.1  234 ACSVVGAAAGG  244 (262)
</pre><pre>Q Consensus       234 ~c~vv~~g~g~  244 (262)
</pre><pre>                      +.++.++++.+
</pre><pre>T Consensus       219 ~~~~g~~~i~~  229 (359)
</pre><pre>T 6W08_A          219 LAGFGVSSIVE  229 (359)
</pre><pre>T ss_dssp             EEEEEEEEEEC
</pre><pre>T ss_pred             HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>TIGR02950 SigM_subfam; RNA polymerase sigma factor, SigM family. This family of RNA polymerase sigma factors is a member of the Sigma-70 subfamily (TIGR02937) and is restricted to certain lineages of the order Bacillales.
</pre><pre>Probab=39.80  E-value=2.4e+02  Score=22.91  Aligned_cols=61  Identities=10%  Similarity=0.107  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    4 QLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         4 ~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      ...+....+...+..|++.+.+.+..+|  ++.. ...-+...|.+...+.++++++.+.|++.
</pre><pre>T Consensus        91 ~~~~~~~~l~~~~~~l~~~~~~il~~~~--~~g~~~~eia~~lgis~~~v~~~~~~~~~~l~~~  152 (154)
</pre><pre>T TIGR02950        91 LIKIEQEEITHHLSRLPENYRTVLILRE--FKEFSYKEIAELLNLSLAKVKSNLFRARKELKKL  152 (154)
</pre><pre>T ss_pred             HHHHHHHHHHHHHhcCCHHHHHHHHHHH--hcCCCHHHHHHHhCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>TIGR03879 near_KaiC_dom; probable regulatory domain. This model describes a common domain shared by two different families of proteins, each of which occurs regularly next to its corresponding partner family, a probable regulatory with homology to KaiC.
</pre><pre>Probab=39.70  E-value=2.4e+02  Score=19.81  Aligned_cols=61  Identities=15%  Similarity=0.121  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc-ccccchHHHHHHH
</pre><pre>Q WP_048874398.1   87 VERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG-VASIRGYSTYVDN  148 (262)
</pre><pre>Q Consensus        87 ~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g-~g~Ip~y~~~i~~  148 (262)
</pre><pre>                      .....++..|.......+....+++. ..+.+.+++|..++-++++... .+-|..+...+..
</pre><pre>T Consensus        10 ~~~~~~~~~l~~~~~~~l~~~~~~~~-~~ia~~~~is~~~v~~~~~~~~~~~~i~~~~~~~~~   71 (73)
</pre><pre>T TIGR03879        10 AERLTWVDSLAEAAAALAREEAGKTA-SEIAEELGRTEQTVRNHLKGETKAGGLVKLAYEKLK   71 (73)
</pre><pre>T ss_pred             HHHHHHHHHhhHHHHHHHHHHcCCCH-HHHHHHHCCCHHHHHHHHHHHHhcCChhHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>3S84_A Apolipoprotein A-IV; four helix bundle, TRANSPORT PROTEIN; HET: SO4; 2.4A {Homo sapiens}
</pre><pre>Probab=39.57  E-value=5.5e+02  Score=23.90  Aligned_cols=97  Identities=10%  Similarity=0.112  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..|......+...|.+...+-...+....+-+..-+       ......+..+++.+...|......+...+.....--.
</pre><pre>T Consensus       172 ~~l~~~~~~~~~~l~~~~~~~~~~l~~~~~~l~~~l-------~~~~~~l~~~~~~~~~~l~~~~~~~~~~l~~~~~~l~  244 (273)
</pre><pre>T 3S84_A          172 ARISASAEELRQRLAPLAEDVRGNLRGNTEGLQKSL-------AELGGHLDQQVEEFRRRVEPYGENFNKALVQQMEQLR  244 (273)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHTSSSCHHHHHHHH-------HHHHHHHHHHHHHHHHHHTTHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhhcHHHHHHHH-------HHHHHHHHHHHHHHHHHhHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKLSRL  106 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~~~~  106 (262)
</pre><pre>                      ..|-.....+-.+|+..+..+..+
</pre><pre>T Consensus       245 ~~l~~~~~~~~~~l~~~~~~~~~~  268 (273)
</pre><pre>T 3S84_A          245 QKLGPHAGDVEGHLSFLEKDLRDK  268 (273)
</pre><pre>T ss_dssp             HHHSCC------------------
</pre><pre>T ss_pred             HHHhHhHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>PF10168.11 ; Nup88 ; Nuclear pore component
</pre><pre>Probab=39.45  E-value=9.9e+02  Score=26.82  Aligned_cols=128  Identities=15%  Similarity=0.128  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      ++.|.+..+...+..-+.-..-.+.+.++...+...+...-.-+........+..++..+...+++++-+++        
</pre><pre>T Consensus       529 L~~l~~~~~~lr~~~~~~~~~~~~~l~~R~~~q~~~~~~Q~~~l~~l~~~~~~l~~~~~~l~~r~e~~~~~q--------  600 (706)
</pre><pre>T H2YPL5_CIOSA/6  529 LQVVTRATQVLRDEYMLKQKLARNAITKRVNLLSDLKLNQLDELHRLGDMKEVLTDNAEKIANDYEDCKERQ--------  600 (706)
</pre><pre>T ss_pred             HHHHHHHHHHHHHhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH--------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHhhcCCCCh-HHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQKLFSELDVLLNKLSRLTLKLKPY-EEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~-~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                              ++|.+|++..+.++......+.+. .++++.|    +.+-..        ++.|..+++++.+-.+..+
</pre><pre>T Consensus       601 --------~~L~~R~~~ll~~l~~~~~~LS~aE~~~~~eL----~~~~~~--------l~~l~~~l~~l~~~~~~~~  657 (706)
</pre><pre>T H2YPL5_CIOSA/6  601 --------ESLMIRLEDVLKRTRDRDPVLSKSEVAMNKEL----RAMKEK--------LWQLDNRIKQLKIKQEYQA  657 (706)
</pre><pre>T ss_pred             --------HHHHHHHHHHHHHHHhcCCCCCHHHHHHHHHH----HHHHHH--------HHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>5JG7_A Fur regulated Salmonella iron transporter; structural genomics, CSGID, periplasmic binding protein, Center for Structural Genomics of Infectious Diseases, METAL TRANSPORT; HET: MSE, GOL; 1.7A {Salmonella typhimurium (strain LT2 / SGSC1412 / ATCC 700720)}
</pre><pre>Probab=39.36  E-value=1.5e+02  Score=25.90  Aligned_cols=35  Identities=14%  Similarity=0.227  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAA   36 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~   36 (262)
</pre><pre>                      ...++..++.+.+.|.++.|+.++++.+|++.+..
</pre><pre>T Consensus       107 p~~~~~~~~~i~~~L~~~~P~~~~~~~~N~~~~~~  141 (160)
</pre><pre>T 5JG7_A          107 AENALIYVDNIRDALVKYDPDNAQIYKQNAERYKA  141 (160)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHChhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>PF11172.10 ; DUF2959 ; Protein of unknown function (DUF2959)
</pre><pre>Probab=39.26  E-value=5.9e+02  Score=24.15  Aligned_cols=139  Identities=10%  Similarity=0.144  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTL   80 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l   80 (262)
</pre><pre>                      +...+++-+.+-..|..|...+..-+.+.|+.|..-+...-....    -+.+|++.+.+.=.++-.-|++++.. ..+-
</pre><pre>T Consensus        27 ~~~a~~q~~~al~~l~~l~~~~~~dl~~~y~~~~~~~~~~~~~a~----~v~~~i~~m~~~a~~~F~eWekel~~~~n~~  102 (190)
</pre><pre>T Q3JB95_NITOC/2   27 QIEAKEQFQSALEQFSSVVNYDGGDLEDLYEELNGEYQASQKKAE----AVHDRIASVENVAGALFQEWEEELDQYDNPS  102 (190)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHcCCCCCHHHHHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHHHHHcCCCHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCCh-HHHHHHhcCcchhhhhhhhhccccccchHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNKLSRLTLKLKPY-EEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDN  148 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~-~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~  148 (262)
</pre><pre>                      -+..=-.+|+.+-.+.+.....+-.-.-.++++ ..|++.-    .-+-|+-+..+.+.|.+....++.
</pre><pre>T Consensus       103 LR~~S~~rl~~t~~~y~~i~~~~~~a~~~~~p~l~~l~D~~----l~Lk~nLna~~i~al~~~~~~~~~  167 (190)
</pre><pre>T Q3JB95_NITOC/2  103 LRRSSQQQLTQTRTRYDKLIRAMKRAEAKIEPVLTTFHDQV----LFLKHNLNARAIASLQGELSTIET  167 (190)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHhhHHHHHHHHHHH----HHHhcCCCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>5XSJ_L Signal transduction histidine kinase, LytS; Two component system, histidine kinase, signal transmission across the membrane, D-xylose uptake, SUGAR BINDING PROTEIN; HET: XYP; 2.202A {Clostridium beijerinckii (strain ATCC 51743 / NCIMB 8052)}
</pre><pre>Probab=39.06  E-value=4.5e+02  Score=22.79  Aligned_cols=96  Identities=9%  Similarity=-0.002  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc
</pre><pre>Q WP_048874398.1   56 FSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG  135 (262)
</pre><pre>Q Consensus        56 ~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g  135 (262)
</pre><pre>                      +++|...++++.....+.+.....-....|...+..+-..++..-...  ..-..-....|++++    .++...+..+-
</pre><pre>T Consensus        22 ln~i~~~~~~~~~~l~~Yl~~~~~~~l~~y~~~~~~l~~~l~~l~~~~--~~~~~~~~~~i~nml----~sy~e~~~~ai   95 (148)
</pre><pre>T 5XSJ_L           22 TNEIKQHVDSSLDNFNQYILNGTPSKKESYNNEVILAKQKIGNLKKNS--DDVNQYILRDLDNTL----DSYIESSKNTI   95 (148)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHCCHHHHHHHHHHHHHHHHHHHHHHHTC--CTTHHHHHHHHHHHH----HHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhCCHhHHHHHHHHHHHHHHHHHHHhcCC--CCcchHHHHHHHHHH----HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ----ccc-cchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  136 ----VAS-IRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus       136 ----~g~-Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                          ... ++.|....+.+.+...+++
</pre><pre>T Consensus        96 ~~~~~~~~~~~y~~~y~e~~~i~~yI~  122 (148)
</pre><pre>T 5XSJ_L           96 SAYENKEGYVFYYDDFVAAKNIASYCD  122 (148)
</pre><pre>T ss_dssp             HHHHTTCCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHcCcchhHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>2DII_A TFIIH basal transcription factor complex p62 subunit; BTF2-p62, General transcription factor IIH polypeptide 1, Nuclear protein, Transcription regulation, structural genomics, NPPSFA, National Project; NMR {Homo sapiens} SCOP: a.240.1.1, l.1.1.1
</pre><pre>Probab=38.64  E-value=1.5e+02  Score=23.15  Aligned_cols=41  Identities=10%  Similarity=0.156  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   54 RYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSE   95 (262)
</pre><pre>Q Consensus        54 ~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~   95 (262)
</pre><pre>                      +.-..+-..=.++.++|++ +-..+.+..++|-+.|+.++.+
</pre><pre>T Consensus        17 e~r~~lL~~np~L~~Lyk~-LV~sgiise~EFW~~r~~l~~~   57 (61)
</pre><pre>T 2DII_A           17 EEKNRMLQEDPVLFQLYKD-LVVSQVISAEEFWANRLNVNSG   57 (61)
</pre><pre>T ss_dssp             HHHHHHHHHCHHHHHHHCC-CCCTTSSCHHHHHHHHCSSCCC
</pre><pre>T ss_pred             HHHHHHHHHCHHHHHHHHH-HHHcCCCCHHHHHHHHhHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>TIGR01636 phage_rinA; phage transcriptional activator, RinA family. This model represents a family of phage proteins, including RinA, a transcriptional activator in staphylococcal phage phi 11.
</pre><pre>Probab=38.32  E-value=1.3e+02  Score=24.53  Aligned_cols=31  Identities=13%  Similarity=0.322  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHF   31 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~   31 (262)
</pre><pre>                      ++..++...+.+..++..|++++...+...|
</pre><pre>T Consensus        65 ~~~~~~~~~~~i~~~~~~l~~~~~~ii~~~y   95 (134)
</pre><pre>T TIGR01636        65 RLWNLERNRDAIENCLNEADEQTRVIIQELY   95 (134)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>d1c02a_ a.24.10.2 (A:) Phosphorelay protein ypd1 {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=38.08  E-value=5.5e+02  Score=23.95  Aligned_cols=98  Identities=9%  Similarity=0.141  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   30 HFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQ-----NQYRTQGTLIGQQFFVERQKLFSELDVLLNKLS  104 (262)
</pre><pre>Q Consensus        30 n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~-----~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~  104 (262)
</pre><pre>                      +|+.|..++.............+...++++.+.+.+|..++.     ..+....+.-...+..-...-+.++=..+....
</pre><pre>T Consensus         9 ~~~~l~~~l~~~~g~~~~~~eLL~~Fiee~~e~~~~L~~al~~~~~~~~l~~~~HkLKGsA~~lGa~~L~~~~~~Ie~~~   88 (166)
</pre><pre>T d1c02a_           9 NWTILNEIISMDDDDSDFSKGLIIQFIDQAQTTFAQMQRQLDGEKNLTELDNLGHFLKGSSAALGLQRIAWVCERIQNLG   88 (166)
</pre><pre>T ss_dssp             CHHHHHHHHGGGGSSTTHHHHHHHHHHHHHHHHHHHHHHHHHTTCCHHHHHHHHHHHHHHHHHTTCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CHHHHHHHHhhcCCCHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHHcChHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             Hhhc-CCCChHHHHHHhcCcchhhhh
</pre><pre>Q WP_048874398.1  105 RLTL-KLKPYEEIKHALGLSSRSIVH  129 (262)
</pre><pre>Q Consensus       105 ~~~~-~l~~~~~lK~~LGlStkS~vh  129 (262)
</pre><pre>                      +... ..|....|-+.|  |.||++.
</pre><pre>T Consensus        89 ~~~~~~~~~~~~~~~~~--~~~~~~~  112 (166)
</pre><pre>T d1c02a_          89 RKMQHFFPNKTELVNTL--SDKSIIN  112 (166)
</pre><pre>T ss_dssp             TTSCCCCCCHHHHHTTC--SSGGGGT
</pre><pre>T ss_pred             HcCCCCCCCchhHhccc--cchhhcC
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>d3fava_ a.25.3.1 (A:) automated matches {Mycobacterium tuberculosis [TaxId: 83332]}
</pre><pre>Probab=37.94  E-value=2.9e+02  Score=20.31  Aligned_cols=74  Identities=8%  Similarity=-0.040  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>Q WP_048874398.1    5 LQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG   78 (262)
</pre><pre>Q Consensus         5 L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g   78 (262)
</pre><pre>                      |+..+.++.+...++...-...-..--.+....-+.++...-....-|.+.+.++.+.|.++.+.-........
</pre><pre>T Consensus         1 l~~~a~~~~~~~~~i~~~~~~l~~~~~~l~~~W~G~a~~af~~~~~~~~~~~~~~~~~L~~~~~~l~~~~~~Y~   74 (74)
</pre><pre>T d3fava_           1 LAQEAGNFERISGDLKTQIDQVESTAGSLQGQWRGAAGTAAQAAVVRFQEAANKQKQELDEISTNIRQAGVQYS   74 (74)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHTTTTCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHC
</pre><pre>T ss_pred             ChHHHHHHHHHHHHHHHHHHHHHHHHHhcccccccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhcC
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>d1pq4a_ c.92.2.2 (A:) Periplasmic zinc binding protein ZnuA {Synechocystis sp. PCC 6803 [TaxId: 1148]}
</pre><pre>Probab=37.70  E-value=2.9e+02  Score=27.07  Aligned_cols=69  Identities=12%  Similarity=0.136  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|..+.|+.++++.+|++.+..-                         |++++..|++.+........
</pre><pre>T Consensus       139 ~~~~~~a~~I~~~L~~~~P~~~~~~~~n~~~~~~~-------------------------l~~l~~~~~~~~~~~~~~~~  193 (289)
</pre><pre>T d1pq4a_         139 TLVKRQATTIAKELAELDPDNRDQYEANLAAFLAE-------------------------LERLNQELGQILQPLPQRKF  193 (289)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHTTCSCCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHhccCCCCcE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.|...-.-|++++
</pre><pre>T Consensus       194 v~~h~~~~Yl~~~~  207 (289)
</pre><pre>T d1pq4a_         194 IVFHPSWAYFARDY  207 (289)
</pre><pre>T ss_dssp             EESSCCCHHHHHHT
</pre><pre>T ss_pred             EEEchhHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>2P7N_A Pathogenicity island 1 effector protein; cvr69, Pathogenicity island 1 effector protein, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium; HET: MSE; 2.8A {Chromobacterium violaceum}
</pre><pre>Probab=37.67  E-value=4e+02  Score=28.72  Aligned_cols=106  Identities=8%  Similarity=0.129  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHH------------HhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAA------------LTNFMDKSLGVVADAGERYFSEIEKKLKAIELAY   70 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~------------~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly   70 (262)
</pre><pre>                      .+|....+.+.+.-..+++++.+.+..-++-+..            .+.......... ..-...+++|.+.+..|..-|
</pre><pre>T Consensus       118 ~~l~~~l~~l~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-~S~~d~~d~I~~~I~~ik~~Y  196 (407)
</pre><pre>T 2P7N_A          118 RLFEGNLKGVAQSGHAMSAEQKQALQSGLDDVFADAPPQARSAGAPMLYSANAAAGQG-MADSDLWDMISDQIGKIKDNY  196 (407)
</pre><pre>T ss_dssp             HHHHHHHHHHHHSCCCCCHHHHHHHHHHHHHHHSSCCHHHHTTSHHHHHHHHHHHHTC-CCHHHHHHHHHHHHHHCCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHhhCCCCCHHHHHHHHHchHHHhhcCChhHHhcCCCccccchhhhcCC-CChHHHHHHHHHHHHHHhhhH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHH
</pre><pre>Q WP_048874398.1   71 QNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIK  117 (262)
</pre><pre>Q Consensus        71 ~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK  117 (262)
</pre><pre>                      ...        ..+...+....|+.+...+.+|..+-..=.+-+++|
</pre><pre>T Consensus       197 L~v--------Ye~iv~kYtdFyq~fn~ilskls~~I~~~~DGn~V~  235 (407)
</pre><pre>T 2P7N_A          197 LGV--------YENVVGQYTDFYKAFSDILSQMANWIKPGGDGNKVK  235 (407)
</pre><pre>T ss_dssp             CHH--------HHHHHHHHHHHHHHHHHHHHHHGGGCC---------
</pre><pre>T ss_pred             HHH--------HHHHHHHHHHHHHHHHHHHHHHHhhcccCCCCCeee
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>4RL5_B Exocyst complex component EXO70A1; exocyst complex, PROTEIN BINDING; HET: MSE; 3.1A {Arabidopsis thaliana}
</pre><pre>Probab=37.25  E-value=9.4e+02  Score=25.96  Aligned_cols=145  Identities=9%  Similarity=0.107  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIE-LAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie-~ly~~~~~~~g~l   80 (262)
</pre><pre>                      |..|...-+.-++.. +-+....-||++|+..|...+..+....-.......++-+.+++..+.+. ..+..-+..-...
</pre><pre>T Consensus       374 i~~L~~~Le~ks~~~-~~~~l~~iFllNN~~yi~~~l~~s~l~~~l~~~~~~~l~~~~~~~~~~y~~~~W~~~l~~l~~~  452 (565)
</pre><pre>T 4RL5_B          374 MQALQNNLDGKSKQY-KDPALTHLFLMNNIHYMVRSVRRSEAKDLLGDDWVQRHRRIVQQHANQYKRVAWTKILQSSSAQ  452 (565)
</pre><pre>T ss_dssp             HHHHHHHHHHTTTTS-SCHHHHHHHHHHHHHHHHHHHTTSCC--SSCSHHHHHHHHHHHHHHHHHHHHHHHHHHHTTSCC
</pre><pre>T ss_pred             HHHHHHHHHHhHHhc-cCHHHHHhhhhchHHHHHHHccchHHHHHhCHHHHHHHHHHHHHHHHHHHHHhhHHHHHhhccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHH--------------HHHHHHHHHHHHHHHHHHHH--hhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHH
</pre><pre>Q WP_048874398.1   81 IGQQF--------------FVERQKLFSELDVLLNKLSR--LTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYST  144 (262)
</pre><pre>Q Consensus        81 ~~~~F--------------~~~R~~Lf~~Ld~~L~~~~~--~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~  144 (262)
</pre><pre>                      ..+..              ...=+..|++.+..+..+.+  +...+|+ +.||+.|   .+.+.+.       -+|.|.+
</pre><pre>T Consensus       453 ~~~~~~~~~~~~~~~~~~~r~~ik~~f~~Fn~~f~~~~~~q~~~~v~D-~~Lr~~l---r~~i~~~-------v~p~Y~~  521 (565)
</pre><pre>T 4RL5_B          453 GLTSSGGGSLEGGNSSGVSRGLLKERFKMFNMQFDELHQRQSQWTVPD-TELRESL---RLAVAEV-------LLPAYRS  521 (565)
</pre><pre>T ss_dssp             ------------------CHHHHHHHHHHHHHHHHHHHHHSSCC-CCS-HHHHHHH---HHHHHHH-------HHHHHHH
</pre><pre>T ss_pred             CCCCCCCCCCCCCCCCccCHHHHHHHHHHHHHHHHHHHHhhccccCCC-HHHHHHH---HHHHHHh-------HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH-------HHHHHHH
</pre><pre>Q WP_048874398.1  145 YVDNASK-------AARFLKM  158 (262)
</pre><pre>Q Consensus       145 ~i~~~ak-------~~k~lk~  158 (262)
</pre><pre>                      -+++...       ..|++|+
</pre><pre>T Consensus       522 f~~~~~~~~~~~k~~~kyik~  542 (565)
</pre><pre>T 4RL5_B          522 FLKRFGPLVESGKNPQKYIKY  542 (565)
</pre><pre>T ss_dssp             HHTTCC---------------
</pre><pre>T ss_pred             HHHHHhhhhccCCChhhhccc
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>6FTJ_1 Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1,RPN1; Protein translocon of the endoplasmic reticulum, PROTEIN TRANSPORT; HET: BMA, MG, MAN, 9UB, NAG; 4.7A {Oryctolagus cuniculus}
</pre><pre>Probab=37.21  E-value=2.5e+02  Score=26.17  Aligned_cols=74  Identities=14%  Similarity=0.170  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc---ccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG---VASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g---~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                      .....+|..++++++..++++-+.. .......-++.+.-.-|.+-..++...   ..+=|.++++++.+-|.-+.+|
</pre><pre>T Consensus        46 ~~~~d~r~~ly~~~e~al~k~K~sk-D~~a~~aa~Kk~d~e~K~~t~~I~~l~~~Lk~~~~~~~eKv~Elqk~d~~~k  122 (162)
</pre><pre>T 6FTJ_1           46 LTLVNKRIGLYRHFDETINRYKQSR-DVSTLNSGKKSLETEHKALTSEIASLQSRLKTEGSDLCDKVSEMQKLDAQVK  122 (162)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHTTTS-CCHHHHHHHHHHHHHHHHHHHHHHHHHHHTSSSCCCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHhcC-CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhcCchHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>6JLU_Q Extrinsic protein in photosystem II; PSII-FCP supercomplex, PLANT PROTEIN; HET: BCR, OEX, KC1, HEM, KC2, SQD, CLA, DGD, PHO, LHG, DD6, A86, LMG, PL9; 3.02A {Chaetoceros gracilis}
</pre><pre>Probab=36.66  E-value=7.1e+02  Score=24.36  Aligned_cols=100  Identities=11%  Similarity=0.022  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ++++.+.....+.+.-+.|.+=+.-.=.++|+-+.+++......+-..-..+.+ ..+..+.++++-+...+.=......
</pre><pre>T Consensus       110 ~~~ei~~y~~~lk~s~~rL~~l~~lI~kk~W~~V~~~Lrgplg~LR~dL~~l~~-~~eakk~ak~lf~~Le~LD~AA~~k  188 (211)
</pre><pre>T 6JLU_Q          110 SAEYLARKKAVLAETKNRLQKIPGYVDKKEWFNVKDELTRYMYETRGAVRSLSS-SVTQKEKAEVFFRALEDTYGAATLK  188 (211)
</pre><pre>T ss_dssp             TSTTHHHHHHHHHHHHHHHHTTTHHHHTTCHHHHHHHTTTTTHHHHHHHHHHCC-SHHHHHHHHHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHhhcchhhchhcHHHHHHHHHHhHHHHHHHHHHHhC-CHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLN  101 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~  101 (262)
</pre><pre>                      +.++....+++..+.||.-|.
</pre><pre>T Consensus       189 d~~~A~~~y~~a~~~ld~~L~  209 (211)
</pre><pre>T 6JLU_Q          189 KGDAVKASNDKAIAALDAFTA  209 (211)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ChHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>d4j42a_ a.25.3.0 (A:) automated matches {Bacillus anthracis [TaxId: 260799]}
</pre><pre>Probab=36.55  E-value=3e+02  Score=19.97  Aligned_cols=72  Identities=8%  Similarity=0.006  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      ....|+..+..+.....+|...-..+-..-..+-...-+.+.........-|.+.+.++.+.|.++.+....
</pre><pre>T Consensus         3 ~~~~l~~~a~~l~~~~~~l~~~~~~l~~~~~~l~~~W~G~a~~af~~~~~~~~~~~~~~~~~l~~~~~~l~~   74 (80)
</pre><pre>T d4j42a_           3 TPEELERIAGNFKNAAGEAQSQINRLEGDINSLEGQWAGATQAKFRGEFIQSKQAMQQFIPILEGISTDLKR   74 (80)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTGGGCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhhcccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>TIGR01417 PTS_I_fam; phosphoenolpyruvate-protein phosphotransferase. This model recognizes a distinct clade of phophoenolpyruvate (PEP)-dependent enzymes.
</pre><pre>Probab=36.45  E-value=3.4e+02  Score=29.73  Aligned_cols=89  Identities=17%  Similarity=0.277  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCH--------HHHHHHHHhHHHHHHH------hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNT--------DEANFFNKHFSTIAAL------TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~--------~ea~fl~~n~~~l~~~------l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      +++.|.++-.++.+.|..|.+        ++++++.-|..++++-      ...-...--.+..+|.+.+++..+.+..+
</pre><pre>T Consensus        38 E~~rl~~Ai~~~~~~L~~l~~~~~~~~~~~~~~Il~ah~~lL~D~~l~~~i~~~I~~~~~~a~~Av~~v~~~~~~~~~~~  117 (565)
</pre><pre>T TIGR01417        38 EISRFLSARAKAKEDLETIKTKAGKTFGQEKAAIFEAHILILEDPELTEEVIELIKKDHKNAEFAAHEVFEGQAKSLEEM  117 (565)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhccCHHHHHHHHHHHHHHhCHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhcCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   67 ELAYQNQYRTQGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        67 e~ly~~~~~~~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ++-|-++       +..++..-.++++..|
</pre><pre>T Consensus       118 ~d~ylre-------R~~Di~di~~rll~~L  140 (565)
</pre><pre>T TIGR01417       118 DDEYLKE-------RAADIRDIGNRLLGHL  140 (565)
</pre><pre>T ss_pred             CCHHHHH-------HHHhHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>7C53_B Spike protein S2',pan-CoVs inhibitor EK1; SARS-CoV-2 HR1, pan-CoVs inhibitor, EK1 peptide, VIRUS, VIRAL PROTEIN; 2.278A {Severe acute respiratory syndrome coronavirus 2}
</pre><pre>Probab=36.35  E-value=1.1e+02  Score=26.61  Aligned_cols=67  Identities=21%  Similarity=0.234  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      |..++....+-.++|..|+.+    +.+||..+..- -......+..+.--+.+.++.+++..++|+.-|.+
</pre><pre>T Consensus        36 L~kiQ~VVN~q~~aL~~L~~q----L~~NFgAISs~g~g~~~~~iN~T~ldl~~Em~~L~evik~LN~S~Id  103 (107)
</pre><pre>T 7C53_B           36 LGKLQDVVNQNAQALNTLVKQ----LSSNFGAISSGGRGGSLDQINVTFLDLEYEMKKLEEAIKKLEESYID  103 (107)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH----HHTC------------CTTCCCCCCCCHHHHHHHHHHHHHHGGGCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH----HHhhcCCcccCCCCCChHHchhhhhhHHHHHHHHHHHHHHHHHhccc
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>7C53_C Spike protein S2',pan-CoVs inhibitor EK1; SARS-CoV-2 HR1, pan-CoVs inhibitor, EK1 peptide, VIRUS, VIRAL PROTEIN; 2.278A {Severe acute respiratory syndrome coronavirus 2}
</pre><pre>Probab=36.35  E-value=1.1e+02  Score=26.61  Aligned_cols=67  Identities=21%  Similarity=0.234  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      |..++....+-.++|..|+.+    +.+||..+..- -......+..+.--+.+.++.+++..++|+.-|.+
</pre><pre>T Consensus        36 L~kiQ~VVN~q~~aL~~L~~q----L~~NFgAISs~g~g~~~~~iN~T~ldl~~Em~~L~evik~LN~S~Id  103 (107)
</pre><pre>T 7C53_C           36 LGKLQDVVNQNAQALNTLVKQ----LSSNFGAISSGGRGGSLDQINVTFLDLEYEMKKLEEAIKKLEESYID  103 (107)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH----HHCC--------------CCCCCCCCCHHHHHHHHHHHHHHHTTCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH----HHhhcCCcccCCCCCChHHchhhhhhHHHHHHHHHHHHHHHHHhccc
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>4I0X_E ESAT-6-like protein MAB_3112; Structural Genomics, PSI-2, Protein Structure Initiative, Integrated Center for Structure and Function Innovation, ISFI, TB Structural; HET: GOL, BME; 1.956A {Mycobacterium abscessus}
</pre><pre>Probab=36.01  E-value=3.3e+02  Score=20.28  Aligned_cols=90  Identities=7%  Similarity=0.027  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|+..+..+......+...-.++...--.+...+-+.++...-.....|...+.++.+.|.++.......       
</pre><pre>T Consensus         1 d~~~l~~~a~~~~~~~~~~~~~~~~~~~~~~~~~~~W~G~a~~~~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   73 (94)
</pre><pre>T 4I0X_E            1 SIDEVGALSKFAASLADQMRAGSNSLDRDVQSLFGVWKGSAADAYRSGWDEMQDGATKVWNALTDIASTLGSN-------   73 (94)
</pre><pre>T ss_dssp             ---CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTTCCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHC----------
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccHHHHHhHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                       ...|.......-..+..
</pre><pre>T Consensus        74 -~~~~~~~d~~~~~~~~~   90 (94)
</pre><pre>T 4I0X_E           74 -AAAFHAQETSTASSITS   90 (94)
</pre><pre>T ss_dssp             ------------------
</pre><pre>T ss_pred             -HHHHHHHhhhhHHhhhh
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>5YFP_B Exocyst complex component SEC5; exocyst, coiled-coil, EXOCYTOSIS; 4.4A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=35.66  E-value=1.4e+03  Score=27.36  Aligned_cols=130  Identities=9%  Similarity=0.197  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHH------------HHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADA------------GERYFSEIEKKLKAIELA   69 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~------------~~~~~~~i~~~L~~ie~l   69 (262)
</pre><pre>                      +..|+...+.....+..++++-...+.+||+-+=..-+.-...-..-...            ....++.+.+.++++...
</pre><pre>T Consensus       126 ~~dL~~~l~~L~~~i~~~~~~lk~LV~~Nf~~FV~ak~tId~v~~~~~~~~~~~~~~~~~~~~~~~~~~L~~~l~~~~~~  205 (971)
</pre><pre>T 5YFP_B          126 FNDLNNSLDRLDSDIQDQSIHLKQLVGKNFTKYVKIKNKLDQIYKEFDEKTNEKNQCDSPKENQINVESLNKKVDEVIRT  205 (971)
</pre><pre>T ss_dssp             STCCCCCHHHHHHHHHHHHHHHHHHTTTTTHHHHHHHCCCCCHHHTTSTTSSSSSSCSCSSSTTCTTHHHHTTTTHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHccHHHHHHHHhhHHHHHHHHHHHHHHHHHHHhccCCcCCCCCchhhHHhHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHH
</pre><pre>Q WP_048874398.1   70 YQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNA  149 (262)
</pre><pre>Q Consensus        70 y~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~  149 (262)
</pre><pre>                      -..-+..         ..+|+.-..+|...++-+-+..+=+.-...|++.+                 .-..|..-++..
</pre><pre>T Consensus       206 a~~~~~p---------ll~~~~k~~~l~~~l~~l~r~k~lf~LP~~L~~~i-----------------~~~~yd~av~dY  259 (971)
</pre><pre>T 5YFP_B          206 TTFKLKP---------LMDNYQKILNYQATKKFIELNKFYFNLPKSLKRCL-----------------TNNDFNEFIIEY  259 (971)
</pre><pre>T ss_dssp             HHHHHTH---------HHHHHHHHHHHHHHHHHHTCSSSSSCHHHHHHHHT-----------------TSSSCSTTTTTH
</pre><pre>T ss_pred             HHHhHHH---------HhhcHHHHHHHHHHHHHHHHHHHHccCcHHHHHHH-----------------HCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHH
</pre><pre>Q WP_048874398.1  150 SKAARFLK  157 (262)
</pre><pre>Q Consensus       150 ak~~k~lk  157 (262)
</pre><pre>                      .|+.+.++
</pre><pre>T Consensus       260 ~kak~l~~  267 (971)
</pre><pre>T 5YFP_B          260 SKGLTLRR  267 (971)
</pre><pre>T ss_dssp             HHHHHHHT
</pre><pre>T ss_pred             HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>5MX5_G Proteasome activator complex subunit 1; proteasome activator, regulator, heptamer, interferon-gamma, hydrolase, structural protein; HET: PO4; 2.9A {Mus musculus}
</pre><pre>Probab=35.07  E-value=4.9e+02  Score=25.47  Aligned_cols=81  Identities=10%  Similarity=0.110  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVER   89 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R   89 (262)
</pre><pre>                      +.|.+.+..+.+ +...+.+..+.+...+...-.-+.-..++.-..-+++-+.|.+++..-...+..     ...||..|
</pre><pre>T Consensus       108 ~~v~~l~~~vk~-e~~~l~~~~~~lk~WI~L~iPkiedGnnfgv~VQe~vl~~l~~~~~~~~~~~~~-----i~~Y~~~R  181 (249)
</pre><pre>T 5MX5_G          108 EKIVVLLQRLKP-EIKDVTEQLNLVTTWLQLQIPRIEDGNNFGVAVQEKVFELMTNLHTKLEGFHTQ-----ISKYFSER  181 (249)
</pre><pre>T ss_dssp             HHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHTCCCCCSSCCHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHhccH-HHHHHHHHHHHHHHHHHhcCCccCCCCCccHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   90 QKLFSEL   96 (262)
</pre><pre>Q Consensus        90 ~~Lf~~L   96 (262)
</pre><pre>                      .++..++
</pre><pre>T Consensus       182 a~lv~k~  188 (249)
</pre><pre>T 5MX5_G          182 GDAVAKA  188 (249)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>PF05277.14 ; DUF726 ; Protein of unknown function (DUF726)
</pre><pre>Probab=34.37  E-value=2e+02  Score=29.27  Aligned_cols=61  Identities=11%  Similarity=0.146  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             hhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHH-HHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1  199 GASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLAC-SVVGAAAGGYIGGEVAGGATEYVM  259 (262)
</pre><pre>Q Consensus       199 ~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c-~vv~~g~g~~~gg~~G~~~G~~~G  259 (262)
</pre><pre>                      .+.++||.+-+..+..+++.|...+|..-+|+|.-.. +..+..+|...=+..++..|-.++
</pre><pre>T Consensus        11 ~~~~~gg~~~~~tgglaap~i~~~~~~~~~~~g~~~~~~~l~~~~g~~~~~~~fg~~g~~l~   72 (340)
</pre><pre>T MIL1_YEAST/536   11 LAMVGGSLVLGLSGGLLAPVIGGGIAAGLSTIGITGATSFLTGVGGTTVVAVSSTAIGANIG   72 (340)
</pre><pre>T ss_pred             HHhHHHHHHHHHhHHhHHHHHHHHHHHhhccccHHHHHHHHHHHHHHHHHHhhhccccchhc
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>4XCP_A Nematode fatty acid retinoid binding protein; Fatty acid retinol binding, retinol-binding protein; HET: MSE, PLM; 2.14A {Necator americanus}
</pre><pre>Probab=33.97  E-value=6e+02  Score=23.13  Aligned_cols=95  Identities=15%  Similarity=0.152  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++...+.+.+-+..|+|+-.+|+.+-++.+..+......+.-.....+.+...++-+..+.|..--++.+...=+.-.
</pre><pre>T Consensus        76 ~K~~~~~~~~~~ki~~LspeaK~Fv~~~~~~~~~l~~~~~~g~~~~~~~ik~~~~~~~~~y~~Ls~~~K~el~~~FP~i~  155 (170)
</pre><pre>T 4XCP_A           76 AKVEKLHAMVKSKIAALGPEAKGFAEKSIEIARGIKARYYTGNEPTKDDLKASVKEVLKLYKAMSDAGKADFGKQFPFLA  155 (170)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTSCHHHHHHHHHHHHHHHHHHHHHHSSSCCCHHHHHHHHHHHHHHHHHSCHHHHHHHHHHCHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhcCHHHHHHHHHHHHHHHHHHhhhhCCCCCCHHHHHHHHHHHHHHHHcCCHHHHHHHHHHChHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                      .-|....-+-|-+.+
</pre><pre>T Consensus       156 ~l~~~~~~~~~~~~~  170 (170)
</pre><pre>T 4XCP_A          156 KVFESGKAAKFAGEN  170 (170)
</pre><pre>T ss_dssp             HHHHTSHHHHHHCC-
</pre><pre>T ss_pred             HHHhCHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>4AKV_A SORTING NEXIN-33; TRANSPORT PROTEIN, ORGANELLE BIOGENESIS; HET: SO4; 2.651A {HOMO SAPIENS}
</pre><pre>Probab=33.69  E-value=8.2e+02  Score=24.20  Aligned_cols=151  Identities=9%  Similarity=0.056  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHh-HHHHHHH--hhhchhhhHHH--------HHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKH-FSTIAAL--TNFMDKSLGVV--------ADAGERYFSEIEKKLKAIELAY   70 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n-~~~l~~~--l~~~s~~~G~~--------s~~~~~~~~~i~~~L~~ie~ly   70 (262)
</pre><pre>                      +..+++..+.+...+..|...-.....++ .++-..+  ++..-...+..        .......++.+.+.+.++..+|
</pre><pre>T Consensus       190 ~~~~~~~~~~l~~~~~~l~~~~~~l~~~~~~~~~~~~~~l~~~l~~l~~~~~~~~~~~~~~l~~~~~~~a~~~~~~~~~~  269 (386)
</pre><pre>T 4AKV_A          190 VDTFKAFSKKMDDSVLQLSTVASELVRKHVGGFRKEFQKLGSAFQAISHSFQMDPPFCSEALNSAISHTGRTYEAIGEMF  269 (386)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTHHHHHHHHHHHHHHHHHHHTTSSGGGSCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHHHHHHHHHHHHHHHHHhccCCcccCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhcC-------------CCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcccc
</pre><pre>Q WP_048874398.1   71 QNQYRTQG-------------TLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVA  137 (262)
</pre><pre>Q Consensus        71 ~~~~~~~g-------------~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g  137 (262)
</pre><pre>                      .++....-             --...+.+..|+..+++++..-.........-.+-..+++.+..-+..+...|..--..
</pre><pre>T Consensus       270 ~~q~~~~~~~l~~~l~~~~~~~~~~~~~~~~r~~~~~~~~~~~~~~~~~k~~~~~~~~~~~~~~~~~~~~~~E~~~f~~~  349 (386)
</pre><pre>T 4AKV_A          270 AEQPKNDLFQMLDTLSLYQGLLSNFPDIIHLQKGAFAKVKESQRMSDEGRMVQDEADGIRRRCRVVGFALQAEMNHFHQR  349 (386)
</pre><pre>T ss_dssp             HHGGGTTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTSSCHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HhCchHhHHHHHHHHHHHHHHHhccHHHHHHhHHHHHHHHHHHHhhhcccccHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccchHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  138 SIRGYSTYVDNASKA  152 (262)
</pre><pre>Q Consensus       138 ~Ip~y~~~i~~~ak~  152 (262)
</pre><pre>                      ....+.+.+...++.
</pre><pre>T Consensus       350 r~~dl~~~l~~~a~~  364 (386)
</pre><pre>T 4AKV_A          350 RELDFKHMMQNYLRQ  364 (386)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>d2o1ea_ c.92.2.0 (A:) automated matches {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=33.65  E-value=3e+02  Score=26.63  Aligned_cols=70  Identities=11%  Similarity=0.133  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|.++.|+.++++.+|++.+..-                         |.+|+..|++.+........
</pre><pre>T Consensus       119 ~~~~~~~~~I~~~L~~~~P~~~~~~~~n~~~~~~~-------------------------l~~l~~~~~~~~~~~~~~~~  173 (282)
</pre><pre>T d2o1ea_         119 VLAQKEVKNITAQIVKQDPDNKEYYEKNSKEYIAK-------------------------LQDLDKLYRTTAKKAEKKEF  173 (282)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHSCSCCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCcccHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHhhHhcCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                      -.+...-.-+.+.+.
</pre><pre>T Consensus       174 v~~h~a~~yl~~~~g  188 (282)
</pre><pre>T d2o1ea_         174 ITQHTAFGYLAKEYG  188 (282)
</pre><pre>T ss_dssp             EESSCTTHHHHHHTT
</pre><pre>T ss_pred             EEecchHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>TIGR03164 UHCUDC; OHCU decarboxylase. Previously thought to only proceed spontaneously, the decarboxylation of 2-oxo-4-hydroxy-4-carboxy--5-ureidoimidazoline (OHCU) has been recently been shown to be catalyzed by this enzyme in Mus musculus.
</pre><pre>Probab=33.60  E-value=3.2e+02  Score=24.86  Aligned_cols=65  Identities=9%  Similarity=0.128  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHH-----HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGE-----RYFSEIEKKLKAIELAYQNQY   74 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~-----~~~~~i~~~L~~ie~ly~~~~   74 (262)
</pre><pre>                      +...+.+..+++++...+...|.-|.......+....-...-=.     ..=.+....|.++++.|.+.|
</pre><pre>T Consensus        38 ~a~~~~~~~~~~~~~~~~l~~hP~lg~~~~~~~~~~~~S~~EQ~~~~l~~~~~~~~~~L~~ln~~Y~~kF  107 (157)
</pre><pre>T TIGR03164        38 AAMVGAVRAASPEQQLALIRAHPDLAGKLAVAGELTAESTSEQASAGLDQLSQEEFARFTRLNNAYRARF  107 (157)
</pre><pre>T ss_pred             HHHHHHHHhCCHHHHHHHHHhCchhhhhhhhccchhHHcHHHHHHcccccCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>3CX3_A Lipoprotein; zinc-binding, Lipoprotein, Transport, LIPID BINDING PROTEIN, METAL BINDING PROTEIN; HET: MSE; 2.4A {Streptococcus pneumoniae} SCOP: c.92.2.0
</pre><pre>Probab=33.31  E-value=3.7e+02  Score=26.04  Aligned_cols=69  Identities=12%  Similarity=0.172  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|..+.|+.+.++.+|++.+.                         ..|.+++..|++.+......+.
</pre><pre>T Consensus       126 ~~~~~~~~~I~~~L~~~~P~~~~~~~~n~~~~~-------------------------~~l~~l~~~~~~~~~~~~~~~~  180 (284)
</pre><pre>T 3CX3_A          126 EKAGEEAQIIADKLSEVDSEHKETYQKNAQAFI-------------------------KKAQELTKKFQPKFEKATQKTF  180 (284)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHSGGGHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHHHSCSCCCE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhhCcccHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHHhhcCCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.+...-.-+++.+
</pre><pre>T Consensus       181 i~~h~~~~y~~~~~  194 (284)
</pre><pre>T 3CX3_A          181 VTQHTAFSYLAKRF  194 (284)
</pre><pre>T ss_dssp             EEEESCCHHHHHHT
</pre><pre>T ss_pred             EEecchHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>PF12920.9 ; TcdA_TcdB_pore ; TcdA/TcdB pore forming domain
</pre><pre>Probab=33.30  E-value=3.5e+02  Score=30.56  Aligned_cols=111  Identities=15%  Similarity=0.168  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             hhhhhhhhccccccchHHHHHHHHHHHHHH-HHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeecccchhchhH
</pre><pre>Q WP_048874398.1  126 SIVHEWSTAGVASIRGYSTYVDNASKAARF-LKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKYSQFGASLAG  204 (262)
</pre><pre>Q Consensus       126 S~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~-lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~g~~~G~~~G  204 (262)
</pre><pre>                      .+++-.+..-.++..-..+.+.++...... +..+  ++++|+.++-...|+|--.+-++.+-......-+-..++-.++
</pre><pre>T Consensus       154 ~l~~lv~~~l~~~~~~~~~~l~~~~~~l~~~~~~g--~~~~l~~~~vg~di~eL~~A~~~~~r~~~~tqL~Fd~~~l~l~  231 (660)
</pre><pre>T Q7MZW9_PHOLL/1  154 KVTALVRTALRGEVVAAETSLKDFASNLGHTVNEG--AGVLFGGAMVGLDAYELAHAENDVQKAVFGTQLAFDSASFVTG  231 (660)
</pre><pre>T ss_pred             HHHHHHHHHHHcChhHhhHhHHHHHHHHHHHHHHH--HHHHHHHHHHHHhHHHHHhCCCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             --HHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHH
</pre><pre>Q WP_048874398.1  205 --GIYGGKYGAMAGAGVCVALGVATGGVGLLACSVV  238 (262)
</pre><pre>Q Consensus       205 --g~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv  238 (262)
</pre><pre>                        ++..+.+|++.+++++-.++||-+|+++-+-+++
</pre><pre>T Consensus       232 ~~~~~a~~~ga~taa~~lg~l~VplaGl~iGi~aLv  267 (660)
</pre><pre>T Q7MZW9_PHOLL/1  232 AAGVGAGLVGASTAGAVLGGASVILGGLAVGFTALA  267 (660)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>7BAS_C CC-Type2-(TgLaId)4-W19BrPhe.; alpha, helical, barrel, pentamer, DE NOVO PROTEIN; HET: 4BF; 1.1A {N/A}
</pre><pre>Probab=33.21  E-value=2.1e+02  Score=19.15  Aligned_cols=21  Identities=29%  Similarity=0.355  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   52 GERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus        52 ~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      +.+.+++|.++|++|....++
</pre><pre>T Consensus         4 iaqtlkeiaktlkeia~tlke   24 (32)
</pre><pre>T 7BAS_C            4 IAQTLKEIAKTLKEIAXTLKE   24 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>2CCY_B CYTOCHROME C; ELECTRON TRANSPORT (HEME PROTEIN); HET: HEC; 1.67A {Phaeospirillum molischianum} SCOP: a.24.3.2
</pre><pre>Probab=32.95  E-value=5.3e+02  Score=21.80  Aligned_cols=100  Identities=8%  Similarity=0.044  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCCh----HHHHHHhcCcchhhhhhhhhccc-------------cccchHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNKLSRLTLKLKPY----EEIKHALGLSSRSIVHEWSTAGV-------------ASIRGYS  143 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~----~~lK~~LGlStkS~vh~w~~~g~-------------g~Ip~y~  143 (262)
</pre><pre>                      ...+....|+.+|++++..+..+...-.+-.++    ...-+.|.-..+.+.+.|.....             ..-+.|.
</pre><pre>T Consensus         4 ~~~~~v~~r~~~M~~~~~~~~~l~~~~~~~~~~d~~~~~~a~~i~~~~~~l~~~f~~~~~~~~~~~a~~~iw~~~~~~F~   83 (128)
</pre><pre>T 2CCY_B            4 KPEDLLKLRQGLMQTLKSQWVPIAGFAAGKADLPADAAQRAENMAMVAKLAPIGWAKGTEALPNGETKPEAFGSKSAEFL   83 (128)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHTSSCCCTTHHHHHHHHHHHHHHGGGGGCTTCTTCTTCCBCGGGTTTTHHHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHhHHHHHHHHcCCCCCCHHHHHHHHHHHHHHhhchhcCCCCCCCCCCCCCChHHhccCHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1  144 TYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTV  181 (262)
</pre><pre>Q Consensus       144 ~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~  181 (262)
</pre><pre>                      +..+...+.+..|...---+= =........+.+.|..
</pre><pre>T Consensus        84 ~~~~~~~~~a~~l~~aa~~~d-~~~~~~~~~~~~~C~~  120 (128)
</pre><pre>T 2CCY_B           84 EGWKALATESTKLAAAAKAGP-DALKAQAAATGKVCKA  120 (128)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHCH-HHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhCC-HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>5XBJ_A Flagellar hook-associated protein FlgK; Bacterial Flagellum, Hook, Filament, Junction protein, BIOSYNTHETIC PROTEIN; 2.448A {Campylobacter jejuni}
</pre><pre>Probab=32.88  E-value=1.1e+03  Score=25.56  Aligned_cols=93  Identities=17%  Similarity=0.060  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhch----hhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMD----KSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s----~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      |..+-.+.+..  +-.+=++..++.+...-+.|...+...+    ..-..+..-+....++|++.|++|.++=++-....
</pre><pre>T Consensus        44 l~~ff~a~~~l--a~~P~~~~~R~~v~~~a~~la~~~~~~~~~l~~~~~~~~~~i~~~V~~iN~ll~~Ia~LN~~I~~~~  121 (511)
</pre><pre>T 5XBJ_A           44 LENYNKAWNDF--ASNPNENATKIALVKASQTLTESVNNTFATLDKIQKKVNDDIKNTVDEINRIGEEIATINKQIYGQE  121 (511)
</pre><pre>T ss_dssp             HHHHHHHHHHH--HHSTTCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTSSS
</pre><pre>T ss_pred             HHHHHHHHHHH--HhCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CC--CccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   78 GT--LIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        78 g~--l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ..  -...+...||.+|+.+|
</pre><pre>T Consensus       122 ~~g~~~~ndL~DqRd~ll~eL  142 (511)
</pre><pre>T 5XBJ_A          122 ALPTEHANELRDRRDELELTL  142 (511)
</pre><pre>T ss_dssp             CCCHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             cCCCCCcchHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>d3gwkc1 a.25.3.0 (C:4-97) automated matches {Streptococcus agalactiae [TaxId: 208435]}
</pre><pre>Probab=32.68  E-value=3.7e+02  Score=19.87  Aligned_cols=90  Identities=9%  Similarity=0.003  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|...+..+......+...-..+-...-.+...+-+.+..........+...+..+.+.|.++.......       
</pre><pre>T Consensus         5 ~~~~l~~~a~~~~~~~~~~~~~~~~~~~~~~~~~~~w~G~a~~a~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   77 (94)
</pre><pre>T d3gwkc1           5 TPEELRSSAQKYTAGSQQVTEVLNLLTQEQAVIDENWDGSTFDSFEAQFNELSPKITEFAQLLEDINQQLLKV-------   77 (94)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHBCSSTTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhccCcHhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                       ...|....+..-+.+++
</pre><pre>T Consensus        78 -~~~~~~~d~~~~~~~~~   94 (94)
</pre><pre>T d3gwkc1          78 -ADIIEQTDADIASQISG   94 (94)
</pre><pre>T ss_dssp             -HHHHHHHHHHHHHHCCC
</pre><pre>T ss_pred             -HHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>PF05823.14 ; Gp-FAR-1 ; Nematode fatty acid retinoid binding protein (Gp-FAR-1)
</pre><pre>Probab=32.65  E-value=5.9e+02  Score=22.20  Aligned_cols=75  Identities=12%  Similarity=0.090  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      ..++.....+.+-+..|+|+-.+|+.+-++.+..+....-.+.......+.+....+-...+.|...-++.+.+.
</pre><pre>T Consensus        55 ~k~~~~~~~~~~k~~~Ls~eak~f~~~~~~~~~~l~~~~~~g~~~~~~~~~~~~~~~~~~y~~Ls~~~k~~i~~~  129 (149)
</pre><pre>T A0A2H2HX60_CAE   55 AKAEKLYAKLQKKVDGLSAEPKAFIQGLILAGRGLHAQAVSGEKIDKATVKSLIETQVKAFKELPAAAQTELKTT  129 (149)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhcCCHHHHHHHHHHHHHHHHhhhhhhcCCCCCHHHHHHHHHHHHHHHHcCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>TIGR01834 PHA_synth_III_E; poly(R)-hydroxyalkanoic acid synthase, class III, PhaE subunit. This model represents the PhaE subunit of the heterodimeric class (class III) of polymerase for poly(R)-hydroxyalkanoic acids (PHAs), carbon and energy storage polymers of many bacteria.
</pre><pre>Probab=32.61  E-value=9.7e+02  Score=24.72  Aligned_cols=131  Identities=13%  Similarity=0.065  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHh--cCCCccHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYF---SEIEKKLKAIELAYQNQYRT--QGTLIGQQ   84 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~---~~i~~~L~~ie~ly~~~~~~--~g~l~~~~   84 (262)
</pre><pre>                      ++.++-+..+.+.-.++...-.+....+......++.....-+..+.   +.+++ .+++-+.+.+.+..  ..-..+++
</pre><pre>T Consensus       175 Re~q~~~~~~~~~~~~yq~~~~~y~~~l~~~~~~a~~~~~~~l~e~~~~g~~~~s-~r~~~d~W~~~~e~~~~~~~~s~e  253 (320)
</pre><pre>T TIGR01834       175 REMQSQLQRLFRDWMEYQQAMADYQLLEADIGYKSFAALMSDLLARAKSGKPVKT-AKALYDLWVIAAEEAYAEVFASEE  253 (320)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCCChhh-HHHHHHHHHHHHHHHHHHHHcCHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   85 FFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKM  158 (262)
</pre><pre>Q Consensus        85 F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~  158 (262)
</pre><pre>                      |..-+.++.+.+...-..+-+      .-..+=+.|||.|++           +|..+.++|..+-+-.+.|+.
</pre><pre>T Consensus       254 f~~~~g~~~~a~~~~~~~~~~------~~e~~l~~l~lPTr~-----------Eid~L~~ri~eL~r~v~~L~~  310 (320)
</pre><pre>T TIGR01834       254 NAKVHGKFINALMRLRIQQQE------IVEALLKMLNLPTRS-----------ELDEAHQRIQQLRREVKSLKK  310 (320)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH------HHHHHHHhCCCCCHH-----------HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>4J2C_C Syntaxin-6; Tethering complex, PROTEIN TRANSPORT; 1.801A {Homo sapiens}
</pre><pre>Probab=32.50  E-value=3.8e+02  Score=21.96  Aligned_cols=101  Identities=14%  Similarity=0.076  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh---cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcC
</pre><pre>Q WP_048874398.1   46 GVVADAGERYFSEIEKKLKAIELAYQNQYRT---QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGL  122 (262)
</pre><pre>Q Consensus        46 G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~---~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGl  122 (262)
</pre><pre>                      |...+.+-...++|...|..+..+|.+....   ....+..+|..-++.|...++..-..+-...-.+..-.+=...++|
</pre><pre>T Consensus         1 ~~~~DPf~~v~~~v~~~l~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~el~~~~~~i~~~l~dL~~si~~v~~~p~kf~i   80 (110)
</pre><pre>T 4J2C_C            1 GAMEDPFFVVKGEVQKAVNTAQGLFQRWTELLQDPSTATREEIDWTTNELRNNLRSIEWDLEDLDETISIVEANPRKFNL   80 (110)
</pre><pre>T ss_dssp             CCCCCTHHHHHHHHHHHHHHHHHHHHHHHHHHHCC---CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHSHHHHCC
</pre><pre>T ss_pred             CCcCChhHHHHHHHHHHHHHHHHHHHHHHHHhhCCCCCCHHHHHHHHHHHHHHHHHHHhhHHhHHHHHHHHHhCchhcCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhhhhhhccccccchHHHHHHH
</pre><pre>Q WP_048874398.1  123 SSRSIVHEWSTAGVASIRGYSTYVDN  148 (262)
</pre><pre>Q Consensus       123 StkS~vh~w~~~g~g~Ip~y~~~i~~  148 (262)
</pre><pre>                      +..-+-.+....-  .+......|..
</pre><pre>T Consensus        81 ~~~El~~Rr~fv~--~~~~~v~~i~~  104 (110)
</pre><pre>T 4J2C_C           81 DATELSIRKAFIT--STRQVVRDMKD  104 (110)
</pre><pre>T ss_dssp             CHHHHHHHHHHHH--HHHHHHHHHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHH--HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>5IJN_N Nuclear pore glycoprotein p62; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=32.49  E-value=5.7e+02  Score=28.13  Aligned_cols=110  Identities=11%  Similarity=0.070  Sum_probs=0.0  Template_Neff=2.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHH---------------cCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHH--HHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALA---------------ELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERY--FSEIEKKL   63 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~---------------~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~--~~~i~~~L   63 (262)
</pre><pre>                      +|+.+..+.+++...|.               +-.+++++-+++.-+.||.=|...+..+...-.-+.+.  -.+-.+-|
</pre><pre>T Consensus       392 ~LdYIErQQdELEslLd~YEkQAE~LLsdi~lq~nDqeREkAY~LAE~LDeQLdsMSeNLsSLIEEINeVSKs~d~dDPL  471 (522)
</pre><pre>T 5IJN_N          392 ELDFILSQQKELEDLLSPLEELVKEQSGTIYLQHADEEREKTYKLAENIDAQLKRMAQDLKDIIEHLNTSGAPADTSDPL  471 (522)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTTSSTHHHHHHHHHHHHSSSCCCCSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTSCCSCTTSSH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhhchhcchhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCCCCCCHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHh-cCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCC
</pre><pre>Q WP_048874398.1   64 KAIELAYQNQYRT-QGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKL  110 (262)
</pre><pre>Q Consensus        64 ~~ie~ly~~~~~~-~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l  110 (262)
</pre><pre>                      .+|-++...++.. +---......+++-.-+++....+.+=..+.+.+
</pre><pre>T Consensus       472 ~QIVKILNsHLsSLqWIDeNstqLs~KLdqL~k~~s~~Rr~~ers~~~  519 (522)
</pre><pre>T 5IJN_N          472 QQICKILNAHMDSLQWIDQNSALLQRKVEEVTKVCEGRRKEQERSFRI  519 (522)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHGGGC-----------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhhchhceee
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>d1t33a1 a.4.1.9 (A:1-88) Putative transcriptional repressor YbiH {Salmonella typhimurium [TaxId: 90371]}
</pre><pre>Probab=32.43  E-value=2e+02  Score=22.73  Aligned_cols=43  Identities=14%  Similarity=0.223  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   87 VERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        87 ~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      ..|++++   +....-+...++. ..-.+|-+.+|+|..++-|++..
</pre><pre>T Consensus        12 ~~~~~il---~~~~~~~~~~g~~-~t~~~Ia~~~gvs~~tiy~~f~~   54 (88)
</pre><pre>T d1t33a1          12 QAKSQLI---AAALAQFGEYGLH-ATTRDIAALAGQNIAAITYYFGS   54 (88)
</pre><pre>T ss_dssp             HHHHHHH---HHHHHHHHHHGGG-SCHHHHHHHHTSCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHH---HHHHHHHHHhCch-hhHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>1ZVA_A E2 glycoprotein; SARS coronavirus, membrane fusion, S2, virus entry, coiled coils, conformational change, VIRAL PROTEIN; 1.5A {SARS coronavirus} SCOP: h.3.3.1
</pre><pre>Probab=32.32  E-value=2.4e+02  Score=23.27  Aligned_cols=71  Identities=13%  Similarity=0.098  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHh---hhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALT---NFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l---~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      |...+....+=.++|..|+.+-..++-..-..+..+.   ........+........++++.++.++|+.-|.+
</pre><pre>T Consensus         2 l~kiq~vvn~~~~al~~l~~qL~~nF~aiSssi~di~~rlt~~ls~lN~t~l~l~~e~~~L~~vik~LN~s~id   75 (77)
</pre><pre>T 1ZVA_A            2 LGKLQDVVNQNAQALNTLVKQLSSNFGAISSVLNDISGGRGGDISGINASVVNIQKEIDRLNEVAKNLNESLID   75 (77)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTSCCCCTTHHHHHHHHHHHHHHHHHHHHHHHHHHHC
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHhhcccccccccccCCCCCCcchhhHHHHHHHHHHHHHHHHHHHHHHhccC
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>1LS4_A Apolipophorin-III; Helix-bundle, exchangeable apolipoprotein, LIPID TRANSPORT; NMR {Locusta migratoria} SCOP: a.63.1.1
</pre><pre>Probab=32.04  E-value=7.7e+02  Score=23.41  Aligned_cols=101  Identities=13%  Similarity=0.100  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHcCCHHHHHHH--HHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCc
</pre><pre>Q WP_048874398.1    5 LQNAKQKVNGALAELNTDEANFF--NKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLI   81 (262)
</pre><pre>Q Consensus         5 L~~~A~~v~~al~~L~~~ea~fl--~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~   81 (262)
</pre><pre>                      |.+-.+++++.+..+..+-.+.+  -++=+++..+-..++.-......+..+.-+++++.++++..-..+.... .....
</pre><pre>T Consensus        26 l~el~~~aq~~i~~l~~~v~~~lglpn~eev~~~lKe~s~~~a~~lq~~~~~l~ee~~~~~~~v~~kl~e~~~~L~~~~P  105 (180)
</pre><pre>T 1LS4_A           26 IAEAVQQLNHTIVNAAHELHETLGLPTPDEALNLLTEQANAFKTKIAEVTTSLKQEAEKHQGSVAEQLNAFARNLNNSIH  105 (180)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHTTCTTTTHHHHHHHHHHHHHHHHHHHHHHHHHTTTTTHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHhCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ----cHHHHHHHHHHHHHHHH---HHHHHHH
</pre><pre>Q WP_048874398.1   82 ----GQQFFVERQKLFSELDV---LLNKLSR  105 (262)
</pre><pre>Q Consensus        82 ----~~~F~~~R~~Lf~~Ld~---~L~~~~~  105 (262)
</pre><pre>                          +|+-.++=..|-.+|+.   .+..++.
</pre><pre>T Consensus       106 ~~~~~~e~~~~a~el~~~l~~~~~~~qti~~  136 (180)
</pre><pre>T 1LS4_A          106 DAATSLNLQDQLNSLQSALTNVGHQWQDIAT  136 (180)
</pre><pre>T ss_dssp             TTSCCCSSSHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             chhhCHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>5W56_A Periplasmic solute binding protein; Solute binding protein, Zinc, ABC Transporter, METAL BINDING PROTEIN; HET: GOL; 2.03A {Paracoccus denitrificans (strain Pd 1222)} SCOP: c.92.2.0
</pre><pre>Probab=31.86  E-value=4e+02  Score=25.89  Aligned_cols=50  Identities=8%  Similarity=0.151  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT   76 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~   76 (262)
</pre><pre>                      ...++..++.+.+.|.++.|+.++++.+|+                         ++..+.|+++++.|++.+..
</pre><pre>T Consensus       120 p~~~~~~~~~I~~~L~~~~P~~~~~~~~n~-------------------------~~~~~~L~~l~~~~~~~~~~  169 (286)
</pre><pre>T 5W56_A          120 VPNAKVYVQNIAAAFCAADAEGCAAYQANA-------------------------ARYIGELDALDTEIRAAIAA  169 (286)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHCGGGHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHChhcHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>d2xpwa1 a.4.1.9 (A:3-67) Tetracyclin repressor (Tet-repressor, TetR) {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=31.61  E-value=2e+02  Score=20.46  Aligned_cols=38  Identities=13%  Similarity=0.226  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   96 LDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        96 Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      ++....-+...+..-+...++-+.+|+|..++.++|+.
</pre><pre>T Consensus         8 ~~~~~~~~~~~~~~~~s~~~ia~~~gis~~~~~~~~~~   45 (65)
</pre><pre>T d2xpwa1           8 IDAALELLNETGIDGLTTRKLAQKLGIEQPTLYWHVKN   45 (65)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHCCC
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>d1ui5a1 a.4.1.9 (A:5-75) A-factor receptor homolog CprB {Streptomyces coelicolor [TaxId: 1902]}
</pre><pre>Probab=31.60  E-value=3.6e+02  Score=19.54  Aligned_cols=69  Identities=9%  Similarity=0.018  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHh-hcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   88 ERQKLFSELDVLLNKLSRL-TLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        88 ~R~~Lf~~Ld~~L~~~~~~-~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                      ++...-.++-..+..+... +..-+....+-+.+|+|..++.+.++....--..-+....+...++.+.+
</pre><pre>T Consensus         2 r~~~~~~~~~~~~~~~~~~~~~~~~s~~~ia~~~gis~~t~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~   71 (71)
</pre><pre>T d1ui5a1           2 RAEQTRATIIGAAADLFDRRGYESTTLSEIVAHAGVTKGALYFHFAAKEDLAHAILEIQSRTSRRLAKDL   71 (71)
</pre><pre>T ss_dssp             CTTTHHHHHHHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSSHHHHHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             chHHHHHHHHHHHHHHHHHHCcccCCHHHHHHHHCCCHHHHHHHcCCHHHHHHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>TIGR03017 EpsF; chain length determinant protein EpsF. Sequences in this family of proteins are members of the chain length determinant family (pfam02706) which includes the wzc protein from E.
</pre><pre>Probab=31.52  E-value=9.8e+02  Score=24.43  Aligned_cols=84  Identities=11%  Similarity=0.036  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHhH--HHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   20 NTDEANFFNKHF--STIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        20 ~~~ea~fl~~n~--~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                      +|+.+..+.+.+  ...++..............++++.++.+++.|.+.++.+.+...+++-....+=.......+..|+
</pre><pre>T Consensus       142 dp~~A~~i~n~~~~~~~~~~~~~~~~~~~~~~~~l~~q~~~~~~~l~~~~~~l~~f~~~~~~~~~~~~~~~~~~~l~~l~  221 (444)
</pre><pre>T TIGR03017       142 DPRFAATVANAFAQAYIDTNIELKVEPAQKAALWFVQQIAALREDLARAQSKLSAYQQEKGIVSSDERLDVERARLNELS  221 (444)
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHhhchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCCCChhHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHH
</pre><pre>Q WP_048874398.1   98 VLLNKL  103 (262)
</pre><pre>Q Consensus        98 ~~L~~~  103 (262)
</pre><pre>                      .++...
</pre><pre>T Consensus       222 ~~~~~~  227 (444)
</pre><pre>T TIGR03017       222 AQLVAA  227 (444)
</pre><pre>T ss_pred             HHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>3HYI_A Protein DUF199/WhiA; LAGLIDADG, homing endonuclease, helix-turn-helix, HTH, TRANSCRIPTION REGULATOR; HET: GOL; 2.34A {Thermotoga maritima}
</pre><pre>Probab=31.47  E-value=8.9e+02  Score=24.05  Aligned_cols=107  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHH
</pre><pre>Q WP_048874398.1    9 KQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVE   88 (262)
</pre><pre>Q Consensus         9 A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~   88 (262)
</pre><pre>                      .+.|.+-|.-+......+-.++..+..++-+...=.+.--++...+..+.-.+.++.|+.+-...               
</pre><pre>T Consensus       175 ~~~I~~fL~~iGA~~~~~~~e~~~i~r~~~n~~nR~~N~d~aN~~kt~~aa~~q~~~I~~l~~~~---------------  239 (295)
</pre><pre>T 3HYI_A          175 IKDILVFLEAIGVQRKLEEIDRIVTERKVIGDVNRTVNFIEANAIRTANSTARQIRAIELIKENM---------------  239 (295)
</pre><pre>T ss_dssp             HHHHHHHHHHTTCCHHHHHHHHHHHHHCCCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHT---------------
</pre><pre>T ss_pred             HHHHHHHHHHcCChHHHHHHHHHHhhhhhhccccChhhHHHHHHHHHHHhHHHHHHHHHHHHHcc---------------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhhcCCCCh--HHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   89 RQKLFSELDVLLNKLSRLTLKLKPY--EEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        89 R~~Lf~~Ld~~L~~~~~~~~~l~~~--~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                         -+..|...|..+.+-.+.-|+.  ..|-+.++||-..+-|..++
</pre><pre>T Consensus       240 ---~l~~l~~~l~~~a~lRl~~P~~SL~eL~~~~~isKs~v~~rlrk  283 (295)
</pre><pre>T 3HYI_A          240 ---GLENLPEDLRRVALVRLRNKELSLRELGKKLNLTKSQIYSKLKR  283 (295)
</pre><pre>T ss_dssp             ---CGGGSCHHHHHHHHHHHHCTTSCHHHHHHTTTCCHHHHHHHHHH
</pre><pre>T ss_pred             ---ChhhCCHHHHHHHHHHHHCCCCCHHHHHHHcCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>PF07200.15 ; Mod_r ; Modifier of rudimentary (Mod(r)) protein
</pre><pre>Probab=31.43  E-value=6e+02  Score=21.98  Aligned_cols=103  Identities=12%  Similarity=0.139  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             HcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   17 AELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        17 ~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ..|+.+|-+.|.++.+.++.++.        ....+......+.+.+.+.+.+-...+..     .+++...|..+-.++
</pre><pre>T Consensus         1 ~~ls~~eL~~Ll~~~~~l~~~~~--------~~~~~~~~~~~~~~l~~~n~~lA~~nl~~-----~~~l~~~r~~l~~~~   67 (146)
</pre><pre>T K7IVX3_NASVI/2    1 DNLTTEELKKLSEDEDLMDEFLE--------KHSKLKDIDSAVEDAIDWVEKTATTNLDQ-----EQELQELREDVTEKV   67 (146)
</pre><pre>T ss_pred             CCCCHHHHHHHHhCHHHHHHHHH--------hCHHHHHHHHHHHHHHHHHHHHHHHHHHh-----HHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhc
</pre><pre>Q WP_048874398.1   97 DVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTA  134 (262)
</pre><pre>Q Consensus        97 d~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~  134 (262)
</pre><pre>                      +..=.....+.-.......+....  |+.++.......
</pre><pre>T Consensus        68 ~~~~~l~~~~~~~~~~~~~~~~~~--s~~~l~~~L~~~  103 (146)
</pre><pre>T K7IVX3_NASVI/2   68 QTVEALKARYDKLIQQYNKLSEVF--APEHIRDCLREA  103 (146)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhcC--CHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>4X6G_A OxyR; OxyR, peroxide, Transcription regulator, LysR, DNA BINDING PROTEIN; HET: GOL, PEO; 2.0A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=31.39  E-value=61  Score=29.58  Aligned_cols=76  Identities=11%  Similarity=0.121  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHhHHHHHHH--------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   26 FFNKHFSTIAAL--------TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        26 fl~~n~~~l~~~--------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      .++.+|+.|..|        ++.++.-.+.+.+.++++++++++.|..  .++.+.-+. .-.-.+..|+..-+.++..+
</pre><pre>T Consensus         4 ~~~~~~~~l~~f~~v~~~~s~~~AA~~l~is~s~vs~~i~~LE~~lg~--~Lf~r~~~~~~lT~~G~~l~~~~~~~~~~~   81 (316)
</pre><pre>T 4X6G_A            4 HHHMTLTELRYIVTLAQEQHFGRAAERCHVSQPTLSVGVKKLEDELGV--LIFERSKSAVRLTPVGEGIVAQAQKVLEQA   81 (316)
</pre><pre>T ss_dssp             --CCCHHHHHHHHHHHHHCSHHHHHHHTTSCHHHHHHHHHHHHHHHTS--CSBCCCSSCCCBCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ccCCCHHHHHHHHHHHHhCCHHHHHHHcCCCHHHHHHHHHHHHHHHCC--eeEEecCCceeeCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   97 DVLLNKL  103 (262)
</pre><pre>Q Consensus        97 d~~L~~~  103 (262)
</pre><pre>                      +.....+
</pre><pre>T Consensus        82 ~~~~~~~   88 (316)
</pre><pre>T 4X6G_A           82 QGIRELA   88 (316)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>4X6G_B OxyR; OxyR, peroxide, Transcription regulator, LysR, DNA BINDING PROTEIN; HET: GOL, PEO; 2.0A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=31.39  E-value=61  Score=29.58  Aligned_cols=76  Identities=11%  Similarity=0.121  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHhHHHHHHH--------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   26 FFNKHFSTIAAL--------TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        26 fl~~n~~~l~~~--------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      .++.+|+.|..|        ++.++.-.+.+.+.++++++++++.|..  .++.+.-+. .-.-.+..|+..-+.++..+
</pre><pre>T Consensus         4 ~~~~~~~~l~~f~~v~~~~s~~~AA~~l~is~s~vs~~i~~LE~~lg~--~Lf~r~~~~~~lT~~G~~l~~~~~~~~~~~   81 (316)
</pre><pre>T 4X6G_B            4 HHHMTLTELRYIVTLAQEQHFGRAAERCHVSQPTLSVGVKKLEDELGV--LIFERSKSAVRLTPVGEGIVAQAQKVLEQA   81 (316)
</pre><pre>T ss_dssp             CSCCCHHHHHHHHHHHHHCCHHHHHHHTTSCHHHHHHHHHHHHHHHTS--CSBCSCSSCCCBCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ccCCCHHHHHHHHHHHHhCCHHHHHHHcCCCHHHHHHHHHHHHHHHCC--eeEEecCCceeeCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   97 DVLLNKL  103 (262)
</pre><pre>Q Consensus        97 d~~L~~~  103 (262)
</pre><pre>                      +.....+
</pre><pre>T Consensus        82 ~~~~~~~   88 (316)
</pre><pre>T 4X6G_B           82 QGIRELA   88 (316)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>6BFU_A Spike protein; coronavirus spike glycoprotein, porcine deltacoronavirus, viral fusion protein, glycan shield, VIRAL PROTEIN, Structural Genomics, Seattle Structural; HET: MAN, NAG, BMA; 3.5A {Porcine deltacoronavirus}
</pre><pre>Probab=31.33  E-value=9.5e+02  Score=29.44  Aligned_cols=80  Identities=14%  Similarity=0.140  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQF   85 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F   85 (262)
</pre><pre>                      |.-|+.-++|+..++..        |+-+.+.+...|.+..+++.++.+-=+-|+.-=..|+.|-++.-..     +.+.
</pre><pre>T Consensus       779 k~iA~sFN~A~~~i~~a--------f~~vn~ai~q~s~g~~t~a~Al~kiQ~vVN~~~~al~~L~~qL~~n-----F~AI  845 (1024)
</pre><pre>T 6BFU_A          779 KILAESFNQAVGNISLA--------LSSVNDAIQQTSEALNTVAIAIKKIQTVVNQQGEALSHLTAQLSNN-----FQAI  845 (1024)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH--------HHTTCCSCCTTCHHHHHHHHHHHHHHHHHHTTTHHHHHHHHGGGCC-----CSSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----hcHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   86 FVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        86 ~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      -.-=+.+++|||.
</pre><pre>T Consensus       846 Sssi~dIy~rLd~  858 (1024)
</pre><pre>T 6BFU_A          846 STSIQDIYNRLEE  858 (1024)
</pre><pre>T ss_dssp             CSCHHHHHHHSCH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>PF05461.13 ; ApoL ; Apolipoprotein L
</pre><pre>Probab=31.26  E-value=3e+02  Score=27.44  Aligned_cols=57  Identities=23%  Similarity=0.144  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             ccchhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  196 SQFGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGATEYV  258 (262)
</pre><pre>Q Consensus       196 g~~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G~~~  258 (262)
</pre><pre>                      +++.|+..|-+.|..      ..+++++...|+|.++++.++.++...+-.....+....+..
</pre><pre>T Consensus        88 ~~~~gss~~i~gg~~------~i~G~~lap~T~G~sl~l~~~G~~~g~~g~~~~~~~~i~~~~  144 (298)
</pre><pre>T F6VUH9_HORSE/1   88 SNVVANSTGAVAGVL------TFLGLALAPVTAGVSLALSATGIGLGAAAGVTTVSTSIVEYS  144 (298)
</pre><pre>T ss_pred             hhhhcccchHHHHHH------HHHHHHHHHHhccHHHHHHHHHHHHHHHHhhhhhHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>PF18249.3 ; Ca_bind_SSO6904 ; Calcium binding protein SSO6904
</pre><pre>Probab=31.22  E-value=2.6e+02  Score=23.77  Aligned_cols=51  Identities=12%  Similarity=0.170  Sum_probs=0.0  Template_Neff=2.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhcC-----CCccHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1   56 FSEIEKKLKAIELAYQNQYRTQG-----TLIGQQFFVERQKLFSELDVLLNKLSRL  106 (262)
</pre><pre>Q Consensus        56 ~~~i~~~L~~ie~ly~~~~~~~g-----~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~  106 (262)
</pre><pre>                      .+.++++|++|+.-|+..-.-+.     --++-+...+.+.+|.-+-..|.++..+
</pre><pre>T Consensus        21 ~~~v~~IL~eie~e~~ks~~ikssiIfvYan~~~~Vk~Nke~y~li~~ilEKys~K   76 (90)
</pre><pre>T W7KWQ0_9CREN/1   21 SAELNSVLEEVEEDFAKSGSLKSSLIFVYANHIEQVRRNKELYDLISAILEKYAPK   76 (90)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhcCCcchhheeehhhhHHHHHhCHHHHHHHHHHHHHHhhh
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>PF10819.10 ; DUF2564 ; Protein of unknown function (DUF2564)
</pre><pre>Probab=31.17  E-value=5.5e+02  Score=21.43  Aligned_cols=64  Identities=14%  Similarity=0.199  Sum_probs=0.0  Template_Neff=3.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHH
</pre><pre>Q WP_048874398.1    5 LQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQ   84 (262)
</pre><pre>Q Consensus         5 L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~   84 (262)
</pre><pre>                      .+.+-+-|-.+=..|+|+                                .++.-.+.+.+-...|.+.......+ .++
</pre><pre>T Consensus        12 V~aAQkmVG~ATmsMD~~--------------------------------qL~~A~~Ai~~Ak~ql~~a~~~~tg~-D~~   58 (78)
</pre><pre>T D5E1E9_BACMQ/1   12 IQSAQKMIGTATMSMSPQ--------------------------------QLEEATNALNDAKAQLQSAKAHGTGV-DEQ   58 (78)
</pre><pre>T ss_pred             HHHHHHHHHHHHccCCHH--------------------------------HHHHHHHHHHHHHHHHHHHHHHCCCC-CHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   85 FFVERQKLFSELDVLLN  101 (262)
</pre><pre>Q Consensus        85 F~~~R~~Lf~~Ld~~L~  101 (262)
</pre><pre>                      |+.+-..++.+.+.+|+
</pre><pre>T Consensus        59 Fl~~~~~~L~~~ehQL~   75 (78)
</pre><pre>T D5E1E9_BACMQ/1   59 FLQQCMQSIQTCEQQLT   75 (78)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>6V8I_BF Tape Measure Protein, gp57; phage tail, tail tip, tape measure protein, VIRAL PROTEIN; 3.7A {Staphylococcus virus 80alpha}
</pre><pre>Probab=31.14  E-value=1.6e+03  Score=26.74  Aligned_cols=251  Identities=8%  Similarity=0.023  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCH-----------HHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNT-----------DEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELA   69 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~-----------~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~l   69 (262)
</pre><pre>                      ++..|+.+.+++++.++.+.+           .+..-+.++.+.++..+...+.  .....-++..+++..+.+++.++.
</pre><pre>T Consensus        13 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~--~~~~~~i~~~~~~~~~~~~~~~~~   90 (1154)
</pre><pre>T 6V8I_BF          13 SVAKFKRQIDSAVKSVQRFKRVADQTKDVELNANDKKLQKTIKVAKKSLDAFSN--KNVKAKLDASIQDLQQKILESNFE   90 (1154)
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHhccccccchhHHHHHHHHHHHHHHHHHhhc--ccccHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcc--------hhhhhhhhhcc-ccccc
</pre><pre>Q WP_048874398.1   70 YQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSS--------RSIVHEWSTAG-VASIR  140 (262)
</pre><pre>Q Consensus        70 y~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlSt--------kS~vh~w~~~g-~g~Ip  140 (262)
</pre><pre>                      .+..-+..-...-+.=+.+-.+=+.+++..|+.+-+.....+..++..+..  ..        +.+-+...+.. .-++.
</pre><pre>T Consensus        91 ~~~l~~~~~~~~~~~~~~~~~~~i~~~~~~l~~l~~~~~~~~v~~~~~~~~--~~~~~~~~~~~~~~~~~~~i~v~~d~~  168 (1154)
</pre><pre>T 6V8I_BF          91 LDKLNSKEASPEVKLQKQKLTKDIAEAENKLSELEKKRVNIDVNADNSKFN--RVLKVSKASLEALNRSKAKAILDVDNS  168 (1154)
</pre><pre>T ss_pred             HHHhhhccCCHHHHHHHHHHHHHHHHHHHHHHHHHHhhhhhhHhcchHHHH--HHHHHHHHHHHHHHHhhccchhhcChH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHhhhHHHHhHHHHHHHHHHHHHhcCChHHccceeeeec-------------------------
</pre><pre>Q WP_048874398.1  141 GYSTYVDNASKAARFLKMGGWLAIGFAGANTTNEVYDACTVGREQSCSKVAVKKY-------------------------  195 (262)
</pre><pre>Q Consensus       141 ~y~~~i~~~ak~~k~lk~gGyvgi~l~~~~s~~~I~eAC~~g~~~~C~k~~~~E~-------------------------  195 (262)
</pre><pre>                      .+..+++++.+..+.+..--.+-+-+|.......+.+.-..=+.-.-.+..-.+.                         
</pre><pre>T Consensus       169 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  248 (1154)
</pre><pre>T 6V8I_BF         169 VANSKIKRTKEELKSIPNKTRSRLDVDTRLSIPTIYAFKKSLDALPNKKTTKVDVDTNGLKKVYAYIIKANDNFQRQMGN  248 (1154)
</pre><pre>T ss_pred             HHHHHHHHHHHHHhcccccCCcccchhhhhhHHHHHHHHHHHhcCCCCCccccccccchhHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             --------ccchhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  196 --------SQFGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGGEVAGGAT  255 (262)
</pre><pre>Q Consensus       196 --------g~~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg~~G~~~G  255 (262)
</pre><pre>                              ++...++..+.+.+..+..+...-.+.-++..-+..+.+.+....+.++..+...+....
</pre><pre>T Consensus       249 ~~~~~~~~g~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~l~~~~~~a~~~~~a  316 (1154)
</pre><pre>T 6V8I_BF         249 LANMFRVFGTVGSNMVGGLLTSSFSILIPVIASVVPVVFALLNAIKVLTGGVLALGGAVAIAGAGFVA  316 (1154)
</pre><pre>T ss_pred             HHHHHHHhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>5MX5_M Proteasome activator complex subunit 2; proteasome activator, regulator, heptamer, interferon-gamma, hydrolase, structural protein; HET: PO4; 2.9A {Mus musculus}
</pre><pre>Probab=31.13  E-value=6.3e+02  Score=24.52  Aligned_cols=81  Identities=14%  Similarity=0.146  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVER   89 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R   89 (262)
</pre><pre>                      +.+.+.+..+.+ +...+.+..+.+...+...-.-+.-..++.-..-++|-+.|.+++......+..     ...|+..|
</pre><pre>T Consensus        98 ~~i~~l~~~lk~-~i~~l~~~~~~vk~WI~l~iP~iedGnnfgv~VQe~vl~~l~~~~~~~~~~~~~-----~~~Y~~~R  171 (239)
</pre><pre>T 5MX5_M           98 EKLLALLALVKP-EVWTLKEKCILVITWIQHLIPKIEDGNDFGVAIQEKVLERVNAVKTKVEAFQTT-----ISKYFSER  171 (239)
</pre><pre>T ss_dssp             HHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHCCCCCSSCCHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHhhhH-HHHHHHHHHHHHHHHHHhhcccccCCCCccHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1   90 QKLFSEL   96 (262)
</pre><pre>Q Consensus        90 ~~Lf~~L   96 (262)
</pre><pre>                      ..+..++
</pre><pre>T Consensus       172 a~lv~k~  178 (239)
</pre><pre>T 5MX5_M          172 GDAVAKA  178 (239)
</pre><pre>T ss_dssp             HHHHHHH
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>4LWS_B Uncharacterized protein; ESX Secretion, Signal Sequence, Type VII Secretion, UNKNOWN FUNCTION; HET: ACT, GOL, MSE; 2.0A {Thermomonospora curvata}
</pre><pre>Probab=31.08  E-value=3.9e+02  Score=19.69  Aligned_cols=89  Identities=6%  Similarity=0.016  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|...++.+......+.....++-.....+...+...+..........|...++.+.+.|.++.....+.       
</pre><pre>T Consensus        14 ~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~w~g~~~~a~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   86 (103)
</pre><pre>T 4LWS_B           14 NFGGLSEGEAQFSMTARALLDELTDLEGKLRAKLDRWDGDAQAAYWNYQKEWDAAAKDMQNVVAQLGVAIREA-------   86 (103)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTGGGSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>T ss_pred             chhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                       ...|...-...-+.++
</pre><pre>T Consensus        87 -~~~~~~~d~~~~~~~~  102 (103)
</pre><pre>T 4LWS_B           87 -HDNYQAAERANTSIWA  102 (103)
</pre><pre>T ss_dssp             -TTCCC-----------
</pre><pre>T ss_pred             -HHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 146
</pre><pre>>TIGR02984 Sig-70_plancto1; RNA polymerase sigma-70 factor, Planctomycetaceae-specific subfamily 1. This group of sigma factors are members of the sigma-70 family (TIGR02937) and are apparently found only in the Planctomycetaceae family including the genuses Gemmata and Pirellula (in which seven sequences are found).
</pre><pre>Probab=31.04  E-value=4.5e+02  Score=22.07  Aligned_cols=63  Identities=14%  Similarity=0.134  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      .....+..+.+...+..|++.+...+.-+|  ++.. ...-+.-.|.+...+.+++.++.+.|++.
</pre><pre>T Consensus       124 ~~~~~~~~~~~~~~l~~l~~~~~~i~~l~~--~~~~s~~eia~~l~~s~~~v~~~~~r~~~~l~~~  187 (189)
</pre><pre>T TIGR02984       124 VAARRESAVRLAQALAKLPEDYREVILLRH--LEGLSFAEVAERMDRSEGAVSMLWVRGLARLRQI  187 (189)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHHH--hcCCCHHHHHHHhCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 147
</pre><pre>>4H0F_B Laminin-binding surface protein; Adhesin, Human Laminin, CELL ADHESION; 2.4A {Streptococcus agalactiae} SCOP: c.92.2.0
</pre><pre>Probab=31.03  E-value=4.5e+02  Score=25.07  Aligned_cols=69  Identities=9%  Similarity=0.120  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|..+.|+.++++.+|+                         ++..+.|.+++..+++.+........
</pre><pre>T Consensus       106 ~~~~~~~~~i~~~L~~~~P~~~~~~~~n~-------------------------~~~~~~l~~l~~~~~~~~~~~~~~~v  160 (264)
</pre><pre>T 4H0F_B          106 VLAGEEAVNIAKELGHLDPKHKDSYTKKA-------------------------KAFKKEAEQLTEEYTQKFKKVRSKTF  160 (264)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHHTCSCCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhHChhcHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHhccCCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.+...-+-+.+.+
</pre><pre>T Consensus       161 v~~~~~~~yl~~~~  174 (264)
</pre><pre>T 4H0F_B          161 VTQHTAFSYLAKRF  174 (264)
</pre><pre>T ss_dssp             EEEESCCHHHHHHT
</pre><pre>T ss_pred             EEeccHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 148
</pre><pre>>2O1E_B YcdH; alpha-beta protein, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, UNKNOWN FUNCTION; HET: MSE; 2.6A {Bacillus subtilis} SCOP: c.92.2.0
</pre><pre>Probab=30.87  E-value=3.6e+02  Score=26.78  Aligned_cols=70  Identities=11%  Similarity=0.133  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|.++.|+.++++.+|++.+..-                         |.+|++.|++.+........
</pre><pre>T Consensus       139 ~~~~~~~~~I~~~L~~~~P~~a~~~~~n~~~~~~~-------------------------L~~l~~~~~~~~~~~~~~~~  193 (312)
</pre><pre>T 2O1E_B          139 VLAQKEVKNITAQIVKQDPDNKEYYEKNSKEYIAK-------------------------LQDLDKLYRTTAKKAEKKEF  193 (312)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHHCSSCCE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCcccHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHhHHhcCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                      -.+...-.-+.+.+.
</pre><pre>T Consensus       194 i~~h~~~~y~~~~~g  208 (312)
</pre><pre>T 2O1E_B          194 ITQHTAFGYLAKEYG  208 (312)
</pre><pre>T ss_dssp             EESSCCCHHHHHHTT
</pre><pre>T ss_pred             EEecchHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 149
</pre><pre>>PF06730.13 ; FAM92 ; FAM92 protein
</pre><pre>Probab=30.83  E-value=7.3e+02  Score=22.74  Aligned_cols=97  Identities=5%  Similarity=-0.014  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ++..|......+...+..+.....+.-....++-..+...++.-    .......+..+.+.++.++..++.........
</pre><pre>T Consensus        19 ~~~~le~~l~~l~~~~~~~~~~~~~l~~~~~~~~~~l~~~~~~E----~~~l~~~l~~~~~~~~~~~~~~~~~~~~~~~~   94 (219)
</pre><pre>T FA92A_HUMAN/1-   19 AVSNVEKHFGELCQIFAAYVRKTARLRDKADLLVNEINAYAATE----TPHLKLGLMNFADEFAKLQDYRQAEVERLEAK   94 (219)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC----CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      --+.+...-. .++.+...|.+
</pre><pre>T Consensus        95 ~~~~l~~y~~-~~~~vk~~l~~  115 (219)
</pre><pre>T FA92A_HUMAN/1-   95 VVEPLKTYGT-IVKMKRDDLKA  115 (219)
</pre><pre>T ss_pred             hHHHHHHHHH-HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 150
</pre><pre>>3A6M_A Protein grpE; GRPE, COILED-COIL, FOUR-HELIX BUNDLE, DIMER, Chaperone, Stress response; 3.23A {Thermus thermophilus}
</pre><pre>Probab=30.81  E-value=6.8e+02  Score=22.38  Aligned_cols=102  Identities=9%  Similarity=0.052  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ++..|+.+.....+.+.++..+..+...........+-..-...---.........+.+-..|-.+-+.+......... 
</pre><pre>T Consensus        13 ~~~~l~~~~~~l~~~l~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ll~~~d~l~~~~~~~~~-   91 (177)
</pre><pre>T 3A6M_A           13 DLEAVGQEAQALEERLKAAEEELKGLKDKYLRLLADFDNYRKRMEEELKAREREGVLKALRALLPVLDDLDRALEFAEA-   91 (177)
</pre><pre>T ss_dssp             GGTTTHHHHSSTTTSTTGGGGTSSSHHHHHHTTTTTTTTHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTGGG-
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccc-
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      ....+...-..++++|...|.++
</pre><pre>T Consensus        92 ~~~~~~~~~~~~~~~l~~~L~~~  114 (177)
</pre><pre>T 3A6M_A           92 SPESIRQGVRAIRDGFFRILAGL  114 (177)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             ChhHHHHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 151
</pre><pre>>6VME_V Vacuolar protein sorting-associated protein 28 homolog; Endosomal transport, HIV release, macroautophagy, PROTEIN TRANSPORT; 2.19A {Homo sapiens} SCOP: a.2.17.0
</pre><pre>Probab=30.70  E-value=6.7e+02  Score=22.24  Aligned_cols=76  Identities=12%  Similarity=0.211  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhcc-c
</pre><pre>Q WP_048874398.1   58 EIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAG-V  136 (262)
</pre><pre>Q Consensus        58 ~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g-~  136 (262)
</pre><pre>                      .|-.+++.||++|.+.     .++.++|-.+-.+|++|....+.  ......+++-+..-+..+++-..-+|+- +.| +
</pre><pre>T Consensus        44 sIi~tle~LEkayvrd-----~i~~~eY~~~C~rLi~qyk~~~~--~~~~~~~~~i~~F~~~y~l~cp~A~~Rl-~~G~P  115 (122)
</pre><pre>T 6VME_V           44 AVVKTMQALEKAYIKD-----CVSPSEYTAACSRLLVQYKAAFR--QVQGSEISSIDEFCRKFRLDCPLAMERI-KEDRP  115 (122)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTT-----CSCHHHHHHHHHHHHHHHHHHHH--HHCSSSCCCHHHHHHHTTCCCHHHHHHH-HHTSC
</pre><pre>T ss_pred             HHHHHHHHHHHHhhcc-----CCCHHHHHHHHHHHHHHHHHHHH--HhcCCCCCCHHHHHHHhCCCChHHHhHh-hcCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccch
</pre><pre>Q WP_048874398.1  137 ASIRG  141 (262)
</pre><pre>Q Consensus       137 g~Ip~  141 (262)
</pre><pre>                      ..++.
</pre><pre>T Consensus       116 atve~  120 (122)
</pre><pre>T 6VME_V          116 ITIKD  120 (122)
</pre><pre>T ss_dssp             TTC--
</pre><pre>T ss_pred             Ccccc
</pre><pre>
</pre><pre>
</pre><pre>No 152
</pre><pre>>6NZK_C Spike surface glycoprotein; Coronavirus spike glycoprotein, sialic acid, HCoV-OC43, fusion protein, VIRAL PROTEIN; HET: BMA, MAN, NAG, MJJ; 2.8A {Human coronavirus OC43}
</pre><pre>Probab=30.67  E-value=1.1e+03  Score=29.64  Aligned_cols=89  Identities=17%  Similarity=0.168  Sum_probs=0.0  Template_Neff=4.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHhcC--
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQ-NQYRTQG--   78 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~-~~~~~~g--   78 (262)
</pre><pre>                      |..+++.-..++.||.+    --+...+|-+.|..++..=+...|    +++..+++|-+-|++||.--| +.+-...  
</pre><pre>T Consensus      1025 i~~Iq~gF~~tn~AL~K----IQ~VVN~qa~aL~~L~~QL~nNFg----AISSSI~dIy~RLD~LEAdaQVDRLItGRL~ 1096 (1322)
</pre><pre>T 6NZK_C         1025 LYAIQEGFDATNSALVK----IQAVVNANAEALNNLLQQLSNRFG----AISASLQEILSRLDALEAEAQIDRLINGRLT 1096 (1322)
</pre><pre>T ss_dssp             HHHHHHTTTSCCHHHHH----HHHHHHHHHHHHHHHHGGGGCCTT----SSCSCHHHHHHHSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHhhc----hhHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ---------CCccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 ---------TLIGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        79 ---------~l~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                               ..+..+-.++|+--.+|++.
</pre><pre>T Consensus      1097 aLNafVtQ~L~~~sevrasr~LA~qKINE 1125 (1322)
</pre><pre>T 6NZK_C         1097 ALNAYVSQQLSDSTLVKFSAAQAMEKVNE 1125 (1322)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 153
</pre><pre>>6ZT1_E CC-Type2-(LaIdGe)4; coiled coil, de novo peptide, multi-state, DE NOVO PROTEIN; HET: 4BF, ACE; 2.01A {N/A}
</pre><pre>Probab=30.54  E-value=2.5e+02  Score=18.81  Aligned_cols=21  Identities=33%  Similarity=0.333  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   52 GERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus        52 ~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      +++.+++|.+.|++|-.+.++
</pre><pre>T Consensus         4 igqalkeigkalkeig~alke   24 (32)
</pre><pre>T 6ZT1_E            4 IGQALKEIGKALKEIGXALKE   24 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 154
</pre><pre>>6ZT1_I CC-Type2-(LaIdGe)4; coiled coil, de novo peptide, multi-state, DE NOVO PROTEIN; HET: 4BF, ACE; 2.01A {N/A}
</pre><pre>Probab=30.54  E-value=2.5e+02  Score=18.81  Aligned_cols=21  Identities=33%  Similarity=0.333  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   52 GERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus        52 ~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      +++.+++|.+.|++|-.+.++
</pre><pre>T Consensus         4 igqalkeigkalkeig~alke   24 (32)
</pre><pre>T 6ZT1_I            4 IGQALKEIGKALKEIGXALKE   24 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 155
</pre><pre>>2HWG_B Phosphoenolpyruvate-protein phosphotransferase; enzyme I, phosphoenolpyruvate:sugar phosphotransferase system, PTS, TRANSFERASE; HET: MSE, OXL, NEP; 2.7A {Escherichia coli}
</pre><pre>Probab=30.53  E-value=7.3e+02  Score=27.10  Aligned_cols=89  Identities=10%  Similarity=0.216  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcC--------CHHHHHHHHHhHHHHHH--HhhhchhhhH----HHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAEL--------NTDEANFFNKHFSTIAA--LTNFMDKSLG----VVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L--------~~~ea~fl~~n~~~l~~--~l~~~s~~~G----~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      +++.|..+-+++.+.|..+        .+++++.+.-|..+|++  |.......+-    ++..++...+++..+.+..+
</pre><pre>T Consensus        39 E~~rl~~A~~~~~~~L~~~~~~~~~~~~~~~~~I~~a~~~iL~D~~~~~~i~~~I~~~~~~a~~Av~~~~~~~~~~~~~~  118 (575)
</pre><pre>T 2HWG_B           39 EVERFLSGRAKASAQLETIKTKAGETFGEEKEAIFEGHIMLLEDEELEQEIIALIKDKHMTADAAAHEVIEGQASALEEL  118 (575)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHTGGGTTTCSSHHHHHHHHHHHHTCHHHHHHHHHHHHHHCCCHHHHHHHHHHHHHHHHHTS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhhcCHHHHHHHHHHHHHHcCHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhcCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   67 ELAYQNQYRTQGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        67 e~ly~~~~~~~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ++-|.++       +..++..-.++++..|
</pre><pre>T Consensus       119 ~d~~~~~-------R~~Di~dl~~rll~~L  141 (575)
</pre><pre>T 2HWG_B          119 DDEYLKE-------RAADVRDIGKRLLRNI  141 (575)
</pre><pre>T ss_dssp             SCHHHHH-------HHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CcHHHHh-------HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 156
</pre><pre>>d2o3fa1 a.4.1.20 (A:1-83) Putative transcriptional regulator YbbH {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=30.47  E-value=3.8e+02  Score=20.02  Aligned_cols=75  Identities=15%  Similarity=0.127  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhcCCC------ChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKLSRLTLKLK------PYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~------~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                      ..+...=..++.+|...-..+.+.-...|      ...++-+.+|+|..++-+.++..+   .++|.+-...+.......
</pre><pre>T Consensus         3 ~~~~~~~~~~~~~l~~~~~~i~~~i~~~~~~~~~~s~~eia~~~~is~~tv~r~~~~l~---~~g~~~~~~~~~~~~~~~   79 (83)
</pre><pre>T d2o3fa1           3 TGGLAIIQSMKHKLPPSERKLADYILAHPHKAIESTVNEISALANSSDAAVIRLCKSLG---LKGFQDLKMRVAGDLAKP   79 (83)
</pre><pre>T ss_dssp             CCHHHHHHHHGGGSCHHHHHHHHHHHHCHHHHHTCCHHHHHHHTTCCHHHHHHHHHHTT---CSSHHHHHHHHHHHHHSG
</pre><pre>T ss_pred             ccHHHHHHHHHhcCCHHHHHHHHHHHHCHHHHhhCCHHHHHHHhCCCHHHHHHHHHHcC---CCCHHHHHHHHHHHHhcc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhh
</pre><pre>Q WP_048874398.1  157 KMGG  160 (262)
</pre><pre>Q Consensus       157 k~gG  160 (262)
</pre><pre>                      ...|
</pre><pre>T Consensus        80 ~~~~   83 (83)
</pre><pre>T d2o3fa1          80 TFQG   83 (83)
</pre><pre>T ss_dssp             GGCC
</pre><pre>T ss_pred             cccC
</pre><pre>
</pre><pre>
</pre><pre>No 157
</pre><pre>>1PQ4_B periplasmic binding protein component of an ABC type zinc uptake transporter; ZnuA, loop, metal-binding, METAL BINDING PROTEIN; 1.9A {Synechocystis sp.} SCOP: c.92.2.2
</pre><pre>Probab=30.30  E-value=3.7e+02  Score=26.40  Aligned_cols=69  Identities=10%  Similarity=0.131  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|..+.|+.++++.+|                         ++...+.|.++++.|++.+.......-
</pre><pre>T Consensus       139 ~~~~~~~~~i~~~L~~~~P~~~~~~~~n-------------------------~~~~~~~l~~l~~~~~~~~~~~~~~~v  193 (291)
</pre><pre>T 1PQ4_B          139 TLVKRQATTIAKELAELDPDNRDQYEAN-------------------------LAAFLAELERLNQELGQILQPLPQRKF  193 (291)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHHTTCSCCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCcccHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHhccCCCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.+...-+-|++.+
</pre><pre>T Consensus       194 v~~h~~~~yl~~~~  207 (291)
</pre><pre>T 1PQ4_B          194 IVFHPSWAYFARDY  207 (291)
</pre><pre>T ss_dssp             EESSCCCHHHHHHT
</pre><pre>T ss_pred             EEecHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 158
</pre><pre>>7JJ8_A Zinc-binding lipoprotein AdcA; AdcA, ZnuA, SBP, ATP-binding cassette transporter, Zn acquisition, metal binding protein; 2.03A {Streptococcus pneumoniae (strain ATCC BAA-255 / R6)}
</pre><pre>Probab=30.24  E-value=6.4e+02  Score=24.71  Aligned_cols=69  Identities=12%  Similarity=0.082  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|..+.|+.++.+.+|++.+..-                         |++++..+++.+........
</pre><pre>T Consensus       120 ~n~~~~a~~i~~~L~~~~p~~a~~~~~n~~~~~~~-------------------------L~~l~~~~~~~~~~~~~~~v  174 (299)
</pre><pre>T 7JJ8_A          120 VRAIKLVEHIRDSLSADYPDKKETFEKNAAAYIEK-------------------------LQSLDKAYAEGLSQAKQKSF  174 (299)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHTTCSCCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHhhcCCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.+...-.-+++.+
</pre><pre>T Consensus       175 v~~h~~~~y~~~~~  188 (299)
</pre><pre>T 7JJ8_A          175 VTQHAAFNYLALDY  188 (299)
</pre><pre>T ss_dssp             EESSCCCHHHHHHH
</pre><pre>T ss_pred             EeechHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 159
</pre><pre>>5ZVK_A Human Coronavirus MERS HR1 motif; MERS, Spike protein, S2 domain, HR1 motif, pan-Coronavirus, VIRAL PROTEIN-INHIBITOR complex; 3.31A {Middle East respiratory syndrome-related coronavirus}
</pre><pre>Probab=30.08  E-value=5.9e+02  Score=21.41  Aligned_cols=73  Identities=11%  Similarity=0.054  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   25 NFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        25 ~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      ++|.+|-..|..-+..+-..+..........+.+|.+...+=-.+... +...=..++..--.-=+.+++|||.
</pre><pre>T Consensus         6 ~vL~~NQk~iA~sFN~A~~~i~~~f~~t~~AL~kiQ~VVN~q~~aL~~-L~~qL~~NFgAISssi~dIy~RLd~   78 (80)
</pre><pre>T 5ZVK_A            6 QVLSENQKLIANKFNQALGAMQTGFTTTNEAFRKVQDAVNNNAQALSK-LASELSNTFGAISASIGDIIQRLDV   78 (80)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHC-----
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHhcHhHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 160
</pre><pre>>5L1X_G hMPV F2 subunit; class I fusion protein, VIRAL PROTEIN; HET: SO4, NAG, FUC; 3.3A {Human metapneumovirus}
</pre><pre>Probab=30.06  E-value=1e+02  Score=25.78  Aligned_cols=42  Identities=7%  Similarity=0.078  Sum_probs=0.0  Template_Neff=2.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHH
</pre><pre>Q WP_048874398.1   50 DAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQK   91 (262)
</pre><pre>Q Consensus        50 ~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~   91 (262)
</pre><pre>                      +.+.+-++.-+++|.||..+.+++.....+...|++..+|++
</pre><pre>T Consensus        47 SLIk~EL~l~knAl~EL~tvs~~~~a~~~r~~~pr~~~~~~~   88 (89)
</pre><pre>T 5L1X_G           47 SLIKTELDLTKSALRELRTVSADQLAREEQIENPRQSKKRKR   88 (89)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHTTCSSGGGCC--------------
</pre><pre>T ss_pred             chhHHHHHHhHHHHHHHHHHcchhHHHHHHhhCchhhhhccc
</pre><pre>
</pre><pre>
</pre><pre>No 161
</pre><pre>>7KIP_B Spike glycoprotein; NL63, in situ, Spike trimer, glycosylation sites, single particle cryo-EM, VIRAL PROTEIN; HET: BMA, MAN, NAG; 3.39A {Human coronavirus NL63}
</pre><pre>Probab=30.01  E-value=9.8e+02  Score=30.25  Aligned_cols=80  Identities=14%  Similarity=0.122  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQF   85 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F   85 (262)
</pre><pre>                      |.-|+.-++||..+++.        |+...+.+...|.+..+++.++.+-=+-|++-=..|+.|-++.-..     +.+.
</pre><pre>T Consensus       973 ~~la~~FN~a~~~i~~~--------f~~~n~a~~~~s~~~~t~~~al~kiq~vvN~~~~al~~l~~qL~~n-----F~Ai 1039 (1356)
</pre><pre>T 7KIP_B          973 KILAASFNKAINNIVAS--------FSSVNDAITQTAEAIHTVTIALNKIQDVVNQQGSALNHLTSQLRHN-----FQAI 1039 (1356)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH--------TTTCCHHHHTTCSSHHHHHHHHHHHHHHHHHHHHHHHHHGGGGSCC-----CSSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH--------hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----hhHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   86 FVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        86 ~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      -.-=+.+++|||.
</pre><pre>T Consensus      1040 Sssi~di~~rLd~ 1052 (1356)
</pre><pre>T 7KIP_B         1040 SNSIQAIYDRLDS 1052 (1356)
</pre><pre>T ss_dssp             CSCHHHHHTTSCH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 162
</pre><pre>>3LK4_K F-actin-capping protein subunit beta isoforms 1 and 2; CapZ, CD2AP, actin filaments, uncapping, actin-filament regulators, protein-protein comple, Actin capping, Actin-binding, Cytoplasm, Cytoskeleton, PROTEIN BINDING; 1.99A {Gallus gallus}
</pre><pre>Probab=30.01  E-value=2.5e+02  Score=27.84  Aligned_cols=56  Identities=9%  Similarity=-0.100  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHH---HHHHHHHHHHHHHhhc
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKL---FSELDVLLNKLSRLTL  108 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~L---f~~Ld~~L~~~~~~~~  108 (262)
</pre><pre>                      ..+...|-+.++++|..|+..+.. ...++...|..-|++|   -.|++..+..-++..+
</pre><pre>T Consensus       207 ~~~a~~i~~~Ie~~E~~~q~~L~~~y~~~~~~~fk~LRR~lPitr~ki~~~~~~~l~~~~  266 (277)
</pre><pre>T 3LK4_K          207 SPHIANIGRLVEDMENKIRSTLNEIYFGKTKDIVNGLRSIDAIPDNQKYKQLQRELSQVL  266 (277)
</pre><pre>T ss_dssp             SCHHHHHHHHHHHHHHHHHHHHCCCCCCCHHHHHHTTC----------------------
</pre><pre>T ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHccccHHHHHHhhcCCCCcchHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 163
</pre><pre>>2BEZ_C Spike glycoprotein; COILED COIL, MEMBRANE FUSION, SEVERE ACUTE RESPIRATORY SYNDROME, VIRAL PROTEIN; 1.6A {Human SARS coronavirus} SCOP: h.3.3.1
</pre><pre>Probab=29.97  E-value=5.8e+02  Score=21.30  Aligned_cols=73  Identities=12%  Similarity=0.011  Sum_probs=0.0  Template_Neff=3.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   25 NFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        25 ~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      ++|.+|-..|..-+..+-..+..........+.+|.+...+=-.+... +...=..++..--.-=+.+|+|||.
</pre><pre>T Consensus         1 dvLq~NQk~lA~sFN~Aig~i~~~f~~~~~AL~kIQ~VVN~q~~aL~~-L~~qL~~NFgAISssi~dIy~RLd~   73 (77)
</pre><pre>T 2BEZ_C            1 NVLYENQKQIANQFNKAISQIQESLTTTSTALGKLQDVVNQNAQALNT-LVKQLSSNFGAISSVLNDILSRLDK   73 (77)
</pre><pre>T ss_dssp             CCSSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CchhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHHcHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 164
</pre><pre>>5AMO_A NOELIN; SIGNALING PROTEIN, OLFM1, DISULFIDE, NEUROBIOLOGY, DEVELOPMENT, AMPA RECEPTOR, BETA PROPELLER; HET: GOL, NAG; 2.4A {MUS MUSCULUS}
</pre><pre>Probab=29.95  E-value=1.3e+03  Score=25.36  Aligned_cols=116  Identities=16%  Similarity=0.222  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcC---CHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAEL---NTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L---~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      |+.+|++.++.+++.++.+   ..+..+.+.+.-+.|.++..--...-...-......|+.+...+++++.+-.+.   +
</pre><pre>T Consensus        77 ~l~~l~~~~~~~~~~~e~~~~k~~~~~~~~~~~~~~l~nlt~rl~~~e~~~~~y~~l~f~~l~~e~~el~~l~~~l---~  153 (471)
</pre><pre>T 5AMO_A           77 QLRQLLEKVQNMSQSIEVLDRRTQRDLQYVEKMENQMKGLETKFKQVEESHKQHLARQFKAIKAKMDELRPLIPVL---E  153 (471)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHhhHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHhhhHHhcHHhchhHhHHHHHHHHHHHHhhhHHH---H
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   78 GTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        78 g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                      ....+...+.+-+.=...+-..+.++-+. ++.-++..+++.+
</pre><pre>T Consensus       154 ~~~~~~~~i~~l~~ei~n~s~~v~~le~~-~d~~~~~~~~~~~  195 (471)
</pre><pre>T 5AMO_A          154 EYKADAKLVLQFKEEVQNLTSVLNELQEE-IGAYDYDELQSRV  195 (471)
</pre><pre>T ss_dssp             ------------------------------------------C
</pre><pre>T ss_pred             HccCChHHHHHHHHHHHHHHHHHHHHHHH-HCCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 165
</pre><pre>>PF00440.25 ; TetR_N ; Bacterial regulatory proteins, tetR family
</pre><pre>Probab=29.87  E-value=1.9e+02  Score=17.29  Aligned_cols=38  Identities=11%  Similarity=0.104  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   96 LDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        96 Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      ++..+..+......-..-+.+-+.+|++...+.|+|+.
</pre><pre>T Consensus         2 ~~~~~~~~~~~~~~~~s~~~~~~~~~~~~~~~~~~f~~   39 (47)
</pre><pre>T P95100_MYCTU/2    2 LRAARELLEETPFAELSVRAISLRAGVARSGFYFYFDS   39 (47)
</pre><pre>T ss_pred             HHHHHHHHHhCChhhCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 166
</pre><pre>>7NRC_A GCN1; Ribosome, Disome, GCN1, Translation, GAAC, ISR, Rbg2, Gir2; HET: 5CT; 3.9A {Saccharomyces cerevisiae S288C}
</pre><pre>Probab=29.82  E-value=2.9e+02  Score=31.75  Aligned_cols=157  Identities=11%  Similarity=0.061  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCHHHHHHHHHhHHHHHHHhhh--chhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHH
</pre><pre>Q WP_048874398.1   12 VNGALAELNTDEANFFNKHFSTIAALTNF--MDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVER   89 (262)
</pre><pre>Q Consensus        12 v~~al~~L~~~ea~fl~~n~~~l~~~l~~--~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R   89 (262)
</pre><pre>                      +..-++.|+|.+..+-.+--..|.++.+.  ...+ -+.++.++..++.+++.-.|+.++-.+.+.+-|...++.-+++-
</pre><pre>T Consensus       275 ~p~LLsALkDpD~rVRRqAAqALGnIGspDvk~lG-asSdtAVp~LieaLdDpdpEVRKlAAEALGkIGd~~R~aaiEKA  353 (1209)
</pre><pre>T 7NRC_A          275 IPVAVSNLDEIAWRTKRGSVQLLGNMAYLDPTQLS-ASVSTIVPEIVGVLNDSHKEVRKAADESLKRFGEVIRNAAIQKL  353 (1209)
</pre><pre>T ss_dssp             HHHHHHTSSSCCHHHHHHHHHHHHHHHHHCCSSCH-HHHHHHHHHHHHHHHCCSSHHHHHHHHHHHHHHHHSCTTSTTTH
</pre><pre>T ss_pred             HHHHHHhccccCHHHHHHHHHHHHcccCCCchhcc-cchhhhHHHHHHHhcCCCHHHHHHHHHHHHHhhhhhhHHHHHhH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHHHhhhHHHHhHHH
</pre><pre>Q WP_048874398.1   90 QKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLKMGGWLAIGFAGA  169 (262)
</pre><pre>Q Consensus        90 ~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvgi~l~~~  169 (262)
</pre><pre>                      ...+-|.-+.-++.+|+...---..+.-+.++=.+-.++.|.-+.|   +.+-..+|++     |..|.+|.+.|-.|..
</pre><pre>T Consensus       354 VpvLLqAL~Dpd~~VRrAAAqAIqtqfvh~i~~~s~alii~ii~~~---m~dr~~~ik~-----~a~~~~g~ma~~~~t~  425 (1209)
</pre><pre>T 7NRC_A          354 VPVLLQAIGDPTKYTEEALDSLIQTQFVHYIDGPSLALIIHIIHRG---MHDRSANIKR-----KACKIVGNMAILVDTK  425 (1209)
</pre><pre>T ss_dssp             HHHHHHTSSCSSSTHHHHHHHHHHTCCCCCCSSHHHHHHHHHHHHT---TSSSCSHHHH-----HHHHHHHHHHTSSCCC
</pre><pre>T ss_pred             HHHHHHHhCCCChHHHHHHHHHHHHhhhhhcCChhHHHHHHHHHHh---hcccchhHHH-----HHHHHHHhcccccchh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHH
</pre><pre>Q WP_048874398.1  170 NTTNEVYD  177 (262)
</pre><pre>Q Consensus       170 ~s~~~I~e  177 (262)
</pre><pre>                      .-..-+++
</pre><pre>T Consensus       426 dlipylq~  433 (1209)
</pre><pre>T 7NRC_A          426 DLIPYLQQ  433 (1209)
</pre><pre>T ss_dssp             CTHHHHHH
</pre><pre>T ss_pred             hhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 167
</pre><pre>>TIGR02959 SigZ; RNA polymerase sigma factor, SigZ family. This family of RNA polymerase sigma factors is a member of the Sigma-70 subfamily (TIGR02937).
</pre><pre>Probab=29.79  E-value=4.5e+02  Score=22.26  Aligned_cols=64  Identities=16%  Similarity=0.173  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      ......+..+.+...+..|++++...+.-+|  ++.+ ...-+.-.|.....+.+++.++.+.|+++
</pre><pre>T Consensus        83 ~~~~~~~~~~~l~~~~~~L~~~~~~i~~l~~--~~~~s~~eIa~~l~is~~~v~~~~~~~~~~l~~~  147 (170)
</pre><pre>T TIGR02959        83 EETFVKELSQCIPPMIKELPDEYREAIRLTE--LEGLSQQEIAEKLGLSLSGAKSRVQRGRKKLKEL  147 (170)
</pre><pre>T ss_pred             hhHHHHHHHHhHHHHHHhCCHHHHHHHHHHH--HcCCCHHHHHHHHCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 168
</pre><pre>>PF01442.20 ; Apolipoprotein ; Apolipoprotein A1/A4/E domain
</pre><pre>Probab=29.77  E-value=7.2e+02  Score=22.34  Aligned_cols=100  Identities=4%  Similarity=-0.082  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..+....+.+++.|.+..++-...+..+.+-+..-+......+..    +.++++.+...|...-..++..+..+..--.
</pre><pre>T Consensus        84 ~~l~~~~~~l~~~L~~~~~~l~~~l~~~~~~l~~~l~~~~~~l~~----l~~~~~~l~~~l~~~~~~~~~~l~~~~~~l~  159 (192)
</pre><pre>T A4YY02_BRASO/1   84 TAITKSLKDLEQASRAAIEQSRQVSVAAVTEMQETSKILRTDTVA----LFERLREGNILLQEVLTGAHDNLNSLERALV  159 (192)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHH---HHHHh
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLN---KLSRL  106 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~---~~~~~  106 (262)
</pre><pre>                      ..+...-..+-.+|+....   .+..+
</pre><pre>T Consensus       160 ~~l~~~~~~l~~~l~~~~~~~~~l~~~  186 (192)
</pre><pre>T A4YY02_BRASO/1  160 ARVADFVTAMNDVTSRNGTATQTLEDQ  186 (192)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhhccchhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 169
</pre><pre>>3DYT_A Sorting nexin-9; 3-helix bundle, BAR domain, PX domain, Phosphoprotein, Protein transport, SH3 domain, Transport, TRANSPORT PROTEIN; HET: MSE; 2.08A {Homo sapiens}
</pre><pre>Probab=29.68  E-value=3.5e+02  Score=25.91  Aligned_cols=142  Identities=11%  Similarity=0.027  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHH
</pre><pre>Q WP_048874398.1    5 LQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQ   84 (262)
</pre><pre>Q Consensus         5 L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~   84 (262)
</pre><pre>                      |...-..+.+++..|......--.+....+...+...+......+.....+.+.....|.+.-.-|.+....     ..+
</pre><pre>T Consensus       202 l~~~~~~l~~~l~~l~~~~~~~~~~~~~~l~~~~~~~a~~~~~~~~~~~~~~~~~~~~l~~~l~~~~~~~~~-----~~~  276 (366)
</pre><pre>T 3DYT_A          202 LPKEYQKIGKALQSLATVFSSSGYQGETDLNDAITEAGKTYEEIASLVAEQPKKDLHFLMECNHEYKGFLGC-----FPD  276 (366)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHTTCCGGGHHHHHHHHHHHHHHHHHHHHHHHSGGGTHHHHHHHHHHHHHHHTT-----HHH
</pre><pre>T ss_pred             ccHHHHHHHHHHHHHHHHhccCCCCCcchHHHHHHHHHHHHHHHHHHHHhCCccchHHHHhhcHHhHHHHhh-----hHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHH
</pre><pre>Q WP_048874398.1   85 FFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASK  151 (262)
</pre><pre>Q Consensus        85 F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak  151 (262)
</pre><pre>                      .+.+|...+.++...-.-..+......+....++.+-.-+..+...+..--......+.+.+...+.
</pre><pre>T Consensus       277 ~l~~r~~~l~k~~~~~k~~~~~~~~~~~~~~~~~~~~~i~~~~~~E~~~f~~~~~~dl~~~l~~~a~  343 (366)
</pre><pre>T 3DYT_A          277 IIGTHKGAIEKVKESDKLVATSKITLQDKQNMVKRVSIMSYALQAEMNHFHSNRIYDYNSVIRLYLE  343 (366)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTHHHHHHTTSSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHhccCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 170
</pre><pre>>6LOD_D Uncharacterized protein ActD; quinol:electron acceptor oxidoreductase, PHOTOSYNTHESIS; HET: EL6, HEC, SF4; 3.2A {Roseiflexus castenholzii (strain DSM 13941 / HLO8)}
</pre><pre>Probab=29.67  E-value=5e+02  Score=23.43  Aligned_cols=56  Identities=13%  Similarity=0.105  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             hchhHHHHHHHHHHHHHHHHH-HHHhcccCc------chHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  200 ASLAGGIYGGKYGAMAGAGVC-VALGVATGG------VGLLACSVVGAAAGGYIGGEVAGGAT  255 (262)
</pre><pre>Q Consensus       200 G~~~Gg~~gg~~g~~~a~~vc-~~~G~~t~G------~g~l~c~vv~~g~g~~~gg~~G~~~G  255 (262)
</pre><pre>                      +.+.|+.+|..++......+- ....+..+|      +..+...+.+.++|+..|+-.|.-.+
</pre><pre>T Consensus        73 ~~~~g~~~G~~~g~~~~~~~~~~~~~~~~~g~~~~~~~~~~~~~~~~~~~g~~~g~~~g~l~~  135 (192)
</pre><pre>T 6LOD_D           73 VIFGAGLLGASGMFALQTWINLVEYPLNIGGRPLFSWPAFIPATFEGMVLLSAFAAVFGMIAA  135 (192)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHCCCCCCCCCCCTTCCSSCSTTSHHHHSHHHHHHHHTTTSTTHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHccCCcccCCCCCCccccHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 171
</pre><pre>>PF15449.8 ; Retinal ; Retinal protein
</pre><pre>Probab=29.54  E-value=1e+03  Score=29.82  Aligned_cols=85  Identities=14%  Similarity=0.187  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhH----------------HHHHHHhhhch----hhhHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHF----------------STIAALTNFMD----KSLGVVADAGERYFSEIEKKLKAIELA   69 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~----------------~~l~~~l~~~s----~~~G~~s~~~~~~~~~i~~~L~~ie~l   69 (262)
</pre><pre>                      ++||+.|.++.+|--..|.+|-                |+|+.+|.|+-    ...|+++...+..|++.-+-|..+..+
</pre><pre>T Consensus       227 EEINQlLeEIA~dGE~LLkEvg~~LAWP~~k~~~~e~PDLLQQLLQYTv~KMq~v~gTVasLt~saLEea~~Yf~S~s~~  306 (1264)
</pre><pre>T G3QKM9_GORGO/1  227 EEIVQLLGEISKDGEVLLQEVREDLAWPLKKREPQEQPDLLQQLLQYTVSKLQVLNGTVASLTGSFLEGSSSYLHSTATH  306 (1264)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhHhccccccccCCCccCCcHHHHHHHHHHHHHHHhHHHHhHhcHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhcCCCccHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   70 YQNQYRTQGTLIGQQFFVERQKLFSELDVL   99 (262)
</pre><pre>Q Consensus        70 y~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~   99 (262)
</pre><pre>                      .++-++.     ...|-++-.+++.+|+..
</pre><pre>T Consensus       307 LeeKLka-----Kr~~eeRL~r~L~~LEaa  331 (1264)
</pre><pre>T G3QKM9_GORGO/1  307 LENKLST-----KRNVDERLLRALGQLESL  331 (1264)
</pre><pre>T ss_pred             HHHHHHH-----HHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 172
</pre><pre>>6PZ8_A S protein; MERS-CoV, coronavirus, DPP4, receptor-binding, membrane fusion, VIRAL PROTEIN-IMMUNE SYSTEM complex; HET: BMA, MAN, NAG; 4.19A {Middle East respiratory syndrome-related coronavirus}
</pre><pre>Probab=29.49  E-value=7.5e+02  Score=27.45  Aligned_cols=73  Identities=11%  Similarity=0.054  Sum_probs=0.0  Template_Neff=3.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   25 NFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        25 ~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      +.|.+|-..|.+-+..+-..+...-......+.+|++...+--.+..+ +-..=..++...-.-=+.+++|||.
</pre><pre>T Consensus       237 ~VL~~NQk~iAnsFN~Ai~~I~~gF~~t~~AL~KIQ~VVN~q~~aL~~-L~~qL~~NFgAISssi~dIy~RLd~  309 (472)
</pre><pre>T 6PZ8_A          237 QVLSENQKLIANKFNQALGAMQTGFTTTNEAFHKVQDAVNNNAQALSK-LASELSNTFGAISASIGDIIQRLDV  309 (472)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHTHHHHSSTTCHHHHHHHHHHHHHHHHHHH-HHHGGGCCSSSSCSCHHHHHHHSCH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHhhchhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 173
</pre><pre>>3RD3_A Probable transcriptional regulator; transcriptional regulator, TRANSCRIPTION; 2.4A {Pseudomonas aeruginosa}
</pre><pre>Probab=29.20  E-value=3.5e+02  Score=22.26  Aligned_cols=53  Identities=21%  Similarity=0.247  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   78 GTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        78 g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      +....+..-..|+++   |+..+.-+..++..--.-.+|-+..|+|..++-|+|..
</pre><pre>T Consensus         1 ~~~~~~~~~~~r~~i---l~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   53 (197)
</pre><pre>T 3RD3_A            1 GSHMKTSYDDTRQHL---LDTGYRIMAVKGFSGVGLNEILQSAGVPKGSFYHYFKS   53 (197)
</pre><pre>T ss_dssp             ---------CHHHHH---HHHHHHHHHHHCSTTCCHHHHHHHHTCCHHHHTTTCSC
</pre><pre>T ss_pred             CCCcccchHHHHHHH---HHHHHHHHHHhCCCCCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 174
</pre><pre>>7M0I_A Fusion glycoprotein F2; VIRAL PROTEIN; HET: NAG; 2.81A {Human metapneumovirus}
</pre><pre>Probab=29.09  E-value=1e+02  Score=25.77  Aligned_cols=42  Identities=7%  Similarity=0.084  Sum_probs=0.0  Template_Neff=2.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHH
</pre><pre>Q WP_048874398.1   50 DAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQK   91 (262)
</pre><pre>Q Consensus        50 ~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~   91 (262)
</pre><pre>                      +.+.+-++.-+++|.||..+.+++.....+...|++..+|++
</pre><pre>T Consensus        47 SLIk~EL~~~knAv~ELktvs~~~~a~~~r~~~pr~~~~~~~   88 (89)
</pre><pre>T 7M0I_A           47 SLIKTELDLTKSALRELKTVSADQLAREEQIENPRQSKKRKR   88 (89)
</pre><pre>T ss_pred             chhHHHHHHhHHHHHHHHHhccchhhHHHHhhCchhhhhccc
</pre><pre>
</pre><pre>
</pre><pre>No 175
</pre><pre>>PF14637.8 ; FNIP_M ; Folliculin-interacting protein middle domain
</pre><pre>Probab=28.96  E-value=3.6e+02  Score=26.34  Aligned_cols=88  Identities=13%  Similarity=0.115  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----------------HhcCCCccH
</pre><pre>Q WP_048874398.1   21 TDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQY-----------------RTQGTLIGQ   83 (262)
</pre><pre>Q Consensus        21 ~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~-----------------~~~g~l~~~   83 (262)
</pre><pre>                      ++..+|+.+||++|+..+..--..+-.+........+.|.++..+..+-..+-+                 .........
</pre><pre>T Consensus        23 ~~~~~f~f~H~~lie~~l~~L~~~v~~a~~~~~~f~~~i~~a~~~f~~~i~~L~~aPRl~~PvWl~l~~~~~~~~~~~~~  102 (224)
</pre><pre>T A8XJ57_CAEBR/4   23 FKHRSFLFQHIPVIEAEMAVLEARIKKSASRKRVFMSYMYNSWGEMMEKMCQIYNAPRLELPYWLALNGKCQLDERTLAV  102 (224)
</pre><pre>T ss_pred             HHHHHHHhhcHHHHHHHHHHHHHHHHHHhcCcchhHHHHHHHHHHHHHHHHHHHhCcccCcCHHHHhcCCCCccchhHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhhc
</pre><pre>Q WP_048874398.1   84 QFFVERQKLFSELDVLLNKLSRLTL  108 (262)
</pre><pre>Q Consensus        84 ~F~~~R~~Lf~~Ld~~L~~~~~~~~  108 (262)
</pre><pre>                      .|..+=..|.+++|+.=.+|+-..+
</pre><pre>T Consensus       103 ~f~~~l~~L~~~~dtK~t~fFlStL  127 (224)
</pre><pre>T A8XJ57_CAEBR/4  103 EFCEHLAYLVRRYDRKETGFFLSNL  127 (224)
</pre><pre>T ss_pred             HHHHHHHHHHHHhCCcccCChHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 176
</pre><pre>>2RT6_A Primosomal replication protein N''; Primosome, Replication restart, PriC, DNA BINDING PROTEIN; NMR {Escherichia coli}
</pre><pre>Probab=28.74  E-value=5.9e+02  Score=21.00  Aligned_cols=76  Identities=17%  Similarity=0.098  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      +|++|..+++.++..+.            +...|+.-+-....      +.+..|+++|+.+|..|...-...       
</pre><pre>T Consensus        13 ~l~~L~~~a~~~d~~~~------------~~~~Fd~~LF~~~~------~~l~~Yv~Ei~~~l~~L~~~~~~~-------   67 (98)
</pre><pre>T 2RT6_A           13 QLATLRQRCAPVSQFAT------------LSARFDRHLFQTRA------TTLQACLDEAGDNLAALRHAVEQQ-------   67 (98)
</pre><pre>T ss_dssp             HHHHHHHHTSTTTTTCC------------CCTTCCTTTSCSSC------CSHHHHHHHHHHHHHHHHHHGGGT-------
</pre><pre>T ss_pred             HHHHHHHHHhhHhhHhh------------hhHhhhHHhcCCCC------CcHHHHHHHHHHHHHHHHHHHHcC-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELDVLLNKLSR  105 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld~~L~~~~~  105 (262)
</pre><pre>                       ..+   ...-|-.+|..++..+.+
</pre><pre>T Consensus        68 -~~~---~~~~l~erl~~Q~~aL~~   88 (98)
</pre><pre>T 2RT6_A           68 -QLP---QVAWLAEHLAAQLEAIAR   88 (98)
</pre><pre>T ss_dssp             -CHH---HHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             -CHH---HHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 177
</pre><pre>>6IN7_B RNA polymerase sigma-H factor; Sigma factor, TRANSCRIPTION; 1.96A {Pseudomonas aeruginosa (strain ATCC 15692 / DSM 22644 / CIP 104116 / JCM 14847 / LMG 12228 / 1C / PRS 101 / PAO1)}
</pre><pre>Probab=28.68  E-value=6e+02  Score=21.11  Aligned_cols=68  Identities=6%  Similarity=0.007  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    4 QLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus         4 ~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      ...+....+...+..|++++...+..+|  ++.+ ...-+...|.+...+.+++.++.+.|++.-.-+..+
</pre><pre>T Consensus       124 ~~~~~~~~~~~~~~~l~~~~~~i~~~~~--~~g~s~~eia~~lg~s~~~v~~~~~r~~~~l~~~l~~~~~e  192 (193)
</pre><pre>T 6IN7_B          124 LRDEIEATVHQTIQQLPEDLRTALTLRE--FEGLSYEDIATVMQCPVGTVRSRIFRAREAIDKALQPLLRE  192 (193)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTSCHHHHHHHHHHH--TTCCCHHHHHHHHTSCHHHHHHHHHHHHHHHHHHHGGGC--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHcCCHHHHHHHHHHH--HCCCCHHHHHHHhCCCHHHHHHHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 178
</pre><pre>>PF10824.10 ; T7SS_ESX_EspC ; Excreted virulence factor EspC, type VII ESX diderm
</pre><pre>Probab=28.67  E-value=4.2e+02  Score=19.33  Aligned_cols=87  Identities=14%  Similarity=0.155  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHH--HHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANF--FNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~f--l~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ..|...++.+.....+|.......  ....-.....+..............|.+.++.+.+.|.++.+.....       
</pre><pre>T Consensus        10 ~~l~~~a~~l~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~l~~~-------   82 (99)
</pre><pre>T A4FPN7_SACEN/1   10 EGLRSHASKVDGVVDRLNTAVDASREVTLNNDAYGILCQPFALMIESVEQNGVETLRKAVDAMREVADNVRAT-------   82 (99)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhhccCCCcchhhhHHHHHhHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                       ...|.......-..++
</pre><pre>T Consensus        83 -a~~y~~~d~~~~~~~~   98 (99)
</pre><pre>T A4FPN7_SACEN/1   83 -AEEYQAVDDDNSSMLG   98 (99)
</pre><pre>T ss_pred             -HHHHHHHHHhHHhhhc
</pre><pre>
</pre><pre>
</pre><pre>No 179
</pre><pre>>PF04353.15 ; Rsd_AlgQ ; Regulator of RNA polymerase sigma(70) subunit, Rsd/AlgQ
</pre><pre>Probab=28.59  E-value=7.3e+02  Score=22.00  Aligned_cols=91  Identities=7%  Similarity=-0.012  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHcCCHH-HHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHH
</pre><pre>Q WP_048874398.1   12 VNGALAELNTD-EANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQ   90 (262)
</pre><pre>Q Consensus        12 v~~al~~L~~~-ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~   90 (262)
</pre><pre>                      +...|..+... ..-...+|+.+...+......-.........++..++..+-+.+.+-..+.-.....-+..+|...-.
</pre><pre>T Consensus        51 ~~~~L~~~~~~L~~HL~~Ed~~lYp~L~~~~~~~~~~~~~~~~~~~~em~~i~~~~~~F~~ky~~~~~~~~~~~F~~~~~  130 (151)
</pre><pre>T M4UIT7_9GAMM/1   51 SQKEITNFCQTLLDYTSTGHFGIYEQIILQCKLDGEQNLKIAQELYSRITTTTDTALNFNDKYAENATEIALLDFDDDLS  130 (151)
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHhcchhHHHHHHHHHHhcchHHHHHHHHHHHHHHHHHHHHHHHHHHccccccHHHHHhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   91 KLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        91 ~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      .++..|...+..
</pre><pre>T Consensus       131 ~i~~~L~~Ri~~  142 (151)
</pre><pre>T M4UIT7_9GAMM/1  131 ELGQKMEERFER  142 (151)
</pre><pre>T ss_pred             HHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 180
</pre><pre>>2H7O_A Protein kinase ypkA; YpkA, YopO, Yersinia, GDI, SIGNALING PROTEIN; 2.0A {Yersinia pseudotuberculosis}
</pre><pre>Probab=28.46  E-value=1.2e+03  Score=24.39  Aligned_cols=96  Identities=13%  Similarity=0.172  Sum_probs=0.0  Template_Neff=2.600
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-----------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-----------TNFMDKSLGVVADAGERYFSEIEKKLKAIELA   69 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-----------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~l   69 (262)
</pre><pre>                      ++..|+.+...+...+..|.+..-..+-.--+.|+.|           -.+...=--..+......+++|+....+|+..
</pre><pre>T Consensus       178 ~v~~LK~~Q~~~~a~~~~l~~~~~sW~~~A~qaLq~Fd~~~PrVkfg~~~~~~~H~qm~sa~A~~alqeVs~~~~e~~~f  257 (303)
</pre><pre>T 2H7O_A          178 QLNTLQQQQESAKAQLSILINRSGSWADVARQSLQRFDSTRPVVKFGTEQYTAIHRQMMAAHAAITLQEVSEFTDDMRNF  257 (303)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHGGGTHHHHHHHHHHHHHHCCCCCCCCCGGGHHHHHHHHHHHHHHHHHHGGGHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHcccchHHHHHHHHHhhhhcCCEEeeccchHHHHHHHHHHHHHHHHHHHHHhhhHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHh-----cCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   70 YQNQYRT-----QGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        70 y~~~~~~-----~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.+.-.-     .+++-.+..-.+|.+|-.++
</pre><pre>T Consensus       258 a~Da~plL~~LG~~pl~de~Lt~qRE~lre~~  289 (303)
</pre><pre>T 2H7O_A          258 TVDSIPLLIQLGRSSLMDEHLVEQREKLRELT  289 (303)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSSCCSCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHhHHHHHHHhCCCCCccchHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 181
</pre><pre>>6J6F_A Envelope glycoprotein; T.marenffei Mp1p, LIPID BINDING PROTEIN; 4.2A {Talaromyces marneffei PM1}
</pre><pre>Probab=28.11  E-value=1e+03  Score=23.46  Aligned_cols=100  Identities=14%  Similarity=0.071  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHhcCC
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGER--YFSEIEKKLKAIELAYQNQYRTQGT   79 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~--~~~~i~~~L~~ie~ly~~~~~~~g~   79 (262)
</pre><pre>                      ...++.....++. ..+++++|+..+...+.-+.......-..+-.--..+.+  .-..|...|..++.........--.
</pre><pre>T Consensus       204 ~~~i~~~~~~~~~-~~~ls~~da~~i~~~~~~l~~~~~~~l~~l~~kk~~f~~~g~~~~v~~~L~~l~~~~~~l~~al~~  282 (313)
</pre><pre>T 6J6F_A          204 IKATLDGVQSLQN-EPPLSSMEALALVGPVQDLSNQILLAIQNLIDKKEPLVQAGFGGKVENNLRQQEEAAQKLSELVST  282 (313)
</pre><pre>T ss_dssp             HHHHHHHHHHTTS-CCCCCHHHHHHHHSHHHHHHHHHHHHHHHHHHTTHHHHHHTCHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhhc-CCCCCHHHHHHhHHHHHHHHHHHHHHHHHHHHcHHHHHHCCChHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   80 LIGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        80 l~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      .-+++....=+.+.++++..+..
</pre><pre>T Consensus       283 k~p~~~~~~a~~~~~~i~~~f~~  305 (313)
</pre><pre>T 6J6F_A          283 KVPHELADISRQLSDGIAAGIKK  305 (313)
</pre><pre>T ss_dssp             HCCTTTHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             cCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 182
</pre><pre>>d4akrb_ e.43.1.2 (B:) automated matches {Dictyostelium discoideum [TaxId: 366501]}
</pre><pre>Probab=28.07  E-value=2.5e+02  Score=27.31  Aligned_cols=40  Identities=13%  Similarity=0.020  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHH
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKL   92 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~L   92 (262)
</pre><pre>                      +.+...|-+.+++.|..|+..+.. .-.++...|..-|+.|
</pre><pre>T Consensus       208 ~~~a~~I~~~I~~~E~~~~~~L~~~y~~~~~~~fk~LRR~l  248 (251)
</pre><pre>T d4akrb_         208 DTHCVNIGKMVEDMESKLRQTLETIYFGKTKEVVNTLRNAT  248 (251)
</pre><pre>T ss_dssp             SCHHHHHHHHHHHHHHHHHHHHHHCCCCCHHHHHHHHSCTT
</pre><pre>T ss_pred             hhHHHHHHHHHHHHHHHHHHHHHHHHccccHHHHHHHhhhc
</pre><pre>
</pre><pre>
</pre><pre>No 183
</pre><pre>>5ZUV_A Spike glycoprotein,Spike glycoprotein,inhibitor EK1; Human Coronavirus 229E, Spike protein, S2 domain, HR1 motif, pan-Coronavirus, VIRAL PROTEIN, INHIBITOR; 2.21A {Human coronavirus 229E}
</pre><pre>Probab=28.04  E-value=6.7e+02  Score=23.03  Aligned_cols=67  Identities=18%  Similarity=0.282  Sum_probs=0.0  Template_Neff=2.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH---------------hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL---------------TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~---------------l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      |...+....+=.++|..|+.+    +.+||..+...               -...-..+.++---...-++++++..+.|
</pre><pre>T Consensus        47 L~KIQ~VVN~qg~aL~~Lt~q----L~~NFgAISSsI~dIy~RLd~leAdaQ~~~~~~iN~t~ldl~~Em~rl~~~~k~l  122 (133)
</pre><pre>T 5ZUV_A           47 LNKIQDVVNQQGNSLNHLTSQ----LRQNFQAISSSIQAIYDRLDTIQSGGRGGSLDQINVTFLDLEYEMKKLEEAIKKL  122 (133)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHTCSSCCCCSCCCCCCCCCCCCCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH----HHHHHhHHHHhHHHHHHHHHHhhcCCCCCchhhhhhhhhHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHH
</pre><pre>Q WP_048874398.1   67 ELAYQN   72 (262)
</pre><pre>Q Consensus        67 e~ly~~   72 (262)
</pre><pre>                      ++-|.+
</pre><pre>T Consensus       123 nes~i~  128 (133)
</pre><pre>T 5ZUV_A          123 EESYID  128 (133)
</pre><pre>T ss_dssp             TTCCCC
</pre><pre>T ss_pred             HHHhcc
</pre><pre>
</pre><pre>
</pre><pre>No 184
</pre><pre>>5ZUV_C Spike glycoprotein,Spike glycoprotein,inhibitor EK1; Human Coronavirus 229E, Spike protein, S2 domain, HR1 motif, pan-Coronavirus, VIRAL PROTEIN, INHIBITOR; 2.21A {Human coronavirus 229E}
</pre><pre>Probab=28.04  E-value=6.7e+02  Score=23.03  Aligned_cols=67  Identities=18%  Similarity=0.282  Sum_probs=0.0  Template_Neff=2.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH---------------hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL---------------TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~---------------l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      |...+....+=.++|..|+.+    +.+||..+...               -...-..+.++---...-++++++..+.|
</pre><pre>T Consensus        47 L~KIQ~VVN~qg~aL~~Lt~q----L~~NFgAISSsI~dIy~RLd~leAdaQ~~~~~~iN~t~ldl~~Em~rl~~~~k~l  122 (133)
</pre><pre>T 5ZUV_C           47 LNKIQDVVNQQGNSLNHLTSQ----LRQNFQAISSSIQAIYDRLDTIQSGGRGGSLDQINVTFLDLEYEMKKLEEAIKKL  122 (133)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHCC------------CCCCCCCCCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH----HHHHHhHHHHhHHHHHHHHHHhhcCCCCCchhhhhhhhhHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHH
</pre><pre>Q WP_048874398.1   67 ELAYQN   72 (262)
</pre><pre>Q Consensus        67 e~ly~~   72 (262)
</pre><pre>                      ++-|.+
</pre><pre>T Consensus       123 nes~i~  128 (133)
</pre><pre>T 5ZUV_C          123 EESYID  128 (133)
</pre><pre>T ss_dssp             TTCCCC
</pre><pre>T ss_pred             HHHhcc
</pre><pre>
</pre><pre>
</pre><pre>No 185
</pre><pre>>6VSJ_A Spike glycoprotein; MHV spike, CEACAM1a, Complex, Glycoprotein, Viral protein; HET: NAG; 3.94A {Murine coronavirus (strain A59)}
</pre><pre>Probab=27.87  E-value=1.3e+03  Score=29.14  Aligned_cols=89  Identities=21%  Similarity=0.252  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHH----------
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQ----------   71 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~----------   71 (262)
</pre><pre>                      |..+++....++.||.+    --+...+|-+.|..++..=+...|    +++..+++|-+-|++||.--|          
</pre><pre>T Consensus       988 i~~Iq~gF~~tn~AL~K----IQ~VVN~~a~aL~~L~~qL~~nFg----AISssi~dIy~RLd~leAdaQvDRLInGRL~ 1059 (1275)
</pre><pre>T 6VSJ_A          988 LGAIQDGFDATNSALGK----IQSVVNANAEALNNLLNQLSNRFG----AISASLQEILTRLEAVEAKAQIDRLINGRLT 1059 (1275)
</pre><pre>T ss_dssp             HHHHHHHHSSCCHHHHH----HHHHHHHHHHHHHHHHHGGGSCCS----SSCSCHHHHHHHSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHhhc----hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             --HHHHhcCCCccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   72 --NQYRTQGTLIGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        72 --~~~~~~g~l~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                        +.|-..-..+..+-.++|+--.+|++.
</pre><pre>T Consensus      1060 aLnafVtq~l~~~~evr~sr~LA~qKiNE 1088 (1275)
</pre><pre>T 6VSJ_A         1060 ALNAYISKQLSDSTLIKVSAAQAIEKVNE 1088 (1275)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 186
</pre><pre>>2KJG_A Archaeal protein SSO6904; hypothetical protein, helical protein, METAL BINDING PROTEIN; NMR {Sulfolobus solfataricus}
</pre><pre>Probab=27.74  E-value=2.5e+02  Score=24.23  Aligned_cols=51  Identities=12%  Similarity=0.180  Sum_probs=0.0  Template_Neff=2.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhcC-----CCccHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q WP_048874398.1   56 FSEIEKKLKAIELAYQNQYRTQG-----TLIGQQFFVERQKLFSELDVLLNKLSRL  106 (262)
</pre><pre>Q Consensus        56 ~~~i~~~L~~ie~ly~~~~~~~g-----~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~  106 (262)
</pre><pre>                      .+.|+++|++|+.-|+..-.-..     --+.-+...+.+.+|.-+-..|.++..+
</pre><pre>T Consensus        21 ~~~v~~IL~eie~e~~ks~~ikssiIfvYan~~~~Vk~Nke~y~lI~~iLEKys~K   76 (99)
</pre><pre>T 2KJG_A           21 FNLVMQILDEIELDLRGSDNIKTSIIYVYSSHLDEIRKNKEFYDMIAEILQRYYKK   76 (99)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHSSCHHHHHHHHHHHTHHHHHHTHHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHcCCCCccHhHeeeehhhHHHHHhCHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 187
</pre><pre>>PF05818.14 ; TraT ; Enterobacterial TraT complement resistance protein
</pre><pre>Probab=27.72  E-value=1.5e+02  Score=28.30  Aligned_cols=41  Identities=20%  Similarity=0.275  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             chhchhHHHHHHHHHHHHHHHHHHHHhcccCcchHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  198 FGASLAGGIYGGKYGAMAGAGVCVALGVATGGVGLLACSVVGAAAGGYIGG  248 (262)
</pre><pre>Q Consensus       198 ~~G~~~Gg~~gg~~g~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~g~~~gg  248 (262)
</pre><pre>                      +.++++|+++|+.          +.......+.+.++.+++++++|..++.
</pre><pre>T Consensus        87 l~~g~~Ga~~Ga~----------~g~~~g~~~~~~~~gg~~Ga~~g~~~~~  127 (215)
</pre><pre>T Q3ILL0_PSET1/3   87 LSRGYEGVAAGSA----------VAFASGSGYRGAAAGGIIGGLVSTATNA  127 (215)
</pre><pre>T ss_pred             HhcCCccchHHHH----------HHHhcCCchhhHhHHHHHHHHhHHhhhh
</pre><pre>
</pre><pre>
</pre><pre>No 188
</pre><pre>>PF06148.13 ; COG2 ; COG (conserved oligomeric Golgi) complex component, COG2
</pre><pre>Probab=27.69  E-value=6.4e+02  Score=21.06  Aligned_cols=100  Identities=9%  Similarity=0.149  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>Q WP_048874398.1    5 LQNAKQKVNGALAELNTDEANFFNKHFSTIAAL------TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG   78 (262)
</pre><pre>Q Consensus         5 L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g   78 (262)
</pre><pre>                      |.+...+....+..|+.+-.+.+.+||+-+-..      +...-..+-.....+...++.+.+.+.+.............
</pre><pre>T Consensus        28 l~~l~~~L~~~~~~l~~~l~~~v~~~y~~~i~~~~~i~~l~~~~~~l~~~l~~l~~~~~~~~~~~~~~~~~l~~~~~~~~  107 (133)
</pre><pre>T COG2_DICDI/53-   28 LESVQKDLREYSKHLDSELIELINKEYQSFFSLSTSLVGFDTILNEFSISQSSIKSEIKSFNNEIVKVRGLVEDKLNEKK  107 (133)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhcchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNKLS  104 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~~~  104 (262)
</pre><pre>                      .++...=.-++-..|.++-..|..++
</pre><pre>T Consensus       108 ~l~~~~~~L~~~~~~~~~~~kl~~~l  133 (133)
</pre><pre>T COG2_DICDI/53-  108 SIEQKKKLLQLYISISETLNNMNHLF  133 (133)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 189
</pre><pre>>6NSI_A Manganese-binding protein; YtgA, solute-binding protein, Chlamydia trachomatis, iron acquisition, ABC transporter, METAL BINDING PROTEIN; 2.00006345565A {Chlamydia trachomatis} SCOP: c.92.2.0
</pre><pre>Probab=27.67  E-value=6.4e+02  Score=24.53  Aligned_cols=69  Identities=10%  Similarity=-0.030  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc--CCC
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ--GTL   80 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~--g~l   80 (262)
</pre><pre>                      ..++..++.+.+.|.++.|+.++++.+|++.+..-                         |+++++.+++.+...  ...
</pre><pre>T Consensus       108 ~~~~~~~~~i~~~L~~~~P~~~~~~~~n~~~~~~~-------------------------L~~l~~~~~~~~~~~~~~~~  162 (287)
</pre><pre>T 6NSI_A          108 RVWGAAVKEMAAALIQQFPQYEEDFQKNADQILSE-------------------------MEELDRWAARSLSTIPEKNR  162 (287)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHTTSCGGGC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhcHHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHhCCCccCc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ..-.+...-.-|++.+
</pre><pre>T Consensus       163 ~~i~~h~~~~yl~~~~  178 (287)
</pre><pre>T 6NSI_A          163 YLVTGHNAFSYFTRRY  178 (287)
</pre><pre>T ss_dssp             CEEBSSSCCHHHHHHH
</pre><pre>T ss_pred             EEEeechHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 190
</pre><pre>>6U7K_B Spike glycoprotein; PEDV, Spike, Coronavirus, Fusion Protein, VIRAL PROTEIN; HET: MAN, NAG, BMA; 3.14A {Porcine epidemic diarrhea virus (strain CV777)}
</pre><pre>Probab=27.53  E-value=1.1e+03  Score=29.86  Aligned_cols=80  Identities=18%  Similarity=0.203  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQF   85 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F   85 (262)
</pre><pre>                      |.-|+.-++||..+++.        |+...+.+...|.+..+++.++.+-=+-|++-=..|+.|-++.-..     +.+.
</pre><pre>T Consensus       994 ~~iA~sFN~a~~~i~~~--------f~~~n~ai~~~s~~~~t~a~Al~kiQ~vVN~~~~al~~l~~qL~~n-----F~AI 1060 (1399)
</pre><pre>T 6U7K_B          994 QLLAESFNSAIGNITSA--------FESVKEAISQTSKGLNTVAHALTKVQEVVNSQGSALNQLTVQLQHN-----FQAI 1060 (1399)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTT--------TTSCGGGCCGGGGSTTSHHHHHHHHHHHHHTTSHHHHHHHHTTSCC-----TTSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----hhhh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   86 FVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        86 ~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      -.-=+.+++|||.
</pre><pre>T Consensus      1061 Sssi~dIy~rLd~ 1073 (1399)
</pre><pre>T 6U7K_B         1061 SSSIDDIYSRLDI 1073 (1399)
</pre><pre>T ss_dssp             CSCHHHHHHSSCH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 191
</pre><pre>>PF09632.12 ; Rac1 ; Rac1-binding domain
</pre><pre>Probab=27.21  E-value=1.2e+03  Score=24.17  Aligned_cols=96  Identities=13%  Similarity=0.172  Sum_probs=0.0  Template_Neff=2.600
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-----------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL-----------TNFMDKSLGVVADAGERYFSEIEKKLKAIELA   69 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-----------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~l   69 (262)
</pre><pre>                      ++..|+.+...+...+..|.+..-..+-.--+.|+.|           -.+...=--..+......+++|+....+|+..
</pre><pre>T Consensus       168 ~v~~LK~~Q~~~~a~~~~l~~~~~sW~~~A~qaLq~Fd~~~PrVkfg~~~~~~~H~qm~sa~A~~alqeVs~~~~e~~~f  247 (293)
</pre><pre>T YPKA_YERPE/440  168 QLNTLQQQQESAKAQLSILINRSGSWADVARQSLQRFDSTRPVVKFGTEQYTAIHRQMMAAHAAITLQEVSEFTDDMRNF  247 (293)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHcccchHHHHHHHHHhhhhcCCeEeeccchHHHHHHHHHHHHHHHHHHHHHhhhHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHh-----cCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   70 YQNQYRT-----QGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        70 y~~~~~~-----~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      -.+.-.-     .+++-.+..-.+|.+|-.++
</pre><pre>T Consensus       248 a~da~plL~~LG~~pl~de~Lt~qRE~lre~~  279 (293)
</pre><pre>T YPKA_YERPE/440  248 TVDSIPLLIQLGRSSLMDEHLVEQREKLRELT  279 (293)
</pre><pre>T ss_pred             HHhHHHHHHHhCCCCCccchHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 192
</pre><pre>>PF04100.14 ; Vps53_N ; Vps53-like, N-terminal
</pre><pre>Probab=27.09  E-value=1.2e+03  Score=24.00  Aligned_cols=142  Identities=12%  Similarity=0.151  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH------hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL------TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYR   75 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~------l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~   75 (262)
</pre><pre>                      +..|+...+.++..|..+..+...-..+-..-++.+      +...=..+.......+..++++-+.+++++.++++.-.
</pre><pre>T Consensus        26 ~~~l~~~~~~ld~~l~~~~~~~~~~~~~~~~~l~~~~~~~~~L~~~i~~i~~~~~~~~~~v~~~~~~i~~ld~ak~nl~~  105 (413)
</pre><pre>T VPS53_HUMAN/40   26 VNKIRLKIRRLDDNIRTVVRGQTNVGQDGRQALEEAQKAIQQLFGKIKDIKDKAEKSEQMVKEITRDIKQLDHAKRHLTT  105 (413)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhcchhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   76 TQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARF  155 (262)
</pre><pre>Q Consensus        76 ~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~  155 (262)
</pre><pre>                      ....++.-+.+.   .+..+|+..+..        .+|...-+.| ...+.+..+...-.  ++|.+.+...++.+....
</pre><pre>T Consensus       106 t~~~L~~l~~l~---~~~~~l~~l~~~--------~~~~~~~~~l-~~~~~l~~~l~~~~--~~~~~~~~~~~l~~l~~~  171 (413)
</pre><pre>T VPS53_HUMAN/40  106 SITTLNHLHMLA---GGVDSLEAMTRR--------RQYGEVANLL-QGVMNVLEHFHKYM--GIPQIRQLSERVKAAQTE  171 (413)
</pre><pre>T ss_pred             HHHHHHHHHHHH---HHHHHHHHHHHh--------CCHHHHHHHH-HHHHHHHHHHHhcc--CCHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HH
</pre><pre>Q WP_048874398.1  156 LK  157 (262)
</pre><pre>Q Consensus       156 lk  157 (262)
</pre><pre>                      |+
</pre><pre>T Consensus       172 l~  173 (413)
</pre><pre>T VPS53_HUMAN/40  172 LG  173 (413)
</pre><pre>T ss_pred             HH
</pre><pre>
</pre><pre>
</pre><pre>No 193
</pre><pre>>5IJN_G NUCLEAR PORE COMPLEX PROTEIN NUP58; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=27.01  E-value=9.4e+02  Score=27.50  Aligned_cols=101  Identities=10%  Similarity=0.012  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh--cC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT--QG   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~--~g   78 (262)
</pre><pre>                      |...|++-.++|.+.|+..  |.+...++|-.-.+.+++..+.-...+..-+++++++..+.++|||++-...-..  ..
</pre><pre>T Consensus       304 QkesLqELmekVKqmLRNT--E~AVRsye~PpGvQ~fYSaPspFFqqTVarFEkrLeEYrQ~IEEIEQlLnS~~nss~qS  381 (599)
</pre><pre>T 5IJN_G          304 NTLNIDKLKIETAQELKNA--EIALRTQKTPPGLQHEYAAPADYFRILVQQFEVQLQQYRQQIEELENHLATQANNSHIT  381 (599)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHT--TTTGGGSSHHHHSCCCCSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHH--HHHHHHhcCCCccccccccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      +-.=+.....-...|--++.++..+
</pre><pre>T Consensus       382 PQsLpKVMSNQHD~FVhVAAKVEsL  406 (599)
</pre><pre>T 5IJN_G          382 PQDLSMAMQKIYQTFVALAAQLQSI  406 (599)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 194
</pre><pre>>5IJO_M Nucleoporin p58/p45; Nuclear pore complex, Nucleocytoplasmic transport, Transport protein; 21.4A {Homo sapiens}
</pre><pre>Probab=27.01  E-value=9.4e+02  Score=27.50  Aligned_cols=101  Identities=10%  Similarity=0.012  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh--cC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRT--QG   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~--~g   78 (262)
</pre><pre>                      |...|++-.++|.+.|+..  |.+...++|-.-.+.+++..+.-...+..-+++++++..+.++|||++-...-..  ..
</pre><pre>T Consensus       304 QkesLqELmekVKqmLRNT--E~AVRsye~PpGvQ~fYSaPspFFqqTVarFEkrLeEYrQ~IEEIEQlLnS~~nss~qS  381 (599)
</pre><pre>T 5IJO_M          304 NTLNIDKLKIETAQELKNA--EIALRTQKTPPGLQHEYAAPADYFRILVQQFEVQLQQYRQQIEELENHLATQANNSHIT  381 (599)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHT--TTTGGGSSHHHHSCCCCSSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHH--HHHHHHhcCCCccccccccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcccCCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      +-.=+.....-...|--++.++..+
</pre><pre>T Consensus       382 PQsLpKVMSNQHD~FVhVAAKVEsL  406 (599)
</pre><pre>T 5IJO_M          382 PQDLSMAMQKIYQTFVALAAQLQSI  406 (599)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 195
</pre><pre>>PF11855.10 ; DUF3375 ; Protein of unknown function (DUF3375)
</pre><pre>Probab=26.95  E-value=6.6e+02  Score=27.15  Aligned_cols=148  Identities=11%  Similarity=0.127  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      +|+.|+++..++++.+..+...+... ++..++.+.+     -.+-..+..+-.=|.+|++.++++.+--.+.+.....-
</pre><pre>T Consensus       137 Ri~~L~~~~~~i~~eI~~i~~G~~~~-l~~~~i~er~-----~~i~~la~~L~~DFr~Ve~~f~~l~r~lr~~~~~~~~~  210 (468)
</pre><pre>T Q8NLY0_CORGL/1  137 RLAELERERDRIERQIEAVHAGEFEV-LTTVQIGDRV-----ADILDLAASIPADFARVRHELSDLNRKLRRQLLDPEDS  210 (468)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHcCCCcc-CCHHHHHHHH-----HHHHHHHHhchHHHHHHHHHHHHHHHHHHHHHhCCCCc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             -------------------ccHHHHHHHHHHHH-HHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccc
</pre><pre>Q WP_048874398.1   81 -------------------IGQQFFVERQKLFS-ELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIR  140 (262)
</pre><pre>Q Consensus        81 -------------------~~~~F~~~R~~Lf~-~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip  140 (262)
</pre><pre>                                         -+..|.+=.+-|.. .....|...+..-+..|.-..|.....-.-+.+++.|...+..-+.
</pre><pre>T Consensus       211 kG~iL~~~l~~~d~l~~s~qGrsF~aF~~~L~~~~~~~~l~~~i~~il~~~~~~~L~~~~r~~L~~l~~~L~~~~~~V~~  290 (468)
</pre><pre>T Q8NLY0_CORGL/1  211 RGDVLEEIFRGVDLIGDSDAGRSFNSFFDVLLDRERSSLIDRWIREVLGRDEAIDLDSKLRTGLYRIFRDMEDASFEVNG  290 (468)
</pre><pre>T ss_pred             chHHHHHHHhcchhHhcCcccHHHHHHHHHHhCHHHHHHHHHHHHHHhcCHHHhcCCHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  141 GYSTYVDNASKAAR  154 (262)
</pre><pre>Q Consensus       141 ~y~~~i~~~ak~~k  154 (262)
</pre><pre>                      .+..-.+++.+...
</pre><pre>T Consensus       291 ~~~~l~~~Lrr~l~  304 (468)
</pre><pre>T Q8NLY0_CORGL/1  291 EMTGLARSLRHYVT  304 (468)
</pre><pre>T ss_pred             HHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 196
</pre><pre>>6ZZ5_BBB SmhB; Pore forming toxin, toxin, Bacterial toxin, Serratia marcescens; HET: SO4, EDO; 1.84A {Serratia marcescens}
</pre><pre>Probab=26.93  E-value=8.6e+02  Score=24.53  Aligned_cols=68  Identities=15%  Similarity=0.148  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHH--HHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTD--EANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIEL   68 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~--ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~   68 (262)
</pre><pre>                      ++++++++-++..+.+..+..+  ....+...++.+...+......+....+.|...-.++.+.+..++.
</pre><pre>T Consensus       232 ~~~~~~~~i~~l~~~l~~~~~~~~~l~~~~~~l~~l~~~~~~a~~~l~~l~~~W~~l~~~l~~l~~~l~~  301 (366)
</pre><pre>T 6ZZ5_BBB        232 VLDKSLSAREKLYRDRSQLNSEVLVASQIGSGYRGLQTQAQSAVTAATQMNNAWDSLTSELETLNANLRK  301 (366)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTTTTSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 197
</pre><pre>>7JMN_O MED15; Transcription, Activation, PolII;{Chaetomium thermophilum (strain DSM 1495 / CBS 144.50 / IMI 039719)}
</pre><pre>Probab=26.86  E-value=9.9e+02  Score=30.82  Aligned_cols=104  Identities=14%  Similarity=0.170  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHc-------CCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHH--HHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAE-------LNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELA--YQN   72 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~-------L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~l--y~~   72 (262)
</pre><pre>                      ++.|++-..++.+.+..       |++|+.+.+..                         .|.++...|.+|++.  ...
</pre><pre>T Consensus      1026 ~~rlk~i~~E~~~~~~~~~~~~Ipmspe~~~~~~~-------------------------~L~~~~~~l~kv~~~~~l~~ 1080 (1625)
</pre><pre>T 7JMN_O         1026 LQKLRNLASEAQLEVSKEHLQPILMTPADQQETRE-------------------------KIAQVKLKMNQIRSMGLLPK 1080 (1625)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhhccCCCCCCCCHHHHHHHHH-------------------------HHHHHHHHHHHHHhhccHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHh-cCCCccHHHHHHHHHHHHHHHHHHH-HHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHH
</pre><pre>Q WP_048874398.1   73 QYRT-QGTLIGQQFFVERQKLFSELDVLLN-KLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNAS  150 (262)
</pre><pre>Q Consensus        73 ~~~~-~g~l~~~~F~~~R~~Lf~~Ld~~L~-~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~a  150 (262)
</pre><pre>                      .|.. +.--.-..||+-|-+|++++...-. .-++-.+.|                           ...++..++..+.
</pre><pre>T Consensus      1081 wy~~t~de~r~R~ffr~R~rl~~Qf~D~~~~~~lkd~fSI---------------------------t~~el~~~~~~l~ 1133 (1625)
</pre><pre>T 7JMN_O         1081 WYYLTNDDNRARQFFRGRFKVVKQFTDGEAMTQLRDTFTI---------------------------TKEELNQYDNLLG 1133 (1625)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHhcCCHHHHHHHHHHHHHHHHHccChhhhcccccccee---------------------------CHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHH
</pre><pre>Q WP_048874398.1  151 KAARFLK  157 (262)
</pre><pre>Q Consensus       151 k~~k~lk  157 (262)
</pre><pre>                      ++.+-|+
</pre><pre>T Consensus      1134 ~m~~dl~ 1140 (1625)
</pre><pre>T 7JMN_O         1134 SMLRDMM 1140 (1625)
</pre><pre>T ss_dssp             -------
</pre><pre>T ss_pred             HHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 198
</pre><pre>>TIGR02952 Sig70_famx2; RNA polymerase sigma-70 factor, TIGR02952 family. This group of sigma factors are members of the sigma-70 family (TIGR02937).
</pre><pre>Probab=26.82  E-value=3.6e+02  Score=22.18  Aligned_cols=59  Identities=22%  Similarity=0.161  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      .+....+.+.+..|++.+...+.-+|  ++.. ...-+...|.+...+.++++++.+.|.+.
</pre><pre>T Consensus       110 ~~~~~~l~~~l~~L~~~~~~il~l~~--~~g~s~~eia~~l~is~~~v~~~~~~~~~~l~~~  169 (170)
</pre><pre>T TIGR02952       110 EEANEKLLKALKILTPKQQHVIALRF--GQNLPIAEVARILGKTEGAVKILQFRAIKKLARQ  169 (170)
</pre><pre>T ss_pred             HHHHHHHHHHHhcCCHHHHHHHHHHh--cCCCCHHHHHHHHCCCHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 199
</pre><pre>>1EQ1_A APOLIPOPHORIN-III; Five helix-bundle, "helix-short helix-helix" recognition motif, LIPID BINDING PROTEIN; NMR {Manduca sexta} SCOP: a.63.1.1
</pre><pre>Probab=26.66  E-value=9e+02  Score=22.47  Aligned_cols=101  Identities=7%  Similarity=-0.002  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHcCCHHHHHHH--HHhHHHHHHHhhhch-------hhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    4 QLQNAKQKVNGALAELNTDEANFF--NKHFSTIAALTNFMD-------KSLGVVADAGERYFSEIEKKLKAIELAYQNQY   74 (262)
</pre><pre>Q Consensus         4 ~L~~~A~~v~~al~~L~~~ea~fl--~~n~~~l~~~l~~~s-------~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~   74 (262)
</pre><pre>                      .|.+-...+++.+..+..+--..+  -++=+++..+-..+.       ..+.-.-..+..+-.++.+.|+++..-..+..
</pre><pre>T Consensus         8 ~l~el~~~aq~~~~~~~~~v~~~l~~p~~eev~~~lke~s~~~~~~l~~~~~klq~~~~~~~~e~k~~l~~~~~kl~e~~   87 (166)
</pre><pre>T 1EQ1_A            8 AFEEMEKHAKEFQKTFSEQFNSLVNSKNTQDFNKALKDGSDSVLQQLSAFSSSLQGAISDANGKAKEALEQARQNVEKTA   87 (166)
</pre><pre>T ss_dssp             CSTHHHHHHHHHHHHHHHHHHHHTSSCCSSCSSHHHHHHHHHHHHHHHHHHHHHHHHHTTCSSTHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHHHHhCCCCHHHHHHHHHHchHHHHHHHHHHHHHHHHHHHhccHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HhcCCCccHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   75 RTQGTLIGQQFFVERQKLFSELDVLLNKLSR  105 (262)
</pre><pre>Q Consensus        75 ~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~  105 (262)
</pre><pre>                      ..-..- .|+-.++=..|-.+|+..+..++.
</pre><pre>T Consensus        88 ~~Lk~~-~Pe~~~~a~el~~~~~~~~~ti~~  117 (166)
</pre><pre>T 1EQ1_A           88 EELRKA-HPDVEKEANAFKDKLQAAVQTTVQ  117 (166)
</pre><pre>T ss_dssp             HGGGGC-SHHHHHTCSSSHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHH-CHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 200
</pre><pre>>PF05508.13 ; Ran-binding ; RanGTP-binding protein
</pre><pre>Probab=26.62  E-value=1.4e+03  Score=24.48  Aligned_cols=90  Identities=11%  Similarity=0.082  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHH--------
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQY--------   74 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~--------   74 (262)
</pre><pre>                      ++|+.-.++.+.-++-++|.        -|+|+-+..-+.+....+-........+|.+-=.+|+.+....-        
</pre><pre>T Consensus        45 ~eL~~lq~rLe~KI~IvsPA--------IDLIel~aaRGNt~Lesal~Lt~~Lr~dI~~f~~~l~~~~~~~~~~~~~~~~  116 (347)
</pre><pre>T C0NRY5_AJECG/1   45 VELQTLQERLQSKIKIISPA--------IDMIELIAARGNTSLESALALTKSLRWDIQNLGLRLSKAVSVEELYRKGAIS  116 (347)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhhHHH--------HHHHHHHhhcCChHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHHcCCcc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ---HhcCCCccHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   75 ---RTQGTLIGQQFFVERQKLFSELDVLL  100 (262)
</pre><pre>Q Consensus        75 ---~~~g~l~~~~F~~~R~~Lf~~Ld~~L  100 (262)
</pre><pre>                         ....+..-....+.=+.|+.++|..+
</pre><pre>T Consensus       117 ~~~~~~~~~~i~~I~~~ik~LL~rIdd~V  145 (347)
</pre><pre>T C0NRY5_AJECG/1  117 VTGRTKLDGEIKSIIKDIRNLLQRIEDAV  145 (347)
</pre><pre>T ss_pred             cccchhhHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 201
</pre><pre>>PF05263.13 ; DUF722 ; Protein of unknown function (DUF722)
</pre><pre>Probab=26.54  E-value=2.5e+02  Score=23.86  Aligned_cols=31  Identities=10%  Similarity=0.086  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHF   31 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~   31 (262)
</pre><pre>                      ++..|....+.|..++..|++++...+...|
</pre><pre>T Consensus        64 ~i~~l~~~~~~i~~~i~~l~~~~~~ii~~ry   94 (134)
</pre><pre>T Q9CIA0_LACLA/1   64 EYIKLKDLMYQFSLWYEPLIKEEKEIIKLKH   94 (134)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhcCCHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 202
</pre><pre>>7CYD_B Spike glycoprotein; Alphacoronavirus, spike glycoprotein, STRUCTURAL PROTEIN; HET: MAN, NAG, BMA;{Human coronavirus 229E}
</pre><pre>Probab=26.53  E-value=1.2e+03  Score=28.99  Aligned_cols=80  Identities=14%  Similarity=0.164  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQF   85 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F   85 (262)
</pre><pre>                      |.-|+.-++|+..+++.        |+-+.+.+...|.+..+++.++.+-=+-|+.-=..|+.|-++.-..     +.+.
</pre><pre>T Consensus       792 k~iA~sFN~A~~~i~~a--------f~~Vn~ai~q~s~g~~t~a~AL~kiQ~VVN~q~~aL~~L~~qL~~n-----FgAI  858 (1116)
</pre><pre>T 7CYD_B          792 KILAASFNKAMTNIVDA--------FTGVNDAITQTSQALQTVATALNKIQDVVNQQGNSLNHLTSQLRQN-----FQAI  858 (1116)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH--------HC---------CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHSC-----SSSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH--------hhhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----HhHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   86 FVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        86 ~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      -.-=+.+++|||.
</pre><pre>T Consensus       859 Sssi~dIy~RLd~  871 (1116)
</pre><pre>T 7CYD_B          859 SSSIQAIYDRLDI  871 (1116)
</pre><pre>T ss_dssp             CSCHHHHHHHCCH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 203
</pre><pre>>6U42_5S FAP106; motile cilia, microtubule doublet, repetitive structure, PROTEIN FIBRIL; HET: GDP, GTP, MG; 3.4A {Chlamydomonas reinhardtii}
</pre><pre>Probab=26.50  E-value=9e+02  Score=23.51  Aligned_cols=65  Identities=15%  Similarity=0.233  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHc------------CCHHHHHHHHH----hHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAE------------LNTDEANFFNK----HFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~------------L~~~ea~fl~~----n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      .++..+.+...+...+            |+++|+..+.+    +|+-+..-+.  ..-+..-+-..-++.+.+++.|++|
</pre><pre>T Consensus       145 ~e~~~~~e~~~~~~~~~~~~~~~~~~~~l~e~ER~~lL~~Lk~~~~el~~e~q--~lp~~~dT~~~~~rK~~lE~~L~~l  222 (240)
</pre><pre>T 6U42_5S         145 DRVAKEKEHFTQYLRMREAPEANAHVSQLSPEDRQQLVRHLKAKWGSVNTAYQ--GLSLSVDSAVKKGRKEAMERELAEI  222 (240)
</pre><pre>T ss_dssp             HHHHC------------------------CHHHHHHHHHHHHHHHHHHHHHHH--HSCSCCCSHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhccchhccccccCCHHHHHHHHHHHHHHHHHHHHHHH--cCCccccCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHH
</pre><pre>Q WP_048874398.1   67 ELA   69 (262)
</pre><pre>Q Consensus        67 e~l   69 (262)
</pre><pre>                      |+.
</pre><pre>T Consensus       223 E~d  225 (240)
</pre><pre>T 6U42_5S         223 ERD  225 (240)
</pre><pre>T ss_dssp             HHH
</pre><pre>T ss_pred             HHH
</pre><pre>
</pre><pre>
</pre><pre>No 204
</pre><pre>>6U42_5V FAP106; motile cilia, microtubule doublet, repetitive structure, PROTEIN FIBRIL; HET: MG, GTP, GDP; 3.4A {Chlamydomonas reinhardtii}
</pre><pre>Probab=26.50  E-value=9e+02  Score=23.51  Aligned_cols=65  Identities=15%  Similarity=0.233  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHc------------CCHHHHHHHHH----hHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAE------------LNTDEANFFNK----HFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~------------L~~~ea~fl~~----n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      .++..+.+...+...+            |+++|+..+.+    +|+-+..-+.  ..-+..-+-..-++.+.+++.|++|
</pre><pre>T Consensus       145 ~e~~~~~e~~~~~~~~~~~~~~~~~~~~l~e~ER~~lL~~Lk~~~~el~~e~q--~lp~~~dT~~~~~rK~~lE~~L~~l  222 (240)
</pre><pre>T 6U42_5V         145 DRVAKEKEHFTQYLRMREAPEANAHVSQLSPEDRQQLVRHLKAKWGSVNTAYQ--GLSLSVDSAVKKGRKEAMERELAEI  222 (240)
</pre><pre>T ss_dssp             HHHHC------------------------CHHHHHHHHHHHHHHHHHHHHHHH--HSCSCCCSHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhccchhccccccCCHHHHHHHHHHHHHHHHHHHHHHH--cCCccccCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHH
</pre><pre>Q WP_048874398.1   67 ELA   69 (262)
</pre><pre>Q Consensus        67 e~l   69 (262)
</pre><pre>                      |+.
</pre><pre>T Consensus       223 E~d  225 (240)
</pre><pre>T 6U42_5V         223 ERD  225 (240)
</pre><pre>T ss_dssp             HHH
</pre><pre>T ss_pred             HHH
</pre><pre>
</pre><pre>
</pre><pre>No 205
</pre><pre>>6PMI_F RNA polymerase sigma factor FliA; sigma28, transcription initiation complex, RpoF, ZNR domain, TRANSCRIPTION; HET: ATP; 3.86A {Escherichia coli O157:H7}
</pre><pre>Probab=26.50  E-value=7.9e+02  Score=22.02  Aligned_cols=69  Identities=13%  Similarity=0.009  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      .+....+.+.+..|++++...+..+|        .......-.+...+-..+.|...++.+.+..++.+.......+
</pre><pre>T Consensus       176 ~~~~~~l~~~l~~L~~~~~~vl~~~~--------~~~~s~~eiA~~lg~s~~~v~~~~~r~~~klr~~~~~~~~~~~  244 (247)
</pre><pre>T 6PMI_F          176 SNLRQRVMEAIETLPEREKLVLTLYY--------QEELNLKEIGAVLEVGESRVSQLHSQAIKRLRTKLGKLLEHHH  244 (247)
</pre><pre>T ss_dssp             HHHHHHHHHHHTTSCHHHHHHHHHHH--------SSCCCSHHHHHHHCSTTHHHHHHHHHHHHHHSSSCCCCC----
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHHh--------CCCCCHHHHHHHhCCCHHHHHHHHHHHHHHHHHHHHHhhcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 206
</pre><pre>>3S84_A Apolipoprotein A-IV; four helix bundle, TRANSPORT PROTEIN; HET: SO4; 2.4A {Homo sapiens}
</pre><pre>Probab=26.48  E-value=9.2e+02  Score=22.48  Aligned_cols=98  Identities=9%  Similarity=0.086  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..|......+...+.+...+-.+.+..+.+-+..-+......+.   ..+..+.+.+.+.|.+........+.....--.
</pre><pre>T Consensus        84 ~~l~~~~~~~~~~l~~~~~~~~~~l~~~~~~l~~~l~~~~~~l~---~~l~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~  160 (273)
</pre><pre>T 3S84_A           84 RVLRENADSLQASLRPHADELKAKIDQNVEELKGRLTPYADEFK---VKIDQTVEELRRSLAPYAQDTQEKLNHQLEGLT  160 (273)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHGGGCSSCHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHcHHHHHHhHHHHHHHHHHHHHHHHHHHHHhhHHHHHHHH---HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      ..+-.....+-.+|+.....+
</pre><pre>T Consensus       161 ~~l~~~~~~l~~~l~~~~~~~  181 (273)
</pre><pre>T 3S84_A          161 FQMKKNAEELKARISASAEEL  181 (273)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 207
</pre><pre>>4I0X_J ESAT-6-like protein MAB_3113; Structural Genomics, PSI-2, Protein Structure Initiative, Integrated Center for Structure and Function Innovation, ISFI, TB Structural; HET: GOL, BME; 1.956A {Mycobacterium abscessus}
</pre><pre>Probab=26.44  E-value=5e+02  Score=19.47  Aligned_cols=89  Identities=7%  Similarity=0.031  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHH-HHH-HhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFST-IAA-LTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~-l~~-~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g   78 (262)
</pre><pre>                      ....|...++.++.....+...-.++-..--.+ ... +...+..........|...++.+.+.|.++.......     
</pre><pre>T Consensus        12 ~~~~l~~~a~~~~~~~~~~~~~~~~~~~~~~~~~~~~~w~g~~~~a~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-----   86 (103)
</pre><pre>T 4I0X_J           12 DLDHIEQVTSRARGFKEFVTENLDQLESRAQKLVQSGQWAGAAAAAYSQAHKEWMDAARELVEGLSQMEEAARTA-----   86 (103)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTC---CCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHcCCCChhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                         ...|...-...-..+.
</pre><pre>T Consensus        87 ---~~~~~~~d~~~~~~~~  102 (103)
</pre><pre>T 4I0X_J           87 ---HGAYSEAQEANLRMAR  102 (103)
</pre><pre>T ss_dssp             ---HC--------------
</pre><pre>T ss_pred             ---HHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 208
</pre><pre>>6JX7_B Feline Infectious Peritonitis Virus Spike Protein; CoV spike protein, VIRAL PROTEIN; HET: BMA, MAN, NAG, FUC; 3.31A {Feline infectious peritonitis virus}
</pre><pre>Probab=26.42  E-value=1.1e+03  Score=30.17  Aligned_cols=80  Identities=10%  Similarity=0.140  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQF   85 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F   85 (262)
</pre><pre>                      |.-|+.-++||..+++.        |+-+.+.+...|.+..+++.++.+-=+-|+.-=..|+.|-++.-..     +.+.
</pre><pre>T Consensus      1065 k~iA~sFN~A~~~i~~A--------f~~Vn~ai~~~s~g~~t~a~AL~kiQ~VVN~~~~aL~~L~~qL~~n-----FgAI 1131 (1468)
</pre><pre>T 6JX7_B         1065 KILANAFNNAIGNITLA--------LGKVSNAITTVSDGFNSMASALTKIQSVVNQQGEALSHLISQLQKN-----FQAI 1131 (1468)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH--------HHSCCSCSCCCCSHHHHHHHHHHHHHHHTTTHHHHHHHHHGGGSCC-----TTSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH--------hhhhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----hhHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   86 FVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        86 ~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      -.-=+.+++|||.
</pre><pre>T Consensus      1132 Sssi~dIy~RLd~ 1144 (1468)
</pre><pre>T 6JX7_B         1132 SSSIAEIYNRLEK 1144 (1468)
</pre><pre>T ss_dssp             CSCHHHHTTTSCH
</pre><pre>T ss_pred             HHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 209
</pre><pre>>1TOA_B PROTEIN (PERIPLASMIC BINDING PROTEIN TROA); PERIPLASMIC BINDING PROTEIN, ZINC BINDING PROTEIN, ABC TRANSPORTER, BINDING PROTEIN; HET: GOL; 1.8A {Treponema pallidum} SCOP: c.92.2.2
</pre><pre>Probab=26.37  E-value=4.7e+02  Score=25.92  Aligned_cols=69  Identities=9%  Similarity=0.071  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc--CCC
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ--GTL   80 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~--g~l   80 (262)
</pre><pre>                      ...+..++.+.+.|.++.|+.++.+.+|++.+.                         +.|++++.-|++.+..-  ...
</pre><pre>T Consensus       144 ~~~~~~~~~Ia~~L~~~~P~~~~~~~~n~~~~~-------------------------~~L~~l~~~~~~~l~~~~~~~~  198 (313)
</pre><pre>T 1TOA_B          144 KLWSYSVKAVYESLCKLLPGKTREFTQRYQAYQ-------------------------QQLDKLDAYVRRKAQSLPAERR  198 (313)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHHHTSCGGGC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhcHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHHhcCCHhhC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ..-.+...-+-|++.+
</pre><pre>T Consensus       199 ~~v~~h~~~~yl~~~~  214 (313)
</pre><pre>T 1TOA_B          199 VLVTAHDAFGYFSRAY  214 (313)
</pre><pre>T ss_dssp             EEEEEESCCHHHHHHH
</pre><pre>T ss_pred             EEEEEcchhHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 210
</pre><pre>>d3aa0a_ e.43.1.1 (A:) Capz alpha-1 subunit {Chicken (Gallus gallus) [TaxId: 9031]}
</pre><pre>Probab=26.37  E-value=2.2e+02  Score=28.14  Aligned_cols=40  Identities=23%  Similarity=0.213  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHh-cCCCccHHHHHHHHHH
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRT-QGTLIGQQFFVERQKL   92 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~-~g~l~~~~F~~~R~~L   92 (262)
</pre><pre>                      ..+...|-+.+++.|..|+..+.. ...++...|..-|++|
</pre><pre>T Consensus       216 ~~~a~~iv~~I~~~E~~~q~~l~~~y~~l~~~~fk~LRR~L  256 (270)
</pre><pre>T d3aa0a_         216 VQTAKEFIKIIENAENEYQTAISENYQTMSDTTFKALRRQL  256 (270)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHCCCCCHHHCCSS
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHhhcCHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 211
</pre><pre>>TIGR02895 spore_sigI; RNA polymerase sigma-I factor. Members of this sigma factor protein family are strictly limited to endospore-forming species in the Firmicutes lineage of bacteria, but are not universally present among such species.
</pre><pre>Probab=26.35  E-value=6.1e+02  Score=22.27  Aligned_cols=68  Identities=10%  Similarity=-0.090  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      ++....+.+.+..|+|.+...+..+|-        ......-.+..++-.-+.|.+.++.+-+-.++..+..+.++
</pre><pre>T Consensus       148 ~~~~~~l~~~l~~L~~~~r~il~l~~~--------~g~s~~eIA~~lgis~~~v~~~~~~~~~~lr~~~~~~~~~~  215 (218)
</pre><pre>T TIGR02895       148 RDTRKKAIKIAKVIVENEELLEYLIRK--------KKLPIKEIEERVRISRKTIERYRKYIIALVIILSGDYTYLK  215 (218)
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHHHH--------CCCCHHHHHHHHCCCHHHHHHHHHHHHHHHHHHHccchhHH
</pre><pre>
</pre><pre>
</pre><pre>No 212
</pre><pre>>7BUL_A General transcription factor IIH subunit 1; DNA repair factor, general transcription factor, nucleotide excision repair, NUCLEAR PROTEIN; NMR {Homo sapiens}
</pre><pre>Probab=26.32  E-value=8.9e+02  Score=22.31  Aligned_cols=67  Identities=9%  Similarity=0.074  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             HHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   28 NKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSE   95 (262)
</pre><pre>Q Consensus        28 ~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~   95 (262)
</pre><pre>                      ....|.+.+.+...-.........=-+.-.++-..=.+|.++|++... .+.+...+|.+.|..|+.+
</pre><pre>T Consensus        89 ~~drd~~kd~l~~~i~~~~~~~~~e~~~r~~LL~~np~L~~l~~elV~-~g~ise~eFW~~R~~Ll~~  155 (161)
</pre><pre>T 7BUL_A           89 VKERDAVKDLLQQLLPKFKRKANKELEEKNRMLQEDPVLFQLYKDLVV-SQVISAEEFWANRLNVNAT  155 (161)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTTTTCCCCCHHHHHHHHHHHHCHHHHHHCCCCCT-TTSSCHHHHHHHHHTCCCH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhHhhchHHHHHHHHHhhhCHHHHHHHHHHHH-cCCCCHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 213
</pre><pre>>3H6P_C ESAT-6-like protein esxR; Four-helix bundle, Structural Genomics, PSI-2, Protein Structure Initiative, TB Structural Genomics Consortium, TBSGC, Integrated Center for; 1.91A {Mycobacterium tuberculosis}
</pre><pre>Probab=26.28  E-value=5e+02  Score=19.34  Aligned_cols=88  Identities=5%  Similarity=0.003  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|+..+..+......+...-.++....-.+...+-..++.........|.+.++.+.+.|.++.....+.       
</pre><pre>T Consensus         7 ~~~~l~~~~~~l~~~~~~~~~~~~~~~~~~~~~~~~w~G~~~~a~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   79 (96)
</pre><pre>T 3H6P_C            7 NYPAMMAHAGDMAGYAGTLQSLGADIASEQAVLSSAWQGDTGITYQGWQTQWNQALEDLVRAYQSMSGTHESN-------   79 (96)
</pre><pre>T ss_dssp             -------------CHHHHHHHHHHHHHHHHHHTGGGCCTTSSSCHHHHHHHHHHHHHHHHHHHHHHHCC-----------
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHcCchhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                       ...|.......-..+
</pre><pre>T Consensus        80 -~~~~~~~d~~~~~~~   94 (96)
</pre><pre>T 3H6P_C           80 -TMAMLARDGAEAAKW   94 (96)
</pre><pre>T ss_dssp             ----------------
</pre><pre>T ss_pred             -HHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 214
</pre><pre>>6GY7_A XaxB; bacterial toxin, pore forming-toxins, TOXIN; 3.4A {Xenorhabdus nematophila (strain ATCC 19061 / DSM 3370 / LMG 1036 / NCIB 9965 / AN6)}
</pre><pre>Probab=26.24  E-value=1.3e+03  Score=24.04  Aligned_cols=145  Identities=13%  Similarity=0.120  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHH-HHHHHHHhHHHHHHH--hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTD-EANFFNKHFSTIAAL--TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~-ea~fl~~n~~~l~~~--l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      ++++|+++-+.+++++.-+... -.+.+.+-....+.+  +...+.-....-.++..-++.+++.|+.|+...+-.    
</pre><pre>T Consensus       189 ~l~~l~~~~~~l~~AI~~ie~~~i~d~~k~~iPt~e~l~~l~~~~pe~~lvk~ai~~gi~~l~k~L~~i~~gi~y~----  264 (367)
</pre><pre>T 6GY7_A          189 KRNKLDADREKIIESQDVIRQNNIADMFKDFIPSAKDIDGLDFTQPKKEAIKQAIKQGAEIARKILGKVSEGLKYI----  264 (367)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHTTHHHHTCSSCCCHHHHHTSSSCHHHHHHHHHHHHHHHHHHHHHHCCCGGGCCHH----
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhCCHHHHhhhhCCCHHHHccCCCCCchHHHHHHHHHHHHHHHHHHHHHHHHhccHH----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   78 GTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus        78 g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                            .....|.+|-.|++..-..+......+.....--+.|  ..=.-+..++..-..++..+.+.+.........++
</pre><pre>T Consensus       265 ------~Lv~aR~~L~~~i~~l~~~~~~~~~~l~~~~~~l~~l--~~i~~l~~~R~~~~~E~~kl~~~~~~f~~~~~~~~  336 (367)
</pre><pre>T 6GY7_A          265 ------DLADARMKLSDQIDQLITETDELKAKIREVELRLSGL--KDVMQIDTERTTLLTEAVKIEQVWISFAEQLHKLS  336 (367)
</pre><pre>T ss_dssp             ------HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ------HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH--HHHhhHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 215
</pre><pre>>3PLT_C Sphingolipid long chain base-responsive protein LSP1; eisosomes, Lsp1, Pil1, bar domain, plasma membrane, self-assembly, phosphoprotein, STRUCTURAL PROTEIN; 2.9A {Saccharomyces cerevisiae}
</pre><pre>Probab=26.17  E-value=1e+03  Score=22.98  Aligned_cols=148  Identities=10%  Similarity=0.057  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      +..|..+-..++.+|..-...|.+-+.+-.+-+..++..-+.....-+..++++...++.++.. |+.....-++...+.
</pre><pre>T Consensus        39 ~~~l~~~~~~as~~L~~Wa~~e~~dl~Dv~~~~~~L~~~~~~~~~~~a~~~~~~R~~lK~I~~~-E~~l~~lr~~~~~l~  117 (234)
</pre><pre>T 3PLT_C           39 MEVVASERREAAKQLSLWGADNDDDVSDVTDKLGVLIYELGELQDQFIDKYDQYRVTLKSIRNI-EASVQPSRDRKEKIT  117 (234)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHTTSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHhhCCccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHhhHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHH----HHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   82 GQQFFVERQ----KLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        82 ~~~F~~~R~----~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                      ...=..+++    .-+.+|...|...-.      ........|....+..+..|=..--+.+..|++.+.-|+...+.+
</pre><pre>T Consensus       118 ~k~~kl~kk~~~~~~l~~l~~el~~~e~------~i~~~e~ele~~Kr~~lk~~l~~~~~a~~ela~k~~iI~~~gk~i  190 (234)
</pre><pre>T 3PLT_C          118 DEIAHLKYKDPQSTKIPVLEQELVRAEA------ESLVAEAQLSNITREKLKAAYSYMFDSLRELSEKFALIAGYGKAL  190 (234)
</pre><pre>T ss_dssp             HHHHHHHHHCCCTTTHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHhhhcCCCCchHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 216
</pre><pre>>2OV3_A Periplasmic binding protein component of an ABC type zinc uptake transporter; ABC Transporter, solute binding domain, zinc transport, TRANSPORT PROTEIN; 2.4A {Synechocystis sp.} SCOP: c.92.2.2
</pre><pre>Probab=25.93  E-value=4.4e+02  Score=25.05  Aligned_cols=70  Identities=10%  Similarity=0.140  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..++..++.+.+.|.++.|+.++.+.+|+                         +...+.|++++.-+++.+........
</pre><pre>T Consensus       107 ~~~~~~~~~i~~~L~~~~P~~~~~~~~n~-------------------------~~~~~~l~~l~~~~~~~~~~~~~~~~  161 (260)
</pre><pre>T 2OV3_A          107 TLVKRQATTIAKELAELDPDNRDQYEANL-------------------------AAFLAELERLNQELGQILQPLPQRKF  161 (260)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHTTCSCCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCcccHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHhccCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                      -.+...-.-+.+.++
</pre><pre>T Consensus       162 v~~h~~~~y~~~~~g  176 (260)
</pre><pre>T 2OV3_A          162 IVFHPSWAYFARDYN  176 (260)
</pre><pre>T ss_dssp             EESSCCCHHHHHHTT
</pre><pre>T ss_pred             EEecHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 217
</pre><pre>>PF09577.12 ; Spore_YpjB ; Sporulation protein YpjB (SpoYpjB)
</pre><pre>Probab=25.92  E-value=9.4e+02  Score=22.40  Aligned_cols=113  Identities=11%  Similarity=0.066  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcC--CHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAEL--NTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN-------   72 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L--~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~-------   72 (262)
</pre><pre>                      +..|...-.++..+|..-  ++++...-...+-+.-+.+......+      |.++...+.+.|.++..+..+       
</pre><pre>T Consensus        47 ~~~l~~~~~~~~~~L~~~~~~~~~~~~~~~~l~l~~dal~~~~~pl------w~~~~~~l~~~l~~~~~a~~~~d~~~a~  120 (226)
</pre><pre>T B7GHS1_ANOFW/2   47 IYVLTATHEEALRAVTASSVPTEQRINQLTQFRLAVDAVHSKHQPL------WKEMKYAVMSAFDKMKQAMEQEDEQSFQ  120 (226)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHHHHHHhcCCCCcH------HHHHHHHHHHHHHHHHHHHHcCCHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ----HHHhcCCCccHHHH-HHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   73 ----QYRTQGTLIGQQFF-VERQKLFSELDVLLNKLSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        73 ----~~~~~g~l~~~~F~-~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                          .+..+=....|... .+-...+.++|..+..+......-|+.....+.|
</pre><pre>T Consensus       121 ~~~~~~~~~w~~iepav~~~~s~~~~~~ie~~l~~l~~~~~~~~~~~~~~~~l  173 (226)
</pre><pre>T B7GHS1_ANOFW/2  121 QAFHQFLQRYELIEPSVKINADREQVQRVSAHISLLEAPAFRQLHATERMKEL  173 (226)
</pre><pre>T ss_pred             HHHHHHHHHHHhHHhhhhcCCCHHHHHHHHHHHHHHHcchhhcCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 218
</pre><pre>>TIGR02989 Sig-70_gvs1; RNA polymerase sigma-70 factor, Rhodopirellula/Verrucomicrobium family. This group of sigma factors are members of the sigma-70 family (TIGR02937) and are abundantly found in the species Rhodopirellula baltica (11), and Verrucomicrobium spinosum (16) and to a lesser extent in Gemmata obscuriglobus (2).
</pre><pre>Probab=25.81  E-value=4.8e+02  Score=21.19  Aligned_cols=59  Identities=14%  Similarity=0.280  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      .+....+.+.+..|++.++..+.-+|  ++.. ...-+.-.|.+...+.+.++++.+.|++.
</pre><pre>T Consensus        99 ~~~~~~~~~~l~~l~~~~~~il~l~~--~~g~~~~eia~~lg~s~~~v~~~~~~~~~~l~~~  158 (159)
</pre><pre>T TIGR02989        99 SDDRQALEGCLEKLPERQRELLDLRY--QRGQKATALAEQTGRTVNAVYKALQRIRVSLREC  158 (159)
</pre><pre>T ss_pred             HHHHHHHHHHHhcCCHHHHHHHHHHH--HcCCCHHHHHHHHCCCHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 219
</pre><pre>>TIGR02947 SigH_actino; RNA polymerase sigma-70 factor, TIGR02947 family. This group of sigma factors are members of the sigma-70 family (TIGR02937).
</pre><pre>Probab=25.43  E-value=5.1e+02  Score=21.91  Aligned_cols=62  Identities=11%  Similarity=0.070  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   10 QKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQ   73 (262)
</pre><pre>Q Consensus        10 ~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~   73 (262)
</pre><pre>                      +.+...+..|++.+...+..+|  ++.+ ...-+.-.|.....+.+.+.++.+.|++.-.-+..+
</pre><pre>T Consensus       123 ~~l~~~l~~L~~~~~~i~~~~~--~~g~s~~eia~~lg~s~~~v~~~~~r~~~~l~~~l~~~~~~  185 (193)
</pre><pre>T TIGR02947       123 QDIKDALQGLPEEFRQAVYLAD--VEGFAYKEIAEIMGTPIGTVMSRLHRGRKQLRKQLVDVAGE  185 (193)
</pre><pre>T ss_pred             HHHHHHHcCCCHHHHHHHHHHH--hcCCCHHHHHHHHCCCHHHHHHHHHHHHHHHHHHHHHHhhh
</pre><pre>
</pre><pre>
</pre><pre>No 220
</pre><pre>>PF05859.14 ; Mis12 ; Mis12 protein
</pre><pre>Probab=25.30  E-value=8e+02  Score=21.41  Aligned_cols=110  Identities=14%  Similarity=0.144  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc---------CCCccHHHHHHHHHHHHHHHHHHHH--------HHHhhcC
</pre><pre>Q WP_048874398.1   47 VVADAGERYFSEIEKKLKAIELAYQNQYRTQ---------GTLIGQQFFVERQKLFSELDVLLNK--------LSRLTLK  109 (262)
</pre><pre>Q Consensus        47 ~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~---------g~l~~~~F~~~R~~Lf~~Ld~~L~~--------~~~~~~~  109 (262)
</pre><pre>                      ..........+.+.+.+.+.-+...+.+...         ......+....-.++..++...+.+        +++.-+.
</pre><pre>T Consensus         9 ~p~~~~d~v~n~v~~~i~~~~~~~e~~l~~~~~~~~~~~~~~~~~~~i~~~~~~l~~~~~~~~d~~~d~~E~y~l~nif~   88 (141)
</pre><pre>T M3JB41_CANMX/9    9 LPLSVIDDVVNAVNAIMYRGITAVDVYLKEQQKLQRKQVFKYLSDEEISVGMGKLESLLESTVDKNFDKYELYCRRNIFS   88 (141)
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhHhhhhccccCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCC----hHHHHHHhcCcchhhhhhhhhcc-ccccchHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  110 LKP----YEEIKHALGLSSRSIVHEWSTAG-VASIRGYSTYVDNASKAARFLK  157 (262)
</pre><pre>Q Consensus       110 l~~----~~~lK~~LGlStkS~vh~w~~~g-~g~Ip~y~~~i~~~ak~~k~lk  157 (262)
</pre><pre>                      +|+    +-.|-+.-+++.+..-.. ...- ..+|..+.++|.+.......|+
</pre><pre>T Consensus        89 IP~~l~~~v~Lp~d~~~~~~~~~~~-~e~~ld~el~~Lr~~i~~~~~~~~~L~  140 (141)
</pre><pre>T M3JB41_CANMX/9   89 IPEDLIGYVRLSHHEGIDFKGDIVS-KKESFADDVKKLEFKISQELQLRKLLK  140 (141)
</pre><pre>T ss_pred             CChhHcccccccccCCCCCCCCccc-cchhHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 221
</pre><pre>>PF15294.8 ; Leu_zip ; Leucine zipper
</pre><pre>Probab=25.30  E-value=1.2e+03  Score=23.64  Aligned_cols=120  Identities=12%  Similarity=0.195  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC--
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG--   78 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g--   78 (262)
</pre><pre>                      +|..|+++-++...-|+.+..+-.+.+.+.-.+-..+-.....-...-...-...+..+.+.+..+..-+...+..+.  
</pre><pre>T Consensus       134 Ei~rL~~EN~~Lk~rl~~le~q~~~~~~ek~~l~~~l~~~~~~~~~~~~~~~~~~~~~le~~~~~l~~~~~~~~~~~~~~  213 (275)
</pre><pre>T A9ULJ7_XENTR/3  134 EISRLQEENDKLKARLKTIEYQATSALGDKIKAESALKDLQKVQDNQKAKQNTQDINILEKTMADLKCDFEKTVNDHTVS  213 (275)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHHHHHHHHHHHHHHHHHHHHHchhhHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCccHHHHHHHHHHHHHHHHHHHH----HHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   79 TLIGQQFFVERQKLFSELDVLLNK----LSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        79 ~l~~~~F~~~R~~Lf~~Ld~~L~~----~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                      ...-..=...-+.-+.++..+|..    +-++-..-+.|.+||++|
</pre><pre>T Consensus       214 ~~~l~~~L~~~k~~l~~vq~qL~laekELekKf~qT~~y~nmKkmL  259 (275)
</pre><pre>T A9ULJ7_XENTR/3  214 QKYLEDNLASTKHELLRVQEQLSSAEKELEKKFQQTAVYRNMKDIL  259 (275)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhHHCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 222
</pre><pre>>d2q9qc_ a.278.1.1 (C:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=25.23  E-value=3.3e+02  Score=23.85  Aligned_cols=40  Identities=13%  Similarity=0.224  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHH-----HHHHcCCHHHHHHHHHhHHHHHHHhhh
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVN-----GALAELNTDEANFFNKHFSTIAALTNF   40 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~-----~al~~L~~~ea~fl~~n~~~l~~~l~~   40 (262)
</pre><pre>                      ++..+++..-.+.     .....||++|.+|+.+|.+++....+.
</pre><pre>T Consensus        83 Rl~kI~~~~~~~~~~l~~~~~~~LS~~E~~f~~~y~~l~~~y~~~  127 (145)
</pre><pre>T d2q9qc_          83 RLLRIRALRWEYGSILPNALRFHMAAEEMEWFNNYKRSLATYMRS  127 (145)
</pre><pre>T ss_dssp             HHHHHHHHHHHTCSSCCHHHHTTSCHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHcCCCCCHHHHhcCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 223
</pre><pre>>5COF_A Uncharacterized protein; Bacteriocin, sporulation, DUF1706, cell cycle, unknown function; HET: MPD, MSE; 1.35A {Escherichia coli (strain UTI89 / UPEC)}
</pre><pre>Probab=25.14  E-value=6.7e+02  Score=20.45  Aligned_cols=144  Identities=14%  Similarity=0.088  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHH----------HHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHH---HHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEA----------NFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIE---KKLKAIEL   68 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea----------~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~---~~L~~ie~   68 (262)
</pre><pre>                      +..+...-+...+.+..+++++.          ..-..-.+++.++..................-....   .....+..
</pre><pre>T Consensus        12 ~~~~~~~~~~~~~~l~~l~~~~~~~~~~~~~~~~~~wsi~~~l~Hl~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~   91 (173)
</pre><pre>T 5COF_A           12 LLAIDKNFSKLISYLNTIPPEITSDKSMDGHAKGTEMSVRDLVSYLLGWNALVVKWIASDAKGLPVDFPETGYKWNQLGL   91 (173)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHSCGGGTTCCCEECSSTTCEECHHHHHHHHHHHHHHHHHHHHHHHTTCCCCSSBTTBCTTCHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhcCCHHHccCcccCCCCCCCCCCHHHHHHHHHHHHHHHHHHHHHhhCCCCCCCCCCCCChhHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhh-hhhhccccccchHHHHHH
</pre><pre>Q WP_048874398.1   69 AYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVH-EWSTAGVASIRGYSTYVD  147 (262)
</pre><pre>Q Consensus        69 ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh-~w~~~g~g~Ip~y~~~i~  147 (262)
</pre><pre>                      ..............++..+.-+....++...|..+....+.-+....     .++..-+++ +       -+..+..|+.
</pre><pre>T Consensus        92 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~l~~~~~~~~~~~~~~~-----~~t~~~~l~~~-------~~~H~~~H~~  159 (173)
</pre><pre>T 5COF_A           92 LAQKFYSDYSELSYELLVAELQTVKNEIVNLINDRTDDILYGRPWYT-----KWTMGRMISFN-------TSSPYANANG  159 (173)
</pre><pre>T ss_dssp             HHHHHHHHHTTSCHHHHHHHHHHHHHHHHHHHHTSCHHHHHSSCSBT-----TBCHHHHHHHH-------THHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHhcCCCHHHHHHHHHHHHHHHHHHHHcCCHHHHhCCCCcC-----CccHHHHHHhh-------chhHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHH
</pre><pre>Q WP_048874398.1  148 NASKAARFLK  157 (262)
</pre><pre>Q Consensus       148 ~~ak~~k~lk  157 (262)
</pre><pre>                      ++..+.+.++
</pre><pre>T Consensus       160 qi~~~~~~~~  169 (173)
</pre><pre>T 5COF_A          160 RLRKWAKNNN  169 (173)
</pre><pre>T ss_dssp             HHHHHHTTTT
</pre><pre>T ss_pred             HHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 224
</pre><pre>>6GRK_I AhlB; Tripartite pore-forming toxin, TOXIN; 2.33A {Aeromonas hydrophila}
</pre><pre>Probab=25.13  E-value=9.7e+02  Score=24.20  Aligned_cols=68  Identities=12%  Similarity=0.144  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHH--HHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTD--EANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIEL   68 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~--ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~   68 (262)
</pre><pre>                      ++++++++-++..+.+..+..+  ....+..+.+.+...+......++...+.|.....++.+.++.|+.
</pre><pre>T Consensus       233 ~~~~~~~~i~~l~~~l~~~~~~~~~l~~~~~~l~~l~~~i~~a~~~l~~l~~~W~~i~~~l~~l~~~l~~  302 (367)
</pre><pre>T 6GRK_I          233 VLHNSLGARQDLYQKRSSLNSEVLIATQIGNGYKGLQVQAQNAVTAATQMSNAWDSLTSDLGSLITDLDK  302 (367)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHSCCSSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 225
</pre><pre>>d2hyja1 a.4.1.9 (A:8-82) Putative transcriptional regulator SCO4940 {Streptomyces coelicolor [TaxId: 1902]}
</pre><pre>Probab=25.00  E-value=5e+02  Score=19.34  Aligned_cols=46  Identities=15%  Similarity=0.099  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHH-HHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   88 ERQKLFSEL-DVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        88 ~R~~Lf~~L-d~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      +......++ +.....+...+..-.....|-+.+|+|..++.++|+.
</pre><pre>T Consensus         2 ~~~~~~~~i~~~~~~~~~~~g~~~~s~~~ia~~~~is~~tiy~~f~~   48 (75)
</pre><pre>T d2hyja1           2 EAQATRGRILGRAAEIASEEGLDGITIGRLAEELEMSKSGVHKHFGT   48 (75)
</pre><pre>T ss_dssp             THHHHHHHHHHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHTTCSS
</pre><pre>T ss_pred             hhHHHHHHHHHHHHHHHHHhCCCcCCHHHHHHHHCCCHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 226
</pre><pre>>2E9X_E DNA replication complex GINS protein PSF1; GINS complex, Eukaryotic DNA replication, REPLICATION; HET: SO4; 2.3A {Homo sapiens} SCOP: a.278.1.1
</pre><pre>Probab=24.89  E-value=3.6e+02  Score=23.29  Aligned_cols=40  Identities=13%  Similarity=0.224  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHH-----HHHHcCCHHHHHHHHHhHHHHHHHhhh
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVN-----GALAELNTDEANFFNKHFSTIAALTNF   40 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~-----~al~~L~~~ea~fl~~n~~~l~~~l~~   40 (262)
</pre><pre>                      ++..++.....+.     +....|++.|.+|+.++.+++...+..
</pre><pre>T Consensus        83 Rl~kI~~~~~~~~~~l~~~~~~~LS~~E~~f~~~y~~l~~~y~~~  127 (149)
</pre><pre>T 2E9X_E           83 RLLRIRALRWEYGSVLPNALRFHMAAEEMEWFNNYKRSLATYMRS  127 (149)
</pre><pre>T ss_dssp             HHHHHHHHHHHHCSSCCHHHHTTSCHHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHhcCCCCHHHHhcCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 227
</pre><pre>>6QAJ_B Endolysin,Transcription intermediary factor 1-beta; Transcriptional repressor  epigenetic silencing  histone H3 lysine 9 methylation (H3K9me3)  endogenous retrovirus ; 2.901A {Enterobacteria phage T4}
</pre><pre>Probab=24.82  E-value=1.3e+03  Score=23.67  Aligned_cols=101  Identities=12%  Similarity=0.061  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH----hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAAL----TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~----l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      +..+....+.+......+..+-...+.+-.+.++..    +..-..........+....+.+++.+.+++......-...
</pre><pre>T Consensus       405 ~~~~~~~~~~~~~~~~~~~~~i~~~~~~l~~~l~~~~~~l~~~l~~~~~~~~~~l~~~~~~l~~~~~~l~~~~~~~~~~~  484 (544)
</pre><pre>T 6QAJ_B          405 TKEVRSSIRQVSDVQKRVQVDVKMAILQIMKELNKRGRVLVNDAQKVTEGQQERLERQHWTMTKIQKHQEHILRFASWAL  484 (544)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHGGG
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCccHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   78 GTLIGQQFFVERQKLFSELDVLLNK  102 (262)
</pre><pre>Q Consensus        78 g~l~~~~F~~~R~~Lf~~Ld~~L~~  102 (262)
</pre><pre>                      ..-...+|......+..+|+..+..
</pre><pre>T Consensus       485 ~~~~~~~~l~~~~~~~~~l~~~~~~  509 (544)
</pre><pre>T 6QAJ_B          485 ESDNNTALLLSKKLIYFQLHRALKM  509 (544)
</pre><pre>T ss_dssp             SCSCGGGGHHHHHHHHTCCCCCCCC
</pre><pre>T ss_pred             hcCCHHHHHHhHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 228
</pre><pre>>d2hkua1 a.4.1.9 (A:18-87) Putative transcriptional regulator RHA1_ro03468 {Rhodococcus sp. RHA1 [TaxId: 101510]}
</pre><pre>Probab=24.80  E-value=3e+02  Score=20.60  Aligned_cols=37  Identities=11%  Similarity=0.103  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhh
</pre><pre>Q WP_048874398.1   96 LDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWST  133 (262)
</pre><pre>Q Consensus        96 Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~  133 (262)
</pre><pre>                      |+....-+...+ .-+.-.+|-+..|+|..++-+++..
</pre><pre>T Consensus         7 ~~~~~~~~~~~g-~~~t~~~ia~~agvs~~t~y~~f~~   43 (70)
</pre><pre>T d2hkua1           7 FTAATELFLEHG-EGVPITQICAAAGAHPNQVTYYYGS   43 (70)
</pre><pre>T ss_dssp             HHHHHHHHHHHC-TTSCHHHHHHHHTCCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHhC-CCCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 229
</pre><pre>>TIGR02954 Sig70_famx3; RNA polymerase sigma-70 factor, TIGR02954 family. This group of sigma factors are members of the sigma-70 family (TIGR02937).
</pre><pre>Probab=24.74  E-value=4.3e+02  Score=21.81  Aligned_cols=58  Identities=16%  Similarity=0.178  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    7 NAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         7 ~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      +....+.+.+..|++.+.+.+.-+|  ++.+ ...-+.-.|.+...+.+.+.++.+.|++.
</pre><pre>T Consensus       108 ~~~~~l~~~l~~L~~~~~~i~~l~~--~~g~s~~eiA~~lg~s~~~v~~~~~~~~~~lr~~  166 (169)
</pre><pre>T TIGR02954       108 DSRLDLYKAIDTLNDKYQTAIILRY--YHDLTIKEIAEVMNKPEGTVKTYLHRALKKLKKR  166 (169)
</pre><pre>T ss_pred             hHHHHHHHHHHhCCHHHHHHHHHHH--hCCCCHHHHHHHhCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 230
</pre><pre>>TIGR03772 anch_rpt_subst; anchored repeat ABC transporter, substrate-binding protein. Members of this protein family are ABC transporter permease subunits as identified by pfam00950, but additionally contain the Actinobacterial insert domain described by TIGR03769.
</pre><pre>Probab=24.71  E-value=5.6e+02  Score=27.68  Aligned_cols=49  Identities=12%  Similarity=0.177  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYR   75 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~   75 (262)
</pre><pre>                      ...++..++.+.+.|.++.|+.++++.+|++.+..                         .|++|.+.|++.+.
</pre><pre>T Consensus       317 P~n~~~~a~~Ia~~L~~ldP~~~~~y~~N~~~~~~-------------------------~L~~L~~~~~~~l~  365 (479)
</pre><pre>T TIGR03772       317 VKNAIAYVEVIRDKLIEVDPRGAQAYRSNASAYIH-------------------------RLERLDTYVRRTIA  365 (479)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHChhhHHHHHHHHHHHHH-------------------------HHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 231
</pre><pre>>TIGR02937 sigma70-ECF; RNA polymerase sigma factor, sigma-70 family. This model encompasses all varieties of the sigma-70 type sigma factors including the ECF subfamily.
</pre><pre>Probab=24.66  E-value=4.9e+02  Score=20.96  Aligned_cols=60  Identities=17%  Similarity=0.277  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      ......+...+..|++.+...+..+|-. ..-...-+.-.|..-+.+.+++.++.+.|++.
</pre><pre>T Consensus        98 ~~~~~~~~~~l~~l~~~~~~i~~~~~~~-~~~~~eia~~l~~s~~~v~~~~~~~~~~l~~~  157 (158)
</pre><pre>T TIGR02937        98 EEEREALREALEKLPEREREVLVLRYLE-GLSYKEIAEILGISVGTVKRRLKRARKKLREL  157 (158)
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHHHHc-CCCHHHHHHHHCCCHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 232
</pre><pre>>3OGI_C Putative ESAT-6-like protein 6; Structural Genomics, PSI-2, Protein Structure Initiative, Integrated Center for Structure and Function Innovation, ISFI, TB Structural; HET: MSE; 2.549A {Mycobacterium tuberculosis}
</pre><pre>Probab=24.60  E-value=6.7e+02  Score=20.30  Aligned_cols=72  Identities=10%  Similarity=0.009  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQN   72 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~   72 (262)
</pre><pre>                      ....|...+..+......|...-.+.-..=-.+-+..-+.++......-..|.+.++++.+.|.++..+..+
</pre><pre>T Consensus         6 ~~~~l~~~~~~i~~~~~~l~~~l~~l~~~v~~l~~~W~G~a~~a~~~~~~~w~~~~~~~~~~L~~~~~~l~~   77 (101)
</pre><pre>T 3OGI_C            6 QFGDVDAHGAMIRAQAGLLEAEHQAIVRDVLAAGDFWGGAGSVACQEFITQLGRNFQVIYEQANAHGQKVQA   77 (101)
</pre><pre>T ss_dssp             -------CHHHHHHHHHHHHHHHHHHHHHHHHTGGGTTCTTSHHHHHHHHHHHHHHHHHTTCC---------
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 233
</pre><pre>>3ZBH_B ESXA; UNKNOWN FUNCTION, TYPE 7 SECRETION, ESAT6 FAMILY PROTEINS, WXG PROTEIN ESS; HET: ACT, PO4; 1.94A {N/A}
</pre><pre>Probab=24.45  E-value=5e+02  Score=18.73  Aligned_cols=89  Identities=9%  Similarity=0.034  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|...++.+......+.....++-.....+-..+.+..+.........+...++.+.+.|.++.......       
</pre><pre>T Consensus        10 ~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~w~g~~~~~~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   82 (99)
</pre><pre>T 3ZBH_B           10 TPEELRGVARQYNVESSNVTELIARLDQMSHTLQGIWEGASSEAFIQQYQELRPSFEKMAVLLNEVGQQLHNS-------   82 (99)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHSCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhcccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                       ...|......+-..++
</pre><pre>T Consensus        83 -~~~~~~~d~~~~~~~~   98 (99)
</pre><pre>T 3ZBH_B           83 -ATILEDTDQQIASQIR   98 (99)
</pre><pre>T ss_dssp             -HHHHHHHHHHHHHCC-
</pre><pre>T ss_pred             -HHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 234
</pre><pre>>3ZBH_G ESXA; UNKNOWN FUNCTION, TYPE 7 SECRETION, ESAT6 FAMILY PROTEINS, WXG PROTEIN ESS; HET: ACT, PO4; 1.94A {N/A}
</pre><pre>Probab=24.45  E-value=5e+02  Score=18.73  Aligned_cols=89  Identities=9%  Similarity=0.034  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCC
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTL   80 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l   80 (262)
</pre><pre>                      ....|...++.+......+.....++-.....+-..+.+..+.........+...++.+.+.|.++.......       
</pre><pre>T Consensus        10 ~~~~l~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~w~g~~~~~~~~~~~~~~~~~~~~~~~l~~~~~~l~~~-------   82 (99)
</pre><pre>T 3ZBH_G           10 TPEELRGVARQYNVESSNVTELIARLDQMSHTLQGIWEGASSEAFIQQYQELRPSFEKMAVLLNEVGQQLHNS-------   82 (99)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHCSHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhhhcccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSELD   97 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~Ld   97 (262)
</pre><pre>                       ...|......+-..++
</pre><pre>T Consensus        83 -~~~~~~~d~~~~~~~~   98 (99)
</pre><pre>T 3ZBH_G           83 -ATILEDTDQQIASQIR   98 (99)
</pre><pre>T ss_dssp             -HHHHHHHHHHTTC---
</pre><pre>T ss_pred             -HHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 235
</pre><pre>>1WYY_A E2 Glycoprotein; membrane fusion, severe acute respiratory syndrome, viral protein; 2.2A {SARS coronavirus} SCOP: h.3.3.1
</pre><pre>Probab=24.35  E-value=1e+03  Score=22.37  Aligned_cols=75  Identities=17%  Similarity=0.272  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCc
</pre><pre>Q WP_048874398.1    2 LKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLI   81 (262)
</pre><pre>Q Consensus         2 l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~   81 (262)
</pre><pre>                      +..+.+....++.+|.+    --+...+|-..|..++..=+...|+.++.+.    +|.+-|++|+.--|--.--.|++.
</pre><pre>T Consensus        31 ~~~i~~~f~~~a~AL~k----iQ~vVN~~~~aL~~L~~qL~~nFgAISssi~----dI~~rLd~l~AdaQvDRLItGR~~  102 (149)
</pre><pre>T 1WYY_A           31 ISQIQESLTTTSTALGK----LQDVVNQNAQALNTLVKQLSSNFGAISSVLN----DILSRLDKVEAEVQIDRLITGGRG  102 (149)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH----HHHHHHHHHHHHC-----------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHHhchhHHHHH----HHHHHHHHHHHHhHHhhHhcCCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cHH
</pre><pre>Q WP_048874398.1   82 GQQ   84 (262)
</pre><pre>Q Consensus        82 ~~~   84 (262)
</pre><pre>                      ++.
</pre><pre>T Consensus       103 s~~  105 (149)
</pre><pre>T 1WYY_A          103 GSD  105 (149)
</pre><pre>T ss_dssp             ---
</pre><pre>T ss_pred             CCC
</pre><pre>
</pre><pre>
</pre><pre>No 236
</pre><pre>>d5x7va_ d.305.1.0 (A:) automated matches {Malaria parasite (Plasmodium falciparum) [TaxId: 5833]}
</pre><pre>Probab=24.35  E-value=2.1e+02  Score=26.38  Aligned_cols=43  Identities=14%  Similarity=0.359  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHh---cCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   54 RYFSEIEKKLKAIELAYQNQYRT---QGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        54 ~~~~~i~~~L~~ie~ly~~~~~~---~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      +.+++|++.+++|+.-+..+...   .-......++.+|+++.+++
</pre><pre>T Consensus         3 ~~L~~iq~~~~~l~~~~~~e~~~l~~~~~~~~~pl~~kR~~ii~~I   48 (189)
</pre><pre>T d5x7va_           3 QDFEDIQKDIEQLDIKCAHEQMNIQKQYDEKKKPLFEKRDEIIQKI   48 (189)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTTS
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhchHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 237
</pre><pre>>6M39_A Spike glycoprotein; SADS-CoV spike, VIRAL PROTEIN; HET: NAG; 3.55A {Swine acute diarrhea syndrome coronavirus}
</pre><pre>Probab=24.35  E-value=1.2e+03  Score=28.45  Aligned_cols=69  Identities=12%  Similarity=-0.003  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhHHHHHHH----hhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   25 NFFNKHFSTIAAL----TNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDV   98 (262)
</pre><pre>Q Consensus        25 ~fl~~n~~~l~~~----l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~   98 (262)
</pre><pre>                      +.|.||-.+|.+-    +...+.+..+++.++.+-=+-|+.-=..|++|-++.-..     +.+.-.-=+.+++|||.
</pre><pre>T Consensus       751 ~vL~~NQk~iA~~FN~a~~~~~~g~~t~~~Al~kiQ~vVN~~~~al~~L~~qL~~n-----FgAISssi~dI~~RLd~  823 (981)
</pre><pre>T 6M39_A          751 NALVDDVNKLANGFNQLTASVSKLALTTSSALQAIQAVVNQNAAQVESLVSGITEN-----FGAISTNFKVISQRLDK  823 (981)
</pre><pre>T ss_dssp             SCCHHHHHHHHHHHHHHHGGGSSCCSTTCSSHHHHHHHHHHHHHHHHHHHHHHHSC-----SSSSCSCHHHHHTTBCS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----hchhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 238
</pre><pre>>TIGR02985 Sig70_bacteroi1; RNA polymerase sigma-70 factor, Bacteroides expansion family 1. This group of sigma factors are members of the sigma-70 family (TIGR02937) and are found primarily in the genus Bacteroides.
</pre><pre>Probab=24.25  E-value=5.7e+02  Score=20.60  Aligned_cols=59  Identities=15%  Similarity=0.187  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHH-hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHFSTIAAL-TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~-l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      .+....+.+.+..|++.+...+.-+|  ++.+ ...-+.-.|.....+.+++.++.+.|++.
</pre><pre>T Consensus       101 ~~~~~~l~~~~~~L~~~~~~i~~l~~--~~~~~~~eIa~~l~~s~~~v~~~~~~~~~~l~~~  160 (161)
</pre><pre>T TIGR02985       101 KELEEIIEEAIEKLPEQCREIFILSR--FEGLSNKEIAEELGISVKTVEYHITKALKFLRKE  160 (161)
</pre><pre>T ss_pred             HHHHHHHHHHHHhCCHHHHHHHHHHH--hcCCCHHHHHHHHCCCHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 239
</pre><pre>>7M5F_A CdiI; toxin, antitoxin, CDI Toxin-Immunity Protein Complexes, Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID; HET: MSE, MLI; 1.59A {Serratia marcescens}
</pre><pre>Probab=24.22  E-value=5.5e+02  Score=20.61  Aligned_cols=44  Identities=20%  Similarity=0.188  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhc----CCC----ccHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQ----GTL----IGQQFFVERQKLFSELDVLL  100 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~----g~l----~~~~F~~~R~~Lf~~Ld~~L  100 (262)
</pre><pre>                      .++.+.|.+.++.|++.+...    ...    ...+|.++-.+|.++|...|
</pre><pre>T Consensus        35 ~~L~~~l~~W~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~g~~La~~L~~el   86 (98)
</pre><pre>T 7M5F_A           35 LGLKNSLEAWAKRYDAILNTDDPALSGFKSVEEEKLFIDDGYKLAELLQEEL   86 (98)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTTCCSSSGGGCCCSSHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHcCCCChhHcCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 240
</pre><pre>>4NNO_A Lipoprotein; Manganese transporter, MRSA, ABC superfamily ATP binding cassette transporter, TRANSPORT PROTEIN; 1.174A {Staphylococcus aureus subsp. aureus} SCOP: c.92.2.0
</pre><pre>Probab=24.06  E-value=8.1e+02  Score=24.04  Aligned_cols=69  Identities=3%  Similarity=-0.046  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcC--CC
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQG--TL   80 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g--~l   80 (262)
</pre><pre>                      ..++..+..+.+.|.++.|+.++.+.+|+                         +...+.|.+++.-|++.+....  ..
</pre><pre>T Consensus       128 ~~~~~~~~~i~~~L~~~~P~~~~~~~~n~-------------------------~~~~~~l~~l~~~~~~~~~~~~~~~~  182 (291)
</pre><pre>T 4NNO_A          128 DNGIKYVKTIQQTFIDNDKKHKADYEKQG-------------------------NKYIAQLEKLNNDSKDKFNDIPKEQR  182 (291)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHHHHH-------------------------HHHHHHHHHHHHHHTTTTTTSCGGGC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhCHhhHHHHHHHH-------------------------HHHHHHHHHHHHHHHHHHhcCCHHHC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   81 IGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        81 ~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ..-.+....+-|.+.+
</pre><pre>T Consensus       183 ~~v~~h~~~~yl~~~~  198 (291)
</pre><pre>T 4NNO_A          183 AMITSEGAFKYFSKQY  198 (291)
</pre><pre>T ss_dssp             EEEEEECCCHHHHHHH
</pre><pre>T ss_pred             eEEEechHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 241
</pre><pre>>5T12_A Phosphoenolpyruvate--protein phosphotransferase; PTSNtr, phosphotransfer, TRANSFERASE; 2.299A {Escherichia coli}
</pre><pre>Probab=23.95  E-value=1.1e+03  Score=22.57  Aligned_cols=88  Identities=15%  Similarity=0.162  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHc--------CCHHHHHHHHHhHHHHHHH------hhhchhhhHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAE--------LNTDEANFFNKHFSTIAAL------TNFMDKSLGVVADAGERYFSEIEKKLKAI   66 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~--------L~~~ea~fl~~n~~~l~~~------l~~~s~~~G~~s~~~~~~~~~i~~~L~~i   66 (262)
</pre><pre>                      +++.|..+-..+.+.|..        +++++++++.-|..+|++=      ...-..+. .+..++...++.+.+.+.++
</pre><pre>T Consensus        39 E~~~~~~a~~~~~~~l~~~~~~~~~~~~~~~~~i~~a~~~~L~D~~~~~~i~~~I~~~~-~a~~Av~~~~~~~~~~~~~~  117 (255)
</pre><pre>T 5T12_A           39 ERERLTGALEEAANEFRRYSKRFAAGAQKETAAIFDLYSHLLSDTRLRRELFAEVDKGS-VAEWAVKTVIEKFAEQFAAL  117 (255)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHTTTTCSHHHHHHHHHHHHHTCHHHHHHHHHHHHTTB-CHHHHHHHHHHHHHHHHHTC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccCHHHHHHHHHHHHHHcCHHHHHHHHHHHHCCC-CHHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhcCCCccHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   67 ELAYQNQYRTQGTLIGQQFFVERQKLFSEL   96 (262)
</pre><pre>Q Consensus        67 e~ly~~~~~~~g~l~~~~F~~~R~~Lf~~L   96 (262)
</pre><pre>                      ++.|.++       +..++..-.++++..|
</pre><pre>T Consensus       118 ~~~~~~~-------r~~Di~dv~~rll~~l  140 (255)
</pre><pre>T 5T12_A          118 SDNYLKE-------RAGDLRALGQRLLFHL  140 (255)
</pre><pre>T ss_dssp             CCHHHHT-------THHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CChhHHH-------HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 242
</pre><pre>>1U5E_A Src-associated adaptor protein; Novel Dimerization Domain, PH Domain, SIGNALING PROTEIN; 2.6A {Mus musculus} SCOP: b.55.1.1
</pre><pre>Probab=23.95  E-value=5.7e+02  Score=22.01  Aligned_cols=45  Identities=16%  Similarity=0.180  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      ++|++.|.+++.--.+.+. ..++.. .-..+|+.|+++|...+..+
</pre><pre>T Consensus         4 ~~~~~ll~~~~~fl~~~l~-~~~ls~-~~~~~~~~l~~~~~~~~~~~   48 (211)
</pre><pre>T 1U5E_A            4 EEIRNLLADVETFVADTLK-GENLSK-KAKEKRESLIKKIKDVKSVY   48 (211)
</pre><pre>T ss_dssp             HHHHHHHHHHHHCCCCCTT-TCBCCH-HHHHHHHHHHHHHHHHGGGG
</pre><pre>T ss_pred             HHHHHHHHhHHHHHHHhhc-CCCCCh-HHHHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 243
</pre><pre>>2OTX_A Src kinase-associated phosphoprotein 2; NOVEL DIMERIZATION DOMAIN, PH DOMAIN, SIGNALING PROTEIN; 2.6A {Mus musculus} SCOP: b.55.1.1
</pre><pre>Probab=23.95  E-value=5.7e+02  Score=22.01  Aligned_cols=45  Identities=16%  Similarity=0.180  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   57 SEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKL  103 (262)
</pre><pre>Q Consensus        57 ~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~  103 (262)
</pre><pre>                      ++|++.|.+++.--.+.+. ..++.. .-..+|+.|+++|...+..+
</pre><pre>T Consensus         4 ~~~~~ll~~~~~fl~~~l~-~~~ls~-~~~~~~~~l~~~~~~~~~~~   48 (211)
</pre><pre>T 2OTX_A            4 EEIRNLLADVETFVADTLK-GENLSK-KAKEKRESLIKKIKDVKSVY   48 (211)
</pre><pre>T ss_dssp             HHHHHHHHHHHHCCCCCTT-TCBCCH-HHHHHHHHHHHHHHHHGGGG
</pre><pre>T ss_pred             HHHHHHHHhHHHHHHHhhc-CCCCCh-HHHHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 244
</pre><pre>>TIGR04449 halocin_C8_dom; halocin C8-like bacteriocin domain. This model describes the 76-amino C-terminal domain of the halocin C8 precursor that actually becomes the mature bacteriocin halocin C8 after export and cleavage, as well as homologous C-terminal regions from many other archaea.
</pre><pre>Probab=23.85  E-value=2.7e+02  Score=22.23  Aligned_cols=30  Identities=43%  Similarity=0.601  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHhcccCcchHHHHHHHHHHH
</pre><pre>Q WP_048874398.1  213 AMAGAGVCVALGVATGGVGLLACSVVGAAA  242 (262)
</pre><pre>Q Consensus       213 ~~~a~~vc~~~G~~t~G~g~l~c~vv~~g~  242 (262)
</pre><pre>                      ......+|..+++++.+++.+.|..+...+
</pre><pre>T Consensus        17 ~~~~~~~C~~l~~~~~~~g~~~C~~~c~~v   46 (69)
</pre><pre>T TIGR04449        17 GAGCAAICGLLGLATFGIGGLACAGFCAAV   46 (69)
</pre><pre>T ss_pred             hhHHHHHHHHHHHhccchHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 245
</pre><pre>>PF10023.11 ; Aminopep ; Putative aminopeptidase
</pre><pre>Probab=23.84  E-value=1.4e+03  Score=23.65  Aligned_cols=102  Identities=15%  Similarity=0.157  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhcCCCccHHHHHHHHHHHHHHHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhh
</pre><pre>Q WP_048874398.1   53 ERYFSEIEKKLKAIELAYQNQYRTQGTLIGQQFFVERQKLFSELDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWS  132 (262)
</pre><pre>Q Consensus        53 ~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~  132 (262)
</pre><pre>                      ......|.+.-++|+.+|.+      .+...+-.++|+++|.++...+..+...-.+.+.+...-.. .+..-.+++...
</pre><pre>T Consensus       208 ~~f~~~i~~~~~~L~~LY~~------~~~~~e~~~~k~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-~lNNA~l~~~~~  280 (320)
</pre><pre>T G0AF90_COLFT/2  208 EEFLALLLQYRQRLASNYAS------ELSDAQKLQQKAAIFLALQVDYQALKVKWGGYAGYDHWFAE-PLSNAHLALVAT  280 (320)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhc------CCCHHHHHHHHHHHHHHHHHHHHHHHHHcCCCCchhhhhcc-CcccceEeHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hccccccchHHHHHHHHHHHHHHHHHhhhHH
</pre><pre>Q WP_048874398.1  133 TAGVASIRGYSTYVDNASKAARFLKMGGWLA  163 (262)
</pre><pre>Q Consensus       133 ~~g~g~Ip~y~~~i~~~ak~~k~lk~gGyvg  163 (262)
</pre><pre>                      ...  ..|.+.+-.+....+.+.++..--+.
</pre><pre>T Consensus       281 Y~~--~~~~f~~l~~~~~dl~~f~~~i~~l~  309 (320)
</pre><pre>T G0AF90_COLFT/2  281 YYD--LVPGFKALLAQQQTLKKFYAAVAGLA  309 (320)
</pre><pre>T ss_pred             Hhh--ccHHHHHHHHhCCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 246
</pre><pre>>d2yvea1 a.4.1.9 (A:1-74) Transcriptional regulator Cgl2612 {Corynebacterium glutamicum [TaxId: 1718]}
</pre><pre>Probab=23.83  E-value=4.6e+02  Score=19.20  Aligned_cols=61  Identities=16%  Similarity=0.268  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhhcCCCChHHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   96 LDVLLNKLSRLTLKLKPYEEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAARFL  156 (262)
</pre><pre>Q Consensus        96 Ld~~L~~~~~~~~~l~~~~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k~l  156 (262)
</pre><pre>                      ++..+..+...+..-+....|-+.+|+|..++.+++.....--..-+...++.+......+
</pre><pre>T Consensus        10 ~~~~~~~~~~~g~~~~s~~~ia~~~~~~~~~i~~~f~~~~~l~~~~~~~~~~~~~~~~~~~   70 (74)
</pre><pre>T d2yvea1          10 LRTAIDYIGEYSLETLSYDSLAEATGLSKSGLIYHFPSRHALLLGMHELLADDWDKELRDI   70 (74)
</pre><pre>T ss_dssp             HHHHHHHHHHSCSTTCCHHHHHHHHCCCHHHHHHHCSSHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHcCchhCCHHHHHHHHCCCHHHHHHhCCCHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 247
</pre><pre>>d3cx3a_ c.92.2.0 (A:) automated matches {Streptococcus pneumoniae [TaxId: 171101]}
</pre><pre>Probab=23.73  E-value=3.4e+02  Score=26.30  Aligned_cols=34  Identities=12%  Similarity=0.278  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHH
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAA   36 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~   36 (262)
</pre><pre>                      ..++..++.+.+.|.++.|+.++++.+|++.+..
</pre><pre>T Consensus       120 ~~~~~~a~~I~~~L~~~~P~~~~~y~~N~~~~~~  153 (277)
</pre><pre>T d3cx3a_         120 EKAGEEAQIIADKLSEVDSEHKETYQKNAQAFIK  153 (277)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHSGGGHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhhChhhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 248
</pre><pre>>3S84_A Apolipoprotein A-IV; four helix bundle, TRANSPORT PROTEIN; HET: SO4; 2.4A {Homo sapiens}
</pre><pre>Probab=23.66  E-value=1e+03  Score=22.14  Aligned_cols=146  Identities=8%  Similarity=0.140  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCcc
</pre><pre>Q WP_048874398.1    3 KQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIELAYQNQYRTQGTLIG   82 (262)
</pre><pre>Q Consensus         3 ~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie~ly~~~~~~~g~l~~   82 (262)
</pre><pre>                      ..|......++..|.+..++-.+.+.++.+-+..-+   ..........+....+++...|.+........+.....--.
</pre><pre>T Consensus        18 ~~l~~~~~~l~~~l~~~~~~l~~~l~~~~~~~~~~l---~~~~~~l~~~l~~~~~~l~~~l~~~~~~l~~~l~~~~~~~~   94 (273)
</pre><pre>T 3S84_A           18 EEIGKELEELRARLLPHANEVSQKIGDNLRELQQRL---EPYADQLRTQVNTQAEQLRRQLTPYAQRMERVLRENADSLQ   94 (273)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHGGGHHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHcHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhhcCCCCh-HHHHHHhcCcchhhhhhhhhccccccchHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1   83 QQFFVERQKLFSELDVLLNKLSRLTLKLKPY-EEIKHALGLSSRSIVHEWSTAGVASIRGYSTYVDNASKAAR  154 (262)
</pre><pre>Q Consensus        83 ~~F~~~R~~Lf~~Ld~~L~~~~~~~~~l~~~-~~lK~~LGlStkS~vh~w~~~g~g~Ip~y~~~i~~~ak~~k  154 (262)
</pre><pre>                      ..+-..-..+..+|+..+..+...   +.++ .+++..|.=....+-.........-...+..+++.+.....
</pre><pre>T Consensus        95 ~~l~~~~~~~~~~l~~~~~~l~~~---l~~~~~~l~~~l~~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~~l~  164 (273)
</pre><pre>T 3S84_A           95 ASLRPHADELKAKIDQNVEELKGR---LTPYADEFKVKIDQTVEELRRSLAPYAQDTQEKLNHQLEGLTFQMK  164 (273)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHHHHHGGGCSSCHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HhHHHHHHHHHHHHHHHHHHHHHh---hHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 249
</pre><pre>>TIGR02835 spore_sigmaE; RNA polymerase sigma-E factor. Members of this family comprise the Firmicutes lineage endospore formation-specific sigma factor SigE, also called SpoIIGB and sigma-29.
</pre><pre>Probab=23.58  E-value=9.6e+02  Score=21.72  Aligned_cols=59  Identities=19%  Similarity=0.366  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHhH----------HHHHHHhhhchhhhHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q WP_048874398.1    6 QNAKQKVNGALAELNTDEANFFNKHF----------STIAALTNFMDKSLGVVADAGERYFSEIEKKLKAIE   67 (262)
</pre><pre>Q Consensus         6 ~~~A~~v~~al~~L~~~ea~fl~~n~----------~~l~~~l~~~s~~~G~~s~~~~~~~~~i~~~L~~ie   67 (262)
</pre><pre>                      .+....+...+..|++.+...+.-+|          .-+...+..+   .......+.+..+.+.+.+.+.+
</pre><pre>T Consensus       166 ~~~~~~l~~~l~~L~~~~r~il~l~~~~~~~~g~s~~eIA~~lg~s---~~~v~~~~~r~~~kL~~~l~~~~  234 (234)
</pre><pre>T TIGR02835       166 EVDRELLRKALAKLNDREKKIMELRFGLVGGTEKTQKEVADMLGIS---QSYISRLEKRILKRLKKEINRME  234 (234)
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCHHHHHHHHHHHhccCCCCCCHHHHHHHHCCC---HHHHHHHHHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 250
</pre><pre>>3GHG_D Fibrinogen alpha chain; triple-stranded coiled coil, beta sheets, alpha helices, Amyloid, Amyloidosis, Blood coagulation, Disease mutation, Glycoprotein, Phosphoprotein, Secreted; HET: BMA, GAL, NAG, MAN, SIA, NDG; 2.9A {Homo sapiens} SCOP: h.1.8.0
</pre><pre>Probab=23.43  E-value=1.9e+03  Score=25.27  Aligned_cols=111  Identities=17%  Similarity=0.226  Sum_probs=0.0  Template_Neff=4.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHcCCHHHHHHHHHhHHHHHHHhhhchhhhHHHHHHHHHHH---HHHHHHHHHHHHHHHHHHHhc
</pre><pre>Q WP_048874398.1    1 QLKQLQNAKQKVNGALAELNTDEANFFNKHFSTIAALTNFMDKSLGVVADAGERYF---SEIEKKLKAIELAYQNQYRTQ   77 (262)
</pre><pre>Q Consensus         1 ~l~~L~~~A~~v~~al~~L~~~ea~fl~~n~~~l~~~l~~~s~~~G~~s~~~~~~~---~~i~~~L~~ie~ly~~~~~~~   77 (262)
</pre><pre>                      +|..+++...+-+++.......-.+++.-+.+.|......-......+-..-.+..   ++|.+-++.|..|..+-..+ 
</pre><pre>T Consensus        65 RI~kI~~~L~~~q~~~~ss~~~tk~~y~~~R~~l~~~~~sd~~Y~elae~Lr~rit~Lk~r~~~q~~~L~~L~~~Vr~Q-  143 (562)
</pre><pre>T 3GHG_D           65 RINKLKNSLFEYQKNNKDSHSLTTNIMEILRGDFSSANNRDNTYNRVSEDLRSRIEVLKRKVIEKVQHIQLLQKNVRAQ-  143 (562)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTTHHHHHHHHHHHHHTTTTTTHHHHHTHHHHHTTTTTTTHHHHHHHHHHHHHHHHHHHHHHHHHHH-
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhhhHHHHHHHHHHHHhhcccccccccHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCccHHHHHHHHHHHHHHHHHH-------HHHHHhhcCCCChHHHHHHh
</pre><pre>Q WP_048874398.1   78 GTLIGQQFFVERQKLFSELDVLL-------NKLSRLTLKLKPYEEIKHAL  120 (262)
</pre><pre>Q Consensus        78 g~l~~~~F~~~R~~Lf~~Ld~~L-------~~~~~~~~~l~~~~~lK~~L  120 (262)
</pre><pre>                              +.+-++|.=.+|-.|       .+.+.+.+....|..|.+.|
</pre><pre>T Consensus       144 --------v~em~RlEVDIDIKLRSCKGSCa~~~~ysVD~eSY~~LqKQL  185 (562)
</pre><pre>T 3GHG_D          144 --------LVDMKRLEVDIDIKIRSCRGSCSRALAREVDLKDYEDQQKQL  185 (562)
</pre><pre>T ss_dssp             --------HHHHHHHHHHHHHHHHTTTTTSSCCCCCCCCHHHHHHHHHHH
</pre><pre>T ss_pred             --------HHHHHhccccHHHHHHhcccchhhccccccChHhHHHHHHHH
</pre><pre>
</pre><pre>
</pre>