<pre>Query         MBL0283831.1/78-125 
</pre><pre>Match_columns 48
</pre><pre>No_of_seqs    109 out of 357
</pre><pre>Neff          4.9518
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Tue Aug 24 15:26:40 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/5734181.hhr -oa3m ../results/5734181.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 6J0M_C Pvc8; assembly, Photorh  96.1   0.059 5.5E-07   37.1   5.9   48    1-48    474-537 (538)
</pre><pre>  2 6RBK_C Afp8; Anti-feeding prop  95.8   0.096 8.9E-07   36.2   5.7   48    1-48    465-528 (529)
</pre><pre>  3 4UHV_A VGRG1, VALINE-GLYCINE R  95.7   0.082 7.6E-07   37.9   5.4   48    1-48    563-611 (651)
</pre><pre>  4 3QR8_A Baseplate assembly prot  95.7    0.15 1.4E-06   31.6   5.8   47    1-48    114-160 (211)
</pre><pre>  5 3VTN_A Protein gp45; beta-heli  95.2     0.4 3.7E-06   26.7   6.1   48    1-48      4-51  (101)
</pre><pre>  6 d1k28a2 b.108.1.2 (A:362-576)   94.3    0.33   3E-06   31.5   4.8   47    1-48    168-214 (215)
</pre><pre>  7 6U9G_B VgrG; T6SS Central Spik  92.8     1.1   1E-05   28.2   5.1   40    3-43    106-145 (189)
</pre><pre>  8 3VTO_B Protein gp45; beta-heli  92.3     2.3 2.1E-05   25.2   5.8   48    1-48     10-57  (115)
</pre><pre>  9 3VTO_C Protein gp45; beta-heli  92.3     2.3 2.1E-05   25.2   5.8   48    1-48     10-57  (115)
</pre><pre> 10 4KU0_C Tail-associated lysozym  91.8     1.9 1.8E-05   23.2   4.7   47    1-48     50-96  (96)
</pre><pre> 11 3AQJ_A Baseplate assembly prot  91.7     3.3 3.1E-05   25.0   6.1   47    1-48     37-83  (134)
</pre><pre> 12 4S37_B Phage baseplate protein  91.5     2.6 2.4E-05   25.7   5.5   41    3-44    114-155 (185)
</pre><pre> 13 4S37_O Phage baseplate protein  91.5     2.6 2.4E-05   25.7   5.5   41    3-44    114-155 (185)
</pre><pre> 14 3QR7_B Baseplate assembly prot  91.0     3.5 3.3E-05   23.6   5.7   47    1-48     18-64  (115)
</pre><pre> 15 3A1M_A chimera of thrombin cle  91.0     2.7 2.5E-05   25.9   5.3   47    1-48     55-101 (139)
</pre><pre> 16 3A1M_E chimera of thrombin cle  91.0     2.7 2.5E-05   25.9   5.3   47    1-48     55-101 (139)
</pre><pre> 17 6P20_C Baseplate central spike  89.8     1.8 1.7E-05   24.1   3.6   45    1-47     66-111 (112)
</pre><pre> 18 PF12059.10 ; DUF3540 ; Protein  89.6     1.1 9.8E-06   29.7   2.9   34   14-47    157-190 (191)
</pre><pre> 19 PF11101.10 ; DUF2884 ; Protein  86.2     2.4 2.2E-05   28.7   3.1   35    3-37     13-48  (227)
</pre><pre> 20 PF04790.15 ; Sarcoglycan_1 ; S  80.6      20 0.00018   25.0   5.7   48    1-48    164-222 (253)
</pre><pre> 21 6ORJ_A PHIKZ164; Membrane punc  80.2      15 0.00014   27.4   5.2   43    2-45    232-274 (293)
</pre><pre> 22 4RU3_A puncturing protein gp41  76.7      20 0.00019   23.5   4.6   38   11-48    145-184 (221)
</pre><pre> 23 PF10106.11 ; DUF2345 ; Unchara  75.2      27 0.00025   20.6   5.9   46    1-47     96-141 (145)
</pre><pre> 24 6SK0_B Putative type VI secret  73.3      31 0.00029   26.5   5.4   47    1-48    714-761 (841)
</pre><pre> 25 6SK0_C Putative type VI secret  73.3      31 0.00029   26.5   5.4   47    1-48    714-761 (841)
</pre><pre> 26 PF16110.7 ; DUF4828 ; Domain o  71.7      20 0.00019   21.6   3.4   20   14-33     18-37  (79)
</pre><pre> 27 TIGR02803 ExbD_1; TonB system   71.2      16 0.00014   21.7   2.9   17   14-30     51-67  (122)
</pre><pre> 28 2JV8_A Uncharacterized protein  68.0      43  0.0004   19.9   4.6   30    4-33      2-31  (73)
</pre><pre> 29 TIGR03361 VI_Rhs_Vgr; type VI   67.8      20 0.00019   25.6   3.3   26    4-29    486-512 (513)
</pre><pre> 30 3WIT_A Putative Vgr protein; T  65.4      51 0.00048   19.9   4.9   48    1-48     25-74  (82)
</pre><pre> 31 2IYJ_B THIOL DISULFIDE INTERCH  63.8      43  0.0004   18.4   3.6   35    6-40     32-66  (75)
</pre><pre> 32 6MVF_A Beta-galactosidase/beta  63.1      28 0.00026   27.0   3.5   31    4-34    268-298 (769)
</pre><pre> 33 d1eeja2 d.17.3.1 (A:1-60) Disu  61.7      42 0.00039   17.9   3.2   32    7-38     29-60  (60)
</pre><pre> 34 PF18545.3 ; HalOD1 ; Halobacte  58.3      57 0.00053   18.7   3.5   21    5-25     54-74  (76)
</pre><pre> 35 d1t3ba2 d.17.3.1 (A:2-60) Disu  58.1      52 0.00048   17.5   3.5   33    6-38     27-59  (59)
</pre><pre> 36 5Z1A_A Putative beta-galactosi  58.0      49 0.00045   24.8   3.9   31    4-34    276-306 (680)
</pre><pre> 37 2ICH_A Putative AttH; STRUCTUR  57.5      31 0.00028   23.4   2.7   23   17-39    170-192 (335)
</pre><pre> 38 2PFU_A Biopolymer transport ex  56.7      33 0.00031   18.9   2.3   17   14-30     23-39  (99)
</pre><pre> 39 TIGR02804 ExbD_2; TonB system   55.7      35 0.00032   20.2   2.4   17   14-30     53-69  (121)
</pre><pre> 40 7CWI_A beta-galactosidase; bet  55.7      71 0.00066   24.7   4.5   31    4-34    268-298 (815)
</pre><pre> 41 PF07143.13 ; CrtC ; CrtC N-ter  55.2      44 0.00041   21.0   2.9   25   17-41    159-183 (186)
</pre><pre> 42 6D1P_B Glycosyl hydrolases fam  55.2      67 0.00062   24.7   4.2   31    4-34    297-327 (814)
</pre><pre> 43 PF07378.13 ; FlbT ; Flagellar   55.0      71 0.00066   20.4   3.8   30   14-43      2-31  (124)
</pre><pre> 44 2G2S_A Green fluorescent prote  54.4      46 0.00043   19.4   2.7   25   11-37     11-35  (66)
</pre><pre> 45 PF02472.18 ; ExbD ; Biopolymer  54.2      38 0.00035   19.8   2.4   17   14-30     56-72  (131)
</pre><pre> 46 d2icha1 b.176.1.1 (A:24-351) A  54.1      44 0.00041   22.8   3.0   24   16-39    167-190 (328)
</pre><pre> 47 6HPD_A Beta-galactosidase (GH2  54.0      75  0.0007   25.2   4.4   31    4-34    315-345 (990)
</pre><pre> 48 3FN9_A Putative beta-galactosi  51.5      94 0.00087   23.4   4.4   31    4-34    271-301 (692)
</pre><pre> 49 3FN9_C Putative beta-galactosi  51.5      94 0.00087   23.4   4.4   31    4-34    271-301 (692)
</pre><pre> 50 d2vzua3 c.1.8.0 (A:336-674) au  51.4      60 0.00056   22.0   3.3   25   12-36      3-27  (339)
</pre><pre> 51 d4npba1 d.17.3.0 (A:22-81) aut  50.3      74 0.00069   17.0   3.4   33    6-38     28-60  (60)
</pre><pre> 52 2GW4_A Kaede; BETA BARREL, LUM  50.3      67 0.00062   18.7   2.9   25   11-37      8-32  (63)
</pre><pre> 53 5EXC_E Green fluorescent prote  50.3      67 0.00062   18.7   2.9   25   11-37      7-31  (62)
</pre><pre> 54 5EXC_H Green fluorescent prote  50.3      67 0.00062   18.7   2.9   25   11-37      7-31  (62)
</pre><pre> 55 3CFA_S GFP-like fluorescent pr  50.2      59 0.00055   18.8   2.7   25   11-37      7-31  (62)
</pre><pre> 56 3CFH_L GFP-like photoswitchabl  50.2      59 0.00055   18.8   2.7   25   11-37      7-31  (62)
</pre><pre> 57 4CU7_A BETA-GALACTOSIDASE; HYD  49.9      84 0.00078   24.4   4.1   31    4-34    280-310 (849)
</pre><pre> 58 3PQI_A gene product 138; Beta-  49.0 1.4E+02  0.0013   19.8   5.4   48    1-48    138-190 (247)
</pre><pre> 59 PF16270.7 ; DUF4923 ; Lipocali  48.8      60 0.00056   21.7   2.9   40    8-47     72-111 (175)
</pre><pre> 60 6M9Y_C Fluorescent protein lan  48.8      62 0.00058   18.6   2.6   22   14-37      6-27  (58)
</pre><pre> 61 6DXU_B Glycosyl hydrolase fami  48.8 1.1E+02  0.0011   23.7   4.6   31    4-34    299-329 (830)
</pre><pre> 62 6J2U_A Tyrosinase co-factor pr  47.9 1.3E+02  0.0012   19.7   4.1   30    4-33     40-78  (118)
</pre><pre> 63 4PFE_A Green fluorescent prote  47.7      52 0.00048   22.2   2.5   25   11-37      9-33  (343)
</pre><pre> 64 3LF4_A Fluorescent Timer Precu  47.7      76 0.00071   18.7   2.9   25   11-37     10-34  (66)
</pre><pre> 65 6LOF_D GFP-like fluorescent ch  46.3      81 0.00075   18.7   2.9   25   11-37     13-37  (68)
</pre><pre> 66 PF11211.10 ; DUF2997 ; Protein  46.3      49 0.00045   17.7   1.9   13   14-26      2-14  (47)
</pre><pre> 67 PF10411.11 ; DsbC_N ; Disulfid  46.2      80 0.00074   16.2   3.4   29    6-34     25-53  (54)
</pre><pre> 68 TIGR02801 tolR; TolR protein.   45.9      60 0.00056   19.3   2.4   17   14-30     53-69  (129)
</pre><pre> 69 5DMY_C Beta-galactosidase; gal  44.9 1.1E+02   0.001   24.1   4.1   31    4-34    299-329 (904)
</pre><pre> 70 3IPF_A uncharacterized protein  44.7 1.1E+02   0.001   17.5   3.3   29   15-43     24-56  (71)
</pre><pre> 71 6QUB_A Beta-galactosidase; TIM  44.5 1.3E+02  0.0012   23.6   4.4   31    4-34    293-323 (885)
</pre><pre> 72 2JWK_A Protein tolR; PROTEIN,   44.0      54  0.0005   16.9   1.8   15   14-28      6-20  (74)
</pre><pre> 73 2JWL_A Protein tolR; PROTEIN,   44.0      54  0.0005   16.9   1.8   15   14-28      6-20  (74)
</pre><pre> 74 PF11012.10 ; DUF2850 ; Protein  42.9   1E+02 0.00093   18.8   3.0   19   12-31     16-34  (78)
</pre><pre> 75 PF20033.1 ; DUF6438 ; Domain o  42.5 1.2E+02  0.0011   17.7   3.2   33    9-41     11-44  (87)
</pre><pre> 76 3RWN_A Gene 9 protein; Knob, C  42.1      81 0.00075   22.1   2.8   22   15-36    126-147 (161)
</pre><pre> 77 6MVH_C Beta-galactosidase; gly  41.8 1.5E+02  0.0013   23.3   4.3   31    4-34    273-303 (780)
</pre><pre> 78 6MVH_D Beta-galactosidase; gly  41.8 1.5E+02  0.0013   23.3   4.3   31    4-34    273-303 (780)
</pre><pre> 79 d1vioa2 d.66.1.5 (A:2-57) Pseu  41.8      67 0.00062   15.5   1.9   29   19-47     22-55  (56)
</pre><pre> 80 PF11685.10 ; DUF3281 ; Protein  41.6      71 0.00066   23.9   2.6   26   16-41    124-150 (267)
</pre><pre> 81 2RE3_A Uncharacterized protein  41.5      51 0.00048   22.4   1.8   22   10-31     38-59  (194)
</pre><pre> 82 6B6L_B Glycosyl hydrolase fami  40.5 1.4E+02  0.0013   23.0   4.0   31    4-34    269-299 (776)
</pre><pre> 83 PF18715.3 ; Phage_spike ; Phag  40.5      96 0.00089   15.5   2.6   21   28-48      3-23  (53)
</pre><pre> 84 3SZE_A Serine protease espP; p  40.4 1.3E+02  0.0012   25.5   4.0   38   11-48    309-346 (968)
</pre><pre> 85 6TYI_Y Biopolymer transport pr  39.1      87  0.0008   20.1   2.5   17   14-30     65-81  (163)
</pre><pre> 86 PF16532.7 ; Phage_tail_NK ; Sf  38.9      97  0.0009   22.0   2.8   22   15-36    140-161 (175)
</pre><pre> 87 PF02563.18 ; Poly_export ; Pol  38.8 1.4E+02  0.0013   16.8   3.4   25    1-25     10-49  (86)
</pre><pre> 88 6U7J_A Beta-glucuronidase; mic  38.8 2.1E+02   0.002   20.7   4.5   30    4-34    274-303 (594)
</pre><pre> 89 6U7J_D Beta-glucuronidase; mic  38.8 2.1E+02   0.002   20.7   4.5   30    4-34    274-303 (594)
</pre><pre> 90 4S36_A Phage baseplate protein  37.9 1.4E+02  0.0013   16.7   5.2   45    3-48     10-57  (90)
</pre><pre> 91 5T9G_C Glycoside Hydrolase; (a  37.8 1.8E+02  0.0016   22.8   4.2   31    4-34    285-315 (846)
</pre><pre> 92 5T9G_D Glycoside Hydrolase; (a  37.8 1.8E+02  0.0016   22.8   4.2   31    4-34    285-315 (846)
</pre><pre> 93 PF18620.3 ; DUF5627 ; Family o  36.3 1.4E+02  0.0013   20.0   3.2   35   14-48     68-114 (146)
</pre><pre> 94 d1kska3 d.66.1.5 (A:1-59) Pseu  36.0      99 0.00091   15.2   2.0   28   19-46     23-55  (59)
</pre><pre> 95 TIGR02892 spore_yabP; sporulat  35.9 1.8E+02  0.0017   17.4   4.4   36    8-43     20-60  (85)
</pre><pre> 96 5VTG_A Translocation and assem  35.8 2.2E+02  0.0021   18.3   4.0   31   14-46    105-135 (184)
</pre><pre> 97 2VZS_B EXO-BETA-D-GLUCOSAMINID  35.3 2.2E+02   0.002   22.8   4.4   33    4-36    326-362 (1032)
</pre><pre> 98 5UJ6_A Glycosyl hydrolases fam  35.1 2.1E+02   0.002   22.2   4.2   31    4-34    287-317 (878)
</pre><pre> 99 6S0X_i 30S ribosomal protein S  35.0      80 0.00074   19.7   1.8   18   13-30     14-32  (127)
</pre><pre>100 PF12059.10 ; DUF3540 ; Protein  34.8 2.6E+02  0.0024   18.9   4.2   38   11-48     60-97  (191)
</pre><pre>101 3GM8_A Glycoside hydrolase fam  34.8 1.9E+02  0.0018   22.5   4.0   31    4-34    260-290 (801)
</pre><pre>102 PF05910.14 ; DUF868 ; Plant pr  34.5 3.4E+02  0.0032   20.1   5.6   43    5-47    156-205 (273)
</pre><pre>103 1RJL_C Outer surface protein B  34.5 1.8E+02  0.0017   18.9   3.3   25    5-29     44-71  (95)
</pre><pre>104 PF11858.10 ; DUF3378 ; Domain   34.3 1.6E+02  0.0015   16.2   3.1   24    6-29     31-54  (75)
</pre><pre>105 4YGU_D putative adhesin; PF109  34.1 2.1E+02   0.002   17.5   5.0   44    5-48    138-183 (184)
</pre><pre>106 1JZ7_D Beta-Galactosidase; TIM  33.5   3E+02  0.0028   22.5   5.0   30    4-34    324-353 (1023)
</pre><pre>107 PF14498.8 ; Glyco_hyd_65N_2 ;   33.3   2E+02  0.0019   19.2   3.6   25   16-40    212-236 (247)
</pre><pre>108 d1p9ka1 d.66.1.6 (A:3-79) Hypo  33.2 1.4E+02  0.0013   15.7   2.4   28   20-47     43-75  (77)
</pre><pre>109 5EUV_B Beta-D-galactosidase; b  33.1 3.1E+02  0.0029   21.3   4.8   33    1-34    231-263 (731)
</pre><pre>110 TIGR03027 pepcterm_export; put  32.9 1.9E+02  0.0018   18.2   3.3   25    1-25      4-33  (165)
</pre><pre>111 2A50_C GFP-like non-fluorescen  32.8 1.3E+02  0.0012   18.3   2.4   22   14-37     21-42  (73)
</pre><pre>112 PF15507.8 ; DUF4649 ; Domain o  32.3      71 0.00066   19.0   1.2   13   22-34     41-53  (68)
</pre><pre>113 5XLJ_A Flagellar hook-associat  31.9 1.9E+02  0.0018   19.1   3.3   29   19-48    162-192 (208)
</pre><pre>114 PF17115.7 ; Toast_rack_N ; N-t  31.8 1.7E+02  0.0016   16.2   2.7   24    5-28      7-30  (91)
</pre><pre>115 PF03633.17 ; Glyco_hydro_65C ;  31.5 1.4E+02  0.0013   14.6   4.2   30    3-32     18-53  (53)
</pre><pre>116 6PAL_A Uncharacterized protein  31.4 2.4E+02  0.0022   19.3   3.7   23   14-36      5-28  (407)
</pre><pre>117 6ZJS_A Beta-galactosidase; gal  31.3 3.5E+02  0.0033   22.1   5.0   31    3-34    300-330 (1010)
</pre><pre>118 3LPF_B Beta-glucuronidase; alp  31.2 3.7E+02  0.0034   19.4   4.7   30    4-34    265-294 (605)
</pre><pre>119 6U7I_B Beta-glucuronidase; mic  30.3 2.7E+02  0.0025   20.3   3.9   30    4-34    275-304 (597)
</pre><pre>120 PF20132.1 ; DUF6522 ; Family o  30.3 2.1E+02  0.0019   17.4   3.0   20    5-24     54-75  (81)
</pre><pre>121 5H5W_B Flagellar hook-associat  30.0 2.2E+02   0.002   19.0   3.3   29   19-48    159-189 (203)
</pre><pre>122 d2wiqa_ d.22.1.0 (A:) automate  29.9 1.7E+02  0.0016   21.2   2.9   25   11-37      9-33  (228)
</pre><pre>123 1H6R_C GREEN FLUORESCENT PROTE  29.8 1.7E+02  0.0015   21.1   2.9   25   11-37     10-34  (236)
</pre><pre>124 PF13186.8 ; SPASM ; Iron-sulfu  29.6 1.2E+02  0.0011   14.3   1.6   15   10-24      2-16  (65)
</pre><pre>125 PF10988.10 ; DUF2807 ; Putativ  29.5 2.4E+02  0.0022   17.6   3.3   37    8-44    164-203 (203)
</pre><pre>126 d2ejoa_ d.22.1.0 (A:) automate  29.4 1.5E+02  0.0014   21.3   2.7   22   14-37      9-30  (217)
</pre><pre>127 3AKO_E Venus; Venus, Fluoresce  29.4 1.8E+02  0.0016   20.3   2.9   25   11-37     31-55  (173)
</pre><pre>128 3AKO_G Venus; Venus, Fluoresce  29.4 1.8E+02  0.0016   20.3   2.9   25   11-37     31-55  (173)
</pre><pre>129 d2rh7a1 d.22.1.1 (A:7-226) Gre  29.4 1.5E+02  0.0014   21.5   2.6   22   14-37      7-28  (220)
</pre><pre>130 2RA9_A Uncharacterized protein  29.4      92 0.00085   20.4   1.5   21   10-30     21-42  (150)
</pre><pre>131 d2c9ja_ d.22.1.0 (A:) automate  29.2 1.5E+02  0.0014   21.3   2.6   22   14-37      3-24  (216)
</pre><pre>132 1WTH_A Tail-associated lysozym  29.2   3E+02  0.0027   22.1   4.3   46    1-47    529-574 (584)
</pre><pre>133 4IAJ_A Conserved domain protei  29.0      81 0.00075   19.2   1.1   13   22-34     48-60  (78)
</pre><pre>134 4IAJ_D Conserved domain protei  29.0      81 0.00075   19.2   1.1   13   22-34     48-60  (78)
</pre><pre>135 6U9G_B VgrG; T6SS Central Spik  28.9   3E+02  0.0027   17.6   4.1   33    2-34    151-189 (189)
</pre><pre>136 d3hn3a3 c.1.8.3 (A:329-632) be  28.9 1.8E+02  0.0017   18.2   2.7   16   19-34      4-19  (304)
</pre><pre>137 5H5T_A Flagellar hook-associat  28.8 2.3E+02  0.0021   18.9   3.2   29   19-48    161-191 (204)
</pre><pre>138 5H5T_C Flagellar hook-associat  28.8 2.3E+02  0.0021   18.9   3.2   29   19-48    161-191 (204)
</pre><pre>139 6ED2_A Glycosyl hydrolase fami  28.7 4.2E+02  0.0039   19.5   4.7   30    4-34    289-318 (631)
</pre><pre>140 PF14734.8 ; DUF4469 ; Domain o  28.6 1.1E+02   0.001   18.5   1.7   21   16-36     15-35  (99)
</pre><pre>141 2JAD_A YELLOW FLUORESCENT PROT  28.4 1.8E+02  0.0016   21.1   2.8   25   11-37     10-34  (362)
</pre><pre>142 PF13109.8 ; AsmA_1 ; AsmA-like  28.2 3.1E+02  0.0028   17.6   5.2   40    5-47    109-148 (214)
</pre><pre>143 3LYC_B Putative pectinase; Str  28.1 3.2E+02   0.003   17.7   4.7   36    7-42    125-165 (241)
</pre><pre>144 3LYC_G Putative pectinase; Str  28.1 3.2E+02   0.003   17.7   4.7   36    7-42    125-165 (241)
</pre><pre>145 6WDR_Q 40S ribosomal protein S  28.0 1.3E+02  0.0012   19.0   1.9   18   13-30     16-34  (127)
</pre><pre>146 d4he4a1 d.22.1.0 (A:2-234) aut  28.0 1.6E+02  0.0015   21.6   2.6   29    7-37      5-33  (233)
</pre><pre>147 d4jkla3 c.1.8.0 (A:273-596) au  27.9 1.7E+02  0.0015   18.6   2.4   16   19-34      4-19  (324)
</pre><pre>148 d5ltra_ d.22.1.0 (A:) automate  27.9 1.7E+02  0.0016   21.2   2.7   23   13-37      7-29  (219)
</pre><pre>149 3OBA_B Beta-galactosidase; TIM  27.8 3.7E+02  0.0034   22.1   4.6   30    4-34    329-358 (1032)
</pre><pre>150 3OBA_C Beta-galactosidase; TIM  27.8 3.7E+02  0.0034   22.1   4.6   30    4-34    329-358 (1032)
</pre><pre>151 3AI5_A yeast enhanced green fl  27.7 1.8E+02  0.0017   20.2   2.7   25   11-37     13-37  (307)
</pre><pre>152 4G6T_B Type III effector HopA1  27.5 1.6E+02  0.0015   18.6   2.2   19   17-35      4-22  (82)
</pre><pre>153 1P4P_A Outer surface protein B  27.5 2.6E+02  0.0024   19.5   3.3   25    5-29     94-121 (145)
</pre><pre>154 d4dkma_ d.22.1.0 (A:) automate  27.4 1.7E+02  0.0016   21.1   2.6   23   13-37      3-25  (216)
</pre><pre>155 6NCW_C Beta-galacturonidase; g  27.4   4E+02  0.0037   19.5   4.4   30    4-34    245-274 (574)
</pre><pre>156 d2p4ma_ d.22.1.1 (A:) automate  27.3 1.8E+02  0.0016   21.1   2.7   22   14-37      9-30  (221)
</pre><pre>157 3BGA_A Beta-galactosidase; Bet  27.2 4.3E+02   0.004   21.8   4.9   30    4-34    327-356 (1010)
</pre><pre>158 d2wura_ d.22.1.1 (A:) automate  27.1 1.9E+02  0.0018   21.1   2.8   25   11-37      8-32  (230)
</pre><pre>159 6JZ2_A Beta-glucuronidase; b-g  27.0 4.8E+02  0.0044   19.4   4.9   30    4-34    292-321 (627)
</pre><pre>160 1JZ8_D Beta-Galactosidase; TIM  27.0 4.1E+02  0.0038   21.9   4.7   30    4-34    324-353 (1023)
</pre><pre>161 d4jgea_ d.22.1.0 (A:) automate  26.9 1.8E+02  0.0017   21.1   2.7   23   13-37      5-27  (219)
</pre><pre>162 PF07873.13 ; YabP ; YabP famil  26.8 2.3E+02  0.0022   15.7   3.7   26    8-33     20-46  (66)
</pre><pre>163 4OHS_D FAR-RED FLUORESCENT PRO  26.7 1.8E+02  0.0016   21.0   2.6   33    3-37      8-40  (237)
</pre><pre>164 4OHS_E FAR-RED FLUORESCENT PRO  26.7 1.8E+02  0.0016   21.0   2.6   33    3-37      8-40  (237)
</pre><pre>165 d3piba1 d.22.1.1 (A:2-229) aut  26.7 1.8E+02  0.0017   21.1   2.6   22   14-37      9-30  (228)
</pre><pre>166 d4dkna_ d.22.1.0 (A:) automate  26.7 1.9E+02  0.0017   20.9   2.7   23   13-37      4-26  (218)
</pre><pre>167 6GP1_A Green to red photoconve  26.6   2E+02  0.0018   18.5   2.5   22   14-37     42-63  (94)
</pre><pre>168 d1nvpc_ b.56.1.1 (C:) Large ch  26.6 1.6E+02  0.0015   16.1   1.9   15   19-33     25-39  (47)
</pre><pre>169 2ZO6_A CYAN-EMITTING GFP-LIKE   26.5 2.1E+02  0.0019   21.0   2.9   33   14-48     40-72  (252)
</pre><pre>170 7CMG_A Por secretion system pr  26.5 1.6E+02  0.0014   22.6   2.4   42    2-43    250-291 (533)
</pre><pre>171 5XYI_Q 40S ribosomal protein S  26.4 1.1E+02   0.001   19.5   1.5   12   20-31     23-34  (140)
</pre><pre>172 6ECA_B Beta-glucuronidase; Gly  26.4 4.3E+02   0.004   19.6   4.4   30    4-34    292-321 (627)
</pre><pre>173 d2ejha_ d.22.1.0 (A:) automate  26.4 2.1E+02   0.002   20.7   2.9   25   11-37      6-30  (218)
</pre><pre>174 1YQ2_F beta-galactosidase; GLY  26.3 4.1E+02  0.0038   21.9   4.6   31    3-34    302-332 (1024)
</pre><pre>175 4DKM_F Amphioxus Green Fluores  26.3 2.2E+02  0.0021   20.1   3.0   33   14-48      4-36  (214)
</pre><pre>176 d2dd7a_ d.22.1.0 (A:) automate  26.3 1.8E+02  0.0017   21.0   2.6   22   14-37      4-25  (217)
</pre><pre>177 PF02626.17 ; CT_A_B ; Carboxyl  26.2 4.3E+02   0.004   18.6   4.5   37    6-48     30-66  (258)
</pre><pre>178 6KL1_A Green fluorescent prote  26.2   2E+02  0.0019   20.7   2.8   25   11-37      9-33  (229)
</pre><pre>179 6ED1_A Glycosyl hydrolase fami  26.1 4.9E+02  0.0045   20.5   4.8   30    4-34    291-320 (879)
</pre><pre>180 6ED1_B Glycosyl hydrolase fami  26.1 4.9E+02  0.0045   20.5   4.8   30    4-34    291-320 (879)
</pre><pre>181 d4q9wa1 d.22.1.0 (A:2-218) aut  26.1 1.9E+02  0.0018   20.8   2.6   22   14-37      8-29  (215)
</pre><pre>182 d2hpwa_ d.22.1.1 (A:) Green fl  26.0 1.9E+02  0.0018   21.0   2.7   25   11-37      9-33  (231)
</pre><pre>183 4V3P_SI 40S ribosomal protein   25.9 1.4E+02  0.0013   18.6   1.8   16   15-30      4-19  (126)
</pre><pre>184 6BYC_A Beta-mannosidase; CARBO  25.9 3.6E+02  0.0033   21.5   4.1   33    4-36    309-348 (862)
</pre><pre>185 d1p4pa_ b.76.1.1 (A:) Outer su  25.6 2.9E+02  0.0027   19.2   3.3   25    5-29     89-116 (140)
</pre><pre>186 PF12791.9 ; RsgI_N ; Anti-sigm  25.6 1.9E+02  0.0017   15.2   2.0   14   11-24      5-18  (50)
</pre><pre>187 d2g6ya_ d.22.1.0 (A:) automate  25.6   2E+02  0.0018   20.8   2.6   22   14-37      3-24  (216)
</pre><pre>188 d1xssa_ d.22.1.0 (A:) automate  25.5   2E+02  0.0018   21.0   2.6   22   14-37      9-30  (226)
</pre><pre>189 4W6T_A fluorescent protein E11  25.4 2.2E+02   0.002   20.5   2.8   25   11-37     11-35  (229)
</pre><pre>190 7BWN_J Chimera of Green fluore  25.2 2.2E+02   0.002   20.9   2.9   25   11-37     10-34  (280)
</pre><pre>191 7BWN_O Chimera of Green fluore  25.2 2.2E+02   0.002   20.9   2.9   25   11-37     10-34  (280)
</pre><pre>192 PF06097.13 ; DUF945 ; Bacteria  25.2      84 0.00078   22.5   0.8   14   19-32    463-476 (477)
</pre><pre>193 1R6V_A subtilisin-like serine   25.1 1.6E+02  0.0014   23.3   2.3   14   21-34    623-636 (671)
</pre><pre>194 5T2A_AK uS9; ribosome, Leishma  25.1 1.4E+02  0.0013   19.4   1.8   18   13-30     23-42  (149)
</pre><pre>195 4DKN_B AMPHIOXUS GREEN FLUORES  25.0 2.5E+02  0.0023   20.0   3.0   33   14-48     10-42  (221)
</pre><pre>196 6D6W_B Beta-galactosidase/beta  25.0 4.6E+02  0.0043   19.3   4.3   30    4-34    289-318 (603)
</pre><pre>197 PF08845.12 ; SymE_toxin ; Toxi  24.8   2E+02  0.0019   14.3   2.8   19    1-19     33-51  (51)
</pre><pre>198 6HR1_B Myosin light chain kina  24.8 1.9E+02  0.0018   21.7   2.6   25   11-37     46-70  (434)
</pre><pre>199 d1v58a2 d.17.3.1 (A:2-61) Thio  24.7 1.6E+02  0.0015   15.9   1.7   21   15-35     40-60  (60)
</pre><pre>200 6WVD_A Green fluorescent prote  24.6 2.1E+02   0.002   22.5   2.8   25   11-37     10-34  (420)
</pre><pre>201 PF14262.8 ; Cthe_2159 ; Carboh  24.6 4.2E+02  0.0039   18.8   4.0   32    3-36    259-290 (292)
</pre><pre>202 6DDT_A Beta-mannosidase; manno  24.5 3.1E+02  0.0029   21.9   3.7   33    4-36    304-345 (868)
</pre><pre>203 3HN3_B Beta-glucuronidase; LYS  24.5 4.6E+02  0.0042   19.3   4.2   30    4-34    298-327 (613)
</pre><pre>204 d3qdpa_ d.212.1.1 (A:) automat  24.5 1.5E+02  0.0014   16.1   1.6   11   14-24     33-43  (88)
</pre><pre>205 d1yzwa_ d.22.1.1 (A:) automate  24.4 2.1E+02   0.002   20.8   2.6   22   14-37      8-29  (221)
</pre><pre>206 4LQT_A Green fluorescent prote  24.4 2.3E+02  0.0021   20.7   2.8   25   11-37     20-44  (246)
</pre><pre>207 6L27_A Green fluorescent prote  24.4 2.3E+02  0.0022   20.3   2.8   25   11-37      8-32  (227)
</pre><pre>208 2HPW_A Green fluorescent prote  24.4 2.2E+02   0.002   20.6   2.7   25   11-37     13-37  (233)
</pre><pre>209 6ITX_C Fiber protein; avian ad  24.1 2.8E+02  0.0026   21.0   3.3   41    8-48     28-69  (301)
</pre><pre>210 6ITX_F Fiber protein; avian ad  24.1 2.8E+02  0.0026   21.0   3.3   41    8-48     28-69  (301)
</pre><pre>211 6RM3_SQ0 uS9; Microsporidia, R  24.0 1.9E+02  0.0018   18.9   2.2   18   13-30     19-38  (155)
</pre><pre>212 TIGR02856 spore_yqfC; sporulat  23.7 1.8E+02  0.0017   17.3   2.0   16   18-33     50-65  (85)
</pre><pre>213 6GJE_C Cubilin; receptor, vita  23.6 3.3E+02   0.003   16.4   3.0   26    5-30     25-53  (110)
</pre><pre>214 PF16475.7 ; DUF5052 ; Domain o  23.5 5.2E+02  0.0048   18.6   4.6   36    3-38     47-103 (204)
</pre><pre>215 PF11739.10 ; DctA-YdbH ; Dicar  23.5 2.8E+02  0.0026   18.3   2.9   42    4-48    138-179 (203)
</pre><pre>216 PF10618.11 ; Tail_tube ; Phage  23.4 2.2E+02  0.0021   17.6   2.3   23   22-44      9-31  (115)
</pre><pre>217 2KW1_A Fas apoptotic inhibitor  23.3 3.3E+02   0.003   16.2   4.1   29    7-35     11-42  (89)
</pre><pre>218 2DD7_A green fluorescent prote  23.3 2.7E+02  0.0025   19.7   3.0   32   14-48      4-36  (216)
</pre><pre>219 7K00_I 30S ribosomal protein S  23.2   1E+02 0.00096   19.5   0.9   11   20-30     25-35  (130)
</pre><pre>220 6XXW_A Beta-glucuronidase; Gly  23.2 5.5E+02  0.0051   18.8   4.7   30    4-34    274-303 (598)
</pre><pre>221 d2c9ia_ d.22.1.1 (A:) automate  23.2 2.3E+02  0.0021   20.7   2.6   22   14-37     10-31  (228)
</pre><pre>222 PF06905.15 ; FAIM1 ; Fas apopt  23.1 4.4E+02  0.0041   17.6   4.3   29    7-35     96-127 (172)
</pre><pre>223 2C9I_B GREEN FLUORESCENT PROTE  23.0 2.6E+02  0.0024   20.0   2.8   33   14-48     10-42  (226)
</pre><pre>224 2C9I_D GREEN FLUORESCENT PROTE  23.0 2.6E+02  0.0024   20.0   2.8   33   14-48     10-42  (226)
</pre><pre>225 4HE4_A Yellow fluorescent prot  23.0 2.6E+02  0.0024   20.3   2.9   25   11-37     10-34  (244)
</pre><pre>226 d2ib5a_ d.22.1.1 (A:) automate  22.9 2.4E+02  0.0022   20.7   2.6   22   14-37      6-27  (228)
</pre><pre>227 PF06236.13 ; MelC1 ; Tyrosinas  22.9 2.2E+02   0.002   18.6   2.3   12   22-33     59-70  (109)
</pre><pre>228 6S68_A Aequorea cf. australis   22.9   3E+02  0.0028   19.7   3.1   36   11-48     10-45  (230)
</pre><pre>229 TIGR03028 EpsE; polysaccharide  22.9   4E+02  0.0037   17.8   3.5   25    1-25      5-34  (239)
</pre><pre>230 3AI4_A yeast enhanced green fl  22.9 2.6E+02  0.0024   20.9   2.9   25   11-37     13-37  (283)
</pre><pre>231 3RD4_B uncharacterized protein  22.8 1.6E+02  0.0015   18.1   1.7   28    3-30     49-79  (100)
</pre><pre>232 PF06739.13 ; SBBP ; Beta-prope  22.8 1.7E+02  0.0016   12.8   1.6   14   14-27     17-30  (37)
</pre><pre>233 2C9J_A GREEN FLUORESCENT PROTE  22.5 3.3E+02   0.003   19.4   3.2   32   14-47      9-40  (223)
</pre><pre>234 6WVE_A Green fluorescent prote  22.3 2.6E+02  0.0024   21.1   2.8   25   11-37     10-34  (312)
</pre><pre>235 6TH6_Al 30S ribosomal protein   22.2 1.2E+02  0.0011   19.3   1.0   11   20-30     23-33  (135)
</pre><pre>236 d2qali1 d.14.1.1 (I:3-129) Rib  22.1 1.8E+02  0.0016   18.9   1.8   16   14-30     17-32  (127)
</pre><pre>237 TIGR04411 T2SS_GspN_Lepto; typ  22.0 4.4E+02  0.0041   17.3   4.6   41    5-48    192-236 (279)
</pre><pre>238 2RH7_C Green fluorescent prote  22.0 2.6E+02  0.0025   20.2   2.7   25   11-37     10-34  (239)
</pre><pre>239 6U1A_A Red fluorescent protein  21.9 2.5E+02  0.0023   20.2   2.6   22   14-37     11-32  (232)
</pre><pre>240 4CVU_A BETA-MANNOSIDASE; HYDRO  21.8 4.7E+02  0.0043   21.4   4.2   33    4-36    340-388 (942)
</pre><pre>241 d2gw3a1 d.22.1.0 (A:1-220) aut  21.8   3E+02  0.0028   20.1   2.9   25   11-37      6-30  (220)
</pre><pre>242 d4ohsa_ d.22.1.0 (A:) automate  21.7 2.6E+02  0.0024   20.4   2.6   22   14-37     10-31  (225)
</pre><pre>243 3IR8_A Large stokes shift fluo  21.7 3.7E+02  0.0034   19.2   3.3   32   14-47     11-42  (221)
</pre><pre>244 4HN7_A Signal transduction pro  21.7 3.8E+02  0.0035   16.4   3.2   22   12-33     43-65  (88)
</pre><pre>245 4Q9W_B GFP-like fluorescent ch  21.5 3.5E+02  0.0033   19.2   3.2   32   14-47      9-40  (218)
</pre><pre>246 PF04872.15 ; Pox_L5 ; Poxvirus  21.4 1.3E+02  0.0012   18.6   1.0   28    7-34     23-56  (78)
</pre><pre>247 3EVP_A Green fluorescent prote  21.4 2.7E+02  0.0026   20.3   2.7   25   11-37    111-135 (243)
</pre><pre>248 6B7T_A Green fluorescent prote  21.4 2.9E+02  0.0027   20.3   2.9   25   11-37     61-85  (246)
</pre><pre>249 6S67_B Aequorea cf. australis   21.4 3.3E+02  0.0031   19.5   3.1   36   11-48     10-45  (230)
</pre><pre>250 6S67_D Aequorea cf. australis   21.4 3.3E+02  0.0031   19.5   3.1   36   11-48     10-45  (230)
</pre><pre>
</pre><pre>No 1
</pre><pre>>6J0M_C Pvc8; assembly, Photorhabdus asymbiotica, PVC, contractile injection system, bacteriophage-like, PROTEIN TRANSPORT; 3.9A {Photorhabdus asymbiotica}
</pre><pre>Probab=96.13  E-value=0.059  Score=37.14  Aligned_cols=48  Identities=21%  Similarity=0.524  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeE----------------EEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKT----------------IDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~----------------I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .....+.+.+++..+++..++.|.|++..                +.+.+.+.|++.|..|++|
</pre><pre>T Consensus       474 ~~~~~i~~~~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~i~i~~~~i~~~  537 (538)
</pre><pre>T 6J0M_C          474 GKEKMIRINAGENQIMLQQDKDISLSTKKELTLKAQTMNATMDKSLAMSGKNSVEIKGAKINLT  537 (538)
</pre><pre>T ss_dssp             TTTTEEEEEBSSSCEEEECSSCCCCBCSSEEEEEESEEEEEESSCEEEEBSSCEEEECSSEEEC
</pre><pre>T ss_pred             CCCCeEEEEeCCceEEEcCCCcEEEcCCCeEEEeecceEEEeCCceeEeecceEEEecceeecC
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>6RBK_C Afp8; Anti-feeding prophage, secretion system, AFP, contractile, VIRUS LIKE PARTICLE, baseplate; 3.4A {Serratia entomophila}
</pre><pre>Probab=95.79  E-value=0.096  Score=36.17  Aligned_cols=48  Identities=15%  Similarity=0.532  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEc----------------CeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLN----------------GKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~----------------G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..+..+++.+++..++++.+|.|.++                +..+.+.+.+.+++.+..|.+|
</pre><pre>T Consensus       465 ~~~~~~~~~~~~~~~~l~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~i~~~~i~~~  528 (529)
</pre><pre>T 6RBK_C          465 TEEKTLRLMAGDNTLTLTGEGNLTMSTPNALQLQADTLGLQADSNLSIAGKQQVEITSAKINMK  528 (529)
</pre><pre>T ss_dssp             TTTTCCEECSSSCCCCCCTTSCCCCCCSSCCCCCCSSCCCCCSSCCCCCCSSCCCCCCSSCCC-
</pre><pre>T ss_pred             CCCCEEEEEcCCcEEEEcCCCcEEEcCCCeEEEecCcEEEEeCCceEEeccCcEEEEeceeeec
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>4UHV_A VGRG1, VALINE-GLYCINE REPEAT PROTEIN G1; STRUCTURAL PROTEIN, VGRG1, VIRULENCE, TOXIN, EFFECTOR, PUNCTURING DEVICE, SPIKE, T6SS, P. AERUGINOSA; HET: CL, NA, MSE; 2.0A {PSEUDOMONAS AERUGINOSA}
</pre><pre>Probab=95.73  E-value=0.082  Score=37.94  Aligned_cols=48  Identities=35%  Similarity=0.590  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCce-eecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKR-IDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~-I~~n   48 (48)
</pre><pre>                      ++++.+++.+|++.+.+..++.+.+.|..+.+.+++.+++.+.. +.++
</pre><pre>T Consensus       563 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  611 (651)
</pre><pre>T 4UHV_A          563 EAGAQIRLVCGKSVVEFNADGTINISGSAFNLYASGNGNIDTGGRLDLN  611 (651)
</pre><pre>T ss_dssp             CCSSEEEEEETTEEEEEESSCCEEEECSCCCCCSCCCCCCCCCSSCCSS
</pre><pre>T ss_pred             EeCCeEEEEeCCcEEEECCCCeEEEeCCcEEEEeCCceEEEeCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>3QR8_A Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; HET: MSE; 2.03A {Enterobacteria phage P2}
</pre><pre>Probab=95.67  E-value=0.15  Score=31.61  Aligned_cols=47  Identities=17%  Similarity=0.189  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..+..+.+..++ .+.+..+|.+.++++.+.+.+...++++++.+.++
</pre><pre>T Consensus       114 ~~~~~i~~~~~~-~i~i~~~g~v~i~~~~~~~~a~~~i~i~~~~~~~~  160 (211)
</pre><pre>T 3QR8_A          114 PETSALTVSGIK-TASVTASGSVTATVPVVMVKASTRVTLDTPEVVCT  160 (211)
</pre><pre>T ss_dssp             TTTTEEEEESCS-CCCCCCSSCCCCCCSCCCCCCSSCCCCCSSCCCCC
</pre><pre>T ss_pred             CCCCeEEEeeCc-eEEEEcCCcEEEEcCeEEEecCCeEEEeCCceEEc
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>3VTN_A Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.75A {Enterobacteria phage Mu}
</pre><pre>Probab=95.25  E-value=0.4  Score=26.68  Aligned_cols=48  Identities=21%  Similarity=0.383  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..|+-+.-.--+..|.+..+|.|.|+...+.+.+...+.+.++.+.++
</pre><pre>T Consensus         4 ~~G~~~~y~~~~~~i~~~~~g~i~i~~~~~~i~~~~~~~~~~~~i~~~   51 (101)
</pre><pre>T 3VTN_A            4 EPGDAGIYHHEGHRIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFT   51 (101)
</pre><pre>T ss_dssp             -------------CEEECTTSCEEECSSCCCCCCSSCCCCCCSCCCCS
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEcCCCEEEEeCCeEEEEcCCceEEeCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>d1k28a2 b.108.1.2 (A:362-576) Tail-associated lysozyme gp5, C-terminal domain {Bacteriophage T4 [TaxId: 10665]}
</pre><pre>Probab=94.34  E-value=0.33  Score=31.47  Aligned_cols=47  Identities=21%  Similarity=0.392  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .++..+.+.+++. +.+...|.+.+..-.+.+.+++.|.++|+.|.||
</pre><pre>T Consensus       168 ~v~g~~~~~v~~~-~~~~v~g~~~~~~g~~~~~~~g~i~i~g~~i~in  214 (215)
</pre><pre>T d1k28a2         168 TVNGNLSWKVAGT-VDWDVGGDWTEKMASMSSISSGQYTIDGSRIDIG  214 (215)
</pre><pre>T ss_dssp             CCSSCCCCCCSSC-CCCCCSSCCCCCCSCCCCCCSSCCCCCCSSCCCC
</pre><pre>T ss_pred             EEcCcEEEEEeCc-eEEEECCcEEEEeccEEEEcCCeEEEeCceeecC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>6U9G_B VgrG; T6SS Central Spike, Complex, TRANSPORT PROTEIN; 3.98A {Francisella tularensis subsp. novicida (strain U112)}
</pre><pre>Probab=92.79  E-value=1.1  Score=28.20  Aligned_cols=40  Identities=25%  Similarity=0.562  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCc
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGK   43 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~   43 (48)
</pre><pre>                      +..+++.+|++.+.+..+ .|.+.+..+.+.+...+.+.+.
</pre><pre>T Consensus       106 ~~~i~l~~~~~~i~i~~~-~i~i~~~~v~i~~~~~~~i~~~  145 (189)
</pre><pre>T 6U9G_B          106 EDKIVIKIGNSLIILDDS-NISLESATINIKSSANINIQAS  145 (189)
</pre><pre>T ss_dssp             TTEEEEEETTEEEEECSS-CEEEEESEEEECCSSCEEECCS
</pre><pre>T ss_pred             CCeEEEEECCEEEEEcCC-CEEEEcCcEEEEcCCceEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>3VTO_B Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=92.35  E-value=2.3  Score=25.16  Aligned_cols=48  Identities=21%  Similarity=0.383  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..|+-+...-.+..|.+..+|.|.+....+.+.+...+.++++.+.++
</pre><pre>T Consensus        10 ~dG~~~~yd~~g~~l~i~~~g~i~i~~~~~~i~a~~~i~~~~~~~~i~   57 (115)
</pre><pre>T 3VTO_B           10 EPGDAGIYHHEGHRIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFT   57 (115)
</pre><pre>T ss_dssp             CTTCEEEECTTSCEEEEETTTEEEEEEEEEEEEEEEEEEEECSEEEES
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEEcCCEEEEeCCeEEEEcCCcEEEeCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>3VTO_C Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=92.35  E-value=2.3  Score=25.16  Aligned_cols=48  Identities=21%  Similarity=0.383  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..|+-+...-.+..|.+..+|.|.+....+.+.+...+.++++.+.++
</pre><pre>T Consensus        10 ~dG~~~~yd~~g~~l~i~~~g~i~i~~~~~~i~a~~~i~~~~~~~~i~   57 (115)
</pre><pre>T 3VTO_C           10 EPGDAGIYHHEGHRIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFT   57 (115)
</pre><pre>T ss_dssp             CTTCEEEECTTSCEEEEETTTEEEEEEEEEEEEEEEEEEEESSEEEES
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEEcCCEEEEeCCeEEEEcCCcEEEeCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>4KU0_C Tail-associated lysozyme; PAAR-repeat motif, membrane piercing, type VI secretion system, T6SS, cell puncturing device, beta-helix, gp5-gp27 protein complex; HET: ELA, STE, PLM, EDO; 1.15A {Enterobacteria phage T4}
</pre><pre>Probab=91.75  E-value=1.9  Score=23.21  Aligned_cols=47  Identities=21%  Similarity=0.392  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .++..+.+.++ ..+.+...+.+.+....-...+.+.|.+.+..+.+|
</pre><pre>T Consensus        50 ~~g~~~~~~~~-~~~~~~~g~~~~i~~~~~~~~~~~~i~i~g~~~~~~   96 (96)
</pre><pre>T 4KU0_C           50 TVNGNLSWKVA-GTVDWDVGGDWTEKMASMSSISSGQYTIDGSRIDIG   96 (96)
</pre><pre>T ss_dssp             EESSCEEEEES-SCEEEEESSEEEEEESEEEEEESSEEEEECSEEEEC
</pre><pre>T ss_pred             EECCcEEEEEe-ceEEEEeCCcEEEEeCceEEEeCceEEEeCceEeeC
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>3AQJ_A Baseplate assembly protein V; bacteriophage, tail spike, iron binding, beta-helix, infection, METAL BINDING PROTEIN; 1.27A {Enterobacteria phage P2}
</pre><pre>Probab=91.72  E-value=3.3  Score=24.96  Aligned_cols=47  Identities=17%  Similarity=0.189  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..+..+.+.+++ .+.+..++.+.+....+.+.++..+.++++.+.++
</pre><pre>T Consensus        37 ~~~~~l~i~~~g-~i~i~~~~~i~~~~~~~~i~~~~~v~~~~~~i~~~   83 (134)
</pre><pre>T 3AQJ_A           37 PETSALTVSGIK-TASVTASGSVTATVPVVMVKASTRVTLDTPEVVCT   83 (134)
</pre><pre>T ss_dssp             TTTTEEEEEEEE-EEEEEEEEEEEEECSEEEEECSSEEEEESSEEEEE
</pre><pre>T ss_pred             CCCCEEEEEeCc-eeEEEcCceEEEECCeEEEEeCcceEEeCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>4S37_B Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: SO4, CL, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=91.48  E-value=2.6  Score=25.68  Aligned_cols=41  Identities=7%  Similarity=0.235  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCc-EEEEcCeEEEEEeccceEEcCce
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDG-TITLNGKTIDVVGSDHIQLDGKR   44 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG-~I~i~G~~I~i~gs~~i~~~~~~   44 (48)
</pre><pre>                      +..+++..++..++++.++ .+.++...+.+++.. ++++++.
</pre><pre>T Consensus       114 ~~~i~i~~~~~~i~~~~~~~~~~~~~~~~~i~~~~-v~i~g~~  155 (185)
</pre><pre>T 4S37_B          114 AHRYRVELPSGTVEVRVGASEVRVSDGAVSLKAPK-ISLEGPV  155 (185)
</pre><pre>T ss_dssp             TTEEEEEESSSEEEEEETTEEEEEESSCEEEECSC-EEEESCE
</pre><pre>T ss_pred             CCEEEEEeCCceEEEEECCcEEEEeCCcEEEECCc-EEEeCCE
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>4S37_O Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: CL, SO4, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=91.48  E-value=2.6  Score=25.68  Aligned_cols=41  Identities=7%  Similarity=0.235  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCc-EEEEcCeEEEEEeccceEEcCce
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDG-TITLNGKTIDVVGSDHIQLDGKR   44 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG-~I~i~G~~I~i~gs~~i~~~~~~   44 (48)
</pre><pre>                      +..+++..++..++++.++ .+.++...+.+++.. ++++++.
</pre><pre>T Consensus       114 ~~~i~i~~~~~~i~~~~~~~~~~~~~~~~~i~~~~-v~i~g~~  155 (185)
</pre><pre>T 4S37_O          114 AHRYRVELPSGTVEVRVGASEVRVSDGAVSLKAPK-ISLEGPV  155 (185)
</pre><pre>T ss_dssp             TTEEEEEESSSEEEEEETTEEEEEESSCEEEECSC-EEEESCE
</pre><pre>T ss_pred             CCEEEEEeCCceEEEEECCcEEEEeCCcEEEECCc-EEEeCCE
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>3QR7_B Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; 0.94A {Enterobacteria phage P2}
</pre><pre>Probab=91.01  E-value=3.5  Score=23.65  Aligned_cols=47  Identities=17%  Similarity=0.189  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..+..+.+...+ .+.++..+.+.+....+.+.+++.+.++++.+.++
</pre><pre>T Consensus        18 ~~~~~l~~~~~g-~i~v~~~~~~~~~~~~~~~~~~~~v~~~~~~~~~~   64 (115)
</pre><pre>T 3QR7_B           18 PETSALTVSGIK-TASVTASGSVTATVPVVMVKASTRVTLDTPEVVCT   64 (115)
</pre><pre>T ss_dssp             TTTTEEEC-CCS-CCCCCCSSCCCCCCSCCCCCCSSCCCCCSSCCCCS
</pre><pre>T ss_pred             CCCCeEEEEecc-ceEEEcCccEEEECCeEEEEeCceEEEeCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>3A1M_A chimera of thrombin cleavage site, Tail-associated lysozyme, Fibritin; bacteriophage T4, Antimicrobial, Bacteriolytic enzyme, Glycosidase, Hydrolase, Late protein, Chaperone, STRUCTURAL PROTEIN; 2.0A {Enterobacteria phage T4}
</pre><pre>Probab=90.98  E-value=2.7  Score=25.93  Aligned_cols=47  Identities=21%  Similarity=0.392  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +++.++.+.+| ..+.+..++.++++...+.+.++..+.+.++.+.++
</pre><pre>T Consensus        55 ~vg~~~~v~~g-~~~~l~~g~~itl~~~~~~~~~~~~i~i~g~~i~l~  101 (139)
</pre><pre>T 3A1M_A           55 TVNGNLSWKVA-GTVDWDVGGDWTEKMASMSSISSGQYTIDGSRIDIG  101 (139)
</pre><pre>T ss_dssp             EESSCEEEEES-SCEEEEESSEEEEEESEEEEEESSEEEEECSEEEET
</pre><pre>T ss_pred             EeCCeEEEEEC-CeEEEEeCCeEEEEecceEEecCceEEEECCeEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>3A1M_E chimera of thrombin cleavage site, Tail-associated lysozyme, Fibritin; bacteriophage T4, Antimicrobial, Bacteriolytic enzyme, Glycosidase, Hydrolase, Late protein, Chaperone, STRUCTURAL PROTEIN; 2.0A {Enterobacteria phage T4}
</pre><pre>Probab=90.98  E-value=2.7  Score=25.93  Aligned_cols=47  Identities=21%  Similarity=0.392  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +++.++.+.+| ..+.+..++.++++...+.+.++..+.+.++.+.++
</pre><pre>T Consensus        55 ~vg~~~~v~~g-~~~~l~~g~~itl~~~~~~~~~~~~i~i~g~~i~l~  101 (139)
</pre><pre>T 3A1M_E           55 TVNGNLSWKVA-GTVDWDVGGDWTEKMASMSSISSGQYTIDGSRIDIG  101 (139)
</pre><pre>T ss_dssp             EESSCEEEEES-SCEEEEESSEEEEEESEEEEEESSEEEEECSEEEEC
</pre><pre>T ss_pred             EeCCeEEEEEC-CeEEEEeCCeEEEEecceEEecCceEEEECCeEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>6P20_C Baseplate central spike complex protein gp5,PHIKZ164; bacteriophage, phiKZ, membrane piercing, central spike, cell puncturing device, PAAR-repeat motif, beta helix, T4 gp5, Pseudomonas; HET: STE, PLM, ELA, EDO; 1.749A {Enterobacteria phage T4}
</pre><pre>Probab=89.79  E-value=1.8  Score=24.11  Aligned_cols=45  Identities=16%  Similarity=0.169  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEe-ccceEEcCceeec
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVG-SDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g-s~~i~~~~~~I~~   47 (48)
</pre><pre>                      .+++.+++.|  ..+.....+.|.+.|..+.+.+ .......++..++
</pre><pre>T Consensus        66 ~~~~~~~~~~--~~~~~~~~~~i~~~g~~~~i~~~~~~~~~~~~~~~~  111 (112)
</pre><pre>T 6P20_C           66 DVGGDWTEKM--ASMSSKSSGTHIQEAGGTMTHKAGGNMLFTAPRYDF  111 (112)
</pre><pre>T ss_dssp             EESSEEEEEE--SEEEEEESSCEEEEESSCEEEEESSCEEEECSCEEE
</pre><pre>T ss_pred             EeCCCEEEEe--cceEEecCCeEEEeeeeEEecCCCccccccccccCC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>PF12059.10 ; DUF3540 ; Protein of unknown function (DUF3540)
</pre><pre>Probab=89.60  E-value=1.1  Score=29.66  Aligned_cols=34  Identities=18%  Similarity=0.391  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      ++....++...+.++...+.+.+.+++||+.|++
</pre><pre>T Consensus       157 ~~~~~a~~~~~~~a~~~~i~a~~~vkiDg~rIhl  190 (191)
</pre><pre>T F3YX79_DESAF/1  157 TQRVLVEDLLTMHSKNAMIVSEEHVTVNAEQIHL  190 (191)
</pre><pre>T ss_pred             eEEEEEEeeEEEEeccEEEeceeceEeChhhccc
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF11101.10 ; DUF2884 ; Protein of unknown function (DUF2884)
</pre><pre>Probab=86.23  E-value=2.4  Score=28.68  Aligned_cols=35  Identities=20%  Similarity=0.461  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeC-CcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVG-KSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG-~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+++.-. ....++..||.++|+|+.+.+..+..
</pre><pre>T Consensus        13 ~~~v~i~~~~~~~~~i~~~g~L~idG~~v~L~~~Qq   48 (227)
</pre><pre>T Q3IDE0_PSET1/3   13 PDELEIFTANKQTMQFNTNGNLTINGESVALNHSQQ   48 (227)
</pre><pre>T ss_pred             CCeEEEEeCCCceEEEcCCCcEEECCEEecCCHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF04790.15 ; Sarcoglycan_1 ; Sarcoglycan complex subunit protein
</pre><pre>Probab=80.61  E-value=20  Score=25.01  Aligned_cols=48  Identities=10%  Similarity=0.220  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcC----------eEEEEEe-ccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNG----------KTIDVVG-SDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G----------~~I~i~g-s~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..++++.++.-..++.++....+.+++          ++|.++. .+.|.++++.|.++
</pre><pre>T Consensus       164 ~~~~dL~leS~~~~l~l~g~egv~i~s~~~~i~~~s~~dI~L~S~~G~I~l~a~~i~l~  222 (253)
</pre><pre>T E3LFZ6_CAERE/7  164 QPDSPLRLESPTRSVSIDAAQDIEILAAAGEVSVNSLLDISISSKQGEIRLESSSIYME  222 (253)
</pre><pre>T ss_pred             CCCCCceeeeCCcEEEEecCCeEEEEEecceEEEEecceEEEEEcCcEEEEecccEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>6ORJ_A PHIKZ164; Membrane puncturing device, beta-helix, OB-fold, trimer, VgrG-like, VIRAL PROTEIN; HET: MSE; 2.37A {Pseudomonas virus phiKZ}
</pre><pre>Probab=80.21  E-value=15  Score=27.38  Aligned_cols=43  Identities=19%  Similarity=0.198  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCcee
</pre><pre>Q MBL0283831.1/7    2 AGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRI   45 (48)
</pre><pre>Q Consensus         2 aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I   45 (48)
</pre><pre>                      +-+++.++||++++.|+++- |..+...+.+..++.....+..+
</pre><pre>T Consensus       232 a~~~I~~~~~~~~~~~~~~~-i~~~t~~~~~~~~~t~~~~~~~~  274 (293)
</pre><pre>T 6ORJ_A          232 ADQYIKLTSGGSTLELNPTE-FKVNTTNTTIKSSGTHIQEAGGT  274 (293)
</pre><pre>T ss_dssp             CSSCEEEEETTEEEEEETTE-EEEECSCCCCCCSSCCCCCCSSC
</pre><pre>T ss_pred             eCceEEEEeCCceEEEcCcc-eEecceeeEeccccceEEecCce
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>4RU3_A puncturing protein gp41; gp41, membrane translocation, phage baseplate, Cell puncturing device, CELL INVASION; 1.172A {Pseudomonas phage SN}
</pre><pre>Probab=76.72  E-value=20  Score=23.52  Aligned_cols=38  Identities=13%  Similarity=0.469  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCc--EEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDG--TITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG--~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ......-.++|  .|.+++..|.|.+++.++++++.+.++
</pre><pre>T Consensus       145 ~~~~~i~~~~G~~~I~i~~~~i~i~~~~~v~i~~~~i~i~  184 (221)
</pre><pre>T 4RU3_A          145 SAAMVIQSTSGATRISIRGDNIKITAPSNVTVDTPQANFT  184 (221)
</pre><pre>T ss_dssp             TTSEEEECTTSSSEEEECSSCEEEECSSCCCCCSSCCCCS
</pre><pre>T ss_pred             cCceEEEcCCCCeEEEEECCeEEEeCCCcEEEcCCceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>PF10106.11 ; DUF2345 ; Uncharacterized protein conserved in bacteria (DUF2345)
</pre><pre>Probab=75.16  E-value=27  Score=20.62  Aligned_cols=46  Identities=13%  Similarity=0.284  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      ...+.+.+.+.+.-..-...+.|.+++..|.+..++.++++ ..+.+
</pre><pre>T Consensus        96 s~~~~~~~~a~~~i~~~~~g~~i~~~~~~i~~~~~g~~~~~-~~~~~  141 (145)
</pre><pre>T X2H4H4_9NEIS/6   96 SVGDEVLVNAKSGIKLASGGAYIEIKGGKISLYSPGPIEYK-GNHQF  141 (145)
</pre><pre>T ss_pred             ecCCEEEEEcCCeEEEEeCCcEEEEeCCEEEEEcCCceEEE-ceeee
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>6SK0_B Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=73.28  E-value=31  Score=26.47  Aligned_cols=47  Identities=19%  Similarity=0.338  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCc-EEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDG-TITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG-~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..+..+.+...+ .|.+...| .|.|+|..|.+.+++.+.+.+....+.
</pre><pre>T Consensus       714 s~~~~i~i~a~~-~i~l~~gg~~i~l~~~~i~~~~~g~~~~~~~~~~~~  761 (841)
</pre><pre>T 6SK0_B          714 SSEGKVTVSANQ-ELILACGGAYIKLSGGNIELGCPGQILLKSTNMQKM  761 (841)
</pre><pre>T ss_dssp             ESSSEEEEEESS-EEEEEETTEEEEEETTEEEEECSSCEEEC-------
</pre><pre>T ss_pred             eCCCEEEEeCCC-EEEEEeCCeEEEEeCCeeEEecCCeEEEEccccccc
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>6SK0_C Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=73.28  E-value=31  Score=26.47  Aligned_cols=47  Identities=19%  Similarity=0.338  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCc-EEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDG-TITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG-~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      ..+..+.+...+ .|.+...| .|.|+|..|.+.+++.+.+.+....+.
</pre><pre>T Consensus       714 s~~~~i~i~a~~-~i~l~~gg~~i~l~~~~i~~~~~g~~~~~~~~~~~~  761 (841)
</pre><pre>T 6SK0_C          714 SSEGKVTVSANQ-ELILACGGAYIKLSGGNIELGCPGQILLKSTNMQKM  761 (841)
</pre><pre>T ss_dssp             ESSSEEEEEESS-EEEEEETTEEEEEETTEEEEECSSCEEEC-------
</pre><pre>T ss_pred             eCCCEEEEeCCC-EEEEEeCCeEEEEeCCeeEEecCCeEEEEccccccc
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>PF16110.7 ; DUF4828 ; Domain of unknown function (DUF4828)
</pre><pre>Probab=71.69  E-value=20  Score=21.62  Aligned_cols=20  Identities=10%  Similarity=0.232  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~   33 (48)
</pre><pre>                      +|++++|.+|+|+++.+..+
</pre><pre>T Consensus        18 ~L~I~~~~~i~id~~~l~~~   37 (79)
</pre><pre>T Q03Q63_LACBA/3   18 RLEISPNLDVQIDDHQLQVR   37 (79)
</pre><pre>T ss_pred             EEEECCCeEEEECCEEeeEE
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>TIGR02803 ExbD_1; TonB system transport protein ExbD, group 1. Members of this family are Gram-negative bacterial inner membrane proteins, generally designated ExbD, related to the TolR family modeled by TIGRFAMs TIGR02801.
</pre><pre>Probab=71.22  E-value=16  Score=21.73  Aligned_cols=17  Identities=12%  Similarity=0.253  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .+.+.++|.|+++|+.+
</pre><pre>T Consensus        51 ~i~i~~~g~i~~~~~~i   67 (122)
</pre><pre>T TIGR02803        51 YVSVKADLSLFVGNDPV   67 (122)
</pre><pre>T ss_pred             EEEEcCCCEEEECCEEe
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>2JV8_A Uncharacterized protein NE1242; solution structure, NESG, STRUCTURAL GENOMICS, UNKNOWN FUNCTION, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium; NMR {Nitrosomonas europaea ATCC 19718}
</pre><pre>Probab=68.03  E-value=43  Score=19.90  Aligned_cols=30  Identities=27%  Similarity=0.361  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~   33 (48)
</pre><pre>                      ...+.+-.+.+|++..+-.++|+|+++.+.
</pre><pre>T Consensus         2 ~~~~e~ykGr~I~i~~~~~v~IDG~~l~v~   31 (73)
</pre><pre>T 2JV8_A            2 THHTEVFEGGTIDIEDDTSLTINGKEISYV   31 (73)
</pre><pre>T ss_dssp             CCEEEEETTEEEEEETTEEEEETTEECCCC
</pre><pre>T ss_pred             CceEEEeCCceEEEeCCcEEEECCEEeceE
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>TIGR03361 VI_Rhs_Vgr; type VI secretion system Vgr family protein. Members of this protein family belong to the Rhs element Vgr protein family (see TIGR01646), but furthermore all are found in genomes with type VI secretion loci.
</pre><pre>Probab=67.78  E-value=20  Score=25.64  Aligned_cols=26  Identities=15%  Similarity=0.184  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeC-CcEEEEccCcEEEEcCeE
</pre><pre>Q MBL0283831.1/7    4 DSITFTVG-KSVLVMKKDGTITLNGKT   29 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG-~~~i~m~kdG~I~i~G~~   29 (48)
</pre><pre>                      +.++|.|+ +..+.+..|++|+|++.+
</pre><pre>T Consensus       486 ~~i~l~~~~~~~~~~~~~~~~~i~~~~  512 (513)
</pre><pre>T TIGR03361       486 EEIYLHAQRDMNTEVENDSTHTVGNNR  512 (513)
</pre><pre>T ss_pred             ceEEEEecCCceEEEcCCceEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>3WIT_A Putative Vgr protein; Triple-stranded beta-helix, All beta proteins, Type 6 secretion system component, STRUCTURAL PROTEIN; 1.95A {Escherichia coli}
</pre><pre>Probab=65.41  E-value=51  Score=19.87  Aligned_cols=48  Identities=21%  Similarity=0.201  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCe--EEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGK--TIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~--~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+.+...+.++.+.+.+.++|-...-|.  .+.+.+++.+.+....+-.+
</pre><pre>T Consensus        25 ~~~k~~~~~v~~s~i~i~~~G~~v~vG~~~t~~v~k~~~~~~g~~~~~~~   74 (82)
</pre><pre>T 3WIT_A           25 TAIKEIMLAVEESKIALTPDGIQLQVGESTVIRLSKDGITIVGGSVFING   74 (82)
</pre><pre>T ss_dssp             --CEEEEEEETTEEEEEETTEEEEEETTTEEEEEETTEEEEECSSCEETT
</pre><pre>T ss_pred             HHHheeeeeeccceEEEeCCcEEEEeCCcEEEEeCCCCcEEecceEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>2IYJ_B THIOL DISULFIDE INTERCHANGE PROTEIN DSBC; DISULFIDE BOND ISOMERASE, ISOMERASE, DSBC, DSBG, PERIPLASMIC, REDOX-ACTIVE CENTER; 2.0A {ESCHERICHIA COLI} SCOP: d.17.3.1
</pre><pre>Probab=63.77  E-value=43  Score=18.39  Aligned_cols=35  Identities=26%  Similarity=0.436  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEE
</pre><pre>Q MBL0283831.1/7    6 ITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQL   40 (48)
</pre><pre>Q Consensus         6 ~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~   40 (48)
</pre><pre>                      .++..++..+-+.+||...+.|.-+.+.+...+++
</pre><pre>T Consensus        32 ~~v~~~~~~~y~~~dg~~~i~G~l~d~~~~~~~nl   66 (75)
</pre><pre>T 2IYJ_B           32 KTVLTNSGVLYITDDGKHIIQGPMYDVSGTAPVNV   66 (75)
</pre><pre>T ss_dssp             EEEEETTEEEEEETTSSEEEESCEEECSSSSCEEH
</pre><pre>T ss_pred             EEEEECCeEEEEcCCCCEEEEcceEeCCCCCceec
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>6MVF_A Beta-galactosidase/beta-glucuronidase; glycoside hydrolase, FMN, beta-glucuronidase, HYDROLASE; HET: FMN; 2.55A {Faecalibacterium prausnitzii L2-6}
</pre><pre>Probab=63.14  E-value=28  Score=27.01  Aligned_cols=31  Identities=29%  Similarity=0.498  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=..+.++.++..+|||+++.+.|
</pre><pre>T Consensus       268 d~~~~~fG~R~~~~~~~~~f~lNG~~~~l~G  298 (769)
</pre><pre>T 6MVF_A          268 DEIYANVGVRSYTVTPDGGFSINGEATPLRG  298 (769)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEeEEEEECCCCCEEECCeeeCcee
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>d1eeja2 d.17.3.1 (A:1-60) Disulfide bond isomerase, DsbC, N-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=61.69  E-value=42  Score=17.88  Aligned_cols=32  Identities=28%  Similarity=0.473  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCcEEEEccCcEEEEcCeEEEEEeccce
</pre><pre>Q MBL0283831.1/7    7 TFTVGKSVLVMKKDGTITLNGKTIDVVGSDHI   38 (48)
</pre><pre>Q Consensus         7 ~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i   38 (48)
</pre><pre>                      ++..+...+-+++||...|.|.-+.+.+...+
</pre><pre>T Consensus        29 ev~~~~~~~Y~~~~g~yli~G~l~d~~~~~~~   60 (60)
</pre><pre>T d1eeja2          29 TVLTNSGVLYITDDGKHIIQGPMYDVSGTAPV   60 (60)
</pre><pre>T ss_dssp             EEEETTEEEEEETTSCEEEESCEEECSSSSCE
</pre><pre>T ss_pred             EEEECCeEEEEcCCCCEEEEeeeEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF18545.3 ; HalOD1 ; Halobacterial output domain 1
</pre><pre>Probab=58.33  E-value=57  Score=18.70  Aligned_cols=21  Identities=24%  Similarity=0.504  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCcEEEEccCcEEEE
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKSVLVMKKDGTITL   25 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~~i~m~kdG~I~i   25 (48)
</pre><pre>                      .+++...+-.+++..||+|.+
</pre><pre>T Consensus        54 ~v~F~~~~~~VtV~~dG~v~v   74 (76)
</pre><pre>T L0AHC4_NATGS/3   54 HVSFVVADHRVTVYSDGEILI   74 (76)
</pre><pre>T ss_pred             EEEEEECCEEEEEecCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>d1t3ba2 d.17.3.1 (A:2-60) Disulfide bond isomerase, DsbC, N-terminal domain {Haemophilus influenzae [TaxId: 727]}
</pre><pre>Probab=58.06  E-value=52  Score=17.49  Aligned_cols=33  Identities=18%  Similarity=0.375  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEeCCcEEEEccCcEEEEcCeEEEEEeccce
</pre><pre>Q MBL0283831.1/7    6 ITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHI   38 (48)
</pre><pre>Q Consensus         6 ~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i   38 (48)
</pre><pre>                      .++..++..+-+++||...|.|.-+.+.+...+
</pre><pre>T Consensus        27 yev~~~~~i~Y~~~dg~yli~G~l~d~~~~~~~   59 (59)
</pre><pre>T d1t3ba2          27 KTAVTDQGILYVSEDGKYLFEGKLYELTNNGPV   59 (59)
</pre><pre>T ss_dssp             ECCEETTEECEEESSSSCEECSCCCCCTTTSCC
</pre><pre>T ss_pred             EEEEECCeEEEEcCCCCEEEEeeeEEcCCCccC
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>5Z1A_A Putative beta-galactosidase; beta-glucuronidase, GH2, HYDROLASE; HET: SJ5; 1.859A {Bacteroides fragilis NCTC 9343}
</pre><pre>Probab=58.03  E-value=49  Score=24.82  Aligned_cols=31  Identities=32%  Similarity=0.425  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=.++.+++++..++||+++.+.|
</pre><pre>T Consensus       276 d~~~~~fG~R~~~~~~~~~~~lNG~~~~l~G  306 (680)
</pre><pre>T 5Z1A_A          276 DSVTQPLGLRYYHTDPDKGFFLNGKHLPLHG  306 (680)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCCEE
</pre><pre>T ss_pred             EEEEEeEeCeEEEEeCCCCEEECCEEeccee
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>2ICH_A Putative AttH; STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-2, LIPID BINDING PROTEIN; HET: GOL, NHE, MSE; 2.0A {Nitrosomonas europaea} SCOP: b.176.1.1
</pre><pre>Probab=57.54  E-value=31  Score=23.44  Aligned_cols=23  Identities=26%  Similarity=0.541  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             EccCcEEEEcCeEEEEEeccceE
</pre><pre>Q MBL0283831.1/7   17 MKKDGTITLNGKTIDVVGSDHIQ   39 (48)
</pre><pre>Q Consensus        17 m~kdG~I~i~G~~I~i~gs~~i~   39 (48)
</pre><pre>                      ++-.|+|+++|+.+.|+|.+-..
</pre><pre>T Consensus       170 ~~v~Gti~i~g~~~~v~G~g~~D  192 (335)
</pre><pre>T 2ICH_A          170 LQVSGIINRQGEDIPVTGTAWLD  192 (335)
</pre><pre>T ss_dssp             EEEEEEEEESSCEEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEECCEEEEEeEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>2PFU_A Biopolymer transport exbD protein; ExbD, TonB system, proton motive force, periplasmic domain, TRANSPORT PROTEIN; NMR {Escherichia coli}
</pre><pre>Probab=56.70  E-value=33  Score=18.88  Aligned_cols=17  Identities=18%  Similarity=0.403  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .|.+..+|.++++|+.+
</pre><pre>T Consensus        23 ~i~i~~~g~~~~~~~~~   39 (99)
</pre><pre>T 2PFU_A           23 YLSVKADNSMFIGNDPV   39 (99)
</pre><pre>T ss_dssp             EEEEETTTEEEETTEEE
</pre><pre>T ss_pred             EEEEcCCCcEEECCEec
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>TIGR02804 ExbD_2; TonB system transport protein ExbD, group 2. Members of this family are Gram-negative bacterial inner membrane proteins, generally designated ExbD, related to the TolR family modeled by TIGRFAMs TIGR02801.
</pre><pre>Probab=55.75  E-value=35  Score=20.18  Aligned_cols=17  Identities=24%  Similarity=0.511  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .+.++++|.++++|+.+
</pre><pre>T Consensus        53 ~i~i~~~g~i~~~~~~~   69 (121)
</pre><pre>T TIGR02804        53 LITITADNQLYFNDKPI   69 (121)
</pre><pre>T ss_pred             EEEEcCCCeEEECCEEC
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>7CWI_A beta-galactosidase; beta-galactosidase, CARBOHYDRATE, HYDROLASE; HET: SO4, GOL; 1.95A {Bacillus circulans}
</pre><pre>Probab=55.71  E-value=71  Score=24.67  Aligned_cols=31  Identities=23%  Similarity=0.453  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=..+.++.++..++||+++.+.|
</pre><pre>T Consensus       268 d~~~~~fG~R~~~~~~~~~~~lNG~~~~l~G  298 (815)
</pre><pre>T 7CWI_A          268 DTYETRFGARFFKFDANEGFSLNGKPMKLYG  298 (815)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEeCEEEEEeCCCCEEECCEEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>PF07143.13 ; CrtC ; CrtC N-terminal lipocalin domain
</pre><pre>Probab=55.21  E-value=44  Score=21.03  Aligned_cols=25  Identities=32%  Similarity=0.739  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             EccCcEEEEcCeEEEEEeccceEEc
</pre><pre>Q MBL0283831.1/7   17 MKKDGTITLNGKTIDVVGSDHIQLD   41 (48)
</pre><pre>Q Consensus        17 m~kdG~I~i~G~~I~i~gs~~i~~~   41 (48)
</pre><pre>                      +.-.|+|.++|+.+.++|.+-..-+
</pre><pre>T Consensus       159 ~~v~Gtl~~~g~~~~v~G~g~~Dh~  183 (186)
</pre><pre>T F3YX61_DESAF/4  159 LRAEGSIEMNGRTMDVSGLAWMDHE  183 (186)
</pre><pre>T ss_pred             eEEEEEEEECCEEEEEEEEEEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>6D1P_B Glycosyl hydrolases family 2, sugar binding domain protein; beta-glucuronidase, HYDROLASE; HET: GOL; 2.35A {Bacteroides uniformis str. 3978 T3 ii}
</pre><pre>Probab=55.17  E-value=67  Score=24.66  Aligned_cols=31  Identities=23%  Similarity=0.291  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.+..++..++||+++.+.|
</pre><pre>T Consensus       297 d~~~~~fG~r~~~~~~~~~~~lNG~~~~l~G  327 (814)
</pre><pre>T 6D1P_B          297 LRTTQHFGFRHFRFEEKGPFYLNGERLLLRG  327 (814)
</pre><pre>T ss_dssp             EEEEEEECBCCEEECTTSCEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEeEEEEEcCCCCEEECCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF07378.13 ; FlbT ; Flagellar protein FlbT
</pre><pre>Probab=54.98  E-value=71  Score=20.40  Aligned_cols=30  Identities=17%  Similarity=0.254  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGK   43 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~   43 (48)
</pre><pre>                      +|.+++.-.+.|||--|.......+.+.++
</pre><pre>T Consensus         2 ~l~LkpgEriiInGavi~n~~~~~l~i~n~   31 (124)
</pre><pre>T Q21AA7_RHOPB/5    2 CVSLRGGERIYVNGAVLRVDRKVTLELIND   31 (124)
</pre><pre>T ss_pred             eeeeCCCCEEEECCEEEEeCCCeEEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>2G2S_A Green fluorescent protein; chromophore, biosynthesis, dehydroalanine, peptide hydrolysis, post-translational modification, LUMINESCENT PROTEIN; 1.2A {Aequorea victoria}
</pre><pre>Probab=54.43  E-value=46  Score=19.42  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        11 ~~~~~k~~~eG~v--Ng~~F~v~GeG~   35 (66)
</pre><pre>T 2G2S_A           11 GVVPILVELDGDV--NGHKFSVSGEGE   35 (66)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCccEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>PF02472.18 ; ExbD ; Biopolymer transport protein ExbD/TolR
</pre><pre>Probab=54.22  E-value=38  Score=19.79  Aligned_cols=17  Identities=6%  Similarity=0.329  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .|.+.+||.|+++|..+
</pre><pre>T Consensus        56 ~I~i~~d~~i~~~~~~~   72 (131)
</pre><pre>T Q3SIR3_THIDA/7   56 EVAVAESGRYQIDDAAL   72 (131)
</pre><pre>T ss_pred             EEEEcCCCcEEECCEEc
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>d2icha1 b.176.1.1 (A:24-351) AttH-related protein NE1406 {Nitrosomonas europaea [TaxId: 915]}
</pre><pre>Probab=54.06  E-value=44  Score=22.85  Aligned_cols=24  Identities=25%  Similarity=0.491  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             EEccCcEEEEcCeEEEEEeccceE
</pre><pre>Q MBL0283831.1/7   16 VMKKDGTITLNGKTIDVVGSDHIQ   39 (48)
</pre><pre>Q Consensus        16 ~m~kdG~I~i~G~~I~i~gs~~i~   39 (48)
</pre><pre>                      .++-.|+|+++|+++.|.|.+-..
</pre><pre>T Consensus       167 ~~~v~G~i~i~G~~~~v~G~g~~D  190 (328)
</pre><pre>T d2icha1         167 HLQVSGIINRQGEDIPVTGTAWLD  190 (328)
</pre><pre>T ss_dssp             EEEEEEEEEESSCEEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEECCEEEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>6HPD_A Beta-galactosidase (GH2); GH2, glycoside hydrolase, glucuronidase, HYDROLASE; 2.43A {Formosa agariphila KMM 3901}
</pre><pre>Probab=54.03  E-value=75  Score=25.23  Aligned_cols=31  Identities=26%  Similarity=0.411  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.+.+++..++||+++.+.|
</pre><pre>T Consensus       315 d~~~~~~G~R~~~~~~~~~f~lNG~~~~l~G  345 (990)
</pre><pre>T 6HPD_A          315 DTYESPLGIRNFDFNKDSGFWINGEQIKLHG  345 (990)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEeeeeeEEEEcCCCcEEECCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>3FN9_A Putative beta-galactosidase; structural genomics, Putative beta-galactosidase, Glycosidase, Hydrolase, PSI-2, Protein Structure Initiative, New York SGX Research Center for; 2.7A {Bacteroides fragilis}
</pre><pre>Probab=51.51  E-value=94  Score=23.35  Aligned_cols=31  Identities=19%  Similarity=0.304  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=-.+.+..++.+++||+++.+.|
</pre><pre>T Consensus       271 d~~~~~~G~r~~~~~~~~~~~lnG~~~~l~G  301 (692)
</pre><pre>T 3FN9_A          271 DEVVQPLGVRKYEIVAGKGFFLNGEKYSMYG  301 (692)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCCEE
</pre><pre>T ss_pred             EEEEEeeeceEEEEEcCCcEEECCEEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>3FN9_C Putative beta-galactosidase; structural genomics, Putative beta-galactosidase, Glycosidase, Hydrolase, PSI-2, Protein Structure Initiative, New York SGX Research Center for; 2.7A {Bacteroides fragilis}
</pre><pre>Probab=51.51  E-value=94  Score=23.35  Aligned_cols=31  Identities=19%  Similarity=0.304  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=-.+.+..++.+++||+++.+.|
</pre><pre>T Consensus       271 d~~~~~~G~r~~~~~~~~~~~lnG~~~~l~G  301 (692)
</pre><pre>T 3FN9_C          271 DEVVQPLGVRKYEIVAGKGFFLNGEKYSMYG  301 (692)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCCEE
</pre><pre>T ss_pred             EEEEEeeeceEEEEEcCCcEEECCEEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>d2vzua3 c.1.8.0 (A:336-674) automated matches {Amycolatopsis orientalis [TaxId: 31958]}
</pre><pre>Probab=51.40  E-value=60  Score=22.00  Aligned_cols=25  Identities=24%  Similarity=0.349  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEccCcEEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7   12 KSVLVMKKDGTITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus        12 ~~~i~m~kdG~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      ...++-+++....+||+++-+.|..
</pre><pre>T Consensus         3 ~~~~~~~~~~~f~vNG~~~~l~G~n   27 (339)
</pre><pre>T d2vzua3           3 KATLNSSGGRQYSVNGKPLLIRGGG   27 (339)
</pre><pre>T ss_dssp             EEEECTTSCEEEEETTEEECEEEEE
</pre><pre>T ss_pred             eeEEecCCCeeEEECCEEEeEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>d4npba1 d.17.3.0 (A:22-81) automated matches {Yersinia pestis [TaxId: 214092]}
</pre><pre>Probab=50.32  E-value=74  Score=17.00  Aligned_cols=33  Identities=33%  Similarity=0.435  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEeCCcEEEEccCcEEEEcCeEEEEEeccce
</pre><pre>Q MBL0283831.1/7    6 ITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHI   38 (48)
</pre><pre>Q Consensus         6 ~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i   38 (48)
</pre><pre>                      .++..++.-+-+++||...|.|.-+.+.+...+
</pre><pre>T Consensus        28 ~ev~~~~~i~Y~~~~g~yli~G~l~d~~~~~~~   60 (60)
</pre><pre>T d4npba1          28 STVMTESGVLYISADGKHLLQGPLYDVSGDQPI   60 (60)
</pre><pre>T ss_dssp             EEEECSSCEEEEETTSCEEEESCCEECSSSSCE
</pre><pre>T ss_pred             EEEEECCeEEEEeCCCCEEEEceeEECCCCccC
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>2GW4_A Kaede; BETA BARREL, LUMINESCENT PROTEIN; HET: RC7, NFA; 1.6A {Trachyphyllia geoffroyi}
</pre><pre>Probab=50.30  E-value=67  Score=18.73  Aligned_cols=25  Identities=16%  Similarity=0.373  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|.|  ||+.+.+.|.+.
</pre><pre>T Consensus         8 ~~m~~k~~meG~V--NGh~F~i~GeG~   32 (63)
</pre><pre>T 2GW4_A            8 PEMKIKLLMEGNV--NGHQFVIEGDGK   32 (63)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCCeEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>5EXC_E Green fluorescent protein; Photoswitchable fluorescent proteins, chromophore, beta-barrel, biomarker, fluorescent protein; HET: MG, GOL, NLW, RC7; 2.14A {Dendronephthya sp. SSAL-2002}
</pre><pre>Probab=50.27  E-value=67  Score=18.68  Aligned_cols=25  Identities=16%  Similarity=0.381  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|.|  ||+.+.+.|.+.
</pre><pre>T Consensus         7 ~~m~~k~~meG~V--NGh~F~i~GeG~   31 (62)
</pre><pre>T 5EXC_E            7 EDMRVKVHMEGNV--NGHAFVIEGEGK   31 (62)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>5EXC_H Green fluorescent protein; Photoswitchable fluorescent proteins, chromophore, beta-barrel, biomarker, fluorescent protein; HET: MG, RC7, NLW, GOL; 2.14A {Dendronephthya sp. SSAL-2002}
</pre><pre>Probab=50.27  E-value=67  Score=18.68  Aligned_cols=25  Identities=16%  Similarity=0.381  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|.|  ||+.+.+.|.+.
</pre><pre>T Consensus         7 ~~m~~k~~meG~V--NGh~F~i~GeG~   31 (62)
</pre><pre>T 5EXC_H            7 EDMRVKVHMEGNV--NGHAFVIEGEGK   31 (62)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>3CFA_S GFP-like fluorescent protein; BETA BARREL, FLUORESCENT PROTEIN; HET: CME, NRQ; 1.75A {Anemonia sulcata}
</pre><pre>Probab=50.21  E-value=59  Score=18.82  Aligned_cols=25  Identities=24%  Similarity=0.446  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +..++.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus         7 ~~m~~k~~~eG~v--NGh~F~i~GeG~   31 (62)
</pre><pre>T 3CFA_S            7 ETMPFRMTMEGTV--NGHHFKCTGKGE   31 (62)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCccEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>3CFH_L GFP-like photoswitchable fluorescent protein; beta barrel, beta can, chromophore, photoactivation, FLUORESCENT PROTEIN; HET: NRQ, CME; 1.75A {Anemonia sulcata}
</pre><pre>Probab=50.21  E-value=59  Score=18.82  Aligned_cols=25  Identities=24%  Similarity=0.446  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +..++.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus         7 ~~m~~k~~~eG~v--NGh~F~i~GeG~   31 (62)
</pre><pre>T 3CFH_L            7 ETMPFRMTMEGTV--NGHHFKCTGKGE   31 (62)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCccEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>4CU7_A BETA-GALACTOSIDASE; HYDROLASE; HET: GIF, SO4, EDO; 2.2A {STREPTOCOCCUS PNEUMONIAE TIGR4}
</pre><pre>Probab=49.90  E-value=84  Score=24.37  Aligned_cols=31  Identities=23%  Similarity=0.273  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=-.+.++.++..++||+++.+.|
</pre><pre>T Consensus       280 d~~~~~fGfR~~~~~~~~~~~lNG~~~~l~G  310 (849)
</pre><pre>T 4CU7_A          280 DAKKDLFGYRYYHWTPNEGFSLNGERIKFHG  310 (849)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETTTEEEETTEEECEEE
</pre><pre>T ss_pred             EEEEEEecCceEEEcCCCCEEECCEEEcceE
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>3PQI_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 2.642A {Bacteriophage phi92}
</pre><pre>Probab=49.05  E-value=1.4e+02  Score=19.85  Aligned_cols=48  Identities=15%  Similarity=0.250  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEc-CeEEEEEecc-ceEEc---CceeecC
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLN-GKTIDVVGSD-HIQLD---GKRIDLN   48 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~-G~~I~i~gs~-~i~~~---~~~I~~n   48 (48)
</pre><pre>                      .....+....++..|.++++|.|.|+ ...+.|.+.. .+.+.   ...+.++
</pre><pre>T Consensus       138 ~~~~~~~~~~~g~~i~~~~~G~i~i~~~~~i~i~~~~~~i~i~~~~~~~~~i~  190 (247)
</pre><pre>T 3PQI_A          138 DSEKVIISNNKQTYASFDPNGNISVYNTQGMKIDMTPNSIVLTDAGGGKLTLQ  190 (247)
</pre><pre>T ss_dssp             CSSCEEEEETTTEEEEECTTSCEEEEETTSCEEEEETTEEEEEETTTEEEEEE
</pre><pre>T ss_pred             CcccEEEEcCCCcEEEECCCCcEEEEeCCceEEEeCCCceEEEeCCCceEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>PF16270.7 ; DUF4923 ; Lipocalin-like domain (DUF4923)
</pre><pre>Probab=48.83  E-value=60  Score=21.67  Aligned_cols=40  Identities=15%  Similarity=0.262  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             EEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7    8 FTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus         8 l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      +..|+..+++++||+...+-....+.|.=.++-+.+.|.+
</pre><pre>T Consensus        72 ~~~~~~t~tF~~DgTft~~~~g~~~~GTys~~~~~~~itl  111 (175)
</pre><pre>T D5EV19_PRER2/2   72 LSASNTYITFNEDGTFSSKIAGTPFNGKYTFDEATQKISL  111 (175)
</pre><pre>T ss_pred             CCCceeEEEECCCCcEEEEECCEEEEEEEEEeCCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>6M9Y_C Fluorescent protein lanFP6A; Fluorescent Protein Branchiostoma floridae Gly-Tyr-Ala tripeptide hydrolysis, FLUORESCENT PROTEIN; 1.35A {Branchiostoma floridae}
</pre><pre>Probab=48.77  E-value=62  Score=18.56  Aligned_cols=22  Identities=27%  Similarity=0.428  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus         6 ~~~~~m~G~V--NG~~F~i~G~G~   27 (58)
</pre><pre>T 6M9Y_C            6 THELHIFGSF--NGVKFDMVGEGT   27 (58)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>6DXU_B Glycosyl hydrolase family 2, TIM barrel domain protein; bacterial enzyme, glycoside hydrolase family 2, GUS, Beta-Glucuronidase, carbohydrate binding protein, HYDROLASE; HET: BCN; 1.9A {Parabacteroides merdae ATCC 43184}
</pre><pre>Probab=48.75  E-value=1.1e+02  Score=23.72  Aligned_cols=31  Identities=26%  Similarity=0.261  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=.++.+..++..++||+++.+.|
</pre><pre>T Consensus       299 d~~~~~fG~R~~~~~~~~~~~lNG~~~~l~G  329 (830)
</pre><pre>T 6DXU_B          299 FTTEERFGFRHTEFKDKGPFFLNGKRLLLRG  329 (830)
</pre><pre>T ss_dssp             EEEEEEECBCCEEECTTSCEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEeceEEEEeCCCcEEECCEEEeeEe
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>6J2U_A Tyrosinase co-factor protein; Citrate synthase, Metallosphaera sedula, OXIDOREDUCTASE; 1.3A {Streptomyces avermitilis} SCOP: d.386.1.0
</pre><pre>Probab=47.93  E-value=1.3e+02  Score=19.73  Aligned_cols=30  Identities=10%  Similarity=0.195  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCc---------EEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDG---------TITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG---------~I~i~G~~I~i~   33 (48)
</pre><pre>                      ..++.+.-+-+|.+..+.         .|+|+|++|++.
</pre><pre>T Consensus        40 ~~f~E~YkGR~I~i~~~~~~~~~~~~~~v~IDGr~Lhvm   78 (118)
</pre><pre>T 6J2U_A           40 QPFDEVYQGRRIEGRATGGGHHHGSGYGVFIDGMELHVM   78 (118)
</pre><pre>T ss_dssp             CCEEEEETTEEEEEEECC--------EEEEETTEEECEE
</pre><pre>T ss_pred             CCceeEECCcEEEEEecCCCCccCCceEEEECCEEceEE
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>4PFE_A Green fluorescent protein; beta barrel, fusion protein, homodimer, IMMUNE SYSTEM, FLUORESCENT PROTEIN; HET: CRO; 2.603A {Aequorea victoria}
</pre><pre>Probab=47.71  E-value=52  Score=22.21  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|.|  ||+.+.|.|.+.
</pre><pre>T Consensus         9 ~~~~~~~~~~~~~--~~~~~~~~g~g~   33 (343)
</pre><pre>T 4PFE_A            9 GVVPILVELDGDV--NGHKFSVRGEGE   33 (343)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccccEEEEEEEEE--CCEEeEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>3LF4_A Fluorescent Timer Precursor Blue102; Fluorescent Timers, Blue-to-red conversion, Chromophore degradation, FLUORESCENT PROTEIN; HET: 0YG; 1.808A {Discosoma sp.}
</pre><pre>Probab=47.66  E-value=76  Score=18.67  Aligned_cols=25  Identities=16%  Similarity=0.409  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ...++.+.-+|.|  ||+.+.+.|.+.
</pre><pre>T Consensus        10 ~~m~~k~~meG~V--NGh~F~i~GeG~   34 (66)
</pre><pre>T 3LF4_A           10 EFMRFKVHMEGSV--NGHEFEIEGEGE   34 (66)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             cCcEEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>6LOF_D GFP-like fluorescent chromoprotein FP538; ZsYellow, fluorescent protein, yellow; HET: NFA, CH7; 2.6A {Zoanthus sp.}
</pre><pre>Probab=46.34  E-value=81  Score=18.67  Aligned_cols=25  Identities=16%  Similarity=0.375  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ....+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        13 ~~m~~k~~m~G~v--Ng~~F~i~GeG~   37 (68)
</pre><pre>T 6LOF_D           13 EEMTMKYHMEGCV--NGHKFVITGEGI   37 (68)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCCeEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>PF11211.10 ; DUF2997 ; Protein of unknown function (DUF2997)
</pre><pre>Probab=46.27  E-value=49  Score=17.72  Aligned_cols=13  Identities=15%  Similarity=0.391  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLN   26 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~   26 (48)
</pre><pre>                      .+++++||+|.+.
</pre><pre>T Consensus         2 ~i~I~~dG~v~~e   14 (47)
</pre><pre>T K9UET1_CHAP6/6    2 EFIIYPDGRVQEK   14 (47)
</pre><pre>T ss_pred             EEEECCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>PF10411.11 ; DsbC_N ; Disulfide bond isomerase protein N-terminus
</pre><pre>Probab=46.20  E-value=80  Score=16.17  Aligned_cols=29  Identities=17%  Similarity=0.326  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    6 ITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         6 ~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      .++..++..+-+.+||+..+.|.-+.+..
</pre><pre>T Consensus        25 ~~v~~~~~~~y~~~dg~~li~G~~~d~~~   53 (54)
</pre><pre>T G7UW74_PSEUP/5   25 REVIAAGQLVYVSEDGKYLLQGSLVDIGQ   53 (54)
</pre><pre>T ss_pred             EEEEECCEEEEEcCCCCEEEEeeeEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>TIGR02801 tolR; TolR protein. The model describes the inner membrane protein TolR, part of the TolR/TolQ complex that transduces energy from the proton-motive force, through TolA, to an outer membrane complex made up of TolB and Pal (peptidoglycan-associated lipoprotein).
</pre><pre>Probab=45.92  E-value=60  Score=19.30  Aligned_cols=17  Identities=35%  Similarity=0.565  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .|.+.+||.++++|..+
</pre><pre>T Consensus        53 ~i~I~~~g~i~~~~~~~   69 (129)
</pre><pre>T TIGR02801        53 TVSVDADGKIYLNDNKI   69 (129)
</pre><pre>T ss_pred             EEEEcCCCcEEECCcee
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>5DMY_C Beta-galactosidase; galactosidase, hydrolase, truncation mutant; 1.95A {Bifidobacterium bifidum}
</pre><pre>Probab=44.94  E-value=1.1e+02  Score=24.13  Aligned_cols=31  Identities=19%  Similarity=0.297  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=-.+.+..++.+++||+++.+.|
</pre><pre>T Consensus       299 d~~~~~fG~R~~~~~~~~~~~lNG~~~~l~G  329 (904)
</pre><pre>T 5DMY_C          299 DTYDTEYGFRWTGFDATSGFSLNGEKVKLKG  329 (904)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEeEEEEEeCCCcEEECCeEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>3IPF_A uncharacterized protein; Q251Q8_DESHY, NESG, DhR8c, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, unknown function; 1.988A {Desulfitobacterium hafniense}
</pre><pre>Probab=44.68  E-value=1.1e+02  Score=17.54  Aligned_cols=29  Identities=14%  Similarity=0.509  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             EEEccCcEEEEcCeEEEEEe----ccceEEcCc
</pre><pre>Q MBL0283831.1/7   15 LVMKKDGTITLNGKTIDVVG----SDHIQLDGK   43 (48)
</pre><pre>Q Consensus        15 i~m~kdG~I~i~G~~I~i~g----s~~i~~~~~   43 (48)
</pre><pre>                      ..-.++|.+.|.|+.+.|..    ++.+.+.|.
</pre><pre>T Consensus        24 ~l~~~~g~l~I~G~~L~i~~~~~~~g~i~I~G~   56 (71)
</pre><pre>T 3IPF_A           24 LLETIQGVLSIKGEKLGIKHLDLKAGQVEVEGL   56 (71)
</pre><pre>T ss_dssp             EEEETTEEEEEEEEEEEEEEEETTTTEEEEEEE
</pre><pre>T ss_pred             EEEeCCeEEEEEcCCeEEEEEECCCCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>6QUB_A Beta-galactosidase; TIM BARREL, GLYCOSIDE HYDROLASE, HYDROLASE, SUGAR BINDING PROTEIN; HET: GAL; 1.95A {Bifidobacterium bifidum}
</pre><pre>Probab=44.47  E-value=1.3e+02  Score=23.62  Aligned_cols=31  Identities=19%  Similarity=0.297  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=.++.++.++..++||+++.+.|
</pre><pre>T Consensus       293 d~~~~~fG~R~~~~~~~~~f~lNG~~~~l~G  323 (885)
</pre><pre>T 6QUB_A          293 DTYDTEYGFRWTGFDATSGFSLNGEKVKLKG  323 (885)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEceEEEEeCCCcEEECCEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>2JWK_A Protein tolR; PROTEIN, Periplasmic domain, membrane, Inner membrane, Protein transport, Transmembrane, Transport, MEMBRANE PROTEIN; NMR {Haemophilus influenzae}
</pre><pre>Probab=44.04  E-value=54  Score=16.92  Aligned_cols=15  Identities=13%  Similarity=0.140  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCe
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGK   28 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~   28 (48)
</pre><pre>                      .+.+.++|.++++|+
</pre><pre>T Consensus         6 ~v~i~~~~~~~~~~~   20 (74)
</pre><pre>T 2JWK_A            6 ILEVAGIGKYAISIG   20 (74)
</pre><pre>T ss_dssp             EEEECSSSCEEEEET
</pre><pre>T ss_pred             EEEEcCCCcEEEeeC
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>2JWL_A Protein tolR; PROTEIN, Periplasmic domain, membrane, Inner membrane, Protein transport, Transmembrane, Transport, MEMBRANE PROTEIN; NMR {Haemophilus influenzae}
</pre><pre>Probab=44.04  E-value=54  Score=16.92  Aligned_cols=15  Identities=13%  Similarity=0.140  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCe
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGK   28 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~   28 (48)
</pre><pre>                      .+.+.++|.++++|+
</pre><pre>T Consensus         6 ~v~i~~~~~~~~~~~   20 (74)
</pre><pre>T 2JWL_A            6 ILEVAGIGKYAISIG   20 (74)
</pre><pre>T ss_dssp             EEEECSSSCEEEEET
</pre><pre>T ss_pred             EEEEcCCCcEEEeeC
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>PF11012.10 ; DUF2850 ; Protein of unknown function (DUF2850)
</pre><pre>Probab=42.89  E-value=1e+02  Score=18.84  Aligned_cols=19  Identities=26%  Similarity=0.576  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEccCcEEEEcCeEEE
</pre><pre>Q MBL0283831.1/7   12 KSVLVMKKDGTITLNGKTID   31 (48)
</pre><pre>Q Consensus        12 ~~~i~m~kdG~I~i~G~~I~   31 (48)
</pre><pre>                      ..++++.++| |++||.-|+
</pre><pre>T Consensus        16 ~d~~~l~~~G-V~~ngrlVt   34 (78)
</pre><pre>T Q9KPQ2_VIBCH/4   16 AQKITIGTQG-IVLNGRLVT   34 (78)
</pre><pre>T ss_pred             ccEEEEcCce-EEECCEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>PF20033.1 ; DUF6438 ; Domain of unknown function (DUF6438)
</pre><pre>Probab=42.48  E-value=1.2e+02  Score=17.68  Aligned_cols=33  Identities=21%  Similarity=0.257  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             EeCCcEEEEccCcEEEEcCeEEEE-EeccceEEc
</pre><pre>Q MBL0283831.1/7    9 TVGKSVLVMKKDGTITLNGKTIDV-VGSDHIQLD   41 (48)
</pre><pre>Q Consensus         9 ~cG~~~i~m~kdG~I~i~G~~I~i-~gs~~i~~~   41 (48)
</pre><pre>                      .|---+|.+.+||.+.++|..-.. .|...-+++
</pre><pre>T Consensus        11 ~CP~y~i~I~~dG~v~f~g~~~~~~~G~~~~~l~   44 (87)
</pre><pre>T A0A0F4PZS6_9GA   11 TCPAYRVDVFSDGVIVYRGDAHTKLVGTYRLPSN   44 (87)
</pre><pre>T ss_pred             eeCcEEEEEcCCCEEEEEecCceeeeEEEEEECC
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>3RWN_A Gene 9 protein; Knob, Cell membrane penetration, bacteriophage Sf6, VIRAL PROTEIN; HET: GLU; 1.0A {Shigella phage Sf6}
</pre><pre>Probab=42.07  E-value=81  Score=22.11  Aligned_cols=22  Identities=27%  Similarity=0.551  Sum_probs=0.0  Template_Neff=3.000
</pre><pre>
</pre><pre>Q ss_pred             EEEccCcEEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7   15 LVMKKDGTITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus        15 i~m~kdG~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +.++|+|.+.-||..|.|+..+
</pre><pre>T Consensus       126 FSVDk~GnlaTNG~~i~I~snG  147 (161)
</pre><pre>T 3RWN_A          126 FSVDKDGFLATNGSTLTIQSNG  147 (161)
</pre><pre>T ss_dssp             ECCCTTSHHHHHCBCEEEEEES
</pre><pre>T ss_pred             EEEcCCCceecCCceEEEEECC
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>6MVH_C Beta-galactosidase; glycoside hydrolase, beta-glucuronidase, Flavin-binder, HYDROLASE; HET: FMN; 2.4A {Roseburia hominis}
</pre><pre>Probab=41.82  E-value=1.5e+02  Score=23.33  Aligned_cols=31  Identities=19%  Similarity=0.385  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=..+.+..++..++||+++.+.|
</pre><pre>T Consensus       273 d~~~~~fGfR~~~~~~~~~f~lNG~~~~l~G  303 (780)
</pre><pre>T 6MVH_C          273 DAVSTRFGCRTFEIDPERGFILNGEEYPLRG  303 (780)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEeeeEEEEEeCCCcEEECCeEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>6MVH_D Beta-galactosidase; glycoside hydrolase, beta-glucuronidase, Flavin-binder, HYDROLASE; HET: FMN; 2.4A {Roseburia hominis}
</pre><pre>Probab=41.82  E-value=1.5e+02  Score=23.33  Aligned_cols=31  Identities=19%  Similarity=0.385  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=..+.+..++..++||+++.+.|
</pre><pre>T Consensus       273 d~~~~~fGfR~~~~~~~~~f~lNG~~~~l~G  303 (780)
</pre><pre>T 6MVH_D          273 DAVSTRFGCRTFEIDPERGFILNGEEYPLRG  303 (780)
</pre><pre>T ss_dssp             -------------------------------
</pre><pre>T ss_pred             EEEEEEeeeEEEEEeCCCcEEECCeEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>d1vioa2 d.66.1.5 (A:2-57) Pseudouridine synthase RsuA N-terminal domain {Haemophilus influenzae [TaxId: 727]}
</pre><pre>Probab=41.75  E-value=67  Score=15.52  Aligned_cols=29  Identities=14%  Similarity=0.459  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEE-----EEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTID-----VVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~-----i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      ..|.|++||+.+.     +.....+.+..+...+
</pre><pre>T Consensus        22 ~~g~v~vn~~~~~~~~~~l~~~~~i~i~~~~~~~   55 (56)
</pre><pre>T d1vioa2          22 RQSAVKINGEIVKSGSVQISQEDEIYFEDELLTW   55 (56)
</pre><pre>T ss_dssp             HTTCEEETTEECCCTTCEECTTSCEEETTEECCS
</pre><pre>T ss_pred             HcCCeEECCEEcccCceecCCCCEEEECCeecee
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>PF11685.10 ; DUF3281 ; Protein of unknown function (DUF3281)
</pre><pre>Probab=41.63  E-value=71  Score=23.87  Aligned_cols=26  Identities=35%  Similarity=0.652  Sum_probs=0.0  Template_Neff=4.000
</pre><pre>
</pre><pre>Q ss_pred             EEccCcEEEEcCeEEEEEec-cceEEc
</pre><pre>Q MBL0283831.1/7   16 VMKKDGTITLNGKTIDVVGS-DHIQLD   41 (48)
</pre><pre>Q Consensus        16 ~m~kdG~I~i~G~~I~i~gs-~~i~~~   41 (48)
</pre><pre>                      .+.-.|+|+++|+.+.|.-+ ..+.++
</pre><pre>T Consensus       124 ~IsVsGtvTvnGktvdl~~~v~~~~id  150 (267)
</pre><pre>T Q5NHH2_FRATT/4  124 SISVSGTITVNGKTVNLASTVPPVTVD  150 (267)
</pre><pre>T ss_pred             EEEEEEEEEECCEEEEccCCCCCeEEe
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>2RE3_A Uncharacterized protein; STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-2, UNKNOWN FUNCTION; HET: MSE; 2.5A {Silicibacter pomeroyi DSS-3}
</pre><pre>Probab=41.46  E-value=51  Score=22.38  Aligned_cols=22  Identities=27%  Similarity=0.445  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             eCCcEEEEccCcEEEEcCeEEE
</pre><pre>Q MBL0283831.1/7   10 VGKSVLVMKKDGTITLNGKTID   31 (48)
</pre><pre>Q Consensus        10 cG~~~i~m~kdG~I~i~G~~I~   31 (48)
</pre><pre>                      ||...|.++++|.-+..|.+|.
</pre><pre>T Consensus        38 ~~~~~i~ID~~G~W~~~g~~I~   59 (194)
</pre><pre>T 2RE3_A           38 NGDIDMRIARDGTWFYQGTPIN   59 (194)
</pre><pre>T ss_dssp             EEEEEEEECTTSCEEETTEECC
</pre><pre>T ss_pred             cccccEEECCCCCEEECCeecC
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>6B6L_B Glycosyl hydrolase family 2, sugar binding domain protein; structural genomics, PSI-Biology, protein structure initiative, midwest center for structural genomics, MCSG, glycosyl hydrolase, HYDROLASE; HET: CSD, MSE, EDO; 2.0A {Bacteroides cellulosilyticus DSM 14838}
</pre><pre>Probab=40.54  E-value=1.4e+02  Score=22.96  Aligned_cols=31  Identities=23%  Similarity=0.474  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=-.+.++.++..++||+++.+.|
</pre><pre>T Consensus       269 d~~~~~~G~R~~~~~~~~~f~lNG~~~~l~G  299 (776)
</pre><pre>T 6B6L_B          269 DEEYISIGIRNIAFSAENGFQLNGKSMKLKG  299 (776)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEeceEEEEeCCCcEEECCEEeceec
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>PF18715.3 ; Phage_spike ; Phage spike trimer
</pre><pre>Probab=40.50  E-value=96  Score=15.46  Aligned_cols=21  Identities=29%  Similarity=0.328  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             eEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   28 KTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        28 ~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +.+.+.++..|.++++.+...
</pre><pre>T Consensus         3 ~~~~i~a~~~v~~~~~~~~~~   23 (53)
</pre><pre>T W1IZ46_9GAMM/1    3 PEITCVASTRITLDTPEVVCT   23 (53)
</pre><pre>T ss_pred             CEEEEEeCceEEEeCCeEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>3SZE_A Serine protease espP; parallel beta-helix, serine protease, HYDROLASE; 2.5A {Escherichia coli O157:H7}
</pre><pre>Probab=40.43  E-value=1.3e+02  Score=25.51  Aligned_cols=38  Identities=24%  Similarity=0.388  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      |...|.+..|-++.++|......|++-+--+++.|+|+
</pre><pre>T Consensus       309 GaG~l~F~~~~~~~~~~~~~t~~gaGi~v~~~~~V~w~  346 (968)
</pre><pre>T 3SZE_A          309 GAGGLVFNDKKTYRVNGDDFTFKGAGVDTRNGSTVEWN  346 (968)
</pre><pre>T ss_dssp             TTCCEEECSSCEEEEECTTCEEESSCEEECTTCEEEEC
</pre><pre>T ss_pred             CceEEEECCCcEEEEcCCccEEEcceEEeCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>6TYI_Y Biopolymer transport protein ExbD; molecular motor, Ton system, membrane protein, TRANSPORT PROTEIN; HET: PEV, PGT;{Escherichia coli (strain K12)}
</pre><pre>Probab=39.08  E-value=87  Score=20.06  Aligned_cols=17  Identities=18%  Similarity=0.403  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .|.+..||+|+++++.+
</pre><pre>T Consensus        65 ~V~i~~dg~i~l~~~~~   81 (163)
</pre><pre>T 6TYI_Y           65 YLSVKADNSMFIGNDPV   81 (163)
</pre><pre>T ss_dssp             -----------------
</pre><pre>T ss_pred             EEEEcCCCeEEECCEeC
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>PF16532.7 ; Phage_tail_NK ; Sf6-type phage tail needle knob or tip of some Caudovirales
</pre><pre>Probab=38.86  E-value=97  Score=22.03  Aligned_cols=22  Identities=27%  Similarity=0.551  Sum_probs=0.0  Template_Neff=3.000
</pre><pre>
</pre><pre>Q ss_pred             EEEccCcEEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7   15 LVMKKDGTITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus        15 i~m~kdG~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +.++|+|.+.-||..+.|+..+
</pre><pre>T Consensus       140 FSVDk~GnlaTNG~~i~I~snG  161 (175)
</pre><pre>T Q716G6_BPSFV/1  140 FSVDKDGFLATNGSTLTIQSNG  161 (175)
</pre><pre>T ss_pred             EEEcCCCceecCCceEEEEECC
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>PF02563.18 ; Poly_export ; Polysaccharide biosynthesis/export protein
</pre><pre>Probab=38.79  E-value=1.4e+02  Score=16.75  Aligned_cols=25  Identities=20%  Similarity=0.359  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEe----------CCc-----EEEEccCcEEEE
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTV----------GKS-----VLVMKKDGTITL   25 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~c----------G~~-----~i~m~kdG~I~i   25 (48)
</pre><pre>                      ..||.+.+++          ...     ...+..||.|.+
</pre><pre>T Consensus        10 ~pgD~l~i~v~~~~~~~~~~~~~~~~~~~~~V~~~G~i~l   49 (86)
</pre><pre>T M1PUB3_DESSD/3   10 DNGDLLYVVIPKEITSEFSPTEKNYFKEKVEVDRHGFIFL   49 (86)
</pre><pre>T ss_pred             CCCCEEEEEecCccCCCCCccccccccCcEEECCCCeEEc
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>6U7J_A Beta-glucuronidase; microbiome, Glycosyl hydrolase, HYDROLASE; 2.2A {uncultured Clostridium sp.} SCOP: b.18.1.0, b.1.4.0, c.1.8.0, l.1.1.1
</pre><pre>Probab=38.78  E-value=2.1e+02  Score=20.67  Aligned_cols=30  Identities=23%  Similarity=0.423  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|-.++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       274 d~~~~~~G~r~v~~~-~~~~~~nG~~~~~~G  303 (594)
</pre><pre>T 6U7J_A          274 DHYTLPFGIRTIQLT-EKQFLINGKPFYFKG  303 (594)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEC-SSCEEETTEECCEEE
</pre><pre>T ss_pred             ceEEeccccEEEEEc-CCEEEECCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>6U7J_D Beta-glucuronidase; microbiome, Glycosyl hydrolase, HYDROLASE; 2.2A {uncultured Clostridium sp.} SCOP: b.18.1.0, l.1.1.1, b.1.4.0, c.1.8.0
</pre><pre>Probab=38.78  E-value=2.1e+02  Score=20.67  Aligned_cols=30  Identities=23%  Similarity=0.423  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|-.++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       274 d~~~~~~G~r~v~~~-~~~~~~nG~~~~~~G  303 (594)
</pre><pre>T 6U7J_D          274 DHYTLPFGIRTIQLT-EKQFLINGKPFYFKG  303 (594)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEC-SSCEEETTEECCEEE
</pre><pre>T ss_pred             ceEEeccccEEEEEc-CCEEEECCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>4S36_A Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; 1.46A {Pseudomonas aeruginosa}
</pre><pre>Probab=37.92  E-value=1.4e+02  Score=16.67  Aligned_cols=45  Identities=18%  Similarity=0.354  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEcc-CcEEEEcCeE--EEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKK-DGTITLNGKT--IDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~k-dG~I~i~G~~--I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      |..++..-....|.++. .|.+.+....  +.+. +..+.++++.+.++
</pre><pre>T Consensus        10 G~~i~yd~~~~~l~i~~~~g~~~i~~~~~~~~i~-~~~i~i~~~~i~~~   57 (90)
</pre><pre>T 4S36_A           10 GASLSYDWAAHRYRVELPSGTVEVRVGASEVRVS-DGAVSLKAPKISLE   57 (90)
</pre><pre>T ss_dssp             SCEEEEETTTTEEEEECSSSEEEEEETTEEEEEE-TTEEEEECSCCCCC
</pre><pre>T ss_pred             CcEEEEeCCCCEEEEEeCCceEEEEeCCceEEEe-cCCEEEeCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>5T9G_C Glycoside Hydrolase; (alpha/beta)6 barrel, glycoside hydrolase, HYDROLASE; HET: GIF, EDO; 2.4A {Bacteroides uniformis}
</pre><pre>Probab=37.77  E-value=1.8e+02  Score=22.82  Aligned_cols=31  Identities=23%  Similarity=0.370  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=..+.++.++..++||+++.+.|
</pre><pre>T Consensus       285 d~~~~~fG~R~i~~~~~~~~~lNG~~~~l~G  315 (846)
</pre><pre>T 5T9G_C          285 DTKETPFGIRSIKFDAKKGFFLNGKNMKIKG  315 (846)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEEECEEE
</pre><pre>T ss_pred             EEEEEEeeeEEEEEeCCCcEEECCEEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>5T9G_D Glycoside Hydrolase; (alpha/beta)6 barrel, glycoside hydrolase, HYDROLASE; HET: GIF, EDO; 2.4A {Bacteroides uniformis}
</pre><pre>Probab=37.77  E-value=1.8e+02  Score=22.82  Aligned_cols=31  Identities=23%  Similarity=0.370  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=..+.++.++..++||+++.+.|
</pre><pre>T Consensus       285 d~~~~~fG~R~i~~~~~~~~~lNG~~~~l~G  315 (846)
</pre><pre>T 5T9G_D          285 DTKETPFGIRSIKFDAKKGFFLNGKNMKIKG  315 (846)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEeeeEEEEEeCCCcEEECCEEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>PF18620.3 ; DUF5627 ; Family of unknown function (DUF5627)
</pre><pre>Probab=36.30  E-value=1.4e+02  Score=19.97  Aligned_cols=35  Identities=29%  Similarity=0.448  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCe--EEEEEeccceEEcCc----------eeecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGK--TIDVVGSDHIQLDGK----------RIDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~--~I~i~gs~~i~~~~~----------~I~~n   48 (48)
</pre><pre>                      .|.++.||.++|...  ...+.|+++...++.          .|.|+
</pre><pre>T Consensus        68 ~Ltfd~~~~~tv~~~~~~~~~sGtG~y~~~g~~~~wg~~~Rd~i~L~  114 (146)
</pre><pre>T R5FAK4_9BACT/1   68 LLSFHDDNTCTVTTETPGCTATGSGTYTIDGAKKAWGDKDRDLIKLD  114 (146)
</pre><pre>T ss_pred             EEEECCCCcEEEEcCCCCceeeceEEEeeCCceeecCCcccceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>d1kska3 d.66.1.5 (A:1-59) Pseudouridine synthase RsuA N-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=36.05  E-value=99  Score=15.16  Aligned_cols=28  Identities=14%  Similarity=0.503  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEE-----EEeccceEEcCceee
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTID-----VVGSDHIQLDGKRID   46 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~-----i~gs~~i~~~~~~I~   46 (48)
</pre><pre>                      +.|.|++||+.+.     +.....+.+++....
</pre><pre>T Consensus        23 ~~g~v~vn~~~v~~~~~~l~~~~~i~i~~~~~~   55 (59)
</pre><pre>T d1kska3          23 RGNRVTVDGEIVRNAAFKLLPEHDVAYDGNPLA   55 (59)
</pre><pre>T ss_dssp             HTTCEEETTEECCCTTCEECTTCCEEETTEEEC
</pre><pre>T ss_pred             hCCcEEECCEEecCCCcccCCCCeEEEcCEecc
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>TIGR02892 spore_yabP; sporulation protein YabP. Members of this protein family are the YabP protein of the bacterial sporulation program, as found in Bacillus subtilis, Clostridium tetani, and other spore-forming members of the Firmicutes.
</pre><pre>Probab=35.88  E-value=1.8e+02  Score=17.39  Aligned_cols=36  Identities=14%  Similarity=0.371  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             EEeCCcEEEEc-cCcEEEEcCeEEEEEecc----ceEEcCc
</pre><pre>Q MBL0283831.1/7    8 FTVGKSVLVMK-KDGTITLNGKTIDVVGSD----HIQLDGK   43 (48)
</pre><pre>Q Consensus         8 l~cG~~~i~m~-kdG~I~i~G~~I~i~gs~----~i~~~~~   43 (48)
</pre><pre>                      +.+....+.+. +.|.+.|.|..+.|.--.    .+.+.|.
</pre><pre>T Consensus        20 ~~~~~~~I~i~~~~g~l~I~G~~L~I~~~~~~~~~i~I~G~   60 (85)
</pre><pre>T TIGR02892        20 ISFDDEEILLETVMGFLTIKGQELKMNKLDVENGQVIIKGF   60 (85)
</pre><pre>T ss_pred             EEECCCEEEEEeCCcEEEEEcCCeEEEEEECCCCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>5VTG_A Translocation and assembly module subunit TamB; Chaperone, Beta-sheet, Periplasm; 1.859A {Escherichia coli}
</pre><pre>Probab=35.78  E-value=2.2e+02  Score=18.32  Aligned_cols=31  Identities=13%  Similarity=0.235  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceee
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRID   46 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~   46 (48)
</pre><pre>                      ++.+.. |.+.+-++++.|. .+.|.+++...+
</pre><pre>T Consensus       105 ~i~i~~-G~~~~~~k~F~i~-~G~i~F~g~~~~  135 (184)
</pre><pre>T 5VTG_A          105 QINIPE-GRFHAYGQDLIVR-KGELLFSGPPDQ  135 (184)
</pre><pre>T ss_dssp             EEEEEE-EEEEETTEEEEEE-EEEEEEEECTTC
</pre><pre>T ss_pred             EEEEec-eEEEEcCeEEEEE-eeEEEEeCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>2VZS_B EXO-BETA-D-GLUCOSAMINIDASE; EXO-BETA-D-GLUCOSAMINIDASE, HYDROLASE, GH2, CSXA, GLUCOSAMINE, GLYCOSIDE HYDROLASE; HET: GCS, GOL; 1.85A {AMYCOLATOPSIS ORIENTALIS} SCOP: b.1.4.1, b.18.1.5, c.1.8.3
</pre><pre>Probab=35.33  E-value=2.2e+02  Score=22.82  Aligned_cols=33  Identities=18%  Similarity=0.255  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEc----cCcEEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMK----KDGTITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~----kdG~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +..+.++|=-++.+.    .+..+.|||++|.+.|..
</pre><pre>T Consensus       326 d~~~~~~G~R~v~~~~~~~~~~~~~~Ng~~~~~~G~~  362 (1032)
</pre><pre>T 2VZS_B          326 DAAKSKFGVRDVKATLNSSGGRQYSVNGKPLLIRGGG  362 (1032)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEEECTTSCEEEEETTEECCEEEEE
</pre><pre>T ss_pred             eeEEEEEEEEEEEEEEeCCCCEEEEECCEEeeEEeee
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>5UJ6_A Glycosyl hydrolases family 2, sugar binding domain protein; Glucuronic-acid Carbohydrate, HYDROLASE; HET: GOL; 1.9A {Bacteroides uniformis str. 3978 T3 ii}
</pre><pre>Probab=35.09  E-value=2.1e+02  Score=22.17  Aligned_cols=31  Identities=23%  Similarity=0.244  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|-..+.++.+...++||+++.+.|
</pre><pre>T Consensus       287 d~~~~~~G~r~~~~~~~~~~~lNG~p~~~~G  317 (878)
</pre><pre>T 5UJ6_A          287 DQSNHYTAFRWFRFDGDEGFFLNGKPYKLRG  317 (878)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEeeeeeEEEEEcCCCcEEECCEEEeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>6S0X_i 30S ribosomal protein S9; Ribosome, antibiotics, resistance, Staphylococcus aureus, exit tunnel, RNA, rProteins, erythromycin; HET: ERY; 2.425A {Staphylococcus aureus}
</pre><pre>Probab=34.99  E-value=80  Score=19.74  Aligned_cols=18  Identities=22%  Similarity=0.510  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcc-CcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   13 SVLVMKK-DGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        13 ~~i~m~k-dG~I~i~G~~I   30 (48)
</pre><pre>                      ++..+.+ +|.|+|||+++
</pre><pre>T Consensus        14 a~v~~~~G~G~i~INg~~~   32 (127)
</pre><pre>T 6S0X_i           14 ARVRLVPGEGNITVNNRDV   32 (127)
</pre><pre>T ss_dssp             EECCCBSSSCCCCBTTBCS
</pre><pre>T ss_pred             EEEEEECCCCeEEECCeeH
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>PF12059.10 ; DUF3540 ; Protein of unknown function (DUF3540)
</pre><pre>Probab=34.85  E-value=2.6e+02  Score=18.86  Aligned_cols=38  Identities=8%  Similarity=0.279  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +...|.+..|-.|....-.+.+.+...+.+.++.++++
</pre><pre>T Consensus        60 ~~~~l~~~g~~~l~a~~G~l~l~a~~~l~l~a~~~~l~   97 (191)
</pre><pre>T F3YX79_DESAF/1   60 TEAIVSFDGPASFRVAQGNLTLAAQGEIGLAAGRVAVD   97 (191)
</pre><pre>T ss_pred             CceEEEeCCCeEEEEcCCcEEEeeCCeEEEeeceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>3GM8_A Glycoside hydrolase family 2, candidate beta-glycosidase; STRUCTURAL GENOMICS, HYDROLASE, Glycosidase, PSI-2, Protein Structure Initiative, New York SGX Research Center for Structural Genomics; HET: GOL; 2.4A {Bacteroides vulgatus}
</pre><pre>Probab=34.76  E-value=1.9e+02  Score=22.48  Aligned_cols=31  Identities=29%  Similarity=0.443  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=..+.+..++..++||+++.+.|
</pre><pre>T Consensus       260 d~~~~~~G~R~v~~~~~~~f~lNG~~~~l~G  290 (801)
</pre><pre>T 3GM8_A          260 DECRLPVGFRQLEFNPVSGFLLNGKSLKIKG  290 (801)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETTTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEEEEEEEeCCCcEEECCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>PF05910.14 ; DUF868 ; Plant protein of unknown function (DUF868)
</pre><pre>Probab=34.51  E-value=3.4e+02  Score=20.07  Aligned_cols=43  Identities=19%  Similarity=0.393  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             eEEEEe-----CCcEEEEccCcEEEEcCeEE--EEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7    5 SITFTV-----GKSVLVMKKDGTITLNGKTI--DVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus         5 ~~~l~c-----G~~~i~m~kdG~I~i~G~~I--~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      ++.+.|     ++..+.+.=||...+.=+.+  .+.|...|.+|+..|++
</pre><pre>T Consensus       156 eI~I~~~~~g~~~~~l~v~iD~k~v~~VkrL~W~FRGN~ti~vdg~~V~v  205 (273)
</pre><pre>T A0A3Q7I695_SOL  156 DITIECDSSVIDDPCLLVRIDSKTVMQVKHLRWKFRGNYTVLIDGLPVEV  205 (273)
</pre><pre>T ss_pred             EEEEEecCCCCCCCeEEEEEcCEEEeEEEeeeEEEEeeEEEEECCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>1RJL_C Outer surface protein B; Beta Sheet, Antibody-Protein Complex, IMMUNE SYSTEM; 2.6A {Mus musculus} SCOP: b.76.1.1
</pre><pre>Probab=34.50  E-value=1.8e+02  Score=18.90  Aligned_cols=25  Identities=32%  Similarity=0.454  Sum_probs=0.0  Template_Neff=2.800
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCc---EEEEccCcEEEEcCeE
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKS---VLVMKKDGTITLNGKT   29 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~---~i~m~kdG~I~i~G~~   29 (48)
</pre><pre>                      .+++..-..   .|++.+||+|+++..+
</pre><pre>T Consensus        44 TLTIs~~skktk~lVF~~~~TITvq~Yd   71 (95)
</pre><pre>T 1RJL_C           44 TLTISADSKKTKDLVFLTDGTITVQQYN   71 (95)
</pre><pre>T ss_dssp             EEEEEETTEEEEEEEECTTSCEEEEEBC
</pre><pre>T ss_pred             eEEEEeCCcccceEEEecCCeEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>PF11858.10 ; DUF3378 ; Domain of unknown function (DUF3378)
</pre><pre>Probab=34.27  E-value=1.6e+02  Score=16.20  Aligned_cols=24  Identities=13%  Similarity=0.351  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEeCCcEEEEccCcEEEEcCeE
</pre><pre>Q MBL0283831.1/7    6 ITFTVGKSVLVMKKDGTITLNGKT   29 (48)
</pre><pre>Q Consensus         6 ~~l~cG~~~i~m~kdG~I~i~G~~   29 (48)
</pre><pre>                      .....++..+++=+.|++.++|++
</pre><pre>T Consensus        31 f~~k~~~~~I~iY~sgK~liQG~~   54 (75)
</pre><pre>T I0JPK2_HALH3/3   31 FAAKTSACTITAYRSGKVLFQGKS   54 (75)
</pre><pre>T ss_pred             EEEEcCCcEEEEEcCCEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>4YGU_D putative adhesin; PF10988 family protein, DUF2807, single-strabded right handed beta-helix fold, Structural Genomics, Joint Center for Structural Genomics; HET: MSE, PEG, PGE, EDO; 2.2A {Bacteroides eggerthii DSM 20697}
</pre><pre>Probab=34.06  E-value=2.1e+02  Score=17.50  Aligned_cols=44  Identities=16%  Similarity=0.225  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCcEEEEccCcEEEEcCeE--EEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKSVLVMKKDGTITLNGKT--IDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~~i~m~kdG~I~i~G~~--I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+...++...+.+...|.|.+.|..  +.+..++.-.+++..+..+
</pre><pre>T Consensus       138 ~~~~~~~~~~~~~~~~g~i~l~g~~~~~~~~~~g~g~i~~~~l~~~  183 (184)
</pre><pre>T 4YGU_D          138 DIHVVCDYLSAQMGGVGSVTLSGSAGRADISKGGIGGVNTDNLKIG  183 (184)
</pre><pre>T ss_dssp             EEEEEEEEEEEEECSSSEEEEEEEEEEEEEEECSSSEEEEEEEEEC
</pre><pre>T ss_pred             EEEEEeCEEEEEEcCceEEEEEeecceEEEEeccceeeecccceec
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>1JZ7_D Beta-Galactosidase; TIM BARREL (ALPHA/BETA BARREL), JELLY-ROLL BARREL, IMMUNOGLOBULIN, BETA SUPERSANDWICH, HYDROLASE; HET: NA, MG, CSO, GAL, DMS; 1.5A {Escherichia coli} SCOP: b.1.4.1, b.18.1.5, b.30.5.1, c.1.8.3
</pre><pre>Probab=33.49  E-value=3e+02  Score=22.53  Aligned_cols=30  Identities=27%  Similarity=0.470  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=-++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       324 d~~~~~~GfR~v~~~-~~~~~lNG~p~~l~G  353 (1023)
</pre><pre>T 1JZ7_D          324 EAEACDVGFREVRIE-NGLLLLNGKPLLIRG  353 (1023)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEcCeEEEEE-CCEEEECCEEEeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>PF14498.8 ; Glyco_hyd_65N_2 ; Glycosyl hydrolase family 65, N-terminal domain
</pre><pre>Probab=33.30  E-value=2e+02  Score=19.25  Aligned_cols=25  Identities=12%  Similarity=0.302  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             EEccCcEEEEcCeEEEEEeccceEE
</pre><pre>Q MBL0283831.1/7   16 VMKKDGTITLNGKTIDVVGSDHIQL   40 (48)
</pre><pre>Q Consensus        16 ~m~kdG~I~i~G~~I~i~gs~~i~~   40 (48)
</pre><pre>                      +...+|++...+..|.|+++..+.+
</pre><pre>T Consensus       212 v~~~~G~~~~~~~~l~v~~a~~v~i  236 (247)
</pre><pre>T Q5WIX9_BACSK/3  212 LLETDGNAAVKNNKLVVEDARYATV  236 (247)
</pre><pre>T ss_pred             EEeCCcEEEEECCEEEEecCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>d1p9ka1 d.66.1.6 (A:3-79) Hypothetical protein YbcJ {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=33.22  E-value=1.4e+02  Score=15.70  Aligned_cols=28  Identities=14%  Similarity=0.546  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCeEEE-----EEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7   20 DGTITLNGKTID-----VVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus        20 dG~I~i~G~~I~-----i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      +|.|.+||+.+.     +...+.+.++.+.+..
</pre><pre>T Consensus        43 ~g~v~vng~~~~~~~~~l~~g~~i~i~~~~~~~   75 (77)
</pre><pre>T d1p9ka1          43 EGQVKVDGAVETRKRCKIVAGQTVSFAGHSVQV   75 (77)
</pre><pre>T ss_dssp             HHHHEETTBCCCCSSCCCCSSEEEEETTEEEEE
</pre><pre>T ss_pred             cCCcEECCEEecCCCceeCCCCEEEECCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>5EUV_B Beta-D-galactosidase; beta-D-galactosidase, dimeric, cold-adapted, enzyme, glycosyl hydrolase, native, hydrolase; HET: PEG, ACT, BTB; 2.4A {Paracoccus sp. 32d}
</pre><pre>Probab=33.13  E-value=3.1e+02  Score=21.26  Aligned_cols=33  Identities=27%  Similarity=0.448  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      ..++..+...|=..+.++.++ .++||+++.+.|
</pre><pre>T Consensus       231 ~~~d~~~~~~G~R~~~~~~~~-~~lNG~~~~l~G  263 (731)
</pre><pre>T 5EUV_B          231 DSGDVTTHRFGFRTAEWTPQG-FLLNGQPMKLRG  263 (731)
</pre><pre>T ss_dssp             TTCCEEEEEECCCCEEEETTE-EEETTEECCEEE
</pre><pre>T ss_pred             CCCceEEEEEEeeEEEEcCCe-EEECCEEeeeee
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>TIGR03027 pepcterm_export; putative polysaccharide export protein, PEP-CTERM sytem-associated. This protein family belongs to the larger set of polysaccharide biosynthesis/export proteins described by pfam02563.
</pre><pre>Probab=32.87  E-value=1.9e+02  Score=18.21  Aligned_cols=25  Identities=32%  Similarity=0.531  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCc-----EEEEccCcEEEE
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKS-----VLVMKKDGTITL   25 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~-----~i~m~kdG~I~i   25 (48)
</pre><pre>                      ..||.+.+..-+.     ...+..||+|++
</pre><pre>T Consensus         4 ~~gD~i~i~~~~~~~~~~~~~V~~~G~i~~   33 (165)
</pre><pre>T TIGR03027         4 GPGDSLNINVWRNPELSGSVPVRPDGKITT   33 (165)
</pre><pre>T ss_pred             CCCCEEEEEecCCccccCceEeCCCCeEEc
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>2A50_C GFP-like non-fluorescent chromoprotein FP595 chain 1; asCP, fluorescent protein, chromoprotein, photochromic protein, reversible photoswitch, LUMINESCENT PROTEIN; HET: NRQ; 1.3A {Anemonia sulcata}
</pre><pre>Probab=32.85  E-value=1.3e+02  Score=18.33  Aligned_cols=22  Identities=23%  Similarity=0.434  Sum_probs=0.0  Template_Neff=4.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=+|.|  ||..+.+.|.+.
</pre><pre>T Consensus        21 ~~k~~meG~V--NGh~F~i~G~G~   42 (73)
</pre><pre>T 2A50_C           21 PFKTTIEGTV--NGHYFKCTGKGE   42 (73)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             cEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>PF15507.8 ; DUF4649 ; Domain of unknown function (DUF4649)
</pre><pre>Probab=32.28  E-value=71  Score=18.98  Aligned_cols=13  Identities=31%  Similarity=0.447  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7   22 TITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus        22 ~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +++++|+++.++|
</pre><pre>T Consensus        41 ~vt~dG~~l~~~g   53 (68)
</pre><pre>T Q8DZ00_STRA5/2   41 KMTYKDKDIAYSG   53 (68)
</pre><pre>T ss_pred             EEEECCEEcccce
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>5XLJ_A Flagellar hook-associated protein 2; Bacterial flagellar cap protein, STRUCTURAL PROTEIN; 1.9A {Serratia marcescens}
</pre><pre>Probab=31.93  E-value=1.9e+02  Score=19.14  Aligned_cols=29  Identities=38%  Similarity=0.673  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEEeccceE--EcCceeecC
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVVGSDHIQ--LDGKRIDLN   48 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~gs~~i~--~~~~~I~~n   48 (48)
</pre><pre>                      +|..+.|+|..+. ..+..+.  ++|-.++|.
</pre><pre>T Consensus       162 ~dA~~~inG~~~t-~~sN~~~~~i~Gv~~~L~  192 (208)
</pre><pre>T 5XLJ_A          162 QDAELTVNGTAIK-RSTNSISDALQGVTIDLK  192 (208)
</pre><pre>T ss_dssp             BCEEEEETTEEEE-ESSSEECSSSTTEEEEEC
</pre><pre>T ss_pred             CceEEEECCEEEe-eCCcEEeeccCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>PF17115.7 ; Toast_rack_N ; N-terminal domain of toast_rack, DUF2154
</pre><pre>Probab=31.78  E-value=1.7e+02  Score=16.23  Aligned_cols=24  Identities=8%  Similarity=0.210  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCcEEEEccCcEEEEcCe
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKSVLVMKKDGTITLNGK   28 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~~i~m~kdG~I~i~G~   28 (48)
</pre><pre>                      ++.|.+|.+++.+.+...-.+.|.
</pre><pre>T Consensus         7 ~l~l~~g~g~l~i~~~~~~l~~~~   30 (91)
</pre><pre>T Q020Q5_SOLUE/3    7 RVDLRMGAGDLQVTDGAQKLVRAD   30 (91)
</pre><pre>T ss_pred             EEEEEecCEEEEEecCCcceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>PF03633.17 ; Glyco_hydro_65C ; Glycosyl hydrolase family 65, C-terminal domain
</pre><pre>Probab=31.51  E-value=1.4e+02  Score=14.63  Aligned_cols=30  Identities=27%  Similarity=0.488  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCc------EEEEcCeEEEE
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDG------TITLNGKTIDV   32 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG------~I~i~G~~I~i   32 (48)
</pre><pre>                      |..+.+++....+.+....      .+.+.|+.+.+
</pre><pre>T Consensus        18 g~~l~v~v~~~~~~i~~~~~~~~~~~~~~~g~~~~l   53 (53)
</pre><pre>T W5TL32_9NOCA/1   18 GHRLTVRVDAEFVDITSDADGADPITIECRGRTVRL   53 (53)
</pre><pre>T ss_pred             CEEEEEEEeCCEEEEEeCCCCCCCEEEEECCEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>6PAL_A Uncharacterized protein; laminarinase, HYDROLASE, endo-glucanase; HET: ACT; 1.818A {Bacteroides uniformis}
</pre><pre>Probab=31.35  E-value=2.4e+02  Score=19.30  Aligned_cols=23  Identities=22%  Similarity=0.518  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCc-EEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDG-TITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG-~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +++-..++ .+++||+++.+.|..
</pre><pre>T Consensus         5 ~~~~~~~~~~~~~nG~~~~~~G~~   28 (407)
</pre><pre>T 6PAL_A            5 VIEKAADGYKLSIDGRETYIKGVG   28 (407)
</pre><pre>T ss_dssp             EEEECSSSEEEEETTEECCEEEEE
</pre><pre>T ss_pred             EEEEcCCcEEEEECCEEEeEEccc
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>6ZJS_A Beta-galactosidase; galactosidase, cold-adapted, psychrophilic, dimeric, mutant, loss of function, HYDROLASE, galactose, complex; HET: LMR, ACT, GAL, MLI; 1.5A {Arthrobacter sp. 32cB}
</pre><pre>Probab=31.25  E-value=3.5e+02  Score=22.11  Aligned_cols=31  Identities=26%  Similarity=0.607  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      ++.++..+|=.++.++ ++.+++||+++.+.|
</pre><pre>T Consensus       300 ~~~~~~~~GfR~v~~~-~~~~~lNG~~~~l~G  330 (1010)
</pre><pre>T 6ZJS_A          300 GESVALQIGFRSIAIE-DAQFKVNGRRILLRG  330 (1010)
</pre><pre>T ss_dssp             SCEEEEEECCCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             CcEEEEEEccEEEEEE-CCEEEECCEEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>3LPF_B Beta-glucuronidase; alpha/beta barrel, sugar-binding domain, beta-sandwich domain, glycosyl hydrolase, beta-glucuronidase inhibitor, Glycosidase, Hydrolase, HYDROLASE-HYDROLASE INHIBITOR complex; HET: Z77, MSE; 2.26A {Escherichia coli} SCOP: b.18.1.0, b.1.4.0, c.1.8.0, l.1.1.1
</pre><pre>Probab=31.20  E-value=3.7e+02  Score=19.44  Aligned_cols=30  Identities=23%  Similarity=0.334  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +.....+|=..+.+. ++.+++||+++.+.|
</pre><pre>T Consensus       265 d~~~~~~G~r~~~~~-~~~~~~nG~~~~~~G  294 (605)
</pre><pre>T 3LPF_B          265 DIYPLRVGIRSVAVK-GEQFLINHKPFYFTG  294 (605)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEE-TTEEEESSCBCCBCE
</pre><pre>T ss_pred             EEEEEEEcceEEEEe-CcEEEeCCEEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>6U7I_B Beta-glucuronidase; microbiome, Glycosyl hydrolase, HYDROLASE; 2.7A {Faecalibacterium prausnitzii} SCOP: b.18.1.0, b.1.4.0, c.1.8.0
</pre><pre>Probab=30.31  E-value=2.7e+02  Score=20.25  Aligned_cols=30  Identities=27%  Similarity=0.395  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=..+.+. ++.+.+||+++.+.|
</pre><pre>T Consensus       275 ~~~~~~~g~r~~~~~-~~~~~~ng~~~~~~G  304 (597)
</pre><pre>T 6U7I_B          275 DVYDQTFGVRSIEVR-GTQVLLNGKPLYFKG  304 (597)
</pre><pre>T ss_dssp             EEEEEEEECCCEEEE-TTEEEETTEEBCEEE
</pre><pre>T ss_pred             ceEeeeEeeEEEEEe-CeEEEECCEEEccEe
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>PF20132.1 ; DUF6522 ; Family of unknown function (DUF6522)
</pre><pre>Probab=30.28  E-value=2.1e+02  Score=17.41  Aligned_cols=20  Identities=40%  Similarity=0.629  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCc--EEEEccCcEEE
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKS--VLVMKKDGTIT   24 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~--~i~m~kdG~I~   24 (48)
</pre><pre>                      .+++.+|..  +++++.+|.|.
</pre><pre>T Consensus        54 Rltf~~~~r~~r~ivd~~G~vl   75 (81)
</pre><pre>T A0A1M7GRH4_9RH   54 RITFYAGARALRLIVDRDGTHL   75 (81)
</pre><pre>T ss_pred             EEEEEECCEEEEEEECCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>5H5W_B Flagellar hook-associated protein 2; Bacterial flagellar cap protein, STRUCTURAL PROTEIN; 2.15A {Escherichia coli}
</pre><pre>Probab=30.04  E-value=2.2e+02  Score=18.97  Aligned_cols=29  Identities=28%  Similarity=0.520  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEEeccceE--EcCceeecC
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVVGSDHIQ--LDGKRIDLN   48 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~gs~~i~--~~~~~I~~n   48 (48)
</pre><pre>                      .|+.+.|+|.++. ..+..+.  ++|-.++|+
</pre><pre>T Consensus       159 ~dA~~~inG~~~~-~~sN~~~d~i~Gv~~~l~  189 (203)
</pre><pre>T 5H5W_B          159 QNAQLTVNNVAIE-NSSNTISDALENITLNLN  189 (203)
</pre><pre>T ss_dssp             BCEEEEETTEEEE-ESSSEEC-CCTTEEEEEC
</pre><pre>T ss_pred             CcEEEEECCEEEE-eCCceeeeccCCEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>d2wiqa_ d.22.1.0 (A:) automated matches {Anthomedusae sp. [TaxId: 328397]}
</pre><pre>Probab=29.88  E-value=1.7e+02  Score=21.20  Aligned_cols=25  Identities=20%  Similarity=0.422  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         9 ~~m~~k~~meG~V--NGh~F~i~GeG~   33 (228)
</pre><pre>T d2wiqa_           9 SDMTFKIFIDGEV--NGQKFTIVADGS   33 (228)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCcEEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>1H6R_C GREEN FLUORESCENT PROTEIN; LUMINESCENT PROTEIN, LUMINESCENCE, GREEN FLUORESCENT PROTEIN, YELLOW-EMISSION; HET: PIA; 1.5A {AEQUOREA VICTORIA} SCOP: l.1.1.1, d.22.1.1
</pre><pre>Probab=29.78  E-value=1.7e+02  Score=21.13  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus        10 ~~~~~k~~~~g~v--ng~~f~i~G~G~   34 (236)
</pre><pre>T 1H6R_C           10 GVVPILVELDGDV--NGHKFSVSGEGE   34 (236)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>PF13186.8 ; SPASM ; Iron-sulfur cluster-binding domain
</pre><pre>Probab=29.59  E-value=1.2e+02  Score=14.27  Aligned_cols=15  Identities=7%  Similarity=0.094  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             eCCcEEEEccCcEEE
</pre><pre>Q MBL0283831.1/7   10 VGKSVLVMKKDGTIT   24 (48)
</pre><pre>Q Consensus        10 cG~~~i~m~kdG~I~   24 (48)
</pre><pre>                      .+...+.+..||.|+
</pre><pre>T Consensus         2 ~~~~~~~i~~~G~v~   16 (65)
</pre><pre>T Q2JJN8_SYNJB/2    2 TRWFDISITATGVVA   16 (65)
</pre><pre>T ss_pred             CccceeEEeCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>PF10988.10 ; DUF2807 ; Putative auto-transporter adhesin, head GIN domain
</pre><pre>Probab=29.52  E-value=2.4e+02  Score=17.61  Aligned_cols=37  Identities=14%  Similarity=0.296  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             EEeCCcEEEEccCcEEEEcCeE---EEEEeccceEEcCce
</pre><pre>Q MBL0283831.1/7    8 FTVGKSVLVMKKDGTITLNGKT---IDVVGSDHIQLDGKR   44 (48)
</pre><pre>Q Consensus         8 l~cG~~~i~m~kdG~I~i~G~~---I~i~gs~~i~~~~~~   44 (48)
</pre><pre>                      +.++...+.+...|.|.+....   +.+.|++.|.+.|.+
</pre><pre>T Consensus       164 ~~~~~~~v~~~~~g~i~v~~~~~~~~~~~~~g~i~~~g~p  203 (203)
</pre><pre>T A9WHL3_CHLAA/4  164 LKASIVEVAVNGSGDVTVWAVDTLDVSISGSGNVRYLGSP  203 (203)
</pre><pre>T ss_pred             cEEeEEEEEEecceEEEEEecCeEEEEEcCceEEEEEccC
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>d2ejoa_ d.22.1.0 (A:) automated matches {Fungia concinna [TaxId: 496660]}
</pre><pre>Probab=29.39  E-value=1.5e+02  Score=21.33  Aligned_cols=22  Identities=23%  Similarity=0.471  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-||+|  ||..+.|.|.+.
</pre><pre>T Consensus         9 ~~k~~meG~v--Ngh~F~i~GeG~   30 (217)
</pre><pre>T d2ejoa_           9 KMRYYMDGSV--NGHEFTIEGEGT   30 (217)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>3AKO_E Venus; Venus, Fluorescent Protein, GFP; HET: CR2, PE8; 2.1A {Plant transformation vector pSITEII-4C1}
</pre><pre>Probab=29.37  E-value=1.8e+02  Score=20.34  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus        31 ~~m~~k~~meG~v--NG~~F~i~GeG~   55 (173)
</pre><pre>T 3AKO_E           31 GVVPILVELDGDV--NGHKFSVSGEGE   55 (173)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>3AKO_G Venus; Venus, Fluorescent Protein, GFP; HET: PE8, CR2; 2.1A {Plant transformation vector pSITEII-4C1}
</pre><pre>Probab=29.37  E-value=1.8e+02  Score=20.34  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus        31 ~~m~~k~~meG~v--NG~~F~i~GeG~   55 (173)
</pre><pre>T 3AKO_G           31 GVVPILVELDGDV--NGHKFSVSGEGE   55 (173)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>d2rh7a1 d.22.1.1 (A:7-226) Green fluorescent protein, GFP {Renilla reniformis [TaxId: 6136]}
</pre><pre>Probab=29.36  E-value=1.5e+02  Score=21.47  Aligned_cols=22  Identities=9%  Similarity=0.252  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         7 ~~k~~~eG~v--Ngh~F~i~GeG~   28 (220)
</pre><pre>T d2rh7a1           7 PTKINLEGLV--GDHAFSMEGVGE   28 (220)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>2RA9_A Uncharacterized protein DUF1285; Structural genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, unknown function; HET: EDO; 1.4A {Shewanella baltica}
</pre><pre>Probab=29.35  E-value=92  Score=20.42  Aligned_cols=21  Identities=19%  Similarity=0.226  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             eCCcEE-EEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   10 VGKSVL-VMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        10 cG~~~i-~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      ||...+ .++++|.-+.+|.+|
</pre><pre>T Consensus        21 ~~~~~ii~ID~~G~W~~~G~~I   42 (150)
</pre><pre>T 2RA9_A           21 CSEVPLFDINALGDWTYLGTSL   42 (150)
</pre><pre>T ss_dssp             -CCCCSEEECTTSCEEETTEEC
</pre><pre>T ss_pred             cccCCcEEECCCCcEEeCCccC
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>d2c9ja_ d.22.1.0 (A:) automated matches {Cerianthus membranaceus [TaxId: 208460]}
</pre><pre>Probab=29.23  E-value=1.5e+02  Score=21.31  Aligned_cols=22  Identities=14%  Similarity=0.252  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ++.+.=+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         3 ~~k~~meG~V--NGh~F~i~GeG~   24 (216)
</pre><pre>T d2c9ja_           3 SVSVYMKGNV--NNHEFEYDGIGG   24 (216)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>1WTH_A Tail-associated lysozyme; Triple-stranded beta-helix, OB fold, pseudohexamer, T4 tail lysozyme, HUB, gp5-gp27, Hydrolase-Structural protein COMPLEX; 2.8A {Enterobacteria phage T4}
</pre><pre>Probab=29.20  E-value=3e+02  Score=22.08  Aligned_cols=46  Identities=22%  Similarity=0.387  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      .++.......++ .+.+...+.+.+.+....+.+++.+.+.|..+++
</pre><pre>T Consensus       529 ~v~g~~~i~~gg-~i~i~sgg~~~i~~g~~~~~asg~V~I~G~ki~~  574 (584)
</pre><pre>T 1WTH_A          529 TVNGNLSWKVAG-TVDWDVGGDWTEKMASMSSISSGQYTIDGSRIDI  574 (584)
</pre><pre>T ss_dssp             CCSSCCCCCCSS-CCCCCCCSCCCCCCSCCCCCCSSCCCCCCSCCCC
</pre><pre>T ss_pred             EECCCEEEEECC-cEEEEEcCCEEEEeCceeEecCeeEEEECceeEe
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>4IAJ_A Conserved domain protein; DUF4649, PF15507 family protein, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY; HET: MSE, EDO, UNL; 1.91A {Streptococcus pneumoniae}
</pre><pre>Probab=28.98  E-value=81  Score=19.25  Aligned_cols=13  Identities=31%  Similarity=0.639  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7   22 TITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus        22 ~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +++++|+++.++|
</pre><pre>T Consensus        48 ~vt~dGk~l~~~G   60 (78)
</pre><pre>T 4IAJ_A           48 KLTYKGHNLDYHG   60 (78)
</pre><pre>T ss_dssp             EEEETTEECCCCS
</pre><pre>T ss_pred             EEEECCEEccccc
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>4IAJ_D Conserved domain protein; DUF4649, PF15507 family protein, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY; HET: EDO, UNL, MSE; 1.91A {Streptococcus pneumoniae}
</pre><pre>Probab=28.98  E-value=81  Score=19.25  Aligned_cols=13  Identities=31%  Similarity=0.639  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7   22 TITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus        22 ~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +++++|+++.++|
</pre><pre>T Consensus        48 ~vt~dGk~l~~~G   60 (78)
</pre><pre>T 4IAJ_D           48 KLTYKGHNLDYHG   60 (78)
</pre><pre>T ss_dssp             EEEETTEEECCCS
</pre><pre>T ss_pred             EEEECCEEccccc
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>6U9G_B VgrG; T6SS Central Spike, Complex, TRANSPORT PROTEIN; 3.98A {Francisella tularensis subsp. novicida (strain U112)}
</pre><pre>Probab=28.91  E-value=3e+02  Score=17.60  Aligned_cols=33  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEEeCC------cEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    2 AGDSITFTVGK------SVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         2 aG~~~~l~cG~------~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      .+..+.+.++.      ..+.+..++.+.++|..+.+.|
</pre><pre>T Consensus       151 ~~~~v~i~a~~~~~~~g~~v~i~~~~~~~v~g~~~~~~~  189 (189)
</pre><pre>T 6U9G_B          151 KSLNNSIKADVNLNAEGLDVNIKGSVTASIKGSAATMVG  189 (189)
</pre><pre>T ss_dssp             CCCSCCCCCC-----------------------------
</pre><pre>T ss_pred             eccceEEEecceEEEEEeeEEEeccceeEEEeeeeEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>d3hn3a3 c.1.8.3 (A:329-632) beta-Glucuronidase, domain 3 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=28.87  E-value=1.8e+02  Score=18.17  Aligned_cols=16  Identities=25%  Similarity=0.420  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +++.+++||+++.+.|
</pre><pre>T Consensus         4 ~~~~~~~~g~~~~~~G   19 (304)
</pre><pre>T d3hn3a3           4 TKSQFLINGKPFYFHG   19 (304)
</pre><pre>T ss_dssp             CSSCEEETTEEECEEE
</pre><pre>T ss_pred             cCCeEEECCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>5H5T_A Flagellar hook-associated protein 2; Bacterial flagellar cap protein, STRUCTURAL PROTEIN; 2.5A {Salmonella typhimurium}
</pre><pre>Probab=28.84  E-value=2.3e+02  Score=18.85  Aligned_cols=29  Identities=21%  Similarity=0.444  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEEeccceE--EcCceeecC
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVVGSDHIQ--LDGKRIDLN   48 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~gs~~i~--~~~~~I~~n   48 (48)
</pre><pre>                      .|..+.|+|.++. ..+..+.  ++|-.++|+
</pre><pre>T Consensus       161 ~dA~~~inG~~~~-s~sN~~~d~i~Gvt~tl~  191 (204)
</pre><pre>T 5H5T_A          161 ENAKLNVNGIDIE-RQSNTVTDAPQGITLTLT  191 (204)
</pre><pre>T ss_dssp             BCEEEEETTEEEE-ESSSEECSTTTTEEEEEC
</pre><pre>T ss_pred             CCeEEEECCEEEE-ecCCeeeeCCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>5H5T_C Flagellar hook-associated protein 2; Bacterial flagellar cap protein, STRUCTURAL PROTEIN; 2.5A {Salmonella typhimurium}
</pre><pre>Probab=28.84  E-value=2.3e+02  Score=18.85  Aligned_cols=29  Identities=21%  Similarity=0.444  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEEeccceE--EcCceeecC
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVVGSDHIQ--LDGKRIDLN   48 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~gs~~i~--~~~~~I~~n   48 (48)
</pre><pre>                      .|..+.|+|.++. ..+..+.  ++|-.++|+
</pre><pre>T Consensus       161 ~dA~~~inG~~~~-s~sN~~~d~i~Gvt~tl~  191 (204)
</pre><pre>T 5H5T_C          161 ENAKLNVNGIDIE-RQSNTVTDAPQGITLTLT  191 (204)
</pre><pre>T ss_dssp             BCEEEEETTEEEE-ESSSEECSTTTTEEEEEC
</pre><pre>T ss_pred             CCeEEEECCEEEE-ecCCeeeeCCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>6ED2_A Glycosyl hydrolase family 2, TIM barrel domain protein; glycosyl hydrolase, HYDROLASE; HET: FMT; 2.3A {Faecalibacterium prausnitzii A2-165} SCOP: b.18.1.0, b.1.4.0, l.1.1.1, c.1.8.0
</pre><pre>Probab=28.74  E-value=4.2e+02  Score=19.51  Aligned_cols=30  Identities=23%  Similarity=0.431  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|=..+.+. ++.+++||+++.+.|
</pre><pre>T Consensus       289 d~~~~~fG~r~~~~~-~~~~~~nG~~~~~~G  318 (631)
</pre><pre>T 6ED2_A          289 DEYLDRIGIRTFEIR-HGRFLLNGSPVYLRG  318 (631)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEeeeccEEEEEE-CCEEEECCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>PF14734.8 ; DUF4469 ; Domain of unknown function (DUF4469) with IG-like fold
</pre><pre>Probab=28.57  E-value=1.1e+02  Score=18.50  Aligned_cols=21  Identities=19%  Similarity=0.355  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             EEccCcEEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7   16 VMKKDGTITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus        16 ~m~kdG~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      .+++.+-+.|.|+.+.+.+++
</pre><pre>T Consensus        15 ~it~G~~~~I~G~~Lk~~~~d   35 (99)
</pre><pre>T A6LGM9_PARD8/1   15 HITPNAPAIIFGSTLLIKGDD   35 (99)
</pre><pre>T ss_pred             ccCCCCCEEEEEeeeEEeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>2JAD_A YELLOW FLUORESCENT PROTEIN GLUTAREDOXIN FUSION PROTEIN; YELLOW FLUORESCENT PROTEIN, ELECTRON TRANSPORT, REDOX- ACTIVE CENTER, YEAST, GRX1P, TRANSPORT, GLUTAREDOXIN; HET: SO4, PIA; 2.7A {AEQUOREA VICTORIA}
</pre><pre>Probab=28.38  E-value=1.8e+02  Score=21.09  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus        10 ~~~~~~~~~~g~~--n~~~f~~~g~g~   34 (362)
</pre><pre>T 2JAD_A           10 GVVPILVELDGDV--NGHKFSVSGEGE   34 (362)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>PF13109.8 ; AsmA_1 ; AsmA-like C-terminal region
</pre><pre>Probab=28.23  E-value=3.1e+02  Score=17.57  Aligned_cols=40  Identities=25%  Similarity=0.488  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      ...+.+-+..+.++.   ..+++..+.+.|++.++++...+++
</pre><pre>T Consensus       109 ~~~~~~~~g~~~~~~---~~~~~~~~~~~~~G~i~l~~~~ld~  148 (214)
</pre><pre>T Q7MRT1_WOLSU/6  109 EAEIGINDRYAVVER---FLLHGTSIDSEGSGVIELESSQIEF  148 (214)
</pre><pre>T ss_pred             EEEEEEcCCEEEEEE---EEEEcceEEEEEEEEEeCCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>3LYC_B Putative pectinase; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesion, CELL ADHESION; HET: MSE; 2.3A {Parabacteroides distasonis}
</pre><pre>Probab=28.11  E-value=3.2e+02  Score=17.74  Aligned_cols=36  Identities=22%  Similarity=0.261  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCcEEEEccCcEEEEcCe-----EEEEEeccceEEcC
</pre><pre>Q MBL0283831.1/7    7 TFTVGKSVLVMKKDGTITLNGK-----TIDVVGSDHIQLDG   42 (48)
</pre><pre>Q Consensus         7 ~l~cG~~~i~m~kdG~I~i~G~-----~I~i~gs~~i~~~~   42 (48)
</pre><pre>                      .+.|..-.|.+...|.|.++..     .+...|++.+.+++
</pre><pre>T Consensus       125 ~~~~~~l~i~~~~~g~i~~~~~~~~~l~~~~~g~g~i~~~g  165 (241)
</pre><pre>T 3LYC_B          125 TASFTNLKIEIAGSGDFVGHKVYCEELNGDMAGSNTIVLGG  165 (241)
</pre><pre>T ss_dssp             CEEEEEEEEEEESSCEEEEEEEEEEEEEEEEESSCEEEEEE
</pre><pre>T ss_pred             eEEecceEEEEeCcceEEEeeEEEeEEEEEEecceEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>3LYC_G Putative pectinase; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesion, CELL ADHESION; HET: MSE; 2.3A {Parabacteroides distasonis}
</pre><pre>Probab=28.11  E-value=3.2e+02  Score=17.74  Aligned_cols=36  Identities=22%  Similarity=0.261  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCcEEEEccCcEEEEcCe-----EEEEEeccceEEcC
</pre><pre>Q MBL0283831.1/7    7 TFTVGKSVLVMKKDGTITLNGK-----TIDVVGSDHIQLDG   42 (48)
</pre><pre>Q Consensus         7 ~l~cG~~~i~m~kdG~I~i~G~-----~I~i~gs~~i~~~~   42 (48)
</pre><pre>                      .+.|..-.|.+...|.|.++..     .+...|++.+.+++
</pre><pre>T Consensus       125 ~~~~~~l~i~~~~~g~i~~~~~~~~~l~~~~~g~g~i~~~g  165 (241)
</pre><pre>T 3LYC_G          125 TASFTNLKIEIAGSGDFVGHKVYCEELNGDMAGSNTIVLGG  165 (241)
</pre><pre>T ss_dssp             CEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
</pre><pre>T ss_pred             eEEecceEEEEeCcceEEEeeEEEeEEEEEEecceEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>6WDR_Q 40S ribosomal protein S16-A; 80S-like, Tsr1, Structural heterogeneity, RIBOSOME; 3.7A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=28.03  E-value=1.3e+02  Score=19.01  Aligned_cols=18  Identities=39%  Similarity=0.593  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcc-CcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   13 SVLVMKK-DGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        13 ~~i~m~k-dG~I~i~G~~I   30 (48)
</pre><pre>                      ++..+.+ .|.|+|||+++
</pre><pre>T Consensus        16 A~v~~~~G~G~i~INg~~~   34 (127)
</pre><pre>T 6WDR_Q           16 AVAHVKAGKGLIKVNGSPI   34 (127)
</pre><pre>T ss_dssp             EEEEEECSEEEEEESSSBT
</pre><pre>T ss_pred             EEEEEEcCCCeEEECCEEH
</pre><pre>
</pre><pre>
</pre><pre>No 146
</pre><pre>>d4he4a1 d.22.1.0 (A:2-234) automated matches {Phialidium sp. [TaxId: 258839]}
</pre><pre>Probab=27.97  E-value=1.6e+02  Score=21.58  Aligned_cols=29  Identities=24%  Similarity=0.455  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7    7 TFTVGKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus         7 ~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .|--|...+.+.-+|.|  ||..+.|.|.+.
</pre><pre>T Consensus         5 ~lf~~~m~~k~~~eG~v--Ngh~F~i~GeG~   33 (233)
</pre><pre>T d4he4a1           5 LLFHGKIPYVVEMEGNV--DGHTFSIRGKGY   33 (233)
</pre><pre>T ss_dssp             GGGSSCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccccCCccEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 147
</pre><pre>>d4jkla3 c.1.8.0 (A:273-596) automated matches {Streptococcus agalactiae [TaxId: 208435]}
</pre><pre>Probab=27.93  E-value=1.7e+02  Score=18.63  Aligned_cols=16  Identities=38%  Similarity=0.520  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +++.++++|+++.+.|
</pre><pre>T Consensus         4 ~~~~~~~~g~~~~~~G   19 (324)
</pre><pre>T d4jkla3           4 TNGQFLLNRKPIYFKG   19 (324)
</pre><pre>T ss_dssp             ETTEEEETTEEBCEEE
</pre><pre>T ss_pred             eCCEEEECCEEEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 148
</pre><pre>>d5ltra_ d.22.1.0 (A:) automated matches {Amphioxus (Branchiostoma lanceolatum) [TaxId: 7740]}
</pre><pre>Probab=27.92  E-value=1.7e+02  Score=21.16  Aligned_cols=23  Identities=30%  Similarity=0.468  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             cEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   13 SVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        13 ~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +.+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         7 ~~~k~~meG~v--Ngh~F~i~GeG~   29 (219)
</pre><pre>T d5ltra_           7 ATHELHIFGSI--NGVDFDMVGQGT   29 (219)
</pre><pre>T ss_dssp             CEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 149
</pre><pre>>3OBA_B Beta-galactosidase; TIM Barrel, Tetramer, GH2, Glycosidase, Hydrolase; HET: GOL; 2.75A {Kluyveromyces lactis}
</pre><pre>Probab=27.84  E-value=3.7e+02  Score=22.13  Aligned_cols=30  Identities=47%  Similarity=0.624  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.++ ++.+++||+++.+.|
</pre><pre>T Consensus       329 ~~~~~~~GfR~v~~~-~~~~~lNG~~~~l~G  358 (1032)
</pre><pre>T 3OBA_B          329 QSIKHHVGFRQVELK-DGNITVNGKDILFRG  358 (1032)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEeeEEEEE-CCEEEECCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 150
</pre><pre>>3OBA_C Beta-galactosidase; TIM Barrel, Tetramer, GH2, Glycosidase, Hydrolase; HET: GOL; 2.75A {Kluyveromyces lactis}
</pre><pre>Probab=27.84  E-value=3.7e+02  Score=22.13  Aligned_cols=30  Identities=47%  Similarity=0.624  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.++ ++.+++||+++.+.|
</pre><pre>T Consensus       329 ~~~~~~~GfR~v~~~-~~~~~lNG~~~~l~G  358 (1032)
</pre><pre>T 3OBA_C          329 QSIKHHVGFRQVELK-DGNITVNGKDILFRG  358 (1032)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEEeeEEEEE-CCEEEECCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 151
</pre><pre>>3AI5_A yeast enhanced green fluorescent protein,Ubiquitin; green fluorescent protein, ubiquitin, fusion protein, Fluorescent Protein, transcription; HET: CR2, EDO; 1.4A {Aequorea victoria}
</pre><pre>Probab=27.68  E-value=1.8e+02  Score=20.21  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        13 ~~~~~~~~~~g~~--n~~~f~~~g~g~   37 (307)
</pre><pre>T 3AI5_A           13 GVVPILVELDGDV--NGHKFSVSGEGE   37 (307)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCccEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 152
</pre><pre>>4G6T_B Type III effector HopA1; Chaperone Effector, secretion chaperone, CHAPERONE; 1.56A {Pseudomonas syringae pv. tomato}
</pre><pre>Probab=27.52  E-value=1.6e+02  Score=18.60  Aligned_cols=19  Identities=21%  Similarity=0.622  Sum_probs=0.0  Template_Neff=3.300
</pre><pre>
</pre><pre>Q ss_pred             EccCcEEEEcCeEEEEEec
</pre><pre>Q MBL0283831.1/7   17 MKKDGTITLNGKTIDVVGS   35 (48)
</pre><pre>Q Consensus        17 m~kdG~I~i~G~~I~i~gs   35 (48)
</pre><pre>                      ++..|.|.++|+.-.|..+
</pre><pre>T Consensus         4 L~~hG~leV~Gk~Y~I~~a   22 (82)
</pre><pre>T 4G6T_B            4 LKANGQLEVDGKRYEIRAA   22 (82)
</pre><pre>T ss_dssp             CCSCEEEEETTEEEEEEEC
</pre><pre>T ss_pred             HHHcCeEEECCEEEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 153
</pre><pre>>1P4P_A Outer surface protein B; INTERMOLECULAR BETA SHEET, EXTENDED BETA SHEET, MEMBRANE PROTEIN; 2.0A {Borrelia burgdorferi} SCOP: b.76.1.1
</pre><pre>Probab=27.49  E-value=2.6e+02  Score=19.50  Aligned_cols=25  Identities=32%  Similarity=0.454  Sum_probs=0.0  Template_Neff=2.500
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCc---EEEEccCcEEEEcCeE
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKS---VLVMKKDGTITLNGKT   29 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~---~i~m~kdG~I~i~G~~   29 (48)
</pre><pre>                      .+++..-..   .|++.+||+|+++-.+
</pre><pre>T Consensus        94 TLTIs~nsKKtkdlVF~~~~TITvQ~Yd  121 (145)
</pre><pre>T 1P4P_A           94 TLTISADSKKTKDLVFLTDGTITVQQYN  121 (145)
</pre><pre>T ss_dssp             EEEEEETTEEEEEEEECTTSCEEEEEBC
</pre><pre>T ss_pred             eEEEEeCCcceeeEEEecCCeEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 154
</pre><pre>>d4dkma_ d.22.1.0 (A:) automated matches {Branchiostoma floridae [TaxId: 7739]}
</pre><pre>Probab=27.40  E-value=1.7e+02  Score=21.09  Aligned_cols=23  Identities=35%  Similarity=0.530  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             cEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   13 SVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        13 ~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         3 ~~~k~~meG~V--NGh~F~i~GeG~   25 (216)
</pre><pre>T d4dkma_           3 TTHEVHVYGSI--NGVEFDLVGSGK   25 (216)
</pre><pre>T ss_dssp             CEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 155
</pre><pre>>6NCW_C Beta-galacturonidase; glycoside hydrolase family 2, beta-galacturonidase, HYDROLASE; HET: GOL; 2.1A {Eisenbergiella tayi}
</pre><pre>Probab=27.38  E-value=4e+02  Score=19.45  Aligned_cols=30  Identities=30%  Similarity=0.443  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|-.++.+..+. +++||+++.+.|
</pre><pre>T Consensus       245 d~~~~~~G~r~~~~~~~~-~~lnG~p~~~~G  274 (574)
</pre><pre>T 6NCW_C          245 DDIIDRVGFREIRTEGKD-ILLNGRKLRIKG  274 (574)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETTE-EEETTEEBCEEE
</pre><pre>T ss_pred             eEEEEEecCEEEEEECCE-EEECCEEeCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 156
</pre><pre>>d2p4ma_ d.22.1.1 (A:) automated matches {Coral (Montipora efflorescens) [TaxId: 105610]}
</pre><pre>Probab=27.30  E-value=1.8e+02  Score=21.08  Aligned_cols=22  Identities=27%  Similarity=0.474  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-||+|  ||..+.|.|.+.
</pre><pre>T Consensus         9 ~~k~~meG~V--Ngh~F~i~GeG~   30 (221)
</pre><pre>T d2p4ma_           9 TYKVYMSGTV--NGHYFEVEGDGK   30 (221)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 157
</pre><pre>>3BGA_A Beta-galactosidase; Beta-Galactosidase, NYSGXRC, Protein Structure Initiative II (PSI-II), Glycosyl hydrolase family 2, Jelly-roll fold, Immunoglobulin-like fold, Tim-barrel; HET: MSE; 2.1A {Bacteroides thetaiotaomicron VPI-5482}
</pre><pre>Probab=27.24  E-value=4.3e+02  Score=21.84  Aligned_cols=30  Identities=30%  Similarity=0.410  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       327 d~~~~~~GfR~v~~~-~~~~~lNG~~~~l~G  356 (1010)
</pre><pre>T 3BGA_A          327 ELTGCEVGFRTSEIK-DGRFCINGVPVLVKG  356 (1010)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEccEEEEEE-CCEEEECCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 158
</pre><pre>>d2wura_ d.22.1.1 (A:) automated matches {Jellyfish (Aequorea victoria) [TaxId: 6100]}
</pre><pre>Probab=27.14  E-value=1.9e+02  Score=21.15  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|.|  ||..+.|.|.+.
</pre><pre>T Consensus         8 ~~m~~k~~~eG~v--Ngh~F~i~GeG~   32 (230)
</pre><pre>T d2wura_           8 GVVPILVELDGDV--NGHKFSVSGEGE   32 (230)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CcceEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 159
</pre><pre>>6JZ2_A Beta-glucuronidase; b-glucuronidase, HYDROLASE; HET: SJ5, MRD, MPD; 1.29A {Ruminococcus gnavus} SCOP: b.1.4.0, c.1.8.0, b.18.1.0
</pre><pre>Probab=26.99  E-value=4.8e+02  Score=19.37  Aligned_cols=30  Identities=20%  Similarity=0.341  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+.+.|-.++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       292 d~~~~~~G~r~~~~~-~~~~~lnG~~~~~~G  321 (627)
</pre><pre>T 6JZ2_A          292 DEYREKIGIRTVRIE-GTKILLNDRPVYLKG  321 (627)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEECCEEEEEE-CcEEEECCEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 160
</pre><pre>>1JZ8_D Beta-Galactosidase; TIM BARREL (ALPHA/BETA BARREL), JELLY-ROLL BARREL, IMMUNOGLOBULIN, BETA SUPERSANDWICH, HYDROLASE; HET: GAL, DMS, MG, BGC, NA; 1.5A {Escherichia coli} SCOP: b.1.4.1, c.1.8.3, b.18.1.5, b.30.5.1
</pre><pre>Probab=26.96  E-value=4.1e+02  Score=21.86  Aligned_cols=30  Identities=27%  Similarity=0.470  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=-++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       324 d~~~~~~GfR~v~~~-~~~~~lNG~~~~l~G  353 (1023)
</pre><pre>T 1JZ8_D          324 EAEACDVGFREVRIE-NGLLLLNGKPLLIRG  353 (1023)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEEEeceEEEEE-CCEEEECCEEEeEee
</pre><pre>
</pre><pre>
</pre><pre>No 161
</pre><pre>>d4jgea_ d.22.1.0 (A:) automated matches {Amphioxus (Branchiostoma lanceolatum) [TaxId: 7740]}
</pre><pre>Probab=26.93  E-value=1.8e+02  Score=21.07  Aligned_cols=23  Identities=30%  Similarity=0.525  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             cEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   13 SVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        13 ~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +.+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         5 ~~~k~~meG~V--Ngh~F~i~GeG~   27 (219)
</pre><pre>T d4jgea_           5 ATHDLHISGSI--NGHEFDLEGSGK   27 (219)
</pre><pre>T ss_dssp             CEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 162
</pre><pre>>PF07873.13 ; YabP ; YabP family
</pre><pre>Probab=26.80  E-value=2.3e+02  Score=15.71  Aligned_cols=26  Identities=12%  Similarity=0.376  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             EEeCCcEEEEc-cCcEEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7    8 FTVGKSVLVMK-KDGTITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus         8 l~cG~~~i~m~-kdG~I~i~G~~I~i~   33 (48)
</pre><pre>                      +.+-...+.+. ++|.+.|.|+++.+.
</pre><pre>T Consensus        20 ~~~~~~~I~l~~~~~~l~I~G~~L~i~   46 (66)
</pre><pre>T R6PCP1_9FIRM/8   20 ISFDEENIVAQTTMGVLIIRGDKLHIN   46 (66)
</pre><pre>T ss_pred             EEEcCCEEEEEECCcEEEEEcCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 163
</pre><pre>>4OHS_D FAR-RED FLUORESCENT PROTEIN AQ143; far-red, Beta Barrel, Red Fluorescent Protein, RFP, FLUORESCENT PROTEIN; HET: CH6, NRQ; 2.19A {Actinia Equina} SCOP: d.22.1.0
</pre><pre>Probab=26.73  E-value=1.8e+02  Score=21.03  Aligned_cols=33  Identities=27%  Similarity=0.505  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      |.-..+--+...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus         8 ~~~~~~~~~~m~~k~~meG~V--Ngh~F~i~GeG~   40 (237)
</pre><pre>T 4OHS_D            8 GSGAPLVTEDMCIKMTMEGTI--NGHHFKCVGEGE   40 (237)
</pre><pre>T ss_dssp             ----CCCCSSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCCCccCCCCceEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 164
</pre><pre>>4OHS_E FAR-RED FLUORESCENT PROTEIN AQ143; far-red, Beta Barrel, Red Fluorescent Protein, RFP, FLUORESCENT PROTEIN; HET: CH6, NRQ; 2.19A {Actinia Equina} SCOP: d.22.1.0
</pre><pre>Probab=26.73  E-value=1.8e+02  Score=21.03  Aligned_cols=33  Identities=27%  Similarity=0.505  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      |.-..+--+...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus         8 ~~~~~~~~~~m~~k~~meG~V--Ngh~F~i~GeG~   40 (237)
</pre><pre>T 4OHS_E            8 GSGAPLVTEDMCIKMTMEGTI--NGHHFKCVGEGE   40 (237)
</pre><pre>T ss_dssp             ----CCSCSSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCCCccCCCCceEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 165
</pre><pre>>d3piba1 d.22.1.1 (A:2-229) automated matches {Sea anemone (Entacmaea quadricolor) [TaxId: 6118]}
</pre><pre>Probab=26.70  E-value=1.8e+02  Score=21.12  Aligned_cols=22  Identities=14%  Similarity=0.311  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         9 ~~k~~meG~V--Ngh~F~i~GeG~   30 (228)
</pre><pre>T d3piba1           9 HMKLYMEGTV--NNHHFKCTSEGE   30 (228)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 166
</pre><pre>>d4dkna_ d.22.1.0 (A:) automated matches {Branchiostoma floridae [TaxId: 7739]}
</pre><pre>Probab=26.66  E-value=1.9e+02  Score=20.94  Aligned_cols=23  Identities=30%  Similarity=0.567  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             cEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   13 SVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        13 ~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +++.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         4 ~~~k~~meG~v--Ngh~F~i~GeG~   26 (218)
</pre><pre>T d4dkna_           4 ATHDIHLHGSI--NGHEFDMVGGGK   26 (218)
</pre><pre>T ss_dssp             CEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 167
</pre><pre>>6GP1_A Green to red photoconvertible GFP-like protein EosFP; Fluorescent protein; HET: IEY, NFA; 1.504A {Lobophyllia hemprichii}
</pre><pre>Probab=26.62  E-value=2e+02  Score=18.51  Aligned_cols=22  Identities=18%  Similarity=0.486  Sum_probs=0.0  Template_Neff=4.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=+|.|  ||..+.+.|.+.
</pre><pre>T Consensus        42 ~~k~~meG~V--NGh~F~i~GeG~   63 (94)
</pre><pre>T 6GP1_A           42 RIKLRMEGNV--NGHHFVIDGDGT   63 (94)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEeE--CCEEeEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 168
</pre><pre>>d1nvpc_ b.56.1.1 (C:) Large chain TOA1, C-terminal domain {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=26.59  E-value=1.6e+02  Score=16.12  Aligned_cols=15  Identities=40%  Similarity=0.636  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i~   33 (48)
</pre><pre>                      +||-+.|+|+++.+.
</pre><pre>T Consensus        25 k~gi~~i~gkd~~F~   39 (47)
</pre><pre>T d1nvpc_          25 KDGIMNLNGRDYIFS   39 (47)
</pre><pre>T ss_dssp             EEEEEEETTEEEEEE
</pre><pre>T ss_pred             cceEEEeCCeeecee
</pre><pre>
</pre><pre>
</pre><pre>No 169
</pre><pre>>2ZO6_A CYAN-EMITTING GFP-LIKE PROTEIN, KUSABIRA-CYAN (KCY); GFP-LIKE PROTEIN, LUMINESCENT PROTEIN, Structural Genomics, RIKEN Structural Genomics/Proteomics Initiative, RSGI, NPPSFA, National Project on Protein; HET: GYS; 1.4A {Fungia concinna} SCOP: d.22.1.0
</pre><pre>Probab=26.48  E-value=2.1e+02  Score=21.00  Aligned_cols=33  Identities=18%  Similarity=0.251  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+.+.-+|+|  ||+.+.+.|.+.-+-......++
</pre><pre>T Consensus        40 ~~k~~meG~V--NGh~F~i~GeG~G~p~~G~~~~~   72 (252)
</pre><pre>T 2ZO6_A           40 KMKYFMDGSV--NGHEFTVEGEGTGKPYEGKHKIT   72 (252)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEEGGGTEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEEecCCceEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 170
</pre><pre>>7CMG_A Por secretion system protein porM/gldM; Type IX secretion system, trans-periplasmic core complex, PROTEIN TRANSPORT; 3.7A {Porphyromonas gingivalis ATCC 33277}
</pre><pre>Probab=26.45  E-value=1.6e+02  Score=22.61  Aligned_cols=42  Identities=14%  Similarity=0.156  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCc
</pre><pre>Q MBL0283831.1/7    2 AGDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGK   43 (48)
</pre><pre>Q Consensus         2 aG~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~   43 (48)
</pre><pre>                      .|+.++..+.=+...=...-.|+++|+++.+.|.+.+++.+.
</pre><pre>T Consensus       250 ~G~~~~a~i~l~~~~~~~~p~i~ing~~i~~~G~~~~~~~~~  291 (533)
</pre><pre>T 7CMG_A          250 SGDTYKANIVLSSVDTTQRPDVFVNGKLLSPENMGLFTATAG  291 (533)
</pre><pre>T ss_dssp             TTSEEEEEEEEEEECSSCCCEEEETTEECCGGGSSEEEEECC
</pre><pre>T ss_pred             CCCcEEEEEEeecCCCCCCCeEEECCEEcCCCCcEEEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 171
</pre><pre>>5XYI_Q 40S ribosomal protein S16, putative; Trichomonas vaginalis ribosome, rRNA, rprotein, RIBOSOME; 3.35A {Trichomonas vaginalis}
</pre><pre>Probab=26.39  E-value=1.1e+02  Score=19.47  Aligned_cols=12  Identities=50%  Similarity=0.852  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCeEEE
</pre><pre>Q MBL0283831.1/7   20 DGTITLNGKTID   31 (48)
</pre><pre>Q Consensus        20 dG~I~i~G~~I~   31 (48)
</pre><pre>                      .|.|+|||+++.
</pre><pre>T Consensus        23 ~G~i~INg~~l~   34 (140)
</pre><pre>T 5XYI_Q           23 SGLIKLNGTPIN   34 (140)
</pre><pre>T ss_dssp             SSEEEETTEETT
</pre><pre>T ss_pred             CceEEECCEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 172
</pre><pre>>6ECA_B Beta-glucuronidase; Glycosyl hydrolase, HYDROLASE; HET: GOL; 2.853A {Lactobacillus rhamnosus} SCOP: b.18.1.0, b.1.4.0, c.1.8.0
</pre><pre>Probab=26.36  E-value=4.3e+02  Score=19.57  Aligned_cols=30  Identities=27%  Similarity=0.368  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|-.++.+. ++.+.+||+++.+.|
</pre><pre>T Consensus       292 d~~~~~~G~r~v~~~-~~~~~lnG~~~~~~G  321 (627)
</pre><pre>T 6ECA_B          292 DTYTDRIGIRTIKIS-GTDILVNDKPIYLKG  321 (627)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             EEEEeceeCEEEEEe-CCeeEECCEEEeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 173
</pre><pre>>d2ejha_ d.22.1.0 (A:) automated matches {Fungia concinna [TaxId: 496660]}
</pre><pre>Probab=26.35  E-value=2.1e+02  Score=20.70  Aligned_cols=25  Identities=24%  Similarity=0.383  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ....+.+.-||+|  ||..+.|.|.+.
</pre><pre>T Consensus         6 ~~mk~k~~meG~v--Ngh~F~i~GeG~   30 (218)
</pre><pre>T d2ejha_           6 PEMKMKYFMDGSV--NGHEFTVEGEGT   30 (218)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCceEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 174
</pre><pre>>1YQ2_F beta-galactosidase; GLYCOSYL HYDROLASE FAMILY 2  TIM BARREL  HEXAMER, HYDROLASE; HET: NA, SO4, PEG; 1.9A {Arthrobacter sp. C2-2} SCOP: b.1.4.1, c.1.8.3, b.18.1.5, b.30.5.1
</pre><pre>Probab=26.33  E-value=4.1e+02  Score=21.93  Aligned_cols=31  Identities=19%  Similarity=0.386  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      |+..+..+|=-.+.+. ++.+++||+++.+.|
</pre><pre>T Consensus       302 ~d~~~~~~GfR~v~~~-~~~~~lNG~~~~l~G  332 (1024)
</pre><pre>T 1YQ2_F          302 AESISVRLGFRTVRIV-GDQFLVNGRRVVFHG  332 (1024)
</pre><pre>T ss_dssp             TEEEEEEECCCCEEEE-TTEEEETTEECCEEE
</pre><pre>T ss_pred             CcEEEEEEeeEEEEEE-CCEEEECCEEEceee
</pre><pre>
</pre><pre>
</pre><pre>No 175
</pre><pre>>4DKM_F Amphioxus Green Fluorescent Protein, GFPc1a; fluorescent protein, beta-can, chromophore; HET: CR2; 1.95A {Branchiostoma floridae} SCOP: d.22.1.0
</pre><pre>Probab=26.31  E-value=2.2e+02  Score=20.10  Aligned_cols=33  Identities=24%  Similarity=0.322  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+.+.=+|.|  ||..+.+.|.+.-+-......+.
</pre><pre>T Consensus         4 ~~~~~~~G~V--NGh~F~i~GeG~G~p~~G~~~~~   36 (214)
</pre><pre>T 4DKM_F            4 THEVHVYGSI--NGVEFDLVGSGKGNPKDGSEEIQ   36 (214)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEEecCCCceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 176
</pre><pre>>d2dd7a_ d.22.1.0 (A:) automated matches {Chiridius poppei [TaxId: 286301]}
</pre><pre>Probab=26.26  E-value=1.8e+02  Score=20.95  Aligned_cols=22  Identities=23%  Similarity=0.522  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ++.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         4 k~k~~meG~V--Ngh~F~i~GeG~   25 (217)
</pre><pre>T d2dd7a_           4 KIESRIHGNL--NGEKFELVGGGV   25 (217)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 177
</pre><pre>>PF02626.17 ; CT_A_B ; Carboxyltransferase domain, subdomain A and B
</pre><pre>Probab=26.22  E-value=4.3e+02  Score=18.63  Aligned_cols=37  Identities=27%  Similarity=0.467  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             EEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    6 ITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         6 ~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +|++.|+.++.+.++-.|.+-|-+..+.      +|++.+.+|
</pre><pre>T Consensus        30 lEi~~~g~~l~~~~~~~iavtGa~~~~~------ing~~~~~~   66 (258)
</pre><pre>T G2J1V3_PSEUL/2   30 LEIVLGPVQLRFLRDGWLALTGADFDAT------LDGAPVWCG   66 (258)
</pre><pre>T ss_pred             EEeecCceEEEEeCCeEEEeecCceEEE------ECCEEccCC
</pre><pre>
</pre><pre>
</pre><pre>No 178
</pre><pre>>6KL1_A Green fluorescent protein; green fluorescent protein, visualization of hydrogen, high resolution, FLUORESCENT PROTEIN; HET: DOD, CRO; 0.851A {Aequorea victoria} SCOP: d.22.1.1
</pre><pre>Probab=26.20  E-value=2e+02  Score=20.67  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus         9 ~~~~~k~~~eg~v--ng~~f~i~GeG~   33 (229)
</pre><pre>T 6KL1_A            9 GVVPILVELDGDV--NGHKFSVSGEGE   33 (229)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 179
</pre><pre>>6ED1_A Glycosyl hydrolase family 2, sugar binding domain protein; Glycosyl hydrolase, HYDROLASE; 2.9A {Bacteroides dorei}
</pre><pre>Probab=26.14  E-value=4.9e+02  Score=20.51  Aligned_cols=30  Identities=27%  Similarity=0.385  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.++.+ ..++||+++.+.|
</pre><pre>T Consensus       291 d~~~~~~G~r~~~~~~~-~~~lNG~~~~l~G  320 (879)
</pre><pre>T 6ED1_A          291 QEVISPVGFRWFSVDKT-GFYLNGKYLKLRG  320 (879)
</pre><pre>T ss_dssp             EEEEEEECBCCEEEETT-EEEETTEECCEEE
</pre><pre>T ss_pred             EEEEEeeccEEEEEcCC-eeEECCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 180
</pre><pre>>6ED1_B Glycosyl hydrolase family 2, sugar binding domain protein; Glycosyl hydrolase, HYDROLASE; 2.9A {Bacteroides dorei}
</pre><pre>Probab=26.14  E-value=4.9e+02  Score=20.51  Aligned_cols=30  Identities=27%  Similarity=0.385  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|=.++.++.+ ..++||+++.+.|
</pre><pre>T Consensus       291 d~~~~~~G~r~~~~~~~-~~~lNG~~~~l~G  320 (879)
</pre><pre>T 6ED1_B          291 QEVISPVGFRWFSVDKT-GFYLNGKYLKLRG  320 (879)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEETT-EEEETTEEECEEE
</pre><pre>T ss_pred             EEEEEeeccEEEEEcCC-eeEECCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 181
</pre><pre>>d4q9wa1 d.22.1.0 (A:2-218) automated matches {Clavularia sp. [TaxId: 86521]}
</pre><pre>Probab=26.05  E-value=1.9e+02  Score=20.85  Aligned_cols=22  Identities=23%  Similarity=0.492  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         8 ~~k~~meG~V--NGh~F~i~GeG~   29 (215)
</pre><pre>T d4q9wa1           8 KIKLKMEGNV--NGYAFVIEGEGE   29 (215)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 182
</pre><pre>>d2hpwa_ d.22.1.1 (A:) Green fluorescent protein, GFP {Clytia gregaria [TaxId: 27801]}
</pre><pre>Probab=25.99  E-value=1.9e+02  Score=21.05  Aligned_cols=25  Identities=12%  Similarity=0.294  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-||.|  ||..+.|.|.+.
</pre><pre>T Consensus         9 ~~m~~k~~meG~v--Ngh~F~i~GeG~   33 (231)
</pre><pre>T d2hpwa_           9 KEIPYITELEGDV--EGMKFIIKGEGT   33 (231)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CcccEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 183
</pre><pre>>4V3P_SI 40S ribosomal protein S16; RIBOSOME; 34.0A {Triticum aestivum}
</pre><pre>Probab=25.92  E-value=1.4e+02  Score=18.64  Aligned_cols=16  Identities=31%  Similarity=0.430  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             EEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   15 LVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        15 i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .....+|.|+|||+++
</pre><pre>T Consensus         4 ~~~~G~G~i~INg~~~   19 (126)
</pre><pre>T 4V3P_SI           4 YCKPGRGLIKVNGVPI   19 (126)
</pre><pre>T ss_dssp             CCCCCCCSSSEECCCC
</pre><pre>T ss_pred             EEECCCceEEECCEEH
</pre><pre>
</pre><pre>
</pre><pre>No 184
</pre><pre>>6BYC_A Beta-mannosidase; CARBOHYDRATE, HYDROLASE-CARBOHYDRATE complex; HET: ACT, GOL, PEG; 1.897A {Xanthomonas axonopodis pv. citri (strain 306)}
</pre><pre>Probab=25.88  E-value=3.6e+02  Score=21.47  Aligned_cols=33  Identities=21%  Similarity=0.384  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCc-------EEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDG-------TITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG-------~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +..+.+.|=-++.+..+.       .+++||+++.+.|..
</pre><pre>T Consensus       309 d~~~~~~GfR~~~~~~~~~~~~~~~~~~lNG~~~~~~G~~  348 (862)
</pre><pre>T 6BYC_A          309 QQIKRVTGLRSVELRREKDRFGKSMEIVINGIPIFAKGAN  348 (862)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEECCEETTEECCEEEETTEEECEEEEE
</pre><pre>T ss_pred             EEEEEEEEeeEEEEeeecCCCCCeEEEEECCEEEeeceec
</pre><pre>
</pre><pre>
</pre><pre>No 185
</pre><pre>>d1p4pa_ b.76.1.1 (A:) Outer surface protein B {Lyme disease spirochete (Borrelia burgdorferi) [TaxId: 139]}
</pre><pre>Probab=25.63  E-value=2.9e+02  Score=19.19  Aligned_cols=25  Identities=32%  Similarity=0.454  Sum_probs=0.0  Template_Neff=2.500
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCc---EEEEccCcEEEEcCeE
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKS---VLVMKKDGTITLNGKT   29 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~---~i~m~kdG~I~i~G~~   29 (48)
</pre><pre>                      .++++....   .|++.+||+|+++-.+
</pre><pre>T Consensus        89 TLTIs~nsKKtk~lVF~~~~TITvQ~Yd  116 (140)
</pre><pre>T d1p4pa_          89 TLTISADSKKTKDLVFLTDGTITVQQYN  116 (140)
</pre><pre>T ss_dssp             EEEEEETTEEEEEEEECTTSCEEEEEBC
</pre><pre>T ss_pred             eEEEEeCCcceeeEEEEcCCeEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 186
</pre><pre>>PF12791.9 ; RsgI_N ; Anti-sigma factor N-terminus
</pre><pre>Probab=25.63  E-value=1.9e+02  Score=15.22  Aligned_cols=14  Identities=29%  Similarity=0.517  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEE
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTIT   24 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~   24 (48)
</pre><pre>                      ++..++|++||...
</pre><pre>T Consensus         5 ~~~~iVlT~~G~F~   18 (50)
</pre><pre>T Q81YQ6_BACAN/7    5 KHSVVVLTPNGEFI   18 (50)
</pre><pre>T ss_pred             CCEEEEEcCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 187
</pre><pre>>d2g6ya_ d.22.1.0 (A:) automated matches {Pontellina plumata [TaxId: 239963]}
</pre><pre>Probab=25.59  E-value=2e+02  Score=20.85  Aligned_cols=22  Identities=27%  Similarity=0.540  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ++.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         3 k~k~~meG~v--Ngh~F~i~GeG~   24 (216)
</pre><pre>T d2g6ya_           3 EIECRITGTL--NGVEFELVGGGE   24 (216)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 188
</pre><pre>>d1xssa_ d.22.1.0 (A:) automated matches {Favia favus [TaxId: 102203]}
</pre><pre>Probab=25.52  E-value=2e+02  Score=20.99  Aligned_cols=22  Identities=18%  Similarity=0.534  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=||+|  ||..+.|.|.+.
</pre><pre>T Consensus         9 ~~~~~meG~V--NGh~F~i~GeG~   30 (226)
</pre><pre>T d1xssa_           9 KMELRMEGAV--NGHKFVITGKGS   30 (226)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 189
</pre><pre>>4W6T_A fluorescent protein E115H/T118H; fluorescent protein; HET: EDO, CRO; 1.604A {synthetic construct}
</pre><pre>Probab=25.43  E-value=2.2e+02  Score=20.47  Aligned_cols=25  Identities=28%  Similarity=0.521  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus        11 ~~~~~k~~~~g~v--ng~~f~i~G~G~   35 (229)
</pre><pre>T 4W6T_A           11 GVVPILIELDGDV--NGHKFFVRGEGE   35 (229)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             cceEEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 190
</pre><pre>>7BWN_J Chimera of Green fluorescent protein and p53dim; Heterocatenane, RECOMBINATION; 2.396A {Aequorea victoria}
</pre><pre>Probab=25.17  E-value=2.2e+02  Score=20.87  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus        10 ~~~~~~~~~~g~v--ng~~f~i~G~G~   34 (280)
</pre><pre>T 7BWN_J           10 GVVPILVELDGDV--NGHKFSVRGEGE   34 (280)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCeEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 191
</pre><pre>>7BWN_O Chimera of Green fluorescent protein and p53dim; Heterocatenane, RECOMBINATION; 2.396A {Aequorea victoria}
</pre><pre>Probab=25.17  E-value=2.2e+02  Score=20.87  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus        10 ~~~~~~~~~~g~v--ng~~f~i~G~G~   34 (280)
</pre><pre>T 7BWN_O           10 GVVPILVELDGDV--NGHKFSVRGEGE   34 (280)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCeEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 192
</pre><pre>>PF06097.13 ; DUF945 ; Bacterial protein of unknown function (DUF945)
</pre><pre>Probab=25.15  E-value=84  Score=22.46  Aligned_cols=14  Identities=29%  Similarity=0.767  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             cCcEEEEcCeEEEE
</pre><pre>Q MBL0283831.1/7   19 KDGTITLNGKTIDV   32 (48)
</pre><pre>Q Consensus        19 kdG~I~i~G~~I~i   32 (48)
</pre><pre>                      ++|.+++||+++.+
</pre><pre>T Consensus       463 ~~G~l~iNG~~l~l  476 (477)
</pre><pre>T H0PVZ9_9RHOO/1  463 RGGELKINGKPVNP  476 (477)
</pre><pre>T ss_pred             eCCEEEECCEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 193
</pre><pre>>1R6V_A subtilisin-like serine protease; subtilisin, sandwich domain, propeptide, HYDROLASE; 1.7A {Fervidobacterium pennivorans} SCOP: c.41.1.1
</pre><pre>Probab=25.09  E-value=1.6e+02  Score=23.30  Aligned_cols=14  Identities=64%  Similarity=1.011  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7   21 GTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus        21 G~I~i~G~~I~i~g   34 (48)
</pre><pre>                      |+++|||.+|-+.|
</pre><pre>T Consensus       623 g~~~~n~~~~~~~~  636 (671)
</pre><pre>T 1R6V_A          623 GTVTLNGEDIPVYG  636 (671)
</pre><pre>T ss_dssp             EEEEETTEEEEEEE
</pre><pre>T ss_pred             EEEEECCceeeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 194
</pre><pre>>5T2A_AK uS9; ribosome, Leishmania donovani; 2.9A {Leishmania donovani}
</pre><pre>Probab=25.09  E-value=1.4e+02  Score=19.43  Aligned_cols=18  Identities=28%  Similarity=0.466  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcc--CcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   13 SVLVMKK--DGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        13 ~~i~m~k--dG~I~i~G~~I   30 (48)
</pre><pre>                      ++..+.+  .|.|+|||.++
</pre><pre>T Consensus        23 A~v~l~~~GsG~i~INg~~l   42 (149)
</pre><pre>T 5T2A_AK          23 AVATVTKAAQCNIKVNGVPL   42 (149)
</pre><pre>T ss_dssp             EEECCCCCSSCCCEETTEEG
</pre><pre>T ss_pred             EEEEEEeCCcCeEEECCEEH
</pre><pre>
</pre><pre>
</pre><pre>No 195
</pre><pre>>4DKN_B AMPHIOXUS GREEN FLUORESCENT PROTEIN, GFPA1; CHROMOPHORE, BETA-CAN, FLUORESCENCE, FLUORESCENT PROTEIN; HET: CR2; 1.35A {Branchiostoma floridae} SCOP: d.22.1.0
</pre><pre>Probab=24.98  E-value=2.5e+02  Score=20.04  Aligned_cols=33  Identities=21%  Similarity=0.331  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+.+.-+|+|  ||+.+.+.|.+.-+-......++
</pre><pre>T Consensus        10 ~~k~~~eG~v--NGh~F~i~GeG~G~p~~G~~~~~   42 (221)
</pre><pre>T 4DKN_B           10 THDIHLHGSI--NGHEFDMVGGGKGDPNAGSLVTT   42 (221)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEEecCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 196
</pre><pre>>6D6W_B Beta-galactosidase/beta-glucuronidase; glycosyl hydrolase, beta-glucuronidase, HYDROLASE; HET: GOL, GCU; 1.8A {Bacteroides uniformis}
</pre><pre>Probab=24.97  E-value=4.6e+02  Score=19.31  Aligned_cols=30  Identities=27%  Similarity=0.289  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+..+|-.++.+. ++..++||+++.+.|
</pre><pre>T Consensus       289 ~~~~~~~g~r~~~~~-~~~~~lnG~~~~~~G  318 (603)
</pre><pre>T 6D6W_B          289 DKVSDEIGFRTIRTE-GIKILLNDKEIFCRG  318 (603)
</pre><pre>T ss_dssp             CEEEEEECCCCEEEE-TTEEEETTEEECEEE
</pre><pre>T ss_pred             CeEEeeEEeEEEEEc-CcEEEECCEEeeeee
</pre><pre>
</pre><pre>
</pre><pre>No 197
</pre><pre>>PF08845.12 ; SymE_toxin ; Toxin SymE, type I toxin-antitoxin system
</pre><pre>Probab=24.81  E-value=2e+02  Score=14.27  Aligned_cols=19  Identities=32%  Similarity=0.508  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCcEEEEcc
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKSVLVMKK   19 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~~i~m~k   19 (48)
</pre><pre>                      ..|+.+.+.+.+..+.+.+
</pre><pre>T Consensus        33 ~~g~~v~v~~~~~~i~i~~   51 (51)
</pre><pre>T Q832D6_ENTFA/1   33 SVGDKITVKCQQGQLTITK   51 (51)
</pre><pre>T ss_pred             CCCCEEEEEEECCEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 198
</pre><pre>>6HR1_B Myosin light chain kinase 2, skeletal/cardiac muscle,Unconventional myosin-X,Green fluorescent protein,Calmodulin-1; fluorescent engineered, STRUCTURAL PROTEIN, fusion protein; HET: TLA, GOL, EDO, CR2; 1.901A {Oryctolagus cuniculus}
</pre><pre>Probab=24.79  E-value=1.9e+02  Score=21.72  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      |-.+..|-.+|++  ||+.+.|+|.+.
</pre><pre>T Consensus        46 ~~~~~~~~~~~~~--~~~~~~~~g~g~   70 (434)
</pre><pre>T 6HR1_B           46 GVVPILVELDGDV--NGHKFSVSGEGE   70 (434)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             cCCceEEEEEEEE--CCeEEEEEEeee
</pre><pre>
</pre><pre>
</pre><pre>No 199
</pre><pre>>d1v58a2 d.17.3.1 (A:2-61) Thiol:disulfide interchange protein DsbG, N-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=24.75  E-value=1.6e+02  Score=15.92  Aligned_cols=21  Identities=19%  Similarity=0.415  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             EEEccCcEEEEcCeEEEEEec
</pre><pre>Q MBL0283831.1/7   15 LVMKKDGTITLNGKTIDVVGS   35 (48)
</pre><pre>Q Consensus        15 i~m~kdG~I~i~G~~I~i~gs   35 (48)
</pre><pre>                      +-+.+||...+.|.-+..+|.
</pre><pre>T Consensus        40 ~Y~t~dG~~~i~G~l~d~~g~   60 (60)
</pre><pre>T d1v58a2          40 IYLTPDGKHAISGYMYNEKGE   60 (60)
</pre><pre>T ss_dssp             EEECTTSSCEEESCCBCTTCC
</pre><pre>T ss_pred             EEECCCCCeEEEEEEEeCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 200
</pre><pre>>6WVD_A Green fluorescent protein, Protein jagunal homolog 1 chimera; JAGN1, Protein jagunal homolog 1, MEMBRANE PROTEIN; HET: CRO, OLC; 2.25A {Aequorea victoria}
</pre><pre>Probab=24.64  E-value=2.1e+02  Score=22.52  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      |...+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus        10 ~~~~~k~~~~G~v--nG~~F~i~GeG~   34 (420)
</pre><pre>T 6WVD_A           10 GVVPILVELDGDV--NGHKFSVRGEGE   34 (420)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             cCeeEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 201
</pre><pre>>PF14262.8 ; Cthe_2159 ; Carbohydrate-binding domain-containing protein Cthe_2159
</pre><pre>Probab=24.61  E-value=4.2e+02  Score=18.84  Aligned_cols=32  Identities=16%  Similarity=0.474  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCcEEEEccCcEEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKSVLVMKKDGTITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      |-.+++.+.+..  +..++.|+|+|=.+.+.+++
</pre><pre>T Consensus       259 gG~i~i~s~~d~--i~s~g~i~I~GG~i~i~~~d  290 (292)
</pre><pre>T H6NTR1_9BACL/9  259 GGTIRIDSADDA--VHSNNSVTIAGGELSITSGD  290 (292)
</pre><pre>T ss_pred             CcEEEEEcCCCC--eEeCCeEEEeCCEEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 202
</pre><pre>>6DDT_A Beta-mannosidase; mannosidase lysosomal glycosidase GH2, HYDROLASE; HET: FUC, NAG, EDO; 2.1A {Mus musculus}
</pre><pre>Probab=24.53  E-value=3.1e+02  Score=21.94  Aligned_cols=33  Identities=21%  Similarity=0.228  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcE---------EEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGT---------ITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~---------I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +..+...|=-++.+..++.         +++||+++.+.|..
</pre><pre>T Consensus       304 d~~~~~~GfR~i~~~~~~~~~~~g~~~~~~lNG~~~~~~G~~  345 (868)
</pre><pre>T 6DDT_A          304 IEKAAKVYFRTVQLIEEGIKGSPGLSFYFKINGLPIFLKGSN  345 (868)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEECCCBTTBCSCCCEEEETTEEECEEEEE
</pre><pre>T ss_pred             EEEEEEEEeeEEEEEeccCCCCCCceEEEEECCEEeeceEec
</pre><pre>
</pre><pre>
</pre><pre>No 203
</pre><pre>>3HN3_B Beta-glucuronidase; LYSOSOMAL ENZYME, ACID HYDROLASE, GLYCOSIDASE, Disease mutation, Glycoprotein, Hydrolase, Lysosome, Mucopolysaccharidosis; HET: BMA, MRD, NAG, GUP, MAN, MPD; 1.7A {Homo sapiens} SCOP: b.1.4.1, c.1.8.3, b.18.1.5
</pre><pre>Probab=24.51  E-value=4.6e+02  Score=19.27  Aligned_cols=30  Identities=27%  Similarity=0.413  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +....++|-.++.+. ++.+++||+++.+.|
</pre><pre>T Consensus       298 d~~~~~~G~r~~~~~-~~~~~~nG~~~~~~G  327 (613)
</pre><pre>T 3HN3_B          298 DFYTLPVGIRTVAVT-KSQFLINGKPFYFHG  327 (613)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEC-SSCEEETTEECCEEE
</pre><pre>T ss_pred             EEEEeceeeEEEEEc-CCeEEECCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 204
</pre><pre>>d3qdpa_ d.212.1.1 (A:) automated matches {Escherichia coli [TaxId: 83333]}
</pre><pre>Probab=24.45  E-value=1.5e+02  Score=16.08  Aligned_cols=11  Identities=18%  Similarity=0.398  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTIT   24 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~   24 (48)
</pre><pre>                      ++++++||.|.
</pre><pre>T Consensus        33 ~~~i~~~G~v~   43 (88)
</pre><pre>T d3qdpa_          33 RIKLAPDGMLL   43 (88)
</pre><pre>T ss_dssp             EEEECTTCCEE
</pre><pre>T ss_pred             EEEECCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 205
</pre><pre>>d1yzwa_ d.22.1.1 (A:) automated matches {Heteractis crispa [TaxId: 175771]}
</pre><pre>Probab=24.45  E-value=2.1e+02  Score=20.76  Aligned_cols=22  Identities=27%  Similarity=0.468  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-+|+|  ||..+.|.|.+.
</pre><pre>T Consensus         8 ~~k~~meG~V--NGh~F~i~GeG~   29 (221)
</pre><pre>T d1yzwa_           8 RIKMYMEGTV--NGHYFKCEGEGD   29 (221)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 206
</pre><pre>>4LQT_A Green fluorescent protein; GFP, Chemical biology, allosteric activation, switchable enzyme, chemical rescue, mutation, FLUORESCENT PROTEIN; HET: CRO, EDO; 1.1A {Aequorea victoria} SCOP: d.22.1.1, l.1.1.1
</pre><pre>Probab=24.44  E-value=2.3e+02  Score=20.69  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus        20 ~~~~~k~~~eg~v--ng~~f~i~G~G~   44 (246)
</pre><pre>T 4LQT_A           20 GVVPILVELDGDV--NGHKFSVRGEGE   44 (246)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 207
</pre><pre>>6L27_A Green fluorescent protein; GFP, mutant, recombinant, FLUORESCENT PROTEIN; HET: GYS, DOD; 0.77A {Aequorea victoria} SCOP: d.22.1.1
</pre><pre>Probab=24.44  E-value=2.3e+02  Score=20.34  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus         8 ~~~~~k~~~~g~v--ng~~f~i~G~G~   32 (227)
</pre><pre>T 6L27_A            8 GVVPILVELDGDV--NGHKFSVSGEGE   32 (227)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 208
</pre><pre>>2HPW_A Green fluorescent protein; Fluorescent protein, GFP, Structural Genomics, PSI, Protein Structure Initiative, Southeast Collaboratory for Structural Genomics, SECSG, LUMINESCENT; HET: GYS; 1.55A {Clytia gregaria} SCOP: d.22.1.1
</pre><pre>Probab=24.39  E-value=2.2e+02  Score=20.63  Aligned_cols=25  Identities=12%  Similarity=0.294  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +.....+.-+|.|  ||+.+.|.|.+.
</pre><pre>T Consensus        13 ~~~~~k~~~eG~V--NGh~F~i~GeG~   37 (233)
</pre><pre>T 2HPW_A           13 KEIPYITELEGDV--EGMKFIIKGEGT   37 (233)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccccEEEEEEEEE--CCeEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 209
</pre><pre>>6ITX_C Fiber protein; avian adenovirus, fiber protein, CELL ADHESION; HET: SO4, GLN; 2.75A {Duck adenovirus 1}
</pre><pre>Probab=24.14  E-value=2.8e+02  Score=21.02  Aligned_cols=41  Identities=10%  Similarity=0.288  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCc-eeecC
</pre><pre>Q MBL0283831.1/7    8 FTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGK-RIDLN   48 (48)
</pre><pre>Q Consensus         8 l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~-~I~~n   48 (48)
</pre><pre>                      +..|...|.+..++.+.+...-+.++.++.+.+... .++||
</pre><pre>T Consensus        28 ~q~~~~gl~v~~~~~l~~s~~~l~v~~~gPl~~t~~~GLtL~   69 (301)
</pre><pre>T 6ITX_C           28 QQMGRGSLNVETRGGLEKSDTGLKIKRAAPLSITSDGELTLA   69 (301)
</pre><pre>T ss_dssp             ------------------------------------------
</pre><pre>T ss_pred             ccccccceeeccCCceeEeCCceEEecCCCEEEcCCCCeEeE
</pre><pre>
</pre><pre>
</pre><pre>No 210
</pre><pre>>6ITX_F Fiber protein; avian adenovirus, fiber protein, CELL ADHESION; HET: GLN, SO4; 2.75A {Duck adenovirus 1}
</pre><pre>Probab=24.14  E-value=2.8e+02  Score=21.02  Aligned_cols=41  Identities=10%  Similarity=0.288  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCc-eeecC
</pre><pre>Q MBL0283831.1/7    8 FTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGK-RIDLN   48 (48)
</pre><pre>Q Consensus         8 l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~-~I~~n   48 (48)
</pre><pre>                      +..|...|.+..++.+.+...-+.++.++.+.+... .++||
</pre><pre>T Consensus        28 ~q~~~~gl~v~~~~~l~~s~~~l~v~~~gPl~~t~~~GLtL~   69 (301)
</pre><pre>T 6ITX_F           28 QQMGRGSLNVETRGGLEKSDTGLKIKRAAPLSITSDGELTLA   69 (301)
</pre><pre>T ss_dssp             ------------------------------------------
</pre><pre>T ss_pred             ccccccceeeccCCceeEeCCceEEecCCCEEEcCCCCeEeE
</pre><pre>
</pre><pre>
</pre><pre>No 211
</pre><pre>>6RM3_SQ0 uS9; Microsporidia, Ribosome, Intracellular Parasite; HET: MG; 3.4A {Vairimorpha necatrix}
</pre><pre>Probab=24.03  E-value=1.9e+02  Score=18.87  Aligned_cols=18  Identities=6%  Similarity=0.045  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcc-C-cEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   13 SVLVMKK-D-GTITLNGKTI   30 (48)
</pre><pre>Q Consensus        13 ~~i~m~k-d-G~I~i~G~~I   30 (48)
</pre><pre>                      ++..+.. + |.|+|||.++
</pre><pre>T Consensus        19 A~v~l~~g~~G~i~INg~~l   38 (155)
</pre><pre>T 6RM3_SQ0         19 ATCKCSESEEMSILVDKVPY   38 (155)
</pre><pre>T ss_dssp             EEEEEECCSSCEEEETTEEG
</pre><pre>T ss_pred             EEEEEECCCCeeEEECCEEH
</pre><pre>
</pre><pre>
</pre><pre>No 212
</pre><pre>>TIGR02856 spore_yqfC; sporulation protein YqfC. This small protein, designated YqfC in Bacillus subtilis, is both restricted to and universal in sporulating species of the Firmcutes, such as Bacillus subtilis and Clostridium perfringens.
</pre><pre>Probab=23.72  E-value=1.8e+02  Score=17.27  Aligned_cols=16  Identities=31%  Similarity=0.607  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             ccCcEEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7   18 KKDGTITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus        18 ~kdG~I~i~G~~I~i~   33 (48)
</pre><pre>                      .++|.+.|.|+.+.|.
</pre><pre>T Consensus        50 ~~~g~l~I~G~~L~i~   65 (85)
</pre><pre>T TIGR02856        50 STNGKITIEGKNFVIK   65 (85)
</pre><pre>T ss_pred             cCCCEEEEEccCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 213
</pre><pre>>6GJE_C Cubilin; receptor, vitamin B12, cubilin, amnionless, AMN, CUBN, kidney, reabsorption, proximal tubule, PROTEIN TRANSPORT; 2.3A {Homo sapiens}
</pre><pre>Probab=23.55  E-value=3.3e+02  Score=16.42  Aligned_cols=26  Identities=23%  Similarity=0.417  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCc---EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKS---VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~---~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      ++-+.++.+   ++.....|.|.+++.++
</pre><pre>T Consensus        25 ~L~~~~~~~~nI~~~~~~~g~i~~~~~dl   53 (110)
</pre><pre>T 6GJE_C           25 NLVFLTGSAQNIEFRTGSLGKIKLNDEDL   53 (110)
</pre><pre>T ss_dssp             EEEEECCTTCEEEEECCTTCCEEETTEEH
</pre><pre>T ss_pred             cEEEEcCCCceeEEEeCCCceEEeCCCCh
</pre><pre>
</pre><pre>
</pre><pre>No 214
</pre><pre>>PF16475.7 ; DUF5052 ; Domain of unknown function (DUF5052)
</pre><pre>Probab=23.50  E-value=5.2e+02  Score=18.63  Aligned_cols=36  Identities=33%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCc-EEEEcc--------CcE------------EEEcCeEEEEEeccce
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKS-VLVMKK--------DGT------------ITLNGKTIDVVGSDHI   38 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~-~i~m~k--------dG~------------I~i~G~~I~i~gs~~i   38 (48)
</pre><pre>                      |+.+.-.-|++ .|.-..        +|.            |+++|+++...|+.-|
</pre><pre>T Consensus        47 G~~i~~~~Gksi~I~~~~~~~~~~~~~G~~~~~~~~SsVi~itidGk~~~~~GstvI  103 (204)
</pre><pre>T R7GNX1_9FIRM/2   47 GNKVFETKGQKIKITGNKVKESSYNSDGSVKETYSLSSVVTITIDGNEIESCGDTCI  103 (204)
</pre><pre>T ss_pred             CCEEEEEEccEEEEccccceeeeccCCCCccccccccceEEEEECCeEEEEeCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 215
</pre><pre>>PF11739.10 ; DctA-YdbH ; Dicarboxylate transport
</pre><pre>Probab=23.47  E-value=2.8e+02  Score=18.34  Aligned_cols=42  Identities=17%  Similarity=0.234  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +++.+..-...+.+..+|...   -+++++|...--.++..|++|
</pre><pre>T Consensus       138 ~n~~y~~l~~~~~~~~~G~~~---l~~~l~G~n~~~~~~~pv~ln  179 (203)
</pre><pre>T A8DVF0_NEMVE/3  138 TDFHFDTLKVGIDYAPDGEMQ---LATQLSGKNPGWNSGHPVNLS  179 (203)
</pre><pre>T ss_pred             hccEEeEEEEEEeeCCCCcEE---EEEEEEeeCCCCcCCCCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 216
</pre><pre>>PF10618.11 ; Tail_tube ; Phage tail tube protein
</pre><pre>Probab=23.43  E-value=2.2e+02  Score=17.60  Aligned_cols=23  Identities=30%  Similarity=0.495  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCeEEEEEeccceEEcCce
</pre><pre>Q MBL0283831.1/7   22 TITLNGKTIDVVGSDHIQLDGKR   44 (48)
</pre><pre>Q Consensus        22 ~I~i~G~~I~i~gs~~i~~~~~~   44 (48)
</pre><pre>                      .|.++|+.+...++-.++..+..
</pre><pre>T Consensus         9 ~i~vdG~~~~~~~~~t~~~gg~~   31 (115)
</pre><pre>T M4RAY2_BIBTR/2    9 TIRFNGKEYPTDNDGSLDVGGKE   31 (115)
</pre><pre>T ss_pred             EEEECCEEEEeCCCeEEcCCCeE
</pre><pre>
</pre><pre>
</pre><pre>No 217
</pre><pre>>2KW1_A Fas apoptotic inhibitory molecule 1; beta-sheets, Apoptosis; NMR {Homo sapiens}
</pre><pre>Probab=23.33  E-value=3.3e+02  Score=16.21  Aligned_cols=29  Identities=24%  Similarity=0.453  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCc--EEEEccC-cEEEEcCeEEEEEec
</pre><pre>Q MBL0283831.1/7    7 TFTVGKS--VLVMKKD-GTITLNGKTIDVVGS   35 (48)
</pre><pre>Q Consensus         7 ~l~cG~~--~i~m~kd-G~I~i~G~~I~i~gs   35 (48)
</pre><pre>                      .+..++.  +++|.|| -.|.+||+.+.-.+.
</pre><pre>T Consensus        11 ~~~l~g~~~rIvL~K~tl~VwvNG~~vet~~~   42 (89)
</pre><pre>T 2KW1_A           11 VLHMDGENFRIVLEKDAMDVWCNGKKLETAGE   42 (89)
</pre><pre>T ss_dssp             EEEETTEEEEEEEETTTTEEEETTEEECCEEE
</pre><pre>T ss_pred             EEEECCeeEEEEEeeCCCEEEECCEEeecceE
</pre><pre>
</pre><pre>
</pre><pre>No 218
</pre><pre>>2DD7_A green fluorescent protein; FLUORESCENT PROTEIN, LUMINESCENT PROTEIN; HET: CXS, CR2; 1.9A {Chiridius poppei} SCOP: d.22.1.0
</pre><pre>Probab=23.32  E-value=2.7e+02  Score=19.68  Aligned_cols=32  Identities=22%  Similarity=0.493  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCce-eecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKR-IDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~-I~~n   48 (48)
</pre><pre>                      .+.+.=+|.|  ||+.+.+.|.+.- ..|.. +.++
</pre><pre>T Consensus         4 ~~k~~~~G~V--NGh~F~i~G~G~g-~~G~~~~~l~   36 (216)
</pre><pre>T 2DD7_A            4 KIESRIHGNL--NGEKFELVGGGVG-EEGRLEIEMK   36 (216)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEE-ETTEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEEe-cccEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 219
</pre><pre>>7K00_I 30S ribosomal protein S9; antibiotics, post-translational modifications, post-transcriptional modifications, RIBOSOME; HET: 2MG, 5MC, 2MA, PAR, 4OC, MEQ, OMC, OMU, PSU, MG, SPD, D2T, 1MG, G7M, UR3, SPM, 3TD, 4D4, 6MZ, 5MU, OMG, MA6, H2U; 1.98A {Escherichia coli}
</pre><pre>Probab=23.24  E-value=1e+02  Score=19.48  Aligned_cols=11  Identities=27%  Similarity=0.773  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   20 DGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        20 dG~I~i~G~~I   30 (48)
</pre><pre>                      +|.|+|||+++
</pre><pre>T Consensus        25 ~G~i~VNg~~~   35 (130)
</pre><pre>T 7K00_I           25 NGKIVINQRSL   35 (130)
</pre><pre>T ss_dssp             SSCEEETTEEH
</pre><pre>T ss_pred             CCeEEECCEEH
</pre><pre>
</pre><pre>
</pre><pre>No 220
</pre><pre>>6XXW_A Beta-glucuronidase; Glycoside hydrolase, Glucuronide, HYDROLASE; HET: TRS; 1.851A {Dictyoglomus thermophilum (strain ATCC 35947 / DSM 3960 / H-6-12)} SCOP: b.18.1.0, b.1.4.0, c.1.8.0
</pre><pre>Probab=23.20  E-value=5.5e+02  Score=18.81  Aligned_cols=30  Identities=17%  Similarity=0.275  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccCcEEEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKDGTITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kdG~I~i~G~~I~i~g   34 (48)
</pre><pre>                      +..+...|-.++.++ ++.+++||+++.+.|
</pre><pre>T Consensus       274 d~~~~~~g~r~v~~~-~~~~~~nG~~~~l~G  303 (598)
</pre><pre>T 6XXW_A          274 DIYRMDIGIRTVRVE-GKQFLINEKPFYFKG  303 (598)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEE-TTEEEETTEEBCEEE
</pre><pre>T ss_pred             eEEEEeeCCeEEEEC-CeEEEECCeeEEeEe
</pre><pre>
</pre><pre>
</pre><pre>No 221
</pre><pre>>d2c9ia_ d.22.1.1 (A:) automated matches {Sea anemone (Anemonia sulcata) [TaxId: 6108]}
</pre><pre>Probab=23.20  E-value=2.3e+02  Score=20.68  Aligned_cols=22  Identities=14%  Similarity=0.356  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=+|+|  ||..+.|.|.+.
</pre><pre>T Consensus        10 ~~k~~meG~V--NGh~F~i~GeG~   31 (228)
</pre><pre>T d2c9ia_          10 RVQLSMEGSV--NYHAFKCTGKGE   31 (228)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 222
</pre><pre>>PF06905.15 ; FAIM1 ; Fas apoptotic inhibitory molecule (FAIM1)
</pre><pre>Probab=23.11  E-value=4.4e+02  Score=17.60  Aligned_cols=29  Identities=24%  Similarity=0.476  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCc--EEEEccC-cEEEEcCeEEEEEec
</pre><pre>Q MBL0283831.1/7    7 TFTVGKS--VLVMKKD-GTITLNGKTIDVVGS   35 (48)
</pre><pre>Q Consensus         7 ~l~cG~~--~i~m~kd-G~I~i~G~~I~i~gs   35 (48)
</pre><pre>                      .+..++.  +|+|.++ -.|++||+.+.-.+.
</pre><pre>T Consensus        96 ~~~l~g~~~rIvl~k~tl~VwvnG~~vet~~~  127 (172)
</pre><pre>T B3S9C3_TRIAD/6   96 AFMLTGEPTRVVFEKDSLDVWVNGEKLNTAGE  127 (172)
</pre><pre>T ss_pred             EEEeCCccEEEEEecCCeEEEECCEEeeceeE
</pre><pre>
</pre><pre>
</pre><pre>No 223
</pre><pre>>2C9I_B GREEN FLUORESCENT PROTEIN ASFP499; FLUORESCENT PROTEIN, BETA-BARREL, BIOLUMINESCENCE, LUMINESCENCE, LUMINESCENT PROTEIN; HET: CRQ; 1.82A {ANEMONIA SULCATA} SCOP: d.22.1.1
</pre><pre>Probab=22.99  E-value=2.6e+02  Score=19.96  Aligned_cols=33  Identities=15%  Similarity=0.200  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+.+.=+|+|  ||+.+.+.|.+.-+.......++
</pre><pre>T Consensus        10 ~~~~~~~G~V--Ngh~F~i~G~G~G~p~~G~~~~~   42 (226)
</pre><pre>T 2C9I_B           10 RVQLSMEGSV--NYHAFKCTGKGEGKPYEGTQSLN   42 (226)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEEccCCccEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 224
</pre><pre>>2C9I_D GREEN FLUORESCENT PROTEIN ASFP499; FLUORESCENT PROTEIN, BETA-BARREL, BIOLUMINESCENCE, LUMINESCENCE, LUMINESCENT PROTEIN; HET: CRQ; 1.82A {ANEMONIA SULCATA} SCOP: d.22.1.1
</pre><pre>Probab=22.99  E-value=2.6e+02  Score=19.96  Aligned_cols=33  Identities=15%  Similarity=0.200  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      .+.+.=+|+|  ||+.+.+.|.+.-+.......++
</pre><pre>T Consensus        10 ~~~~~~~G~V--Ngh~F~i~G~G~G~p~~G~~~~~   42 (226)
</pre><pre>T 2C9I_D           10 RVQLSMEGSV--NYHAFKCTGKGEGKPYEGTQSLN   42 (226)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEEccCCccEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 225
</pre><pre>>4HE4_A Yellow fluorescent protein; phiYFP, yellow fluorescent protein, mutant variant of phiYFP, Thr-Tyr-Gly chromophore, beta-barrel, Biomarker, Thr-Tyr-Gly bicyclic chromophore, FLUORESCENT; HET: CRO; 2.05A {Phialidium sp. SL-2003} SCOP: l.1.1.1, d.22.1.0
</pre><pre>Probab=22.97  E-value=2.6e+02  Score=20.30  Aligned_cols=25  Identities=28%  Similarity=0.574  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|.|  ||+.+.+.|.+.
</pre><pre>T Consensus        10 ~~m~~k~~meG~V--NGh~F~i~GeG~   34 (244)
</pre><pre>T 4HE4_A           10 GKIPYVVEMEGNV--DGHTFSIRGKGY   34 (244)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCCeEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 226
</pre><pre>>d2ib5a_ d.22.1.1 (A:) automated matches {Cnidopus japonicus [TaxId: 380086]}
</pre><pre>Probab=22.95  E-value=2.4e+02  Score=20.70  Aligned_cols=22  Identities=14%  Similarity=0.231  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-||.|  ||..+.|.|.+.
</pre><pre>T Consensus         6 ~~k~~meG~V--Ngh~F~i~GeG~   27 (228)
</pre><pre>T d2ib5a_           6 RIKLYMEGTV--NNHHFMCEAEGE   27 (228)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 227
</pre><pre>>PF06236.13 ; MelC1 ; Tyrosinase co-factor MelC1
</pre><pre>Probab=22.91  E-value=2.2e+02  Score=18.59  Aligned_cols=12  Identities=17%  Similarity=0.468  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCeEEEEE
</pre><pre>Q MBL0283831.1/7   22 TITLNGKTIDVV   33 (48)
</pre><pre>Q Consensus        22 ~I~i~G~~I~i~   33 (48)
</pre><pre>                      .++|+|++|++.
</pre><pre>T Consensus        59 ~v~IDGr~Lhvm   70 (109)
</pre><pre>T K4RFH5_STRDJ/1   59 GVFVDGVELHVM   70 (109)
</pre><pre>T ss_pred             EEEECCEEeeEE
</pre><pre>
</pre><pre>
</pre><pre>No 228
</pre><pre>>6S68_A Aequorea cf. australis fluorescent protein 2 (AausFP2); GFP-like fluorescent protein, FLUORESCENT PROTEIN; HET: PIA; 2.06A {Aequorea australis}
</pre><pre>Probab=22.90  E-value=3e+02  Score=19.71  Aligned_cols=36  Identities=22%  Similarity=0.447  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +...+.+.-+|.|  ||+.+.+.|.+.-+-......++
</pre><pre>T Consensus        10 ~~m~~k~~~eG~V--Ngh~F~i~GeG~g~p~~G~~~~~   45 (230)
</pre><pre>T 6S68_A           10 GKIPFVVELEGDV--NGEVFSVRGNGWGDGSTGKLEIK   45 (230)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             CCccEEEEEEEEE--CCEEEEEEEEEEecCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 229
</pre><pre>>TIGR03028 EpsE; polysaccharide export protein EpsE. Sequences in this family of proteins are members of a polysaccharide export protein family (pfam02563) which includes the wza protein from E.
</pre><pre>Probab=22.87  E-value=4e+02  Score=17.76  Aligned_cols=25  Identities=28%  Similarity=0.392  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEeCCc-----EEEEccCcEEEE
</pre><pre>Q MBL0283831.1/7    1 TAGDSITFTVGKS-----VLVMKKDGTITL   25 (48)
</pre><pre>Q Consensus         1 ~aG~~~~l~cG~~-----~i~m~kdG~I~i   25 (48)
</pre><pre>                      ..||.+.+.+-+.     ...+..||+|++
</pre><pre>T Consensus         5 ~~gD~l~i~v~~~~~~~~~~~V~~dG~i~~   34 (239)
</pre><pre>T TIGR03028         5 GPGDVLRITVFQQPDLTTDTRVSESGSITF   34 (239)
</pre><pre>T ss_pred             CCCCEEEEEeCCCCCCCCCEEECCCceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 230
</pre><pre>>3AI4_A yeast enhanced green fluorescent protein,DNA polymerase iota; UBM, Ubiquitin-binding motif, GFP, Fusion, Fluorescent Protein, replication; HET: CR2; 1.6A {Aequorea victoria}
</pre><pre>Probab=22.86  E-value=2.6e+02  Score=20.88  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.|.|.+.
</pre><pre>T Consensus        13 ~~~~~k~~~~g~v--ng~~F~i~GeG~   37 (283)
</pre><pre>T 3AI4_A           13 GVVPILVELDGDV--NGHKFSVSGEGE   37 (283)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 231
</pre><pre>>3RD4_B uncharacterized protein; Structural Genomics, PSI-Biology, Northeast Structural Genomics Consortium, NESG, PROPEN_03304, Unknown function; 2.9A {Proteus penneri}
</pre><pre>Probab=22.79  E-value=1.6e+02  Score=18.11  Aligned_cols=28  Identities=14%  Similarity=0.341  Sum_probs=0.0  Template_Neff=6.800
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEeCCc---EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7    3 GDSITFTVGKS---VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus         3 G~~~~l~cG~~---~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      |+.+++.|...   .|....-|..+.+|..+
</pre><pre>T Consensus        49 G~~~ef~V~~~~Y~~L~eGd~G~LtyqG~rf   79 (100)
</pre><pre>T 3RD4_B           49 RKDLVLNIDKSSYQNIEPEMKGRLFMQGSRF   79 (100)
</pre><pre>T ss_dssp             CEEEEEEECHHHHTTCCTTCEEEEEEETTEE
</pre><pre>T ss_pred             CceEEEEEcHHHcccCCCCCEEEEEEeCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 232
</pre><pre>>PF06739.13 ; SBBP ; Beta-propeller repeat
</pre><pre>Probab=22.75  E-value=1.7e+02  Score=12.77  Aligned_cols=14  Identities=7%  Similarity=0.143  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcC
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNG   27 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G   27 (48)
</pre><pre>                      .+.++++|.+++-|
</pre><pre>T Consensus        17 ~i~~d~~g~~~v~G   30 (37)
</pre><pre>T Q8F8X3_LEPIN/4   17 KIGIDFNRNLYTTG   30 (37)
</pre><pre>T ss_pred             eEEEcCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 233
</pre><pre>>2C9J_A GREEN FLUORESCENT PROTEIN FP512; FLUORESCENT PROTEIN, BETA-BARREL, BIOLUMINESCENCE, LUMINESCENCE, LUMINESCENT PROTEIN; HET: CRQ; 1.35A {CERIANTHUS MEMBRANACEUS} SCOP: d.22.1.0
</pre><pre>Probab=22.52  E-value=3.3e+02  Score=19.42  Aligned_cols=32  Identities=13%  Similarity=0.201  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      .+.+.=+|.|  ||+.+.+.|.+.-+.......+
</pre><pre>T Consensus         9 ~~~~~~~G~V--Ngh~F~i~GeG~g~p~~G~~~~   40 (223)
</pre><pre>T 2C9J_A            9 SVSVYMKGNV--NNHEFEYDGIGGGDPNSGQFSL   40 (223)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEeecCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 234
</pre><pre>>6WVE_A Green fluorescent protein,Chicken Signal Peptidase Complex Subunit 1,Green fluorescent protein; Signal Peptidase Complex Subunit 1(SPCS1), membrane protein; HET: CRO; 2.429A {Aequorea victoria}
</pre><pre>Probab=22.34  E-value=2.6e+02  Score=21.10  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        10 ~~~~~k~~~eg~v--ng~~F~i~GeG~   34 (312)
</pre><pre>T 6WVE_A           10 GVVPILVELDGDV--NGHKFSVRGEGE   34 (312)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 235
</pre><pre>>6TH6_Al 30S ribosomal protein S9; T. kodakarensis, ac4C, Ribosome, cryo-EM; HET: OMC, 6MZ, LV2, 5MU, OMU, OMG, MA6, 4AC, A2M, 5MC, ZN, LHH, 4SU; 2.55A {Thermococcus kodakarensis (strain ATCC BAA-918 / JCM 12380 / KOD1)}
</pre><pre>Probab=22.24  E-value=1.2e+02  Score=19.31  Aligned_cols=11  Identities=27%  Similarity=0.649  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   20 DGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        20 dG~I~i~G~~I   30 (48)
</pre><pre>                      +|.|+|||+++
</pre><pre>T Consensus        23 ~G~i~INg~~l   33 (135)
</pre><pre>T 6TH6_Al          23 KGRVRINHKPV   33 (135)
</pre><pre>T ss_dssp             SSCEEETTEEG
</pre><pre>T ss_pred             CceEEECCEEH
</pre><pre>
</pre><pre>
</pre><pre>No 236
</pre><pre>>d2qali1 d.14.1.1 (I:3-129) Ribosomal protein S9 {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=22.14  E-value=1.8e+02  Score=18.85  Aligned_cols=16  Identities=19%  Similarity=0.522  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEE
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTI   30 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I   30 (48)
</pre><pre>                      .+. ...|.|+|||+++
</pre><pre>T Consensus        17 ~l~-~G~G~i~VNg~~l   32 (127)
</pre><pre>T d2qali1          17 FIK-PGNGKIVINQRSL   32 (127)
</pre><pre>T ss_pred             EEE-CCCCeEEECCEeH
</pre><pre>
</pre><pre>
</pre><pre>No 237
</pre><pre>>TIGR04411 T2SS_GspN_Lepto; type II secretion system protein N, Leptospira/Geobacter-type. Members of this family are the N (or GspN) protein of type II secretion systems (T2SS) as found in Leptospira, Geobacter, Myxococcus, and several other genera.
</pre><pre>Probab=22.01  E-value=4.4e+02  Score=17.26  Aligned_cols=41  Identities=17%  Similarity=0.277  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCcEEEEccCcEEEEcCeEEEEEeccceEEcC----ceeecC
</pre><pre>Q MBL0283831.1/7    5 SITFTVGKSVLVMKKDGTITLNGKTIDVVGSDHIQLDG----KRIDLN   48 (48)
</pre><pre>Q Consensus         5 ~~~l~cG~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~----~~I~~n   48 (48)
</pre><pre>                      ...+.+.+..+.++.   +.+++..+.+.+++.++++.    ..++++
</pre><pre>T Consensus       192 ~~~~~~~~~~l~i~~---l~~~~~~~~~~~~G~~~l~~~~~~~~~dl~  236 (279)
</pre><pre>T TIGR04411       192 RGRIRLKKGKATVER---LQLDGPDAYLRLTGTIRLSPPLEFSRLNLK  236 (279)
</pre><pre>T ss_pred             EEEEEEECCEEEEEE---EEEeCCCeEEEEEEEEEcCCCCccceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 238
</pre><pre>>2RH7_C Green fluorescent protein; fluorescent protein; HET: CRO; 1.5A {Renilla reniformis} SCOP: d.22.1.1
</pre><pre>Probab=21.98  E-value=2.6e+02  Score=20.24  Aligned_cols=25  Identities=8%  Similarity=0.170  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        10 ~~m~~k~~meG~V--NGh~F~i~GeG~   34 (239)
</pre><pre>T 2RH7_C           10 EVMPTKINLEGLV--GDHAFSMEGVGE   34 (239)
</pre><pre>T ss_dssp             SEEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCccEEEEEEEEE--CCEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 239
</pre><pre>>6U1A_A Red fluorescent protein; fluorescent protein, alternative routes of post-translational modifications, hydrolysis; HET: PQ4, NRQ; 1.09A {Entacmaea quadricolor} SCOP: d.22.1.1
</pre><pre>Probab=21.92  E-value=2.5e+02  Score=20.23  Aligned_cols=22  Identities=14%  Similarity=0.314  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        11 ~~k~~~~G~v--Ngh~F~i~GeG~   32 (232)
</pre><pre>T 6U1A_A           11 PMKLYMEGTV--NNHHFKCTSEGE   32 (232)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 240
</pre><pre>>4CVU_A BETA-MANNOSIDASE; HYDROLASE; HET: BMA, CD, MAN, NAG, PG4; 1.9A {TRICHODERMA HARZIANUM}
</pre><pre>Probab=21.81  E-value=4.7e+02  Score=21.43  Aligned_cols=33  Identities=18%  Similarity=0.307  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEeCCcEEEEccC-------------c---EEEEcCeEEEEEecc
</pre><pre>Q MBL0283831.1/7    4 DSITFTVGKSVLVMKKD-------------G---TITLNGKTIDVVGSD   36 (48)
</pre><pre>Q Consensus         4 ~~~~l~cG~~~i~m~kd-------------G---~I~i~G~~I~i~gs~   36 (48)
</pre><pre>                      +..+.++|=-++.+..+             |   .+.+||++|.+.|..
</pre><pre>T Consensus       340 d~~~~~~G~R~i~~~~~~~~~~~~~~~~~~g~~f~f~vNG~~i~~~G~n  388 (942)
</pre><pre>T 4CVU_A          340 LVSQRRVGFRTILFSSGNITDAQIASGITPGNNWHFEINGHEFYAKGAN  388 (942)
</pre><pre>T ss_dssp             EEEEEEECCCCEEEECSBCCHHHHHHTCCSBBCCEEEETTEECCEEEEE
</pre><pre>T ss_pred             EEEEEEEEeEEEEEEcccCChhHHhcCCCCCCcEEEEECCEEeeeceee
</pre><pre>
</pre><pre>
</pre><pre>No 241
</pre><pre>>d2gw3a1 d.22.1.0 (A:1-220) automated matches {Trachyphyllia geoffroyi [TaxId: 196280]}
</pre><pre>Probab=21.79  E-value=3e+02  Score=20.12  Aligned_cols=25  Identities=16%  Similarity=0.373  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      ...++.+.-+|.|  ||..+.|.|.+.
</pre><pre>T Consensus         6 ~~mk~k~~meG~v--ngh~F~i~GeG~   30 (220)
</pre><pre>T d2gw3a1           6 PEMKIKLLMEGNV--NGHQFVIEGDGK   30 (220)
</pre><pre>T ss_dssp             SSEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CCceEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 242
</pre><pre>>d4ohsa_ d.22.1.0 (A:) automated matches {European sea anemone (Actinia equina) [TaxId: 6106]}
</pre><pre>Probab=21.65  E-value=2.6e+02  Score=20.37  Aligned_cols=22  Identities=36%  Similarity=0.649  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      .+.+.=+|+|  ||.++.|.|.+.
</pre><pre>T Consensus        10 ~~k~~meG~V--NGh~F~i~GeG~   31 (225)
</pre><pre>T d4ohsa_          10 CIKMTMEGTI--NGHHFKCVGEGE   31 (225)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 243
</pre><pre>>3IR8_A Large stokes shift fluorescent protein; beta barrel, FLUORESCENT PROTEIN; HET: CRQ; 1.63A {Montipora sp. 20} SCOP: d.22.1.1
</pre><pre>Probab=21.65  E-value=3.7e+02  Score=19.20  Aligned_cols=32  Identities=19%  Similarity=0.226  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      .+.+.=+|.|  ||+.+.+.|.+.-+.......+
</pre><pre>T Consensus        11 ~~~~~~~G~V--NGh~F~v~G~G~G~~~~G~~~~   42 (221)
</pre><pre>T 3IR8_A           11 TYKVYMSGTV--NGHYFEVEGDGKGKPYEGEQTV   42 (221)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEE
</pre><pre>T ss_pred             EEEEEEEEEE--CCEEEEEEEEEEeccCccEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 244
</pre><pre>>4HN7_A Signal transduction protein pmrD; Signaling protein; 2.352A {Escherichia coli}
</pre><pre>Probab=21.65  E-value=3.8e+02  Score=16.36  Aligned_cols=22  Identities=18%  Similarity=0.132  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEccCcEEEEc-CeEEEEE
</pre><pre>Q MBL0283831.1/7   12 KSVLVMKKDGTITLN-GKTIDVV   33 (48)
</pre><pre>Q Consensus        12 ~~~i~m~kdG~I~i~-G~~I~i~   33 (48)
</pre><pre>                      ++.|+-.+||.+.|| |+.-.++
</pre><pre>T Consensus        43 Gd~L~p~~~g~~~IN~~~~~~i~   65 (88)
</pre><pre>T 4HN7_A           43 GDLLSPLQNALYCINREKLHTVK   65 (88)
</pre><pre>T ss_dssp             TCEEEEEETTEEEETTEEEEEEE
</pre><pre>T ss_pred             CCEEEEcCCeeEEEeCCCceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 245
</pre><pre>>4Q9W_B GFP-like fluorescent chromoprotein cFP484; beta barrel, fluorescent protein, artificial metalloenzyme, metalloenzyme, bioconjugation, protein design, thermostable, temperature stable, solvent stable, robust; HET: PIA; 1.0A {Clavularia sp.} SCOP: l.1.1.1, d.22.1.0
</pre><pre>Probab=21.51  E-value=3.5e+02  Score=19.20  Aligned_cols=32  Identities=16%  Similarity=0.242  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEccCcEEEEcCeEEEEEeccceEEcCceeec
</pre><pre>Q MBL0283831.1/7   14 VLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDL   47 (48)
</pre><pre>Q Consensus        14 ~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~   47 (48)
</pre><pre>                      .+.+.=+|.|  ||+.+.+.|.+.-+.......+
</pre><pre>T Consensus         9 ~~~~~~~G~V--NG~~F~i~G~G~g~p~~G~~~~   40 (218)
</pre><pre>T 4Q9W_B            9 KIKLKMEGNV--NGYAFVIEGEGEGKPYDGTNTI   40 (218)
</pre><pre>T ss_dssp             EEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEE
</pre><pre>T ss_pred             eEEEEEEEEE--CCEEEEEEEEEEecCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 246
</pre><pre>>PF04872.15 ; Pox_L5 ; Poxvirus L5 protein family
</pre><pre>Probab=21.42  E-value=1.3e+02  Score=18.58  Aligned_cols=28  Identities=7%  Similarity=0.152  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             EEEeCCcEEEEc--cCcE----EEEcCeEEEEEe
</pre><pre>Q MBL0283831.1/7    7 TFTVGKSVLVMK--KDGT----ITLNGKTIDVVG   34 (48)
</pre><pre>Q Consensus         7 ~l~cG~~~i~m~--kdG~----I~i~G~~I~i~g   34 (48)
</pre><pre>                      +|.|-+..+.|.  .++.    ..+||++|.+..
</pre><pre>T Consensus        23 ~L~C~gn~L~I~~lp~~~~~~ALslng~PI~~~~   56 (78)
</pre><pre>T Q6TUU9_YMTV5/5   23 FLFCKENKMMIGGLPNNITTEALSVNKLPIVIDN   56 (78)
</pre><pre>T ss_pred             eeeeeCCeeEeecCCCCCcccchhhCCCCccccC
</pre><pre>
</pre><pre>
</pre><pre>No 247
</pre><pre>>3EVP_A Green fluorescent protein,Green fluorescent protein; circular-permutated, EGFP, Chromophore, Luminescence, Photoprotein, SIGNALING PROTEIN; HET: CRO; 1.453A {Aequorea victoria}
</pre><pre>Probab=21.40  E-value=2.7e+02  Score=20.28  Aligned_cols=25  Identities=28%  Similarity=0.589  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus       111 ~~~~~k~~~~g~v--ng~~F~i~GeG~  135 (243)
</pre><pre>T 3EVP_A          111 GVVPILVELDGDV--NGHKFSVSGEGE  135 (243)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             ccceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 248
</pre><pre>>6B7T_A Green fluorescent protein,Green fluorescent protein; Superfolder GFP, Truncated GFP, His-tag, FLUORESCENT PROTEIN; HET: GYS; 1.91A {Aequorea victoria}
</pre><pre>Probab=21.37  E-value=2.9e+02  Score=20.27  Aligned_cols=25  Identities=28%  Similarity=0.575  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccc
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDH   37 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~   37 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.
</pre><pre>T Consensus        61 ~~~~~k~~~eG~v--ng~~F~v~GeG~   85 (246)
</pre><pre>T 6B7T_A           61 GVVPILVELDGDV--NGHKFSVRGEGE   85 (246)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEE
</pre><pre>T ss_pred             CcceEEEEEEEEE--CCEEEEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 249
</pre><pre>>6S67_B Aequorea cf. australis fluorescent protein 1 (AausFP1); GFP-like fluorescent protein, FLUORESCENT PROTEIN; HET: CRO, GOL; 2.47A {Aequorea australis}
</pre><pre>Probab=21.36  E-value=3.3e+02  Score=19.50  Aligned_cols=36  Identities=22%  Similarity=0.392  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.-+-.-....+.
</pre><pre>T Consensus        10 ~~m~~k~~~eG~V--Ngh~F~i~GeG~G~p~~G~~~~~   45 (230)
</pre><pre>T 6S67_B           10 EKIPYVVEMEGDV--EGMKFSVRGKGHGDANTGKIEAS   45 (230)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             CCCcEEEEEEEEE--CCEEEEEEEEEEecCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 250
</pre><pre>>6S67_D Aequorea cf. australis fluorescent protein 1 (AausFP1); GFP-like fluorescent protein, FLUORESCENT PROTEIN; HET: GOL, CRO; 2.47A {Aequorea australis}
</pre><pre>Probab=21.36  E-value=3.3e+02  Score=19.50  Aligned_cols=36  Identities=22%  Similarity=0.392  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEccCcEEEEcCeEEEEEeccceEEcCceeecC
</pre><pre>Q MBL0283831.1/7   11 GKSVLVMKKDGTITLNGKTIDVVGSDHIQLDGKRIDLN   48 (48)
</pre><pre>Q Consensus        11 G~~~i~m~kdG~I~i~G~~I~i~gs~~i~~~~~~I~~n   48 (48)
</pre><pre>                      +...+.+.-+|+|  ||+.+.+.|.+.-+-.-....+.
</pre><pre>T Consensus        10 ~~m~~k~~~eG~V--Ngh~F~i~GeG~G~p~~G~~~~~   45 (230)
</pre><pre>T 6S67_D           10 EKIPYVVEMEGDV--EGMKFSVRGKGHGDANTGKIEAS   45 (230)
</pre><pre>T ss_dssp             SCEEEEEEEEEEE--TTEEEEEEEEEEEETTTTEEEEE
</pre><pre>T ss_pred             CCCcEEEEEEEEE--CCEEEEEEEEEEecCCCcEEEEE
</pre><pre>
</pre><pre>
</pre>