<pre>Query         FD01847867_02109 
</pre><pre>Match_columns 133
</pre><pre>No_of_seqs    87 out of 117
</pre><pre>Neff          3.62345
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Tue Aug 24 15:42:47 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/6906908.hhr -oa3m ../results/6906908.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF20148.1 ; DUF6531 ; Domain o  99.2 4.4E-11 4.1E-16   72.4   4.2   55   73-131     1-56  (75)
</pre><pre>  2 6P0X_B Salmonella plasmid viru  70.4      10 9.4E-05   32.9   2.8   45   75-124     6-51  (333)
</pre><pre>  3 d1bf2a2 b.71.1.1 (A:638-750) I  60.7      20 0.00019   25.8   2.5   27   86-112    50-77  (113)
</pre><pre>  4 6OX6_A Tas1; type VI secretion  58.0      42 0.00039   30.4   4.4   53   10-62    134-195 (439)
</pre><pre>  5 PF03534.15 ; SpvB ; Salmonella  53.8      28 0.00026   29.3   2.5   30   93-124    14-45  (289)
</pre><pre>  6 3F1X_A Serine acetyltransferas  48.0      32  0.0003   30.4   2.1   24    5-28    268-291 (310)
</pre><pre>  7 7BW9_A Serine acetyltransferas  43.9      40 0.00037   30.2   2.1   24    5-28    240-263 (336)
</pre><pre>  8 7BW9_B Serine acetyltransferas  43.9      40 0.00037   30.2   2.1   24    5-28    240-263 (336)
</pre><pre>  9 PF18390.3 ; GlgX_C ; Glycogen   40.4      53 0.00049   22.3   1.8   24   86-112    43-66  (85)
</pre><pre> 10 d2vhea_ b.81.1.8 (A:) Acetyltr  39.2      70 0.00065   25.1   2.5   25    4-28    159-183 (194)
</pre><pre> 11 d4r36a_ b.81.1.0 (A:) automate  38.2      56 0.00052   27.5   2.0   23    6-28    155-177 (255)
</pre><pre> 12 4G3Q_A Bifunctional protein Gl  38.0      56 0.00052   28.9   2.1   24    5-28    428-451 (501)
</pre><pre> 13 d4m98a_ b.81.1.0 (A:) automate  37.8      83 0.00077   24.0   2.7   24    5-28    171-194 (205)
</pre><pre> 14 TIGR03308 phn_thr-fam; phospho  36.8      68 0.00063   25.8   2.2   25    4-28    124-148 (204)
</pre><pre> 15 TIGR01853 lipid_A_lpxD; UDP-3-  33.0      84 0.00078   27.0   2.3   25    4-28    269-293 (324)
</pre><pre> 16 4HZD_A CysE, serine acetyltran  32.6      82 0.00076   27.7   2.2   24    5-28    220-243 (282)
</pre><pre> 17 4M98_A Pilin glycosylation pro  31.5 1.3E+02  0.0012   22.9   2.8   25    4-28    173-197 (208)
</pre><pre> 18 2GGO_A 401aa long hypothetical  30.6   1E+02 0.00093   25.6   2.3   24    5-28    373-396 (401)
</pre><pre> 19 d4n69a_ b.81.1.0 (A:) automate  30.1      95 0.00088   26.3   2.1   24    5-28    209-232 (243)
</pre><pre> 20 d4eqya_ b.81.1.0 (A:) automate  30.0      97  0.0009   25.2   2.1   24    5-28    156-179 (261)
</pre><pre> 21 4E79_C UDP-3-O-acylglucosamine  29.3   1E+02 0.00097   26.3   2.2   24    5-28    280-304 (357)
</pre><pre> 22 1SSQ_D Serine acetyltransferas  29.0      97  0.0009   27.0   2.0   24    5-28    206-229 (267)
</pre><pre> 23 4N6A_B Serine Acetyltransferas  28.9   1E+02 0.00097   27.1   2.2   24    5-28    221-244 (286)
</pre><pre> 24 2IU8_C UDP-3-O-[3-HYDROXYMYRIS  28.7   1E+02 0.00097   27.1   2.2   24    5-28    303-326 (374)
</pre><pre> 25 5VMK_A Bifunctional protein Gl  28.3   1E+02 0.00096   26.5   2.0   24    5-28    416-439 (462)
</pre><pre> 26 5VMK_B Bifunctional protein Gl  28.3   1E+02 0.00096   26.5   2.0   24    5-28    416-439 (462)
</pre><pre> 27 1SSM_A Serine acetyltransferas  28.3 1.1E+02 0.00099   25.9   2.1   24    5-28    206-229 (242)
</pre><pre> 28 1SSM_B Serine acetyltransferas  28.3 1.1E+02 0.00099   25.9   2.1   24    5-28    206-229 (242)
</pre><pre> 29 6WYE_B Serine acetyltransferas  27.5 1.1E+02   0.001   27.4   2.1   24    5-28    235-258 (293)
</pre><pre> 30 d3r0sa_ b.81.1.0 (A:) automate  27.2 1.1E+02  0.0011   25.3   2.1   24    5-28    156-179 (256)
</pre><pre> 31 3NZ2_C Hexapeptide-repeat cont  27.0 1.2E+02  0.0011   23.3   2.0   24    5-28    149-172 (195)
</pre><pre> 32 6SUP_A TcdB2,TccC3,Cell divisi  26.9      73 0.00068   34.6   1.2   17  106-124    27-43  (2128)
</pre><pre> 33 TIGR03991 alt_bact_glmU; UDP-N  25.9      97  0.0009   26.5   1.5   24    5-28    271-294 (337)
</pre><pre> 34 d1ssqa_ b.81.1.6 (A:) Serine a  25.6 1.3E+02  0.0012   25.1   2.0   24    5-28    206-229 (241)
</pre><pre> 35 3Q1X_A Serine acetyltransferas  25.4 1.3E+02  0.0012   27.2   2.1   24    5-28    240-263 (313)
</pre><pre> 36 PF18836.3 ; B_solenoid_ydck ;   23.4 1.5E+02  0.0014   16.7   1.4   15    7-21      1-15  (17)
</pre><pre> 37 TIGR01172 cysE; serine O-acety  22.7 1.7E+02  0.0016   22.5   2.1   25    4-28    129-153 (162)
</pre><pre> 38 d3mc4a_ b.81.1.0 (A:) automate  22.6 1.5E+02  0.0014   25.0   2.0   25    5-29    211-235 (249)
</pre><pre> 39 3PMO_A UDP-3-O-[3-hydroxymyris  22.2 1.7E+02  0.0016   26.3   2.3   24    5-28    297-321 (372)
</pre><pre> 40 3HSQ_B Acyl-[acyl-carrier-prot  22.1 1.7E+02  0.0016   24.0   2.2   24    5-28    156-179 (259)
</pre><pre> 41 3I3X_C Acyl-[acyl-carrier-prot  22.1 1.7E+02  0.0016   24.0   2.2   24    5-28    156-179 (259)
</pre><pre> 42 1HM9_B UDP-N-ACETYLGLUCOSAMINE  21.8 1.7E+02  0.0015   25.4   2.1   25    5-29    419-443 (468)
</pre><pre> 43 PF06688.13 ; DUF1187 ; Protein  21.6 1.8E+02  0.0016   21.4   1.9   15   96-110    10-24  (61)
</pre><pre> 44 d4e6ua_ b.81.1.0 (A:) automate  21.3 1.8E+02  0.0016   24.2   2.1   24    5-28    156-179 (258)
</pre><pre> 45 6L7I_F TcdB1; protein transpor  21.0 1.1E+02   0.001   32.7   1.0   17  106-124    58-74  (1476)
</pre><pre> 46 6PXA_I Chloramphenicol acetylt  20.8 1.8E+02  0.0017   24.4   2.1   24    5-28    138-161 (221)
</pre><pre> 47 6PXA_L Chloramphenicol acetylt  20.8 1.8E+02  0.0017   24.4   2.1   24    5-28    138-161 (221)
</pre><pre> 48 3T57_A UDP-N-acetylglucosamine  20.8 1.8E+02  0.0017   24.3   2.1   24    5-28    185-208 (305)
</pre><pre> 49 TIGR04008 WcaF; colanic acid b  20.7 2.1E+02  0.0019   22.7   2.3   25    4-28    140-164 (180)
</pre><pre> 50 6P89_C UDP-3-O-(3-hydroxymyris  20.4 1.9E+02  0.0018   24.3   2.2   24    5-28    277-301 (343)
</pre><pre> 51 PF09937.11 ; DUF2169 ; Unchara  20.3 1.2E+02  0.0011   27.0   1.0   25   98-122   114-148 (293)
</pre><pre> 52 d3hsqa_ b.81.1.0 (A:) automate  20.2 2.1E+02  0.0019   23.9   2.3   26    4-29    155-180 (259)
</pre><pre> 53 6AG8_C Maltose O-acetyltransfe  20.1 2.2E+02  0.0021   21.9   2.3   25    4-28    145-169 (191)
</pre><pre> 54 4FCE_A Bifunctional protein Gl  20.0   2E+02  0.0018   25.0   2.2   24    5-28    414-437 (459)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF20148.1 ; DUF6531 ; Domain of unknown function (DUF6531)
</pre><pre>Probab=99.21  E-value=4.4e-11  Score=72.36  Aligned_cols=55  Identities=35%  Similarity=0.593  Sum_probs=50.3  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             CCCeeccCCCcccCCCcccccccCCc-CCeeEEEEeeCCCCcCCCcCCccccceeEEeee
</pre><pre>Q FD01847867_021   73 NRPVNAATGAKYLAGDDDVDFSLPGH-FPLEWQRTYSSRDERTEGMFGRGWSVRRTTRMK  131 (133)
</pre><pre>Q Consensus        73 ~~PVh~aTGaKvL~g~edlDF~LPg~-lPl~WQR~Y~S~d~r~~GlfG~GWs~p~e~~l~  131 (133)
</pre><pre>                      ++||++.||.|++.   +.||.+++. +|+.|+|.|+|.... .+.||.||+..|+.+++
</pre><pre>T Consensus         1 g~pV~~~~g~~~~~---~~d~~~~g~~~~~~~~r~y~s~~~~-~~~~g~gw~~~~~~~~~   56 (75)
</pre><pre>T A0A085WJS1_9DE    1 NSTVNMATGNLREQ---WTDLTLPAVGLPITWRRTYASRGNG-MTELGYGWMHSYGMYLR   56 (75)
</pre><pre>T ss_pred             CCceeCCCCcEEEE---EeceEccCCCCCEEEEEEEcCCCCC-cCCCccCeecccCcEEE
</pre><pre>Confidence            47999999999885   899999999 999999999999776 78999999999998875
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>6P0X_B Salmonella plasmid virulence: SpvB; salmonella effector, SpvB, ADP-ribosyltransferase, UNKNOWN FUNCTION; HET: AKR, BTB, GOL, MLY; 2.4A {Salmonella typhimurium (strain LT2 / SGSC1412 / ATCC 700720)}
</pre><pre>Probab=70.42  E-value=10  Score=32.93  Aligned_cols=45  Identities=24%  Similarity=0.294  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CeeccCCCcccCCCcccccccCC-cCCeeEEEEeeCCCCcCCCcCCccccc
</pre><pre>Q FD01847867_021   75 PVNAATGAKYLAGDDDVDFSLPG-HFPLEWQRTYSSRDERTEGMFGRGWSV  124 (133)
</pre><pre>Q Consensus        75 PVh~aTGaKvL~g~edlDF~LPg-~lPl~WQR~Y~S~d~r~~GlfG~GWs~  124 (133)
</pre><pre>                      ||...||+=-++   --.|.+|. -+.+.-+..|+|...  .+.+|.||++
</pre><pre>T Consensus         6 ~v~~~tG~~~is---iPl~~~~~~~~~~~isL~Y~s~g~--~g~vG~GW~L   51 (333)
</pre><pre>T 6P0X_B            6 SQSGPDGLASIT---LPLPISAERGFAPALALHYSSGGG--NGPFGVGWSC   51 (333)
</pre><pre>T ss_dssp             ------CCCCEE---EECCCCCSSSCCCCCEEEECTTCC--CSTTCSSEEC
</pre><pre>T ss_pred             ccCCCCCceeee---eecccccCCCcCCceEEEEcCCCC--CCceeeccee
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>d1bf2a2 b.71.1.1 (A:638-750) Isoamylase {Pseudomonas amyloderamosa [TaxId: 32043]}
</pre><pre>Probab=60.69  E-value=20  Score=25.83  Aligned_cols=27  Identities=26%  Similarity=0.389  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCcccccccCCcC-CeeEEEEeeCCCC
</pre><pre>Q FD01847867_021   86 AGDDDVDFSLPGHF-PLEWQRTYSSRDE  112 (133)
</pre><pre>Q Consensus        86 ~g~edlDF~LPg~l-Pl~WQR~Y~S~d~  112 (133)
</pre><pre>                      .+.++.+|.||..- ...|++..++...
</pre><pre>T Consensus        50 ~~~~~~~f~LP~~~~~~~W~~~~dT~~~   77 (113)
</pre><pre>T d1bf2a2          50 GWSSSVTFTLPAPPSGTQWYRVTDTCDW   77 (113)
</pre><pre>T ss_dssp             CSSSCEEEECCCCSSSSCEEEEEECSGG
</pre><pre>T ss_pred             CCCCceEEeCCCCCCCCceEEEEeCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>6OX6_A Tas1; type VI secretion, bacterial warfare, (p)ppApp, alarmone, Pseudomonas aeruginosa, Structural Genomics, Center for Structural Genomics of; HET: ACT, MSE; 2.17A {Pseudomonas aeruginosa}
</pre><pre>Probab=58.02  E-value=42  Score=30.37  Aligned_cols=53  Identities=21%  Similarity=0.158  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             cCCcEeCCCeeEEEecCCCCCH------HHHHHHHHHHhhC---cchHhhhchHHHHHHHHH
</pre><pre>Q FD01847867_021   10 SGDVRIGGPLLVVRDIKSGKSQ------ITLVTTIALTFMQ---PGRASAKSACFMMGLGID   62 (133)
</pre><pre>Q Consensus        10 S~nVrIGG~~vvVRdI~SGK~p------~~~~~~~~~~ll~---~~k~~~~l~C~~~~~~~~   62 (133)
</pre><pre>                      |+||+|||.+.++++|.+...|      +..+++.+.+++.   .+-......|...+...+
</pre><pre>T Consensus       134 s~~v~~g~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  195 (439)
</pre><pre>T 6OX6_A          134 SQNTFIGGPTERVAFVLDLEEWLHTGLEALGLAALAGGLLLAAMAGVAALVGVVAIGGLMMG  195 (439)
</pre><pre>T ss_dssp             --------------------------------------------------------------
</pre><pre>T ss_pred             CCcEEECCcceeeeeecchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF03534.15 ; SpvB ; Salmonella virulence plasmid 65kDa B protein
</pre><pre>Probab=53.81  E-value=28  Score=29.32  Aligned_cols=30  Identities=23%  Similarity=0.489  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             cccCCc--CCeeEEEEeeCCCCcCCCcCCccccc
</pre><pre>Q FD01847867_021   93 FSLPGH--FPLEWQRTYSSRDERTEGMFGRGWSV  124 (133)
</pre><pre>Q Consensus        93 F~LPg~--lPl~WQR~Y~S~d~r~~GlfG~GWs~  124 (133)
</pre><pre>                      +.+|++  +.+.-...|+|...  .|.+|.||+|
</pre><pre>T Consensus        14 ~~~~~~~g~~~~isL~Y~s~~~--~g~~G~GWsL   45 (289)
</pre><pre>T S8C346_DACHA/7   14 STSSSRGGFGPLLNFAYNSGSG--NGPYGFGWAI   45 (289)
</pre><pre>T ss_pred             ecCCCCCccCCeeEEEEeCCCC--CCCceEceEe
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>3F1X_A Serine acetyltransferase; NESG X-Ray BvR62 A6KZB9 A6KZB9_BACV8, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, Transferase; 2.0A {Bacteroides vulgatus ATCC 8482}
</pre><pre>Probab=48.02  E-value=32  Score=30.38  Aligned_cols=24  Identities=21%  Similarity=0.537  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||+.+++.+++..+
</pre><pre>T Consensus       268 ~~v~IG~~~~Iga~svV~~~v~~~  291 (310)
</pre><pre>T 3F1X_A          268 GRVTIGKGATVGGNIWVTENVPAG  291 (310)
</pre><pre>T ss_dssp             SSCEECTTCEECSSCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>7BW9_A Serine acetyltransferase 1, putative; Left handed beta-helical (LBH) domain, serine acetyltransferase, Cysteine biosynthesis, Cysteine synthase Complex., TRANSFERASE; HET: CYS; 2.131A {Entamoeba histolytica}
</pre><pre>Probab=43.91  E-value=40  Score=30.17  Aligned_cols=24  Identities=17%  Similarity=0.493  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||..+++.+|+..+
</pre><pre>T Consensus       240 ggv~IG~~~iIga~svVt~~vp~~  263 (336)
</pre><pre>T 7BW9_A          240 GNITIGSNVKIGANCWITQNIDQD  263 (336)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCcEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>7BW9_B Serine acetyltransferase 1, putative; Left handed beta-helical (LBH) domain, serine acetyltransferase, Cysteine biosynthesis, Cysteine synthase Complex., TRANSFERASE; HET: CYS; 2.131A {Entamoeba histolytica}
</pre><pre>Probab=43.91  E-value=40  Score=30.17  Aligned_cols=24  Identities=17%  Similarity=0.493  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||..+++.+|+..+
</pre><pre>T Consensus       240 ggv~IG~~~iIga~svVt~~vp~~  263 (336)
</pre><pre>T 7BW9_B          240 GNITIGSNVKIGANCWITQNIDQD  263 (336)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCcEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>PF18390.3 ; GlgX_C ; Glycogen debranching enzyme C-terminal domain
</pre><pre>Probab=40.40  E-value=53  Score=22.28  Aligned_cols=24  Identities=25%  Similarity=0.361  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCCcccccccCCcCCeeEEEEeeCCCC
</pre><pre>Q FD01847867_021   86 AGDDDVDFSLPGHFPLEWQRTYSSRDE  112 (133)
</pre><pre>Q Consensus        86 ~g~edlDF~LPg~lPl~WQR~Y~S~d~  112 (133)
</pre><pre>                      +..++..|.||..   .|++..++..+
</pre><pre>T Consensus        43 ~~~~~~~f~LP~~---~W~~~~dt~~~   66 (85)
</pre><pre>T A0A077ZLZ1_TRI   43 ATLEVTEIVLPAG---EWHAIPPFAGE   66 (85)
</pre><pre>T ss_pred             CCCCcEEEECCCC---ceEEeccCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>d2vhea_ b.81.1.8 (A:) Acetyltransferase PglD {Campylobacter jejuni [TaxId: 197]}
</pre><pre>Probab=39.18  E-value=70  Score=25.15  Aligned_cols=25  Identities=16%  Similarity=0.299  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      ..++.|+++++|+..+++.+||..+
</pre><pre>T Consensus       159 ~~~~~Ig~~~~I~~~~~v~~~v~~~  183 (194)
</pre><pre>T d2vhea_         159 LPNLSLADDSILGGGATLVKNQDEK  183 (194)
</pre><pre>T ss_dssp             CTTCEECTTCEECTTCEECSCBCSC
</pre><pre>T ss_pred             CCCCEECCCCEECCCCEEeCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>d4r36a_ b.81.1.0 (A:) automated matches {Bacteroides fragilis [TaxId: 272559]}
</pre><pre>Probab=38.21  E-value=56  Score=27.49  Aligned_cols=23  Identities=30%  Similarity=0.223  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             cceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    6 GVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         6 g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++|++++.||+.+++.+|+...
</pre><pre>T Consensus       155 ~~~IG~~~~IgagsvV~~dvpp~  177 (255)
</pre><pre>T d4r36a_         155 FCRVGGYVMIQGGCRFSKDIPPY  177 (255)
</pre><pre>T ss_dssp             TCEECTTCEECTTCEESSBBCTT
</pre><pre>T ss_pred             CCEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>4G3Q_A Bifunctional protein GlmU; ACETYLTRANSFERASE, BIFUNCTIONAL, PYROPHOSPHORYLASE, ROSSMANN-LIKE FOLD, LEFT-HANDED-BETA-HELIX, CELL SHAPE, CELL WALL BIOGENESIS/DEGRADATION, METAL-BINDING, MULTIFUNCTIONAL ENZYME, NUCLEOTIDYLTRANSFERASE, PEPTIDOGLYCAN; HET: UD1; 1.9A {Mycobacterium tuberculosis} SCOP: c.68.1.0, b.81.1.0
</pre><pre>Probab=37.99  E-value=56  Score=28.93  Aligned_cols=24  Identities=21%  Similarity=0.343  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++|++||..+++.+||.++
</pre><pre>T Consensus       428 ~g~~ig~~~~igagsvv~~~v~~~  451 (501)
</pre><pre>T 4G3Q_A          428 APVTIGDGAYTGAGTVVREDVPPG  451 (501)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCcEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>d4m98a_ b.81.1.0 (A:) automated matches {Neisseria gonorrhoeae [TaxId: 242231]}
</pre><pre>Probab=37.78  E-value=83  Score=23.98  Aligned_cols=24  Identities=29%  Similarity=0.441  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|+++++||-.+++-+++.++
</pre><pre>T Consensus       171 ~~~~i~~~~~i~~~~~v~~~~~~~  194 (205)
</pre><pre>T d4m98a_         171 QQTTVGSGVTAGAGAVIVCDIPDG  194 (205)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEEecccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>TIGR03308 phn_thr-fam; phosphonate metabolim protein, transferase hexapeptide repeat family. This family of proteins contains copies of the Bacterial transferase hexapeptide repeat family (pfam00132) and is only found in operons encoding the phosphonate C-P lyase system (GenProp0232).
</pre><pre>Probab=36.82  E-value=68  Score=25.78  Aligned_cols=25  Identities=20%  Similarity=0.295  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      ..++.|++|+.||..+++.+|+.++
</pre><pre>T Consensus       124 ~~g~~IG~~~~Ig~~s~v~~~vp~~  148 (204)
</pre><pre>T TIGR03308       124 LPGVTIGNGAVIAAGAVVTKDVAPY  148 (204)
</pre><pre>T ss_pred             CCCCEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>TIGR01853 lipid_A_lpxD; UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase LpxD. This model describes LpxD, an enzyme for the biosynthesis of lipid A, a component oflipopolysaccharide (LPS) in the outer membrane outer leaflet of most Gram-negative bacteria.
</pre><pre>Probab=32.96  E-value=84  Score=27.04  Aligned_cols=25  Identities=20%  Similarity=0.308  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .+++.|+++++|++.+++.+|+.++
</pre><pre>T Consensus       269 ~~~v~Ig~~~~v~~gs~v~~~v~~~  293 (324)
</pre><pre>T TIGR01853       269 AGHLEIGDNVTIGAKSGVTKSIPPP  293 (324)
</pre><pre>T ss_pred             CCCcEECCCCEECCCCEEcCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>4HZD_A CysE, serine acetyltransferase; Left handed beta-helical (LBH) domain, Cysteine biosynthesis, serine acetyltransferase, TRANSFERASE; HET: COA; 1.87A {Brucella abortus} SCOP: b.81.1.0
</pre><pre>Probab=32.57  E-value=82  Score=27.69  Aligned_cols=24  Identities=13%  Similarity=0.288  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|+++++||-.+++.+||.++
</pre><pre>T Consensus       220 g~v~IG~~~~Iga~svV~~~vp~~  243 (282)
</pre><pre>T 4HZD_A          220 GNIQVGQCSKIAAGSVVLKSVPHN  243 (282)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>4M98_A Pilin glycosylation protein; left-handed beta-helix, rossmann fold, acetyltransferase, Transferase; 1.67A {Neisseria gonorrhoeae} SCOP: b.81.1.0
</pre><pre>Probab=31.49  E-value=1.3e+02  Score=22.93  Aligned_cols=25  Identities=28%  Similarity=0.429  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .+++.|++|+.||...++.+++...
</pre><pre>T Consensus       173 ~~~~~i~~~~~i~~~~~v~~~~~~~  197 (208)
</pre><pre>T 4M98_A          173 RQQTTVGSGVTAGAGAVIVCDIPDG  197 (208)
</pre><pre>T ss_dssp             CTTCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCCEECCCCEECCCCEEeceeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>2GGO_A 401aa long hypothetical glucose-1-phosphate thymidylyltransferase; BETA BARREL, TRANSFERASE; 1.8A {Sulfolobus tokodaii}
</pre><pre>Probab=30.62  E-value=1e+02  Score=25.59  Aligned_cols=24  Identities=33%  Similarity=0.423  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++++.||..+++-+||..+
</pre><pre>T Consensus       373 ~~~~i~~~~~i~~~~~v~~~~~~~  396 (401)
</pre><pre>T 2GGO_A          373 PGVKIGAYARIYPGAVVNRDVGYG  396 (401)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEeCCCcEEcCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>d4n69a_ b.81.1.0 (A:) automated matches {Soybean (Glycine max) [TaxId: 3847]}
</pre><pre>Probab=30.08  E-value=95  Score=26.30  Aligned_cols=24  Identities=13%  Similarity=0.387  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||..+++.+++..+
</pre><pre>T Consensus       209 ~~~~Ig~~~~Igags~v~~~v~~~  232 (243)
</pre><pre>T d4n69a_         209 GNIKIGEGAKVGAGSVVLIDVPPR  232 (243)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCcEECCCCEECCCCEECCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>d4eqya_ b.81.1.0 (A:) automated matches {Burkholderia thailandensis [TaxId: 271848]}
</pre><pre>Probab=29.99  E-value=97  Score=25.23  Aligned_cols=24  Identities=25%  Similarity=0.354  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .++.|++++.||..+++.+|+.++
</pre><pre>T Consensus       156 ~~~~Ig~~~~I~~~s~v~~~vp~~  179 (261)
</pre><pre>T d4eqya_         156 QYVRIGAHSMLGGASALVQDIPPF  179 (261)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSBBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEeCCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>4E79_C UDP-3-O-acylglucosamine N-acyltransferase; lipopolysaccaride synthesis, TRANSFERASE; 2.66A {Acinetobacter baumannii}
</pre><pre>Probab=29.35  E-value=1e+02  Score=26.25  Aligned_cols=24  Identities=21%  Similarity=0.530  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEec-CCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDI-KSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI-~SG   28 (133)
</pre><pre>                      .+++|++++.||..+++.+|+ ..+
</pre><pre>T Consensus       280 ~~~~ig~~~~ig~~s~v~~~v~~~~  304 (357)
</pre><pre>T 4E79_C          280 GHLSIADNVTLTGMSMVTKNISEAG  304 (357)
</pre><pre>T ss_dssp             SSCEECTTEEECTTCEECSCBCSCE
</pre><pre>T ss_pred             CCcEECCCCEECCCCEEcCCcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>1SSQ_D Serine acetyltransferase; LEFT-HANDED PARALLEL BETA HELIX, transferase; HET: CYS; 1.85A {Haemophilus influenzae} SCOP: b.81.1.6
</pre><pre>Probab=29.03  E-value=97  Score=26.98  Aligned_cols=24  Identities=17%  Similarity=0.285  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||-.+++.+|+..+
</pre><pre>T Consensus       206 ~~v~IG~~~~Iga~svV~~~v~~~  229 (267)
</pre><pre>T 1SSQ_D          206 GNIEVGKYAKIGANSVVLNPVPEY  229 (267)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>4N6A_B Serine Acetyltransferase Apoenzyme; acetyltransferase, transferase; 1.75A {Glycine max} SCOP: b.81.1.0
</pre><pre>Probab=28.87  E-value=1e+02  Score=27.13  Aligned_cols=24  Identities=13%  Similarity=0.387  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||-.+++.+++..+
</pre><pre>T Consensus       221 ~~v~IG~~~~IgagsvV~~~v~~~  244 (286)
</pre><pre>T 4N6A_B          221 GNIKIGEGAKVGAGSVVLIDVPPR  244 (286)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEEcCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>2IU8_C UDP-3-O-[3-HYDROXYMYRISTOYL] GLUCOSAMINE N-ACYLTRANSFERASE; TRANSFERASE, UDP-3- O-ACYL-GLUCOSAMINE N-ACYLTRANSFERASE, ACYLTRANSFERASE, LIPID A BIOSYNTHESIS, LEFT-HANDED BETA HELIX, COMPLEX WITH UDPGLCNAC, ENZYME, HOMOTRIMER; HET: SO4, UD1, PLM; 2.2A {CHLAMYDIA TRACHOMATIS}
</pre><pre>Probab=28.73  E-value=1e+02  Score=27.08  Aligned_cols=24  Identities=17%  Similarity=0.160  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++++.||..+++.+|+.+.
</pre><pre>T Consensus       303 ~g~~Ig~~~~vg~~svV~~~v~~~  326 (374)
</pre><pre>T 2IU8_C          303 GHICIADHVIMMAQTGVTKSITSP  326 (374)
</pre><pre>T ss_dssp             SSCEECTTEEECTTCEECSCBCSC
</pre><pre>T ss_pred             CCCEECCCCEECCCCEEcCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>5VMK_A Bifunctional protein GlmU; NIAID, cSMRT, citrate, structural genomics, mutli-domain protein, bifunctional protein, Seattle Structural Genomics Center for Infectious Disease; HET: CIT, PG4; 2.55A {Acinetobacter baumannii} SCOP: b.81.1.0, c.68.1.0
</pre><pre>Probab=28.30  E-value=1e+02  Score=26.54  Aligned_cols=24  Identities=13%  Similarity=0.360  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +|+.|+++++||..+++.+|+.+.
</pre><pre>T Consensus       416 ~~~~ig~~~~v~~~~~v~~~~~~~  439 (462)
</pre><pre>T 5VMK_A          416 APVTIGNGATVGAGSVITKDVAEQ  439 (462)
</pre><pre>T ss_dssp             SSEEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCcEECCCCEECCCCEeCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>5VMK_B Bifunctional protein GlmU; NIAID, cSMRT, citrate, structural genomics, mutli-domain protein, bifunctional protein, Seattle Structural Genomics Center for Infectious Disease; HET: CIT, PG4; 2.55A {Acinetobacter baumannii} SCOP: c.68.1.0, b.81.1.0
</pre><pre>Probab=28.30  E-value=1e+02  Score=26.54  Aligned_cols=24  Identities=13%  Similarity=0.360  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +|+.|+++++||..+++.+|+.+.
</pre><pre>T Consensus       416 ~~~~ig~~~~v~~~~~v~~~~~~~  439 (462)
</pre><pre>T 5VMK_B          416 APVTIGNGATVGAGSVITKDVAEQ  439 (462)
</pre><pre>T ss_dssp             SSEEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCcEECCCCEECCCCEeCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>1SSM_A Serine acetyltransferase; LEFT-HANDED PARALLEL BETA HELIX, transferase; HET: MSE; 2.15A {Haemophilus influenzae} SCOP: b.81.1.6
</pre><pre>Probab=28.27  E-value=1.1e+02  Score=25.93  Aligned_cols=24  Identities=17%  Similarity=0.285  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|+++++||-.+++.+|+..+
</pre><pre>T Consensus       206 ~~v~IG~~~~Iga~svV~~~v~~~  229 (242)
</pre><pre>T 1SSM_A          206 GNIEVGKYAKIGANSVVLNPVPEY  229 (242)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>1SSM_B Serine acetyltransferase; LEFT-HANDED PARALLEL BETA HELIX, transferase; HET: MSE; 2.15A {Haemophilus influenzae} SCOP: b.81.1.6
</pre><pre>Probab=28.27  E-value=1.1e+02  Score=25.93  Aligned_cols=24  Identities=17%  Similarity=0.285  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|+++++||-.+++.+|+..+
</pre><pre>T Consensus       206 ~~v~IG~~~~Iga~svV~~~v~~~  229 (242)
</pre><pre>T 1SSM_B          206 GNIEVGKYAKIGANSVVLNPVPEY  229 (242)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>6WYE_B Serine acetyltransferase; CysE, Serine acetyltransferase, Neisseria, Gonorrhoea, TRANSFERASE; HET: LMR; 2.01A {Neisseria gonorrhoeae}
</pre><pre>Probab=27.52  E-value=1.1e+02  Score=27.35  Aligned_cols=24  Identities=21%  Similarity=0.518  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||-.+++..||..+
</pre><pre>T Consensus       235 g~v~IG~~~~IgagsvV~~dvp~~  258 (293)
</pre><pre>T 6WYE_B          235 GNIRIGSNAKIGAGSVVVSDVPPS  258 (293)
</pre><pre>T ss_pred             CCcEECCCCEECCCCEECCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>d3r0sa_ b.81.1.0 (A:) automated matches {Campylobacter jejuni [TaxId: 192222]}
</pre><pre>Probab=27.21  E-value=1.1e+02  Score=25.30  Aligned_cols=24  Identities=25%  Similarity=0.189  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++++.||..+++.+|+...
</pre><pre>T Consensus       156 ~gv~Ig~~~~I~~~s~v~~dv~~~  179 (256)
</pre><pre>T d3r0sa_         156 QFVKVGEGCMIAGASALSQDIVPF  179 (256)
</pre><pre>T ss_dssp             TTCEECTTCEECSSCBBCSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>3NZ2_C Hexapeptide-repeat containing-acetyltransferase; Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID, beta-helix, methyltransferase, TRANSFERASE; HET: MSE, BU1, ACO, ACY, GOL; 2.35A {Vibrio cholerae O1 biovar eltor} SCOP: b.81.1.0
</pre><pre>Probab=26.99  E-value=1.2e+02  Score=23.31  Aligned_cols=24  Identities=17%  Similarity=0.340  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .++.|++|++||-.+++..+|.+.
</pre><pre>T Consensus       149 ~~~~ig~~~~i~~~~~v~~~i~~~  172 (195)
</pre><pre>T 3NZ2_C          149 QGVTIGARSVVAANSVVNQDVPPD  172 (195)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSCBCSS
</pre><pre>T ss_pred             CCCEECCCCEECCCCEeCCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>6SUP_A TcdB2,TccC3,Cell division control protein 42 homolog; Toxins, Tc Toxins, TOXIN; 2.0A {Photorhabdus luminescens}
</pre><pre>Probab=26.91  E-value=73  Score=34.61  Aligned_cols=17  Identities=35%  Similarity=0.790  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             EeeCCCCcCCCcCCccccc
</pre><pre>Q FD01847867_021  106 TYSSRDERTEGMFGRGWSV  124 (133)
</pre><pre>Q Consensus       106 ~Y~S~d~r~~GlfG~GWs~  124 (133)
</pre><pre>                      .|||....  |+||.||+|
</pre><pre>T Consensus        27 ~Y~S~~gn--G~~G~GWsL   43 (2128)
</pre><pre>T 6SUP_A           27 NYNSGAGN--SPFGLGWDC   43 (2128)
</pre><pre>T ss_dssp             EEETTCCC--CTTCTTEEC
</pre><pre>T ss_pred             EEecCCCC--CCceecEee
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>TIGR03991 alt_bact_glmU; UDP-N-acetylglucosamine diphosphorylase/glucosamine-1-phosphate N-acetyltransferase.
</pre><pre>Probab=25.85  E-value=97  Score=26.53  Aligned_cols=24  Identities=21%  Similarity=0.061  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++++.|++.+++-+|+.++
</pre><pre>T Consensus       271 ~g~~Ig~~~~v~~~s~v~~~v~~~  294 (337)
</pre><pre>T TIGR03991       271 TGTVVGVGSNIFGSGFPPKYVPSF  294 (337)
</pre><pre>T ss_pred             CCCEECCCCEECCCCccCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>d1ssqa_ b.81.1.6 (A:) Serine acetyltransferase {Haemophilus influenzae [TaxId: 727]}
</pre><pre>Probab=25.63  E-value=1.3e+02  Score=25.08  Aligned_cols=24  Identities=17%  Similarity=0.285  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||..+++-.|+.++
</pre><pre>T Consensus       206 ~~~~Ig~~~~I~~~s~v~~~vp~~  229 (241)
</pre><pre>T d1ssqa_         206 GNIEVGKYAKIGANSVVLNPVPEY  229 (241)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>3Q1X_A Serine acetyltransferase; Cysteine biosynthesis, serine acetyltransferas, Left handed beta helix, Acetyl transferase, OASS, TRANSFERASE; 1.59A {Entamoeba histolytica}
</pre><pre>Probab=25.42  E-value=1.3e+02  Score=27.20  Aligned_cols=24  Identities=33%  Similarity=0.531  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      +++.|++|++||..+++.+++..+
</pre><pre>T Consensus       240 ggv~IG~~~~Iga~svV~~~v~~~  263 (313)
</pre><pre>T 3Q1X_A          240 GNIIVGSHVRIGANCWIDRDVDSN  263 (313)
</pre><pre>T ss_dssp             SSCEECSSEEECTTCEECSCBCSS
</pre><pre>T ss_pred             CCcEECCCCEECCCCEEcCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF18836.3 ; B_solenoid_ydck ; YdcK Beta solenoid repeat
</pre><pre>Probab=23.40  E-value=1.5e+02  Score=16.69  Aligned_cols=15  Identities=13%  Similarity=0.364  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             ceecCCcEeCCCeeE
</pre><pre>Q FD01847867_021    7 VHVSGDVRIGGPLLV   21 (133)
</pre><pre>Q Consensus         7 ~~vS~nVrIGG~~vv   21 (133)
</pre><pre>                      .+||.|++|-+..++
</pre><pre>T Consensus         1 AqIsgnA~V~~s~v~   15 (17)
</pre><pre>T A8AGI5_CITK8/1    1 ARISNNVTVQNSVVR   15 (17)
</pre><pre>T ss_pred             CceecceEEecCEEe
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>TIGR01172 cysE; serine O-acetyltransferase. Cysteine biosynthesis [Amino acid biosynthesis, Serine family].
</pre><pre>Probab=22.68  E-value=1.7e+02  Score=22.49  Aligned_cols=25  Identities=24%  Similarity=0.554  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .+++.|++|+.||-..++..++.++
</pre><pre>T Consensus       129 ~~~~~Ig~~~~Ig~~~~i~~~v~~~  153 (162)
</pre><pre>T TIGR01172       129 LGNIEVGENAKIGANSVVLKDVPPG  153 (162)
</pre><pre>T ss_pred             cCCeEECCCCEECCCcEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>d3mc4a_ b.81.1.0 (A:) automated matches {Brucella melitensis [TaxId: 359391]}
</pre><pre>Probab=22.60  E-value=1.5e+02  Score=25.03  Aligned_cols=25  Identities=12%  Similarity=0.254  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSGK   29 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SGK   29 (133)
</pre><pre>                      +++.|+++++||..+++..++..+.
</pre><pre>T Consensus       211 ~~~~Ig~~~~Ig~~svV~~~vp~~~  235 (249)
</pre><pre>T d3mc4a_         211 GNIQVGQCSKIAAGSVVLKSVPHNV  235 (249)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTTE
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCcCCCe
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>3PMO_A UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase; Lipid A biosynthesis pathway, TRANSFERASE; 1.3A {Pseudomonas aeruginosa}
</pre><pre>Probab=22.22  E-value=1.7e+02  Score=26.26  Aligned_cols=24  Identities=25%  Similarity=0.510  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEec-CCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDI-KSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI-~SG   28 (133)
</pre><pre>                      +++.|++|++||..+++.+|+ ..+
</pre><pre>T Consensus       297 ~~v~Ig~~~~vg~~s~V~~~v~~~~  321 (372)
</pre><pre>T 3PMO_A          297 GHIEICDNVFVTGMTMVTRSITEPG  321 (372)
</pre><pre>T ss_dssp             SSCEECSSEEECTTCEECSCBCSCE
</pre><pre>T ss_pred             CCCEECCCCEECCCCEEcCccCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>3HSQ_B Acyl-[acyl-carrier-protein]--UDP-N-acetylglucosamine O-acyltransferase; L.interrogans LPXA, LPXA, LPXA crystal structure, Acyltransferase, Lipid A biosynthesis, Lipid synthesis, Transferase; 2.1A {Leptospira interrogans} SCOP: b.81.1.0
</pre><pre>Probab=22.05  E-value=1.7e+02  Score=23.98  Aligned_cols=24  Identities=21%  Similarity=0.145  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .++.|++++.|++..++.+|+...
</pre><pre>T Consensus       156 ~~~~ig~~~~i~~~s~v~~~vp~~  179 (259)
</pre><pre>T 3HSQ_B          156 QFCFVGDYSMVAGLAKVVQDVPPY  179 (259)
</pre><pre>T ss_dssp             TTCEECTTCEECSSCEECSBBCTT
</pre><pre>T ss_pred             CCCEECCCcEECCCCEECCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>3I3X_C Acyl-[acyl-carrier-protein]--UDP-N-acetylglucosamine O-acyltransferase; LpxA, Leptospira Interrogans LpxA, L. interrogans LpxA, L. interrogans LpxA product complex, Acyltransferase, Lipid A biosynthesis; HET: U22; 2.1A {Leptospira interrogans} SCOP: b.81.1.0
</pre><pre>Probab=22.05  E-value=1.7e+02  Score=23.98  Aligned_cols=24  Identities=21%  Similarity=0.145  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .++.|++++.|++..++.+|+...
</pre><pre>T Consensus       156 ~~~~ig~~~~i~~~s~v~~~vp~~  179 (259)
</pre><pre>T 3I3X_C          156 QFCFVGDYSMVAGLAKVVQDVPPY  179 (259)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSBBCTT
</pre><pre>T ss_pred             CCCEECCCcEECCCCEECCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>1HM9_B UDP-N-ACETYLGLUCOSAMINE-1-PHOSPHATE URIDYLTRANSFERASE; ACETYLTRANSFERASE, BIFUNCTIONAL, DRUG DESIGN, PYROPHOSPHORYLASE, Rossmann-like fold, left-handed-beta-helix, trimer, domain-interchange, TRANSFERASE; HET: UD1, ACO; 1.75A {Streptococcus pneumoniae} SCOP: b.81.1.4, c.68.1.5
</pre><pre>Probab=21.83  E-value=1.7e+02  Score=25.43  Aligned_cols=25  Identities=12%  Similarity=0.298  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSGK   29 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SGK   29 (133)
</pre><pre>                      .|+.|+++++||...++.+|+..+.
</pre><pre>T Consensus       419 ~~~~i~~~~~i~~~~~v~~~~~~~~  443 (468)
</pre><pre>T 1HM9_B          419 APVELGDNSLVGAGSTITKDVPADA  443 (468)
</pre><pre>T ss_dssp             SSCEECTTCEECTTCEECSCBCTTC
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF06688.13 ; DUF1187 ; Protein of unknown function (DUF1187)
</pre><pre>Probab=21.61  E-value=1.8e+02  Score=21.40  Aligned_cols=15  Identities=33%  Similarity=0.663  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             CCcCCeeEEEEeeCC
</pre><pre>Q FD01847867_021   96 PGHFPLEWQRTYSSR  110 (133)
</pre><pre>Q Consensus        96 Pg~lPl~WQR~Y~S~  110 (133)
</pre><pre>                      ||..|..|.|+=++.
</pre><pre>T Consensus        10 ~G~~Pv~W~ryS~~k   24 (61)
</pre><pre>T C8UPE6_ECO1A/2   10 EGGTPTNWTRYSKSK   24 (61)
</pre><pre>T ss_pred             CCCCceEEEEEcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>d4e6ua_ b.81.1.0 (A:) automated matches {Acinetobacter baumannii [TaxId: 470]}
</pre><pre>Probab=21.27  E-value=1.8e+02  Score=24.16  Aligned_cols=24  Identities=21%  Similarity=0.361  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .++.|++++.|+..+++.++|.+.
</pre><pre>T Consensus       156 ~g~~Ig~~~~I~~~s~v~~~vp~~  179 (258)
</pre><pre>T d4e6ua_         156 QFCKIDSYSMIGGASLILKDVPAY  179 (258)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSBBCTT
</pre><pre>T ss_pred             CCCEECCCCEEcCCCEECCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>6L7I_F TcdB1; protein transportation, TcdA1, TcdB1, TccC2, TOXIN-PROTEIN TRANSPORT complex; 2.9A {Photorhabdus luminescens}
</pre><pre>Probab=21.01  E-value=1.1e+02  Score=32.74  Aligned_cols=17  Identities=35%  Similarity=0.796  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             EeeCCCCcCCCcCCccccc
</pre><pre>Q FD01847867_021  106 TYSSRDERTEGMFGRGWSV  124 (133)
</pre><pre>Q Consensus       106 ~Y~S~d~r~~GlfG~GWs~  124 (133)
</pre><pre>                      .|||....  |+||.||+|
</pre><pre>T Consensus        58 ~Y~S~~gn--G~~G~GW~l   74 (1476)
</pre><pre>T 6L7I_F           58 NYNSGTGN--SPFGLGWDC   74 (1476)
</pre><pre>T ss_dssp             EEETTCCE--ETTEESEEC
</pre><pre>T ss_pred             EEcCCCCC--CCceeeeee
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>6PXA_I Chloramphenicol acetyltransferase; Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID, TRANSFERASE; HET: CL, TCH, SO4, GOL, ACT, FMT; 1.82A {Aliivibrio fischeri (strain ATCC 700601 / ES114)} SCOP: b.81.1.0, l.1.1.1
</pre><pre>Probab=20.81  E-value=1.8e+02  Score=24.44  Aligned_cols=24  Identities=21%  Similarity=0.479  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++++.||..+++-+++..+
</pre><pre>T Consensus       138 pgv~IG~~~vIgagsvV~~~v~~~  161 (221)
</pre><pre>T 6PXA_I          138 PGVHIGDGAIIGARAVITKNVAPY  161 (221)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>6PXA_L Chloramphenicol acetyltransferase; Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID, TRANSFERASE; HET: ACT, CL, FMT, GOL, TCH, SO4; 1.82A {Aliivibrio fischeri (strain ATCC 700601 / ES114)} SCOP: b.81.1.0
</pre><pre>Probab=20.81  E-value=1.8e+02  Score=24.44  Aligned_cols=24  Identities=21%  Similarity=0.479  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|++++.||..+++-+++..+
</pre><pre>T Consensus       138 pgv~IG~~~vIgagsvV~~~v~~~  161 (221)
</pre><pre>T 6PXA_L          138 PGVHIGDGAIIGARAVITKNVAPY  161 (221)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>3T57_A UDP-N-acetylglucosamine O-acyltransferase domain-containing protein; left-handed parallel beta helix, acyltransferase, lipid A biosynthesis, lipid synthesis, acylation, TRANSFERASE; 2.1A {Arabidopsis thaliana} SCOP: b.81.1.0
</pre><pre>Probab=20.76  E-value=1.8e+02  Score=24.33  Aligned_cols=24  Identities=25%  Similarity=0.283  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .++.|++++.||..+++.+|+...
</pre><pre>T Consensus       185 ~~~~ig~~~~i~~~s~v~~~v~~~  208 (305)
</pre><pre>T 3T57_A          185 QFCHIGSFAFIGGGSVVSQDVPKY  208 (305)
</pre><pre>T ss_dssp             TTCEECTTCEECTTCEECSBBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCeeCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>TIGR04008 WcaF; colanic acid biosynthesis acetyltransferase WcaF. This gene is one of the glycosyl transferases involved in the biosynthesis of colanic acid, an exopolysaccharide expressed in Enterobacteraceae species.
</pre><pre>Probab=20.65  E-value=2.1e+02  Score=22.68  Aligned_cols=25  Identities=16%  Similarity=0.348  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      ..|+.|++++.||..+++..++.++
</pre><pre>T Consensus       140 ~~g~~ig~~~~ig~~s~v~~~~~~~  164 (180)
</pre><pre>T TIGR04008       140 APGVTIGDGTVVGARSSVFKSLPAN  164 (180)
</pre><pre>T ss_pred             CCCCEECCCCEECCCcEecCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>6P89_C UDP-3-O-(3-hydroxymyristoyl)glucosamine N-acyltransferase; lps synthesis, TRANSFERASE, TRANSFERASE-INHIBITOR complex; HET: O3V, O4A; 1.4A {Escherichia coli}
</pre><pre>Probab=20.36  E-value=1.9e+02  Score=24.26  Aligned_cols=24  Identities=25%  Similarity=0.470  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEec-CCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDI-KSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI-~SG   28 (133)
</pre><pre>                      .++.|++++.||..+++.+++ ..+
</pre><pre>T Consensus       277 ~g~~Ig~~~~i~~~s~v~~~~~~~~  301 (343)
</pre><pre>T 6P89_C          277 GHMEICDKVTVTGMGMVMRPITEPG  301 (343)
</pre><pre>T ss_dssp             SSEEECSSEEECTTCEECSCBCSCE
</pre><pre>T ss_pred             CCCEeCCCcEEcCCcEEcCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>PF09937.11 ; DUF2169 ; Uncharacterized protein conserved in bacteria (DUF2169)
</pre><pre>Probab=20.27  E-value=1.2e+02  Score=27.03  Aligned_cols=25  Identities=20%  Similarity=0.380  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             cCCeeEEEEeeCCC----------CcCCCcCCccc
</pre><pre>Q FD01847867_021   98 HFPLEWQRTYSSRD----------ERTEGMFGRGW  122 (133)
</pre><pre>Q Consensus        98 ~lPl~WQR~Y~S~d----------~r~~GlfG~GW  122 (133)
</pre><pre>                      .+||.|++.|--.|          ....+..|.||
</pre><pre>T Consensus       114 ~~pL~ye~AfGG~~~~~~~~~~~~~~~~NP~G~G~  148 (293)
</pre><pre>T A9GHK2_SORC5/2  114 RSPLVYERAFGGVLKYDPEKGPLEIDQRNPVGIGF  148 (293)
</pre><pre>T ss_pred             EeecccccccCCcCCCCCCCCCcccCCCCCcccCC
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>d3hsqa_ b.81.1.0 (A:) automated matches {Leptospira interrogans [TaxId: 173]}
</pre><pre>Probab=20.15  E-value=2.1e+02  Score=23.85  Aligned_cols=26  Identities=19%  Similarity=0.141  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSGK   29 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SGK   29 (133)
</pre><pre>                      ..++.|++++.|+..+++-+||.++.
</pre><pre>T Consensus       155 ~~g~~Ig~~~~i~~gs~v~~~v~~~~  180 (259)
</pre><pre>T d3hsqa_         155 HQFCFVGDYSMVAGLAKVVQDVPPYS  180 (259)
</pre><pre>T ss_dssp             CTTCEECTTCEECSSEEECSBBCTTE
</pre><pre>T ss_pred             CCCCEECCCCEECCCCEECCccCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>6AG8_C Maltose O-acetyltransferase; maltose O-acetyltransferase, TRANSFERASE; HET: TRS; 1.34A {Escherichia coli (strain K12)} SCOP: b.81.1.0
</pre><pre>Probab=20.14  E-value=2.2e+02  Score=21.87  Aligned_cols=25  Identities=20%  Similarity=0.456  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             cccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    4 ENGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         4 ~~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      ..++.|++++.|+..+++.+++.++
</pre><pre>T Consensus       145 ~~~~~ig~~~~i~~~~~i~~~v~~~  169 (191)
</pre><pre>T 6AG8_C          145 NPGVTIGDNVVVASGAVVTKDVPDN  169 (191)
</pre><pre>T ss_dssp             CTTCEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCCEECCCCEECCCCEEcCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>4FCE_A Bifunctional protein GlmU; glmU. CSGID, NIAID, Structural Genomics, National Institute of Allergy and Infectious Diseases, Center for Structural Genomics; HET: GP1, EDO; 1.955A {Yersinia pestis CO92} SCOP: c.68.1.5, b.81.1.0
</pre><pre>Probab=20.02  E-value=2e+02  Score=25.04  Aligned_cols=24  Identities=29%  Similarity=0.405  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ccceecCCcEeCCCeeEEEecCCC
</pre><pre>Q FD01847867_021    5 NGVHVSGDVRIGGPLLVVRDIKSG   28 (133)
</pre><pre>Q Consensus         5 ~g~~vS~nVrIGG~~vvVRdI~SG   28 (133)
</pre><pre>                      .|+.|+.|++||..+++-+++..+
</pre><pre>T Consensus       414 ~g~~i~~~~~i~~~s~v~~~~~~~  437 (459)
</pre><pre>T 4FCE_A          414 APVTVANGATIGAGTTVTRDVAEN  437 (459)
</pre><pre>T ss_dssp             SSEEECTTCEECTTCEECSCBCTT
</pre><pre>T ss_pred             CCCEECCCCEECCCCEECCCCCCC
</pre><pre>
</pre><pre>
</pre>