<pre>Query         FD01848339_03324 
</pre><pre>Match_columns 113
</pre><pre>No_of_seqs    27 out of 33
</pre><pre>Neff          2.92253
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Tue Aug 24 15:45:20 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/5356680.hhr -oa3m ../results/5356680.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 6J0M_C Pvc8; assembly, Photorh  97.7  0.0016 1.5E-08   52.5  11.1   89   21-112   446-538 (538)
</pre><pre>  2 6RBK_C Afp8; Anti-feeding prop  97.7  0.0022   2E-08   51.8  11.1   91   21-112   437-529 (529)
</pre><pre>  3 3QR8_A Baseplate assembly prot  95.0     0.6 5.6E-06   34.3   8.0   62   49-111    98-160 (211)
</pre><pre>  4 3QR7_B Baseplate assembly prot  93.7     2.6 2.4E-05   28.6   8.5   61   50-111     3-64  (115)
</pre><pre>  5 PF12059.10 ; DUF3540 ; Protein  92.1     2.9 2.7E-05   33.2   7.9   51   61-111    40-97  (191)
</pre><pre>  6 6P2A_C CHIMERA OF BACTERIOPHAG  92.0     1.9 1.8E-05   32.8   6.5   46   66-111    36-87  (148)
</pre><pre>  7 6P2A_E CHIMERA OF BACTERIOPHAG  92.0     1.9 1.8E-05   32.8   6.5   46   66-111    36-87  (148)
</pre><pre>  8 4UHV_A VGRG1, VALINE-GLYCINE R  91.0     2.1   2E-05   37.0   6.5   62   49-111   565-626 (651)
</pre><pre>  9 6U9G_B VgrG; T6SS Central Spik  90.7      10 9.3E-05   28.6   9.0   78   31-113    83-160 (189)
</pre><pre> 10 3VTN_A Protein gp45; beta-heli  90.5     2.6 2.4E-05   27.7   5.3   46   65-111     4-51  (101)
</pre><pre> 11 PF10106.11 ; DUF2345 ; Unchara  89.8     9.5 8.8E-05   26.7   9.6   80   33-113    58-144 (145)
</pre><pre> 12 3AQJ_A Baseplate assembly prot  88.9      12 0.00012   26.9   8.1   60   51-111    23-83  (134)
</pre><pre> 13 4KU0_C Tail-associated lysozym  88.5     8.6 7.9E-05   24.4   6.6   48   66-113    44-91  (96)
</pre><pre> 14 PF04790.15 ; Sarcoglycan_1 ; S  85.9     9.1 8.5E-05   32.0   6.8   57   56-112   164-223 (253)
</pre><pre> 15 d1biaa2 b.34.1.1 (A:271-317) B  85.4     3.4 3.2E-05   23.8   3.1   32   57-88     16-47  (47)
</pre><pre> 16 3VTO_B Protein gp45; beta-heli  85.1     9.6 8.9E-05   26.9   5.7   46   65-111    10-57  (115)
</pre><pre> 17 3VTO_C Protein gp45; beta-heli  85.1     9.6 8.9E-05   26.9   5.7   46   65-111    10-57  (115)
</pre><pre> 18 6P20_C Baseplate central spike  84.0      18 0.00017   23.9   6.8   47   66-113    52-98  (112)
</pre><pre> 19 PF10106.11 ; DUF2345 ; Unchara  81.6     9.1 8.4E-05   26.7   4.5   47   61-107     9-55  (145)
</pre><pre> 20 6SK0_B Putative type VI secret  81.2      42 0.00039   30.9   9.3   78   34-113   686-763 (841)
</pre><pre> 21 6SK0_C Putative type VI secret  81.2      42 0.00039   30.9   9.3   78   34-113   686-763 (841)
</pre><pre> 22 4S37_B Phage baseplate protein  80.9      34 0.00031   24.9   7.8   56   56-111    94-152 (185)
</pre><pre> 23 4S37_O Phage baseplate protein  80.9      34 0.00031   24.9   7.8   56   56-111    94-152 (185)
</pre><pre> 24 d1pwaa_ b.42.1.1 (A:) Fibrobla  80.0     8.8 8.2E-05   29.1   4.3   46   23-70     24-70  (136)
</pre><pre> 25 d1k28a2 b.108.1.2 (A:362-576)   79.1      23 0.00021   27.7   6.3   46   66-111   162-207 (215)
</pre><pre> 26 3A1M_A chimera of thrombin cle  79.0      24 0.00022   26.2   6.1   46   67-113    58-103 (139)
</pre><pre> 27 3A1M_E chimera of thrombin cle  79.0      24 0.00022   26.2   6.1   46   67-113    58-103 (139)
</pre><pre> 28 7CBM_a Flagellar basal-body ro  78.3      34 0.00032   27.7   7.2   74   26-104    73-146 (251)
</pre><pre> 29 7CBM_b Flagellar basal-body ro  78.3      34 0.00032   27.7   7.2   74   26-104    73-146 (251)
</pre><pre> 30 d2dxua2 b.34.1.1 (A:189-235) B  73.5      19 0.00017   21.2   3.6   33   57-89     13-46  (47)
</pre><pre> 31 3PQI_A gene product 138; Beta-  72.4      41 0.00038   26.6   6.1   47   52-98    143-192 (247)
</pre><pre> 32 6SK0_B Putative type VI secret  70.1      63 0.00058   29.8   7.5  101    2-105   439-564 (841)
</pre><pre> 33 6SK0_C Putative type VI secret  70.1      63 0.00058   29.8   7.5  101    2-105   439-564 (841)
</pre><pre> 34 d1v1ha1 b.83.1.1 (A:319-392) A  69.4      48 0.00044   24.0   5.5   61   38-103    13-73  (74)
</pre><pre> 35 d4cjma_ b.42.1.0 (A:) automate  65.4      35 0.00033   25.4   4.4   35   36-70     31-67  (129)
</pre><pre> 36 4CJM_A FIBROBLAST GROWTH FACTO  65.2      43 0.00039   24.5   4.7   44   27-71     24-69  (141)
</pre><pre> 37 d1ijta_ b.42.1.1 (A:) Fibrobla  64.9      40 0.00037   25.2   4.6   46   23-70     22-68  (128)
</pre><pre> 38 2P39_A Fibroblast growth facto  64.7      49 0.00045   25.3   5.1   46   23-70     33-79  (155)
</pre><pre> 39 4M0N_A Conserved hypothetical   63.2 1.3E+02  0.0012   23.6  10.1   90   20-113    10-114 (232)
</pre><pre> 40 2J3P_B HEPARIN-BINDING GROWTH   61.1      67 0.00062   23.4   5.0   53   23-76     21-74  (134)
</pre><pre> 41 d2p39a_ b.42.1.0 (A:) automate  60.4      61 0.00057   25.2   5.0   49   26-76     32-81  (142)
</pre><pre> 42 4S36_A Phage baseplate protein  60.1      90 0.00083   20.9   6.2   46   65-111     8-57  (90)
</pre><pre> 43 d1q1ua_ b.42.1.1 (A:) Fibrobas  60.0      35 0.00033   25.9   3.6   35   36-70     34-69  (138)
</pre><pre> 44 3PQH_A gene product 138; Beta-  59.9 1.2E+02  0.0011   22.2   8.9   73   23-111    11-84  (127)
</pre><pre> 45 3F1R_A Fibroblast growth facto  59.9      36 0.00034   28.0   3.9   35   36-70     93-128 (211)
</pre><pre> 46 PF12503.10 ; CMV_1a_C ; Cucumb  59.7      33  0.0003   26.3   3.4   32   40-72     27-65  (81)
</pre><pre> 47 1WTH_A Tail-associated lysozym  59.5 1.7E+02  0.0016   28.0   8.5   65   45-110   505-574 (584)
</pre><pre> 48 1IJT_A fibroblast growth facto  58.6      56 0.00052   23.4   4.3   46   23-70     22-68  (128)
</pre><pre> 49 6M6E_A Fibroblast growth facto  58.2      89 0.00083   22.2   5.2   46   23-70     24-70  (128)
</pre><pre> 50 PF14563.8 ; DUF4444 ; Domain o  58.0      48 0.00045   20.2   3.4   29   57-85     11-39  (42)
</pre><pre> 51 d2fdbm1 b.42.1.1 (M:34-180) Fi  57.8      59 0.00055   24.9   4.5   35   36-70     48-84  (147)
</pre><pre> 52 4PEU_A Uncharacterized protein  57.6      57 0.00053   28.0   4.8   49   65-113   148-196 (313)
</pre><pre> 53 3OGF_B de novo designed dimeri  54.1 1.1E+02   0.001   20.3   4.8   35   36-70     35-70  (90)
</pre><pre> 54 PF12445.10 ; FliC ; Flagellin   53.7      34 0.00031   27.2   2.8   40   63-102    81-120 (125)
</pre><pre> 55 PF07481.13 ; DUF1521 ; Domain   53.4   2E+02  0.0018   24.2   7.0   67   47-113    16-97  (169)
</pre><pre> 56 d3hqxa_ b.82.1.0 (A:) automate  52.8 1.3E+02  0.0012   22.6   5.4   59   50-108    24-89  (105)
</pre><pre> 57 d1ihka_ b.42.1.1 (A:) Fibrobla  51.9      63 0.00059   25.6   4.0   35   36-70     39-74  (157)
</pre><pre> 58 d1nuna_ b.42.1.1 (A:) Fibrobla  51.5      72 0.00067   24.3   4.1   35   36-70     38-73  (139)
</pre><pre> 59 5W21_B Fibroblast growth facto  49.3      95 0.00088   26.4   4.8   46   23-70     79-125 (226)
</pre><pre> 60 4PEU_A Uncharacterized protein  49.3 3.1E+02  0.0029   23.9   8.3   80   31-113   203-284 (313)
</pre><pre> 61 3O4C_A de novo designed beta-t  49.0 1.1E+02   0.001   22.4   4.5   35   36-70     45-80  (142)
</pre><pre> 62 PF14262.8 ; Cthe_2159 ; Carboh  47.8 3.1E+02  0.0029   23.4   8.0   69   43-111    51-130 (292)
</pre><pre> 63 2XSK_A CSGC; CHAPERONE; HET: A  45.6 1.5E+02  0.0014   22.1   4.8   67   41-113     8-74  (110)
</pre><pre> 64 6SK0_B Putative type VI secret  45.4      64  0.0006   29.8   3.5   51   57-107   622-673 (841)
</pre><pre> 65 6SK0_C Putative type VI secret  45.4      64  0.0006   29.8   3.5   51   57-107   622-673 (841)
</pre><pre> 66 4UHV_A VGRG1, VALINE-GLYCINE R  44.9 4.1E+02  0.0038   24.0  11.9   99   15-113   450-582 (651)
</pre><pre> 67 d3vgfa3 b.71.1.1 (A:491-557) G  43.7   1E+02 0.00097   22.8   3.7   48   66-113     9-62  (67)
</pre><pre> 68 PF00167.20 ; FGF ; Fibroblast   43.1 1.5E+02  0.0014   22.3   4.5   54   22-76     21-79  (136)
</pre><pre> 69 6MJP_C Lipopolysaccharide expo  43.0 2.3E+02  0.0021   20.5   5.9   48   63-112    86-138 (191)
</pre><pre> 70 PF04773.15 ; FecR ; FecR prote  42.9   2E+02  0.0018   19.7   6.4   70   42-111     2-78  (104)
</pre><pre> 71 5VTG_A Translocation and assem  42.7 2.8E+02  0.0026   21.5   6.3   51   61-113    68-125 (184)
</pre><pre> 72 PF10988.10 ; DUF2807 ; Putativ  42.3 2.6E+02  0.0024   20.9   6.8   64   41-104   110-198 (203)
</pre><pre> 73 PF11142.10 ; DUF2917 ; Protein  42.2   2E+02  0.0019   19.6   6.5   52   54-105     2-63  (65)
</pre><pre> 74 4OW4_A Beta-terfoil designed b  41.0   2E+02  0.0019   19.6   4.6   35   36-70     30-65  (123)
</pre><pre> 75 PF13540.8 ; RCC1_2 ; Regulator  39.5      84 0.00078   14.5   2.4   18   59-76     10-27  (30)
</pre><pre> 76 1RG8_A Heparin-binding growth   39.4 1.4E+02  0.0013   22.4   3.9   35   36-70     45-80  (146)
</pre><pre> 77 6KTR_C Fibroblast growth facto  39.1   1E+02 0.00094   25.8   3.5   46   23-70     42-88  (218)
</pre><pre> 78 3GVZ_A Uncharacterized protein  38.8 1.4E+02  0.0013   24.2   4.1   44   30-73     45-94  (299)
</pre><pre> 79 PF18287.3 ; Hfx_Cass5 ; Integr  37.6      78 0.00072   23.6   2.4   19   60-78      5-23  (77)
</pre><pre> 80 1NUN_A Fibroblast growth facto  37.5 1.7E+02  0.0016   21.8   4.1   35   36-70     43-78  (145)
</pre><pre> 81 TIGR04248 SCM_PqqD_rel; SynChl  37.2 1.1E+02 0.00098   20.1   2.8   33   35-67      1-33  (84)
</pre><pre> 82 PF12545.10 ; DUF3739 ; Filamen  36.8 3.9E+02  0.0036   21.4   6.4   69   29-102    26-107 (111)
</pre><pre> 83 2EAY_A Biotin [acetyl-CoA-carb  36.7 1.5E+02  0.0014   23.7   3.9   34   57-90    200-233 (233)
</pre><pre> 84 3IF4_D Integron Cassette Prote  36.4      81 0.00075   25.4   2.5   20   59-78     28-47  (119)
</pre><pre> 85 3MY2_A Lipopolysaccharide expo  35.5 3.4E+02  0.0032   20.3   5.9   47   63-111    70-121 (175)
</pre><pre> 86 3IZO_G Fiber; human adenovirus  34.9 3.8E+02  0.0036   26.3   6.8   67   12-105    16-82  (581)
</pre><pre> 87 3IZO_H Fiber; human adenovirus  34.9 3.8E+02  0.0036   26.3   6.8   67   12-105    16-82  (581)
</pre><pre> 88 5NY0_A L. reuteris SRRP bindin  34.9 4.5E+02  0.0042   23.2   6.7   75   30-104   239-319 (368)
</pre><pre> 89 PF10610.11 ; Tafi-CsgC ; Thin   34.8 2.7E+02  0.0025   20.0   4.7   67   41-113    12-78  (99)
</pre><pre> 90 3LYC_B Putative pectinase; Str  34.8 3.9E+02  0.0036   20.7   6.8   64   41-104   151-219 (241)
</pre><pre> 91 3LYC_G Putative pectinase; Str  34.8 3.9E+02  0.0036   20.7   6.8   64   41-104   151-219 (241)
</pre><pre> 92 TIGR04409 LptC_YrbK; LPS expor  34.3 3.5E+02  0.0033   20.2   7.2   52   61-112    47-102 (180)
</pre><pre> 93 PF13100.8 ; OstA_2 ; OstA-like  33.6   3E+02  0.0028   19.2   4.8   43   69-112     4-48  (159)
</pre><pre> 94 TIGR00121 birA_ligase; birA, b  33.3 3.4E+02  0.0031   21.9   5.4   48   35-88    190-237 (237)
</pre><pre> 95 3JX8_A Putative lipoprotein; S  32.1 4.4E+02  0.0041   20.6   6.9   65   41-105   161-230 (250)
</pre><pre> 96 3JX8_F Putative lipoprotein; S  32.1 4.4E+02  0.0041   20.6   6.9   65   41-105   161-230 (250)
</pre><pre> 97 PF09992.11 ; NAGPA ; Phosphodi  31.6 1.4E+02  0.0013   22.5   3.0   36   37-73     84-119 (191)
</pre><pre> 98 2N59_A Putative uncharacterize  31.3 3.9E+02  0.0036   19.7   5.6   67   41-113     6-72  (106)
</pre><pre> 99 3BFM_A Biotin protein ligase-l  31.1 1.8E+02  0.0017   23.3   3.7   32   57-88    203-234 (235)
</pre><pre>100 4OEE_A Fibroblast growth facto  30.7 2.4E+02  0.0023   21.6   4.1   47   23-70     45-92  (155)
</pre><pre>101 2P23_B Fibroblast growth facto  30.6 2.1E+02  0.0019   23.4   3.9   47   23-71     42-89  (194)
</pre><pre>102 1BIA_A BirA BIFUNCTIONAL PROTE  30.4 2.1E+02  0.0019   24.2   4.0   36   57-92    286-321 (321)
</pre><pre>103 PF02237.19 ; BPL_C ; Biotin pr  30.4   2E+02  0.0019   16.2   3.2   30   58-87     18-47  (48)
</pre><pre>104 1NNX_A Protein ygiW; ygiW, Str  30.2   4E+02  0.0037   19.5   5.7   59   49-110    44-106 (109)
</pre><pre>105 PF06739.13 ; SBBP ; Beta-prope  29.8 1.8E+02  0.0016   15.3   2.5   16   58-73     15-30  (37)
</pre><pre>106 d2oyza1 b.82.1.22 (A:2-94) Unc  29.7 3.8E+02  0.0035   19.5   4.7   58   49-106    14-75  (93)
</pre><pre>107 PF03961.15 ; FapA ; Flagellar   29.6 5.2E+02  0.0048   23.4   6.3   58   45-104   209-270 (441)
</pre><pre>108 6ORJ_A PHIKZ164; Membrane punc  29.5 8.3E+02  0.0077   23.0   9.7   77   37-113   188-264 (293)
</pre><pre>109 2FDB_N fibroblast growth facto  28.6 3.4E+02  0.0032   20.8   4.5   47   23-70     47-95  (164)
</pre><pre>110 3PET_B Putative adhesin; RIGHT  28.2 4.8E+02  0.0045   19.9   7.1   64   41-104   132-200 (221)
</pre><pre>111 3P6J_A de novo designed beta-t  28.2 3.9E+02  0.0037   18.8   4.5   50   26-75     93-142 (142)
</pre><pre>112 1PWA_A Fibroblast growth facto  27.9 3.3E+02  0.0031   21.0   4.4   46   23-70     30-76  (162)
</pre><pre>113 4V02_C PROBABLE SEPTUM SITE-DE  27.8 5.5E+02  0.0051   20.3   5.9   52   57-110    14-76  (128)
</pre><pre>114 6SK0_B Putative type VI secret  27.0 2.5E+02  0.0023   26.3   4.2   91   20-111   629-720 (841)
</pre><pre>115 6SK0_C Putative type VI secret  27.0 2.5E+02  0.0023   26.3   4.2   91   20-111   629-720 (841)
</pre><pre>116 d1bfga_ b.42.1.1 (A:) Basic FG  26.9 2.9E+02  0.0027   20.6   3.9   35   36-70     30-65  (126)
</pre><pre>117 3JX8_A Putative lipoprotein; S  26.8 5.5E+02  0.0051   20.1   7.4   79   34-112    59-160 (250)
</pre><pre>118 3JX8_F Putative lipoprotein; S  26.8 5.5E+02  0.0051   20.1   7.4   79   34-112    59-160 (250)
</pre><pre>119 d1pg6a_ b.82.5.2 (A:) Hypothet  26.5 6.2E+02  0.0058   20.6   6.9  102    6-112    76-185 (227)
</pre><pre>120 PF12992.9 ; DUF3876 ; Domain o  26.5 4.5E+02  0.0042   18.9   6.2   65   38-102    23-92  (93)
</pre><pre>121 PF15911.7 ; WD40_3 ; WD domain  26.3 3.7E+02  0.0035   17.9   4.6   35   40-74      1-36  (57)
</pre><pre>122 TIGR01644 phage_P2_V; phage ba  25.9 5.6E+02  0.0052   19.8   5.5   47   65-111    87-138 (190)
</pre><pre>123 6XGR_B YSD1_22 major tail prot  25.6 6.2E+02  0.0058   22.3   6.0   87   15-101   150-260 (381)
</pre><pre>124 6XGR_F YSD1_22 major tail prot  25.6 6.2E+02  0.0058   22.3   6.0   87   15-101   150-260 (381)
</pre><pre>125 d1rg8a1 b.42.1.1 (A:1-137) Aci  25.6   4E+02  0.0037   20.4   4.4   35   36-70     39-74  (137)
</pre><pre>126 d1qqka_ b.42.1.1 (A:) Keratino  25.4 3.9E+02  0.0036   20.2   4.3   35   36-70     31-66  (132)
</pre><pre>127 3V9F_C Two-component system se  24.8 7.5E+02   0.007   20.9   6.4   47   33-79    597-651 (781)
</pre><pre>128 4Q35_A LPS-assembly protein Lp  24.2 1.1E+03    0.01   22.7   8.4   84   12-111    37-127 (802)
</pre><pre>129 4RU3_A puncturing protein gp41  24.1 6.7E+02  0.0062   20.1   5.7   56   49-111   128-184 (221)
</pre><pre>130 1Q7F_A brain tumor CG10719-PA;  23.8   5E+02  0.0046   18.5   6.0   54   28-81      2-55  (286)
</pre><pre>131 1QQL_A FIBROBLAST GROWTH FACTO  23.6 2.2E+02  0.0021   20.9   2.7   35   36-70     39-74  (140)
</pre><pre>132 PF06715.14 ; Gp5_C ; Gp5 C-ter  23.2   3E+02  0.0028   15.7   2.9   24   83-106     1-24  (24)
</pre><pre>133 1RWR_A Filamentous hemagglutin  22.6 5.9E+02  0.0054   21.1   5.1   59   51-113   229-291 (301)
</pre><pre>134 6NU2_AW 28S ribosomal protein   22.3 1.8E+02  0.0017   22.6   2.1   19   67-85     26-44  (97)
</pre><pre>135 6RIQ_A MinC; Bacterial cell di  22.2 7.6E+02   0.007   20.0   7.2   65   14-87     12-78  (144)
</pre><pre>136 6RIQ_B MinC; Bacterial cell di  22.2 7.6E+02   0.007   20.0   7.2   65   14-87     12-78  (144)
</pre><pre>137 5V1V_B TbiB1; Lasso Peptide, R  21.5 3.8E+02  0.0036   16.3   3.2   34   34-67      3-36  (91)
</pre><pre>138 6ITX_C Fiber protein; avian ad  21.3 8.8E+02  0.0082   22.6   6.3   62   47-113    25-87  (301)
</pre><pre>139 6ITX_F Fiber protein; avian ad  21.3 8.8E+02  0.0082   22.6   6.3   62   47-113    25-87  (301)
</pre><pre>140 2DXU_A biotin--[acetyl-CoA-car  21.2 4.5E+02  0.0042   21.0   4.1   33   57-89    201-234 (235)
</pre><pre>141 6JF2_A Flagellar basal-body ro  21.1 7.7E+02  0.0072   19.7   6.0   66   32-104    52-118 (185)
</pre><pre>142 4OPW_A Uncharacterized protein  21.1 7.4E+02  0.0068   19.4   9.0   79   34-112    57-150 (241)
</pre><pre>143 1IHK_A GLIA-ACTIVATING FACTOR;  20.6 2.9E+02  0.0027   21.8   3.0   35   36-70     56-91  (174)
</pre><pre>144 PF10246.11 ; MRP-S35 ; Mitocho  20.4 1.9E+02  0.0017   22.7   1.9   19   67-85     29-47  (105)
</pre><pre>145 PF03775.18 ; MinC_C ; Septum f  20.3 6.7E+02  0.0062   18.7   6.1   54   54-109     5-64  (101)
</pre><pre>
</pre><pre>No 1
</pre><pre>>6J0M_C Pvc8; assembly, Photorhabdus asymbiotica, PVC, contractile injection system, bacteriophage-like, PROTEIN TRANSPORT; 3.9A {Photorhabdus asymbiotica}
</pre><pre>Probab=97.74  E-value=0.0016  Score=52.52  Aligned_cols=89  Identities=12%  Similarity=0.242  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCcceeeecCC----ceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEe
</pre><pre>Q FD01848339_033   21 APSMPSTSATKDFIYSRNG----LMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIE   96 (113)
</pre><pre>Q Consensus        21 aP~~p~~s~tkD~V~~RNg----llE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~   96 (113)
</pre><pre>                      .|..+.-.........|+|    .|......+.+.+..  ....|-|+.++.+.+.+.+.+-|.+ .++++++++++.++
</pre><pre>T Consensus       446 ~~~~~~~~~~~~~~~~~~g~~~~~l~~~~~~~~i~~~~--~~~~~~~~~~~~i~~~~~~~~~~~~-~~~~~~~~~~~~~~  522 (538)
</pre><pre>T 6J0M_C          446 APFEPTQDNREKVLIVKKGEAQQQLVIDGKEKMIRINA--GENQIMLQQDKDISLSTKKELTLKA-QTMNATMDKSLAMS  522 (538)
</pre><pre>T ss_dssp             CSSCCCTTCCEEEECEECSSCEECCEEETTTTEEEEEB--SSSCEEEECSSCCCCBCSSEEEEEE-SEEEEEESSCEEEE
</pre><pre>T ss_pred             CCCCCCCccccEEEEEecCCceeEEEEECCCCeEEEEe--CCceEEEcCCCcEEEcCCCeEEEee-cceEEEeCCceeEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eCeeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   97 AGGVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        97 a~g~~~i~A~~v~idk  112 (113)
</pre><pre>                      +++.+.|+|..|.|++
</pre><pre>T Consensus       523 ~~~~i~i~~~~i~~~~  538 (538)
</pre><pre>T 6J0M_C          523 GKNSVEIKGAKINLTQ  538 (538)
</pre><pre>T ss_dssp             BSSCEEEECSSEEEC-
</pre><pre>T ss_pred             ecceEEEecceeecCC
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>6RBK_C Afp8; Anti-feeding prophage, secretion system, AFP, contractile, VIRUS LIKE PARTICLE, baseplate; 3.4A {Serratia entomophila}
</pre><pre>Probab=97.68  E-value=0.0022  Score=51.81  Aligned_cols=91  Identities=14%  Similarity=0.221  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCcceeeecCCc--eEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeC
</pre><pre>Q FD01848339_033   21 APSMPSTSATKDFIYSRNGL--MEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAG   98 (113)
</pre><pre>Q Consensus        21 aP~~p~~s~tkD~V~~RNgl--lE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~   98 (113)
</pre><pre>                      .|..+....+...+..+++-  .++.-...+-+|+-...+..|-|+.+|.+.+.+.+++.|.+ .++++++++++.+.+.
</pre><pre>T Consensus       437 ~~~~~~~~~~~~~~~~~~~~~~~~~~~d~~~~~~~~~~~~~~~~l~~~~~~~~~~~~~i~~~~-~~~~~~~~~~~~~~~~  515 (529)
</pre><pre>T 6RBK_C          437 APFPPDEKNNRKGLIVSQADQTQALMIDTEEKTLRLMAGDNTLTLTGEGNLTMSTPNALQLQA-DTLGLQADSNLSIAGK  515 (529)
</pre><pre>T ss_dssp             CSSCCCSSCCBCCCCCBSSSCBCSEECBTTTTCCEECSSSCCCCCCTTSCCCCCCSSCCCCCC-SSCCCCCSSCCCCCCS
</pre><pre>T ss_pred             CCCCCCcccCceEEEEecCCCceEEEEeCCCCEEEEEcCCcEEEEcCCCcEEEcCCCeEEEec-CcEEEEeCCceEEecc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   99 GVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        99 g~~~i~A~~v~idk  112 (113)
</pre><pre>                      +.+.|++..|+|++
</pre><pre>T Consensus       516 ~~~~i~~~~i~~~~  529 (529)
</pre><pre>T 6RBK_C          516 QQVEITSAKINMKK  529 (529)
</pre><pre>T ss_dssp             SCCCCCCSSCCC--
</pre><pre>T ss_pred             CcEEEEeceeeecC
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>3QR8_A Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; HET: MSE; 2.03A {Enterobacteria phage P2}
</pre><pre>Probab=95.00  E-value=0.6  Score=34.29  Aligned_cols=62  Identities=15%  Similarity=0.155  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ceeEEEcCCCCEEEecC-CCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   49 AWSVTHLGNGTTIGFNE-AGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        49 G~siTh~asGs~iG~Ne-sGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      .........|..|-+++ .+++++...+.+.|.+.|+++++ ...+.+++.+.+.|++..+.+.
</pre><pre>T Consensus        98 ~~~~~~~~~G~~i~~~~~~~~i~~~~~~~i~i~~~g~v~i~-~~~~~~~a~~~i~i~~~~~~~~  160 (211)
</pre><pre>T 3QR8_A           98 DALHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTAT-VPVVMVKASTRVTLDTPEVVCT  160 (211)
</pre><pre>T ss_dssp             SCEEEECTTSCEEEEETTTTEEEEESCSCCCCCCSSCCCCC-CSCCCCCCSSCCCCCSSCCCCC
</pre><pre>T ss_pred             CcEEEEcCCCCEEEEECCCCeEEEeeCceEEEEcCCcEEEE-cCeEEEecCCeEEEeCCceEEc
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>3QR7_B Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; 0.94A {Enterobacteria phage P2}
</pre><pre>Probab=93.72  E-value=2.6  Score=28.60  Aligned_cols=61  Identities=15%  Similarity=0.183  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEEcCCCCEEEecCCCCEEEEe-cCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   50 WSVTHLGNGTTIGFNEAGELFAIS-QGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        50 ~siTh~asGs~iG~NesGq~yi~s-~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ...+....|+.+-.++.++.+.+. .+.+.+.+.+.++++ ...+.+.+.+.+.+.+..+.++
</pre><pre>T Consensus         3 ~~~~~~~dG~~~~yd~~~~~l~~~~~g~i~v~~~~~~~~~-~~~~~~~~~~~v~~~~~~~~~~   64 (115)
</pre><pre>T 3QR7_B            3 ALHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTAT-VPVVMVKASTRVTLDTPEVVCT   64 (115)
</pre><pre>T ss_dssp             CCEEECTTSCEEEEETTTTEEEC-CCSCCCCCCSSCCCCC-CSCCCCCCSSCCCCCSSCCCCS
</pre><pre>T ss_pred             eEEEEcCCCCEEEEeCCCCeEEEEeccceEEEcCccEEEE-CCeEEEEeCceEEEeCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF12059.10 ; DUF3540 ; Protein of unknown function (DUF3540)
</pre><pre>Probab=92.15  E-value=2.9  Score=33.18  Aligned_cols=51  Identities=31%  Similarity=0.543  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             EEecCCCCEEEEe------cCceEEEecCCeEEEe-CcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   61 IGFNEAGELFAIS------QGGAFISAAGDMNIKA-GGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        61 iG~NesGq~yi~s------~~~~fi~~~G~~t~k~-~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      +-..++|+.||++      ....-|...|++.+.+ .|++.|.+...|.|.|.++.++
</pre><pre>T Consensus        40 v~~~~~~~~yIlaVLer~~~~~~~l~~~g~~~l~a~~G~l~l~a~~~l~l~a~~~~l~   97 (191)
</pre><pre>T F3YX79_DESAF/1   40 LSVDGAGSAFILSILRRPEGTEAIVSFDGPASFRVAQGNLTLAAQGEIGLAAGRVAVD   97 (191)
</pre><pre>T ss_pred             EEEcCCCcEEEEEEEECCCCCceEEEeCCCeEEEEcCCcEEEeeCCeEEEeeceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>6P2A_C CHIMERA OF BACTERIOPHAGE OBP GP146 AND A T4 GP5; Bacteriophage OBP, membrane piercing, central spike, cell puncturing device, beta helix, T4 gp5, contractile injection system; HET: STE, ELA, PLM; 1.9A {Enterobacteria phage T4}
</pre><pre>Probab=92.01  E-value=1.9  Score=32.83  Aligned_cols=46  Identities=17%  Similarity=0.367  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEe------eeeeee
</pre><pre>Q FD01848339_033   66 AGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRA------AEVKVD  111 (113)
</pre><pre>Q Consensus        66 sGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A------~~v~id  111 (113)
</pre><pre>                      .|...+.-.++..+...|+++++..|++.+++++.+.+.|      +.+.++
</pre><pre>T Consensus        36 ~g~~~~~V~Gn~~i~v~G~~~~~v~G~~~~~vgg~~~~~~~~~~~~g~iti~   87 (148)
</pre><pre>T 6P2A_C           36 KGDATTLVEGNQTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSKGNVTHE   87 (148)
</pre><pre>T ss_dssp             SSCEEEEESSCEEEEESSCEEEEESSCEEEEESSEEEEEEEEEEEEEEEEEE
</pre><pre>T ss_pred             cCCeEEEEeCCEEEEEcCCEEEEEeCcEEEEeCCcEEEEeceeecCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>6P2A_E CHIMERA OF BACTERIOPHAGE OBP GP146 AND A T4 GP5; Bacteriophage OBP, membrane piercing, central spike, cell puncturing device, beta helix, T4 gp5, contractile injection system; HET: ELA, PLM, STE; 1.9A {Enterobacteria phage T4}
</pre><pre>Probab=92.01  E-value=1.9  Score=32.83  Aligned_cols=46  Identities=17%  Similarity=0.367  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEe------eeeeee
</pre><pre>Q FD01848339_033   66 AGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRA------AEVKVD  111 (113)
</pre><pre>Q Consensus        66 sGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A------~~v~id  111 (113)
</pre><pre>                      .|...+.-.++..+...|+++++..|++.+++++.+.+.|      +.+.++
</pre><pre>T Consensus        36 ~g~~~~~V~Gn~~i~v~G~~~~~v~G~~~~~vgg~~~~~~~~~~~~g~iti~   87 (148)
</pre><pre>T 6P2A_E           36 KGDATTLVEGNQTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSKGNVTHE   87 (148)
</pre><pre>T ss_dssp             SSCEEEEESSCEEEEESSCEEEEESSCEEEEESSEEEEEEEEEEEEEEEEEE
</pre><pre>T ss_pred             cCCeEEEEeCCEEEEEcCCEEEEEeCcEEEEeCCcEEEEeceeecCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>4UHV_A VGRG1, VALINE-GLYCINE REPEAT PROTEIN G1; STRUCTURAL PROTEIN, VGRG1, VIRULENCE, TOXIN, EFFECTOR, PUNCTURING DEVICE, SPIKE, T6SS, P. AERUGINOSA; HET: CL, NA, MSE; 2.0A {PSEUDOMONAS AERUGINOSA}
</pre><pre>Probab=90.97  E-value=2.1  Score=37.03  Aligned_cols=62  Identities=21%  Similarity=0.359  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   49 AWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        49 G~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      +-.++....++.+-++.++.+-+.+. ++-+.++++++++.++.+.+.+.|.+.+++..+.+.
</pre><pre>T Consensus       565 ~~~~~~~~~~~~~~~~~~~~~~~~~~-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  626 (651)
</pre><pre>T 4UHV_A          565 GAQIRLVCGKSVVEFNADGTINISGS-AFNLYASGNGNIDTGGRLDLNSGGASEVDAKGKGVQ  626 (651)
</pre><pre>T ss_dssp             SSEEEEEETTEEEEEESSCCEEEECS-CCCCCSCCCCCCCCCSSCCSSCSSCSCCCCTTCCCH
</pre><pre>T ss_pred             CCeEEEEeCCcEEEECCCCeEEEeCC-cEEEEeCCceEEEeCCcEEEEeCCCceeccCCCcce
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>6U9G_B VgrG; T6SS Central Spike, Complex, TRANSPORT PROTEIN; 3.98A {Francisella tularensis subsp. novicida (strain U112)}
</pre><pre>Probab=90.74  E-value=10  Score=28.55  Aligned_cols=78  Identities=14%  Similarity=0.205  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             cceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeee
</pre><pre>Q FD01848339_033   31 KDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKV  110 (113)
</pre><pre>Q Consensus        31 kD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~i  110 (113)
</pre><pre>                      .+.+....+-++....++-..|.    +..|-+...+....+.++.+-+.+.. +.+++.+++.+++.+.+.+....|.+
</pre><pre>T Consensus        83 ~~~~~~~~~~~~~~~~~~~i~i~----~~~i~l~~~~~~i~i~~~~i~i~~~~-v~i~~~~~~~i~~~~~~~~~~~~v~i  157 (189)
</pre><pre>T 6U9G_B           83 DSKISITEKEILLATKKSSIMLS----EDKIVIKIGNSLIILDDSNISLESAT-INIKSSANINIQASQNIDIKSLNNSI  157 (189)
</pre><pre>T ss_dssp             SCCEEECSSCEEEEETTEEEEEE----TTEEEEEETTEEEEECSSCEEEEESE-EEECCSSCEEECCSSCCCCCCCSCCC
</pre><pre>T ss_pred             CcEEEEeCCEEEEEcCCCEEEEc----CCeEEEEECCEEEEEcCCCEEEEcCc-EEEEcCCceEEecCCceEeeccceEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eCC
</pre><pre>Q FD01848339_033  111 DKG  113 (113)
</pre><pre>Q Consensus       111 dkg  113 (113)
</pre><pre>                      +-+
</pre><pre>T Consensus       158 ~a~  160 (189)
</pre><pre>T 6U9G_B          158 KAD  160 (189)
</pre><pre>T ss_dssp             CCC
</pre><pre>T ss_pred             Eec
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>3VTN_A Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.75A {Enterobacteria phage Mu}
</pre><pre>Probab=90.46  E-value=2.6  Score=27.74  Aligned_cols=46  Identities=15%  Similarity=0.188  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEEecCceEEE--ecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   65 EAGELFAISQGGAFIS--AAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        65 esGq~yi~s~~~~fi~--~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ++|++.++....-||.  ..|.+.+++. .+.+++...+.+.+..+.+.
</pre><pre>T Consensus         4 ~~G~~~~y~~~~~~i~~~~~g~i~i~~~-~~~i~~~~~~~~~~~~i~~~   51 (101)
</pre><pre>T 3VTN_A            4 EPGDAGIYHHEGHRIRLTKDGRCIITCK-TVEVYADESMTVDTPRTTFT   51 (101)
</pre><pre>T ss_dssp             -------------CEEECTTSCEEECSS-CCCCCCSSCCCCCCSCCCCS
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEcCCCEEEEeCC-eEEEEcCCceEEeCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>PF10106.11 ; DUF2345 ; Uncharacterized protein conserved in bacteria (DUF2345)
</pre><pre>Probab=89.77  E-value=9.5  Score=26.65  Aligned_cols=80  Identities=10%  Similarity=0.224  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eeeecCCceEEEecCCceeEE------EcCCCCEEEecCCC-CEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   33 FIYSRNGLMEIRNAGGAWSVT------HLGNGTTIGFNEAG-ELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        33 ~V~~RNgllE~RtagGG~siT------h~asGs~iG~NesG-q~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      .++.+.+-++...+.|-.++.      ...+...+.+.... .+.+.+...+-+.++|....-.++.|.+.+.|.+.++ 
</pre><pre>T Consensus        58 ~~~~~~~~~~~~a~~g~~~~~a~~~~~~~~a~~~~~~~s~~~~~~~~a~~~i~~~~~g~~i~~~~~~i~~~~~g~~~~~-  136 (145)
</pre><pre>T X2H4H4_9NEIS/6   58 QVLAQTEDLNLVAGKKTARLASHGSAVEITAAKKISVQSVGDEVLVNAKSGIKLASGGAYIEIKGGKISLYSPGPIEYK-  136 (145)
</pre><pre>T ss_pred             eeeeccCCeEEEECCCcEEEEeCCCceEEEecCcEEEEecCCEEEEEcCCeEEEEeCCcEEEEeCCEEEEEcCCceEEE-
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeeeeCC
</pre><pre>Q FD01848339_033  106 AEVKVDKG  113 (113)
</pre><pre>Q Consensus       106 ~~v~idkg  113 (113)
</pre><pre>                      +.+.+..+
</pre><pre>T Consensus       137 ~~~~~~~~  144 (145)
</pre><pre>T X2H4H4_9NEIS/6  137 GNHQFQGP  144 (145)
</pre><pre>T ss_pred             ceeeecCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>3AQJ_A Baseplate assembly protein V; bacteriophage, tail spike, iron binding, beta-helix, infection, METAL BINDING PROTEIN; 1.27A {Enterobacteria phage P2}
</pre><pre>Probab=88.95  E-value=12  Score=26.88  Aligned_cols=60  Identities=15%  Similarity=0.225  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             eEEEcCCCCEEEecCCCCEEEEe-cCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   51 SVTHLGNGTTIGFNEAGELFAIS-QGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        51 siTh~asGs~iG~NesGq~yi~s-~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      .......|+.|-.+..+.-+.+. .+.+-|.+.+.++++.. .+.+++.+.+.+.+..+.+.
</pre><pre>T Consensus        23 ~~~~~~dG~~~~yd~~~~~l~i~~~g~i~i~~~~~i~~~~~-~~~i~~~~~v~~~~~~i~~~   83 (134)
</pre><pre>T 3AQJ_A           23 LHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTATVP-VVMVKASTRVTLDTPEVVCT   83 (134)
</pre><pre>T ss_dssp             EEEECTTSCEEEEETTTTEEEEEEEEEEEEEEEEEEEEECS-EEEEECSSEEEEESSEEEEE
</pre><pre>T ss_pred             EEEEcCCCCEEEEeCCCCEEEEEeCceeEEEcCceEEEECC-eEEEEeCcceEEeCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>4KU0_C Tail-associated lysozyme; PAAR-repeat motif, membrane piercing, type VI secretion system, T6SS, cell puncturing device, beta-helix, gp5-gp27 protein complex; HET: ELA, STE, PLM, EDO; 1.15A {Enterobacteria phage T4}
</pre><pre>Probab=88.47  E-value=8.6  Score=24.43  Aligned_cols=48  Identities=17%  Similarity=0.284  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   66 AGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        66 sGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      .+...+......++.+++++++.+++.+.+.++..+.+.++.|.|...
</pre><pre>T Consensus        44 ~~~~~~~~g~~~~~~~~~~~~~~~g~~~~i~~~~~~~~~~~~i~i~g~   91 (96)
</pre><pre>T 4KU0_C           44 EGNQTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSISSGQYTIDGS   91 (96)
</pre><pre>T ss_dssp             SSCEEEEESSCEEEEESSCEEEEESSEEEEEESEEEEEESSEEEEECS
</pre><pre>T ss_pred             eCCEEEEECCcEEEEEeceEEEEeCCcEEEEeCceEEEeCceEEEeCc
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF04790.15 ; Sarcoglycan_1 ; Sarcoglycan complex subunit protein
</pre><pre>Probab=85.87  E-value=9.1  Score=31.96  Aligned_cols=57  Identities=12%  Similarity=0.228  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEecCCCC-EEEEecCceEEEe-cCCeEEEeCcceeEee-CeeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   56 GNGTTIGFNEAGE-LFAISQGGAFISA-AGDMNIKAGGNMGIEA-GGVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        56 asGs~iG~NesGq-~yi~s~~~~fi~~-~G~~t~k~~g~i~l~a-~g~~~i~A~~v~idk  112 (113)
</pre><pre>                      ..+..+-+.-..+ +.+.++..+++.+ .+++++.+..+|.|++ .|++.+.|..|.+++
</pre><pre>T Consensus       164 ~~~~dL~leS~~~~l~l~g~egv~i~s~~~~i~~~s~~dI~L~S~~G~I~l~a~~i~l~~  223 (253)
</pre><pre>T E3LFZ6_CAERE/7  164 QPDSPLRLESPTRSVSIDAAQDIEILAAAGEVSVNSLLDISISSKQGEIRLESSSIYMEK  223 (253)
</pre><pre>T ss_pred             CCCCCceeeeCCcEEEEecCCeEEEEEecceEEEEecceEEEEEcCcEEEEecccEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>d1biaa2 b.34.1.1 (A:271-317) Biotin repressor/biotin holoenzyme synthetase, C-terminal domain {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=85.36  E-value=3.4  Score=23.80  Aligned_cols=32  Identities=19%  Similarity=0.342  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCeEEE
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDMNIK   88 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k   88 (113)
</pre><pre>                      .|.-+|+|++|+|.+...+..+....|++.+.
</pre><pre>T Consensus        16 ~G~~~gI~~~G~L~l~~~g~~~~~~~g~i~~~   47 (47)
</pre><pre>T d1biaa2          16 FGISRGIDKQGALLLEQDGIIKPWMGGEISLR   47 (47)
</pre><pre>T ss_dssp             EEEEEEECTTSCEEEEETTEEEEESSCEEEEC
</pre><pre>T ss_pred             EEEEEeeCCCCcEEEEECCEEEEEEeeEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>3VTO_B Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=85.08  E-value=9.6  Score=26.91  Aligned_cols=46  Identities=11%  Similarity=0.137  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEEecC--ceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   65 EAGELFAISQG--GAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        65 esGq~yi~s~~--~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      .+|+++++...  .+.+...|.+.+... ++.+.+.+.+.+.+.++.++
</pre><pre>T Consensus        10 ~dG~~~~yd~~g~~l~i~~~g~i~i~~~-~~~i~a~~~i~~~~~~~~i~   57 (115)
</pre><pre>T 3VTO_B           10 EPGDAGIYHHEGHRIRLTKDGRCIITCK-TVEVYADESMTVDTPRTTFT   57 (115)
</pre><pre>T ss_dssp             CTTCEEEECTTSCEEEEETTTEEEEEEE-EEEEEEEEEEEEECSEEEES
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEEcCCEEEEeCC-eEEEEcCCcEEEeCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>3VTO_C Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=85.08  E-value=9.6  Score=26.91  Aligned_cols=46  Identities=11%  Similarity=0.137  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEEecC--ceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   65 EAGELFAISQG--GAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        65 esGq~yi~s~~--~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      .+|+++++...  .+.+...|.+.+... ++.+.+.+.+.+.+.++.++
</pre><pre>T Consensus        10 ~dG~~~~yd~~g~~l~i~~~g~i~i~~~-~~~i~a~~~i~~~~~~~~i~   57 (115)
</pre><pre>T 3VTO_C           10 EPGDAGIYHHEGHRIRLTKDGRCIITCK-TVEVYADESMTVDTPRTTFT   57 (115)
</pre><pre>T ss_dssp             CTTCEEEECTTSCEEEEETTTEEEEEEE-EEEEEEEEEEEEESSEEEES
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEEcCCEEEEeCC-eEEEEcCCcEEEeCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>6P20_C Baseplate central spike complex protein gp5,PHIKZ164; bacteriophage, phiKZ, membrane piercing, central spike, cell puncturing device, PAAR-repeat motif, beta helix, T4 gp5, Pseudomonas; HET: STE, PLM, ELA, EDO; 1.749A {Enterobacteria phage T4}
</pre><pre>Probab=84.04  E-value=18  Score=23.88  Aligned_cols=47  Identities=21%  Similarity=0.165  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   66 AGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        66 sGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      .+...+...++..+.+++.++++. ..+.+.+.+.+.+.+..+.|+.+
</pre><pre>T Consensus        52 ~~~~~~~~~~~~~~~~~~~~~~~~-~~~~~~~~~~i~~~g~~~~i~~~   98 (112)
</pre><pre>T 6P20_C           52 NGNLSWKVAGTVDWDVGGDWTEKM-ASMSSKSSGTHIQEAGGTMTHKA   98 (112)
</pre><pre>T ss_dssp             SSCEEEEESSCEEEEESSEEEEEE-SEEEEEESSCEEEEESSCEEEEE
</pre><pre>T ss_pred             cCCEEEEEcCeEEEEeCCCEEEEe-cceEEecCCeEEEeeeeEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF10106.11 ; DUF2345 ; Uncharacterized protein conserved in bacteria (DUF2345)
</pre><pre>Probab=81.63  E-value=9.1  Score=26.75  Aligned_cols=47  Identities=19%  Similarity=0.182  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeee
</pre><pre>Q FD01848339_033   61 IGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAE  107 (113)
</pre><pre>Q Consensus        61 iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~  107 (113)
</pre><pre>                      +.++...-+-+.++.++.+.++.++.+.++.++.+.+++.+.+.|.+
</pre><pre>T Consensus         9 ~~i~a~~g~~~~~~~~~~~~a~~~~~~~ag~~~~~~ag~~~~~~a~~   55 (145)
</pre><pre>T X2H4H4_9NEIS/6    9 ILTSAVAGMAQVSPKSILNSVGQDIHQQAGKDVFTGAGENITQTAVK   55 (145)
</pre><pre>T ss_pred             EEEecccceeecCcccEEEeeCCceEEeeCCceEehhhhhheeeecc
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>6SK0_B Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=81.19  E-value=42  Score=30.87  Aligned_cols=78  Identities=12%  Similarity=0.206  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   34 IYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        34 V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      +....|-+++....|-.++.  +.+.---....|.+.|.+.+.+.+.++|....-.++.|.+.+.|.+.++++...+.-+
</pre><pre>T Consensus       686 ~~~~~g~v~~~a~~~~~~~~--a~~~i~i~s~~~~i~i~a~~~i~l~~gg~~i~l~~~~i~~~~~g~~~~~~~~~~~~g~  763 (841)
</pre><pre>T 6SK0_B          686 LLAAKGRVNIQAQSDDLSMS--SQQNLDIQSSEGKVTVSANQELILACGGAYIKLSGGNIELGCPGQILLKSTNMQKMGP  763 (841)
</pre><pre>T ss_dssp             EEESSSCEEEEETTSCEEEE--ESSCEEEEESSSEEEEEESSEEEEEETTEEEEEETTEEEEECSSCEEEC---------
</pre><pre>T ss_pred             eEEccCCeEEEEcCCceeEe--ecCCeEEEeCCCEEEEeCCCEEEEEeCCeEEEEeCCeeEEecCCeEEEEcccccccCC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>6SK0_C Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=81.19  E-value=42  Score=30.87  Aligned_cols=78  Identities=12%  Similarity=0.206  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   34 IYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        34 V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      +....|-+++....|-.++.  +.+.---....|.+.|.+.+.+.+.++|....-.++.|.+.+.|.+.++++...+.-+
</pre><pre>T Consensus       686 ~~~~~g~v~~~a~~~~~~~~--a~~~i~i~s~~~~i~i~a~~~i~l~~gg~~i~l~~~~i~~~~~g~~~~~~~~~~~~g~  763 (841)
</pre><pre>T 6SK0_C          686 LLAAKGRVNIQAQSDDLSMS--SQQNLDIQSSEGKVTVSANQELILACGGAYIKLSGGNIELGCPGQILLKSTNMQKMGP  763 (841)
</pre><pre>T ss_dssp             EEESSSCEEEEETTSCEEEE--ESSCEEEEESSSEEEEEESSEEEEEETTEEEEEETTEEEEECSSCEEEC---------
</pre><pre>T ss_pred             eEEccCCeEEEEcCCceeEe--ecCCeEEEeCCCEEEEeCCCEEEEEeCCeEEEEeCCeeEEecCCeEEEEcccccccCC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>4S37_B Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: SO4, CL, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=80.93  E-value=34  Score=24.86  Aligned_cols=56  Identities=7%  Similarity=0.175  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEecCCCCEEEEecC--ceEEEecCC-eEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   56 GNGTTIGFNEAGELFAISQG--GAFISAAGD-MNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        56 asGs~iG~NesGq~yi~s~~--~~fi~~~G~-~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ....++-..++|..+.+.+.  .+.|...+. ++++..+....-+++.+.|++.+|.|+
</pre><pre>T Consensus        94 ~~~~~~~~~~~g~~~~~d~~~~~i~i~~~~~~i~~~~~~~~~~~~~~~~~i~~~~v~i~  152 (185)
</pre><pre>T 4S37_B           94 SASSETWRFDDGASLSYDWAAHRYRVELPSGTVEVRVGASEVRVSDGAVSLKAPKISLE  152 (185)
</pre><pre>T ss_dssp             CTTEEEEECTTSCEEEEETTTTEEEEEESSSEEEEEETTEEEEEESSCEEEECSCEEEE
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEECCCCEEEEEeCCceEEEEECCcEEEEeCCcEEEECCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>4S37_O Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: CL, SO4, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=80.93  E-value=34  Score=24.86  Aligned_cols=56  Identities=7%  Similarity=0.175  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEecCCCCEEEEecC--ceEEEecCC-eEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   56 GNGTTIGFNEAGELFAISQG--GAFISAAGD-MNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        56 asGs~iG~NesGq~yi~s~~--~~fi~~~G~-~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ....++-..++|..+.+.+.  .+.|...+. ++++..+....-+++.+.|++.+|.|+
</pre><pre>T Consensus        94 ~~~~~~~~~~~g~~~~~d~~~~~i~i~~~~~~i~~~~~~~~~~~~~~~~~i~~~~v~i~  152 (185)
</pre><pre>T 4S37_O           94 SASSETWRFDDGASLSYDWAAHRYRVELPSGTVEVRVGASEVRVSDGAVSLKAPKISLE  152 (185)
</pre><pre>T ss_dssp             CTTEEEEECTTSCEEEEETTTTEEEEEESSSEEEEEETTEEEEEESSCEEEECSCEEEE
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEECCCCEEEEEeCCceEEEEECCcEEEEeCCcEEEECCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>d1pwaa_ b.42.1.1 (A:) Fibroblast growth factor-19 (FGF19) {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=80.02  E-value=8.8  Score=29.07  Aligned_cols=46  Identities=11%  Similarity=0.151  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...+..|.|  -+...+||+++.+-| ..|.+..++.-+.||..|.||
</pre><pre>T Consensus        24 ~~G~V~gt~~--~~~~s~l~i~s~~~g~v~I~g~~s~~YLcmn~~G~L~   70 (136)
</pre><pre>T d1pwaa_          24 ADGVVDCARG--QSAHSLLEIKAVALRTVAIKGVHSVRYLCMGADGKMQ   70 (136)
</pre><pre>T ss_dssp             TTSBEEEESS--CCGGGCEEEEEEETTEEEEEETTTCCEEEECGGGCEE
</pre><pre>T ss_pred             CCCeEeccCC--CCcccceEEEEEccCceEEEccccceEEEeCCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>d1k28a2 b.108.1.2 (A:362-576) Tail-associated lysozyme gp5, C-terminal domain {Bacteriophage T4 [TaxId: 10665]}
</pre><pre>Probab=79.14  E-value=23  Score=27.72  Aligned_cols=46  Identities=17%  Similarity=0.301  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   66 AGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        66 sGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      .|...+...+.+.+.+++.+++..+|.+.+.+++.+.+.++.|.|+
</pre><pre>T Consensus       162 ~g~~~~~v~g~~~~~v~~~~~~~v~g~~~~~~g~~~~~~~g~i~i~  207 (215)
</pre><pre>T d1k28a2         162 EGNQTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSISSGQYTID  207 (215)
</pre><pre>T ss_dssp             SSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSCCCCCCSSCCCCC
</pre><pre>T ss_pred             eCCEEEEEcCcEEEEEeCceEEEECCcEEEEeccEEEEcCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>3A1M_A chimera of thrombin cleavage site, Tail-associated lysozyme, Fibritin; bacteriophage T4, Antimicrobial, Bacteriolytic enzyme, Glycosidase, Hydrolase, Late protein, Chaperone, STRUCTURAL PROTEIN; 2.0A {Enterobacteria phage T4}
</pre><pre>Probab=79.01  E-value=24  Score=26.16  Aligned_cols=46  Identities=20%  Similarity=0.214  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   67 GELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        67 Gq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      +.+.+...+.+-+.+++.++++. +.+.+.+.+.+.+.+..+.++-+
</pre><pre>T Consensus        58 ~~~~v~~g~~~~l~~g~~itl~~-~~~~~~~~~~i~i~g~~i~l~~g  103 (139)
</pre><pre>T 3A1M_A           58 GNLSWKVAGTVDWDVGGDWTEKM-ASMSSISSGQYTIDGSRIDIGSV  103 (139)
</pre><pre>T ss_dssp             SCEEEEESSCEEEEESSEEEEEE-SEEEEEESSEEEEECSEEEETTB
</pre><pre>T ss_pred             CeEEEEECCeEEEEeCCeEEEEe-cceEEecCceEEEECCeEEEccC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>3A1M_E chimera of thrombin cleavage site, Tail-associated lysozyme, Fibritin; bacteriophage T4, Antimicrobial, Bacteriolytic enzyme, Glycosidase, Hydrolase, Late protein, Chaperone, STRUCTURAL PROTEIN; 2.0A {Enterobacteria phage T4}
</pre><pre>Probab=79.01  E-value=24  Score=26.16  Aligned_cols=46  Identities=20%  Similarity=0.214  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   67 GELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        67 Gq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      +.+.+...+.+-+.+++.++++. +.+.+.+.+.+.+.+..+.++-+
</pre><pre>T Consensus        58 ~~~~v~~g~~~~l~~g~~itl~~-~~~~~~~~~~i~i~g~~i~l~~g  103 (139)
</pre><pre>T 3A1M_E           58 GNLSWKVAGTVDWDVGGDWTEKM-ASMSSISSGQYTIDGSRIDIGSV  103 (139)
</pre><pre>T ss_dssp             SCEEEEESSCEEEEESSEEEEEE-SEEEEEESSEEEEECSEEEEC--
</pre><pre>T ss_pred             CeEEEEECCeEEEEeCCeEEEEe-cceEEecCceEEEECCeEEEccC
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>7CBM_a Flagellar basal-body rod protein FlgF; Flagella, Distal rod, FlgG, FlgF, MOTOR PROTEIN; 3.2A {Salmonella typhimurium (strain LT2 / SGSC1412 / ATCC 700720)}
</pre><pre>Probab=78.28  E-value=34  Score=27.72  Aligned_cols=74  Identities=27%  Similarity=0.420  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   26 STSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        26 ~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      .....-|+.+.-+|++.+++..|-...|...   ++.++++|.|..  .+..++..++.+.+-...++.+...|.+.++
</pre><pre>T Consensus        73 ~T~~~~d~ai~g~g~F~v~~~~g~~~yTR~G---~f~~d~~g~lv~--~g~~v~~~~~~i~ip~~~~~~i~~~G~v~~~  146 (251)
</pre><pre>T 7CBM_a           73 YTSRPLDVALQQDGWLVVQAADGAEGYTRNG---NIQVGPTGQLTI--QGHPVIGEGGPITVPEGSEITIAADGTISAL  146 (251)
</pre><pre>T ss_dssp             ECCCTTBEECCTTEEEEEECTTSCEEEECCC---BCEECTTSBEEE--TTEEBBBSSSBCBCCSSCEEEECTTSEEEEE
</pre><pre>T ss_pred             ecCCcceEEecCCceEEEEcCCCCeEEEEEe---eEEECCCCeEEE--CCeeeeCCCCCcccCCCCceEECCCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>7CBM_b Flagellar basal-body rod protein FlgF; Flagella, Distal rod, FlgG, FlgF, MOTOR PROTEIN; 3.2A {Salmonella typhimurium (strain LT2 / SGSC1412 / ATCC 700720)}
</pre><pre>Probab=78.28  E-value=34  Score=27.72  Aligned_cols=74  Identities=27%  Similarity=0.420  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   26 STSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        26 ~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      .....-|+.+.-+|++.+++..|-...|...   ++.++++|.|..  .+..++..++.+.+-...++.+...|.+.++
</pre><pre>T Consensus        73 ~T~~~~d~ai~g~g~F~v~~~~g~~~yTR~G---~f~~d~~g~lv~--~g~~v~~~~~~i~ip~~~~~~i~~~G~v~~~  146 (251)
</pre><pre>T 7CBM_b           73 YTSRPLDVALQQDGWLVVQAADGAEGYTRNG---NIQVGPTGQLTI--QGHPVIGEGGPITVPEGSEITIAADGTISAL  146 (251)
</pre><pre>T ss_dssp             ECCCTTBEECCTTEEEEEECSSSCEEEECCC---BCEECTTCCEEE--TTEEBCBSSSSCCCCSSCCCEECSSSEEEEC
</pre><pre>T ss_pred             ecCCcceEEecCCceEEEEcCCCCeEEEEEe---eEEECCCCeEEE--CCeeeeCCCCCcccCCCCceEECCCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>d2dxua2 b.34.1.1 (A:189-235) Biotin--[acetyl-CoA-carboxylase] ligase C-terminal domain {Pyrococcus horikoshii [TaxId: 53953]}
</pre><pre>Probab=73.47  E-value=19  Score=21.22  Aligned_cols=33  Identities=18%  Similarity=0.313  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCc-eEEEecCCeEEEe
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGG-AFISAAGDMNIKA   89 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~-~fi~~~G~~t~k~   89 (113)
</pre><pre>                      .|...|++++|.|.+-.... .+.-..|++.+..
</pre><pre>T Consensus        13 ~G~~~gI~~~G~L~l~~~~g~~~~~~~g~v~~~~   46 (47)
</pre><pre>T d2dxua2          13 EGIAEDIDDFGRLIIRLDSGEVKKVIYGDVSLRF   46 (47)
</pre><pre>T ss_dssp             EEEEEEECTTSCEEEECTTSCEEEECCSSCCCEE
</pre><pre>T ss_pred             EEEEEeeCCCceEEEEeCCCCEEEEEEEeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>3PQI_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 2.642A {Bacteriophage phi92}
</pre><pre>Probab=72.39  E-value=41  Score=26.58  Aligned_cols=47  Identities=17%  Similarity=0.365  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EEEcCCCCEEEecCCCCEEEEecCceEEEecC-CeEE--EeCcceeEeeC
</pre><pre>Q FD01848339_033   52 VTHLGNGTTIGFNEAGELFAISQGGAFISAAG-DMNI--KAGGNMGIEAG   98 (113)
</pre><pre>Q Consensus        52 iTh~asGs~iG~NesGq~yi~s~~~~fi~~~G-~~t~--k~~g~i~l~a~   98 (113)
</pre><pre>                      +.....|+.|-++++|.+.|.+.+++-|.+.+ .+++  ...+++.+++.
</pre><pre>T Consensus       143 ~~~~~~g~~i~~~~~G~i~i~~~~~i~i~~~~~~i~i~~~~~~~~~i~~~  192 (247)
</pre><pre>T 3PQI_A          143 IISNNKQTYASFDPNGNISVYNTQGMKIDMTPNSIVLTDAGGGKLTLQGG  192 (247)
</pre><pre>T ss_dssp             EEEETTTEEEEECTTSCEEEEETTSCEEEEETTEEEEEETTTEEEEEETT
</pre><pre>T ss_pred             EEEcCCCcEEEECCCCcEEEEeCCceEEEeCCCceEEEeCCCceEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>6SK0_B Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=70.09  E-value=63  Score=29.81  Aligned_cols=101  Identities=11%  Similarity=0.127  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCccccCCCCCCCCCCCC---------CCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCC---E
</pre><pre>Q FD01848339_033    2 VPPEASGQGTGYTPPDVD---------GAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGE---L   69 (113)
</pre><pre>Q Consensus         2 v~pEasg~G~~y~p~~vd---------GaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq---~   69 (113)
</pre><pre>                      ..|-++..-+.|.||++.         |-|..|-+.   -.++....--....+..-...-.+.+|..|-|++...   +
</pre><pre>T Consensus       439 ~~~~ag~~~G~~~~p~~g~eV~v~f~~gd~~~p~i~---g~~~~~~~~~~~~~~~~~~~~~~t~~g~~l~~~d~~~~~~i  515 (841)
</pre><pre>T 6SK0_B          439 AKPYSGDAYGFHFPLIQGTEVAIAFEEGDPDRPYIA---HALHDSRHVDHVTDKNGTRNVIRTPANNKLRMEDKRGEEHI  515 (841)
</pre><pre>T ss_dssp             -------------------------------------------------------CCEEEEECTTCCEEEEECCTTSCEE
</pre><pre>T ss_pred             eeccccCCCceeeccCCCCeeEEEEecCCCCCCEEE---EeeccccCCCCCcCCCCcccceecccccEEEEEcCCCceEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEecC--ceEEEecCC-----------eEEEeCcceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   70 FAISQG--GAFISAAGD-----------MNIKAGGNMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        70 yi~s~~--~~fi~~~G~-----------~t~k~~g~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      ++.++.  ...++-+..           +.+.+.+.+.+.++..+.|++
</pre><pre>T Consensus       516 ~l~~~~~~~~~l~~~~~~~~~~~~~~~~~~l~~~~~~~i~a~~~~~i~~  564 (841)
</pre><pre>T 6SK0_B          516 KLSTEYGGKTQLNLGHNVDASRELRGEGAELRTDDWISIRGGKGIFISA  564 (841)
</pre><pre>T ss_dssp             EEEESSSSSEEEEEEEEECTTSCEEEEEEEEEESSEEEEEETTEEEEEC
</pre><pre>T ss_pred             EEEecCCCccEeEcceeeccccccccCcEEEEeCCcceEecccceEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>6SK0_C Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=70.09  E-value=63  Score=29.81  Aligned_cols=101  Identities=11%  Similarity=0.127  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCccccCCCCCCCCCCCC---------CCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCC---E
</pre><pre>Q FD01848339_033    2 VPPEASGQGTGYTPPDVD---------GAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGE---L   69 (113)
</pre><pre>Q Consensus         2 v~pEasg~G~~y~p~~vd---------GaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq---~   69 (113)
</pre><pre>                      ..|-++..-+.|.||++.         |-|..|-+.   -.++....--....+..-...-.+.+|..|-|++...   +
</pre><pre>T Consensus       439 ~~~~ag~~~G~~~~p~~g~eV~v~f~~gd~~~p~i~---g~~~~~~~~~~~~~~~~~~~~~~t~~g~~l~~~d~~~~~~i  515 (841)
</pre><pre>T 6SK0_C          439 AKPYSGDAYGFHFPLIQGTEVAIAFEEGDPDRPYIA---HALHDSRHVDHVTDKNGTRNVIRTPANNKLRMEDKRGEEHI  515 (841)
</pre><pre>T ss_dssp             -------------------------------------------------------CCEEEEECTTCCEEEEECCTTSCEE
</pre><pre>T ss_pred             eeccccCCCceeeccCCCCeeEEEEecCCCCCCEEE---EeeccccCCCCCcCCCCcccceecccccEEEEEcCCCceEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEecC--ceEEEecCC-----------eEEEeCcceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   70 FAISQG--GAFISAAGD-----------MNIKAGGNMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        70 yi~s~~--~~fi~~~G~-----------~t~k~~g~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      ++.++.  ...++-+..           +.+.+.+.+.+.++..+.|++
</pre><pre>T Consensus       516 ~l~~~~~~~~~l~~~~~~~~~~~~~~~~~~l~~~~~~~i~a~~~~~i~~  564 (841)
</pre><pre>T 6SK0_C          516 KLSTEYGGKTQLNLGHNVDASRELRGEGAELRTDDWISIRGGKGIFISA  564 (841)
</pre><pre>T ss_dssp             EEEESSSSSEEEEEEEEECTTSCEEEEEEEEEESSEEEEEETTEEEEEC
</pre><pre>T ss_pred             EEEecCCCccEeEcceeeccccccccCcEEEEeCCcceEecccceEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>d1v1ha1 b.83.1.1 (A:319-392) Adenovirus {Human adenovirus type 2 [TaxId: 10515]}
</pre><pre>Probab=69.42  E-value=48  Score=23.95  Aligned_cols=61  Identities=20%  Similarity=0.261  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEE
</pre><pre>Q FD01848339_033   38 NGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGI  103 (113)
</pre><pre>Q Consensus        38 NgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i  103 (113)
</pre><pre>                      |+-|... .|+|..+.+    ..+-+|.+.++.+-....+.+...+.+.+|-+..+.|.+.|.+.+
</pre><pre>T Consensus        13 n~~L~lk-~G~GL~~~~----g~l~~~~~~~l~l~~G~GL~~~~n~~l~vKlG~GL~fds~G~i~~   73 (74)
</pre><pre>T d1v1ha1          13 NTAIAIN-AGKGLEFDT----NTSESPDINPIKTKIGSGIDYNENGAMITKLGAGLSFDNSGAITI   73 (74)
</pre><pre>T ss_dssp             TTEEEEC-BCTTEEECT----TSTTTTTSCCEEECBCTTEEECTTSCEEECBCTTCEECTTSCEES
</pre><pre>T ss_pred             CCceEEe-cCCCEEEcC----CCcccCCCCCccceeCCCEEEcCCCCeEEecCCCcEECCCCceee
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>d4cjma_ b.42.1.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=65.35  E-value=35  Score=25.39  Aligned_cols=35  Identities=9%  Similarity=0.129  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecC--CceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAG--GAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~Rtag--GG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      ++..+||+++.+  |=..|-...++--+-||..|+||
</pre><pre>T Consensus        31 ~~~~~l~i~s~~~~g~V~I~~~~s~~yLcmn~~G~l~   67 (129)
</pre><pre>T d4cjma_          31 DKYAQLLVETDTFGSQVRIKGKETEFYLCMNRKGKLV   67 (129)
</pre><pre>T ss_dssp             CGGGCEEEEECTTSSEEEEEETTTCCEEEECTTCCEE
</pre><pre>T ss_pred             CcccEEEEEEcCCCCeEEEEeeCCCeEEeEcCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>4CJM_A FIBROBLAST GROWTH FACTOR 18; SIGNALING PROTEIN, SIGNALLING; HET: SO4; 2.7A {HOMO SAPIENS} SCOP: b.42.1.0
</pre><pre>Probab=65.20  E-value=43  Score=24.47  Aligned_cols=44  Identities=11%  Similarity=0.052  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCcceeeecCCceEEEecCCc--eeEEEcCCCCEEEecCCCCEEE
</pre><pre>Q FD01848339_033   27 TSATKDFIYSRNGLMEIRNAGGA--WSVTHLGNGTTIGFNEAGELFA   71 (113)
</pre><pre>Q Consensus        27 ~s~tkD~V~~RNgllE~RtagGG--~siTh~asGs~iG~NesGq~yi   71 (113)
</pre><pre>                      +..|.| -.+++.+|+++..+-+  ..|....++.-+.||+.|++|.
</pre><pre>T Consensus        24 V~gt~~-~~~~~s~l~~~~~~~~~~v~i~~~~s~~yL~m~~~G~l~~   69 (141)
</pre><pre>T 4CJM_A           24 ISARGE-DGDKYAQLLVETDTFGSQVRIKGKETEFYLCMNRKGKLVG   69 (141)
</pre><pre>T ss_dssp             EEEEEC-TTCGGGCEEEEECTTSSEEEEEETTTCCEEEECTTCCEEE
</pre><pre>T ss_pred             EEcccC-CCCCccEEEEEEeCCCCEEEEEecCCceEEEEcCCCCEEe
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>d1ijta_ b.42.1.1 (A:) Fibroblast growth factor 4 (FGF4) {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=64.85  E-value=40  Score=25.16  Aligned_cols=46  Identities=15%  Similarity=0.283  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecC-CceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAG-GAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~Rtag-GG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...+..|.|.  +++.+|++++.. |=..|.+..++.-+-||+.|++|
</pre><pre>T Consensus        22 ~~G~V~gt~~~--~~~~~l~~~~~~~g~v~i~~~~s~~yLc~n~~G~l~   68 (128)
</pre><pre>T d1ijta_          22 PDGRIGGAHAD--TRDSLLELSPVERGVVSIFGVASRFFVAMSSKGKLY   68 (128)
</pre><pre>T ss_dssp             TTSCEEEESSC--CGGGCEEEEEEETTEEEEEETTTTEEEEECTTCCEE
</pre><pre>T ss_pred             CCCcEEcccCC--CCCceeEEeecccCeEEEEeeecceEEEEcCCceEe
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>2P39_A Fibroblast growth factor 23; atypical beta-trefoil fold, SIGNALING PROTEIN; HET: GU4, YYJ; 1.5A {Homo sapiens} SCOP: b.42.1.0
</pre><pre>Probab=64.68  E-value=49  Score=25.29  Aligned_cols=46  Identities=11%  Similarity=0.128  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...+..+.|.  +...+|++++.+.| ..|-...++.-+.||..|.||
</pre><pre>T Consensus        33 ~~G~V~g~~~~--~~~~~l~i~s~~~g~v~I~g~~s~~yLcmn~~G~L~   79 (155)
</pre><pre>T 2P39_A           33 KNGHVDGAPHQ--TIYSALMIRSEDAGFVVITGVMSRRYLCMDFRGNIF   79 (155)
</pre><pre>T ss_dssp             TTSCEEEESSC--CTTTCEEEEECGGGEEEEEETTTTEEEEECTTSCEE
</pre><pre>T ss_pred             CCCeEecCCCC--CCCeEEEEEEcCCCeEEEeecccceEEEEcCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>4M0N_A Conserved hypothetical protein, putative anti-sigma factor; FecR protein, PF04773 family, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY; 1.65A {Parabacteroides distasonis}
</pre><pre>Probab=63.18  E-value=1.3e+02  Score=23.59  Aligned_cols=90  Identities=18%  Similarity=0.267  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEec-----CCCCEEEEecCceEEEecCC----eEEEeC
</pre><pre>Q FD01848339_033   20 GAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFN-----EAGELFAISQGGAFISAAGD----MNIKAG   90 (113)
</pre><pre>Q Consensus        20 GaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~N-----esGq~yi~s~~~~fi~~~G~----~t~k~~   90 (113)
</pre><pre>                      +...........++.-.++...++.-.+|.  ......+|+|-++     ...++++  .|.+|+...-+    +.|+..
</pre><pre>T Consensus        10 ~~~~~~~~~~~~~i~t~~~~~~~~~l~dgs--~v~L~~~S~i~~~~~~~~~~~~v~L--~G~a~~~v~~~~~~~~~V~t~   85 (232)
</pre><pre>T 4M0N_A           10 KQHTSASDHLLTEISVNHGEHKQVTLPDGT--VVHLNAGTVMRYPTEFTSDIRLVEM--EGEAFFNVMRDEGKPFIVRTR   85 (232)
</pre><pre>T ss_dssp             ----------CEEEECCTTCCEEEECTTSC--EEEECTTCEEEECSSCCSSSEEEEE--EEEEEEEECCCTTSCEEEECS
</pre><pre>T ss_pred             CcccCCCcceeEEEEcCCCceEEEECCCCC--EEEeCCCCEEEccCCCCCCeeEEEE--EEEEEEEEeeCCCCCEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cceeEeeCeeeEEEe------eeeeeeCC
</pre><pre>Q FD01848339_033   91 GNMGIEAGGVLGIRA------AEVKVDKG  113 (113)
</pre><pre>Q Consensus        91 g~i~l~a~g~~~i~A------~~v~idkg  113 (113)
</pre><pre>                      .-.-.-.++.|.|..      .+|.+-+|
</pre><pre>T Consensus        86 ~~~i~~~gt~f~v~~~~~~~~~~v~v~~G  114 (232)
</pre><pre>T 4M0N_A           86 QADVKVLGASFNVKAYQEDELMAVSVRTG  114 (232)
</pre><pre>T ss_dssp             SCEEEESSEEEEEEECTTSSEEEEEEEEE
</pre><pre>T ss_pred             CcEEEEcCEEEEEEEcCCCCcEEEEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>2J3P_B HEPARIN-BINDING GROWTH FACTOR 1; GROWTH FACTOR, DEVELOPMENTAL PROTEIN, ACIDIC FIBROBLAST GROWTH FACTOR, DIFFERENTIATION, HEPARIN-BINDING, MITOGEN, CYTOKINE, ANGIOGENESIS, BETA-TREFOIL; HET: SO4; 1.4A {RATTUS NORVEGICUS} SCOP: b.42.1.1
</pre><pre>Probab=61.14  E-value=67  Score=23.36  Aligned_cols=53  Identities=9%  Similarity=0.168  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEEEEecCc
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELFAISQGG   76 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~yi~s~~~   76 (113)
</pre><pre>                      +...+..|.|. .++..+|++++.+.| ..|-+..++.-+.||..|++|.....+
</pre><pre>T Consensus        21 ~~g~V~gt~~~-~~~~~~l~~~~~~~~~v~i~~~~s~~yl~mn~~G~l~~~~~~~   74 (134)
</pre><pre>T 2J3P_B           21 PDGTVDGTRDR-SDQHIQLQLSAESAGEVYIKGTETGQYLAMDTEGLLYGSQTPN   74 (134)
</pre><pre>T ss_dssp             TTSCEEEESCT-TCTTSCEEEEEEETTEEEEEETTTCCEEEECTTSCEEEESSCC
</pre><pre>T ss_pred             CCCcEecccCC-CCCceEEEEEeCCCceEEEEcccceeEEEECCCCCEEeeCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>d2p39a_ b.42.1.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=60.37  E-value=61  Score=25.21  Aligned_cols=49  Identities=10%  Similarity=0.095  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEEEEecCc
</pre><pre>Q FD01848339_033   26 STSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELFAISQGG   76 (113)
</pre><pre>Q Consensus        26 ~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~yi~s~~~   76 (113)
</pre><pre>                      .+..|.|  .++..+|++++.+.| ..|-...++.-+-||..|+||......
</pre><pre>T Consensus        32 ~V~gt~~--~s~~s~l~i~s~~~g~v~Ikg~~s~~YLcmn~~G~L~~~~~~~   81 (142)
</pre><pre>T d2p39a_          32 HVDGAPH--QTIYSALMIRSEDAGFVVITGVMSRRYLCMDFRGNIFGSHYFD   81 (142)
</pre><pre>T ss_dssp             CEEEESS--CCTTTCEEEEECGGGEEEEEETTTTEEEEECTTSCEEEESSCC
</pre><pre>T ss_pred             eEEcCCC--CCccceeEEEEcCCCeEEEEeeeccceEeeccCCcEEeeeecC
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>4S36_A Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; 1.46A {Pseudomonas aeruginosa}
</pre><pre>Probab=60.11  E-value=90  Score=20.90  Aligned_cols=46  Identities=11%  Similarity=0.417  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEEecC--ceEEEe-cCCeEEEeCcc-eeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   65 EAGELFAISQG--GAFISA-AGDMNIKAGGN-MGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        65 esGq~yi~s~~--~~fi~~-~G~~t~k~~g~-i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ++|..+.+...  .+.|.. +|.+.++..+. +.++ .+.+.+.+..+.++
</pre><pre>T Consensus         8 ~dG~~i~yd~~~~~l~i~~~~g~~~i~~~~~~~~i~-~~~i~i~~~~i~~~   57 (90)
</pre><pre>T 4S36_A            8 DDGASLSYDWAAHRYRVELPSGTVEVRVGASEVRVS-DGAVSLKAPKISLE   57 (90)
</pre><pre>T ss_dssp             TTSCEEEEETTTTEEEEECSSSEEEEEETTEEEEEE-TTEEEEECSCCCCC
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEEeCCceEEEEeCCceEEEe-cCCEEEeCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>d1q1ua_ b.42.1.1 (A:) Fibrobast growth factor homologous factor 1 (FHF1b, FGF12b) {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=60.03  E-value=35  Score=25.90  Aligned_cols=35  Identities=17%  Similarity=0.250  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      ++..+||+++.+-| ..|-+..++.-+-||+.|+||
</pre><pre>T Consensus        34 ~~~~~l~i~s~~~g~v~I~~~~s~~ylcmn~~G~l~   69 (138)
</pre><pre>T d1q1ua_          34 SDYTLFNLIPVGLRVVAIQGVKASLYVAMNGEGYLY   69 (138)
</pre><pre>T ss_dssp             SGGGCEEEEEEETTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CcceeEEEEEeCcceEEEeechhcceEeecCCceEe
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>3PQH_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 1.295A {Bacteriophage phi92}
</pre><pre>Probab=59.94  E-value=1.2e+02  Score=22.18  Aligned_cols=73  Identities=12%  Similarity=0.166  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeec-CCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeee
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSR-NGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVL  101 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~R-NgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~  101 (113)
</pre><pre>                      ..|+.+.+.|-+..| |+--..+-...|..|......++|-++ .|.+.|..++.+.+...+               +.+
</pre><pre>T Consensus        11 ~~~~p~~~~~~~~~~~~~~~~~~~~~dG~~i~yd~~~~~l~i~-~g~i~i~~~~~~~~~~~~---------------~~i   74 (127)
</pre><pre>T 3PQH_A           11 TDSSEEVDSEKVIISNNKQTYASFDPNGNISVYNTQGMKIDMT-PNSIVLTDAGGGKLTLQG---------------GTM   74 (127)
</pre><pre>T ss_dssp             ---------CCEEEEETTTEEEEECTTSCEEEEETTSCEEEEE-TTEEEEEETTTEEEEEET---------------TEE
</pre><pre>T ss_pred             CCCCCCCCcccEEEEcCCcceeeeCCCCcEEEEcCCCcEEEec-CCEEEEEcCCCcEEEECC---------------CeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEEeeeeeee
</pre><pre>Q FD01848339_033  102 GIRAAEVKVD  111 (113)
</pre><pre>Q Consensus       102 ~i~A~~v~id  111 (113)
</pre><pre>                      .+.+..+.|+
</pre><pre>T Consensus        75 ~~~~~~v~i~   84 (127)
</pre><pre>T 3PQH_A           75 TYKGGTVNLN   84 (127)
</pre><pre>T ss_dssp             EEESSCEEET
</pre><pre>T ss_pred             EEeCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>3F1R_A Fibroblast growth factor 20; beta-trefoil fold, Growth factor, Polymorphism, Secreted, HORMONE; HET: SO4; 2.5A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=59.87  E-value=36  Score=28.01  Aligned_cols=35  Identities=31%  Similarity=0.590  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecC-CceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAG-GAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~Rtag-GG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +++.+||+.+-+ |-.+|....++--+.||+.|+||
</pre><pre>T Consensus        93 s~~~~le~~s~~~g~V~Irgv~s~~YL~mn~~G~Ly  128 (211)
</pre><pre>T 3F1R_A           93 SLFGILEFISVAVGLVSIRGVDSGLYLGMNDKGELY  128 (211)
</pre><pre>T ss_dssp             CSSSEEEEEEEETTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CccceEEEEeecccceeeeeecccceEeecCCCcEe
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>PF12503.10 ; CMV_1a_C ; Cucumber mosaic virus 1a protein C terminal
</pre><pre>Probab=59.66  E-value=33  Score=26.29  Aligned_cols=32  Identities=28%  Similarity=0.613  Sum_probs=0.0  Template_Neff=2.700
</pre><pre>
</pre><pre>Q ss_pred             ceEEEecCCceeEEEcCCCCEEEecCCC-------CEEEE
</pre><pre>Q FD01848339_033   40 LMEIRNAGGAWSVTHLGNGTTIGFNEAG-------ELFAI   72 (113)
</pre><pre>Q Consensus        40 llE~RtagGG~siTh~asGs~iG~NesG-------q~yi~   72 (113)
</pre><pre>                      |+-++-+.|.|.. .+.---++|+||+|       |+||.
</pre><pre>T Consensus        27 lvNvH~~~G~W~~-P~~~~Y~VGyNe~GLG~K~~~ElYiV   65 (81)
</pre><pre>T 1A_BMV/590-670   27 MVNVHLANGNWLY-PKKYDYTVGYNEHGLGPKHADETYIV   65 (81)
</pre><pre>T ss_pred             ceeEEecCCcEec-ccceEEEeeecCCCCccCCCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>1WTH_A Tail-associated lysozyme; Triple-stranded beta-helix, OB fold, pseudohexamer, T4 tail lysozyme, HUB, gp5-gp27, Hydrolase-Structural protein COMPLEX; 2.8A {Enterobacteria phage T4}
</pre><pre>Probab=59.49  E-value=1.7e+02  Score=28.04  Aligned_cols=65  Identities=18%  Similarity=0.172  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             ecCCceeEEEcCC-----CCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeee
</pre><pre>Q FD01848339_033   45 NAGGAWSVTHLGN-----GTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKV  110 (113)
</pre><pre>Q Consensus        45 tagGG~siTh~as-----Gs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~i  110 (113)
</pre><pre>                      +.+|+..++-...     +....+.-.+...+...+.+.+.+++..+++.+..+.+ +.+.+.|.+..+++
</pre><pre>T Consensus       505 ~I~G~~~vtv~g~~~~~i~g~~ti~v~g~~~i~~gg~i~i~sgg~~~i~~g~~~~~-asg~V~I~G~ki~~  574 (584)
</pre><pre>T 1WTH_A          505 IVEGNADITVKGDATTLVEGNQTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSI-SSGQYTIDGSRIDI  574 (584)
</pre><pre>T ss_dssp             CCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCCSCCCCCCSCCCCC-CSSCCCCCCSCCCC
</pre><pre>T ss_pred             EEcCCEEEEEcCCeEEEEccCEEEEECCCEEEEECCcEEEEEcCCEEEEeCceeEe-cCeeEEEECceeEe
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>1IJT_A fibroblast growth factor 4; b-trefoil fold, HORMONE-GROWTH FACTOR COMPLEX; HET: SO4; 1.8A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=58.56  E-value=56  Score=23.38  Aligned_cols=46  Identities=13%  Similarity=0.231  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCce-eEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGAW-SVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG~-siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...+..|.|.  +.+.+++++..+.+. .|....++.-+.||..|.+|
</pre><pre>T Consensus        22 ~~G~V~gt~~~--~~~~~~~~~~~~~~~v~i~~~~s~~yl~~~~~G~l~   68 (128)
</pre><pre>T 1IJT_A           22 PDGRIGGAHAD--TRDSLLELSPVERGVVSIFGVASRFFVAMSSKGKLY   68 (128)
</pre><pre>T ss_dssp             TTSCEEEESSC--CGGGCEEEEEEETTEEEEEETTTTEEEEECTTCCEE
</pre><pre>T ss_pred             CCCcEecccCC--CCCceeEEEEecCCeEEEEeehhHeEEEEcCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>6M6E_A Fibroblast growth factor 21; Anti-diabetes, beta-trefoil conformation, beta-klotho binding, Metabolic regulator, HORMONE; NMR {Homo sapiens}
</pre><pre>Probab=58.25  E-value=89  Score=22.24  Aligned_cols=46  Identities=11%  Similarity=0.180  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...++.+.|.  +.+.+|++++.+-+ ..|....++.-+.||..|.+|
</pre><pre>T Consensus        24 ~~g~v~g~~~~--~~~~~l~~~~~~~~~v~i~~~~s~~ylc~~~~G~l~   70 (128)
</pre><pre>T 6M6E_A           24 EDGTVGGAADQ--SPESLLQLKALKPGVIQILGVKTSRFLCQRPDGALY   70 (128)
</pre><pre>T ss_dssp             TTSCEEEESSC--CGGGEEEEEEEETTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CCCeecccCCC--CCCceEEEEEeCCCeEEEEecCCCeEEEECCCCCeE
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>PF14563.8 ; DUF4444 ; Domain of unknown function (DUF4444)
</pre><pre>Probab=58.02  E-value=48  Score=20.23  Aligned_cols=29  Identities=14%  Similarity=0.208  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCe
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDM   85 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~   85 (113)
</pre><pre>                      +|.-.|++++|.|.+-.......-..+++
</pre><pre>T Consensus        11 ~G~~~gld~~G~Lll~~~~g~~~i~~~~v   39 (42)
</pre><pre>T Q5LSG4_RUEPO/1   11 TGTFLGVDEKFGMLIRTGADTRLIPLTAL   39 (42)
</pre><pre>T ss_pred             eEEEEEECCCCCEEEEeCCCeEEEEhhee
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>d2fdbm1 b.42.1.1 (M:34-180) Fibroblast growth factor 8, FGF8 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=57.83  E-value=59  Score=24.95  Aligned_cols=35  Identities=17%  Similarity=0.143  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCc--eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGA--WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG--~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +.+.+|++++..-|  ..|-+..++.-|.||..|+||
</pre><pre>T Consensus        48 ~~~~~l~i~s~~~g~~v~i~~~~s~~YLc~n~~G~l~   84 (147)
</pre><pre>T d2fdbm1          48 DPFAKLIVETDTFGSRVRVRGAETGLYICMNKKGKLI   84 (147)
</pre><pre>T ss_dssp             CGGGCEEEEEEETTTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CCccEEEEEEcCCCCeEEEEeeCCCeEEeEecCCcee
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>4PEU_A Uncharacterized protein; Beta-helix, Polysaccharide lyase, carbohydrate-binding, calcium-binding, UNKNOWN FUNCTION; 1.8001A {Clostridium thermocellum}
</pre><pre>Probab=57.63  E-value=57  Score=28.02  Aligned_cols=49  Identities=20%  Similarity=0.241  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   65 EAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        65 esGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      .+...-|++.+++.|...|.+++++...--+.+.+.|.|..+.|.+..+
</pre><pre>T Consensus       148 ~~~~~~i~~~~~ltI~G~GtL~i~~~~~~gI~~~~~l~I~~gti~i~~~  196 (313)
</pre><pre>T 4PEU_A          148 NKPNACIYSRDDITIKGNGNLTVNANFNNGIGTSNDLKITGGNITVKAF  196 (313)
</pre><pre>T ss_dssp             CCCCEEEEESSEEEEEESSEEEEEECSSEEEEESSEEEECSCEEEEEES
</pre><pre>T ss_pred             CCCCceEEeCCcEEEecCcEEEEecCCCCccCCCCcEEEeCCeEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>3OGF_B de novo designed dimeric trefoil-fold sub-domain which forms homo-trimer assembly; beta-trefoil, DE NOVO PROTEIN; HET: SO4; 2.864A {synthetic construct}
</pre><pre>Probab=54.13  E-value=1.1e+02  Score=20.27  Aligned_cols=35  Identities=17%  Similarity=0.228  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecC-CceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAG-GAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~Rtag-GG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      ++..+|+++..+ +-..|....++.-+.||..|.+|
</pre><pre>T Consensus        35 ~~~~~l~~~~~~~~~v~i~g~~s~~~l~~~~~g~l~   70 (90)
</pre><pre>T 3OGF_B           35 DPHIQFQISPEGNGEVLLKSTETGQYLRINPDGTVD   70 (90)
</pre><pre>T ss_dssp             CGGGCBEEETTCSCCEEEEETTTTEEEEECTTSCEE
</pre><pre>T ss_pred             CCCceEEEeecCCCcEEEEEcCCCcEEEECCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>PF12445.10 ; FliC ; Flagellin protein
</pre><pre>Probab=53.71  E-value=34  Score=27.19  Aligned_cols=40  Identities=18%  Similarity=0.322  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             ecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeE
</pre><pre>Q FD01848339_033   63 FNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLG  102 (113)
</pre><pre>Q Consensus        63 ~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~  102 (113)
</pre><pre>                      ++.+|+|.-...-+.|+...|.++..++..+.+.+.|+|.
</pre><pre>T Consensus        81 vd~dG~ltt~~t~~y~~~~dGsvt~~~G~~vyv~adGklT  120 (125)
</pre><pre>T A2V706_ECOLX/3   81 TQTDGTLAAKNETEIFLQKDGSITNNSGKAVYVQEDGKFT  120 (125)
</pre><pre>T ss_pred             EcCCCceeccceeeEEEccCCcEEcCCCceeEECCCCCEe
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>PF07481.13 ; DUF1521 ; Domain of Unknown Function (DUF1521)
</pre><pre>Probab=53.41  E-value=2e+02  Score=24.17  Aligned_cols=67  Identities=13%  Similarity=0.200  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCceeEEEcCCCCEEEecCC--CCEE-EEecCceEEEecCCeEEEeCcceeEeeCeeeEEEee------------eeeee
</pre><pre>Q FD01848339_033   47 GGAWSVTHLGNGTTIGFNEA--GELF-AISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAA------------EVKVD  111 (113)
</pre><pre>Q Consensus        47 gGG~siTh~asGs~iG~Nes--Gq~y-i~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~------------~v~id  111 (113)
</pre><pre>                      ++||.|+-..+.+.|-|.+.  |+.+ |.+...+.++......++-...+.|.-+=++.|...            +|.|.
</pre><pre>T Consensus        16 gdgY~l~~~~~~s~~~I~n~~tG~~t~IwGDPHvd~~g~~~~dF~~~~TF~L~DGTkITv~t~~~~~~~~~t~~~~vtIt   95 (169)
</pre><pre>T W0AEV6_9SPHN/8   16 GDGYTMRLDERSSEIMIHNEKTNEWTRIWGDPHVDVNGERAFDFWGTTTFTLENGTKVTIDTEQWGGNPDAYVASKVTVT   95 (169)
</pre><pre>T ss_pred             CCCeEEEEcCCCCEEEEEeCCCCceEEEEeCcceecCCCcceeeeCcEEEECCCCCEEEEECccCCCCCCceeeeeEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CC
</pre><pre>Q FD01848339_033  112 KG  113 (113)
</pre><pre>Q Consensus       112 kg  113 (113)
</pre><pre>                      +|
</pre><pre>T Consensus        96 ~G   97 (169)
</pre><pre>T W0AEV6_9SPHN/8   96 NG   97 (169)
</pre><pre>T ss_pred             eC
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>d3hqxa_ b.82.1.0 (A:) automated matches {Acinetobacter sp. [TaxId: 62977]}
</pre><pre>Probab=52.79  E-value=1.3e+02  Score=22.61  Aligned_cols=59  Identities=15%  Similarity=0.159  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             eeEEEcCC-CCEEEecCCCC---EEEEecCceEEEecCCeEEEeCc---ceeEeeCeeeEEEeeee
</pre><pre>Q FD01848339_033   50 WSVTHLGN-GTTIGFNEAGE---LFAISQGGAFISAAGDMNIKAGG---NMGIEAGGVLGIRAAEV  108 (113)
</pre><pre>Q Consensus        50 ~siTh~as-Gs~iG~NesGq---~yi~s~~~~fi~~~G~~t~k~~g---~i~l~a~g~~~i~A~~v  108 (113)
</pre><pre>                      ..+..... ..++|..+.|+   =|...+.+.|.=..|.+++..++   ...|+++..|.|-|...
</pre><pre>T Consensus        24 ~~~~~~~g~~~tlgv~~pG~~~~~~ht~~~E~~~Vl~G~L~v~~~g~~~~~~l~~Gd~~~Vp~g~~   89 (105)
</pre><pre>T d3hqxa_          24 HTVQFEDGTKKTLGVILPTEQPLTFETHVPERMEIISGECRVKIADSTESELFRAGQSFYVPGNSL   89 (105)
</pre><pre>T ss_dssp             EEEECTTSCEEEEEEECCCSSCEEEECSSCEEEEEEESEEEEEETTCSSCEEEETTCEEEECTTCE
</pre><pre>T ss_pred             EEEEeCCCCEEEEEEEccCCCCceecCCCCeEEEEEEEEEEEEeCCccCEEEECCCCEEEECCCCe
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>d1ihka_ b.42.1.1 (A:) Fibroblast growth factor 9, FGF9 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=51.91  E-value=63  Score=25.56  Aligned_cols=35  Identities=34%  Similarity=0.578  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +++.+||+++.+-| ..|-+..++.=+-||..|.||
</pre><pre>T Consensus        39 ~~~s~l~i~s~~~g~V~I~g~~s~~yLcmn~~G~L~   74 (157)
</pre><pre>T d1ihka_          39 SRFGILEFISIAVGLVSIRGVDSGLYLGMNEKGELY   74 (157)
</pre><pre>T ss_dssp             CGGGCEEEEEEETTEEEEEETTTCCEEEECTTCCEE
</pre><pre>T ss_pred             CCceeEEEEEecCCeEEEeeeccceeeeecCCceEE
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>d1nuna_ b.42.1.1 (A:) Fibroblast growth factor-10, FGF10 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=51.49  E-value=72  Score=24.26  Aligned_cols=35  Identities=17%  Similarity=0.341  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +++.+|++++.+-| ..|.+..++.=+-||+.|+||
</pre><pre>T Consensus        38 ~~~~~l~i~s~~~g~v~I~~~~s~~yLc~n~~G~l~   73 (139)
</pre><pre>T d1nuna_          38 CPYSILEITSVEIGVVAVKAINSNYYLAMNKKGKLY   73 (139)
</pre><pre>T ss_dssp             CTTSCEEEEEEETTEEEEEETTTTEEEEECTTSBEE
</pre><pre>T ss_pred             CCceeEEEEEeccCeEEEEeeecCceEEEecCCeeE
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>5W21_B Fibroblast growth factor 23; complex, ligand, receptor, co-receptor, HYDROLASE-PROTEIN BINDING complex; HET: NAG; 3.0A {Homo sapiens}
</pre><pre>Probab=49.27  E-value=95  Score=26.39  Aligned_cols=46  Identities=11%  Similarity=0.128  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...+..|.|-  ++..+|++++...| ..|-...++.-+.||..|++|
</pre><pre>T Consensus        79 ~dG~V~Gt~~~--s~~~~L~i~s~~~g~V~Irg~~s~~YLcmn~~G~Ly  125 (226)
</pre><pre>T 5W21_B           79 KNGHVDGAPHQ--TIYSALMIRSEDAGFVVITGVMSRRYLCMDFRGNIF  125 (226)
</pre><pre>T ss_dssp             TTSCEEEESSC--CSTTCEEEEECSSSEEEEEETTTTEEEEECTTSCEE
</pre><pre>T ss_pred             CCCcccCCCCc--cCceEEEEEeCCCCcEEEeeecceeeEEEecCCcee
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>4PEU_A Uncharacterized protein; Beta-helix, Polysaccharide lyase, carbohydrate-binding, calcium-binding, UNKNOWN FUNCTION; 1.8001A {Clostridium thermocellum}
</pre><pre>Probab=49.26  E-value=3.1e+02  Score=23.87  Aligned_cols=80  Identities=13%  Similarity=0.210  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             cceeeecCCceEEEecCCceeEE--EcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeee
</pre><pre>Q FD01848339_033   31 KDFIYSRNGLMEIRNAGGAWSVT--HLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEV  108 (113)
</pre><pre>Q Consensus        31 kD~V~~RNgllE~RtagGG~siT--h~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v  108 (113)
</pre><pre>                      ++.+.-.+|-+.+...+.|....  .......|-++ .|.+.+.+..+. |.+.++++| .+|.+.+.+.+.---....|
</pre><pre>T Consensus       203 ~g~i~I~gG~v~i~~~~~gi~~~~~~~~~~g~i~I~-gG~v~~~~~~~g-i~~~~~i~I-~gG~v~~~~~~~~~~~~~~i  279 (313)
</pre><pre>T 4PEU_A          203 NGSVTISGGNIDITAGADGIKVENTEEPHKGYVNIT-GGTIKIRAKDDA-IDSVRSVSI-NNADVKVSVGGKDVKCEGVL  279 (313)
</pre><pre>T ss_dssp             SSEEEECSCEEEEEESSCSEECCCCSSTTCSEEEEC-SSEEEEEESSCS-EEESSEEEE-CSSEEEEEESSCSEEESSEE
</pre><pre>T ss_pred             CCeEEEeCCEEEEecCCCceeecCCCCCCCcEEEEe-CCEEEEEeCCCc-cccCCeEEE-eCCEEEEEeCCCceeeCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeCC
</pre><pre>Q FD01848339_033  109 KVDKG  113 (113)
</pre><pre>Q Consensus       109 ~idkg  113 (113)
</pre><pre>                      .|+.|
</pre><pre>T Consensus       280 ~i~gG  284 (313)
</pre><pre>T 4PEU_A          280 NIAEG  284 (313)
</pre><pre>T ss_dssp             EECTT
</pre><pre>T ss_pred             EEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>3O4C_A de novo designed beta-trefoil architecture with symmetric primary structure; beta-trefoil, DE NOVO PROTEIN; HET: SO4; 1.75A {synthetic construct}
</pre><pre>Probab=49.02  E-value=1.1e+02  Score=22.40  Aligned_cols=35  Identities=14%  Similarity=0.164  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +.+.+|++.+.+.| ..|-...++.-+.||++|.++
</pre><pre>T Consensus        45 ~~~~~l~~~~~~~~~v~i~~~~s~~yl~~~~~G~l~   80 (142)
</pre><pre>T 3O4C_A           45 DVHIQFQISPEGNGEVLLKSTETGQYLRINPDGTVD   80 (142)
</pre><pre>T ss_dssp             CGGGCEEEEEETTTEEEEEETTTCCEEEECTTSBEE
</pre><pre>T ss_pred             CCceeEEEecCCCceEEEEEecCccEEEECCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>PF14262.8 ; Cthe_2159 ; Carbohydrate-binding domain-containing protein Cthe_2159
</pre><pre>Probab=47.79  E-value=3.1e+02  Score=23.39  Aligned_cols=69  Identities=19%  Similarity=0.273  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEecCCceeEEEcCCCCEEEecCCCCEE-----------EEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   43 IRNAGGAWSVTHLGNGTTIGFNEAGELF-----------AISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        43 ~RtagGG~siTh~asGs~iG~NesGq~y-----------i~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ++-....-.+.....|+.--+.+.....           |.+.+++.|...|.+++....+--+.+.+.|.|+-+.+.|.
</pre><pre>T Consensus        51 i~i~~~~~~~i~~~~gt~n~l~~~~~~~~~~~~~~~~~~I~~~~~L~I~G~GtL~i~~~~~~gI~~~~~l~I~~gti~i~  130 (292)
</pre><pre>T H6NTR1_9BACL/9   51 IQVMQADKTVITVQEGTANAVSDGTQYAAVDDSGEPNAAIFSKDDLTFNGSGTLTVQGNYNNGITSKDDLKITGGTIKVT  130 (292)
</pre><pre>T ss_pred             EEEecCCeEEEEECCCceEEEccCceeeccCCCCCCCceEEeCCCEEEccCcEEEEeCCCCCCEEeCCcEEEcCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>2XSK_A CSGC; CHAPERONE; HET: ACT; 1.7A {ESCHERICHIA COLI}
</pre><pre>Probab=45.61  E-value=1.5e+02  Score=22.07  Aligned_cols=67  Identities=9%  Similarity=0.157  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             eEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   41 MEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        41 lE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      |++...+|.+.|+-.....+=.   ++++-+.-...   ..+|..+.+-.|++.|.++....+.-..|.+..+
</pre><pre>T Consensus         8 I~~~~~~~~~~i~~~v~~~~~~---~~~Y~l~v~k~---g~~Gss~s~QsG~f~l~~g~~~~Ls~~~ls~~~~   74 (110)
</pre><pre>T 2XSK_A            8 FNTTQQGDMYTIIPEVTLTQSX---LXRVQILSLRE---GSSGQSQTKQEKTLSLPANQPIALTKLSLNISPD   74 (110)
</pre><pre>T ss_dssp             EEEEEETTEEEEEEEEECSSCE---EEEEEEEEEEE---CSSCEEEEEEEEEEEECSSSEEECCCEEEECCTT
</pre><pre>T ss_pred             EEEEEECCEEEEEEEEEECCCC---CeeEEEEEEEE---CCCCceEEeeeeEEEcCCCCcEEeEEEEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>6SK0_B Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=45.43  E-value=64  Score=29.75  Aligned_cols=51  Identities=18%  Similarity=0.214  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCeE-EEeCcceeEeeCeeeEEEeee
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDMN-IKAGGNMGIEAGGVLGIRAAE  107 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~t-~k~~g~i~l~a~g~~~i~A~~  107 (113)
</pre><pre>                      ....+-+...+.+-+.+++++.+.+++++. +.+++++.+.+.+.+.+.|++
</pre><pre>T Consensus       622 ~~~~~~~~~~~~i~~~~~~~~~~~a~~~~~~~~ag~~~~~~~~~~~~~~a~~  673 (841)
</pre><pre>T 6SK0_B          622 TAPGMLLHAPDGIGMVSARALRIASGSESVGIMSGDNTDITAGQSFTVVAEG  673 (841)
</pre><pre>T ss_dssp             SSSCEEEECTTCEEEEESSCEEEEESSSCEEEEESSCEEEEESSCEEEEESS
</pre><pre>T ss_pred             cCCeeeeecCCcceeccHhhhhHhccCccEEEEeCCceEEeecceEEEEEhh
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>6SK0_C Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=45.43  E-value=64  Score=29.75  Aligned_cols=51  Identities=18%  Similarity=0.214  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCeE-EEeCcceeEeeCeeeEEEeee
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDMN-IKAGGNMGIEAGGVLGIRAAE  107 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~t-~k~~g~i~l~a~g~~~i~A~~  107 (113)
</pre><pre>                      ....+-+...+.+-+.+++++.+.+++++. +.+++++.+.+.+.+.+.|++
</pre><pre>T Consensus       622 ~~~~~~~~~~~~i~~~~~~~~~~~a~~~~~~~~ag~~~~~~~~~~~~~~a~~  673 (841)
</pre><pre>T 6SK0_C          622 TAPGMLLHAPDGIGMVSARALRIASGSESVGIMSGDNTDITAGQSFTVVAEG  673 (841)
</pre><pre>T ss_dssp             SSSCEEEECTTCEEEEESSCEEEEESSSCEEEEESSCEEEEESSCEEEEESS
</pre><pre>T ss_pred             cCCeeeeecCCcceeccHhhhhHhccCccEEEEeCCceEEeecceEEEEEhh
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>4UHV_A VGRG1, VALINE-GLYCINE REPEAT PROTEIN G1; STRUCTURAL PROTEIN, VGRG1, VIRULENCE, TOXIN, EFFECTOR, PUNCTURING DEVICE, SPIKE, T6SS, P. AERUGINOSA; HET: CL, NA, MSE; 2.0A {PSEUDOMONAS AERUGINOSA}
</pre><pre>Probab=44.87  E-value=4.1e+02  Score=23.96  Aligned_cols=99  Identities=14%  Similarity=0.079  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCcceeeecC---------CceEEEecCCceeEEEcCCCC-----------------------EEE
</pre><pre>Q FD01848339_033   15 PPDVDGAPSMPSTSATKDFIYSRN---------GLMEIRNAGGAWSVTHLGNGT-----------------------TIG   62 (113)
</pre><pre>Q Consensus        15 p~~vdGaP~~p~~s~tkD~V~~RN---------gllE~RtagGG~siTh~asGs-----------------------~iG   62 (113)
</pre><pre>                      +......|...+-.++...+..|.         +.+......|...|.....+.                       .+-
</pre><pre>T Consensus       450 ~~~~~~~p~~~~~~~~~~~~~~~~~~~g~~~~~~~~~~~d~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  529 (651)
</pre><pre>T 4UHV_A          450 YNAEQTVPYELPANATQSGMKSRSSKGGTPANFNEIRMEDKKGAEQLYIHAERNQDNLVENDASLSVGHDRNKSIGHDEL  529 (651)
</pre><pre>T ss_dssp             CCSSSCCSSCTTTSSSCEEEECCCCSSCCTTCCCEEEEECCTTCCEEEEECSSCCCCCCSSCCCCCCSSCCCCCCSSCCC
</pre><pre>T ss_pred             ccCCCCCCCCCCcccceeEEEecCcCCCCCCCccEEEeeCCCCceeEEEeeccCcccccccCcEEEecCCCccccCCceE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeee--eeeeCC
</pre><pre>Q FD01848339_033   63 FNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAE--VKVDKG  113 (113)
</pre><pre>Q Consensus        63 ~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~--v~idkg  113 (113)
</pre><pre>                      ++-.+...+...++..+..+|+.++...+++.+++.+++.++++.  +.+..+
</pre><pre>T Consensus       530 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  582 (651)
</pre><pre>T 4UHV_A          530 ARIGNNRTRAVKLNDTLLVGGAKSDSVTGTYLIEAGAQIRLVCGKSVVEFNAD  582 (651)
</pre><pre>T ss_dssp             CCCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSSEEEEEETTEEEEEESS
</pre><pre>T ss_pred             EEecCCeEEEEecCeEEEECCceEEEecCcEEEEeCCeEEEEeCCcEEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>d3vgfa3 b.71.1.1 (A:491-557) Glycosyltrehalose trehalohydrolase {Sulfolobus solfataricus, km1 [TaxId: 2287]}
</pre><pre>Probab=43.68  E-value=1e+02  Score=22.75  Aligned_cols=48  Identities=21%  Similarity=0.449  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEEecC----ceEEEecCCeEEEeCcceeEeeCeee--EEEeeeeeeeCC
</pre><pre>Q FD01848339_033   66 AGELFAISQG----GAFISAAGDMNIKAGGNMGIEAGGVL--GIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        66 sGq~yi~s~~----~~fi~~~G~~t~k~~g~i~l~a~g~~--~i~A~~v~idkg  113 (113)
</pre><pre>                      .||-++|-.+    .+|+.+-..+.||..|.+-|.+...|  .|+-+...+|||
</pre><pre>T Consensus         9 ngenwliikgreyfslyvfskssievkysgtlllssnnsfpqhieegkyefdkg   62 (67)
</pre><pre>T d3vgfa3           9 NGENWLIIKGREYFSLYVFSKSSIEVKYSGTLLLSSNNSFPQHIEEGKYEFDKG   62 (67)
</pre><pre>T ss_dssp             ECSSEEEEECSSCEEEEESSCEEEEECCCEEEEEESSSCSCSEECSEEEEECSE
</pre><pre>T ss_pred             cccceEEEeceeEEEEEEEeCccEEEEeeeeEEEecCCCCchhhhcCceeeccc
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>PF00167.20 ; FGF ; Fibroblast growth factor
</pre><pre>Probab=43.06  E-value=1.5e+02  Score=22.32  Aligned_cols=54  Identities=19%  Similarity=0.265  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CCCC----CCCCCcceeeecCCceEEEecC-CceeEEEcCCCCEEEecCCCCEEEEecCc
</pre><pre>Q FD01848339_033   22 PSMP----STSATKDFIYSRNGLMEIRNAG-GAWSVTHLGNGTTIGFNEAGELFAISQGG   76 (113)
</pre><pre>Q Consensus        22 P~~p----~~s~tkD~V~~RNgllE~Rtag-GG~siTh~asGs~iG~NesGq~yi~s~~~   76 (113)
</pre><pre>                      |+..    .+..+.+. .+.+.+++++..+ |-..|....++.-+.||..|.+|......
</pre><pre>T Consensus        21 ~~~g~~~~~V~g~~~~-~~~~~~l~~~~~~~~~v~i~~~~s~~ylcm~~~G~l~~~~~~~   79 (136)
</pre><pre>T LE756_CAEEL/72   21 PDDGSTRVKVHGTKEE-SSKFSIVEFVSVAMSLVSIRGVETKNFICMDPSGKLYATPSSN   79 (136)
</pre><pre>T ss_pred             CCCCCcceEEEcccCC-CCCCcEEEEEEeCceeEEEeccccceEEEECCCCCEEecCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>6MJP_C Lipopolysaccharide export system protein LptC; lipopolysaccharide transport, ABC-transporter, LIPID TRANSPORT; HET: PG4, MA4, LMT, AE3, JU7, GOL; 2.85A {Vibrio cholerae}
</pre><pre>Probab=42.95  E-value=2.3e+02  Score=20.48  Aligned_cols=48  Identities=10%  Similarity=0.207  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ecCCC-CEEEEecCceEEEecCCeEEEeCcceeEee----CeeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   63 FNEAG-ELFAISQGGAFISAAGDMNIKAGGNMGIEA----GGVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        63 ~NesG-q~yi~s~~~~fi~~~G~~t~k~~g~i~l~a----~g~~~i~A~~v~idk  112 (113)
</pre><pre>                      +.+++ ..+.+....+.++.. + .+...|++.++.    ...+.|++.++.+|.
</pre><pre>T Consensus        86 ~~~~~~~~~~i~a~~~~~~~~-~-~~~l~g~V~i~~~~~~~~~~~i~t~~l~~~~  138 (191)
</pre><pre>T 6MJP_C           86 YREGTLQEWEITARRGILSKD-Q-VLTLYDDVLAKNLLPDSGFDTLTTSEMSIQL  138 (191)
</pre><pre>T ss_dssp             EETTTEEEEEEECSEEEECTT-C-EEEECSSEEEEECCTTSSCCEEEESCEEEET
</pre><pre>T ss_pred             EcCCCCceEEEEEeEEEEeCC-C-EEEEeCCEEEEeCCCCCCceEEEcceEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>PF04773.15 ; FecR ; FecR protein
</pre><pre>Probab=42.90  E-value=2e+02  Score=19.73  Aligned_cols=70  Identities=11%  Similarity=0.116  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCC--eEEEeCc-----ceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   42 EIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGD--MNIKAGG-----NMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        42 E~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~--~t~k~~g-----~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      .++|..++...-....|+.+-|++.-++-|.......-.....  +.+..+.     .+.+.....|.|..+...+.
</pre><pre>T Consensus         2 ~i~T~~~~~~~l~~~dg~~v~l~~~s~l~i~~~~~~~~~~~~~~~l~l~~G~~~~~~~~~~~~~~~~~i~t~~~~~~   78 (104)
</pre><pre>T A8ZRT1_DESOH/7    2 RIRVGDQSRIELRLPDGSFVRFDSNSVFELKAMAVDSRDNRRDINVNVVFGKIWASVSKWVSGSSRFDVSMKTTTAG   78 (104)
</pre><pre>T ss_pred             EEEcCCCcEEEEEeCCCCEEEeCCCcEEEEEEEEEcCCCCeeEEEEEEEeCeeeeEEEEecCCCCcEEEEcCceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>5VTG_A Translocation and assembly module subunit TamB; Chaperone, Beta-sheet, Periplasm; 1.859A {Escherichia coli}
</pre><pre>Probab=42.68  E-value=2.8e+02  Score=21.45  Aligned_cols=51  Identities=20%  Similarity=0.287  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             EEecCCCCEEEEecCceEEEecCCeEEEeC-------cceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   61 IGFNEAGELFAISQGGAFISAAGDMNIKAG-------GNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        61 iG~NesGq~yi~s~~~~fi~~~G~~t~k~~-------g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      +-++-..+++|.+.. +.+...|++.++..       |++.+.. |.+.+...++.|++|
</pre><pre>T Consensus        68 l~v~~~~~~~i~~~~-l~~~~~G~l~l~~~~~~~~~~G~i~i~~-G~~~~~~k~F~i~~G  125 (184)
</pre><pre>T 5VTG_A           68 LIVHVGNNVRIDAFG-LKARLTGDLNVVQDKQGLGLNGQINIPE-GRFHAYGQDLIVRKG  125 (184)
</pre><pre>T ss_dssp             EEEEEEEEEEEEETT-EEEEEEEEEEEEEETTEEEEEEEEEEEE-EEEEETTEEEEEEEE
</pre><pre>T ss_pred             EEEEeCCcEEEEECC-EEEEEEEEEEEEECCCceEEEEEEEEec-eEEEEcCeEEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>PF10988.10 ; DUF2807 ; Putative auto-transporter adhesin, head GIN domain
</pre><pre>Probab=42.27  E-value=2.6e+02  Score=20.87  Aligned_cols=64  Identities=17%  Similarity=0.312  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             eEEE-ecCCceeEE--------------------EcCCCCEEEecCCCCEEEE----ecCceEEEecCCeEEEeCcceeE
</pre><pre>Q FD01848339_033   41 MEIR-NAGGAWSVT--------------------HLGNGTTIGFNEAGELFAI----SQGGAFISAAGDMNIKAGGNMGI   95 (113)
</pre><pre>Q Consensus        41 lE~R-tagGG~siT--------------------h~asGs~iG~NesGq~yi~----s~~~~fi~~~G~~t~k~~g~i~l   95 (113)
</pre><pre>                      |.++ +.+|-..+.                    ....-..+-++.+|.+.+.    ..-.+..+..|++.+....++.+
</pre><pre>T Consensus       110 l~i~~~~~g~v~~~~~~~~~~~~~~~~~~~~~~~g~~~~~~i~~~g~g~i~~~~~~~~~~~v~~~~~g~i~v~~~~~~~~  189 (203)
</pre><pre>T A9WHL3_CHLAA/4  110 LSVRISGSGDMILPNVAAKTILAEVNSSGLMEAAGTTDRLHVKVSGSGDLLAEKLKASIVEVAVNGSGDVTVWAVDTLDV  189 (203)
</pre><pre>T ss_pred             EEEEEeccceeEecceEEeeEEEEecCCceEEEeeEecEEEEEEecCceeeeeccEEeEEEEEEecceEEEEEecCeEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeCeeeEEE
</pre><pre>Q FD01848339_033   96 EAGGVLGIR  104 (113)
</pre><pre>Q Consensus        96 ~a~g~~~i~  104 (113)
</pre><pre>                      ++.+.-.|+
</pre><pre>T Consensus       190 ~~~~~g~i~  198 (203)
</pre><pre>T A9WHL3_CHLAA/4  190 SISGSGNVR  198 (203)
</pre><pre>T ss_pred             EEcCceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>PF11142.10 ; DUF2917 ; Protein of unknown function (DUF2917)
</pre><pre>Probab=42.19  E-value=2e+02  Score=19.64  Aligned_cols=52  Identities=21%  Similarity=0.387  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             EcCCCCEEEec--CCCCEEEEecCceEEEecC--------CeEEEeCcceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   54 HLGNGTTIGFN--EAGELFAISQGGAFISAAG--------DMNIKAGGNMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        54 h~asGs~iG~N--esGq~yi~s~~~~fi~~~G--------~~t~k~~g~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      ....|..+-+.  ..+.-...-+|.+.|...|        |+-+..+-.+.+.+++.+-|.|
</pre><pre>T Consensus         2 ~L~~g~~~~l~~~~~~~~l~~~~G~~WlT~~g~~~~~~~~D~~L~~Ge~~~l~~~~~vvvea   63 (65)
</pre><pre>T A2SL07_METPP/2    2 RLVRGQTLRLQAARNTRWLRLREGRLWVTAEGRLGGQIPEDWWLAAGQSLRLPAGTEVLAEG   63 (65)
</pre><pre>T ss_pred             eeCCCCEEEEEeCCCCeEEEEEecEEEEEeCCCCCCcCCCCEEECCCCEEEECCCCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>4OW4_A Beta-terfoil designed by folding nucleus symmetric expansion ("Phifoil"); Beta-trefoil, folding nucleus symmetric expansion, protein design, de novo, top-down symmetric deconstruction, DE NOVO PROTEIN; 2.148A {synthetic construct}
</pre><pre>Probab=40.97  E-value=2e+02  Score=19.58  Aligned_cols=35  Identities=9%  Similarity=0.148  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecC-CceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAG-GAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~Rtag-GG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +++.+++++..+ +-+.|-...++.-+.|+++|.++
</pre><pre>T Consensus        30 ~~~~~~~~~~~~~~~~~i~~~~~~~~l~~~~~g~~~   65 (123)
</pre><pre>T 4OW4_A           30 DQHIQLQLSAESVGEVYIKSTETGQYLRILPDGTVD   65 (123)
</pre><pre>T ss_dssp             CGGGCEEEEEEETTEEEEEETTTCCEEEECTTSBEE
</pre><pre>T ss_pred             CCceeEEEEccCCCcEEEEEcCCCCEEEECCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>PF13540.8 ; RCC1_2 ; Regulator of chromosome condensation (RCC1) repeat
</pre><pre>Probab=39.55  E-value=84  Score=14.53  Aligned_cols=18  Identities=22%  Similarity=0.514  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             CEEEecCCCCEEEEecCc
</pre><pre>Q FD01848339_033   59 TTIGFNEAGELFAISQGG   76 (113)
</pre><pre>Q Consensus        59 s~iG~NesGq~yi~s~~~   76 (113)
</pre><pre>                      .++.+..+|++|..+...
</pre><pre>T Consensus        10 ~~~~~~~~g~~~~~g~~~   27 (30)
</pre><pre>T D2V5M1_NAEGR/2   10 TTLVLTESGHVYSCGDSS   27 (30)
</pre><pre>T ss_pred             eEEEEECCCcEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>1RG8_A Heparin-binding growth factor 1; BETA-TREFOIL, HORMONE-GROWTH FACTOR COMPLEX; HET: FMT; 1.1A {Homo sapiens} SCOP: l.1.1.1, b.42.1.1
</pre><pre>Probab=39.43  E-value=1.4e+02  Score=22.36  Aligned_cols=35  Identities=9%  Similarity=0.218  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +++.+|++++.+-+ ..|....++.-+.||..|.+|
</pre><pre>T Consensus        45 ~~~~~l~~~~~~~~~v~i~g~~s~~ylcmn~~G~l~   80 (146)
</pre><pre>T 1RG8_A           45 DQHIQLQLSAESVGEVYIKSTETGQYLAMDTDGLLY   80 (146)
</pre><pre>T ss_dssp             CTTCCEEEEEEETTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CCceEEEEEEccCceEEEcccccceEEEECCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>6KTR_C Fibroblast growth factor 19; complex, growth factor, antibody, CELL CYCLE, IMMUNE SYSTEM-PROTEIN BINDING complex; HET: SO4; 2.59775753879A {Homo sapiens} SCOP: b.42.1.0
</pre><pre>Probab=39.11  E-value=1e+02  Score=25.82  Aligned_cols=46  Identities=11%  Similarity=0.144  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +......+.+.  +...+|++++..-| ..|....++.=+.||..|++|
</pre><pre>T Consensus        42 ~dG~V~Gt~~~--~~~s~L~i~sv~~g~V~I~gv~s~~YLCmn~~G~Ly   88 (218)
</pre><pre>T 6KTR_C           42 ADGVVDCARGQ--SAHSLLEIKAVALRTVAIKGVHSVRYLCMGADGKMQ   88 (218)
</pre><pre>T ss_dssp             TTSBEEEESSC--CGGGEEEEEESSSSEEEEEETTTTEEEEECGGGCEE
</pre><pre>T ss_pred             CCCeEeeeCCC--CcCceEEEEEeccceeEEcceeeEeEEEECCCCCeE
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>3GVZ_A Uncharacterized protein CV2077; alpha-beta protein, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, UNKNOWN FUNCTION; HET: MSE; 2.8A {Chromobacterium violaceum ATCC 12472}
</pre><pre>Probab=38.78  E-value=1.4e+02  Score=24.17  Aligned_cols=44  Identities=14%  Similarity=0.221  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CcceeeecCCceEEEecCCceeEEEcCC------CCEEEecCCCCEEEEe
</pre><pre>Q FD01848339_033   30 TKDFIYSRNGLMEIRNAGGAWSVTHLGN------GTTIGFNEAGELFAIS   73 (113)
</pre><pre>Q Consensus        30 tkD~V~~RNgllE~RtagGG~siTh~as------Gs~iG~NesGq~yi~s   73 (113)
</pre><pre>                      +.|+-......+.+....+|+.+.....      |...|||+.|-.+...
</pre><pre>T Consensus        45 N~D~~~~~~~~~~~~~~~~g~~~~~~~~~~~~~~g~~~G~N~~Gl~~~~~   94 (299)
</pre><pre>T 3GVZ_A           45 NRDWKPDHAQSLRLLHPEHGYAYLGLYADNGSEPGIKAGVNQKGLAVVAA   94 (299)
</pre><pre>T ss_dssp             EEEECSCSCCEEEEECCSSSCCEEEEECCSSSSCSEEEEEETTSCEEEEE
</pre><pre>T ss_pred             ecCCCCccCCeEEEECCCCCeeEEEEEecCCCCCceeeEEecCCEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>PF18287.3 ; Hfx_Cass5 ; Integron Cassette Protein Hfx_Cass5
</pre><pre>Probab=37.56  E-value=78  Score=23.55  Aligned_cols=19  Identities=26%  Similarity=0.534  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             EEEecCCCCEEEEecCceE
</pre><pre>Q FD01848339_033   60 TIGFNEAGELFAISQGGAF   78 (113)
</pre><pre>Q Consensus        60 ~iG~NesGq~yi~s~~~~f   78 (113)
</pre><pre>                      .|+++++|+|||.-....|
</pre><pre>T Consensus         5 ~I~Id~~g~L~v~p~~~~f   23 (77)
</pre><pre>T V5BEQ7_9GAMM/4    5 IIGIDESKRLYVTPMREEF   23 (77)
</pre><pre>T ss_pred             EEEECCCCCEEEEcCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>1NUN_A Fibroblast growth factor-10; BETA-TREFOIL FOLD, IMMUNOGLOBULIN-LIKE DOMAIN, HORMONE-GROWTH FACTOR-MEMBRANE PROTEIN COMPLEX; HET: SO4, MSE, 15P; 2.9A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=37.55  E-value=1.7e+02  Score=21.77  Aligned_cols=35  Identities=17%  Similarity=0.341  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCce-eEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGAW-SVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG~-siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +.+.+|++++.+-+. .|....++.-+.||+.|++|
</pre><pre>T Consensus        43 ~~~~~l~~~~~~~~~v~i~~~~s~~yL~m~~~G~l~   78 (145)
</pre><pre>T 1NUN_A           43 CPYSILEITSVEIGVVAVKAINSNYYLAMNKKGKLY   78 (145)
</pre><pre>T ss_dssp             CTTSCEEEEEEETTEEEEEETTTTEEEEECTTSBEE
</pre><pre>T ss_pred             CCCceEEEEEEecceEEEEEcccceEEEeeccCcee
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>TIGR04248 SCM_PqqD_rel; SynChlorMet cassette protein ScmD. A biosynthesis cassette found in Syntrophobacter fumaroxidans MPOB, Chlorobium limicola DSM 245, Methanocella paludicola SANAE, and delta proteobacterium NaphS2 contains two PqqE-like radical SAM/SPASM domain proteins, a PqqD homolog, and a conserved hypothetical protein.
</pre><pre>Probab=37.21  E-value=1.1e+02  Score=20.14  Aligned_cols=33  Identities=9%  Similarity=0.066  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             eecCCceEEEecCCceeEEEcCCCCEEEecCCC
</pre><pre>Q FD01848339_033   35 YSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAG   67 (113)
</pre><pre>Q Consensus        35 ~~RNgllE~RtagGG~siTh~asGs~iG~NesG   67 (113)
</pre><pre>                      |++.+++-++..++-+-+.+..++..+-||+.+
</pre><pre>T Consensus         1 ~~~~~~~~~~~~~~~~~i~~~~~~~~~~ln~~~   33 (84)
</pre><pre>T TIGR04248         1 IANPLIVLREEFDDWAILFDPDDGQSFGLNPVS   33 (84)
</pre><pre>T ss_pred             CCCCccEEEEEeCCEEEEEeCCCCcEEEeCHHH
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>PF12545.10 ; DUF3739 ; Filamentous haemagglutinin family outer membrane protein
</pre><pre>Probab=36.77  E-value=3.9e+02  Score=21.39  Aligned_cols=69  Identities=14%  Similarity=0.244  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             CCcceeeecCCceEEE----ecCCceeEEEcCCCCEEEecCCCCEEEEecC------ceEEEecCCeEEEeCcce---eE
</pre><pre>Q FD01848339_033   29 ATKDFIYSRNGLMEIR----NAGGAWSVTHLGNGTTIGFNEAGELFAISQG------GAFISAAGDMNIKAGGNM---GI   95 (113)
</pre><pre>Q Consensus        29 ~tkD~V~~RNgllE~R----tagGG~siTh~asGs~iG~NesGq~yi~s~~------~~fi~~~G~~t~k~~g~i---~l   95 (113)
</pre><pre>                      |.-.++++.+|.+.+.    ..|.|....+...+..     .|.+|++.+.      ++.|.++|++++-+.--+   ++
</pre><pre>T Consensus        26 pp~~~~~d~~g~~~~~~~~~~~GSGI~tl~~~~~~~-----~G~v~L~AP~G~IdAgdAGIr~~g~v~i~A~~v~na~NI  100 (111)
</pre><pre>T A4YQ81_BRASO/4   26 PAIQLVYDQNAFPSIDPGGYVTGSGIGTLQASESAA-----PSSLYLLAPRGVIDFGTAGVRSSGNAVIVAPVIANASNF  100 (111)
</pre><pre>T ss_pred             CCceeeeCCCCCeEECCCCCcCCCceeecccCCCCC-----CCceeeEeCCcEEEcCCcceeecCcEEEEccEEeCccce
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeCeeeE
</pre><pre>Q FD01848339_033   96 EAGGVLG  102 (113)
</pre><pre>Q Consensus        96 ~a~g~~~  102 (113)
</pre><pre>                      ++.|...
</pre><pre>T Consensus       101 ~~~G~~~  107 (111)
</pre><pre>T A4YQ81_BRASO/4  101 QVSGSTT  107 (111)
</pre><pre>T ss_pred             Eeeceee
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>2EAY_A Biotin [acetyl-CoA-carboxylase] ligase; Biotin Biosynthesis, Dimer, Structural Genomics, NPPSFA, National Project on Protein Structural and Functional Analyses, RIKEN Structural; HET: MSE; 1.95A {Aquifex aeolicus}
</pre><pre>Probab=36.69  E-value=1.5e+02  Score=23.68  Aligned_cols=34  Identities=18%  Similarity=0.469  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCeEEEeC
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDMNIKAG   90 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~   90 (113)
</pre><pre>                      +|.-.|++++|.+.+...+....-..|++.+...
</pre><pre>T Consensus       200 ~g~~~gi~~~G~L~l~~~g~~~~~~~g~v~~~~~  233 (233)
</pre><pre>T 2EAY_A          200 TGKLVGLSEKGGALILTEEGIKEILSGEFSLRRS  233 (233)
</pre><pre>T ss_dssp             EEEEEEECTTSCEEEEETTEEEEECSCCEEEC--
</pre><pre>T ss_pred             EEEEEEeCCCCCEEEEeCCceEEEEeceEEEeeC
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>3IF4_D Integron Cassette Protein Hfx_Cass5; Integron Cassette Protein Mobile Metagenome Structural Genomics PSI-2 Protein Structure Initiative Midwest Center for Structural Genomics; HET: MSE; 2.181A {unidentified}
</pre><pre>Probab=36.37  E-value=81  Score=25.41  Aligned_cols=20  Identities=15%  Similarity=0.375  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             CEEEecCCCCEEEEecCceE
</pre><pre>Q FD01848339_033   59 TTIGFNEAGELFAISQGGAF   78 (113)
</pre><pre>Q Consensus        59 s~iG~NesGq~yi~s~~~~f   78 (113)
</pre><pre>                      +.|+++++|+|||.-...-|
</pre><pre>T Consensus        28 ~eI~Id~~grL~V~P~~~~f   47 (119)
</pre><pre>T 3IF4_D           28 AAIEIDGTGRIHVTPGESQF   47 (119)
</pre><pre>T ss_dssp             EEEEECTTCCEEEEESSCCC
</pre><pre>T ss_pred             eEEEECCCCCEEEEeCCCcC
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>3MY2_A Lipopolysaccharide export system protein lptC; LptC, lipopolysaccharide export pathway, Structural Genomics, Scottish Structural Proteomics Facility, SSPF, TRANSPORT PROTEIN; HET: MSE; 2.2A {Escherichia coli}
</pre><pre>Probab=35.48  E-value=3.4e+02  Score=20.33  Aligned_cols=47  Identities=11%  Similarity=0.197  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             ecCCC-CEEEEecCceEEEecCCeEEEeCcceeEeeCe----eeEEEeeeeeee
</pre><pre>Q FD01848339_033   63 FNEAG-ELFAISQGGAFISAAGDMNIKAGGNMGIEAGG----VLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        63 ~NesG-q~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g----~~~i~A~~v~id  111 (113)
</pre><pre>                      +++.| +.+.+.+....++.. + .+...|++.+....    .+.|.+.++.+|
</pre><pre>T Consensus        70 ~~~~g~~~~~i~a~~~~~~~~-~-~~~l~g~V~i~~~~~~~~~~~i~t~~l~~~  121 (175)
</pre><pre>T 3MY2_A           70 FDKDKIPTWSVKADKAKLTND-R-MLYLYGHVEVNALVPDSQLRRITTDNAQIN  121 (175)
</pre><pre>T ss_dssp             ECTTCCEEEEEECSEEEECTT-S-EEEEEEEEEEEECSSSCSEEEEEEEEEEEE
</pre><pre>T ss_pred             EcCCCCceEEEEecEEEEcCC-C-eEEEeCCEEEEeCCCCCCcEEEEeccEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>3IZO_G Fiber; human adenovirus fiber tail, pentameric penton base, trimeric fiber, VIRAL PROTEIN; 3.6A {Human adenovirus 5}
</pre><pre>Probab=34.94  E-value=3.8e+02  Score=26.31  Aligned_cols=67  Identities=24%  Similarity=0.377  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCc
</pre><pre>Q FD01848339_033   12 GYTPPDVDGAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGG   91 (113)
</pre><pre>Q Consensus        12 ~y~p~~vdGaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g   91 (113)
</pre><pre>                      ||.++.  |+|-.|.+.|-   .+.-|||.|--                     .|.|.+-...-+.++ .|.+++|.+.
</pre><pre>T Consensus        16 Py~~~~--g~~~~p~i~PP---f~~~~Gl~e~~---------------------~g~LsL~~~~PL~~~-~g~L~Lk~g~   68 (581)
</pre><pre>T 3IZO_G           16 PYDTET--GPPTVPFLTPP---FVSPNGFQESP---------------------PGVLSLRLSEPLVTS-NGMLALKMGN   68 (581)
</pre><pre>T ss_pred             CCCCCC--CCCCCCccCCC---eeCCCCceeCC---------------------CCcEEEEeCCCEEEe-CCeEEEeeCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   92 NMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        92 ~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      -+.+...|.|.+++
</pre><pre>T Consensus        69 GL~l~~~g~L~~~~   82 (581)
</pre><pre>T 3IZO_G           69 GLSLDEAGNLTSQN   82 (581)
</pre><pre>T ss_pred             CeEECCCCCEeecc
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>3IZO_H Fiber; human adenovirus fiber tail, pentameric penton base, trimeric fiber, VIRAL PROTEIN; 3.6A {Human adenovirus 5}
</pre><pre>Probab=34.94  E-value=3.8e+02  Score=26.31  Aligned_cols=67  Identities=24%  Similarity=0.377  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCc
</pre><pre>Q FD01848339_033   12 GYTPPDVDGAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGG   91 (113)
</pre><pre>Q Consensus        12 ~y~p~~vdGaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g   91 (113)
</pre><pre>                      ||.++.  |+|-.|.+.|-   .+.-|||.|--                     .|.|.+-...-+.++ .|.+++|.+.
</pre><pre>T Consensus        16 Py~~~~--g~~~~p~i~PP---f~~~~Gl~e~~---------------------~g~LsL~~~~PL~~~-~g~L~Lk~g~   68 (581)
</pre><pre>T 3IZO_H           16 PYDTET--GPPTVPFLTPP---FVSPNGFQESP---------------------PGVLSLRLSEPLVTS-NGMLALKMGN   68 (581)
</pre><pre>T ss_pred             CCCCCC--CCCCCCccCCC---eeCCCCceeCC---------------------CCcEEEEeCCCEEEe-CCeEEEeeCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   92 NMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        92 ~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      -+.+...|.|.+++
</pre><pre>T Consensus        69 GL~l~~~g~L~~~~   82 (581)
</pre><pre>T 3IZO_H           69 GLSLDEAGNLTSQN   82 (581)
</pre><pre>T ss_pred             CeEECCCCCEeecc
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>5NY0_A L. reuteris SRRP binding region; beta-solenoid, L. reuteri, adhesin, CELL ADHESION; 2.0A {Lactobacillus reuteri 100-23}
</pre><pre>Probab=34.87  E-value=4.5e+02  Score=23.22  Aligned_cols=75  Identities=7%  Similarity=0.178  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             CcceeeecCCceEEEecCCceeEEEc--CCCCEEEecCCCCEEEEecCceEEEecCC----eEEEeCcceeEeeCeeeEE
</pre><pre>Q FD01848339_033   30 TKDFIYSRNGLMEIRNAGGAWSVTHL--GNGTTIGFNEAGELFAISQGGAFISAAGD----MNIKAGGNMGIEAGGVLGI  103 (113)
</pre><pre>Q Consensus        30 tkD~V~~RNgllE~RtagGG~siTh~--asGs~iG~NesGq~yi~s~~~~fi~~~G~----~t~k~~g~i~l~a~g~~~i  103 (113)
</pre><pre>                      ...+....+.-|...+.+....+...  ...++|-|.+...++|.+...-.+..++.    +.+...+.+.|+.+....+
</pre><pre>T Consensus       239 ~~~~~v~~~a~v~i~~~~~~~~~~~~~~~~~~~i~i~~~s~l~i~~~~~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~  318 (368)
</pre><pre>T 5NY0_A          239 SNSFTVGEGSTVKLSAKVDNVRILPYEDSNTANVSFAKGSDVTLHAGTGSNLRMGASISNQIDFNGKATFIKDSGAYANT  318 (368)
</pre><pre>T ss_dssp             EEEEEECTTCEEEEEESSEEEEECCSSSSCEEEEEECTTCEEEEEESSSCSEEECCSSEEEEEECSEEEEEECC------
</pre><pre>T ss_pred             CCeEEeCCCCEEEEecCCCCceecCccCCCCCeEEECCCCEEEEEeCCCCCeEeecccCcEEEECCCceEEEcCCcccCc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             E
</pre><pre>Q FD01848339_033  104 R  104 (113)
</pre><pre>Q Consensus       104 ~  104 (113)
</pre><pre>                      .
</pre><pre>T Consensus       319 ~  319 (368)
</pre><pre>T 5NY0_A          319 A  319 (368)
</pre><pre>T ss_dssp             -
</pre><pre>T ss_pred             c
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>PF10610.11 ; Tafi-CsgC ; Thin aggregative fimbriae synthesis protein
</pre><pre>Probab=34.85  E-value=2.7e+02  Score=20.00  Aligned_cols=67  Identities=6%  Similarity=0.117  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             eEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   41 MEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        41 lE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      |++...++.+.|+......+-.   ++.+-+.-...   ..+|+.+.+-.|.+.+.+++...+.-..|.++.+
</pre><pre>T Consensus        12 i~~~~~~~~~~i~~~v~~~~~~---~~~Y~l~v~k~---g~sG~s~~~Q~G~f~l~~~~~~~Ls~~~l~~~~~   78 (99)
</pre><pre>T A8AI37_CITK8/6   12 FNTTQQGEIYTIVPQVTLSKPC---VCEVQIFAVRD---GKGGQSRTQQKTTLSLPANQPIDLTKLSLNISPE   78 (99)
</pre><pre>T ss_pred             EEEEEeCCeEEEEEEEEECCCe---EEEEEEEEEEe---CCCCceeeceeeEEEcCCCCcEEeeEEEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>3LYC_B Putative pectinase; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesion, CELL ADHESION; HET: MSE; 2.3A {Parabacteroides distasonis}
</pre><pre>Probab=34.76  E-value=3.9e+02  Score=20.74  Aligned_cols=64  Identities=11%  Similarity=0.068  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eEEEec-CCceeEEEcCCCCEEEecCCCCEEEEe--cCceEEEe--cCCeEEEeCcceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   41 MEIRNA-GGAWSVTHLGNGTTIGFNEAGELFAIS--QGGAFISA--AGDMNIKAGGNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        41 lE~Rta-gGG~siTh~asGs~iG~NesGq~yi~s--~~~~fi~~--~G~~t~k~~g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      |..... .|-..+.....-..+-+..+|.+.+..  .+.+.+..  .|++.+.....+.+...|.-.|.
</pre><pre>T Consensus       151 l~~~~~g~g~i~~~g~~~~~~~~~~~~g~i~~~~l~~~~~~~~~~~~g~i~v~~~~~l~~~~~g~g~i~  219 (241)
</pre><pre>T 3LYC_B          151 LNGDMAGSNTIVLGGTVGIAEFSIAGSGTVRAFDCTMDELECKIAGSGDIEAFVVNKIKAEIAGSGSVK  219 (241)
</pre><pre>T ss_dssp             EEEEEESSCEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEESSCEEE
</pre><pre>T ss_pred             EEEEEecceEEEEEeEEEEEEEEEecceEEEEEeeEEeEEEEEEeccceEEEEEEeeEEEEEeceeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>3LYC_G Putative pectinase; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesion, CELL ADHESION; HET: MSE; 2.3A {Parabacteroides distasonis}
</pre><pre>Probab=34.76  E-value=3.9e+02  Score=20.74  Aligned_cols=64  Identities=11%  Similarity=0.068  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eEEEec-CCceeEEEcCCCCEEEecCCCCEEEEe--cCceEEEe--cCCeEEEeCcceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   41 MEIRNA-GGAWSVTHLGNGTTIGFNEAGELFAIS--QGGAFISA--AGDMNIKAGGNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        41 lE~Rta-gGG~siTh~asGs~iG~NesGq~yi~s--~~~~fi~~--~G~~t~k~~g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      |..... .|-..+.....-..+-+..+|.+.+..  .+.+.+..  .|++.+.....+.+...|.-.|.
</pre><pre>T Consensus       151 l~~~~~g~g~i~~~g~~~~~~~~~~~~g~i~~~~l~~~~~~~~~~~~g~i~v~~~~~l~~~~~g~g~i~  219 (241)
</pre><pre>T 3LYC_G          151 LNGDMAGSNTIVLGGTVGIAEFSIAGSGTVRAFDCTMDELECKIAGSGDIEAFVVNKIKAEIAGSGSVK  219 (241)
</pre><pre>T ss_dssp             EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
</pre><pre>T ss_pred             EEEEEecceEEEEEeEEEEEEEEEecceEEEEEeeEEeEEEEEEeccceEEEEEEeeEEEEEeceeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>TIGR04409 LptC_YrbK; LPS export ABC transporter periplasmic protein LptC. Members of this family are LptC, a periplasmic protein tethered to the inner membrane in the lipopolysaccharide (LPS)exporter transenvelope complex (Lpt), which is responsible for conducting LPS to the outer leaflet of the outer membrane in most Gram-negative bacteria.
</pre><pre>Probab=34.34  E-value=3.5e+02  Score=20.19  Aligned_cols=52  Identities=13%  Similarity=0.107  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             EEecCCCC-EEEEecCceEEEecCCeEEEeCcceeEeeCee---eEEEeeeeeeeC
</pre><pre>Q FD01848339_033   61 IGFNEAGE-LFAISQGGAFISAAGDMNIKAGGNMGIEAGGV---LGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        61 iG~NesGq-~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~---~~i~A~~v~idk  112 (113)
</pre><pre>                      ..++++|+ .|.+.+..+..+...+...-.+-.+.+.....   +.|+|.+..+|.
</pre><pre>T Consensus        47 ~~~~~~G~~~~~i~a~~~~~~~~~~~~~l~~~~~~~~~~~~~~~~~i~a~~~~~~~  102 (180)
</pre><pre>T TIGR04409        47 TQYDEDGKLQYRLTADKAEHYPDDGTTEFTNPVLILFDEDGKPTWTITADRAKLNK  102 (180)
</pre><pre>T ss_pred             EEECCCCCEEEEEEEeeEEEeCCCCeEEEECCEEEEECCCCCEEEEEEECEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>PF13100.8 ; OstA_2 ; OstA-like protein
</pre><pre>Probab=33.63  E-value=3e+02  Score=19.21  Aligned_cols=43  Identities=7%  Similarity=-0.038  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCceEEEe--cCCeEEEeCcceeEeeCeeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   69 LFAISQGGAFISA--AGDMNIKAGGNMGIEAGGVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        69 ~yi~s~~~~fi~~--~G~~t~k~~g~i~l~a~g~~~i~A~~v~idk  112 (113)
</pre><pre>                      ++++....+....  ..+-.+...|++.++. +.+.|+|..+.+++
</pre><pre>T Consensus         4 i~~i~a~~~~~~~~~~~~~~~~~~g~v~i~~-~~~~l~ad~~~~~~   48 (159)
</pre><pre>T A6KXH5_BACV8/4    4 VYLLHSDVLKKSPLNPDPDAQILIGNVAFRH-DSVYMYCDSACFYE   48 (159)
</pre><pre>T ss_pred             EEEEechhceecCCCCCCCeEEEEecEEEec-CCeEEEEeEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>TIGR00121 birA_ligase; birA, biotin-[acetyl-CoA-carboxylase] ligase region. This model represents the biotin--acetyl-CoA-carboxylase ligase region of biotin--acetyl-CoA-carboxylase ligase.
</pre><pre>Probab=33.34  E-value=3.4e+02  Score=21.87  Aligned_cols=48  Identities=17%  Similarity=0.344  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             eecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEE
</pre><pre>Q FD01848339_033   35 YSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIK   88 (113)
</pre><pre>Q Consensus        35 ~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k   88 (113)
</pre><pre>                      +.++..+.+...++.+.      |.-.|++++|.|.+...+....-..|++.+.
</pre><pre>T Consensus       190 ~~~g~~V~v~~~~~~~~------G~~~gId~~G~L~i~~~~~~~~~~sg~v~~~  237 (237)
</pre><pre>T TIGR00121       190 AHIGREVSLTTGNGEIE------GIARGIDKDGALLLEDGGGIKKIISGEISLR  237 (237)
</pre><pre>T ss_pred             cccCCeEEEecCCCeEE------EEEEEECCCCCEEEEeCCeeEEEEeeeEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>3JX8_A Putative lipoprotein; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesin, CELL ADHESION; HET: MSE, SO4, GOL; 2.16A {Parabacteroides distasonis ATCC 8503}
</pre><pre>Probab=32.08  E-value=4.4e+02  Score=20.58  Aligned_cols=65  Identities=17%  Similarity=0.206  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eEEEec-CCceeEEEcCCCCEEEecCCCCEEEE----ecCceEEEecCCeEEEeCcceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   41 MEIRNA-GGAWSVTHLGNGTTIGFNEAGELFAI----SQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        41 lE~Rta-gGG~siTh~asGs~iG~NesGq~yi~----s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      |..... .|-..+.....-..+-+..+|++.+.    ....+..+..|++.+...+.+.++..|.-.|..
</pre><pre>T Consensus       161 l~~~~~~~g~i~l~g~~~~~~~~~~~~g~i~~~~~~~~~~~~~~~g~g~i~~~~~~~l~~~~~g~g~i~~  230 (250)
</pre><pre>T 3JX8_A          161 LSCSLASSGEIEVIGTVDRASFNVAGSGEIKAFDCQARKAECNIASSGEISVYATQILDANIVGSGEIHY  230 (250)
</pre><pre>T ss_dssp             EEEEEESSCEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEESSCEEEE
</pre><pre>T ss_pred             EEEEEecCceEEEEEEeceEEEEEeCceeEEeeeeEEEEEEEEEccCcEEEEEEeeeEEEEEecceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>3JX8_F Putative lipoprotein; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesin, CELL ADHESION; HET: GOL, SO4, MSE; 2.16A {Parabacteroides distasonis ATCC 8503}
</pre><pre>Probab=32.08  E-value=4.4e+02  Score=20.58  Aligned_cols=65  Identities=17%  Similarity=0.206  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eEEEec-CCceeEEEcCCCCEEEecCCCCEEEE----ecCceEEEecCCeEEEeCcceeEeeCeeeEEEe
</pre><pre>Q FD01848339_033   41 MEIRNA-GGAWSVTHLGNGTTIGFNEAGELFAI----SQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRA  105 (113)
</pre><pre>Q Consensus        41 lE~Rta-gGG~siTh~asGs~iG~NesGq~yi~----s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A  105 (113)
</pre><pre>                      |..... .|-..+.....-..+-+..+|++.+.    ....+..+..|++.+...+.+.++..|.-.|..
</pre><pre>T Consensus       161 l~~~~~~~g~i~l~g~~~~~~~~~~~~g~i~~~~~~~~~~~~~~~g~g~i~~~~~~~l~~~~~g~g~i~~  230 (250)
</pre><pre>T 3JX8_F          161 LSCSLASSGEIEVIGTVDRASFNVAGSGEIKAFDCQARKAECNIASSGEISVYATQILDANIVGSGEIHY  230 (250)
</pre><pre>T ss_dssp             EEEEEESSCEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEESSCEEEE
</pre><pre>T ss_pred             EEEEEecCceEEEEEEeceEEEEEeCceeEEeeeeEEEEEEEEEccCcEEEEEEeeeEEEEEecceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>PF09992.11 ; NAGPA ; Phosphodiester glycosidase
</pre><pre>Probab=31.63  E-value=1.4e+02  Score=22.51  Aligned_cols=36  Identities=19%  Similarity=0.158  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             cCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEe
</pre><pre>Q FD01848339_033   37 RNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAIS   73 (113)
</pre><pre>Q Consensus        37 RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s   73 (113)
</pre><pre>                      |||-+. ......|..+....-+.||++.+|+++++.
</pre><pre>T Consensus        84 ~~G~~~-~~~~~~~~~~~~~~Rt~ig~~~~g~~~~~~  119 (191)
</pre><pre>T A0A374AHK9_9BA   84 DGGKVV-EYTSGSSFYTRRHPRTLIGKRADGTVVMAV  119 (191)
</pre><pre>T ss_pred             eCCEEc-ccCCCCCccccCCCeEEEEEcCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>2N59_A Putative uncharacterized protein CsgH; UNKNOWN FUNCTION; NMR {Rhodopseudomonas palustris DX-1}
</pre><pre>Probab=31.32  E-value=3.9e+02  Score=19.71  Aligned_cols=67  Identities=18%  Similarity=0.202  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             eEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   41 MEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        41 lE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      |+....++++.|+.......   .-++++-+.-...   ..+|..+..-.|++.|.+.....+.-..|.++.+
</pre><pre>T Consensus         6 I~~~~~~~~~~i~~~v~~~~---~~~~~Y~l~v~~~---g~~G~s~~~QsG~f~l~~~~~~~Ls~~~l~~~~~   72 (106)
</pre><pre>T 2N59_A            6 VEAAVSGGHVTLQGVITAVR---DGAGSYKLAVDKA---GAAGTSRIKQAGAFTAIAEQRVTVGNVVLDYSSA   72 (106)
</pre><pre>T ss_dssp             CEEEEETTEEEEEEEEEESS---SEEEEEEEEEEEE---ETTEEEEEEEEEEEEECBTEEEEEEEEEEECCTT
</pre><pre>T ss_pred             EEEEEECCeEEEEEEEEECC---CeeEEEEEEEEEc---CCCCceEEEEEeeEEeCCCCEEEEEEEEEeCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>3BFM_A Biotin protein ligase-like protein of unknown function; Structural genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, unknown function; HET: MSE, 2PE; 1.7A {Silicibacter sp.}
</pre><pre>Probab=31.14  E-value=1.8e+02  Score=23.32  Aligned_cols=32  Identities=13%  Similarity=0.217  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCeEEE
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDMNIK   88 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k   88 (113)
</pre><pre>                      .|.-.|+|++|.+.+........-..|++.++
</pre><pre>T Consensus       203 ~g~~~gi~~~G~l~l~~~~g~~~~~~~~v~~~  234 (235)
</pre><pre>T 3BFM_A          203 SGTFLGVDEDFGMLLRDETTTHLIPLTTVLVQ  234 (235)
</pre><pre>T ss_dssp             EEEEEEECTTCCEEEECSSCEEEECGGGGSCC
</pre><pre>T ss_pred             eeEEEeeCCCCCEEEECCCCcEEEecceeEEe
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>4OEE_A Fibroblast growth factor 2; Heparin/Heparin Sulfate Binding, PROTEIN BINDING; HET: IDY, GNX; 1.5A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=30.66  E-value=2.4e+02  Score=21.63  Aligned_cols=47  Identities=9%  Similarity=0.157  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...+..+.|. .+...+|+++..+.| ..|.+..++.-+.||..|++|
</pre><pre>T Consensus        45 ~dG~V~gt~~~-~~~~s~l~~~~~~~g~v~i~~~~s~~yLcmn~~G~l~   92 (155)
</pre><pre>T 4OEE_A           45 PDGRVDGVREK-SDPHIKLQLQAEERGVVSIKGVSANRYLAMKEDGRLL   92 (155)
</pre><pre>T ss_dssp             TTSCEEEESCT-TCTTCCEEEEEEETTEEEEEETTTTEEEEECTTSCEE
</pre><pre>T ss_pred             CCCcEEcccCC-CCccceeEEEEcCCcEEEEcccccceeEEEcCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>2P23_B Fibroblast growth factor 19; atypical beta-trefoil fold, SIGNALING PROTEIN; 1.8A {Homo sapiens} SCOP: b.42.1.0
</pre><pre>Probab=30.58  E-value=2.1e+02  Score=23.39  Aligned_cols=47  Identities=11%  Similarity=0.145  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc-eeEEEcCCCCEEEecCCCCEEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA-WSVTHLGNGTTIGFNEAGELFA   71 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG-~siTh~asGs~iG~NesGq~yi   71 (113)
</pre><pre>                      +...+..|.|.  +.+.+|++++.+-| ..|....++.-+.||..|.+|.
</pre><pre>T Consensus        42 ~dG~V~Gt~~~--~~~~~l~i~s~~~g~v~I~g~~s~~YLcm~~~G~L~~   89 (194)
</pre><pre>T 2P23_B           42 ADGVVDCARGQ--SAHSLLEIKAVALRTVAIKGVHSVRYLCMGADGKMQG   89 (194)
</pre><pre>T ss_dssp             TTSBEEEESSC--CTTTCEEEEEEETTEEEEEETTTCCEEEECGGGCEEE
</pre><pre>T ss_pred             CCCeEeccCCC--CCCcEEEEEEEecceeEecceeeeeEEEECCCCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>1BIA_A BirA BIFUNCTIONAL PROTEIN; TRANSCRIPTION REGULATION; 2.3A {Escherichia coli} SCOP: b.34.1.1, a.4.5.1, d.104.1.2
</pre><pre>Probab=30.42  E-value=2.1e+02  Score=24.24  Aligned_cols=36  Identities=17%  Similarity=0.319  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcc
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGN   92 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~   92 (113)
</pre><pre>                      .|.-.|++++|.|.+...+....-..|++.+....+
</pre><pre>T Consensus       286 ~g~~~gi~~~G~L~l~~~g~~~~~~~g~i~~~~~~~  321 (321)
</pre><pre>T 1BIA_A          286 FGISRGIDKQGALLLEQDGIIKPWMGGEISLRSAEK  321 (321)
</pre><pre>T ss_dssp             EEEEEEECTTSCEEEEETTEEEEESSCEEEEC----
</pre><pre>T ss_pred             EEEEEeeCCCCCEEEEECCEEEEEecceEEEeeCCC
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>PF02237.19 ; BPL_C ; Biotin protein ligase C terminal domain
</pre><pre>Probab=30.37  E-value=2e+02  Score=16.19  Aligned_cols=30  Identities=23%  Similarity=0.432  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCEEEecCCCCEEEEecCceEEEecCCeEE
</pre><pre>Q FD01848339_033   58 GTTIGFNEAGELFAISQGGAFISAAGDMNI   87 (113)
</pre><pre>Q Consensus        58 Gs~iG~NesGq~yi~s~~~~fi~~~G~~t~   87 (113)
</pre><pre>                      |...+++++|.+.+.....-..-..|++.+
</pre><pre>T Consensus        18 g~~~~i~~~G~l~v~~~~g~~~~~~~~i~~   47 (48)
</pre><pre>T BICOA_NEIMB/27   18 GTVKGVDGQGVLHLETAEGKQTVVSGEISL   47 (48)
</pre><pre>T ss_pred             EEEEeeCCCCcEEEEcCCCcEEEEEeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>1NNX_A Protein ygiW; ygiW, Structural Genomics, hypothetical protein, OB-fold, Structure 2 Function Project, S2F, UNKNOWN FUNCTION; 1.45A {Escherichia coli} SCOP: b.40.10.1
</pre><pre>Probab=30.20  E-value=4e+02  Score=19.54  Aligned_cols=59  Identities=8%  Similarity=0.170  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEEcCCCCEEEecCCCCEEEEecCceE----EEecCCeEEEeCcceeEeeCeeeEEEeeeeee
</pre><pre>Q FD01848339_033   49 AWSVTHLGNGTTIGFNEAGELFAISQGGAF----ISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKV  110 (113)
</pre><pre>Q Consensus        49 G~siTh~asGs~iG~NesGq~yi~s~~~~f----i~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~i  110 (113)
</pre><pre>                      |+-+.......-+=-+.+|+++|.-..+.|    +..+-.+.|.  |.+.-+. +...|.|..|.+
</pre><pre>T Consensus        44 G~I~~~~~~~~y~~~D~tG~I~V~i~~~~~~~~~~~~g~~V~V~--G~v~~~~-~~~~i~a~~i~~  106 (109)
</pre><pre>T 1NNX_A           44 GNIVERISDDLYVFKDASGTINVDIDHKRWNGVTVTPKDTVEIQ--GEVDKDW-NSVEIDVKQIRK  106 (109)
</pre><pre>T ss_dssp             EEEEEEEETTEEEEEETTEEEEEECCGGGSTTCCCCTTSCEEEE--EEEEEET-TEEEEEEEEEEE
</pre><pre>T ss_pred             EEEEEEecCCEEEEEcCCCEEEEEeCchhhCCCcCCCCCEEEEE--EEEeCCC-CeEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>PF06739.13 ; SBBP ; Beta-propeller repeat
</pre><pre>Probab=29.77  E-value=1.8e+02  Score=15.34  Aligned_cols=16  Identities=19%  Similarity=0.343  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCEEEecCCCCEEEEe
</pre><pre>Q FD01848339_033   58 GTTIGFNEAGELFAIS   73 (113)
</pre><pre>Q Consensus        58 Gs~iG~NesGq~yi~s   73 (113)
</pre><pre>                      ...+.++.+|++|+.+
</pre><pre>T Consensus        15 ~~~i~~d~~g~~~v~G   30 (37)
</pre><pre>T Q8F8X3_LEPIN/4   15 PGKIGIDFNRNLYTTG   30 (37)
</pre><pre>T ss_pred             cceEEEcCCCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>d2oyza1 b.82.1.22 (A:2-94) Uncharacterized protein VPA0057 {Vibrio parahaemolyticus [TaxId: 670]}
</pre><pre>Probab=29.66  E-value=3.8e+02  Score=19.46  Aligned_cols=58  Identities=16%  Similarity=0.291  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEEcCCCCEEEecCCCCE-EEEecCceEEEecCCeEEEeCc---ceeEeeCeeeEEEee
</pre><pre>Q FD01848339_033   49 AWSVTHLGNGTTIGFNEAGEL-FAISQGGAFISAAGDMNIKAGG---NMGIEAGGVLGIRAA  106 (113)
</pre><pre>Q Consensus        49 G~siTh~asGs~iG~NesGq~-yi~s~~~~fi~~~G~~t~k~~g---~i~l~a~g~~~i~A~  106 (113)
</pre><pre>                      +|.+.....-.++|.-..|++ |.....+.|.=..|.+.+...+   ...|.++-.|.|.+.
</pre><pre>T Consensus        14 s~~~~~~~~~~~~g~~~pG~~h~ht~~~E~~~vl~G~l~i~~~~~~~~~~l~~Gd~~~vp~g   75 (93)
</pre><pre>T d2oyza1          14 SLGFNQHGQDVSVGVMLPGEYTFGTQAPERMTVVKGALVVKRVGEADWTTYSSGESFDVEGN   75 (93)
</pre><pre>T ss_dssp             EEEEEETTEEEEEEEECSEEEEEEESSCEEEEEEESEEEEEETTCSSCEEEETTCEEEECSS
</pre><pre>T ss_pred             EEEEEcCCceEEEEEECCceEecCCCCCEEEEEEEEEEEEEECCcccEEEEcCCCEEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>PF03961.15 ; FapA ; Flagellar Assembly Protein A
</pre><pre>Probab=29.58  E-value=5.2e+02  Score=23.36  Aligned_cols=58  Identities=24%  Similarity=0.307  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             ecCCceeEEEcCCCCEEEecCCCCEEEE-ecCc---eEEEecCCeEEEeCcceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   45 NAGGAWSVTHLGNGTTIGFNEAGELFAI-SQGG---AFISAAGDMNIKAGGNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        45 tagGG~siTh~asGs~iG~NesGq~yi~-s~~~---~fi~~~G~~t~k~~g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      .+.|-..|...-.+++|--  .|.++|. +--.   ..|.++|++.++..-+-.+++.|++.|.
</pre><pre>T Consensus       209 ~a~g~i~i~G~v~~a~I~a--~g~i~i~~gi~~~~~~~I~a~g~i~~~~~~~~~i~a~g~i~i~  270 (441)
</pre><pre>T Q8EQV6_OCEIH/7  209 KAEGDIRIHGVVEGAKIIA--GGSVFIMEGIAGKNIGSIQAGNDIILGYCNHSFIQAGNDLYVE  270 (441)
</pre><pre>T ss_pred             EECCCEEEeCeEeccEEEE--CCCEEEecCeecCCccEEEeCCcEEEEeeecCEEEeCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>6ORJ_A PHIKZ164; Membrane puncturing device, beta-helix, OB-fold, trimer, VgrG-like, VIRAL PROTEIN; HET: MSE; 2.37A {Pseudomonas virus phiKZ}
</pre><pre>Probab=29.53  E-value=8.3e+02  Score=22.99  Aligned_cols=77  Identities=9%  Similarity=0.085  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             cCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   37 RNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        37 RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      ++|.+.+...-|-+-.-+.....---.|.+|-++-+--.++++.+-.++.+..+|+.----...+..+-..+.++.+
</pre><pre>T Consensus       188 ~~G~~~l~Dd~gN~~~LdS~e~~~~~~N~d~S~i~inKk~I~~~a~~~I~~~~~~~~~~~~~~~i~~~t~~~~~~~~  264 (293)
</pre><pre>T 6ORJ_A          188 GTGAVYILDDIGNRFELVSKDKRLMLMNADNSFVKVEKKAIDLNADQYIKLTSGGSTLELNPTEFKVNTTNTTIKSS  264 (293)
</pre><pre>T ss_dssp             TTTEEEEEESSSCEEEEETTTTEEEEECTTSCEEEEETTEEEEECSSCEEEEETTEEEEEETTEEEEECSCCCCCCS
</pre><pre>T ss_pred             CCeEEEEEeCCCCEEEEEcCCCeEEEEcCCCCEEEecccEEEEEeCceEEEEeCCceEEEcCcceEecceeeEeccc
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>2FDB_N fibroblast growth factor 8 isoform B; beta-trefoil fold  immunoglobulin fold, HORMONE-GROWTH FACTOR-TRANSFERASE COMPLEX; 2.28A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=28.60  E-value=3.4e+02  Score=20.81  Aligned_cols=47  Identities=15%  Similarity=0.065  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEEecCCc--eeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIRNAGGA--WSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~RtagGG--~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +...++.+.|. .+...++++.+.+.+  ..|....++.-+.||..|++|
</pre><pre>T Consensus        47 ~~G~V~g~~~~-~~~~s~l~~~~~~~~~~v~i~~~~s~~yl~~~~~G~l~   95 (164)
</pre><pre>T 2FDB_N           47 ANKRINAMAED-GDPFAKLIVETDTFGSRVRVRGAETGLYICMNKKGKLI   95 (164)
</pre><pre>T ss_dssp             TTSCEEEEECT-TCGGGCEEEEEEETTTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CCCeEEcccCC-CCCCcEEEEEEcCCCCEEEEEecCceeEEEECCCccEE
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>3PET_B Putative adhesin; RIGHT-HANDED BETA-HELIX, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, CELL ADHESION; HET: MSE, PG4; 2.07A {Bacteroides fragilis NCTC 9343}
</pre><pre>Probab=28.22  E-value=4.8e+02  Score=19.86  Aligned_cols=64  Identities=16%  Similarity=0.200  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             eEEEe-cCCceeEEEcCCCCEEEecCCCCEEEEec--CceEEEe--cCCeEEEeCcceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   41 MEIRN-AGGAWSVTHLGNGTTIGFNEAGELFAISQ--GGAFISA--AGDMNIKAGGNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        41 lE~Rt-agGG~siTh~asGs~iG~NesGq~yi~s~--~~~fi~~--~G~~t~k~~g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      |...+ ..|-..+.....-..+-+..+|.+.+..-  ..+.+.+  .|++.+...+.+.+++.|.-.|.
</pre><pre>T Consensus       132 l~~~~~~~g~i~l~g~~~~~~~~~sg~g~i~~~~~~~~~~~~~~~~~g~i~v~~~~~l~~~~~g~g~i~  200 (221)
</pre><pre>T 3PET_B          132 LNVQSMGVGDVKLEGTAQIAALHSKGVGNIEAGNLRANAVEASSQGVGDITCNATESIDAAVRGVGSIK  200 (221)
</pre><pre>T ss_dssp             EEEEECSBSCEEEEEEEEEEEEEECSBSCEEEEEEEEEEEEEEECSBSCEEEEEEEEEEEEECSBSCEE
</pre><pre>T ss_pred             EEEEEccccEEEEEEEEeEEEEEEcCceeEEecceEecEEEEEEcccCeEEEEecCeEEEEEecccEEE
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>3P6J_A de novo designed beta-trefoil architecture with symmetric primary structure; beta-trefoil, DE NOVO PROTEIN; 1.35A {N/A}
</pre><pre>Probab=28.16  E-value=3.9e+02  Score=18.82  Aligned_cols=50  Identities=12%  Similarity=0.110  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecC
</pre><pre>Q FD01848339_033   26 STSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQG   75 (113)
</pre><pre>Q Consensus        26 ~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~   75 (113)
</pre><pre>                      .+...++....+.-+.++....|.+.|.....|.-++++.+|.+.-....
</pre><pre>T Consensus        93 ~l~~~~~~~~~~~~f~~~~~~~g~~~i~~~~~g~~l~~~~~G~~~~~~~~  142 (142)
</pre><pre>T 3P6J_A           93 TVDGTRDRSDTHIQFQISPEGNGEVLLKSTETGQYLRINPDGTVDGTRDR  142 (142)
</pre><pre>T ss_dssp             BEEEECCTTCGGGCEEEEECSTTCEEEEETTTCCEEEECTTSBEEEEC--
</pre><pre>T ss_pred             CEEccCCCCCCceEEEEEEcCCcEEEEEEcCCCcEEEECCCCCeeeccCC
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>1PWA_A Fibroblast growth factor-19; BETA TREFOIL, DISULPHIDE BONDS, HORMONE-GROWTH FACTOR COMPLEX; 1.3A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=27.94  E-value=3.3e+02  Score=21.03  Aligned_cols=46  Identities=13%  Similarity=0.151  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCcceeeecCCceEEE-ecCCceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   23 SMPSTSATKDFIYSRNGLMEIR-NAGGAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        23 ~~p~~s~tkD~V~~RNgllE~R-tagGG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +....+.+.|  -+...+|++. ...|-..|....++.=+.||..|.+|
</pre><pre>T Consensus        30 ~~G~V~g~~~--~~~~~~l~i~s~~~~~v~I~g~~s~~ylcmn~~G~l~   76 (162)
</pre><pre>T 1PWA_A           30 ADGVVDCARG--QSAHSLLEIKAVALRTVAIKGVHSVRYLCMGADGKMQ   76 (162)
</pre><pre>T ss_dssp             TTSBEEEESS--CCGGGCEEEEEEETTEEEEEETTTCCEEEECGGGCEE
</pre><pre>T ss_pred             CCCeEeccCC--CCCCceEEEEEecCCeEEEcccccceEEEECCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>4V02_C PROBABLE SEPTUM SITE-DETERMINING PROTEIN MINC; CELL CYCLE, BACTERIAL CELL DIVISION, FTSZ, MIN SYSTEM; HET: ATP; 2.7A {AQUIFEX AEOLICUS}
</pre><pre>Probab=27.75  E-value=5.5e+02  Score=20.35  Aligned_cols=52  Identities=19%  Similarity=0.312  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEe--cCceEEEecCCeEEEeCcceeEeeC--ee-------eEEEeeeeee
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAIS--QGGAFISAAGDMNIKAGGNMGIEAG--GV-------LGIRAAEVKV  110 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s--~~~~fi~~~G~~t~k~~g~i~l~a~--g~-------~~i~A~~v~i  110 (113)
</pre><pre>                      +|..|-.  +|.+.|++  ...+.|.+.|++.|...=.=.+.|+  |+       ..+.+.+|.|
</pre><pre>T Consensus        14 sGq~i~~--~gdvii~GdV~~Ga~V~a~Gni~V~G~l~G~v~AG~~g~~~a~I~a~~~~~~~I~I   76 (128)
</pre><pre>T 4V02_C           14 AGQRIEH--RGDILILGDVNKDAEVLAGGNIIVMGKLRGVAKAGLIGDHSAVIVALKMEPQLLQI   76 (128)
</pre><pre>T ss_dssp             TTCEEEE--SSEEEESSCBCTTCEEEESSCEEEEEEECSEEEECTTSCTTCCEEEEEECCSEEEE
</pre><pre>T ss_pred             CCCEEEE--CCcEEEECCCCCCCEEEECCCEEEeeeeCCEEEcCCCCCccEEEEEEecCcceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>6SK0_B Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=27.05  E-value=2.5e+02  Score=26.27  Aligned_cols=91  Identities=22%  Similarity=0.251  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEe-CcceeEeeC
</pre><pre>Q FD01848339_033   20 GAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKA-GGNMGIEAG   98 (113)
</pre><pre>Q Consensus        20 GaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~-~g~i~l~a~   98 (113)
</pre><pre>                      .+|..-.++..++....=.+-+...++++.++++ ......+.-.++=.++....+--++.+.|++.+.+ .+++.+.+.
</pre><pre>T Consensus       629 ~~~~~i~~~~~~~~~~~a~~~~~~~~ag~~~~~~-~~~~~~~~a~~~~~~~~~~~g~~~~~~~g~v~~~a~~~~~~~~a~  707 (841)
</pre><pre>T 6SK0_B          629 HAPDGIGMVSARALRIASGSESVGIMSGDNTDIT-AGQSFTVVAEGAVSLLSRNQGMQLLAAKGRVNIQAQSDDLSMSSQ  707 (841)
</pre><pre>T ss_dssp             ECTTCEEEEESSCEEEEESSSCEEEEESSCEEEE-ESSCEEEEESSCEEEEESSSCEEEEESSSCEEEEETTSCEEEEES
</pre><pre>T ss_pred             ecCCcceeccHhhhhHhccCccEEEEeCCceEEe-ecceEEEEEhhhHHHHHHhhcceeEEccCCeEEEEcCCceeEeec
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeEEEeeeeeee
</pre><pre>Q FD01848339_033   99 GVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        99 g~~~i~A~~v~id  111 (113)
</pre><pre>                      +.+.|.+.+-.|.
</pre><pre>T Consensus       708 ~~i~i~s~~~~i~  720 (841)
</pre><pre>T 6SK0_B          708 QNLDIQSSEGKVT  720 (841)
</pre><pre>T ss_dssp             SCEEEEESSSEEE
</pre><pre>T ss_pred             CCeEEEeCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>6SK0_C Putative type VI secretion protein; Puncture, spike, tip, carrier, inhibitor, CHAPERONE; 2.3A {Escherichia coli}
</pre><pre>Probab=27.05  E-value=2.5e+02  Score=26.27  Aligned_cols=91  Identities=22%  Similarity=0.251  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEe-CcceeEeeC
</pre><pre>Q FD01848339_033   20 GAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKA-GGNMGIEAG   98 (113)
</pre><pre>Q Consensus        20 GaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~-~g~i~l~a~   98 (113)
</pre><pre>                      .+|..-.++..++....=.+-+...++++.++++ ......+.-.++=.++....+--++.+.|++.+.+ .+++.+.+.
</pre><pre>T Consensus       629 ~~~~~i~~~~~~~~~~~a~~~~~~~~ag~~~~~~-~~~~~~~~a~~~~~~~~~~~g~~~~~~~g~v~~~a~~~~~~~~a~  707 (841)
</pre><pre>T 6SK0_C          629 HAPDGIGMVSARALRIASGSESVGIMSGDNTDIT-AGQSFTVVAEGAVSLLSRNQGMQLLAAKGRVNIQAQSDDLSMSSQ  707 (841)
</pre><pre>T ss_dssp             ECTTCEEEEESSCEEEEESSSCEEEEESSCEEEE-ESSCEEEEESSCEEEEESSSCEEEEESSSCEEEEETTSCEEEEES
</pre><pre>T ss_pred             ecCCcceeccHhhhhHhccCccEEEEeCCceEEe-ecceEEEEEhhhHHHHHHhhcceeEEccCCeEEEEcCCceeEeec
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeEEEeeeeeee
</pre><pre>Q FD01848339_033   99 GVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        99 g~~~i~A~~v~id  111 (113)
</pre><pre>                      +.+.|.+.+-.|.
</pre><pre>T Consensus       708 ~~i~i~s~~~~i~  720 (841)
</pre><pre>T 6SK0_C          708 QNLDIQSSEGKVT  720 (841)
</pre><pre>T ss_dssp             SCEEEEESSSEEE
</pre><pre>T ss_pred             CCeEEEeCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>d1bfga_ b.42.1.1 (A:) Basic FGF (FGF2) {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=26.93  E-value=2.9e+02  Score=20.60  Aligned_cols=35  Identities=14%  Similarity=0.318  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEE-ecCCceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIR-NAGGAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~R-tagGG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      ++..+++++ ...|=..|.+..++-=+-||..|++|
</pre><pre>T Consensus        30 ~~~~~~~i~~~~~g~v~i~~~~s~~ylc~n~~G~l~   65 (126)
</pre><pre>T d1bfga_          30 DPHIKLQLQAEERGVVSIKGVSANRYLAMKEDGRLL   65 (126)
</pre><pre>T ss_dssp             CGGGCEEEEECSTTEEEEEETTTTEEEEECTTSCEE
</pre><pre>T ss_pred             CCccEEEEEEcccCceEEEeeeccceEeecCCceEE
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>3JX8_A Putative lipoprotein; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesin, CELL ADHESION; HET: MSE, SO4, GOL; 2.16A {Parabacteroides distasonis ATCC 8503}
</pre><pre>Probab=26.75  E-value=5.5e+02  Score=20.06  Aligned_cols=79  Identities=16%  Similarity=0.220  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eeecCCceEEEecC-------CceeEEEcCCCCEEEecCCCCEEEEecCceEE---EecCCeEEEeC-------------
</pre><pre>Q FD01848339_033   34 IYSRNGLMEIRNAG-------GAWSVTHLGNGTTIGFNEAGELFAISQGGAFI---SAAGDMNIKAG-------------   90 (113)
</pre><pre>Q Consensus        34 V~~RNgllE~Rtag-------GG~siTh~asGs~iG~NesGq~yi~s~~~~fi---~~~G~~t~k~~-------------   90 (113)
</pre><pre>                      +..+++.|+++...       ..|.+....---+|-+..--++.+.+.+++.+   -...++.+...             
</pre><pre>T Consensus        59 ~~~~~~~L~i~~~~~~~~~~~~~~~~~~~~~~i~i~~~~l~~l~~~~~~~v~~~~~~~~~~l~l~~~~~g~i~~~~~~~~  138 (250)
</pre><pre>T 3JX8_A           59 TEVEGGTLKIYPKSIKKGFNNNSYDLRPTVYKIKSNSKELKELNTVGSGSFIISKPTKVNRMEINMAGSGNVELRGPVKG  138 (250)
</pre><pre>T ss_dssp             EEEETTEEEEEECBCTTTTTSCBCEEECSCEEEEEEESCCSEEEEESSCEEEECSCEEEEEEEEEEESSCEEEECSCEEE
</pre><pre>T ss_pred             EEEECCEEEEEeCccCCCCCCCCeeecCCeEEEEEeCCCcceEEEecceEEEEcCCcccceEEEEEeCCceEEecCCEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cceeEeeCeeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   91 GNMGIEAGGVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        91 g~i~l~a~g~~~i~A~~v~idk  112 (113)
</pre><pre>                      .++.|...+.-.|....+..++
</pre><pre>T Consensus       139 ~~l~i~~~~~g~i~~~~~~~~~  160 (250)
</pre><pre>T 3JX8_A          139 YKLECNMAGSGNIIAKDIQLDN  160 (250)
</pre><pre>T ss_dssp             EEEEEEEESSCEEEEEEEEEEE
</pre><pre>T ss_pred             cEEEEEEcCCceEEEeeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>3JX8_F Putative lipoprotein; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesin, CELL ADHESION; HET: GOL, SO4, MSE; 2.16A {Parabacteroides distasonis ATCC 8503}
</pre><pre>Probab=26.75  E-value=5.5e+02  Score=20.06  Aligned_cols=79  Identities=16%  Similarity=0.220  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eeecCCceEEEecC-------CceeEEEcCCCCEEEecCCCCEEEEecCceEE---EecCCeEEEeC-------------
</pre><pre>Q FD01848339_033   34 IYSRNGLMEIRNAG-------GAWSVTHLGNGTTIGFNEAGELFAISQGGAFI---SAAGDMNIKAG-------------   90 (113)
</pre><pre>Q Consensus        34 V~~RNgllE~Rtag-------GG~siTh~asGs~iG~NesGq~yi~s~~~~fi---~~~G~~t~k~~-------------   90 (113)
</pre><pre>                      +..+++.|+++...       ..|.+....---+|-+..--++.+.+.+++.+   -...++.+...             
</pre><pre>T Consensus        59 ~~~~~~~L~i~~~~~~~~~~~~~~~~~~~~~~i~i~~~~l~~l~~~~~~~v~~~~~~~~~~l~l~~~~~g~i~~~~~~~~  138 (250)
</pre><pre>T 3JX8_F           59 TEVEGGTLKIYPKSIKKGFNNNSYDLRPTVYKIKSNSKELKELNTVGSGSFIISKPTKVNRMEINMAGSGNVELRGPVKG  138 (250)
</pre><pre>T ss_dssp             EEEETTEEEEEECBCTTTTTSCBCEEECSCEEEEEEESCCSEEEEESSCEEEECSCEEEEEEEEEEESSCEEEECSCEEE
</pre><pre>T ss_pred             EEEECCEEEEEeCccCCCCCCCCeeecCCeEEEEEeCCCcceEEEecceEEEEcCCcccceEEEEEeCCceEEecCCEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cceeEeeCeeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   91 GNMGIEAGGVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        91 g~i~l~a~g~~~i~A~~v~idk  112 (113)
</pre><pre>                      .++.|...+.-.|....+..++
</pre><pre>T Consensus       139 ~~l~i~~~~~g~i~~~~~~~~~  160 (250)
</pre><pre>T 3JX8_F          139 YKLECNMAGSGNIIAKDIQLDN  160 (250)
</pre><pre>T ss_dssp             EEEEEEEESSCEEEEEEEEEEE
</pre><pre>T ss_pred             cEEEEEEcCCceEEEeeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>d1pg6a_ b.82.5.2 (A:) Hypothetical protein SPyM3_0169 (SPyM18_0222, SPS0176) {Streptococcus pyogenes [TaxId: 1314]}
</pre><pre>Probab=26.53  E-value=6.2e+02  Score=20.57  Aligned_cols=102  Identities=14%  Similarity=0.257  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCCCCCCCCCCCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEE-ecCCC---CEEEEecCceEEEe
</pre><pre>Q FD01848339_033    6 ASGQGTGYTPPDVDGAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIG-FNEAG---ELFAISQGGAFISA   81 (113)
</pre><pre>Q Consensus         6 asg~G~~y~p~~vdGaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG-~NesG---q~yi~s~~~~fi~~   81 (113)
</pre><pre>                      +.|.|.-+-.|...|.-..-.+.+ .++..++..+|-...   +.++...-.+-.-+ +...|   .+-+.+++.+||++
</pre><pre>T Consensus        76 ~~g~g~v~l~~~~~g~I~~i~L~~-~~~~v~~~~~lA~~~---~v~~~~~~~~~~~~~~~g~~gl~~~~l~G~G~v~l~~  151 (227)
</pre><pre>T d1pg6a_          76 SNGDGKLALAPNTPGQIVALELGE-KQYRLNDGAFLALDG---SAQYKMERQNIGKALFGGQGGLFVMTTEGLGTLLANS  151 (227)
</pre><pre>T ss_dssp             ESSSEEEEECCSSSBCEEEEEESS-CCEEEETTCEEEEET---TSEEEEEECCCC-------CCCEEEEEESSEEEEEEC
</pre><pre>T ss_pred             ecCCEEEEEeCCCCCCEEEEEeCC-ceEEEeCCeEEEEec---ceeEEEEecCcccccccCccceEEEEEeCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCC---eEEEeCcceeEeeCeeeEEEee-eeeeeC
</pre><pre>Q FD01848339_033   82 AGD---MNIKAGGNMGIEAGGVLGIRAA-EVKVDK  112 (113)
</pre><pre>Q Consensus        82 ~G~---~t~k~~g~i~l~a~g~~~i~A~-~v~idk  112 (113)
</pre><pre>                      .|+   +.++ +..+.++..--+..+.. ++.+.+
</pre><pre>T Consensus       152 ~G~i~~~~L~-ge~~~Vd~~~lva~~~~~~~~~~~  185 (227)
</pre><pre>T d1pg6a_         152 FGSIKKITLD-GGTMTIDNAHVVAWSRELDYDIHL  185 (227)
</pre><pre>T ss_dssp             SBCEEEEEEE-EEEEEEEGGGEEEEETTSEEEEEC
</pre><pre>T ss_pred             CCcEEEEEcC-CCeEEEeCCcEEEEEccceEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>PF12992.9 ; DUF3876 ; Domain of unknown function, B. Theta Gene description (DUF3876)
</pre><pre>Probab=26.53  E-value=4.5e+02  Score=18.92  Aligned_cols=65  Identities=18%  Similarity=0.107  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             CCceEEEecCCceeEEEcCCC-CEEEecCCCCEEEEecCceEEEec-CCeEE---EeCcceeEeeCeeeE
</pre><pre>Q FD01848339_033   38 NGLMEIRNAGGAWSVTHLGNG-TTIGFNEAGELFAISQGGAFISAA-GDMNI---KAGGNMGIEAGGVLG  102 (113)
</pre><pre>Q Consensus        38 NgllE~RtagGG~siTh~asG-s~iG~NesGq~yi~s~~~~fi~~~-G~~t~---k~~g~i~l~a~g~~~  102 (113)
</pre><pre>                      +-.|+++..|++|.|+..... .+-.+...=-......+.+||+.+ |.+.|   +...-+.|...|++.
</pre><pre>T Consensus        23 ~p~v~I~r~g~~y~i~~~~~~~~~~~~~~~~y~~~~~~g~l~i~~g~g~~~i~Yd~~~D~L~ls~~G~Y~   92 (93)
</pre><pre>T Q8A5A2_BACTN/4   23 NPTVIIYRNGKEYLLSIVYVSETTKQASPATYEIQQDGSQYFIATASKRLSVDYDPAKDVLSISSLGDYL   92 (93)
</pre><pre>T ss_pred             CCcEEEEECCcEEEEEEEEeCCCCceeeCeEEEEEEeCCEEEEEcCCcEEEEEeeCcCCEEEEcCCccee
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>PF15911.7 ; WD40_3 ; WD domain, G-beta repeat
</pre><pre>Probab=26.27  E-value=3.7e+02  Score=17.91  Aligned_cols=35  Identities=9%  Similarity=0.052  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             ceEEEecCCceeEEEcCCCCEEEe-cCCCCEEEEec
</pre><pre>Q FD01848339_033   40 LMEIRNAGGAWSVTHLGNGTTIGF-NEAGELFAISQ   74 (113)
</pre><pre>Q Consensus        40 llE~RtagGG~siTh~asGs~iG~-NesGq~yi~s~   74 (113)
</pre><pre>                      +.|.|...|=-+|--..+|+++-+ ++.++.|++.+
</pre><pre>T Consensus         1 v~e~~h~~~I~~i~p~~~Gtr~vfiD~~~~~~~ynp   36 (57)
</pre><pre>T J9JNN3_ACYPI/5    1 AFTYKHHFGIVNLYPNQEGIRCVIVDAKNEAYLYLP   36 (57)
</pre><pre>T ss_pred             CceEEcCceEeEEEECCCCCEEEEEeCCCCEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>TIGR01644 phage_P2_V; phage baseplate assembly protein V. This model describes a family of phage (and bacteriocin) proteins related to the phage P2 V gene product, which forms the small spike at the tip of the tail.
</pre><pre>Probab=25.90  E-value=5.6e+02  Score=19.81  Aligned_cols=47  Identities=21%  Similarity=0.308  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEEecCceEEE---ecCCeEEEeCcceeEeeCe--eeEEEeeeeeee
</pre><pre>Q FD01848339_033   65 EAGELFAISQGGAFIS---AAGDMNIKAGGNMGIEAGG--VLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        65 esGq~yi~s~~~~fi~---~~G~~t~k~~g~i~l~a~g--~~~i~A~~v~id  111 (113)
</pre><pre>                      +.++..+......+|.   .++.+++...+.+.+++..  .+.+.+..+.++
</pre><pre>T Consensus        87 ~~~~~~~~~~~g~~i~~~~~~~~l~i~~~~~~~~~~~~~~~~~~~~~~~~~~  138 (190)
</pre><pre>T TIGR01644        87 SAGDTVTYFADGARIEYDKAAHALTVCGIKTVYLDAGDGTTVTVDAPQITFT  138 (190)
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEeCCCCEEEEeeceeEEEeCCCCceEEecCCEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>6XGR_B YSD1_22 major tail protein; Bacteriophage tail, helical assembly, VIRAL PROTEIN; 3.5A {Bacteriophage sp.}
</pre><pre>Probab=25.61  E-value=6.2e+02  Score=22.34  Aligned_cols=87  Identities=14%  Similarity=0.035  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCcceeee-cCCceEEEecCCce-----------------------eEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   15 PPDVDGAPSMPSTSATKDFIYS-RNGLMEIRNAGGAW-----------------------SVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        15 p~~vdGaP~~p~~s~tkD~V~~-RNgllE~RtagGG~-----------------------siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      .+.+........+.+..||..+ +.|++++...+|..                       .-.....=.-.|.|++|+-+
</pre><pre>T Consensus       150 ~~~~~~~~g~~~~~~gtdY~vd~~~G~i~~~~~~~~i~~~~~v~v~Yt~~a~~~~~i~~~~~~~~~~l~~~g~n~~g~~~  229 (381)
</pre><pre>T 6XGR_B          150 SGDITSIVGATVVNPAGNYEIDLEAGRIYIEPDSTDLSGNVQIAVQYDVDAQKRTLVIGKSNMVYGALRMISDNPVGLNK  229 (381)
</pre><pre>T ss_dssp             SSCSTTSTTCEECCSSSSEEEETTTTEEEECSSCGGGCSSEEEEEEECBCCCEEEEEECCCCCEEEEEEEEECCSSSCCE
</pre><pre>T ss_pred             CCcceeecCceeecCCCcEEEECCCceEEEcCCCCCCCCCceEEEEEeecCCeeEEEEecCCeEEEEEEEEEeCCCCCce
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEecCceEEEecCCeEEEeCcceeEeeCeee
</pre><pre>Q FD01848339_033   71 AISQGGAFISAAGDMNIKAGGNMGIEAGGVL  101 (113)
</pre><pre>Q Consensus        71 i~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~  101 (113)
</pre><pre>                      ++---.+-+...|++.+....-..++-.+++
</pre><pre>T Consensus       230 ~~~~~~~~~~~~~~~~l~~~~~~~~~l~g~~  260 (381)
</pre><pre>T 6XGR_B          230 NYYFPKVSIAPDGDYALKGDDWQVMSFTFKA  260 (381)
</pre><pre>T ss_dssp             EEEEEEEEEEECSEEECBSSSCCCCCEEEEC
</pre><pre>T ss_pred             EEEeeceEEcCCCCcccccCCcEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>6XGR_F YSD1_22 major tail protein; Bacteriophage tail, helical assembly, VIRAL PROTEIN; 3.5A {Bacteriophage sp.}
</pre><pre>Probab=25.61  E-value=6.2e+02  Score=22.34  Aligned_cols=87  Identities=14%  Similarity=0.035  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCcceeee-cCCceEEEecCCce-----------------------eEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   15 PPDVDGAPSMPSTSATKDFIYS-RNGLMEIRNAGGAW-----------------------SVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        15 p~~vdGaP~~p~~s~tkD~V~~-RNgllE~RtagGG~-----------------------siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      .+.+........+.+..||..+ +.|++++...+|..                       .-.....=.-.|.|++|+-+
</pre><pre>T Consensus       150 ~~~~~~~~g~~~~~~gtdY~vd~~~G~i~~~~~~~~i~~~~~v~v~Yt~~a~~~~~i~~~~~~~~~~l~~~g~n~~g~~~  229 (381)
</pre><pre>T 6XGR_F          150 SGDITSIVGATVVNPAGNYEIDLEAGRIYIEPDSTDLSGNVQIAVQYDVDAQKRTLVIGKSNMVYGALRMISDNPVGLNK  229 (381)
</pre><pre>T ss_dssp             SSCSTTSTTCEECCSSSSEEEETTTTEEEECSSCGGGCSSEEEEEEECBCCCEEEEEECCCCCEEEEEEEEECCSSSCCE
</pre><pre>T ss_pred             CCcceeecCceeecCCCcEEEECCCceEEEcCCCCCCCCCceEEEEEeecCCeeEEEEecCCeEEEEEEEEEeCCCCCce
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEecCceEEEecCCeEEEeCcceeEeeCeee
</pre><pre>Q FD01848339_033   71 AISQGGAFISAAGDMNIKAGGNMGIEAGGVL  101 (113)
</pre><pre>Q Consensus        71 i~s~~~~fi~~~G~~t~k~~g~i~l~a~g~~  101 (113)
</pre><pre>                      ++---.+-+...|++.+....-..++-.+++
</pre><pre>T Consensus       230 ~~~~~~~~~~~~~~~~l~~~~~~~~~l~g~~  260 (381)
</pre><pre>T 6XGR_F          230 NYYFPKVSIAPDGDYALKGDDWQVMSFTFKA  260 (381)
</pre><pre>T ss_dssp             EEEEEEEEEEECSEEECBSSSCCCCCEEEEC
</pre><pre>T ss_pred             EEEeeceEEcCCCCcccccCCcEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>d1rg8a1 b.42.1.1 (A:1-137) Acidic FGF (FGF1) {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=25.55  E-value=4e+02  Score=20.42  Aligned_cols=35  Identities=11%  Similarity=0.307  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecC-CceeEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAG-GAWSVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~Rtag-GG~siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +.+-+|++++.. |-..|-+..++.-+-||..|+||
</pre><pre>T Consensus        39 ~~~~~l~i~~~~~g~v~i~~~~s~~YLcmn~~G~l~   74 (137)
</pre><pre>T d1rg8a1          39 DQHIQLQLSAESVGEVYIKSTETGQYLAMDTDGLLY   74 (137)
</pre><pre>T ss_dssp             CTTCCEEEEEEETTEEEEEETTTCCEEEECTTSCEE
</pre><pre>T ss_pred             CCceEEEEEecccCceEEEeccCccEEEeCCCCcee
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>d1qqka_ b.42.1.1 (A:) Keratinocyte growth factor, FGF7 {Norway rat (Rattus norvegicus) [TaxId: 10116]}
</pre><pre>Probab=25.35  E-value=3.9e+02  Score=20.20  Aligned_cols=35  Identities=20%  Similarity=0.387  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCcee-EEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGAWS-VTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG~s-iTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      ++..+|++++.+-|.- |-+..++.=+-||..|+||
</pre><pre>T Consensus        31 ~~~~~l~i~s~~~g~v~i~~~~s~~YLc~n~~G~l~   66 (132)
</pre><pre>T d1qqka_          31 NSYNIMEIRTVAVGIVAIKGVESEYYLAMNKEGKLY   66 (132)
</pre><pre>T ss_dssp             CSTTEEEEEECSSSEEEEEETTTCCCCEECTTCCEE
</pre><pre>T ss_pred             CCcceeEEEEecCCeEEEEecccccEEeecCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>3V9F_C Two-component system sensor histidine kinase/response regulator, hybrid (One-component system); beta-propeller, beta-sandwich, TRANSCRIPTION; HET: MSE; 3.3A {Bacteroides thetaiotaomicron}
</pre><pre>Probab=24.80  E-value=7.5e+02  Score=20.92  Aligned_cols=47  Identities=11%  Similarity=0.137  Sum_probs=0.0  Template_Neff=13.400
</pre><pre>
</pre><pre>Q ss_pred             eeeecCCceEEEecCCceeEEEcCCCC--------EEEecCCCCEEEEecCceEE
</pre><pre>Q FD01848339_033   33 FIYSRNGLMEIRNAGGAWSVTHLGNGT--------TIGFNEAGELFAISQGGAFI   79 (113)
</pre><pre>Q Consensus        33 ~V~~RNgllE~RtagGG~siTh~asGs--------~iG~NesGq~yi~s~~~~fi   79 (113)
</pre><pre>                      ++...+|+......++.|.......+.        .+-...+|.+|+-+...++.
</pre><pre>T Consensus       597 ~~~t~~g~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~g~l~~~t~~Gl~~  651 (781)
</pre><pre>T 3V9F_C          597 WASTNTGISCYITSKKCFYTYDHSNNIPQGSFISGCVTKDHNGLIYFGSINGLCF  651 (781)
</pre><pre>T ss_dssp             EEECSSSEEEEETTTTEEEEECGGGTCCSSCEEEEEEEECTTCCEEEEETTEEEE
</pre><pre>T ss_pred             EEEeCCceEEEECCCCEEEEeccccCCCCCceeeeeEEECCCCcEEEEecCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>4Q35_A LPS-assembly protein LptD; complex, 26 beita-sheet, membrane protein, LPS biogenesis; HET: C8E, SO4, LDA; 2.393A {Shigella flexneri}
</pre><pre>Probab=24.19  E-value=1.1e+03  Score=22.67  Aligned_cols=84  Identities=11%  Similarity=0.074  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCC-------
</pre><pre>Q FD01848339_033   12 GYTPPDVDGAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGD-------   84 (113)
</pre><pre>Q Consensus        12 ~y~p~~vdGaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~-------   84 (113)
</pre><pre>                      .|.++...+.+...++.-+.|.+.....-. + .+-|...|+...             ..+.+..+.++...+       
</pre><pre>T Consensus        37 ~~~~~~~~~~~~~~~~~i~AD~~~~~~~~~-~-~~~GnV~i~~~~-------------~~l~Ad~i~~~~~~~~~~~g~~  101 (802)
</pre><pre>T 4Q35_A           37 SYDRPLVQGDTNDLPVTINADHAKGDYPDD-A-VFTGSVDIMQGN-------------SRLQADEVQLHQKEAPGQPEPV  101 (802)
</pre><pre>T ss_dssp             CCCSCCCCSCSSSCCCEEECSEEESCTTTE-E-EEESSEEEEETT-------------EEEEESCEEEECCCSCSCC--C
</pre><pre>T ss_pred             CCCCCccCCCCCCCcEEEEEeeeEecCCCe-E-EEEecEEEEeCC-------------EEEEEeEEEEECCCCCCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   85 MNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        85 ~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      -.+...|++.++. +.+.++|.++.+|
</pre><pre>T Consensus       102 ~~i~a~G~V~~~~-~~~~~~~~~~~~~  127 (802)
</pre><pre>T 4Q35_A          102 RTVDALGNVHYDD-NQVILKGPKGWAN  127 (802)
</pre><pre>T ss_dssp             CEEEECSSEEEEC-SSEEEEESEEEEE
</pre><pre>T ss_pred             cEEEEeCCEEEeC-CCeEEEeCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>4RU3_A puncturing protein gp41; gp41, membrane translocation, phage baseplate, Cell puncturing device, CELL INVASION; 1.172A {Pseudomonas phage SN}
</pre><pre>Probab=24.10  E-value=6.7e+02  Score=20.07  Aligned_cols=56  Identities=18%  Similarity=0.224  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             ceeEEEcCCCCEEEecCCCCEEEEecCc-eEEEecCCeEEEeCcceeEeeCeeeEEEeeeeeee
</pre><pre>Q FD01848339_033   49 AWSVTHLGNGTTIGFNEAGELFAISQGG-AFISAAGDMNIKAGGNMGIEAGGVLGIRAAEVKVD  111 (113)
</pre><pre>Q Consensus        49 G~siTh~asGs~iG~NesGq~yi~s~~~-~fi~~~G~~t~k~~g~i~l~a~g~~~i~A~~v~id  111 (113)
</pre><pre>                      ++.+...-+...+--.+.++++|..... ..|....       ++|.+++.+.+.|.|..+.|+
</pre><pre>T Consensus       128 ~~~~~~~~~~~~~~~~~~~~~~i~~~~G~~~I~i~~-------~~i~i~~~~~v~i~~~~i~i~  184 (221)
</pre><pre>T 4RU3_A          128 GLFIPDVFRRYTINSEDSAAMVIQSTSGATRISIRG-------DNIKITAPSNVTVDTPQANFT  184 (221)
</pre><pre>T ss_dssp             EEEEECCTTCCCCCGGGTTSEEEECTTSSSEEEECS-------SCEEEECSSCCCCCSSCCCCS
</pre><pre>T ss_pred             eEEEeeecCCCccCCCCcCceEEEcCCCCeEEEEEC-------CeEEEeCCCcEEEcCCceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>1Q7F_A brain tumor CG10719-PA; brat, NHL domain, NHL repeat, beta-propeller, TRANSLATION; 1.95A {Drosophila melanogaster} SCOP: b.68.9.1
</pre><pre>Probab=23.80  E-value=5e+02  Score=18.52  Aligned_cols=54  Identities=7%  Similarity=0.044  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             CCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEe
</pre><pre>Q FD01848339_033   28 SATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISA   81 (113)
</pre><pre>Q Consensus        28 s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~   81 (113)
</pre><pre>                      ++..-.+.+.++.+...-...|..-.+...-..|.++.+|++|+....+-.|..
</pre><pre>T Consensus         2 ~~~~~~~~~~~~~~~~~~~~~g~~~~~~~~~~~i~~~~~g~~~v~~~~~~~i~~   55 (286)
</pre><pre>T 1Q7F_A            2 THMKSQIKRQRMIYHCKFGEFGVMEGQFTEPSGVAVNAQNDIIVADTNNHRIQI   55 (286)
</pre><pre>T ss_dssp             ------CCCEEECEEEEECCBSSSTTCBSCEEEEEECTTCCEEEEEGGGTEEEE
</pre><pre>T ss_pred             cchhhheeecCcEEEEeeCccceeeccccCceEEEECCCCcEEEEECCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>1QQL_A FIBROBLAST GROWTH FACTOR 7/1 CHIMERA; BETA-TREFOIL, HORMONE-GROWTH FACTOR COMPLEX; 2.3A {Rattus norvegicus} SCOP: b.42.1.1
</pre><pre>Probab=23.58  E-value=2.2e+02  Score=20.88  Aligned_cols=35  Identities=20%  Similarity=0.407  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCce-eEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGAW-SVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG~-siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +.+.+|++++.+-+. .|....++.-+.||..|++|
</pre><pre>T Consensus        39 ~~~~~l~~~~~~~~~v~i~~~~s~~yL~~~~~G~l~   74 (140)
</pre><pre>T 1QQL_A           39 NSYNIMEIRTVAVGIVAIKGVESEYYLAMNKEGKLY   74 (140)
</pre><pre>T ss_dssp             CSTTCEEEEEEETTEEEEEETTTCCCCEECTTSCEE
</pre><pre>T ss_pred             CCCceeEEEEEcceeEEEeccchhHheeeecCCeEe
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>PF06715.14 ; Gp5_C ; Gp5 C-terminal repeat (3 copies)
</pre><pre>Probab=23.18  E-value=3e+02  Score=15.74  Aligned_cols=24  Identities=13%  Similarity=0.155  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCeEEEeCcceeEeeCeeeEEEee
</pre><pre>Q FD01848339_033   83 GDMNIKAGGNMGIEAGGVLGIRAA  106 (113)
</pre><pre>Q Consensus        83 G~~t~k~~g~i~l~a~g~~~i~A~  106 (113)
</pre><pre>                      |+.+....|+..+..+|+.+++..
</pre><pre>T Consensus         1 gn~t~~V~gn~t~~V~gn~t~~V~   24 (24)
</pre><pre>T A0A220NTH3_9CA    1 ANLTANVKGNAELMIEGNTTETIK   24 (24)
</pre><pre>T ss_pred             CCeeEEeecCEEEEecCcEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>1RWR_A Filamentous hemagglutinin; Type V secretion, beta-helix, adhesins, filamentous hemagglutinin, TPS domain, CELL ADHESION; 1.72A {Bordetella pertussis} SCOP: b.80.1.11
</pre><pre>Probab=22.57  E-value=5.9e+02  Score=21.14  Aligned_cols=59  Identities=19%  Similarity=0.417  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             eEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEe----CcceeEeeCeeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   51 SVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKA----GGNMGIEAGGVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        51 siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~----~g~i~l~a~g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      .|.-..+..-++++-.|.+.  +++++.|++.|++....    .++|.|++++  .+.++.+.-..+
</pre><pre>T Consensus       229 ~i~l~~~~~g~~v~~~G~l~--~~~~~~i~~~G~l~~~~~~~~~~~i~~~~~~--~~~~g~i~~~~~  291 (301)
</pre><pre>T 1RWR_A          229 HITLVSSDSGLGVRQLGSLS--SPSAITVSSQGEIALGDATVQRGPLSLKGAG--VVSAGKLASGGG  291 (301)
</pre><pre>T ss_dssp             EEEEEECSTTCEEEECSCEE--ESSEEEEEESSCEEEEEEEESSSEEEEEESS--CEEEEEEEETTC
</pre><pre>T ss_pred             EEEEEECCCCCceEecceEe--cCCcEEEcCCCeEEECCceeecCCEEEEecc--eEecceEEecCc
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>6NU2_AW 28S ribosomal protein S28, mitochondrial; mitochondrial ribosome recycling Factor, mtRRF, 55S, RIBOSOME; HET: MG; 3.9A {Homo sapiens}
</pre><pre>Probab=22.32  E-value=1.8e+02  Score=22.60  Aligned_cols=19  Identities=26%  Similarity=0.641  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             CCEEEEecCceEEEecCCe
</pre><pre>Q FD01848339_033   67 GELFAISQGGAFISAAGDM   85 (113)
</pre><pre>Q Consensus        67 Gq~yi~s~~~~fi~~~G~~   85 (113)
</pre><pre>                      |++|.+-..++||+-||+.
</pre><pre>T Consensus        26 G~I~hvv~dDLYIDfGgKF   44 (97)
</pre><pre>T 6NU2_AW          26 GRIFHIVENDLYIDFGGKF   44 (97)
</pre><pre>T ss_dssp             EEEEEECSSEEEEECSSSS
</pre><pre>T ss_pred             EeEEEeccCceEEecCCCe
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>6RIQ_A MinC; Bacterial cell division, PROTEIN FIBRIL; HET: MG, ATP; 3.1A {Pseudomonas aeruginosa}
</pre><pre>Probab=22.17  E-value=7.6e+02  Score=20.01  Aligned_cols=65  Identities=22%  Similarity=0.308  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEe--cCceEEEecCCeEE
</pre><pre>Q FD01848339_033   14 TPPDVDGAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAIS--QGGAFISAAGDMNI   87 (113)
</pre><pre>Q Consensus        14 ~p~~vdGaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s--~~~~fi~~~G~~t~   87 (113)
</pre><pre>                      ..|.+.-.++.|..++++.--...+..+-++        .+.-+|.+|-.+. +.+.|++  ...+.|.+.|++.|
</pre><pre>T Consensus        12 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ii~--------~~vRSGq~i~~~g-~dlvIlGdV~~GA~V~A~GnI~V   78 (144)
</pre><pre>T 6RIQ_A           12 PSPSVGEARPEPAKAEEKPAEPVSRPTKVVK--------TPVRGGMQIYAAG-GDLIVLAAVSPGAELLADGNIHV   78 (144)
</pre><pre>T ss_dssp             -------------------------CCEEEC--------SCBCTTCEEEESS-SCEEECSCBCTTCEEEESSCEEE
</pre><pre>T ss_pred             CCCCccccCCCCccCCCCCCCCCCCCeEEEc--------cccCCCCEEEECC-CcEEEEcCCCCCCEEEeCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>6RIQ_B MinC; Bacterial cell division, PROTEIN FIBRIL; HET: ATP, MG; 3.1A {Pseudomonas aeruginosa}
</pre><pre>Probab=22.17  E-value=7.6e+02  Score=20.01  Aligned_cols=65  Identities=22%  Similarity=0.308  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCcceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEe--cCceEEEecCCeEE
</pre><pre>Q FD01848339_033   14 TPPDVDGAPSMPSTSATKDFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAIS--QGGAFISAAGDMNI   87 (113)
</pre><pre>Q Consensus        14 ~p~~vdGaP~~p~~s~tkD~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s--~~~~fi~~~G~~t~   87 (113)
</pre><pre>                      ..|.+.-.++.|..++++.--...+..+-++        .+.-+|.+|-.+. +.+.|++  ...+.|.+.|++.|
</pre><pre>T Consensus        12 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ii~--------~~vRSGq~i~~~g-~dlvIlGdV~~GA~V~A~GnI~V   78 (144)
</pre><pre>T 6RIQ_B           12 PSPSVGEARPEPAKAEEKPAEPVSRPTKVVK--------TPVRGGMQIYAAG-GDLIVLAAVSPGAELLADGNIHV   78 (144)
</pre><pre>T ss_dssp             -------------------------CCEEEC--------SCBCTTCEEEESS-SCEEECSCBCTTCEEEESSCEEE
</pre><pre>T ss_pred             CCCCccccCCCCccCCCCCCCCCCCCeEEEc--------cccCCCCEEEECC-CcEEEEcCCCCCCEEEeCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>5V1V_B TbiB1; Lasso Peptide, RiPP, Peptide Binding, PROTEIN BINDING; 1.35A {Thermobaculum terrenum (strain ATCC BAA-798 / YNP1)}
</pre><pre>Probab=21.52  E-value=3.8e+02  Score=16.35  Aligned_cols=34  Identities=18%  Similarity=0.262  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             eeecCCceEEEecCCceeEEEcCCCCEEEecCCC
</pre><pre>Q FD01848339_033   34 IYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAG   67 (113)
</pre><pre>Q Consensus        34 V~~RNgllE~RtagGG~siTh~asGs~iG~NesG   67 (113)
</pre><pre>                      .++++.-+..+..+|++.+.+..++..+-+|+.+
</pre><pre>T Consensus         3 ~~~~~~~~~~~~~~~~~~l~~~~~~~~~~l~~~~   36 (91)
</pre><pre>T 5V1V_B            3 AMKISDAVVSAHIDDEVVLLHLQTGTYFGLDAVG   36 (91)
</pre><pre>T ss_dssp             -CEECTTEEEEEETTEEEEEETTTTEEEEECHHH
</pre><pre>T ss_pred             ceeecCCeEEEEeCCeEEEEECCCCcEEeecHHH
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>6ITX_C Fiber protein; avian adenovirus, fiber protein, CELL ADHESION; HET: SO4, GLN; 2.75A {Duck adenovirus 1}
</pre><pre>Probab=21.34  E-value=8.8e+02  Score=22.60  Aligned_cols=62  Identities=16%  Similarity=0.295  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             CCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeC-eeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   47 GGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAG-GVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        47 gGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~-g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      .||...   ..+. |+.+.++-|-|.. ..+=|...|-+++.+.+.+.|+-. ..|.|.++++.+-++
</pre><pre>T Consensus        25 ~Gg~q~---~~~g-l~v~~~~~l~~s~-~~l~v~~~gPl~~t~~~GLtL~yD~~~F~V~~G~L~l~~~   87 (301)
</pre><pre>T 6ITX_C           25 TGGQQM---GRGS-LNVETRGGLEKSD-TGLKIKRAAPLSITSDGELTLAYDSTDFQVTENGLALKVS   87 (301)
</pre><pre>T ss_dssp             -----------------------------------------------------------------CCC
</pre><pre>T ss_pred             cccccc---cccc-eeeccCCceeEeC-CceEEecCCCEEEcCCCCeEeEeCCCceEEECCEEEeecC
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>6ITX_F Fiber protein; avian adenovirus, fiber protein, CELL ADHESION; HET: GLN, SO4; 2.75A {Duck adenovirus 1}
</pre><pre>Probab=21.34  E-value=8.8e+02  Score=22.60  Aligned_cols=62  Identities=16%  Similarity=0.295  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             CCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeCcceeEeeC-eeeEEEeeeeeeeCC
</pre><pre>Q FD01848339_033   47 GGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAGGNMGIEAG-GVLGIRAAEVKVDKG  113 (113)
</pre><pre>Q Consensus        47 gGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~g~i~l~a~-g~~~i~A~~v~idkg  113 (113)
</pre><pre>                      .||...   ..+. |+.+.++-|-|.. ..+=|...|-+++.+.+.+.|+-. ..|.|.++++.+-++
</pre><pre>T Consensus        25 ~Gg~q~---~~~g-l~v~~~~~l~~s~-~~l~v~~~gPl~~t~~~GLtL~yD~~~F~V~~G~L~l~~~   87 (301)
</pre><pre>T 6ITX_F           25 TGGQQM---GRGS-LNVETRGGLEKSD-TGLKIKRAAPLSITSDGELTLAYDSTDFQVTENGLALKVS   87 (301)
</pre><pre>T ss_dssp             -------------------------------------------------------------CCCCTTC
</pre><pre>T ss_pred             cccccc---cccc-eeeccCCceeEeC-CceEEecCCCEEEcCCCCeEeEeCCCceEEECCEEEeecC
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>2DXU_A biotin--[acetyl-CoA-carboxylase] ligase; Biotin Biosynthesis, Dimer, Structural Genomics, NPPSFA, National Project on Protein Structural and Functional Analyses, RIKEN Structural; HET: BT5; 1.28A {Pyrococcus horikoshii} SCOP: b.34.1.1, d.104.1.2
</pre><pre>Probab=21.25  E-value=4.5e+02  Score=21.03  Aligned_cols=33  Identities=18%  Similarity=0.301  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEecCCCCEEEEec-CceEEEecCCeEEEe
</pre><pre>Q FD01848339_033   57 NGTTIGFNEAGELFAISQ-GGAFISAAGDMNIKA   89 (113)
</pre><pre>Q Consensus        57 sGs~iG~NesGq~yi~s~-~~~fi~~~G~~t~k~   89 (113)
</pre><pre>                      .|.-.|+|++|.+.+... +....-..|++.+..
</pre><pre>T Consensus       201 ~g~~~gi~~~G~L~l~~~~g~~~~~~~g~~~~~~  234 (235)
</pre><pre>T 2DXU_A          201 EGIAEDIDDFGRLIIRLDSGEVKKVIYGDVSLRF  234 (235)
</pre><pre>T ss_dssp             EEEEEEECTTSCEEEECTTSCEEEECCSSCCCEE
</pre><pre>T ss_pred             EEEEEEeCCCCCEEEEeCCCCEEEEEcCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>6JF2_A Flagellar basal-body rod protein FlgG; Bacterial flagellum, distal rod, STRUCTURAL PROTEIN; HET: MSE; 2.0A {Salmonella typhimurium (strain LT2 / SGSC1412 / ATCC 700720)}
</pre><pre>Probab=21.10  E-value=7.7e+02  Score=19.71  Aligned_cols=66  Identities=18%  Similarity=0.277  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             ceeeecCCceEEEecCCceeEEEcCCCCEEEecCCCCEEEEecCceEEEecCCeEEEeC-cceeEeeCeeeEEE
</pre><pre>Q FD01848339_033   32 DFIYSRNGLMEIRNAGGAWSVTHLGNGTTIGFNEAGELFAISQGGAFISAAGDMNIKAG-GNMGIEAGGVLGIR  104 (113)
</pre><pre>Q Consensus        32 D~V~~RNgllE~RtagGG~siTh~asGs~iG~NesGq~yi~s~~~~fi~~~G~~t~k~~-g~i~l~a~g~~~i~  104 (113)
</pre><pre>                      |+-+.-++++.++...|....|...   ++.+++.|++......-++    +.+.+-.. .++.++..|.+.+.
</pre><pre>T Consensus        52 d~ai~g~gff~v~~~~g~~~ytR~G---~f~~~~~g~l~~~~G~~v~----~~i~i~~~~~~~~i~~~G~v~~~  118 (185)
</pre><pre>T 6JF2_A           52 DVAIKGQGFFQVMLPDGTSAYTRDG---SFQVDQNGQLVTAGGFQVQ----PAITIPANALSITIGRDGVVSVT  118 (185)
</pre><pre>T ss_dssp             BEEEESBCBEEEECTTSCEEEESCC---BCEECTTSBEECTTSCEES----SCCBCCTTEEEEEECTTCEEEEE
</pre><pre>T ss_pred             cEEEcCCCeEEEECCCCCEEEEcCe---EEEECCCCcEEeCCCcEec----CCcccCCCcceeEeCCCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>4OPW_A Uncharacterized protein; PF10988 family protein, DUF2807, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY; HET: EDO, MSE; 1.75A {Parabacteroides merdae ATCC 43184}
</pre><pre>Probab=21.07  E-value=7.4e+02  Score=19.44  Aligned_cols=79  Identities=15%  Similarity=0.202  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             eeecCCceEEEecCCceeE-------EEcCCCCEEEecCCCCEEE---EecCceEEEecCCeEEEeC-----cceeEeeC
</pre><pre>Q FD01848339_033   34 IYSRNGLMEIRNAGGAWSV-------THLGNGTTIGFNEAGELFA---ISQGGAFISAAGDMNIKAG-----GNMGIEAG   98 (113)
</pre><pre>Q Consensus        34 V~~RNgllE~RtagGG~si-------Th~asGs~iG~NesGq~yi---~s~~~~fi~~~G~~t~k~~-----g~i~l~a~   98 (113)
</pre><pre>                      +..+++-|.++.....|.-       -+...=..|-++.+|++.+   +....+.|...|.-.++..     .++.++..
</pre><pre>T Consensus        57 ~~~~~~~L~I~~~~~~~~~~~~~~v~i~~p~l~~i~~~~~~~i~~~~~~~~~~l~i~~~g~g~i~~~~~~~~~~l~i~~~  136 (241)
</pre><pre>T 4OPW_A           57 IDIQDRVLTVGFKGAKVDHFTKFIVKTNSKWLKEVKASGNANFIANSPLKGDELKINANSNCLVQLKQKVEVGKLDLNVS  136 (241)
</pre><pre>T ss_dssp             EEEETTEEEEEECSSEEEECSCCEEEEEESCCCEEEEESSCEEEECSCEEEEEEEEEECTTCEEEECSCEEEEEEEEEEE
</pre><pre>T ss_pred             EEEECCEEEEEeCCeeecCcccEEEEEecCCcceeEEecceEEEEcCCcccCeEEEEEeCCceEEECCeEEEcEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeEEEeeeeeeeC
</pre><pre>Q FD01848339_033   99 GVLGIRAAEVKVDK  112 (113)
</pre><pre>Q Consensus        99 g~~~i~A~~v~idk  112 (113)
</pre><pre>                      |.-.|....+..++
</pre><pre>T Consensus       137 ~~g~i~~~~~~~~~  150 (241)
</pre><pre>T 4OPW_A          137 GSANMVVNELKTDK  150 (241)
</pre><pre>T ss_dssp             TTEEEEESCEECSE
</pre><pre>T ss_pred             cCceEEEeeeEece
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>1IHK_A GLIA-ACTIVATING FACTOR; b-trefoil fold, HORMONE-GROWTH FACTOR COMPLEX; HET: PO4; 2.2A {Homo sapiens} SCOP: b.42.1.1
</pre><pre>Probab=20.62  E-value=2.9e+02  Score=21.77  Aligned_cols=35  Identities=34%  Similarity=0.596  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             ecCCceEEEecCCce-eEEEcCCCCEEEecCCCCEE
</pre><pre>Q FD01848339_033   36 SRNGLMEIRNAGGAW-SVTHLGNGTTIGFNEAGELF   70 (113)
</pre><pre>Q Consensus        36 ~RNgllE~RtagGG~-siTh~asGs~iG~NesGq~y   70 (113)
</pre><pre>                      +.+.+|++.+.+-|. .|-...++.-+.||..|.+|
</pre><pre>T Consensus        56 ~~~~~l~i~s~~~~~v~I~g~~s~~yLcmn~~G~l~   91 (174)
</pre><pre>T 1IHK_A           56 SRFGILEFISIAVGLVSIRGVDSGLYLGMNEKGELY   91 (174)
</pre><pre>T ss_dssp             CGGGCEEEEEEETTEEEEEETTTCCEEEECTTCCEE
</pre><pre>T ss_pred             CCCeEEEEEEEecCcEEEccccceeeeeeccCceEe
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>PF10246.11 ; MRP-S35 ; Mitochondrial ribosomal protein MRP-S35
</pre><pre>Probab=20.38  E-value=1.9e+02  Score=22.74  Aligned_cols=19  Identities=16%  Similarity=0.244  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             CCEEEEecCceEEEecCCe
</pre><pre>Q FD01848339_033   67 GELFAISQGGAFISAAGDM   85 (113)
</pre><pre>Q Consensus        67 Gq~yi~s~~~~fi~~~G~~   85 (113)
</pre><pre>                      |+++.+-..++||+-||..
</pre><pre>T Consensus        29 G~I~~vv~dDLYIDfGgKF   47 (105)
</pre><pre>T A8X0G7_CAEBR/3   29 GKIVNRIQEDVYIDFGLKF   47 (105)
</pre><pre>T ss_pred             EEEEEecCCCeEEeCCCce
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>PF03775.18 ; MinC_C ; Septum formation inhibitor MinC, C-terminal domain
</pre><pre>Probab=20.34  E-value=6.7e+02  Score=18.70  Aligned_cols=54  Identities=22%  Similarity=0.303  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             EcCCCCEEEecCCCCEEEEe--cCceEEEecCCeEEEeCcceeEeeCeeeE----EEeeeee
</pre><pre>Q FD01848339_033   54 HLGNGTTIGFNEAGELFAIS--QGGAFISAAGDMNIKAGGNMGIEAGGVLG----IRAAEVK  109 (113)
</pre><pre>Q Consensus        54 h~asGs~iG~NesGq~yi~s--~~~~fi~~~G~~t~k~~g~i~l~a~g~~~----i~A~~v~  109 (113)
</pre><pre>                      +..+|..+-..  |.+.|.+  ...+.|.++|++.|...=.=.+.|+....    |.|..+.
</pre><pre>T Consensus         5 ~vrsG~~v~~~--gdi~i~G~V~~ga~V~a~Gnv~V~g~v~G~i~AG~~g~~~~~I~a~~~~   64 (101)
</pre><pre>T D9S283_THEOJ/9    5 TLRSGQRIAYS--GNVVVVGDVNPGSEIIAKGDILVFGALRGMAHAGAEGDENSIVVAFRLQ   64 (101)
</pre><pre>T ss_pred             ccCCCCEEEEC--CCEEEECCcCCCCEEEECCCEEEeceeCeEEEecCCCCCCcEEEEEEeC
</pre><pre>
</pre><pre>
</pre>