<pre>Query         MBG6247344.1/91-180 
</pre><pre>Match_columns 90
</pre><pre>No_of_seqs    106 out of 179
</pre><pre>Neff          3.75549
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Tue Aug 24 15:28:26 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/1508968.hhr -oa3m ../results/1508968.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 TIGR01644 phage_P2_V; phage ba  99.2 1.4E-09 1.3E-14   77.1  10.2   83    4-87     90-190 (190)
</pre><pre>  2 4S36_A Phage baseplate protein  99.1 1.6E-09 1.5E-14   68.6   8.7   34    4-37      2-36  (90)
</pre><pre>  3 3QR7_B Baseplate assembly prot  99.1 9.2E-09 8.6E-14   66.6  10.7   80    4-83      3-110 (115)
</pre><pre>  4 3AQJ_A Baseplate assembly prot  99.1 1.2E-08 1.2E-13   69.2  11.1   88    1-90     19-134 (134)
</pre><pre>  5 3QR8_A Baseplate assembly prot  99.0 1.4E-08 1.3E-13   70.9  10.4   85    3-89     98-210 (211)
</pre><pre>  6 3PQI_A gene product 138; Beta-  98.9 2.8E-08 2.6E-13   72.9   8.4   89    1-90    138-243 (247)
</pre><pre>  7 3PQH_A gene product 138; Beta-  98.9   4E-08 3.7E-13   66.9   8.5   89    1-90     18-123 (127)
</pre><pre>  8 3VTN_A Protein gp45; beta-heli  98.8 1.3E-07 1.2E-12   59.7   8.5   80    9-90      3-101 (101)
</pre><pre>  9 3VTO_B Protein gp45; beta-heli  98.8   2E-07 1.9E-12   62.3   8.6   79    9-90      9-106 (115)
</pre><pre> 10 3VTO_C Protein gp45; beta-heli  98.8   2E-07 1.9E-12   62.3   8.6   79    9-90      9-106 (115)
</pre><pre> 11 4RU3_A puncturing protein gp41  98.7 2.1E-07 1.9E-12   68.1   8.7   74    1-89    144-219 (221)
</pre><pre> 12 4S37_B Phage baseplate protein  98.5 4.3E-06   4E-11   57.3   8.4   76    1-76     94-185 (185)
</pre><pre> 13 4S37_O Phage baseplate protein  98.5 4.3E-06   4E-11   57.3   8.4   76    1-76     94-185 (185)
</pre><pre> 14 PF18715.3 ; Phage_spike ; Phag  98.0 6.3E-05 5.8E-10   43.5   5.2   40   28-67      1-53  (53)
</pre><pre> 15 6P2A_C CHIMERA OF BACTERIOPHAG  97.2   0.008 7.4E-08   43.6   8.0   65   24-90     57-147 (148)
</pre><pre> 16 6P2A_E CHIMERA OF BACTERIOPHAG  97.2   0.008 7.4E-08   43.6   8.0   65   24-90     57-147 (148)
</pre><pre> 17 PF18946.2 ; Apex ; GpV Apex mo  90.3    0.57 5.2E-06   21.1   1.6   22   68-89      1-22  (23)
</pre><pre> 18 6J0M_C Pvc8; assembly, Photorh  84.2      17 0.00015   29.6   7.3   66    3-68    455-538 (538)
</pre><pre> 19 PF14913.8 ; DPCD ; DPCD protei  80.6       5 4.7E-05   32.4   3.3   25    2-26     18-44  (190)
</pre><pre> 20 6RBK_C Afp8; Anti-feeding prop  77.1      46 0.00042   27.3   7.6   66    3-68    458-529 (529)
</pre><pre> 21 PF20000.1 ; fvmX2 ; FtsH terna  67.0      26 0.00024   29.8   4.3   31   11-41     77-107 (260)
</pre><pre> 22 PF04773.15 ; FecR ; FecR prote  58.2      94 0.00087   20.5   7.2   62    4-66     10-96  (104)
</pre><pre> 23 5CL2_A Sporulation-control pro  56.3      64 0.00059   25.0   4.5   37   36-72     10-48  (252)
</pre><pre> 24 PF07202.15 ; Tcp10_C ; T-compl  55.3      47 0.00044   18.5   2.8   21    6-26      6-26  (35)
</pre><pre> 25 d2gpia1 d.354.1.1 (A:1-90) Hyp  54.4      38 0.00035   22.9   2.8   21    7-29      5-25  (90)
</pre><pre> 26 PF20003.1 ; fvmX5 ; FtsH terna  52.9      98 0.00091   24.9   5.1   39    5-43     58-96  (188)
</pre><pre> 27 d2psba1 c.153.1.1 (A:13-308) U  52.8 2.4E+02  0.0022   23.5   7.7   70    3-72    170-255 (296)
</pre><pre> 28 PF14481.8 ; Fimbrial_PilY2 ; T  50.6   1E+02 0.00095   20.4   4.2   61   12-73     29-91  (99)
</pre><pre> 29 5LHX_A Serine/threonine-protei  50.0      69 0.00064   22.2   3.5   25    1-25     24-50  (93)
</pre><pre> 30 6M9Y_B Fluorescent protein lan  47.8      61 0.00057   25.8   3.3   47    8-71     19-71  (172)
</pre><pre> 31 d1mbya_ d.223.1.1 (A:) Serine/  45.7      99 0.00091   19.2   3.4   25    1-25     14-38  (75)
</pre><pre> 32 1MBY_A serine/threonine kinase  44.5      84 0.00078   20.7   3.2   22    4-25     24-45  (88)
</pre><pre> 33 PF07273.14 ; DUF1439 ; Protein  42.0 1.1E+02  0.0011   22.2   3.8   23   13-35     75-97  (154)
</pre><pre> 34 PF00659.20 ; POLO_box ; POLO b  40.7 1.2E+02  0.0011   17.9   3.2   22    4-25     13-34  (65)
</pre><pre> 35 PF12134.10 ; PRP8_domainIV ; P  40.6      59 0.00055   27.0   2.4   35    4-39     30-72  (230)
</pre><pre> 36 PF10989.10 ; DUF2808 ; Protein  40.6 2.3E+02  0.0021   20.9   5.1   51    1-54     40-102 (142)
</pre><pre> 37 PF13619.8 ; KTSC ; KTSC domain  40.4      78 0.00072   19.0   2.3   15   13-27      6-20  (58)
</pre><pre> 38 PF12992.9 ; DUF3876 ; Domain o  38.6   1E+02 0.00097   21.1   3.0   24    5-28     63-87  (93)
</pre><pre> 39 2GW4_D Kaede; BETA BARREL, LUM  38.0 1.6E+02  0.0015   23.4   4.2   44    3-71     26-69  (162)
</pre><pre> 40 PF10703.11 ; MoaF ; MoaF N-ter  37.4 1.7E+02  0.0015   20.3   3.8   45    3-47     27-72  (110)
</pre><pre> 41 PF10049.11 ; DUF2283 ; Protein  36.2      71 0.00066   19.0   1.7   14   14-27      1-14  (48)
</pre><pre> 42 4MPZ_A Sas-4; beta-spine, cent  35.9 2.4E+02  0.0022   21.7   4.8   43    4-46    111-157 (209)
</pre><pre> 43 PF10807.10 ; DUF2541 ; Protein  35.8 2.1E+02  0.0019   20.9   4.3   34    2-35     70-115 (132)
</pre><pre> 44 PF14220.8 ; DUF4329 ; Domain o  35.5      83 0.00077   21.9   2.2   21    5-25    116-136 (136)
</pre><pre> 45 6YWS_h Mitochondrial ribosomal  34.8 1.1E+02   0.001   20.9   2.6   18    4-21     55-72  (98)
</pre><pre> 46 PF16463.7 ; Phage_TTP_13 ; Pha  34.3 1.9E+02  0.0018   19.9   3.7   47    1-48     85-134 (137)
</pre><pre> 47 5EXC_I Green fluorescent prote  34.2   2E+02  0.0019   23.1   4.3   44    3-71     26-69  (170)
</pre><pre> 48 5EXC_M Green fluorescent prote  34.2   2E+02  0.0019   23.1   4.3   44    3-71     26-69  (170)
</pre><pre> 49 PF06832.14 ; BiPBP_C ; Penicil  34.2 1.4E+02  0.0013   18.9   2.9   24    4-27     11-37  (87)
</pre><pre> 50 6OVM_R Siderophore-interacting  34.2 3.3E+02  0.0031   20.4   5.2   44    3-46     17-61  (219)
</pre><pre> 51 d2auwa2 d.331.1.1 (A:4-87) Hyp  33.7 1.1E+02   0.001   20.4   2.5   15    4-18     16-30  (84)
</pre><pre> 52 2K4Q_A Major tail protein V; g  33.6 1.6E+02  0.0015   20.8   3.3   47    1-47    107-153 (156)
</pre><pre> 53 6NU2_k 39S ribosomal protein L  33.4 1.4E+02  0.0013   19.1   2.8   18    4-21     51-68  (84)
</pre><pre> 54 PF08813.13 ; Phage_tail_3 ; Ph  33.0 1.4E+02  0.0013   21.6   3.1   45    4-48    113-158 (161)
</pre><pre> 55 2ZMU_A Fluorescent protein; GF  32.7 1.7E+02  0.0016   24.3   3.8   44    3-71     89-132 (223)
</pre><pre> 56 PF19720.1 ; DUF6214 ; Family o  32.5      82 0.00076   25.8   2.0   16    1-16     17-32  (173)
</pre><pre> 57 d3qx1a1 d.15.1.2 (A:571-650) a  32.4 1.1E+02 0.00099   19.0   2.1   16    1-16      2-17  (80)
</pre><pre> 58 PF19686.1 ; DUF6188 ; Family o  31.7 1.3E+02  0.0012   20.3   2.5   18    4-21     71-88  (115)
</pre><pre> 59 3RWA_E Fluorescent protein FP4  31.6 2.1E+02  0.0019   24.2   4.1   44    3-71    158-201 (233)
</pre><pre> 60 3RWA_G Fluorescent protein FP4  31.6 2.1E+02  0.0019   24.2   4.1   44    3-71    158-201 (233)
</pre><pre> 61 4RGI_A Uncharacterized protein  31.6 1.2E+02  0.0011   19.4   2.4   15    3-17     24-38  (73)
</pre><pre> 62 PF01353.24 ; GFP ; Green fluor  31.4   2E+02  0.0018   23.6   3.9   44    3-71     82-125 (214)
</pre><pre> 63 d5lhxa_ d.223.1.0 (A:) automat  31.4 2.2E+02   0.002   18.7   3.5   25    1-25     19-45  (87)
</pre><pre> 64 d1i42a_ d.15.1.2 (A:) p47 {Nor  31.2 1.3E+02  0.0012   19.7   2.4   15    4-18     16-30  (89)
</pre><pre> 65 2A50_D GFP-like non-fluorescen  31.1 2.2E+02   0.002   22.8   4.0   44    3-71     26-69  (168)
</pre><pre> 66 6DEJ_B GFP-like non-fluorescen  30.9 1.9E+02  0.0017   24.3   3.8   44    3-71     98-141 (235)
</pre><pre> 67 PF07070.13 ; Spo0M ; SpoOM pro  30.8 3.3E+02   0.003   20.2   4.7   37   36-72     13-51  (208)
</pre><pre> 68 2Y3M_B PROTEIN TRANSPORT PROTE  30.5 1.4E+02  0.0013   20.3   2.6   20    9-28    132-151 (175)
</pre><pre> 69 PF06835.15 ; LptC ; Lipopolysa  30.4 3.4E+02  0.0032   18.8   5.8   61   12-74     87-158 (175)
</pre><pre> 70 2Y9K_D PROTEIN INVG; PROTEIN T  30.1 1.1E+02   0.001   20.0   2.0   20    9-28     99-118 (137)
</pre><pre> 71 2Y9K_E PROTEIN INVG; PROTEIN T  30.1 1.1E+02   0.001   20.0   2.0   20    9-28     99-118 (137)
</pre><pre> 72 5R0D_A Pre-mRNA-splicing facto  29.9 1.7E+02  0.0016   24.9   3.4   35    4-39     29-71  (258)
</pre><pre> 73 PF07593.14 ; UnbV_ASPIC ; ASPI  29.9 1.6E+02  0.0014   17.6   2.5   18    1-18     41-58  (65)
</pre><pre> 74 3L6I_B Uncharacterized lipopro  29.6 2.2E+02  0.0021   21.6   3.7   23   13-35     87-109 (181)
</pre><pre> 75 PF09981.11 ; DUF2218 ; Unchara  29.6 2.2E+02   0.002   18.0   3.2   23    5-27     33-55  (89)
</pre><pre> 76 PF10387.11 ; DUF2442 ; Protein  28.6 1.7E+02  0.0016   18.3   2.6   17    4-20     11-27  (76)
</pre><pre> 77 d1sxva1 b.40.5.0 (A:1-159) aut  28.3 2.4E+02  0.0022   21.4   3.7   24    4-27      4-29  (159)
</pre><pre> 78 2RH7_C Green fluorescent prote  28.2 2.5E+02  0.0023   23.7   4.1   44    3-71     91-134 (239)
</pre><pre> 79 PF06097.13 ; DUF945 ; Bacteria  28.0   2E+02  0.0018   23.9   3.4   23   49-71    452-474 (477)
</pre><pre> 80 PF13600.8 ; DUF4140 ; N-termin  27.6 1.8E+02  0.0017   19.4   2.7   38    6-43      1-44  (100)
</pre><pre> 81 2W68_B SIALIDASE; CBM, CALCIUM  27.6 1.4E+02  0.0013   23.8   2.5   20   14-33    136-158 (195)
</pre><pre> 82 2W68_C SIALIDASE; CBM, CALCIUM  27.6 1.4E+02  0.0013   23.8   2.5   20   14-33    136-158 (195)
</pre><pre> 83 PF13100.8 ; OstA_2 ; OstA-like  27.5 3.7E+02  0.0034   18.2   6.3   57   14-72     43-104 (159)
</pre><pre> 84 PF06155.14 ; GBBH-like_N ; Gam  27.4 1.5E+02  0.0014   18.1   2.2   17    3-19      8-24  (85)
</pre><pre> 85 4JK8_A Pre-mRNA-processing-spl  27.2 1.3E+02  0.0012   24.7   2.3   28   11-39     36-63  (222)
</pre><pre> 86 6GP1_B Green to red photoconve  27.2 2.7E+02  0.0025   22.2   3.9   41    3-68     26-66  (163)
</pre><pre> 87 4DXM_A GREEN FLUORESCENT PROTE  26.8 2.5E+02  0.0023   23.4   3.8   44    3-71     87-130 (229)
</pre><pre> 88 d2a46a_ d.22.1.0 (A:) automate  26.6 2.1E+02  0.0019   24.2   3.4   47    8-71     84-134 (220)
</pre><pre> 89 PF18544.3 ; Polo_box_3 ; Polo   26.6 2.5E+02  0.0024   21.2   3.5   23    4-27     25-47  (96)
</pre><pre> 90 PF19367.1 ; DUF5943 ; Domain o  26.5 1.6E+02  0.0015   18.9   2.2   29    9-37      1-29  (96)
</pre><pre> 91 PF14157.8 ; YmzC ; YmzC-like p  26.3 1.4E+02  0.0013   20.5   1.9   13   14-26     40-52  (60)
</pre><pre> 92 2KXJ_A UBX domain-containing p  26.3 1.5E+02  0.0014   18.3   2.0   16    1-16     11-26  (90)
</pre><pre> 93 PF12970.9 ; DUF3858 ; Domain o  26.3 3.4E+02  0.0032   17.5   5.4   42    4-67     40-81  (116)
</pre><pre> 94 d1s3si_ d.15.1.2 (I:) p47 {Nor  25.9 1.4E+02  0.0013   18.4   1.8   12    5-16     22-33  (50)
</pre><pre> 95 d3o2ga1 d.392.1.1 (A:1-95) Gam  25.9 1.4E+02  0.0013   19.7   2.0   17    3-19     15-31  (95)
</pre><pre> 96 3E9L_A Pre-mRNA-processing-spl  25.6 1.1E+02   0.001   25.5   1.8   28   11-39     45-72  (257)
</pre><pre> 97 3NED_A PAmCherry1 protein; RFP  25.5 2.8E+02  0.0026   23.4   3.9   44    3-71    104-147 (242)
</pre><pre> 98 d1rl6a1 d.141.1.1 (A:7-81) Rib  25.2   2E+02  0.0018   18.5   2.5   20    7-28      2-21  (75)
</pre><pre> 99 2ICR_B Red fluorescent protein  25.2 3.2E+02   0.003   23.0   4.2   44    3-71     99-144 (237)
</pre><pre>100 d2cqla1 d.141.1.1 (A:1-87) Rib  25.0 2.8E+02  0.0026   18.4   3.2   25    2-28      5-29  (87)
</pre><pre>101 d2icra_ d.22.1.0 (A:) automate  24.8 3.4E+02  0.0031   23.1   4.3   43    3-68     90-132 (228)
</pre><pre>102 3LF4_B Fluorescent Timer Precu  24.8 3.1E+02  0.0029   21.9   3.8   44    3-71     26-69  (164)
</pre><pre>103 2YMS_B OUTER MEMBRANE PROTEIN   24.8 2.5E+02  0.0023   15.4   3.1   21    5-25     50-70  (74)
</pre><pre>104 1XMZ_A GFP-like chromoprotein   24.6 2.9E+02  0.0027   23.3   3.9   44    3-71     99-142 (241)
</pre><pre>105 PF18161.3 ; ISP1_C ; ISP1 C-te  24.5 3.1E+02  0.0029   20.7   3.6   24    4-27     11-38  (107)
</pre><pre>106 d2qi2a1 b.38.4.1 (A:1-126) Cel  24.3 1.7E+02  0.0015   21.9   2.3   16   14-29     69-84  (126)
</pre><pre>107 4ZBL_A Photosesitizer mKillerO  24.0 2.9E+02  0.0027   23.3   3.7   44    3-71    104-147 (240)
</pre><pre>108 d2wura_ d.22.1.1 (A:) automate  23.8 3.3E+02   0.003   23.3   4.0   41    3-68     92-132 (230)
</pre><pre>109 PF17479.4 ; DUF3048_C ; Protei  23.7 4.9E+02  0.0046   18.4   7.6   68    5-72      1-81  (112)
</pre><pre>110 PF18315.3 ; VCH_CASS14 ; Integ  23.6 2.1E+02   0.002   21.3   2.6   25   14-60     62-86  (97)
</pre><pre>111 3PIB_B eqFP578 fluorescent pro  23.6 3.5E+02  0.0033   22.7   4.1   44    3-71     95-138 (236)
</pre><pre>112 3PET_B Putative adhesin; RIGHT  23.5 5.5E+02  0.0051   18.9   8.0   69    4-73     82-170 (221)
</pre><pre>113 PF18045.3 ; ISP3_C ; ISP3 C-te  23.4 3.4E+02  0.0031   20.5   3.6   24    4-27     14-41  (110)
</pre><pre>114 d5x3pa_ d.15.1.0 (A:) automate  23.4 2.7E+02  0.0025   17.2   2.7   20    4-23      5-24  (80)
</pre><pre>115 6VU9_A Threonine--tRNA ligase;  23.4 1.6E+02  0.0014   25.5   2.2   17    5-21     10-26  (641)
</pre><pre>116 6X2D_B Ribonuclease E; Structu  23.1 2.9E+02  0.0027   20.1   3.2   31    1-32      1-31  (115)
</pre><pre>117 3IR8_A Large stokes shift fluo  22.9 3.2E+02   0.003   22.6   3.7   44    3-71     89-132 (221)
</pre><pre>118 4HE4_A Yellow fluorescent prot  22.8 3.5E+02  0.0032   22.9   4.0   44    3-71     90-133 (244)
</pre><pre>119 2AUW_A hypothetical protein NE  22.7 2.2E+02   0.002   20.2   2.5   15    4-18     21-35  (170)
</pre><pre>120 3CGL_B GFP-like fluorescent ch  22.6 3.4E+02  0.0031   23.0   3.9   44    3-71    102-145 (241)
</pre><pre>121 2A46_A GFP-like fluorescent ch  22.5 3.5E+02  0.0032   23.0   3.9   44    3-71    104-147 (238)
</pre><pre>122 PF03958.19 ; Secretin_N ; Bact  22.4 2.8E+02  0.0026   15.9   2.5   20    9-28     23-42  (63)
</pre><pre>123 TIGR03353 VI_chp_4; type VI se  22.4 1.4E+02  0.0013   25.8   1.7   21    1-21     56-76  (439)
</pre><pre>124 PF11765.10 ; Hyphal_reg_CWP ;   22.3 2.1E+02   0.002   24.0   2.7   22    6-27    238-268 (331)
</pre><pre>125 2HPW_A Green fluorescent prote  22.2 3.6E+02  0.0033   22.8   3.9   41    3-68     93-133 (233)
</pre><pre>126 6M9Z_D Fluorescent protein lan  22.0 3.5E+02  0.0032   22.6   3.8   44    3-71     86-129 (230)
</pre><pre>127 6S68_A Aequorea cf. australis   22.0 3.8E+02  0.0036   22.3   4.0   44    3-71     90-133 (230)
</pre><pre>128 4Q9W_B GFP-like fluorescent ch  22.0 3.6E+02  0.0034   22.2   3.9   44    3-71     87-130 (218)
</pre><pre>129 2C9J_A GREEN FLUORESCENT PROTE  22.0 3.7E+02  0.0034   22.3   3.9   44    3-71     87-130 (223)
</pre><pre>130 2G6Y_B green fluorescent prote  21.7 3.8E+02  0.0035   22.1   3.9   44    3-71     83-126 (217)
</pre><pre>131 2G6Y_C green fluorescent prote  21.7 3.8E+02  0.0035   22.1   3.9   44    3-71     83-126 (217)
</pre><pre>132 PF06505.13 ; XylR_N ; Activato  21.7 2.3E+02  0.0022   18.5   2.3   25   13-37      4-28  (101)
</pre><pre>133 4DKN_B AMPHIOXUS GREEN FLUORES  21.6 3.8E+02  0.0036   22.2   3.9   44    3-71     88-131 (221)
</pre><pre>134 3KF8_D Protein ten1; OB fold,   21.6 1.7E+02  0.0016   22.1   1.9   16   12-27     30-45  (123)
</pre><pre>135 5GZT_A Chitinase; HYDROLASE; H  21.5 5.9E+02  0.0055   18.5   4.9   47   16-72     46-97  (108)
</pre><pre>136 PF03463.17 ; eRF1_1 ; eRF1 dom  21.4 2.2E+02   0.002   20.3   2.3   16   14-29     72-87  (130)
</pre><pre>137 3AI5_A yeast enhanced green fl  21.3 4.3E+02   0.004   21.4   4.1   41    3-68     95-135 (307)
</pre><pre>138 4OHS_D FAR-RED FLUORESCENT PRO  21.3 3.6E+02  0.0033   22.7   3.7   44    3-71     97-140 (237)
</pre><pre>139 4OHS_E FAR-RED FLUORESCENT PRO  21.3 3.6E+02  0.0033   22.7   3.7   44    3-71     97-140 (237)
</pre><pre>140 4NKB_B Probable serine/threoni  21.2 3.2E+02   0.003   22.6   3.4   23    4-27    153-175 (230)
</pre><pre>141 PF03968.16 ; LptD_N ; LptA/(Lp  21.0 4.6E+02  0.0043   17.1   3.9   34   12-48     72-105 (111)
</pre><pre>142 PF06905.15 ; FAIM1 ; Fas apopt  21.0 6.5E+02   0.006   19.5   4.8   65    4-73     59-124 (172)
</pre><pre>143 PF16461.7 ; Phage_TTP_12 ; Lam  21.0 2.8E+02  0.0026   19.0   2.7   45    4-48     90-134 (135)
</pre><pre>144 d1xmza_ d.22.1.1 (A:) automate  20.9 3.9E+02  0.0036   22.8   3.9   44    3-71     87-130 (229)
</pre><pre>145 PF13101.8 ; DUF3945 ; Protein   20.9 2.3E+02  0.0022   18.2   2.2   14   13-26      5-18  (59)
</pre><pre>146 4DKM_F Amphioxus Green Fluores  20.9 3.8E+02  0.0035   22.1   3.8   44    3-71     82-125 (214)
</pre><pre>147 d4dkna_ d.22.1.0 (A:) automate  20.9 3.8E+02  0.0035   22.7   3.8   44    3-71     85-128 (218)
</pre><pre>148 4LD1_A Uncharacterized protein  20.8 2.6E+02  0.0024   19.3   2.5   20    5-24    163-182 (184)
</pre><pre>149 d4ohsa_ d.22.1.0 (A:) automate  20.8 3.7E+02  0.0034   22.9   3.7   44    3-71     93-136 (225)
</pre><pre>150 2C9I_B GREEN FLUORESCENT PROTE  20.7 3.7E+02  0.0035   22.3   3.7   44    3-71     88-131 (226)
</pre><pre>151 2C9I_D GREEN FLUORESCENT PROTE  20.7 3.7E+02  0.0035   22.3   3.7   44    3-71     88-131 (226)
</pre><pre>152 d2ejoa_ d.22.1.0 (A:) automate  20.7 3.8E+02  0.0035   22.7   3.8   44    3-71     91-134 (217)
</pre><pre>153 2IB5_B Chromo protein; beta ba  20.5 3.8E+02  0.0035   22.5   3.7   44    3-71     91-134 (233)
</pre><pre>154 d3cgla_ d.22.1.1 (A:) automate  20.5   4E+02  0.0037   22.7   3.9   44    3-71     93-136 (226)
</pre><pre>155 d4dkma_ d.22.1.0 (A:) automate  20.5   4E+02  0.0037   22.5   3.9   44    3-71     84-127 (216)
</pre><pre>156 6LOF_B GFP-like fluorescent ch  20.5 5.8E+02  0.0054   20.4   4.5   46    3-71     26-71  (164)
</pre><pre>157 6GP0_A Green to red photoconve  20.4 3.9E+02  0.0037   23.0   3.9   41    3-68    120-160 (257)
</pre><pre>158 4MPZ_A Sas-4; beta-spine, cent  20.4 2.6E+02  0.0024   21.5   2.6   20    5-24    186-205 (209)
</pre><pre>159 4MPZ_A Sas-4; beta-spine, cent  20.3 3.8E+02  0.0036   20.6   3.5   23    5-27     37-59  (209)
</pre><pre>160 5OXZ_B NEQ528; Intein complex   20.3 2.2E+02   0.002   17.7   1.8   18   56-73     13-30  (33)
</pre><pre>161 4G08_A Protein InvG; Ring-buil  20.2 2.3E+02  0.0021   19.3   2.1   20    9-28    113-132 (159)
</pre><pre>
</pre><pre>No 1
</pre><pre>>TIGR01644 phage_P2_V; phage baseplate assembly protein V. This model describes a family of phage (and bacteriocin) proteins related to the phage P2 V gene product, which forms the small spike at the tip of the tail.
</pre><pre>Probab=99.15  E-value=1.4e-09  Score=77.08  Aligned_cols=83  Identities=39%  Similarity=0.685  Sum_probs=65.3  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEceeEEEEEec--cEEEeeCCeEe-cC------c---------eEeeeeeecCCeEE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVGMKTARVSAE--TRILLDTPEVQ-KG------G---------EMHGNIAHSGGAFT   65 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As--~sITldtP~V~-~G------g---------~m~G~ithsgG~~s   65 (90)
</pre><pre>                      +..+.|+||..++|+++...|.+...++++++|.  .++++++|.++ .|      +         .+.|+++. .|.++
</pre><pre>T Consensus        90 ~~~~~~~~g~~i~~~~~~~~l~i~~~~~~~~~~~~~~~~~~~~~~~~~~g~~~~~~~~~~~~~g~~~~~g~~~~-~g~v~  168 (190)
</pre><pre>T TIGR01644        90 DTVTYFADGARIEYDKAAHALTVCGIKTVYLDAGDGTTVTVDAPQITFTGGVTTLKGPETVITGNVTIDGTLHT-GGDAS  168 (190)
</pre><pre>T ss_pred             cEEEEeCCCCEEEEeCCCCEEEEeeceeEEEeCCCCceEEecCCEEEEecceEEecCCceEEEeeEEEcCceee-cCcEE
</pre><pre>Confidence            5678899999999999999999988888888875  46778888776 11      1         13344443 47889
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ECCEEeeeeCCCcccCCCCCCC
</pre><pre>Q MBG6247344.1/9   66 SNGVQIDDHDHGAVQRGGSWTE   87 (90)
</pre><pre>Q Consensus        66 sNGv~v~~H~HggV~~Gg~~T~   87 (90)
</pre><pre>                      .||+.++.|.|.++++|+..|+
</pre><pre>T Consensus       169 ~~g~~~~~H~H~~~~~G~~~t~  190 (190)
</pre><pre>T TIGR01644       169 SNGISLDKHGHTGVHSGGTTSG  190 (190)
</pre><pre>T ss_pred             ECCEEeccccccCCCCCCCCCC
</pre><pre>Confidence            9999999999999999987663
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>4S36_A Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; 1.46A {Pseudomonas aeruginosa}
</pre><pre>Probab=99.13  E-value=1.6e-09  Score=68.60  Aligned_cols=34  Identities=26%  Similarity=0.140  Sum_probs=27.4  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEc-eeEEEEEec
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVG-MKTARVSAE   37 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~G-iKTAtV~As   37 (90)
</pre><pre>                      +..++|+||.+++|+++.+.|.+.. .+.+.+.+.
</pre><pre>T Consensus         2 ~~~~~~~dG~~i~yd~~~~~l~i~~~~g~~~i~~~   36 (90)
</pre><pre>T 4S36_A            2 SETWRFDDGASLSYDWAAHRYRVELPSGTVEVRVG   36 (90)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEETTTTEEEEECSSSEEEEEET
</pre><pre>T ss_pred             ceEEEeCCCcEEEEeCCCCEEEEEeCCceEEEEeC
</pre><pre>Confidence            4677899999999999999999987 356666553
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>3QR7_B Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; 0.94A {Enterobacteria phage P2}
</pre><pre>Probab=99.08  E-value=9.2e-09  Score=66.61  Aligned_cols=80  Identities=40%  Similarity=0.618  Sum_probs=45.7  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEceeEEEEEeccE---------------EEeeCCeEe-cCc------------eEee
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVGMKTARVSAETR---------------ILLDTPEVQ-KGG------------EMHG   55 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~s---------------ITldtP~V~-~Gg------------~m~G   55 (90)
</pre><pre>                      +++++|+||.++.||++++.|.+...+.+.+.+...               +.+.++.+. .+.            .+.+
</pre><pre>T Consensus         3 ~~~~~~~dG~~~~yd~~~~~l~~~~~g~i~v~~~~~~~~~~~~~~~~~~~~v~~~~~~~~~~g~~~~g~~~~~g~~~~~g   82 (115)
</pre><pre>T 3QR7_B            3 ALHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTATVPVVMVKASTRVTLDTPEVVCTNRLITGTLEVQKGGTMRG   82 (115)
</pre><pre>T ss_dssp             CCEEECTTSCEEEEETTTTEEEC-CCSCCCCCCSSCCCCCCSCCCCCCSSCCCCCSSCCCCSSCCCCSSCCCSSCCCCCS
</pre><pre>T ss_pred             eEEEEcCCCCEEEEeCCCCeEEEEeccceEEEcCccEEEECCeEEEEeCceEEEeCCeEEEeeeEEECCEEEcCCcEEEE
</pre><pre>Confidence            457789999999999999988886544443333100               111111111 100            1112
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeecCCeEEECCEEeeeeCCCcccCCC
</pre><pre>Q MBG6247344.1/9   56 NIAHSGGAFTSNGVQIDDHDHGAVQRGG   83 (90)
</pre><pre>Q Consensus        56 ~ithsgG~~ssNGv~v~~H~HggV~~Gg   83 (90)
</pre><pre>                      .+....+++..++.++..|.|..+..|.
</pre><pre>T Consensus        83 ~~~~~~~~~~~~~~~~~~h~h~~~~~g~  110 (115)
</pre><pre>T 3QR7_B           83 NIEHTGGELSSNGKVLHTHKHPGDSGGT  110 (115)
</pre><pre>T ss_dssp             CCCCCSSCCEETTEETTTCCCBCSSSSB
</pre><pre>T ss_pred             eEEEeCceEEeCCEEecceeCCCCCCCC
</pre><pre>Confidence            2333025678888999999998766665
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>3AQJ_A Baseplate assembly protein V; bacteriophage, tail spike, iron binding, beta-helix, infection, METAL BINDING PROTEIN; 1.27A {Enterobacteria phage P2}
</pre><pre>Probab=99.05  E-value=1.2e-08  Score=69.19  Aligned_cols=88  Identities=38%  Similarity=0.549  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEE---cee------------EEEEEeccEEEeeCCeEe-cC------------ce
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAV---GMK------------TARVSAETRILLDTPEVQ-KG------------GE   52 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~---GiK------------TAtV~As~sITldtP~V~-~G------------g~   52 (90)
</pre><pre>                      +-....+.|+||++++||.+...|++.   +++            .+++++..+++++.|.+. .|            ..
</pre><pre>T Consensus        19 ~~~~~~~~~~dG~~~~yd~~~~~l~i~~~g~i~i~~~~~i~~~~~~~~i~~~~~v~~~~~~i~~~g~~~~g~~~~~g~~~   98 (134)
</pre><pre>T 3AQJ_A           19 SADALHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTATVPVVMVKASTRVTLDTPEVVCTNRLITGTLEVQKGGT   98 (134)
</pre><pre>T ss_dssp             BCSCEEEECTTSCEEEEETTTTEEEEEEEEEEEEEEEEEEEEECSEEEEECSSEEEEESSEEEEEEEEEEEEEEESSCEE
</pre><pre>T ss_pred             CCCcEEEEcCCCCEEEEeCCCCEEEEEeCceeEEEcCceEEEECCeEEEEeCcceEEeCCcEEEeccEEECCeeeccCcE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EeeeeeecCCeEEECCEEeeeeCCCcccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   53 MHGNIAHSGGAFTSNGVQIDDHDHGAVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        53 m~G~ithsgG~~ssNGv~v~~H~HggV~~Gg~~T~G~r   90 (90)
</pre><pre>                      +.|+++..++.+..+++.+..|.|.+++.|.  |.++.
</pre><pre>T Consensus        99 ~~g~i~~~~~~~~~~~~~~~~h~h~~~~~g~--t~~~~  134 (134)
</pre><pre>T 3AQJ_A           99 MRGNIEHTGGELSSNGKVLHTHKHPGDSGGT--TGSPL  134 (134)
</pre><pre>T ss_dssp             EESCEEEESSCEEETTEETTTCCCBCTTSSB--CCSCC
</pre><pre>T ss_pred             EECeEEEeCCceeeCCEEeccccCCCCCCCC--CCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>3QR8_A Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; HET: MSE; 2.03A {Enterobacteria phage P2}
</pre><pre>Probab=99.02  E-value=1.4e-08  Score=70.94  Aligned_cols=85  Identities=41%  Similarity=0.641  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEc---------------eeEEEEEeccEEEeeCCeEe-------------cCceEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVG---------------MKTARVSAETRILLDTPEVQ-------------KGGEMH   54 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~G---------------iKTAtV~As~sITldtP~V~-------------~Gg~m~   54 (90)
</pre><pre>                      ....+.|.||+.++||++...|.+..               .+...++|...+++++|.+.             .+..++
</pre><pre>T Consensus        98 ~~~~~~~~~G~~i~~~~~~~~i~~~~~~~i~i~~~g~v~i~~~~~~~~a~~~i~i~~~~~~~~~~~~~g~~~~~~~~~~~  177 (211)
</pre><pre>T 3QR8_A           98 DALHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTATVPVVMVKASTRVTLDTPEVVCTNRLITGTLEVQKGGTMR  177 (211)
</pre><pre>T ss_dssp             SCEEEECTTSCEEEEETTTTEEEEESCSCCCCCCSSCCCCCCSCCCCCCSSCCCCCSSCCCCCSCC--------------
</pre><pre>T ss_pred             CcEEEEcCCCCEEEEECCCCeEEEeeCceEEEEcCCcEEEEcCeEEEecCCeEEEeCCceEEcCcEEeCcEEEccCCEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeeecCCeEEECCEEeeeeCCCcccCCCCCCCcC
</pre><pre>Q MBG6247344.1/9   55 GNIAHSGGAFTSNGVQIDDHDHGAVQRGGSWTEGT   89 (90)
</pre><pre>Q Consensus        55 G~ithsgG~~ssNGv~v~~H~HggV~~Gg~~T~G~   89 (90)
</pre><pre>                      |+++..+|.++.+|+.+..|.|.++.  ++.|+.+
</pre><pre>T Consensus       178 G~~~~~~~~~~~~~~~~~~H~H~~~~--~g~t~~p  210 (211)
</pre><pre>T 3QR8_A          178 GNIEHTGGELSSNGKVLHTHKHPGDS--GGTTGSP  210 (211)
</pre><pre>T ss_dssp             -----------------------------------
</pre><pre>T ss_pred             ccEEEcCCcEEeCCEEeccccCCCCC--CCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>3PQI_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 2.642A {Bacteriophage phi92}
</pre><pre>Probab=98.89  E-value=2.8e-08  Score=72.86  Aligned_cols=89  Identities=21%  Similarity=0.415  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEEceeEEEEEe-----------ccEEEeeCCeEe-cCc--eEeee-eeecCCeEE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAVGMKTARVSA-----------ETRILLDTPEVQ-KGG--EMHGN-IAHSGGAFT   65 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~A-----------s~sITldtP~V~-~Gg--~m~G~-ithsgG~~s   65 (90)
</pre><pre>                      +..++++.|++|+.|+|++ ++.+.+..-..++|+|           ..+++++.|.++ .|+  .++|+ ++..|+.++
</pre><pre>T Consensus       138 ~~~~~~~~~~~g~~i~~~~-~G~i~i~~~~~i~i~~~~~~i~i~~~~~~~~~i~~~~~~~~g~~~~~~g~~i~~~g~~~~  216 (247)
</pre><pre>T 3PQI_A          138 DSEKVIISNNKQTYASFDP-NGNISVYNTQGMKIDMTPNSIVLTDAGGGKLTLQGGTMTYKGGTVNLNGLTITPDGRMTD  216 (247)
</pre><pre>T ss_dssp             CSSCEEEEETTTEEEEECT-TSCEEEEETTSCEEEEETTEEEEEETTTEEEEEETTEEEEESSCEEETTEEECTTSCEEC
</pre><pre>T ss_pred             CcccEEEEcCCCcEEEECC-CCcEEEEeCCceEEEeCCCceEEEeCCCceEEEcCCeeeecCCeEEEcCcEEcCCCCEEc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ECCEEeeeeCCC--cccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   66 SNGVQIDDHDHG--AVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        66 sNGv~v~~H~Hg--gV~~Gg~~T~G~r   90 (90)
</pre><pre>                      .||+.+..|.|.  +|+.|++.|+.+.
</pre><pre>T Consensus       217 ~~g~~~~~H~H~~~~v~~G~~~t~~~~  243 (247)
</pre><pre>T 3PQI_A          217 SGGIGLHTHTHPVRGVETGGSTVTSDK  243 (247)
</pre><pre>T ss_dssp             TTSCBTTTCCCCCCSCCCCSCCCCCCS
</pre><pre>T ss_pred             CCCccccceeecCCcccCCCCCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>3PQH_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 1.295A {Bacteriophage phi92}
</pre><pre>Probab=98.88  E-value=4e-08  Score=66.91  Aligned_cols=89  Identities=21%  Similarity=0.350  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEe---------ecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe---cCc--eEee-eeeecCCeEE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVIT---------ALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ---KGG--EMHG-NIAHSGGAFT   65 (90)
</pre><pre>Q Consensus         1 s~~~~~i~---------fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~---~Gg--~m~G-~ithsgG~~s   65 (90)
</pre><pre>                      +-..+++.         |+||++++||+++..|.+. =..+.|++...+++..+.-.   .++  .+.| +++..++.+.
</pre><pre>T Consensus        18 ~~~~~~~~~~~~~~~~~~~dG~~i~yd~~~~~l~i~-~g~i~i~~~~~~~~~~~~~~i~~~~~~v~i~G~~~~~~g~~~~   96 (127)
</pre><pre>T 3PQH_A           18 DSEKVIISNNKQTYASFDPNGNISVYNTQGMKIDMT-PNSIVLTDAGGGKLTLQGGTMTYKGGTVNLNGLTITPDGRMTD   96 (127)
</pre><pre>T ss_dssp             --CCEEEEETTTEEEEECTTSCEEEEETTSCEEEEE-TTEEEEEETTTEEEEEETTEEEEESSCEEETTEEECTTSCEEC
</pre><pre>T ss_pred             CcccEEEEcCCcceeeeCCCCcEEEEcCCCcEEEec-CCEEEEEcCCCcEEEECCCeEEEeCCeEEEeceEEecCCCeee
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ECCEEeeeeCC--CcccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   66 SNGVQIDDHDH--GAVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        66 sNGv~v~~H~H--ggV~~Gg~~T~G~r   90 (90)
</pre><pre>                      .+|+.+..|.|  .++..|+..+....
</pre><pre>T Consensus        97 ~~g~~~~~h~h~~~~~~~g~~~~~~~~  123 (127)
</pre><pre>T 3PQH_A           97 SGGIGLHTHTHPVRGVETGGSTVTSDK  123 (127)
</pre><pre>T ss_dssp             TTSCBTTTCCCCCCSCCCCSCCCCCCS
</pre><pre>T ss_pred             eCCEEcCCceecCCCcccCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>3VTN_A Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.75A {Enterobacteria phage Mu}
</pre><pre>Probab=98.81  E-value=1.3e-07  Score=59.66  Aligned_cols=80  Identities=19%  Similarity=0.205  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEceeEEEEE-------eccEEEeeCCeEe--cCceEeeeeeecC----------CeEEECCE
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVGMKTARVS-------AETRILLDTPEVQ--KGGEMHGNIAHSG----------GAFTSNGV   69 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~GiKTAtV~-------As~sITldtP~V~--~Gg~m~G~ithsg----------G~~ssNGv   69 (90)
</pre><pre>                      ||||+++.||.....|.+..-..+.|+       |...+++++|.+.  ..-.+.|+++..|          +.+..+++
</pre><pre>T Consensus         3 ~~~G~~~~y~~~~~~i~~~~~g~i~i~~~~~~i~~~~~~~~~~~~i~~~g~~~~~g~~~~~g~~~~~g~~~~~~~~~~~~   82 (101)
</pre><pre>T 3VTN_A            3 MEPGDAGIYHHEGHRIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFTGDVEIQKGLGVKGKSQFDSNITAPDAIINGK   82 (101)
</pre><pre>T ss_dssp             --------------CEEECTTSCEEECSSCCCCCCSSCCCCCCSCCCCSSCCCCSSCCCCCSCCCCSSCCCCSCCCBTTB
</pre><pre>T ss_pred             CCCCcEEEEeCCCCEEEEcCCCEEEEeCCeEEEEcCCceEEeCCcEEEeccEEEcCCeEEcCcceEecceecCceEECCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EeeeeCCCcccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   70 QIDDHDHGAVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        70 ~v~~H~HggV~~Gg~~T~G~r   90 (90)
</pre><pre>                      .+..|.|.++.  ++.|++++
</pre><pre>T Consensus        83 ~~~~h~h~~~~--~~~t~~~~  101 (101)
</pre><pre>T 3VTN_A           83 STDKHIHRGDS--GGTTGPMQ  101 (101)
</pre><pre>T ss_dssp             CSSCC----------------
</pre><pre>T ss_pred             EcCCcccCCCC--CCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>3VTO_B Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=98.76  E-value=2e-07  Score=62.27  Aligned_cols=79  Identities=18%  Similarity=0.171  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEceeEEEE-------EeccEEEeeCCeEe--cCceEeeeeeec----------CCeEEECCE
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVGMKTARV-------SAETRILLDTPEVQ--KGGEMHGNIAHS----------GGAFTSNGV   69 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~GiKTAtV-------~As~sITldtP~V~--~Gg~m~G~iths----------gG~~ssNGv   69 (90)
</pre><pre>                      ||||+++.||.+...|.+..-.+..+       .|...+++++|.+.  ..-.+.|.++..          +|.+..+|+
</pre><pre>T Consensus         9 ~~dG~~~~yd~~g~~l~i~~~g~i~i~~~~~~i~a~~~i~~~~~~~~i~g~~~~~g~~~~~g~~~~~g~~~~g~~~~~g~   88 (115)
</pre><pre>T 3VTO_B            9 LEPGDAGIYHHEGHRIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFTGDVEIQKGLGVKGKSQFDSNITAPDAIINGK   88 (115)
</pre><pre>T ss_dssp             CCTTCEEEECTTSCEEEEETTTEEEEEEEEEEEEEEEEEEEECSEEEESSEEEESSCEEESSCEEESSCEEESCEEETTE
</pre><pre>T ss_pred             CCCCcEEEEeCCCCEEEEEcCCEEEEeCCeEEEEcCCcEEEeCCcEEEEecEEEccceEecCceeeecceecCcEEECCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EeeeeCCCcccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   70 QIDDHDHGAVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        70 ~v~~H~HggV~~Gg~~T~G~r   90 (90)
</pre><pre>                      .+..|.|.   .|++.|+++.
</pre><pre>T Consensus        89 ~~~~h~h~---~~~~~t~~~~  106 (115)
</pre><pre>T 3VTO_B           89 STDKHIHR---GDSGGTTGPM  106 (115)
</pre><pre>T ss_dssp             ETTTCBCB---CTTSSBCCCC
</pre><pre>T ss_pred             EcCCceec---CCCCCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>3VTO_C Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=98.76  E-value=2e-07  Score=62.27  Aligned_cols=79  Identities=18%  Similarity=0.171  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEceeEEEE-------EeccEEEeeCCeEe--cCceEeeeeeec----------CCeEEECCE
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVGMKTARV-------SAETRILLDTPEVQ--KGGEMHGNIAHS----------GGAFTSNGV   69 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~GiKTAtV-------~As~sITldtP~V~--~Gg~m~G~iths----------gG~~ssNGv   69 (90)
</pre><pre>                      ||||+++.||.+...|.+..-.+..+       .|...+++++|.+.  ..-.+.|.++..          +|.+..+|+
</pre><pre>T Consensus         9 ~~dG~~~~yd~~g~~l~i~~~g~i~i~~~~~~i~a~~~i~~~~~~~~i~g~~~~~g~~~~~g~~~~~g~~~~g~~~~~g~   88 (115)
</pre><pre>T 3VTO_C            9 LEPGDAGIYHHEGHRIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFTGDVEIQKGLGVKGKSQFDSNITAPDAIINGK   88 (115)
</pre><pre>T ss_dssp             CCTTCEEEECTTSCEEEEETTTEEEEEEEEEEEEEEEEEEEESSEEEESSEEEESSCEEESSCEEESSCEEESCEEETTE
</pre><pre>T ss_pred             CCCCcEEEEeCCCCEEEEEcCCEEEEeCCeEEEEcCCcEEEeCCcEEEEecEEEccceEecCceeeecceecCcEEECCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EeeeeCCCcccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   70 QIDDHDHGAVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        70 ~v~~H~HggV~~Gg~~T~G~r   90 (90)
</pre><pre>                      .+..|.|.   .|++.|+++.
</pre><pre>T Consensus        89 ~~~~h~h~---~~~~~t~~~~  106 (115)
</pre><pre>T 3VTO_C           89 STDKHIHR---GDSGGTTGPM  106 (115)
</pre><pre>T ss_dssp             ETTTCBCB---CTTSSBCCCC
</pre><pre>T ss_pred             EcCCceec---CCCCCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>4RU3_A puncturing protein gp41; gp41, membrane translocation, phage baseplate, Cell puncturing device, CELL INVASION; 1.172A {Pseudomonas phage SN}
</pre><pre>Probab=98.74  E-value=2.1e-07  Score=68.08  Aligned_cols=74  Identities=22%  Similarity=0.442  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCc-cEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEeeeeCCCcc
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDG-ARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQIDDHDHGAV   79 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDG-A~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v~~H~HggV   79 (90)
</pre><pre>                      +...++|.|+|| ..|+++++          .+.|+++..+++++|.+    +++|+++.. |.+++||+.+..|.|.||
</pre><pre>T Consensus       144 ~~~~~~i~~~~G~~~I~i~~~----------~i~i~~~~~v~i~~~~i----~i~G~v~~~-g~v~~~g~~~~~H~H~gv  208 (221)
</pre><pre>T 4RU3_A          144 DSAAMVIQSTSGATRISIRGD----------NIKITAPSNVTVDTPQA----NFTGSVTIA-NTLVVNGVNVNNHGHLEN  208 (221)
</pre><pre>T ss_dssp             GTTSEEEECTTSSSEEEECSS----------CEEEECSSCCCCCSSCC----CCSSCCCCS-SCCEETTEESTTCCCBCT
</pre><pre>T ss_pred             CcCceEEEcCCCCeEEEEECC----------eEEEeCCCcEEEcCCce----EEeccEEEc-CCEEECCEEcceeEccCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCCC-CCcC
</pre><pre>Q MBG6247344.1/9   80 QRGGSW-TEGT   89 (90)
</pre><pre>Q Consensus        80 ~~Gg~~-T~G~   89 (90)
</pre><pre>                      ..|++. |+++
</pre><pre>T Consensus       209 ~~g~~~~t~~p  219 (221)
</pre><pre>T 4RU3_A          209 NPPDARTKGGM  219 (221)
</pre><pre>T ss_dssp             TTTTCBCSSSC
</pre><pre>T ss_pred             CCCCcCCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>4S37_B Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: SO4, CL, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=98.45  E-value=4.3e-06  Score=57.33  Aligned_cols=76  Identities=25%  Similarity=0.347  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEEceeE-EEEEecc--------EEEeeCCeEe-cCc-eEeeeeeecC-----CeE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAVGMKT-ARVSAET--------RILLDTPEVQ-KGG-EMHGNIAHSG-----GAF   64 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~GiKT-AtV~As~--------sITldtP~V~-~Gg-~m~G~ithsg-----G~~   64 (90)
</pre><pre>                      +.....+.|+||..++||++++.+.+..-+. .++++..        ++++++|.++ .|. .+.|+++.+|     |.+
</pre><pre>T Consensus        94 ~~~~~~~~~~~g~~~~~d~~~~~i~i~~~~~~i~~~~~~~~~~~~~~~~~i~~~~v~i~g~~~~~g~~~~~g~~~~~~~~  173 (185)
</pre><pre>T 4S37_B           94 SASSETWRFDDGASLSYDWAAHRYRVELPSGTVEVRVGASEVRVSDGAVSLKAPKISLEGPVEIAGTLTVSGDILGGGSI  173 (185)
</pre><pre>T ss_dssp             CTTEEEEECTTSCEEEEETTTTEEEEEESSSEEEEEETTEEEEEESSCEEEECSCEEEESCEEEESCEEESSCEEESSCE
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEECCCCEEEEEeCCceEEEEECCcEEEEeCCcEEEECCcEEEeCCEEEEccEEEeCcEEEccEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EECCEEeeeeCC
</pre><pre>Q MBG6247344.1/9   65 TSNGVQIDDHDH   76 (90)
</pre><pre>Q Consensus        65 ssNGv~v~~H~H   76 (90)
</pre><pre>                      +.||+.+..|.|
</pre><pre>T Consensus       174 ~~~~~~~~~h~h  185 (185)
</pre><pre>T 4S37_B          174 IDTAGNSNHHTH  185 (185)
</pre><pre>T ss_dssp             ECCC--------
</pre><pre>T ss_pred             EcCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>4S37_O Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: CL, SO4, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=98.45  E-value=4.3e-06  Score=57.33  Aligned_cols=76  Identities=25%  Similarity=0.347  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEEceeE-EEEEecc--------EEEeeCCeEe-cCc-eEeeeeeecC-----CeE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAVGMKT-ARVSAET--------RILLDTPEVQ-KGG-EMHGNIAHSG-----GAF   64 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~GiKT-AtV~As~--------sITldtP~V~-~Gg-~m~G~ithsg-----G~~   64 (90)
</pre><pre>                      +.....+.|+||..++||++++.+.+..-+. .++++..        ++++++|.++ .|. .+.|+++.+|     |.+
</pre><pre>T Consensus        94 ~~~~~~~~~~~g~~~~~d~~~~~i~i~~~~~~i~~~~~~~~~~~~~~~~~i~~~~v~i~g~~~~~g~~~~~g~~~~~~~~  173 (185)
</pre><pre>T 4S37_O           94 SASSETWRFDDGASLSYDWAAHRYRVELPSGTVEVRVGASEVRVSDGAVSLKAPKISLEGPVEIAGTLTVSGDILGGGSI  173 (185)
</pre><pre>T ss_dssp             CTTEEEEECTTSCEEEEETTTTEEEEEESSSEEEEEETTEEEEEESSCEEEECSCEEEESCEEEESCEEESSCEEESSCE
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEECCCCEEEEEeCCceEEEEECCcEEEEeCCcEEEECCcEEEeCCEEEEccEEEeCcEEEccEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EECCEEeeeeCC
</pre><pre>Q MBG6247344.1/9   65 TSNGVQIDDHDH   76 (90)
</pre><pre>Q Consensus        65 ssNGv~v~~H~H   76 (90)
</pre><pre>                      +.||+.+..|.|
</pre><pre>T Consensus       174 ~~~~~~~~~h~h  185 (185)
</pre><pre>T 4S37_O          174 IDTAGNSNHHTH  185 (185)
</pre><pre>T ss_dssp             ECCC--------
</pre><pre>T ss_pred             EcCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF18715.3 ; Phage_spike ; Phage spike trimer
</pre><pre>Probab=97.97  E-value=6.3e-05  Score=43.51  Aligned_cols=40  Identities=55%  Similarity=0.851  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             ceeEEEEEeccEEEeeCCeEe-------------cCceEeeeeeecCCeEEEC
</pre><pre>Q MBG6247344.1/9   28 GMKTARVSAETRILLDTPEVQ-------------KGGEMHGNIAHSGGAFTSN   67 (90)
</pre><pre>Q Consensus        28 GiKTAtV~As~sITldtP~V~-------------~Gg~m~G~ithsgG~~ssN   67 (90)
</pre><pre>                      ++|++.|+|+.+|++++|+++             .|+.+.|++++.+|.+.+|
</pre><pre>T Consensus         1 ~~~~~~i~a~~~v~~~~~~~~~~~~~~~~~~~~~~g~~i~g~v~~~~g~~~~~   53 (53)
</pre><pre>T W1IZ46_9GAMM/1    1 TAPEITCVASTRITLDTPEVVCTQLMSTGNLIVRNGGKMTGNIEHTGGSFSSN   53 (53)
</pre><pre>T ss_pred             CCCEEEEEeCceEEEeCCeEEEcCcEEEceEEEecCcEEEcceEEeCceeeeC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>6P2A_C CHIMERA OF BACTERIOPHAGE OBP GP146 AND A T4 GP5; Bacteriophage OBP, membrane piercing, central spike, cell puncturing device, beta helix, T4 gp5, contractile injection system; HET: STE, ELA, PLM; 1.9A {Enterobacteria phage T4}
</pre><pre>Probab=97.18  E-value=0.008  Score=43.64  Aligned_cols=65  Identities=25%  Similarity=0.357  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             eEEEceeEEEEEeccEEEeeC------CeEe--------------cCceEeeeeeecC------CeEEECCEEeeeeCCC
</pre><pre>Q MBG6247344.1/9   24 LEAVGMKTARVSAETRILLDT------PEVQ--------------KGGEMHGNIAHSG------GAFTSNGVQIDDHDHG   77 (90)
</pre><pre>Q Consensus        24 L~v~GiKTAtV~As~sITldt------P~V~--------------~Gg~m~G~ithsg------G~~ssNGv~v~~H~Hg   77 (90)
</pre><pre>                      ++|.|  ...+++...++..+      +.+.              ..-.|.|+|++.+      |.+..+|+.+..|.|.
</pre><pre>T Consensus        57 ~~v~G--~~~~~vgg~~~~~~~~~~~~g~iti~g~~~~~G~~~~~g~~~~~G~i~~~~~~~~~~~~~~~~g~~~~~h~H~  134 (148)
</pre><pre>T 6P2A_C           57 WKVAG--TVDWDVGGDWTEKMASMSSKGNVTHEGNYNQLGNYTVQGNVGIQGAFSQFGGAGSVEGGWTIDNIRYLGHRHG  134 (148)
</pre><pre>T ss_dssp             EEESS--CEEEEESSEEEEEEEEEEEEEEEEEESCEEEESCEEEESCEEEESCEEEESSCEEECSCEEETTEEGGGCEEC
</pre><pre>T ss_pred             EEEeC--cEEEEeCCcEEEEeceeecCCeEEEEcceeEEeEEEEeCCeEEEeeEEeeCCCccccCcEEECCEEecCcccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   78 AVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        78 gV~~Gg~~T~G~r   90 (90)
</pre><pre>                      +++.|+..|..+.
</pre><pre>T Consensus       135 ~~~~g~~~t~~~~  147 (148)
</pre><pre>T 6P2A_C          135 GVQSGGSKTDTPS  147 (148)
</pre><pre>T ss_dssp             SECBCSBCCCCEE
</pre><pre>T ss_pred             CccCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>6P2A_E CHIMERA OF BACTERIOPHAGE OBP GP146 AND A T4 GP5; Bacteriophage OBP, membrane piercing, central spike, cell puncturing device, beta helix, T4 gp5, contractile injection system; HET: ELA, PLM, STE; 1.9A {Enterobacteria phage T4}
</pre><pre>Probab=97.18  E-value=0.008  Score=43.64  Aligned_cols=65  Identities=25%  Similarity=0.357  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             eEEEceeEEEEEeccEEEeeC------CeEe--------------cCceEeeeeeecC------CeEEECCEEeeeeCCC
</pre><pre>Q MBG6247344.1/9   24 LEAVGMKTARVSAETRILLDT------PEVQ--------------KGGEMHGNIAHSG------GAFTSNGVQIDDHDHG   77 (90)
</pre><pre>Q Consensus        24 L~v~GiKTAtV~As~sITldt------P~V~--------------~Gg~m~G~ithsg------G~~ssNGv~v~~H~Hg   77 (90)
</pre><pre>                      ++|.|  ...+++...++..+      +.+.              ..-.|.|+|++.+      |.+..+|+.+..|.|.
</pre><pre>T Consensus        57 ~~v~G--~~~~~vgg~~~~~~~~~~~~g~iti~g~~~~~G~~~~~g~~~~~G~i~~~~~~~~~~~~~~~~g~~~~~h~H~  134 (148)
</pre><pre>T 6P2A_E           57 WKVAG--TVDWDVGGDWTEKMASMSSKGNVTHEGNYNQLGNYTVQGNVGIQGAFSQFGGAGSVEGGWTIDNIRYLGHRHG  134 (148)
</pre><pre>T ss_dssp             EEESS--CEEEEESSEEEEEEEEEEEEEEEEEESCEEEESCEEEESCEEEESCEEEESSCEEECSCEEETTEEGGGCEEC
</pre><pre>T ss_pred             EEEeC--cEEEEeCCcEEEEeceeecCCeEEEEcceeEEeEEEEeCCeEEEeeEEeeCCCccccCcEEECCEEecCcccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccCCCCCCCcCC
</pre><pre>Q MBG6247344.1/9   78 AVQRGGSWTEGTK   90 (90)
</pre><pre>Q Consensus        78 gV~~Gg~~T~G~r   90 (90)
</pre><pre>                      +++.|+..|..+.
</pre><pre>T Consensus       135 ~~~~g~~~t~~~~  147 (148)
</pre><pre>T 6P2A_E          135 GVQSGGSKTDTPS  147 (148)
</pre><pre>T ss_dssp             SECEEEECCCCEE
</pre><pre>T ss_pred             CccCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>PF18946.2 ; Apex ; GpV Apex motif
</pre><pre>Probab=90.27  E-value=0.57  Score=21.08  Aligned_cols=22  Identities=59%  Similarity=0.927  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CEEeeeeCCCcccCCCCCCCcC
</pre><pre>Q MBG6247344.1/9   68 GVQIDDHDHGAVQRGGSWTEGT   89 (90)
</pre><pre>Q Consensus        68 Gv~v~~H~HggV~~Gg~~T~G~   89 (90)
</pre><pre>                      ++.+..|.|.++..|+..+..+
</pre><pre>T Consensus         1 ~~~~~~h~h~~~~~g~~~~~~~   22 (23)
</pre><pre>T A0A109RVX8_9BU    1 GVVLDRHTHGGVQSGGSSTGGP   22 (23)
</pre><pre>T ss_pred             CccceecccCCccCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>6J0M_C Pvc8; assembly, Photorhabdus asymbiotica, PVC, contractile injection system, bacteriophage-like, PROTEIN TRANSPORT; 3.9A {Photorhabdus asymbiotica}
</pre><pre>Probab=84.16  E-value=17  Score=29.60  Aligned_cols=66  Identities=12%  Similarity=0.214  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCcc---EEEEecCCCceEEEc---------eeEEEEEeccEEEeeCCeEe--cCceE----eeeeeecCCeE
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGA---RFRYDAAAGALEAVG---------MKTARVSAETRILLDTPEVQ--KGGEM----HGNIAHSGGAF   64 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA---~~~Yep~~gaL~v~G---------iKTAtV~As~sITldtP~V~--~Gg~m----~G~ithsgG~~   64 (90)
</pre><pre>                      ....+.+++|.   .|.||.+.+.+.+..         -+...++|..++++.++++.  .++.|    .++|+-.|..+
</pre><pre>T Consensus       455 ~~~~~~~~~g~~~~~l~~~~~~~~i~~~~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~i~i~~~~i  534 (538)
</pre><pre>T 6J0M_C          455 REKVLIVKKGEAQQQLVIDGKEKMIRINAGENQIMLQQDKDISLSTKKELTLKAQTMNATMDKSLAMSGKNSVEIKGAKI  534 (538)
</pre><pre>T ss_dssp             CEEEECEECSSCEECCEEETTTTEEEEEBSSSCEEEECSSCCCCBCSSEEEEEESEEEEEESSCEEEEBSSCEEEECSSE
</pre><pre>T ss_pred             ccEEEEEecCCceeEEEEECCCCeEEEEeCCceEEEcCCCcEEEcCCCeEEEeecceEEEeCCceeEeecceEEEeccee
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EECC
</pre><pre>Q MBG6247344.1/9   65 TSNG   68 (90)
</pre><pre>Q Consensus        65 ssNG   68 (90)
</pre><pre>                      .+|.
</pre><pre>T Consensus       535 ~~~~  538 (538)
</pre><pre>T 6J0M_C          535 NLTQ  538 (538)
</pre><pre>T ss_dssp             EEC-
</pre><pre>T ss_pred             ecCC
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF14913.8 ; DPCD ; DPCD protein family
</pre><pre>Probab=80.56  E-value=5  Score=32.45  Aligned_cols=25  Identities=20%  Similarity=0.275  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             cceeEEeecCccEE--EEecCCCceEE
</pre><pre>Q MBG6247344.1/9    2 LQEMVITALDGARF--RYDAAAGALEA   26 (90)
</pre><pre>Q Consensus         2 ~~~~~i~fPDGA~~--~Yep~~gaL~v   26 (90)
</pre><pre>                      ...+|..||||..+  |||.+++.|.+
</pre><pre>T Consensus        18 rrkVHy~f~DG~EmvEEYd~~t~~Ll~   44 (190)
</pre><pre>T W5N166_LEPOC/6   18 KRKVHYLFSDGKEMAEEYDIKTDELIV   44 (190)
</pre><pre>T ss_pred             EEEEEEEcCCCceEEEEEeCCCCeeEE
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>6RBK_C Afp8; Anti-feeding prophage, secretion system, AFP, contractile, VIRUS LIKE PARTICLE, baseplate; 3.4A {Serratia entomophila}
</pre><pre>Probab=77.14  E-value=46  Score=27.25  Aligned_cols=66  Identities=11%  Similarity=0.181  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe----cCceEee--eeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ----KGGEMHG--NIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~----~Gg~m~G--~ithsgG~~ssNG   68 (90)
</pre><pre>                      ......-.++..|+.+.....|++..-+...++|..+|++...++.    +.=+|.+  +++-+++.+.+|.
</pre><pre>T Consensus       458 ~~~~~~d~~~~~~~~~~~~~~~~l~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~i~~~~i~~~~  529 (529)
</pre><pre>T 6RBK_C          458 TQALMIDTEEKTLRLMAGDNTLTLTGEGNLTMSTPNALQLQADTLGLQADSNLSIAGKQQVEITSAKINMKK  529 (529)
</pre><pre>T ss_dssp             BCSEECBTTTTCCEECSSSCCCCCCTTSCCCCCCSSCCCCCCSSCCCCCSSCCCCCCSSCCCCCCSSCCC--
</pre><pre>T ss_pred             ceEEEEeCCCCEEEEEcCCcEEEEcCCCcEEEcCCCeEEEecCcEEEEeCCceEEeccCcEEEEeceeeecC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>PF20000.1 ; fvmX2 ; FtsH ternary system domain X2
</pre><pre>Probab=66.98  E-value=26  Score=29.78  Aligned_cols=31  Identities=26%  Similarity=0.304  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CccEEEEecCCCceEEEceeEEEEEeccEEE
</pre><pre>Q MBG6247344.1/9   11 DGARFRYDAAAGALEAVGMKTARVSAETRIL   41 (90)
</pre><pre>Q Consensus        11 DGA~~~Yep~~gaL~v~GiKTAtV~As~sIT   41 (90)
</pre><pre>                      ||-.+.|||+++.|.|.=--+..|+|..+++
</pre><pre>T Consensus        77 ~gg~v~ydp~t~elei~a~l~~~V~a~~~a~  107 (260)
</pre><pre>T A0A1G7X4D6_9AC   77 PGGRIAYWPDTGELEISVELSTEIEAEGEGV  107 (260)
</pre><pre>T ss_pred             CCeEEEEECCCCEEEEEEEeeceEEEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>PF04773.15 ; FecR ; FecR protein
</pre><pre>Probab=58.17  E-value=94  Score=20.50  Aligned_cols=62  Identities=18%  Similarity=0.272  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCc-----------------eE-EEce----eEEEEEeccEEEeeCCeEe---cCceEeeeee
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGA-----------------LE-AVGM----KTARVSAETRILLDTPEVQ---KGGEMHGNIA   58 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~ga-----------------L~-v~Gi----KTAtV~As~sITldtP~V~---~Gg~m~G~it   58 (90)
</pre><pre>                      ...|.|+||..+.-+|.+.-                 |. ..|-    -.+.+.....+.+.+|...   .|+.+.-.+ 
</pre><pre>T Consensus        10 ~~~l~~~dg~~v~l~~~s~l~i~~~~~~~~~~~~~~~l~l~~G~~~~~~~~~~~~~~~~~i~t~~~~~~~~gt~f~v~v-   88 (104)
</pre><pre>T A8ZRT1_DESOH/7   10 RIELRLPDGSFVRFDSNSVFELKAMAVDSRDNRRDINVNVVFGKIWASVSKWVSGSSRFDVSMKTTTAGVRGTAYRVNV-   88 (104)
</pre><pre>T ss_pred             EEEEEeCCCCEEEeCCCcEEEEEEEEEcCCCCeeEEEEEEEeCeeeeEEEEecCCCCcEEEEcCceEEeEeCeEEEEEE-
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ecCCeEEE
</pre><pre>Q MBG6247344.1/9   59 HSGGAFTS   66 (90)
</pre><pre>Q Consensus        59 hsgG~~ss   66 (90)
</pre><pre>                      ...+....
</pre><pre>T Consensus        89 ~~~~~~~v   96 (104)
</pre><pre>T A8ZRT1_DESOH/7   89 NSDDSAVV   96 (104)
</pre><pre>T ss_pred             CCCCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>5CL2_A Sporulation-control protein spo0M; sporulation, Spo0M, PROTEIN BINDING; 2.3A {Bacillus subtilis (strain 168)}
</pre><pre>Probab=56.33  E-value=64  Score=25.01  Aligned_cols=37  Identities=19%  Similarity=0.356  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             eccEEEeeCCeEecCceEeeeeeecCCe--EEECCEEee
</pre><pre>Q MBG6247344.1/9   36 AETRILLDTPEVQKGGEMHGNIAHSGGA--FTSNGVQID   72 (90)
</pre><pre>Q Consensus        36 As~sITldtP~V~~Gg~m~G~ithsgG~--~ssNGv~v~   72 (90)
</pre><pre>                      |...+.++.+.+..|..++|.|.-.||+  ..++++.+.
</pre><pre>T Consensus        10 ~~v~i~l~~~~~~~Ge~v~G~v~i~~g~~~~~i~~i~v~   48 (252)
</pre><pre>T 5CL2_A           10 AKVDTILEKDAYFPGEEVQGTVHVKGGKIAQDIRYIDLQ   48 (252)
</pre><pre>T ss_dssp             CEEEEEESCSEECTTCEEEEEEEEECCSSCEEEEEEEEE
</pre><pre>T ss_pred             eEEEEEECCCeecCCCEEEEEEEEEcCCCcEEEcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>PF07202.15 ; Tcp10_C ; T-complex protein 10 C-terminus
</pre><pre>Probab=55.29  E-value=47  Score=18.54  Aligned_cols=21  Identities=10%  Similarity=0.112  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             EEeecCccEEEEecCCCceEE
</pre><pre>Q MBG6247344.1/9    6 VITALDGARFRYDAAAGALEA   26 (90)
</pre><pre>Q Consensus         6 ~i~fPDGA~~~Yep~~gaL~v   26 (90)
</pre><pre>                      .+.||||.+.+-.++......
</pre><pre>T Consensus         6 e~~~pdG~v~~~~~~G~~~i~   26 (35)
</pre><pre>T A0A3M7RLS4_BRA    6 ETRLPNGTLIKVDKNGDKVIE   26 (35)
</pre><pre>T ss_pred             EEECCCCCEEEECCCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>d2gpia1 d.354.1.1 (A:1-90) Hypothetical protein Shew3726 {Shewanella loihica [TaxId: 359303]}
</pre><pre>Probab=54.38  E-value=38  Score=22.87  Aligned_cols=21  Identities=14%  Similarity=0.173  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             EeecCccEEEEecCCCceEEEce
</pre><pre>Q MBG6247344.1/9    7 ITALDGARFRYDAAAGALEAVGM   29 (90)
</pre><pre>Q Consensus         7 i~fPDGA~~~Yep~~gaL~v~Gi   29 (90)
</pre><pre>                      |.|||..  +||.+.+++...++
</pre><pre>T Consensus         5 i~F~~~~--~~d~~~~~V~F~a~   25 (90)
</pre><pre>T d2gpia1           5 IIFTEQL--TWDVQLSAIHFTAQ   25 (90)
</pre><pre>T ss_dssp             EEECCCC--EEETTTTEEEEEEE
</pre><pre>T ss_pred             eeeCCCc--eEECCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>PF20003.1 ; fvmX5 ; FtsH ternary system domain X5
</pre><pre>Probab=52.92  E-value=98  Score=24.94  Aligned_cols=39  Identities=18%  Similarity=0.167  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEee
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLD   43 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITld   43 (90)
</pre><pre>                      .+..=.||-.+.|||+++.|.|.=--+..|++..+.+..
</pre><pre>T Consensus        58 ~~r~~~~g~~v~~dp~t~~l~i~a~~~~~v~~e~e~~~~   96 (188)
</pre><pre>T L7U9B5_MYXSD/1   58 AVRVEADGVRVAVDVTTGDVSVTLSDESSVNLVATGEVR   96 (188)
</pre><pre>T ss_pred             EEEEeeCCEEEEEECCCCEEEEEEeccceEEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>d2psba1 c.153.1.1 (A:13-308) Uncharacterized protein YerB {Bacillus subtilis [TaxId: 1423]}
</pre><pre>Probab=52.77  E-value=2.4e+02  Score=23.54  Aligned_cols=70  Identities=14%  Similarity=0.191  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCc-----cEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe----------cCceEeeeeeecC-CeEEE
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDG-----ARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ----------KGGEMHGNIAHSG-GAFTS   66 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDG-----A~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~----------~Gg~m~G~ithsg-G~~ss   66 (90)
</pre><pre>                      .++.|.|+++     ..++||++++.-.-.==-...+.+...-.|...+|.          ..|.+.=++.-+| +-+-+
</pre><pre>T Consensus       170 ~~v~i~~~~~~~~~~~~~~Yd~~~~~Y~R~~~G~~~~D~~tg~ql~~~NVIV~~~~~~~~d~~g~~~~~~~g~G~a~if~  249 (296)
</pre><pre>T d2psba1         170 YNVRVDYGTNNVTNLVEYNYDKKAEFYTRSSDGVITTDRETGKPVAMQNIFIVEASHHIIDQDGRRDIDLESGGKGLLFQ  249 (296)
</pre><pre>T ss_dssp             CEEEEECCCSSCCCEEEEEEETTTTEEEEEETTEECEETTTTEECCBSEEEEEECEEEECSSSCCEEEECSSCEEEEEEE
</pre><pre>T ss_pred             eEEEEEccCCCCceeEEEEEECCCCEEEEEeCCeeeEeCCCCCeeeeEEEEEEEceEEEeCCCCcEEEEeecceEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCEEee
</pre><pre>Q MBG6247344.1/9   67 NGVQID   72 (90)
</pre><pre>Q Consensus        67 NGv~v~   72 (90)
</pre><pre>                      ||..+.
</pre><pre>T Consensus       250 dGk~~~  255 (296)
</pre><pre>T d2psba1         250 HGNVIE  255 (296)
</pre><pre>T ss_dssp             TTEEEE
</pre><pre>T ss_pred             CCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>PF14481.8 ; Fimbrial_PilY2 ; Type 4 fimbrial biogenesis protein PilY2
</pre><pre>Probab=50.57  E-value=1e+02  Score=20.44  Aligned_cols=61  Identities=8%  Similarity=-0.026  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             ccEEEEecCCCceEEEceeEEEEEeccEE--EeeCCeEecCceEeeeeeecCCeEEECCEEeee
</pre><pre>Q MBG6247344.1/9   12 GARFRYDAAAGALEAVGMKTARVSAETRI--LLDTPEVQKGGEMHGNIAHSGGAFTSNGVQIDD   73 (90)
</pre><pre>Q Consensus        12 GA~~~Yep~~gaL~v~GiKTAtV~As~sI--TldtP~V~~Gg~m~G~ithsgG~~ssNGv~v~~   73 (90)
</pre><pre>                      |.+.++|++.+.|++.| +...+.+...+  ......++.|..+.-++.-.++......+.+..
</pre><pre>T Consensus        29 G~V~~vd~~~~~i~I~g-~~~~~~~~~~~~~~~~~~~l~~G~~V~v~~~~~~~~~~~~~I~~~~   91 (99)
</pre><pre>T W9T792_9PSED/2   29 GTISDVQLGNNLIVVEK-TTYSLPNKTALNGTPAILQLQPGYLIGFSGTEASPHSIIDSIYLYP   91 (99)
</pre><pre>T ss_pred             EEEEEEECCCCEEEECC-EEEeCCCCceecCCChHHhCCCCCEEEEEEecCCCceeEEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>5LHX_A Serine/threonine-protein kinase PLK4; Polo box domain, Centriole, transferase, structural protein; 1.53A {Drosophila melanogaster} SCOP: d.223.1.0
</pre><pre>Probab=50.01  E-value=69  Score=22.19  Aligned_cols=25  Identities=16%  Similarity=0.207  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecC--CCceE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAA--AGALE   25 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~--~gaL~   25 (90)
</pre><pre>                      |-.++.+.|-||.++..++.  ...+.
</pre><pre>T Consensus        24 s~G~i~V~FnDGs~l~v~~~~~~~~v~   50 (93)
</pre><pre>T 5LHX_A           24 SHGVVQVQFYDGSVVSVIPSMQGGGIT   50 (93)
</pre><pre>T ss_dssp             TTSCEEEEETTSCEEEECCGGGTCCEE
</pre><pre>T ss_pred             CCCeEEEEECCCCEEEEeCCCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>6M9Y_B Fluorescent protein lanFP6A; Fluorescent Protein Branchiostoma floridae Gly-Tyr-Ala tripeptide hydrolysis, FLUORESCENT PROTEIN; 1.35A {Branchiostoma floridae}
</pre><pre>Probab=47.81  E-value=61  Score=25.77  Aligned_cols=47  Identities=23%  Similarity=0.436  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             eecCc------cEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    8 TALDG------ARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         8 ~fPDG------A~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .||||      ..+.||           .-+++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        19 sfP~G~~yt~eRt~~FE-----------Dgg~~t~~~~it~------e~~~~~~~~~~~G~~Fp~dGpvm   71 (172)
</pre><pre>T 6M9Y_B           19 AMQDESGYQVHRTLQYE-----------DGAFVTANLRYTY------EGSHIKGEFQVIGTGFPPDGPVM   71 (172)
</pre><pre>T ss_dssp             HHHTTCCEEEEEEEEET-----------TSCEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             hcCCCCCcEEEEEEEEC-----------CCCEEEEEEEEEE------eCCEEEEEEEEEeccCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>d1mbya_ d.223.1.1 (A:) Serine/threonine-protein kinase Sak C-terminal domain {Mouse (Mus musculus) [TaxId: 10090]}
</pre><pre>Probab=45.67  E-value=99  Score=19.20  Aligned_cols=25  Identities=12%  Similarity=0.232  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALE   25 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~   25 (90)
</pre><pre>                      |-..+.+.|.||..+..++....+.
</pre><pre>T Consensus        14 s~g~~~v~F~D~t~l~~~~~~~~v~   38 (75)
</pre><pre>T d1mbya_          14 TSGAVWVQFNDGSQLVMQAGVSSIS   38 (75)
</pre><pre>T ss_dssp             SEEEEEEEETTSCEEEEEEEEEEEE
</pre><pre>T ss_pred             CCCeEEEEECCCCEEEEECCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>1MBY_A serine/threonine kinase; POLO DOMAIN, STRAND EXCHANGE, TRANSFERASE; 2.0A {Mus musculus} SCOP: d.223.1.1
</pre><pre>Probab=44.45  E-value=84  Score=20.70  Aligned_cols=22  Identities=14%  Similarity=0.288  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALE   25 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~   25 (90)
</pre><pre>                      ++.+.|.||..+..++....+.
</pre><pre>T Consensus        24 ~i~v~F~Dgs~l~~~~~~~~~~   45 (88)
</pre><pre>T 1MBY_A           24 AVWVQFNDGSQLVMQAGVSSIS   45 (88)
</pre><pre>T ss_dssp             EEEEEETTSCEEEEEEEEEEEE
</pre><pre>T ss_pred             cEEEEECCCCEEEEECCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>PF07273.14 ; DUF1439 ; Protein of unknown function (DUF1439)
</pre><pre>Probab=41.96  E-value=1.1e+02  Score=22.25  Aligned_cols=23  Identities=22%  Similarity=0.234  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             cEEEEecCCCceEEEceeEEEEE
</pre><pre>Q MBG6247344.1/9   13 ARFRYDAAAGALEAVGMKTARVS   35 (90)
</pre><pre>Q Consensus        13 A~~~Yep~~gaL~v~GiKTAtV~   35 (90)
</pre><pre>                      +.++|||++++|.+..++--.++
</pre><pre>T Consensus        75 g~~~Yd~~~~~iyL~~~~i~~~~   97 (154)
</pre><pre>T Q6LIP7_PHOPR/2   75 AIPFYDKDEGAIFLKDLNVESLN   97 (154)
</pre><pre>T ss_pred             EeEEEECCCCEEEEEEeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF00659.20 ; POLO_box ; POLO box duplicated region
</pre><pre>Probab=40.68  E-value=1.2e+02  Score=17.88  Aligned_cols=22  Identities=14%  Similarity=0.160  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALE   25 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~   25 (90)
</pre><pre>                      .+.+.|.|+..+..++....+.
</pre><pre>T Consensus        13 ~~~v~f~D~t~~~~~~~~~~~~   34 (65)
</pre><pre>T D2VZP4_NAEGR/6   13 CVGAVYNDSSKMTYMPATEQIN   34 (65)
</pre><pre>T ss_pred             eEEEEeeCCCEEEEeCCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>PF12134.10 ; PRP8_domainIV ; PRP8 domain IV core
</pre><pre>Probab=40.62  E-value=59  Score=26.97  Aligned_cols=35  Identities=20%  Similarity=0.147  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             eeEEeec--------CccEEEEecCCCceEEEceeEEEEEeccE
</pre><pre>Q MBG6247344.1/9    4 EMVITAL--------DGARFRYDAAAGALEAVGMKTARVSAETR   39 (90)
</pre><pre>Q Consensus         4 ~~~i~fP--------DGA~~~Yep~~gaL~v~GiKTAtV~As~s   39 (90)
</pre><pre>                      .++-+||        .|+.+.|+|.+|.|-+. |--+.+.|+++
</pre><pre>T Consensus        30 ~i~~~~~G~~~~~ping~i~i~~p~tg~l~lk-vih~sv~~gqK   72 (230)
</pre><pre>T Q6CCC5_YARLI/1   30 AAHTTFEGNFTNKPLNGAVFILNPKTGQLFLK-VVHTSTWAGQK   72 (230)
</pre><pre>T ss_pred             EeeEcCCCceecceeCcEEEEeccccCcEEEe-eeccchHhhhc
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF10989.10 ; DUF2808 ; Protein of unknown function (DUF2808)
</pre><pre>Probab=40.59  E-value=2.3e+02  Score=20.89  Aligned_cols=51  Identities=10%  Similarity=0.182  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccE-EEEecCCCceEE---EceeEEEEEeccEEEeeCCeEe--------cCceEe
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGAR-FRYDAAAGALEA---VGMKTARVSAETRILLDTPEVQ--------KGGEMH   54 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~-~~Yep~~gaL~v---~GiKTAtV~As~sITldtP~V~--------~Gg~m~   54 (90)
</pre><pre>                      +|+++.|.+||+.. +.|+++.-.+..   .| |...+++. ++. +.-.++        -|.+++
</pre><pre>T Consensus        40 ~l~~l~i~~p~~~~~~~~~~~~v~v~~~~~~g-~~i~~~~~-~~~-~~~~i~v~f~~PV~pg~~l~  102 (142)
</pre><pre>T B0C9L7_ACAM1/3   40 PLQRVMFNQQEGVDSISFNLKRTQAYRNKQKK-QLLPIAAA-STD-EKGAISIFFDPPISPGETVT  102 (142)
</pre><pre>T ss_pred             CceEEEEEeCCCcceeeecccccEEEECCCCC-ccceeeee-EeC-CCCeEEEEeCCCCCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>PF13619.8 ; KTSC ; KTSC domain
</pre><pre>Probab=40.37  E-value=78  Score=18.99  Aligned_cols=15  Identities=33%  Similarity=0.463  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             cEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9   13 ARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus        13 A~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      ..+.||++++.|.|.
</pre><pre>T Consensus         6 ~~v~Yd~~~~~L~V~   20 (58)
</pre><pre>T I4F247_9ACTN/7    6 ATVGYDEEQRIMEIV   20 (58)
</pre><pre>T ss_pred             eEEEEECCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>PF12992.9 ; DUF3876 ; Domain of unknown function, B. Theta Gene description (DUF3876)
</pre><pre>Probab=38.63  E-value=1e+02  Score=21.14  Aligned_cols=24  Identities=21%  Similarity=0.281  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCcc-EEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    5 MVITALDGA-RFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA-~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      +.+....|+ .|.||+++..|++++
</pre><pre>T Consensus        63 l~i~~g~g~~~i~Yd~~~D~L~ls~   87 (93)
</pre><pre>T Q8A5A2_BACTN/4   63 YFIATASKRLSVDYDPAKDVLSISS   87 (93)
</pre><pre>T ss_pred             EEEEcCCcEEEEEeeCcCCEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>2GW4_D Kaede; BETA BARREL, LUMINESCENT PROTEIN; HET: RC7, NFA; 1.6A {Trachyphyllia geoffroyi}
</pre><pre>Probab=37.98  E-value=1.6e+02  Score=23.37  Aligned_cols=44  Identities=27%  Similarity=0.354  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||+++.                   |...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        26 ~eRt~~FEDgG~~t-------------------~~~~isl------e~~~~~~~v~~~G~nFp~dGpvm   69 (162)
</pre><pre>T 2GW4_D           26 WERSLMFEDGGVCI-------------------ATNDITL------KGDTFFNKVRFDGVNFPPNGPVM   69 (162)
</pre><pre>T ss_dssp             EEEEEEETTSCEEE-------------------EEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCCcEEE-------------------EEEEEEE------eCCEEEEEEEEEeeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF10703.11 ; MoaF ; MoaF N-terminal domain
</pre><pre>Probab=37.43  E-value=1.7e+02  Score=20.27  Aligned_cols=45  Identities=16%  Similarity=0.032  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEec-CCCceEEEceeEEEEEeccEEEeeCCeE
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDA-AAGALEAVGMKTARVSAETRILLDTPEV   47 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep-~~gaL~v~GiKTAtV~As~sITldtP~V   47 (90)
</pre><pre>                      +.+.+.|+||..+++.- +.+.|+...+.-..--++.+.....=+|
</pre><pre>T Consensus        27 ~~~~~~~~~G~~~~~~f~~~~~l~w~~~~g~~~g~~~~~~y~~~~v   72 (110)
</pre><pre>T C7YXT9_NECH7/1   27 RNIEHTFDNGYKISHLFGDAGNLTWKILEGEDKGLTGEAEYEAYEV   72 (110)
</pre><pre>T ss_pred             cEEEEEeCCCcEEEEEEccCCeEEEEEecCCCCCCceeEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>PF10049.11 ; DUF2283 ; Protein of unknown function (DUF2283)
</pre><pre>Probab=36.25  E-value=71  Score=18.99  Aligned_cols=14  Identities=36%  Similarity=0.387  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      .+.|||+...|.+.
</pre><pre>T Consensus         1 ~i~YD~~~D~l~i~   14 (48)
</pre><pre>T B0JGR7_MICAN/2    1 KIDYDRAANAAYIR   14 (48)
</pre><pre>T ss_pred             CeEEeCCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>4MPZ_A Sas-4; beta-spine, centrosome biogenesis, STRUCTURAL PROTEIN; 2.701A {Drosophila melanogaster}
</pre><pre>Probab=35.95  E-value=2.4e+02  Score=21.67  Aligned_cols=43  Identities=9%  Similarity=-0.012  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCC----ceEEEceeEEEEEeccEEEeeCCe
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAG----ALEAVGMKTARVSAETRILLDTPE   46 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~g----aL~v~GiKTAtV~As~sITldtP~   46 (90)
</pre><pre>                      ...+.||||....+.|+..    ...-.--...+.....+..+..|.
</pre><pre>T Consensus       111 ~~~i~~pdG~~~~~~~dg~~~~~~~~~~dGt~~~~~~dG~~~~~~pd  157 (209)
</pre><pre>T 4MPZ_A          111 TVEIHFPNNSIKIVDPSDTEKLEEWRYADGTHLVQLRNGDKILNLPN  157 (209)
</pre><pre>T ss_dssp             CEEEECSSCEEEECC----CEEEEEECTTCCEEEEETTSCEEEECTT
</pre><pre>T ss_pred             cEEEEeCCCcEEEECCCCCceeEEEEcCCCCEEEEEeCCCEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF10807.10 ; DUF2541 ; Protein of unknown function (DUF2541)
</pre><pre>Probab=35.82  E-value=2.1e+02  Score=20.95  Aligned_cols=34  Identities=15%  Similarity=0.253  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             cceeEEeecCccEEEEec--------CCCceEEEc----eeEEEEE
</pre><pre>Q MBG6247344.1/9    2 LQEMVITALDGARFRYDA--------AAGALEAVG----MKTARVS   35 (90)
</pre><pre>Q Consensus         2 ~~~~~i~fPDGA~~~Yep--------~~gaL~v~G----iKTAtV~   35 (90)
</pre><pre>                      |..+.+.|.||+..+++.        .+..+.+.|    |+.+.+.
</pre><pre>T Consensus        70 i~~v~V~f~nG~~~~~~~~~~l~~G~~s~~i~l~g~~r~I~~I~~~  115 (132)
</pre><pre>T B8HV24_CYAP4/6   70 LKQVTLVYGNGEVDRLPIVKRLNQGNETGWIAVQGNKRCISRITIV  115 (132)
</pre><pre>T ss_pred             EEEEEEEECCCCeEEEeccceeCCCCccccEEcCCCccceEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>PF14220.8 ; DUF4329 ; Domain of unknown function (DUF4329)
</pre><pre>Probab=35.54  E-value=83  Score=21.87  Aligned_cols=21  Identities=10%  Similarity=0.031  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCceE
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGALE   25 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL~   25 (90)
</pre><pre>                      ..+.-|||+.+.|+|..+...
</pre><pre>T Consensus       116 ~Yl~tp~G~l~ky~~~~~~~~  136 (136)
</pre><pre>T B8F9G4_DESAL/1  116 GYLVCFKPSMEEIDPRTSWLK  136 (136)
</pre><pre>T ss_pred             eEEEecCCceEEeecCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>6YWS_h Mitochondrial ribosomal protein L44; Neurospora crassa, translating Mitoribosomes, tRNA, mRNA, mL108, TRANSLATION; HET: NAD, MG, K, SPM;{Neurospora crassa OR74A}
</pre><pre>Probab=34.84  E-value=1.1e+02  Score=20.89  Aligned_cols=18  Identities=17%  Similarity=0.287  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCC
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAA   21 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~   21 (90)
</pre><pre>                      .+.+.|.||....++++.
</pre><pre>T Consensus        55 ~i~v~f~dg~~~~i~~~~   72 (98)
</pre><pre>T 6YWS_h           55 LLYVKFKDGKEMNLDVEN   72 (98)
</pre><pre>T ss_dssp             EEEEEETTSCEEEEETTT
</pre><pre>T ss_pred             EEEEEEeCCcEEEEEccc
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>PF16463.7 ; Phage_TTP_13 ; Phage tail tube protein family
</pre><pre>Probab=34.32  E-value=1.9e+02  Score=19.88  Aligned_cols=47  Identities=2%  Similarity=-0.133  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEE---ceeEEEEEeccEEEeeCCeEe
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAV---GMKTARVSAETRILLDTPEVQ   48 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~---GiKTAtV~As~sITldtP~V~   48 (90)
</pre><pre>                      ....+.|.||||..+.|+-.-..+...   +=..-.++++-++.++ +.+.
</pre><pre>T Consensus        85 ~~~~~~i~~~~g~~~~~~~~v~~~~~~~~~~~~~~~~~~t~~~~~~-~~~~  134 (137)
</pre><pre>T I2B9D9_SHIBC/1   85 QTVQFYVELPNQRTVLMLLALSGWQMNDISAPASEAIQITVQGKQN-KIEW  134 (137)
</pre><pre>T ss_pred             CeEEEEEEeCCCCEEEEEEEecCcccCCCCCCccccEEEEEEEeec-ceEE
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>5EXC_I Green fluorescent protein; Photoswitchable fluorescent proteins, chromophore, beta-barrel, biomarker, fluorescent protein; HET: MG, GOL, NLW, RC7; 2.14A {Dendronephthya sp. SSAL-2002}
</pre><pre>Probab=34.23  E-value=2e+02  Score=23.10  Aligned_cols=44  Identities=25%  Similarity=0.297  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.++|.||+++.                   +...|++      +++.+.++++-.|-.|.-||-+.
</pre><pre>T Consensus        26 ~eRt~~FEDgG~~t-------------------~~~~isl------e~~~~~~~v~~~G~nFp~dGpvm   69 (170)
</pre><pre>T 5EXC_I           26 WERTMTYEDKGICT-------------------IRSDISL------EGDCFFQNVRFNGMNFPPNGPVM   69 (170)
</pre><pre>T ss_dssp             EEEEEEETTSCEEE-------------------EEEEEEC------STTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCCcEEE-------------------EEEEEEE------eCCEEEEEEEEeecCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>5EXC_M Green fluorescent protein; Photoswitchable fluorescent proteins, chromophore, beta-barrel, biomarker, fluorescent protein; HET: MG, RC7, NLW, GOL; 2.14A {Dendronephthya sp. SSAL-2002}
</pre><pre>Probab=34.23  E-value=2e+02  Score=23.10  Aligned_cols=44  Identities=25%  Similarity=0.297  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.++|.||+++.                   +...|++      +++.+.++++-.|-.|.-||-+.
</pre><pre>T Consensus        26 ~eRt~~FEDgG~~t-------------------~~~~isl------e~~~~~~~v~~~G~nFp~dGpvm   69 (170)
</pre><pre>T 5EXC_M           26 WERTMTYEDKGICT-------------------IRSDISL------EGDCFFQNVRFNGMNFPPNGPVM   69 (170)
</pre><pre>T ss_dssp             EEEEEEETTCCEEE-------------------EEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCCcEEE-------------------EEEEEEE------eCCEEEEEEEEeecCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>PF06832.14 ; BiPBP_C ; Penicillin-Binding Protein C-terminus Family
</pre><pre>Probab=34.21  E-value=1.4e+02  Score=18.90  Aligned_cols=24  Identities=13%  Similarity=-0.010  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             eeEEeec-CccEEEEecC--CCceEEE
</pre><pre>Q MBG6247344.1/9    4 EMVITAL-DGARFRYDAA--AGALEAV   27 (90)
</pre><pre>Q Consensus         4 ~~~i~fP-DGA~~~Yep~--~gaL~v~   27 (90)
</pre><pre>                      .+.|.+| ||++|..++.  ...|.+.
</pre><pre>T Consensus        11 ~~~I~~P~~g~~~~~~~~~~~~~i~~~   37 (87)
</pre><pre>T Q316S6_DESAG/6   11 PPRILHPQQGATVESLGQQAGEGIALR   37 (87)
</pre><pre>T ss_pred             CCEEEcCCCCCEEEeCCCCCCcceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>6OVM_R Siderophore-interacting protein; signaling protein, transcriptional regulation, periplasmic protein, iron transport regulation; HET: MSE, TLA; 1.6A {Pseudomonas capeferrum}
</pre><pre>Probab=34.17  E-value=3.3e+02  Score=20.38  Aligned_cols=44  Identities=16%  Similarity=0.306  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEe-ccEEEeeCCe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSA-ETRILLDTPE   46 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~A-s~sITldtP~   46 (90)
</pre><pre>                      ....+.||||..+.-++.+.--.-..-....++- +.++-++.+.
</pre><pre>T Consensus        17 ~~~~i~l~dgs~v~L~~~S~l~~~~~~~~~~~~l~~G~~~~~v~~   61 (219)
</pre><pre>T 6OVM_R           17 EQRRLTLADGTQVQLNTDSALNVAFDQQARRLRLVRGEMLITRPA   61 (219)
</pre><pre>T ss_dssp             CCEEEECTTSCEEEECTTCEEEEEECSSCEEEEEEESEEEEESCC
</pre><pre>T ss_pred             eeEEEECCCCCEEEECCCCcEEEEecCCceEEEEEeceEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>d2auwa2 d.331.1.1 (A:4-87) Hypothetical protein NE0471 N-terminal domain {Nitrosomonas europaea [TaxId: 915]}
</pre><pre>Probab=33.73  E-value=1.1e+02  Score=20.40  Aligned_cols=15  Identities=20%  Similarity=0.180  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEe
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYD   18 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Ye   18 (90)
</pre><pre>                      .|.|.|.||.+..||
</pre><pre>T Consensus        16 ~L~i~f~dG~~~~~D   30 (84)
</pre><pre>T d2auwa2          16 RLRTTWSTGEVLEVD   30 (84)
</pre><pre>T ss_dssp             EEEEEETTSCEEEEE
</pre><pre>T ss_pred             EEEEEEcCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>2K4Q_A Major tail protein V; gpV, Bacteriophage Lambda, Major tail protein, VIRAL PROTEIN; NMR {Enterobacteria phage lambda}
</pre><pre>Probab=33.56  E-value=1.6e+02  Score=20.79  Aligned_cols=47  Identities=4%  Similarity=-0.021  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEV   47 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V   47 (90)
</pre><pre>                      ....+.|.+|||..++|+..-..+...+=+...++++-++.++-+-+
</pre><pre>T Consensus       107 ~~~~~~i~~~~g~~~~~~~~vt~~~~~~~~~~~~~~s~tl~~~G~~~  153 (156)
</pre><pre>T 2K4Q_A          107 DTRAYKIRFPNGTVDVFRGWVSSIGKAVTAKEVITRTVKVTNVGRPS  153 (156)
</pre><pre>T ss_dssp             CCEEEEEECTTSCEEEEEECBCCCCCCEEETTEEEEEEEECCSCSCS
</pre><pre>T ss_pred             CceEEEEEcCCCcEEEEEEEEeeeCCccCCCcceEEEEEEEEcccce
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>6NU2_k 39S ribosomal protein L53, mitochondrial; mitochondrial ribosome recycling Factor, mtRRF, 55S, RIBOSOME; HET: MG; 3.9A {Homo sapiens}
</pre><pre>Probab=33.36  E-value=1.4e+02  Score=19.13  Aligned_cols=18  Identities=22%  Similarity=0.135  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCC
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAA   21 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~   21 (90)
</pre><pre>                      .+.+.|-||....++++.
</pre><pre>T Consensus        51 ~i~~~y~ng~~~~i~~~~   68 (84)
</pre><pre>T 6NU2_k           51 CVDVLFGDGHRLIMRGAH   68 (84)
</pre><pre>T ss_dssp             CEEECCTTSCCEEECCTT
</pre><pre>T ss_pred             EEEEEEcCCCEEEEEcCC
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>PF08813.13 ; Phage_tail_3 ; Phage tail tube protein, TTP
</pre><pre>Probab=33.01  E-value=1.4e+02  Score=21.65  Aligned_cols=45  Identities=18%  Similarity=0.099  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCce-EEEceeEEEEEeccEEEeeCCeEe
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGAL-EAVGMKTARVSAETRILLDTPEVQ   48 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL-~v~GiKTAtV~As~sITldtP~V~   48 (90)
</pre><pre>                      .+.|.||||..+.|.-.-..+ ...+-....++++-++++..+-+.
</pre><pre>T Consensus       113 ~~~i~~~~g~~~~~~g~v~~~~~~~~~~~~~~~~t~ti~~~g~~~~  158 (161)
</pre><pre>T Q6LHS9_PHOPR/5  113 VVRATLPNGATLYYHAYVSVNETPSMTKDQVMAIAVTISLIVRPTR  158 (161)
</pre><pre>T ss_pred             EEEEEeCCCCeEEEEEEEEecCCCCCCCCceeEEEEEEEEecCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>2ZMU_A Fluorescent protein; GFP-LIKE PROTEIN, LUMINESCENT PROTEIN, Structural Genomics, RIKEN Structural Genomics/Proteomics Initiative, RSGI, NPPSFA, National Project on Protein; HET: CFY; 1.65A {Fungia concinna} SCOP: d.22.1.0
</pre><pre>Probab=32.71  E-value=1.7e+02  Score=24.33  Aligned_cols=44  Identities=23%  Similarity=0.331  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||+                   +++|...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        89 ~eRt~~FEDgg-------------------~~t~~~~is~------e~~~i~~~~~~~G~~Fp~dGpvm  132 (223)
</pre><pre>T 2ZMU_A           89 WERSLEFEDGG-------------------SASVSAHISL------RGNTFYHKSKFTGVNFPADGPIM  132 (223)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCCc-------------------EEEEEEEEEE------eCCEEEEEEEEEEeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>PF19720.1 ; DUF6214 ; Family of unknown function (DUF6214)
</pre><pre>Probab=32.50  E-value=82  Score=25.83  Aligned_cols=16  Identities=25%  Similarity=0.339  Sum_probs=0.0  Template_Neff=3.800
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFR   16 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~   16 (90)
</pre><pre>                      +...+.+.|+|||.++
</pre><pre>T Consensus        17 ~Wf~VrL~f~DGA~vd   32 (173)
</pre><pre>T A0A2A3HAA7_9AC   17 SWFDVRLVFADGAQVE   32 (173)
</pre><pre>T ss_pred             CceEEEEecCCCCeeE
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>d3qx1a1 d.15.1.2 (A:571-650) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=32.37  E-value=1.1e+02  Score=19.02  Aligned_cols=16  Identities=13%  Similarity=0.268  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFR   16 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~   16 (90)
</pre><pre>                      ....+.|.||||.++.
</pre><pre>T Consensus         2 ~~~~i~vr~p~g~~~~   17 (80)
</pre><pre>T d3qx1a1           2 PVSKLRIRTPSGEFLE   17 (80)
</pre><pre>T ss_dssp             CEEEEEEECTTSCEEE
</pre><pre>T ss_pred             CcEEEEEECCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>PF19686.1 ; DUF6188 ; Family of unknown function (DUF6188)
</pre><pre>Probab=31.67  E-value=1.3e+02  Score=20.27  Aligned_cols=18  Identities=17%  Similarity=0.189  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCC
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAA   21 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~   21 (90)
</pre><pre>                      .+.|.|.||++++..|+.
</pre><pre>T Consensus        71 ~L~i~F~~g~~L~v~~~~   88 (115)
</pre><pre>T A0A3D9T2X0_9NO   71 ALLLEFDDGSSVSAVPRA   88 (115)
</pre><pre>T ss_pred             CEEEEECCCCEEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>3RWA_E Fluorescent protein FP480; GFP-like Fluoresent proteins, mkate, circular permutated, FLUORESCENT PROTEIN; HET: NRQ; 1.67A {Entacmaea quadricolor}
</pre><pre>Probab=31.63  E-value=2.1e+02  Score=24.25  Aligned_cols=44  Identities=32%  Similarity=0.333  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||+                   .++|...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus       158 ~eRt~~FEDGg-------------------~~t~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  201 (233)
</pre><pre>T 3RWA_E          158 WERVTTYEDGG-------------------VLTATQDTSL------QDGCLIYNVKIRGVNFPSNGPVM  201 (233)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEccCC-------------------EEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>3RWA_G Fluorescent protein FP480; GFP-like Fluoresent proteins, mkate, circular permutated, FLUORESCENT PROTEIN; HET: NRQ; 1.67A {Entacmaea quadricolor}
</pre><pre>Probab=31.63  E-value=2.1e+02  Score=24.25  Aligned_cols=44  Identities=32%  Similarity=0.333  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||+                   .++|...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus       158 ~eRt~~FEDGg-------------------~~t~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  201 (233)
</pre><pre>T 3RWA_G          158 WERVTTYEDGG-------------------VLTATQDTSL------QDGCLIYNVKIRGVNFPSNGPVM  201 (233)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEccCC-------------------EEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>4RGI_A Uncharacterized protein; Chicago Center for Functional Annotation, Structural Genomics, PSI-Biology, Midwest Center for Structural Genomics, MCSG, alpha-beta sandwich; HET: GOL, MSE; 1.732A {Yersinia pestis}
</pre><pre>Probab=31.61  E-value=1.2e+02  Score=19.37  Aligned_cols=15  Identities=13%  Similarity=0.363  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEE
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRY   17 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Y   17 (90)
</pre><pre>                      ..|.|.|.||...+|
</pre><pre>T Consensus        24 ~~L~V~f~~g~~y~Y   38 (73)
</pre><pre>T 4RGI_A           24 RMLEIQFREQGTYQY   38 (73)
</pre><pre>T ss_dssp             TEEEEEETTTEEEEE
</pre><pre>T ss_pred             CEEEEEECCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>PF01353.24 ; GFP ; Green fluorescent protein
</pre><pre>Probab=31.42  E-value=2e+02  Score=23.58  Aligned_cols=44  Identities=18%  Similarity=0.376  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .+-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        82 ~eRt~~feDg-------------------g~~~~~~~is~------~~~~l~~~~~~~G~~fp~dgpvm  125 (214)
</pre><pre>T C3YKQ3_BRAFL/5   82 VYRVFDFEDG-------------------GKLTTEFKYSY------EGSHIKADMKLMGSGFPDDGPVM  125 (214)
</pre><pre>T ss_pred             EEEEEEeccC-------------------CEEEEEEEEEE------eCCEEEEEEEEEecCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>d5lhxa_ d.223.1.0 (A:) automated matches {Fruit fly (Drosophila melanogaster) [TaxId: 7227]}
</pre><pre>Probab=31.38  E-value=2.2e+02  Score=18.72  Aligned_cols=25  Identities=16%  Similarity=0.207  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecC--CCceE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAA--AGALE   25 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~--~gaL~   25 (90)
</pre><pre>                      |=..+.+.|.||.++.-+|.  ...+.
</pre><pre>T Consensus        19 s~g~v~v~F~Dgs~l~l~~~~~~~~v~   45 (87)
</pre><pre>T d5lhxa_          19 SHGVVQVQFYDGSVVSVIPSMQGGGIT   45 (87)
</pre><pre>T ss_dssp             TTSCEEEEETTSCEEEECCGGGTCCEE
</pre><pre>T ss_pred             CCCeEEEEECCCCEEEEecCCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>d1i42a_ d.15.1.2 (A:) p47 {Norway rat (Rattus norvegicus) [TaxId: 10116]}
</pre><pre>Probab=31.20  E-value=1.3e+02  Score=19.75  Aligned_cols=15  Identities=27%  Similarity=0.414  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEe
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYD   18 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Ye   18 (90)
</pre><pre>                      .+.|.||||.++...
</pre><pre>T Consensus        16 ~i~iR~p~G~~~~~~   30 (89)
</pre><pre>T d1i42a_          16 NIQIRLADGGRLVQK   30 (89)
</pre><pre>T ss_dssp             EEEEEETTTCCEEEE
</pre><pre>T ss_pred             EEEEECCCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>2A50_D GFP-like non-fluorescent chromoprotein FP595 chain 2; asCP, fluorescent protein, chromoprotein, photochromic protein, reversible photoswitch, LUMINESCENT PROTEIN; HET: NRQ; 1.3A {Anemonia sulcata}
</pre><pre>Probab=31.07  E-value=2.2e+02  Score=22.76  Aligned_cols=44  Identities=23%  Similarity=0.266  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.++|.||                   ++++|...|++      +++.+.++++-.|-.|.-||-+.
</pre><pre>T Consensus        26 ~eRt~~FEDG-------------------G~~t~~~~is~------e~~~~~~~v~~~G~nFp~dGpVm   69 (168)
</pre><pre>T 2A50_D           26 WERTTTYEDG-------------------GFLTAHQDTSL------DGDCLVYKVKILGNNFPADGPVM   69 (168)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEecCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>6DEJ_B GFP-like non-fluorescent chromoprotein; HcRed, red, far-red, FLUORESCENT PROTEIN; HET: 12P, CRU, SO4; 1.6279A {Heteractis crispa} SCOP: d.22.1.1
</pre><pre>Probab=30.90  E-value=1.9e+02  Score=24.31  Aligned_cols=44  Identities=23%  Similarity=0.283  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        98 ~eRt~~FEDg-------------------g~~t~~~~is~------e~~~~~~~~~~~G~~Fp~dgpvm  141 (235)
</pre><pre>T 6DEJ_B           98 WERTTTYEDG-------------------GILTAHQDTSL------EGNCLIYKVKVHGTNFPADGPVM  141 (235)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------ECCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>PF07070.13 ; Spo0M ; SpoOM protein
</pre><pre>Probab=30.82  E-value=3.3e+02  Score=20.25  Aligned_cols=37  Identities=27%  Similarity=0.395  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             eccEEEeeCCeEecCceEeeeeeecCC--eEEECCEEee
</pre><pre>Q MBG6247344.1/9   36 AETRILLDTPEVQKGGEMHGNIAHSGG--AFTSNGVQID   72 (90)
</pre><pre>Q Consensus        36 As~sITldtP~V~~Gg~m~G~ithsgG--~~ssNGv~v~   72 (90)
</pre><pre>                      |.-.+.++.+.+..|+.++|.|.-.+|  ...++.+.+.
</pre><pre>T Consensus        13 ~~v~~~l~~~~~~~Ge~v~G~v~i~~g~~~~~i~~i~l~   51 (208)
</pre><pre>T E3E849_PAEPS/4   13 AKVDTLVDQTVYVPGEELSGIIRIKGGKIDQEIGKIDIE   51 (208)
</pre><pre>T ss_pred             eEEEEEECCCeEeCCCEEEEEEEEEcCCCcEEEeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>2Y3M_B PROTEIN TRANSPORT PROTEIN HOFQ; TRANSPORT PROTEIN, SECRETIN, DNA UPTAKE, COMPETENCE; HET: GOL; 2.3A {AGGREGATIBACTER ACTINOMYCETEMCOMITANS}
</pre><pre>Probab=30.47  E-value=1.4e+02  Score=20.29  Aligned_cols=20  Identities=10%  Similarity=0.097  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      |+++..+.+++.++.|.|.|
</pre><pre>T Consensus       132 ~~~~~~v~~~~~~n~l~v~~  151 (175)
</pre><pre>T 2Y3M_B          132 LSPNGSITFDDRSNLLLIQD  151 (175)
</pre><pre>T ss_dssp             SCTTCEEEEETTTTEEEEEE
</pre><pre>T ss_pred             cCCCceEEEeCCCCeEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>PF06835.15 ; LptC ; Lipopolysaccharide-assembly, LptC-related
</pre><pre>Probab=30.39  E-value=3.4e+02  Score=18.84  Aligned_cols=61  Identities=10%  Similarity=0.073  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             ccEEEEecCCCceEEEceeEEEEEec------cEEEeeCCeEe-----cCceEeeeeeecCCeEEECCEEeeee
</pre><pre>Q MBG6247344.1/9   12 GARFRYDAAAGALEAVGMKTARVSAE------TRILLDTPEVQ-----KGGEMHGNIAHSGGAFTSNGVQIDDH   74 (90)
</pre><pre>Q Consensus        12 GA~~~Yep~~gaL~v~GiKTAtV~As------~sITldtP~V~-----~Gg~m~G~ithsgG~~ssNGv~v~~H   74 (90)
</pre><pre>                      .....|+.++..+.+.|  .+.+...      ..+++.++.+.     +--...|++.-..+.+.+.|..+...
</pre><pre>T Consensus        87 a~~~~~~~~~~~~~l~g--~V~i~~~~~~~~~~~~~i~~~~l~~~~~~~~~~~~~~v~i~~~~~~~~g~~~~~~  158 (175)
</pre><pre>T Q3J7F9_NITOC/4   87 AERGLTNSQGDEIDLLG--EVIIRQLGADSKTSNMKILTQNVRVKPRIKYAETQQPVTLLNSFGKTHSIGARVY  158 (175)
</pre><pre>T ss_pred             EeEEEECCCCCEEEEeC--CEEEEECCCCCCCCceEEEeceEEEecCCCeEEeCCCEEEEcCCcEEEEEeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>2Y9K_D PROTEIN INVG; PROTEIN TRANSPORT, TYPE III SECRETION SYSTEM, OUTER MEMBRANE RING, SECRETIN FAMILY, C15 FOLD; 8.3A {SALMONELLA ENTERICA SUBSP. ENTERICA SEROVAR TYPHIMURIUM}
</pre><pre>Probab=30.15  E-value=1.1e+02  Score=20.02  Aligned_cols=20  Identities=20%  Similarity=0.184  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      ++++..+.+++.++.|.|.|
</pre><pre>T Consensus        99 ~~~~~~~~~~~~~~~l~v~~  118 (137)
</pre><pre>T 2Y9K_D           99 YNKNYPLRGDNRKGTFYVSG  118 (137)
</pre><pre>T ss_dssp             CCSSSCEEECSSTTEEEEEE
</pre><pre>T ss_pred             CCCCCCcEEeCCCCeEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>2Y9K_E PROTEIN INVG; PROTEIN TRANSPORT, TYPE III SECRETION SYSTEM, OUTER MEMBRANE RING, SECRETIN FAMILY, C15 FOLD; 8.3A {SALMONELLA ENTERICA SUBSP. ENTERICA SEROVAR TYPHIMURIUM}
</pre><pre>Probab=30.15  E-value=1.1e+02  Score=20.02  Aligned_cols=20  Identities=20%  Similarity=0.184  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      ++++..+.+++.++.|.|.|
</pre><pre>T Consensus        99 ~~~~~~~~~~~~~~~l~v~~  118 (137)
</pre><pre>T 2Y9K_E           99 YNKNYPLRGDNRKGTFYVSG  118 (137)
</pre><pre>T ss_dssp             CCSSSCEEECSSTTEEEEEE
</pre><pre>T ss_pred             CCCCCCcEEeCCCCeEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>5R0D_A Pre-mRNA-splicing factor 8; FragMAX, FragMAXapp, fragment screening, RNaseH like domain, VHS like domain, U5 snRNP assembly, SPLICING; HET: R8Y; 1.27A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)} SCOP: c.55.3.14, l.1.1.1
</pre><pre>Probab=29.90  E-value=1.7e+02  Score=24.94  Aligned_cols=35  Identities=20%  Similarity=0.184  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             eeEEeec--------CccEEEEecCCCceEEEceeEEEEEeccE
</pre><pre>Q MBG6247344.1/9    4 EMVITAL--------DGARFRYDAAAGALEAVGMKTARVSAETR   39 (90)
</pre><pre>Q Consensus         4 ~~~i~fP--------DGA~~~Yep~~gaL~v~GiKTAtV~As~s   39 (90)
</pre><pre>                      .++-+||        .|+.+.|+|.+|.|-+. +--+.+.|+++
</pre><pre>T Consensus        29 ~i~~~~~Gn~~~~ping~i~i~~p~tg~l~lk-vih~sv~~gqK   71 (258)
</pre><pre>T 5R0D_A           29 TVHKTFEGNVATKAINGCIFTLNPKTGHLFLK-IIHTSVWAGQK   71 (258)
</pre><pre>T ss_dssp             EEEECTTSCEEEEEECEEEEEECTTTCEEEEE-EECGGGGTTCS
</pre><pre>T ss_pred             EeEEecCCceeeeeeCCeEEEEccCcCcEEEE-EeecchhcCcc
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>PF07593.14 ; UnbV_ASPIC ; ASPIC and UnbV
</pre><pre>Probab=29.90  E-value=1.6e+02  Score=17.65  Aligned_cols=18  Identities=17%  Similarity=0.329  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEe
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYD   18 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Ye   18 (90)
</pre><pre>                      .+.++.|.+|||..-.|.
</pre><pre>T Consensus        41 ~~~~i~v~Wp~g~~~~~~   58 (65)
</pre><pre>T A2U4A3_9FLAO/4   41 KINQLIINWPSGHVDSYS   58 (65)
</pre><pre>T ss_pred             ceEEEEEEcCCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>3L6I_B Uncharacterized lipoprotein yceB; NESG, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, Cell membrane, Lipoprotein, Membrane, Palmitate; HET: MSE; 2.011A {Escherichia coli}
</pre><pre>Probab=29.63  E-value=2.2e+02  Score=21.57  Aligned_cols=23  Identities=22%  Similarity=0.212  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             cEEEEecCCCceEEEceeEEEEE
</pre><pre>Q MBG6247344.1/9   13 ARFRYDAAAGALEAVGMKTARVS   35 (90)
</pre><pre>Q Consensus        13 A~~~Yep~~gaL~v~GiKTAtV~   35 (90)
</pre><pre>                      +.++|||+++++.+.-++--.++
</pre><pre>T Consensus        87 g~~~Yd~~~~aiyL~~~~i~~~~  109 (181)
</pre><pre>T 3L6I_B           87 ALPVFDKEKGAIFLKEMEVVDAT  109 (181)
</pre><pre>T ss_dssp             EEEEEETTTTEEEEEEEEEEEEE
</pre><pre>T ss_pred             EeEEEECCCCeEEEEEeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>PF09981.11 ; DUF2218 ; Uncharacterized protein conserved in bacteria (DUF2218)
</pre><pre>Probab=29.60  E-value=2.2e+02  Score=18.03  Aligned_cols=23  Identities=17%  Similarity=0.020  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      +.+.||+|..+...+....|.+.
</pre><pre>T Consensus        33 ~~~~~~~g~~~~l~~~~~~l~i~   55 (89)
</pre><pre>T I3TPS3_TISMK/6   33 ARIPFSGTSCCIMQASEDRLGIV   55 (89)
</pre><pre>T ss_pred             EEEEeCCCCEEEEEEeCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>PF10387.11 ; DUF2442 ; Protein of unknown function (DUF2442)
</pre><pre>Probab=28.64  E-value=1.7e+02  Score=18.35  Aligned_cols=17  Identities=24%  Similarity=0.300  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecC
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAA   20 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~   20 (90)
</pre><pre>                      .+.+.|.||.+..+|.+
</pre><pre>T Consensus        11 ~l~i~~~dG~~~~~dl~   27 (76)
</pre><pre>T Q2W6T4_MAGSA/5   11 KLHLRFQDGAEGEVDCS   27 (76)
</pre><pre>T ss_pred             EEEEEECCCCEEEEECh
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>d1sxva1 b.40.5.0 (A:1-159) automated matches {Mycobacterium tuberculosis [TaxId: 1773]}
</pre><pre>Probab=28.28  E-value=2.4e+02  Score=21.42  Aligned_cols=24  Identities=17%  Similarity=0.385  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCc--cEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDG--ARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDG--A~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      +++|.-|-|  +.++||++++.|.+.
</pre><pre>T Consensus         4 ~~vVEip~gs~~k~e~d~~~~~~~~~   29 (159)
</pre><pre>T d1sxva1           4 DVTIEIPKGQRNKYEVDHETGRVRLD   29 (159)
</pre><pre>T ss_dssp             EEEEEECTTCCEEC-CCTTTCCCCCC
</pre><pre>T ss_pred             EEEEEEcCCCCCeeEEECCCCcEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>2RH7_C Green fluorescent protein; fluorescent protein; HET: CRO; 1.5A {Renilla reniformis} SCOP: d.22.1.1
</pre><pre>Probab=28.20  E-value=2.5e+02  Score=23.65  Aligned_cols=44  Identities=20%  Similarity=0.247  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        91 ~eRt~~FeDg-------------------g~~t~~~~is~------e~~~~~~~~~~~G~nFp~dgpvm  134 (239)
</pre><pre>T 2RH7_C           91 YERNIRYQDG-------------------GTAIVKSDISL------EDGKFIVNVDFKAKDLRRMGPVM  134 (239)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------C-CCEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------cEEEEEEEEEE------ECCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>PF06097.13 ; DUF945 ; Bacterial protein of unknown function (DUF945)
</pre><pre>Probab=28.02  E-value=2e+02  Score=23.90  Aligned_cols=23  Identities=22%  Similarity=0.455  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             cCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9   49 KGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus        49 ~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      +++.+.-++.-..|.+++||.++
</pre><pre>T Consensus       452 ~~~~~~~~~~~~~G~l~iNG~~l  474 (477)
</pre><pre>T H0PVZ9_9RHOO/1  452 EQGFVKTALEFRGGELKINGKPV  474 (477)
</pre><pre>T ss_pred             eCCEEEEEEEEeCCEEEECCEEC
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>PF13600.8 ; DUF4140 ; N-terminal domain of unknown function (DUF4140)
</pre><pre>Probab=27.64  E-value=1.8e+02  Score=19.42  Aligned_cols=38  Identities=16%  Similarity=0.045  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             EEeecCccEEEEec------CCCceEEEceeEEEEEeccEEEee
</pre><pre>Q MBG6247344.1/9    6 VITALDGARFRYDA------AAGALEAVGMKTARVSAETRILLD   43 (90)
</pre><pre>Q Consensus         6 ~i~fPDGA~~~Yep------~~gaL~v~GiKTAtV~As~sITld   43 (90)
</pre><pre>                      +..|||||.+....      ....|.+.|+.......+-.+...
</pre><pre>T Consensus         1 Vtvy~~~A~v~r~~~~~l~~G~~~i~i~~Lp~~i~~~Sl~v~~~   44 (100)
</pre><pre>T A7HCS6_ANADF/3    1 VTVYRQSARVTRVAKVELPDGPTRVVLEGLPDELDDDSIRVEGK   44 (100)
</pre><pre>T ss_pred             CEEecCCeEEEEEEEEEcCCCCeEEEEcCCCCccCccceEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>2W68_B SIALIDASE; CBM, CALCIUM, SECRETED, HYDROLASE, GLYCOSIDASE, SIALIC ACID, CARBOHYDRATE-BINDING-DOMAIN; HET: SIA, BGC, GAL; 2.5A {VIBRIO CHOLERAE}
</pre><pre>Probab=27.60  E-value=1.4e+02  Score=23.77  Aligned_cols=20  Identities=5%  Similarity=0.012  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCC---ceEEEceeEEE
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAG---ALEAVGMKTAR   33 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~g---aL~v~GiKTAt   33 (90)
</pre><pre>                      .|+|||.++   .|.|.|.+.++
</pre><pre>T Consensus       136 eL~ydp~t~~~A~l~vDG~~i~s  158 (195)
</pre><pre>T 2W68_B          136 ELVFLPGSNPSASFYFDGKLIRD  158 (195)
</pre><pre>T ss_dssp             EEEEECSSSCEEEEEETTEEEEE
</pre><pre>T ss_pred             EEEEcCCCCCeEEEEECCEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>2W68_C SIALIDASE; CBM, CALCIUM, SECRETED, HYDROLASE, GLYCOSIDASE, SIALIC ACID, CARBOHYDRATE-BINDING-DOMAIN; HET: GAL, BGC, SIA; 2.5A {VIBRIO CHOLERAE}
</pre><pre>Probab=27.60  E-value=1.4e+02  Score=23.77  Aligned_cols=20  Identities=5%  Similarity=0.012  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCC---ceEEEceeEEE
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAG---ALEAVGMKTAR   33 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~g---aL~v~GiKTAt   33 (90)
</pre><pre>                      .|+|||.++   .|.|.|.+.++
</pre><pre>T Consensus       136 eL~ydp~t~~~A~l~vDG~~i~s  158 (195)
</pre><pre>T 2W68_C          136 ELVFLPGSNPSASFYFDGKLIRD  158 (195)
</pre><pre>T ss_dssp             EEEEECSSSCEEEEEESSCEEEE
</pre><pre>T ss_pred             EEEEcCCCCCeEEEEECCEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>PF13100.8 ; OstA_2 ; OstA-like protein
</pre><pre>Probab=27.51  E-value=3.7e+02  Score=18.23  Aligned_cols=57  Identities=11%  Similarity=0.050  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe-----cCceEeeeeeecCCeEEECCEEee
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ-----KGGEMHGNIAHSGGAFTSNGVQID   72 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~-----~Gg~m~G~ithsgG~~ssNGv~v~   72 (90)
</pre><pre>                      .++|+++++.+.+.|-  +.+.-...+++.+-...     +-..+.|++.-..+.+.+.+-.+.
</pre><pre>T Consensus        43 ~~~~~~~~~~~~~~g~--v~~~~~~~~~~~~~~~~~~~~~~~~~~~g~v~~~~~~~~~~~~~~~  104 (159)
</pre><pre>T A6KXH5_BACV8/4   43 SACFYEKTNSLEAFDN--VKMVQGDTLFLYGDYLFYDGNTQIAQVRYNVRMENKNTTLLTDSLN  104 (159)
</pre><pre>T ss_pred             EEEEECCCCEEEEEec--EEEEeCCeeEEEEeEEEEECCCCEEEEecCEEEECCCeEEEeCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>PF06155.14 ; GBBH-like_N ; Gamma-butyrobetaine hydroxylase-like, N-terminal
</pre><pre>Probab=27.41  E-value=1.5e+02  Score=18.07  Aligned_cols=17  Identities=18%  Similarity=0.077  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEec
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDA   19 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep   19 (90)
</pre><pre>                      ..+.+.|+||..+.|.+
</pre><pre>T Consensus         8 ~~l~v~~~d~~~~~~~~   24 (85)
</pre><pre>T C6X7R2_METGS/1    8 RLLEITFDDNTTCMLSC   24 (85)
</pre><pre>T ss_pred             CeEEEEeCCCCeEEEeH
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>4JK8_A Pre-mRNA-processing-splicing factor 8; metalloprotein, conformational change, RNA BINDING PROTEIN; HET: GOL; 1.15A {Homo sapiens} SCOP: c.55.3.14
</pre><pre>Probab=27.23  E-value=1.3e+02  Score=24.75  Aligned_cols=28  Identities=25%  Similarity=0.352  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CccEEEEecCCCceEEEceeEEEEEeccE
</pre><pre>Q MBG6247344.1/9   11 DGARFRYDAAAGALEAVGMKTARVSAETR   39 (90)
</pre><pre>Q Consensus        11 DGA~~~Yep~~gaL~v~GiKTAtV~As~s   39 (90)
</pre><pre>                      .|+.+.|+|.+|.|-+. +--+++.|+++
</pre><pre>T Consensus        36 ng~i~~~~p~tg~l~~k-v~h~s~~~~qK   63 (222)
</pre><pre>T 4JK8_A           36 NGAIFIFNPRTGQLFLK-IIHTSVWAGQK   63 (222)
</pre><pre>T ss_dssp             CEEEEEECTTTCEEEEE-EECGGGGTTCS
</pre><pre>T ss_pred             ccEEEEEccCcceEEEE-Eeccchhcccc
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>6GP1_B Green to red photoconvertible GFP-like protein EosFP; Fluorescent protein; HET: IEY, NFA; 1.504A {Lobophyllia hemprichii}
</pre><pre>Probab=27.20  E-value=2.7e+02  Score=22.21  Aligned_cols=41  Identities=27%  Similarity=0.407  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNG   68 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||
</pre><pre>T Consensus        26 ~eRt~~FEDg-------------------g~~t~~~~is~------e~~~~~~~v~~~G~nFp~dG   66 (163)
</pre><pre>T 6GP1_B           26 WERSLTFEDG-------------------GICNARNDITM------EGDTFYNKVRFYGTNFPANG   66 (163)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTS
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>4DXM_A GREEN FLUORESCENT PROTEIN; BETA BARREL, LUMINESCENT PROTEIN; HET: CR8, SO4; 1.4A {Synthetic Construct}
</pre><pre>Probab=26.84  E-value=2.5e+02  Score=23.36  Aligned_cols=44  Identities=30%  Similarity=0.353  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        87 ~eRt~~feDg-------------------g~lt~~~~is~------~~~~l~~~~~~~G~~Fp~dgpvm  130 (229)
</pre><pre>T 4DXM_A           87 WERSMTFEDG-------------------GICTATSDITL------EGDCFFYEIRFDGVNFPPNGPVM  130 (229)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>d2a46a_ d.22.1.0 (A:) automated matches {Anemonia majano [TaxId: 105399]}
</pre><pre>Probab=26.63  E-value=2.1e+02  Score=24.25  Aligned_cols=47  Identities=26%  Similarity=0.391  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             eecCc----cEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    8 TALDG----ARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         8 ~fPDG----A~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .||+|    ..+.||           .-++++|+..|++      +++.+..+++-.|-.|..||.+.
</pre><pre>T Consensus        84 sfPeG~t~eRt~~FE-----------DGg~~t~~~~isl------~~~~~~~~~~~~G~nFp~dgPVm  134 (220)
</pre><pre>T d2a46a_          84 AFPDGMSYERTFTYE-----------DGGVATASWEISL------KGNCFEHKSTFHGVNFPADGPVM  134 (220)
</pre><pre>T ss_dssp             GTTTCEEEEEEEEET-----------TSCEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             hCCCCeeEEEEEEEc-----------CCcEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>PF18544.3 ; Polo_box_3 ; Polo box domain
</pre><pre>Probab=26.55  E-value=2.5e+02  Score=21.20  Aligned_cols=23  Identities=13%  Similarity=0.176  Sum_probs=0.0  Template_Neff=3.700
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      .+.|.|||| ++---..++.+.++
</pre><pre>T Consensus        25 dfRikf~d~-~~VqK~~s~~v~~~   47 (96)
</pre><pre>T G0NAV2_CAEBE/1   25 DLRIKCEKG-VIVRRKENGEIYNS   47 (96)
</pre><pre>T ss_pred             cEEEEecCc-eEEEEcCCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>PF19367.1 ; DUF5943 ; Domain of unknown function (DUF5943)
</pre><pre>Probab=26.46  E-value=1.6e+02  Score=18.92  Aligned_cols=29  Identities=14%  Similarity=0.091  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEceeEEEEEec
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVGMKTARVSAE   37 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~GiKTAtV~As   37 (90)
</pre><pre>                      |.-+..++||++.+.|...|...+.+.+.
</pre><pre>T Consensus         1 ~~~~~~~~~d~~~g~l~~~~~r~v~~~~~   29 (96)
</pre><pre>T G0ACQ5_COLFT/1    1 MQPQLPIEVDASTGVWTTDKLPMLYVPRH   29 (96)
</pre><pre>T ss_pred             CCCCCCcEEeCCCCeEEeCCceeEEeeHH
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>PF14157.8 ; YmzC ; YmzC-like protein
</pre><pre>Probab=26.34  E-value=1.4e+02  Score=20.54  Aligned_cols=13  Identities=15%  Similarity=0.537  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCCceEE
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAGALEA   26 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~gaL~v   26 (90)
</pre><pre>                      .++|||++..|+.
</pre><pre>T Consensus        40 Iy~Yd~~~Nel~l   52 (60)
</pre><pre>T A0A6L7H7A7_BAC   40 IFKFDPDTNEIKL   52 (60)
</pre><pre>T ss_pred             EEEEcCCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>2KXJ_A UBX domain-containing protein 4; UBXD2, UBX, VCP, p97, PROTEIN BINDING; NMR {Homo sapiens}
</pre><pre>Probab=26.31  E-value=1.5e+02  Score=18.30  Aligned_cols=16  Identities=19%  Similarity=0.434  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFR   16 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~   16 (90)
</pre><pre>                      ....+.|.||||.++.
</pre><pre>T Consensus        11 ~~~~i~ir~p~G~~~~   26 (90)
</pre><pre>T 2KXJ_A           11 TVARIQFRLPDGSSFT   26 (90)
</pre><pre>T ss_dssp             SEEEEEEECTTSCEEE
</pre><pre>T ss_pred             CEEEEEEECCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>PF12970.9 ; DUF3858 ; Domain of Unknown Function with PDB structure (DUF3858)
</pre><pre>Probab=26.30  E-value=3.4e+02  Score=17.48  Aligned_cols=42  Identities=10%  Similarity=-0.003  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEEC
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSN   67 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssN   67 (90)
</pre><pre>                      ++.|.+|+|-.+++-|+  .+.+..                    ..+.+.-.+++.|+.+++.
</pre><pre>T Consensus        40 ~~~i~lP~~~~~~~~p~--~~~~~~--------------------~~~~~~~~~~~~~~~l~~~   81 (116)
</pre><pre>T Q5L7X0_BACFN/4   40 TYTVDTPAGMEPVTRPM--EKKMDN--------------------AVGTLTISIRPEGAKTRVI   81 (116)
</pre><pre>T ss_pred             EEEEECCCCceEeecCc--ceeEec--------------------CCeEEEEEEEecCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>d1s3si_ d.15.1.2 (I:) p47 {Norway rat (Rattus norvegicus) [TaxId: 10116]}
</pre><pre>Probab=25.89  E-value=1.4e+02  Score=18.37  Aligned_cols=12  Identities=33%  Similarity=0.526  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEE
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFR   16 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~   16 (90)
</pre><pre>                      +.|.||||.++.
</pre><pre>T Consensus        22 iqiRlp~G~~~~   33 (50)
</pre><pre>T d1s3si_          22 IQIRLADGGRLV   33 (50)
</pre><pre>T ss_dssp             ------------
</pre><pre>T ss_pred             EEEEcCCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>d3o2ga1 d.392.1.1 (A:1-95) Gamma-butyrobetaine dioxygenase (BBOX1) N-terminal domain {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=25.88  E-value=1.4e+02  Score=19.74  Aligned_cols=17  Identities=29%  Similarity=0.180  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEec
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDA   19 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep   19 (90)
</pre><pre>                      ..+.|.|+||...+|.+
</pre><pre>T Consensus        15 ~~l~v~~~dg~~~~~~~   31 (95)
</pre><pre>T d3o2ga1          15 HLMQILWYDEEESLYPA   31 (95)
</pre><pre>T ss_dssp             TEEEEEETTSCEEEEEH
</pre><pre>T ss_pred             CeEEEEeCCCCEEecch
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>3E9L_A Pre-mRNA-processing-splicing factor 8; nucleotidyl transfer, Disease mutation, mRNA processing, mRNA splicing, Nucleus, Phosphoprotein, Retinitis pigmentosa, RNA-binding, Sensory transduction, Spliceosome; 1.95A {Homo sapiens} SCOP: c.55.3.14
</pre><pre>Probab=25.63  E-value=1.1e+02  Score=25.47  Aligned_cols=28  Identities=25%  Similarity=0.352  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             CccEEEEecCCCceEEEceeEEEEEeccE
</pre><pre>Q MBG6247344.1/9   11 DGARFRYDAAAGALEAVGMKTARVSAETR   39 (90)
</pre><pre>Q Consensus        11 DGA~~~Yep~~gaL~v~GiKTAtV~As~s   39 (90)
</pre><pre>                      .|+.+.|+|.+|.|-+. +--+++.|+++
</pre><pre>T Consensus        45 ng~i~i~~p~tg~l~~k-vih~sv~~~~k   72 (257)
</pre><pre>T 3E9L_A           45 NGAIFIFNPRTGQLFLK-IIHTSVWAGQK   72 (257)
</pre><pre>T ss_dssp             CEEEEEECTTTCEEEEE-EECGGGGTTCS
</pre><pre>T ss_pred             ccEEEEEcCCcceEEEe-eechhhhcccc
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>3NED_A PAmCherry1 protein; RFP, mCherry, beta barrel, FLUORESCENT PROTEIN; HET: NRQ, EYG, CH6; 0.95A {Discosoma sp.} SCOP: d.22.1.1
</pre><pre>Probab=25.52  E-value=2.8e+02  Score=23.40  Aligned_cols=44  Identities=27%  Similarity=0.351  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .+-.+.|.||+                   ++++...|++      +++.+..+++-.|-.|..||.+.
</pre><pre>T Consensus       104 ~eRt~~FeDGg-------------------~~t~~~~is~------e~~~~~~~~~~~G~nFp~dgpvm  147 (242)
</pre><pre>T 3NED_A          104 WERVMNFEDGG-------------------VVTVTQDSSL------QDGEFIYKVKLRGTNFPSDGPVM  147 (242)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCCc-------------------EEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>d1rl6a1 d.141.1.1 (A:7-81) Ribosomal protein L6 {Bacillus stearothermophilus [TaxId: 1422]}
</pre><pre>Probab=25.25  E-value=2e+02  Score=18.47  Aligned_cols=20  Identities=15%  Similarity=0.272  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             EeecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    7 ITALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         7 i~fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      |..|+|-.++|+  .+.|.+.|
</pre><pre>T Consensus         2 i~iP~~v~v~~~--~~~l~v~G   21 (75)
</pre><pre>T d1rl6a1           2 IEIPAGVTVTVN--GNTVTVKG   21 (75)
</pre><pre>T ss_dssp             EECCTTCEEEEE--TTEEEEEE
</pre><pre>T ss_pred             cccCCCcEEEEe--CCEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>2ICR_B Red fluorescent protein zoanRFP; red fluorescent protein, ZRFP574, Zoanthus sp., chromophore structure, CYS-PHE link, FLUORESCENT PROTEIN; HET: SO4, XYG; 1.51A {Zoanthus sp.} SCOP: d.22.1.0
</pre><pre>Probab=25.16  E-value=3.2e+02  Score=23.01  Aligned_cols=44  Identities=16%  Similarity=0.150  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEE--EeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRI--LLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sI--TldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||+                   ++++...|  ++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        99 ~eRt~~FEDGG-------------------~lt~~~~i~~~~------e~~~i~~~~~~~G~~Fp~dGpvm  144 (237)
</pre><pre>T 2ICR_B           99 WHRSFRFEDGA-------------------VCICSADITVNV------RENCIYHESTFYGVNFPADGPVM  144 (237)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEEEG------GGTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEecCCC-------------------EEEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>d2cqla1 d.141.1.1 (A:1-87) Ribosomal protein L6 {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=25.04  E-value=2.8e+02  Score=18.43  Aligned_cols=25  Identities=8%  Similarity=0.169  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             cceeEEeecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    2 LQEMVITALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         2 ~~~~~i~fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      +.+..|..|+|-.++|+  .+.|.+.|
</pre><pre>T Consensus         5 ~~~~~i~IP~~v~v~~~--~~~v~v~G   29 (87)
</pre><pre>T d2cqla1           5 LSNQTVDIPENVDITLK--GRTVIVKG   29 (87)
</pre><pre>T ss_dssp             CCCEEECCCSSCEEEEE--TTEEEEEE
</pre><pre>T ss_pred             EeeeEEECCCCcEEEEe--CCEEEEEc
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>d2icra_ d.22.1.0 (A:) automated matches {Zoanthus sp. [TaxId: 105402]}
</pre><pre>Probab=24.83  E-value=3.4e+02  Score=23.09  Aligned_cols=43  Identities=16%  Similarity=0.190  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNG   68 (90)
</pre><pre>                      .|-.+.|.||+                   .++|+..|+++-    +++.+..+++-.|-.|..||
</pre><pre>T Consensus        90 ~eRt~~FEDGg-------------------~~t~~~~is~~~----~~~~i~~~v~~~G~nFp~dG  132 (228)
</pre><pre>T d2icra_          90 WHRSFRFEDGA-------------------VCICSADITVNV----RENCIYHESTFYGVNFPADG  132 (228)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEEET----TTTEEEEEEEEEEECCCTTS
</pre><pre>T ss_pred             EEEEEEEcCCc-------------------EEEEEEEEEEEE----eCCEEEEEEEEEeecCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>3LF4_B Fluorescent Timer Precursor Blue102; Fluorescent Timers, Blue-to-red conversion, Chromophore degradation, FLUORESCENT PROTEIN; HET: 0YG; 1.808A {Discosoma sp.}
</pre><pre>Probab=24.80  E-value=3.1e+02  Score=21.86  Aligned_cols=44  Identities=27%  Similarity=0.351  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.++|.||                   +++++.-.|++      +++.+.++++-.|-.|.-||-+.
</pre><pre>T Consensus        26 ~eRt~~FEDg-------------------g~~t~~~~is~------e~~~~~~~~~~~G~~Fp~dGpVm   69 (164)
</pre><pre>T 3LF4_B           26 WERVMNFEDG-------------------GVVTVTQDSSL------QDGEFIYKVKLRGTNFPSDGPVM   69 (164)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEccC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>2YMS_B OUTER MEMBRANE PROTEIN ASSEMBLY FACTOR BAMB; CHAPERONE, PROPELLER STRUCTURE, ASSEMBLED FROM FRAGMENTS, NINE BLADES; 2.101A {ESCHERICHIA COLI}
</pre><pre>Probab=24.78  E-value=2.5e+02  Score=15.38  Aligned_cols=21  Identities=24%  Similarity=0.400  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCceE
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGALE   25 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL~   25 (90)
</pre><pre>                      +.+...++.++.+|++++.+.
</pre><pre>T Consensus        50 ~~~~~~~~~~~~~~~~~g~~~   70 (74)
</pre><pre>T 2YMS_B           50 VLIHTSNGQLQALNEADGAVK   70 (74)
</pre><pre>T ss_dssp             EEEEBTTSCEEEEETTTCCEE
</pre><pre>T ss_pred             EEEEecCCeEEEEECCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>1XMZ_A GFP-like chromoprotein FP595; FLUORESCENT PROTEIN, CHROMOPHORE STRUCTURE, ASCP595, LUMINESCENT PROTEIN; HET: CRK, BME; 1.38A {Anemonia sulcata} SCOP: d.22.1.1
</pre><pre>Probab=24.65  E-value=2.9e+02  Score=23.27  Aligned_cols=44  Identities=23%  Similarity=0.266  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +.+++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        99 ~eRt~~feDG-------------------g~lt~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  142 (241)
</pre><pre>T 1XMZ_A           99 WERTTTYEDG-------------------GFLTAHQDTSL------DGDCLVYKVKILGNNFPADGPVM  142 (241)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>PF18161.3 ; ISP1_C ; ISP1 C-terminal
</pre><pre>Probab=24.48  E-value=3.1e+02  Score=20.66  Aligned_cols=24  Identities=25%  Similarity=0.535  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEE----EecCCCceEEE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFR----YDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~----Yep~~gaL~v~   27 (90)
</pre><pre>                      .+.+.||||-.++    ||.+...|.++
</pre><pre>T Consensus        11 ~v~vll~DGs~l~C~l~l~~~~~~l~ls   38 (107)
</pre><pre>T B9PKK4_TOXGV/6   11 AVLVLLQDGTRLQCILHYNEADSSLSIS   38 (107)
</pre><pre>T ss_pred             EEEEEcCCCCeEEEEEEEEcCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>d2qi2a1 b.38.4.1 (A:1-126) Cell division protein pelota {Thermoplasma acidophilum [TaxId: 2303]}
</pre><pre>Probab=24.32  E-value=1.7e+02  Score=21.94  Aligned_cols=16  Identities=13%  Similarity=0.385  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCCceEEEce
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAGALEAVGM   29 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~gaL~v~Gi   29 (90)
</pre><pre>                      .++|+|.++.|.+.|.
</pre><pre>T Consensus        69 ~ief~~~~~~Lri~G~   84 (126)
</pre><pre>T d2qi2a1          69 KIEFQDFDNRLRILGT   84 (126)
</pre><pre>T ss_dssp             CTTCSCEEEEEEECC-
</pre><pre>T ss_pred             EEeeeCCCCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>4ZBL_A Photosesitizer mKillerOrange; Fluorescent protein, Phototoxicity, Beta-barrel, QWG chromophore; HET: 4M9, GOL, CIT; 1.57A {Hydrozoa}
</pre><pre>Probab=23.95  E-value=2.9e+02  Score=23.28  Aligned_cols=44  Identities=9%  Similarity=0.138  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +.++|...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus       104 ~eRt~~FEDG-------------------g~~t~~~~is~------e~~~i~~~~~~~G~~Fp~dgpvm  147 (240)
</pre><pre>T 4ZBL_A          104 IDRTVRFEND-------------------GTMTSHHTYEL------DDTCVVSRITVNCDGFQPDGPIM  147 (240)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEESCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEecC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccC
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>d2wura_ d.22.1.1 (A:) automated matches {Jellyfish (Aequorea victoria) [TaxId: 6100]}
</pre><pre>Probab=23.76  E-value=3.3e+02  Score=23.32  Aligned_cols=41  Identities=22%  Similarity=0.335  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNG   68 (90)
</pre><pre>                      .|-.+.|.||++                   ++|+..|++      +++.+..+++-.|-.|..||
</pre><pre>T Consensus        92 ~eR~~~FeDGg~-------------------~t~~~~is~------~~~~~~~~~~~~g~nFp~dG  132 (230)
</pre><pre>T d2wura_          92 QERTIFFKDDGN-------------------YKTRAEVKF------EGDTLVNRIELKGIDFKEDG  132 (230)
</pre><pre>T ss_dssp             EEEEEEETTSCE-------------------EEEEEEEEE------ETTEEEEEEEEEEECCCTTS
</pre><pre>T ss_pred             EEEEEEEeCCcE-------------------EEEEEEEEE------eCCEEEEEEEEEEeCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>PF17479.4 ; DUF3048_C ; Protein of unknown function (DUF3048) C-terminal domain
</pre><pre>Probab=23.75  E-value=4.9e+02  Score=18.43  Aligned_cols=68  Identities=19%  Similarity=0.239  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCc--cEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe----------cCceEeeeeeecC-CeEEECCEEe
</pre><pre>Q MBG6247344.1/9    5 MVITALDG--ARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ----------KGGEMHGNIAHSG-GAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         5 ~~i~fPDG--A~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~----------~Gg~m~G~ithsg-G~~ssNGv~v   71 (90)
</pre><pre>                      +.|.|+..  ..++||++++.-.-.=--...+.+...--+.+.+|.          ..|.+.-++.-+| +.+-+||..+
</pre><pre>T Consensus         1 i~v~~~~~~~~~~~yd~~~~~Y~R~~~g~~~~d~~~g~~i~~~Nviv~~~~~~~~d~~g~~~~~~~g~G~a~~~~~G~~~   80 (112)
</pre><pre>T C0Z4B8_BREBN/2    1 VDLTFHSLNKAAFTYDAEQKKYMRFTAGKPHLDLSDKKQLSTTNLLVISAKHRVLDKEGRLSVDVIGPGDGYLFQQGKAR   80 (112)
</pre><pre>T ss_pred             CeeeeCCCccEEEEEECcCCeEEEEeCCEEeEeCCCCcEEEEeEEEEEEeeEEEeCCCCcEEEEEEceEEEEEEECCEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             e
</pre><pre>Q MBG6247344.1/9   72 D   72 (90)
</pre><pre>Q Consensus        72 ~   72 (90)
</pre><pre>                      .
</pre><pre>T Consensus        81 ~   81 (112)
</pre><pre>T C0Z4B8_BREBN/2   81 K   81 (112)
</pre><pre>T ss_pred             E
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>PF18315.3 ; VCH_CASS14 ; Integron cassette protein VCH_CASS1 chain
</pre><pre>Probab=23.60  E-value=2.1e+02  Score=21.35  Aligned_cols=25  Identities=36%  Similarity=0.739  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeec
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHS   60 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~iths   60 (90)
</pre><pre>                      .|.||++++.+.+                      ..++++|.+-|+
</pre><pre>T Consensus        62 afsYnh~t~~IEl----------------------R~~s~~G~vl~s   86 (97)
</pre><pre>T A9GUG2_SORC5/1   62 AFSYNHKAGTIEM----------------------REGSTRGAVLHS   86 (97)
</pre><pre>T ss_pred             EEEEECCCCeEEE----------------------ecCCccCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>3PIB_B eqFP578 fluorescent protein; Red fluorescent protein, beta-barrel, biomarker, Met-Tyr-Gly chromophore, sea anemone, FLUORESCENT PROTEIN; HET: NRQ, GOL; 1.154A {Entacmaea quadricolor} SCOP: d.22.1.1, l.1.1.1
</pre><pre>Probab=23.57  E-value=3.5e+02  Score=22.72  Aligned_cols=44  Identities=30%  Similarity=0.345  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+..+++-.|-.|..||-+.
</pre><pre>T Consensus        95 ~eRt~~feDg-------------------g~~t~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  138 (236)
</pre><pre>T 3PIB_B           95 WERITTYEDG-------------------GVLTATQDTSL------QNGCIIYNVKINGVNFPSNGSVM  138 (236)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------ECCEEEEEEEEEeeccCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>3PET_B Putative adhesin; RIGHT-HANDED BETA-HELIX, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, CELL ADHESION; HET: MSE, PG4; 2.07A {Bacteroides fragilis NCTC 9343}
</pre><pre>Probab=23.54  E-value=5.5e+02  Score=18.92  Aligned_cols=69  Identities=12%  Similarity=0.238  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEc-eeE--EEEEeccEEEeeCCeEe----------------cCceEeeeeeecC-Ce
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVG-MKT--ARVSAETRILLDTPEVQ----------------KGGEMHGNIAHSG-GA   63 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~G-iKT--AtV~As~sITldtP~V~----------------~Gg~m~G~ithsg-G~   63 (90)
</pre><pre>                      ++.|..|.-..++.+ ..+.+.+.+ ++.  ..++++..-.++.-.+.                .|-.-..++..+| |.
</pre><pre>T Consensus        82 ~i~i~~p~l~~i~~~-~~g~i~~~~~l~~~~l~i~~~~~g~i~~~~~~~~~l~~~~~~~g~i~l~g~~~~~~~~~sg~g~  160 (221)
</pre><pre>T 3PET_B           82 KITITSPTLNGISFK-GVGDVHIENGLTTDNLDIESKGVGNVDIQSLTCQKLNVQSMGVGDVKLEGTAQIAALHSKGVGN  160 (221)
</pre><pre>T ss_dssp             EEEEEESCCCEEEEC-SBSCEEECSCEECSEEEEEECSBSCEEEEEEECSEEEEEECSBSCEEEEEEEEEEEEEECSBSC
</pre><pre>T ss_pred             EEEEEcCCcceeEEe-ccEEEEEcCcEEcCcEEEEEcceeEEEEeeeEcCEEEEEEccccEEEEEEEEeEEEEEEcCcee
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEECCEEeee
</pre><pre>Q MBG6247344.1/9   64 FTSNGVQIDD   73 (90)
</pre><pre>Q Consensus        64 ~ssNGv~v~~   73 (90)
</pre><pre>                      |..+......
</pre><pre>T Consensus       161 i~~~~~~~~~  170 (221)
</pre><pre>T 3PET_B          161 IEAGNLRANA  170 (221)
</pre><pre>T ss_dssp             EEEEEEEEEE
</pre><pre>T ss_pred             EEecceEecE
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>PF18045.3 ; ISP3_C ; ISP3 C-terminal
</pre><pre>Probab=23.42  E-value=3.4e+02  Score=20.53  Aligned_cols=24  Identities=8%  Similarity=0.342  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEE----EecCCCceEEE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFR----YDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~----Yep~~gaL~v~   27 (90)
</pre><pre>                      .+.+.||||-.++    |+++...|.++
</pre><pre>T Consensus        14 ~v~Vll~DGs~l~C~l~l~~~~~~l~ls   41 (110)
</pre><pre>T Q4N9K6_THEPA/3   14 DVIVLLEDGTKLSCTLHVNCETSLVRIA   41 (110)
</pre><pre>T ss_pred             EEEEEeCCCCeEEEEEEEECCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>d5x3pa_ d.15.1.0 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=23.41  E-value=2.7e+02  Score=17.19  Aligned_cols=20  Identities=20%  Similarity=0.288  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCc
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGA   23 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~ga   23 (90)
</pre><pre>                      .+.|.||||.++.-.-...+
</pre><pre>T Consensus         5 ~i~ir~p~g~~~~~~f~~~~   24 (80)
</pre><pre>T d5x3pa_           5 QLMLRYPDGKREQITLPEQA   24 (80)
</pre><pre>T ss_dssp             EEEEECTTSCEEEEEEETTS
</pre><pre>T ss_pred             EEEEECCCCCeEEEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>6VU9_A Threonine--tRNA ligase; AARS, aminoacyl tRNA synthetase, NIAID, National Institute of Allergy and Infectious Diseases, thrS, antibiotic resistance, borrelidin; HET: SO4; 2.2A {Stenotrophomonas maltophilia (strain K279a)}
</pre><pre>Probab=23.40  E-value=1.6e+02  Score=25.47  Aligned_cols=17  Identities=29%  Similarity=0.501  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCC
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAA   21 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~   21 (90)
</pre><pre>                      +.|+||||...+|++.+
</pre><pre>T Consensus        10 ~~~~~~~~~~~~~~~~~   26 (641)
</pre><pre>T 6VU9_A           10 INITLPDGSRREFENPV   26 (641)
</pre><pre>T ss_dssp             -CEECTTSCEECCSSCE
</pre><pre>T ss_pred             eEEECCCCCeEEeCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>6X2D_B Ribonuclease E; Structural Genomics, Center for Structural Genomics of Infectious Diseases, CSGID, RNA BINDING PROTEIN; HET: MSE; 1.85A {Vibrio cholerae serotype O1 (strain ATCC 39315 / El Tor Inaba N16961)}
</pre><pre>Probab=23.15  E-value=2.9e+02  Score=20.14  Aligned_cols=31  Identities=19%  Similarity=0.268  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCCCceEEEceeEE
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAAGALEAVGMKTA   32 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~gaL~v~GiKTA   32 (90)
</pre><pre>                      |+..-.+.+|+|..+..|+ +.||++--|.++
</pre><pre>T Consensus         1 ~~~~~~v~L~~Gg~l~ie~-TeAlt~IDVns~   31 (115)
</pre><pre>T 6X2D_B            1 SNAQREVRLPSGGSIVIDP-TEALTSIDINSA   31 (115)
</pre><pre>T ss_dssp             ---CCEEECTTSCEEEEEE-CSSEEEEEEECC
</pre><pre>T ss_pred             CCCCCEEECCCCcEEEEec-CCcEEEEEcccc
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>3IR8_A Large stokes shift fluorescent protein; beta barrel, FLUORESCENT PROTEIN; HET: CRQ; 1.63A {Montipora sp. 20} SCOP: d.22.1.1
</pre><pre>Probab=22.86  E-value=3.2e+02  Score=22.61  Aligned_cols=44  Identities=25%  Similarity=0.295  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .+-.+.|.||                   +.+++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        89 ~eRt~~feDg-------------------g~~~~~~~is~------~~~~i~~~~~~~G~~Fp~dGpvm  132 (221)
</pre><pre>T 3IR8_A           89 WERSMNFEDG-------------------AVCTVSNDSSI------QGNCFIYNVKISGENFPPNGPVM  132 (221)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>4HE4_A Yellow fluorescent protein; phiYFP, yellow fluorescent protein, mutant variant of phiYFP, Thr-Tyr-Gly chromophore, beta-barrel, Biomarker, Thr-Tyr-Gly bicyclic chromophore, FLUORESCENT; HET: CRO; 2.05A {Phialidium sp. SL-2003} SCOP: l.1.1.1, d.22.1.0
</pre><pre>Probab=22.79  E-value=3.5e+02  Score=22.89  Aligned_cols=44  Identities=18%  Similarity=0.339  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +.+++...|++      +++.+..+++-.|-.|..||-+.
</pre><pre>T Consensus        90 ~eRt~~FeDG-------------------g~~t~~~~is~------e~~~~~~~v~~~G~~Fp~dGpVm  133 (244)
</pre><pre>T 4HE4_A           90 QERTITFEGD-------------------GNFKTRAEVTF------ENGSVYNRVKLNGQGFKKDGHVL  133 (244)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEeccCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>2AUW_A hypothetical protein NE0471; alpha-beta structure, Structural Genomics, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, UNKNOWN FUNCTION; HET: MSE, GOL; 1.85A {Nitrosomonas europaea} SCOP: a.35.1.10, d.331.1.1
</pre><pre>Probab=22.74  E-value=2.2e+02  Score=20.17  Aligned_cols=15  Identities=20%  Similarity=0.180  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEe
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYD   18 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Ye   18 (90)
</pre><pre>                      .+.|.|.||.+..||
</pre><pre>T Consensus        21 ~l~v~~~dG~~~~~d   35 (170)
</pre><pre>T 2AUW_A           21 RLRTTWSTGEVLEVD   35 (170)
</pre><pre>T ss_dssp             EEEEEETTSCEEEEE
</pre><pre>T ss_pred             EEEEEECCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>3CGL_B GFP-like fluorescent chromoprotein dsFP483; beta barrel, Chromophore, Luminescence, Photoprotein, FLUORESCENT PROTEIN; HET: CRQ; 2.09A {Discosoma striata} SCOP: d.22.1.1
</pre><pre>Probab=22.64  E-value=3.4e+02  Score=23.03  Aligned_cols=44  Identities=25%  Similarity=0.278  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus       102 ~eRt~~FEDg-------------------G~~t~~~~is~------e~~~~~~~~~~~G~~Fp~dgpvm  145 (241)
</pre><pre>T 3CGL_B          102 WERSMHFEDG-------------------GLCCITNDISL------TGNCFYYDIKFTGLNFPPNGPVV  145 (241)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEecCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>2A46_A GFP-like fluorescent chromoprotein amFP486; Beta Barrel, LUMINESCENT PROTEIN; HET: CR7; 1.65A {Anemonia majano} SCOP: d.22.1.0
</pre><pre>Probab=22.47  E-value=3.5e+02  Score=22.95  Aligned_cols=44  Identities=27%  Similarity=0.318  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+..+++-.|-.|..||-+.
</pre><pre>T Consensus       104 ~eRt~~FEDG-------------------g~~t~~~~isl------e~~~i~~~~~~~G~~Fp~dGpvm  147 (238)
</pre><pre>T 2A46_A          104 YERTFTYEDG-------------------GVATASWEISL------KGNCFEHKSTFHGVNFPADGPVM  147 (238)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>PF03958.19 ; Secretin_N ; Bacterial type II/III secretion system short domain
</pre><pre>Probab=22.43  E-value=2.8e+02  Score=15.90  Aligned_cols=20  Identities=25%  Similarity=0.225  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      ++.+..+..++.++.|.+.|
</pre><pre>T Consensus        23 ~~~~~~~~~~~~~~~l~i~g   42 (63)
</pre><pre>T Q9I3W2_PSEAE/2   23 EKSGLSVMADGARSGLMLKG   42 (63)
</pre><pre>T ss_pred             ccCCceEEEeCCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>TIGR03353 VI_chp_4; type VI secretion protein, VC_A0114 family. Work by Mougous, et al. (2006), describes IAHP-related loci as a type VI secretion system ().
</pre><pre>Probab=22.40  E-value=1.4e+02  Score=25.82  Aligned_cols=21  Identities=19%  Similarity=0.162  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CcceeEEeecCccEEEEecCC
</pre><pre>Q MBG6247344.1/9    1 SLQEMVITALDGARFRYDAAA   21 (90)
</pre><pre>Q Consensus         1 s~~~~~i~fPDGA~~~Yep~~   21 (90)
</pre><pre>                      ++.++..-||||..|+|+.+.
</pre><pre>T Consensus        56 ~i~~~~ai~pdG~~i~~~~~~   76 (439)
</pre><pre>T TIGR03353        56 ALTSASGIFPDGTPFDIPGDD   76 (439)
</pre><pre>T ss_pred             EEEEEEEECCCCCEEECCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>PF11765.10 ; Hyphal_reg_CWP ; Hyphally regulated cell wall protein N-terminal
</pre><pre>Probab=22.32  E-value=2.1e+02  Score=23.99  Aligned_cols=22  Identities=18%  Similarity=0.229  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             EEeecCc---------cEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9    6 VITALDG---------ARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         6 ~i~fPDG---------A~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      .|.|++.         ..++||+++|-|++.
</pre><pre>T Consensus       238 ~I~~~~~~~~~~~~~~~~~~Y~~~tGiLtv~  268 (331)
</pre><pre>T M3IL73_CANMX/1  238 KVGINLPLVGSLLPPLPSYDYDEKTGILTLR  268 (331)
</pre><pre>T ss_pred             EEEEeccccCCCCCCCCceEEeCCCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>2HPW_A Green fluorescent protein; Fluorescent protein, GFP, Structural Genomics, PSI, Protein Structure Initiative, Southeast Collaboratory for Structural Genomics, SECSG, LUMINESCENT; HET: GYS; 1.55A {Clytia gregaria} SCOP: d.22.1.1
</pre><pre>Probab=22.17  E-value=3.6e+02  Score=22.75  Aligned_cols=41  Identities=20%  Similarity=0.432  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNG   68 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||
</pre><pre>T Consensus        93 ~eR~~~FEDg-------------------g~~t~~~~is~------~~~~i~~~~~~~G~~Fp~dG  133 (233)
</pre><pre>T 2HPW_A           93 QDRIISFDND-------------------GQYDVKAKVTY------ENGTLYNRVTVKGTGFKSNG  133 (233)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEESCCTTS
</pre><pre>T ss_pred             EEEEEEEeCC-------------------CEEEEEEEEEE------ECCEEEEEEEEEEecCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>6M9Z_D Fluorescent protein lanFP6G; Fluorescent Protein Branchiostoma floridae Gly-Tyr-Gly chromophore, FLUORESCENT PROTEIN; HET: CR2; 1.2A {Branchiostoma floridae} SCOP: d.22.1.0
</pre><pre>Probab=22.03  E-value=3.5e+02  Score=22.60  Aligned_cols=44  Identities=25%  Similarity=0.401  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        86 ~eRt~~feDg-------------------g~lt~~~~is~------~~~~l~~~~~~~G~~Fp~dgpvm  129 (230)
</pre><pre>T 6M9Z_D           86 VHRTLQYEDG-------------------AFVTANLRYTY------EGSHIKGEFQVIGTGFPPDGPVM  129 (230)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEECCC-------------------CEEEEEEEEEE------ECCEEEEEEEEEeecCCCCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>6S68_A Aequorea cf. australis fluorescent protein 2 (AausFP2); GFP-like fluorescent protein, FLUORESCENT PROTEIN; HET: PIA; 2.06A {Aequorea australis}
</pre><pre>Probab=22.01  E-value=3.8e+02  Score=22.32  Aligned_cols=44  Identities=16%  Similarity=0.339  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        90 ~eRt~~feDg-------------------g~~~~~~~is~------~~~~~~~~~~~~G~~Fp~dGpvm  133 (230)
</pre><pre>T 6S68_A           90 QERKISYEND-------------------GTFDVRQEVTY------ENGALYNRVKFNGSGFRKDGNVL  133 (230)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEESCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------CEEEEEEEEEE------ECCEEEEEEEEEEeccCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>4Q9W_B GFP-like fluorescent chromoprotein cFP484; beta barrel, fluorescent protein, artificial metalloenzyme, metalloenzyme, bioconjugation, protein design, thermostable, temperature stable, solvent stable, robust; HET: PIA; 1.0A {Clavularia sp.} SCOP: l.1.1.1, d.22.1.0
</pre><pre>Probab=21.98  E-value=3.6e+02  Score=22.23  Aligned_cols=44  Identities=25%  Similarity=0.246  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .+-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        87 ~eRt~~feDg-------------------g~l~~~~~is~------e~~~i~~~~~~~G~~F~~dgpvm  130 (218)
</pre><pre>T 4Q9W_B           87 WERTMTFEDK-------------------GIVKVKSDISL------EEDSFIYEIYLKGENFPPNGPVM  130 (218)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>2C9J_A GREEN FLUORESCENT PROTEIN FP512; FLUORESCENT PROTEIN, BETA-BARREL, BIOLUMINESCENCE, LUMINESCENCE, LUMINESCENT PROTEIN; HET: CRQ; 1.35A {CERIANTHUS MEMBRANACEUS} SCOP: d.22.1.0
</pre><pre>Probab=21.96  E-value=3.7e+02  Score=22.28  Aligned_cols=44  Identities=25%  Similarity=0.343  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .+-.+.|.||                   +.+++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        87 ~eRt~~feDg-------------------g~~t~~~~it~------e~~~~~~~~~~~G~~F~~dgpvm  130 (223)
</pre><pre>T 2C9J_A           87 WNRRIEFEDG-------------------GVINMSSDITY------KDKVLHGDVWALGVNFPPNGPVM  130 (223)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>2G6Y_B green fluorescent protein 2; green fluorescent protein, natural chromophore, rapid maturation, beta-can, LUMINESCENT PROTEIN; HET: CR2; 1.6A {Pontellina plumata} SCOP: l.1.1.1, d.22.1.0
</pre><pre>Probab=21.72  E-value=3.8e+02  Score=22.10  Aligned_cols=44  Identities=16%  Similarity=0.254  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        83 ~eRt~~feDg-------------------g~~~~~~~is~------e~~~~~~~~~~~G~~Fp~dgpvm  126 (217)
</pre><pre>T 2G6Y_B           83 NTRIEKYEDG-------------------GVLHVSFSYRY------EAGRVIGDFKVMGTGFPEDSVIF  126 (217)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTCGGG
</pre><pre>T ss_pred             EEEEEEeCCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>2G6Y_C green fluorescent protein 2; green fluorescent protein, natural chromophore, rapid maturation, beta-can, LUMINESCENT PROTEIN; HET: CR2; 1.6A {Pontellina plumata} SCOP: d.22.1.0
</pre><pre>Probab=21.72  E-value=3.8e+02  Score=22.10  Aligned_cols=44  Identities=16%  Similarity=0.254  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        83 ~eRt~~feDg-------------------g~~~~~~~is~------e~~~~~~~~~~~G~~Fp~dgpvm  126 (217)
</pre><pre>T 2G6Y_C           83 NTRIEKYEDG-------------------GVLHVSFSYRY------EAGRVIGDFKVMGTGFPEDSVIF  126 (217)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTCGGG
</pre><pre>T ss_pred             EEEEEEeCCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>PF06505.13 ; XylR_N ; Activator of aromatic catabolism
</pre><pre>Probab=21.72  E-value=2.3e+02  Score=18.52  Aligned_cols=25  Identities=12%  Similarity=0.211  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             cEEEEecCCCceEEEceeEEEEEec
</pre><pre>Q MBG6247344.1/9   13 ARFRYDAAAGALEAVGMKTARVSAE   37 (90)
</pre><pre>Q Consensus        13 A~~~Yep~~gaL~v~GiKTAtV~As   37 (90)
</pre><pre>                      ..+++||+.+.+...|.+.+.+.+.
</pre><pre>T Consensus         4 ~~~~~~~~~g~~~~~g~r~v~~~~~   28 (101)
</pre><pre>T A1WLM8_VEREI/1    4 KLLRFEPDSGAIWLGERRMVLLHTA   28 (101)
</pre><pre>T ss_pred             hceEEcCCCCeEEECCEEEEEEeHH
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>4DKN_B AMPHIOXUS GREEN FLUORESCENT PROTEIN, GFPA1; CHROMOPHORE, BETA-CAN, FLUORESCENCE, FLUORESCENT PROTEIN; HET: CR2; 1.35A {Branchiostoma floridae} SCOP: d.22.1.0
</pre><pre>Probab=21.63  E-value=3.8e+02  Score=22.24  Aligned_cols=44  Identities=20%  Similarity=0.381  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        88 ~eRt~~feDg-------------------g~~t~~~~is~------e~~~~~~~~~~~G~~Fp~dgpvm  131 (221)
</pre><pre>T 4DKN_B           88 VYRVFDFEDG-------------------GKLSTEFKYSY------EGSHIKADMKLMGSGFPDDGPVM  131 (221)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEccC-------------------CEEEEEEEEEE------eCCEEEEEEEEEeccCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>3KF8_D Protein ten1; OB fold, STRUCTURAL PROTEIN; 2.4A {Candida tropicalis MYA-3404}
</pre><pre>Probab=21.60  E-value=1.7e+02  Score=22.14  Aligned_cols=16  Identities=0%  Similarity=-0.128  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             ccEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9   12 GARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus        12 GA~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      |-+.+||+.++.|.+.
</pre><pre>T Consensus        30 GcV~~Yd~~ta~L~L~   45 (123)
</pre><pre>T 3KF8_D           30 AQVKDFIPHESTIVID   45 (123)
</pre><pre>T ss_dssp             EEEEEEETTTTEEEEE
</pre><pre>T ss_pred             EEEEEEcCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>5GZT_A Chitinase; HYDROLASE; HET: FMT; 2.1A {Paenibacillus sp. FPU-7}
</pre><pre>Probab=21.54  E-value=5.9e+02  Score=18.52  Aligned_cols=47  Identities=23%  Similarity=0.404  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             EEecCCCceEEEceeEEEEEeccEEEeeCCeEe-cCceEeeeeeecC----CeEEECCEEee
</pre><pre>Q MBG6247344.1/9   16 RYDAAAGALEAVGMKTARVSAETRILLDTPEVQ-KGGEMHGNIAHSG----GAFTSNGVQID   72 (90)
</pre><pre>Q Consensus        16 ~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~-~Gg~m~G~ithsg----G~~ssNGv~v~   72 (90)
</pre><pre>                      .|.++.+.-.+.|          .+.+.+|.|. +.-.+.|++.-..    |.++++++.|.
</pre><pre>T Consensus        46 ~y~~~~~~~~v~g----------nv~I~~~~v~L~n~~i~GdL~i~~gvg~g~v~L~nv~V~   97 (108)
</pre><pre>T 5GZT_A           46 TYGPASGTETIDG----------NVKVTVPGVTLRNLVIKGDLLLSEGVGSGDVTLDKVSVH   97 (108)
</pre><pre>T ss_dssp             SCSCSSSCEECCS----------CEECCSTTCCEESEEBSSCEECTGGGTTSCEEEESCBBS
</pre><pre>T ss_pred             cccCCCCceEecC----------CEEEcCCCeEeeccEEECcEEEcCCcCCCcEEEECeEEe
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>PF03463.17 ; eRF1_1 ; eRF1 domain 1
</pre><pre>Probab=21.37  E-value=2.2e+02  Score=20.31  Aligned_cols=16  Identities=19%  Similarity=0.586  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEecCCCceEEEce
</pre><pre>Q MBG6247344.1/9   14 RFRYDAAAGALEAVGM   29 (90)
</pre><pre>Q Consensus        14 ~~~Yep~~gaL~v~Gi   29 (90)
</pre><pre>                      .++|+|.++.|.+.|.
</pre><pre>T Consensus        72 ~~~~~~~~~~L~i~G~   87 (130)
</pre><pre>T D2UZ11_NAEGR/1   72 KIDFDPNGGSFRVSGK   87 (130)
</pre><pre>T ss_pred             EEEecCCCCeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>3AI5_A yeast enhanced green fluorescent protein,Ubiquitin; green fluorescent protein, ubiquitin, fusion protein, Fluorescent Protein, transcription; HET: CR2, EDO; 1.4A {Aequorea victoria}
</pre><pre>Probab=21.28  E-value=4.3e+02  Score=21.40  Aligned_cols=41  Identities=22%  Similarity=0.335  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNG   68 (90)
</pre><pre>                      .|-.+.|.||+                   +++|...|++      +++.+.+++.-.|-.|..||
</pre><pre>T Consensus        95 ~~r~~~f~d~~-------------------~~~~~~~~~~------~~~~~~~~~~~~g~~~~~~~  135 (307)
</pre><pre>T 3AI5_A           95 QERTIFFKDDG-------------------NYKTRAEVKF------EGDTLVNRIELKGIDFKEDG  135 (307)
</pre><pre>T ss_dssp             EEEEEEETTSC-------------------EEEEEEEEEE------ETTEEEEEEEEEEECCCTTS
</pre><pre>T ss_pred             EEEEEEEecCc-------------------EEEEEEEEEE------eCCEEEEEEEEEEecCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>4OHS_D FAR-RED FLUORESCENT PROTEIN AQ143; far-red, Beta Barrel, Red Fluorescent Protein, RFP, FLUORESCENT PROTEIN; HET: CH6, NRQ; 2.19A {Actinia Equina} SCOP: d.22.1.0
</pre><pre>Probab=21.26  E-value=3.6e+02  Score=22.73  Aligned_cols=44  Identities=20%  Similarity=0.268  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        97 ~eRt~~FeDG-------------------g~~t~~~~is~------~~~~~~~~~~~~G~~F~~dgpvm  140 (237)
</pre><pre>T 4OHS_D           97 WERTQIFEDG-------------------GSLTIHQDTSL------QGNNFIFKVNVIGANFPANGPVM  140 (237)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>4OHS_E FAR-RED FLUORESCENT PROTEIN AQ143; far-red, Beta Barrel, Red Fluorescent Protein, RFP, FLUORESCENT PROTEIN; HET: CH6, NRQ; 2.19A {Actinia Equina} SCOP: d.22.1.0
</pre><pre>Probab=21.26  E-value=3.6e+02  Score=22.73  Aligned_cols=44  Identities=20%  Similarity=0.268  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        97 ~eRt~~FeDG-------------------g~~t~~~~is~------~~~~~~~~~~~~G~~F~~dgpvm  140 (237)
</pre><pre>T 4OHS_E           97 WERTQIFEDG-------------------GSLTIHQDTSL------QGNNFIFKVNVIGANFPANGPVM  140 (237)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>4NKB_B Probable serine/threonine-protein kinase zyg-1; cryptic polo box, Centriole biogenesis, centrosomes, TRANSFERASE; HET: DTT; 2.3A {Caenorhabditis elegans}
</pre><pre>Probab=21.16  E-value=3.2e+02  Score=22.65  Aligned_cols=23  Identities=9%  Similarity=0.081  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      .+.+.| ||+++.|...++.+++.
</pre><pre>T Consensus       153 Dfe~~F-~G~kv~~~~~~~~i~i~  175 (230)
</pre><pre>T 4NKB_B          153 DLRIKL-PRSVIVRKMDNGEIFNC  175 (230)
</pre><pre>T ss_dssp             CEEEEE-TTEEEEECTTTCCEEEC
</pre><pre>T ss_pred             cEEEEe-CCceEEEEcCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>PF03968.16 ; LptD_N ; LptA/(LptD N-terminal domain) LPS transport protein
</pre><pre>Probab=21.05  E-value=4.6e+02  Score=17.09  Aligned_cols=34  Identities=12%  Similarity=0.142  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe
</pre><pre>Q MBG6247344.1/9   12 GARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ   48 (90)
</pre><pre>Q Consensus        12 GA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~   48 (90)
</pre><pre>                      +..+.|+.+.+.+.+.|  .+.+. .....+.++.++
</pre><pre>T Consensus        72 a~~~~~~~~~~~~~~~g--~~~~~-~~~~~~~a~~i~  105 (111)
</pre><pre>T Q8P714_XANCP/3   72 ADNIEYKVADEIIILTG--NYKVD-SPKGTNAGQRMV  105 (111)
</pre><pre>T ss_pred             eeEEEEEcCCCEEEEEc--ceEEe-CCCCeEEcceEE
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>PF06905.15 ; FAIM1 ; Fas apoptotic inhibitory molecule (FAIM1)
</pre><pre>Probab=21.00  E-value=6.5e+02  Score=19.54  Aligned_cols=65  Identities=17%  Similarity=0.189  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe-cCceEeeeeeecCCeEEECCEEeee
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ-KGGEMHGNIAHSGGAFTSNGVQIDD   73 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~-~Gg~m~G~ithsgG~~ssNGv~v~~   73 (90)
</pre><pre>                      ++.|...++-.++|+     |.|.|..-.+.....+=.+.+=++. .|...+=-+.+..-.+-.||..+++
</pre><pre>T Consensus        59 ~i~I~~~~~~~y~Y~-----L~VdG~~~~~~~~~~~k~~~~W~~~l~g~~~rIvl~k~tl~VwvnG~~vet  124 (172)
</pre><pre>T B3S9C3_TRIAD/6   59 TINIDAAGGFAYEYS-----LEVNGKSLKKFVENRKKSSKTWAFMLTGEPTRVVFEKDSLDVWVNGEKLNT  124 (172)
</pre><pre>T ss_pred             EEEEEeCCCceEEEE-----EEECCEEHHHhhhcccccceeEEEEeCCccEEEEEecCCeEEEECCEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>PF16461.7 ; Phage_TTP_12 ; Lambda phage tail tube protein, TTP
</pre><pre>Probab=20.96  E-value=2.8e+02  Score=19.04  Aligned_cols=45  Identities=4%  Similarity=-0.034  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             eeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEe
</pre><pre>Q MBG6247344.1/9    4 EMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQ   48 (90)
</pre><pre>Q Consensus         4 ~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~   48 (90)
</pre><pre>                      .+.+.+|||+.+.|...=..+...+=..-.++++-++.++.+-+.
</pre><pre>T Consensus        90 ~~~~~~~~g~~~~~~~~v~~~~~~~~~~~~~~~s~t~~~~g~~~~  134 (135)
</pre><pre>T A0A0B8XVC4_9SP   90 AYKIRFPNGTVDVFRGWVSSIGKAVTAKEVITRTVKVTNVGRPSM  134 (135)
</pre><pre>T ss_pred             EEEEEcCCCcEEEEEEEEEEccCCccCCCeEEEEEEEEEeccccc
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>d1xmza_ d.22.1.1 (A:) automated matches {Sea anemone (Anemonia sulcata) [TaxId: 6108]}
</pre><pre>Probab=20.94  E-value=3.9e+02  Score=22.83  Aligned_cols=44  Identities=23%  Similarity=0.266  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|..+.|.||                   ++++++..|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        87 ~eRt~~FEDG-------------------gv~t~~~~is~------~~~~~~~~~~~~G~nFp~dGpVm  130 (229)
</pre><pre>T d1xmza_          87 WERTTTYEDG-------------------GFLTAHQDTSL------DGDCLVYKVKILGNNFPADGPVM  130 (229)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEecCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>PF13101.8 ; DUF3945 ; Protein of unknown function (DUF3945)
</pre><pre>Probab=20.92  E-value=2.3e+02  Score=18.21  Aligned_cols=14  Identities=14%  Similarity=0.128  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             cEEEEecCCCceEE
</pre><pre>Q MBG6247344.1/9   13 ARFRYDAAAGALEA   26 (90)
</pre><pre>Q Consensus        13 A~~~Yep~~gaL~v   26 (90)
</pre><pre>                      +.++|||+++.|..
</pre><pre>T Consensus         5 ~~v~~D~~Tn~l~~   18 (59)
</pre><pre>T Q8ABK7_BACTN/2    5 SFVSVDKVTNELVS   18 (59)
</pre><pre>T ss_pred             eEEEEeCCCCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 146
</pre><pre>>4DKM_F Amphioxus Green Fluorescent Protein, GFPc1a; fluorescent protein, beta-can, chromophore; HET: CR2; 1.95A {Branchiostoma floridae} SCOP: d.22.1.0
</pre><pre>Probab=20.90  E-value=3.8e+02  Score=22.11  Aligned_cols=44  Identities=25%  Similarity=0.455  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        82 ~eRt~~feDg-------------------g~~~~~~~is~------e~~~~~~~~~~~G~~Fp~dgpvm  125 (214)
</pre><pre>T 4DKM_F           82 VHRTIQFEDG-------------------ASLTGNYRYSY------DGGHIKGEFHVVGSGFPADGPVM  125 (214)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------ECCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 147
</pre><pre>>d4dkna_ d.22.1.0 (A:) automated matches {Branchiostoma floridae [TaxId: 7739]}
</pre><pre>Probab=20.88  E-value=3.8e+02  Score=22.66  Aligned_cols=44  Identities=20%  Similarity=0.381  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|..+.|.||                   +.++++..|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        85 ~eRt~~FEDG-------------------g~~t~~~~is~------e~~~~~~~~~~~G~nFp~dgpVm  128 (218)
</pre><pre>T d4dkna_          85 VYRVFDFEDG-------------------GKLSTEFKYSY------EGSHIKADMKLMGSGFPDDGPVM  128 (218)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEECCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 148
</pre><pre>>4LD1_A Uncharacterized protein; beta sheet, G-box, Centriole organisation, structural protein, protein binding; HET: PGE, PG4, MPD; 1.44A {Danio rerio}
</pre><pre>Probab=20.81  E-value=2.6e+02  Score=19.34  Aligned_cols=20  Identities=10%  Similarity=-0.071  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCce
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGAL   24 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL   24 (90)
</pre><pre>                      +.+.|+||....++|+...|
</pre><pre>T Consensus       163 ~~~~~~~G~~~~~~~~G~~~  182 (184)
</pre><pre>T 4LD1_A          163 QETQYPTGRVRLKDPQGKVI  182 (184)
</pre><pre>T ss_dssp             EEEECCC-------------
</pre><pre>T ss_pred             EEEEcCCCCEEEECCCCCEe
</pre><pre>
</pre><pre>
</pre><pre>No 149
</pre><pre>>d4ohsa_ d.22.1.0 (A:) automated matches {European sea anemone (Actinia equina) [TaxId: 6106]}
</pre><pre>Probab=20.78  E-value=3.7e+02  Score=22.85  Aligned_cols=44  Identities=20%  Similarity=0.268  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   ++++|+..|++      +++.+..+++-.|-.|..||.+.
</pre><pre>T Consensus        93 ~eRt~~FEDG-------------------g~~t~~~~is~------e~~~~~~~~~~~G~nFp~dGpVm  136 (225)
</pre><pre>T d4ohsa_          93 WERTQIFEDG-------------------GSLTIHQDTSL------QGNNFIFKVNVIGANFPANGPVM  136 (225)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 150
</pre><pre>>2C9I_B GREEN FLUORESCENT PROTEIN ASFP499; FLUORESCENT PROTEIN, BETA-BARREL, BIOLUMINESCENCE, LUMINESCENCE, LUMINESCENT PROTEIN; HET: CRQ; 1.82A {ANEMONIA SULCATA} SCOP: d.22.1.1
</pre><pre>Probab=20.66  E-value=3.7e+02  Score=22.29  Aligned_cols=44  Identities=30%  Similarity=0.320  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        88 ~eRt~~feDg-------------------g~~~~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  131 (226)
</pre><pre>T 2C9I_B           88 WERVSTYEDG-------------------GVLSATQETSL------QGDCIICKVKVLGTNFPANGPVM  131 (226)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 151
</pre><pre>>2C9I_D GREEN FLUORESCENT PROTEIN ASFP499; FLUORESCENT PROTEIN, BETA-BARREL, BIOLUMINESCENCE, LUMINESCENCE, LUMINESCENT PROTEIN; HET: CRQ; 1.82A {ANEMONIA SULCATA} SCOP: d.22.1.1
</pre><pre>Probab=20.66  E-value=3.7e+02  Score=22.29  Aligned_cols=44  Identities=30%  Similarity=0.320  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +++++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        88 ~eRt~~feDg-------------------g~~~~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  131 (226)
</pre><pre>T 2C9I_D           88 WERVSTYEDG-------------------GVLSATQETSL------QGDCIICKVKVLGTNFPANGPVM  131 (226)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 152
</pre><pre>>d2ejoa_ d.22.1.0 (A:) automated matches {Fungia concinna [TaxId: 496660]}
</pre><pre>Probab=20.66  E-value=3.8e+02  Score=22.66  Aligned_cols=44  Identities=23%  Similarity=0.331  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   ++++++..|++      +++.+..+++-.|-.|..||.+.
</pre><pre>T Consensus        91 ~eRt~~FeDG-------------------g~~t~~~~is~------~~~~~~~~~~~~G~nFp~dGpvm  134 (217)
</pre><pre>T d2ejoa_          91 WERSLEFEDG-------------------GSASVSAHISL------RGNTFYHKSKFTGVNFPADGPIM  134 (217)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEeCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 153
</pre><pre>>2IB5_B Chromo protein; beta barrel, alpha helix, chromoprotein, chromophore, blue, GFP-like protein, LUMINESCENT PROTEIN; HET: MSE, CRQ, PO4; 1.8A {Cnidopus japonicus} SCOP: d.22.1.1
</pre><pre>Probab=20.47  E-value=3.8e+02  Score=22.46  Aligned_cols=44  Identities=27%  Similarity=0.289  Sum_probs=0.0  Template_Neff=6.200
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +.+++...|++      +++.+.++++-.|-.|..||.+.
</pre><pre>T Consensus        91 ~eRt~~feDg-------------------g~~~~~~~is~------~~~~~~~~~~~~G~~Fp~dgpvm  134 (233)
</pre><pre>T 2IB5_B           91 WERTTIYEDG-------------------AYLTTQQETKL------DGNCLVYNIKILGCNFPPNGPVM  134 (233)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEEecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 154
</pre><pre>>d3cgla_ d.22.1.1 (A:) automated matches {Discosoma striata}
</pre><pre>Probab=20.46  E-value=4e+02  Score=22.69  Aligned_cols=44  Identities=25%  Similarity=0.278  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|..+.|.||                   ++++++..|++      +++.+..+++-.|-.|..||-+.
</pre><pre>T Consensus        93 ~eRt~~FEDG-------------------gv~t~~~~is~------~~~~~~~~v~~~G~nFp~dGPVm  136 (226)
</pre><pre>T d3cgla_          93 WERSMHFEDG-------------------GLCCITNDISL------TGNCFYYDIKFTGLNFPPNGPVV  136 (226)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEeecCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 155
</pre><pre>>d4dkma_ d.22.1.0 (A:) automated matches {Branchiostoma floridae [TaxId: 7739]}
</pre><pre>Probab=20.46  E-value=4e+02  Score=22.51  Aligned_cols=44  Identities=25%  Similarity=0.455  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||                   +.++++..|++      +++.+.++++-.|-.|.-||.+.
</pre><pre>T Consensus        84 ~eRt~~FEDG-------------------g~~t~~~~is~------e~~~~~~~~~~~G~nFp~dGpVm  127 (216)
</pre><pre>T d4dkma_          84 VHRTIQFEDG-------------------ASLTGNYRYSY------DGGHIKGEFHVVGSGFPADGPVM  127 (216)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEEcCC-------------------CEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 156
</pre><pre>>6LOF_B GFP-like fluorescent chromoprotein FP538; ZsYellow, fluorescent protein, yellow; HET: NFA, CH7; 2.6A {Zoanthus sp.}
</pre><pre>Probab=20.46  E-value=5.8e+02  Score=20.39  Aligned_cols=46  Identities=15%  Similarity=0.153  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECCEEe
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNGVQI   71 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNGv~v   71 (90)
</pre><pre>                      .|-.+.|.||++                 ++..+.-++++      +++.+.++++-.|-.|..||-+.
</pre><pre>T Consensus        26 ~eRt~~FEDGG~-----------------~t~~~~i~~~~------e~~~i~~~~~~~G~gFp~dGpvm   71 (164)
</pre><pre>T 6LOF_B           26 WGRSFLFEDGAV-----------------CICNVDITVSV------KENCIYHKSIFNGVNFPADGPVM   71 (164)
</pre><pre>T ss_dssp             EEEEEEETTSCE-----------------EEEEEEEEEET------TTTEEEEEEEEEEECCCTTSTTT
</pre><pre>T ss_pred             EEEEEEECCCcE-----------------EEEEEEEEEEE------eCCEEEEEEEEEeeCCCCCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 157
</pre><pre>>6GP0_A Green to red photoconvertible GFP-like protein EosFP; Fluorescent protein; HET: CR8, IEY, NFA; 1.5A {Lobophyllia hemprichii} SCOP: l.1.1.1, d.22.1.0
</pre><pre>Probab=20.41  E-value=3.9e+02  Score=22.99  Aligned_cols=41  Identities=27%  Similarity=0.407  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             ceeEEeecCccEEEEecCCCceEEEceeEEEEEeccEEEeeCCeEecCceEeeeeeecCCeEEECC
</pre><pre>Q MBG6247344.1/9    3 QEMVITALDGARFRYDAAAGALEAVGMKTARVSAETRILLDTPEVQKGGEMHGNIAHSGGAFTSNG   68 (90)
</pre><pre>Q Consensus         3 ~~~~i~fPDGA~~~Yep~~gaL~v~GiKTAtV~As~sITldtP~V~~Gg~m~G~ithsgG~~ssNG   68 (90)
</pre><pre>                      .|-.+.|.||                   ++++|...|++      +++.+.++++-.|-.|..||
</pre><pre>T Consensus       120 ~eRt~~FEDG-------------------g~lt~~~~is~------e~~~i~~~~~~~G~~Fp~dG  160 (257)
</pre><pre>T 6GP0_A          120 WERSLTFEDG-------------------GICNARNDITM------EGDTFYNKVRFYGTNFPANG  160 (257)
</pre><pre>T ss_dssp             EEEEEEETTS-------------------CEEEEEEEEEE------ETTEEEEEEEEEEECCCTTS
</pre><pre>T ss_pred             EEEEEEEeCC-------------------cEEEEEEEEEE------eCCEEEEEEEEEEecCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 158
</pre><pre>>4MPZ_A Sas-4; beta-spine, centrosome biogenesis, STRUCTURAL PROTEIN; 2.701A {Drosophila melanogaster}
</pre><pre>Probab=20.35  E-value=2.6e+02  Score=21.55  Aligned_cols=20  Identities=10%  Similarity=-0.069  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCce
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGAL   24 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL   24 (90)
</pre><pre>                      ..+.||||.+..++++...+
</pre><pre>T Consensus       186 ~~~~~~dG~~~~~~~dG~~~  205 (209)
</pre><pre>T 4MPZ_A          186 QETRYSNGRVRLKDKDGKLI  205 (209)
</pre><pre>T ss_dssp             --------------------
</pre><pre>T ss_pred             EEEEeCCCCEEEECCCCCEE
</pre><pre>
</pre><pre>
</pre><pre>No 159
</pre><pre>>4MPZ_A Sas-4; beta-spine, centrosome biogenesis, STRUCTURAL PROTEIN; 2.701A {Drosophila melanogaster}
</pre><pre>Probab=20.30  E-value=3.8e+02  Score=20.63  Aligned_cols=23  Identities=17%  Similarity=0.024  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             eEEeecCccEEEEecCCCceEEE
</pre><pre>Q MBG6247344.1/9    5 MVITALDGARFRYDAAAGALEAV   27 (90)
</pre><pre>Q Consensus         5 ~~i~fPDGA~~~Yep~~gaL~v~   27 (90)
</pre><pre>                      ..+.||||.+.+++|+.....+.
</pre><pre>T Consensus        37 ~~~~~~dG~~~~~~~dG~~~~~~   59 (209)
</pre><pre>T 4MPZ_A           37 KDIWYPNGNLKKISADGMNLRML   59 (209)
</pre><pre>T ss_dssp             EEEECTTSCEEEECTTSCEEEEE
</pre><pre>T ss_pred             EEEEeCCCCEEEEeCCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 160
</pre><pre>>5OXZ_B NEQ528; Intein complex 1, SPLICING; 1.2A {Nanoarchaeum equitans (strain Kin4-M)}
</pre><pre>Probab=20.28  E-value=2.2e+02  Score=17.67  Aligned_cols=18  Identities=22%  Similarity=0.608  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             eeeecCCeEEECCEEeee
</pre><pre>Q MBG6247344.1/9   56 NIAHSGGAFTSNGVQIDD   73 (90)
</pre><pre>Q Consensus        56 ~ithsgG~~ssNGv~v~~   73 (90)
</pre><pre>                      ++.-..|+|-.||.++|.
</pre><pre>T Consensus        13 dlstesgkfyvnglvlhn   30 (33)
</pre><pre>T 5OXZ_B           13 DLSTESGKFYVNGLVLHN   30 (33)
</pre><pre>T ss_dssp             EEEETTSEEEETTEEEEC
</pre><pre>T ss_pred             EccCCCceEEEeeEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 161
</pre><pre>>4G08_A Protein InvG; Ring-building motif, protein secretion, PrgH, CELL INVASION; 1.801A {Salmonella enterica subsp. enterica serovar Typhimurium}
</pre><pre>Probab=20.16  E-value=2.3e+02  Score=19.34  Aligned_cols=20  Identities=20%  Similarity=0.184  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             ecCccEEEEecCCCceEEEc
</pre><pre>Q MBG6247344.1/9    9 ALDGARFRYDAAAGALEAVG   28 (90)
</pre><pre>Q Consensus         9 fPDGA~~~Yep~~gaL~v~G   28 (90)
</pre><pre>                      +.+...+.+|+.++.|.|.|
</pre><pre>T Consensus       113 ~~~~~~~~~~~~~n~l~v~~  132 (159)
</pre><pre>T 4G08_A          113 YNKNYPLRGDNRKGTFYVSG  132 (159)
</pre><pre>T ss_dssp             CCTTSCCEECTTSSEEEEEE
</pre><pre>T ss_pred             CCCCCCcEEeCCCCeEEEEc
</pre><pre>
</pre><pre>
</pre>