<pre>Query         WP_214195415.1/96-205 
</pre><pre>Match_columns 110
</pre><pre>No_of_seqs    25 out of 33
</pre><pre>Neff          3.35768
</pre><pre>Searched_HMMs 107766
</pre><pre>Date          Tue Aug 24 15:48:22 2021
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/scope70/scope70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -d /cluster/toolkit/production/databases/hh-suite/TIGRFAM/TIGRFAM -o ../results/9108424.hhr -oa3m ../results/9108424.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 4S36_A Phage baseplate protein  99.4 2.3E-11 2.1E-16   79.3  11.1   85    4-96      1-90  (90)
</pre><pre>  2 3QR8_A Baseplate assembly prot  99.2   2E-10 1.8E-15   82.4   9.4   36    3-40     97-132 (211)
</pre><pre>  3 3QR7_B Baseplate assembly prot  99.2 2.8E-10 2.6E-15   75.8   9.5   35    3-39      1-35  (115)
</pre><pre>  4 3AQJ_A Baseplate assembly prot  99.2   7E-10 6.5E-15   77.5  10.0  105    2-107    19-134 (134)
</pre><pre>  5 4S37_B Phage baseplate protein  98.9 1.5E-07 1.3E-12   66.4  11.2   87    2-96     94-185 (185)
</pre><pre>  6 4S37_O Phage baseplate protein  98.9 1.5E-07 1.3E-12   66.4  11.2   87    2-96     94-185 (185)
</pre><pre>  7 3VTN_A Protein gp45; beta-heli  98.7 4.6E-07 4.2E-12   59.0   9.0   94   10-107     3-101 (101)
</pre><pre>  8 3VTO_B Protein gp45; beta-heli  98.7 4.9E-07 4.5E-12   62.4   8.5   96    9-108     8-107 (115)
</pre><pre>  9 3VTO_C Protein gp45; beta-heli  98.7 4.9E-07 4.5E-12   62.4   8.5   96    9-108     8-107 (115)
</pre><pre> 10 TIGR01644 phage_P2_V; phage ba  98.5 5.6E-06 5.2E-11   61.0  10.5   95    2-105    87-190 (190)
</pre><pre> 11 3PQH_A gene product 138; Beta-  98.3 1.5E-05 1.4E-10   56.3   8.4   90    2-107    18-123 (127)
</pre><pre> 12 3PQI_A gene product 138; Beta-  98.1 6.9E-05 6.4E-10   57.2   8.7   96    2-109   138-245 (247)
</pre><pre> 13 4RU3_A puncturing protein gp41  96.8   0.019 1.7E-07   44.1   7.2   71    2-107   144-220 (221)
</pre><pre> 14 6P2A_C CHIMERA OF BACTERIOPHAG  96.1    0.13 1.2E-06   38.8   8.2   83   25-107    47-147 (148)
</pre><pre> 15 6P2A_E CHIMERA OF BACTERIOPHAG  96.1    0.13 1.2E-06   38.8   8.2   83   25-107    47-147 (148)
</pre><pre> 16 PF04519.15 ; Bactofilin ; Poly  92.2     1.3 1.2E-05   29.8   5.2   48   28-78     12-59  (97)
</pre><pre> 17 PF10833.10 ; DUF2572 ; Protein  90.6       2 1.9E-05   35.0   5.8   52   27-78    140-191 (220)
</pre><pre> 18 PF19119.2 ; DUF5803 ; Family o  89.5     3.1 2.9E-05   34.5   6.2   49    7-55     46-106 (198)
</pre><pre> 19 PF13619.8 ; KTSC ; KTSC domain  86.6     4.1 3.8E-05   25.3   4.1   26   14-39      6-31  (58)
</pre><pre> 20 PF03961.15 ; FapA ; Flagellar   85.2      19 0.00017   31.7   8.6   73    4-80    144-223 (441)
</pre><pre> 21 6RBK_C Afp8; Anti-feeding prop  84.9      13 0.00012   31.1   7.3   72    2-76    446-523 (529)
</pre><pre> 22 6RIA_K bactofilin; Prokaryotic  84.4     6.2 5.8E-05   28.2   4.7   42   36-78     30-72  (123)
</pre><pre> 23 6RIA_L bactofilin; Prokaryotic  84.4     6.2 5.8E-05   28.2   4.7   42   36-78     30-72  (123)
</pre><pre> 24 PF18715.3 ; Phage_spike ; Phag  81.0      18 0.00016   21.7   5.3   47   26-73      4-52  (53)
</pre><pre> 25 2N3D_A Bactofilin A; BacA, Bac  79.0      18 0.00016   28.9   5.8   47   28-78     51-97  (184)
</pre><pre> 26 4RGI_A Uncharacterized protein  78.4      13 0.00012   24.6   4.2   25   15-39     16-40  (73)
</pre><pre> 27 PF20000.1 ; fvmX2 ; FtsH terna  73.9      30 0.00028   30.5   6.3   56    2-58     68-125 (260)
</pre><pre> 28 2PI2_F Replication protein A 1  71.2      67 0.00062   24.7   6.9   58   13-70     50-111 (142)
</pre><pre> 29 5OPQ_C 3,6-anhydro-D-galactosi  68.2      51 0.00047   30.0   6.6   49    2-50    634-689 (693)
</pre><pre> 30 PF12182.10 ; DUF3642 ; Bacteri  68.1      50 0.00046   25.2   5.5   50   12-64      6-59  (86)
</pre><pre> 31 6J0M_C Pvc8; assembly, Photorh  67.3      97  0.0009   26.3   7.6   65    3-76    454-532 (538)
</pre><pre> 32 3KDF_C Replication protein A 1  66.4      86  0.0008   22.8   6.8   58   13-70     29-90  (121)
</pre><pre> 33 1EW4_A CYAY PROTEIN; Friedreic  66.2      25 0.00023   25.4   3.6   28   10-37     24-51  (106)
</pre><pre> 34 6RIA_K bactofilin; Prokaryotic  65.0      39 0.00037   24.3   4.4   45   33-78     78-122 (123)
</pre><pre> 35 6RIA_L bactofilin; Prokaryotic  65.0      39 0.00037   24.3   4.4   45   33-78     78-122 (123)
</pre><pre> 36 PF13600.8 ; DUF4140 ; N-termin  64.1      33 0.00031   23.6   3.8   28    7-39      1-29  (100)
</pre><pre> 37 PF20003.1 ; fvmX5 ; FtsH terna  64.0      39 0.00036   28.1   4.8   34   12-45     64-97  (188)
</pre><pre> 38 PF10049.11 ; DUF2283 ; Protein  61.9      26 0.00024   21.6   2.7   17   16-32      2-18  (48)
</pre><pre> 39 PF07202.15 ; Tcp10_C ; T-compl  61.3      51 0.00047   19.1   3.7   29    3-32      2-30  (35)
</pre><pre> 40 PF12992.9 ; DUF3876 ; Domain o  61.0      36 0.00033   24.2   3.6   32    3-36     58-92  (93)
</pre><pre> 41 d1hf2a1 b.80.3.1 (A:100-206) C  57.2      23 0.00022   26.8   2.3   35   44-78     11-46  (107)
</pre><pre> 42 d2icya1 b.81.1.4 (A:384-466) U  57.1      16 0.00015   26.5   1.4   26   51-76     37-62  (83)
</pre><pre> 43 4HS5_B Protein CyaY; pFXN, fra  56.9      55 0.00051   23.5   4.0   27   12-38     24-50  (105)
</pre><pre> 44 PF15998.7 ; DUF4773 ; Domain o  56.3      72 0.00066   23.3   4.6   30   15-56     27-56  (116)
</pre><pre> 45 d4hs5a_ d.82.2.0 (A:) automate  55.8      47 0.00044   24.5   3.6   22   16-37     28-49  (105)
</pre><pre> 46 4GNX_X Putative uncharacterize  52.2 1.6E+02  0.0015   21.4   6.2   60   13-74     25-94  (114)
</pre><pre> 47 4JPD_A Protein CyaY; SSGCID, S  51.5      79 0.00073   23.0   4.1   29   10-38     29-57  (112)
</pre><pre> 48 d4d6ga2 b.24.2.0 (A:839-1003)   51.4      62 0.00057   26.9   4.0   25   14-38    138-162 (165)
</pre><pre> 49 4LD1_A Uncharacterized protein  50.9 1.1E+02   0.001   21.8   4.7   36    7-43    146-181 (184)
</pre><pre> 50 6QVI_B ComZ; competence pilin,  50.7 1.3E+02  0.0012   27.8   6.1   52   26-77    346-419 (534)
</pre><pre> 51 5VTG_A Translocation and assem  48.6 1.6E+02  0.0015   22.7   5.6   43   24-69     65-110 (184)
</pre><pre> 52 3K8R_A Uncharacterized protein  47.8      85 0.00079   23.7   4.0   24   14-37     28-51  (124)
</pre><pre> 53 3PET_B Putative adhesin; RIGHT  47.5 2.2E+02  0.0021   21.6   6.7   68    6-73    152-220 (221)
</pre><pre> 54 d2pi2e_ b.40.4.3 (E:) automate  47.5   2E+02  0.0018   21.0   7.0   58   13-70     28-89  (117)
</pre><pre> 55 4MPZ_A Sas-4; beta-spine, cent  47.5 2.1E+02  0.0019   22.8   6.1   49    2-50     61-115 (209)
</pre><pre> 56 4OPW_A Uncharacterized protein  47.3 2.4E+02  0.0022   22.0   6.7   67    6-72    172-238 (241)
</pre><pre> 57 d4qmea1 b.98.1.1 (A:2-188) Ami  44.6      76 0.00071   23.5   3.2   27   12-38     62-88  (187)
</pre><pre> 58 3JX8_A Putative lipoprotein; S  44.1 2.7E+02  0.0025   21.7   6.7   66    7-72    182-247 (250)
</pre><pre> 59 3JX8_F Putative lipoprotein; S  44.1 2.7E+02  0.0025   21.7   6.7   66    7-72    182-247 (250)
</pre><pre> 60 PF16986.7 ; CzcE ; Heavy-metal  43.0 1.9E+02  0.0018   20.0   4.8   32    8-39     25-58  (80)
</pre><pre> 61 d1ew4a_ d.82.2.1 (A:) CyaY {Es  43.0   1E+02 0.00094   22.9   3.7   26   12-37     26-51  (106)
</pre><pre> 62 TIGR03422 mito_frataxin; frata  41.7   1E+02 0.00095   22.7   3.5   22   16-37     29-50  (97)
</pre><pre> 63 TIGR03421 FeS_CyaY; iron donor  41.4   1E+02 0.00096   22.8   3.5   22   16-37     27-48  (102)
</pre><pre> 64 4YGU_D putative adhesin; PF109  41.3 2.5E+02  0.0023   20.5   5.5   60   21-80    123-184 (184)
</pre><pre> 65 6CUJ_B Gna2132; Vaccine, Neiss  40.5 2.2E+02   0.002   22.1   5.2   82   15-97     67-159 (161)
</pre><pre> 66 2N3D_A Bactofilin A; BacA, Bac  40.5 1.4E+02  0.0013   24.1   4.4   33   34-67    104-136 (184)
</pre><pre> 67 6FCO_A Mitochondrial frataxin-  40.5 1.1E+02  0.0011   23.5   3.7   26   13-38     32-58  (126)
</pre><pre> 68 6FCO_F Mitochondrial frataxin-  40.5 1.1E+02  0.0011   23.5   3.7   26   13-38     32-58  (126)
</pre><pre> 69 d1jjua5 b.61.4.1 (A:166-273) Q  40.4 1.6E+02  0.0015   23.0   4.5   52   26-92     27-80  (108)
</pre><pre> 70 6JP4_B Alginate lyase; alpha b  39.6   1E+02 0.00094   27.4   3.7   27   12-38    741-767 (770)
</pre><pre> 71 4PEU_A Uncharacterized protein  38.9 1.3E+02  0.0012   25.9   4.2   31   26-57     67-103 (313)
</pre><pre> 72 1ZDX_A Outer membrane usher pr  38.7      82 0.00076   21.1   2.5   19   12-30     74-92  (101)
</pre><pre> 73 4MPZ_A Sas-4; beta-spine, cent  38.5 3.3E+02  0.0031   21.7   6.0   50    2-51    138-199 (209)
</pre><pre> 74 4H09_B Hypothetical leucine ri  37.9 1.6E+02  0.0015   22.4   4.0   36   13-49      1-36  (379)
</pre><pre> 75 4H09_E Hypothetical leucine ri  37.9 1.6E+02  0.0015   22.4   4.0   36   13-49      1-36  (379)
</pre><pre> 76 PF16378.7 ; DUF4988 ; Domain o  37.8 1.3E+02  0.0012   23.9   3.8   24   14-37    162-185 (186)
</pre><pre> 77 PF01723.18 ; Chorion_1 ; Chori  36.1      70 0.00065   26.8   2.2   38   35-72     90-129 (172)
</pre><pre> 78 2X8N_A CV0863; NON-UNIFORM SAM  35.8 1.6E+02  0.0015   22.0   3.8   24   14-37     28-51  (111)
</pre><pre> 79 3OEQ_A Frataxin homolog, mitoc  35.6 1.4E+02  0.0013   22.4   3.5   29   10-38     43-71  (123)
</pre><pre> 80 4PEU_A Uncharacterized protein  34.6 5.5E+02  0.0051   22.5   7.4   77    2-78     92-185 (313)
</pre><pre> 81 PF01491.18 ; Frataxin_Cyay ; F  34.5 1.5E+02  0.0014   21.3   3.4   22   16-37     34-55  (110)
</pre><pre> 82 TIGR02892 spore_yabP; sporulat  34.4 3.3E+02   0.003   19.8   5.1   33   21-54     23-62  (85)
</pre><pre> 83 PF18952.2 ; DUF5696 ; Family o  34.2   2E+02  0.0019   26.1   4.7   33    1-33    579-614 (615)
</pre><pre> 84 PF16241.7 ; DUF4900 ; Domain o  34.0 1.7E+02  0.0016   20.6   3.5   40   30-69     48-87  (88)
</pre><pre> 85 PF01696.19 ; Adeno_E1B_55K ; A  33.3 1.2E+02  0.0012   25.1   3.1   27   26-52     61-87  (383)
</pre><pre> 86 d4ec2a_ d.82.2.1 (A:) C-termin  33.2 1.5E+02  0.0014   22.6   3.3   23   16-38     40-62  (112)
</pre><pre> 87 TIGR01222 minC; septum site-de  32.8      92 0.00086   25.7   2.4   34   45-78    122-157 (217)
</pre><pre> 88 5NZG_A UDP-glucose pyrophospho  32.4 2.3E+02  0.0021   25.7   4.8   69   28-97    437-505 (505)
</pre><pre> 89 d1ze3d1 b.167.1.1 (D:1-125) Ou  31.5 1.3E+02  0.0012   21.5   2.6   19   12-30     98-116 (125)
</pre><pre> 90 d3t3la_ d.82.2.1 (A:) automate  31.4 2.1E+02  0.0019   22.0   3.8   26   12-37     30-56  (121)
</pre><pre> 91 d4b0ma_ b.167.1.0 (A:) automat  31.4 1.4E+02  0.0013   21.8   2.8   19   12-30     98-116 (135)
</pre><pre> 92 PF15490.8 ; Ten1_2 ; Telomere-  30.9 3.7E+02  0.0034   19.3   7.0   53   13-65     28-91  (117)
</pre><pre> 93 5HMB_A Azi13; azinomycin biosy  30.2 2.4E+02  0.0022   19.9   3.8   51   44-96     91-141 (141)
</pre><pre> 94 PF06832.14 ; BiPBP_C ; Penicil  30.2 1.6E+02  0.0015   19.3   2.8   29    6-38     44-72  (87)
</pre><pre> 95 d2wmfa2 b.24.2.1 (A:426-589) S  30.2 1.9E+02  0.0017   24.2   3.7   24   15-38    140-163 (164)
</pre><pre> 96 4UHV_A VGRG1, VALINE-GLYCINE R  29.8 6.3E+02  0.0058   22.9   6.8   80    6-85    483-586 (651)
</pre><pre> 97 d2fgza4 b.71.1.1 (A:966-1083)   29.4 3.5E+02  0.0032   20.6   4.7   46   16-61     46-99  (118)
</pre><pre> 98 TIGR01643 YD_repeat_2x; YD rep  29.2 1.7E+02  0.0016   15.1   3.7   38    1-38      5-42  (42)
</pre><pre> 99 PF13954.8 ; PapC_N ; PapC N-te  28.9 1.7E+02  0.0016   21.3   3.0   19   12-30     96-114 (146)
</pre><pre>100 PF03775.18 ; MinC_C ; Septum f  28.0 1.4E+02  0.0013   22.0   2.4   31   47-77     12-42  (101)
</pre><pre>101 1PG6_A Hypothetical protein Sp  27.8 5.3E+02  0.0049   21.4   5.8   39   15-53     16-56  (243)
</pre><pre>102 d1pg6a_ b.82.5.2 (A:) Hypothet  27.8 3.6E+02  0.0034   21.8   4.8   41    3-43    168-209 (227)
</pre><pre>103 PF07481.13 ; DUF1521 ; Domain   27.5 4.2E+02  0.0039   22.3   5.2   42    8-50     65-117 (169)
</pre><pre>104 PF07481.13 ; DUF1521 ; Domain   27.4 6.6E+02  0.0062   21.2   6.9   55    2-56      7-66  (169)
</pre><pre>105 PF14734.8 ; DUF4469 ; Domain o  27.1 1.4E+02  0.0013   21.8   2.2   29   18-46      5-34  (99)
</pre><pre>106 4B0M_A F1 CAPSULE-ANCHORING PR  27.1 1.8E+02  0.0016   21.1   2.7   18   13-30     99-116 (136)
</pre><pre>107 PF07873.13 ; YabP ; YabP famil  27.0 3.7E+02  0.0034   18.0   5.4   33   21-54     23-62  (66)
</pre><pre>108 5NFI_B Minor fimbrium anchorin  26.9 5.3E+02  0.0049   21.0   5.5   44    6-49     29-80  (281)
</pre><pre>109 3LF4_B Fluorescent Timer Precu  26.6 3.7E+02  0.0035   22.3   4.7   31   26-57     29-59  (164)
</pre><pre>110 PF13159.8 ; DUF3994 ; Domain o  25.9 2.6E+02  0.0025   21.5   3.6   40    1-41     50-96  (99)
</pre><pre>111 3BWU_D Outer membrane usher pr  25.4 1.8E+02  0.0017   20.7   2.5   18   13-30     99-116 (125)
</pre><pre>112 4XIN_B LpqH orthologue; SSGCID  25.0 3.2E+02  0.0029   21.1   3.9   29   12-41     75-104 (126)
</pre><pre>113 PF05481.14 ; Myco_19_kDa ; Myc  24.9   3E+02  0.0027   20.7   3.6   29    8-41     63-92  (110)
</pre><pre>114 PF07532.13 ; Big_4 ; Bacterial  24.6 2.8E+02  0.0026   15.9   3.9   37    3-40     17-54  (58)
</pre><pre>115 PF09922.11 ; DUF2154 ; Cell wa  24.3 4.5E+02  0.0042   18.3   4.2   36   15-51     41-76  (113)
</pre><pre>116 d2z8ga2 b.80.1.0 (A:185-564) a  24.0 2.7E+02  0.0025   24.0   3.7   54   11-64     28-89  (380)
</pre><pre>117 1ZDV_A Outer membrane usher pr  23.8 2.3E+02  0.0022   19.6   2.8   19   12-30     74-92  (115)
</pre><pre>118 6D6V_F Telomerase holoenzyme T  23.7 5.5E+02  0.0051   19.0   5.6   61   13-74     30-100 (121)
</pre><pre>119 4LD1_A Uncharacterized protein  23.5 5.2E+02  0.0048   18.6   5.3   40    2-41     14-53  (184)
</pre><pre>120 d3e1za_ b.1.26.1 (A:) Chagasin  23.5 3.5E+02  0.0033   19.2   3.6   30    4-34      1-30  (109)
</pre><pre>121 PF18482.3 ; Pih1_fungal_CS ; F  23.3 2.6E+02  0.0024   20.7   3.0   18   15-32     34-51  (90)
</pre><pre>122 3ML3_A Outer membrane protein   23.3 6.1E+02  0.0056   19.8   5.1   77    1-80      2-94  (189)
</pre><pre>123 PF04357.15 ; TamB ; TamB, inne  23.2 7.1E+02  0.0066   22.3   6.0   52   23-75      1-74  (418)
</pre><pre>124 3SZE_A Serine protease espP; p  23.2 3.7E+02  0.0034   27.8   4.9   50   26-75    680-755 (968)
</pre><pre>125 TIGR00686 phnA; alkylphosphona  23.2 3.9E+02  0.0036   21.3   4.1   62    8-69     12-80  (109)
</pre><pre>126 d1yoxa1 b.82.5.2 (A:5-244) Hyp  22.9 5.8E+02  0.0054   21.1   5.1   41    3-43    172-214 (240)
</pre><pre>127 2OTN_A Diaminopimelate epimera  22.8 5.4E+02   0.005   21.0   4.9   40   13-52    266-307 (308)
</pre><pre>128 PF12708.9 ; Pectate_lyase_3 ;   22.5 3.8E+02  0.0035   19.2   3.6   33   18-51     36-68  (229)
</pre><pre>129 5G5O_C LH3 HEXON-INTERLACING C  22.3 3.2E+02   0.003   23.4   3.7   26   26-52     46-71  (373)
</pre><pre>130 PF18544.3 ; Polo_box_3 ; Polo   22.1 3.3E+02  0.0031   21.5   3.5   62   14-91      3-67  (96)
</pre><pre>131 3DA0_B Cleaved chimeric recept  22.1 2.5E+02  0.0024   19.8   2.7   33   32-64      1-33  (136)
</pre><pre>132 PF19917.1 ; bpX1 ; MoxR-vWA-be  22.0 5.7E+02  0.0053   18.5   4.8   41    6-46      2-54  (83)
</pre><pre>133 5WZQ_A Alpha-N-acetylgalactosa  21.9 3.3E+02   0.003   24.8   3.9   26    2-27    601-628 (640)
</pre><pre>134 d2i5vo_ b.76.1.1 (O:) automate  21.6 8.9E+02  0.0082   22.3   6.2   66    1-78    103-171 (246)
</pre><pre>135 PF10988.10 ; DUF2807 ; Putativ  21.2 6.5E+02   0.006   18.9   5.7   53    6-58    150-202 (203)
</pre><pre>136 PF09917.11 ; DUF2147 ; Unchara  21.2 3.2E+02   0.003   19.9   3.1   31    9-39     70-101 (116)
</pre><pre>137 4QDG_A Putative adhesin; PF088  21.1 6.2E+02  0.0057   21.3   5.0   36    3-39     36-80  (310)
</pre><pre>138 6D0T_A BB1; beta-barrel, desig  20.6 7.1E+02  0.0066   20.6   5.1   45   34-78     37-83  (120)
</pre><pre>139 6Z1P_BR Iron donor protein Cya  20.6 3.5E+02  0.0032   21.4   3.4   22   16-37     73-94  (143)
</pre><pre>140 d4ts4a2 b.46.1.0 (A:204-308) a  20.4 4.6E+02  0.0043   17.6   3.6   26   13-38     63-88  (105)
</pre><pre>141 1YOX_B hypothetical protein PA  20.2 8.9E+02  0.0083   20.2   5.7   38   15-53     11-50  (250)
</pre><pre>142 PF08307.13 ; Glyco_hydro_98C ;  20.2 3.9E+02  0.0036   23.5   3.9   24   15-38    230-253 (257)
</pre><pre>143 6QI5_Q Protein LH3; adenovirus  20.1   4E+02  0.0037   22.3   3.8   25   26-51     46-70  (370)
</pre><pre>144 6QI5_S Protein LH3; adenovirus  20.1   4E+02  0.0037   22.3   3.8   25   26-51     46-70  (370)
</pre><pre>145 6GHU_A Type II secretion syste  20.0 3.1E+02  0.0028   18.3   2.7   15   15-29     23-37  (78)
</pre><pre>
</pre><pre>No 1
</pre><pre>>4S36_A Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; 1.46A {Pseudomonas aeruginosa}
</pre><pre>Probab=99.40  E-value=2.3e-11  Score=79.34  Aligned_cols=85  Identities=21%  Similarity=0.453  Sum_probs=56.1  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             cceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecce-Ee---cCEEEE-eeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1    4 CEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGT-LD---GPVQIT-GKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus         4 ~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~-ld---G~v~it-~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      ++|+++|.||+.|+||+++|.|+|.++. |.+.|.+.+. +.   +.++|. +.+++.|.+.+.+++.+.+++.+.+++.
</pre><pre>T Consensus         1 ~~~~~~~~dG~~i~yd~~~~~l~i~~~~-g~~~i~~~~~~~~i~~~~i~i~~~~i~~~g~~~~~g~~~~~g~i~~~g~~~   79 (90)
</pre><pre>T 4S36_A            1 SSETWRFDDGASLSYDWAAHRYRVELPS-GTVEVRVGASEVRVSDGAVSLKAPKISLEGPVEIAGTLTVSGDILGGGSII   79 (90)
</pre><pre>T ss_dssp             CEEEEECTTSCEEEEETTTTEEEEECSS-SEEEEEETTEEEEEETTEEEEECSCCCCCSCCCCSSCCCCSSCCCCSSCCC
</pre><pre>T ss_pred             CceEEEeCCCcEEEEeCCCCEEEEEeCC-ceEEEEeCCceEEEecCCEEEeCCeEEEeccEEEeCcEEEEceEEecCeEE
</pre><pre>Confidence            4689999999999999999999999864 7888888762 11   234444 3455555555555555555555554443
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCCCcchhhhhhccCCCC
</pre><pre>Q WP_214195415.1   79 DAKGKMSAIRTTYNGHTH   96 (110)
</pre><pre>Q Consensus        79 D~~gsms~iReiyN~H~H   96 (110)
</pre><pre>                      .       -...++.|.|
</pre><pre>T Consensus        80 ~-------~g~~~~~h~h   90 (90)
</pre><pre>T 4S36_A           80 D-------TAGNSNHHTH   90 (90)
</pre><pre>T ss_dssp             C-------SSCCCCCSCC
</pre><pre>T ss_pred             e-------CCccCCCCCC
</pre><pre>Confidence            2       2334466665
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>3QR8_A Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; HET: MSE; 2.03A {Enterobacteria phage P2}
</pre><pre>Probab=99.24  E-value=2e-10  Score=82.40  Aligned_cols=36  Identities=22%  Similarity=0.304  Sum_probs=27.5  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEec
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGN   40 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~   40 (110)
</pre><pre>                      .+.++++++||..+.||++.+.+.+...  +.+.|.++
</pre><pre>T Consensus        97 ~~~~~~~~~~G~~i~~~~~~~~i~~~~~--~~i~i~~~  132 (211)
</pre><pre>T 3QR8_A           97 ADALHIRFPDGAVIEYEPETSALTVSGI--KTASVTAS  132 (211)
</pre><pre>T ss_dssp             SSCEEEECTTSCEEEEETTTTEEEEESC--SCCCCCCS
</pre><pre>T ss_pred             CCcEEEEcCCCCEEEEECCCCeEEEeeC--ceEEEEcC
</pre><pre>Confidence            4567888999999999999999988762  44544443
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>3QR7_B Baseplate assembly protein V; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, Tail spike, VIRAL PROTEIN; 0.94A {Enterobacteria phage P2}
</pre><pre>Probab=99.24  E-value=2.8e-10  Score=75.79  Aligned_cols=35  Identities=23%  Similarity=0.304  Sum_probs=27.2  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEe
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVG   39 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g   39 (110)
</pre><pre>                      ++.++++|+||+++.||++.+.|.+...  +.+.+.+
</pre><pre>T Consensus         1 ~~~~~~~~~dG~~~~yd~~~~~l~~~~~--g~i~v~~   35 (115)
</pre><pre>T 3QR7_B            1 ADALHIRFPDGAVIEYEPETSALTVSGI--KTASVTA   35 (115)
</pre><pre>T ss_dssp             CCCCEEECTTSCEEEEETTTTEEEC-CC--SCCCCCC
</pre><pre>T ss_pred             CCeEEEEcCCCCEEEEeCCCCeEEEEec--cceEEEc
</pre><pre>Confidence            3578899999999999999999998864  4554544
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>3AQJ_A Baseplate assembly protein V; bacteriophage, tail spike, iron binding, beta-helix, infection, METAL BINDING PROTEIN; 1.27A {Enterobacteria phage P2}
</pre><pre>Probab=99.19  E-value=7e-10  Score=77.53  Aligned_cols=105  Identities=19%  Similarity=0.266  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecC---------EEEEe-eEEeCCceeeeeeEEeccce
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGP---------VQITG-KLTVKGKTDLLANTHIEGNL   71 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~---------v~it~-~ltv~g~~~~~~n~~v~gni   71 (110)
</pre><pre>                      ++.++.++|.||+.+.||++++.|+|....+..+...+.-.+..+         +++.. .+++.|.+++ +++.+.+.+
</pre><pre>T Consensus        19 ~~~~~~~~~~dG~~~~yd~~~~~l~i~~~g~i~i~~~~~i~~~~~~~~i~~~~~v~~~~~~i~~~g~~~~-g~~~~~g~~   97 (134)
</pre><pre>T 3AQJ_A           19 SADALHIRFPDGAVIEYEPETSALTVSGIKTASVTASGSVTATVPVVMVKASTRVTLDTPEVVCTNRLIT-GTLEVQKGG   97 (134)
</pre><pre>T ss_dssp             BCSCEEEECTTSCEEEEETTTTEEEEEEEEEEEEEEEEEEEEECSEEEEECSSEEEEESSEEEEEEEEEE-EEEEESSCE
</pre><pre>T ss_pred             CCCcEEEEcCCCCEEEEeCCCCEEEEEeCceeEEEcCceEEEECCeEEEEeCcceEEeCCcEEEeccEEE-CCeeeccCc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ecCceeecCCCcchhhhhhccCCCCC-CCCCCCCCCC
</pre><pre>Q WP_214195415.1   72 TASDEISDAKGKMSAIRTTYNGHTHR-ETNSTTRKPD  107 (110)
</pre><pre>Q Consensus        72 ~a~~eVsD~~gsms~iReiyN~H~H~-d~gg~T~~PN  107 (110)
</pre><pre>                      .+.++|....+.+...+..|+.|.|. ...|.|.+|.
</pre><pre>T Consensus        98 ~~~g~i~~~~~~~~~~~~~~~~h~h~~~~~g~t~~~~  134 (134)
</pre><pre>T 3AQJ_A           98 TMRGNIEHTGGELSSNGKVLHTHKHPGDSGGTTGSPL  134 (134)
</pre><pre>T ss_dssp             EEESCEEEESSCEEETTEETTTCCCBCTTSSBCCSCC
</pre><pre>T ss_pred             EEECeEEEeCCceeeCCEEeccccCCCCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>4S37_B Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: SO4, CL, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=98.87  E-value=1.5e-07  Score=66.39  Aligned_cols=87  Identities=21%  Similarity=0.439  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEee-EEeCCceeeeeeEEeccceecCce
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITGK-LTVKGKTDLLANTHIEGNLTASDE   76 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~~-ltv~g~~~~~~n~~v~gni~a~~e   76 (110)
</pre><pre>                      .+.++.++|.||+++.||++++.++|..+.| .++|...+    .-.+.+.|+.. +++.|++.+++++.+.|.+.+.++
</pre><pre>T Consensus        94 ~~~~~~~~~~~g~~~~~d~~~~~i~i~~~~~-~i~~~~~~~~~~~~~~~~~i~~~~v~i~g~~~~~g~~~~~g~~~~~~~  172 (185)
</pre><pre>T 4S37_B           94 SASSETWRFDDGASLSYDWAAHRYRVELPSG-TVEVRVGASEVRVSDGAVSLKAPKISLEGPVEIAGTLTVSGDILGGGS  172 (185)
</pre><pre>T ss_dssp             CTTEEEEECTTSCEEEEETTTTEEEEEESSS-EEEEEETTEEEEEESSCEEEECSCEEEESCEEEESCEEESSCEEESSC
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEECCCCEEEEEeCCc-eEEEEECCcEEEEeCCcEEEECCcEEEeCCEEEEccEEEeCcEEEccE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eecCCCcchhhhhhccCCCC
</pre><pre>Q WP_214195415.1   77 ISDAKGKMSAIRTTYNGHTH   96 (110)
</pre><pre>Q Consensus        77 VsD~~gsms~iReiyN~H~H   96 (110)
</pre><pre>                      |.+.       ..-||.|.|
</pre><pre>T Consensus       173 ~~~~-------~~~~~~h~h  185 (185)
</pre><pre>T 4S37_B          173 IIDT-------AGNSNHHTH  185 (185)
</pre><pre>T ss_dssp             EECC-------C--------
</pre><pre>T ss_pred             EEcC-------CCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>4S37_O Phage baseplate protein; Cell Puncturing Device, Trimer, beta-helix, Iron-binding, Outer cell membrane piercing, METAL BINDING PROTEIN; HET: CL, SO4, PGE, EDO; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=98.87  E-value=1.5e-07  Score=66.39  Aligned_cols=87  Identities=21%  Similarity=0.439  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEee-EEeCCceeeeeeEEeccceecCce
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITGK-LTVKGKTDLLANTHIEGNLTASDE   76 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~~-ltv~g~~~~~~n~~v~gni~a~~e   76 (110)
</pre><pre>                      .+.++.++|.||+++.||++++.++|..+.| .++|...+    .-.+.+.|+.. +++.|++.+++++.+.|.+.+.++
</pre><pre>T Consensus        94 ~~~~~~~~~~~g~~~~~d~~~~~i~i~~~~~-~i~~~~~~~~~~~~~~~~~i~~~~v~i~g~~~~~g~~~~~g~~~~~~~  172 (185)
</pre><pre>T 4S37_O           94 SASSETWRFDDGASLSYDWAAHRYRVELPSG-TVEVRVGASEVRVSDGAVSLKAPKISLEGPVEIAGTLTVSGDILGGGS  172 (185)
</pre><pre>T ss_dssp             CTTEEEEECTTSCEEEEETTTTEEEEEESSS-EEEEEETTEEEEEESSCEEEECSCEEEESCEEEESCEEESSCEEESSC
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEECCCCEEEEEeCCc-eEEEEECCcEEEEeCCcEEEECCcEEEeCCEEEEccEEEeCcEEEccE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eecCCCcchhhhhhccCCCC
</pre><pre>Q WP_214195415.1   77 ISDAKGKMSAIRTTYNGHTH   96 (110)
</pre><pre>Q Consensus        77 VsD~~gsms~iReiyN~H~H   96 (110)
</pre><pre>                      |.+.       ..-||.|.|
</pre><pre>T Consensus       173 ~~~~-------~~~~~~h~h  185 (185)
</pre><pre>T 4S37_O          173 IIDT-------AGNSNHHTH  185 (185)
</pre><pre>T ss_dssp             EECC-------C--------
</pre><pre>T ss_pred             EEcC-------CCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>3VTN_A Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.75A {Enterobacteria phage Mu}
</pre><pre>Probab=98.72  E-value=4.6e-07  Score=58.98  Aligned_cols=94  Identities=19%  Similarity=0.283  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             eCCCCEEEeecccCeEEEEECCCceEEEEecc-eEec--CEEEEee-EEeCCceeeeeeEEeccceecCceeecCCCcch
</pre><pre>Q WP_214195415.1   10 YEDGTEVRYNRESHALTITLAEGGTYRIVGNG-TLDG--PVQITGK-LTVKGKTDLLANTHIEGNLTASDEISDAKGKMS   85 (110)
</pre><pre>Q Consensus        10 Y~DGTev~Ydr~~haLtitLaeGGt~~I~g~g-~ldG--~v~it~~-ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsms   85 (110)
</pre><pre>                      |.||+.+.||+..+  .|.|..+|.+.|.++. .|..  .+.+... +++.|.+.+.+++.+.+.....+.+  ..+.+.
</pre><pre>T Consensus         3 ~~~G~~~~y~~~~~--~i~~~~~g~i~i~~~~~~i~~~~~~~~~~~~i~~~g~~~~~g~~~~~g~~~~~g~~--~~~~~~   78 (101)
</pre><pre>T 3VTN_A            3 MEPGDAGIYHHEGH--RIRLTKDGRCIITCKTVEVYADESMTVDTPRTTFTGDVEIQKGLGVKGKSQFDSNI--TAPDAI   78 (101)
</pre><pre>T ss_dssp             ----------------CEEECTTSCEEECSSCCCCCCSSCCCCCCSCCCCSSCCCCSSCCCCCSCCCCSSCC--CCSCCC
</pre><pre>T ss_pred             CCCCcEEEEeCCCC--EEEEcCCCEEEEeCCeEEEEcCCceEEeCCcEEEeccEEEcCCeEEcCcceEecce--ecCceE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhhhccCCCCC-CCCCCCCCCC
</pre><pre>Q WP_214195415.1   86 AIRTTYNGHTHR-ETNSTTRKPD  107 (110)
</pre><pre>Q Consensus        86 ~iReiyN~H~H~-d~gg~T~~PN  107 (110)
</pre><pre>                      ..+..++.|.|. ..++.|..|.
</pre><pre>T Consensus        79 ~~~~~~~~h~h~~~~~~~t~~~~  101 (101)
</pre><pre>T 3VTN_A           79 INGKSTDKHIHRGDSGGTTGPMQ  101 (101)
</pre><pre>T ss_dssp             BTTBCSSCC--------------
</pre><pre>T ss_pred             ECCEEcCCcccCCCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>3VTO_B Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=98.67  E-value=4.9e-07  Score=62.36  Aligned_cols=96  Identities=20%  Similarity=0.276  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             eeCCCCEEEeecccCeEEEEECCCceEEEEecc---eEecCEEEEee-EEeCCceeeeeeEEeccceecCceeecCCCcc
</pre><pre>Q WP_214195415.1    9 VYEDGTEVRYNRESHALTITLAEGGTYRIVGNG---TLDGPVQITGK-LTVKGKTDLLANTHIEGNLTASDEISDAKGKM   84 (110)
</pre><pre>Q Consensus         9 rY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g---~ldG~v~it~~-ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsm   84 (110)
</pre><pre>                      .|.||+.+.||++.+.|++.  .+|.+.|.++.   ..++.+.+... +++.|.+.+.+++.+.+.++..+++  ..+.+
</pre><pre>T Consensus         8 ~~~dG~~~~yd~~g~~l~i~--~~g~i~i~~~~~~i~a~~~i~~~~~~~~i~g~~~~~g~~~~~g~~~~~g~~--~~g~~   83 (115)
</pre><pre>T 3VTO_B            8 NLEPGDAGIYHHEGHRIRLT--KDGRCIITCKTVEVYADESMTVDTPRTTFTGDVEIQKGLGVKGKSQFDSNI--TAPDA   83 (115)
</pre><pre>T ss_dssp             SCCTTCEEEECTTSCEEEEE--TTTEEEEEEEEEEEEEEEEEEEECSEEEESSEEEESSCEEESSCEEESSCE--EESCE
</pre><pre>T ss_pred             CCCCCcEEEEeCCCCEEEEE--cCCEEEEeCCeEEEEcCCcEEEeCCcEEEEecEEEccceEecCceeeecce--ecCcE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhhhhccCCCCCCCCCCCCCCCc
</pre><pre>Q WP_214195415.1   85 SAIRTTYNGHTHRETNSTTRKPDQ  108 (110)
</pre><pre>Q Consensus        85 s~iReiyN~H~H~d~gg~T~~PNQ  108 (110)
</pre><pre>                      ...+..++.|.|...++.|+.|=|
</pre><pre>T Consensus        84 ~~~g~~~~~h~h~~~~~~t~~~~~  107 (115)
</pre><pre>T 3VTO_B           84 IINGKSTDKHIHRGDSGGTTGPMQ  107 (115)
</pre><pre>T ss_dssp             EETTEETTTCBCBCTTSSBCCCCB
</pre><pre>T ss_pred             EECCEEcCCceecCCCCCCCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>3VTO_C Protein gp45; beta-helix, central spike, Mu phage, METAL BINDING PROTEIN; 1.44A {Enterobacteria phage Mu}
</pre><pre>Probab=98.67  E-value=4.9e-07  Score=62.36  Aligned_cols=96  Identities=20%  Similarity=0.276  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             eeCCCCEEEeecccCeEEEEECCCceEEEEecc---eEecCEEEEee-EEeCCceeeeeeEEeccceecCceeecCCCcc
</pre><pre>Q WP_214195415.1    9 VYEDGTEVRYNRESHALTITLAEGGTYRIVGNG---TLDGPVQITGK-LTVKGKTDLLANTHIEGNLTASDEISDAKGKM   84 (110)
</pre><pre>Q Consensus         9 rY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g---~ldG~v~it~~-ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsm   84 (110)
</pre><pre>                      .|.||+.+.||++.+.|++.  .+|.+.|.++.   ..++.+.+... +++.|.+.+.+++.+.+.++..+++  ..+.+
</pre><pre>T Consensus         8 ~~~dG~~~~yd~~g~~l~i~--~~g~i~i~~~~~~i~a~~~i~~~~~~~~i~g~~~~~g~~~~~g~~~~~g~~--~~g~~   83 (115)
</pre><pre>T 3VTO_C            8 NLEPGDAGIYHHEGHRIRLT--KDGRCIITCKTVEVYADESMTVDTPRTTFTGDVEIQKGLGVKGKSQFDSNI--TAPDA   83 (115)
</pre><pre>T ss_dssp             CCCTTCEEEECTTSCEEEEE--TTTEEEEEEEEEEEEEEEEEEEESSEEEESSEEEESSCEEESSCEEESSCE--EESCE
</pre><pre>T ss_pred             CCCCCcEEEEeCCCCEEEEE--cCCEEEEeCCeEEEEcCCcEEEeCCcEEEEecEEEccceEecCceeeecce--ecCcE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhhhhccCCCCCCCCCCCCCCCc
</pre><pre>Q WP_214195415.1   85 SAIRTTYNGHTHRETNSTTRKPDQ  108 (110)
</pre><pre>Q Consensus        85 s~iReiyN~H~H~d~gg~T~~PNQ  108 (110)
</pre><pre>                      ...+..++.|.|...++.|+.|=|
</pre><pre>T Consensus        84 ~~~g~~~~~h~h~~~~~~t~~~~~  107 (115)
</pre><pre>T 3VTO_C           84 IINGKSTDKHIHRGDSGGTTGPMQ  107 (115)
</pre><pre>T ss_dssp             EETTEETTTCBCBCTTSSBCCCCB
</pre><pre>T ss_pred             EECCEEcCCceecCCCCCCCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>TIGR01644 phage_P2_V; phage baseplate assembly protein V. This model describes a family of phage (and bacteriocin) proteins related to the phage P2 V gene product, which forms the small spike at the tip of the tail.
</pre><pre>Probab=98.49  E-value=5.6e-06  Score=61.03  Aligned_cols=95  Identities=23%  Similarity=0.363  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecc-----eEecCEEEEeeEEeCCc--eeeeeeEEeccceecC
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNG-----TLDGPVQITGKLTVKGK--TDLLANTHIEGNLTAS   74 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g-----~ldG~v~it~~ltv~g~--~~~~~n~~v~gni~a~   74 (110)
</pre><pre>                      .+.+.+.+|.||+.|+|+++++.|++.  ..+.++++++.     .-...+++++.+++.+.  ..+.+++++.|++.+.
</pre><pre>T Consensus        87 ~~~~~~~~~~~g~~i~~~~~~~~l~i~--~~~~~~~~~~~~~~~~~~~~~~~~~g~~~~~~~~~~~~~g~~~~~g~~~~~  164 (190)
</pre><pre>T TIGR01644        87 SAGDTVTYFADGARIEYDKAAHALTVC--GIKTVYLDAGDGTTVTVDAPQITFTGGVTTLKGPETVITGNVTIDGTLHTG  164 (190)
</pre><pre>T ss_pred             CCCcEEEEeCCCCEEEEeCCCCEEEEe--eceeEEEeCCCCceEEecCCEEEEecceEEecCCceEEEeeEEEcCceeec
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ceeecCCCcchhhhhhccCCCCC--CCCCCCCC
</pre><pre>Q WP_214195415.1   75 DEISDAKGKMSAIRTTYNGHTHR--ETNSTTRK  105 (110)
</pre><pre>Q Consensus        75 ~eVsD~~gsms~iReiyN~H~H~--d~gg~T~~  105 (110)
</pre><pre>                      +++...       +..|+.|.|.  ..|+.+..
</pre><pre>T Consensus       165 g~v~~~-------g~~~~~H~H~~~~~G~~~t~  190 (190)
</pre><pre>T TIGR01644       165 GDASSN-------GISLDKHGHTGVHSGGTTSG  190 (190)
</pre><pre>T ss_pred             CcEEEC-------CEEeccccccCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>3PQH_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 1.295A {Bacteriophage phi92}
</pre><pre>Probab=98.30  E-value=1.5e-05  Score=56.28  Aligned_cols=90  Identities=21%  Similarity=0.281  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CccceeEe---------eCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEecc-ce
</pre><pre>Q WP_214195415.1    2 RECEHITV---------YEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEG-NL   71 (110)
</pre><pre>Q Consensus         2 s~~eHvtr---------Y~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~g-ni   71 (110)
</pre><pre>                      ++++++++         |.||++|+||+.++.|+|.   +|.++|...+.+.-.+. ...+++.+.     .+.+.| ++
</pre><pre>T Consensus        18 ~~~~~~~~~~~~~~~~~~~dG~~i~yd~~~~~l~i~---~g~i~i~~~~~~~~~~~-~~~i~~~~~-----~v~i~G~~~   88 (127)
</pre><pre>T 3PQH_A           18 DSEKVIISNNKQTYASFDPNGNISVYNTQGMKIDMT---PNSIVLTDAGGGKLTLQ-GGTMTYKGG-----TVNLNGLTI   88 (127)
</pre><pre>T ss_dssp             --CCEEEEETTTEEEEECTTSCEEEEETTSCEEEEE---TTEEEEEETTTEEEEEE-TTEEEEESS-----CEEETTEEE
</pre><pre>T ss_pred             CcccEEEEcCCcceeeeCCCCcEEEEcCCCcEEEec---CCEEEEEcCCCcEEEEC-CCeEEEeCC-----eEEEeceEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ecCc-eeecCCCcchhhhhhccCCCC--C---CCCCCCCCCC
</pre><pre>Q WP_214195415.1   72 TASD-EISDAKGKMSAIRTTYNGHTH--R---ETNSTTRKPD  107 (110)
</pre><pre>Q Consensus        72 ~a~~-eVsD~~gsms~iReiyN~H~H--~---d~gg~T~~PN  107 (110)
</pre><pre>                      ...+ .+......+       ..|.|  .   ..++.+..|.
</pre><pre>T Consensus        89 ~~~g~~~~~~g~~~-------~~h~h~~~~~~~g~~~~~~~~  123 (127)
</pre><pre>T 3PQH_A           89 TPDGRMTDSGGIGL-------HTHTHPVRGVETGGSTVTSDK  123 (127)
</pre><pre>T ss_dssp             CTTSCEECTTSCBT-------TTCCCCCCSCCCCSCCCCCCS
</pre><pre>T ss_pred             ecCCCeeeeCCEEc-------CCceecCCCcccCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>3PQI_A gene product 138; Beta-Helix, OB-Fold, Phage Baseplate, Iron-Binding, Cell membrane piercing, VIRAL PROTEIN; 2.642A {Bacteriophage phi92}
</pre><pre>Probab=98.09  E-value=6.9e-05  Score=57.18  Aligned_cols=96  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEe----CCceeeeeeE-Eeccc-eecCc
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTV----KGKTDLLANT-HIEGN-LTASD   75 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv----~g~~~~~~n~-~v~gn-i~a~~   75 (110)
</pre><pre>                      ++++++..+.+|+.|+|++ ++.+.|.-+..-++.+..           +.+++    .+.+.++++. ++.++ +...+
</pre><pre>T Consensus       138 ~~~~~~~~~~~g~~i~~~~-~G~i~i~~~~~i~i~~~~-----------~~i~i~~~~~~~~~i~~~~~~~~g~~~~~~g  205 (247)
</pre><pre>T 3PQI_A          138 DSEKVIISNNKQTYASFDP-NGNISVYNTQGMKIDMTP-----------NSIVLTDAGGGKLTLQGGTMTYKGGTVNLNG  205 (247)
</pre><pre>T ss_dssp             CSSCEEEEETTTEEEEECT-TSCEEEEETTSCEEEEET-----------TEEEEEETTTEEEEEETTEEEEESSCEEETT
</pre><pre>T ss_pred             CcccEEEEcCCCcEEEECC-CCcEEEEeCCceEEEeCC-----------CceEEEeCCCceEEEcCCeeeecCCeEEEcC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             e-eecCCCcchhhhhhccCCCCC-----CCCCCCCCCCcC
</pre><pre>Q WP_214195415.1   76 E-ISDAKGKMSAIRTTYNGHTHR-----ETNSTTRKPDQQ  109 (110)
</pre><pre>Q Consensus        76 e-VsD~~gsms~iReiyN~H~H~-----d~gg~T~~PNQk  109 (110)
</pre><pre>                      + ++...+.|..-+..++.|.|.     .+++.|..|.++
</pre><pre>T Consensus       206 ~~i~~~g~~~~~~g~~~~~H~H~~~~v~~G~~~t~~~~~~  245 (247)
</pre><pre>T 3PQI_A          206 LTITPDGRMTDSGGIGLHTHTHPVRGVETGGSTVTSDKPN  245 (247)
</pre><pre>T ss_dssp             EEECTTSCEECTTSCBTTTCCCCCCSCCCCSCCCCCCSCC
</pre><pre>T ss_pred             cEEcCCCCEEcCCCccccceeecCCcccCCCCCccCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>4RU3_A puncturing protein gp41; gp41, membrane translocation, phage baseplate, Cell puncturing device, CELL INVASION; 1.172A {Pseudomonas phage SN}
</pre><pre>Probab=96.76  E-value=0.019  Score=44.09  Aligned_cols=71  Identities=14%  Similarity=0.217  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCC-CEEEeecccCeEEEEECCCceEEEEecc-eEecCEEEEeeEEeCCceeeeeeEEeccceecCceeec
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDG-TEVRYNRESHALTITLAEGGTYRIVGNG-TLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEISD   79 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DG-Tev~Ydr~~haLtitLaeGGt~~I~g~g-~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVsD   79 (110)
</pre><pre>                      +..+++.++.|| +.|.++..    .|++..++.++|.++. ++.|.+++.                  +++.+.+    
</pre><pre>T Consensus       144 ~~~~~~i~~~~G~~~I~i~~~----~i~i~~~~~v~i~~~~i~i~G~v~~~------------------g~v~~~g----  197 (221)
</pre><pre>T 4RU3_A          144 DSAAMVIQSTSGATRISIRGD----NIKITAPSNVTVDTPQANFTGSVTIA------------------NTLVVNG----  197 (221)
</pre><pre>T ss_dssp             GTTSEEEECTTSSSEEEECSS----CEEEECSSCCCCCSSCCCCSSCCCCS------------------SCCEETT----
</pre><pre>T ss_pred             CcCceEEEcCCCCeEEEEECC----eEEEeCCCcEEEcCCceEEeccEEEc------------------CCEEECC----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCcchhhhhhccCCCCC---CCCCC-CCCCC
</pre><pre>Q WP_214195415.1   80 AKGKMSAIRTTYNGHTHR---ETNST-TRKPD  107 (110)
</pre><pre>Q Consensus        80 ~~gsms~iReiyN~H~H~---d~gg~-T~~PN  107 (110)
</pre><pre>                               .-++.|.|.   ..++. |..|+
</pre><pre>T Consensus       198 ---------~~~~~H~H~gv~~g~~~~t~~p~  220 (221)
</pre><pre>T 4RU3_A          198 ---------VNVNNHGHLENNPPDARTKGGMI  220 (221)
</pre><pre>T ss_dssp             ---------EESTTCCCBCTTTTTCBCSSSCC
</pre><pre>T ss_pred             ---------EEcceeEccCCCCCCcCCCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>6P2A_C CHIMERA OF BACTERIOPHAGE OBP GP146 AND A T4 GP5; Bacteriophage OBP, membrane piercing, central spike, cell puncturing device, beta helix, T4 gp5, contractile injection system; HET: STE, ELA, PLM; 1.9A {Enterobacteria phage T4}
</pre><pre>Probab=96.10  E-value=0.13  Score=38.83  Aligned_cols=83  Identities=23%  Similarity=0.321  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCceEEEEecc--eEecCEEEEe-------eEEeCCceeeeeeEEeccceecCceeecCC------Ccchhhhh
</pre><pre>Q WP_214195415.1   25 LTITLAEGGTYRIVGNG--TLDGPVQITG-------KLTVKGKTDLLANTHIEGNLTASDEISDAK------GKMSAIRT   89 (110)
</pre><pre>Q Consensus        25 LtitLaeGGt~~I~g~g--~ldG~v~it~-------~ltv~g~~~~~~n~~v~gni~a~~eVsD~~------gsms~iRe   89 (110)
</pre><pre>                      +++++...-+++|.|.-  ...|...+..       .+++.+.....+++.+.|++.+.+.|...-      +.+..-..
</pre><pre>T Consensus        47 ~~i~v~G~~~~~v~G~~~~~vgg~~~~~~~~~~~~g~iti~g~~~~~G~~~~~g~~~~~G~i~~~~~~~~~~~~~~~~g~  126 (148)
</pre><pre>T 6P2A_C           47 QTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSKGNVTHEGNYNQLGNYTVQGNVGIQGAFSQFGGAGSVEGGWTIDNI  126 (148)
</pre><pre>T ss_dssp             EEEEESSCEEEEESSCEEEEESSEEEEEEEEEEEEEEEEEESCEEEESCEEEESCEEEESCEEEESSCEEECSCEEETTE
</pre><pre>T ss_pred             EEEEEcCCEEEEEeCcEEEEeCCcEEEEeceeecCCeEEEEcceeEEeEEEEeCCeEEEeeEEeeCCCccccCcEEECCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hccCCCCC---CCCCCCCCCC
</pre><pre>Q WP_214195415.1   90 TYNGHTHR---ETNSTTRKPD  107 (110)
</pre><pre>Q Consensus        90 iyN~H~H~---d~gg~T~~PN  107 (110)
</pre><pre>                      .++.|.|.   ..++.|.+|.
</pre><pre>T Consensus       127 ~~~~h~H~~~~~g~~~t~~~~  147 (148)
</pre><pre>T 6P2A_C          127 RYLGHRHGGVQSGGSKTDTPS  147 (148)
</pre><pre>T ss_dssp             EGGGCEECSECBCSBCCCCEE
</pre><pre>T ss_pred             EecCccccCccCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>6P2A_E CHIMERA OF BACTERIOPHAGE OBP GP146 AND A T4 GP5; Bacteriophage OBP, membrane piercing, central spike, cell puncturing device, beta helix, T4 gp5, contractile injection system; HET: ELA, PLM, STE; 1.9A {Enterobacteria phage T4}
</pre><pre>Probab=96.10  E-value=0.13  Score=38.83  Aligned_cols=83  Identities=23%  Similarity=0.321  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             EEEEECCCceEEEEecc--eEecCEEEEe-------eEEeCCceeeeeeEEeccceecCceeecCC------Ccchhhhh
</pre><pre>Q WP_214195415.1   25 LTITLAEGGTYRIVGNG--TLDGPVQITG-------KLTVKGKTDLLANTHIEGNLTASDEISDAK------GKMSAIRT   89 (110)
</pre><pre>Q Consensus        25 LtitLaeGGt~~I~g~g--~ldG~v~it~-------~ltv~g~~~~~~n~~v~gni~a~~eVsD~~------gsms~iRe   89 (110)
</pre><pre>                      +++++...-+++|.|.-  ...|...+..       .+++.+.....+++.+.|++.+.+.|...-      +.+..-..
</pre><pre>T Consensus        47 ~~i~v~G~~~~~v~G~~~~~vgg~~~~~~~~~~~~g~iti~g~~~~~G~~~~~g~~~~~G~i~~~~~~~~~~~~~~~~g~  126 (148)
</pre><pre>T 6P2A_E           47 QTNTVNGNLSWKVAGTVDWDVGGDWTEKMASMSSKGNVTHEGNYNQLGNYTVQGNVGIQGAFSQFGGAGSVEGGWTIDNI  126 (148)
</pre><pre>T ss_dssp             EEEEESSCEEEEESSCEEEEESSEEEEEEEEEEEEEEEEEESCEEEESCEEEESCEEEESCEEEESSCEEECSCEEETTE
</pre><pre>T ss_pred             EEEEEcCCEEEEEeCcEEEEeCCcEEEEeceeecCCeEEEEcceeEEeEEEEeCCeEEEeeEEeeCCCccccCcEEECCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hccCCCCC---CCCCCCCCCC
</pre><pre>Q WP_214195415.1   90 TYNGHTHR---ETNSTTRKPD  107 (110)
</pre><pre>Q Consensus        90 iyN~H~H~---d~gg~T~~PN  107 (110)
</pre><pre>                      .++.|.|.   ..++.|.+|.
</pre><pre>T Consensus       127 ~~~~h~H~~~~~g~~~t~~~~  147 (148)
</pre><pre>T 6P2A_E          127 RYLGHRHGGVQSGGSKTDTPS  147 (148)
</pre><pre>T ss_dssp             EGGGCEECSECEEEECCCCEE
</pre><pre>T ss_pred             EecCccccCccCCCCCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>PF04519.15 ; Bactofilin ; Polymer-forming cytoskeletal
</pre><pre>Probab=92.20  E-value=1.3  Score=29.77  Aligned_cols=48  Identities=10%  Similarity=0.163  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             EECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   28 TLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        28 tLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      .|.-.+.++|.|  .+.|++...+.+.|.+...+.+++.+ +++...++|.
</pre><pre>T Consensus        12 ~~~~~~~i~i~g--~i~g~v~~~g~v~v~~~~~v~g~i~~-~~i~i~G~v~   59 (97)
</pre><pre>T E6W760_DESIS/2   12 EIRSETDVFIDG--KFKGDIYAKKTISVITGGNVDGTFNA-DKMVISGKVQ   59 (97)
</pre><pre>T ss_pred             EEEeCCcEEEEE--EEEEEEEeCCeEEECCCcEEEeEEEe-eEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>PF10833.10 ; DUF2572 ; Protein of unknown function (DUF2572)
</pre><pre>Probab=90.63  E-value=2  Score=35.05  Aligned_cols=52  Identities=17%  Similarity=0.293  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             EEECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   27 ITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        27 itLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      |-+.++|.+.+.+++.+.|-|-.++.+++.|...+.|.+.+.+++...+.+.
</pre><pre>T Consensus       140 iliv~~g~~~~~~~~~f~GiIi~~g~l~i~G~~~I~Gavis~g~l~~~g~~~  191 (220)
</pre><pre>T B0UVK0_HISS2/5  140 LYWVEEKSAELEIQGNINAVVIAEGDLKLTGKGKISGTVITGGKLSIEPAVK  191 (220)
</pre><pre>T ss_pred             EEEEECCeEEEcCCcEEEEEEEEeCeEEEcCCcEEEEEEEECCCccCCCCeE
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>PF19119.2 ; DUF5803 ; Family of unknown function (DUF5803)
</pre><pre>Probab=89.55  E-value=3.1  Score=34.48  Aligned_cols=49  Identities=18%  Similarity=0.284  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             eEeeCCCCEE-------EeecccCeEEEEECCC--ceEEEEecc---eEecCEEEEeeEEe
</pre><pre>Q WP_214195415.1    7 ITVYEDGTEV-------RYNRESHALTITLAEG--GTYRIVGNG---TLDGPVQITGKLTV   55 (110)
</pre><pre>Q Consensus         7 vtrY~DGTev-------~Ydr~~haLtitLaeG--Gt~~I~g~g---~ldG~v~it~~ltv   55 (110)
</pre><pre>                      .++|+|||.+       .+.+......|+++++  |++..+++.   .+.-++...++..|
</pre><pre>T Consensus        46 ~~~y~ngTvv~~~~~~~~v~~~~~~~~I~fp~~~~G~vAyt~~~~~k~f~~~~~~~gs~~V  106 (198)
</pre><pre>T C7NXL0_HALMD/4   46 QYRYPNGTVVTADHSALSVSQTRERTIIDVPNETAGQVAFTAPRQGKRYSTPVFVEGATNL  106 (198)
</pre><pre>T ss_pred             EEECCCCCeeccCCCcceEEecCcEEEEECCCCCCeEEEEEeecCCCEEEEeeecCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF13619.8 ; KTSC ; KTSC domain
</pre><pre>Probab=86.65  E-value=4.1  Score=25.27  Aligned_cols=26  Identities=23%  Similarity=0.561  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CEEEeecccCeEEEEECCCceEEEEe
</pre><pre>Q WP_214195415.1   14 TEVRYNRESHALTITLAEGGTYRIVG   39 (110)
</pre><pre>Q Consensus        14 Tev~Ydr~~haLtitLaeGGt~~I~g   39 (110)
</pre><pre>                      ..|.||++++.|.|.+..|.+|...+
</pre><pre>T Consensus         6 ~~v~Yd~~~~~L~V~f~~g~~y~y~~   31 (58)
</pre><pre>T I4F247_9ACTN/7    6 ATVGYDEEQRIMEIVFRDGRVYHFLE   31 (58)
</pre><pre>T ss_pred             eEEEEECCCCEEEEEECCCcEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF03961.15 ; FapA ; Flagellar Assembly Protein A
</pre><pre>Probab=85.23  E-value=19  Score=31.71  Aligned_cols=73  Identities=12%  Similarity=0.253  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             cceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEe-------cCEEEEeeEEeCCceeeeeeEEeccceecCce
</pre><pre>Q WP_214195415.1    4 CEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLD-------GPVQITGKLTVKGKTDLLANTHIEGNLTASDE   76 (110)
</pre><pre>Q Consensus         4 ~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ld-------G~v~it~~ltv~g~~~~~~n~~v~gni~a~~e   76 (110)
</pre><pre>                      ++.+..-.||..+ |-.....+.+.   .+.+.|.-.=.++       |++.+.+.+.|+|.+.-...+.+.|+|.+.|.
</pre><pre>T Consensus       144 g~~~~~~~dg~~i-~A~~~G~~~~~---~~~i~V~~~~~i~g~v~~~tG~i~~~g~v~I~G~V~~g~~v~a~g~i~i~G~  219 (441)
</pre><pre>T Q8EQV6_OCEIH/7  144 GKNVVWKEDVHSF-FAASDGQISIN---NQQIQVLPVFKVNESISMKTGNLDFVGNIEIHGDVPSGFEMKAEGDIRIHGV  219 (441)
</pre><pre>T ss_pred             CCCEEECCCCCEE-EECcccEEEEE---CCEEEEEceEEEeccccCCcCCEEEeccEEEeCCcCCCeEEEECCCEEEeCe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eecC
</pre><pre>Q WP_214195415.1   77 ISDA   80 (110)
</pre><pre>Q Consensus        77 VsD~   80 (110)
</pre><pre>                      |.++
</pre><pre>T Consensus       220 v~~a  223 (441)
</pre><pre>T Q8EQV6_OCEIH/7  220 VEGA  223 (441)
</pre><pre>T ss_pred             Eecc
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>6RBK_C Afp8; Anti-feeding prophage, secretion system, AFP, contractile, VIRUS LIKE PARTICLE, baseplate; 3.4A {Serratia entomophila}
</pre><pre>Probab=84.88  E-value=13  Score=31.13  Aligned_cols=72  Identities=15%  Similarity=0.246  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCC-CEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEe-eEEeCCceeeeeeEEeccceecCc
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDG-TEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITG-KLTVKGKTDLLANTHIEGNLTASD   75 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DG-Tev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~-~ltv~g~~~~~~n~~v~gni~a~~   75 (110)
</pre><pre>                      .....+++.+++ .++.||.+.+.+.+. ..+..++|.+.+    .-++.+.|.. +++++.+..+.  +..+++|...+
</pre><pre>T Consensus       446 ~~~~~~~~~~~~~~~~~~d~~~~~~~~~-~~~~~~~l~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~--~~~~~~~~i~~  522 (529)
</pre><pre>T 6RBK_C          446 NRKGLIVSQADQTQALMIDTEEKTLRLM-AGDNTLTLTGEGNLTMSTPNALQLQADTLGLQADSNLS--IAGKQQVEITS  522 (529)
</pre><pre>T ss_dssp             CBCCCCCBSSSCBCSEECBTTTTCCEEC-SSSCCCCCCTTSCCCCCCSSCCCCCCSSCCCCCSSCCC--CCCSSCCCCCC
</pre><pre>T ss_pred             CceEEEEecCCCceEEEEeCCCCEEEEE-cCCcEEEEcCCCcEEEcCCCeEEEecCcEEEEeCCceE--EeccCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             e
</pre><pre>Q WP_214195415.1   76 E   76 (110)
</pre><pre>Q Consensus        76 e   76 (110)
</pre><pre>                      .
</pre><pre>T Consensus       523 ~  523 (529)
</pre><pre>T 6RBK_C          523 A  523 (529)
</pre><pre>T ss_dssp             S
</pre><pre>T ss_pred             c
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>6RIA_K bactofilin; Prokaryotic cytoskeletons, PROTEIN FIBRIL; 3.5A {Thermus thermophilus}
</pre><pre>Probab=84.45  E-value=6.2  Score=28.25  Aligned_cols=42  Identities=17%  Similarity=0.290  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             EEEecceEecCEEEEeeEEeCCceeeeee-EEeccceecCceee
</pre><pre>Q WP_214195415.1   36 RIVGNGTLDGPVQITGKLTVKGKTDLLAN-THIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        36 ~I~g~g~ldG~v~it~~ltv~g~~~~~~n-~~v~gni~a~~eVs   78 (110)
</pre><pre>                      .|...|.+.|++.+.+++++.+...+.++ +.. .++...+.|.
</pre><pre>T Consensus        30 ~i~i~g~i~g~v~~~g~i~i~~~~~v~G~~i~~-~~~~i~G~v~   72 (123)
</pre><pre>T 6RIA_K           30 QVRIDGLVRGSVLVEGELEVGPTGRVEGERVEA-RSVLIHGEVK   72 (123)
</pre><pre>T ss_dssp             EEEECSEEESCBCCEEEEEECTTCEEEEEEEEC-SEEEESSEEE
</pre><pre>T ss_pred             eEEEEeEEEceEEECCeEEECCCcEEEccEEEe-cEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>6RIA_L bactofilin; Prokaryotic cytoskeletons, PROTEIN FIBRIL; 3.5A {Thermus thermophilus}
</pre><pre>Probab=84.45  E-value=6.2  Score=28.25  Aligned_cols=42  Identities=17%  Similarity=0.290  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             EEEecceEecCEEEEeeEEeCCceeeeee-EEeccceecCceee
</pre><pre>Q WP_214195415.1   36 RIVGNGTLDGPVQITGKLTVKGKTDLLAN-THIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        36 ~I~g~g~ldG~v~it~~ltv~g~~~~~~n-~~v~gni~a~~eVs   78 (110)
</pre><pre>                      .|...|.+.|++.+.+++++.+...+.++ +.. .++...+.|.
</pre><pre>T Consensus        30 ~i~i~g~i~g~v~~~g~i~i~~~~~v~G~~i~~-~~~~i~G~v~   72 (123)
</pre><pre>T 6RIA_L           30 QVRIDGLVRGSVLVEGELEVGPTGRVEGERVEA-RSVLIHGEVK   72 (123)
</pre><pre>T ss_dssp             EEEECSEEESCEECEEEEEECTTCEEEEEEEEC-SEEEESSEEE
</pre><pre>T ss_pred             eEEEEeEEEceEEECCeEEECCCcEEEccEEEe-cEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>PF18715.3 ; Phage_spike ; Phage spike trimer
</pre><pre>Probab=81.02  E-value=18  Score=21.68  Aligned_cols=47  Identities=13%  Similarity=0.144  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecc-eEecCEEEEeeEEeCCceeeeeeEEec-cceec
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNG-TLDGPVQITGKLTVKGKTDLLANTHIE-GNLTA   73 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g-~ldG~v~it~~ltv~g~~~~~~n~~v~-gni~a   73 (110)
</pre><pre>                      ++++.....+++.++- .+.|.+.+. .+++.+...+.+++.+. +++.+
</pre><pre>T Consensus         4 ~~~i~a~~~v~~~~~~~~~~~~~~~~-~~~~~~g~~i~g~v~~~~g~~~~   52 (53)
</pre><pre>T W1IZ46_9GAMM/1    4 EITCVASTRITLDTPEVVCTQLMSTG-NLIVRNGGKMTGNIEHTGGSFSS   52 (53)
</pre><pre>T ss_pred             EEEEEeCceEEEeCCeEEEcCcEEEc-eEEEecCcEEEcceEEeCceeee
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>2N3D_A Bactofilin A; BacA, Bactofilin, cell shape, cytoskeleton, beta helix, STRUCTURAL PROTEIN; NMR {Caulobacter crescentus}
</pre><pre>Probab=79.02  E-value=18  Score=28.92  Aligned_cols=47  Identities=21%  Similarity=0.415  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             EECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   28 TLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        28 tLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      .|.-.|.++|  .|.+.|++.. ++|+|.+...+.+++... +|.+.|.|.
</pre><pre>T Consensus        51 ~i~~~g~i~i--~G~v~G~i~~-~~i~I~~~~~v~G~v~a~-~i~i~G~v~   97 (184)
</pre><pre>T 2N3D_A           51 GVTGEGELQI--DGVVKGDVRV-GRLTVGETGHVEGSVYAE-AVEVRGRVV   97 (184)
</pre><pre>T ss_dssp             EEECCSEEEE--CSEEEEEEEE-EEEEECSSCCBEEEEEEE-EEEESSEEE
</pre><pre>T ss_pred             eEEeCCeEEE--EEEEEcEEEE-eEEEECCCEEEEeEEEec-EEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>4RGI_A Uncharacterized protein; Chicago Center for Functional Annotation, Structural Genomics, PSI-Biology, Midwest Center for Structural Genomics, MCSG, alpha-beta sandwich; HET: GOL, MSE; 1.732A {Yersinia pestis}
</pre><pre>Probab=78.38  E-value=13  Score=24.60  Aligned_cols=25  Identities=32%  Similarity=0.705  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEEe
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIVG   39 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~g   39 (110)
</pre><pre>                      .|.||.+++.|.|.+.+|..|.-.+
</pre><pre>T Consensus        16 ~v~Yd~~~~~L~V~f~~g~~y~Y~~   40 (73)
</pre><pre>T 4RGI_A           16 SIGYDPDNRMLEIQFREQGTYQYLG   40 (73)
</pre><pre>T ss_dssp             EEEEETTTTEEEEEETTTEEEEEES
</pre><pre>T ss_pred             EEEEECCCCEEEEEECCCcEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>PF20000.1 ; fvmX2 ; FtsH ternary system domain X2
</pre><pre>Probab=73.92  E-value=30  Score=30.52  Aligned_cols=56  Identities=18%  Similarity=0.251  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecc--eEecCEEEEeeEEeCCc
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNG--TLDGPVQITGKLTVKGK   58 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g--~ldG~v~it~~ltv~g~   58 (110)
</pre><pre>                      .++.+.....+| .|.||+.++.|.|++.--=.++-.|.-  +|.|.|..+-.-...|.
</pre><pre>T Consensus        68 ~~~g~r~~~~gg-~v~ydp~t~elei~a~l~~~V~a~~~a~~~~~G~~~~ev~ae~~g~  125 (260)
</pre><pre>T A0A1G7X4D6_9AC   68 VGDCHLADVPGG-RIAYWPDTGELEISVELSTEIEAEGEGVVTIGGEVREDATAEAEAF  125 (260)
</pre><pre>T ss_pred             eCCeEEEecCCe-EEEEECCCCEEEEEEEeeceEEEEeeeEEEeeeEEeeeeeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>2PI2_F Replication protein A 14 kDa subunit; full-length RPA14/32, ssDNA binding protein, OB-fold, Dioxane, REPLICATION, DNA BINDING PROTEIN; HET: DIO; 2.0A {Homo sapiens} SCOP: b.40.4.3
</pre><pre>Probab=71.22  E-value=67  Score=24.70  Aligned_cols=58  Identities=16%  Similarity=0.212  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEeeEEeCCceeeeeeEEeccc
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITGKLTVKGKTDLLANTHIEGN   70 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~~ltv~g~~~~~~n~~v~gn   70 (110)
</pre><pre>                      |..++|+.+...++|.-++|+.++|.-.-    .+.+-|||.+++.-.+.+.+..-+....+
</pre><pre>T Consensus        50 GkV~~~d~~~~~~~l~~~Dg~~V~V~l~~~~~~~~~~~vEViG~V~~~~~I~~~~~~~~g~~  111 (142)
</pre><pre>T 2PI2_F           50 GRLEKIHPTGKMFILSDGEGKNGTIELMEPLDEEISGIVEVVGRVTAKATILCTSYVQFKED  111 (142)
</pre><pre>T ss_dssp             EEEEEECTTSSEEEEECTTSCEEEEEESSCCSSCCCSEEEEEEEECTTSCEEEEEEEECCCS
</pre><pre>T ss_pred             EEEEEEeCCCCEEEEEeCCCCEEEEEEcCCCCCCCCCEEEEEEEECCCCeEEEEEEEEcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>5OPQ_C 3,6-anhydro-D-galactosidase; glycoside hydrolase 3, 6-anhydro-D-galactosidase 3, 6-anhydro-D-galactose carrageenan Zobellia galactanivorans, HYDROLASE; HET: MPD, TRS; 1.7A {Zobellia galactanivorans}
</pre><pre>Probab=68.18  E-value=51  Score=29.99  Aligned_cols=49  Identities=22%  Similarity=0.217  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             Ccc--ceeEeeCCCCEEEeecccCeEEE----EECCCceEEEEecc-eEecCEEEE
</pre><pre>Q WP_214195415.1    2 REC--EHITVYEDGTEVRYNRESHALTI----TLAEGGTYRIVGNG-TLDGPVQIT   50 (110)
</pre><pre>Q Consensus         2 s~~--eHvtrY~DGTev~Ydr~~haLti----tLaeGGt~~I~g~g-~ldG~v~it   50 (110)
</pre><pre>                      +++  .-.++|+||++|--|..+...++    +|++.|-+.+.+.+ ..-|--.|+
</pre><pre>T Consensus       634 ~~~~~v~~t~y~nG~~v~vN~~~~~~~~~~g~~v~~~~~~v~~~~~~~~~~~~~~~  689 (693)
</pre><pre>T 5OPQ_C          634 SADYKVQRSRFSSGTEVIANLGPVAQKIEGGISIPGYGYRIQMKDGSLKTGHFQVS  689 (693)
</pre><pre>T ss_dssp             STTSCEEEEEETTTEEEEEECSSSCEECTTSCEECTTEEEEECTTSCEEEEEEEEE
</pre><pre>T ss_pred             cCCcceEEEEeCCCCEEEEEcCCCceEEeCcEEeCCCeEEEEcCCCCeEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>PF12182.10 ; DUF3642 ; Bacterial lipoprotein
</pre><pre>Probab=68.12  E-value=50  Score=25.18  Aligned_cols=50  Identities=22%  Similarity=0.209  Sum_probs=0.0  Template_Neff=3.800
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeeccc----CeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeee
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRES----HALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLAN   64 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~----haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n   64 (110)
</pre><pre>                      ||+-+..|++.    .+||+... |.+++|+=. .-+|.-+|+.+|. .....+.|+
</pre><pre>T Consensus         6 dGtYv~~d~~~~dtdD~~Tl~~~-G~~gt~t~~-d~Dg~~~i~v~ld-~nql~~iGd   59 (86)
</pre><pre>T A5VKA8_LACRD/7    6 TGRYVYADKNNPDTNDSLTWRKE-GEHYLINLE-DRDVTNPLTATLD-NNKLIVTGS   59 (86)
</pre><pre>T ss_pred             CceEEEcCCCCCCCCCeEEEEEe-CCeEEEEEe-cCCCCCCEEEEEe-CCcEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>6J0M_C Pvc8; assembly, Photorhabdus asymbiotica, PVC, contractile injection system, bacteriophage-like, PROTEIN TRANSPORT; 3.9A {Photorhabdus asymbiotica}
</pre><pre>Probab=67.31  E-value=97  Score=26.34  Aligned_cols=65  Identities=11%  Similarity=0.147  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCC---EEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeee-----------EEec
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGT---EVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLAN-----------THIE   68 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGT---ev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n-----------~~v~   68 (110)
</pre><pre>                      ......+..+|.   +++||...+.++|.. ..+.+.+..++        .-+|...+++.+.++           +..+
</pre><pre>T Consensus       454 ~~~~~~~~~~g~~~~~l~~~~~~~~i~~~~-~~~~~~~~~~~--------~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~  524 (538)
</pre><pre>T 6J0M_C          454 NREKVLIVKKGEAQQQLVIDGKEKMIRINA-GENQIMLQQDK--------DISLSTKKELTLKAQTMNATMDKSLAMSGK  524 (538)
</pre><pre>T ss_dssp             CCEEEECEECSSCEECCEEETTTTEEEEEB-SSSCEEEECSS--------CCCCBCSSEEEEEESEEEEEESSCEEEEBS
</pre><pre>T ss_pred             cccEEEEEecCCceeEEEEECCCCeEEEEe-CCceEEEcCCC--------cEEEcCCCeEEEeecceEEEeCCceeEeec
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cceecCce
</pre><pre>Q WP_214195415.1   69 GNLTASDE   76 (110)
</pre><pre>Q Consensus        69 gni~a~~e   76 (110)
</pre><pre>                      ++|...++
</pre><pre>T Consensus       525 ~~i~i~~~  532 (538)
</pre><pre>T 6J0M_C          525 NSVEIKGA  532 (538)
</pre><pre>T ss_dssp             SCEEEECS
</pre><pre>T ss_pred             ceEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>3KDF_C Replication protein A 14 kDa subunit; Wheat germ cell free, protein complex, Center for Eukaryotic Structural Genomics, PSI, replication protein A, Homo; HET: EDO, MSE; 1.975A {Homo sapiens} SCOP: b.40.4.3
</pre><pre>Probab=66.43  E-value=86  Score=22.80  Aligned_cols=58  Identities=16%  Similarity=0.212  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEeeEEeCCceeeeeeEEeccc
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITGKLTVKGKTDLLANTHIEGN   70 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~~ltv~g~~~~~~n~~v~gn   70 (110)
</pre><pre>                      |..++|+.....++|.-++|+.++|.-..    .+.+-+||.+++.-.+.+.+..-+....+
</pre><pre>T Consensus        29 G~v~~~~~~~~~~~l~~~d~~~V~V~l~~~~~~~~~~~vEviG~V~~~~~i~~~~~~~~~~~   90 (121)
</pre><pre>T 3KDF_C           29 GRLEKIHPTGKMFILSDGEGKNGTIELMEPLDEEISGIVEVVGRVTAKATILCTSYVQFKED   90 (121)
</pre><pre>T ss_dssp             EEEEEECTTSSEEEEECTTSCEEEEECSSCCSSCCCSEEEEEEEECTTSCEEEEEEEECCCS
</pre><pre>T ss_pred             EEEEEEcCCCCEEEEEeCCCCEEEEEECCCCCCCCCcEEEEEEEECCCCeEEEEEEEEcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>1EW4_A CYAY PROTEIN; Friedreich ataxia, frataxin family, CyaY, iron homeostasis, UNKNOWN FUNCTION; 1.4A {Escherichia coli} SCOP: d.82.2.1
</pre><pre>Probab=66.16  E-value=25  Score=25.41  Aligned_cols=28  Identities=21%  Similarity=0.429  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             eCCCCEEEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   10 YEDGTEVRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        10 Y~DGTev~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +.++..+-++...+.|+|.++++++|.|
</pre><pre>T Consensus        24 ~~~~~~~d~~~~~~vl~i~~~~~~~~Vi   51 (106)
</pre><pre>T 1EW4_A           24 WDGDSDIDCEINGGVLTITFENGSKIII   51 (106)
</pre><pre>T ss_dssp             CCSSSCCEEEEETTEEEEECTTSCEEEE
</pre><pre>T ss_pred             cCCCCCeEeeeeCCeEEEEECCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>6RIA_K bactofilin; Prokaryotic cytoskeletons, PROTEIN FIBRIL; 3.5A {Thermus thermophilus}
</pre><pre>Probab=64.96  E-value=39  Score=24.31  Aligned_cols=45  Identities=7%  Similarity=0.083  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   33 GTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        33 Gt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      +.+.|.+...+.|++.. ..+.+.+..++++.+.............
</pre><pre>T Consensus        78 ~~l~i~~~a~i~G~v~~-~~i~i~~ga~i~G~i~~~~~~~~~~~~~  122 (123)
</pre><pre>T 6RIA_K           78 EKVVLSKTARFTGQLKA-QALEVEAGAVFVGQSVAGEHKALEAPKE  122 (123)
</pre><pre>T ss_dssp             SEEEECTTCEEEEEEEE-EEEEEC----------------------
</pre><pre>T ss_pred             EEEEECCCeEEEeEEEe-CeEEECCCCEEEEEEEeCCccccCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>6RIA_L bactofilin; Prokaryotic cytoskeletons, PROTEIN FIBRIL; 3.5A {Thermus thermophilus}
</pre><pre>Probab=64.96  E-value=39  Score=24.31  Aligned_cols=45  Identities=7%  Similarity=0.083  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             ceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   33 GTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        33 Gt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      +.+.|.+...+.|++.. ..+.+.+..++++.+.............
</pre><pre>T Consensus        78 ~~l~i~~~a~i~G~v~~-~~i~i~~ga~i~G~i~~~~~~~~~~~~~  122 (123)
</pre><pre>T 6RIA_L           78 EKVVLSKTARFTGQLKA-QALEVEAGAVFVGQSVAGEHKALEAPKE  122 (123)
</pre><pre>T ss_dssp             SEEEECTTCEEEEEEEE-EEEEEC----------------------
</pre><pre>T ss_pred             EEEEECCCeEEEeEEEe-CeEEECCCCEEEEEEEeCCccccCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF13600.8 ; DUF4140 ; N-terminal domain of unknown function (DUF4140)
</pre><pre>Probab=64.06  E-value=33  Score=23.61  Aligned_cols=28  Identities=25%  Similarity=0.472  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             eEeeCCCCEEEeecccCeEEEEECCC-ceEEEEe
</pre><pre>Q WP_214195415.1    7 ITVYEDGTEVRYNRESHALTITLAEG-GTYRIVG   39 (110)
</pre><pre>Q Consensus         7 vtrY~DGTev~Ydr~~haLtitLaeG-Gt~~I~g   39 (110)
</pre><pre>                      ||.|.||++|..     ..++.|+.| ..+.|.+
</pre><pre>T Consensus         1 Vtvy~~~A~v~r-----~~~~~l~~G~~~i~i~~   29 (100)
</pre><pre>T A7HCS6_ANADF/3    1 VTVYRQSARVTR-----VAKVELPDGPTRVVLEG   29 (100)
</pre><pre>T ss_pred             CEEecCCeEEEE-----EEEEEcCCCCeEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>PF20003.1 ; fvmX5 ; FtsH ternary system domain X5
</pre><pre>Probab=63.97  E-value=39  Score=28.08  Aligned_cols=34  Identities=21%  Similarity=0.436  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEECCCceEEEEecceEec
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLAEGGTYRIVGNGTLDG   45 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG   45 (110)
</pre><pre>                      ||-.|.||+++..|+|++..-=.+.+.+.+...+
</pre><pre>T Consensus        64 ~g~~v~~dp~t~~l~i~a~~~~~v~~e~e~~~~~   97 (188)
</pre><pre>T L7U9B5_MYXSD/1   64 DGVRVAVDVTTGDVSVTLSDESSVNLVATGEVRT   97 (188)
</pre><pre>T ss_pred             CCEEEEEECCCCEEEEEEeccceEEEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>PF10049.11 ; DUF2283 ; Protein of unknown function (DUF2283)
</pre><pre>Probab=61.85  E-value=26  Score=21.65  Aligned_cols=17  Identities=35%  Similarity=0.561  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCC
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEG   32 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeG   32 (110)
</pre><pre>                      ++||++...|.|.|.++
</pre><pre>T Consensus         2 i~YD~~~D~l~i~~~~~   18 (48)
</pre><pre>T B0JGR7_MICAN/2    2 IDYDRAANAAYIRRFEG   18 (48)
</pre><pre>T ss_pred             eEEeCCCCEEEEEECCC
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>PF07202.15 ; Tcp10_C ; T-complex protein 10 C-terminus
</pre><pre>Probab=61.32  E-value=51  Score=19.13  Aligned_cols=29  Identities=14%  Similarity=0.207  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCCEEEeecccCeEEEEECCC
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGTEVRYNRESHALTITLAEG   32 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGTev~Ydr~~haLtitLaeG   32 (110)
</pre><pre>                      +....+.|.||+..+...+ ..-.|.++.|
</pre><pre>T Consensus         2 ~~~~e~~~pdG~v~~~~~~-G~~~i~f~nG   30 (35)
</pre><pre>T A0A3M7RLS4_BRA    2 DGSEETRLPNGTLIKVDKN-GDKVIEYPNK   30 (35)
</pre><pre>T ss_pred             CCeEEEECCCCCEEEECCC-CCEEEEcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF12992.9 ; DUF3876 ; Domain of unknown function, B. Theta Gene description (DUF3876)
</pre><pre>Probab=61.01  E-value=36  Score=24.19  Aligned_cols=32  Identities=16%  Similarity=0.267  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCC---EEEeecccCeEEEEECCCceEE
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGT---EVRYNRESHALTITLAEGGTYR   36 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGT---ev~Ydr~~haLtitLaeGGt~~   36 (110)
</pre><pre>                      ..+.+..+..|-   .|.||++.+.|+  |+.+|.|+
</pre><pre>T Consensus        58 ~~~g~l~i~~g~g~~~i~Yd~~~D~L~--ls~~G~Y~   92 (93)
</pre><pre>T Q8A5A2_BACTN/4   58 QDGSQYFIATASKRLSVDYDPAKDVLS--ISSLGDYL   92 (93)
</pre><pre>T ss_pred             EeCCEEEEEcCCcEEEEEeeCcCCEEE--EcCCccee
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>d1hf2a1 b.80.3.1 (A:100-206) Cell-division inhibitor MinC, C-terminal domain {Thermotoga maritima [TaxId: 2336]}
</pre><pre>Probab=57.19  E-value=23  Score=26.78  Aligned_cols=35  Identities=20%  Similarity=0.253  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             ec-CEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   44 DG-PVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        44 dG-~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      .| .+++.+.+.|.|++.-.+.+.+.|||.+-|.|.
</pre><pre>T Consensus        11 sGq~i~~~g~vvI~G~V~~G~~V~a~G~I~V~G~v~   46 (107)
</pre><pre>T d1hf2a1          11 SGQTVVHSGDVIVFGNVNKGAEILAGGSVVVFGKAQ   46 (107)
</pre><pre>T ss_dssp             TTEEEEESSCEEESSCBCTTCEEEESSCEEEEEEEC
</pre><pre>T ss_pred             CCCEEEECCcEEEeCCcCCCCEEEeCCcEEEEeeEE
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>d2icya1 b.81.1.4 (A:384-466) UDP-glucose pyrophosphorylase 2 (UDPGP 2) {Thale cress (Arabidopsis thaliana) [TaxId: 3702]}
</pre><pre>Probab=57.13  E-value=16  Score=26.54  Aligned_cols=26  Identities=19%  Similarity=0.316  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             eeEEeCCceeeeeeEEeccceecCce
</pre><pre>Q WP_214195415.1   51 GKLTVKGKTDLLANTHIEGNLTASDE   76 (110)
</pre><pre>Q Consensus        51 ~~ltv~g~~~~~~n~~v~gni~a~~e   76 (110)
</pre><pre>                      ..|+|+|++.+..++.+.|++...+.
</pre><pre>T Consensus        37 ~sL~V~Gdv~Fg~~V~lkG~V~i~n~   62 (83)
</pre><pre>T d2icya1          37 DSLKVSGDVWFGSSIVLKGKVTVAAK   62 (83)
</pre><pre>T ss_dssp             EEEEEESEEEECSSCEEEEEEEEECC
</pre><pre>T ss_pred             CeeEEEeeEEeCCCcEEEEEEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>4HS5_B Protein CyaY; pFXN, frataxin, CyaY, Psychromonas ingrahamii, Frataxin-like domain, CyaY protein family (pfam ID PF01491), Iron homeostasis-redox balance; 1.45A {Psychromonas ingrahamii} SCOP: d.82.2.0
</pre><pre>Probab=56.90  E-value=55  Score=23.52  Aligned_cols=27  Identities=22%  Similarity=0.421  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      .+..+-|++....|+|++++++.|.|.
</pre><pre>T Consensus        24 ~~~~~d~~~~~~vL~i~~~~~~~~vin   50 (105)
</pre><pre>T 4HS5_B           24 SGADVDYDQNGSLLTLEFENHTKLIIN   50 (105)
</pre><pre>T ss_dssp             HTCSEEEEEETTEEEEEETTCCEEEEE
</pre><pre>T ss_pred             cCCceEEEecCCeEEEEECCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>PF15998.7 ; DUF4773 ; Domain of unknown function (DUF4773)
</pre><pre>Probab=56.28  E-value=72  Score=23.31  Aligned_cols=30  Identities=23%  Similarity=0.420  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeC
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVK   56 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~   56 (110)
</pre><pre>                      .+.|+++.-+|.++|            +++|.+.++.++.+.
</pre><pre>T Consensus        27 ~~~y~~~~~~l~~~l------------~~n~~~i~~~~is~~   56 (116)
</pre><pre>T K7IP58_NASVI/1   27 NVTYDPDEFAFTAKL------------SMNEQLLFTRTVSGK   56 (116)
</pre><pre>T ss_pred             EEEEeCccceEEEEE------------EECCEEEEEEEeeCC
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>d4hs5a_ d.82.2.0 (A:) automated matches {Psychromonas ingrahamii [TaxId: 357804]}
</pre><pre>Probab=55.75  E-value=47  Score=24.54  Aligned_cols=22  Identities=23%  Similarity=0.392  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +.|+....-|+|.++++|+|.|
</pre><pre>T Consensus        28 ~d~~~~~gvL~i~~~~~~~~Vi   49 (105)
</pre><pre>T d4hs5a_          28 VDYDQNGSLLTLEFENHTKLII   49 (105)
</pre><pre>T ss_dssp             EEEEEETTEEEEEETTSCEEEE
</pre><pre>T ss_pred             eEEEecCCEEEEEECCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>4GNX_X Putative uncharacterized protein; ssDNA binding, DNA BINDING PROTEIN-DNA complex; 2.8A {Ustilago maydis}
</pre><pre>Probab=52.19  E-value=1.6e+02  Score=21.37  Aligned_cols=60  Identities=15%  Similarity=0.232  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEeeEEeCCce------eeeeeEEeccceecC
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITGKLTVKGKT------DLLANTHIEGNLTAS   74 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~~ltv~g~~------~~~~n~~v~gni~a~   74 (110)
</pre><pre>                      |..++++  ...+++.-++|+.+++.-..    .+..-|||.+.+.=.+.+      .+.-.+....++...
</pre><pre>T Consensus        25 Gkv~~~~--~~~~~l~~~d~~~V~v~~~~~~~~~~~~~vEviG~V~~~~~i~~~~~~~~~~~~~~g~~~D~~   94 (114)
</pre><pre>T 4GNX_X           25 GKVHKVT--GNTLLMQTSDLGNVEIAMTPDSDVSSSTFVEVTGKVSDAGSSFQANQIREFTTVDCGHDVDLT   94 (114)
</pre><pre>T ss_dssp             EEEEEEE--TTEEEEEETTSCEEEEECCSSCCGGGCSEEEEEEEECTTSSCEEEEEEEECSCCC----CCHH
</pre><pre>T ss_pred             EEEEEEe--CCEEEEEeCCCCEEEEEECCCCCCCCCcEEEEEEEEcCCCceeecCeeEEEEEEECCCCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>4JPD_A Protein CyaY; SSGCID, Seattle Structural Genomics Center for Infectious Disease, CyaY, Frataxin, iron binding protein, METAL BINDING PROTEIN; 1.9A {Burkholderia cenocepacia}
</pre><pre>Probab=51.51  E-value=79  Score=23.02  Aligned_cols=29  Identities=17%  Similarity=0.382  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             eCCCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   10 YEDGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        10 Y~DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      +..+..+-|++....|+|++++++.|.|.
</pre><pre>T Consensus        29 ~~~~~~~d~~~~~~vL~i~~~~~~~~vin   57 (112)
</pre><pre>T 4JPD_A           29 NDGDHDIDLERNGSVLTLTFENGSKIIVN   57 (112)
</pre><pre>T ss_dssp             HHTTCCCEEEEETTEEEEECTTSCEEEEE
</pre><pre>T ss_pred             cCCCCcEEEEeeCCEEEEEeCCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>d4d6ga2 b.24.2.0 (A:839-1003) automated matches {Streptococcus pneumoniae [TaxId: 406556]}
</pre><pre>Probab=51.39  E-value=62  Score=26.86  Aligned_cols=25  Identities=20%  Similarity=0.522  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             CEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   14 TEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        14 Tev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      .+|.||.+.+.+||++.-.|.+.|.
</pre><pre>T Consensus       138 ~~v~~d~~tg~~Tltv~hNG~vd~~  162 (165)
</pre><pre>T d4d6ga2         138 PSLNFDQKNHMVTITINSNGNLEFE  162 (165)
</pre><pre>T ss_dssp             CEEEEETTTTEEEEEEEESSEEEEE
</pre><pre>T ss_pred             ceEEEeCCCCeEEEEEecCceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>4LD1_A Uncharacterized protein; beta sheet, G-box, Centriole organisation, structural protein, protein binding; HET: PGE, PG4, MPD; 1.44A {Danio rerio}
</pre><pre>Probab=50.88  E-value=1.1e+02  Score=21.82  Aligned_cols=36  Identities=17%  Similarity=0.149  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             eEeeCCCCEEEeecccCeEEEEECCCceEEEEecceE
</pre><pre>Q WP_214195415.1    7 ITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTL   43 (110)
</pre><pre>Q Consensus         7 vtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~l   43 (110)
</pre><pre>                      ...|.||+.+.|+.+.. +.++.++|..+.|.-.|.+
</pre><pre>T Consensus       146 ~~~~~dG~~~~~~~~G~-~~~~~~~G~~~~~~~~G~~  181 (184)
</pre><pre>T 4LD1_A          146 RREYPDGTVKTVYSDGR-QETQYPTGRVRLKDPQGKV  181 (184)
</pre><pre>T ss_dssp             EEECTTSCEEEEETTSC-EEEECCC------------
</pre><pre>T ss_pred             EEEcCCCCEEEEeCCCC-EEEEcCCCCEEEECCCCCE
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>6QVI_B ComZ; competence pilin, DNA receptor, beta-solenoid domain, pseudopilin-like domain, DNA BINDING PROTEIN; 2.72A {Thermus thermophilus (strain HB27 / ATCC BAA-163 / DSM 7039)}
</pre><pre>Probab=50.73  E-value=1.3e+02  Score=27.82  Aligned_cols=52  Identities=27%  Similarity=0.459  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEEECCC--ceEEEE---ecc--eEecCEEEEe-eEEeCCceeeee--------eEEe------ccceecCcee
</pre><pre>Q WP_214195415.1   26 TITLAEG--GTYRIV---GNG--TLDGPVQITG-KLTVKGKTDLLA--------NTHI------EGNLTASDEI   77 (110)
</pre><pre>Q Consensus        26 titLaeG--Gt~~I~---g~g--~ldG~v~it~-~ltv~g~~~~~~--------n~~v------~gni~a~~eV   77 (110)
</pre><pre>                      +.+.+.+  |.+.+.   ...  +|.|.+-|++ +|++++...+.+        .+.+      .|+|.+.+.+
</pre><pre>T Consensus       346 ~~~~~~~~~~~~~~~~~~~~~~l~~~G~i~v~G~~i~i~~~i~~~g~~~~~~~~tIva~~~~~~~G~I~i~~~~  419 (534)
</pre><pre>T 6QVI_B          346 TFTRLDGSRGGFRYTYTGGQELLEVFGDVVLEGIDAVLNRPVDYRAQSGSAKSATLAVLKLGGNGGNLDINGNL  419 (534)
</pre><pre>T ss_dssp             EEECTTSCEEEEEEECCSSSEEEEEESCEEEESCEEEECSCEEEEEEBTTBSEEEEEEEEETTEECCEEECSCE
</pre><pre>T ss_pred             eeecCCCCcceEEEecCCCCCeeEEEeEEEEeCccEEECCcEEeecCCCCCceeEEEEEecCCCCCcEEEcCCe
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>5VTG_A Translocation and assembly module subunit TamB; Chaperone, Beta-sheet, Periplasm; 1.859A {Escherichia coli}
</pre><pre>Probab=48.65  E-value=1.6e+02  Score=22.65  Aligned_cols=43  Identities=19%  Similarity=0.238  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             eEEEEECCCceEEEEecc---eEecCEEEEeeEEeCCceeeeeeEEecc
</pre><pre>Q WP_214195415.1   24 ALTITLAEGGTYRIVGNG---TLDGPVQITGKLTVKGKTDLLANTHIEG   69 (110)
</pre><pre>Q Consensus        24 aLtitLaeGGt~~I~g~g---~ldG~v~it~~ltv~g~~~~~~n~~v~g   69 (110)
</pre><pre>                      .|.+.+.-+..++|.+++   .+.|++.|.+..   ++..+.|++.+..
</pre><pre>T Consensus        65 ~ldl~v~~~~~~~i~~~~l~~~~~G~l~l~~~~---~~~~~~G~i~i~~  110 (184)
</pre><pre>T 5VTG_A           65 NSNLIVHVGNNVRIDAFGLKARLTGDLNVVQDK---QGLGLNGQINIPE  110 (184)
</pre><pre>T ss_dssp             EEEEEEEEEEEEEEEETTEEEEEEEEEEEEEET---TEEEEEEEEEEEE
</pre><pre>T ss_pred             EEeEEEEeCCcEEEEECCEEEEEEEEEEEEECC---CceEEEEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>3K8R_A Uncharacterized protein; UNKNOWN FUNCTION, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2; HET: MSE; 2.75A {Rhodospirillum rubrum ATCC 11170}
</pre><pre>Probab=47.84  E-value=85  Score=23.67  Aligned_cols=24  Identities=29%  Similarity=0.613  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CEEEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   14 TEVRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        14 Tev~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      ..|+||++...|-|.|.+|-.+.+
</pre><pre>T Consensus        28 ~~v~~~~~~~~l~i~l~dG~~~~~   51 (124)
</pre><pre>T 3K8R_A           28 AKARYDRSSARVIVDLENGCTFAF   51 (124)
</pre><pre>T ss_dssp             EEEEEETTTTEEEEEETTSCEEEE
</pre><pre>T ss_pred             EEEEEECCCCEEEEEECCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>3PET_B Putative adhesin; RIGHT-HANDED BETA-HELIX, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, CELL ADHESION; HET: MSE, PG4; 2.07A {Bacteroides fragilis NCTC 9343}
</pre><pre>Probab=47.50  E-value=2.2e+02  Score=21.63  Aligned_cols=68  Identities=15%  Similarity=0.131  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             eeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCcee-eeeeEEeccceec
</pre><pre>Q WP_214195415.1    6 HITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTD-LLANTHIEGNLTA   73 (110)
</pre><pre>Q Consensus         6 HvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~-~~~n~~v~gni~a   73 (110)
</pre><pre>                      .+.....|..--++-....+.+.....|.+.+...+.|+..+.=.+.+.+.|... +.....-.|+|..
</pre><pre>T Consensus       152 ~~~~sg~g~i~~~~~~~~~~~~~~~~~g~i~v~~~~~l~~~~~g~g~i~~~g~~~~~~~~~sg~G~i~~  220 (221)
</pre><pre>T 3PET_B          152 ALHSKGVGNIEAGNLRANAVEASSQGVGDITCNATESIDAAVRGVGSIKYKGSPTIKSLSKKGVGTIKN  220 (221)
</pre><pre>T ss_dssp             EEEECSBSCEEEEEEEEEEEEEEECSBSCEEEEEEEEEEEEECSBSCEEEESCCSEEEEEECSBSCEEE
</pre><pre>T ss_pred             EEEEcCceeEEecceEecEEEEEEcccCeEEEEecCeEEEEEecccEEEEEecCceEEEEeccceeeee
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>d2pi2e_ b.40.4.3 (E:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=47.50  E-value=2e+02  Score=21.03  Aligned_cols=58  Identities=16%  Similarity=0.212  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecc----eEecCEEEEeeEEeCCceeeeeeEEeccc
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNG----TLDGPVQITGKLTVKGKTDLLANTHIEGN   70 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g----~ldG~v~it~~ltv~g~~~~~~n~~v~gn   70 (110)
</pre><pre>                      |+.++++.+...+.+.-++|+.++|.-+-    .+.+-|||.+.+.=.+.+....-.....+
</pre><pre>T Consensus        28 GkV~~~~~~~~~~~~~~~D~~~V~V~~~~~~~~~~~~~vEViG~V~~~~~i~~~~~~~~~~~   89 (117)
</pre><pre>T d2pi2e_          28 GRLEKIHPTGKMFILSDGEGKNGTIELMEPLDEEISGIVEVVGRVTAKATILCTSYVQFKED   89 (117)
</pre><pre>T ss_dssp             EEEEEECTTSSEEEEECTTSCEEEEECSSCCSSCCCSEEEEEEEECTTSCEEEEEEEECCCS
</pre><pre>T ss_pred             EEEEEEeCCCCEEEEEeCCCCEEEEEEcCCCCCCCCcEEEEEEEEcCCCeEEEEEEEEccCC
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>4MPZ_A Sas-4; beta-spine, centrosome biogenesis, STRUCTURAL PROTEIN; 2.701A {Drosophila melanogaster}
</pre><pre>Probab=47.46  E-value=2.1e+02  Score=22.83  Aligned_cols=49  Identities=22%  Similarity=0.251  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CccceeEe-eCCCCEEEeecccCeEEEEECCCceEEEEecc-----eEecCEEEE
</pre><pre>Q WP_214195415.1    2 RECEHITV-YEDGTEVRYNRESHALTITLAEGGTYRIVGNG-----TLDGPVQIT   50 (110)
</pre><pre>Q Consensus         2 s~~eHvtr-Y~DGTev~Ydr~~haLtitLaeGGt~~I~g~g-----~ldG~v~it   50 (110)
</pre><pre>                      ........ +.||..+.|.........++++|-....-..|     .-+|.++|.
</pre><pre>T Consensus        61 ~nG~~~~~~~~dG~~~~~~~~~~~~~~~~~dG~~~~~~pdG~~~~~~~dG~~~i~  115 (209)
</pre><pre>T 4MPZ_A           61 FNKDIKETNIREGTVKYYYAETNTWHTSYLDGLEILEFPNGQTEHRRKDGTVEIH  115 (209)
</pre><pre>T ss_dssp             SSSCEEEEETTTTEEEEEETTTTEEEEECSSSCEEEECTTSCEEEECTTSCEEEE
</pre><pre>T ss_pred             eCCceEEEEcCCCeEEEEEcCCCeEEEEeCCCcEEEEcCCCCEEEEeCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>4OPW_A Uncharacterized protein; PF10988 family protein, DUF2807, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-BIOLOGY; HET: EDO, MSE; 1.75A {Parabacteroides merdae ATCC 43184}
</pre><pre>Probab=47.32  E-value=2.4e+02  Score=22.00  Aligned_cols=67  Identities=15%  Similarity=0.258  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             eeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEecccee
</pre><pre>Q WP_214195415.1    6 HITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLT   72 (110)
</pre><pre>Q Consensus         6 HvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~   72 (110)
</pre><pre>                      .+.-.+.|..-..+-....+.+.....|...+.....|+..+.=...+.+.|..+++......|+|.
</pre><pre>T Consensus       172 ~~~~~g~~~i~~~~l~~~~~~i~~~~~~~i~v~~~~~l~~~~~g~g~i~~~g~p~~~~~~~~~g~i~  238 (241)
</pre><pre>T 4OPW_A          172 DYSITTDGEIMAFGVAVPEVNCKITGKGSAQIHPTDNLKATIVGKGNIRYKGPTAVQQKVIGKGTVE  238 (241)
</pre><pre>T ss_dssp             EEEEEESCEEEEEEEEEEEEEEEEEESCEEEEEEEEEEEEEEESSCEEEECSSCEEEEEEESSCCEE
</pre><pre>T ss_pred             EEEEcCCceEEEcceEecEEEEEEecceeEEEeecCcEEEEEcCCeEEEEeCCCceeEEEecceeEE
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>d4qmea1 b.98.1.1 (A:2-188) Aminopeptidase N (APN) N-terminal domain {Neisseria meningitidis [TaxId: 122586]}
</pre><pre>Probab=44.55  E-value=76  Score=23.48  Aligned_cols=27  Identities=22%  Similarity=0.364  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      +|..+.|.++.+.|.|.|+.++.++|.
</pre><pre>T Consensus        62 ~~~~~~~~~~~~~l~i~~~~~~~~~l~   88 (187)
</pre><pre>T d4qmea1          62 NGAAADYVLEGETLTIAGVPSERFTVE   88 (187)
</pre><pre>T ss_dssp             TTEECCCEEETTEEEECCCCSSCEEEE
</pre><pre>T ss_pred             CCEEceEEEeCCEEEEecCCCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>3JX8_A Putative lipoprotein; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesin, CELL ADHESION; HET: MSE, SO4, GOL; 2.16A {Parabacteroides distasonis ATCC 8503}
</pre><pre>Probab=44.13  E-value=2.7e+02  Score=21.70  Aligned_cols=66  Identities=15%  Similarity=0.294  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEecccee
</pre><pre>Q WP_214195415.1    7 ITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLT   72 (110)
</pre><pre>Q Consensus         7 vtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~   72 (110)
</pre><pre>                      +.....|..--.+-....+.+.....|.+.+...+.|+..+.=...+++.|...+.......|+|.
</pre><pre>T Consensus       182 ~~~~~~g~i~~~~~~~~~~~~~~~g~g~i~~~~~~~l~~~~~g~g~i~~~g~~~~~~~~s~~g~i~  247 (250)
</pre><pre>T 3JX8_A          182 FNVAGSGEIKAFDCQARKAECNIASSGEISVYATQILDANIVGSGEIHYKGDPEISKSIMGSGSIN  247 (250)
</pre><pre>T ss_dssp             EEEESSCEEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEESSCEEEEESCCEEEEEEESSCEEE
</pre><pre>T ss_pred             EEEeCceeEEeeeeEEEEEEEEEccCcEEEEEEeeeEEEEEecceEEEEEcCCccceEEecceeeE
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>3JX8_F Putative lipoprotein; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, Lipoprotein, Adhesin, CELL ADHESION; HET: GOL, SO4, MSE; 2.16A {Parabacteroides distasonis ATCC 8503}
</pre><pre>Probab=44.13  E-value=2.7e+02  Score=21.70  Aligned_cols=66  Identities=15%  Similarity=0.294  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             eEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEecccee
</pre><pre>Q WP_214195415.1    7 ITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLT   72 (110)
</pre><pre>Q Consensus         7 vtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~   72 (110)
</pre><pre>                      +.....|..--.+-....+.+.....|.+.+...+.|+..+.=...+++.|...+.......|+|.
</pre><pre>T Consensus       182 ~~~~~~g~i~~~~~~~~~~~~~~~g~g~i~~~~~~~l~~~~~g~g~i~~~g~~~~~~~~s~~g~i~  247 (250)
</pre><pre>T 3JX8_F          182 FNVAGSGEIKAFDCQARKAECNIASSGEISVYATQILDANIVGSGEIHYKGDPEISKSIMGSGSIN  247 (250)
</pre><pre>T ss_dssp             EEEESSCEEEEEEEEEEEEEEEEESSCEEEEEEEEEEEEEEESSCEEEEESCCEEEEEEESSCEEE
</pre><pre>T ss_pred             EEEeCceeEEeeeeEEEEEEEEEccCcEEEEEEeeeEEEEEecceEEEEEcCCccceEEecceeeE
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>PF16986.7 ; CzcE ; Heavy-metal resistance protein CzcE
</pre><pre>Probab=43.03  E-value=1.9e+02  Score=19.96  Aligned_cols=32  Identities=6%  Similarity=0.039  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             EeeCCCCEEEeecccCeEEEEECC--CceEEEEe
</pre><pre>Q WP_214195415.1    8 TVYEDGTEVRYNRESHALTITLAE--GGTYRIVG   39 (110)
</pre><pre>Q Consensus         8 trY~DGTev~Ydr~~haLtitLae--GGt~~I~g   39 (110)
</pre><pre>                      .....|..|++.-....++|++.+  -|.|.+.|
</pre><pre>T Consensus        25 i~v~~Ge~V~f~~~~~s~~~~f~~~~~G~~~~~C   58 (80)
</pre><pre>T D0IXY8_COMT2/5   25 LNLVCGETVEFTNGKEQFIWKFDVMGHRPVDLSK   58 (80)
</pre><pre>T ss_pred             EEECCCCEEEEEcCCeEEEEEEecCCCccEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>d1ew4a_ d.82.2.1 (A:) CyaY {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=42.98  E-value=1e+02  Score=22.92  Aligned_cols=26  Identities=23%  Similarity=0.374  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +...+-++....-|+|.++.+|+|.|
</pre><pre>T Consensus        26 ~~~d~d~~~~~gvL~i~~~~~~~~Vi   51 (106)
</pre><pre>T d1ew4a_          26 GDSDIDCEINGGVLTITFENGSKIII   51 (106)
</pre><pre>T ss_dssp             SSSCCEEEEETTEEEEECTTSCEEEE
</pre><pre>T ss_pred             CCCCeEEEEeCCEEEEEeCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>TIGR03422 mito_frataxin; frataxin. Frataxin is a mitochondrial protein, mutation of which leads to the disease Friedreich's ataxia.
</pre><pre>Probab=41.70  E-value=1e+02  Score=22.66  Aligned_cols=22  Identities=36%  Similarity=0.428  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +-+++...-|+|.++++|+|.|
</pre><pre>T Consensus        29 ~D~~~~~~vL~i~~~~~~~~vi   50 (97)
</pre><pre>T TIGR03422        29 FDVEYSSGVLTLELPSVGTYVI   50 (97)
</pre><pre>T ss_pred             eEEEeeCCEEEEEeCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>TIGR03421 FeS_CyaY; iron donor protein CyaY. Members of this protein family are the iron-sulfur cluster (FeS) metabolism protein CyaY, a homolog of eukaryotic frataxin.
</pre><pre>Probab=41.39  E-value=1e+02  Score=22.85  Aligned_cols=22  Identities=27%  Similarity=0.475  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +-|+....-|+|.++++|+|.|
</pre><pre>T Consensus        27 ~D~~~~~gvL~i~~~~~~~~Vi   48 (102)
</pre><pre>T TIGR03421        27 IDCERQGGVLTLTFENGSQIII   48 (102)
</pre><pre>T ss_pred             eEEEeeCCEEEEEeCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>4YGU_D putative adhesin; PF10988 family protein, DUF2807, single-strabded right handed beta-helix fold, Structural Genomics, Joint Center for Structural Genomics; HET: MSE, PEG, PGE, EDO; 2.2A {Bacteroides eggerthii DSM 20697}
</pre><pre>Probab=41.31  E-value=2.5e+02  Score=20.51  Aligned_cols=60  Identities=15%  Similarity=0.123  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             ccCeEEEEECCCceEEEEec-ceEecCEEEEeeEEeCCcee-eeeeEEeccceecCceeecC
</pre><pre>Q WP_214195415.1   21 ESHALTITLAEGGTYRIVGN-GTLDGPVQITGKLTVKGKTD-LLANTHIEGNLTASDEISDA   80 (110)
</pre><pre>Q Consensus        21 ~~haLtitLaeGGt~~I~g~-g~ldG~v~it~~ltv~g~~~-~~~n~~v~gni~a~~eVsD~   80 (110)
</pre><pre>                      ....|++.+..+|.+.+.+. +.+.-.+.=.+.+.+.|... +...+.-.++|.+..=.++.
</pre><pre>T Consensus       123 ~~~~~~~~~~~~g~i~~~~~~~~~~~~~~~~g~i~l~g~~~~~~~~~~g~g~i~~~~l~~~~  184 (184)
</pre><pre>T 4YGU_D          123 TCNVLKVALRGVGSADIHVVCDYLSAQMGGVGSVTLSGSAGRADISKGGIGGVNTDNLKIGR  184 (184)
</pre><pre>T ss_dssp             ECSEEEEEECTTCEEEEEEEEEEEEEEECSSSEEEEEEEEEEEEEEECSSSEEEEEEEEEC-
</pre><pre>T ss_pred             EcCEEEEEEeCceEEEEEEEeCEEEEEEcCceEEEEEeecceEEEEeccceeeecccceecC
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>6CUJ_B Gna2132; Vaccine, Neisseria, Meningitis, Disorder, LIPID BINDING PROTEIN; 1.805A {Neisseria meningitidis}
</pre><pre>Probab=40.55  E-value=2.2e+02  Score=22.12  Aligned_cols=82  Identities=15%  Similarity=0.182  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceE---EEEecceEecCEEEEeeE------EeCCceeeeeeEEeccceecCc--eeecCCCc
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTY---RIVGNGTLDGPVQITGKL------TVKGKTDLLANTHIEGNLTASD--EISDAKGK   83 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~---~I~g~g~ldG~v~it~~l------tv~g~~~~~~n~~v~gni~a~~--eVsD~~gs   83 (110)
</pre><pre>                      ++..|-.+..++.+|..+...   .+.-.+.|+++ .+..++      .+.|.---..--.+.|.+....  +-....|.
</pre><pre>T Consensus        67 ~l~vdF~~~~v~G~i~~~~~~~~~~~~~~g~i~~~-~f~g~~~~~~~g~~~G~FyGp~a~EigG~f~~~~~~~~~~~~G~  145 (161)
</pre><pre>T 6CUJ_B           67 AAKVDFGSKSVDGIIDSGDDLHMGTQKFKAAIDGN-GFKGTWTENGGGDVSGRFYGPAGEEVAGKYSYRPTDAEKGGFGV  145 (161)
</pre><pre>T ss_dssp             EEEEETTTTEEEEEEECSCTTTTCEEEEEEEEETT-EEEEEESSSCSSCEEEEEESTTCCEEEEEEEECC-----CEEEE
</pre><pre>T ss_pred             EEEEECCCCEEEEEEeCCCCccceeEEEEEEEeCC-eEEeEeecCCceEEEEEEECCCCCeeEEEEEEecCCCCceEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhhhccCCCCC
</pre><pre>Q WP_214195415.1   84 MSAIRTTYNGHTHR   97 (110)
</pre><pre>Q Consensus        84 ms~iReiyN~H~H~   97 (110)
</pre><pre>                      ..+.|+..+.|.|.
</pre><pre>T Consensus       146 F~a~~~~~~~~~~~  159 (161)
</pre><pre>T 6CUJ_B          146 FAGKKEQDLEHHHH  159 (161)
</pre><pre>T ss_dssp             EEEEECCC------
</pre><pre>T ss_pred             EEEEeCCCCCcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>2N3D_A Bactofilin A; BacA, Bactofilin, cell shape, cytoskeleton, beta helix, STRUCTURAL PROTEIN; NMR {Caulobacter crescentus}
</pre><pre>Probab=40.53  E-value=1.4e+02  Score=24.06  Aligned_cols=33  Identities=15%  Similarity=0.253  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             eEEEEecceEecCEEEEeeEEeCCceeeeeeEEe
</pre><pre>Q WP_214195415.1   34 TYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHI   67 (110)
</pre><pre>Q Consensus        34 t~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v   67 (110)
</pre><pre>                      .+.|.+.+.+.|++.. ..|.+.+...+.|++..
</pre><pre>T Consensus       104 ~v~l~~~a~V~G~i~~-~~l~i~~ga~i~G~i~~  136 (184)
</pre><pre>T 2N3D_A          104 QVRLYGTSYVDGDITH-EQLAMETGAFFQGRSLK  136 (184)
</pre><pre>T ss_dssp             EEEECTTCEEEEEEEE-SEEEECTTCBCCSCEEE
</pre><pre>T ss_pred             EEEEcCCeEEEcEEEe-CeEEECCCCEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>6FCO_A Mitochondrial frataxin-like protein; iron sulphur cluster, iron chaperone, Friedreich's ataxia, TRANSPORT PROTEIN; HET: MLA; 2.03A {Chaetomium thermophilum var. thermophilum DSM 1495}
</pre><pre>Probab=40.49  E-value=1.1e+02  Score=23.48  Aligned_cols=26  Identities=35%  Similarity=0.452  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEE-CCCceEEEE
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITL-AEGGTYRIV   38 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitL-aeGGt~~I~   38 (110)
</pre><pre>                      +..+.+++....|+|++ +++|+|.|.
</pre><pre>T Consensus        32 ~~~~dvd~~~gvL~i~~g~~~~~~Vin   58 (126)
</pre><pre>T 6FCO_A           32 REDVDVEYQSGVLTLNMGPEVGTYVIN   58 (126)
</pre><pre>T ss_dssp             CTTCEEEEETTEEEEECCTTTCEEEEE
</pre><pre>T ss_pred             CCCeEEEecCCeEEEEEcCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>6FCO_F Mitochondrial frataxin-like protein; iron sulphur cluster, iron chaperone, Friedreich's ataxia, TRANSPORT PROTEIN; HET: MLA; 2.03A {Chaetomium thermophilum var. thermophilum DSM 1495}
</pre><pre>Probab=40.49  E-value=1.1e+02  Score=23.48  Aligned_cols=26  Identities=35%  Similarity=0.452  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEE-CCCceEEEE
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITL-AEGGTYRIV   38 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitL-aeGGt~~I~   38 (110)
</pre><pre>                      +..+.+++....|+|++ +++|+|.|.
</pre><pre>T Consensus        32 ~~~~dvd~~~gvL~i~~g~~~~~~Vin   58 (126)
</pre><pre>T 6FCO_F           32 REDVDVEYQSGVLTLNMGPEVGTYVIN   58 (126)
</pre><pre>T ss_dssp             CTTCEEEEETTEEEEECCTTTCEEEEE
</pre><pre>T ss_pred             CCCeEEEecCCeEEEEEcCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>d1jjua5 b.61.4.1 (A:166-273) Quinohemoprotein amine dehydrogenase A chain, domain 3 {Paracoccus denitrificans [TaxId: 266]}
</pre><pre>Probab=40.40  E-value=1.6e+02  Score=22.98  Aligned_cols=52  Identities=19%  Similarity=0.268  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEC-CCceEEEEecc-eEecCEEEEeeEEeCCceeeeeeEEeccceecCceeecCCCcchhhhhhcc
</pre><pre>Q WP_214195415.1   26 TITLA-EGGTYRIVGNG-TLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEISDAKGKMSAIRTTYN   92 (110)
</pre><pre>Q Consensus        26 titLa-eGGt~~I~g~g-~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsms~iReiyN   92 (110)
</pre><pre>                      +.++. ....|.+...+ -++|.+..+++-+|-+.-+..+++++.               =.++|+||+
</pre><pre>T Consensus        27 ~mtv~~~~d~y~~~~~~~~~dG~~~~~G~a~vYtGYeWRa~~~~~---------------~~~~RqV~a   80 (108)
</pre><pre>T d1jjua5          27 RLVLKKAGEDYEVTMTLDFADGSRSFSGTGRILGAGEWRATLSDG---------------TVTIRQIFA   80 (108)
</pre><pre>T ss_dssp             EEEEEEETTEEEEEEEEEETTEEEEEEEEEEEETTTEEEEEEEET---------------TEEEEEEEE
</pre><pre>T ss_pred             EEEEEECCCcEEEEEEEEeCCCcEEEEEEEEEecceEEEEEEecC---------------CeEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>6JP4_B Alginate lyase; alpha barrel+beta sandwich, LYASE; HET: ACT, EDO; 2.069A {Defluviitalea phaphyphila}
</pre><pre>Probab=39.56  E-value=1e+02  Score=27.39  Aligned_cols=27  Identities=30%  Similarity=0.528  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      ||..+.+.+....++|.|+.+|.+.|.
</pre><pre>T Consensus       741 ~g~~~~~~~~~~~~~~~~~~~g~~~~~  767 (770)
</pre><pre>T 6JP4_B          741 NGENIEFSVEDGYTVIQVAEGGDININ  767 (770)
</pre><pre>T ss_dssp             TTEECCCEEETTEEEEEESSCEEEEEE
</pre><pre>T ss_pred             CCEEEEEEEeCCeEEEEEeccEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>4PEU_A Uncharacterized protein; Beta-helix, Polysaccharide lyase, carbohydrate-binding, calcium-binding, UNKNOWN FUNCTION; 1.8001A {Clostridium thermocellum}
</pre><pre>Probab=38.87  E-value=1.3e+02  Score=25.93  Aligned_cols=31  Identities=23%  Similarity=0.492  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEe------eEEeCC
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITG------KLTVKG   57 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~------~ltv~g   57 (110)
</pre><pre>                      ++++.++|+|+|+|... +|.+.|..      +|+..+
</pre><pre>T Consensus        67 ~~~I~~~G~Y~isG~~~-~~~i~v~~~~~~~v~LtL~~  103 (313)
</pre><pre>T 4PEU_A           67 EVKITKPGMYTFSGTWN-DGQILVDIGKEFEAVLVLDG  103 (313)
</pre><pre>T ss_dssp             EEEECSCEEEEEEEEES-SCEEEEECCTTCEEEEEEEE
</pre><pre>T ss_pred             EEEEcCCeEEEEEEEeC-CCeEEEEcCCCceEEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>1ZDX_A Outer membrane usher protein fimD; BETA SHEET, ALPHA HELIX, Membrane Protein; NMR {Escherichia coli} SCOP: b.167.1.1
</pre><pre>Probab=38.73  E-value=82  Score=21.09  Aligned_cols=19  Identities=11%  Similarity=0.270  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      .|..++||..+..|.|++|
</pre><pre>T Consensus        74 ~~~~~~~d~~~~~L~i~~P   92 (101)
</pre><pre>T 1ZDX_A           74 QDATAHLDVGQQRLNLTIP   92 (101)
</pre><pre>T ss_dssp             SSCEEEEETTTTEEEEECC
</pre><pre>T ss_pred             CCeEEEeeccccEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>4MPZ_A Sas-4; beta-spine, centrosome biogenesis, STRUCTURAL PROTEIN; 2.701A {Drosophila melanogaster}
</pre><pre>Probab=38.55  E-value=3.3e+02  Score=21.72  Aligned_cols=50  Identities=12%  Similarity=0.233  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEE-------EEECCCceEEEEecc-----eEecCEEEEe
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALT-------ITLAEGGTYRIVGNG-----TLDGPVQITG   51 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLt-------itLaeGGt~~I~g~g-----~ldG~v~it~   51 (110)
</pre><pre>                      .+..+.+.+.||.++.+.+....-.       +.+++|-...+..+|     .-+|.+.+.+
</pre><pre>T Consensus       138 ~dGt~~~~~~dG~~~~~~pdG~~~~~~~~g~~~~~pdG~~~~~~~dG~~~~~~~dG~~~~~~  199 (209)
</pre><pre>T 4MPZ_A          138 ADGTHLVQLRNGDKILNLPNGQKEIHTKLNKRREYPDGTVKLVYPDGSQETRYSNGRVRLKD  199 (209)
</pre><pre>T ss_dssp             TTCCEEEEETTSCEEEECTTSCEEEECC----------------------------------
</pre><pre>T ss_pred             CCCCEEEEEeCCCEEEEcCCCCEEEEcCCCcEEEcCCCCEEEECCCCCEEEEeCCCCEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>4H09_B Hypothetical leucine rich repeat protein; TWO LRR_5 DOMAINS, PF13306 FAMILY, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE; HET: MSE, PEG, EDO; 2.5A {Eubacterium ventriosum}
</pre><pre>Probab=37.87  E-value=1.6e+02  Score=22.45  Aligned_cols=36  Identities=28%  Similarity=0.525  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecceEecCEEE
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQI   49 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~i   49 (110)
</pre><pre>                      |.++.|+- +..++..+.++|.+.|.+.+.+...+.+
</pre><pre>T Consensus         1 ~~~~~~~~-~~~~~~~~~~~~~~~~~~~~~~~~~~~~   36 (379)
</pre><pre>T 4H09_B            1 GASITYNV-SSTIKGVLTDDGVLTISGTGAMPDYTKI   36 (379)
</pre><pre>T ss_dssp             CCEEEEEC-SSSEEEEEETTSEEEEEESEECCCCSSG
</pre><pre>T ss_pred             CCcceeec-cceeeEEECCCCeEEEeeceeCCCceee
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>4H09_E Hypothetical leucine rich repeat protein; TWO LRR_5 DOMAINS, PF13306 FAMILY, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE; HET: MSE, PEG, EDO; 2.5A {Eubacterium ventriosum}
</pre><pre>Probab=37.87  E-value=1.6e+02  Score=22.45  Aligned_cols=36  Identities=28%  Similarity=0.525  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecceEecCEEE
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQI   49 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~i   49 (110)
</pre><pre>                      |.++.|+- +..++..+.++|.+.|.+.+.+...+.+
</pre><pre>T Consensus         1 ~~~~~~~~-~~~~~~~~~~~~~~~~~~~~~~~~~~~~   36 (379)
</pre><pre>T 4H09_E            1 GASITYNV-SSTIKGVLTDDGVLTISGTGAMPDYTKI   36 (379)
</pre><pre>T ss_dssp             -CCEEEEC-SSSEEEEECTTSEEEEEESEECCCCSSG
</pre><pre>T ss_pred             CCcceeec-cceeeEEECCCCeEEEeeceeCCCceee
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>PF16378.7 ; DUF4988 ; Domain of unknown function
</pre><pre>Probab=37.82  E-value=1.3e+02  Score=23.94  Aligned_cols=24  Identities=38%  Similarity=0.563  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CEEEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   14 TEVRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        14 Tev~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      ..|.||.+...++|+|++|..++|
</pre><pre>T Consensus       162 ~~v~~~~~~~~~~~~l~~g~~i~l  185 (186)
</pre><pre>T R6EJA9_9BACE/2  162 DRVVHNKETGNMNITLVTGETFSI  185 (186)
</pre><pre>T ss_pred             EEEEEeCCCCEEEEEECCCCEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>PF01723.18 ; Chorion_1 ; Chorion protein
</pre><pre>Probab=36.11  E-value=70  Score=26.80  Aligned_cols=38  Identities=18%  Similarity=0.292  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEecceEecCEEEEeeEEeCCceeeeeeEEe--cccee
</pre><pre>Q WP_214195415.1   35 YRIVGNGTLDGPVQITGKLTVKGKTDLLANTHI--EGNLT   72 (110)
</pre><pre>Q Consensus        35 ~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v--~gni~   72 (110)
</pre><pre>                      +.|..+..++|+|.|.++|-+-|.+.+++.+-.  .+.|+
</pre><pre>T Consensus        90 ~sv~sen~~eG~vaV~G~LPflGtv~veG~lPt~G~GaV~  129 (172)
</pre><pre>T CHCB1_BOMMO/1-   90 LTVASENTIEGNLGIFGQLPFLGAVATDGAFSTGGIGAVL  129 (172)
</pre><pre>T ss_pred             eeeecCCeeeeeEEEecCCCcceeEEeccccccCceEEeE
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>2X8N_A CV0863; NON-UNIFORM SAMPLING, MULTIDIMENSIONAL DECOMPOSITION, ABACUS, FRAGMENT MONTE CARLO, STRUCTURAL GENOMICS, NORTHEAST STRUCTURAL GENOMICS CONSORTIUM, NESG, ONTARIO; NMR {CHROMOBACTERIUM VIOLACEUM ATCC 12472}
</pre><pre>Probab=35.79  E-value=1.6e+02  Score=21.96  Aligned_cols=24  Identities=21%  Similarity=0.397  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             CEEEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   14 TEVRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        14 Tev~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      ..++|+...+.|.|.|.+|-.+.|
</pre><pre>T Consensus        28 ~~~~~~~~~~~l~i~l~dGr~~~~   51 (111)
</pre><pre>T 2X8N_A           28 RSVQYRSEEGVIVFILANDRELKF   51 (111)
</pre><pre>T ss_dssp             EEEEEETTTTEEEEEETTCCEEEE
</pre><pre>T ss_pred             eEEEEEEcCCEEEEEECCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>3OEQ_A Frataxin homolog, mitochondrial; ALPHA/BETA SANDWICH, METALLOCHAPERONE, IRON-STORAGE, TRANSPORT PROTEIN; 2.96A {Saccharomyces cerevisiae} SCOP: d.82.2.1
</pre><pre>Probab=35.62  E-value=1.4e+02  Score=22.37  Aligned_cols=29  Identities=17%  Similarity=0.131  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             eCCCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   10 YEDGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        10 Y~DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      +.+...+-+++....|+|.++++++|.|.
</pre><pre>T Consensus        43 ~~~~~~~dv~~~~~vL~i~~~~~~~~vin   71 (123)
</pre><pre>T 3OEQ_A           43 AHPDCIPDVELSHGVMTLEIPAFGTYVIN   71 (123)
</pre><pre>T ss_dssp             SCSSCCCEEEECSSEEEEECTTTCCEEEE
</pre><pre>T ss_pred             hCCCCCceeEecCCeEEEEeCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>4PEU_A Uncharacterized protein; Beta-helix, Polysaccharide lyase, carbohydrate-binding, calcium-binding, UNKNOWN FUNCTION; 1.8001A {Clostridium thermocellum}
</pre><pre>Probab=34.59  E-value=5.5e+02  Score=22.46  Aligned_cols=77  Identities=16%  Similarity=0.301  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEee-------cccCeEEEEECCCceEEEEecceEe----------cCEEEEeeEEeCCceeeeee
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYN-------RESHALTITLAEGGTYRIVGNGTLD----------GPVQITGKLTVKGKTDLLAN   64 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Yd-------r~~haLtitLaeGGt~~I~g~g~ld----------G~v~it~~ltv~g~~~~~~n   64 (110)
</pre><pre>                      ...+.++-+-||..|.-.       .....++|+|++|..-.++......          +-+...+.|++.|.-.+...
</pre><pre>T Consensus        92 ~~~~~v~LtL~~v~i~~~~~~~i~i~~~~~~~i~l~~gt~N~l~~~~~~~~~~~~~~~~~~~i~~~~~ltI~G~GtL~i~  171 (313)
</pre><pre>T 4PEU_A           92 GKEFEAVLVLDGVNITNTKSAPIYIKSAEKVKIELADGKDNVLTDAEFYEFEDPQDNKPNACIYSRDDITIKGNGNLTVN  171 (313)
</pre><pre>T ss_dssp             CTTCEEEEEEEEEEEECSSSCSEEEEECSEEEEEECTTCEEEEECCSSCCCSSTTCCCCCEEEEESSEEEEEESSEEEEE
</pre><pre>T ss_pred             CCCceEEEEEcCeEEEeCCCCCEEEEcCCeEEEEECCCCeeEeeccccccCCCCCCCCCCceEEeCCcEEEecCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEeccceecCceee
</pre><pre>Q WP_214195415.1   65 THIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        65 ~~v~gni~a~~eVs   78 (110)
</pre><pre>                      ..-..-|.+.+.|.
</pre><pre>T Consensus       172 ~~~~~gI~~~~~l~  185 (313)
</pre><pre>T 4PEU_A          172 ANFNNGIGTSNDLK  185 (313)
</pre><pre>T ss_dssp             ECSSEEEEESSEEE
</pre><pre>T ss_pred             cCCCCccCCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>PF01491.18 ; Frataxin_Cyay ; Frataxin-like domain
</pre><pre>Probab=34.54  E-value=1.5e+02  Score=21.33  Aligned_cols=22  Identities=36%  Similarity=0.498  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +.+|.....|+|.++++|+|.|
</pre><pre>T Consensus        34 ~dvd~~~~vL~i~~~~~~~~vi   55 (110)
</pre><pre>T G7DVC6_MIXOS/5   34 CEVEYSSGVLTLSLGSKGTYVI   55 (110)
</pre><pre>T ss_pred             eEEEeeCCEEEEEECCceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>TIGR02892 spore_yabP; sporulation protein YabP. Members of this protein family are the YabP protein of the bacterial sporulation program, as found in Bacillus subtilis, Clostridium tetani, and other spore-forming members of the Firmicutes.
</pre><pre>Probab=34.36  E-value=3.3e+02  Score=19.79  Aligned_cols=33  Identities=24%  Similarity=0.425  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             ccCeEEEEECCCceEEEEecc----eE---ecCEEEEeeEE
</pre><pre>Q WP_214195415.1   21 ESHALTITLAEGGTYRIVGNG----TL---DGPVQITGKLT   54 (110)
</pre><pre>Q Consensus        21 ~~haLtitLaeGGt~~I~g~g----~l---dG~v~it~~lt   54 (110)
</pre><pre>                      +...+.+.++ .|.++|+|++    .+   +|.+.|++.+.
</pre><pre>T Consensus        23 ~~~~I~i~~~-~g~l~I~G~~L~I~~~~~~~~~i~I~G~I~   62 (85)
</pre><pre>T TIGR02892        23 DDEEILLETV-MGFLTIKGQELKMNKLDVENGQVIIKGFIS   62 (85)
</pre><pre>T ss_pred             CCCEEEEEeC-CcEEEEEcCCeEEEEEECCCCEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>PF18952.2 ; DUF5696 ; Family of unknown function (DUF5696)
</pre><pre>Probab=34.18  E-value=2e+02  Score=26.06  Aligned_cols=33  Identities=15%  Similarity=0.109  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCccceeEeeCCCCEEEeecccCeEEE---EECCCc
</pre><pre>Q WP_214195415.1    1 GRECEHITVYEDGTEVRYNRESHALTI---TLAEGG   33 (110)
</pre><pre>Q Consensus         1 gs~~eHvtrY~DGTev~Ydr~~haLti---tLaeGG   33 (110)
</pre><pre>                      ..++...++|+||++|--|......++   +|++.|
</pre><pre>T Consensus       579 ~d~~v~~t~y~nG~~v~vN~~~~~~~~~g~~v~~~~  614 (615)
</pre><pre>T A9NEW9_ACHLI/2  579 NFNNVYKVGYSNNLEMIINYNLFSITVEGVTIPAMD  614 (615)
</pre><pre>T ss_pred             CCCCEEEEEEcCCeEEEEECCCceEEECCEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>PF16241.7 ; DUF4900 ; Domain of unknown function (DUF4900)
</pre><pre>Probab=33.95  E-value=1.7e+02  Score=20.57  Aligned_cols=40  Identities=23%  Similarity=0.292  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEecc
</pre><pre>Q WP_214195415.1   30 AEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEG   69 (110)
</pre><pre>Q Consensus        30 aeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~g   69 (110)
</pre><pre>                      +.|+.+...+.-+++|||---+.|.+.|...+.+.++..|
</pre><pre>T Consensus        48 ~~g~~v~f~~~~~i~GpvHsN~~i~~~g~p~F~g~vts~g   87 (88)
</pre><pre>T D7BJP2_MEISD/1   48 ARGNLHLFDGGVVLEGRLHVAGLLGVRDNPWLAGAVTSGS   87 (88)
</pre><pre>T ss_pred             CCCCeeeeCCCcEEEceEEcCCcEEEcCCCeEccceEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>PF01696.19 ; Adeno_E1B_55K ; Adenovirus EB1 55K protein / large t-antigen
</pre><pre>Probab=33.30  E-value=1.2e+02  Score=25.12  Aligned_cols=27  Identities=15%  Similarity=0.317  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEee
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITGK   52 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~~   52 (110)
</pre><pre>                      +|.|++|++|.|..+=.+..+++|.+.
</pre><pre>T Consensus        61 ~i~l~~G~~Y~i~~~l~i~~~vti~G~   87 (383)
</pre><pre>T Q83459_ADEP4/2   61 KVALRPGAEYVLSSGFCISTCCYVIGR   87 (383)
</pre><pre>T ss_pred             cEEECCCCEEEeCCCEEeCCCEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>d4ec2a_ d.82.2.1 (A:) C-terminal domain of frataxin {Baker's yeast (Saccharomyces cerevisiae) [TaxId: 4932]}
</pre><pre>Probab=33.21  E-value=1.5e+02  Score=22.56  Aligned_cols=23  Identities=22%  Similarity=0.245  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      +-+++...-|+|.++++++|.|.
</pre><pre>T Consensus        40 ~d~e~~~gvL~i~~~~~~~~vin   62 (112)
</pre><pre>T d4ec2a_          40 PDVELSHGVMTLEIPAFGTYVIN   62 (112)
</pre><pre>T ss_dssp             CCEEECSSEEEEEETTTEEEEEE
</pre><pre>T ss_pred             ceeEEeCCEEEEEeCCCeEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>TIGR01222 minC; septum site-determining protein MinC. The minC protein assists in correct placement of the septum for cell division by inhibiting septum formation at other sites.
</pre><pre>Probab=32.84  E-value=92  Score=25.67  Aligned_cols=34  Identities=24%  Similarity=0.405  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             c-CEEEE-eeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   45 G-PVQIT-GKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        45 G-~v~it-~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      | .+++. +.|.|-|++.-.+.+.+.|||.+-|.+.
</pre><pre>T Consensus       122 GQ~i~~~~g~i~i~g~v~~ga~v~a~g~i~v~g~~~  157 (217)
</pre><pre>T TIGR01222       122 GQQIYAKHGDLIVLGNVNAGAEVLADGNIHVYGKLR  157 (217)
</pre><pre>T ss_pred             CCeEEECCCCEEEECCCCCCCEEEeCCcEEEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>5NZG_A UDP-glucose pyrophosphorylase; NTP-transferase, Pathogen, Allostery, Catalysis, transferase; HET: UPG, EDO; 1.6A {Leishmania major}
</pre><pre>Probab=32.38  E-value=2.3e+02  Score=25.72  Aligned_cols=69  Identities=19%  Similarity=0.241  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             EECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceeecCCCcchhhhhhccCCCCC
</pre><pre>Q WP_214195415.1   28 TLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEISDAKGKMSAIRTTYNGHTHR   97 (110)
</pre><pre>Q Consensus        28 tLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsms~iReiyN~H~H~   97 (110)
</pre><pre>                      .|.+.-.++|.|+=.+.-.|+|.+...+.. ..+-.++.+..+......|-...-++.++|-.-.-|.|.
</pre><pre>T Consensus       437 ~~~~~~~~~I~g~v~iG~~v~I~~~~~I~~-~~ig~~~~I~~~~~i~~~iv~~~~~~~~~~~~~~~~~~~  505 (505)
</pre><pre>T 5NZG_A          437 SLVECKRVTVKGLVQFGAGNVLTGTVTIEN-TDSASAFVIPDGAKLNDTTASPQQSTNKMRPLEHHHHHH  505 (505)
</pre><pre>T ss_dssp             BCTTEEEEEEESSEECCTTCEEEEEEEEEC-SCTTSCEECCTTCEEEEEEECC-----------------
</pre><pre>T ss_pred             CcccCcEEEEeCCEEECCCCEEEccEEEeC-CCCCCCEEeCCCCEEcCceeCCCcCCCCccCCcccCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>d1ze3d1 b.167.1.1 (D:1-125) Outer membrane usher protein FimD {Escherichia coli [TaxId: 562]}
</pre><pre>Probab=31.50  E-value=1.3e+02  Score=21.50  Aligned_cols=19  Identities=11%  Similarity=0.270  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      .|..++||..+..|.|++|
</pre><pre>T Consensus        98 ~~~~~~~d~~~q~L~i~vP  116 (125)
</pre><pre>T d1ze3d1          98 QDATAHLDVGQQRLNLTIP  116 (125)
</pre><pre>T ss_dssp             TTCEEEEETTTTEEEEECC
</pre><pre>T ss_pred             CCEEEEEecccCEEEEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>d3t3la_ d.82.2.1 (A:) automated matches {Human (Homo sapiens) [TaxId: 9606]}
</pre><pre>Probab=31.42  E-value=2.1e+02  Score=21.99  Aligned_cols=26  Identities=31%  Similarity=0.310  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEEC-CCceEEE
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLA-EGGTYRI   37 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLa-eGGt~~I   37 (110)
</pre><pre>                      +.-.+-++....-|+|.++ ++|+|.|
</pre><pre>T Consensus        30 ~~~d~d~e~~~gvL~i~~~~~~~~~vI   56 (121)
</pre><pre>T d3t3la_          30 TFEDYDVSFGSGVLTVKLGGDLGTYVI   56 (121)
</pre><pre>T ss_dssp             SCTTCEEEEETTEEEEECCTTCCEEEE
</pre><pre>T ss_pred             CCcCeEEecCCCEEEEEECCCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>d4b0ma_ b.167.1.0 (A:) automated matches {Yersinia pestis [TaxId: 632]}
</pre><pre>Probab=31.37  E-value=1.4e+02  Score=21.79  Aligned_cols=19  Identities=11%  Similarity=0.338  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      .|..++||..+..|.|++|
</pre><pre>T Consensus        98 ~~~~~~~d~~~q~L~i~vP  116 (135)
</pre><pre>T d4b0ma_          98 PHSDVQFYFNQQKLSLIVP  116 (135)
</pre><pre>T ss_dssp             SSCEEEEETTTTEEEEECC
</pre><pre>T ss_pred             CCeEEEEECccCEEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>PF15490.8 ; Ten1_2 ; Telomere-capping, CST complex subunit
</pre><pre>Probab=30.86  E-value=3.7e+02  Score=19.30  Aligned_cols=53  Identities=11%  Similarity=0.186  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEE---ECCCceEEEEec------ceEecCEEEEeeEEe--CCceeeeeeE
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTIT---LAEGGTYRIVGN------GTLDGPVQITGKLTV--KGKTDLLANT   65 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtit---LaeGGt~~I~g~------g~ldG~v~it~~ltv--~g~~~~~~n~   65 (110)
</pre><pre>                      |....||..+..++++   -.+++.+.+.-.      -.+..-|+|-+.+.-  .+...+.+.+
</pre><pre>T Consensus        28 G~v~~~d~~~~~~~l~~~~~~~~~~v~V~~~~~~~~~~~~~~~v~viG~v~~~~~~~~~l~a~~   91 (117)
</pre><pre>T H3BD46_LATCH/3   28 GRLISYDAEQSEVILATQHDSSQYQVIVSTTFVEPFQPHFGTPYITLGEIQKKEGDQVQIKARI   91 (117)
</pre><pre>T ss_pred             EEEEEEECCCCEEEEEecCCCCCeEEEEEeeecCCCCCCCCCcEEEEEEEEEecCCeeEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>5HMB_A Azi13; azinomycin biosynthesis, polyketide synthase, thioesterase, naphthoate, HYDROLASE; HET: SO4; 2.151A {Streptomyces sahachiroi} SCOP: d.38.1.0, l.1.1.1
</pre><pre>Probab=30.24  E-value=2.4e+02  Score=19.92  Aligned_cols=51  Identities=18%  Similarity=0.251  Sum_probs=0.0  Template_Neff=10.300
</pre><pre>
</pre><pre>Q ss_pred             ecCEEEEeeEEeCCceeeeeeEEeccceecCceeecCCCcchhhhhhccCCCC
</pre><pre>Q WP_214195415.1   44 DGPVQITGKLTVKGKTDLLANTHIEGNLTASDEISDAKGKMSAIRTTYNGHTH   96 (110)
</pre><pre>Q Consensus        44 dG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsms~iReiyN~H~H   96 (110)
</pre><pre>                      ++++.++..+.-.++..+.-.+.+..  ..+..++.+..+++..|+-|.+|.|
</pre><pre>T Consensus        91 ~~~~~~~~~~~~~~~~~~~~~~~i~~--~~~~~~a~~~~~~~~~~~~~~~~~~  141 (141)
</pre><pre>T 5HMB_A           91 DGRLTSTALPVHRGATQQVWSVETVD--AAGRLVARGQVRLHNLRLEHHHHHH  141 (141)
</pre><pre>T ss_dssp             GCEEEEEEEEEEECSSEEEEEEEEEC--TTCCEEEEEEEEEEEEEC-------
</pre><pre>T ss_pred             CCeEEEEEEEEEeCCCEEEEEEEEEC--CCCCEEEEEEEEEEecCCCCcCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>PF06832.14 ; BiPBP_C ; Penicillin-Binding Protein C-terminus Family
</pre><pre>Probab=30.23  E-value=1.6e+02  Score=19.26  Aligned_cols=29  Identities=10%  Similarity=0.051  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             eeEeeCCCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1    6 HITVYEDGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus         6 HvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      .+.-|-||..+    ....+.+.+.+-|.++|.
</pre><pre>T Consensus        44 ~v~w~lng~~~----~~~~~~~~~~~~G~h~l~   72 (87)
</pre><pre>T Q316S6_DESAG/6   44 PLRWVVNGLPL----PEDQHYWLPDSEGFARLT   72 (87)
</pre><pre>T ss_pred             CEEEEECCEEC----CCCCcEEcCCCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>d2wmfa2 b.24.2.1 (A:426-589) Sp4GH98 {Streptococcus pneumoniae TIGR4 [TaxId: 170187]}
</pre><pre>Probab=30.16  E-value=1.9e+02  Score=24.19  Aligned_cols=24  Identities=21%  Similarity=0.440  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      ...||.+.+.+||++.-.|.+.|.
</pre><pre>T Consensus       140 s~~w~~~~g~~titI~hNG~vd~~  163 (164)
</pre><pre>T d2wmfa2         140 TENWDENTHVYTITVNHNGMVEMS  163 (164)
</pre><pre>T ss_dssp             EEEEETTTTEEEEEEEESSEEEEE
</pre><pre>T ss_pred             eEEEeCCCCEEEEEEeeCceEeee
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>4UHV_A VGRG1, VALINE-GLYCINE REPEAT PROTEIN G1; STRUCTURAL PROTEIN, VGRG1, VIRULENCE, TOXIN, EFFECTOR, PUNCTURING DEVICE, SPIKE, T6SS, P. AERUGINOSA; HET: CL, NA, MSE; 2.0A {PSEUDOMONAS AERUGINOSA}
</pre><pre>Probab=29.84  E-value=6.3e+02  Score=22.94  Aligned_cols=80  Identities=15%  Similarity=0.089  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             eeEeeCC--CCEEEeecccCeEEEEECCCceEEEEecc--eEecCEEEE--------------------eeEEeCCceee
</pre><pre>Q WP_214195415.1    6 HITVYED--GTEVRYNRESHALTITLAEGGTYRIVGNG--TLDGPVQIT--------------------GKLTVKGKTDL   61 (110)
</pre><pre>Q Consensus         6 HvtrY~D--GTev~Ydr~~haLtitLaeGGt~~I~g~g--~ldG~v~it--------------------~~ltv~g~~~~   61 (110)
</pre><pre>                      |..+|.|  |.+..|....+.+.+.....-.+.+.+..  .+.++..+.                    ..+++.|+..+
</pre><pre>T Consensus       483 ~~~~~~d~~~~~~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  562 (651)
</pre><pre>T 4UHV_A          483 NEIRMEDKKGAEQLYIHAERNQDNLVENDASLSVGHDRNKSIGHDELARIGNNRTRAVKLNDTLLVGGAKSDSVTGTYLI  562 (651)
</pre><pre>T ss_dssp             CEEEEECCTTCCEEEEECSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCCCCSSCCCC
</pre><pre>T ss_pred             cEEEeeCCCCceeEEEeeccCcccccccCcEEEecCCCccccCCceEEEecCCeEEEEecCeEEEECCceEEEecCcEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             eeeEEeccceecCceeecCCCcch
</pre><pre>Q WP_214195415.1   62 LANTHIEGNLTASDEISDAKGKMS   85 (110)
</pre><pre>Q Consensus        62 ~~n~~v~gni~a~~eVsD~~gsms   85 (110)
</pre><pre>                      .+.-.+.-......-...+.+++.
</pre><pre>T Consensus       563 ~~~~~~~~~~~~~~~~~~~~~~~~  586 (651)
</pre><pre>T 4UHV_A          563 EAGAQIRLVCGKSVVEFNADGTIN  586 (651)
</pre><pre>T ss_dssp             CCSSEEEEEETTEEEEEESSCCEE
</pre><pre>T ss_pred             EeCCeEEEEeCCcEEEECCCCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>d2fgza4 b.71.1.1 (A:966-1083) Pullulanase PulA {Klebsiella pneumoniae [TaxId: 573]}
</pre><pre>Probab=29.40  E-value=3.5e+02  Score=20.57  Aligned_cols=46  Identities=15%  Similarity=0.140  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEEEecceEecCE--------EEEeeEEeCCceee
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRIVGNGTLDGPV--------QITGKLTVKGKTDL   61 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v--------~it~~ltv~g~~~~   61 (110)
</pre><pre>                      |-||.....-+|.|+.|+.|.+--...-..+.        ...++++|-.-+.+
</pre><pre>T Consensus        46 V~~Na~~~~~ti~lp~g~~w~l~~~~~~~~~~~~~~~~~~~~~g~~tVP~~S~~   99 (118)
</pre><pre>T d2fgza4          46 VAINAAPESRTLQDFAGTSLQLSAIQQAAGDRSLASGVQVAADGSVTLPAWSVA   99 (118)
</pre><pre>T ss_dssp             EEEECSSSCEEECTTTTSCCEECHHHHHTGGGSTTTTCEECTTSCEEECTTEEE
</pre><pre>T ss_pred             EEEeCCCCcEEEcccCCCceEEecceeccCCccceeceeEecCCEEEECCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>TIGR01643 YD_repeat_2x; YD repeat (two copies). This model describes two tandem copies of a 21-residue extracellular repeat found in Gram-negative, Gram-positive, and animal proteins.
</pre><pre>Probab=29.20  E-value=1.7e+02  Score=15.05  Aligned_cols=38  Identities=26%  Similarity=0.324  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             CCccceeEeeCCCCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1    1 GRECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus         1 gs~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      |...........-+...||....-+.++++.|-...+.
</pre><pre>T Consensus         5 g~~~~~~~~~~~~~~~~yd~~g~~~~~~~~~~~~~~~~   42 (42)
</pre><pre>T TIGR01643         5 GRLTGSTDADGTTTRYTYDAAGRLVEITDADGGSTRYE   42 (42)
</pre><pre>T ss_pred             CCeeeEECCCCCEEEEEEcCCCCEEEEEeCCCCEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>PF13954.8 ; PapC_N ; PapC N-terminal domain
</pre><pre>Probab=28.87  E-value=1.7e+02  Score=21.30  Aligned_cols=19  Identities=0%  Similarity=0.142  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      .|..++||..+..|.|++|
</pre><pre>T Consensus        96 ~~~~~~~d~~~q~L~l~iP  114 (146)
</pre><pre>T H2IVA4_RAHAC/4   96 PTARWNLDVNTQTFRLDVP  114 (146)
</pre><pre>T ss_pred             CCeEEEEECCCCEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>PF03775.18 ; MinC_C ; Septum formation inhibitor MinC, C-terminal domain
</pre><pre>Probab=28.02  E-value=1.4e+02  Score=22.02  Aligned_cols=31  Identities=13%  Similarity=0.295  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             EEEEeeEEeCCceeeeeeEEeccceecCcee
</pre><pre>Q WP_214195415.1   47 VQITGKLTVKGKTDLLANTHIEGNLTASDEI   77 (110)
</pre><pre>Q Consensus        47 v~it~~ltv~g~~~~~~n~~v~gni~a~~eV   77 (110)
</pre><pre>                      ++..+.+.+.|.+.-.+.+...|||.+-+.+
</pre><pre>T Consensus        12 v~~~gdi~i~G~V~~ga~V~a~Gnv~V~g~v   42 (101)
</pre><pre>T D9S283_THEOJ/9   12 IAYSGNVVVVGDVNPGSEIIAKGDILVFGAL   42 (101)
</pre><pre>T ss_pred             EEECCCEEEECCcCCCCEEEECCCEEEecee
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>1PG6_A Hypothetical protein SpyM3_0169; STRUCTURAL GENOMICS  Streptococcus pyogenes  SpyM3_0169, PSI, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG; HET: MSE; 1.7A {Streptococcus pyogenes} SCOP: b.82.5.2
</pre><pre>Probab=27.77  E-value=5.3e+02  Score=21.36  Aligned_cols=39  Identities=21%  Similarity=0.258  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEEecc--eEecCEEEEeeE
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIVGNG--TLDGPVQITGKL   53 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~g~g--~ldG~v~it~~l   53 (110)
</pre><pre>                      .|..+....-|.|+|++|..+.+...-  -.+|+|++...+
</pre><pre>T Consensus        16 ~i~~~~~~~~l~v~L~~ge~v~~~~gs~~~~~~~v~~~~~~   56 (243)
</pre><pre>T 1PG6_A           16 TIDQNMQFPLVEIDLEHGGSVYLQQGSMVYHTENVTLNTKL   56 (243)
</pre><pre>T ss_dssp             EECSSCSSCCEEEEECTTCEEEEETTCEEEECTTEEEECTT
</pre><pre>T ss_pred             EeccCCCCCEEEEEeCCCCEEEEeCCcEEEEeCCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>d1pg6a_ b.82.5.2 (A:) Hypothetical protein SPyM3_0169 (SPyM18_0222, SPS0176) {Streptococcus pyogenes [TaxId: 1314]}
</pre><pre>Probab=27.75  E-value=3.6e+02  Score=21.82  Aligned_cols=41  Identities=17%  Similarity=0.316  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCCEEEeecccCeEE-EEECCCceEEEEecceE
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGTEVRYNRESHALT-ITLAEGGTYRIVGNGTL   43 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGTev~Ydr~~haLt-itLaeGGt~~I~g~g~l   43 (110)
</pre><pre>                      +..|+..|.+.-.+++.+...-.+ +.-.+|-.++++|+|++
</pre><pre>T Consensus       168 d~~~lva~~~~~~~~~~~~~~~~~~~~~ge~~~~~~~GpG~V  209 (227)
</pre><pre>T d1pg6a_         168 DNAHVVAWSRELDYDIHLENGFMQSIGTGEGVVNTFRGHGEI  209 (227)
</pre><pre>T ss_dssp             EGGGEEEEETTSEEEEECSSCHHHHHHHTCCCEEEEEEEEEE
</pre><pre>T ss_pred             eCCcEEEEEccceEEEEEeCceeeeeecCCeeEEEEEeCeEE
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>PF07481.13 ; DUF1521 ; Domain of Unknown Function (DUF1521)
</pre><pre>Probab=27.48  E-value=4.2e+02  Score=22.26  Aligned_cols=42  Identities=24%  Similarity=0.309  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             EeeCCCCEEEeecc----------cCeEEEEECCCceEEEEecc-eEecCEEEE
</pre><pre>Q WP_214195415.1    8 TVYEDGTEVRYNRE----------SHALTITLAEGGTYRIVGNG-TLDGPVQIT   50 (110)
</pre><pre>Q Consensus         8 trY~DGTev~Ydr~----------~haLtitLaeGGt~~I~g~g-~ldG~v~it   50 (110)
</pre><pre>                      +...|||.|..++.          ...|+|+- ..-.+.|+|-. .-.|+++|+
</pre><pre>T Consensus        65 F~L~DGTkITv~t~~~~~~~~~t~~~~vtIt~-G~~~~~v~gl~~~~~g~l~v~  117 (169)
</pre><pre>T W0AEV6_9SPHN/8   65 FTLENGTKVTIDTEQWGGNPDAYVASKVTVTN-GDRAIVVDGISQNEIGDLEIS  117 (169)
</pre><pre>T ss_pred             EECCCCCEEEEECccCCCCCCceeeeeEEEEe-CCeEEEEecccCCCccccEEE
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>PF07481.13 ; DUF1521 ; Domain of Unknown Function (DUF1521)
</pre><pre>Probab=27.43  E-value=6.6e+02  Score=21.18  Aligned_cols=55  Identities=22%  Similarity=0.231  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECC-CceEEEEecceEe----cCEEEEeeEEeC
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAE-GGTYRIVGNGTLD----GPVQITGKLTVK   56 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLae-GGt~~I~g~g~ld----G~v~it~~ltv~   56 (110)
</pre><pre>                      +...-.....||-.|.++.....++|+=++ |-.++|-|..+++    ....|.+.+++.
</pre><pre>T Consensus         7 ~~~~~~i~lgdgY~l~~~~~~s~~~I~n~~tG~~t~IwGDPHvd~~g~~~~dF~~~~TF~   66 (169)
</pre><pre>T W0AEV6_9SPHN/8    7 SEHTGNIDLGDGYTMRLDERSSEIMIHNEKTNEWTRIWGDPHVDVNGERAFDFWGTTTFT   66 (169)
</pre><pre>T ss_pred             cCCceEEEcCCCeEEEEcCCCCEEEEEeCCCCceEEEEeCcceecCCCcceeeeCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>PF14734.8 ; DUF4469 ; Domain of unknown function (DUF4469) with IG-like fold
</pre><pre>Probab=27.13  E-value=1.4e+02  Score=21.78  Aligned_cols=29  Identities=10%  Similarity=0.240  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             eecccCeEEEEECCCceEEEEecc-eEecC
</pre><pre>Q WP_214195415.1   18 YNRESHALTITLAEGGTYRIVGNG-TLDGP   46 (110)
</pre><pre>Q Consensus        18 Ydr~~haLtitLaeGGt~~I~g~g-~ldG~   46 (110)
</pre><pre>                      ||..+....=+|..|+.++|.|.. .++|.
</pre><pre>T Consensus         5 ~D~~t~~~~~~it~G~~~~I~G~~Lk~~~~   34 (99)
</pre><pre>T A6LGM9_PARD8/1    5 TDSTTKEKNLHITPNAPAIIFGSTLLIKGD   34 (99)
</pre><pre>T ss_pred             EeCCCCccCCccCCCCCEEEEEeeeEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>4B0M_A F1 CAPSULE-ANCHORING PROTEIN; PROTEIN TRANSPORT, CHAPERONE-USHER PATHWAY, PILI ASSEMBLY; 1.8A {YERSINIA PESTIS} SCOP: b.167.1.0
</pre><pre>Probab=27.11  E-value=1.8e+02  Score=21.06  Aligned_cols=18  Identities=11%  Similarity=0.370  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      +..+.||....+|.|++|
</pre><pre>T Consensus        99 ~~~~~~d~~~~~L~i~iP  116 (136)
</pre><pre>T 4B0M_A           99 HSDVQFYFNQQKLSLIVP  116 (136)
</pre><pre>T ss_dssp             SCEEEEETTTTEEEEECC
</pre><pre>T ss_pred             CeEEEEECccCEEEEEcC
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>PF07873.13 ; YabP ; YabP family
</pre><pre>Probab=26.98  E-value=3.7e+02  Score=18.05  Aligned_cols=33  Identities=24%  Similarity=0.472  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             ccCeEEEEECCCceEEEEecc----eEe---cCEEEEeeEE
</pre><pre>Q WP_214195415.1   21 ESHALTITLAEGGTYRIVGNG----TLD---GPVQITGKLT   54 (110)
</pre><pre>Q Consensus        21 ~~haLtitLaeGGt~~I~g~g----~ld---G~v~it~~lt   54 (110)
</pre><pre>                      ....+.+.+ .+|.++|.|++    .++   +.+.|++.+.
</pre><pre>T Consensus        23 ~~~~I~l~~-~~~~l~I~G~~L~i~~~~~~~~~i~I~G~I~   62 (66)
</pre><pre>T R6PCP1_9FIRM/8   23 DEENIVAQT-TMGVLIIRGDKLHINSLNLEKGTLNIDGNVT   62 (66)
</pre><pre>T ss_pred             cCCEEEEEE-CCcEEEEEcCCEEEEEEECCCCEEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>5NFI_B Minor fimbrium anchoring subunit Mfa2; FIMBRIA, ADHESIN, PERIODONTITIS, CELL ADHESION; HET: MSE; 2.508A {Porphyromonas gingivalis (strain ATCC 33277 / DSM 20709 / CIP 103683 / JCM 12257 / NCTC 11834 / 2561)}
</pre><pre>Probab=26.91  E-value=5.3e+02  Score=21.02  Aligned_cols=44  Identities=25%  Similarity=0.375  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             eeEee-CCCCEEEeecccC-------eEEEEECCCceEEEEecceEecCEEE
</pre><pre>Q WP_214195415.1    6 HITVY-EDGTEVRYNRESH-------ALTITLAEGGTYRIVGNGTLDGPVQI   49 (110)
</pre><pre>Q Consensus         6 HvtrY-~DGTev~Ydr~~h-------aLtitLaeGGt~~I~g~g~ldG~v~i   49 (110)
</pre><pre>                      +++.| ++|..+..-....       .+.+.|.+-|+|+|.+=|-++....+
</pre><pre>T Consensus        29 ~l~vFD~~G~~v~~~~~~~~~l~~~~~~~~~l~~~G~Y~iVaw~n~~~~~~~   80 (281)
</pre><pre>T 5NFI_B           29 NVYAFDKDGILRSANVFEDVQLSAAKEWLIPLEKDGLYTIFAWGNIDDHYNI   80 (281)
</pre><pre>T ss_dssp             EEEEECTTSBEEEEEEECSCCCCTTCCEEEEECSCEEEEEEEEECCCTTEEE
</pre><pre>T ss_pred             EEEEECCCCcEEEEEEeCCccccccceeEeecCCCcEEEEEEEEeCCCceee
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>3LF4_B Fluorescent Timer Precursor Blue102; Fluorescent Timers, Blue-to-red conversion, Chromophore degradation, FLUORESCENT PROTEIN; HET: 0YG; 1.808A {Discosoma sp.}
</pre><pre>Probab=26.61  E-value=3.7e+02  Score=22.26  Aligned_cols=31  Identities=16%  Similarity=0.469  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEeeEEeCC
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITGKLTVKG   57 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~~ltv~g   57 (110)
</pre><pre>                      +++..+||+++....=+|+|. .|...+++.|
</pre><pre>T Consensus        29 t~~FEDgg~~t~~~~is~e~~-~~~~~~~~~G   59 (164)
</pre><pre>T 3LF4_B           29 VMNFEDGGVVTVTQDSSLQDG-EFIYKVKLRG   59 (164)
</pre><pre>T ss_dssp             EEEETTSCEEEEEEEEEEETT-EEEEEEEEEE
</pre><pre>T ss_pred             EEEEccCcEEEEEEEEEEeCC-EEEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>PF13159.8 ; DUF3994 ; Domain of unknown function (DUF3994)
</pre><pre>Probab=25.90  E-value=2.6e+02  Score=21.50  Aligned_cols=40  Identities=25%  Similarity=0.512  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCccceeE--eeCCCCEE-----EeecccCeEEEEECCCceEEEEecc
</pre><pre>Q WP_214195415.1    1 GRECEHIT--VYEDGTEV-----RYNRESHALTITLAEGGTYRIVGNG   41 (110)
</pre><pre>Q Consensus         1 gs~~eHvt--rY~DGTev-----~Ydr~~haLtitLaeGGt~~I~g~g   41 (110)
</pre><pre>                      |.+..-|.  . .||+|+     .|.-..-...|.|-+.|+|++-+.|
</pre><pre>T Consensus        50 gid~d~v~~ni-kDGkELvG~WG~~~~~~f~~~l~lk~DgTye~Y~~g   96 (99)
</pre><pre>T Q813D6_BACCR/1   50 GINTDNVMKNV-KDGTELIGNWGFQGTDGFNMSLILKGDKTFETYGKG   96 (99)
</pre><pre>T ss_pred             CCChhHHhhhc-cCCceeeeeceeeCCCceEEEEEEcCCCcEEEeccC
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>3BWU_D Outer membrane usher protein FimD, N-terminal domain; Usher, N-terminal domain, ternary complex with chaperone and pilus subunit, CHAPERONE, STRUCTURAL PROTEIN, MEBRANE PROTEIN, STRUCTURAL; HET: EDO; 1.76A {Escherichia coli} SCOP: b.167.1.1
</pre><pre>Probab=25.38  E-value=1.8e+02  Score=20.68  Aligned_cols=18  Identities=11%  Similarity=0.268  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      |..+.||....+|.|++|
</pre><pre>T Consensus        99 ~~~~~~d~~~~~L~i~iP  116 (125)
</pre><pre>T 3BWU_D           99 DATAHLDVGQQRLNLTIP  116 (125)
</pre><pre>T ss_dssp             TCEEEEETTTTEEEEECC
</pre><pre>T ss_pred             CeEEEEecccCEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>4XIN_B LpqH orthologue; SSGCID, LpqH, Mycobacterium avium, Structural Genomics, Seattle Structural Genomics Center for Infectious Disease, UNKNOWN FUNCTION; 1.5A {Mycobacterium avium (strain 104)}
</pre><pre>Probab=24.95  E-value=3.2e+02  Score=21.06  Aligned_cols=29  Identities=21%  Similarity=0.481  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccC-eEEEEECCCceEEEEecc
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESH-ALTITLAEGGTYRIVGNG   41 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~h-aLtitLaeGGt~~I~g~g   41 (110)
</pre><pre>                      ||..+.|..... .-+++- +|.+|+|+|.+
</pre><pre>T Consensus        75 ~G~~~~y~~~~~G~A~vt~-dG~tYtItGta  104 (126)
</pre><pre>T 4XIN_B           75 SGFSGMYSRGGSGSADMSM-TGDKFTISGTA  104 (126)
</pre><pre>T ss_dssp             TTEEEEEESSSSCCEEEEE-ETTEEEEEEEE
</pre><pre>T ss_pred             CCeEEeEEcCCccceEEEE-ECCEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>PF05481.14 ; Myco_19_kDa ; Mycobacterium 19 kDa lipoprotein antigen
</pre><pre>Probab=24.90  E-value=3e+02  Score=20.72  Aligned_cols=29  Identities=28%  Similarity=0.534  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             EeeCC-CCEEEeecccCeEEEEECCCceEEEEecc
</pre><pre>Q WP_214195415.1    8 TVYED-GTEVRYNRESHALTITLAEGGTYRIVGNG   41 (110)
</pre><pre>Q Consensus         8 trY~D-GTev~Ydr~~haLtitLaeGGt~~I~g~g   41 (110)
</pre><pre>                      .+..| | .+.|...   .+++. +|.+|+|+|.+
</pre><pre>T Consensus        63 i~~~~~G-t~~~~~~---a~vt~-~G~tYtItGta   92 (110)
</pre><pre>T D5UWN3_TSUPD/5   63 ILDPDAG-TLTNDES---FAVTA-DGRTYRLEGEL   92 (110)
</pre><pre>T ss_pred             EEcCCCC-CceecCC---eEEEE-eCCEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>PF07532.13 ; Big_4 ; Bacterial Ig-like domain (group 4)
</pre><pre>Probab=24.56  E-value=2.8e+02  Score=15.92  Aligned_cols=37  Identities=22%  Similarity=0.239  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCCEEEeecccCeE-EEEECCCceEEEEec
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGTEVRYNRESHAL-TITLAEGGTYRIVGN   40 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGTev~Ydr~~haL-titLaeGGt~~I~g~   40 (110)
</pre><pre>                      +..-..+|+||......-.= .+ ...+..-|++++.|.
</pre><pre>T Consensus        17 p~~v~~~~~~g~~~~~~v~W-~~~~~~~~~~g~~~~~G~   54 (58)
</pre><pre>T A0Q3B3_CLONN/1   17 PKTVEAVMSDGSIKNVPVKW-NKTSIDTSNAGTFSIEGK   54 (58)
</pre><pre>T ss_pred             CCeEEEEEcCCCEEEEEeee-cCCccccCCCEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>PF09922.11 ; DUF2154 ; Cell wall-active antibiotics response 4TMS YvqF
</pre><pre>Probab=24.34  E-value=4.5e+02  Score=18.33  Aligned_cols=36  Identities=6%  Similarity=-0.010  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEEecceEecCEEEEe
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITG   51 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~   51 (110)
</pre><pre>                      .|.-+.....+.|.||++-.+++.+...+ |.+.+.+
</pre><pre>T Consensus        41 ~i~i~~~~G~~~i~vP~~~~v~v~~~~~~-g~~~~~~   76 (113)
</pre><pre>T Q5WCK1_BACSK/1   41 IVIVRGFIGRIQLYVPYDAGVSVEHSAIF-GQTVVFD   76 (113)
</pre><pre>T ss_pred             EEEEEEEEeeEEEEecCCceEEEEEEEEE-EeEEECC
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>d2z8ga2 b.80.1.0 (A:185-564) automated matches {Aspergillus niger [TaxId: 5061]}
</pre><pre>Probab=24.02  E-value=2.7e+02  Score=24.01  Aligned_cols=54  Identities=15%  Similarity=0.197  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCEEEeecccCeEE----EEEC-CCceEEEEecceEecCEEE---EeeEEeCCceeeeee
</pre><pre>Q WP_214195415.1   11 EDGTEVRYNRESHALT----ITLA-EGGTYRIVGNGTLDGPVQI---TGKLTVKGKTDLLAN   64 (110)
</pre><pre>Q Consensus        11 ~DGTev~Ydr~~haLt----itLa-eGGt~~I~g~g~ldG~v~i---t~~ltv~g~~~~~~n   64 (110)
</pre><pre>                      .+|.+|...+....+.    |.|+ ++-++.+.+..+|.+....   ...+++.|.-.+.++
</pre><pre>T Consensus        28 ~~g~tv~~p~G~y~~~~~~~i~l~~s~~~l~~~~ga~i~~~~~~~~~~~~i~I~G~G~idg~   89 (380)
</pre><pre>T d2z8ga2          28 TSASTVVFNPGVYYFTGHDHMVLSSSVTWVYFAPGAYVKGAVEFLSTASEVKASGHGVLSGE   89 (380)
</pre><pre>T ss_dssp             CCCSEEEECSEEEECBTTBCEEECTTCCEEEECTTEEEESCEEECCCSSEEEEESSCEEECT
</pre><pre>T ss_pred             CCCCEEEECCeEEEcCCCceEEcCCCceEEEECCCcEEEeEEEeCCCCCcEEEEEeceecCC
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>1ZDV_A Outer membrane usher protein fimD; BETA SHEET, ALPHA HELIX, MEMBRANE PROTEIN; NMR {Escherichia coli} SCOP: b.167.1.1
</pre><pre>Probab=23.76  E-value=2.3e+02  Score=19.62  Aligned_cols=19  Identities=11%  Similarity=0.270  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CCCEEEeecccCeEEEEEC
</pre><pre>Q WP_214195415.1   12 DGTEVRYNRESHALTITLA   30 (110)
</pre><pre>Q Consensus        12 DGTev~Ydr~~haLtitLa   30 (110)
</pre><pre>                      .|..++||.....|.|++|
</pre><pre>T Consensus        74 ~~~~~~~d~~~~~L~i~iP   92 (115)
</pre><pre>T 1ZDV_A           74 QDATAHLDVGQQRLNLTIP   92 (115)
</pre><pre>T ss_dssp             SSCEEEEETTTTEEEEECC
</pre><pre>T ss_pred             CCeEEEEecccCEEEEEec
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>6D6V_F Telomerase holoenzyme TEB heterotrimer Teb3 subunit; Telomerase, telomere, REPLICATION;{Tetrahymena thermophila}
</pre><pre>Probab=23.65  E-value=5.5e+02  Score=18.98  Aligned_cols=61  Identities=15%  Similarity=0.090  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEE---ECCCceEEEE----ec-ceEecCEEEEeeEEeCCceeeeeeEEec--cceecC
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTIT---LAEGGTYRIV----GN-GTLDGPVQITGKLTVKGKTDLLANTHIE--GNLTAS   74 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtit---LaeGGt~~I~----g~-g~ldG~v~it~~ltv~g~~~~~~n~~v~--gni~a~   74 (110)
</pre><pre>                      |..++++... .+++.   -++||.++|.    .+ -...+-|||.+++.=.+.+...--+...  .++...
</pre><pre>T Consensus        30 G~v~~~~~~~-~~~l~~~~~~d~~~V~v~~~~~~~~~~~~~~vEV~G~V~~~~~i~~~~~~~~~~~~~~D~~  100 (121)
</pre><pre>T 6D6V_F           30 GNVCNEDQND-SLVIEFGPTGLNQHVVIDNYRRVDLNNTTKFVEIRGVVLNQNIVSCEELTEFEQKDPFDFD  100 (121)
</pre><pre>T ss_dssp             EEEEECCTTS-SEEEEECSSSSCCEEEEECCSSCCTTTSCSEEEEEECEETTTEECCCCCCCCCCSSCCCHH
</pre><pre>T ss_pred             EEEEEeCCCC-eEEEEEeCCCCCcEEEEEecCCCCCCCCCCEEEEEEEEcCCCeEEEEEEEECCCCCCCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>4LD1_A Uncharacterized protein; beta sheet, G-box, Centriole organisation, structural protein, protein binding; HET: PGE, PG4, MPD; 1.44A {Danio rerio}
</pre><pre>Probab=23.46  E-value=5.2e+02  Score=18.58  Aligned_cols=40  Identities=10%  Similarity=0.308  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecc
</pre><pre>Q WP_214195415.1    2 RECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNG   41 (110)
</pre><pre>Q Consensus         2 s~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g   41 (110)
</pre><pre>                      .++.-+..|.||....+.+....+.+..++|-...+...|
</pre><pre>T Consensus        14 ~dG~~~~~~~~G~~~~~~~~g~~~~~~~~~g~~~~~~~~G   53 (184)
</pre><pre>T 4LD1_A           14 PDGGRLVVFPNGTRKELSADGQTVKVMFFNGDVKHTMPDQ   53 (184)
</pre><pre>T ss_dssp             TTTCEEEECTTSCEEEECTTSSCEEEECTTSCEEEECTTS
</pre><pre>T ss_pred             CCCCEEEEcCCCCEEEECCCCCEEEEEeCCCCEEEECCCC
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>d3e1za_ b.1.26.1 (A:) Chagasin {Trypanosoma cruzi [TaxId: 5693]}
</pre><pre>Probab=23.46  E-value=3.5e+02  Score=19.19  Aligned_cols=30  Identities=17%  Similarity=0.228  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             cceeEeeCCCCEEEeecccCeEEEEECCCce
</pre><pre>Q WP_214195415.1    4 CEHITVYEDGTEVRYNRESHALTITLAEGGT   34 (110)
</pre><pre>Q Consensus         4 ~eHvtrY~DGTev~Ydr~~haLtitLaeGGt   34 (110)
</pre><pre>                      +..++...+|.+|.-.... .|+|.|++.++
</pre><pre>T Consensus         1 ~~~~t~~~~~~~i~v~~g~-~~~i~L~~n~s   30 (109)
</pre><pre>T d3e1za_           1 SHKVTKAHNGATLTVAVGE-LVEIQLPSNPT   30 (109)
</pre><pre>T ss_dssp             CCCBCGGGTTCEEEECTTC-EEEEEEEECGG
</pre><pre>T ss_pred             CceeecccCCcEEEEcCCC-EEEEEeCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>PF18482.3 ; Pih1_fungal_CS ; Fungal Pih1 CS domain
</pre><pre>Probab=23.34  E-value=2.6e+02  Score=20.71  Aligned_cols=18  Identities=28%  Similarity=0.373  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCC
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEG   32 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeG   32 (110)
</pre><pre>                      +|+|+..++.|.|+..+.
</pre><pre>T Consensus        34 ~v~Y~~~~n~L~I~~~~~   51 (90)
</pre><pre>T A7TS28_VANPO/1   34 SIKYNNSSNELFIKNINL   51 (90)
</pre><pre>T ss_pred             EEEEECCCCEEEEEeCCc
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>3ML3_A Outer membrane protein icsA autotransporter; BETA HELIX, BETA SANDWICH, PROTEIN TRANSPORT; 2.0A {Shigella flexneri}
</pre><pre>Probab=23.30  E-value=6.1e+02  Score=19.75  Aligned_cols=77  Identities=14%  Similarity=0.067  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCccceeEeeCCCCEEEeecccCeEEEEEC------CCceEEEEecceEecCEEE---------EeeEEeCCceee-eee
</pre><pre>Q WP_214195415.1    1 GRECEHITVYEDGTEVRYNRESHALTITLA------EGGTYRIVGNGTLDGPVQI---------TGKLTVKGKTDL-LAN   64 (110)
</pre><pre>Q Consensus         1 gs~~eHvtrY~DGTev~Ydr~~haLtitLa------eGGt~~I~g~g~ldG~v~i---------t~~ltv~g~~~~-~~n   64 (110)
</pre><pre>                      +++..|+..|.-   +-=+.-.++-||.+.      -++.+.|.+.+.|...-.|         ..+|+|.|.... .+.
</pre><pre>T Consensus         2 ~~~~~~~~~~~~---~~~~~~~~~Gtl~~~~~~~~~~~~~~~~~~~~~l~~~G~i~l~~~~~~~~~tl~v~G~~~g~gG~   78 (189)
</pre><pre>T 3ML3_A            2 GSSHHHHHHSSG---LVPRGSHMSGTVLINNINAPFLPDPVIVTGNMTLEKNGHVILNNSSSNVGQTYVQKGNWHGKGGI   78 (189)
</pre><pre>T ss_dssp             -----------------CCSEEECSEEECCCCCSSSSCCCEEEESCEEECTTCEEECCSCGGGCCCEEEEESCEEEEEEE
</pre><pre>T ss_pred             CCcccccccccc---cCCCceecceEEEeCCCCCCCCCCcEEEecCCeEecCcEEEeCCCCCCCCCEEEEEcceEecCcE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             EEeccceecCceeecC
</pre><pre>Q WP_214195415.1   65 THIEGNLTASDEISDA   80 (110)
</pre><pre>Q Consensus        65 ~~v~gni~a~~eVsD~   80 (110)
</pre><pre>                      +.+..++.....-+|.
</pre><pre>T Consensus        79 ~~l~~~l~~~~~~sd~   94 (189)
</pre><pre>T 3ML3_A           79 LSLGAVLGNDNSKTDR   94 (189)
</pre><pre>T ss_dssp             EEEEECCSSTTCCBCE
</pre><pre>T ss_pred             EEEEeEcCCCCCCCcE
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>PF04357.15 ; TamB ; TamB, inner membrane protein subunit of TAM complex
</pre><pre>Probab=23.23  E-value=7.1e+02  Score=22.26  Aligned_cols=52  Identities=21%  Similarity=0.333  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CeEEEEECCCceEEEEecceE-------ecCEEEEe------------eEEeCCceeeee---eEEeccceecCc
</pre><pre>Q WP_214195415.1   23 HALTITLAEGGTYRIVGNGTL-------DGPVQITG------------KLTVKGKTDLLA---NTHIEGNLTASD   75 (110)
</pre><pre>Q Consensus        23 haLtitLaeGGt~~I~g~g~l-------dG~v~it~------------~ltv~g~~~~~~---n~~v~gni~a~~   75 (110)
</pre><pre>                      +.|+++. .+|.+.+.|.=.|       .-++.++.            ...+.|++.+.+   ...+.|+|.+..
</pre><pre>T Consensus         1 ~~l~~~~-~~G~l~~~G~i~l~~~~~~~~~~l~~~~~~~l~l~~~~~~~~~~sg~l~i~G~~~~~~i~G~i~i~~   74 (418)
</pre><pre>T Q8YZJ8_NOSS1/1    1 QGIQATY-NQGLVSASGNLPIFATGETVTNPLTVAIQNQLNFQVAGLYTGDVSGNAVIRGTALRPRIGGEITLSN   74 (418)
</pre><pre>T ss_pred             CceEEEe-cCeEEEEEEEEeeeCCCCCCCCCeEEEEcCCeeeecCCceEEEEeeeEEEEeccCCCEEEeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>3SZE_A Serine protease espP; parallel beta-helix, serine protease, HYDROLASE; 2.5A {Escherichia coli O157:H7}
</pre><pre>Probab=23.19  E-value=3.7e+02  Score=27.75  Aligned_cols=50  Identities=14%  Similarity=0.292  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEeeEEeCC--------------------------ceeeeeeEEeccceecCc
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITGKLTVKG--------------------------KTDLLANTHIEGNLTASD   75 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~~ltv~g--------------------------~~~~~~n~~v~gni~a~~   75 (110)
</pre><pre>                      .++|.+|++++..+.-.-+|.+.|.++|++.+                          ...+.....+.|+|.+..
</pre><pre>T Consensus       680 ~Ltl~~g~~lt~~~g~~s~g~v~i~g~LtL~~~~~~~~~~~~~~~~~~g~~~l~g~~a~l~~~~~a~~~G~I~~~~  755 (968)
</pre><pre>T 3SZE_A          680 KATIENGADVTTQSGMFSTSDISISGNLSMTGNPDKDNKFEPSIYLNDASYLLTDDSARLVAKNKASVVGDIHSTK  755 (968)
</pre><pre>T ss_dssp             EEEECTTCEEEECSCEEESSCEEESSEEEEECEECTTSCEECEEEEEEEEEEECSTTCEEEEESSEEEEEEEEESS
</pre><pre>T ss_pred             eeEEecCCeeEecCccccCCcEEEcCeEEeCCCCCCCCCcccceEecCceEEecCCceeEEEcCceEEEceeEEec
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>TIGR00686 phnA; alkylphosphonate utilization operon protein PhnA. The protein family includes an uncharacterized member designated phnA in Escherichia coli, part of a large operon associated with alkylphosphonate uptake and carbon-phosphorus bond cleavage.
</pre><pre>Probab=23.16  E-value=3.9e+02  Score=21.29  Aligned_cols=62  Identities=24%  Similarity=0.236  Sum_probs=0.0  Template_Neff=4.500
</pre><pre>
</pre><pre>Q ss_pred             EeeCCCCEEEeecccCeEEEEE---CCCceEEEEecc--eEecC-EEEEeeEEeCCce-eeeeeEEecc
</pre><pre>Q WP_214195415.1    8 TVYEDGTEVRYNRESHALTITL---AEGGTYRIVGNG--TLDGP-VQITGKLTVKGKT-DLLANTHIEG   69 (110)
</pre><pre>Q Consensus         8 trY~DGTev~Ydr~~haLtitL---aeGGt~~I~g~g--~ldG~-v~it~~ltv~g~~-~~~~n~~v~g   69 (110)
</pre><pre>                      +.|.||...-|..=+|.....-   ++.....--.+|  .-+|+ |++.+.|.|+|.. .++....|+.
</pre><pre>T Consensus        12 ~ty~dg~~~vCpeCghEWa~a~~~~~~~~~~~~DanG~~L~~GD~V~~iKDLkVKGss~~~K~GTkvkn   80 (109)
</pre><pre>T TIGR00686        12 YTYHDGTQLICPSCLYEWNENEVNDDDDELIVKDCNGNLLANGDSVILIKDLKVKGSSLVLKKGTKIKN   80 (109)
</pre><pre>T ss_pred             ceecCCcEEECCccccccccccCCCCCccceEEcCCCCccCCCCeEEEEeecCcCCccceeccCceEEe
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>d1yoxa1 b.82.5.2 (A:5-244) Hypothetical protein PA3696 {Pseudomonas aeruginosa [TaxId: 287]}
</pre><pre>Probab=22.85  E-value=5.8e+02  Score=21.06  Aligned_cols=41  Identities=15%  Similarity=0.337  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeCCCCEEEeecccCeEEEEECCCceE--EEEecceE
</pre><pre>Q WP_214195415.1    3 ECEHITVYEDGTEVRYNRESHALTITLAEGGTY--RIVGNGTL   43 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~--~I~g~g~l   43 (110)
</pre><pre>                      +..|+..|.+.-++...+...-....+..-|-+  +++|+|++
</pre><pre>T Consensus       172 d~~~lvA~~~~~~~~~~~~~~~~~~~~~geg~~~~~~~GpG~v  214 (240)
</pre><pre>T d1yoxa1         172 DTGCLVAFTDGIDYDVQLAGGLKSMLFGGEGLLLTTLKGSGTV  214 (240)
</pre><pre>T ss_dssp             EGGGEEEECSSEEEEEEECC-----------CEEEEEEEEEEE
</pre><pre>T ss_pred             eCCeEEEEeCCceEEEEEeccceeeeecccceEEEEEEEeeEE
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>2OTN_A Diaminopimelate epimerase; Bacillus anthracis diaminopimelate epimerase, DAP, lysine metabolism, lanthionine, ISOMERASE; 2.4A {Bacillus anthracis str.}
</pre><pre>Probab=22.76  E-value=5.4e+02  Score=21.03  Aligned_cols=40  Identities=20%  Similarity=0.255  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEEecc--eEecCEEEEee
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIVGNG--TLDGPVQITGK   52 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~g~g--~ldG~v~it~~   52 (110)
</pre><pre>                      +..++..+....|.+.+.+++.+.|.|+.  ..+|.+.+.+.
</pre><pre>T Consensus       266 ~~~~~i~q~~g~~~v~~~~~~~v~v~G~~~~v~~g~~~~~~~  307 (308)
</pre><pre>T 2OTN_A          266 GKEITVHLAGGDLMIAWTEEGNVLMKGPAEVICRGVYEYKIE  307 (308)
</pre><pre>T ss_dssp             TCCEEEECSSCEEEEEECTTSCEEEEEECEEEEEEEEEEEEC
</pre><pre>T ss_pred             CCeEEEEeCCceEEEEECCCCcEEEEEecEEEEEEEEEcccc
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>PF12708.9 ; Pectate_lyase_3 ; Pectate lyase superfamily protein
</pre><pre>Probab=22.52  E-value=3.8e+02  Score=19.22  Aligned_cols=33  Identities=18%  Similarity=0.083  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             eecccCeEEEEECCCceEEEEecceEecCEEEEe
</pre><pre>Q WP_214195415.1   18 YNRESHALTITLAEGGTYRIVGNGTLDGPVQITG   51 (110)
</pre><pre>Q Consensus        18 Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~   51 (110)
</pre><pre>                      .+.....-+|.|++| +|.|..+-.+...+.|.+
</pre><pre>T Consensus        36 ~~~~~~g~~i~~~~G-~y~~~~~i~~~~~~~i~g   68 (229)
</pre><pre>T Q7S602_NEUCR/1   36 DGSTTSPALVYFPPG-TYLVSRPIIDYYYTQIIG   68 (229)
</pre><pre>T ss_pred             CcccCCCCEEEECCc-eEEecCcEEeCCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>5G5O_C LH3 HEXON-INTERLACING CAPSID PROTEIN; VIRAL PROTEIN, CAPSID PROTEIN, BETA-HELIX; HET: ACT, GOL; 2.0A {SNAKE ADENOVIRUS 1}
</pre><pre>Probab=22.27  E-value=3.2e+02  Score=23.36  Aligned_cols=26  Identities=31%  Similarity=0.468  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEee
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITGK   52 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~~   52 (110)
</pre><pre>                      +|.|++|++|++ .+=.|+..++|.+.
</pre><pre>T Consensus        46 ~I~l~~G~~Y~~-~~l~i~~~vtI~G~   71 (373)
</pre><pre>T 5G5O_C           46 VVFLVPGAEYNW-KNVVIRKPVWIYGN   71 (373)
</pre><pre>T ss_dssp             EEEECTTCEEEC-CSEEECSCCEEECT
</pre><pre>T ss_pred             EEEECCCCEEEe-eeEEEcCCeEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>PF18544.3 ; Polo_box_3 ; Polo box domain
</pre><pre>Probab=22.12  E-value=3.3e+02  Score=21.45  Aligned_cols=62  Identities=19%  Similarity=0.299  Sum_probs=0.0  Template_Neff=3.700
</pre><pre>
</pre><pre>Q ss_pred             CEEEeeccc---CeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceeeeeeEEeccceecCceeecCCCcchhhhhh
</pre><pre>Q WP_214195415.1   14 TEVRYNRES---HALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDLLANTHIEGNLTASDEISDAKGKMSAIRTT   90 (110)
</pre><pre>Q Consensus        14 Tev~Ydr~~---haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVsD~~gsms~iRei   90 (110)
</pre><pre>                      ..|.|.|.+   . -+..|-|.|.++|+-+         .+ |-|+  -..+++|.+.- =+-..+|++  .+|..+.|+
</pre><pre>T Consensus         3 ~KIvf~~p~qfp~-a~AklMENGdfRikf~---------d~-~~Vq--K~~s~~v~~~~-~~~k~~vs~--~~l~~fnev   66 (96)
</pre><pre>T G0NAV2_CAEBE/1    3 TKITYRRPPQFPD-AHAQLMENGDLRIKCE---------KG-VIVR--RKENGEIYNSI-GNRKEEVHG--SVLMKVKEV   66 (96)
</pre><pre>T ss_pred             eeEEecCCCCCCC-cceeeeecCcEEEEec---------Cc-eEEE--EcCCCCEEEEe-CCeeeeech--hHHhHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             c
</pre><pre>Q WP_214195415.1   91 Y   91 (110)
</pre><pre>Q Consensus        91 y   91 (110)
</pre><pre>                      |
</pre><pre>T Consensus        67 ~   67 (96)
</pre><pre>T G0NAV2_CAEBE/1   67 Y   67 (96)
</pre><pre>T ss_pred             H
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>3DA0_B Cleaved chimeric receptor binding protein from bacteriophages TP901-1 and p2; lactococcal phage p2, lactococcal phage TP901-1 receptor binding protein, VIRAL PROTEIN; 1.65A {Lactococcus phage TP901-1} SCOP: b.108.1.0, l.1.1.1, b.21.1.0
</pre><pre>Probab=22.09  E-value=2.5e+02  Score=19.83  Aligned_cols=33  Identities=27%  Similarity=0.470  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CceEEEEecceEecCEEEEeeEEeCCceeeeee
</pre><pre>Q WP_214195415.1   32 GGTYRIVGNGTLDGPVQITGKLTVKGKTDLLAN   64 (110)
</pre><pre>Q Consensus        32 GGt~~I~g~g~ldG~v~it~~ltv~g~~~~~~n   64 (110)
</pre><pre>                      |-.+..+|+.++.|.-.+++.++|.|.......
</pre><pre>T Consensus         1 ~~~v~~tg~e~i~G~k~f~~~~~v~g~~~~~~~   33 (136)
</pre><pre>T 3DA0_B            1 GNVVHKTGDETIAGKKTFTGNVEVNGSLTLPVQ   33 (136)
</pre><pre>T ss_dssp             -CBCBSSSCEEECSCEEECSCEEESSCEECCEE
</pre><pre>T ss_pred             CCeeecCCCceeCCceeecCceEEcCceecceE
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>PF19917.1 ; bpX1 ; MoxR-vWA-beta-propeller ternary system domain bpX1
</pre><pre>Probab=21.96  E-value=5.7e+02  Score=18.48  Aligned_cols=41  Identities=17%  Similarity=0.285  Sum_probs=0.0  Template_Neff=6.800
</pre><pre>
</pre><pre>Q ss_pred             eeEeeCCCCEEEeec-----------ccCeEEEEECCCceEE-EEecceEecC
</pre><pre>Q WP_214195415.1    6 HITVYEDGTEVRYNR-----------ESHALTITLAEGGTYR-IVGNGTLDGP   46 (110)
</pre><pre>Q Consensus         6 HvtrY~DGTev~Ydr-----------~~haLtitLaeGGt~~-I~g~g~ldG~   46 (110)
</pre><pre>                      +..+++||-+|..|+           +=-.++|.|..+..+- ....|.+-|+
</pre><pre>T Consensus         2 ~~f~~~dGs~~~~D~rG~L~L~ss~~~ipei~i~~v~~~~~a~w~sdg~~~G~   54 (83)
</pre><pre>T A0A3S0JG31_9BA    2 RPHTWPDGSQAFVDTRGLLHLRSADPALPDVPLVLIIDRPTAAWAADGTVCGP   54 (83)
</pre><pre>T ss_pred             CcEecCCCcEEEEcCCCeEEEEeCCCCCCeEEEEEEcCCceeeeecCCeeecc
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>5WZQ_A Alpha-N-acetylgalactosaminidase; Glycoside Hydrolase, HYDROLASE; HET: GOL; 1.9A {Bifidobacterium bifidum}
</pre><pre>Probab=21.92  E-value=3.3e+02  Score=24.80  Aligned_cols=26  Identities=19%  Similarity=0.305  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             Ccc--ceeEeeCCCCEEEeecccCeEEE
</pre><pre>Q WP_214195415.1    2 REC--EHITVYEDGTEVRYNRESHALTI   27 (110)
</pre><pre>Q Consensus         2 s~~--eHvtrY~DGTev~Ydr~~haLti   27 (110)
</pre><pre>                      +++  ...|+|+||++|.-|......++
</pre><pre>T Consensus       601 ~~~~~v~~t~y~~G~~v~vN~~~~~~~~  628 (640)
</pre><pre>T 5WZQ_A          601 GGDPLVQRSVFADGTAVTCDFHAQTYEV  628 (640)
</pre><pre>T ss_dssp             GGCTTEEEEEETTCCEEEEETTTTEEEE
</pre><pre>T ss_pred             CCCCceEEEEeCCCcEEEEECCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>d2i5vo_ b.76.1.1 (O:) automated matches {Lyme disease spirochete (Borrelia burgdorferi) [TaxId: 139]}
</pre><pre>Probab=21.62  E-value=8.9e+02  Score=22.34  Aligned_cols=66  Identities=30%  Similarity=0.467  Sum_probs=0.0  Template_Neff=2.200
</pre><pre>
</pre><pre>Q ss_pred             CCccceeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCceee---eeeEEeccceecCcee
</pre><pre>Q WP_214195415.1    1 GRECEHITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGKTDL---LANTHIEGNLTASDEI   77 (110)
</pre><pre>Q Consensus         1 gs~~eHvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~~~~---~~n~~v~gni~a~~eV   77 (110)
</pre><pre>                      |.-+|-+..-++||.+.|-.        +..-|  .-++..+|++ +.++++|.+. ++.+   +|.|+..-.|.-+++|
</pre><pre>T Consensus       103 g~lseK~itRaNgT~LEYt~--------m~~~~--~~KAkE~LK~-~~LEG~L~~~-Kttl~ikEgTVtL~k~I~k~g~v  170 (246)
</pre><pre>T d2i5vo_         103 GEVSEKIITRADGTRLEYTG--------IKSDG--SGKAKEVLKG-YVLEGTLTAE-KTTLVVKEGTVTLSKNISKSGEV  170 (246)
</pre><pre>T ss_dssp             SCEEEEEEECTTSCEEEEES--------CCTTS--CEEEEEEETT-EEEEEEECSS-EEEEEEEETTEEEEEEEETTCCE
</pre><pre>T ss_pred             CCceeeEEEccCCcEEEEec--------ceeCC--Cceeeeeecc-cEEEEEEcCC-ceEEEEEecEEEEEeeeccCccE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             e
</pre><pre>Q WP_214195415.1   78 S   78 (110)
</pre><pre>Q Consensus        78 s   78 (110)
</pre><pre>                      .
</pre><pre>T Consensus       171 ~  171 (246)
</pre><pre>T d2i5vo_         171 S  171 (246)
</pre><pre>T ss_dssp             E
</pre><pre>T ss_pred             E
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>PF10988.10 ; DUF2807 ; Putative auto-transporter adhesin, head GIN domain
</pre><pre>Probab=21.21  E-value=6.5e+02  Score=18.88  Aligned_cols=53  Identities=17%  Similarity=0.281  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             eeEeeCCCCEEEeecccCeEEEEECCCceEEEEecceEecCEEEEeeEEeCCc
</pre><pre>Q WP_214195415.1    6 HITVYEDGTEVRYNRESHALTITLAEGGTYRIVGNGTLDGPVQITGKLTVKGK   58 (110)
</pre><pre>Q Consensus         6 HvtrY~DGTev~Ydr~~haLtitLaeGGt~~I~g~g~ldG~v~it~~ltv~g~   58 (110)
</pre><pre>                      .+..-+.|..--++.....+.+.+...|.++|..++.|+..+.=..++.+.|+
</pre><pre>T Consensus       150 ~i~~~g~g~i~~~~~~~~~~~v~~~~~g~i~v~~~~~~~~~~~~~g~i~~~g~  202 (203)
</pre><pre>T A9WHL3_CHLAA/4  150 HVKVSGSGDLLAEKLKASIVEVAVNGSGDVTVWAVDTLDVSISGSGNVRYLGS  202 (203)
</pre><pre>T ss_pred             EEEEecCceeeeeccEEeEEEEEEecceEEEEEecCeEEEEEcCceEEEEEcc
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>PF09917.11 ; DUF2147 ; Uncharacterized protein conserved in bacteria (DUF2147)
</pre><pre>Probab=21.15  E-value=3.2e+02  Score=19.94  Aligned_cols=31  Identities=19%  Similarity=0.224  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             eeCCCCEEEeeccc-CeEEEEECCCceEEEEe
</pre><pre>Q WP_214195415.1    9 VYEDGTEVRYNRES-HALTITLAEGGTYRIVG   39 (110)
</pre><pre>Q Consensus         9 rY~DGTev~Ydr~~-haLtitLaeGGt~~I~g   39 (110)
</pre><pre>                      +|.+|+.+.-+... ...+|+|.++++++|+|
</pre><pre>T Consensus        70 ~w~~G~i~dP~~g~~y~~~i~l~~~~~L~v~g  101 (116)
</pre><pre>T A6H0P6_FLAPJ/2   70 EYNGGKILDPETGNIYKCFIKLAGKDKLEVRG  101 (116)
</pre><pre>T ss_pred             ceeccEEECCCCCCEEEEEEEEcCCCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>4QDG_A Putative adhesin; PF08842 family, Fimbrillin-A associated anchor proteins Mfa1 and Mfa2, Structural Genomics, Joint Center for Structural Genomics; HET: MSE; 2.2A {Bacteroides thetaiotaomicron}
</pre><pre>Probab=21.06  E-value=6.2e+02  Score=21.27  Aligned_cols=36  Identities=25%  Similarity=0.578  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             ccceeEeeC-CCCEEEeecccCe--------EEEEECCCceEEEEe
</pre><pre>Q WP_214195415.1    3 ECEHITVYE-DGTEVRYNRESHA--------LTITLAEGGTYRIVG   39 (110)
</pre><pre>Q Consensus         3 ~~eHvtrY~-DGTev~Ydr~~ha--------LtitLaeGGt~~I~g   39 (110)
</pre><pre>                      .+-+++.|. +|..+..-.....        +.+.|++| .|+|.+
</pre><pre>T Consensus        36 ~~~~l~vfd~~G~~v~~~~~~~~~l~~~~~~~~l~Lp~G-~Y~~Va   80 (310)
</pre><pre>T 4QDG_A           36 DCLTLLVYDENGNYVDTRIVTGTELQDENYRMKLDLKQG-NYHFVA   80 (310)
</pre><pre>T ss_dssp             CCEEEEEEETTCCEEEEEEECSGGGGCTTCCEEEECCSS-EEEEEE
</pre><pre>T ss_pred             CEEEEEEECCCCCEEEEEEEcCCccccCCeEEEeeeCCc-eEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>6D0T_A BB1; beta-barrel, design, DE NOVO PROTEIN; 1.63A {synthetic construct}
</pre><pre>Probab=20.63  E-value=7.1e+02  Score=20.62  Aligned_cols=45  Identities=18%  Similarity=0.244  Sum_probs=0.0  Template_Neff=1.900
</pre><pre>
</pre><pre>Q ss_pred             eEEEEecc-eEec-CEEEEeeEEeCCceeeeeeEEeccceecCceee
</pre><pre>Q WP_214195415.1   34 TYRIVGNG-TLDG-PVQITGKLTVKGKTDLLANTHIEGNLTASDEIS   78 (110)
</pre><pre>Q Consensus        34 t~~I~g~g-~ldG-~v~it~~ltv~g~~~~~~n~~v~gni~a~~eVs   78 (110)
</pre><pre>                      ++.|+..| .-|| |++=.++++|+-..++..+++-..++.|.|.+.
</pre><pre>T Consensus        37 t~di~~qGt~sDGrPi~G~Gkitvktp~t~~~~~ty~~~~k~qGqlt   83 (120)
</pre><pre>T 6D0T_A           37 EIEIRFKGQSSDGRPVEGRGSIEVRSPYEYRFEMQSSDGARWEGTLQ   83 (120)
</pre><pre>T ss_dssp             EEEEEEEEEETTCCEEEEEEEEECCSSSSCEEEEEETTSCEEEEEEE
</pre><pre>T ss_pred             EEEEEEEEEecCCccceEEeEEEEcCCCEEEEEEEcCCCCEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>6Z1P_BR Iron donor protein CyaY; Mitochondrial ribosome, ribosome, mitochondria, ciliate, tetrahymena; HET: MG, ATP; 3.7A {Tetrahymena thermophila (strain SB210)}
</pre><pre>Probab=20.62  E-value=3.5e+02  Score=21.43  Aligned_cols=22  Identities=14%  Similarity=0.141  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             EEeecccCeEEEEECCCceEEE
</pre><pre>Q WP_214195415.1   16 VRYNRESHALTITLAEGGTYRI   37 (110)
</pre><pre>Q Consensus        16 v~Ydr~~haLtitLaeGGt~~I   37 (110)
</pre><pre>                      +-+|.....|+|.++++|+|.|
</pre><pre>T Consensus        73 ~Dvd~s~gvLti~~~~~~~~Vi   94 (143)
</pre><pre>T 6Z1P_BR          73 SDVQISDGVIKVTTQGEKTFVI   94 (143)
</pre><pre>T ss_dssp             EEEEECSSEEEEEESSSCEEEE
</pre><pre>T ss_pred             eEEEeeCCEEEEEECCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>d4ts4a2 b.46.1.0 (A:204-308) automated matches {Zebrafish (Danio rerio) [TaxId: 7955]}
</pre><pre>Probab=20.40  E-value=4.6e+02  Score=17.60  Aligned_cols=26  Identities=12%  Similarity=0.079  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCEEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   13 GTEVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        13 GTev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      |+.+.|....+.|.|..++|..+.|.
</pre><pre>T Consensus        63 G~~~~~~~~~~~l~v~~~dg~~l~i~   88 (105)
</pre><pre>T d4ts4a2          63 GASRAALVTKNGLVLFGNDGKMLLVK   88 (105)
</pre><pre>T ss_dssp             TCSSCEEEETTEEEEECTTSCEEEEE
</pre><pre>T ss_pred             CcceeeEEeCCeEEEEccCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>1YOX_B hypothetical protein PA3696; STRUCTURAL GENOMICS  PSI, PROTEIN STRUCTURE INITIATIVE, MIDWEST CENTER FOR STRUCTURAL GENOMICS, MCSG, STRUCTURAL GENOMICS, UNKNOWN; HET: MSE; 2.3A {Pseudomonas aeruginosa} SCOP: l.1.1.1, b.82.5.2
</pre><pre>Probab=20.21  E-value=8.9e+02  Score=20.23  Aligned_cols=38  Identities=16%  Similarity=0.279  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEEecc--eEecCEEEEeeE
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIVGNG--TLDGPVQITGKL   53 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~g~g--~ldG~v~it~~l   53 (110)
</pre><pre>                      +|.-++ ...|.|+|+++..+.+.-.-  -.+|+|++...+
</pre><pre>T Consensus        11 ~I~g~~-~~~l~v~L~~~~~v~~~~gsl~~~~~~v~~~~~~   50 (250)
</pre><pre>T 1YOX_B           11 RILGES-MQTVEIELDPGETVIAEAGAMNYMTGDIRFTARM   50 (250)
</pre><pre>T ss_dssp             EEECSS-SCEEEEEECTTCEEEECTTCEEEEETTEEEECCC
</pre><pre>T ss_pred             EEecCC-CCEEEEEeCCCCeEEEeCCcEEEeeCCeEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>PF08307.13 ; Glyco_hydro_98C ; Glycosyl hydrolase family 98 C-terminal domain
</pre><pre>Probab=20.20  E-value=3.9e+02  Score=23.48  Aligned_cols=24  Identities=21%  Similarity=0.440  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEECCCceEEEE
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITLAEGGTYRIV   38 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitLaeGGt~~I~   38 (110)
</pre><pre>                      ++.||.+.+.+||++.-.|.+.|+
</pre><pre>T Consensus       230 s~~wd~~~~~~TitV~hNG~Vd~~  253 (257)
</pre><pre>T Q8CY62_STRR6/3  230 TENWDENTHVYTITVNHNGMVEMS  253 (257)
</pre><pre>T ss_pred             eEEEeCCCCEEEEEEecCceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>6QI5_Q Protein LH3; adenovirus atadenovirus virus evolution, VIRUS;{Lizard adenovirus 2}
</pre><pre>Probab=20.12  E-value=4e+02  Score=22.35  Aligned_cols=25  Identities=32%  Similarity=0.492  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEe
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITG   51 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~   51 (110)
</pre><pre>                      +|.|++|++|.+ ..=.|..+++|.+
</pre><pre>T Consensus        46 ~I~l~~G~~Y~~-~~l~i~~~vti~G   70 (370)
</pre><pre>T 6QI5_Q           46 VVYLVPGAQYHW-KNILIEKPVWIYG   70 (370)
</pre><pre>T ss_dssp             EEECCTTCCEEC-CSCEECSCCEEEE
</pre><pre>T ss_pred             EEEECCCCEEEE-eeEEeCCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>6QI5_S Protein LH3; adenovirus atadenovirus virus evolution, VIRUS;{Lizard adenovirus 2}
</pre><pre>Probab=20.12  E-value=4e+02  Score=22.35  Aligned_cols=25  Identities=32%  Similarity=0.492  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             EEEECCCceEEEEecceEecCEEEEe
</pre><pre>Q WP_214195415.1   26 TITLAEGGTYRIVGNGTLDGPVQITG   51 (110)
</pre><pre>Q Consensus        26 titLaeGGt~~I~g~g~ldG~v~it~   51 (110)
</pre><pre>                      +|.|++|++|.+ ..=.|..+++|.+
</pre><pre>T Consensus        46 ~I~l~~G~~Y~~-~~l~i~~~vti~G   70 (370)
</pre><pre>T 6QI5_S           46 VVYLVPGAQYHW-KNILIEKPVWIYG   70 (370)
</pre><pre>T ss_dssp             EEECCTTCCEEC-CSCEECSCCEEEE
</pre><pre>T ss_pred             EEEECCCCEEEE-eeEEeCCCeEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>6GHU_A Type II secretion system protein L; ferredoxin fold, dimer, GspL, type II secretion system, TRANSPORT PROTEIN; 2.0A {Pseudomonas aeruginosa}
</pre><pre>Probab=20.04  E-value=3.1e+02  Score=18.31  Aligned_cols=15  Identities=0%  Similarity=0.306  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             EEEeecccCeEEEEE
</pre><pre>Q WP_214195415.1   15 EVRYNRESHALTITL   29 (110)
</pre><pre>Q Consensus        15 ev~Ydr~~haLtitL   29 (110)
</pre><pre>                      .++|++..+.|++.|
</pre><pre>T Consensus        23 ~l~y~~~~~~L~l~l   37 (78)
</pre><pre>T 6GHU_A           23 QLDFSAARGDVALQV   37 (78)
</pre><pre>T ss_dssp             EEEEETTTCSEEEEE
</pre><pre>T ss_pred             EEEEeCCCCeEEEEE
</pre><pre>
</pre><pre>
</pre>